<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.08.939942v5.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.08.939942v5.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.08.939942v5.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/04/08/2020.02.08.939942.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.08.939942v5"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.08.939942"/>
  <meta name="DC.Date" content="2022-04-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Nanopore sequencing enables the efficient and unbiased measurement of transcriptomes from any sample. However, current methods for transcript identification and quantification rely of mapping reads to a reference genome, which precludes the study of species with a partial or missing reference or the identification of disease-specific transcripts not readily identifiable from a reference. Here we present RATTLE, a tool to perform reference-free reconstruction and quantification of transcripts using only Nanopore reads. Using simulated data and experimental data from isoform spike-ins, human tissues, and cell lines, we show that RATTLE accurately determines transcript sequences and their abundances, and shows good scalability with the number of transcripts. RATTLE provides unprecedented access to transcriptomes from any sample and species without relying on a reference or additional technologies.

### Competing Interest Statement

E.E. has received support from Oxford Nanopore Technologies (ONT) to present the results from this manuscript at scientific conferences. However, ONT played no role in the algorithm or software developments, study design, analysis, or preparation of the manuscript."/>
  <meta name="DC.Contributor" content="Ivan de la Rubia"/>
  <meta name="DC.Contributor" content="Akanksha Srivastava"/>
  <meta name="DC.Contributor" content="Wenjing Xue"/>
  <meta name="DC.Contributor" content="Joel A Indi"/>
  <meta name="DC.Contributor" content="Silvia Carbonell-Sala"/>
  <meta name="DC.Contributor" content="Julien Lagarde"/>
  <meta name="DC.Contributor" content="M Mar Albà"/>
  <meta name="DC.Contributor" content="Eduardo Eyras"/>
  <meta name="article:published_time" content="2022-04-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Nanopore sequencing enables the efficient and unbiased measurement of transcriptomes from any sample. However, current methods for transcript identification and quantification rely of mapping reads to a reference genome, which precludes the study of species with a partial or missing reference or the identification of disease-specific transcripts not readily identifiable from a reference. Here we present RATTLE, a tool to perform reference-free reconstruction and quantification of transcripts using only Nanopore reads. Using simulated data and experimental data from isoform spike-ins, human tissues, and cell lines, we show that RATTLE accurately determines transcript sequences and their abundances, and shows good scalability with the number of transcripts. RATTLE provides unprecedented access to transcriptomes from any sample and species without relying on a reference or additional technologies.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.08.939942v5"/>
  <meta name="citation_id" content="2020.02.08.939942v5"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.939942v5"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.939942v5.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.939942v5.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.08.939942"/>
  <meta name="citation_num_pages" content="28"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.08.939942"/>
  <meta name="citation_author" content="Ivan de la Rubia"/>
  <meta name="citation_author_institution" content="EMBL Australia Partner Laboratory Network at the Australian National University"/>
  <meta name="citation_author_institution" content="Pompeu Fabra University (UPF)"/>
  <meta name="citation_author" content="Akanksha Srivastava"/>
  <meta name="citation_author_institution" content="EMBL Australia Partner Laboratory Network at the Australian National University"/>
  <meta name="citation_author_institution" content="Australian National University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4553-9232"/>
  <meta name="citation_author" content="Wenjing Xue"/>
  <meta name="citation_author_institution" content="EMBL Australia Partner Laboratory Network at the Australian National University"/>
  <meta name="citation_author_institution" content="Australian National University"/>
  <meta name="citation_author" content="Joel A Indi"/>
  <meta name="citation_author_institution" content="EMBL Australia Partner Laboratory Network at the Australian National University"/>
  <meta name="citation_author_institution" content="Universidade de Lisboa"/>
  <meta name="citation_author" content="Silvia Carbonell-Sala"/>
  <meta name="citation_author_institution" content="Pompeu Fabra University (UPF)"/>
  <meta name="citation_author_institution" content="Centre for Regulatory Genomics (CRG)"/>
  <meta name="citation_author" content="Julien Lagarde"/>
  <meta name="citation_author_institution" content="Pompeu Fabra University (UPF)"/>
  <meta name="citation_author_institution" content="Centre for Regulatory Genomics (CRG)"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0290-7445"/>
  <meta name="citation_author" content="M Mar Albà"/>
  <meta name="citation_author_institution" content="Pompeu Fabra University (UPF)"/>
  <meta name="citation_author_institution" content="Catalan Institution for Research and Advanced Studies (ICREA)"/>
  <meta name="citation_author_institution" content="Hospital del Mar Medical Research Institute (IMIM)"/>
  <meta name="citation_author_email" content="malba@imim.es"/>
  <meta name="citation_author_email" content="eduardo.eyras@anu.edu.au"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7963-7375"/>
  <meta name="citation_author" content="Eduardo Eyras"/>
  <meta name="citation_author_institution" content="EMBL Australia Partner Laboratory Network at the Australian National University"/>
  <meta name="citation_author_institution" content="Australian National University"/>
  <meta name="citation_author_institution" content="Catalan Institution for Research and Advanced Studies (ICREA)"/>
  <meta name="citation_author_institution" content="Hospital del Mar Medical Research Institute (IMIM)"/>
  <meta name="citation_author_email" content="malba@imim.es"/>
  <meta name="citation_author_email" content="eduardo.eyras@anu.edu.au"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0793-6218"/>
  <meta name="citation_reference" content="Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nat. Biotechnol. 2011;29:644–52."/>
  <meta name="citation_reference" content="Faherty SL, Villanueva-Cañas JL, Blanco MB, Albà MM, Yoder AD. Transcriptomics in the wild: Hibernation physiology in free-ranging dwarf lemurs. Mol. Ecol. [Internet]. 2018;27:709–22. Available from: http://www.ncbi.nlm.nih.gov/pubmed/29319912"/>
  <meta name="citation_reference" content="Pinskaya M, Saci Z, Gallopin M, Gabriel M, Nguyen HT, Firlej V, et al. Reference-free transcriptome exploration reveals novel RNAs for prostate cancer diagnosis. Life Sci. alliance [Internet]. 2019;2. Available from: http://www.ncbi.nlm.nih.gov/pubmed/31732695"/>
  <meta name="citation_reference" content="Hsieh P-H, Oyang Y-J, Chen C-Y. Effect of de novo transcriptome assembly on transcript quantification. Sci. Rep. [Internet]. 2019;9:8304. Available from: http://www.ncbi.nlm.nih.gov/pubmed/31165774"/>
  <meta name="citation_reference" content="Workman RE, Tang AD, Tang PS, Jain M, Tyson JR, Razaghi R, et al. Nanopore native RNA sequencing of a human poly(A) transcriptome. Nat. Methods [Internet]. 2019;16:1297–305. Available from: http://www.ncbi.nlm.nih.gov/pubmed/31740818"/>
  <meta name="citation_reference" content="Sessegolo C, Cruaud C, Da Silva C, Cologne A, Dubarry M, Derrien T, et al. Transcriptome profiling of mouse samples using nanopore sequencing of cDNA and RNA molecules. Sci. Rep. [Internet]. 2019;9:14908. Available from: http://biorxiv.org/content/early/2019/07/16/575142.abstract"/>
  <meta name="citation_reference" content="Wyman D, Mortazavi A. TranscriptClean: variant-aware correction of indels, mismatches and splice junctions in long-read transcripts. Bioinformatics [Internet]. 2019;35:340–2. Available from: http://www.ncbi.nlm.nih.gov/pubmed/29912287"/>
  <meta name="citation_reference" content="Kovaka S, Zimin A V, Pertea GM, Razaghi R, Salzberg SL, Pertea M. Transcriptome assembly from long-read RNA-seq alignments with StringTie2. Genome Biol. [Internet]. 2019;20:278. Available from: http://biorxiv.org/content/early/2019/07/08/694554.abstract"/>
  <meta name="citation_reference" content="Tang AD, Soulette CM, van Baren MJ, Hart K, Hrabeta-Robinson E, Wu CJ, et al. Full-length transcript characterization of SF3B1 mutation in chronic lymphocytic leukemia reveals downregulation of retained introns. Nat. Commun. [Internet]. 2020;11:1438. Available from: http://www.ncbi.nlm.nih.gov/pubmed/32188845"/>
  <meta name="citation_reference" content="Bushmanova E, Antipov D, Lapidus A, Prjibelski AD. rnaSPAdes: a de novo transcriptome assembler and its application to RNA-Seq data. Gigascience [Internet]. 2019;8. Available from: http://www.ncbi.nlm.nih.gov/pubmed/31494669"/>
  <meta name="citation_reference" content="Fu S, Ma Y, Yao H, Xu Z, Chen S, Song J, et al. IDP-denovo: de novo transcriptome assembly and isoform annotation by hybrid sequencing. Bioinformatics [Internet]. 2018;34:2168–76. Available from: http://www.ncbi.nlm.nih.gov/pubmed/29905763"/>
  <meta name="citation_reference" content="Koren S, Walenz BP, Berlin K, Miller JR, Bergman NH, Phillippy AM. Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. Genome Res. 2017;27:722–36."/>
  <meta name="citation_reference" content="Salmela L, Walve R, Rivals E, Ukkonen E. Accurate self-correction of errors in long reads using de Bruijn graphs. Bioinformatics. 2017;33:799–806."/>
  <meta name="citation_reference" content="Xiao C-L, Chen Y, Xie S-Q, Chen K-N, Wang Y, Han Y, et al. MECAT: fast mapping, error correction, and de novo assembly for single-molecule sequencing reads. Nat. Methods [Internet]. 2017;14:1072–4. Available from: http://www.ncbi.nlm.nih.gov/pubmed/28945707"/>
  <meta name="citation_reference" content="Xie Y, Wu G, Tang J, Luo R, Patterson J, Liu S, et al. SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-Seq reads. Bioinformatics [Internet]. 2014;30:1660–6. Available from: http://www.ncbi.nlm.nih.gov/pubmed/24532719"/>
  <meta name="citation_reference" content="Marchet C, Lecompte L, Silva C Da, Cruaud C, Aury J-M, Nicolas J, et al. De novo clustering of long reads by gene from transcriptomics data. Nucleic Acids Res. [Internet]. 2018; Available from: http://www.ncbi.nlm.nih.gov/pubmed/30260405"/>
  <meta name="citation_reference" content="Sahlin K, Medvedev P. De novo clustering of long-read transcriptome data using a greedy, quality-value based algorithm. Int. Conf. Res. Comput. Mol. Biol. Springer; 2019. p. 227–42."/>
  <meta name="citation_reference" content="Morisse P, Marchet C, Limasset A, Lecroq T, Lefebvre A. Scalable long read self-correction and assembly polishing with multiple sequence alignment. Sci. Rep. [Internet]. 2021;11:761. Available from: http://www.ncbi.nlm.nih.gov/pubmed/33436980"/>
  <meta name="citation_reference" content="Sahlin K, Medvedev P. Error correction enables use of Oxford Nanopore technology for reference-free transcriptome analysis. Nat. Commun. [Internet]. 2021;12:2. Available from: http://www.ncbi.nlm.nih.gov/pubmed/33397972"/>
  <meta name="citation_reference" content="Singh M, Al-Eryani G, Carswell S, Ferguson JM, Blackburn J, Barton K, et al. High-throughput targeted long-read single cell sequencing reveals the clonal and transcriptional landscape of lymphocytes. Nat. Commun. [Internet]. 2019;10:3120. Available from: http://www.ncbi.nlm.nih.gov/pubmed/31311926"/>
  <meta name="citation_reference" content="Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34:3094–100."/>
  <meta name="citation_reference" content="Tardaguila M, de la Fuente L, Marti C, Pereira C, Pardo-Palacios FJ, Del Risco H, et al. SQANTI: extensive characterization of long-read transcript sequences for quality control in full-length transcriptome identification and quantification. Genome Res. [Internet]. 2018; Available from: http://www.ncbi.nlm.nih.gov/pubmed/29440222"/>
  <meta name="citation_reference" content="Wyman D, Balderrama-Gutierrez G, Reese F, Jiang S, Rahmanian S, Zeng W, et al. A technology-agnostic long-read analysis pipeline for transcriptome discovery and quantification. bioRxiv [Internet]. 2019;672931. Available from: http://biorxiv.org/content/early/2019/06/18/672931.abstract"/>
  <meta name="citation_reference" content="Nip KM, Chiu R, Yang C, Chu J, Mohamadi H, Warren RL, et al. RNA-Bloom enables reference-free and reference-guided sequence assembly for single-cell transcriptomes. Genome Res. [Internet]. 2020;30:1191–200. Available from: http://www.ncbi.nlm.nih.gov/pubmed/32817073"/>
  <meta name="citation_reference" content="Trincado JL, Entizne JC, Hysenaj G, Singh B, Skalic M, Elliott DJ, et al. SUPPA2: Fast, accurate, and uncertainty-aware differential splicing analysis across multiple conditions. Genome Biol. 2018;19."/>
  <meta name="citation_reference" content="Lebrigand K, Magnone V, Barbry P, Waldmann R. High throughput error corrected Nanopore single cell transcriptome sequencing. Nat. Commun. [Internet]. 2020;11:4025. Available from: http://biorxiv.org/content/early/2019/11/05/831495.abstract"/>
  <meta name="citation_reference" content="Acera Mateos P, Burgio G, Preiss T, Shirokikh NE, Eyras E. Identification of m6A and m5C RNA modifications at single-molecule resolution from Nanopore sequencing [PREPRINT]. BIORXIV/2022/484124 [Internet]. 2022; Available from: https://doi.org/10.1101/2022.03.14.484124"/>
  <meta name="citation_reference" content="Ruiz-Reche A, Srivastava A, Indi JA, de la Rubia I, Eyras E. ReorientExpress: reference-free orientation of nanopore cDNA reads with deep learning. Genome Biol. [Internet]. 2019;20:260. Available from: http://biorxiv.org/content/early/2019/02/18/553321.abstract"/>
  <meta name="citation_reference" content="Lee C, Grasso C, Sharlow MF. Multiple sequence alignment using partial order graphs. Bioinformatics [Internet]. 2002;18:452–64. Available from: http://www.ncbi.nlm.nih.gov/pubmed/11934745"/>
  <meta name="citation_reference" content="Li Y, Han R, Bi C, Li M, Wang S, Gao X. DeepSimulator: a deep simulator for Nanopore sequencing. Bioinformatics. 2018;34:2899–908."/>
  <meta name="citation_reference" content="Fu L, Niu B, Zhu Z, Wu S, Li W. CD-HIT: accelerated for clustering the next-generation sequencing data. Bioinformatics [Internet]. 2012;28:3150–2. Available from: http://www.ncbi.nlm.nih.gov/pubmed/23060610"/>
  <meta name="citation_reference" content="Vinh NX, Epps J, Bailey J. Information theoretic measures for clusterings comparison: Variants, properties, normalization and correction for chance. J. Mach. Learn. Res. 2010;11:2837–54."/>
  <meta name="citation_reference" content="Rosenberg A, Hirschberg J. V-measure: A conditional entropy-based external cluster evaluation measure. Proc. 2007 Jt. Conf. Empir. methods Nat. Lang. Process. Comput. Nat. Lang. Learn. 2007. p. 410–20."/>
  <meta name="citation_reference" content="Barrett CL, DeBoever C, Jepsen K, Saenz CC, Carson DA, Frazer KA. Systematic transcriptome analysis reveals tumor-specific isoforms for ovarian cancer diagnosis and therapy. Proc. Natl. Acad. Sci. U. S. A. [Internet]. 2015;112:E3050–7. Available from: http://www.ncbi.nlm.nih.gov/pubmed/26015570"/>
  <meta name="citation_reference" content="Gleeson J, Leger A, Prawer YDJ, Lane TA, Harrison PJ, Haerty W, et al. Accurate expression quantification from nanopore direct RNA sequencing with NanoCount. Nucleic Acids Res. [Internet]. 2021; Available from: http://www.ncbi.nlm.nih.gov/pubmed/34850115"/>
  <meta name="citation_reference" content="Pratanwanich PN, Yao F, Chen Y, Koh CWQ, Wan YK, Hendra C, et al. Identification of differential RNA modifications from nanopore direct RNA sequencing with xPore. Nat. Biotechnol. [Internet]. 2021;2020.06.18.160010-2020.06.18.160010. Available from: http://biorxiv.org/content/early/2020/06/20/2020.06.18.160010.abstract"/>
  <meta name="twitter:title" content="RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Nanopore sequencing enables the efficient and unbiased measurement of transcriptomes from any sample. However, current methods for transcript identification and quantification rely of mapping reads to a reference genome, which precludes the study of species with a partial or missing reference or the identification of disease-specific transcripts not readily identifiable from a reference. Here we present RATTLE, a tool to perform reference-free reconstruction and quantification of transcripts using only Nanopore reads. Using simulated data and experimental data from isoform spike-ins, human tissues, and cell lines, we show that RATTLE accurately determines transcript sequences and their abundances, and shows good scalability with the number of transcripts. RATTLE provides unprecedented access to transcriptomes from any sample and species without relying on a reference or additional technologies.

### Competing Interest Statement

E.E. has received support from Oxford Nanopore Technologies (ONT) to present the results from this manuscript at scientific conferences. However, ONT played no role in the algorithm or software developments, study design, analysis, or preparation of the manuscript."/>
  <meta name="og-title" property="og:title" content="RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.08.939942v5"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Nanopore sequencing enables the efficient and unbiased measurement of transcriptomes from any sample. However, current methods for transcript identification and quantification rely of mapping reads to a reference genome, which precludes the study of species with a partial or missing reference or the identification of disease-specific transcripts not readily identifiable from a reference. Here we present RATTLE, a tool to perform reference-free reconstruction and quantification of transcripts using only Nanopore reads. Using simulated data and experimental data from isoform spike-ins, human tissues, and cell lines, we show that RATTLE accurately determines transcript sequences and their abundances, and shows good scalability with the number of transcripts. RATTLE provides unprecedented access to transcriptomes from any sample and species without relying on a reference or additional technologies.

### Competing Interest Statement

E.E. has received support from Oxford Nanopore Technologies (ONT) to present the results from this manuscript at scientific conferences. However, ONT played no role in the algorithm or software developments, study design, analysis, or preparation of the manuscript."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-04-08"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.08.939942v5"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2488778"/>
  <title>
   RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4OKkAFFrSIg55Wvg_n-HEr6A5u7uBAt6oAn2ALQEHyU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2488778","apath":"\/biorxiv\/early\/2022\/04\/08\/2020.02.08.939942.atom","pisa":"biorxiv;2020.02.08.939942v5","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.08.939942v5"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.08.939942v5","title":"RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing","identifier":"node\/2488778"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220K6UnRU91VnhoPhIFV\\\\\\\/KsQGrawY9D2gyBBHOLJAIxs9pLID+65o4kd3CZj5pZce+bCBb3NX2UxUYuOLxbSNDyHt\\\\\\\/lyLb4KDZqFeQ3u7cSzyaH+nEN199x0qi+KlPIOJFYQkp7ZPsO2JGLhNR0kHsPKdNMRElEL+CVeA\\\\\\\/cEj88LYGoewSDBbGX1rXY2VbOyt+09miAGCpN2Ixm2Jxk+wq4+mpDfQaj73Y2NG6kpwQ52gpk+jC5Co4TAoB+Sc7E+9eL0PkJUjuFLZUpaBX99nVBmp3DkTlSub5dbw9feoDlxLSfRiTmVNS3+BQYVUStbmVBBq9cwhZCtStzhx\\\\\\\/dlVXPBZkLOpFmGIZDXsBpbe7ormBAIejGQtQLIkLvCOEwNlqrWGp8Fn8olcJYtcH4wlXYYMrXRuINYjXup2tkyhcm3Mr7eyWmifEh445Xf1CZYxwrEIFChLZgdx87X2IW5sV\\\\\\\/+qKXBjg\\\\\\\/aqV6BGGfOScVNbviwRtCZe8iwu3hgG1AIst+NMfFbf\\\\\\\/zrzSgVbzYIxtqeGG4gEFGO8gLn2zviLiGi6oaztEUkry8ZA0aJGyqj89q7B00NCI0m1ZeqmFRSbssW\\\\\\\/LMMMc6k88AFveQFEYNrwESS3ngMnABqXhWTyEDXu23QdUlGznyo7CFcZqW48aWI7cXeFGSx8JMF1RRGadXqTv693oiqPLsWZmTRXgtUrem4YL4xdbJjlIj34Jv9ijLgcEtjrL5lVI71En+RbjGTruEu+fykPyPmVgemu9ituV7NQN0ufizifV7Y5qDnLXMkFLrIF7x9Tbei6q9p5Vm9JcuSj+ZtZHuxQntR\\\\\\\/wrbGr3WKmEtHfoOc6iaZoulYUaqPakQce9Y9teFaNDiOcuqH09Vgj+Z5cUGDQXQAzM1rKs1XzIzT4nSLrt9lxkwTm82xEmCd6XiR2FveqfoRK5\\\\\\\/iMPnKIEpHwVGzIUDJF1AU8E8ksCP2v6uM8QjWUQQYo6t759QY+qn1gt6saHBXCukz2BX22NRiybwwy9MCIJ34uxWhTP62IQhAf7l0j+1rwXMXfazCJvJt7LIiGhgF34eFShkEURPCjGivt64dZzMlf+\\\\\\\/YHroDG1\\\\\\\/IjfapLXJUYnqUxuHRb4wQASZPPs0s7lNT9+qSP3soYhmw+9G0vjAA99ClDDJfcBE9MFCV2r0f1CmDPZCb9OW+6wM5fr3eAmFoMbkAwFxFDEV2UyZxG10rOZmXH6fB6TmvWsVbfY51zCZOwhBWJlOOnx8JqHEN7Qkk2DyJBBHcE3T\\\\\\\/g6PVy1jluslTclnmhAdRZvAeIBbhCDEMH56xtsEybHXRRCCVfRcBtavyjgZTl34ucpRsOXnnQPbxSUFwl5BB+Z+1LPb7SMqViVqCmjARlkfB6QvAL4uU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022gqKIN9Z3ZlWI\\\\\\\/euXF9DI0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00221d3bd1ca177e954a44015308d48cb5431b4984fd3deb2672e91be804fa272ec6\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.08.939942v5.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2488778 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.08.939942v5.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_195069170">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_195069170" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1331569070" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-y_iaItLGAj9kWsA95G56v9601Gob56iVXgkV373Oa9Q"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2488778" data-node-nid="2488778" id="node-2488778--2624026287" data-pisa="biorxiv;2020.02.08.939942v5" data-pisa-master="biorxiv;2020.02.08.939942" data-apath="/biorxiv/early/2022/04/08/2020.02.08.939942.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Ivan
                      </span>
                      <span class="nlm-surname">
                       de la Rubia
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-4553-9232" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Akanksha
                      </span>
                      <span class="nlm-surname">
                       Srivastava
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Wenjing
                      </span>
                      <span class="nlm-surname">
                       Xue
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Joel A
                      </span>
                      <span class="nlm-surname">
                       Indi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Silvia
                      </span>
                      <span class="nlm-surname">
                       Carbonell-Sala
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0002-0290-7445" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Julien
                      </span>
                      <span class="nlm-surname">
                       Lagarde
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0002-7963-7375" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       M Mar
                      </span>
                      <span class="nlm-surname">
                       Albà
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0003-0793-6218" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Eduardo
                      </span>
                      <span class="nlm-surname">
                       Eyras
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.02.08.939942
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2488778--2624026287" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Ivan de la Rubia
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        EMBL Australia Partner Laboratory Network at the Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Pompeu Fabra University (UPF)
                       </span>
                       , E08003 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ivan%2Bde%2Bla%2BRubia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=de%20la%20Rubia%20I&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AIvan%2Bde%2Bla%2BRubia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Akanksha Srivastava
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        EMBL Australia Partner Laboratory Network at the Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Akanksha%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Srivastava%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAkanksha%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4553-9232" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Akanksha Srivastava
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Wenjing Xue
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        EMBL Australia Partner Laboratory Network at the Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Wenjing%2BXue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Xue%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWenjing%2BXue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Joel A Indi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        EMBL Australia Partner Laboratory Network at the Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Universidade de Lisboa
                       </span>
                       , Lisboa,
                       <span class="nlm-country">
                        Portugal
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Joel%2BA%2BIndi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Indi%20JA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJoel%2BA%2BIndi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Silvia Carbonell-Sala
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Pompeu Fabra University (UPF)
                       </span>
                       , E08003 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Centre for Regulatory Genomics (CRG)
                       </span>
                       , E08001 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Silvia%2BCarbonell-Sala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Carbonell-Sala%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ASilvia%2BCarbonell-Sala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Julien Lagarde
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Pompeu Fabra University (UPF)
                       </span>
                       , E08003 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Centre for Regulatory Genomics (CRG)
                       </span>
                       , E08001 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Julien%2BLagarde%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lagarde%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJulien%2BLagarde%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-0290-7445" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Julien Lagarde
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     M Mar Albà
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Pompeu Fabra University (UPF)
                       </span>
                       , E08003 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Catalan Institution for Research and Advanced Studies (ICREA)
                       </span>
                       , E08010 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Hospital del Mar Medical Research Institute (IMIM)
                       </span>
                       , E08001 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=M%2BMar%2BAlb%C3%A0%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Alb%C3%A0%20MM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AM%2BMar%2BAlb%25C3%25A0%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-7963-7375" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for M Mar Albà
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:malba@imim.es" class="" data-icon-position="" data-hide-link-title="0">
                        malba@imim.es
                       </a>
                       <a href="mailto:eduardo.eyras@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">
                        eduardo.eyras@anu.edu.au
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Eduardo Eyras
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        EMBL Australia Partner Laboratory Network at the Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Australian National University
                       </span>
                       , Acton ACT 2601, Canberra,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Catalan Institution for Research and Advanced Studies (ICREA)
                       </span>
                       , E08010 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Hospital del Mar Medical Research Institute (IMIM)
                       </span>
                       , E08001 Barcelona,
                       <span class="nlm-country">
                        Spain
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Eduardo%2BEyras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Eyras%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AEduardo%2BEyras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-0793-6218" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Eduardo Eyras
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:malba@imim.es" class="" data-icon-position="" data-hide-link-title="0">
                        malba@imim.es
                       </a>
                       <a href="mailto:eduardo.eyras@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">
                        eduardo.eyras@anu.edu.au
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.02.08.939942v5" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2488778" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2488778/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.08.939942v5.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2488778" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2488778/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.08.939942v5.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2488778" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2488778/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.08.939942v5.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2488778" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2488778/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.08.939942v5.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2488778" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2488778/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.08.939942v5.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2488778" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2488778/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.02.08.939942v5.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2488778" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2488778/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Nanopore sequencing enables the efficient and unbiased measurement of transcriptomes from any sample. However, current methods for transcript identification and quantification rely of mapping reads to a reference genome, which precludes the study of species with a partial or missing reference or the identification of disease-specific transcripts not readily identifiable from a reference. Here we present RATTLE, a tool to perform reference-free reconstruction and quantification of transcripts using only Nanopore reads. Using simulated data and experimental data from isoform spike-ins, human tissues, and cell lines, we show that RATTLE accurately determines transcript sequences and their abundances, and shows good scalability with the number of transcripts. RATTLE provides unprecedented access to transcriptomes from any sample and species without relying on a reference or additional technologies.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Background
                            </h2>
                            <p id="p-6">
                             The direct interrogation of transcriptome has proven effective to study species without an available genome reference [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ] or non-model organisms of ecological relevance [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ], or to identify cancer-specific RNAs with diagnostic relevance [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. However, these approaches have suffered from the limitations of the short sequencing reads, which lead to uncertainties in the reconstruction of transcripts [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. Single-molecule long- read sequencing with Oxford Nanopore Technologies (ONT) enables the direct measurement of native RNA molecules, often identify complete known and novel transcripts [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ], and accurately estimate transcript abundances [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. These advantages coupled with the ease of sample preparation and the availability of portable platforms, makes this technology very compelling for the cost-effective and unbiased measurement of transcriptomes from any sample and species. However, current analysis methods rely on the comparison with a reference genome [
                             <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             –
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ] or on the use of multiple sequencing technologies [
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ], which precludes the possibility of cost-effective studies. Methods for de novo DNA assembly [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             –
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ] cannot be directly transferred to transcriptomes, since they do not model genes with multiple transcript isoforms. Similarly, methods developed for Illumina [
                             <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ] rely on their high sequence accuracy, which is currently lacking in individual Nanopore reads. To address these challenges, we have developed RATTLE, a tool to perform reference-free reconstruction and quantification of transcripts using only Nanopore long reads. Using simulated data, isoform spike-ins, and sequencing data from human tissues and cell lines, we show that RATTLE is competitive at recovering transcript sequences and their abundances despite not using any information from the reference. RATTLE lays the foundation for a multitude of potential new applications of Nanopore transcriptomics.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              RATTLE algorithm
                             </h3>
                             <p id="p-7">
                              RATTLE starts by building read clusters that represent potential genes. To circumvent the quadratic complexity of an all-vs-all comparison of reads, RATTLE performs a greedy deterministic clustering using a two-step k-mer based similarity measure (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ). The first step consists of a fast comparison of k-mers (k=6) shared between every two reads (Supp. Fig. 1a), whereas the second step solves the Longest Increasing Subsequence (LIS) problem to find the longest list of collinear matching k-mers between a pair of reads, which defines the RATTLE similarity score (Supp. Fig. 1b). Clusters are greedily generated by comparing reads to a representative of each existing cluster at every step of the iteration. This process generates read clusters, where each cluster represents a potential gene containing reads originating from all possible transcript isoforms.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F1.large.jpg?width=800&height=600&carousel=1" title="RATTLE proceeds through three main steps: clustering, error-correction, and transcript polishing. In the clustering step, gene-like and transcript-like clusters are produced. Error correction is performed at the level of transcript clusters. At the transcript- polishing step, transcript consensus sequences and their abundances are produced." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478826943" data-figure-caption='<div class="highwire-markup">RATTLE proceeds through three main steps: clustering, error-correction, and transcript polishing. In the clustering step, gene-like and transcript-like clusters are produced. Error correction is performed at the level of transcript clusters. At the transcript- polishing step, transcript consensus sequences and their abundances are produced.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F1.medium.gif" width="138" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F1.medium.gif" width="138" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <p id="p-8" class="first-child">
                                RATTLE proceeds through three main steps: clustering, error-correction, and transcript polishing. In the clustering step, gene-like and transcript-like clusters are produced. Error correction is performed at the level of transcript clusters. At the transcript- polishing step, transcript consensus sequences and their abundances are produced.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-9">
                              Gene clusters are subsequently split into sub-clusters defining candidate transcripts. These transcript clusters are built by determining whether every pair of reads in a gene cluster is more likely to originate from different transcript isoforms rather than from the same isoform. This is estimated using a threshold for the maximum variance allowed (--iso-max-variance) for the distribution of gap-lengths between the previously calculated co-linear matching k-mers (Supp. Fig. 1c). RATTLE performs error correction within each one of the transcript clusters by generating a block-guided multiple sequence alignment (MSA). Each read is assessed for error correction using the base quality for each base and the average quality of the consensus at each MSA column. RATTLE then builds the final transcripts after a polishing step to refine the cluster definitions. The final transcript sequence is obtained as a consensus from the final transcript cluster and the abundance is defined as the total read count in that cluster. More details are provided in the Methods section.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Evaluation of read clustering into genes and transcripts
                             </h3>
                             <p id="p-10">
                              To illustrate the ability of RATTLE similarity score to perform read clustering, we simulated reads from multiple annotated transcripts using the read length distribution observed in a Nanopore cDNA sequencing run (Supp. Fig. 1d). RATTLE similarity score separated reads originating from different transcripts better than using the score based on a sequence alignment using Minimap2 [
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ], which is based on minimizers (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2a
                              </a>
                              ). This indicates that although minimizers provide an efficient comparison between sequences, RATTLE similarity score may be more robust for comparisons between error-prone reads. We further considered reads simulated from two transcripts from the same gene differing by only by an internal alternative exon (Supp. Fig. 1e). RATTLE similarity score separated reads originating from each of the two transcript isoforms better than using the number of matches in an alignment as similarity measure (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2b
                              </a>
                              ). Furthermore, we simulated reads from three pairs of transcripts differing by one internal alternative exon using 10 different exon lengths between 25 and 150, and for 7 different values of the -iso-max-variance parameter. The results showed that RATTLE correctly separated reads from two isoforms for alternative exons longer than 50-65 nt using --iso- max-variance 5 or higher (Supp. Fig. 2) (Supp. Table S1).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F2.large.jpg?width=800&height=600&carousel=1" title="(a) Comparison of the RATTLE similarity score (x axis), based on the longest increasing subsequence, with a similarity score calculated from Minimap2 (y axis), using k=6. Each dot represents a comparison between a pair of reads simulated from the same (orange) or different (blue) transcripts from two different genes. The distribution of values for each comparison is given as box plots along each axis. (b) Number of common bases between two reads (x-axis) and variance in the distribution of gap-length differences between adjacent matching k-mers between the same two reads (y axis). Reads were simulated from two transcript isoforms from the same gene differing only by an internal exon (Supp. Fig. 1e). Each dot is colored according to whether the reads originated from the same transcript, 1-1 (red) or 2-2 (blue) or not, 1-2 (green). (c) Clustering accuracy of RATTLE, CARNAC, and isONclust in terms of the V-measure (y axis), using simulated reads. Simulations (x-axis) were performed with a single (SI) or with multiple (MI) transcript isoforms per gene, using a different number of total transcripts (t) and a different number of reads per transcript (r), indicated as SIt-r or MIt-r. Other accuracy metrics are provided in Supp. Table S2. (d) Clustering accuracy using spike-in isoform RNA variants (SIRVs) as reference. The plot shows the V-measure (y axis) for RATTLE, CARNAC, and isONclust using SIRV reads from four of the tested samples (x axis): three using the cDNA-seq protocol (cDNA1, 2, 3) and one using the dRNA protocol (RNA1) (Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478826943" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(a)</strong> Comparison of the RATTLE similarity score (x axis), based on the longest increasing subsequence, with a similarity score calculated from Minimap2 (y axis), using k=6. Each dot represents a comparison between a pair of reads simulated from the same (orange) or different (blue) transcripts from two different genes. The distribution of values for each comparison is given as box plots along each axis. <strong>(b)</strong> Number of common bases between two reads (x-axis) and variance in the distribution of gap-length differences between adjacent matching k-mers between the same two reads (y axis). Reads were simulated from two transcript isoforms from the same gene differing only by an internal exon (Supp. Fig. 1e). Each dot is colored according to whether the reads originated from the same transcript, 1-1 (red) or 2-2 (blue) or not, 1-2 (green). <strong>(c)</strong> Clustering accuracy of RATTLE, CARNAC, and isONclust in terms of the V-measure (y axis), using simulated reads. Simulations (x-axis) were performed with a single (SI) or with multiple (MI) transcript isoforms per gene, using a different number of total transcripts (<em>t</em>) and a different number of reads per transcript (<em>r</em>), indicated as SI<em>t</em>-<em>r</em> or MI<em>t</em>-<em>r</em>. Other accuracy metrics are provided in Supp. Table S2. <strong>(d)</strong> Clustering accuracy using spike-in isoform RNA variants (SIRVs) as reference. The plot shows the V-measure (y axis) for RATTLE, CARNAC, and isONclust using SIRV reads from four of the tested samples (x axis): three using the cDNA-seq protocol (cDNA1, 2, 3) and one using the dRNA protocol (RNA1) (Methods).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F2.medium.gif" width="440" height="434"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F2.medium.gif" width="440" height="434"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <p id="p-11" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Comparison of the RATTLE similarity score (x axis), based on the longest increasing subsequence, with a similarity score calculated from Minimap2 (y axis), using k=6. Each dot represents a comparison between a pair of reads simulated from the same (orange) or different (blue) transcripts from two different genes. The distribution of values for each comparison is given as box plots along each axis.
                                <strong>
                                 (b)
                                </strong>
                                Number of common bases between two reads (x-axis) and variance in the distribution of gap-length differences between adjacent matching k-mers between the same two reads (y axis). Reads were simulated from two transcript isoforms from the same gene differing only by an internal exon (Supp. Fig. 1e). Each dot is colored according to whether the reads originated from the same transcript, 1-1 (red) or 2-2 (blue) or not, 1-2 (green).
                                <strong>
                                 (c)
                                </strong>
                                Clustering accuracy of RATTLE, CARNAC, and isONclust in terms of the V-measure (y axis), using simulated reads. Simulations (x-axis) were performed with a single (SI) or with multiple (MI) transcript isoforms per gene, using a different number of total transcripts (t) and a different number of reads per transcript (r), indicated as SIt-r or MIt-r. Other accuracy metrics are provided in Supp. Table S2.
                                <strong>
                                 (d)
                                </strong>
                                Clustering accuracy using spike-in isoform RNA variants (SIRVs) as reference. The plot shows the V-measure (y axis) for RATTLE, CARNAC, and isONclust using SIRV reads from four of the tested samples (x axis): three using the cDNA-seq protocol (cDNA1, 2, 3) and one using the dRNA protocol (RNA1) (Methods).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-12">
                              To test the accuracy in the identification of gene clusters, we compared RATTLE with two other methods that cluster long reads, CARNAC [
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ] and isONclust [
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ]. We built several reference datasets of simulated reads from multiple genes with one or more transcripts per gene and with a variable number of reads per transcript. RATTLE showed higher accuracy at recovering gene clusters in most of the comparisons and using different metrics (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2c
                              </a>
                              ) (Supp. Table S2). At the transcript level, RATTLE performed better than CARNAC and isONclust (Supp. Table S2). Moreover, RATTLE was faster than CARNAC and isONclust in all tested datasets (Supp. Table S2).
                             </p>
                             <p id="p-13">
                              We further assessed the accuracy of RATTLE at recovering gene clusters using Spike-in isoform RNA Variants (SIRVs). The SIRV genome (SIRVome) is organized into 7 different genes containing a total set of 69 transcripts with known coordinates, sequences, and abundances. We performed 4 different experiments with added SIRVs: PCR-based cDNA sequencing (cDNA-seq) from human brain (two independent replicates, cDNA1, cDNA2) and human heart (cDNA3) samples, and direct RNA (dRNA-seq) from the same heart sample (RNA1). We first used the SIRV transcripts aggregated per gene to evaluate clustering at the gene level. RATTLE was faster and showed higher accuracy in the identification of SIRV genes compared with the other two methods (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2d
                              </a>
                              ) (Supp. Table S3). Similar results were found using dRNA and cDNA reads for SIRVs in mouse samples [
                              <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ] (Supp. Table S3). CARNAC and isONclust appeared to be more sensitive to the dynamic range of SIRV abundances. Furthermore, RATTLE achieved high accuracy at clustering reads into SIRV transcripts using multiple metrics (Supp. Table S3).
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Evaluation of sequence accuracy after error correction
                             </h3>
                             <p id="p-14">
                              To test RATTLE accuracy in error correction in Nanopore reads without using a reference, we used the same SIRV reads and compared RATTLE results with methods developed for long-read self-correction. We used CONSENT [
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ] and isONcorrect [
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ], and the self-correction step from Canu [
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ], which has been used previously for the correction of cDNA reads [
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. Reads were mapped to the SIRVome with Minimap2 [
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ] before and after correction by each method. As a comparison, we included TranscriptClean [
                              <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ] to correct the reads mapped to the genome reference without using the annotation. After correction, all methods improved the percentage identity with the SIRV transcript isoforms both for cDNA (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3a
                              </a>
                              ) and dRNA (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3b
                              </a>
                              ) reads (Supp. Fig. 3). All methods also showed a decrease in the error rates in cDNA (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3c
                              </a>
                              ) and dRNA (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3d
                              </a>
                              ) reads (Supp. Fig. 4). Both RATTLE and isONcorrect achieved results close to TranscriptClean, which had the best improvement over raw reads. TranscriptClean results were similar for cDNA and dRNA reads. However, self-correction with dRNA reads did not improve percentage identity and decrease error-rate as much as with cDNA reads. Notably, RATTLE was faster than all other methods in all tested samples, with runtimes ranging from 1.64 minutes (mins) (for 14,958 reads) to 123.9 mins (for 214,107 reads) (Supp. Table S4). Furthermore, RATTLE corrected approximately as many reads as TranscriptClean and isONcorrect, and more than CONSENT (Supp. Table S4). We also performed error correction using reads from the human transcriptome. RATTLE showed similar improvements over raw reads to isONcorrect and TranscriptClean (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Figs. 3e
                              </a>
                              and
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               3f
                              </a>
                              ) (Supp. Table S5).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F3.large.jpg?width=800&height=600&carousel=1" title="We show the percentage identity distributions of all SIRV reads before (Raw) and after correcting with RATTLE, CONSENT, isONcorrect, Canu, and TranscriptClean (Clean) for Nanopore cDNA (sample cDNA2) (a) and dRNA reads (sample RNA1) (b). Percentage identity was calculated as the number of nucleotide matches divided by the total length of the aligned region. Other samples are shown in Supp. Fig. 3. We also show the error rate distribution of SIRV reads before (Raw) and after correction with RATTLE, CONSENT, isONcorrect, Canu, and TranscriptClean (Clean) for the same cDNA (c) and RNA (d) samples. The error rate was calculated as the sum of insertions, deletions, and substitutions divided by the length of the read. Other samples are shown in Supp. Fig. 4. Finally, we show the distribution of percentage identity (e) and error rate (f) for reads from the human transcriptome before (Raw) and after correcting with RATTLE, CONSENT, isONcorrect, and TranscriptClean (Clean) for Human heart dRNA-seq (RNA1). Percentage identity and error rate was calculated as in (a-d)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478826943" data-figure-caption='<div class="highwire-markup">We show the percentage identity distributions of all SIRV reads before (Raw) and after correcting with RATTLE, CONSENT, isONcorrect, Canu, and TranscriptClean (Clean) for Nanopore cDNA (sample cDNA2) (a) and dRNA reads (sample RNA1) (b). Percentage identity was calculated as the number of nucleotide matches divided by the total length of the aligned region. Other samples are shown in Supp. Fig. 3. We also show the error rate distribution of SIRV reads before (Raw) and after correction with RATTLE, CONSENT, isONcorrect, Canu, and TranscriptClean (Clean) for the same cDNA (c) and RNA (d) samples. The error rate was calculated as the sum of insertions, deletions, and substitutions divided by the length of the read. Other samples are shown in Supp. Fig. 4. Finally, we show the distribution of percentage identity (e) and error rate (f) for reads from the human transcriptome before (Raw) and after correcting with RATTLE, CONSENT, isONcorrect, and TranscriptClean (Clean) for Human heart dRNA-seq (RNA1). Percentage identity and error rate was calculated as in (a-d).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F3.medium.gif" width="440" height="312"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F3.medium.gif" width="440" height="312"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <p id="p-15" class="first-child">
                                We show the percentage identity distributions of all SIRV reads before (Raw) and after correcting with RATTLE, CONSENT, isONcorrect, Canu, and TranscriptClean (Clean) for Nanopore cDNA (sample cDNA2) (a) and dRNA reads (sample RNA1) (b). Percentage identity was calculated as the number of nucleotide matches divided by the total length of the aligned region. Other samples are shown in Supp. Fig. 3. We also show the error rate distribution of SIRV reads before (Raw) and after correction with RATTLE, CONSENT, isONcorrect, Canu, and TranscriptClean (Clean) for the same cDNA (c) and RNA (d) samples. The error rate was calculated as the sum of insertions, deletions, and substitutions divided by the length of the read. Other samples are shown in Supp. Fig. 4. Finally, we show the distribution of percentage identity (e) and error rate (f) for reads from the human transcriptome before (Raw) and after correcting with RATTLE, CONSENT, isONcorrect, and TranscriptClean (Clean) for Human heart dRNA-seq (RNA1). Percentage identity and error rate was calculated as in (a-d).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F4.large.jpg?width=800&height=600&carousel=1" title="(a) Left panel: recall of unique SIRV introns and intron-chains obtained by mapping reads to the SIRV genome before (Raw) and after correction with RATTLE, CONSENT, Canu, isONcorrect, and TranscriptClean (Clean) for the dRNA-seq (RNA 1) and the cDNA (cDNA 1, 2, 3) samples. Recall was calculated as the fraction of unique annotated introns or intron-chains exactly found by each method. Middle panel: Precision values for introns and intron-chains for the same methods and datasets. Precision was calculated as the fraction of unique introns or intron-chains predicted by reads that matched exactly the SIRV annotation. Right panel: Read-precision for introns and intron-chains for the same methods and datasets. Read-precision was calculated as the fraction of all introns or intron-chains predicted in reads that corresponded to SIRV introns or intron chains. Similar plots for internal and external exons are given in Supp. Fig. 5. Only cases with >5 reads support were considered. (b) Same accuracy plots as in (a) for the same methods and datasets but using expressed Gencode transcripts as reference. (c) We plot the recall (green), precision (red) and read-precision (blue) of the SIRV introns (y axis), as a function of the number of minimum reads supporting the predictions (x axis). We indicate for each case the threshold at which a precision (red) of approximately 0.95 is achieved. For that threshold we indicate the corresponding recall (green) and read-precision (blue). The plot corresponds to the dataset cDNA2. Results for other samples are available in Supp. Fig. 7. (d) Same accuracy plots as in (c) but using expressed Gencode transcripts as reference. Results for other samples are available in Supp. Fig. 9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478826943" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(a)</strong> Left panel: recall of unique SIRV introns and intron-chains obtained by mapping reads to the SIRV genome before (Raw) and after correction with RATTLE, CONSENT, Canu, isONcorrect, and TranscriptClean (Clean) for the dRNA-seq (RNA 1) and the cDNA (cDNA 1, 2, 3) samples. Recall was calculated as the fraction of unique annotated introns or intron-chains exactly found by each method. Middle panel: Precision values for introns and intron-chains for the same methods and datasets. Precision was calculated as the fraction of unique introns or intron-chains predicted by reads that matched exactly the SIRV annotation. Right panel: Read-precision for introns and intron-chains for the same methods and datasets. Read-precision was calculated as the fraction of all introns or intron-chains predicted in reads that corresponded to SIRV introns or intron chains. Similar plots for internal and external exons are given in Supp. Fig. 5. Only cases with >5 reads support were considered. <strong>(b)</strong> Same accuracy plots as in (a) for the same methods and datasets but using expressed Gencode transcripts as reference. <strong>(c)</strong> We plot the recall (green), precision (red) and read-precision (blue) of the SIRV introns (y axis), as a function of the number of minimum reads supporting the predictions (x axis). We indicate for each case the threshold at which a precision (red) of approximately 0.95 is achieved. For that threshold we indicate the corresponding recall (green) and read-precision (blue). The plot corresponds to the dataset cDNA2. Results for other samples are available in Supp. Fig. 7. <strong>(d)</strong> Same accuracy plots as in (c) but using expressed Gencode transcripts as reference. Results for other samples are available in Supp. Fig. 9.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F4.medium.gif" width="440" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F4.medium.gif" width="440" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <p id="p-16" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Left panel: recall of unique SIRV introns and intron-chains obtained by mapping reads to the SIRV genome before (Raw) and after correction with RATTLE, CONSENT, Canu, isONcorrect, and TranscriptClean (Clean) for the dRNA-seq (RNA 1) and the cDNA (cDNA 1, 2, 3) samples. Recall was calculated as the fraction of unique annotated introns or intron-chains exactly found by each method. Middle panel: Precision values for introns and intron-chains for the same methods and datasets. Precision was calculated as the fraction of unique introns or intron-chains predicted by reads that matched exactly the SIRV annotation. Right panel: Read-precision for introns and intron-chains for the same methods and datasets. Read-precision was calculated as the fraction of all introns or intron-chains predicted in reads that corresponded to SIRV introns or intron chains. Similar plots for internal and external exons are given in Supp. Fig. 5. Only cases with >5 reads support were considered.
                                <strong>
                                 (b)
                                </strong>
                                Same accuracy plots as in (a) for the same methods and datasets but using expressed Gencode transcripts as reference.
                                <strong>
                                 (c)
                                </strong>
                                We plot the recall (green), precision (red) and read-precision (blue) of the SIRV introns (y axis), as a function of the number of minimum reads supporting the predictions (x axis). We indicate for each case the threshold at which a precision (red) of approximately 0.95 is achieved. For that threshold we indicate the corresponding recall (green) and read-precision (blue). The plot corresponds to the dataset cDNA2. Results for other samples are available in Supp. Fig. 7.
                                <strong>
                                 (d)
                                </strong>
                                Same accuracy plots as in (c) but using expressed Gencode transcripts as reference. Results for other samples are available in Supp. Fig. 9.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Evaluation of exon-intron structures after error correction
                             </h3>
                             <p id="p-17">
                              We next evaluated the ability to accurately recover exon-intron structures after mapping self-corrected reads to a genome reference. To this end, we used simple and easily interpretable metrics to determine the accuracy of the exon-intron structures. We first analyzed the exact matches to introns and intron-chains, defined as an ordered sequence of introns in an annotated transcript or mapped read (equivalent to FSM in the SQANTI nomenclature [
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ]). Using the SIRV reads and the SIRVome as reference, RATTLE recovered slightly fewer introns and intron-chains than other methods (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4a
                              </a>
                              , left panel) but displayed precision values similar to isONcorrect and higher than the other methods (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4a
                              </a>
                              , middle panel). The precision was generally low for all meethods, suggesting that despite the read correction, the proportion of unique false positive introns and intron-chains was still high for all methods. To further investigate this, we calculated a read-precision metric, defined as the proportion of reads supporting correctly identified annotated features. All tools showed an increase in read-precision with respect to precision (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4a
                              </a>
                              , right panel), indicating that most of the wrongly predicted features are supported by a low number of reads. Indeed, false positive introns generally showed lower read support than true positives (Supp. Fig. 5).
                             </p>
                             <p id="p-18">
                              We performed a similar analysis mapping sequencing reads to the human genome and testing the exact matches to the exon-intron structures of the human gene annotation. As no ground truth was available, we used as reference the exon-intron structures of Gencode transcripts that had evidence of expression in each sample. Compared to the analysis performed with the SIRVs, recall was relatively low for all methods, reflecting the higher transcriptome complexity of this dataset. RATTLE recovered a similar proportion of introns and intron-chains as the other tools, despite not using any information from the reference (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4b
                              </a>
                              , left panel). In the case of precision (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig. 4b
                              </a>
                              , middle panel) and read-precision (
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Fig. 4b
                              </a>
                              , right panel), RATTLE and isONcorrect achieved the highest values, suggesting that self-correction may help in discarding potential false positives.
                             </p>
                             <p id="p-19">
                              We also performed comparisons to annotated exons, separated according to whether they were internal or external in the annotated transcript, using either SIRVs (Supp. Fig. 6) (Supp. Table S6) or the Gencode human transcriptome (Supp. Fig. 7) (Supp. Table S7). In general, the results recapitulated those observed for introns but we could also observe that external exons were less well recovered than internal ones, something expected given that the first 15 nucleotides at the 5’end of mRNAs are typically not recovered with ONT [
                              <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ,
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ].
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Impact of the number of reads supporting a transcript
                             </h3>
                             <p id="p-20">
                              Our analyses indicated that it is fundamental to consider the number of supporting reads to establish the reliability of the predictions. To establish the accuracy in terms of read support, we calculated the recall, precision, and read-precision for the exact matches with SIRV introns at different thresholds of read support, and estimated the minimal read support needed to achieve a precision of 0.95 (i.e. 5% of the predictions would be false). All correction methods showed an improvement over the raw reads (
                              <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                               Fig. 4c
                              </a>
                              ) (Supp. Fig. 8). RATTLE and isONcorrect were the methods that required the least read support to achieve 0.95 precision. Moreover, the recall at these thresholds for RATTLE and isONcorrect was also higher than for the other methods. We did the same analysis using the expressed Gencode human transcripts as reference and found again that RATTLE and isONcorrect required the least read support for 0.95 precision and achieved the highest recalls at these thresholds (
                              <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                               Fig. 4d
                              </a>
                              ) (Supp. Fig. 9). These results indicate that RATTLE and isONcorrect generally require lower read support than other methods to ascertain the reliability of the introns defined by their read corrections.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Evaluation of transcript abundance estimation
                             </h3>
                             <p id="p-21">
                              We next tested the accuracy of RATTLE transcripts to recover the known abundances of the SIRV transcript isoforms. We compared RATTLE predictions with StringTie2 [
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ], FLAIR [
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ], and TALON [
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ], which use the genome and annotation references to delineate transcript isoforms and their abundances. Despite not using any information from the SIRV genome or annotation, the correlation with SIRV isoform abundances achieved by RATTLE was comparable to the other methods (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5a
                              </a>
                              ) (Supp. Fig. 10) (Supp. Table S8). Interestingly, using a simple approach based on counting reads mapped to the transcripts achieved higher correlations than other methods for dRNA and cDNA reads (Supp. Fig. 10) (Supp. Table S8). In agreement with previous similar comparisons [
                              <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ], the correlation for dRNA reads was generally higher than for cDNA for all methods (Supp. Fig. 10) (Supp. Table S8), indicating that dRNA reads produce better transcript abundance estimates than cDNA reads.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F5.large.jpg?width=800&height=600&carousel=1" title="(a) Comparison of the transcript abundances (y axis) predicted by RATTLE, FLAIR, StringTie2, and TALON, with the abundances of the SIRV spike-in transcript isoforms (x axis). Each panel shows the Pearson correlation r for the comparison of the abundance values. Units on the y-axis vary according to the method: RATTLE provides abundances as read counts per million, like TALON and FLAIR. StringTie2 produces a TPM value. SIRV data corresponds to the RNA1 sample. Correlations for other datasets are provided in Supp. Fig. 10. (b) Distribution of the lengths (y-axis, log10 scale) of the transcripts predicted by each method compared with the transcripts from the reference (Gencode v29) using cDNA and dRNA. StringTie2 was run with (stringtie) and without (stringtie_no) the annotation. The dashed line indicates the median value for the reference transcripts. (c) Distribution of the proportion of the annotated transcript covered (y-axis) by the best matching predicted transcript from each method. The dashed line indicates 0.75. (d) Correlation (y-axis) of the predicted transcript abundances from two replicate cDNA and dRNA sequencing experiments from the Nanopore sequencing consortium. For each method, we give the number of predicted transcripts that were matched to the reference annotation in both replicates (x-axis). Colors are like for (b) and (c). (e) Saturation plot of transcripts using an incremental number of dRNA reads as input. Left panel: for each input (x-axis), we give the total number of transcripts predicted by each method (y-axis) with an expression of >5 reads. Right panel: for each input (x-axis), we give the total number of annotated transcripts matched by the predicted transcripts from the left panel with a coverage of at least 75%. The number of transcripts obtained by each method and input is given in Supp. Table S10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478826943" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(a)</strong> Comparison of the transcript abundances (y axis) predicted by RATTLE, FLAIR, StringTie2, and TALON, with the abundances of the SIRV spike-in transcript isoforms (x axis). Each panel shows the Pearson correlation <em>r</em> for the comparison of the abundance values. Units on the y-axis vary according to the method: RATTLE provides abundances as read counts per million, like TALON and FLAIR. StringTie2 produces a TPM value. SIRV data corresponds to the RNA1 sample. Correlations for other datasets are provided in Supp. Fig. 10. <strong>(b)</strong> Distribution of the lengths (y-axis, log10 scale) of the transcripts predicted by each method compared with the transcripts from the reference (Gencode v29) using cDNA and dRNA. StringTie2 was run with (stringtie) and without (stringtie_no) the annotation. The dashed line indicates the median value for the reference transcripts. <strong>(c)</strong> Distribution of the proportion of the annotated transcript covered (y-axis) by the best matching predicted transcript from each method. The dashed line indicates 0.75. <strong>(d)</strong> Correlation (y-axis) of the predicted transcript abundances from two replicate cDNA and dRNA sequencing experiments from the Nanopore sequencing consortium. For each method, we give the number of predicted transcripts that were matched to the reference annotation in both replicates (x-axis). Colors are like for (b) and (c). <strong>(e)</strong> Saturation plot of transcripts using an incremental number of dRNA reads as input. Left panel: for each input (x-axis), we give the total number of transcripts predicted by each method (y-axis) with an expression of >5 reads. Right panel: for each input (x-axis), we give the total number of annotated transcripts matched by the predicted transcripts from the left panel with a coverage of at least 75%. The number of transcripts obtained by each method and input is given in Supp. Table S10.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F5.medium.gif" width="440" height="285"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F5.medium.gif" width="440" height="285"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <p id="p-22" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Comparison of the transcript abundances (y axis) predicted by RATTLE, FLAIR, StringTie2, and TALON, with the abundances of the SIRV spike-in transcript isoforms (x axis). Each panel shows the Pearson correlation r for the comparison of the abundance values. Units on the y-axis vary according to the method: RATTLE provides abundances as read counts per million, like TALON and FLAIR. StringTie2 produces a TPM value. SIRV data corresponds to the RNA1 sample. Correlations for other datasets are provided in Supp. Fig. 10.
                                <strong>
                                 (b)
                                </strong>
                                Distribution of the lengths (y-axis, log10 scale) of the transcripts predicted by each method compared with the transcripts from the reference (Gencode v29) using cDNA and dRNA. StringTie2 was run with (stringtie) and without (stringtie_no) the annotation. The dashed line indicates the median value for the reference transcripts.
                                <strong>
                                 (c)
                                </strong>
                                Distribution of the proportion of the annotated transcript covered (y-axis) by the best matching predicted transcript from each method. The dashed line indicates 0.75.
                                <strong>
                                 (d)
                                </strong>
                                Correlation (y-axis) of the predicted transcript abundances from two replicate cDNA and dRNA sequencing experiments from the Nanopore sequencing consortium. For each method, we give the number of predicted transcripts that were matched to the reference annotation in both replicates (x-axis). Colors are like for (b) and (c).
                                <strong>
                                 (e)
                                </strong>
                                Saturation plot of transcripts using an incremental number of dRNA reads as input. Left panel: for each input (x-axis), we give the total number of transcripts predicted by each method (y-axis) with an expression of >5 reads. Right panel: for each input (x-axis), we give the total number of annotated transcripts matched by the predicted transcripts from the left panel with a coverage of at least 75%. The number of transcripts obtained by each method and input is given in Supp. Table S10.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-23">
                              We next decided to study the properties of the RATTLE outputs using human transcriptomes. As no ground truth was available for the human transcriptome, we had to devise a way to perform controlled comparisons. Using the same human brain and heart samples as before, we first studied the length distributions of the transcripts predicted by each method. RATTLE transcripts showed similar length distributions to reference- based methods (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5b
                              </a>
                              ) (Supp. Fig. 11). Moreover, all methods produced transcripts that were generally shorter than the annotation (Gencode v29) (median length indicated as a dashed line in
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5b
                              </a>
                              ). To compare all tools on similar conditions, we remapped the transcripts predicted by each tool directly to the Gencode transcriptome using Minimap2. Looking at the coverage of the annotated transcripts by the matching predicted transcripts, all tools showed a bimodal distribution (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5c
                              </a>
                              ) (Supp. Fig. 12). This behaviour was similar across the tools for dRNA, but RATTLE and FLAIR showed lower coverage values for cDNA. Our analyses also suggested that transcript matches with a coverage of at least 75% (dashed line in
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig. 5c
                              </a>
                              ) may provide a good correspondence between predictions and annotations. We thus used this strategy to determine the reproducibility of the abundances predicted by each method across replicates of cDNA and dRNA sequencing from a lymphoblastoid cell line [
                              <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ]. The transcripts predicted by RATTLE, FLAIR, StringTie2, and TALON were directly mapped to the human transcriptome annotation, keeping only one-to-one matches with at least 75% coverage (Methods). The abundance of the predicted transcripts assigned to the same annotated transcript was then compared across replicates. Although RATTLE generally recovered fewer transcripts (Supp. Table S9), it showed similar correlation values to the reference-based methods (
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Fig. 5d
                              </a>
                              ). These correlation values were also stable across other coverage cutoffs (Supp. Fig. 13). Moreover, the correlation values for all methods were generally higher for dRNA than for cDNA reads (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig. 3d
                              </a>
                              ) (Supp. Table S9).
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              RATTLE scalability
                             </h3>
                             <p id="p-24">
                              To investigate the relationship between the number of predicted transcripts and the number of input reads, we used data from multiple dRNA runs in the same cell line to generate input datasets of size 100k, 250k, 500k, 750k, 1M, 2M, and 3M reads. The total number of transcripts increased with the number of reads although in a sub-linear manner, except for Talon (
                              <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                               Fig. 5e
                              </a>
                              left panel) (Supp. Table S10). We further measured the number of predicted transcripts that had a matched annotated transcript using the same approach as above. The curves relating the number of transcripts and the number of input reads showed a tendency to saturate and were similar for the different methods (
                              <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                               Fig. 5e
                              </a>
                              right panel). We further compared RATTLE and RNA-Bloom [
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ] using input datasets from 100k to 3M reads from a different experiment (Methods). This analysis showed that RATTLE transcripts are longer than RNA-Bloom transcripts for all input sizes and converged to the expected length of the annotation (Supp. Fig. 14). To further compare both tools, we calculated the number of transcripts produced by both methods for each input size. The number of RATTLE transcripts increased in a sublinear manner, whereas the number of RNA-Bloom transcripts increased at a much faster rate with the input reads (Supp. Fig. 15)
                             </p>
                             <p id="p-25">
                              Finally, we studied the running time and memory usage for all RATTLE steps, i.e., clustering, read correction, and transcript polishing. We first tested five different configurations for the parameters of the clustering step with 3 different input sizes, 100k, 250k, 500k reads, using the SIRV transcriptome to test the accuracy of each run (Supp. Table S11). Using different metrics, these tests showed that RATTLE recovered SIRV genes and transcripts with similar high accuracy across all parameter configurations and input sizes. We also observed that the memory usage was only dependent on the number of input reads. On the other hand, the running time was slightly lower for parameter configurations that resulted in fewer iterations of the greedy clustering algorithm (see configuration c5 in Supp. Table S11).
                             </p>
                             <p id="p-26">
                              Next, we calculated the memory usage and running times for input datasets of 100k, 250k, 500k, 750k, 1M, 2M, and 3M reads. The read correction step showed a larger memory peak than the clustering step for an input size of <1M reads (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 6a
                              </a>
                              ), similarly to the pattern previously observed with the SIRV reads. However, the clustering step was close to a linear form for the clustering step and showed higher memory usage beyond 1M input reads. The running time for all RATTLE steps together was slightly over linear, and much faster than quadratic time (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig. 6b
                              </a>
                              ) (Supp. Table S12). Although RNA-Bloom was generally faster for these inputs, it required much higher memory (Supp. Table S12).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F6.large.jpg?width=800&height=600&carousel=1" title="(a) Maximum memory usage in Gigabytes (y axis) as a function of the number of input reads (x axis) for RATTLE clustering (red) and read correction (blue) steps. (b) Sum of the CPU time in seconds (y axis) for the RATTLE clustering, correction, and polishing steps for an increasing number of input reads (x axis). RATTLE time (red) is compared with the linear (green) and quadratic (blue) times. The O(n) and O(n2) graphs are extrapolated from the first value. For (a) and (b), RATTLE was run with the parameter configuration c5 described in (Supp. Table S11). Memory and timing values are given in Supp. Table S12." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-478826943" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(a)</strong> Maximum memory usage in Gigabytes (y axis) as a function of the number of input reads (x axis) for RATTLE clustering (red) and read correction (blue) steps. <strong>(b)</strong> Sum of the CPU time in seconds (y axis) for the RATTLE clustering, correction, and polishing steps for an increasing number of input reads (x axis). RATTLE time (red) is compared with the linear (green) and quadratic (blue) times. The O(n) and O(n<sup>2</sup>) graphs are extrapolated from the first value. For (a) and (b), RATTLE was run with the parameter configuration c5 described in (Supp. Table S11). Memory and timing values are given in Supp. Table S12.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F6.medium.gif" width="440" height="143"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F6.medium.gif" width="440" height="143"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/08/2020.02.08.939942/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <p id="p-27" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Maximum memory usage in Gigabytes (y axis) as a function of the number of input reads (x axis) for RATTLE clustering (red) and read correction (blue) steps.
                                <strong>
                                 (b)
                                </strong>
                                Sum of the CPU time in seconds (y axis) for the RATTLE clustering, correction, and polishing steps for an increasing number of input reads (x axis). RATTLE time (red) is compared with the linear (green) and quadratic (blue) times. The O(n) and O(n
                                <sup>
                                 2
                                </sup>
                                ) graphs are extrapolated from the first value. For (a) and (b), RATTLE was run with the parameter configuration c5 described in (Supp. Table S11). Memory and timing values are given in Supp. Table S12.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-28">
                             Our analyses provide evidence that despite the current limitations of using ONT reads without a reference genome, RATTLE represents a significant breakthrough in the capacity to perform reference-free transcriptomics with Nanopore reads, without using additional technologies. Using multiple data types and tests, RATTLE shows a competitive accuracy at building genes and transcripts, as well as at estimating their abundances. RATTLE generally achieved a high precision in the exon-intron structures produced by corrected reads and predicted transcripts, which will be fundamental to reliably identify new genes and transcript isoforms in samples without an available reference. In our tests, RATTLE performed comparably to reference- based methods, and self-correction did not impact the ability of RATTLE to recover exon-intron structures with high precision.
                            </p>
                            <p id="p-29">
                             Several limitations remain for reference-free long-read transcriptomics. RATTLE generally recovered fewer known transcripts compared to the reference-based methods. This indicates a limitation in the resolution of clusters when real biological differences are comparable to sequencing errors in reads. In particular, we observed that there is a lower limit in the size of alternative exons that RATTLE can separate between pairs of individual reads, which impacts the identification of transcript isoforms and their abundances. We also observed that quantification with cDNA reads, in comparison with dRNA reads, showed in general high variability across methods, lower accuracy in comparison with the true abundances, and lower correlation across replicates. Our analyses also indicated that simply counting mapped reads can accomplish high accuracy. This suggests that, despite having lower sequence accuracy, dRNA may capture better the lengths and abundances of RNA molecules, and hence may be more suitable for reference-free transcriptomics. Given that abundance is a relative measure and as the length and accuracy of dRNA reads improve with time, accurate quantification might be achieved by simply counting reads mapped to transcripts. In turn, this will also improve the reference-free estimation of abundances with RATTLE.
                            </p>
                            <p id="p-30">
                             In comparison with other read clustering and error correction methods, RATTLE was generally faster. RATTLE algorithms and data structures facilitate the efficient comparison of reads for clustering, and the storage and processing for transcript prediction. Although other approaches may be faster at clustering reads into candidate transcripts, RATTLE achieves higher accuracy in terms of transcript lengths and abundances. RATTLE thus provides competitive reference-free transcriptome analysis for the most common outputs from a MinION platform (500k-3M reads). Further improvements may be required to make a reference-free approach competitive for much larger input datasets, such as those from the PromethION platform, which may yield over 20M dRNA reads.
                            </p>
                            <p id="p-31">
                             RATTLE modularity, with the ability to parameterize each step, means it can be easily adapted to any sample type. Additionally, RATTLE rich output, including information about the predicted transcripts and genes, as well as the reads used to build each transcript, will prove valuable in downstream analyses, including the study of differential transcript usage [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ], the analysis of single-cell long-read sequencing [
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ], and the identification of RNA modifications in non-model species [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ]. RATTLE lays the foundation of exciting developments in long-read transcriptomics. Considering the rapid growth of the interest in ONT RNA sequencing in a variety of new samples and systems, RATTLE can become a valuable tool for the scientific community.
                            </p>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             Data availability
                            </h2>
                            <p id="p-32">
                             Nanopore sequencing data generated in this study has been deposited in the European Nucleotide Archive (ENA) under the umbrella study PRJEB40410 (
                             <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB40410">
                              https://www.ebi.ac.uk/ena/browser/view/PRJEB40410
                             </a>
                             ) including the FASTQ files (
                             <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB39835">
                              https://www.ebi.ac.uk/ena/browser/view/PRJEB39835
                             </a>
                             ) and signal data (
                             <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB40335">
                              https://www.ebi.ac.uk/ena/browser/view/PRJEB40335
                             </a>
                             ). Datasets from mouse used in this study were obtained ENA under study accession PRJEB27590 (
                             <a href="http://www.ebi.ac.uk/ena/data/view/PRJEB27590">
                              http://www.ebi.ac.uk/ena/data/view/PRJEB27590
                             </a>
                             ) (runs ERR2680375 and ERR2680377), and from the Nanopore sequencing consortium from
                             <a href="https://github.com/nanopore-wgs-consortium/NA12878">
                              https://github.com/nanopore-wgs-consortium/NA12878
                             </a>
                             (under nanopore-human-transcriptome). Direct RNA sequencing data from HEK293 cells was obtained from ENA, under the accession PRJEB40872 (
                             <a href="http://www.ebi.ac.uk/ena/data/view/PRJEB40872">
                              http://www.ebi.ac.uk/ena/data/view/PRJEB40872
                             </a>
                             ). Data files with the simulated sets of reads, as well as the output files generated by the different assembly programs have been deposited in figshare (
                             <a href="https://figshare.com/projects/RATTLE_Paper_Data/113580">
                              https://figshare.com/projects/RATTLE_Paper_Data/113580
                             </a>
                             ).
                            </p>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             Software availability
                            </h2>
                            <p id="p-33">
                             RATTLE is written in C++ and is available at
                             <a href="https://github.com/comprna/RATTLE">
                              https://github.com/comprna/RATTLE
                             </a>
                             under the GPL-3.0 license.
                            </p>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             Potential competing interests
                            </h2>
                            <p id="p-34">
                             Eduardo Eyras has received support from Oxford Nanopore Technologies (ONT) to present the results from this manuscript at scientific conferences. However, ONT played no role in the algorithm or software developments, study design, analysis, or preparation of the manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             Methods
                            </h2>
                            <div class="section" id="sec-16">
                             <h3>
                              RATTLE clustering algorithm
                             </h3>
                             <p id="p-35">
                              Before running RATTLE, reads were pre-processed with porechop (
                              <a href="https://github.com/rrwick/Porechop">
                               https://github.com/rrwick/Porechop
                              </a>
                              ) and those of length 150nt or shorter were filtered out. RATTLE starts by performing a greedy clustering. RATTLE sorts the reads in descending order by their length and processes them one at a time in that order. In the first iteration, RATTLE creates a new cluster with the first unclustered read. All subsequent unclustered reads are then compared against each existing cluster and assigned greedily if the scores are above a certain threshold. Otherwise, a new cluster is created. In each subsequent iteration, the threshold is decreased by a step change, and clusters are created or merged as initially. This process is continued until the threshold used is reduced to a limit. To ensure fast computation and circumvent the quadratic time complexity of an all-vs-all comparison, comparisons are performed using a representative read from each cluster, which is defined by the position in the ranking of read lengths within the cluster and can be set as a parameter by the user. In our analyses, we used the read at the position 0.15 x (number of reads in the cluster). At each iteration, and for each existing cluster, all other clusters and unclustered reads are compared to this cluster using the cluster representative. When two clusters are similar above the set thresholds, a new cluster is formed with the reads from these two clusters.
                             </p>
                             <p id="p-36">
                              To reduce memory usage and for efficient calculation, reads are compared using a two-step similarity calculation. The first one provides a fast approximate comparison using bit-vectors, whereas the second one provides a more sensitive comparison if a threshold is passed. For the bit-vector comparison, sequence k-mers in reads are hashed into 32-bit integers with the hashing function H(A)=0, H(C)=1, H(G)=2, H(T)=3, such that for any k-mer s=b1…bk, the hash H(s) = 4
                              <sup>
                               k-1
                              </sup>
                              H(b1) + 4
                              <sup>
                               k-2
                              </sup>
                              H(b2) + … + H(bk) maps each k-mer to a unique position of a vector that is set to 1, i.e. a k-mer bit-vector. Each read is converted in this way into a bit-vector where the positions in the vector corresponding to the hashed k-mer in the read are set to 1. The first similarity score between a pair of reads is obtained through an AND operation on their bit-vectors and calculated as the number of common k-mers divided by the maximum number of k-mers in either read. Extraction and hashing of k-mers is performed only once per read in linear time, and the vector operations are performed in constant time.
                             </p>
                             <p id="p-37">
                              If this first score is above a previously set threshold, a second similarity calculation is performed. For the second metric, all k-mers from both reads are extracted. Now k can be chosen on the command line, and k- mers are hashed as before. The intersection of the k-mers from both reads and their positions in each read are used to generate a list of triplets (s, p1, p2), where s is the hashed k-mer, p1 is the position of this k-mer in the first read, and p2 is the position of the same k-mer in the second read. These triplets are then sorted by p1 and the Longest Increasing Subsequence (LIS) problem is solved with dynamic programming for p2. This produces the longest set of common co-linear k-mers between a pair of reads. The similarity value is defined as the number of bases covered by these co-linear common k-mers over the length of the shortest read in the pair. If the orientation for cDNA reads is unknown [
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                              ], RATTLE tests both relative orientations for each pair of reads by default, or only the given strand if the --rna option is used. As a result, all reads within a cluster have the same orientation.
                             </p>
                             <p id="p-38">
                              The number of iterations to be performed for clustering is specified in the command line by setting the initial (defined by -B) and final (defined by -b) thresholds for the first bitvector-based score (default 0.4 to 0.2) and a decreasing step change (defined by -f) (default 0.05). For default parameters, iterations are thus performed for thresholds 0.4, 0.35, 0.3, 0.25, and 0.2. A final comparison is done using a threshold of 0.0, i.e., all remaining singletons and all cluster representatives are compared to each other using the LIS-based score. The LIS-based score threshold remains fixed over the entire clustering process. In our analyses, it was required to be 0.2 or larger. Analyses shown here were carried out for k=10 (can be changed with -k). Please see
                              <a href="https://github.com/comprna/RATTLE">
                               https://github.com/comprna/RATTLE
                              </a>
                              for a complete description of all parameters.
                             </p>
                            </div>
                            <div class="section" id="sec-17">
                             <h3>
                              RATTLE transcript-cluster identification and error correction
                             </h3>
                             <p id="p-39">
                              Initial read clusters produced by the algorithm described above are considered to approximately correspond to genes, i.e., gene-clusters. Reads within each cluster are then separated into subclusters according to whether they are likely to originate from different transcript isoforms to form transcript-clusters. RATTLE considers the relative distances between co-linear k-mers calculated from the LIS-based score. Two reads in the same gene-cluster are separated into different transcript-clusters if the distribution of the relative distances between co-linear matching k-mers has a variance greater than a given threshold. That is, if co-linear matching k-mers calculated from the LIS algorithm show relative distances that would be compatible with a difference in exon content. The value 25 was used for the analyses, but it can be also modified as an input parameter.
                             </p>
                             <p id="p-40">
                              RATTLE performs read correction within each transcript-cluster in two steps. First, each cluster with N reads is separated into blocks, each with R reads. In our analyses, R was set to 200. If R≤N<2R, the cluster is split in half, and if N<R, we take a single block. To avoid length bias, blocks are built in parallel from the reads in the cluster sorted by length: to build K blocks, block 1 is made from reads in positions 1, K+1, 2K+1, …; block 2 from reads in positions 2, K+2, 2K+2, … ; and block K from reads in positions K, 2K, 3K, …. A multiple sequence alignment (MSA) is then built from each block using SIMD partial order alignment (SPOA) (
                              <a href="https://github.com/rvaser/spoa">
                               https://github.com/rvaser/spoa
                              </a>
                              ) [
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ]. A consensus from each column in the MSA is then extracted in the following way: for each read and each base of the read, the base is changed to the consensus if the consensus base occurs with at least 60% frequency, but not if this base has an error probability (obtained from the FASTQ file) less than or equal to 1/3 times the average for the consensus base in that position of the alignment. Indels are treated similarly, but without the error constraint. This is only performed using aligned positions. That is, for each base in a given read, other reads of the MSA contribute to the correction if they have a base or an internal gap in that position. The consensus sequences from all blocks are then realigned with SPOA to obtain a final MSA for the transcript-cluster and an associated consensus is obtained as before. Only transcript-clusters with a minimum number of reads are corrected and considered for further analysis. Here we used transcript-clusters with more than 5 reads. The frequency of the consensus, error-probability cutoff, and the minimum number of reads for a transcript cluster can be set up as input parameters. We observed that in MSAs many reads had a few bases wrongly aligned at the terminal regions. To fix these cases so that they do not affect the correction step, RATTLE identifies small blocks (<10nt) followed by large gaps (larger than or equal to 20 positions) at both ends of each aligned read and removes them. A block is defined as a subsequence that has gaps shorter than or equal to 3nt. RATTLE keeps removing blocks that satisfy these conditions at both ends of every aligned read until there no more such blocks are found.
                             </p>
                            </div>
                            <div class="section" id="sec-18">
                             <h3>
                              RATTLE transcript polishing and quantification
                             </h3>
                             <p id="p-41">
                              To define the final list of transcripts, RATTLE performs a final polishing step of the transcript-cluster definitions. This is done to correct possible splitting of reads originating from the same transcript into different transcript-clusters (under-clustering). For this, RATTLE uses the same 2-step greedy clustering described above on the transcript-clusters using the representative read from each cluster in the comparison. In each of the resulting clusters, an MSA column consensus is calculated, with abundance given by all the reads contained in the final cluster. Additionally, the transcripts are given a gene ID that corresponds to the gene- clusters to which they belong. When two transcript clusters are merged, if they are part of the same original gene-cluster, the resulting transcript stays in the same gene. If they are part of different genes, the gene with more transcripts absorbs the transcripts from the other gene to become one single gene.
                             </p>
                             <p id="p-42">
                              RATTLE outputs different files at different stages of its execution. In the clustering step, it can either output gene-clusters or transcript-clusters in binary files. These files can then be used to extract a CSV file containing each read ID and the cluster it belongs to. The same binary files are also used for the correction step, which outputs three files: one with the corrected reads, one with those that are left uncorrected, and one containing the consensus sequence for each cluster from the input (in FASTQ format). Finally, the transcript-cluster polishing step receives as input the consensus sequences from the correction step and outputs a new file in FASTQ format with the final transcriptome. The quantification of each transcript is included in the header line. The quality per base in each FASTQ entry is calculated as the average of the PHRED quality scores from the bases in each column of the MSA in the transcript-cluster.
                             </p>
                            </div>
                            <div class="section" id="sec-19">
                             <h3>
                              Simulated reads and clusters
                             </h3>
                             <p id="p-43">
                              We developed a wrapper script (available at
                              <a href="https://github.com/comprna/RATTLE">
                               https://github.com/comprna/RATTLE
                              </a>
                              ) for DeepSimulator [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ] to simulate a specific number of reads per transcript considering a read length distribution. The length distribution was calculated from a human cDNA sequencing sample from the Nanopore consortium [
                              <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ]. To simulate the read sequences, we used the Gencode transcript annotation (v29), filtering out pseudogenes and genes from non-standard chromosomes, and removing transcripts that showed > 95% percentage identity with other transcripts using CD-HIT [
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ]. We then randomly selected different numbers of genes and transcripts to simulate reads. We considered genes with one single transcript isoform (SI), or genes with multiple isoforms (MI). For each case, various datasets were simulated using different numbers of reads per transcript and different numbers of transcripts per gene. To determine the accuracy of the clustering we used the adjusted rand index, which is a measure of the similarity between two cluster sets corrected for chance [
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ]. Additionally, we used homogeneity, completeness, and the V-measure [
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ]. Homogeneity is maximal when each cluster contains only elements of the same class. Completeness is maximal when all the elements of a single class are in the same cluster. The V-measure is the harmonic mean of completeness and homogeneity. We compared the clusters predicted by each method with the simulated clusters as the reference set. We run isONclust [
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ] (options: --ont --t 24), CARNAC-LR [
                              <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ] with Minimap2 overlaps (options: -t 24 -x ava-ont), and RATTLE clustering (options: -t 24 -k 10 -s 0.20 –v 1000000 –iso-score-threshold 0.30 –iso-kmer-size 11 –iso-max-variance 25 –p 0.15).
                             </p>
                            </div>
                            <div class="section" id="sec-20">
                             <h3>
                              MinION sequencing with SIRVs
                             </h3>
                             <p id="p-44">
                              We performed Nanopore sequencing on two total RNA samples from human brain (Ambion - product num. AM7962; lot num. 1887911) and heart
                              <strong>
                               (
                              </strong>
                              Ambion - product num. AM7966; lot num. 1866106). Unless otherwise noted, kit-based protocols described below followed the manufacturer’s instructions. Regular quality controls using qBIT, nanodrop and Bioanalyzer were performed according to the manufacturer’s protocols to assess the length and the concentration of the samples. rRNA depletion was performed using Ribo-Zero rRNA Removal Kit Human/Mouse/Rat (Epicentre - Illumina). 12 ug of total RNA from each sample were prepared and divided into 3 aliquots (4 ug of total RNA each). 8ul of a 1:100 dilution (1 ng total) of synthetic controls (E2 mix lot number 001418 from SIRV-set, Lexogen) were added to each total RNA aliquot. The resulting ribosomal depleted RNAs were purified using 1.8X Agencourt RNAClean XP beads (Beckman Coulter). Samples were finally resuspended with 11 ul of RNA-free water and stored at -80°C. The cDNA was prepared using 50 ng of rRNA depleted RNA. The Takara Bio cDNA synthesis kit based on SMART (Switching Mechanism at 5’ End of RNA Template) technology coupled with PCR amplification was used to generate high yields of full-length double-stranded cDNA.
                             </p>
                             <p id="p-45">
                              The sequencing libraries were prepared using 1 ug of full-length double-stranded cDNA following the standard ONT protocol SQK-LSK108 for an aliquot of the brain sample (cDNA 2), protocol SQK-LSK109 for 1 aliquot of the brain sample (cDNA 3) and 1 aliquot of the heart sample (cDNA 4). The direct RNA sequencing library was prepared using 500 ng of previously prepared ribosomal depleted sample (RiboZero kit, catalog num. MRZH11124, Epicentre-Illumina) from heart (RNA 2) with standard direct RNA ONT protocol SQK-RNA002, following manufacturer’s instructions, including the RT step. The final libraries were loaded on an R9.4.1 flowcell, and standard ONT scripts available in MinKNOW were used for a total of 48 hours run for each flowcell. ONT sequencing data was basecalled using Guppy 2.3.1+9514fbc (options: -- qscore_filtering --min_qscore 8 --flowcell FLO-MIN106 --kit <kit> --records_per_fastq 0 --recursive -- cpu_threads_per_caller 14 --num_callers 1), where <kit> is the corresponding protocol, as specified above (SQK-LSK109, SQK-LSK108 or SQK-RNA002).
                             </p>
                             <p id="p-46">
                              To select reads corresponding to SIRVs, we run porechop (
                              <a href="https://github.com/rrwick/Porechop">
                               https://github.com/rrwick/Porechop
                              </a>
                              ) and mapped the reads to the SIRV genome (SIRVome) with Minimap2 (options: -t 24 -cx splice --splice-flank=no -- secondary=no). To extract the subset of reads with a hit on the SIRVome and being at least 150nt in length we used seqtk (
                              <a href="https://github.com/lh3/seqtk">
                               https://github.com/lh3/seqtk
                              </a>
                              , option subseq). We also used data from cDNA (ERR2680377) (cDNA_br_mm) and direct RNA (ERR2680375) (RNA_br_mm) sequencing of mouse brain including the E2 SIRVs [
                              <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ], as well as samples from the Nanopore sequencing consortium [
                              <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ].
                             </p>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              Clustering accuracy analysis
                             </h3>
                             <p id="p-47">
                              We first built SIRV isoform clusters by mapping reads to SIRV isoforms with Minimap2 and selecting for each read the SIRV isoform with the best mapping score. All reads that mapped to the same SIRV gene were then considered a gene-cluster. We then clustered reads with RATTLE, CARNAC, and isONclust and measured the accuracy of the predicted clusters by comparing them with the built SIRV gene clusters using the same metrics as with the simulated data. We used subsets of 25,000 reads (subsampled with seqtk) from each sample since we could not make CARNAC run for some of the complete datasets.
                             </p>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              Assessment of error correction accuracy
                             </h3>
                             <p id="p-48">
                              Reads were mapped to the SIRV transcripts with Minimap2 before and after read correction. Here we used the complete dataset of SIRV reads. Each read was assigned to the best matching transcript according to the mapping score. From the CIGAR string of the SAM output, the error rate was calculated as the sum of insertions, deletions, and substitutions divided by the length of the read, and the percentage identity as the number of nucleotide matches divided by the total length of the aligned region. We compared RATTLE (options: -t24 –g 0.3 –m 0.3 –s 200) with CONSENT [
                              <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ] (options: consent-correct --type ONT), Canu [
                              <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ] (options: minReadLength=200 stopOnLowCoverage=0.10 executiveMemory=16 executiveThreads=24), isONcorrect [
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ] (options: --t 24), and TranscriptClean [
                              <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ]. TranscriptClean was run with default parameters using as input the reads mapped with Minimap2 (options: -t 12 -cx splice --splice-flank=no --secondary=no), but with no exon-intron information from the SIRV annotation.
                             </p>
                             <p id="p-49">
                              To perform the accuracy analysis of the SIRV annotation features, PAF files from the mapping were compared with the annotation using ssCheck (available at
                              <a href="https://github.com/comprna/RATTLE/">
                               https://github.com/comprna/RATTLE/
                              </a>
                              ). ssCheck works by comparing annotation features in the mapped reads with the annotation and calculates the number of unique features as well as the total number of features predicted in the mapped reads. As annotation features, we used introns, intron-chains. internal exons and external exons. An intron-chain was defined as an ordered sequence of introns in an annotated transcript or mapped read. The recall was calculated as the fraction of unique annotated features correctly found; precision was calculated as the fraction of unique predicted features that were in the annotation and read-precision was calculated as the fraction of the total number of predicted features in reads that corresponded to annotated features. Read-precision is affected by abundance levels but better reflects the accuracy per read. We developed ssCheck to be able to calculate the read-precision, since gffcompare (
                              <a href="https://ccb.jhu.edu/software/stringtie/gffcompare.shtml">
                               https://ccb.jhu.edu/software/stringtie/gffcompare.shtml
                              </a>
                              ) merges identical intron chains with different identifiers, which precludes this calculation. Additionally, ssCheck calculates accuracy metrics for internal and external exons, which was not possible with gffcompare.
                             </p>
                             <p id="p-50">
                              To perform the comparison with the human transcriptome we considered the Gencode transcripts (v29) after removing pseudogenes, genes from non-standard chromosomes, and transcripts with >95% percentage identity with other transcripts. We only used transcripts that had evidence of expression of >5 reads from mapping the reads to the transcriptome using Minimap2 (options: -t 24 -x map-ont --secondary=no) to avoid spurious matches to transcripts with shared sequence but without evidence of expression in the sample, as done previously [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. We then mapped the reads corrected by each method to the human genome with Minimap2 (options: -t 24 -cx splice --secondary=no). Considering the various genomic features in the expressed annotated transcriptome (introns, intron-chains, internal exons, external exons), we calculated the recall, precision, and read-precision with ssCheck as before.
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              Assessment of transcriptome sequence and abundance accuracy
                             </h3>
                             <p id="p-51">
                              We used FLAIR [
                              <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] (options: align, correct, collapse, quantify –tpm), StringTie2 [
                              <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ] (options: -p 24 –L) and TALON [
                              <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ] (talon_initialize_database, talon, talon_summarize, talon_abundance, talon_create_GTF) with the cDNA and dRNA reads mapped to the SIRV genome with Minimap2 (options: -t 24 –ax splice –splice- flank=no –secondary=no, with –MD tag for TALON). These methods perform read correction (FLAIR and TALON) and transcript quantification (FLAIR, StringTie2 and TALON) of annotated and novel transcripts using the mapped reads with the help of the annotation. For the same samples, RATTLE was run for clustering (options: -t 24, -k 10, -s 0.20 –v 1000000 –iso-score-threshold 0.30 –iso-kmer-size 11 –iso-max-variance 25 –p 0.15), read correction (options: -t24 –g 0.3 –m 0.3 –s 200) and transcript polishing (options: -t 24). As an additional comparison, we mapped raw reads directly to SIRV isoforms with Minimap2 (options: -ax map- ont -t24) and estimated abundances with NanoCount [
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ], which assigns reads to isoforms with an expectation-maximization (EM) algorithm. We also assigned reads directly to SIRV isoforms by mapping them with Minimap2 (options: -t 24 -cx map-ont --secondary=no) and simply counting the number of mapped reads per transcript. FLAIR, StringTie2, and TALON provide the SIRV isoform ID with the predicted abundance. If more than one predicted transcript mapped to the same annotation, we only considered the prediction with the highest abundance. To assess the accuracy of RATTLE, we matched transcripts predicted by RATTLE to the SIRV isoforms using Minimap2 (options: -cx map-ont --secondary=no), assigning each SIRV to the best matching RATTLE transcript. If more than one transcript matched the same SIRV isoform, we selected the RATTLE transcript with the highest abundance.
                             </p>
                             <p id="p-52">
                              To assess the accuracy of the transcriptome in terms of exon-intron structures, we compared the predicted transcripts mapped to the genome with the Gencode transcripts with evidence of expression (>5 reads) in the same sample. To make this analysis comparable across methods, all predicted transcriptomes were mapped against the genome with Minimap2 with Minimap2 (options: -t 24 -cx splice --secondary=no). We then calculated the exact matches of the predicted intron-exon structures (introns, intron chains, internal exons, external exons) with the annotation for that sample using ssCheck as before.
                             </p>
                            </div>
                            <div class="section" id="sec-24">
                             <h3>
                              Correlation of human transcriptomes
                             </h3>
                             <p id="p-53">
                              We used data from cDNA and dRNA sequencing from the Nanopore consortium [
                              <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ]: the samples from Johns Hopkins (cDNA 2 replicates, dRNA 2 replicates) and UCSC (cDNA 2 replicates, dRNA 2 replicates). The resulting transcriptome from each method, FLAIR, TALON, RATTLE, and StringTie2 (with and without using the annotation), was then mapped to the expressed annotated transcripts using Minimap2 (options: -t 24 -x map-ont --secondary=no). To perform an accurate assignment of predicted to annotated transcripts, for each predicted transcript we only kept the best match if it covered at least 75% of the annotated transcript. Moreover, for each annotated transcript, we only assigned the best matching predicted transcript. If more than one passed the filter, we assigned the predicted transcript with the highest abundance. That is, the assignments of predicted and annotated transcripts were one-to-one. The correlation was calculated using the abundances of the predicted transcripts in the two replicates matched to the same annotated transcript.
                             </p>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              Scalability analysis
                             </h3>
                             <p id="p-54">
                              We used 30 pooled direct RNA sequencing runs from the Nanopore sequencing consortium [
                              <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ] (
                              <a href="https://github.com/nanopore-wgs-consortium/NA12878/blob/master/RNA.md">
                               https://github.com/nanopore-wgs-consortium/NA12878/blob/master/RNA.md
                              </a>
                              ) and generated subsamples of 100k, 250k, 500k, 750k, 1M, 2M, and 3M reads, using with seqtk (
                              <a href="https://github.com/lh3/seqtk">
                               https://github.com/lh3/seqtk
                              </a>
                              ). We run FLAIR, TALON and StringTie (with and without annotation) as before on all these samples using the same options as before. RATTLE was run in all these samples (options: -B 0.5 -b 0.3 -f 0.2 --iso) using a machine with 24 cores. We first considered the total number of transcripts predicted by each method for each input size with an expression >5 reads. For each input size we also performed a one-to-one assignment of the predicted transcripts to the transcriptome annotation as described above to calculate the total number of identified annotated transcripts for each input size. We also run RATTLE (options: -B 0.5 -b 0.3 -f 0.2 --iso) and RNA-Bloom [
                              <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ] (option: -long) in datasets of size 100k, 250k, 500k, 750k, 1M, 2M, and 3M reads, by subsampling reads from three dRNA replicates from HEK293 cells [
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ]. The memory usage and running times for RATTLE (Supp. Table S12) were obtained for these input datasets.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-26">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-56">
                             EE and IdlR designed the algorithms in RATTLE with inputs from MMA. IdlR prototyped and implemented the algorithms. IdlR carried out the benchmarking analyses with some contributions from JAI. AS and WX contributed to the software development. SCS and JL generated the experimental data. EE, IdlR, and MMA wrote the paper with inputs from all authors.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-55">
                             We thank Roderic Guigó for providing access to the samples with SIRVs. We also thank the Nanopore sequencing consortium for providing access to their Nanopore transcriptomic sequencing runs. We acknowledge funding support funding from the Australian Research Council with ID DP210102385; from the Spanish Government and FEDER with grants BFU2015-65235-P (MMA), BIO2017-85364-R (EE), PGC2018-094091-B-I00 (MMA); and from the Catalan Government (AGAUR) with grant SGR2017-1020 (MMA). IdlR was supported by grant PRE2018-083413 from the Spanish Government.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               We have performed additional benchmarking analyses and have clarified some of the algorithmic details.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="https://github.com/comprna/RATTLE">
                                https://github.com/comprna/RATTLE
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.1" data-doi="10.1038/nbt.1883">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grabherr
                                  </span>
                                  <span class="cit-name-given-names">
                                   MG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haas
                                  </span>
                                  <span class="cit-name-given-names">
                                   BJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yassour
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levin
                                  </span>
                                  <span class="cit-name-given-names">
                                   JZ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thompson
                                  </span>
                                  <span class="cit-name-given-names">
                                   DA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amit
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Full-length transcriptome assembly from RNA-Seq data without a reference genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Biotechnol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  644
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  52
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.1883&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21572440&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F08%2F2020.02.08.939942.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faherty
                                  </span>
                                  <span class="cit-name-given-names">
                                   SL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Villanueva-Cañas
                                  </span>
                                  <span class="cit-name-given-names">
                                   JL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blanco
                                  </span>
                                  <span class="cit-name-given-names">
                                   MB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Albà
                                  </span>
                                  <span class="cit-name-given-names">
                                   MM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yoder
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Transcriptomics in the wild: Hibernation physiology in free-ranging dwarf lemurs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Ecol. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  709
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29319912">
                                  http://www.ncbi.nlm.nih.gov/pubmed/29319912
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2B%255BInternet%255D%26rft.volume%253D27%26rft.spage%253D709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pinskaya
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saci
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gallopin
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gabriel
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nguyen
                                  </span>
                                  <span class="cit-name-given-names">
                                   HT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Firlej
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Reference-free transcriptome exploration reveals novel RNAs for prostate cancer diagnosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Life Sci. alliance [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/31732695">
                                  http://www.ncbi.nlm.nih.gov/pubmed/31732695
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hsieh
                                  </span>
                                  <span class="cit-name-given-names">
                                   P-H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oyang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y-J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   C-Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Effect of de novo transcriptome assembly on transcript quantification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  8304
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/31165774">
                                  http://www.ncbi.nlm.nih.gov/pubmed/31165774
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%2B%255BInternet%255D%26rft.volume%253D9%26rft.spage%253D8304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Workman
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   PS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jain
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tyson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Razaghi
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Nanopore native RNA sequencing of a human poly(A) transcriptome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1297
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  305
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/31740818">
                                  http://www.ncbi.nlm.nih.gov/pubmed/31740818
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%2B%255BInternet%255D%26rft.volume%253D16%26rft.spage%253D1297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sessegolo
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cruaud
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Da Silva
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cologne
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dubarry
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Derrien
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Transcriptome profiling of mouse samples using nanopore sequencing of cDNA and RNA molecules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  14908
                                 </span>
                                 . Available from:
                                 <a href="http://biorxiv.org/content/early/2019/07/16/575142.abstract">
                                  http://biorxiv.org/content/early/2019/07/16/575142.abstract
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%2B%255BInternet%255D%26rft.volume%253D9%26rft.spage%253D14908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wyman
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mortazavi
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  TranscriptClean: variant-aware correction of indels, mismatches and splice junctions in long-read transcripts
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  340
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29912287">
                                  http://www.ncbi.nlm.nih.gov/pubmed/29912287
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%255BInternet%255D%26rft.volume%253D35%26rft.spage%253D340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kovaka
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zimin
                                  </span>
                                  <span class="cit-name-given-names">
                                   A V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pertea
                                  </span>
                                  <span class="cit-name-given-names">
                                   GM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Razaghi
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   SL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pertea
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Transcriptome assembly from long-read RNA-seq alignments with StringTie2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  278
                                 </span>
                                 . Available from:
                                 <a href="http://biorxiv.org/content/early/2019/07/08/694554.abstract">
                                  http://biorxiv.org/content/early/2019/07/08/694554.abstract
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2B%255BInternet%255D%26rft.volume%253D20%26rft.spage%253D278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Soulette
                                  </span>
                                  <span class="cit-name-given-names">
                                   CM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Baren
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hart
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hrabeta-Robinson
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  <span class="cit-name-given-names">
                                   CJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Full-length transcript characterization of SF3B1 mutation in chronic lymphocytic leukemia reveals downregulation of retained introns
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1438
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/32188845">
                                  http://www.ncbi.nlm.nih.gov/pubmed/32188845
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%2B%255BInternet%255D%26rft.volume%253D11%26rft.spage%253D1438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bushmanova
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Antipov
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lapidus
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prjibelski
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  rnaSPAdes: a de novo transcriptome assembler and its application to RNA-Seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gigascience [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/31494669">
                                  http://www.ncbi.nlm.nih.gov/pubmed/31494669
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fu
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Song
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  IDP-denovo: de novo transcriptome assembly and isoform annotation by hybrid sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2168
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  76
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29905763">
                                  http://www.ncbi.nlm.nih.gov/pubmed/29905763
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%255BInternet%255D%26rft.volume%253D34%26rft.spage%253D2168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.12" data-doi="10.1101/gr.215087.116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walenz
                                  </span>
                                  <span class="cit-name-given-names">
                                   BP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berlin
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bergman
                                  </span>
                                  <span class="cit-name-given-names">
                                   NH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillippy
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  722
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  36
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMDgvMjAyMC4wMi4wOC45Mzk5NDIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.08.939942v5.13" data-doi="10.1093/bioinformatics/btw321">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salmela
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walve
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rivals
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ukkonen
                                  </span>
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Accurate self-correction of errors in long reads using de Bruijn graphs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  799
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  806
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D799%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw321&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xiao
                                  </span>
                                  <span class="cit-name-given-names">
                                   C-L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  <span class="cit-name-given-names">
                                   S-Q
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   K-N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  MECAT: fast mapping, error correction, and de novo assembly for single-molecule sequencing reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1072
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/28945707">
                                  http://www.ncbi.nlm.nih.gov/pubmed/28945707
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%2B%255BInternet%255D%26rft.volume%253D14%26rft.spage%253D1072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Luo
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patterson
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-Seq reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1660
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24532719">
                                  http://www.ncbi.nlm.nih.gov/pubmed/24532719
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%255BInternet%255D%26rft.volume%253D30%26rft.spage%253D1660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.08.939942v5.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marchet
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lecompte
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Silva C
                                  </span>
                                  <span class="cit-name-surname">
                                   Da
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cruaud
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aury
                                  </span>
                                  <span class="cit-name-given-names">
                                   J-M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nicolas
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  De novo clustering of long reads by gene from transcriptomics data
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Nucleic Acids Res. [Internet]
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ; Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/30260405">
                                  http://www.ncbi.nlm.nih.gov/pubmed/30260405
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.08.939942v5.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sahlin
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Medvedev
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  De novo clustering of long-read transcriptome data using a greedy, quality-value based algorithm
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Int. Conf. Res. Comput. Mol. Biol
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ;
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 . p.
                                 <span class="cit-fpage">
                                  227
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  42
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morisse
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marchet
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Limasset
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lecroq
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lefebvre
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Scalable long read self-correction and assembly polishing with multiple sequence alignment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  761
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/33436980">
                                  http://www.ncbi.nlm.nih.gov/pubmed/33436980
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%2B%255BInternet%255D%26rft.volume%253D11%26rft.spage%253D761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sahlin
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Medvedev
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Error correction enables use of Oxford Nanopore technology for reference-free transcriptome analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/33397972">
                                  http://www.ncbi.nlm.nih.gov/pubmed/33397972
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%2B%255BInternet%255D%26rft.volume%253D12%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Al-Eryani
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carswell
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferguson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blackburn
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barton
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  High-throughput targeted long-read single cell sequencing reveals the clonal and transcriptional landscape of lymphocytes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  3120
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/31311926">
                                  http://www.ncbi.nlm.nih.gov/pubmed/31311926
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%2B%255BInternet%255D%26rft.volume%253D10%26rft.spage%253D3120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.21" data-doi="10.1093/bioinformatics/bty191">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Minimap2: pairwise alignment for nucleotide sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  3094
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  100
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29750242&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F08%2F2020.02.08.939942.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.08.939942v5.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tardaguila
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de la Fuente
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marti
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pereira
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pardo-Palacios
                                  </span>
                                  <span class="cit-name-given-names">
                                   FJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Del Risco
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SQANTI: extensive characterization of long-read transcript sequences for quality control in full-length transcriptome identification and quantification
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Genome Res. [Internet]
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ; Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29440222">
                                  http://www.ncbi.nlm.nih.gov/pubmed/29440222
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.08.939942v5.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wyman
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Balderrama-Gutierrez
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reese
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rahmanian
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zeng
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A technology-agnostic long-read analysis pipeline for transcriptome discovery and quantification
                                 </span>
                                 .
                                 <span class="cit-source">
                                  bioRxiv [Internet]
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-fpage">
                                  672931
                                 </span>
                                 . Available from:
                                 <a href="http://biorxiv.org/content/early/2019/06/18/672931.abstract">
                                  http://biorxiv.org/content/early/2019/06/18/672931.abstract
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nip
                                  </span>
                                  <span class="cit-name-given-names">
                                   KM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chiu
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chu
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mohamadi
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Warren
                                  </span>
                                  <span class="cit-name-given-names">
                                   RL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  RNA-Bloom enables reference-free and reference-guided sequence assembly for single-cell transcriptomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1191
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  200
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/32817073">
                                  http://www.ncbi.nlm.nih.gov/pubmed/32817073
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%2B%255BInternet%255D%26rft.volume%253D30%26rft.spage%253D1191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trincado
                                  </span>
                                  <span class="cit-name-given-names">
                                   JL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Entizne
                                  </span>
                                  <span class="cit-name-given-names">
                                   JC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hysenaj
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Skalic
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elliott
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SUPPA2: Fast, accurate, and uncertainty-aware differential splicing analysis across multiple conditions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lebrigand
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Magnone
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barbry
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Waldmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  High throughput error corrected Nanopore single cell transcriptome sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  4025
                                 </span>
                                 . Available from:
                                 <a href="http://biorxiv.org/content/early/2019/11/05/831495.abstract">
                                  http://biorxiv.org/content/early/2019/11/05/831495.abstract
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%2B%255BInternet%255D%26rft.volume%253D11%26rft.spage%253D4025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.08.939942v5.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Acera Mateos
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgio
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Preiss
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shirokikh
                                  </span>
                                  <span class="cit-name-given-names">
                                   NE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eyras
                                  </span>
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Identification of m6A and m5C RNA modifications at single-molecule resolution from Nanopore sequencing [PREPRINT]
                                 </span>
                                 .
                                 <span class="cit-source">
                                  BIORXIV/2022/484124 [Internet]
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ; Available from:
                                 <a href="https://doi.org/10.1101/2022.03.14.484124">
                                  https://doi.org/10.1101/2022.03.14.484124
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruiz-Reche
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Srivastava
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Indi
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de la Rubia
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eyras
                                  </span>
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  ReorientExpress: reference-free orientation of nanopore cDNA reads with deep learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  260
                                 </span>
                                 . Available from:
                                 <a href="http://biorxiv.org/content/early/2019/02/18/553321.abstract">
                                  http://biorxiv.org/content/early/2019/02/18/553321.abstract
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2B%255BInternet%255D%26rft.volume%253D20%26rft.spage%253D260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grasso
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharlow
                                  </span>
                                  <span class="cit-name-given-names">
                                   MF
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Multiple sequence alignment using partial order graphs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  452
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  64
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/11934745">
                                  http://www.ncbi.nlm.nih.gov/pubmed/11934745
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%255BInternet%255D%26rft.volume%253D18%26rft.spage%253D452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.30" data-doi="org/10.1093/bioinformatics/bty223">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bi
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  DeepSimulator: a deep simulator for Nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2899
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  908
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D2899%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1093%252Fbioinformatics%252Fbty223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=org/10.1093/bioinformatics/bty223&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fu
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niu
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  CD-HIT: accelerated for clustering the next-generation sequencing data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  3150
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610">
                                  http://www.ncbi.nlm.nih.gov/pubmed/23060610
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%255BInternet%255D%26rft.volume%253D28%26rft.spage%253D3150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vinh
                                  </span>
                                  <span class="cit-name-given-names">
                                   NX
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Epps
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bailey
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Information theoretic measures for clusterings comparison: Variants, properties, normalization and correction for chance
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Mach. Learn. Res
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2837
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  54
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes%26rft.volume%253D11%26rft.spage%253D2837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hirschberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  V-measure: A conditional entropy-based external cluster evaluation measure
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. 2007 Jt. Conf. Empir. methods Nat. Lang. Process. Comput. Nat. Lang. Learn
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 . p.
                                 <span class="cit-fpage">
                                  410
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  20
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.08.939942v5.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrett
                                  </span>
                                  <span class="cit-name-given-names">
                                   CL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   DeBoever
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jepsen
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saenz
                                  </span>
                                  <span class="cit-name-given-names">
                                   CC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carson
                                  </span>
                                  <span class="cit-name-given-names">
                                   DA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frazer
                                  </span>
                                  <span class="cit-name-given-names">
                                   KA
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Systematic transcriptome analysis reveals tumor-specific isoforms for ovarian cancer diagnosis and therapy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U. S. A. [Internet]
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  E3050
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 . Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26015570">
                                  http://www.ncbi.nlm.nih.gov/pubmed/26015570
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA.%2B%255BInternet%255D%26rft.volume%253D112%26rft.spage%253D3050E%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.08.939942v5.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gleeson
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leger
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prawer
                                  </span>
                                  <span class="cit-name-given-names">
                                   YDJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lane
                                  </span>
                                  <span class="cit-name-given-names">
                                   TA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harrison
                                  </span>
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haerty
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Accurate expression quantification from nanopore direct RNA sequencing with NanoCount
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Nucleic Acids Res. [Internet]
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ; Available from:
                                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/34850115">
                                  http://www.ncbi.nlm.nih.gov/pubmed/34850115
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.08.939942v5.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pratanwanich
                                  </span>
                                  <span class="cit-name-given-names">
                                   PN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koh
                                  </span>
                                  <span class="cit-name-given-names">
                                   CWQ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wan
                                  </span>
                                  <span class="cit-name-given-names">
                                   YK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hendra
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Identification of differential RNA modifications from nanopore direct RNA sequencing with xPore
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Nat. Biotechnol. [Internet]
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;2020.06.18.160010-2020.06.18.160010. Available from:
                                 <a href="http://biorxiv.org/content/early/2020/06/20/2020.06.18.160010.abstract">
                                  http://biorxiv.org/content/early/2020/06/20/2020.06.18.160010.abstract
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.08.939942v5" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.04.06.487011v1" title="Discovery and Visualization of Age-dependent Patterns in the Diurnal Transcriptome of Drosophila" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2020.06.08.140939v3" title="Rhythmic coordination of hippocampal-prefrontal ensembles for odor-place associative memory and decision making" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted April 08, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.02.08.939942v5.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.02.08.939942v5.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/04/08/2020.02.08.939942.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.02.08.939942v5.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.02.08.939942v5" class="active" data-icon-position="" data-hide-link-title="0">
                                   RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2488778"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-o3rgFvBzdGQJwBO9u6IKH-JOEZTPDsuXOKUpKqqLA0U"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484467048"/>
                                   <input type="hidden" name="captcha_token" value="b3a6b60b8314fe69dc2c81976c57566f"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2488778--3" data-node-nid="2488778" id="node-2488778--41306784256" data-pisa="biorxiv;2020.02.08.939942v5" data-pisa-master="biorxiv;2020.02.08.939942" data-seqnum="2488778" data-apath="/biorxiv/early/2022/04/08/2020.02.08.939942.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Ivan
                                  </span>
                                  <span class="nlm-surname">
                                   de la Rubia
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Akanksha
                                  </span>
                                  <span class="nlm-surname">
                                   Srivastava
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Wenjing
                                  </span>
                                  <span class="nlm-surname">
                                   Xue
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Joel A
                                  </span>
                                  <span class="nlm-surname">
                                   Indi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Silvia
                                  </span>
                                  <span class="nlm-surname">
                                   Carbonell-Sala
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Julien
                                  </span>
                                  <span class="nlm-surname">
                                   Lagarde
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   M Mar
                                  </span>
                                  <span class="nlm-surname">
                                   Albà
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Eduardo
                                  </span>
                                  <span class="nlm-surname">
                                   Eyras
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.02.08.939942;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.02.08.939942
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.08.939942v5" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&title=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&title=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&text=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&t=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&title=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&title=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&title=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2488778--3" data-node-nid="2488778" id="citation-node-2488778--41219978523" data-pisa="biorxiv;2020.02.08.939942v5" data-pisa-master="biorxiv;2020.02.08.939942" data-seqnum="2488778" data-apath="/biorxiv/early/2022/04/08/2020.02.08.939942.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     RATTLE: Reference-free reconstruction and quantification of transcriptomes from Nanopore sequencing
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Ivan
                                      </span>
                                      <span class="nlm-surname">
                                       de la Rubia
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Akanksha
                                      </span>
                                      <span class="nlm-surname">
                                       Srivastava
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Wenjing
                                      </span>
                                      <span class="nlm-surname">
                                       Xue
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Joel A
                                      </span>
                                      <span class="nlm-surname">
                                       Indi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Silvia
                                      </span>
                                      <span class="nlm-surname">
                                       Carbonell-Sala
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Julien
                                      </span>
                                      <span class="nlm-surname">
                                       Lagarde
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       M Mar
                                      </span>
                                      <span class="nlm-surname">
                                       Albà
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Eduardo
                                      </span>
                                      <span class="nlm-surname">
                                       Eyras
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.02.08.939942;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.02.08.939942
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=de+la+Rubia%2C+Ivan&rft.au=Srivastava%2C+Akanksha&rft.au=Xue%2C+Wenjing&rft.au=Indi%2C+Joel+A&rft.au=Carbonell-Sala%2C+Silvia&rft.au=Lagarde%2C+Julien&rft.au=Alb%C3%A0%2C+M+Mar&rft.au=Eyras%2C+Eduardo">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2488778/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2488778/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2488778/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2488778/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2488778/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2488778/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2488778/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2488778/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2488778/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2488778/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2488778/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2488778/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&count=horizontal&via=&text=RATTLE%3A%20Reference-free%20reconstruction%20and%20quantification%20of%20transcriptomes%20from%20Nanopore%20sequencing&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.08.939942v5&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.02.08.939942v5" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
