<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.10.14.339804v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.10.14.339804v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.10.14.339804v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/embed/graphic-3.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/14/2020.10.14.339804.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.10.14.339804v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Higher-order epistasis and phenotypic prediction"/>
  <meta name="DC.Identifier" content="10.1101/2020.10.14.339804"/>
  <meta name="DC.Date" content="2022-03-14"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Contemporary high-throughput mutagenesis experiments are providing an increasingly detailed view of the complex patterns of genetic interaction that occur between multiple mutations within a single protein or regulatory element. By simultaneously measuring the effects of thousands of combinations of mutations, these experiments have revealed that the genotype-phenotype relationship typically reflects genetic interactions not only between pairs of sites, but also higher-order interactions among larger numbers of sites. However, modeling and understanding these higher-order interactions remains challenging. Here, we present a method for reconstructing sequence-to-function mappings from partially observed data that can accommodate all orders of genetic interaction. The main idea is to make predictions for unobserved genotypes that match the type and extent of epistasis found in the observed data. This information on the type and extent of epistasis can be extracted by considering how phenotypic correlations change as a function of mutational distance, which is equivalent to estimating the fraction of phenotypic variance due to each order of genetic interaction (additive, pairwise, three-way, etc.). Using these estimated variance components, we then define an empirical Bayes prior that in expectation matches the observed pattern of epistasis, and reconstruct the genotype-phenotype mapping by conducting Gaussian process regression under this prior. To demonstrate the power of this approach, we present an application to the antibody-binding domain GB1 and also provide a detailed exploration of a dataset consisting of high-throughput measurements for the splicing efficiency of human pre-mRNA 5 ′ splice sites, for which we also validate our model predictions via additional low-throughput experiments.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Juannan Zhou"/>
  <meta name="DC.Contributor" content="Mandy S. Wong"/>
  <meta name="DC.Contributor" content="Wei-Chia Chen"/>
  <meta name="DC.Contributor" content="Adrian R. Krainer"/>
  <meta name="DC.Contributor" content="Justin B. Kinney"/>
  <meta name="DC.Contributor" content="David M. McCandlish"/>
  <meta name="article:published_time" content="2022-03-14"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Higher-order epistasis and phenotypic prediction"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Contemporary high-throughput mutagenesis experiments are providing an increasingly detailed view of the complex patterns of genetic interaction that occur between multiple mutations within a single protein or regulatory element. By simultaneously measuring the effects of thousands of combinations of mutations, these experiments have revealed that the genotype-phenotype relationship typically reflects genetic interactions not only between pairs of sites, but also higher-order interactions among larger numbers of sites. However, modeling and understanding these higher-order interactions remains challenging. Here, we present a method for reconstructing sequence-to-function mappings from partially observed data that can accommodate all orders of genetic interaction. The main idea is to make predictions for unobserved genotypes that match the type and extent of epistasis found in the observed data. This information on the type and extent of epistasis can be extracted by considering how phenotypic correlations change as a function of mutational distance, which is equivalent to estimating the fraction of phenotypic variance due to each order of genetic interaction (additive, pairwise, three-way, etc.). Using these estimated variance components, we then define an empirical Bayes prior that in expectation matches the observed pattern of epistasis, and reconstruct the genotype-phenotype mapping by conducting Gaussian process regression under this prior. To demonstrate the power of this approach, we present an application to the antibody-binding domain GB1 and also provide a detailed exploration of a dataset consisting of high-throughput measurements for the splicing efficiency of human pre-mRNA <b>5</b>′ splice sites, for which we also validate our model predictions via additional low-throughput experiments.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.10.14.339804v3"/>
  <meta name="citation_id" content="2020.10.14.339804v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.10.14.339804v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.14.339804v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.14.339804v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.10.14.339804"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.10.14.339804"/>
  <meta name="citation_author" content="Juannan Zhou"/>
  <meta name="citation_author_institution" content="Department of Biology, University of Florida"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1373-4746"/>
  <meta name="citation_author" content="Mandy S. Wong"/>
  <meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor"/>
  <meta name="citation_author" content="Wei-Chia Chen"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author" content="Adrian R. Krainer"/>
  <meta name="citation_author_institution" content="Cold Spring Harbor Laboratory, Cold Spring Harbor"/>
  <meta name="citation_author" content="Justin B. Kinney"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1897-3778"/>
  <meta name="citation_author" content="David M. McCandlish"/>
  <meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory"/>
  <meta name="citation_author_email" content="mccandlish@cshl.edu"/>
  <meta name="citation_reference" content="Patrick C Phillips. “Epistasis—the essential role of gene interactions in the structure and evolution of genetic systems”. In: Nat. Rev. Genet. 9.11 (2008), pp. 855–867."/>
  <meta name="citation_reference" content="Dmitry A Kondrashov and Fyodor A Kondrashov. “Topological features of rugged fitness landscapes in sequence space”. In: Trends Genet. 31.1 (2015), pp. 24–33."/>
  <meta name="citation_reference" content="Júlia Domingo, Pablo Baeza-Centurion, and Ben Lehner. “The Causes and Consequences of Genetic Interactions (Epistasis)”. In: Annu. Rev. Genomics Hum. Genet. 20 (2019)."/>
  <meta name="citation_reference" content="Douglas M Fowler et al. “High-resolution mapping of protein sequence-function relationships”. In: Nat. Methods 7.9 (2010), pp. 741–746."/>
  <meta name="citation_reference" content="Lea M Starita et al. “Activity-enhancing mutations in an E3 ubiquitin ligase identified by high-throughput mutagenesis”. In: Proc. Natl. Acad. Sci. U.S.A. 110.14 (2013), E1263–E1272."/>
  <meta name="citation_reference" content="Daniel Melamed et al. “Deep mutational scanning of an RRM domain of the Saccharomyces cerevisiae poly (A)-binding protein”. In: RNA 19.11 (2013), pp. 1537–1551."/>
  <meta name="citation_reference" content="C Anders Olson, Nicholas C Wu, and Ren Sun. “A comprehensive biophysical description of pairwise epistasis throughout an entire protein domain”. In: Curr. Biol. 24.22 (2014), pp. 2643–2651."/>
  <meta name="citation_reference" content="Michael B Doud, Orr Ashenberg, and Jesse D Bloom. “Site-specific amino acid preferences are mostly conserved in two closely related protein homologs”. In: Mol. Biol. Evol. 32.11 (2015), pp. 2944–2960."/>
  <meta name="citation_reference" content="Anna I Podgornaia and Michael T Laub. “Pervasive degeneracy and epistasis in a protein-protein interface”. In: Science 347.6222 (2015), pp. 673–677."/>
  <meta name="citation_reference" content="Karen S Sarkisyan et al. “Local fitness landscape of the green fluorescent protein”. In: Nature 533.7603 (2016), p. 397."/>
  <meta name="citation_reference" content="Barrett Steinberg and Marc Ostermeier. “Shifting fitness and epistatic landscapes reflect trade-offs along an evolutionary pathway”. In: J Mol Biol. 428.13 (2016), pp. 2730–2743."/>
  <meta name="citation_reference" content="Claudia Bank et al. “On the (un)predictability of a large intragenic fitness landscape”. In: Proc. Natl. Acad. Sci. U.S.A. 113.49 (2016), pp. 14085–14090."/>
  <meta name="citation_reference" content="Tyler N Starr, Lora K Picton, and Joseph W Thornton. “Alternative evolutionary histories in the sequence space of an ancient protein.” In: Nature 549.7672 (Sept. 2017), pp. 409–413."/>
  <meta name="citation_reference" content="Victoria O Pokusaeva et al. “An experimental assay of the interactions of amino acids from orthologous sequences shaping a complex fitness landscape”. In: PLos Genet. 15.4 (2019), e1008079."/>
  <meta name="citation_reference" content="Calin Plesa et al. “Multiplexed gene synthesis in emulsions for exploring protein functional landscapes”. In: Science 359.6373 (2018), pp. 343–347."/>
  <meta name="citation_reference" content="Drew S Tack et al. “The genotype-phenotype landscape of an allosteric protein”. In: Molecular systems biology 17.3 (2021), e10179."/>
  <meta name="citation_reference" content="Tyler N Starr et al. “Deep mutational scanning of SARS-CoV-2 receptor binding domain reveals constraints on folding and ACE2 binding”. In: Cell 182.5 (2020), pp. 1295–1310."/>
  <meta name="citation_reference" content="Louisa Gonzalez Somermeyer et al. “Heterogeneity of the GFP fitness landscape and data-driven protein design”. In: bioRxiv (2021). doi: 10.1101/2021.12.08.471728. eprint: https://www.biorxiv.org/content/early/2021/12/09/2021.12.08.471728.full.pdf. url: https://www.biorxiv.org/content/early/2021/12/09/2021.12.08.471728."/>
  <meta name="citation_reference" content="J N Pitt and A R Ferré-D’Amaré. “Rapid Construction of Empirical RNA Fitness Landscapes”. In: Science 330.6002 (2010), pp. 376–379. doi: 10.1126/science.1192001. url: http://www.sciencemag.org/content/330/6002/376.abstract."/>
  <meta name="citation_reference" content="José I Jiménez et al. “Comprehensive experimental fitness landscape and evolutionary network for small RNA”. In: Proc. Natl. Acad. Sci. U.S.A. 110.37 (2013), pp. 14984–14989."/>
  <meta name="citation_reference" content="Olga Puchta et al. “Network of epistatic interactions within a yeast snoRNA”. In: Science 352.6287 (2016), pp. 840–844."/>
  <meta name="citation_reference" content="Chuan Li et al. “The fitness landscape of a tRNA gene”. In: Science 352.6287 (2016), pp. 837–840."/>
  <meta name="citation_reference" content="Júlia Domingo, Guillaume Diss, and Ben Lehner. “Pairwise and higher-order genetic interactions during the evolution of a tRNA”. In: Nature 558.7708 (2018), p. 117."/>
  <meta name="citation_reference" content="Valerie WC Soo et al. “Fitness landscape of a dynamic RNA structure”. In: PLoS genetics 17.2 (2021), e1009353."/>
  <meta name="citation_reference" content="Rupali P Patwardhan et al. “High-resolution analysis of DNA regulatory elements by synthetic saturation mutagenesis”. In: Nature Biotechnology 27.12 (2009), pp. 1173–1175."/>
  <meta name="citation_reference" content="Justin B Kinney et al. “Using deep sequencing to characterize the biophysical mechanism of a transcriptional regulatory sequence”. In: Proc. Natl. Acad. Sci. U.S.A. 107.20 (2010), pp. 9158–9163."/>
  <meta name="citation_reference" content="Shengdong Ke et al. “Quantitative evaluation of all hexamers as exonic splicing elements”. In: Genome research 21.8 (2011), pp. 1360–1374."/>
  <meta name="citation_reference" content="Alexander B Rosenberg et al. “Learning the sequence determinants of alternative splicing from millions of random sequences”. In: Cell 163.3 (2015), pp. 698–711."/>
  <meta name="citation_reference" content="Philippe Julien et al. “The complete local genotype–phenotype landscape for the alternative splicing of a human exon”. In: Nat. Commun. 7 (2016), p. 11558."/>
  <meta name="citation_reference" content="Shengdong Ke et al. “Saturation mutagenesis reveals manifold determinants of exon definition”. In: Genome Res. 28.1 (2018), pp. 11–24."/>
  <meta name="citation_reference" content="Mandy S Wong, Justin B Kinney, and Adrian R Krainer. “Quantitative Activity Profile and Context Dependence of All Human 5′ Splice Sites”. In: Mol. Cell (2018)."/>
  <meta name="citation_reference" content="Daniel M Weinreich et al. “Should evolutionary geneticists worry about higher-order epistasis?” In: Curr. Opin. Genet. Dev. 23.6 (2013), pp. 700–707."/>
  <meta name="citation_reference" content="Johannes Neidhart, Ivan G Szendro, and Joachim Krug. “Exact results for amplitude spectra of fitness landscapes”. In: J. Theor. Biol. 332 (2013), pp. 218–227."/>
  <meta name="citation_reference" content="Tyler N Starr and Joseph W Thornton. “Epistasis in protein evolution”. In: Protein Sci. 25.7 (2016), pp. 1204–1218."/>
  <meta name="citation_reference" content="Zachary R Sailer and Michael J Harms. “Detecting High-Order Epistasis in Nonlinear Genotype-Phenotype Maps”. In: Genetics 205.3 (2017), pp. 1079–1088."/>
  <meta name="citation_reference" content="Zachary R Sailer and Michael J Harms. “High-order epistasis shapes evolutionary trajectories”. In: PLoS Comput. Biol. 13.5 (2017), e1005541."/>
  <meta name="citation_reference" content="Nicholas Wu et al. “Adaptation in protein fitness landscapes is facilitated by indirect paths”. In: eLife 5 (2016), e16965."/>
  <meta name="citation_reference" content="Julian Echave and Claus O Wilke. “Biophysical models of protein evolution: understanding the patterns of evolutionary sequence divergence”. In: Annu. Rev. Biophys. 46 (2017), pp. 85–103."/>
  <meta name="citation_reference" content="Frank J Poelwijk, Michael Socolich, and Rama Ranganathan. “Learning the pattern of epistasis linking genotype and phenotype in a protein”. In: Nat. Commun. 10.1 (2019), pp. 1–11."/>
  <meta name="citation_reference" content="Aneth S Canale et al. “Evolutionary mechanisms studied through protein fitness landscapes”. In: Curr. Opin. Struct. Biol. 48 (2018), pp. 141–148."/>
  <meta name="citation_reference" content="Daniel M. Weinreich et al. “The Influence of Higher-Order Epistasis on Biological Fitness Landscape Topography”. In: J. Stat. Phys. 172.1 (2018), pp. 208–225. issn: 00224715. doi: 10.1007/s10955-018-1975-3. url: https://doi.org/10.1007/s10955-018-1975-3."/>
  <meta name="citation_reference" content="Jay F Storz. “Compensatory mutations and epistasis for protein function”. In: Curr. Opin. Struct. Biol. 50 (2018), pp. 18–25."/>
  <meta name="citation_reference" content="Charlotte M Miton, Karol Buda, and Nobuhiko Tokuriki. “Epistasis and intramolecular networks in protein evolution”. In: Current opinion in structural biology 69 (2021), pp. 160–168."/>
  <meta name="citation_reference" content="Gloria Yang et al. “Higher-order epistasis shapes the fitness landscape of a xenobiotic-degrading enzyme”. In: Nature Chemical Biology 15.11 (2019), pp. 1120–1128."/>
  <meta name="citation_reference" content="Jakub Otwinowski, David Martin McCandlish, and Joshua B Plotkin. “Inferring the shape of global epistasis”. In: Proceedings of the National Academy of Sciences 115.32 (2018), E7550–E7558."/>
  <meta name="citation_reference" content="Gautam Reddy and Michael M Desai. “Global epistasis emerges from a generic model of a complex trait”. In: Elife 10 (2021), e64740."/>
  <meta name="citation_reference" content="Carl Edward Rasmussen and Christopher K I Williams. Gaussian processes for machine learning. Vol. 1. MIT press Cambridge, 2006."/>
  <meta name="citation_reference" content="Bradley P Carlin and Thomas A Louis. Bayes and empirical Bayes methods for data analysis. Vol. 88. Chapman & Hall/CRC Boca Raton, 2000."/>
  <meta name="citation_reference" content="citation_journal_title=Handbook of Markov chain Monte Carlo;citation_pages=113-162;citation_year=2011"/>
  <meta name="citation_reference" content="R A Fisher. “The Correlation Between Relatives on the Supposition of Mendelian Inheritance”. In: Trans R Soc Edinburgh 52.02 (1918), pp. 399–433."/>
  <meta name="citation_reference" content="Trevor Hinkley et al. “A systems analysis of mutational effects in HIV-1 protease and reverse transcriptase”. In: Nature Genetics 43.5 (2011), pp. 487–489."/>
  <meta name="citation_reference" content="Manfred Eigen, John McCaskill, and Peter Schuster. “The molecular quasi-species”. In: Adv. Chem. Phys 75 (1989), pp. 149–263."/>
  <meta name="citation_reference" content="Peter F Stadler and Robert Happel. “Random field models for fitness landscapes”. In: J. Math. Biol. 38.5 (1999), pp. 435–478."/>
  <meta name="citation_reference" content="Edward D Weinberger. “Fourier and Taylor series on fitness landscapes”. In: Biol Cybern 65.5 (1991), pp. 321–330."/>
  <meta name="citation_reference" content="Luca Ferretti et al. “Measuring epistasis in fitness landscapes: The correlation of fitness effects of mutations”. In: Journal of Theoretical Biology 396 (2016), pp. 132–143. issn: 10958541. doi: 10.1016/j.jtbi.2016.01.037."/>
  <meta name="citation_reference" content="Robert Happel and Peter F Stadler. “Canonical approximation of fitness landscapes”. In: Complexity 2.1 (1996), pp. 53–58."/>
  <meta name="citation_reference" content="Peter F Stadler. “Landscapes and their correlation functions”. In: J. Math. Chem. 20.1 (1996), pp. 1–45."/>
  <meta name="citation_reference" content="Peter F Stadler. “Fitness landscapes”. In: Biological Evolution and Statistical Physics. Springer, 2002, pp. 183–204."/>
  <meta name="citation_reference" content="Juannan Zhou and David M McCandlish. “Minimum epistasis interpolation for sequence-function relationships”. In: Nature communications 11.1 (2020), pp. 1–14."/>
  <meta name="citation_reference" content="Tinghua Wang, Dongyan Zhao, and Shengfeng Tian. “An overview of kernel alignment and its applications”. In: Artificial Intelligence Review 43.2 (2015), pp. 179–192."/>
  <meta name="citation_reference" content="John P Cunningham, Krishna V Shenoy, and Maneesh Sahani. “Fast Gaussian process methods for point process intensity estimation”. In: Proceedings of the 25th international conference on Machine learning. 2008, pp. 192–199."/>
  <meta name="citation_reference" content="Peter F Stadler. “Random walks and orthogonal functions associated with highly symmetric graphs”. In: Discrete mathematics 145.1-3 (1995), pp. 229–237."/>
  <meta name="citation_reference" content="DG Higman. “Intersection matrices for finite permutation groups”. In: Journal of Algebra 6 (1967), pp. 22–42."/>
  <meta name="citation_reference" content="Daniel M Weinreich et al. “The influence of higher-order epistasis on biological fitness landscape topography”. In: Journal of Statistical Physics 172.1 (2018), pp. 208–225."/>
  <meta name="citation_reference" content="Amirali Aghazadeh et al. “Epistatic Net allows the sparse spectral regularization of deep neural networks for inferring fitness functions”. In: Nature communications 12.1 (2021), pp. 1–10."/>
  <meta name="citation_reference" content="David H Brookes, Amirali Aghazadeh, and Jennifer Listgarten. “On the sparsity of fitness functions and implications for learning”. In: Proceedings of the National Academy of Sciences 119.1 (2022)."/>
  <meta name="citation_reference" content="Hui Zou and Trevor Hastie. “Regularization and variable selection via the elastic net”. In: Journal of the royal statistica 67.2 (2005), pp. 301–320."/>
  <meta name="citation_reference" content="Yasushi Kondo et al. “Crystal structure of human U1 snRNP, a small nuclear ribonucleoprotein particle, reveals the mechanism of 5′splice site recognition”. In: Elife 4 (2015), e04986."/>
  <meta name="citation_reference" content="David M McCandlish. “Visualizing fitness landscapes”. In: Evolution 65.6 (2011), pp. 1544–1558."/>
  <meta name="citation_reference" content="Ronald R Coifman and Stéphane Lafon. “Diffusion maps”. In: Appl. and Comp. Harmonic Anal. 21.1 (2006), pp. 5–30."/>
  <meta name="citation_reference" content="David M McCandlish. “Long-term evolution on complex fitness landscapes when mutation is weak”. In: Heredity 121.5 (2018), pp. 449–465."/>
  <meta name="citation_reference" content="Xavier Roca et al. “Features of 5′-splice-site efficiency derived from disease-causing mutations and comparative genomics”. In: Genome research 18.1 (2008), pp. 77–87."/>
  <meta name="citation_reference" content="Ido Carmel et al. “Comparative analysis detects dependencies among the 5′splice-site positions”. In: RNA 10.5 (2004), pp. 828–840."/>
  <meta name="citation_reference" content="Chris Burge and Samuel Karlin. “Prediction of complete gene structures in human genomic DNA”. In: Journal of Molecular Biology 268.1 (1997), pp. 78–94."/>
  <meta name="citation_reference" content="Gene Yeo and Christopher B Burge. “Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals”. In: Journal of computational biology 11.2-3 (2004), pp. 377–394."/>
  <meta name="citation_reference" content="Jiazi Tan et al. “Noncanonical registers and base pairs in human 5′splice-site selection”. In: Nucleic acids research 44.8 (2016), pp. 3908–3921."/>
  <meta name="citation_reference" content="Philip A Romero, Andreas Krause, and Frances H Arnold. “Navigating the protein fitness landscape with Gaussian processes”. In: Proc. Natl. Acad. Sci. U.S.A. 110.3 (2013),E193–E201."/>
  <meta name="citation_reference" content="Alexey S Kondrashov. “Selection against harmful mutations in large sexual and asexual populations”. In: Genetics Research 40.3 (1982), pp. 325–332."/>
  <meta name="citation_reference" content="Santiago F Elena and Richard E Lenski. “Test of synergistic interactions among deleterious mutations in bacteria”. In: Nature 390.6658 (1997), pp. 395–398."/>
  <meta name="citation_reference" content="Sebastian Bonhoeffer et al. “Evidence for positive epistasis in HIV-1”. In: Science 306.5701 (2004), pp. 1547–1550."/>
  <meta name="citation_reference" content="Shimon Bershtein et al. “Robustness–epistasis link shapes the fitness landscape of a randomly drifting protein”. In: Nature 444.7121 (2006), pp. 929–932."/>
  <meta name="citation_reference" content="Alexander J Smola and Risi Kondor. “Kernels and regularization on graphs”. In: COLT. Vol. 2777. Springer. 2003, pp. 144–158."/>
  <meta name="citation_reference" content="Wei-Chia Chen et al. “Field-theoretic density estimation for biological sequence space with applications to 5 splice site diversity and aneuploidy in cancer”. In: Proceedings of the National Academy of Sciences 118.40 (2021)."/>
  <meta name="citation_reference" content="Justin B Kinney and David M McCandlish. “Massively Parallel Assays and Quantitative Sequence– Function Relationships”. In: Annu Rev Genomics Hum Genet 20 (2019)."/>
  <meta name="citation_reference" content="Karen S Sarkisyan et al. “Local fitness landscape of the green fluorescent protein”. In: Nature 533.7603 (2016), pp. 397–401."/>
  <meta name="citation_reference" content="Ammar Tareen et al. “MAVE-NN: Quantitative Modeling of Genotype-Phenotype Maps as Information Bottlenecks”. In: BioRxiv (2020)."/>
  <meta name="citation_reference" content="Bernhard Schölkopf, Ralf Herbrich, and Alex J Smola. “A generalized representer theorem”. In: International conference on computational learning theory. Springer. 2001, pp. 416–426."/>
  <meta name="twitter:title" content="Higher-order epistasis and phenotypic prediction"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Contemporary high-throughput mutagenesis experiments are providing an increasingly detailed view of the complex patterns of genetic interaction that occur between multiple mutations within a single protein or regulatory element. By simultaneously measuring the effects of thousands of combinations of mutations, these experiments have revealed that the genotype-phenotype relationship typically reflects genetic interactions not only between pairs of sites, but also higher-order interactions among larger numbers of sites. However, modeling and understanding these higher-order interactions remains challenging. Here, we present a method for reconstructing sequence-to-function mappings from partially observed data that can accommodate all orders of genetic interaction. The main idea is to make predictions for unobserved genotypes that match the type and extent of epistasis found in the observed data. This information on the type and extent of epistasis can be extracted by considering how phenotypic correlations change as a function of mutational distance, which is equivalent to estimating the fraction of phenotypic variance due to each order of genetic interaction (additive, pairwise, three-way, etc.). Using these estimated variance components, we then define an empirical Bayes prior that in expectation matches the observed pattern of epistasis, and reconstruct the genotype-phenotype mapping by conducting Gaussian process regression under this prior. To demonstrate the power of this approach, we present an application to the antibody-binding domain GB1 and also provide a detailed exploration of a dataset consisting of high-throughput measurements for the splicing efficiency of human pre-mRNA 5 ′ splice sites, for which we also validate our model predictions via additional low-throughput experiments.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Higher-order epistasis and phenotypic prediction"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.10.14.339804v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Contemporary high-throughput mutagenesis experiments are providing an increasingly detailed view of the complex patterns of genetic interaction that occur between multiple mutations within a single protein or regulatory element. By simultaneously measuring the effects of thousands of combinations of mutations, these experiments have revealed that the genotype-phenotype relationship typically reflects genetic interactions not only between pairs of sites, but also higher-order interactions among larger numbers of sites. However, modeling and understanding these higher-order interactions remains challenging. Here, we present a method for reconstructing sequence-to-function mappings from partially observed data that can accommodate all orders of genetic interaction. The main idea is to make predictions for unobserved genotypes that match the type and extent of epistasis found in the observed data. This information on the type and extent of epistasis can be extracted by considering how phenotypic correlations change as a function of mutational distance, which is equivalent to estimating the fraction of phenotypic variance due to each order of genetic interaction (additive, pairwise, three-way, etc.). Using these estimated variance components, we then define an empirical Bayes prior that in expectation matches the observed pattern of epistasis, and reconstruct the genotype-phenotype mapping by conducting Gaussian process regression under this prior. To demonstrate the power of this approach, we present an application to the antibody-binding domain GB1 and also provide a detailed exploration of a dataset consisting of high-throughput measurements for the splicing efficiency of human pre-mRNA 5 ′ splice sites, for which we also validate our model predictions via additional low-throughput experiments.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-14"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.10.14.339804v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2446672"/>
  <title>
   Higher-order epistasis and phenotypic prediction | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"xAxaf6a5YOGZL0DFrOvoI529J2PJCeOb90_Ct1Xe_xk"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2446672","apath":"\/biorxiv\/early\/2022\/03\/14\/2020.10.14.339804.atom","pisa":"biorxiv;2020.10.14.339804v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.10.14.339804v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.10.14.339804v3","title":"Higher-order epistasis and phenotypic prediction","identifier":"node\/2446672"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222dJ+9KvJ6MFQyLyuXm+DpMf8i\\\\\\\/0GoenH1gn3fGuXcc+mpydh94n\\\\\\\/YmQfHM05gMJmJGzeSvPofXxk7Udep38S3ICOSAaxRRPDP36dM3fDzL6DH6stCkdB9cznWF4TF173JixH5\\\\\\\/oWZh+nmXFztD4OaOaGoSYcw9ba5fPJmzre4wQWSDhGqgPv0w+iVohzthMN7px9q\\\\\\\/aLhq0ItMYS12QAopo\\\\\\\/trM6\\\\\\\/dy5U0vlpH+nGWVLIdjWvii7L9cMg0yMmqbvUwJ8DRYa5SstdyccV+76wQX0zXpWKpQJlRv8baSOzukzAVeO\\\\\\\/Rh5E311mDyVwAYlvImqfpiQB1zpJol4EIsrMU\\\\\\\/L6hubTuBDzX+P5gXSGIxtKzVSgYSzfw+0LWvYwVQFqvMJ72NLh1C3siTDPpZKUK9o+t\\\\\\\/50e0F\\\\\\\/HbxdKVrh8CTdldPSWwMy5BHmxDd3XC\\\\\\\/4cFnmgMmCWLd8TEyZvAeIVYfreTPsR8v214czD\\\\\\\/3Dfj3L5ja\\\\\\\/74Z6yzJYvwA64sIwGvSuJmbZXyIJRP9lKTJxU6OAXhXc23GAedrCppzTEqjvz2eAY+XtFMMoDxdw\\\\\\\/dOu1ACBGOzZeEM+OEK1d\\\\\\\/oUMocHxxxZ4fphJbmPXDMgCSLTkQYSNwn0LZWOaS\\\\\\\/RfCqfxuyNpE\\\\\\\/I\\\\\\\/TlUkXL\\\\\\\/9l9nWEzjmeyIfZEQhfX4rQFUCLLiap2IxHFJm6mKZ9irReB+I\\\\\\\/6\\\\\\\/06cHc6sSOjs8OmM5RmlVeTy0omRgwk4DXpakTGH51X5+\\\\\\\/hR12qRRH2HdGxyD88cfGpsv1lNGnv9Q9e\\\\\\\/EVQjP56xD3hQ9mz4mjGywrpxltJ\\\\\\\/\\\\\\\/l3EXOtJq19t7Es1P\\\\\\\/zhImYgiS90mK55yFkwESiz7PdV7J3glrsEDOSf4Oj0+HSynRpQXjoBXTogznv\\\\\\\/nWdECQTyjPzw5N8pa7VGvusbcTLwCbU8xfXsbQ0\\\\\\\/TG57jrnBaIp05oKhz9F4XJQMhWXFU9Wem0v7Bvp\\\\\\\/yOi+p0sLJbr3LIXBHV4Xgew25vvDsLOIeZMo7LHXrNfj9II3flSEf8KQztbLEOPCdV8d7RJwDgCpkA3H2Kn3\\\\\\\/MkOzsQ5O5qy9ZqmfcmqBsA+3we4DQRzMbo9H2nrHzDiKaONU+Yomv+wYmu\\\\\\\/eyIWhxgf22OMrVG2e8onOoeCy3LjRIciwMKqWK5m4Y1V4zGYyQKRrKr0ZZy6JgEeCAA8f5HvUYXp88Vp0O9kEkDLECLwK2XE743kT\\\\\\\/E+hcn1bKcq81NLQ76xYU+uNC7uuNsFd+7Ubf0uQtG2BvC\\\\\\\/wFnKDpGENY9AfkD354DzreYU9pu4rrL1Gc9FBWj3XSmsje9+BAezRCK\\\\\\\/w7R1tqq2+AfmQwq5TW2vwySVGI85mY4aNQkrh1Og1B66s1mTKlB8IlV1VrhkCCg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022blSoh9QcvjxdbSKgWklayA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022d8e41d80efc2cda7ae5eabb23bc6f0b6c6db3476a5d736053d18502ff65f4c75\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.10.14.339804v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2446672 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.10.14.339804v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_288667374">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_288667374" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1159701474" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-dSV0THCsrYgpZTNO407bYLx6oSLH-hrtMZFWQ64Z1vg"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2446672" data-node-nid="2446672" id="node-2446672--21100043830" data-pisa="biorxiv;2020.10.14.339804v3" data-pisa-master="biorxiv;2020.10.14.339804" data-apath="/biorxiv/early/2022/03/14/2020.10.14.339804.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Higher-order epistasis and phenotypic prediction
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-1373-4746" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Juannan
                      </span>
                      <span class="nlm-surname">
                       Zhou
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Mandy S.
                      </span>
                      <span class="nlm-surname">
                       Wong
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Wei-Chia
                      </span>
                      <span class="nlm-surname">
                       Chen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Adrian R.
                      </span>
                      <span class="nlm-surname">
                       Krainer
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0003-1897-3778" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Justin B.
                      </span>
                      <span class="nlm-surname">
                       Kinney
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       David M.
                      </span>
                      <span class="nlm-surname">
                       McCandlish
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.10.14.339804
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2446672--21100043830" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Juannan Zhou
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        a
                       </span>
                       <span class="nlm-institution">
                        Department of Biology, University of Florida
                       </span>
                       , Gainesville, FL, 32611
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        b
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Juannan%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhou%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJuannan%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-1373-4746" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Juannan Zhou
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Mandy S. Wong
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        c
                       </span>
                       <span class="nlm-institution">
                        Cold Spring Harbor Laboratory, Cold Spring Harbor
                       </span>
                       , NY 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mandy%2BS.%2BWong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wong%20MS&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMandy%2BS.%2BWong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Wei-Chia Chen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        b
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Wei-Chia%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Chen%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWei-Chia%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Adrian R. Krainer
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        c
                       </span>
                       <span class="nlm-institution">
                        Cold Spring Harbor Laboratory, Cold Spring Harbor
                       </span>
                       , NY 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Adrian%2BR.%2BKrainer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Krainer%20AR&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAdrian%2BR.%2BKrainer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Justin B. Kinney
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        b
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Justin%2BB.%2BKinney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kinney%20JB&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJustin%2BB.%2BKinney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-1897-3778" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Justin B. Kinney
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     David M. McCandlish
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        b
                       </span>
                       <span class="nlm-institution">
                        Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY 11724
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=David%2BM.%2BMcCandlish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=McCandlish%20DM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ADavid%2BM.%2BMcCandlish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:mccandlish@cshl.edu" class="" data-icon-position="" data-hide-link-title="0">
                        mccandlish@cshl.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.10.14.339804v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2446672" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2446672/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.14.339804v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2446672" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2446672/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.14.339804v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2446672" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2446672/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.14.339804v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2446672" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2446672/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.14.339804v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2446672" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2446672/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.14.339804v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2446672" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2446672/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.10.14.339804v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2446672" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2446672/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-7">
                             Contemporary high-throughput mutagenesis experiments are providing an increasingly detailed view of the complex patterns of genetic interaction that occur between multiple mutations within a single protein or regulatory element. By simultaneously measuring the effects of thousands of combinations of mutations, these experiments have revealed that the genotype-phenotype relationship typically reflects genetic interactions not only between pairs of sites, but also higher-order interactions among larger numbers of sites. However, modeling and understanding these higher-order interactions remains challenging. Here, we present a method for reconstructing sequence-to-function mappings from partially observed data that can accommodate all orders of genetic interaction. The main idea is to make predictions for unobserved genotypes that match the type and extent of epistasis found in the observed data. This information on the type and extent of epistasis can be extracted by considering how phenotypic correlations change as a function of mutational distance, which is equivalent to estimating the fraction of phenotypic variance due to each order of genetic interaction (additive, pairwise, three-way, etc.). Using these estimated variance components, we then define an empirical Bayes prior that in expectation matches the observed pattern of epistasis, and reconstruct the genotype-phenotype mapping by conducting Gaussian process regression under this prior. To demonstrate the power of this approach, we present an application to the antibody-binding domain GB1 and also provide a detailed exploration of a dataset consisting of high-throughput measurements for the splicing efficiency of human pre-mRNA
                             <strong>
                              5
                             </strong>
                             ′ splice sites, for which we also validate our model predictions via additional low-throughput experiments.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-10">
                             Understanding the relationship between genotype and phenotype is difficult because the effects of a mutation often depend on which other mutations are already present in the sequence, a phenomenon known as epistasis [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. Recent advances in high-throughput mutagenesis and phenotyping have for the first time provided a detailed view of these complex genetic interactions, by allowing phenotypic measurements for the effects of tens of thousands of combinations of mutations within individual proteins [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             –
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], RNAs [
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             –
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ], and regulatory or splicing elements [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             –
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ]. Importantly, it has now become clear that the data from these experiments cannot be captured by considering simple pairwise interactions, but rather higher-order genetic interactions between three, four, or even all sites within a functional element are empirically common [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             –
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              44
                             </a>
                             ] and indeed often expected based on first-principles biophysical considerations [
                             <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ,
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ,
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              36
                             </a>
                             ,
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              41
                             </a>
                             ,
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ,
                             <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                              46
                             </a>
                             ]. However, the enormous number of possible combinations of mutations makes these higher-order interactions both difficult to conceptualize and challenging to incorporate into predictive models.
                            </p>
                            <p id="p-11">
                             From a very basic perspective, data from combinatorial mutagenesis experiments provide us with observations of phenotypic values for individual genotypes, the effects of specific mutations on specific genetic backgrounds, epistatic coefficients between pairs of mutations on specific backgrounds, etc. The essential problem in modeling data like this then comes down to the question of how to combine these observed quantities to make phenotypic predictions for unobserved genotypes. That is, given that we have already seen the results of a specific mutation in several different genetic backgrounds, how should we combine these observations to predict its effect in a new background?
                            </p>
                            <p id="p-12">
                             Here, we provide an answer to this question based on the intuition that, when making these predictions, we should focus on the observed effects of mutations that are nearby in sequence space to the genetic background we are making a prediction for, rather than observations of mutational effects that are more distant. We do this by considering a key comprehensible aspect of higher-order epistasis, namely the decay in the predictability of mutational effects, epistatic coefficients of double mutants, and observed phenotypes, as one moves through sequence space. We show analytically that the shape of how this predictability decays as a function of distance is completely determined by the fraction of phenotypic variance due to each order of genetic interaction (additive, pair-wise, three-way, etc.). Thus, rather than conceptualizing higher-order epistasis in terms of innumerable interaction terms between larger and larger numbers of sites, we suggest that: (1) we can understand a great deal about higher-order epistasis by considering simple diagrams showing how the correlations between mutational effects, epistatic coefficients, etc. decay as a function of genetic distance; and (2) these same diagrams suggest a method for making phenotypic predictions by weighting our observations in terms of the degree of information they provide for mutations on a genetic background of interest.
                            </p>
                            <p id="p-13">
                             We implement these ideas in terms of a Gaussian process regression [
                             <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                              47
                             </a>
                             ] framework with an empirical Bayes [
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              48
                             </a>
                             ] prior. Specifically, we use the observed pattern of decay in phenotypic correlation as a function of genetic distance to estimate the fraction of variance due to each order of interaction in our observed data. We then use these point estimates of the variance components to construct a prior distribution over all possible sequence-to-function mappings, where the expected decay in the predictability of mutational effects in the prior matches that observed in the data. Finally, we conduct Bayesian inference under this prior, using Hamiltonian Monte Carlo [
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              49
                             </a>
                             ] to sample from the resulting high-dimensional posterior distribution. The end result is a procedure that automatically weights the contributions of our observations to our predictions in the manner suggested by the overall form of higher-order epistasis present in the data, while simultaneously accounting for the effects of measurement noise and quantifying the uncertainty in our predictions.
                            </p>
                            <p id="p-14">
                             To demonstrate the performance of this technique, we apply our method to two datasets. The first dataset is derived from a combinatorial mutagenesis experiment for protein G [
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              37
                             </a>
                             ], a streptococcal antibody-binding protein that has served as a model system for studies of the genotype-phenotype map in proteins. The second dataset consists of high-throughput measurements of the splicing efficiency of human 5′ splice sites [
                             <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ], which are RNA sequence elements crucial for the assembly of the spliceosome during pre-mRNA splicing. For this latter dataset, we also present low-throughput validation measurements for our model predictions, as well as a qualitative exploration of the complex patterns of epistasis in splicing efficiency observed in this system.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <p id="p-15">
                             Experimental observations of the genotype-phenotype map typically consist of measurements of phenotypic values for a subset of possible genotypes. From these observations, we can calculate a number of more derived quantities, such as mutational effects (i.e., the difference in phenotype between two mutationally adjacent genotypes) and double-mutant epistatic coefficients (i.e., the difference between the observed phenotype of a double mutant and its expected phenotype based on the sum of the single-mutant effects). The central question of phenotypic prediction is then deciding how to combine these various mutational effects, local epistatic coefficients and individual phenotypic values to produce accurate predictions for the phenotypic values of unmeasured genotypes.
                            </p>
                            <p id="p-16">
                             Different prediction methods typically reflect different overall strategies for how to combine these experimentally derived quantities. For example, when we fit an additive or non-epistatic model [
                             <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                              50
                             </a>
                             ], we are implementing a strategy based on the assumption that the phenotypic effects of observed mutations are the same regardless of the presence/absence of other mutations. Thus, fitting an additive model can be thought of as a generalization of the simple heuristic procedure of making predictions by: (1) averaging over all the times the effect of each possible point mutation is observed; and then (2) adding up these average effects to make a prediction for any given genotype. In a similar way, it is easy to show that while a pairwise interaction model [
                             <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                              51
                             </a>
                             ] allows the effects of individual mutations to vary across genetic backgrounds, the epistatic interaction observed in double mutants for any specific pair of mutations is constant across backgrounds (see SI Appendix). Thus, fitting a pairwise model is conceptually closely related to the heuristic of determining the interaction between a pair of mutations by averaging over the local epistatic coefficients for this pair of mutations that are observed in the data and then assuming that this pair of mutations has the same interaction regardless of what genetic background these mutations occur on.
                            </p>
                            <p id="p-17">
                             Putting the underlying strategies of additive and pairwise interaction models in these terms helps clarify the deficiencies of these models. Both models assume that only interactions between a certain number of mutations are relevant to prediction (i.e., additive effects of single mutations in non-epistatic models and interactions between two sites in pairwise interaction models). And both models assume that these interactions or mutational effects do not vary over sequence space, first by pooling information across all observed sequences to estimate these interactions or mutational effects, and then by making predictions that extrapolate these observations to all possible genetic backgrounds—even to areas of sequence space where we have little or no data.
                            </p>
                            <p id="p-18">
                             Here we introduce a prediction method corresponding to a different heuristic, one that implements the intuitions that: (1) all orders of genetic interaction can be important and helpful in making predictions; and (2) observations of mutational effects and epistatic coefficients in nearby genetic backgrounds should influence our predictions more than observations in distant genetic backgrounds.
                            </p>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Genetic interactions and the predictability of mutational effects
                             </h3>
                             <p id="p-19">
                              To implement a strategy of this type, it will be helpful to present some general results concerning higher-order epistasis. We begin by considering the case where all phenotypic values are known, before proceeding to our main problem of predicting unknown phenotypic values.
                             </p>
                             <p id="p-20">
                              Our first task is to understand the relationship between the spatial scale of smoothness or predictability in the genotype-phenotype map, the amount of higher order epistasis, and the typical magnitude of epistatic interactions of various orders. These features of the genotype-phenotype map are illustrated for a simulated complete genotype-phenotype mapping in
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1A-C
                              </a>
                              .
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1A
                              </a>
                              shows the distance correlation function (ref. [
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ,
                              <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                              ,
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ] and SI Appendix), which plots how correlations between phenotypic values drop off as one moves through sequence space.
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                              shows the decomposition of the genotype-phenotype map into variance components, where the fraction of variance due to a particular interaction order is defined to be increase in the R
                              <sup>
                               2
                              </sup>
                              of a least squares fit when one adds interaction terms of that order to a model that already includes all interactions of lower order (SI Appendix). This set of variance components is also known in the literature as the normalized amplitude spectrum [
                              <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ,
                              <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ].
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Figure 1C
                              </a>
                              shows how large the individual regression coefficients of a given order (ref. [
                              <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ,
                              <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ,
                              <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ] and SI Appendix) tend to be, by plotting the mean square regression coefficient size as a function of interaction order (more precisely, the regression coefficients for order k are expressed with respect to an arbitrary orthonormal basis that spans the orthogonal complement of the space of all (k - 1)th order interaction models within the space of k-th order models).
                             </p>
                             <div id="F1" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F1.large.jpg?width=800&height=600&carousel=1" title="Summary statistics of a simulated genotype-phenotype map on sequences of length 8 with 4 alleles per site. (A) Distance correlation function for phenotypes (ρ). This is equal to the correlation between phenotypic values averaged over all pairs of sequences separated by the specified number of mutations (Hamming distance). (B) Decomposition of the genotype-phenotype map into variance components. Specifically, the k-th order variance component is equal to the increase in R2 of a least squares fit when when one adds k-th order interaction terms to a model only containing interactions up to order k − 1. (C) Mean squared regression coefficients of different orders (λ). (D) Distance correlation function for mutational effects (γ1), (E) distance correlation function for local pairwise epistatic coefficients (γ2), and (F) distance correlation function for local three-way interactions (γ3). Formulas for calculating these statistics can be found in SI Appendix." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1224697913" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Summary statistics of a simulated genotype-phenotype map on sequences of length 8 with 4 alleles per site. (A) Distance correlation function for phenotypes (<em>ρ</em>). This is equal to the correlation between phenotypic values averaged over all pairs of sequences separated by the specified number of mutations (Hamming distance). (B) Decomposition of the genotype-phenotype map into variance components. Specifically, the <em>k</em>-th order variance component is equal to the increase in <em>R</em><sup>2</sup> of a least squares fit when when one adds <em>k</em>-th order interaction terms to a model only containing interactions up to order <em>k</em> − 1. (C) Mean squared regression coefficients of different orders (<em>λ</em>). (D) Distance correlation function for mutational effects (<em>γ</em><sub>1</sub>), (E) distance correlation function for local pairwise epistatic coefficients (<em>γ</em><sub>2</sub>), and (F) distance correlation function for local three-way interactions (<em>γ</em><sub>3</sub>). Formulas for calculating these statistics can be found in <em>SI Appendix</em>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F1.medium.gif" width="440" height="251"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F1.medium.gif" width="440" height="251"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <p id="p-21" class="first-child">
                                Summary statistics of a simulated genotype-phenotype map on sequences of length 8 with 4 alleles per site. (A) Distance correlation function for phenotypes (ρ). This is equal to the correlation between phenotypic values averaged over all pairs of sequences separated by the specified number of mutations (Hamming distance). (B) Decomposition of the genotype-phenotype map into variance components. Specifically, the k-th order variance component is equal to the increase in R
                                <sup>
                                 2
                                </sup>
                                of a least squares fit when when one adds k-th order interaction terms to a model only containing interactions up to order k − 1. (C) Mean squared regression coefficients of different orders (λ). (D) Distance correlation function for mutational effects (γ
                                <sub>
                                 1
                                </sub>
                                ), (E) distance correlation function for local pairwise epistatic coefficients (γ
                                <sub>
                                 2
                                </sub>
                                ), and (F) distance correlation function for local three-way interactions (γ
                                <sub>
                                 3
                                </sub>
                                ). Formulas for calculating these statistics can be found in SI Appendix.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              Because our goal is to understand how to combine the mutational effects, observed double-mutant epistatic coefficients, etc., we can also plot how the predictability of these effects drops off as we move through sequence space [
                              <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              ]. The distance correlation functions for mutational effects, local pairwise epistatic coefficients, and local three-way interactions are shown in
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Figure 1D-F
                              </a>
                              , respectively.
                             </p>
                             <p id="p-23">
                              These pictures, particularly the plots of correlations as a function of distance in genotypic space, are quite informative for our intuitive goal of determining how to combine our observations of mutational effects, local epistatic coefficients, etc. when making predictions. We see for example from
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Figure 1D
                              </a>
                              that, for this particular genotype-phenotype map, mutational effects remain moderately correlated across all of sequence space, dropping from having a Pearson correlation coefficient of roughly 0.5 in adjacent genetic backgrounds to a correlation coefficient of roughly 0.2 in maximally distant backgrounds. However, from
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Figure 1E
                              </a>
                              we see that the predictability of interactions in double-mutants decays much more rapidly, and so our observations of double-mutant interactions are only really informative in genetic backgrounds up to approximately two mutations away. Similarly,
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Figure 1F
                              </a>
                              shows that observed triple-mutant interactions are only substantially informative in immediately adjacent genetic backgrounds. These results suggest that when making predictions, it might, e.g., be sensible to extrapolate our observations of mutational effects throughout sequence space, but only allow our observations of interactions in local double and triple mutant cycles to influence our predictions in relatively nearby genetic backgrounds.
                             </p>
                             <p id="p-24">
                              How can we convert these intuitions based on examining the decay in the consistency of observed interactions into a rigorous method of phenotypic prediction? The key in answering this question lies in the fact that all 6 panels of
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              are actually intimately related with each other and with previously proposed methods for phenotypic prediction. In particular, it is classically known that the three pictures in
                              <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                               Figure 1A-C
                              </a>
                              actually contain identical information, in the sense that for any given genotype-phenotype map, having any one of the panels in the top row of
                              <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              allows us to compute the other two (ref. [
                              <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ,
                              <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ,
                              <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                               56
                              </a>
                              ], SI Appendix). Moreover, recent results on the distance correlation function of mutational effects (previously denoted by γ, [
                              <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              ]) have shown that
                              <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                               Figure 1D
                              </a>
                              can be derived from any of
                              <a id="xref-fig-1-13" class="xref-fig" href="#F1">
                               Figure 1A-C
                              </a>
                              . Here, we extend these results to arbitrary orders of interaction, so that
                              <a id="xref-fig-1-14" class="xref-fig" href="#F1">
                               Figure 1E-F
                              </a>
                              and their higher-order analogs can likewise be derived from any of
                              <a id="xref-fig-1-15" class="xref-fig" href="#F1">
                               Figure 1A-C
                              </a>
                              (SI Appendix). In total, these results show that all the information on how mutational effects and local epistatic interactions (of arbitrary order) generalize across genetic backgrounds is in fact captured by any one of the pictures in
                              <a id="xref-fig-1-16" class="xref-fig" href="#F1">
                               Figure 1A-C
                              </a>
                              and can hence be specified using only ℓ − 1 parameters, where ℓ is the sequence length.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Higher-order epistasis and the geometry of genotype-phenotype maps
                             </h3>
                             <p id="p-25">
                              To see how exactly
                              <a id="xref-fig-1-17" class="xref-fig" href="#F1">
                               Figures 1A-F
                              </a>
                              are related, we need to develop a deeper understanding of the relationship between higher-order epistasis and the decay of phenotypic predictability across increasingly divergent genetic backgrounds. If we again consider constructing a least-squares fit of a genotype-phenotype map using up to k-th order interactions and compare it to the least squares fit using only up to (k − 1)th order interactions, the difference between these two fits represents the pure k-th order interaction component of the genotype-phenotype map, in the sense that it is the component that can be expressed using k-th order interactions but cannot be constructed using only lower-order interactions. The crucial observation is that these pure k-th order components each have a very specific geometry, and as a result the distance correlation function for any such k-th order component must take a very specific shape. Technically, these shapes are given by a set of orthogonal polynomials known as the Krawtchouk polynomials [
                              <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ,
                              <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">
                               56
                              </a>
                              –
                              <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ], but for our purposes it suffices to look at these functions visually,
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figures 2A
                              </a>
                              and
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               2B
                              </a>
                              .
                             </p>
                             <div id="F2" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F2.large.jpg?width=800&height=600&carousel=1" title="The relationship between the distance correlation function (ρ) and higher-order epistasis. The example shown considers sequences of length 8 with 4 alleles per site. (A) Distance correlation functions for interaction orders contributing to positive local correlations. (B) Distance correlation functions for interactions contributing to negative local correlations. The critical interaction order dividing these two groups of interactions is given by the expected number of mutational differences between two random sequences (which is  in this case). Note that the values plotted in (A) and (B) do not depend on the specifics of the genotype-phenotype map, but rather depend only on the sequence length ℓ and number of alleles per site. These values are given by the appropriately normalized Krawtchouk polynomials (SI Appendix). (C) The distance correlation function of a complete genotype-phenotype map is a weighted average of curves shown in (A) and (B), with the weights given by the list of variance components (vk, k = 1, ···, ℓ, shown in parentheses in the legend). The distance correlation function and weights shown correspond to the genotype-phenotype map from Figure 1 and each curve is shaded proportionally to its weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1224697913" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The relationship between the distance correlation function (<em>ρ</em>) and higher-order epistasis. The example shown considers sequences of length 8 with 4 alleles per site. (A) Distance correlation functions for interaction orders contributing to positive local correlations. (B) Distance correlation functions for interactions contributing to negative local correlations. The critical interaction order dividing these two groups of interactions is given by the expected number of mutational differences between two random sequences (which is  in this case). Note that the values plotted in (A) and (B) do not depend on the specifics of the genotype-phenotype map, but rather depend only on the sequence length <em>ℓ</em> and number of alleles per site. These values are given by the appropriately normalized Krawtchouk polynomials (<em>SI Appendix</em>). (C) The distance correlation function of a complete genotype-phenotype map is a weighted average of curves shown in (A) and (B), with the weights given by the list of variance components (<em>v</em><sub><em>k</em></sub>, <em>k</em> = 1, ···, <em>ℓ</em>, shown in parentheses in the legend). The distance correlation function and weights shown correspond to the genotype-phenotype map from Figure 1 and each curve is shaded proportionally to its weight.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F2.medium.gif" width="440" height="115"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F2.medium.gif" width="440" height="115"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <p id="p-26" class="first-child">
                                The relationship between the distance correlation function (ρ) and higher-order epistasis. The example shown considers sequences of length 8 with 4 alleles per site. (A) Distance correlation functions for interaction orders contributing to positive local correlations. (B) Distance correlation functions for interactions contributing to negative local correlations. The critical interaction order dividing these two groups of interactions is given by the expected number of mutational differences between two random sequences (which is
                                <span class="inline-formula" id="inline-formula-1">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/F2/embed/inline-graphic-1.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/F2/embed/inline-graphic-1.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                in this case). Note that the values plotted in (A) and (B) do not depend on the specifics of the genotype-phenotype map, but rather depend only on the sequence length ℓ and number of alleles per site. These values are given by the appropriately normalized Krawtchouk polynomials (SI Appendix). (C) The distance correlation function of a complete genotype-phenotype map is a weighted average of curves shown in (A) and (B), with the weights given by the list of variance components (v
                                <sub>
                                 k
                                </sub>
                                , k = 1, ···, ℓ, shown in parentheses in the legend). The distance correlation function and weights shown correspond to the genotype-phenotype map from
                                <a id="xref-fig-1-18" class="xref-fig" href="#F1">
                                 Figure 1
                                </a>
                                and each curve is shaded proportionally to its weight.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-27">
                              We see that these functions split naturally into two groups, with different qualitative interpretations depending on the order of the interaction. The first of these groups consists of orders of interaction that contribute positive local correlations, so that genotypes that are near to each other in sequence space tend to have similar phenotypes. These functions are shown in
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Figure 2A
                              </a>
                              , and we can see that the main qualitative effect of increasing interaction order among this group is that these locally positive correlations decay increasingly rapidly.
                             </p>
                             <p id="p-28">
                              The second of these groups consists of orders of interaction that exhibit negative local correlations, i.e., they make mutationally adjacent sequences tend to have anti-correlated values. These orders of interaction are shown in
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Figure 2B
                              </a>
                              , and qualitatively they oscillate increasingly rapidly as the order increases. There is an easy mnemonic for whether a particular interaction order contributes locally positive or locally negative correlations, since if the order of interaction is less than the expected distance between two random sequences, then that order contributes locally positive correlations, otherwise it contributes negative local correlations. (If the order is equal to the expected distance between random sequences, then the correlation at distance 1 is zero, but it will be negative at distance 2, see SI Appendix).
                             </p>
                             <p id="p-29">
                              It turns out that the distance correlation function of a complete genotype-phenotype map is simply a weighted average of these curves, with the weights given by the list of variance components (illustrated by
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Figure 2C
                              </a>
                              ). Therefore, having
                              <a id="xref-fig-1-19" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                              allows us to draw
                              <a id="xref-fig-1-20" class="xref-fig" href="#F1">
                               Figure 1A
                              </a>
                              . Conversely, given the distance correlation in
                              <a id="xref-fig-1-21" class="xref-fig" href="#F1">
                               Figure 1A
                              </a>
                              , we can identify the unique linear combination of curves in
                              <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                               Figure 2A-B
                              </a>
                              that produce it, which allows us to draw
                              <a id="xref-fig-1-22" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                              . At the same time, given the mean squared regression coefficient λ
                              <sub>
                               k
                              </sub>
                              for each order k, we can calculate the total variance due to k-th order interaction by multiplying λ
                              <sub>
                               k
                              </sub>
                              with the number of interaction terms of order k. This allows us to convert between the quantities in
                              <a id="xref-fig-1-23" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                              and
                              <a id="xref-fig-1-24" class="xref-fig" href="#F1">
                               C
                              </a>
                              . Finally, once we have the vector of λ
                              <sub>
                               k
                              </sub>
                              , we can calculate the distance correlation function for how local genetic interactions of order j generalize over sequence space, γ
                              <sub>
                               j
                              </sub>
                              . Specifically, we find that the contribution of kth-order interaction to γ
                              <sub>
                               j
                              </sub>
                              takes a fixed shape that is given by a Krawtchouk polynomial of order k − j for a sequence of length ℓ − j, so that the calculation of the curve γ
                              <sub>
                               j
                              </sub>
                              is exactly the same as calculating the distance correlation function for a sequence of length ℓ − j whose ℓ − j λ
                              <sub>
                               k
                              </sub>
                              s are simply λ
                              <sub>
                               j
                              </sub>
                              ,…, λ
                              <sub>
                               ℓ
                              </sub>
                              (see Supplemental Figure S1 for an example and SI Appendix for details).
                             </p>
                             <p id="p-30">
                              In summary, we see that the reason the fraction of variance due to each order of interaction determines how phenotypic observations, mutational effects, local double-mutant epistatic coefficients, etc. generalize is that each order of interaction makes a characteristic contribution to the shape of the distance correlation functions for each of these quantities. Moreover, the qualitative nature of these contributions is that the correlations decay more and more quickly with genetic background as the order of interactions increases, so that we can think of an increasing fraction of higher-order interactions as resulting in an increasingly rapid decay of predictability with genetic distance, with the exact values of the variance components determining the precise shape of the decay.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Bayesian phenotypic prediction
                             </h3>
                             <p id="p-31">
                              Now that we understand how higher-order epistasis relates to how far in sequence space we can generalize our experimental observations, our next goal is to develop an inference procedure that can incorporate our specific beliefs about how quickly phenotypic predictability decays. One way to incorporate such beliefs is via Bayesian inference, where these beliefs can be incorporated by choosing an appropriate prior. In this context, the prior takes the form of a probability distribution over all possible genotype-phenotype maps, and we wish to construct a prior whose mass is concentrated on genotype-phenotype maps where the phenotypic predictability decays in the desired manner.
                             </p>
                             <p id="p-32">
                              In fact, our analysis so far suggests a natural family of such priors. Specifically, given the fraction of variance due to each order of interaction (as shown schematically in
                              <a id="xref-fig-1-25" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                              ), we can draw epistatic interaction coefficients from a zero-mean normal distribution with variance given by the values in
                              <a id="xref-fig-1-26" class="xref-fig" href="#F1">
                               Figure 1C
                              </a>
                              , which results in a genotype-phenotype map that in expectation produces the patterns of correlation shown in
                              <a id="xref-fig-1-27" class="xref-fig" href="#F1">
                               Figure 1A
                              </a>
                              and
                              <a id="xref-fig-1-28" class="xref-fig" href="#F1">
                               Figure 1D-F
                              </a>
                              . Models of this form are actually well-studied in the theoretical literature on fitness landscapes, where they are known as “random field models” [
                              <a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ,
                              <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">
                               56
                              </a>
                              ], and take the form of a family of multivariate Gaussian distributions parameterized in terms of the amount of variance due to each order of genetic interaction.
                             </p>
                             <p id="p-33">
                              Importantly, various previously developed methods can be subsumed as particular (limiting) cases of Bayesian inference under this family of priors. For example, the solutions of the additive model and our recently proposed method of minimum epistasis interpolation [
                              <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                               59
                              </a>
                              ] both arise as the maximum a posteriori (MAP) estimates in particular limiting cases where the prior fraction of variance due to additive effects goes to 1 (see Supplemental Figure S2). Similarly, the pairwise model [
                              <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ] arises as the MAP estimate in the limit where the total fraction of variance due to additive and pairwise effects goes to 1 (Supplemental Figure S2). Thus, in a rigorous manner we can view these previously proposed methods as encoding specific assumptions about how the predictability of mutational effects, epistatic coefficients and phenotypic values changes as we move through sequence space, where these assumptions take the form of particular shapes for the curves in
                              <a id="xref-fig-1-29" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              or equivalently specific prior assumptions about the fraction of phenotypic variance due to each order of interaction.
                             </p>
                             <p id="p-34">
                              Finally, a key fact about this family of priors is that they are Gaussian, and so under the assumption that experimental errors are normally distributed, we can do inference under this prior using Gaussian process regression (see [
                              <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ] for a review), which allows us to write down analytical expressions for the corresponding posterior distribution. In particular, suppose our prior distribution is a mean zero Gaussian with covariance matrix
                              <strong>
                               K
                              </strong>
                              ,
                              <strong>
                               y
                              </strong>
                              is our vector of observations and
                              <strong>
                               E
                              </strong>
                              is a diagonal matrix with estimates of the variance due to experimental noise for each of our observations down the main diagonal. Then the posterior distribution for our vector of predicted phenotypes
                              <strong>
                               f
                              </strong>
                              is normally distributed with mean
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and covariance matrix
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <strong>
                               K
                              </strong>
                              <sub>
                               BB
                              </sub>
                              is the submatrix of
                              <strong>
                               K
                              </strong>
                              indexed by the set of observed sequences B, and
                              <strong>
                               K
                              </strong>
                              <sub>
                               B·
                              </sub>
                              and
                              <strong>
                               K
                              </strong>
                              <sub>
                               ·B
                              </sub>
                              are the submatrices of
                              <strong>
                               K
                              </strong>
                              consisting, respectively, of the rows and columns indexed by members of B.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Estimating variance components from partial data
                             </h3>
                             <p id="p-35">
                              To summarize the previous section, if we know the fraction of phenotypic variation due to each order of epistatic interaction, then we can derive a simple method of making phenotypic predictions that uses the corresponding covariance structure to appropriately generalize from observed phenotypic effects, double-mutant epistatic interactions, phenotypic values, etc. While several existing methods of phenotypic prediction essentially come down to making specific assumptions about these variance components, our analysis suggests that a natural approach would be to make our predictions using variance components estimated from the data itself, i.e., an empirical Bayes approach in which we determine what prior to use by looking at the covariance structure of our observations. Conceptually, we want to make phenotypic predictions by assuming that the decay in the predictability of mutational effects, local epistatic interactions, etc. for the true genotype-phenotype mapping is similar outside our data as it is within our data. Practically, we can implement this idea by doing inference under a prior consisting of random genotype-phenotype maps where the effects of mutations (and epistatic coefficients, etc.) decay with genetic distance in the same way as in our data.
                             </p>
                             <p id="p-36">
                              A naïve implementation of this approach would be to simply use our observed distance correlation function to build the covariance matrix
                              <strong>
                               K
                              </strong>
                              for our prior by setting the covariance between each pair of sequences at distance d equal to the covariance between sequences at distance d in our data. However, there is a subtle problem with this idea. To see what the difficulty is, recall that the distance correlation in phenotypes is a weighted sum of distance correlation curves of various orders with the weights equal to the fraction of variance due to each order of interaction, as shown in
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              .
                             </p>
                             <p id="p-37">
                              The fact that these weights need to be positive and sum to one puts strong constraints on the shape that the correlation function can take for a function defined over all of sequence space. For example, positive local correlations cannot decay any more slowly than they would for a purely additive model. However, for incompletely sampled sequence spaces, these constraints need not hold (e.g., if the sampling consisted of several clusters of sequences with identical phenotypes separated from each other with missing sequences, one could have a perfect correlation within the smaller distance classes). Unfortunately, using such a function to define a the matrix
                              <strong>
                               K
                              </strong>
                              would not result in a valid prior (in particular,
                              <strong>
                               K
                              </strong>
                              would not be positive definite, see SI Appendix). Thus, rather than using the observed covariance function to define our prior, we instead attempt to find the closest valid prior. We do this using weighted least squares, where the squared error for the correlation at distance d is weighted by the number of pairs of sequences at distance d (SI Appendix); this technique is formally equivalent to the idea of choosing a prior based on “kernel alignment” in the Gaussian processes literature, see ref. [
                              <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                               60
                              </a>
                              ]. In addition, when producing this weighted least squares estimate, we also take into account the magnitude of the experimental noise so as to distinguish between experimental uncertainty and the influence of any true uncorrelated component of the genotype-phenotype map, and apply regularization to ensure that the resulting prior includes interactions of all orders (SI Appendix).
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Practical implementation
                             </h3>
                             <p id="p-38">
                              So far we have described a two-step method for phenotypic prediction: in the first step we estimate the variance components of the data via a weighted least squares fit to the observed phenotypic correlation function; then, in the second step, we conduct Gaussian process regression under the corresponding prior, so as to encapsulate the assumption that the effects of mutations, etc. generalize outside the data in a manner similar to how they generalize within the data. However, to construct a practical implementation of this approach we must also address certain computational difficulties that arise as a consequence of the very large size of biological sequence space.
                             </p>
                             <p id="p-39">
                              Specifically, the major challenge in solving
                              <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. 1
                              </a>
                              and
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               2
                              </a>
                              is that the computation involves inverting the m × m dense matrix
                              <strong>
                               K
                              </strong>
                              <sub>
                               BB
                              </sub>
                              , where m is the number of observed sequences. The complexity of this problem scales cubically with m in time and quadratically with m in space. As a result, Gaussian process regression becomes computationally expensive when the number of observed sequences m is larger than several thousand [
                              <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                               61
                              </a>
                              ].
                             </p>
                             <p id="p-40">
                              To circumvent this difficulty, we provide an implementation that leverages the symmetries of sequence space to allow practical computations for sequence spaces containing up to low millions of sequences. The basic strategy is to rephrase our problem so that the solution can be found iteratively using only sparse matrix-vector multiplication.
                             </p>
                             <p id="p-41">
                              In particular, notice that
                              <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. 1
                              </a>
                              can be solved by first finding a vector
                              <strong>
                               α
                              </strong>
                              that satisfies (
                              <strong>
                               K
                              </strong>
                              <sub>
                               BB
                              </sub>
                              +
                              <strong>
                               E
                              </strong>
                              )
                              <strong>
                               α
                              </strong>
                              =
                              <strong>
                               y
                              </strong>
                              . Also, notice that the matrix
                              <strong>
                               K
                              </strong>
                              <sub>
                               BB
                              </sub>
                              is a principle submatrix of
                              <strong>
                               K
                              </strong>
                              , so that we can write
                              <strong>
                               K
                              </strong>
                              <sub>
                               BB
                              </sub>
                              = (
                              <strong>
                               I
                              </strong>
                              <sub>
                               ·B
                              </sub>
                              )
                              <sup>
                               T
                              </sup>
                              <strong>
                               KI
                              </strong>
                              <sub>
                               ·B
                              </sub>
                              where
                              <strong>
                               I
                              </strong>
                              <sub>
                               ·B
                              </sub>
                              consists of the columns of the identity matrix
                              <strong>
                               I
                              </strong>
                              that correspond to our set of observed sequences B. Since the entries of
                              <strong>
                               K
                              </strong>
                              depend only on the Hamming distance between the corresponding sequences,
                              <strong>
                               K
                              </strong>
                              can expressed as a polynomial in the graph Laplacian (i.e., the matrix
                              <strong>
                               L
                              </strong>
                              whose i, j-th entry is −1 if i is adjacent to j, ℓ(
                              <strong>
                               α
                              </strong>
                              − 1) if i = j, and 0 otherwise) that is,
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              [
                              <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                               62
                              </a>
                              ,
                              <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                               63
                              </a>
                              ] for some b
                              <sub>
                               0
                              </sub>
                              ,…, b
                              <sub>
                               ℓ
                              </sub>
                              that we can find analytically, and thus
                              <strong>
                               Kx
                              </strong>
                              can be found by iteratively applying the sparse matrix
                              <strong>
                               L
                              </strong>
                              to
                              <strong>
                               x
                              </strong>
                              at most ℓ times. Using these results, we can rewrite our original equation (
                              <strong>
                               K
                              </strong>
                              <sub>
                               BB
                              </sub>
                              +
                              <strong>
                               E
                              </strong>
                              )
                              <strong>
                               α
                              </strong>
                              =
                              <strong>
                               y
                              </strong>
                              using only sparse matrices as
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , which we solve using the conjugate gradient algorithm.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Application to protein G
                             </h3>
                             <p id="p-42">
                              We first apply our method to a dataset derived from a deep mutational scanning study of the IgG-binding domain of streptococcal protein G (GB1) [
                              <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ]. This experiment assayed nearly all possible combinations of mutations at four sites (V39, D40, G41, and V54; 20
                              <sup>
                               4
                              </sup>
                              = 160, 000 protein variants), where pairs of mutations at these sites were previously shown to exhibit strong interactions [
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ]. The library of protein variants was sequenced before and after binding to IgG-Fc beads and the binding scores were determined as the log enrichment ratio (logarithm of ratio of counts before and after selection, normalized by subtracting the log ratio of the wild-type). Due to incomplete coverage of the input library, these data lack binding scores for 6.6% of possible variants.
                             </p>
                             <p id="p-43">
                              We began by inferring the variance components of the GB1 landscape from the empirical autocorrelation function (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3A
                              </a>
                              ) using our least squares procedure applied to the empirical distance correlation function of all available data (93.6% of all possible sequences, see also SI Appendix for details). In
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 3B
                              </a>
                              , we see that the vast majority of the variance in the data is estimated to be explained by the additive, pairwise, and 3-way components (59%, 37%, and 4% of total variance, respectively), while the estimated contribution of the 4th order component is negligible.
                             </p>
                             <div id="F3" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F3.large.jpg?width=800&height=600&carousel=1" title="Analyses of the GB1 combinatorial mutagenesis dataset [37]. (A) Distance correlation of phenotypic values. (B) Variance components. (C) Distance correlation of the effects of single mutations. In A-C, gray represents statistics of the prior distribution inferred from the full dataset consisting of 149, 361 genotypes (93.6% of all possible sequences), black represents the posterior statistics estimated based on 2, 000 Hamiltonian Monte Carlo samples. Error bars indicate 95% credible intervals. (D) Comparison of model performance in terms of out-of-sample R2 for a range of training sample sizes calculated for 5 replicates. Additive models were fit using ordinary least squares. Pairwise and 3-way regression models were fit using elastic net regularization with regularization parameters chosen by 10-fold cross-validation (SI Appendix). The global epistasis model assumes the binding score is a nonlinear transformation of an unobserved additive phenotype and was fit following ref. [45]. Error bars represent one standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1224697913" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analyses of the GB1 combinatorial mutagenesis dataset [37]. (A) Distance correlation of phenotypic values. (B) Variance components. (C) Distance correlation of the effects of single mutations. In A-C, gray represents statistics of the prior distribution inferred from the full dataset consisting of 149, 361 genotypes (93.6% of all possible sequences), black represents the posterior statistics estimated based on 2, 000 Hamiltonian Monte Carlo samples. Error bars indicate 95% credible intervals. (D) Comparison of model performance in terms of out-of-sample <em>R</em><sup>2</sup> for a range of training sample sizes calculated for 5 replicates. Additive models were fit using ordinary least squares. Pairwise and 3-way regression models were fit using elastic net regularization with regularization parameters chosen by 10-fold cross-validation (<em>SI Appendix</em>). The global epistasis model assumes the binding score is a nonlinear transformation of an unobserved additive phenotype and was fit following ref. [45]. Error bars represent one standard deviation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F3.medium.gif" width="440" height="406"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F3.medium.gif" width="440" height="406"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <p id="p-44" class="first-child">
                                Analyses of the GB1 combinatorial mutagenesis dataset [
                                <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">
                                 37
                                </a>
                                ]. (A) Distance correlation of phenotypic values. (B) Variance components. (C) Distance correlation of the effects of single mutations. In A-C, gray represents statistics of the prior distribution inferred from the full dataset consisting of 149, 361 genotypes (93.6% of all possible sequences), black represents the posterior statistics estimated based on 2, 000 Hamiltonian Monte Carlo samples. Error bars indicate 95% credible intervals. (D) Comparison of model performance in terms of out-of-sample R
                                <sup>
                                 2
                                </sup>
                                for a range of training sample sizes calculated for 5 replicates. Additive models were fit using ordinary least squares. Pairwise and 3-way regression models were fit using elastic net regularization with regularization parameters chosen by 10-fold cross-validation (SI Appendix). The global epistasis model assumes the binding score is a nonlinear transformation of an unobserved additive phenotype and was fit following ref. [
                                <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                                 45
                                </a>
                                ]. Error bars represent one standard deviation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-45">
                              We can use the results from the previous section to understand the practical meaning of these estimates for our task of phenotypic prediction. For example, in
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 3C
                              </a>
                              , we plot the correlation of mutational effects as a function of Hamming distance [
                              <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              ] (SI Appendix). We observe that the correlation of the effect of a random mutation is 0.72 between two genetic backgrounds that differ by one mutation, and 0.32 for two maximally distinct backgrounds (Hamming distance = 3). This decay is characteristic of non-additivity and shows that while the effects of point mutations remain positively correlated across sequence space, the extent of this correlation is approximately twice as high in nearby sequences as opposed to maximally distant sequences, and that therefore, when making predictions, we should give local observations of mutational effects approximately twice as strong a weight as distant observations of mutational effects.
                             </p>
                             <p id="p-46">
                              At a broader scale, our analysis above also provides qualitative insights into the overall structure of the genotype-phenotype map. For example, we stated above that the orders of epistatic interaction can be divided into locally correlated and locally anti-correlated groups, depending on whether the order of the interaction is less or greater than the expected distance between two random sequences. Random protein sequences of length 4 differ at
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              sites on average, so interaction orders 1 through 3 correspond to the genotype-phenotype map being locally correlated, whereas order 4 controls the strength of local anti-correlation. Thus, our estimated variance components suggest that the GB1 genotype-phenotype map is dominated by locally positive correlations, with essentially no anti-correlated component.
                             </p>
                             <p id="p-47">
                              Within our overall inference procedure, the estimated variance components discussed above are used to construct a prior probability distribution over all genotype-phenotype maps where, in expectation, mutational effects, epistatic interactions and observed phenotypes generalize across sequence space in the same manner as observed in the data. The resulting prior distribution is based only based on the coarse summary statistics provided by the distance correlation function, and so the next step is to calculate the posterior distribution using the fine-scale information from the individual observations. Specifically, we calculated the MAP solution using all available data and drew 2000 samples from the resulting posterior distribution. Using a 2021 MacBook laptop with 32 Gb RAM, calculating the MAP solution took less than one minute, and we could also produce samples from the posterior distribution at a rate of 1-2 samples per minute.
                             </p>
                             <p id="p-48">
                              One immediate question about this posterior distribution is the extent to which its distance correlation function and variance components are similar or different from that of the prior (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Figure 3A-C
                              </a>
                              ). We find that the posterior gives very tight estimates of the variance components and correlation structure of the true genotype-phenotype map, but that these estimates differ somewhat from the prior, with the 3rd order interactions being roughly 1.6 times as strong in the posterior (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Figure 3B
                              </a>
                              ), which results in a slightly faster decay in the predictability of mutational effects as we move through sequence space (
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Figure 3C
                              </a>
                              ). Thus, we conclude that our prior distribution provided a qualitatively reasonable estimate of the overall statistical features of the data, but that the rough estimates used to define the prior can be further refined using our full inference procedure.
                             </p>
                             <p id="p-49">
                              Obviously, another important question is the performance of the predictions made by our method. Since the GB1 landscape is relatively well sampled, we were able to assess this performance for a large range of sampling regimes, from quite sparse to extremely dense, by using our method to make predictions for randomly sampled held-out data with increasing amounts of training data. Critically, the variance component estimates were re-computed for each of these random samples in order to provide a realistic test of the entire inference pipeline in the low-data regime. For comparison we also fit an additive model using ordinary least squares and regularized pairwise and 3-way regression models. Since both L
                              <sub>
                               1
                              </sub>
                              and L
                              <sub>
                               2
                              </sub>
                              regularized regression have been used to infer genotype-phenotype maps [
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ,
                              <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ,
                              <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                               64
                              </a>
                              –
                              <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                               66
                              </a>
                              ], here we fit the pairwise and three-way models using elastic net regression (see SI Appendix) where the penalty term for model complexity is a mixture of L
                              <sub>
                               1
                              </sub>
                              and L
                              <sub>
                               2
                              </sub>
                              norms [
                              <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">
                               67
                              </a>
                              ] and the relative weight of the two penalties is chosen via crossvalidation. In addition to the linear regression models, we also fit a global epistasis model [
                              <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ] where the binding score is modeled as a nonlinear transformation of a latent additive phenotype on which each possible mutation has a background-independent effect (SI Appendix).
                             </p>
                             <p id="p-50">
                              We compared the predictive accuracy of these five models by plotting out-of-sample R
                              <sup>
                               2
                              </sup>
                              against a wide range of training sample sizes (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Figure 3D
                              </a>
                              ). We see that the out-of-sample R
                              <sup>
                               2
                              </sup>
                              of the additive model and the global epistasis model stay nearly constant, regardless of training sample size, consistent with their low number of model parameters and low flexibility. The modest R
                              <sup>
                               2
                              </sup>
                              for the global epistasis model also indicates a substantial degree of specific epistasis (i.e., interactions between specific subsets of sites, as opposed to interactions due to a global nonlinearity [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]). In terms of the regression models that do include these specific interactions, the pairwise model is among the top models for low training sample size, but fails to improve beyond 20% training data, while the 3-way model performs strongly with a large amount of data, but under-performs when data are sparse. We see that our empirical variance component regression (VC regression) method performs equivalently to the pairwise model at low data density and similarly to the three-way model at high data density (remaining marginally superior at very high sampling), and thus provides the strongest overall performance.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Application to human 5′ splice site data
                             </h3>
                             <p id="p-51">
                              To provide an application of our method to a nucleic acid genotype-phenotype map, we turn to an analysis of a high-throughput splicing assay that measured the activity of nearly all possible 5′ splice sites [
                              <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ]. The 5′ splice site (5′ss) is a 9-nucleotide sequence that spans the exon-intron junction. It comprises 3 nt at the end of the upstream exon (denoted as positions −3 to −1) and 6 nt at the beginning of the intron (coded +1 to +6). The consensus 5′ss sequence in humans is CAG/GUAAGU, with the slash denoting the exon-intron junction. At the beginning of the splicing reaction, the 5′ss is recognized by the U1 snRNP of the spliceosome through direct base pairing between the 5′ss and the U1 snRNA [
                              <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">
                               68
                              </a>
                              ], whose 5′ sequence is complementary to the consensus 5′ss sequence. In ref. [
                              <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ], the authors used a massively parallel splicing assay to estimate the splicing efficiency of 93.8% of the 32,768 possible 5′ss sequences of the form NNN/GYNNNN for intron 7 of the gene SMN1, using a barcoded minigene library transiently transfected into human cells. Splicing efficiency was measured in units of percent spliced in (PSI), which was estimated as the ratio between the exon inclusion read count and the total read count (which comprises both exon inclusion and exon skipping reads) divided by the corresponding ratio for the consensus sequence, expressed as a percentage. Computational times were somewhat faster for this dataset than for GB1, with the MAP estimate taking roughly 25 seconds on a 2021 MacBook and samples from the posterior distribution being produced at a rate of roughly 10 samples per minute.
                             </p>
                             <p id="p-52">
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4A
                              </a>
                              shows the distance correlation function of PSI for the observed sequences. These correlations drop off quite rapidly, with sequences differing at 5 or more positions having PSIs that are essentially uncorrelated. The associated estimated variance components are shown in
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Figure 4B
                              </a>
                              . These indicate that pairwise interactions account for the largest proportion of the sample variance (42.2%), but that there are also substantial higher-order interactions, with the variance due to 5-way interactions (13.7%) being comparable to those of the additive and three-way component. The orders of genetic interaction corresponding to locally negative correlations (order ≥ 6, since the Hamming distance between two random sequences is equal to
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/14/2020.10.14.339804/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) are estimated to play a relatively small but perhaps non-negligible role, accounting for 6.0% of the total variance. In
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Figure 4C
                              </a>
                              , we found the correlation of mutational effects for two backgrounds that differ by one mutation is roughly 50% but decays to roughly zero for distant genetic backgrounds. Sampling from the posterior distribution, we see that the statistical characteristics of the posterior again have very small credible intervals and remain similar to those estimated using our least squares procedure, with a slightly increased contribution of pair-wise and three-way interactions and a decreased contribution of five-way interactions (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Figure 4B
                              </a>
                              ). Overall, the splicing landscape appears to be dominated by interactions of order 2 through 5, resulting in positive correlations between the splicing activity of nearby genotypes, but a relatively limited ability to generalize our observations to distant regions of sequence space. This qualitative behavior is consistent with our mechanistic intuition, e.g., mutations that substantially decrease U1 snRNA binding in the context of a functional splice site are likely to have no impact in an already non-functional sequence context.
                             </p>
                             <div id="F4" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F4.large.jpg?width=800&height=600&carousel=1" title="Analyses of the SMN1 5′ss combinatorial mutagenesis dataset. (A) Distance correlation function of the splicing phenotype (PSI). (B) Variance components. (C) Distance correlation of single-mutant effects. Gray represents statistics of the prior distribution inferred from the full dataset consisting of 30, 732 genotypes (93.8% of all possible splice sites), black represents the posterior statistics estimated using 2000 Hamiltonian Monte Carlo samples. Error bars indicate 95% credible intervals. (D) Out-of-sample R2 of the five models plotted against a range of training sample sizes. Error bars represent one standard deviation calculated for 5 replicates for each sample size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1224697913" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Analyses of the <em>SMN1</em> 5′ss combinatorial mutagenesis dataset. (A) Distance correlation function of the splicing phenotype (PSI). (B) Variance components. (C) Distance correlation of single-mutant effects. Gray represents statistics of the prior distribution inferred from the full dataset consisting of 30, 732 genotypes (93.8% of all possible splice sites), black represents the posterior statistics estimated using 2000 Hamiltonian Monte Carlo samples. Error bars indicate 95% credible intervals. (D) Out-of-sample <em>R</em><sup>2</sup> of the five models plotted against a range of training sample sizes. Error bars represent one standard deviation calculated for 5 replicates for each sample size.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F4.medium.gif" width="440" height="418"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F4.medium.gif" width="440" height="418"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <p id="p-53" class="first-child">
                                Analyses of the SMN1 5′ss combinatorial mutagenesis dataset. (A) Distance correlation function of the splicing phenotype (PSI). (B) Variance components. (C) Distance correlation of single-mutant effects. Gray represents statistics of the prior distribution inferred from the full dataset consisting of 30, 732 genotypes (93.8% of all possible splice sites), black represents the posterior statistics estimated using 2000 Hamiltonian Monte Carlo samples. Error bars indicate 95% credible intervals. (D) Out-of-sample R
                                <sup>
                                 2
                                </sup>
                                of the five models plotted against a range of training sample sizes. Error bars represent one standard deviation calculated for 5 replicates for each sample size.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-54">
                              To see how our model performs when greater or lesser amounts of data are available, we compared the predictive power of the same five models as in
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Figure 3D
                              </a>
                              by plotting their out-of-sample R
                              <sup>
                               2
                              </sup>
                              against a wide range of training sample sizes (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Figure 4D
                              </a>
                              ). The rank order of the models is largely consistent throughout the sampling range. More importantly, we see that the variance component model adapts to increasing data density at a much faster rate than the other models. For example, with low sampling density (training sample size < 20% of all possible sequences) the three-way model has similar performance as our model, but the performance gap between the two models quickly widens as the training data become dense. The variance component model is able to achieve a final R
                              <sup>
                               2
                              </sup>
                              = 0.83 with 93% of the sequence space assigned as training data (n = 30474), compared with the three-way model R
                              <sup>
                               2
                              </sup>
                              = 0.72. This difference in model performance is consistent with the observation of substantial contribution of higher-order interactions (k > 3), which the low-order regression model is unable to accommodate.
                             </p>
                             <p id="p-55">
                              Another question is the qualitative nature of the genetic interactions captured by our model. We note that the global epistasis model provides a remarkably good fit to the data, considering that it has only a few more parameters than a simple additive model. In Supplemental Figure S3, we see that the global epistasis model approximates the splicing landscape with a sigmoid-like function that maps an unobserved additive trait to the PSI scale. This is as we might expect under a simple biophysical model where each position in the splice site makes a context-independent contribution to the binding energy of the U1 snRNA to the 5′ss, and then this binding energy is mapped via a nonlinear function to PSI [
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ]. However, we also note that this simple model fails to capture some important features of the data, most notably a group of false-negative sequences that are predicted to be non-functional by the global epistasis model but experimentally show moderate to high measured PSI (Supplemental Figure S4A). Using the variance component regression, we were able to accurately predict these outlier sequences (Supplemental Figure S4B). We thus conclude that while the global epistasis model provides a relatively simple first-pass understanding of the landscape of 5′ss activity, our empirical variance component regression is able to capture more of the fine-scale features of this particular genotype-phenotype map.
                             </p>
                             <p id="p-56">
                              Although predictions on held-out data provide one means of testing model performance, a stronger test is to conduct low-throughput experiments to validate the predictions of our method on sequences that were not measured in the original experiment. The SMN1 dataset provides a suitable case study for this application, since the original dataset does not report the PSI of 2036 sequences (6.2% of all possible 5′ss) due to low read counts. To assess the predictive power of our method for these truly missing sequences, we first made predictions for all unsampled sequences using all available data. We then selected 40 unsampled sequences whose predicted values are evenly distributed on the PSI scale. The true PSIs of these sequences were then quantitatively measured using low-throughput radioactive RT-PCR [
                              <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ]; see Materials and Methodsand
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5A
                              </a>
                              . Overall, our method achieved a reasonable qualitative agreement with the low-throughput measurements (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Figure 5B
                              </a>
                              ), but differs systematically in that the transition between nearly 0 and nearly 100 PSI is more rapid in the low-throughput measurement than in our predictions. Intuitively, we can understand the source of this discrepancy in terms of the geometry of the splicing landscape, which features a bimodal distribution of PSIs with separate modes near 0 and 100 [
                              <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ] and a sharp transition between these two sets of sequences in sequence space (Supplemental Figure S3). Because phenotypic observations generalize farther in most regions of sequence space than they do near this boundary between low and high PSI, our method tends to smooth anomalously sharp features of this type. This results in out-of-sample predictions that are more smoothly graded, rather than threshold-like, in the vicinity of this boundary.
                             </p>
                             <div id="F5" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F5.large.jpg?width=800&height=600&carousel=1" title="Manual validation of predicted PSI for 40 unmeasured SMN1 5′ss. (A) Gel images of manually validated sequences. For each lane, the top band corresponds to mRNA product containing exon 7 (exon inclusion), while the bottom band correspond to mRNA product without exon 7 (exon skipping). Percent spliced in (PSI) is indicated below each lane. Gel images are representative of triplicates. (B) Scatterplot showing measured PSI values versus PSI values predicted by the variance component regression. Horizontal error bars correspond to one standard deviation of the posterior distribution. Vertical error bars correspond to one standard deviation around the mean PSI estimated using three replicates in the manual validation. Since unlike the high-throughput measurements, the low-throughput PSIs are inherently restricted to the range 0–100, in this analysis we likewise cap the predicted PSIs to lie in this same range (see Supplemental Figure S5 for the unrestricted predictions)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1224697913" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Manual validation of predicted PSI for 40 unmeasured <em>SMN1</em> 5′ss. (A) Gel images of manually validated sequences. For each lane, the top band corresponds to mRNA product containing exon 7 (exon inclusion), while the bottom band correspond to mRNA product without exon 7 (exon skipping). Percent spliced in (PSI) is indicated below each lane. Gel images are representative of triplicates. (B) Scatterplot showing measured PSI values versus PSI values predicted by the variance component regression. Horizontal error bars correspond to one standard deviation of the posterior distribution. Vertical error bars correspond to one standard deviation around the mean PSI estimated using three replicates in the manual validation. Since unlike the high-throughput measurements, the low-throughput PSIs are inherently restricted to the range 0–100, in this analysis we likewise cap the predicted PSIs to lie in this same range (see Supplemental Figure S5 for the unrestricted predictions).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F5.medium.gif" width="440" height="156"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F5.medium.gif" width="440" height="156"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <p id="p-57" class="first-child">
                                Manual validation of predicted PSI for 40 unmeasured SMN1 5′ss. (A) Gel images of manually validated sequences. For each lane, the top band corresponds to mRNA product containing exon 7 (exon inclusion), while the bottom band correspond to mRNA product without exon 7 (exon skipping). Percent spliced in (PSI) is indicated below each lane. Gel images are representative of triplicates. (B) Scatterplot showing measured PSI values versus PSI values predicted by the variance component regression. Horizontal error bars correspond to one standard deviation of the posterior distribution. Vertical error bars correspond to one standard deviation around the mean PSI estimated using three replicates in the manual validation. Since unlike the high-throughput measurements, the low-throughput PSIs are inherently restricted to the range 0–100, in this analysis we likewise cap the predicted PSIs to lie in this same range (see Supplemental Figure S5 for the unrestricted predictions).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Structure of the SMN1 splicing landscape
                             </h3>
                             <p id="p-58">
                              Besides making accurate phenotypic predictions, it is important to understand the qualitative features of a genotype-phenotype map, both with regard to how the underlying mechanisms result in observed genetic interactions and how these genetic interactions affect other processes, such as molecular evolution and disease. For simple models, such as pairwise interaction models or global epistasis models, extracting these qualitative insights can often be achieved by examining the inferred model parameters. Here, we take a different approach and attempt to understand these major qualitative features by constructing visualizations based on the entire inferred activity landscape. Because we previously conducted a detailed analysis of this type for the GB1 dataset [see
                              <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">
                               59
                              </a>
                              ], we focus here on the inferred activity landscape for the 5′ss.
                             </p>
                             <p id="p-59">
                              In particular, our visualization method [
                              <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">
                               69
                              </a>
                              ] is based on constructing a model of molecular evolution under the assumption that natural selection is acting to preserve the molecular function measured in the assay. The resulting visualization optimally represents the expected time it takes to evolve from one sequence to another (SI Appendix), and naturally produces clusters of genotypes where the long-term evolutionary dynamics are similar for a population starting at any genotype in that cluster (e.g., genotypes on the slopes leading up to a fitness peak will tend to be plotted near that peak). To make such a visualization for our splicing data, we built a model of molecular evolution based on the MAP estimate obtained above (SI Appendix). We then used the subdominant eigenvectors of the transition matrix for this model as coordinates for the genotypes in a low-dimensional representation; these coordinates are known as diffusion axes [
                              <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">
                               70
                              </a>
                              ], since they relate closely to how the probability distribution describing the genotypic state of a population evolving under the combined action of selection, mutation, and genetic drift is likely to diffuse through sequence space [
                              <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">
                               69
                              </a>
                              ,
                              <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">
                               71
                              </a>
                              ].
                             </p>
                             <p id="p-60">
                              The resulting visualization using the first three diffusion axes is shown in
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 6A
                              </a>
                              and
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               B
                              </a>
                              . Here, genotypes are points (colored by the number of times that particular 5′ss is used in the human genome, Materials and Methods) and edges connect genotypes that differ by single point mutations. Remarkably, each of these diffusion axes turns out to have an interpretable meaning. First, diffusion axis 1 separates functional 5′ss (large positive values) from non-functional 5′ss (negative values), as can be seen in
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Figure 6C
                              </a>
                              , which plots the estimated PSI against diffusion axis 1. Second, diffusion axis 2 captures the typical physical location of consensus nucleotides within high-activity 5′ss. Specifically, as one moves up diffusion axis 2, the mean position of consensus nucleotides shifts from the exonic portion (5′-end) of the splice site to the intronic (3′) portion (
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               Figure 6D
                              </a>
                              ). This reflects a previously observed “seesaw” linkage pattern [
                              <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">
                               72
                              </a>
                              –
                              <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">
                               75
                              </a>
                              ] between the intronic and exonic portions of the splice site, where non-consensus nucleotides are typically clustered in one or the other of these regions, but not both. Finally, we find that diffusion axis 3 encodes whether or not mutations are present at the +3 position (
                              <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                               Figure 6B
                              </a>
                              inset), where 5′ss with the consensus A are plotted at negative values on diffusion axis 3, and 5′ss with mutant nucleotides on +3 are plotted at positive values of diffusion axis 3.
                             </p>
                             <div id="F6" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F6.large.jpg?width=800&height=600&carousel=1" title="Visualization of the SMN1 splicing landscape reconstructed using Empirical Variance Component regression. Genotypes are plotted using the dimensionality reduction technique from [69] (see SI Appendix). (A) Visualization of all 32,768 splice sites using diffusion axes 1 and 2. Two splice sites are connected by an edge if they differ by a point mutation. (B) Visualization of all 32,768 splice sites using diffusion axes 1 and 3. (C) PSI versus diffusion axis 1; we see that diffusion axis 1 separates high PSI versus low PSI sequences. Genotypes are colored in (A-C) according to the number of times they are observed as annotated splice sites in the human reference genome (hg38); see panel (A) inset for the color scale and a histogram of the numbers of counts. Gray dots represent sequences not present as functional splice sites (65.9% of all sequences of the form NNN/GYNNNN). (D) Diffusion axis 2 versus the average physical position of the consensus nucleotides of the 818 splice sites with predicted PSI > 80%. (E) Visualization of the 818 splice sites with predicted PSI > 80% using diffusion axes 2 and 3. Genotypes colored as in (A-C). (F) Abstracted version of panel E. Splice sites are grouped by mutational states (consensus vs. mutated) at positions -1, -2, +3, +4, +5, and +6. Each dot corresponds to a group of sequences with a prescribed pattern of consensus or mutated states on the six sites. Two groups are connected by an edge if they differ in mutational state at exactly one site. Gray lines represent differences at position -1, +3, and +5. Black lines represent differences at positions -2, +4, and +6. Only groups containing splice sites with > 80% PSI are shown, resulting in six (in)complete cubes with black edges, each representing a combination of mutational states on the three major sites -1, +3, and +5. The incompleteness of a cube indicates the absence of a combination of mutational states at position -2, +4, and +6. Note that no cubes contain both -1 and +5 mutant states, indicating a major incompatibility between mutations at these two sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1224697913" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Visualization of the <em>SMN1</em> splicing landscape reconstructed using Empirical Variance Component regression. Genotypes are plotted using the dimensionality reduction technique from [69] (see <em>SI Appendix</em>). (A) Visualization of all 32,768 splice sites using diffusion axes 1 and 2. Two splice sites are connected by an edge if they differ by a point mutation. (B) Visualization of all 32,768 splice sites using diffusion axes 1 and 3. (C) PSI versus diffusion axis 1; we see that diffusion axis 1 separates high PSI versus low PSI sequences. Genotypes are colored in (A-C) according to the number of times they are observed as annotated splice sites in the human reference genome (hg38); see panel (A) inset for the color scale and a histogram of the numbers of counts. Gray dots represent sequences not present as functional splice sites (65.9% of all sequences of the form NNN/GYNNNN). (D) Diffusion axis 2 versus the average physical position of the consensus nucleotides of the 818 splice sites with predicted PSI > 80%. (E) Visualization of the 818 splice sites with predicted PSI > 80% using diffusion axes 2 and 3. Genotypes colored as in (A-C). (F) Abstracted version of panel E. Splice sites are grouped by mutational states (consensus vs. mutated) at positions -1, -2, +3, +4, +5, and +6. Each dot corresponds to a group of sequences with a prescribed pattern of consensus or mutated states on the six sites. Two groups are connected by an edge if they differ in mutational state at exactly one site. Gray lines represent differences at position -1, +3, and +5. Black lines represent differences at positions -2, +4, and +6. Only groups containing splice sites with > 80% PSI are shown, resulting in six (in)complete cubes with black edges, each representing a combination of mutational states on the three major sites -1, +3, and +5. The incompleteness of a cube indicates the absence of a combination of mutational states at position -2, +4, and +6. Note that no cubes contain both -1 and +5 mutant states, indicating a major incompatibility between mutations at these two sites.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F6.medium.gif" width="349" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F6.medium.gif" width="349" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/14/2020.10.14.339804/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <p id="p-61" class="first-child">
                                Visualization of the SMN1 splicing landscape reconstructed using Empirical Variance Component regression. Genotypes are plotted using the dimensionality reduction technique from [
                                <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">
                                 69
                                </a>
                                ] (see SI Appendix). (A) Visualization of all 32,768 splice sites using diffusion axes 1 and 2. Two splice sites are connected by an edge if they differ by a point mutation. (B) Visualization of all 32,768 splice sites using diffusion axes 1 and 3. (C) PSI versus diffusion axis 1; we see that diffusion axis 1 separates high PSI versus low PSI sequences. Genotypes are colored in (A-C) according to the number of times they are observed as annotated splice sites in the human reference genome (hg38); see panel (A) inset for the color scale and a histogram of the numbers of counts. Gray dots represent sequences not present as functional splice sites (65.9% of all sequences of the form NNN/GYNNNN). (D) Diffusion axis 2 versus the average physical position of the consensus nucleotides of the 818 splice sites with predicted PSI > 80%. (E) Visualization of the 818 splice sites with predicted PSI > 80% using diffusion axes 2 and 3. Genotypes colored as in (A-C). (F) Abstracted version of panel E. Splice sites are grouped by mutational states (consensus vs. mutated) at positions -1, -2, +3, +4, +5, and +6. Each dot corresponds to a group of sequences with a prescribed pattern of consensus or mutated states on the six sites. Two groups are connected by an edge if they differ in mutational state at exactly one site. Gray lines represent differences at position -1, +3, and +5. Black lines represent differences at positions -2, +4, and +6. Only groups containing splice sites with > 80% PSI are shown, resulting in six (in)complete cubes with black edges, each representing a combination of mutational states on the three major sites -1, +3, and +5. The incompleteness of a cube indicates the absence of a combination of mutational states at position -2, +4, and +6. Note that no cubes contain both -1 and +5 mutant states, indicating a major incompatibility between mutations at these two sites.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-62">
                              To reveal more detailed structures in the sub-landscape of high-activity 5′ss, we focus on the 818 5′ss with PSI>80. These sequences are plotted using diffusion axis 2 and 3 in
                              <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                               Figure 6E
                              </a>
                              .
                              <a id="xref-fig-6-7" class="xref-fig" href="#F6">
                               Figure 6F
                              </a>
                              further groups these high-activity 5′ss by their mutant states (consensus vs. mutant) at six positions −1, −2, +3, +4, +5, and +6 and represents each group by a dot (see also Supplemental Figure S6). On a coarse level, we see two main groups of 5′ss separated along diffusion axis 3 that correspond to sequences with the canonical A (bottom) and mutant nucleotides (predominantly G, top) at position +3, and also a distinction into three main groups along diffusion axis 2, where the three groups correspond to mutant +5 (left), consensus for both +5 and −1 (center), and mutant −1 (right). This separation between sequences containing mutations at positions +5 and −1 (see also
                              <a id="xref-fig-6-8" class="xref-fig" href="#F6">
                               Figure 6A
                              </a>
                              , inset) arises due to a previously noted incompatibility between mutations at these two sites [
                              <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ,
                              <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">
                               73
                              </a>
                              –
                              <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">
                               75
                              </a>
                              ], so that evolutionary trajectories that maintain high levels of splicing must typically wait for a reversion of a +5 mutation before fixing a mutation at the −1 position (and vice versa).
                             </p>
                             <p id="p-63">
                              On a finer scale,
                              <a id="xref-fig-6-9" class="xref-fig" href="#F6">
                               Figure 6F
                              </a>
                              and Supplemental Figure S6 show that each functional combination of mutant states on the three major positions (−1, +3, +5) can be thought of as defining a cube (plotted with dark edges) corresponding to the 2
                              <sup>
                               3
                              </sup>
                              = 8 possible mutant states on the three minor positions (+2, +4, +6). Whereas having either a single −1 or +5 mutation is compatible with having many different combinations of mutations at the three minor positions (complete cubes on the bottom half of
                              <a id="xref-fig-6-10" class="xref-fig" href="#F6">
                               Figure 6F
                              </a>
                              and Supplemental Figure S6), in a +3 mutant background combined with either a −1 or +5 mutation, most combinations of minor mutations result in low activity (two partial cubes on the top half of
                              <a id="xref-fig-6-11" class="xref-fig" href="#F6">
                               Figure 6F
                              </a>
                              and Supplemental Figure S6). While in broad strokes this pattern is consistent with a global epistasis model in that minor mutations are mostly tolerable except in weak genetic backgrounds, the global epistasis model further requires that any specific mutation that is tolerated in a weaker background must also be tolerated in a stronger background. However, we instead observe a pattern where in weak backgrounds mutations are only tolerable if an adjacent major site is already mutated (Supplemental Figure S6). In particular, −2 mutations are often tolerable in a −1+3 background but never tolerable in a +3+5 background, and +6 mutations are often tolerable in a +3+5 background but not in a −1+3 background. More specifically, the deleterious effect of the +6 mutation in the −1+3 mutant background is almost completely abrogated in the +3+5 mutant background (Supplemental Figure S7), consistent with previous findings [
                              <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">
                               74
                              </a>
                              ]. Likewise, we observe that −2 mutations are often tolerated in the −1+3 mutant background (median effect=−18.6 PSI, calculated for sequences with consensus bases on all other positions and with PSI > 80), but have much larger effects when +3+5 are mutated (median effect= 93.8 PSI). The global epistasis model is even more clearly violated by the fact that the +3+5+6 mutant background can also tolerate certain +4 mutations that would not be tolerable in the absence of a +6 mutation (Supplemental Figure S6). Specifically, these +4 mutations result in two highly functional 5′ss: CAG/GUUGUA, which binds to U1 snRNA using a noncanonical geometry known as an asymmetric loop [
                              <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">
                               76
                              </a>
                              ], and AAG/GUGGAC, which does not seem to correspond to any known alternative binding geometry, but occurs as an annotated splice site 14 times in the human genome (Materials and Methods) and has a high level of splicing activity as confirmed via low-throughput validation (Supplemental Figure S8).
                             </p>
                             <p id="p-64">
                              In summary, we conclude that the 5′ss activity landscape contains many qualitatively different types of genetic interactions. At broad scale, the splicing landscape can be understood in light of the global epistasis model, where PSI is modeled as a nonlinear function of an underlying additive phenotype, and interactions between major mutations arise due to the sharp threshold of the global nonlinearity. However, at a finer scale we discover that the effect of a mutation can be strongly modulated by other mutations in ways that are incompatible with the global epistasis model, both due to specific pairwise interactions, such as the interaction between the +5 and +6 positions, but also due to more complex interactions associated with changes in the physical geometry of U1 snRNA binding [
                              <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">
                               76
                              </a>
                              ].
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-65">
                             In this paper, we address the problem of how to model the complex genetic interactions observed in high-throughput mutagenesis experiments in order to predict phenotypic values for unmeasured genotypes. Our method is based on the simple idea that the type and extent of epistasis that we predict outside our observed data should be similar to the type and extent of epistasis observed in the data itself. We show that this information about the type and extent of epistasis can be extracted from how correlations between phenotypic values decay as one moves through sequence space. Specifically, we show that: (1) this same distance correlation function also determines the degree to which our observations of mutational effects, double-mutant epistatic coefficients, and observed interactions among three or more mutants generalize across increasingly distant genetic backgrounds; and (2) the distance correlation function can be parameterized in terms of the fraction of phenotypic variance due to each order of genetic interaction (i.e., the ℓ variance components, where ℓ is the sequence length). By estimating these variance components from the data, we can construct a prior distribution over all possible genotype-phenotype maps that is concentrated on the subset of genotype-phenotype maps where the effects of mutations generalize in the same manner as occurs in our observed data. Conducting Bayesian inference under this prior then produces phenotypic estimates that reflect the belief that the extent and types of epistasis in unobserved regions of sequence space are similar to the extent and type of epistasis in regions of sequence space that we have already observed.
                            </p>
                            <p id="p-66">
                             One way to understand our contribution here is to see it as an integration between practical Gaussian process-based methods for analyzing genotype-phenotype maps [
                             <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">
                              77
                             </a>
                             ] and the classical spectral theory of fitness landscapes [
                             <a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">
                              53
                             </a>
                             ,
                             <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">
                              54
                             </a>
                             ,
                             <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ], which provides the most sophisticated mathematical theory of genetic interactions currently available. Within this theoretical literature, so-called “random field models” identical to the family of priors we propose have been extensively studied [
                             <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             ,
                             <a id="xref-ref-53-7" class="xref-bibr" href="#ref-53">
                              53
                             </a>
                             ,
                             <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ], and we have leveraged this existing knowledge to craft priors that encode comprehensible beliefs about the structure of high-dimensional genotype-phenotype maps.
                            </p>
                            <p id="p-67">
                             Our results here also provide some significant additions to the spectral theory of fitness landscapes that help to provide a more intuitive view of this complex area of mathematical theory. First, we suggest that higher-order epistatic interactions can be qualitatively classified into two types, corresponding to interactions that result in locally positive correlations or locally negative correlations. The idea of an anti-correlated component to a genotype-phenotype map has been discussed previously in the literature in terms of the “eggbox” component [
                             <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">
                              55
                             </a>
                             ] which is perfectly anti-correlated between adjacent genotypes (i.e., whether the phenotypic value is high or low flips with each step one takes through sequence space, similar to the alternating peaks and valleys of an egg carton). Our analysis shows that there is actually a whole set of orders of genetic interaction with a similar character, corresponding to all orders of genetic interaction higher than the average number of differences between two random sequences. However, our main interest is in the components that produce locally positive correlations (which appear more likely to arise under most conceivable physical mechanisms), with the balance between these higher-order locally correlated components controlling how precisely phenotypic correlations decay with increasing Hamming distance.
                            </p>
                            <p id="p-68">
                             Second, we defined a summary statistic γ
                             <sub>
                              k
                             </sub>
                             (d) which, beyond simple phenotypic correlations, measures how mutational effects (k = 1) or epistatic coefficients (k > 1) decay as the distance d between genetic backgrounds increases. The correlation of mutational effects as a function of distance between genetic backgrounds has been previously termed γ(d), which is used to measure the ruggedness of the landscape [
                             <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">
                              55
                             </a>
                             ]. Here we generalize this measure to epistatic coefficients of any order, and show that the distance correlation of epistatic coefficients of order k is in fact determined solely by the components of the landscape of order larger than k (see SI Appendix, where we provide a simple formula showing the relationship between different orders). This result can also help us understand why our method out-performs pairwise and three-way epistatic models. Specifically, we show that models that include only up to k-th order epistatic interactions in fact make the very strong assumption that any observed k-th order interactions generalize across all genetic backgrounds. Incorporating higher-order interactions is then equivalent to relaxing this strong assumption and allowing these lower-order interactions to change as one moves through sequence space.
                            </p>
                            <p id="p-69">
                             Third, in the SI Appendix we provide some additional results to better understand the possible geometries produced by any given order of genetic interaction. In particular, we consider the mean phenotype as a function of Hamming distance from some focal genotype, which is a classical coarse descriptor of genotype-phenotype map and fitness landscape structure [e.g.,
                             <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">
                              78
                             </a>
                             –
                             <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">
                              80
                             </a>
                             ]. We show that for a pure k-th order interaction this mean function is in fact equal to the distance correlation function up to a multiplicative constant. As a consequence of this result, the distance mean function for a model containing up to only k-th order terms must be a k-th order polynomial, so that e.g., in a pairwise interaction model the mean fitness at a given distance from a focal genotype is always a quadratic function of distance. However from a biological perspective, we might often expect mean fitness to take more complex shapes, such as a sigmoid [
                             <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ,
                             <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">
                              81
                             </a>
                             ] (which obviously cannot be well-approximated by a quadratic), providing an explanation for the need to incorporate higher-order interactions in order to provide qualitatively reasonable fits.
                            </p>
                            <p id="p-70">
                             The method we propose here also has some commonalities with minimum epistasis interpolation [
                             <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">
                              59
                             </a>
                             ], another method we recently proposed for phenotypic prediction that includes genetic interactions of all orders, but the two methods differ in their aims. Minimum epistasis interpolation aims to produce a highly conservative reconstruction of the genotype-phenotype map by making the effects of mutations as consistent as possible between adjacent genetic backgrounds. In contrast, empirical variance composnent regression aims to produce a more realistic reconstruction, where the extent and type of epistasis present in the reconstruction should be similar to the extent and type of epistasis present in the data itself. Depending on the needs of the user, both minimum epistasis interpolation and empirical variance component regression can be conducted either in a Bayesian manner or as a form of L
                             <sub>
                              2
                             </sub>
                             -regularized regression [
                             <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">
                              82
                             </a>
                             ] (where our MAP estimate is equivalent to the L
                             <sub>
                              2
                             </sub>
                             regularized solution, SI Appendix). From a regularization perspective, the main difference between these methods is that they penalize different orders of genetic interaction differently, either with a penalty that increases quadratically with the order of interaction, in the case of minimum epistasis interpolation [see also
                             <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">
                              83
                             </a>
                             ], or a penalty determined by the empirically estimated variance components, in the case of empirical variance component regression. In Supplemental Figure S9, we provide a comparison of model performance between these two methods in the GB1 and SMN1 dataset. We observe that empirical variance component regression consistently outperforms minimum epistasis interpolation in both the GB1 and SMN1 datasets. This difference in model performance is likely due to the misalignment between the quadratically increasing penalty imposed by minimum epistatsis interpolation and the actual variance components in the data. Overall, empirical variance component regression is likely the superior method if high predictive performance is desirable. On the other hand, minimum epistasis interpolation is a more conservative approach and has many simple theoretical properties [
                             <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">
                              59
                             </a>
                             ]. Thus, it should be preferred when theoretical guarantees for model behavior are more important.
                            </p>
                            <p id="p-71">
                             One potential limitation of our approach is our choice to select the hyperparameters based on the point estimates supplied by our training data, i.e., by kernel alignment [
                             <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">
                              60
                             </a>
                             ]. It may well be possible to produce more accurate predictions by choosing hyperparameters by maximizing the evidence [
                             <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">
                              47
                             </a>
                             ] or via a hierarchical Bayesian model where we integrate over our uncertainty in the values of these hyperparameters, at the cost of a much greater computational burden. However, an advantage of the empirical Bayes approach is that it provides a clear conceptual separation between the first step of estimating the type and extent of epistasis present in a set of phenotypic observations, and the second step of using these estimates to make additional phenotypic predictions.
                            </p>
                            <p id="p-72">
                             Another limitation concerning empirical variance component regression is that it is unable to explicitly model any overall nonlinearity that may be present in the genotype-phenotype map, i.e., it does not explicitly model nonspecific or global epistasis [
                             <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                              34
                             </a>
                             ,
                             <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ,
                             <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ,
                             <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">
                              84
                             </a>
                             –
                             <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">
                              86
                             </a>
                             ]. Rather, empirical variance component regression must learn any such global structure based on consistent patterns in the observations themselves. For instance, whereas the global epistasis model is able to easily handle the saturation of PSI at 0 and 100%, empirical variance component regression must learn these flatter regions based on the consistently small effects of mutations in particular regions of sequence space, rather than via an overall nonlinearity that is assumed by the structure of the model. Incorporating the possibility of such global nonlinearities would be an important extension to the methods presented here.
                            </p>
                            <p id="p-73">
                             A final limitation concerns the applicability of the method we propose to very large datasets. In our implementation, we take advantage of the isotropic property of the prior distribution (i.e. that covariance depends only on Hamming distance) and the highly symmetric graph structure of sequence space. This allows us to express the covariance matrix and its inverse as polynomials in the highly sparse matrix known as the graph Laplacian, which makes inference possible on sequence spaces containing up to low millions of sequences. However, due to the exponential growth of biological sequence space as a function of sequence length, practically this still limits us to nucleic acid sequences of length 11 or less, and amino acid sequences of length 5 or less. Using the kernel trick [
                             <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">
                              87
                             </a>
                             ], it is possible to work with much longer sequences, but at the cost of only being able to accommodate up to low tens of thousands of observed sequences, due to the resulting dense kernel matrix. Although here we have successfully analyzed datasets that contain tens to hundreds of thousands of sequences, more work is needed to scale the methods proposed here to even larger datasets and sequence spaces.
                            </p>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Low-throughput validation of unsampled SMN1 5′ss
                             </h3>
                             <p id="p-74">
                              To assess the predictive accuracy of our method for the activity of truly unsampled splice sites, we selected 40 5′ss absent in the SMN1 dataset that are evenly distributed on the predicted PSI scale. We quantified the splicing activities of the selected 5′ss in the context of an SMN1 minigene that spans exons 6-8 with the variable 5′ss residing in intron 7. The minigene construct is the same as the one used to generate the high-throughput data [
                              <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ] (minigene sequence is available at
                              <a href="https://github.com/jbkinney/15_splicing">
                               https://github.com/jbkinney/15_splicing
                              </a>
                              ). Specifically, minigenes containing variable 5′ss were inserted in to the pcDNA5/FRT expression vector (Invitrogen). 1 μg of minigene plasmid was then transiently transfected into HeLa cells, which were collected after 48 hr. RNA was isolated from the minigene-expressing HeLa cells using Trizol (Life Technologies) and treated with RQ1 RNase-free DNase (Promega). cDNA was made using Improm-II Reverse Transcription System (Promega), following the manufacturer′s instructions. The splicing isoforms were then amplified with minigene-specific primers (F: CTGGCTAACTAGAGAACCCACTGC; R: GGCAACTAGAAGGCACAGTCG) and
                              <sup>
                               32
                              </sup>
                              P-labeled dCTP using Q5 High-Fidelity DNA Polymerase (New England Biolabs) following the manufacturer′s instructions. PCR products were separated on a 5.5% non-denaturing polyacrylamide gel and were detected using a Typhoon FLA7000 phosphorimager. Finally, we used ImageJ (NIH) to quantify isoform abundance, in the process accounting for cytosine content. All 5′ss were assessed in triplicate.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Acquisition of human 5′ splice sites
                             </h3>
                             <p id="p-75">
                              Human 5′ss were extracted from GENCODE Release 34 (GRCh38.p13) (available at
                              <a href="https://www.gencodegenes.org/human/">
                               https://www.gencodegenes.org/human/
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Code availability
                             </h3>
                             <p id="p-76">
                              A python command-line interface, vcregression, that implements the Empirical Variance Component method described here is available at
                              <a href="https://github.com/davidmccandlish/vcregression">
                               https://github.com/davidmccandlish/vcregression
                              </a>
                              .
                             </p>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-77">
                             This work was funded by NIH grant R35GM133613, an Alfred P. Sloan Research Fellowship (awarded to D.M.M.), as well as funding from the Simons Center for Quantitative Biology at Cold Spring Harbor Laboratory. J.B.K. acknowledges support from NIH grant R35GM133777. M.S.W. and A.R.K. acknowledge support from NIH grant R37GM42699.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               <a class="rev-xref" href="#xref-fn-1-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                1
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 juannanzhou{at}ufl.edu
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-others" id="fn-4">
                              <p id="p-4">
                               J.Z., M.S.W., W.-C.C., A.R.K., J.B.K. and D.M.M. designed research. J.Z., M.S.W., W.-C.C., and J.B.K. performed research. J.Z., M.S.W., W.-C.C., A.R.K., J.B.K. and D.M.M. wrote the paper.
                              </p>
                              <p id="p-5">
                               The authors have no competing interests to declare.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-5">
                              <p id="p-9">
                               <a href="https://github.com/davidmccandlish/vcregression">
                                https://github.com/davidmccandlish/vcregression
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.1" data-doi="10.1038/nrg2452">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Patrick C
                                  </span>
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Epistasis—the essential role of gene interactions in the structure and evolution of genetic systems
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet.
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  855
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  867
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPhillips%26rft.auinit1%253DP.%2BC.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D855%26rft.epage%253D867%26rft.atitle%253DEpistasis--the%2Bessential%2Brole%2Bof%2Bgene%2Binteractions%2Bin%2Bthe%2Bstructure%2Band%2Bevolution%2Bof%2Bgenetic%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2452%26rft_id%253Dinfo%253Apmid%252F18852697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg2452&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18852697&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000260162500010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.2" data-doi="10.1016/j.tig.2014.09.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dmitry A
                                  </span>
                                  <span class="cit-name-surname">
                                   Kondrashov
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fyodor A
                                  </span>
                                  <span class="cit-name-surname">
                                   Kondrashov
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Topological features of rugged fitness landscapes in sequence space
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Trends Genet.
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  24
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  33
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D31%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.09.009%26rft_id%253Dinfo%253Apmid%252F25438718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.09.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25438718&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Júlia
                                  </span>
                                  <span class="cit-name-surname">
                                   Domingo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pablo
                                  </span>
                                  <span class="cit-name-surname">
                                   Baeza-Centurion
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ben
                                  </span>
                                  <span class="cit-name-surname">
                                   Lehner
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  The Causes and Consequences of Genetic Interactions (Epistasis)
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Annu. Rev. Genomics Hum. Genet.
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.4" data-doi="10.1038/nmeth.1492">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Douglas M
                                  </span>
                                  <span class="cit-name-surname">
                                   Fowler
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  High-resolution mapping of protein sequence-function relationships
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  741
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  746
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DFowler%26rft.auinit1%253DD.%2BM.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D741%26rft.epage%253D746%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bprotein%2Bsequence-function%2Brelationships.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1492%26rft_id%253Dinfo%253Apmid%252F20711194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.1492&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20711194&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000281429200020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.5" data-doi="10.1073/pnas.1303309110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lea M
                                  </span>
                                  <span class="cit-name-surname">
                                   Starita
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Activity-enhancing mutations in an E3 ubiquitin ligase identified by high-throughput mutagenesis
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A.
                                 </abbr>
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  14
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  E1263
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E1272
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1303309110%26rft_id%253Dinfo%253Apmid%252F23509263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzE0L0UxMjYzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTQvMjAyMC4xMC4xNC4zMzk4MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.6" data-doi="10.1261/rna.040709.113">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel
                                  </span>
                                  <span class="cit-name-surname">
                                   Melamed
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Deep mutational scanning of an RRM domain of the Saccharomyces cerevisiae poly (A)-binding protein
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  RNA
                                 </abbr>
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1537
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1551
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.040709.113%26rft_id%253Dinfo%253Apmid%252F24064791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8xNTM3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTQvMjAyMC4xMC4xNC4zMzk4MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.7" data-doi="10.1016/j.cub.2014.09.072">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C Anders
                                  </span>
                                  <span class="cit-name-surname">
                                   Olson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas C
                                  </span>
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ren
                                  </span>
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  A comprehensive biophysical description of pairwise epistasis throughout an entire protein domain
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Curr. Biol.
                                 </abbr>
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  22
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  2643
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2651
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D24%26rft.spage%253D2643%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2014.09.072%26rft_id%253Dinfo%253Apmid%252F25455030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cub.2014.09.072&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25455030&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.8" data-doi="10.1093/molbev/msv167">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael B
                                  </span>
                                  <span class="cit-name-surname">
                                   Doud
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Orr
                                  </span>
                                  <span class="cit-name-surname">
                                   Ashenberg
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jesse D
                                  </span>
                                  <span class="cit-name-surname">
                                   Bloom
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Site-specific amino acid preferences are mostly conserved in two closely related protein homologs
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Biol. Evol.
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  2944
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2960
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv167%26rft_id%253Dinfo%253Apmid%252F26226986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msv167&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26226986&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.9" data-doi="10.1126/science.1257360">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anna I
                                  </span>
                                  <span class="cit-name-surname">
                                   Podgornaia
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael T
                                  </span>
                                  <span class="cit-name-surname">
                                   Laub
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Pervasive degeneracy and epistasis in a protein-protein interface
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  347
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6222
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  673
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  677
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPodgornaia%26rft.auinit1%253DA.%2BI.%26rft.volume%253D347%26rft.issue%253D6222%26rft.spage%253D673%26rft.epage%253D677%26rft.atitle%253DPervasive%2Bdegeneracy%2Band%2Bepistasis%2Bin%2Ba%2Bprotein-protein%2Binterface%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1257360%26rft_id%253Dinfo%253Apmid%252F25657251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDcvNjIyMi82NzMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.10" data-doi="10.1038/nature17995">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Karen S
                                  </span>
                                  <span class="cit-name-surname">
                                   Sarkisyan
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Local fitness landscape of the green fluorescent protein
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  533
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7603
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  397
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D533%26rft.spage%253D397%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17995%26rft_id%253Dinfo%253Apmid%252F27193686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature17995&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27193686&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.11" data-doi="10.1016/j.jmb.2016.04.033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Barrett
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinberg
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marc
                                  </span>
                                  <span class="cit-name-surname">
                                   Ostermeier
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Shifting fitness and epistatic landscapes reflect trade-offs along an evolutionary pathway
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  J Mol Biol.
                                 </abbr>
                                 <span class="cit-vol">
                                  428
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  13
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  2730
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2743
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BBiol.%26rft.volume%253D428%26rft.spage%253D2730%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2016.04.033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jmb.2016.04.033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.12" data-doi="10.1073/pnas.1612676113">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Claudia
                                  </span>
                                  <span class="cit-name-surname">
                                   Bank
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  On the (un)predictability of a large intragenic fitness landscape
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A.
                                 </abbr>
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  49
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  14085
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14090
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1612676113%26rft_id%253Dinfo%253Apmid%252F27864516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQ5LzE0MDg1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTQvMjAyMC4xMC4xNC4zMzk4MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.13" data-doi="10.1038/nature23902">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tyler N
                                  </span>
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lora K
                                  </span>
                                  <span class="cit-name-surname">
                                   Picton
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joseph W
                                  </span>
                                  <span class="cit-name-surname">
                                   Thornton
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Alternative evolutionary histories in the sequence space of an ancient protein
                                 </span>
                                 .” In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  549
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7672
                                 </span>
                                 (
                                 <span class="cit-month">
                                  Sept
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  409
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  413
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D549%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature23902%26rft_id%253Dinfo%253Apmid%252F28902834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature23902&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28902834&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Victoria O
                                  </span>
                                  <span class="cit-name-surname">
                                   Pokusaeva
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  An experimental assay of the interactions of amino acids from orthologous sequences shaping a complex fitness landscape
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  PLos Genet.
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e1008079
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLos%2BGenet.%26rft.volume%253D15%26rft.spage%253De1008079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.15" data-doi="10.1126/science.aao5167">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Calin
                                  </span>
                                  <span class="cit-name-surname">
                                   Plesa
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Multiplexed gene synthesis in emulsions for exploring protein functional landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  359
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6373
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  343
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  347
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aao5167%26rft_id%253Dinfo%253Apmid%252F29301959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTkvNjM3My8zNDMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.16" data-doi="10.15252/msb.202010179">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Drew S
                                  </span>
                                  <span class="cit-name-surname">
                                   Tack
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  The genotype-phenotype landscape of an allosteric protein
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular systems biology
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e10179
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bsystems%2Bbiology%26rft.volume%253D17%26rft.spage%253De10179%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.202010179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.15252/msb.202010179&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tyler N
                                  </span>
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Deep mutational scanning of SARS-CoV-2 receptor binding domain reveals constraints on folding and ACE2 binding
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  182
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1295
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1310
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D182%26rft.spage%253D1295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.18" data-doi="10.1101/2021.12.08.471728">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Louisa Gonzalez
                                  </span>
                                  <span class="cit-name-surname">
                                   Somermeyer
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Heterogeneity of the GFP fitness landscape and data-driven protein design
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.12.08.471728
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  .
                                 </span>
                                 eprint:
                                 <a href="https://www.biorxiv.org/content/early/2021/12/09/2021.12.08.471728.full.pdf">
                                  https://www.biorxiv.org/content/early/2021/12/09/2021.12.08.471728.full.pdf
                                 </a>
                                 . url:
                                 <a href="https://www.biorxiv.org/content/early/2021/12/09/2021.12.08.471728">
                                  https://www.biorxiv.org/content/early/2021/12/09/2021.12.08.471728
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.12.08.471728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4xMi4wOC40NzE3Mjh2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzE0LzIwMjAuMTAuMTQuMzM5ODA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.19" data-doi="10.1126/science.1192001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J N
                                  </span>
                                  <span class="cit-name-surname">
                                   Pitt
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A R
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferré-D’Amaré
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Rapid Construction of Empirical RNA Fitness Landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  330
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6002
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  376
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  379
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.1192001
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  .
                                 </span>
                                 url:
                                 <a href="http://www.sciencemag.org/content/330/6002/376.abstract">
                                  http://www.sciencemag.org/content/330/6002/376.abstract
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPitt%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D330%26rft.issue%253D6002%26rft.spage%253D376%26rft.epage%253D379%26rft.atitle%253DRapid%2BConstruction%2Bof%2BEmpirical%2BRNA%2BFitness%2BLandscapes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1192001%26rft_id%253Dinfo%253Apmid%252F20947767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMi8zNzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.20" data-doi="10.1073/pnas.1307604110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   José I
                                  </span>
                                  <span class="cit-name-surname">
                                   Jiménez
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Comprehensive experimental fitness landscape and evolutionary network for small RNA
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A.
                                 </abbr>
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  37
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  14984
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14989
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1307604110%26rft_id%253Dinfo%253Apmid%252F23980164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzM3LzE0OTg0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTQvMjAyMC4xMC4xNC4zMzk4MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.21" data-doi="10.1126/science.aaf0965">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Olga
                                  </span>
                                  <span class="cit-name-surname">
                                   Puchta
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Network of epistatic interactions within a yeast snoRNA
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  352
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6287
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  840
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  844
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf0965%26rft_id%253Dinfo%253Apmid%252F27080103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTIvNjI4Ny84NDAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.22" data-doi="10.1126/science.aae0568">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chuan
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  The fitness landscape of a tRNA gene
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  352
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6287
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  837
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  840
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aae0568%26rft_id%253Dinfo%253Apmid%252F27080104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTIvNjI4Ny84MzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.23" data-doi="10.1038/s41586-018-0170-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Júlia
                                  </span>
                                  <span class="cit-name-surname">
                                   Domingo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Guillaume
                                  </span>
                                  <span class="cit-name-surname">
                                   Diss
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ben
                                  </span>
                                  <span class="cit-name-surname">
                                   Lehner
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Pairwise and higher-order genetic interactions during the evolution of a tRNA
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  558
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7708
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  117
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D558%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0170-7%26rft_id%253Dinfo%253Apmid%252F29849145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0170-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29849145&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Valerie WC
                                  </span>
                                  <span class="cit-name-surname">
                                   Soo
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Fitness landscape of a dynamic RNA structure
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e1009353
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bgenetics%26rft.volume%253D17%26rft.spage%253D1009353e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.25" data-doi="10.1038/nbt.1589">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rupali P
                                  </span>
                                  <span class="cit-name-surname">
                                   Patwardhan
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  High-resolution analysis of DNA regulatory elements by synthetic saturation mutagenesis
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Biotechnology
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1173
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1175
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DPatwardhan%26rft.auinit1%253DR.%2BP.%26rft.volume%253D27%26rft.issue%253D12%26rft.spage%253D1173%26rft.epage%253D1175%26rft.atitle%253DHigh-resolution%2Banalysis%2Bof%2BDNA%2Bregulatory%2Belements%2Bby%2Bsynthetic%2Bsaturation%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1589%26rft_id%253Dinfo%253Apmid%252F19915551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.1589&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19915551&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000272708300034&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.26" data-doi="10.1073/pnas.1004290107">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Justin B
                                  </span>
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Using deep sequencing to characterize the biophysical mechanism of a transcriptional regulatory sequence
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A.
                                 </abbr>
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  9158
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9163
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1004290107%26rft_id%253Dinfo%253Apmid%252F20439748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzIwLzkxNTgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.27" data-doi="10.1101/gr.119628.110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shengdong
                                  </span>
                                  <span class="cit-name-surname">
                                   Ke
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Quantitative evaluation of all hexamers as exonic splicing elements
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1360
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1374
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.119628.110%26rft_id%253Dinfo%253Apmid%252F21659425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTM2MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzE0LzIwMjAuMTAuMTQuMzM5ODA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.28" data-doi="10.1016/j.cell.2015.09.054">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander B
                                  </span>
                                  <span class="cit-name-surname">
                                   Rosenberg
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Learning the sequence determinants of alternative splicing from millions of random sequences
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  163
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  698
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  711
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D698%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.09.054%26rft_id%253Dinfo%253Apmid%252F26496609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.09.054&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26496609&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philippe
                                  </span>
                                  <span class="cit-name-surname">
                                   Julien
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  The complete local genotype–phenotype landscape for the alternative splicing of a human exon
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun.
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  11558
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D11558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.30" data-doi="10.1101/gr.219683.116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shengdong
                                  </span>
                                  <span class="cit-name-surname">
                                   Ke
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Saturation mutagenesis reveals manifold determinants of exon definition
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res.
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  11
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  24
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.219683.116%26rft_id%253Dinfo%253Apmid%252F29242188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjI4LzEvMTEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mandy S
                                  </span>
                                  <span class="cit-name-surname">
                                   Wong
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Justin B
                                  </span>
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Adrian R
                                  </span>
                                  <span class="cit-name-surname">
                                   Krainer
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Quantitative Activity Profile and Context Dependence of All Human 5′ Splice Sites
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Cell
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.32" data-doi="10.1016/j.gde.2013.10.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel M
                                  </span>
                                  <span class="cit-name-surname">
                                   Weinreich
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Should evolutionary geneticists worry about higher-order epistasis?
                                 </span>
                                 ” In:
                                 <abbr class="cit-jnl-abbrev">
                                  Curr. Opin. Genet. Dev.
                                 </abbr>
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  700
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  707
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D23%26rft.spage%253D700%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2013.10.007%26rft_id%253Dinfo%253Apmid%252F24290990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.gde.2013.10.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24290990&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.33" data-doi="10.1016/j.jtbi.2013.05.002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Johannes
                                  </span>
                                  <span class="cit-name-surname">
                                   Neidhart
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ivan G
                                  </span>
                                  <span class="cit-name-surname">
                                   Szendro
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joachim
                                  </span>
                                  <span class="cit-name-surname">
                                   Krug
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Exact results for amplitude spectra of fitness landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  J. Theor. Biol.
                                 </abbr>
                                 <span class="cit-vol">
                                  332
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  218
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  227
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BTheor.%2BBiol.%26rft.volume%253D332%26rft.spage%253D218%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2013.05.002%26rft_id%253Dinfo%253Apmid%252F23685065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2013.05.002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23685065&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000321988500023&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.34" data-doi="10.1002/pro.2897">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tyler N
                                  </span>
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joseph W
                                  </span>
                                  <span class="cit-name-surname">
                                   Thornton
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Epistasis in protein evolution
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Sci.
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1204
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1218
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BSci.%26rft.volume%253D25%26rft.spage%253D1204%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.2897%26rft_id%253Dinfo%253Apmid%252F26833806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/pro.2897&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26833806&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.35" data-doi="10.1534/genetics.116.195214">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zachary R
                                  </span>
                                  <span class="cit-name-surname">
                                   Sailer
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael J
                                  </span>
                                  <span class="cit-name-surname">
                                   Harms
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Detecting High-Order Epistasis in Nonlinear Genotype-Phenotype Maps
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  205
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1079
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1088
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.195214%26rft_id%253Dinfo%253Apmid%252F28100592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS8zLzEwNzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.36" data-doi="10.1371/journal.pcbi.1005541">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zachary R
                                  </span>
                                  <span class="cit-name-surname">
                                   Sailer
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael J
                                  </span>
                                  <span class="cit-name-surname">
                                   Harms
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  High-order epistasis shapes evolutionary trajectories
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput. Biol.
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e1005541
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol.%26rft.volume%253D13%26rft.spage%253De1005541%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1005541%26rft_id%253Dinfo%253Apmid%252F28505183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1005541&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28505183&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.37" data-doi="10.7554/eLife.16965">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas
                                  </span>
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Adaptation in protein fitness landscapes is facilitated by indirect paths
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e16965
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De16965%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.16965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.16965&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.38" data-doi="10.1146/annurev-biophys-070816-033819">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Julian
                                  </span>
                                  <span class="cit-name-surname">
                                   Echave
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Claus O
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilke
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Biophysical models of protein evolution: understanding the patterns of evolutionary sequence divergence
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Annu. Rev. Biophys.
                                 </abbr>
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  85
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  103
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiophys.%26rft.volume%253D46%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biophys-070816-033819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev-biophys-070816-033819&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.39" data-doi="10.1038/s41467-019-09078-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Frank J
                                  </span>
                                  <span class="cit-name-surname">
                                   Poelwijk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Socolich
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rama
                                  </span>
                                  <span class="cit-name-surname">
                                   Ranganathan
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Learning the pattern of epistasis linking genotype and phenotype in a protein
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun.
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-09078-0%26rft_id%253Dinfo%253Apmid%252F30602773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-019-09078-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30602773&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aneth S
                                  </span>
                                  <span class="cit-name-surname">
                                   Canale
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Evolutionary mechanisms studied through protein fitness landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Curr. Opin. Struct. Biol.
                                 </abbr>
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  141
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  148
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BStruct.%2BBiol.%26rft.volume%253D48%26rft.spage%253D141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.41" data-doi="10.1007/s10955-018-1975-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Weinreich
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  The Influence of Higher-Order Epistasis on Biological Fitness Landscape Topography
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  J. Stat. Phys.
                                 </abbr>
                                 <span class="cit-vol">
                                  172
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  208
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  225
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  00224715
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1007/s10955-018-1975-3
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  .
                                 </span>
                                 url:
                                 <a href="https://doi.org/10.1007/s10955-018-1975-3">
                                  https://doi.org/10.1007/s10955-018-1975-3
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BPhys.%26rft.volume%253D172%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10955-018-1975-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10955-018-1975-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.42" data-doi="10.1016/j.sbi.2017.10.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jay F
                                  </span>
                                  <span class="cit-name-surname">
                                   Storz
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Compensatory mutations and epistasis for protein function
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Curr. Opin. Struct. Biol.
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  18
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BStruct.%2BBiol.%26rft.volume%253D50%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2017.10.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.sbi.2017.10.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Charlotte M
                                  </span>
                                  <span class="cit-name-surname">
                                   Miton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Karol
                                  </span>
                                  <span class="cit-name-surname">
                                   Buda
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nobuhiko
                                  </span>
                                  <span class="cit-name-surname">
                                   Tokuriki
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Epistasis and intramolecular networks in protein evolution
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Current opinion in structural biology
                                 </abbr>
                                 <span class="cit-vol">
                                  69
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  160
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  168
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.volume%253D69%26rft.spage%253D160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gloria
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Higher-order epistasis shapes the fitness landscape of a xenobiotic-degrading enzyme
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Chemical Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1120
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1128
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BChemical%2BBiology%26rft.volume%253D15%26rft.spage%253D1120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.45" data-doi="10.1073/pnas.1804015115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jakub
                                  </span>
                                  <span class="cit-name-surname">
                                   Otwinowski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   McCandlish
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua B
                                  </span>
                                  <span class="cit-name-surname">
                                   Plotkin
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Inferring the shape of global epistasis
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  32
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  E7550
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E7558
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1804015115%26rft_id%253Dinfo%253Apmid%252F30037990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzMyL0U3NTUwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTQvMjAyMC4xMC4xNC4zMzk4MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [46].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gautam
                                  </span>
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael M
                                  </span>
                                  <span class="cit-name-surname">
                                   Desai
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Global epistasis emerges from a generic model of a complex trait
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e64740
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D10%26rft.spage%253De64740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [47].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.14.339804v3.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carl Edward
                                  </span>
                                  <span class="cit-name-surname">
                                   Rasmussen
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher K I
                                  </span>
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Gaussian processes for machine learning
                                 </span>
                                 . Vol.
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  MIT press Cambridge
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [48].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.14.339804v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bradley P
                                  </span>
                                  <span class="cit-name-surname">
                                   Carlin
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas A
                                  </span>
                                  <span class="cit-name-surname">
                                   Louis
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Bayes and empirical Bayes methods for data analysis
                                 </span>
                                 . Vol.
                                 <span class="cit-vol">
                                  88
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Chapman & Hall/CRC Boca Raton
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [49].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.49">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    Steve
                                   </span>
                                   <span class="cit-name-surname">
                                    Brooks
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Radford M
                                  </span>
                                  <span class="cit-name-surname">
                                   Neal
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-chapter-title">
                                  MCMC using Hamiltonian dynamics
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Handbook of Markov chain Monte Carlo
                                 </abbr>
                                 . Ed. by
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   Steve
                                  </span>
                                  <span class="cit-name-surname">
                                   Brooks
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  113
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  162
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [50].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R A
                                  </span>
                                  <span class="cit-name-surname">
                                   Fisher
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  The Correlation Between Relatives on the Supposition of Mendelian Inheritance
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Trans R Soc Edinburgh
                                 </abbr>
                                 <span class="cit-vol">
                                  52
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  02
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1918
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  399
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  433
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrans%2BR%2BSoc%2BEdinburgh%26rft.volume%253D52%26rft.spage%253D399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [51].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.51" data-doi="10.1038/ng.795">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Trevor
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinkley
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  A systems analysis of mutational effects in HIV-1 protease and reverse transcriptase
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  43
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  487
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  489
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHinkley%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D487%26rft.epage%253D489%26rft.atitle%253DA%2Bsystems%2Banalysis%2Bof%2Bmutational%2Beffects%2Bin%2BHIV-1%2Bprotease%2Band%2Breverse%2Btranscriptase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.795%26rft_id%253Dinfo%253Apmid%252F21441930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.795&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21441930&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [52].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.52">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Manfred
                                  </span>
                                  <span class="cit-name-surname">
                                   Eigen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John
                                  </span>
                                  <span class="cit-name-surname">
                                   McCaskill
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                  <span class="cit-name-surname">
                                   Schuster
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  The molecular quasi-species
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Adv. Chem. Phys
                                 </abbr>
                                 <span class="cit-vol">
                                  75
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1989
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  149
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  263
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAdv.%2BChem.%2BPhys%26rft.volume%253D75%26rft.spage%253D149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [53].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.53" data-doi="10.1007/s002850050156">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter F
                                  </span>
                                  <span class="cit-name-surname">
                                   Stadler
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert
                                  </span>
                                  <span class="cit-name-surname">
                                   Happel
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Random field models for fitness landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  J. Math. Biol.
                                 </abbr>
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  435
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  478
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BMath.%2BBiol.%26rft.volume%253D38%26rft.spage%253D435%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002850050156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s002850050156&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [54].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.54" data-doi="10.1007/BF00216965">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edward D
                                  </span>
                                  <span class="cit-name-surname">
                                   Weinberger
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Fourier and Taylor series on fitness landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Biol Cybern
                                 </abbr>
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  321
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  330
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiol%2BCybern%26rft.volume%253D65%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00216965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/BF00216965&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [55].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.55" data-doi="10.1016/j.jtbi.2016.01.037">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Luca
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferretti
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Measuring epistasis in fitness landscapes: The correlation of fitness effects of mutations
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Theoretical Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  396
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  132
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  143
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  10958541
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.jtbi.2016.01.037
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.volume%253D396%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2016.01.037%26rft_id%253Dinfo%253Apmid%252F26854875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2016.01.037&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26854875&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [56].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.56">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert
                                  </span>
                                  <span class="cit-name-surname">
                                   Happel
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter F
                                  </span>
                                  <span class="cit-name-surname">
                                   Stadler
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Canonical approximation of fitness landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Complexity
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  53
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  58
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComplexity%26rft.volume%253D2%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [57].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.57">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter F
                                  </span>
                                  <span class="cit-name-surname">
                                   Stadler
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Landscapes and their correlation functions
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  J. Math. Chem.
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  45
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BMath.%2BChem.%26rft.volume%253D20%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [58].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.14.339804v3.58">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter F
                                  </span>
                                  <span class="cit-name-surname">
                                   Stadler
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-chapter-title">
                                  Fitness landscapes
                                 </span>
                                 ”. In:
                                 <span class="cit-source">
                                  Biological Evolution and Statistical Physics
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  183
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  204
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [59].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.59">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Juannan
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David M
                                  </span>
                                  <span class="cit-name-surname">
                                   McCandlish
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Minimum epistasis interpolation for sequence-function relationships
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [60].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tinghua
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dongyan
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shengfeng
                                  </span>
                                  <span class="cit-name-surname">
                                   Tian
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  An overview of kernel alignment and its applications
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Artificial Intelligence Review
                                 </abbr>
                                 <span class="cit-vol">
                                  43
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  179
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  192
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DArtificial%2BIntelligence%2BReview%26rft.volume%253D43%26rft.spage%253D179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [61].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John P
                                  </span>
                                  <span class="cit-name-surname">
                                   Cunningham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Krishna V
                                  </span>
                                  <span class="cit-name-surname">
                                   Shenoy
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maneesh
                                  </span>
                                  <span class="cit-name-surname">
                                   Sahani
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Fast Gaussian process methods for point process intensity estimation
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the 25th international conference on Machine learning
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  192
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  199
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [62].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.62">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter F
                                  </span>
                                  <span class="cit-name-surname">
                                   Stadler
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Random walks and orthogonal functions associated with highly symmetric graphs
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Discrete mathematics
                                 </abbr>
                                 <span class="cit-vol">
                                  145
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1-3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  229
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  237
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDiscrete%2Bmathematics%26rft.volume%253D145%26rft.spage%253D229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [63].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.63">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   DG
                                  </span>
                                  <span class="cit-name-surname">
                                   Higman
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Intersection matrices for finite permutation groups
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Algebra
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1967
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  22
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  42
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAlgebra%26rft.volume%253D6%26rft.spage%253D22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [64].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.64" data-doi="10.1007/s10955-018-1975-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel M
                                  </span>
                                  <span class="cit-name-surname">
                                   Weinreich
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  The influence of higher-order epistasis on biological fitness landscape topography
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Statistical Physics
                                 </abbr>
                                 <span class="cit-vol">
                                  172
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  208
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  225
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BPhysics%26rft.volume%253D172%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10955-018-1975-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10955-018-1975-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [65].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.65">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Amirali
                                  </span>
                                  <span class="cit-name-surname">
                                   Aghazadeh
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Epistatic Net allows the sparse spectral regularization of deep neural networks for inferring fitness functions
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D12%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [66].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference [66] in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.66">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David H
                                  </span>
                                  <span class="cit-name-surname">
                                   Brookes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Amirali
                                  </span>
                                  <span class="cit-name-surname">
                                   Aghazadeh
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jennifer
                                  </span>
                                  <span class="cit-name-surname">
                                   Listgarten
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  On the sparsity of fitness functions and implications for learning
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  119
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [67].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference [67] in text" id="ref-67">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.67">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hui
                                  </span>
                                  <span class="cit-name-surname">
                                   Zou
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Trevor
                                  </span>
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Regularization and variable selection via the elastic net
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the royal statistica
                                 </abbr>
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  301
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  320
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2Broyal%2Bstatistica%26rft.volume%253D67%26rft.spage%253D301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [68].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference [68] in text" id="ref-68">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.68" data-doi="10.7554/eLife.04986">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yasushi
                                  </span>
                                  <span class="cit-name-surname">
                                   Kondo
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Crystal structure of human U1 snRNP, a small nuclear ribonucleoprotein particle, reveals the mechanism of 5′splice site recognition
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e04986
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D4%26rft.spage%253De04986%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.04986%26rft_id%253Dinfo%253Apmid%252F25555158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.04986&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25555158&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [69].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference [69] in text" id="ref-69">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.69" data-doi="10.1111/j.1558-5646.2011.01236.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David M
                                  </span>
                                  <span class="cit-name-surname">
                                   McCandlish
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Visualizing fitness landscapes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1544
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1558
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DMcCandlish%26rft.auinit1%253DD.%2BM.%26rft.volume%253D65%26rft.issue%253D6%26rft.spage%253D1544%26rft.epage%253D1558%26rft.atitle%253DVisualizing%2Bfitness%2Blandscapes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01236.x%26rft_id%253Dinfo%253Apmid%252F21644947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01236.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21644947&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [70].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference [70] in text" id="ref-70">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.70">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ronald R
                                  </span>
                                  <span class="cit-name-surname">
                                   Coifman
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stéphane
                                  </span>
                                  <span class="cit-name-surname">
                                   Lafon
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Diffusion maps
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Appl. and Comp. Harmonic Anal.
                                 </abbr>
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  5
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAppl.%2Band%2BComp.%2BHarmonic%2BAnal.%26rft.volume%253D21%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [71].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference [71] in text" id="ref-71">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.71">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David M
                                  </span>
                                  <span class="cit-name-surname">
                                   McCandlish
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Long-term evolution on complex fitness landscapes when mutation is weak
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Heredity
                                 </abbr>
                                 <span class="cit-vol">
                                  121
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  449
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  465
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D121%26rft.spage%253D449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [72].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference [72] in text" id="ref-72">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.72" data-doi="10.1101/gr.6859308">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xavier
                                  </span>
                                  <span class="cit-name-surname">
                                   Roca
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Features of 5′-splice-site efficiency derived from disease-causing mutations and comparative genomics
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  77
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  87
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6859308%26rft_id%253Dinfo%253Apmid%252F18032726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE4LzEvNzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNC8yMDIwLjEwLjE0LjMzOTgwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [73].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference [73] in text" id="ref-73">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.73" data-doi="10.1261/rna.5196404">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ido
                                  </span>
                                  <span class="cit-name-surname">
                                   Carmel
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Comparative analysis detects dependencies among the 5′splice-site positions
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  RNA
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  828
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  840
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DCARMEL%26rft.auinit1%253DI.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D828%26rft.epage%253D840%26rft.atitle%253DComparative%2Banalysis%2Bdetects%2Bdependencies%2Bamong%2Bthe%2B5%2527%2Bsplice-site%2Bpositions%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.5196404%26rft_id%253Dinfo%253Apmid%252F15100438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjEwLzUvODI4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTQvMjAyMC4xMC4xNC4zMzk4MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [74].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference [74] in text" id="ref-74">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.74" data-doi="10.1006/jmbi.1997.0951">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chris
                                  </span>
                                  <span class="cit-name-surname">
                                   Burge
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Samuel
                                  </span>
                                  <span class="cit-name-surname">
                                   Karlin
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Prediction of complete gene structures in human genomic DNA
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  268
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  78
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  94
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBurge%26rft.auinit1%253DC.%26rft.volume%253D268%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D94%26rft.atitle%253DPrediction%2Bof%2Bcomplete%2Bgene%2Bstructures%2Bin%2Bhuman%2Bgenomic%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.0951%26rft_id%253Dinfo%253Apmid%252F9149143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.0951&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9149143&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1997WV87100009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [75].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference [75] in text" id="ref-75">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.75" data-doi="10.1089/1066527041410418">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gene
                                  </span>
                                  <span class="cit-name-surname">
                                   Yeo
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher B
                                  </span>
                                  <span class="cit-name-surname">
                                   Burge
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of computational biology
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2-3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  377
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  394
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DYeo%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D2-3%26rft.spage%253D377%26rft.epage%253D394%26rft.atitle%253DMaximum%2Bentropy%2Bmodeling%2Bof%2Bshort%2Bsequence%2Bmotifs%2Bwith%2Bapplications%2Bto%2BRNA%2Bsplicing%2Bsignals.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F1066527041410418%26rft_id%253Dinfo%253Apmid%252F15285897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1089/1066527041410418&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15285897&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000222588300010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [76].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference [76] in text" id="ref-76">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.76" data-doi="10.1093/nar/gkw163">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jiazi
                                  </span>
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Noncanonical registers and base pairs in human 5′splice-site selection
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  3908
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3921
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw163%26rft_id%253Dinfo%253Apmid%252F26969736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkw163&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26969736&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [77].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference [77] in text" id="ref-77">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.77" data-doi="10.1073/pnas.1215251110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philip A
                                  </span>
                                  <span class="cit-name-surname">
                                   Romero
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andreas
                                  </span>
                                  <span class="cit-name-surname">
                                   Krause
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Frances H
                                  </span>
                                  <span class="cit-name-surname">
                                   Arnold
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Navigating the protein fitness landscape with Gaussian processes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A.
                                 </abbr>
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  E193
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E201
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215251110%26rft_id%253Dinfo%253Apmid%252F23277561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzMvRTE5MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzE0LzIwMjAuMTAuMTQuMzM5ODA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [78].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference [78] in text" id="ref-78">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.78">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexey S
                                  </span>
                                  <span class="cit-name-surname">
                                   Kondrashov
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Selection against harmful mutations in large sexual and asexual populations
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics Research
                                 </abbr>
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1982
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  325
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  332
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D40%26rft.spage%253D325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [79].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.79" data-doi="10.1038/37108">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Santiago F
                                  </span>
                                  <span class="cit-name-surname">
                                   Elena
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard E
                                  </span>
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Test of synergistic interactions among deleterious mutations in bacteria
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  390
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6658
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  395
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  398
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DElena%26rft.auinit1%253DS.%2BF.%26rft.volume%253D390%26rft.issue%253D6658%26rft.spage%253D395%26rft.epage%253D398%26rft.atitle%253DTest%2Bof%2Bsynergistic%2Binteractions%2Bamong%2Bdeleterious%2Bmutations%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F37108%26rft_id%253Dinfo%253Apmid%252F9389477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/37108&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9389477&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1997YH54900061&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [80].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference [80] in text" id="ref-80">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.80" data-doi="10.1126/science.1101786">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sebastian
                                  </span>
                                  <span class="cit-name-surname">
                                   Bonhoeffer
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Evidence for positive epistasis in HIV-1
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  306
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5701
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1547
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1550
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBonhoeffer%26rft.auinit1%253DS.%26rft.volume%253D306%26rft.issue%253D5701%26rft.spage%253D1547%26rft.epage%253D1550%26rft.atitle%253DEvidence%2Bfor%2BPositive%2BEpistasis%2Bin%2BHIV-1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101786%26rft_id%253Dinfo%253Apmid%252F15567861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTcwMS8xNTQ3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTQvMjAyMC4xMC4xNC4zMzk4MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [81].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference [81] in text" id="ref-81">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.81" data-doi="10.1038/nature05385">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shimon
                                  </span>
                                  <span class="cit-name-surname">
                                   Bershtein
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Robustness–epistasis link shapes the fitness landscape of a randomly drifting protein
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  444
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7121
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  929
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  932
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBershtein%26rft.auinit1%253DS.%26rft.volume%253D444%26rft.issue%253D7121%26rft.spage%253D929%26rft.epage%253D932%26rft.atitle%253DRobustness-epistasis%2Blink%2Bshapes%2Bthe%2Bfitness%2Blandscape%2Bof%2Ba%2Brandomly%2Bdrifting%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05385%26rft_id%253Dinfo%253Apmid%252F17122770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature05385&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17122770&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000242805400056&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [82].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference [82] in text" id="ref-82">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.14.339804v3.82">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander J
                                  </span>
                                  <span class="cit-name-surname">
                                   Smola
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Risi
                                  </span>
                                  <span class="cit-name-surname">
                                   Kondor
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-chapter-title">
                                  Kernels and regularization on graphs
                                 </span>
                                 ”. In:
                                 <span class="cit-source">
                                  COLT
                                 </span>
                                 . Vol.
                                 <span class="cit-vol">
                                  2777
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  Springer
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  144
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  158
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCOLT%26rft.volume%253D2777%26rft.spage%253D144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [83].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference [83] in text" id="ref-83">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.83">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wei-Chia
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Field-theoretic density estimation for biological sequence space with applications to 5 splice site diversity and aneuploidy in cancer
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  40
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [84].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference [84] in text" id="ref-84">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.84">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Justin B
                                  </span>
                                  <span class="cit-name-surname">
                                   Kinney
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David M
                                  </span>
                                  <span class="cit-name-surname">
                                   McCandlish
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Massively Parallel Assays and Quantitative Sequence– Function Relationships
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Annu Rev Genomics Hum Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [85].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.14.339804v3.85" data-doi="10.1038/nature17995">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Karen S
                                  </span>
                                  <span class="cit-name-surname">
                                   Sarkisyan
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Local fitness landscape of the green fluorescent protein
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  533
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7603
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  397
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  401
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D533%26rft.spage%253D397%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17995%26rft_id%253Dinfo%253Apmid%252F27193686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature17995&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27193686&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F14%2F2020.10.14.339804.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [86].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference [86] in text" id="ref-86">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.14.339804v3.86">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ammar
                                  </span>
                                  <span class="cit-name-surname">
                                   Tareen
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  MAVE-NN: Quantitative Modeling of Genotype-Phenotype Maps as Information Bottlenecks
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  BioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [87].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference [87] in text" id="ref-87">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.14.339804v3.87">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bernhard
                                  </span>
                                  <span class="cit-name-surname">
                                   Schölkopf
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ralf
                                  </span>
                                  <span class="cit-name-surname">
                                   Herbrich
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alex J
                                  </span>
                                  <span class="cit-name-surname">
                                   Smola
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-chapter-title">
                                  A generalized representer theorem
                                 </span>
                                 ”. In:
                                 <span class="cit-source">
                                  International conference on computational learning theory
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  Springer
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  416
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  426
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.10.14.339804v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.03.12.484105v1" title="Invasiveness of Cancer Populations in a Two-dimensional Percolation cluster: a Stochastic Mathematical Approach" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.14.484209v1" title="Inorganic sulfur fixation via a new homocysteine synthase allows yeast cells to cooperatively compensate for methionine auxotrophy" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 14, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.10.14.339804v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.10.14.339804v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/03/14/2020.10.14.339804.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.10.14.339804v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.10.14.339804v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Higher-order epistasis and phenotypic prediction
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2446672"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-sGwtKaupQR-NDL2908zK2EarGPwMi2BSGAKd5NOwFpE"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484482327"/>
                                   <input type="hidden" name="captcha_token" value="28387fbbdbf1e43826be01fb96968229"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2446672--3" data-node-nid="2446672" id="node-2446672--4246597134" data-pisa="biorxiv;2020.10.14.339804v3" data-pisa-master="biorxiv;2020.10.14.339804" data-seqnum="2446672" data-apath="/biorxiv/early/2022/03/14/2020.10.14.339804.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Higher-order epistasis and phenotypic prediction
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Juannan
                                  </span>
                                  <span class="nlm-surname">
                                   Zhou
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Mandy S.
                                  </span>
                                  <span class="nlm-surname">
                                   Wong
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Wei-Chia
                                  </span>
                                  <span class="nlm-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Adrian R.
                                  </span>
                                  <span class="nlm-surname">
                                   Krainer
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Justin B.
                                  </span>
                                  <span class="nlm-surname">
                                   Kinney
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   David M.
                                  </span>
                                  <span class="nlm-surname">
                                   McCandlish
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.10.14.339804;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.10.14.339804
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.10.14.339804v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&title=Higher-order%20epistasis%20and%20phenotypic%20prediction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&title=Higher-order%20epistasis%20and%20phenotypic%20prediction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&text=Higher-order%20epistasis%20and%20phenotypic%20prediction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&t=Higher-order%20epistasis%20and%20phenotypic%20prediction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&title=Higher-order%20epistasis%20and%20phenotypic%20prediction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&title=Higher-order%20epistasis%20and%20phenotypic%20prediction&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&title=Higher-order%20epistasis%20and%20phenotypic%20prediction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2446672--5" data-node-nid="2446672" id="citation-node-2446672--6851022014" data-pisa="biorxiv;2020.10.14.339804v3" data-pisa-master="biorxiv;2020.10.14.339804" data-seqnum="2446672" data-apath="/biorxiv/early/2022/03/14/2020.10.14.339804.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Higher-order epistasis and phenotypic prediction
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Juannan
                                      </span>
                                      <span class="nlm-surname">
                                       Zhou
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Mandy S.
                                      </span>
                                      <span class="nlm-surname">
                                       Wong
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Wei-Chia
                                      </span>
                                      <span class="nlm-surname">
                                       Chen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Adrian R.
                                      </span>
                                      <span class="nlm-surname">
                                       Krainer
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Justin B.
                                      </span>
                                      <span class="nlm-surname">
                                       Kinney
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       David M.
                                      </span>
                                      <span class="nlm-surname">
                                       McCandlish
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.10.14.339804;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.10.14.339804
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Higher-order%20epistasis%20and%20phenotypic%20prediction&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Zhou%2C+Juannan&rft.au=Wong%2C+Mandy+S.&rft.au=Chen%2C+Wei-Chia&rft.au=Krainer%2C+Adrian+R.&rft.au=Kinney%2C+Justin+B.&rft.au=McCandlish%2C+David+M.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2446672/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2446672/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2446672/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2446672/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2446672/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2446672/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2446672/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2446672/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2446672/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2446672/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2446672/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2446672/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&count=horizontal&via=&text=Higher-order%20epistasis%20and%20phenotypic%20prediction&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.10.14.339804v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.10.14.339804v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Evolutionary Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
