<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.11.16.382945v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.11.16.382945v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.11.16.382945v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/01/04/2020.11.16.382945.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.11.16.382945v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Population genetics of transposable element load: a mechanistic account of observed overdispersion"/>
  <meta name="DC.Identifier" content="10.1101/2020.11.16.382945"/>
  <meta name="DC.Date" content="2022-01-04"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="In an empirical analysis of transposable element (TE) abundance within natural populations of Mimulus guttatus and Drosophila melanogaster , we found a surprisingly high variance of TE count (e.g., variance-to-mean ratio on the order of 10 to 300). To obtain insight regarding those evolutionary genetic mechanisms that are may underlie the overdispersed population distributions of TE abundance, we developed a mathematical model of TE population genetics that includes the dynamics of element proliferation and purifying selection on TE load. The modeling approach begins with a master equation for a birth-death process and extends the predictions of the classical theory of TE dynamics in several ways. In particular, moment-based analyses of time-dependent and equilibrium population distributions of TE load reveal that overdispersion is likely to arise via copy-and-paste proliferation dynamics, especially when the elementary processes of proliferation and excision are first-order and approximately balanced. Parameter studies and analytic work confirm this result and further suggest that overdispersed population distributions of TE abundance are unlikely to be a consequence of purifying selection on total element load.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Ronald D. Smith"/>
  <meta name="DC.Contributor" content="Joshua R. Puzey"/>
  <meta name="DC.Contributor" content="Gregory D. Conradi Smith"/>
  <meta name="article:published_time" content="2022-01-04"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Population genetics of transposable element load: a mechanistic account of observed overdispersion"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>In an empirical analysis of transposable element (TE) abundance within natural populations of <i>Mimulus guttatus</i> and <i>Drosophila melanogaster</i>, we found a surprisingly high variance of TE count (e.g., variance-to-mean ratio on the order of 10 to 300). To obtain insight regarding those evolutionary genetic mechanisms that are may underlie the overdispersed population distributions of TE abundance, we developed a mathematical model of TE population genetics that includes the dynamics of element proliferation and purifying selection on TE load. The modeling approach begins with a master equation for a birth-death process and extends the predictions of the classical theory of TE dynamics in several ways. In particular, moment-based analyses of time-dependent and equilibrium population distributions of TE load reveal that overdispersion is likely to arise via copy-and-paste proliferation dynamics, especially when the elementary processes of proliferation and excision are first-order and approximately balanced. Parameter studies and analytic work confirm this result and further suggest that overdispersed population distributions of TE abundance are unlikely to be a consequence of purifying selection on total element load.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.11.16.382945v3"/>
  <meta name="citation_id" content="2020.11.16.382945v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.11.16.382945v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.16.382945v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.16.382945v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.11.16.382945"/>
  <meta name="citation_num_pages" content="53"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.11.16.382945"/>
  <meta name="citation_author" content="Ronald D. Smith"/>
  <meta name="citation_author_institution" content="Department of Applied Science, William & Mary"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2736-0092"/>
  <meta name="citation_author" content="Joshua R. Puzey"/>
  <meta name="citation_author_institution" content="Department of Biology, William & Mary"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8019-9993"/>
  <meta name="citation_author" content="Gregory D. Conradi Smith"/>
  <meta name="citation_author_institution" content="Department of Applied Science, William & Mary"/>
  <meta name="citation_author_email" content="greg@wm.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1054-6790"/>
  <meta name="citation_reference" content="Carolina Bartolomé, Xulio Maside, and Brian Charlesworth. On the abundance and distribution of transposable elements in the genome of Drosophila melanogaster. Molecular Biology and Evolution, 19(6):926–937, 2002."/>
  <meta name="citation_reference" content="Guillaume Bourque, Kathleen H Burns, Mary Gehring, Vera Gorbunova, Andrei Seluanov, Molly Hammell, Michaël Imbeault, Zsuzsanna Izsvák, Henry L Levin, Todd S Macfarlan, et al. Ten things you should know about transposable elements. Genome Biology, 19(1):199, 2018."/>
  <meta name="citation_reference" content="John F.Y. Brookfield and Richard M. Badge. Population genetics models of transposable elements. Genetica, 100(1-3):281–294, 1997."/>
  <meta name="citation_reference" content="Michael George Bulmer. The Mathematical Theory of Quantitative Genetics. Clarendon Press, 1980."/>
  <meta name="citation_reference" content="Brian Charlesworth and Deborah Charlesworth. The population dynamics of transposable elements. Genetics Research, 42(1):1–27, 1983."/>
  <meta name="citation_reference" content="Brian Charlesworth and Deborah Charlesworth. Elements of Evolutionary Genetics. W. H. Freeman, January 2010."/>
  <meta name="citation_reference" content="Julie M Cridland, Stuart J Macdonald, Anthony D Long, and Kevin R Thornton. Abundance and distribution of transposable elements in two Drosophila QTL mapping resources. Molecular Biology and Evolution, 30(10):2311–2327, 2013."/>
  <meta name="citation_reference" content="Gregory Deceliere. The dynamics of transposable elements in structured populations. Genetics, 169(1):467–474, 2004."/>
  <meta name="citation_reference" content="Laurent Duret, Gabriel Marais, and Christian Biémont. Transposons but not retrotransposons are located preferentially in regions of high recombination rate in Caenorhabditis elegans. Genetics, 156(4):1661–1669, 2000."/>
  <meta name="citation_reference" content="Crispin Gardiner. Stochastic Methods: A Handbook for the Natural and Social Sciences. Springer, 4th edition, 2009."/>
  <meta name="citation_reference" content="John H. Gillespie. Population Genetics: A Concise Guide. The Johns Hopkins University Press, 2004."/>
  <meta name="citation_reference" content="Dan Graur. Molecular and Genome Evolution. Sinauer Associates, 2016."/>
  <meta name="citation_reference" content="Uffe Hellsten, Kevin M Wright, Jerry Jenkins, Shengqiang Shu, Yaowu Yuan, Susan R Wessler, Jeremy Schmutz, John H Willis, and Daniel S Rokhsar. Fine-scale variation in meiotic recombination in mimulus inferred from population shotgun sequencing. Proceedings of the National Academy of Sciences, 110(48):19478–19482, 2013."/>
  <meta name="citation_reference" content="Tyler V Kent, Jasmina Uzunović, and Stephen I Wright. Coevolution between transposable elements and recombination. Philosophical Transactions of the Royal Society B: Biological Sciences, 372(1736):20160458, 2017."/>
  <meta name="citation_reference" content="Arnaud Le Rouzic and Grégory Deceliere. Models of the population genetics of transposable elements. Genetics Research, 85(3):171–181, 2005."/>
  <meta name="citation_reference" content="Trudy FC Mackay. Transposable elements and fitness in Drosophila melanogaster. Genome, 31(1): 284–295, 1989."/>
  <meta name="citation_reference" content="Ryan E Mills, E Andrew Bennett, Rebecca C Iskow, and Scott E Devine. Which transposable elements are active in the human genome? Trends in Genetics, 23(4):183–191, 2007."/>
  <meta name="citation_reference" content="EG Pasyukova, SV Nuzhdin, TV Morozova, and TFC Mackay. Accumulation of transposable elements in the genome of Drosophila melanogaster is associated with a decrease in fitness. Journal of Heredity, 95(4):284–290, 2004."/>
  <meta name="citation_reference" content="Nathan M Springer, Kai Ying, Yan Fu, Tieming Ji, Cheng-Ting Yeh, Yi Jia, Wei Wu, Todd Richmond, Jacob Kitzman, Heidi Rosenbaum, A. Leonardo Iniguez, W. Brad Barbazuk, Jeffrey A. Jeddeloh, Dan Nettleton, and Patrick S. Schnable. Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content. PLoS Genetics, 5(11):e1000734, 2009."/>
  <meta name="citation_reference" content="Ashley Troth, Joshua R Puzey, Rebecca S Kim, John H Willis, and John K Kelly. Selective trade-offs maintain alleles underpinning complex trait variation in plants. Science, 361(6401):475–478, 2018."/>
  <meta name="citation_reference" content="Mario Vallejo-Marín, Arielle M Cooley, Michelle Yuequi Lee, Madison Folmer, Michael R McKain, and Joshua R Puzey. Strongly asymmetric hybridization barriers shape the origin of a new polyploid species and its hybrid ancestor. American Journal of Botany, 103(7):1272–1288, 2016."/>
  <meta name="citation_reference" content="Nicolaas Godfried Van Kampen. Stochastic Processes in Physics and Chemistry. North Holland, 3rd edition, 2007."/>
  <meta name="citation_reference" content="Thomas Wicker, François Sabot, Aurélie Hua-Van, Jeffrey L Bennetzen, Pierre Capy, Boulos Chalhoub, Andrew Flavell, Philippe Leroy, Michele Morgante, Olivier Panaud, Etienne Paux, Phillip SanMiguel, and Alan H. Schulman. A unified classification system for eukaryotic transposable elements. Nature Reviews Genetics, 8(12):973–982, 2007."/>
  <meta name="citation_reference" content="Brian Charlesworth and Deborah Charlesworth. The population dynamics of transposable elements. Genetics Research, 42(1):1–27, 1983."/>
  <meta name="citation_reference" content="Crispin Gardiner. Stochastic Methods: A Handbook for the Natural and Social Sciences. Springer, 4th edition, 2009."/>
  <meta name="citation_reference" content="Nicolaas Godfried Van Kampen. S’tochastic Processes in Physics and Chemistry. North Holland, 3rd edition, 2007."/>
  <meta name="twitter:title" content="Population genetics of transposable element load: a mechanistic account of observed overdispersion"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="In an empirical analysis of transposable element (TE) abundance within natural populations of Mimulus guttatus and Drosophila melanogaster , we found a surprisingly high variance of TE count (e.g., variance-to-mean ratio on the order of 10 to 300). To obtain insight regarding those evolutionary genetic mechanisms that are may underlie the overdispersed population distributions of TE abundance, we developed a mathematical model of TE population genetics that includes the dynamics of element proliferation and purifying selection on TE load. The modeling approach begins with a master equation for a birth-death process and extends the predictions of the classical theory of TE dynamics in several ways. In particular, moment-based analyses of time-dependent and equilibrium population distributions of TE load reveal that overdispersion is likely to arise via copy-and-paste proliferation dynamics, especially when the elementary processes of proliferation and excision are first-order and approximately balanced. Parameter studies and analytic work confirm this result and further suggest that overdispersed population distributions of TE abundance are unlikely to be a consequence of purifying selection on total element load.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Population genetics of transposable element load: a mechanistic account of observed overdispersion"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.11.16.382945v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="In an empirical analysis of transposable element (TE) abundance within natural populations of Mimulus guttatus and Drosophila melanogaster , we found a surprisingly high variance of TE count (e.g., variance-to-mean ratio on the order of 10 to 300). To obtain insight regarding those evolutionary genetic mechanisms that are may underlie the overdispersed population distributions of TE abundance, we developed a mathematical model of TE population genetics that includes the dynamics of element proliferation and purifying selection on TE load. The modeling approach begins with a master equation for a birth-death process and extends the predictions of the classical theory of TE dynamics in several ways. In particular, moment-based analyses of time-dependent and equilibrium population distributions of TE load reveal that overdispersion is likely to arise via copy-and-paste proliferation dynamics, especially when the elementary processes of proliferation and excision are first-order and approximately balanced. Parameter studies and analytic work confirm this result and further suggest that overdispersed population distributions of TE abundance are unlikely to be a consequence of purifying selection on total element load.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-01-04"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.11.16.382945v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2335923"/>
  <title>
   Population genetics of transposable element load: a mechanistic account of observed overdispersion | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"QgFwxsxZw0uGkIIbVnT7bqCREVpY1sbzFHLVB6EqDcU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2335923","apath":"\/biorxiv\/early\/2022\/01\/04\/2020.11.16.382945.atom","pisa":"biorxiv;2020.11.16.382945v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.11.16.382945v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.11.16.382945v3","title":"Population genetics of transposable element load: a mechanistic account of observed overdispersion","identifier":"node\/2335923"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t1yW7XWlM51uRVs\\\\\\\/DKIHh4nZyNfGlMHFgHEaWGrsY\\\\\\\/57WF0JEAFH6Crf8o1R3x0lzHaHuQMY5eD5HFpBOkI6jG9+e9FcaLq8BDDB+AtpkKRYve3lIK79EAYZQGZRybfpVUcYNp\\\\\\\/6pU0GlVmdyvu064zMN1kPAP6iv6YkaT10FAj7jcYEp1W7dBdyVsdAltwnS3980UIPTFeVMPRynE0a\\\\\\\/28HkBBxMFBdJ\\\\\\\/zgmNi+7R5N\\\\\\\/QHE5eU7UwcU4paCZoKLFRrX1zFekpJ5FYA4+8lNNcoOL51G0rMM3ao8Pjwj74jjhtsLs81yOT1ZW7pY37wfJZyqpSgF7dSwIu8d+INYAWRZBijGvfhLSFW1Ne9a\\\\\\\/wbbafpmOAM\\\\\\\/AA9yxnwgjDq8E++Vs0KJcCGcqB8eb+JhZ9w+WS3Lip7JOKG0bL1mcHpOgPHGVLUDzFDIJfCnIC\\\\\\\/wbHXxDyHwTkEveRslXfvtASjNAYOZSu\\\\\\\/k6nZQ\\\\\\\/NjHfLjwVhwsX6B08soct1fJtA35Z1D6oXGDLOzikdtPA0wmtu7jJv89lLpfuu+Qckjb6CKcxydP5L42PZ5zrwSQT7E5PqX5KlHWejhSxTngBIJU0N6F\\\\\\\/VZ7gf63qNW7R7Qs1\\\\\\\/Qs+h9gBn9wyWoda0SiHlqknbo\\\\\\\/bQPsBE8mNE5ZYDhVUqNyffgH\\\\\\\/\\\\\\\/w47xB7WIBt3IlZ4GbiAwzUST9YS7hBODpro3U4sFcbsU69\\\\\\\/QQQgK6jiV2FfrOY0u21Fn55JgRVvgpVvz6Po+u+QaiSTTXU0RDOEnGCGn2sfdVsUjWO7voWwEEDWGhhhevECRep2ySum+5zlu7EKLpOonkHtbHB2M+hHlBJBhZN\\\\\\\/YRfUz+Zpm2wtmM18+d7pjCx\\\\\\\/6jJfavH9880v3GsA0fpp+AsknTtoTH0W2K3HUiF0OqJGIiPKOxSe6H1XjPubGRwSkeRrleZDakW34lLhIUK\\\\\\\/7J7lgz42Efoip4ecIyY\\\\\\\/yDNvu2VNrn02MGnr3Y0XVd3QaI6Aq1mtoFN0hM46Kr5BqPGWwn9J4W\\\\\\\/ohgcSJblta4\\\\\\\/KtZej0KRv5heigotcjPX++KT5fwEhL6k0OiBQt7w9YksAdB2W2Fn9CQPctOOjk+uf9OkU0bhz3kzWDpKJoF2\\\\\\\/J40t77k\\\\\\\/s46xASbjKT83p6dOYkXSZVD+7yCXYv\\\\\\\/o0edix+qo9pU1NPNjS5JhwbmpEBIHCD5Yt5UTfS9FeqaOIApdXSYTPgEH9wTWHH0+3WlDzyHGGSnF3Tqy97VIyOTL4AvTKsO4mykJ0KtM\\\\\\\/L367P6cJ4ZStXHuX\\\\\\\/\\\\\\\/WNXjZNY6vjNkwMW+qA7lgIeB1DmHShll\\\\\\\/srdCYwPLn5Ga7d18QW7AgOJ7aT3hp7eFa\\\\\\\/BrgjFCRJ+i5hFH1vQRvu+ndjS0fz5sxAv94w=\\\u0022,\\\u0022iv\\\u0022:\\\u0022jabR\\\\\\\/\\\\\\\/AP6LyZfscqDU\\\\\\\/9LQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00225df4629c972b94ca4402933229ed3d930e9202e380bbefeebd7c0d74f5bca21c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.11.16.382945v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2335923 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.11.16.382945v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2136936013">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2136936013" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_586531291" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-eRC-IKdb_dZLwXpNsEwHezjIjq3qGJoOQHjg3hapLYo"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2335923" data-node-nid="2335923" id="node-2335923--2976601232" data-pisa="biorxiv;2020.11.16.382945v3" data-pisa-master="biorxiv;2020.11.16.382945" data-apath="/biorxiv/early/2022/01/04/2020.11.16.382945.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Population genetics of transposable element load: a mechanistic account of observed overdispersion
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-2736-0092" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ronald D.
                      </span>
                      <span class="nlm-surname">
                       Smith
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0001-8019-9993" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Joshua R.
                      </span>
                      <span class="nlm-surname">
                       Puzey
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-1054-6790" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Gregory D.
                      </span>
                      <span class="nlm-surname">
                       Conradi Smith
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.11.16.382945
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2335923--2976601232" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Ronald D. Smith
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Applied Science, William & Mary
                       </span>
                       , Williamsburg, VA 23187
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ronald%2BD.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Smith%20RD&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARonald%2BD.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2736-0092" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ronald D. Smith
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Joshua R. Puzey
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Biology, William & Mary
                       </span>
                       , Williamsburg, VA 23187
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Joshua%2BR.%2BPuzey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Puzey%20JR&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJoshua%2BR.%2BPuzey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-8019-9993" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Joshua R. Puzey
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Gregory D. Conradi Smith
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Applied Science, William & Mary
                       </span>
                       , Williamsburg, VA 23187
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Gregory%2BD.%2BConradi%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Conradi%20Smith%20GD&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AGregory%2BD.%2BConradi%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-1054-6790" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Gregory D. Conradi Smith
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:greg@wm.edu" class="" data-icon-position="" data-hide-link-title="0">
                        greg@wm.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.11.16.382945v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2335923" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2335923/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.16.382945v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2335923" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2335923/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.16.382945v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2335923" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2335923/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.16.382945v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2335923" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2335923/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.11.16.382945v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2335923" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2335923/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             In an empirical analysis of transposable element (TE) abundance within natural populations of Mimulus guttatus and Drosophila melanogaster, we found a surprisingly high variance of TE count (e.g., variance-to-mean ratio on the order of 10 to 300). To obtain insight regarding those evolutionary genetic mechanisms that are may underlie the overdispersed population distributions of TE abundance, we developed a mathematical model of TE population genetics that includes the dynamics of element proliferation and purifying selection on TE load. The modeling approach begins with a master equation for a birth-death process and extends the predictions of the classical theory of TE dynamics in several ways. In particular, moment-based analyses of time-dependent and equilibrium population distributions of TE load reveal that overdispersion is likely to arise via copy-and-paste proliferation dynamics, especially when the elementary processes of proliferation and excision are first-order and approximately balanced. Parameter studies and analytic work confirm this result and further suggest that overdispersed population distributions of TE abundance are unlikely to be a consequence of purifying selection on total element load.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-5">
                             The genomics revolution has revealed that a significant portion of eukaryotic genomes consists of transposable elements (TEs, also called mobile DNA elements or transposons). Notable examples include the human and maize genomes, 44% and 85% of which are TE sequences (
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Mills et al., 2007
                             </a>
                             ;
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              Springer et al., 2009
                             </a>
                             ). Various mobility mechanisms enable TEs to proliferate and/or change position within a genome. The effect of TEs can range from having little to no consequence on phenotype to being powerful mutagens (
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              Bourque et al., 2018
                             </a>
                             ). In addition to the innate tendency of TEs to proliferate, factors such as recombination, epigenetics, and selection contribute to the complex genomic distribution and demography of these elements (
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              Kent et al., 2017
                             </a>
                             ). While it is clear that TEs have been an integral part of the long-term evolution of genome architecture, much about the role of TEs in evolution remains unknown. Knowledge of the dynamics of TE abundance in natural populations is an important step toward an increasing understanding of how genomes evolve.
                            </p>
                            <p id="p-6">
                             The pioneering and influential population genetic theory of TEs developed by
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              Charlesworth and Charlesworth (1983)
                             </a>
                             used a combination of mathematical analysis, computer simulation, and a limited amount of experimental data, to give theoretical insight into TE dynamics and demographics. This modeling considered a single family of TEs with a drift-diffusion representation of TE proliferation, with either no selection or weak selection acting on total TE copy number. This modeling approach has informed our understanding of the population genetics of TEs for several decades. However, the classical theory does not reproduce experimentally observed within-population variances that often greatly exceed the population mean. The cause of this discrepancy is that the classical modeling approach assumes a binomial distribution of within-population TE loads, which constrains the population variance to be no greater than the population mean.
                            </p>
                            <p id="p-7">
                             This paper begins with a brief review of classical TE population genetics. This is followed by an analysis of TE demography derived from genome-sequence data of two natural populations (Mimulus guttatus and Drosophila melanogaster). Notably, in both cases, we observe that the within-population variance of TE load is highly overdispersed. Because these empirical results violate the predictions of classical TE modeling, we developed a more realistic master equation formulation of the population distribution of TE loads in a large randomly mating population. This alternative population genetic framework simultaneously and self-consistently predicts both the mean and variance of within-population TE load. This model of TE population genetics is then interrogated in order to elucidate those evolutionary genetic mechanisms that have the greatest influence on the population variance of TE load. In particular, moment-based analyses of time-dependent and equilibrium population distributions of TE load reveal that overdispersion is likely to arise via copy-and-paste proliferation dynamics, especially when the elementary processes of proliferation and excision are first-order and balanced. Parameter studies and analytic work confirm this result and further suggest that overdispersed population distributions of TE abundance are probably not a consequence of purifying selection on total element load.
                            </p>
                            <div id="sec-2" class="subsection">
                             <h3>
                              Classical population genetics of TEs
                             </h3>
                             <p id="p-8">
                              A good starting point for discussing the evolutionary dynamics of TEs is an influential paper by
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               Charlesworth and Charlesworth (1983)
                              </a>
                              and subsequent work with a similar modeling approach (
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               Brookfield and Badge, 1997
                              </a>
                              ;
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               Charlesworth and Charlesworth, 2010
                              </a>
                              ;
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               Deceliere, 2004
                              </a>
                              ;
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               Le Rouzic and Deceliere, 2005
                              </a>
                              ). This classical theory represents a chromosome as a finite set of m available insertion sites (loci) per haploid genome, each of which can either be occupied by a transposable element (or not). For a single family of TEs, the state of an infinite diploid population at a given chromosomal site i, for i = 1, 2,…, m, is described by its frequency, x
                              <sub>
                               i
                              </sub>
                              , where 0 ≤ x
                              <sub>
                               i
                              </sub>
                              ≤ 1. Assuming insertion sites exhibit no linkage disequilibrium, the set of frequencies,
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , describes the state of the population. The mean copy number of TEs per individual is
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where the factor of 2 accounts for diploidy.
                             </p>
                             <p id="p-9">
                              The evolutionarily neutral version of the classical theory includes two processes affecting TE load (gain and loss). Gain of TEs is represented by a proliferation rate (per individual per element per generation) in the germ line of an individual with n elements. This proliferation rate, denoted u
                              <sub>
                               n
                              </sub>
                              , is typically assumed to be a decreasing function of TE load (du
                              <sub>
                               n
                              </sub>
                              /dn < 0). Loss of TEs is represented by a first-order excision rate constant (per individual per element per generation) denoted by ν. The change (per generation) in the mean TE copy number per individual is thus
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <strong>
                               n
                              </strong>
                              is the diploid TE load of a randomly sampled individual, the expected value is taken over individuals in the population, and
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the population mean of TE copy number. Expanding
                              <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. 1
                              </a>
                              around the mean TE load gives the following second-order approximation,
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where V
                              <sub>
                               n
                              </sub>
                              denotes the population variance in TE copy number (
                              <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                               Charlesworth and Charlesworth, 1983
                              </a>
                              ). If the higher order terms that scale the population variance are negligible, the change in mean TE copy number per generation is
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . For this neutral model of TE population dynamics, one concludes that
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              will approach a (stable) equilibrium value satisfying
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              provided
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-10">
                              To extend this model of TE population genetics to include the effect of natural selection, it is customary to assume a viability function, w
                              <sub>
                               n
                              </sub>
                              , that is a decreasing function of genome-wide TE load (dw
                              <sub>
                               n
                              </sub>
                              /dn < 0). Approximating the mean fitness of the population (E[w
                              <sub>
                               n
                              </sub>
                              ]) by the fitness of an individual with an average number of copies
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Eq. 2
                              </a>
                              can be extended to include the effect of selection on TE load (
                              <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                               Charlesworth and Charlesworth, 2010
                              </a>
                              ),
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-11">
                              As a specific example, consider the proliferation rate function u
                              <sub>
                               n
                              </sub>
                              = ξ
                              <sub>
                               0
                              </sub>
                              /n with ξ
                              <sub>
                               0
                              </sub>
                              > 0 and the selection function w
                              <sub>
                               n
                              </sub>
                              = e
                              <sup>
                               −γn
                              </sup>
                              for γ > 0 (viability is a decreasing function of TE copy number). Because du
                              <sub>
                               n
                              </sub>
                              /dn = −ξ
                              <sub>
                               0
                              </sub>
                              /n
                              <sup>
                               2
                              </sup>
                              and d
                              <sup>
                               2
                              </sup>
                              u
                              <sub>
                               n
                              </sub>
                              /dn
                              <sup>
                               2
                              </sup>
                              = 2ξ
                              <sub>
                               0
                              </sub>
                              /n
                              <sup>
                               3
                              </sup>
                              , the higher order terms involving derivatives of u
                              <sub>
                               n
                              </sub>
                              evaluate to zero. Consequently,
                              <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                               Eq. 3
                              </a>
                              becomes
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-12">
                              Substituting
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and setting
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              gives
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Solving for the equilibrium mean TE load gives,
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-13">
                              This result is biologically meaningful for ξ
                              <sub>
                               0
                              </sub>
                              > γV
                              <sub>
                               n
                              </sub>
                              . As expected, the equilibrium TE load is an increasing function of the proliferation rate constant, ξ
                              <sub>
                               0
                              </sub>
                              , and a decreasing function of the excision rate constant, ν. Furthermore, stronger selection against TE load (greater γ) decreases the mean value of the equilibrium TE load in the population.
                             </p>
                            </div>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Population variance in the classical model
                             </h3>
                             <p id="p-14">
                              Analysis of the classical model of TE population genetics proceeds in an ad hoc manner by making further assumptions regarding the population variance (V
                              <sub>
                               n
                              </sub>
                              is a parameter in
                              <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">
                               Eqs. 2
                              </a>
                              –
                              <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                               4
                              </a>
                              ). For example,
                              <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">
                               Charlesworth and Charlesworth (1983)
                              </a>
                              assume the population variance takes the form
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where D is a matrix of linkage disequilibrium coefficients (
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               Bulmer, 1980
                              </a>
                              ), and
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the variance in element frequency across loci (see
                              <a id="xref-sec-19-1" class="xref-sec" href="#sec-19">
                               Supplemental Material
                              </a>
                              ,
                              <a id="xref-sec-20-1" class="xref-sec" href="#sec-20">
                               Section S1
                              </a>
                              ). If one further assumes that linkage effects are small enough to be ignored, then
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-15">
                              <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">
                               Charlesworth and Charlesworth (1983)
                              </a>
                              argue that for a large enough population, one expects the variance in element frequency across loci to be eventually become negligable,
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and, consequently, the equilibrium population variance of TE load should approach that of a binomial distribution,
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-16">
                              In that case, assuming occupiable loci are not limiting
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the population variance will be well-approximately by the mean
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Substituting this value into
                              <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">
                               Eq. 4
                              </a>
                              , the classical model indicates that the equilibrium TE load will be
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-17">
                              As in
                              <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">
                               Eq. 4
                              </a>
                              , the equilibrium TE load is an increasing function of the proliferation rate constant (ξ
                              <sub>
                               0
                              </sub>
                              ), and a decreasing function of both the excision rate constant (ν) and the strength of selection against TE load (γ).
                             </p>
                             <p id="p-18">
                              The classical model (
                              <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">
                               Eqs. 3
                              </a>
                              –
                              <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                               8
                              </a>
                              ) has informed expectations regarding the population genetics of TEs for several decades. For example, an extension of this classical theory predicts that in a finite population of effective size N
                              <sub>
                               e
                              </sub>
                              , the the stationary distribution of TE frequency (x) will take the form ρ(x) ∝ x
                              <sup>
                               a–1
                              </sup>
                              (1 – x)
                              <sup>
                               b–1
                              </sup>
                              where
                              <span class="inline-formula" id="inline-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               Le Rouzic and Deceliere, 2005
                              </a>
                              ). For u
                              <sub>
                               n
                              </sub>
                              = ξ
                              <sub>
                               0
                              </sub>
                              /n, w
                              <sub>
                               n
                              </sub>
                              = e
                              <sup>
                               −γn
                              </sup>
                              , and
                              <span class="inline-formula" id="inline-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , this gives a = 4N
                              <sub>
                               e
                              </sub>
                              ξ
                              <sub>
                               0
                              </sub>
                              and b = 4N
                              <sub>
                               e
                              </sub>
                              (ν+γ). On the other hand, the classical approach to modeling TE population genetics has obvious limitations. For one thing, the derivation and analysis of the classical model makes assumptions about the population variance (V
                              <sub>
                               n
                              </sub>
                              in
                              <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                               Eqs. 5
                              </a>
                              –
                              <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                               7
                              </a>
                              ) that may not be consistent with experimental observations (see
                              <a id="xref-sec-4-1" class="xref-sec" href="#sec-4">
                               Results
                              </a>
                              ). Furthermore, the population variance of TE load ought to be an emergent property of a model constructed for the purpose of understanding the population genetics of TEs, rather than a modeling assumption imposed upon a preexisting framework (
                              <a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">
                               Eq. 7
                              </a>
                              ).
                             </p>
                             <p id="p-19">
                              The remainder of this paper addresses these two issues in detail. We begin by presenting empirical evidence that population variance of TEs is neither binomial nor well-approximated by the mean. This motivates the presentation of an alternative population genetic framework that simultaneously and self-consistently predicts both the population variance and the mean TE load. This model of TE population genetics is then interrogated in order to elucidate those evolutionary genetic mechanisms that have the greatest influence on the population variance of TE load.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-4">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Dispersion of TE loads in the classical model
                             </h3>
                             <p id="p-20">
                              In the classical modeling of TE population genetics discussed above, analytical results are obtained by assuming a randomly mating population with a binomial distribution of TE loads,
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with mean
                              <span class="inline-formula" id="inline-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and variance
                              <span class="inline-formula" id="inline-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">
                               Eqs. 5
                              </a>
                              –
                              <a id="xref-disp-formula-8-3" class="xref-disp-formula" href="#disp-formula-8">
                               7
                              </a>
                              ). A simple measure of the variability of TE load within a population is the index of dispersion (Fano factor) given by
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-21">
                              Substituting the mean and variance of the binomial distribution into
                              <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">
                               Eq. 10
                              </a>
                              , it is apparent that the classical model of TE population genetics predicts (i.e., assumes) a Fano factor that is less than one,
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-22">
                              In fact, when the number of sites occupied by TEs is small compared to the total number of occupiable loci (m → ∞ with
                              <span class="inline-formula" id="inline-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              fixed), the Fano factor approaches one from below (Fano[
                              <strong>
                               n
                              </strong>
                              ] → 1). In this limit, the binomial distribution of
                              <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                               Eq. 9
                              </a>
                              is well-approximated by
                              <span class="inline-formula" id="inline-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . If it were the case that the TE load within a population were Poisson distributed, then the mean and variance of TE load would be equal
                              <span class="inline-formula" id="inline-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and the index of dispersion would be Fano[
                              <strong>
                               n
                              </strong>
                              ] = 1. With our expectations set by this prediction of the classical model of TE population genetics, empirical observations of a Fano factor greater than one (Fano[
                              <strong>
                               n
                              </strong>
                              ] > 1) would indicate overdispersion of TE load within a population.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Overdispersion of empirical TE counts
                             </h3>
                             <p id="p-23">
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figs. 1
                              </a>
                              and
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               2
                              </a>
                              present analyses of two data sets, both of which demonstrate that the variance of TE load in experimentally studied populations can be far greater than would be predicted by classical models of TE population genetics. The first data set (analyzed in
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ) consists of whole-genome sequence data from 164 lines of Mimulus guttatus derived from a naturally occurring population (hundreds of thousands of individuals) in Iron Mountain, Oregon, USA (
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               Troth et al., 2018
                              </a>
                              ). To estimate TE copy numbers, we compared the coverage of each TE to the average coverage of single copy genes (see Appendix for details). Although Mimulus guttatus is our primary interest, we also analyzed, for comparison, a data set (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ) that consists of genomic DNA sequencing data from 131 lines of Drosophila melanogaster (derived from a large population in Raleigh, North Carolina) obtained from the Drosophila Genetic Reference Panel (
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               Cridland et al., 2013
                              </a>
                              , Supplementary Material S5).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F1.large.jpg?width=800&height=600&carousel=1" title="A: Mean-variance plot of TE copy number in a M. guttatus population compared to theoretical expectation (Poisson line). For each of ten different families of TEs, the index of dispersion is in the range 10 < Fano[n] < 300. For each family, the vertical bars show 95% bootstrap confidence interval of the population variance of TE load. B: Estimated TE copy number for 164 M. guttatus individuals. TE counts are separated by class (red, Class I, retrotransposon; blue, Class II, DNA transposon). The variability in TE load can be observed in the counts from individuals (bottom) as well as histograms (top). The overdispersion in TE load is apparent in the deviation of the observed counts (red and blue histograms) from the corresponding Poisson distributions (gray lines). The vertical dashed lines show the population mean of TE load." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">A: Mean-variance plot of TE copy number in a <em>M. guttatus</em> population compared to theoretical expectation (Poisson line). For each of ten different families of TEs, the index of dispersion is in the range 10 < Fano[<strong>n</strong>] < 300. For each family, the vertical bars show 95% bootstrap confidence interval of the population variance of TE load. B: Estimated TE copy number for 164 <em>M. guttatus</em> individuals. TE counts are separated by class (red, Class I, retrotransposon; blue, Class II, DNA transposon). The variability in TE load can be observed in the counts from individuals (bottom) as well as histograms (top). The overdispersion in TE load is apparent in the deviation of the observed counts (red and blue histograms) from the corresponding Poisson distributions (gray lines). The vertical dashed lines show the population mean of TE load.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F1.medium.gif" width="440" height="209"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F1.medium.gif" width="440" height="209"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <p id="p-24" class="first-child">
                                A: Mean-variance plot of TE copy number in a M. guttatus population compared to theoretical expectation (Poisson line). For each of ten different families of TEs, the index of dispersion is in the range 10 < Fano[
                                <strong>
                                 n
                                </strong>
                                ] < 300. For each family, the vertical bars show 95% bootstrap confidence interval of the population variance of TE load. B: Estimated TE copy number for 164 M. guttatus individuals. TE counts are separated by class (red, Class I, retrotransposon; blue, Class II, DNA transposon). The variability in TE load can be observed in the counts from individuals (bottom) as well as histograms (top). The overdispersion in TE load is apparent in the deviation of the observed counts (red and blue histograms) from the corresponding Poisson distributions (gray lines). The vertical dashed lines show the population mean of TE load.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F2.large.jpg?width=800&height=600&carousel=1" title="A: Mean-variance plot of TE copy number in D. melanogaster compared to theoretical expectation (Poisson line). For each of four different families of TEs, the index of dispersion is in the range 1 < Fano[n] < 200. Vertical bars show 95% bootstrap confidence intervals. B: Estimated TE copy number for 131 D. melanogaster individuals. TE counts are separated by class (red, Class I, retrotransposon; blue, Class II, DNA transposon)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">A: Mean-variance plot of TE copy number in <em>D. melanogaster</em> compared to theoretical expectation (Poisson line). For each of four different families of TEs, the index of dispersion is in the range 1 < Fano[<strong>n</strong>] < 200. Vertical bars show 95% bootstrap confidence intervals. B: Estimated TE copy number for 131 <em>D. melanogaster</em> individuals. TE counts are separated by class (red, Class I, retrotransposon; blue, Class II, DNA transposon).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F2.medium.gif" width="440" height="209"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F2.medium.gif" width="440" height="209"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <p id="p-25" class="first-child">
                                A: Mean-variance plot of TE copy number in D. melanogaster compared to theoretical expectation (Poisson line). For each of four different families of TEs, the index of dispersion is in the range 1 < Fano[
                                <strong>
                                 n
                                </strong>
                                ] < 200. Vertical bars show 95% bootstrap confidence intervals. B: Estimated TE copy number for 131 D. melanogaster individuals. TE counts are separated by class (red, Class I, retrotransposon; blue, Class II, DNA transposon).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F3.large.jpg?width=800&height=600&carousel=1" title="Example rates of transposable element gain (gn, dashed) and loss (ℓn, dotted) are shown cyan. These functions of TE load given by Eqs. 20–21 intersect (balance) when 17% of the insertion sites are occupied (gray dot-dashed line). Parameters: η0 = 20, η = 3, ν = 2.5, m = 5 × 105. The net rate of change (blue curve) is zero for a TE load of n = 8.3 × 104 (open circle), which is on the order of that found for transposons (Class II elements) in Mimulus (e.g., LINE and LTR, see Fig. 1B left)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Example rates of transposable element gain (<em>g<sub>n</sub></em>, dashed) and loss (<em>ℓ<sub>n</sub></em>, dotted) are shown cyan. These functions of TE load given by Eqs. 20–21 intersect (balance) when 17% of the insertion sites are occupied (gray dot-dashed line). Parameters: <em>η</em><sub>0</sub> = 20, <em>η</em> = 3, <em>ν</em> = 2.5, <em>m</em> = 5 × 10<sup>5</sup>. The net rate of change (blue curve) is zero for a TE load of <em>n</em> = 8.3 × 10<sup>4</sup> (open circle), which is on the order of that found for transposons (Class II elements) in <em>Mimulus</em> (e.g., LINE and LTR, see Fig. 1B left).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F3.medium.gif" width="440" height="353"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F3.medium.gif" width="440" height="353"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <p id="p-26" class="first-child">
                                Example rates of transposable element gain (g
                                <sub>
                                 n
                                </sub>
                                , dashed) and loss (ℓ
                                <sub>
                                 n
                                </sub>
                                , dotted) are shown cyan. These functions of TE load given by
                                <a id="xref-disp-formula-21-1" class="xref-disp-formula" href="#disp-formula-21">
                                 Eqs. 20
                                </a>
                                –
                                <a id="xref-disp-formula-22-1" class="xref-disp-formula" href="#disp-formula-22">
                                 21
                                </a>
                                intersect (balance) when 17% of the insertion sites are occupied (gray dot-dashed line). Parameters: η
                                <sub>
                                 0
                                </sub>
                                = 20, η = 3, ν = 2.5, m = 5 × 10
                                <sup>
                                 5
                                </sup>
                                . The net rate of change (blue curve) is zero for a TE load of n = 8.3 × 10
                                <sup>
                                 4
                                </sup>
                                (open circle), which is on the order of that found for transposons (Class II elements) in Mimulus (e.g., LINE and LTR, see
                                <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                 Fig. 1B
                                </a>
                                left).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-27">
                              Comparison of the marker locations and histograms in
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Figs. 1
                              </a>
                              and
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               2
                              </a>
                              with the gray lines labelled Poisson shows that in both species, Mimulus guttatus and Drosophila melanogaster, the population distribution of TE load is overdispersed (the variance of TE load is greater than the mean TE load). In D. melanogaster, this overdispersion is greater for Class I TEs (retrotransposons) with an RNA intermediate than Class II TEs (DNA transposons). (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2B
                              </a>
                              ). The corresponding Fano factors (
                              <a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">
                               Eq. 10
                              </a>
                              ) are 16 and 2.7, respectively; these values indicate overdispersion (see
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ).
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig. 1A
                              </a>
                              shows that overdispersion of TE load is even more pronounced in M. guttatus. In this case, the Fano factors are 61 for Class I TEs (red symbols: LINE, SINE and LTR), and 646 for Class II TEs (blue symbols: CACTA, Helitron, Tcl-Mariner, PIF-Harbinger, hAT and MULE).
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              shows the estimated number of Class I and II TEs in each of the 164 lines of M. guttatus (horizontal bar graph). In both cases, the variance (illustrated by the width of red and blue histograms) is far greater than the variance that would be consistent with classical model of TE population genetics (gray curves). Taken together,
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Figs. 1
                              </a>
                              and
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               2
                              </a>
                              show that in both species, Mimulus guttatus and Drosophila melanogaster, and for both classes of TEs (retrotransposons and DNA transposons), the distribution of TE load within the studied population is highly overdispersed.
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2338362/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2338362/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2338362" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1:
                               </span>
                               <p id="p-28" class="first-child">
                                Empirically observed mean, variance, and index of dispersion (Fano factor) of the population distribution of TE load in 164 M. guttatus and 131 D. melanogaster individuals (cf.
                                <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                                 Figs. 1
                                </a>
                                and
                                <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                                 2
                                </a>
                                ). Class I elements (retrotransposons) proliferate in a staged manner that involves an RNA intermediate, while Class II elements (DNA transposons) do not utilize an RNA intermediate (see, e.g., Ch. 9 of
                                <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                                 Graur (2016)
                                </a>
                                for review).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Overdispersion is not explained by distinct TE families
                             </h3>
                             <p id="p-29">
                              The overdispersion documented in
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Figs. 1
                              </a>
                              and
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               2
                              </a>
                              cannot be explained away as a simple consequence of the heterogeneity of the properties of distinct TE types. Consider two families of TEs with loads across individuals in the population given by the random variables
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              and
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              . Denoting the mean TE loads of these families by
                              <span class="inline-formula" id="inline-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the corresponding Fano factors are
                              <span class="inline-formula" id="inline-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . If these two families were not distinguished, the observed mean load would be given by a composite count,
                              <strong>
                               x
                              </strong>
                              =
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              +
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              , with mean
                              <span class="inline-formula" id="inline-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and variance,
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-30">
                              Substituting
                              <span class="inline-formula" id="inline-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and dividing by
                              <span class="inline-formula" id="inline-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              gives an expression for the composite index of dispersion,
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-31">
                              Assuming that the within-population loads for the two families of TEs are independent, the covariance will be zero (Cov[
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              ] = 0). In that case, the composite Fano factor is a weighted average of Fano factors for each family,
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which takes values in the range min(F
                              <sub>
                               1
                              </sub>
                              , F
                              <sub>
                               2
                              </sub>
                              ) ≤ F ≤ max(F
                              <sub>
                               1
                              </sub>
                              , F
                              <sub>
                               2
                              </sub>
                              ). A similar argument allows us to conclude that for TE families with independent proliferation and excision dynamics, the dispersion of TE load that results when families are not distinguished is always less than the overdispersion of at least one of the TE families (see
                              <a id="xref-sec-21-1" class="xref-sec" href="#sec-21">
                               Section S2
                              </a>
                              of the Supplemental Materials for further discussion).
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Master equation for TE population dynamics
                             </h3>
                             <p id="p-32">
                              Our modeling aims to clarify the observed overdispersion of TE load in Mimulus guttatus and Drosophila melanogaster, following classical TE population genetics, but with a few important modifications. Because the variance in TE load is not the result of heterogeneity in TE types (see above), our analysis will focus on a single TE family.
                             </p>
                             <p id="p-33">
                              Let p
                              <sub>
                               n
                              </sub>
                              (t) denote the probability that a randomly sampled haploid genome (gamete) has a TE count of n at time t. Prior to considerations of selection, our neutral model of TE population dynamics will take the form of a skip-free birth-death process with gain and loss rates denoted g
                              <sub>
                               n
                              </sub>
                              and ℓ
                              <sub>
                               n
                              </sub>
                              . The discrete state space for haploid TE load is
                              <strong>
                               n
                              </strong>
                              ∈ {0, 1, 2,…, m} and the state-transition diagram of the stochastic process is
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F4.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F4.medium.gif" width="440" height="59"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F4.medium.gif" width="440" height="59"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure4" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <p id="p-34">
                              The master equation for this stochastic process is the following system of m +1 differential equations,
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-35">
                              The expected value of TE load of a randomly sampled diploid genotype is
                              <span class="disp-formula" id="disp-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the mean TE load of a randomly sampled haploid gamete. By differentiating
                              <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">
                               Eq. 18
                              </a>
                              to obtain
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and substituting
                              <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                               Eqs. 15
                              </a>
                              –
                              <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">
                               17
                              </a>
                              , the master equation formulation can be shown to be consistent with the classical model (see
                              <a id="xref-sec-22-1" class="xref-sec" href="#sec-22">
                               Section S3
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Master equation predicts the variance of TE load
                             </h3>
                             <p id="p-36">
                              One feature of the master equation formulation (
                              <a id="xref-disp-formula-16-2" class="xref-disp-formula" href="#disp-formula-16">
                               Eqs. 15
                              </a>
                              –
                              <a id="xref-disp-formula-18-2" class="xref-disp-formula" href="#disp-formula-18">
                               17
                              </a>
                              ) is that the dynamics of the population variance of TE load are an emergent property of the model. To illustrate, let us assume that TE excision occurs with first order rate constant, then the loss rate as a function of n is
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-37">
                              Let us further assume that the rate of gain for a single family of TEs takes the form
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where η is the copy-and-paste rate per transposon (a first-order rate constant characterizing proliferation of TEs), η
                              <sub>
                               0
                              </sub>
                              is a zeroth order rate constant, n is the TE copy number, and m is the number of occupiable loci (in a haploid gamete). Substituting these constitutive relations for the loss and grain rates, ℓ
                              <sub>
                               n
                              </sub>
                              and g
                              <sub>
                               n
                              </sub>
                              , into
                              <a id="xref-disp-formula-16-3" class="xref-disp-formula" href="#disp-formula-16">
                               Eqs. 15
                              </a>
                              –
                              <a id="xref-disp-formula-18-3" class="xref-disp-formula" href="#disp-formula-18">
                               17
                              </a>
                              gives
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-38">
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 4
                              </a>
                              shows representative numerical solutions of this master equation for the population dynamics of TE load. When the copy-and-paste rate constant is zero (η = 0) and occupiable loci are not limiting
                              <span class="inline-formula" id="inline-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the stationary probability distribution is well-approximated by a Poisson distribution with
                              <span class="inline-formula" id="inline-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and Fano[
                              <strong>
                               n
                              </strong>
                              ] ≈ 1 (blue histograms). For both Mimulus- and Drosophila-like parameters, no overdispersion is observed when η = 0. These results should be compared to the green and red histograms, for which the copy-and-pate rate is nonzero (see caption for parameters). Notably, an increase in the copy-and-paste rate leads to significant overdispersion of the TE load for both simulated populations (Fano[
                              <strong>
                               n
                              </strong>
                              ] ranging from 1 to 100).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F5.large.jpg?width=800&height=600&carousel=1" title="Top: Stationary population distributions of TE load in haploid genomes (gametes) calculated using the evolutionarily neutral master equation model (Eqs. 22–24). For mean loads similar to Mimulus (left) and Drosophila (right), no overdispersion is observed in simulations absent copy-and- paste transposition (η = 0, Fano[n] ≈ 1). Green and red histograms show overdispersed population distributions of TE load that are obtained when copy-and-paste transposition is included. Mimulus parameters: ν = 0.1, m = 109; η0, η = 2000, 0 (blue), 200, 0.095 (red), 20, 0.099 (green). Drosophila parameters: ν = 0.1, m = 5000; η0, η = 20, 0 (blue), 1, 0.1 (green), 2, 0.1 (red). See Section S6 for numerical methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Top: Stationary population distributions of TE load in haploid genomes (gametes) calculated using the evolutionarily neutral master equation model (Eqs. 22–24). For mean loads similar to <em>Mimulus</em> (left) and <em>Drosophila</em> (right), no overdispersion is observed in simulations absent copy-and- paste transposition (<em>η</em> = 0, Fano[<strong>n</strong>] ≈ 1). Green and red histograms show overdispersed population distributions of TE load that are obtained when copy-and-paste transposition is included. <em>Mimulus</em> parameters: <em>ν</em> = 0.1, <em>m</em> = 10<sup>9</sup>; <em>η</em><sub>0</sub>, <em>η</em> = 2000, 0 (blue), 200, 0.095 (red), 20, 0.099 (green). <em>Drosophila</em> parameters: <em>ν</em> = 0.1, <em>m</em> = 5000; <em>η</em><sub>0</sub>, η = 20, 0 (blue), 1, 0.1 (green), 2, 0.1 (red). See Section S6 for numerical methods.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F5.medium.gif" width="440" height="219"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F5.medium.gif" width="440" height="219"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <p id="p-39" class="first-child">
                                Top: Stationary population distributions of TE load in haploid genomes (gametes) calculated using the evolutionarily neutral master equation model (
                                <a id="xref-disp-formula-23-1" class="xref-disp-formula" href="#disp-formula-23">
                                 Eqs. 22
                                </a>
                                –
                                <a id="xref-disp-formula-25-1" class="xref-disp-formula" href="#disp-formula-25">
                                 24
                                </a>
                                ). For mean loads similar to Mimulus (left) and Drosophila (right), no overdispersion is observed in simulations absent copy-and- paste transposition (η = 0, Fano[
                                <strong>
                                 n
                                </strong>
                                ] ≈ 1). Green and red histograms show overdispersed population distributions of TE load that are obtained when copy-and-paste transposition is included. Mimulus parameters: ν = 0.1, m = 10
                                <sup>
                                 9
                                </sup>
                                ; η
                                <sub>
                                 0
                                </sub>
                                , η = 2000, 0 (blue), 200, 0.095 (red), 20, 0.099 (green). Drosophila parameters: ν = 0.1, m = 5000; η
                                <sub>
                                 0
                                </sub>
                                , η = 20, 0 (blue), 1, 0.1 (green), 2, 0.1 (red). See
                                <a id="xref-sec-32-1" class="xref-sec" href="#sec-32">
                                 Section S6
                                </a>
                                for numerical methods.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Moment equations for mean and variance of TE load
                             </h3>
                             <p id="p-40">
                              The previous section showed that the evolutionarily neutral master equation model provides information about the population variance of TE load that is unavailable in classical theory. Because this realism comes at the expense of a more complex model formulation (
                              <a id="xref-disp-formula-23-2" class="xref-disp-formula" href="#disp-formula-23">
                               Eqs. 22
                              </a>
                              –
                              <a id="xref-disp-formula-25-2" class="xref-disp-formula" href="#disp-formula-25">
                               24
                              </a>
                              compared to
                              <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">
                               Eq. 2
                              </a>
                              ), we derived ordinary differential equations (ODEs) that summarize the dynamics of the mean and variance of the population distribution of diploid TE loads predicted by the master equation.
                              <a id="xref-sec-22-2" class="xref-sec" href="#sec-22">
                               Section S3
                              </a>
                              of the Supplementary Material shows that the mean and variance of TE load solve the following ODEs,
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-41">
                              The term
                              <span class="inline-formula" id="inline-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              that appears in
                              <a id="xref-disp-formula-27-1" class="xref-disp-formula" href="#disp-formula-27">
                               Eq. 26
                              </a>
                              is the third central moment of the within-population diploid TE load. Analysis of this system of ODEs and the third central moment is provided below.
                             </p>
                             <p id="p-42">
                              If number of occupiable loci are not limiting
                              <span class="inline-formula" id="inline-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we may take the limit of
                              <a id="xref-disp-formula-26-1" class="xref-disp-formula" href="#disp-formula-26">
                               Eqs. 25
                              </a>
                              –
                              <a id="xref-disp-formula-27-2" class="xref-disp-formula" href="#disp-formula-27">
                               26
                              </a>
                              as m → ∞ to obtain simpler equations for the mean and variance,
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-43">
                              This reduced system of ODEs is linear and, for large m, the equation for the variance (
                              <a id="xref-disp-formula-29-1" class="xref-disp-formula" href="#disp-formula-29">
                               Eq. 28
                              </a>
                              ) does not depend on the third central moment. The steady-state solution of
                              <a id="xref-disp-formula-28-1" class="xref-disp-formula" href="#disp-formula-28">
                               Eqs. 27
                              </a>
                              –
                              <a id="xref-disp-formula-29-2" class="xref-disp-formula" href="#disp-formula-29">
                               28
                              </a>
                              is given by
                              <span class="disp-formula" id="disp-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is physical provided ν > η, that is, when m is large, the rate of excision ν must be greater than the copy-and-past rate constant η for a biologically meaningful solution with
                              <span class="inline-formula" id="inline-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (mean TE load must be non-negative). This steady state is stable because the Jacobian of
                              <a id="xref-disp-formula-28-2" class="xref-disp-formula" href="#disp-formula-28">
                               Eqs. 27
                              </a>
                              –
                              <a id="xref-disp-formula-29-3" class="xref-disp-formula" href="#disp-formula-29">
                               28
                              </a>
                              , given by the 2 × 2 matrix with entries J
                              <sub>
                               11
                              </sub>
                              = −(ν – η), J
                              <sub>
                               12
                              </sub>
                              = 0, J
                              <sub>
                               21
                              </sub>
                              = ν + η, J
                              <sub>
                               22
                              </sub>
                              = −2(ν – η), has real valued eigenvalues λ
                              <sub>
                               +
                              </sub>
                              = −(ν – η) < 0 and λ
                              <sub>
                               −
                              </sub>
                              = 2λ
                              <sub>
                               +
                              </sub>
                              < 0.
                             </p>
                             <p id="p-44">
                              The values for the steady-state mean and variance of TE load given by
                              <a id="xref-disp-formula-30-1" class="xref-disp-formula" href="#disp-formula-30">
                               Eqs. 29
                              </a>
                              –
                              <a id="xref-disp-formula-31-1" class="xref-disp-formula" href="#disp-formula-31">
                               30
                              </a>
                              correspond to the following index of dispersion,
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-45">
                              Notably, the condition for a stable steady state (ν > η) implies an index of dispersion greater than unity (Fano[
                              <strong>
                               n
                              </strong>
                              ] > 1) for any nonzero copy-and-paste rate constant (η > 0). For this reason, we conclude that copy-and-paste proliferation dynamics will result in an overdispersed steady-state population distribution of TE loads provided the number of occupiable loci are not limiting
                              <span class="inline-formula" id="inline-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Further analysis of the moment equations (
                              <a id="xref-disp-formula-26-2" class="xref-disp-formula" href="#disp-formula-26">
                               Eqs. 25
                              </a>
                              –
                              <a id="xref-disp-formula-27-3" class="xref-disp-formula" href="#disp-formula-27">
                               26
                              </a>
                              ) shows that overdispersion will not occur in the absence of copy-and-paste dynamics (see the η = 0 case in
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ).
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2338355/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2338355/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2338355" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2:
                               </span>
                               <p id="p-46" class="first-child">
                                The evolutionarily neutral moment equations (
                                <a id="xref-disp-formula-26-3" class="xref-disp-formula" href="#disp-formula-26">
                                 Eqs. 25
                                </a>
                                –
                                <a id="xref-disp-formula-27-4" class="xref-disp-formula" href="#disp-formula-27">
                                 26
                                </a>
                                ) for the mean and variance of TE load make predictions in various limits (see
                                <a id="xref-sec-24-1" class="xref-sec" href="#sec-24">
                                 Sections S3.2
                                </a>
                                –
                                <a id="xref-sec-25-1" class="xref-sec" href="#sec-25">
                                 S3.3
                                </a>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-47">
                              This preliminary analysis of an evolutionarily neutral master equation for TE proliferation (
                              <a id="xref-disp-formula-23-3" class="xref-disp-formula" href="#disp-formula-23">
                               Eqs. 22
                              </a>
                              –
                              <a id="xref-disp-formula-25-3" class="xref-disp-formula" href="#disp-formula-25">
                               24
                              </a>
                              ) indicates that a nonzero copy-and-paste rate may lead to an overdispersed population distribution of TE load (
                              <a id="xref-disp-formula-32-1" class="xref-disp-formula" href="#disp-formula-32">
                               Eq. 31
                              </a>
                              ). That is, copy-and-paste TE dynamics is one possible explanation for our empirical observations of overdispersed TE counts (
                              <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                               Figs. 1
                              </a>
                              and
                              <a id="xref-fig-2-8" class="xref-fig" href="#F2">
                               2
                              </a>
                              ). Furthermore, this analysis predicts that a large index of dispersion may be a consequence of balanced dynamics of TE gain and loss (i.e., Fano[
                              <strong>
                               n
                              </strong>
                              ] → ∞ as ν decreases to η in
                              <a id="xref-disp-formula-32-2" class="xref-disp-formula" href="#disp-formula-32">
                               Eq. 31
                              </a>
                              ). While the divergence in the analytical result is an artifact of taking the m → ∞ limit, a parameter study of the master equation model (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 5
                              </a>
                              ) confirms that overdispersion is most pronounced (Fano[
                              <strong>
                               n
                              </strong>
                              ] maximized) when m is large and the dynamics of TE gain and loss are approximately balanced (η ≈ ν).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F6.large.jpg?width=800&height=600&carousel=1" title="Parameter studies of the neutral master equation model showing the mean , variance , and index of dispersion (Fano[n]) of within-population TE load as a function of the copy- and-paste rate constant (η). Parameters: m = 4 × 103 and as in legend. Cyan curves indicate analytical approximations using ν = 0.1 that are valid in the limit as m → ∞ (see Table 2). These approximations are most accurate for small η/ν and diverge as η approaches ν from below (cyan arrowheads). These calculations were accelerated using a Fokker-Planck approximation to Eqs. 22–24 (see Section S6)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Parameter studies of the neutral master equation model showing the mean , variance , and index of dispersion (Fano[<strong>n</strong>]) of within-population TE load as a function of the copy- and-paste rate constant (<em>η</em>). Parameters: <em>m</em> = 4 × 10<sup>3</sup> and as in legend. Cyan curves indicate analytical approximations using <em>ν</em> = 0.1 that are valid in the limit as <em>m</em> → ∞ (see Table 2). These approximations are most accurate for small <em>η</em>/<em>ν</em> and diverge as <em>η</em> approaches <em>ν</em> from below (cyan arrowheads). These calculations were accelerated using a Fokker-Planck approximation to Eqs. 22–24 (see Section S6).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F6.medium.gif" width="385" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F6.medium.gif" width="385" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <p id="p-48" class="first-child">
                                Parameter studies of the neutral master equation model showing the mean
                                <span class="inline-formula" id="inline-formula-36">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F6/embed/inline-graphic-36.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F6/embed/inline-graphic-36.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , variance
                                <span class="inline-formula" id="inline-formula-37">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F6/embed/inline-graphic-37.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F6/embed/inline-graphic-37.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , and index of dispersion (Fano[
                                <strong>
                                 n
                                </strong>
                                ]) of within-population TE load as a function of the copy- and-paste rate constant (η). Parameters: m = 4 × 10
                                <sup>
                                 3
                                </sup>
                                and as in legend. Cyan curves indicate analytical approximations using ν = 0.1 that are valid in the limit as m → ∞ (see
                                <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                                 Table 2
                                </a>
                                ). These approximations are most accurate for small η/ν and diverge as η approaches ν from below (cyan arrowheads). These calculations were accelerated using a Fokker-Planck approximation to
                                <a id="xref-disp-formula-23-4" class="xref-disp-formula" href="#disp-formula-23">
                                 Eqs. 22
                                </a>
                                –
                                <a id="xref-disp-formula-25-4" class="xref-disp-formula" href="#disp-formula-25">
                                 24
                                </a>
                                (see
                                <a id="xref-sec-32-2" class="xref-sec" href="#sec-32">
                                 Section S6
                                </a>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Influence of selection on overdispersion
                             </h3>
                             <p id="p-49">
                              To investigate the effect of purifying selection on the population variance of TE load, we assume a selection coefficient (w
                              <sub>
                               n
                              </sub>
                              ) that depends on total diploid TE load (n) with dw
                              <sub>
                               n
                              </sub>
                              /dn < 0 (higher load is less viable). For concreteness, let
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where s is the strength of selection against TE load. When the neutral model (
                              <a id="xref-disp-formula-23-5" class="xref-disp-formula" href="#disp-formula-23">
                               Eqs. 22
                              </a>
                              –
                              <a id="xref-disp-formula-25-5" class="xref-disp-formula" href="#disp-formula-25">
                               24
                              </a>
                              ) is modified to include selection, the master equation becomes
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for 1 ≤ n ≤ m. The first term in this expression represents each load probability p
                              <sub>
                               n
                              </sub>
                              relaxing to a target probability
                              <span class="inline-formula" id="inline-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              given by
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where w
                              <sub>
                               i+j
                              </sub>
                              = (1 – s)
                              <sup>
                               i+j
                              </sup>
                              . The equations for for dp
                              <sub>
                               0
                              </sub>
                              /dt and dp
                              <sub>
                               m
                              </sub>
                              /dt have fewer gain/loss terms than
                              <a id="xref-disp-formula-34-1" class="xref-disp-formula" href="#disp-formula-34">
                               Eq. 33
                              </a>
                              , but are analogous (cf.
                              <a id="xref-disp-formula-23-6" class="xref-disp-formula" href="#disp-formula-23">
                               Eqs. 22
                              </a>
                              and
                              <a id="xref-disp-formula-25-6" class="xref-disp-formula" href="#disp-formula-25">
                               24
                              </a>
                              ). The parameter α that occurs in
                              <a id="xref-disp-formula-34-2" class="xref-disp-formula" href="#disp-formula-34">
                               Eq. 33
                              </a>
                              is the inverse of the generation time. The quantity
                              <span class="inline-formula" id="inline-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the mean fitness under the assumption of random mating (
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               Gillespie, 2004
                              </a>
                              ).
                             </p>
                             <p id="p-50">
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Fig. 6
                              </a>
                              shows steady-state distributions of haploid (top row) and diploid (bottom) TE loads calculated using
                              <a id="xref-disp-formula-34-3" class="xref-disp-formula" href="#disp-formula-34">
                               Eq. 33
                              </a>
                              both with and without of selection on diploid load. As expected, for both Mimulus- and Drosophila-like mean TE loads, the effect of weak selection (red and green histograms) is to decrease the TE load in the population as compared to the neutral model (blue histograms). This decrease in mean TE load occurs for a wide range of generation times (1/α) and selection coefficients (s). More important (and less obvious) is the impact of selection on the variance of TE load and overdispersion. Using Drosophila parameters,
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Fig. 6
                              </a>
                              (top right) shows an example simulation (green histogram) in which selection leads to increased dispersion (the Fano factor increases from 1 to 8.66). However, in a second case (red histogram), selection increases the index of dispersion only slightly (to a Fano factor of 1.06). Notably, in three representative simulations using Mimulus parameters, selection does not increase the dispersion of TE load (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Fig. 6
                              </a>
                              , left). This observation is consistent with the moment-based analysis presented in the following section.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F7.large.jpg?width=800&height=600&carousel=1" title="Stationary population distributions of TE abundance with and without selection given by numerical solution of Fokker-Planck equation associated to the master equations (Eqs. 33–34). Parameters: selection coefficient s as in legends. Mimulus: ν = 0.1, m = 109; η0, η = 2000, 0 (blue), 200, 0.095 (red), 20, 0.099 (green). Drosophila: ν = 0.1, m = 5000; η0, η = 20, 0 (blue), 1, 0.1 (green), 2, 0.1 (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Stationary population distributions of TE abundance with and without selection given by numerical solution of Fokker-Planck equation associated to the master equations (Eqs. 33–34). Parameters: selection coefficient s as in legends. <em>Mimulus</em>: <em>ν</em> = 0.1, <em>m</em> = 10<sup>9</sup>; <em>η</em><sub>0</sub>, <em>η</em> = 2000, 0 (blue), 200, 0.095 (red), 20, 0.099 (green). <em>Drosophila</em>: <em>ν</em> = 0.1, <em>m</em> = 5000; <em>η</em><sub>0</sub>, <em>η</em> = 20, 0 (blue), 1, 0.1 (green), 2, 0.1 (red).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F7.medium.gif" width="440" height="384"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F7.medium.gif" width="440" height="384"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <p id="p-51" class="first-child">
                                Stationary population distributions of TE abundance with and without selection given by numerical solution of Fokker-Planck equation associated to the master equations (
                                <a id="xref-disp-formula-34-4" class="xref-disp-formula" href="#disp-formula-34">
                                 Eqs. 33
                                </a>
                                –
                                <a id="xref-disp-formula-35-1" class="xref-disp-formula" href="#disp-formula-35">
                                 34
                                </a>
                                ). Parameters: selection coefficient s as in legends. Mimulus: ν = 0.1, m = 10
                                <sup>
                                 9
                                </sup>
                                ; η
                                <sub>
                                 0
                                </sub>
                                , η = 2000, 0 (blue), 200, 0.095 (red), 20, 0.099 (green). Drosophila: ν = 0.1, m = 5000; η
                                <sub>
                                 0
                                </sub>
                                , η = 20, 0 (blue), 1, 0.1 (green), 2, 0.1 (red).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Moment equations with selection
                             </h3>
                             <p id="p-52">
                              For a deeper understanding of the impact of selection on the distribution of TE load in a population, one may begin with
                              <a id="xref-disp-formula-34-5" class="xref-disp-formula" href="#disp-formula-34">
                               Eqs. 33
                              </a>
                              –
                              <a id="xref-disp-formula-35-2" class="xref-disp-formula" href="#disp-formula-35">
                               34
                              </a>
                              and derive the dynamics of the mean and variance of TE load under the action of simple selection functions. For example, in the limit of weak selection 0 < s ≪ 1,
                              <a id="xref-disp-formula-33-1" class="xref-disp-formula" href="#disp-formula-33">
                               Eq. 32
                              </a>
                              is well-approximated by w
                              <sub>
                               n
                              </sub>
                              = 1 – sn. In this case, as derived in
                              <a id="xref-sec-27-1" class="xref-sec" href="#sec-27">
                               Section S4
                              </a>
                              , the dynamics of the mean and variance of TE load solve
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-53">
                              These ODEs may be compared to the moment equations for the neutral model (
                              <a id="xref-disp-formula-28-3" class="xref-disp-formula" href="#disp-formula-28">
                               Eqs. 27
                              </a>
                              –
                              <a id="xref-disp-formula-29-4" class="xref-disp-formula" href="#disp-formula-29">
                               28
                              </a>
                              ). As expected, the influence of selection on the mean TE load is proportional to the population variance (through the factor
                              <span class="inline-formula" id="inline-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in
                              <a id="xref-disp-formula-36-1" class="xref-disp-formula" href="#disp-formula-36">
                               Eq. 35
                              </a>
                              ). Similarly, the influence of selection on the population variance is proportional to the third central moment of the diploid load (through the factor
                              <span class="inline-formula" id="inline-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in
                              <a id="xref-disp-formula-37-1" class="xref-disp-formula" href="#disp-formula-37">
                               Eq. 36
                              </a>
                              ). In both cases, the quantity
                              <span class="inline-formula" id="inline-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the mean fitness of the population, i.e.,
                              <span class="inline-formula" id="inline-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-54">
                              Under the assumption that the mean TE load is much smaller than the number of loci
                              <span class="inline-formula" id="inline-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we may simplify the moment equations with selection (
                              <a id="xref-disp-formula-36-2" class="xref-disp-formula" href="#disp-formula-36">
                               Eqs. 35
                              </a>
                              –
                              <a id="xref-disp-formula-37-2" class="xref-disp-formula" href="#disp-formula-37">
                               36
                              </a>
                              ) by taking the limit m → ∞ to obtain
                              <span class="disp-formula" id="disp-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-55">
                              Setting the left side of
                              <a id="xref-disp-formula-38-1" class="xref-disp-formula" href="#disp-formula-38">
                               Eq. 37
                              </a>
                              to zero, we observe that the steady-state mean and variance are related as follows,
                              <span class="disp-formula" id="disp-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-56">
                              Comparing this expression to
                              <a id="xref-disp-formula-30-2" class="xref-disp-formula" href="#disp-formula-30">
                               Eq. 29
                              </a>
                              , noting that the variance is nonnegative
                              <span class="inline-formula" id="inline-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we see that the effect of weak selection (0 < s ≪ 1) is to decrease the mean TE load in the population as compared to the neutral model (as expected). Similar analysis of
                              <a id="xref-disp-formula-39-1" class="xref-disp-formula" href="#disp-formula-39">
                               Eq. 38
                              </a>
                              shows how selection may impact the variance of TE load and, consequently, the index of overdispersion. Setting the left side of
                              <a id="xref-disp-formula-39-2" class="xref-disp-formula" href="#disp-formula-39">
                               Eq. 38
                              </a>
                              to zero and solving for the steady-state variance, gives
                              <span class="disp-formula" id="disp-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the first term on the right side, 2η
                              <sub>
                               0
                              </sub>
                              ν/(ν – η)
                              <sup>
                               2
                              </sup>
                              , is the variance of TE load in the absence of selection.
                              <a id="xref-disp-formula-41-1" class="xref-disp-formula" href="#disp-formula-41">
                               Eq. 40
                              </a>
                              shows that the effect of selection is to either decrease or increase the population variance of TE load, depending on the sign of the third central moment
                              <span class="inline-formula" id="inline-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , which is consistent with the master equation simulations shown in
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               Fig. 6
                              </a>
                              .
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Moment closure and the
                              <span class="inline-formula" id="inline-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              phase plane
                             </h3>
                             <p id="p-57">
                              In their current form, the moment equations (
                              <a id="xref-disp-formula-36-3" class="xref-disp-formula" href="#disp-formula-36">
                               Eqs. 35
                              </a>
                              –
                              <a id="xref-disp-formula-37-3" class="xref-disp-formula" href="#disp-formula-37">
                               36
                              </a>
                              ) are an open system of ODEs, because the equation for the variance
                              <span class="inline-formula" id="inline-formula-48">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              depends on
                              <span class="inline-formula" id="inline-formula-49">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the unknown third central moment. As discussed in
                              <a id="xref-sec-28-1" class="xref-sec" href="#sec-28">
                               Section S5
                              </a>
                              , a moment closure technique that is applicable in this situation assumes that the third central moment of the diploid load is algebraic function of the mean and variance,
                              <span class="disp-formula" id="disp-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-58">
                              We investigated two possibilities for this function based on the properties of the beta-binomial and negative binomial distributions. The beta-binomial moment closure, derived in
                              <a id="xref-sec-31-1" class="xref-sec" href="#sec-31">
                               Section S5.3
                              </a>
                              , is a complicated expression involving the mean, variance, and number of loci m,
                              <span class="disp-formula" id="disp-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-59">
                              Moment closure motivated by the properties of the negative binomial distribution results in a simpler expression that does not involve the number of loci m,
                              <span class="disp-formula" id="disp-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-60">
                              Although the beta-binomial closure (
                              <a id="xref-disp-formula-43-1" class="xref-disp-formula" href="#disp-formula-43">
                               Eq. 42
                              </a>
                              ) is arguably a better approximation, in our experience it does not perform markedly better than the negative binomial closure (
                              <a id="xref-disp-formula-44-1" class="xref-disp-formula" href="#disp-formula-44">
                               Eq. 43
                              </a>
                              ), as assessed through comparison of moment ODE and master equation simulations. In the analysis that follows, we use the negative binomial closure, motivated by its simplicity and the fact the two expressions coincide when the number of loci are not limiting (to see this, observe that ψ
                              <sub>
                               BB
                              </sub>
                              → ψ
                              <sub>
                               NB
                              </sub>
                              as m → ∞). When the algebraic relationship representing the negative binomial closure (
                              <a id="xref-disp-formula-44-2" class="xref-disp-formula" href="#disp-formula-44">
                               Eq. 43
                              </a>
                              ) is substituted into
                              <a id="xref-disp-formula-26-4" class="xref-disp-formula" href="#disp-formula-26">
                               Eqs. 25
                              </a>
                              –
                              <a id="xref-disp-formula-27-5" class="xref-disp-formula" href="#disp-formula-27">
                               26
                              </a>
                              , we obtain the following system of ODEs for the mean and variance of diploid load under the influence of selection:
                              <span class="disp-formula" id="disp-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-54.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-54.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-55.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-55.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-61">
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Fig. 7A
                              </a>
                              presents a representative (n,
                              <span class="inline-formula" id="inline-formula-50">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) phase plane for the dynamics of the mean and variance of TE load predicted by
                              <a id="xref-disp-formula-45-1" class="xref-disp-formula" href="#disp-formula-45">
                               Eqs. 44
                              </a>
                              –
                              <a id="xref-disp-formula-46-1" class="xref-disp-formula" href="#disp-formula-46">
                               45
                              </a>
                              . The red and green lines are the nullclines for the mean and variance, respectively, with intersection corresponding to the steady state. This calculation uses parameters resulting in a steady-state TE load similar to our empirical observations of Mimulus guttatus (counts on the order of 10
                              <sup>
                               5
                              </sup>
                              ). This steady state predicted by the moment equations is located far above the broken black line denoting
                              <span class="inline-formula" id="inline-formula-51">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and Fano factor of 1. The blue curves show two solutions that use initial conditions for which the population variance is equal to the mean (solutions were obtained by numerically integrating
                              <a id="xref-disp-formula-45-2" class="xref-disp-formula" href="#disp-formula-45">
                               Eqs. 44
                              </a>
                              –
                              <a id="xref-disp-formula-46-2" class="xref-disp-formula" href="#disp-formula-46">
                               45
                              </a>
                              ). Interestingly, these solutions show that dynamics of TE load can include a transient phase in which the index of dispersion is far greater or less than the steady-state value.
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F8.large.jpg?width=800&height=600&carousel=1" title="The phase plane for the dynamics of mean (n) and variance  of TE load (Eqs. 44–45). The red and green curves are the nullclines for the mean and variance, respectively, with intersection corresponding to the steady state (open circle). The blue trajectories show the dynamics of equilibration. A: Mean loads similar to Mimulus. Parameters: ν = 2.5, m = 5 × 105, η0 = 20 and η = 3 with no selection (α = 0, s = 0). B: Mean loads similar to Drosophila. Parameters: ν = 0.1, m = 5000, η0 = 1 and η = 0.1. Dotted nullclines: no selection. Dashed: s = 10-4, α = 5. Solid: s = 10-4, α = 20. Note that increased selection on TE load (gray arrow) decreases the index of dispersion (Fano[n])." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The phase plane for the dynamics of mean (<em>n</em>) and variance  of TE load (Eqs. 44–45). The red and green curves are the nullclines for the mean and variance, respectively, with intersection corresponding to the steady state (open circle). The blue trajectories show the dynamics of equilibration. A: Mean loads similar to <em>Mimulus</em>. Parameters: <em>ν</em> = 2.5, <em>m</em> = 5 × 10<sup>5</sup>, <em>η</em><sub>0</sub> = 20 and <em>η</em> = 3 with no selection (<em>α</em> = 0, <em>s</em> = 0). B: Mean loads similar to <em>Drosophila</em>. Parameters: <em>ν</em> = 0.1, <em>m</em> = 5000, <em>η</em><sub>0</sub> = 1 and <em>η</em> = 0.1. Dotted nullclines: no selection. Dashed: <em>s</em> = 10<sup>-4</sup>, <em>α</em> = 5. Solid: <em>s</em> = 10<sup>-4</sup>, <em>α</em> = 20. Note that increased selection on TE load (gray arrow) decreases the index of dispersion (Fano[<strong>n</strong>]).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F8.medium.gif" width="293" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F8.medium.gif" width="293" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7:
                               </span>
                               <p id="p-62" class="first-child">
                                The phase plane for the dynamics of mean (n) and variance
                                <span class="inline-formula" id="inline-formula-52">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F8/embed/inline-graphic-52.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F8/embed/inline-graphic-52.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                of TE load (
                                <a id="xref-disp-formula-45-3" class="xref-disp-formula" href="#disp-formula-45">
                                 Eqs. 44
                                </a>
                                –
                                <a id="xref-disp-formula-46-3" class="xref-disp-formula" href="#disp-formula-46">
                                 45
                                </a>
                                ). The red and green curves are the nullclines for the mean and variance, respectively, with intersection corresponding to the steady state (open circle). The blue trajectories show the dynamics of equilibration. A: Mean loads similar to Mimulus. Parameters: ν = 2.5, m = 5 × 10
                                <sup>
                                 5
                                </sup>
                                , η
                                <sub>
                                 0
                                </sub>
                                = 20 and η = 3 with no selection (α = 0, s = 0). B: Mean loads similar to Drosophila. Parameters: ν = 0.1, m = 5000, η
                                <sub>
                                 0
                                </sub>
                                = 1 and η = 0.1. Dotted nullclines: no selection. Dashed: s = 10
                                <sup>
                                 -4
                                </sup>
                                , α = 5. Solid: s = 10
                                <sup>
                                 -4
                                </sup>
                                , α = 20. Note that increased selection on TE load (gray arrow) decreases the index of dispersion (Fano[
                                <strong>
                                 n
                                </strong>
                                ]).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-63">
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               Fig. 7B
                              </a>
                              shows how the nullclines for the mean and variance of TE load depend on the strength of selection in three cases with parameters corresopnding to TE loads similar to Drosophila melanogaster (counts on the order of 100). As the strength of selection increases, both the mean and variance of TE load decrease, in such a manner that the index of dispersion (Fano[
                              <strong>
                               n
                              </strong>
                              ]) decreases.
                             </p>
                             <p id="p-64">
                              Although the model obtained by moment closure and the phase plane analysis of
                              <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                               Fig. 7
                              </a>
                              does not assume
                              <span class="inline-formula" id="inline-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we may consider
                              <a id="xref-disp-formula-45-4" class="xref-disp-formula" href="#disp-formula-45">
                               Eqs. 44
                              </a>
                              –
                              <a id="xref-disp-formula-46-4" class="xref-disp-formula" href="#disp-formula-46">
                               45
                              </a>
                              in the limit as m → ∞,
                              <span class="disp-formula" id="disp-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-57.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-57.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-48">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-65">
                              Setting the left sides of
                              <a id="xref-disp-formula-45-5" class="xref-disp-formula" href="#disp-formula-45">
                               Eqs. 44
                              </a>
                              –
                              <a id="xref-disp-formula-46-5" class="xref-disp-formula" href="#disp-formula-46">
                               45
                              </a>
                              to zero, and assuming weak selection (0 ≤ s ≪ 1), we can derive first-order accurate asymptotic expressions for the steady-state mean and variance,
                              <span class="disp-formula" id="disp-formula-49">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-50">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-60.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-60.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-66">
                              Because υ/(ν – η)
                              <sup>
                               2
                              </sup>
                              > 0, this expression indicates that weak selection decreases the mean TE load, consistent with our intuition. Similarly, the factor (ν + η)/(ν — η)
                              <sup>
                               2
                              </sup>
                              is positive, so we conclude that weak selection decreases the population variance when m is large. As for the index of dispersion, this analysis indicates that under weak selection the Fano factor is
                              <span class="disp-formula" id="disp-formula-51">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-61.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-61.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-67">
                              Because η/(ν – η)
                              <sup>
                               2
                              </sup>
                              is positive any nonzero copy-and-paste rate (η > 0), we conclude that the Fano factor is also expected to decrease, because weak selection causes the within-population variance of TE load to decrease more than the mean. This conclusion—i.e., selection on diploid TE load is unlikely to be responsible for overdispersion—is consistent with the numerical parameter studies summarized in
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Fig. 8
                              </a>
                              that were enabled by the moment equations with selection (
                              <a id="xref-disp-formula-36-4" class="xref-disp-formula" href="#disp-formula-36">
                               Eqs. 35
                              </a>
                              –
                              <a id="xref-disp-formula-37-4" class="xref-disp-formula" href="#disp-formula-37">
                               36
                              </a>
                              ) and beta-binomial moment closure (
                              <a id="xref-disp-formula-43-2" class="xref-disp-formula" href="#disp-formula-43">
                               Eq. 42
                              </a>
                              ).
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F9.large.jpg?width=800&height=600&carousel=1" title="The moment equations derived under the assumption of weak selection (Eqs. 35–36) with beta-binomial moment closure (Eq. 42) enabled these parameter studies of the mean, variance, and dispersion of TE load as a function of generation time (1/α). Parameters: ν = 0.1, η0 = 10, η = 0.1, and as in legend." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The moment equations derived under the assumption of weak selection (Eqs. 35–36) with beta-binomial moment closure (Eq. 42) enabled these parameter studies of the mean, variance, and dispersion of TE load as a function of generation time (1/<em>α</em>). Parameters: <em>ν</em> = 0.1, <em>η</em><sub>0</sub> = 10, <em>η</em> = 0.1, and as in legend.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F9.medium.gif" width="349" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F9.medium.gif" width="349" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8:
                               </span>
                               <p id="p-68" class="first-child">
                                The moment equations derived under the assumption of weak selection (
                                <a id="xref-disp-formula-36-5" class="xref-disp-formula" href="#disp-formula-36">
                                 Eqs. 35
                                </a>
                                –
                                <a id="xref-disp-formula-37-5" class="xref-disp-formula" href="#disp-formula-37">
                                 36
                                </a>
                                ) with beta-binomial moment closure (
                                <a id="xref-disp-formula-43-3" class="xref-disp-formula" href="#disp-formula-43">
                                 Eq. 42
                                </a>
                                ) enabled these parameter studies of the mean, variance, and dispersion of TE load as a function of generation time (1/α). Parameters: ν = 0.1, η
                                <sub>
                                 0
                                </sub>
                                = 10, η = 0.1, and as in legend.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-69">
                             Although mathematical modeling has informed our understanding of the population genetics of transposable elements (TEs) for several decades, classical theory has emphasized analytical results that assume a binomial distribution of TE loads (see
                             <a id="xref-sec-1-1" class="xref-sec" href="#sec-1">
                              Introduction
                             </a>
                             ). Because the variance of a binomial distribution is less than or equal to its mean, the classical theory effectively assumes that the population distribution of TE loads are underdispersed (Fano[
                             <strong>
                              n
                             </strong>
                             ] ≤ 1).
                            </p>
                            <p id="p-70">
                             In an empirical analysis of TE copy number in two natural populations (M. guttatus and D. melanogaster), we found—in both cases—that the population distribution of TE loads was dramatically overdispersed (
                             <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                              Table 1
                             </a>
                             ,
                             <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                              Figs. 1
                             </a>
                             and
                             <a id="xref-fig-2-9" class="xref-fig" href="#F2">
                              2
                             </a>
                             ). Because the classical theory of TE population genetics is not applicable to this situation, we extended this theory and explored mechanisms that may be responsible for the empirically observed overdispersion. The master equation model presented here predicts the entire distribution function of TE loads, and from this distribution we calculate the mean, variance, and index of dispersion as a function of model parameters.
                            </p>
                            <p id="p-71">
                             Prior to considerations of selection, the parameters of the neutral model encode assumptions regarding the dynamics of TE proliferation (e.g., copy-and-paste and excision rate constants) as well as an estimate of the maximum number of loci that may be occupied by TEs. Using parameter sets that yield TE counts in the empirically observed ranges (tens of thousands for M. guttatus, hundreds for D. melanogaster), we found—in both cases—that copy-and-paste TE proliferation dynamics often resulted in overdispersed TE loads (
                             <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                              Fig. 4
                             </a>
                             ). Moment-based analysis of the neutral model suggests that overdispersed population distributions are to be expected when the copy-and-paste transposition rate constant (η) and excision rate constant (ν) are balanced (i.e., η ≈ ν, see
                             <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                              Fig. 5
                             </a>
                             and
                             <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                              Table 1
                             </a>
                             ).
                            </p>
                            <p id="p-72">
                             Next, we extended the master equation model to include purifying selection on TE load. For a parameter set corresponding to M. guttatus, selection decreased the mean and variance of TE load; however, because the variance decreased more than the mean, purifying selection had the effect of decreasing the index of dispersion (
                             <a id="xref-fig-7-5" class="xref-fig" href="#F7">
                              Fig. 6
                             </a>
                             , left). For a parameter set corresponding to D. melanogaster, we found that purifying selection, when sufficiently strong, may lead to an increased index of dispersion of TE load (
                             <a id="xref-fig-7-6" class="xref-fig" href="#F7">
                              Fig. 6
                             </a>
                             , right). In both M. guttatus and D. melanogaster parameter regimes, our simulations (
                             <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                              Fig. 8
                             </a>
                             ) and analysis (
                             <a id="xref-disp-formula-49-1" class="xref-disp-formula" href="#disp-formula-49">
                              Eqs. 48
                             </a>
                             –
                             <a id="xref-disp-formula-51-1" class="xref-disp-formula" href="#disp-formula-51">
                              50
                             </a>
                             ) agree that weak purifying selection decreases both the mean and variance of TE load in such a way that the index of dispersion is unchanged or slightly increases. Moment-based analysis of the master equation confirmed that weak selection usually has the effect of decreasing the index of dispersion.
                            </p>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Comparison of M. guttatus and D. melanogaster
                             </h3>
                             <p id="p-73">
                              Class I elements (retrotransposons) proliferate in a staged manner that involves an RNA intermediate, white Class II elements (DNA transposons) do not utilize an RNA intermediate (
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               Wicker et al., 2007
                              </a>
                              ). In our empirical analysis of TE load in D. melanogaster, we compared these two broad classes of TEs. We found that retrotransposons were 6-fold more highly overdispersed than DNA transposons (see
                              <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              and
                              <a id="xref-fig-2-10" class="xref-fig" href="#F2">
                               Fig. 2B
                              </a>
                              ). Conversely, our empirical analysis of TE load in M. guttatus shows that, in this natural population, DNA transposons are far more overdispersed than retrotransposons. These contrasting empirical results from M. guttatus and D. melanogaster suggest that it is the effective, and perhaps context-dependent, copy-and-past rate (η) of a TE family—as opposed to the mobility mechanism or TE class distinction—that is most relevant to the distribution of within-population TE load.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Limitations of the model
                             </h3>
                             <p id="p-74">
                              The master equation for the dynamics of TE proliferation presented here extends the classical theory in several ways. Most importantly, in both the master equation and moment-based simulations, the relationship between the population variance and mean is a prediction of the model (as opposed to a modeling assumption, as in the classical theory). This feature of the model enables parameter studies exploring how the dynamics of TE proliferation and purifying selection influence the dispersion of within-population TE loads.
                             </p>
                             <p id="p-75">
                              One limitation of our model is the harsh (but common) assumption that selection acts on overall TE load (
                              <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">
                               Charlesworth and Charlesworth, 1983
                              </a>
                              ;
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               Brookfield and Badge, 1997
                              </a>
                              ;
                              <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">
                               Charlesworth and Charlesworth, 2010
                              </a>
                              ;
                              <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                               Deceliere, 2004
                              </a>
                              ;
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               Le Rouzic and Deceliere, 2005
                              </a>
                              ). This choice is consistent with the finding that most TE insertions have negative fitness consequences and are located outside of genes (
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               Bartolomé et al., 2002
                              </a>
                              ;
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               Duret et al., 2000
                              </a>
                              ;
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               Mackay, 1989
                              </a>
                              ;
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               Pasyukova et al., 2004
                              </a>
                              ). On the other hand, many TEs are located in heterochromatic regions of the genome. It is unlikely that these large masses of TEs have fitness consequences comparable to TEs that are proximal to genes. In future work, our model could be extended to include variability in the selective cost of TE insertions, inactivating mutations that lead to nonautonomous TEs, dead-on-arrival TE insertion, and other phenomena that, for simplicity, were not included in this study.
                             </p>
                             <p id="p-76">
                              Arguably, the most significant limitation of the master equation model is that the dynamics of recombination are not represented. Indeed, the population distribution of TE load is modeled without any representation of the location of TEs within the genome. To the extent that recombination promotes linkage equilibrium, one expects that recombination will decrease the dispersion of TE load and, consequently, this aspect of recombination dynamics is unlikely to be responsible for empirically observed overdispersion. We recommend interpreting the master equation and moment-based models as representations of the dynamics of a single linkage class of TEs, with the tacit understanding that the index of dispersion for a genome composed of multiple linkages classes will be less than the model prediction. Admittedly, this viewpoint does not account for the fact that recombination is less frequent in regions of the genome that have a high density of TEs. Recombination hotspots exist in M. guttatus that may impact patterns of TE inheritance and population variance (
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               Hellsten et al., 2013
                              </a>
                              ). However, studying the influence of density-dependent recombination on the dispersion of TE load is beyond the scope of this paper, as it would require a modeling framework that is explicitly spatial.
                             </p>
                             <p id="p-77">
                              We note that events involving the loss or gain of multiple TEs (as could occur via ectopic recombination or other mechanisms) are expected to contribute to overdispersion. To see this, consider a master equation simulation in which the gain and loss of TEs occurs in blocks of size b. If there is no other change to the model, we may reinterpret the random variable
                              <strong>
                               n
                              </strong>
                              as the number of blocks of TEs in a randomly sampled diploid genome. In that case, the mean and variance of TE count are increased by a factor of b and b
                              <sup>
                               2
                              </sup>
                              , respectively. The Fano factor, given by the ratio of variance to mean, increases by a factor of b,
                              <span class="disp-formula" id="disp-formula-52">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-63.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-63.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-78">
                              This scaling implies that block-wise inheritance of TEs is expected to increase the index of dispersion by a factor proportional to a representative block size. This intriguing and relatively simple explanation for empirically observed overdispersion could be studied using an explicitly spatial model of TE population genetics, preferably one that includes a mechanistic account of ectopic recombination and perhaps other genome rearrangements.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-17">
                            <h2 class="">
                             Appendix: Analysis of overdispersion of TE counts
                            </h2>
                            <p id="p-79">
                             <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                              Figs. 1
                             </a>
                             and
                             <a id="xref-fig-2-11" class="xref-fig" href="#F2">
                              2
                             </a>
                             present analyses of two data sets, both of which indicate that the variance of TE load in experimentally studied populations can be far greater than would be predicted by classical models of TE population genetics. The first data set (analyzed in
                             <a id="xref-fig-1-13" class="xref-fig" href="#F1">
                              Fig. 1
                             </a>
                             ) consists of whole-genome sequence data from 164 lines of Mimulus guttatus derived from a naturally occurring population (hundreds of thousands of individuals) in Iron Mountain, Oregon, USA (
                             <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                              Troth et al., 2018
                             </a>
                             ). To estimate TE copy numbers, we compared the coverage of each TE to the average coverage of genes understood to be present in single copy. These short reads were first aligned to a composite genomic database consisting of M. guttatus coding sequences, the mitochondrial genome, M. luteus chloroplast, and a file of approximately 1400 TE sequences (
                             <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                              Hellsten et al., 2013
                             </a>
                             ;
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              Vallejo-Marín et al., 2016
                             </a>
                             ). The M. luteus chloroplast genome was used because it was completely assembled, M. luteus is closely related to M. guttatus, and chloroplast sequences evolve slowly making this a reasonable reference. Next, the whole genome sequencing data from the aforementioned 164 individuals was mapped to this combined reference using Bowtie 2 (
                             <a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">
                              http://bowtie-bio.sourceforge.net/bowtie2/index.shtml
                             </a>
                             ) in its
                             <kbd>
                              --very-sensitive-local
                             </kbd>
                             mode. After this, Picard Tools was used to mark and remove duplicate reads (
                             <a href="https://broadinstitute.github.io/picard/">
                              https://broadinstitute.github.io/picard/
                             </a>
                             ). The remaining reads were then filtered using samtools view (
                             <a href="http://www.htslib.org/">
                              http://www.htslib.org/
                             </a>
                             ) with options
                             <kbd>
                              -h -q 10 -F 0×904
                             </kbd>
                             to exclude reads that were low quality, non-primary, or supplementary. The final list of read counts was processed using a custom Python script to create an array of reads per feature per individual. TE copy numbers were estimated by first removing the reads mapping to mitochondrial, chloroplast, and rRNA genes. The remaining genes were assumed to exist in single copy per chromosome, and the average coverage per feature j (i.e., gene or transposon) in individual i was computed as c
                             <sub>
                              ij
                             </sub>
                             = r
                             <sub>
                              ij
                             </sub>
                             l
                             <sub>
                              ij
                             </sub>
                             /k
                             <sub>
                              j
                             </sub>
                             , where r
                             <sub>
                              ij
                             </sub>
                             and l
                             <sub>
                              ij
                             </sub>
                             are the number and length of reads to feature j in individual i, and k
                             <sub>
                              j
                             </sub>
                             is the annotated length of feature j in the reference genome. To control for genes that might be present in more than single copy, the top 99th percentile of genes were removed (the remaining set of 33,233 genes is denoted G) and the average coverage was computed as
                             <span class="inline-formula" id="inline-formula-54">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-54.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-54.gif"/>
                               </noscript>
                              </span>
                             </span>
                             . Finally, the copy number of all TE features in each individual was estimated as ĉ
                             <sub>
                              ij
                             </sub>
                             = c
                             <sub>
                              ij
                             </sub>
                             /g
                             <sub>
                              ij
                             </sub>
                             .
                            </p>
                            <p id="p-80">
                             The second data set (analyzed in
                             <a id="xref-fig-2-12" class="xref-fig" href="#F2">
                              Fig. 2
                             </a>
                             ) comes from an analysis of 131 lines of Drosophila melanogaster obtained from the Drosophila Genetic Reference Panel (
                             <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                              Cridland et al., 2013
                             </a>
                             , Supplementary Material S5). The individual lines were derived from a large population in Raleigh, North Carolina. In a previously published analysis, Cridland et al. used genomic DNA sequencing to identify over 17,000 TE insertions across these lines. For each insertion (locus), in each individual, this previous work provides a call of present, absent, or indeterminate. Because the vast majority of TE insertions were determined to be rare (83% are present in only one line), we treated loci with indeterminate calls as absent. Elements that were not previously identified as transposons (DNA intermediates) or retrotransposons (RNA intermediate) were excluded. Chromosome 4 was excluded from this analysis, because it is known to have a number of peculiar features (e.g., small size and lack of recombination).
                            </p>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             Conflict of Interest
                            </h2>
                            <p id="p-82">
                             The authors declare that they have no conflict of interest.
                            </p>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Supplemental Materials
                            </h2>
                            <div class="section" id="sec-20">
                             <h3>
                              S1 Comments on the classical model
                             </h3>
                             <p id="p-83">
                              In classical models of TE population genetics (
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               Charlesworth and Charlesworth, 1983
                              </a>
                              ), the state of an infinite diploid population at a given chromosomal site i, for 0 ≤ i ≤ m, is described by its frequency, x
                              <sub>
                               i
                              </sub>
                              , where 0 ≤ x
                              <sub>
                               i
                              </sub>
                              ≤ 1. Assuming insertion sites exhibit no linkage disequilibrium, the set of frequencies,
                              <span class="inline-formula" id="inline-formula-55">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-55.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-55.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , describes the state of the population. The TE load of a randomly sampled diploid individual is a random number
                              <strong>
                               n
                              </strong>
                              given by
                              <span class="disp-formula" id="disp-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-64.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-64.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <strong>
                               x
                              </strong>
                              <sub>
                               i
                              </sub>
                              and
                              <strong>
                               y
                              </strong>
                              <sub>
                               i
                              </sub>
                              are pairs of i.i.d. Bernoulli random variables with parameters x
                              <sub>
                               i
                              </sub>
                              . Thus, E[
                              <strong>
                               x
                              </strong>
                              <sub>
                               i
                              </sub>
                              ] = E[
                              <strong>
                               y
                              </strong>
                              <sub>
                               i
                              </sub>
                              ] = x¿ and the mean copy number of TEs per individual is
                              <span class="disp-formula" id="disp-formula-54">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-65.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-65.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where in the last equality we have written n in terms of the number of loci and the mean frequency,
                              <span class="inline-formula" id="inline-formula-56">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-56.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-56.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The variance of a Bernoulli random variable with parameter x
                              <sub>
                               i
                              </sub>
                              is x
                              <sub>
                               i
                              </sub>
                              (1 – x
                              <sub>
                               i
                              </sub>
                              ). As a consequence, the variance of TE load is
                              <span class="disp-formula" id="disp-formula-55">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-66.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-66.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-84">
                              The diploid load given by
                              <a id="xref-disp-formula-53-1" class="xref-disp-formula" href="#disp-formula-53">
                               Eq. S1
                              </a>
                              is the sum of two i.i.d. Poisson-binomial random variables, that is,
                              <strong>
                               n
                              </strong>
                              =
                              <strong>
                               X
                              </strong>
                              +
                              <strong>
                               Y
                              </strong>
                              where
                              <span class="inline-formula" id="inline-formula-57">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-57.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-57.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the sum of independent Bernoulli random variables that are not necessarily independent (and similarly for
                              <span class="inline-formula" id="inline-formula-58">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ). It is well-known that
                              <span class="disp-formula" id="disp-formula-56">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-67.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-67.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-59">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the “variance” among the parameters of the Poisson-binomial distribution,
                              <span class="inline-formula" id="inline-formula-60">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-60.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-60.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (i.e., the variability of frequencies of occupation of the TE loci). Using Var[
                              <strong>
                               Y
                              </strong>
                              ] = Var[
                              <strong>
                               X
                              </strong>
                              ] and Var[
                              <strong>
                               n
                              </strong>
                              ] = 2Var[
                              <strong>
                               X
                              </strong>
                              ], and
                              <a id="xref-disp-formula-56-1" class="xref-disp-formula" href="#disp-formula-56">
                               Eq. S2
                              </a>
                              , we see that the variance of TE load is
                              <span class="disp-formula" id="disp-formula-57">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-68.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-68.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-85">
                              Substituting
                              <span class="inline-formula" id="inline-formula-61">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-61.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-61.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in the above expression gives
                              <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                               Eq. 6
                              </a>
                              .
                             </p>
                             <p id="p-86">
                              To extend this model of TE population genetics to include the effect of natural selection,
                              <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                               Charlesworth and Charlesworth (1983)
                              </a>
                              assume a viability function, w
                              <sub>
                               n
                              </sub>
                              , that is a decreasing function of total genomewide TE load (dw
                              <sub>
                               n
                              </sub>
                              /dn < 0). The effect of selection is to decrease the occupation frequency at each loci in a manner that is proportional to
                              <span class="inline-formula" id="inline-formula-62">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-62.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-62.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , which is the variance of a Bernoulli random variable with parameter x
                              <sub>
                               i
                              </sub>
                              , and also proportional the derivative, with respect to x
                              <sub>
                               i
                              </sub>
                              , of the mean fitness of the population,
                              <span class="inline-formula" id="inline-formula-63">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-63.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-63.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,
                              <span class="disp-formula" id="disp-formula-58">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-69.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-69.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-87">
                              The second equality is obtained using
                              <span class="inline-formula" id="inline-formula-64">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-64.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-64.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and noting that
                              <span class="inline-formula" id="inline-formula-65">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-65.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-65.gif"/>
                                </noscript>
                               </span>
                              </span>
                              implies
                              <span class="inline-formula" id="inline-formula-66">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-66.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-66.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Summing over all sites gives
                              <span class="disp-formula" id="disp-formula-59">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-70.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-70.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-88">
                              Using
                              <span class="inline-formula" id="inline-formula-67">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-67.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-67.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and approximating the mean fitness of the population
                              <span class="inline-formula" id="inline-formula-68">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-68.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-68.gif"/>
                                </noscript>
                               </span>
                              </span>
                              by the fitness of an individual with an average number of copies
                              <span class="inline-formula" id="inline-formula-69">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-69.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-69.gif"/>
                                </noscript>
                               </span>
                              </span>
                              gives the first term of
                              <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">
                               Eq. 3
                              </a>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              S2 Statistics of indistinguishable TE families
                             </h3>
                             <p id="p-89">
                              For TE families with independent proliferation and excision dynamics, the dispersion of TE load that results when families are not distinguished is always less than the overdispersion of at least one of the TE families. To see this, consider the generalization of
                              <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">
                               Eq. 12
                              </a>
                              to an arbitrary number of TE families,
                              <span class="disp-formula" id="disp-formula-60">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-71.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-71.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-90">
                              Substituting
                              <span class="inline-formula" id="inline-formula-70">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-70.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-70.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and dividing by
                              <span class="inline-formula" id="inline-formula-71">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-71.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-71.gif"/>
                                </noscript>
                               </span>
                              </span>
                              gives an expression for the composite index of dispersion,
                              <span class="disp-formula" id="disp-formula-61">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-72.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-72.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-91">
                              Assuming that the within-population loads for the families of TEs are independent, the covariances will be zero. In that case,
                              <span class="disp-formula" id="disp-formula-62">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-73.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-73.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-72">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-72.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-72.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Thus, the composite Fano factor F is in the range, min
                              <sub>
                               i
                              </sub>
                              F
                              <sub>
                               i
                              </sub>
                              ≤ F ≤ max
                              <sub>
                               i
                              </sub>
                              F
                              <sub>
                               i
                              </sub>
                              .
                             </p>
                             <p id="p-92">
                              The situation is more complicated when the dynamics of TE families are not independent. For simplicity, consider the case of two TE families. Beginning with
                              <a id="xref-disp-formula-13-2" class="xref-disp-formula" href="#disp-formula-13">
                               Eq. 12
                              </a>
                              and the definition of the correlation of two random variables,
                              <span class="disp-formula" id="disp-formula-63">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-74.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-74.gif"/>
                                </noscript>
                               </span>
                              </span>
                              the composite Fano factor may be expressed as follows,
                              <span class="disp-formula" id="disp-formula-64">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-75.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-75.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-65">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-76.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-76.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-73">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-73.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-73.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the fraction of TEs in the second family, takes values between 0 and 1, and the correlation function, Corr[
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              ] takes values between −1 and 1. When there is no correlation (Corr[
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              ] = 0), the composite Fano factor is a weighted average of F
                              <sub>
                               1
                              </sub>
                              and F
                              <sub>
                               2
                              </sub>
                              , namely,
                              <span class="disp-formula" id="disp-formula-66">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-77.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-77.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F10.large.jpg?width=800&height=600&carousel=1" title="The composite index of dispersion, Fano[x1 + x2], as a function of the fraction of TEs in the second of two families , depends on the Fano factors of both families and the correlation coefficient (Corr[x1, x2]) for the loads of the families within the population (see Eqs. S3–S4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-440826032" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The composite index of dispersion, Fano[<strong>x</strong><sub>1</sub> + <strong>x</strong><sub>2</sub>], as a function of the fraction of TEs in the second of two families , depends on the Fano factors of both families and the correlation coefficient (Corr[<strong>x</strong><sub>1</sub>, <strong>x</strong><sub>2</sub>]) for the loads of the families within the population (see Eqs. S3–S4).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F10.medium.gif" width="440" height="206"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F10.medium.gif" width="440" height="206"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/04/2020.11.16.382945/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S1:
                               </span>
                               <p id="p-93" class="first-child">
                                The composite index of dispersion, Fano[
                                <strong>
                                 x
                                </strong>
                                <sub>
                                 1
                                </sub>
                                +
                                <strong>
                                 x
                                </strong>
                                <sub>
                                 2
                                </sub>
                                ], as a function of the fraction of TEs in the second of two families
                                <span class="inline-formula" id="inline-formula-74">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F10/embed/inline-graphic-74.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/F10/embed/inline-graphic-74.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , depends on the Fano factors of both families and the correlation coefficient (Corr[
                                <strong>
                                 x
                                </strong>
                                <sub>
                                 1
                                </sub>
                                ,
                                <strong>
                                 x
                                </strong>
                                <sub>
                                 2
                                </sub>
                                ]) for the loads of the families within the population (see
                                <a id="xref-disp-formula-64-1" class="xref-disp-formula" href="#disp-formula-64">
                                 Eqs. S3
                                </a>
                                –
                                <a id="xref-disp-formula-65-1" class="xref-disp-formula" href="#disp-formula-65">
                                 S4
                                </a>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-94">
                              For fixed ρ, F is maximized when Corr[
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              ] = 1,
                              <span class="disp-formula" id="disp-formula-67">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-79.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-79.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-95">
                              Similarly, F is minimized when Corr[
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              ] = −1,
                              <span class="disp-formula" id="disp-formula-68">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-80.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-80.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-96">
                              It can be shown that max
                              <sub>
                               ρ
                              </sub>
                              F
                              <sub>
                               +
                              </sub>
                              = F
                              <sub>
                               1
                              </sub>
                              + F
                              <sub>
                               2
                              </sub>
                              and min
                              <sub>
                               ρ
                              </sub>
                              F
                              <sub>
                               −
                              </sub>
                              = 0. For less extreme values of the correlation, we have Corr[
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              ] > 0 ⇒ F > F
                              <sub>
                               0
                              </sub>
                              and Corr[
                              <strong>
                               x
                              </strong>
                              <sub>
                               1
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               2
                              </sub>
                              ] < 0 ⇒ F < F
                              <sub>
                               0
                              </sub>
                              for all 0 ≤ ρ ≤ 1 (see
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Fig. S1
                              </a>
                              ). These calculations show that the composite Fano factor can either increase or decrease when families of TEs are lumped into larger groups, or split into smaller groups. The composite Fano factor will be greater or less than the weighted average given by
                              <a id="xref-disp-formula-66-1" class="xref-disp-formula" href="#disp-formula-66">
                               Eq. S5
                              </a>
                              depending on whether the TE loads are positively or negatively correlated in the population.
                             </p>
                             <p id="p-97">
                              For example, the overdispersion observed documented in
                              <a id="xref-fig-1-14" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              can be analyzed at the level of family, class, or in toto. At the level of class,
                              <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              shows Fano factors for M. guttatus of 61 for Class I and 646 for Class II. When the two classes of TEs are combined, the overall Fano factor is 525, which is an intermediate value (61 < 525 < 646). This composite Fano factor is slightly higher than would be obtained under the assumption of independence, which is (1 – ρ)61 + ρ646 = 513 (
                              <a id="xref-disp-formula-66-2" class="xref-disp-formula" href="#disp-formula-66">
                               Eq. S5
                              </a>
                              ), where we have used the proportion of TEs in each class (ρ = 0.772). This calculation is consistent with the small positive correlation between the TE loads for the two classes (Corr[
                              <strong>
                               x
                              </strong>
                              <sub>
                               Class I
                              </sub>
                              ,
                              <strong>
                               x
                              </strong>
                              <sub>
                               Class II
                              </sub>
                              ] = 0.0734).
                             </p>
                             <p id="p-98">
                              When analyzed at the level of family, the Fano factors for Class I elements of M. guttatus range from 7 to 31, while for Class II elements the range is 12 to 228 (see
                              <a id="xref-fig-1-15" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              of main text). These ranges are less than the indices of dispersion observed at the level of class (61 and 646, respectively), because the loads for TE families (within each class) are positively correlated. The symmetric matrix of correlation coefficients at the level of family are shown below.
                             </p>
                             <div id="T3" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2338363/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2338363/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2338363" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                             </div>
                             <p id="p-99">
                              The six Class II elements (DNA transposons) are listed first, followed by four Class I elements (retro-transposons). The on- and off-diagonal blocks of this matrix correspond to comparisons of TE families within and between classes. Note that the small positive and negative correlations are predominantly located within the off-diagonal blocks, where loads of Class I and II elements are being compared.
                             </p>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              S3 Derivation of moment equations with gain and loss terms
                             </h3>
                             <p id="p-100">
                              Let
                              <strong>
                               n
                              </strong>
                              be a random variable representing TE load of a randomly sampled diploid individual, and
                              <strong>
                               x
                              </strong>
                              and
                              <strong>
                               y
                              </strong>
                              be random variables representing the TE load of randomly sampled haploid genomes (gametes). Define the moments of the probability distribution of haploid TE loads (given by p
                              <sub>
                               i
                              </sub>
                              for 0 ≤ i ≤ m) as
                              <span class="disp-formula" id="disp-formula-69">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-82.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-82.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where μ
                              <sub>
                               0
                              </sub>
                              = 1 (conservation of probability), μ
                              <sub>
                               1
                              </sub>
                              = E[
                              <strong>
                               x
                              </strong>
                              ] and
                              <span class="inline-formula" id="inline-formula-75">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-75.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-75.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . We assume random mating for which the diploid TE load is the sum of two i.i.d. gametic loads (
                              <strong>
                               n
                              </strong>
                              =
                              <strong>
                               x
                              </strong>
                              +
                              <strong>
                               y
                              </strong>
                              ). In the case, the mean and variance of the within-population diploid TE load are related to μ
                              <sub>
                               2
                              </sub>
                              and μ
                              <sub>
                               1
                              </sub>
                              as follows,
                              <span class="disp-formula" id="disp-formula-70">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-83.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-83.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-71">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-84.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-84.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-101">
                              Using
                              <a id="xref-disp-formula-70-1" class="xref-disp-formula" href="#disp-formula-70">
                               Eqs. S7
                              </a>
                              –
                              <a id="xref-disp-formula-71-1" class="xref-disp-formula" href="#disp-formula-71">
                               S8
                              </a>
                              we see that the Fano factor of the diploid load is
                              <span class="disp-formula" id="disp-formula-72">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-85.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-85.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-102">
                              Because the factor of two occurs in both the numerator and denominator of the above expression, the dispersion of the diploid load is equal to that of the haploid load, i.e. Fano[
                              <strong>
                               n
                              </strong>
                              ] = Fano[
                              <strong>
                               x
                              </strong>
                              ].
                             </p>
                             <div class="section" id="sec-23">
                              <h4>
                               S3.1 ODEs for the dynamics of μ
                               <sub>
                                1
                               </sub>
                               and μ
                               <sub>
                                2
                               </sub>
                              </h4>
                              <p id="p-103">
                               The moment equations for the haploid TE load are derived by differentiating
                               <a id="xref-disp-formula-69-1" class="xref-disp-formula" href="#disp-formula-69">
                                Eq. S6
                               </a>
                               to obtain
                               <span class="disp-formula" id="disp-formula-73">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-86.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-86.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-104">
                               Substituting
                               <a id="xref-disp-formula-23-7" class="xref-disp-formula" href="#disp-formula-23">
                                Eqs. 22
                               </a>
                               –
                               <a id="xref-disp-formula-25-7" class="xref-disp-formula" href="#disp-formula-25">
                                24
                               </a>
                               into the above expression gives
                               <span class="disp-formula" id="disp-formula-74">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-87.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-87.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-105">
                               The terms V
                               <sub>
                                q
                               </sub>
                               are evaluated as follows,
                               <span class="disp-formula" id="disp-formula-75">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-88.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-88.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where we use υ
                               <sub>
                                0
                               </sub>
                               = 0. For q = 1, [(n – 1)
                               <sup>
                                q
                               </sup>
                               – n
                               <sup>
                                q
                               </sup>
                               ] = −1; thus, V
                               <sub>
                                1
                               </sub>
                               is given by
                               <span class="disp-formula" id="disp-formula-76">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-89.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-89.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where we use υ
                               <sub>
                                n
                               </sub>
                               = νn. Similarly,
                               <span class="disp-formula" id="disp-formula-77">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-90.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-90.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-106">
                               Using q = 1, u
                               <sub>
                                m
                               </sub>
                               = 0, [(n + 1)
                               <sup>
                                q
                               </sup>
                               – n
                               <sup>
                                q
                               </sup>
                               ] = +1, and u
                               <sub>
                                n
                               </sub>
                               = (η
                               <sub>
                                0
                               </sub>
                               + ηn)(1 – n/m), we obtain
                               <span class="disp-formula" id="disp-formula-78">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-91.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-91.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-107">
                               Thus, U
                               <sub>
                                1
                               </sub>
                               is given by
                               <span class="disp-formula" id="disp-formula-79">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-92.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-92.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-108">
                               Combining the expression for U
                               <sub>
                                1
                               </sub>
                               and V
                               <sub>
                                1
                               </sub>
                               gives the ODE for μ
                               <sub>
                                1
                               </sub>
                               , the first moment of haploid TE load,
                               <span class="disp-formula" id="disp-formula-80">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-93.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-93.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-109">
                               Similar calculations give dμ
                               <sub>
                                0
                               </sub>
                               /dt = 0 (conservation of probability) and
                               <span class="disp-formula" id="disp-formula-81">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-94.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-94.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-82">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-95.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-95.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-110">
                               Using these expressions, the ODE for the second moment, dμ
                               <sub>
                                2
                               </sub>
                               /dt = V
                               <sub>
                                2
                               </sub>
                               + U
                               <sub>
                                2
                               </sub>
                               , is found to be
                               <span class="disp-formula" id="disp-formula-83">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-96.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-96.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-111">
                               Eqs. S11–S14 are the first two ODEs in a sequence for which dμ
                               <sub>
                                1
                               </sub>
                               /dt depends on μ
                               <sub>
                                1
                               </sub>
                               and μ
                               <sub>
                                2
                               </sub>
                               , dμ
                               <sub>
                                2
                               </sub>
                               /dt depends on μ
                               <sub>
                                1
                               </sub>
                               , μ
                               <sub>
                                2
                               </sub>
                               and μ
                               <sub>
                                3
                               </sub>
                               , and so on, as follows,
                               <span class="disp-formula" id="disp-formula-84">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-97.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-97.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-85">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-98.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-98.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-86">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-99.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-99.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-112">
                               <a id="xref-sec-28-2" class="xref-sec" href="#sec-28">
                                Section S5
                               </a>
                               shows how this open system of ODEs can be closed by assuming an algebraic relationship between the third moment and those of lower order. The influence of selection on the both the master equation and moment equation models is discussed in
                               <a id="xref-sec-27-2" class="xref-sec" href="#sec-27">
                                Section S4
                               </a>
                               . The following two sections (
                               <a id="xref-sec-24-2" class="xref-sec" href="#sec-24">
                                Sections S3.2
                               </a>
                               and
                               <a id="xref-sec-25-2" class="xref-sec" href="#sec-25">
                                S3.3
                               </a>
                               ) explore parameter regimes for which the moment equations decouple and it is possible to derive analytical steady states for the population mean and variance of TE load.
                              </p>
                             </div>
                             <div class="section" id="sec-24">
                              <h4>
                               S3.2 Moment ODEs in absence of copy-and-paste transposition
                              </h4>
                              <p id="p-113">
                               When there is no copy-and-paste transposition (η = 0),
                               <a id="xref-disp-formula-80-1" class="xref-disp-formula" href="#disp-formula-80">
                                Eqs. S11
                               </a>
                               –
                               <a id="xref-disp-formula-83-1" class="xref-disp-formula" href="#disp-formula-83">
                                S14
                               </a>
                               simplify as follows:
                               <span class="disp-formula" id="disp-formula-87">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-100.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-100.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-114">
                               Notice that the dependence of dμ
                               <sub>
                                1
                               </sub>
                               /dt on μ
                               <sub>
                                2
                               </sub>
                               , and dμ
                               <sub>
                                2
                               </sub>
                               /dt on μ
                               <sub>
                                3
                               </sub>
                               , vanishes when η = 0. Regrouping terms gives
                               <span class="disp-formula" id="disp-formula-88">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-101.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-101.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-115">
                               This system has steady state given by
                               <span class="disp-formula" id="disp-formula-89">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-102.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-102.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-116">
                               The central moment
                               <span class="inline-formula" id="inline-formula-76">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-76.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-76.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , the variance in haploid load, is thus
                               <span class="disp-formula" id="disp-formula-90">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-103.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-103.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-117">
                               Noting that
                               <span class="inline-formula" id="inline-formula-77">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-77.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-77.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-78">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-78.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-78.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we find
                               <span class="disp-formula" id="disp-formula-91">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-104.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-104.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-118">
                               The index of dispersion for the diploid load is thus
                               <span class="disp-formula" id="disp-formula-92">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-105.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-105.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div class="section" id="sec-25">
                              <h4>
                               S3.3 Moment ODEs when occupiable loci are not limiting
                              </h4>
                              <p id="p-119">
                               When occupiable loci are not limiting (μ
                               <sub>
                                1
                               </sub>
                               ≪ m), we may consider
                               <a id="xref-disp-formula-80-2" class="xref-disp-formula" href="#disp-formula-80">
                                Eqs. S11
                               </a>
                               –
                               <a id="xref-disp-formula-83-2" class="xref-disp-formula" href="#disp-formula-83">
                                S14
                               </a>
                               in the limit as m → ∞,
                               <span class="disp-formula" id="disp-formula-93">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-106.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-106.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-120">
                               Note that the large m limit uncouples the moment ODEs. Regrouping terms, we see that
                               <span class="disp-formula" id="disp-formula-94">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-107.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-107.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-121">
                               Provided ν > η, this system has the stable steady state given by
                               <span class="disp-formula" id="disp-formula-95">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-108.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-108.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-122">
                               The central moment
                               <span class="inline-formula" id="inline-formula-79">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-79.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-79.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , the variance in haploid load, is thus
                               <span class="disp-formula" id="disp-formula-96">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-109.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-109.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-123">
                               Noting that
                               <span class="inline-formula" id="inline-formula-80">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-80.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-80.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-81">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-81.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-81.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               gives
                               <a id="xref-disp-formula-30-3" class="xref-disp-formula" href="#disp-formula-30">
                                Eqs. 29
                               </a>
                               –
                               <a id="xref-disp-formula-31-2" class="xref-disp-formula" href="#disp-formula-31">
                                30
                               </a>
                               , namely,
                               <span class="disp-formula" id="disp-formula-97">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-110.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-110.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-124">
                               The index of dispersion for the diploid load is thus
                               <span class="disp-formula" id="disp-formula-98">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-111.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-111.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div class="section" id="sec-26">
                              <h4>
                               S3.4 Central moment equations
                              </h4>
                              <p id="p-125">
                               Recall that the open system of moment equations for the probability distribution of TE load take the form
                               <span class="disp-formula" id="disp-formula-99">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-112.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-112.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where dμ
                               <sub>
                                3
                               </sub>
                               /dt = f
                               <sub>
                                3
                               </sub>
                               (μ
                               <sub>
                                2
                               </sub>
                               , μ
                               <sub>
                                3
                               </sub>
                               , μ
                               <sub>
                                4
                               </sub>
                               ), and so on (cf.
                               <a id="xref-disp-formula-84-1" class="xref-disp-formula" href="#disp-formula-84">
                                Eqs. S15
                               </a>
                               –
                               <a id="xref-disp-formula-86-1" class="xref-disp-formula" href="#disp-formula-86">
                                S17
                               </a>
                               ). Rearranging terms in the equations for the first two moments gives
                               <span class="disp-formula" id="disp-formula-100">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-113.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-113.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-101">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-114.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-114.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-126">
                               It is convenient to express
                               <a id="xref-disp-formula-100-1" class="xref-disp-formula" href="#disp-formula-100">
                                Eqs. S18
                               </a>
                               –
                               <a id="xref-disp-formula-101-1" class="xref-disp-formula" href="#disp-formula-101">
                                S19
                               </a>
                               in terms of the central moments. The first central moment of the haploid TE load is the mean, μ
                               <sub>
                                1
                               </sub>
                               = E[
                               <strong>
                                x
                               </strong>
                               ]. The second central moment is the variance
                               <span class="disp-formula" id="disp-formula-102">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-115.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-115.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-127">
                               The third central moment is
                               <span class="disp-formula" id="disp-formula-103">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-116.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-116.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-128">
                               To find an ODE for the dynamics of the variance, we differentiate
                               <span class="inline-formula" id="inline-formula-82">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-82.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-82.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to obtain
                               <span class="disp-formula" id="disp-formula-104">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-117.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-117.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-129">
                               Substituting
                               <a id="xref-disp-formula-100-2" class="xref-disp-formula" href="#disp-formula-100">
                                Eqs. S18
                               </a>
                               –
                               <a id="xref-disp-formula-101-2" class="xref-disp-formula" href="#disp-formula-101">
                                S19
                               </a>
                               into this expression we obtain,
                               <span class="disp-formula" id="disp-formula-105">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-118.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-118.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-106">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-119.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-119.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-130">
                               Using
                               <span class="inline-formula" id="inline-formula-83">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-83.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-83.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-84">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-84.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-84.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and
                               <span class="inline-formula" id="inline-formula-85">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-85.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-85.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ,
                               <a id="xref-disp-formula-105-1" class="xref-disp-formula" href="#disp-formula-105">
                                Eqs. S22
                               </a>
                               –
                               <a id="xref-disp-formula-106-1" class="xref-disp-formula" href="#disp-formula-106">
                                S23
                               </a>
                               may be transformed into equations for the mean and variance of diploid load. To see this, write
                               <span class="inline-formula" id="inline-formula-86">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-86.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-86.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-87">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-87.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-87.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and differentiate to obtain
                               <span class="disp-formula" id="disp-formula-107">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-120.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-120.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-131">
                               Substitution gives
                               <span class="disp-formula" id="disp-formula-108">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-121.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-121.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where we gave used
                               <a id="xref-disp-formula-103-1" class="xref-disp-formula" href="#disp-formula-103">
                                Eq. S20
                               </a>
                               . After simplifying, these equations become
                               <span class="disp-formula" id="disp-formula-109">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-122.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-122.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-110">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-123.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-123.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-132">
                               Taking the limit as m → ∞ gives
                               <a id="xref-disp-formula-28-4" class="xref-disp-formula" href="#disp-formula-28">
                                Eqs. 27
                               </a>
                               –
                               <a id="xref-disp-formula-29-5" class="xref-disp-formula" href="#disp-formula-29">
                                28
                               </a>
                               .
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-27">
                             <h3>
                              S4 Selection in the master equation and moment equation models
                             </h3>
                             <p id="p-133">
                              In the master equation formulation, p
                              <sub>
                               n
                              </sub>
                              is the probability of randomly sampling a gamete with a TE load of n. Under the assumption of random mating, selection leads to the following probabilities for each load in the next generation,
                              <span class="disp-formula" id="disp-formula-111">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-124.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-124.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="disp-formula" id="disp-formula-112">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-125.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-125.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the mean fitness of the diploid population. Selection may be included in the master equations for TE load (
                              <a id="xref-disp-formula-23-8" class="xref-disp-formula" href="#disp-formula-23">
                               Eqs. 22
                              </a>
                              –
                              <a id="xref-disp-formula-25-8" class="xref-disp-formula" href="#disp-formula-25">
                               24
                              </a>
                              ) as follows,
                              <span class="disp-formula" id="disp-formula-113">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-126.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-126.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where for typographical convenience we do not write the reaction terms involving u
                              <sub>
                               n
                              </sub>
                              and υ
                              <sub>
                               n
                              </sub>
                              (these are indicated by…). In the weak selection limit, w
                              <sub>
                               n
                              </sub>
                              = (1 – s)
                              <sup>
                               n
                              </sup>
                              ≈ 1 – sn and the mean fitness
                              <span class="inline-formula" id="inline-formula-88">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-88.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-88.gif"/>
                                </noscript>
                               </span>
                              </span>
                              becomes
                              <span class="disp-formula" id="disp-formula-114">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-127.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-127.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-134">
                              Thus, weak selection can be included in the moment equations as follows
                              <span class="disp-formula" id="disp-formula-115">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-128.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-128.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-135">
                              This expression leads to the following differential equation for the first moment in the weak selection limit,
                              <span class="disp-formula" id="disp-formula-116">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-129.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-129.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-117">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-130.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-130.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the quantity in brackets is the variance in haploid load
                              <span class="inline-formula" id="inline-formula-89">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-89.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-89.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and V
                              <sub>
                               1
                              </sub>
                              and U
                              <sub>
                               1
                              </sub>
                              are given by
                              <a id="xref-disp-formula-76-1" class="xref-disp-formula" href="#disp-formula-76">
                               Eqs. S9
                              </a>
                              –
                              <a id="xref-disp-formula-79-1" class="xref-disp-formula" href="#disp-formula-79">
                               S10
                              </a>
                              . A similar calculation gives the dynamics of the second moment of the haploid load, moment
                              <span class="disp-formula" id="disp-formula-118">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-131.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-131.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where V
                              <sub>
                               2
                              </sub>
                              and U
                              <sub>
                               2
                              </sub>
                              are given by
                              <a id="xref-disp-formula-81-1" class="xref-disp-formula" href="#disp-formula-81">
                               Eqs. S12
                              </a>
                              –
                              <a id="xref-disp-formula-82-1" class="xref-disp-formula" href="#disp-formula-82">
                               S13
                              </a>
                              . Using
                              <a id="xref-disp-formula-104-1" class="xref-disp-formula" href="#disp-formula-104">
                               Eq. S21
                              </a>
                              , an ODE for the variance in haploid load is found,
                              <span class="disp-formula" id="disp-formula-119">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-132.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-132.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-136">
                              Using
                              <span class="inline-formula" id="inline-formula-90">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-90.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-90.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-91">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-91.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-91.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , and
                              <span class="inline-formula" id="inline-formula-92">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-92.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-92.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we see that
                              <a id="xref-disp-formula-117-1" class="xref-disp-formula" href="#disp-formula-117">
                               Eq. S27
                              </a>
                              is equivalent to
                              <span class="disp-formula" id="disp-formula-120">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-133.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-133.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-137">
                              Using
                              <span class="inline-formula" id="inline-formula-93">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-93.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-93.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-94">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-94.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-94.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we obtain
                              <span class="disp-formula" id="disp-formula-121">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-134.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-134.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-138">
                              Combining these results for the effect of selection with the reaction terms of the neutral model (
                              <a id="xref-disp-formula-109-1" class="xref-disp-formula" href="#disp-formula-109">
                               Eqs. S24
                              </a>
                              –
                              <a id="xref-disp-formula-110-1" class="xref-disp-formula" href="#disp-formula-110">
                               S25
                              </a>
                              ), we obtain the following equations for the mean and variance of diploid load under the influence of selection:
                              <span class="disp-formula" id="disp-formula-122">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-135.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-135.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-123">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-136.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-136.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-139">
                              Taking the limit of
                              <a id="xref-disp-formula-122-1" class="xref-disp-formula" href="#disp-formula-122">
                               Eqs. S28
                              </a>
                              –
                              <a id="xref-disp-formula-123-1" class="xref-disp-formula" href="#disp-formula-123">
                               S29
                              </a>
                              as m → ∞ gives
                              <a id="xref-disp-formula-38-2" class="xref-disp-formula" href="#disp-formula-38">
                               Eqs. 37
                              </a>
                              –
                              <a id="xref-disp-formula-39-3" class="xref-disp-formula" href="#disp-formula-39">
                               38
                              </a>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-28">
                             <h3>
                              S5 Moment closure
                             </h3>
                             <p id="p-140">
                              To analyze solutions of
                              <a id="xref-disp-formula-122-2" class="xref-disp-formula" href="#disp-formula-122">
                               Eqs. S28
                              </a>
                              –
                              <a id="xref-disp-formula-123-2" class="xref-disp-formula" href="#disp-formula-123">
                               S29
                              </a>
                              without assuming that m is large or η is zero, the dependence of
                              <span class="inline-formula" id="inline-formula-95">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-95.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-95.gif"/>
                                </noscript>
                               </span>
                              </span>
                              on
                              <span class="inline-formula" id="inline-formula-96">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-96.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-96.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (equivalently, the dependence of
                              <span class="inline-formula" id="inline-formula-97">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-97.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-97.gif"/>
                                </noscript>
                               </span>
                              </span>
                              on
                              <span class="inline-formula" id="inline-formula-98">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-98.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-98.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) must be accounted for. This is accomplished using the technique of moment closure, whereby we assume an algebraic relationship of the form μ
                              <sub>
                               3
                              </sub>
                              = ψ(μ
                              <sub>
                               2
                              </sub>
                              , μ
                              <sub>
                               1
                              </sub>
                              ) or
                              <span class="disp-formula" id="disp-formula-124">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-137.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-137.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-141">
                              One way to motivate a particular choice of algebraic relationship ψ is to select a distribution with properties similar to those exhibited by the master equation simulations. Next, one derives the relation
                              <a id="xref-disp-formula-124-1" class="xref-disp-formula" href="#disp-formula-124">
                               Eq. S30
                              </a>
                              that would be exact if the model truly exhibited the selected distribution.
                             </p>
                             <div class="section" id="sec-29">
                              <h4>
                               S5.1 Negative binomial closure
                              </h4>
                              <p id="p-142">
                               One possibility we have investigated is the negative binomial distribution. This choice is motivated by a few key properties. First, the negative binomial distribution is supported on the non-negative integers. The probability mass function for a negative binomial random variable,
                               <strong>
                                X
                               </strong>
                               ~ NB(r, p) for r > 0 and p ∈ [0, 1], is
                               <span class="disp-formula" id="disp-formula-125">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-138.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-138.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-143">
                               Second, overdispersion is a propery of the the negative binomial distribution. The mean μ
                               <sub>
                                1
                               </sub>
                               = E[
                               <strong>
                                X
                               </strong>
                               ], variance
                               <span class="inline-formula" id="inline-formula-99">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-99.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-99.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and index of dispersion
                               <span class="inline-formula" id="inline-formula-100">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-100.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-100.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are given by
                               <span class="disp-formula" id="disp-formula-126">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-139.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-139.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-144">
                               The third central moment is
                               <span class="disp-formula" id="disp-formula-127">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-140.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-140.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-145">
                               Inverting the above expressions to give
                               <span class="disp-formula" id="disp-formula-128">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-141.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-141.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-146">
                               Substituting into
                               <a id="xref-disp-formula-127-1" class="xref-disp-formula" href="#disp-formula-127">
                                Eq. S31
                               </a>
                               gives
                               <span class="disp-formula" id="disp-formula-129">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-142.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-142.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-147">
                               The corresponding expression for the third central moment of the diploid load is
                               <span class="disp-formula" id="disp-formula-130">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-143.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-143.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div class="section" id="sec-30">
                              <h4>
                               S5.2 Negative binomial closure: analysis of the weak selection limit
                              </h4>
                              <p id="p-148">
                               When
                               <a id="xref-disp-formula-38-3" class="xref-disp-formula" href="#disp-formula-38">
                                Eqs. 37
                               </a>
                               –
                               <a id="xref-disp-formula-39-4" class="xref-disp-formula" href="#disp-formula-39">
                                38
                               </a>
                               are modified consistent with the negative binomial moment closure (
                               <a id="xref-sec-29-1" class="xref-sec" href="#sec-29">
                                Section S5.1
                               </a>
                               ), we obtain the closed system,
                               <span class="disp-formula" id="disp-formula-131">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-144.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-144.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-149">
                               Setting the left sides to zero and clearing the denominators gives
                               <span class="disp-formula" id="disp-formula-132">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-145.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-145.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-150">
                               Assuming asymptotic expansions of the form
                               <span class="inline-formula" id="inline-formula-101">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-101.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-101.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-102">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-102.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-102.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we obtain
                               <span class="disp-formula" id="disp-formula-133">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-146.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-146.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-151">
                               The zeroth order equations are
                               <span class="disp-formula" id="disp-formula-134">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-147.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-147.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-152">
                               Assuming
                               <span class="inline-formula" id="inline-formula-103">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-103.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-103.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we find
                               <span class="disp-formula" id="disp-formula-135">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-148.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-148.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               consistent with the neutral model (
                               <a id="xref-disp-formula-30-4" class="xref-disp-formula" href="#disp-formula-30">
                                Eqs. 29
                               </a>
                               –
                               <a id="xref-disp-formula-31-3" class="xref-disp-formula" href="#disp-formula-31">
                                30
                               </a>
                               ). The first-order equations are
                               <span class="disp-formula" id="disp-formula-136">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-149.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-149.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-153">
                               In the first equation, the expression in brackets evaluates to zero, so
                               <span class="disp-formula" id="disp-formula-137">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-150.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-150.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-154">
                               After some algebra, we find that the second equation yields,
                               <span class="disp-formula" id="disp-formula-138">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-151.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-151.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-155">
                               The above expressions may be combined to form the following two-term approximation for the mean and variance of TE load,
                               <span class="disp-formula" id="disp-formula-139">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-152.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-152.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-156">
                               Because υ/(ν – η)
                               <sup>
                                2
                               </sup>
                               > 0, the equation for
                               <span class="inline-formula" id="inline-formula-104">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-104.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-104.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               indicates that weak selection decreases the mean TE load, consistent with our intuition. In the equation for
                               <span class="inline-formula" id="inline-formula-105">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-105.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-105.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , the factor (ν + η)/(ν – η)
                               <sup>
                                2
                               </sup>
                               is positive, so we conclude that weak selection also decreases the population variance. As for the index of dispersion, this analysis indicates that under weak selection the Fano factor is well-approximated by
                               <span class="disp-formula" id="disp-formula-140">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-153.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-153.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-157">
                               That is, under weak selection, the Fano factor is expected to decrease, because weak selection causes variance to decrease more than the mean.
                              </p>
                             </div>
                             <div class="section" id="sec-31">
                              <h4>
                               S5.3 Beta-binomial closure
                              </h4>
                              <p id="p-158">
                               For comparison to the negative binomial closure, we have worked through the possibility of choosing ψ to be the function that would be correct if the actual distribution of TE loads were beta-binomial distributed. The probability mass function for a beta-binomial random variable,
                               <strong>
                                X
                               </strong>
                               ~ BB(α, β) for α > 0 and β > 0 on the interval 0 to m is
                               <span class="disp-formula" id="disp-formula-141">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-154.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-154.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-106">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-106.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-106.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the beta function. The first three raw moments of a betabinomial random variable are
                               <span class="disp-formula" id="disp-formula-142">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-155.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-155.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               while the variance is
                               <span class="disp-formula" id="disp-formula-143">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-156.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-156.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-159">
                               Using
                               <a id="xref-disp-formula-103-2" class="xref-disp-formula" href="#disp-formula-103">
                                Eq. S20
                               </a>
                               , it can be shown that the third central moment of a beta-binomial random variable is
                               <span class="disp-formula" id="disp-formula-144">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-157.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-157.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-160">
                               Inverting
                               <a id="xref-disp-formula-146-1" class="xref-disp-formula" href="#disp-formula-146">
                                Eqs. S35
                               </a>
                               –
                               <a id="xref-disp-formula-146-2" class="xref-disp-formula" href="#disp-formula-146">
                                S35
                               </a>
                               gives
                               <span class="disp-formula" id="disp-formula-145">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-158.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-158.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-161">
                               From these values we calculate the following algebraic relationship for the third central moment in terms of the mean and variance of the haploid TE load,
                               <span class="disp-formula" id="disp-formula-146">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-159.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-159.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-162">
                               For the mean and variance of the diploid TE load, the corresponding expression is
                               <span class="disp-formula" id="disp-formula-147">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-160.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-160.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-163">
                               This beta-binomial closure represented by this expression for the third central moment of diploid TE load is arguably preferable to the negative binomial closure discussed in
                               <a id="xref-sec-29-2" class="xref-sec" href="#sec-29">
                                Section S5.1
                               </a>
                               , because a betabinomial random variable has finite support (values between 0 and m as in
                               <a id="xref-disp-formula-141-1" class="xref-disp-formula" href="#disp-formula-141">
                                Eq. S34
                               </a>
                               ). On the other hand, the negative binomial closure results in a simpler expression that often gives approximately the same nullclines and solution trajectories (
                               <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                                Fig. 7
                               </a>
                               ). It is notable that the expression for
                               <span class="inline-formula" id="inline-formula-107">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-107.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-107.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               obtained using the beta-binomial distribution (
                               <a id="xref-disp-formula-147-1" class="xref-disp-formula" href="#disp-formula-147">
                                Eq. S36
                               </a>
                               ) is well-approximated by the negative binomial result (
                               <a id="xref-disp-formula-130-1" class="xref-disp-formula" href="#disp-formula-130">
                                Eq. S33
                               </a>
                               ) when the number of loci are not limiting
                               <span class="inline-formula" id="inline-formula-108">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-108.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-108.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . To see this, one may compare
                               <a id="xref-disp-formula-129-1" class="xref-disp-formula" href="#disp-formula-129">
                                Eqs. S32
                               </a>
                               and
                               <a id="xref-disp-formula-146-3" class="xref-disp-formula" href="#disp-formula-146">
                                S35
                               </a>
                               and show that
                               <span class="inline-formula" id="inline-formula-109">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-109.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-109.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               as m → ∞.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-32">
                             <h3>
                              S6 Numerical methods
                             </h3>
                             <p id="p-164">
                              The master equation model given by
                              <a id="xref-disp-formula-34-6" class="xref-disp-formula" href="#disp-formula-34">
                               Eq. 33
                              </a>
                              is a system of m +1 ordinary differential equations. When m is sufficiently small, it is straightforward to use a relaxation method to calculate the limiting probability distributions for the master equation. Because the number of ODEs in the master equation grows with the number of occupiable loci, it can be more efficient, especially when m is large, to numerically solve for the limiting probability distribution of the associated Fokker-Planck equation. Writing ρ(n, t) dn = Pr[n ≤
                              <strong>
                               n
                              </strong>
                              ≤ n + dn] for the time-dependent probability density function for TE load, ρ(n, t) solves the following Fokker-Planck equation (
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               Gardiner, 2009
                              </a>
                              ;
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               Van Kampen, 2007
                              </a>
                              ),
                              <span class="disp-formula" id="disp-formula-148">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-161.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-161.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-165">
                              In these expressions,
                              <strong>
                               n
                              </strong>
                              is the random variable (TE load of a randomly sampled haploid genome) and n is the independent variable of the probability density ρ(n, t). The drift and diffusion terms of the Fokker-Planck equation are
                              <span class="disp-formula" id="disp-formula-149">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-162.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-162.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-150">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-163.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-163.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where we have used the gain and loss terms given by u(n) = (η
                              <sub>
                               0
                              </sub>
                              +η
                              <sub>
                               n
                              </sub>
                              )(1–n/m) and ν(n) = νn (
                              <a id="xref-disp-formula-21-2" class="xref-disp-formula" href="#disp-formula-21">
                               Eqs. 20
                              </a>
                              –
                              <a id="xref-disp-formula-22-2" class="xref-disp-formula" href="#disp-formula-22">
                               21
                              </a>
                              ). Writing
                              <a id="xref-disp-formula-148-1" class="xref-disp-formula" href="#disp-formula-148">
                               Eq. S37
                              </a>
                              in conservative form as ∂
                              <sub>
                               ρ
                              </sub>
                              /∂t = −∂ϕ/∂n, where ϕ(n) is the probability flux, we find
                              <span class="disp-formula" id="disp-formula-151">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-164.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-164.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-166">
                              For a steady-state solution
                              <span class="inline-formula" id="inline-formula-110">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-110.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-110.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with no flux boundary conditions, setting ϕ(n) = 0 leads to the analytical solution
                              <span class="disp-formula" id="disp-formula-152">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-165.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-165.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where b(n) is given by
                              <a id="xref-disp-formula-150-1" class="xref-disp-formula" href="#disp-formula-150">
                               Eq. S39
                              </a>
                              , and θ is a normalization constant such that
                              <span class="inline-formula" id="inline-formula-111">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-111.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/inline-graphic-111.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-153">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-166.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/04/2020.11.16.382945/embed/graphic-166.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-167">
                              In fact, U(n) may be any antiderivative satisfying U′ = a/b, because the normalization constant θ absorbs the arbitrary constant of integration.
                             </p>
                             <p id="p-168">
                              Several numerical methods were used to simulate the models of TE population dynamics defined by the master equation for the neutral model (
                              <a id="xref-disp-formula-23-9" class="xref-disp-formula" href="#disp-formula-23">
                               Eqs. 22
                              </a>
                              –
                              <a id="xref-disp-formula-25-9" class="xref-disp-formula" href="#disp-formula-25">
                               24
                              </a>
                              ) and the master equation that accounts for selection (
                              <a id="xref-disp-formula-34-7" class="xref-disp-formula" href="#disp-formula-34">
                               Eq. 33
                              </a>
                              ).
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 4
                              </a>
                              used a flux-limiting numerical scheme and the method of lines to integrate the Fokker-Planck equation (
                              <a id="xref-disp-formula-148-2" class="xref-disp-formula" href="#disp-formula-148">
                               Eq. S37
                              </a>
                              ) until a limiting value was reached.
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Fig. 5
                              </a>
                              was obtained using the analytical steady state of the Fokker-Planck equation (
                              <a id="xref-disp-formula-152-1" class="xref-disp-formula" href="#disp-formula-152">
                               Eq. S41
                              </a>
                              ).
                              <a id="xref-fig-7-7" class="xref-fig" href="#F7">
                               Fig. 6
                              </a>
                              used Monte Carlo simulation of drift-diffusion process associated to the Fokker-Planck equation (
                              <a id="xref-disp-formula-148-3" class="xref-disp-formula" href="#disp-formula-148">
                               Eq. S37
                              </a>
                              ). Comparing these results to the flux-limiting numerical scheme revealed that, for some parameter sets that included strong selection, the infinite population model exhibits periodic solutions that are rarely observed in large finite populations.
                              <a id="xref-fig-8-5" class="xref-fig" href="#F8">
                               Fig. 7
                              </a>
                              was calculated using the moment equations with selection and negative binomial moment closure (
                              <a id="xref-disp-formula-45-6" class="xref-disp-formula" href="#disp-formula-45">
                               Eqs. 44
                              </a>
                              –
                              <a id="xref-disp-formula-46-6" class="xref-disp-formula" href="#disp-formula-46">
                               45
                              </a>
                              ). Beta-binomial moment closure leads to very similar results unless m is quite small (on the order of 100).
                              <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                               Fig. 8
                              </a>
                              used moment equations with selection and Beta-binomial closure (
                              <a id="xref-disp-formula-36-6" class="xref-disp-formula" href="#disp-formula-36">
                               Eqs. 35
                              </a>
                              –
                              <a id="xref-disp-formula-37-6" class="xref-disp-formula" href="#disp-formula-37">
                               36
                              </a>
                              with
                              <a id="xref-disp-formula-43-4" class="xref-disp-formula" href="#disp-formula-43">
                               Eq. 42
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-81">
                             The work was supported in part by National Science Foundation Grant DMS 1951646 to GDCS, DEB 1754080 to JRP, DEB 2031275 to GDCS, JRP and Arielle Cooley (Whitman College), and National Insitutes of Health NCCIH Grant R01 AT010816 to GDCS and Christopher Del Negro (William & Mary).
                            </p>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.1" data-doi="10.1093/oxfordjournals.molbev.a004150">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carolina
                                  </span>
                                  <span class="cit-name-surname">
                                   Bartolomé
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xulio
                                  </span>
                                  <span class="cit-name-surname">
                                   Maside
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  On the abundance and distribution of transposable elements in the genome of Drosophila melanogaster
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  926
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  937
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBartolome%26rft.auinit1%253DC.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D926%26rft.epage%253D937%26rft.atitle%253DOn%2Bthe%2BAbundance%2Band%2BDistribution%2Bof%2BTransposable%2BElements%2Bin%2Bthe%2BGenome%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004150%26rft_id%253Dinfo%253Apmid%252F12032249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a004150&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12032249&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000176211700015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.2" data-doi="10.1186/s13059-018-1577-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Guillaume
                                  </span>
                                  <span class="cit-name-surname">
                                   Bourque
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kathleen H
                                  </span>
                                  <span class="cit-name-surname">
                                   Burns
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mary
                                  </span>
                                  <span class="cit-name-surname">
                                   Gehring
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vera
                                  </span>
                                  <span class="cit-name-surname">
                                   Gorbunova
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrei
                                  </span>
                                  <span class="cit-name-surname">
                                   Seluanov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Molly
                                  </span>
                                  <span class="cit-name-surname">
                                   Hammell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michaël
                                  </span>
                                  <span class="cit-name-surname">
                                   Imbeault
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zsuzsanna
                                  </span>
                                  <span class="cit-name-surname">
                                   Izsvák
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Henry L
                                  </span>
                                  <span class="cit-name-surname">
                                   Levin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Todd S
                                  </span>
                                  <span class="cit-name-surname">
                                   Macfarlan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Ten things you should know about transposable elements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  199
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D19%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-018-1577-z%26rft_id%253Dinfo%253Apmid%252F30454069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-018-1577-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30454069&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.3" data-doi="10.1023/A:1018310418744">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John F.Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brookfield
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Badge
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Population genetics models of transposable elements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetica
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  100
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1-3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  281
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  294
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DBrookfield%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D100%26rft.issue%253D1-3%26rft.spage%253D281%26rft.epage%253D294%26rft.atitle%253DPopulation%2Bgenetics%2Bmodels%2Bof%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1018310418744%26rft_id%253Dinfo%253Apmid%252F9440281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1023/A:1018310418744&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9440281&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000071304100029&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.16.382945v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael George
                                  </span>
                                  <span class="cit-name-surname">
                                   Bulmer
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  The Mathematical Theory of Quantitative Genetics
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Clarendon Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1980
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Deborah
                                  </span>
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The population dynamics of transposable elements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1983
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D42%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.16.382945v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Deborah
                                  </span>
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Elements of Evolutionary Genetics
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  W. H. Freeman
                                 </span>
                                 ,
                                 <span class="cit-month">
                                  January
                                 </span>
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.7" data-doi="10.1093/molbev/mst129">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Julie M
                                  </span>
                                  <span class="cit-name-surname">
                                   Cridland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stuart J
                                  </span>
                                  <span class="cit-name-surname">
                                   Macdonald
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anthony D
                                  </span>
                                  <span class="cit-name-surname">
                                   Long
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kevin R
                                  </span>
                                  <span class="cit-name-surname">
                                   Thornton
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Abundance and distribution of transposable elements in two Drosophila QTL mapping resources
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2311
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2327
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst129%26rft_id%253Dinfo%253Apmid%252F23883524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mst129&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23883524&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000326744700007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gregory
                                  </span>
                                  <span class="cit-name-surname">
                                   Deceliere
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The dynamics of transposable elements in structured populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  169
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  467
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  474
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D169%26rft.spage%253D467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Laurent
                                  </span>
                                  <span class="cit-name-surname">
                                   Duret
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gabriel
                                  </span>
                                  <span class="cit-name-surname">
                                   Marais
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian
                                  </span>
                                  <span class="cit-name-surname">
                                   Biémont
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Transposons but not retrotransposons are located preferentially in regions of high recombination rate in Caenorhabditis elegans
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  156
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1661
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1669
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D1661%26rft.epage%253D1669%26rft.atitle%253DTransposons%2Bbut%2BNot%2BRetrotransposons%2BAre%2BLocated%2BPreferentially%2Bin%2BRegions%2Bof%2BHigh%2BRecombination%2BRate%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F11102365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzE2NjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wNC8yMDIwLjExLjE2LjM4Mjk0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-book no-rev-xref" id="cit-2020.11.16.382945v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Crispin
                                  </span>
                                  <span class="cit-name-surname">
                                   Gardiner
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Stochastic Methods: A Handbook for the Natural and Social Sciences
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-edition">
                                  4th
                                 </span>
                                 edition,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.16.382945v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Population Genetics: A Concise Guide
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  The Johns Hopkins University Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.16.382945v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dan
                                  </span>
                                  <span class="cit-name-surname">
                                   Graur
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Molecular and Genome Evolution
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Sinauer Associates
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.13" data-doi="10.1073/pnas.1319032110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Uffe
                                  </span>
                                  <span class="cit-name-surname">
                                   Hellsten
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kevin M
                                  </span>
                                  <span class="cit-name-surname">
                                   Wright
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jerry
                                  </span>
                                  <span class="cit-name-surname">
                                   Jenkins
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shengqiang
                                  </span>
                                  <span class="cit-name-surname">
                                   Shu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yaowu
                                  </span>
                                  <span class="cit-name-surname">
                                   Yuan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Susan R
                                  </span>
                                  <span class="cit-name-surname">
                                   Wessler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeremy
                                  </span>
                                  <span class="cit-name-surname">
                                   Schmutz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John H
                                  </span>
                                  <span class="cit-name-surname">
                                   Willis
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel S
                                  </span>
                                  <span class="cit-name-surname">
                                   Rokhsar
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Fine-scale variation in meiotic recombination in mimulus inferred from population shotgun sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  48
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  19478
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  19482
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1319032110%26rft_id%253Dinfo%253Apmid%252F24225854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ4LzE5NDc4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDQvMjAyMC4xMS4xNi4zODI5NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.14" data-doi="10.1098/rstb.2016.0458">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tyler V
                                  </span>
                                  <span class="cit-name-surname">
                                   Kent
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jasmina
                                  </span>
                                  <span class="cit-name-surname">
                                   Uzunović
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephen I
                                  </span>
                                  <span class="cit-name-surname">
                                   Wright
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Coevolution between transposable elements and recombination
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Philosophical Transactions of the Royal Society B: Biological Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  372
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1736
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  20160458
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0458%26rft_id%253Dinfo%253Apmid%252F29109221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1098/rstb.2016.0458&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29109221&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Arnaud
                                  </span>
                                  <span class="cit-name-surname">
                                   Le Rouzic
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Grégory
                                  </span>
                                  <span class="cit-name-surname">
                                   Deceliere
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Models of the population genetics of transposable elements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  85
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  171
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  181
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D85%26rft.spage%253D171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Trudy FC
                                  </span>
                                  <span class="cit-name-surname">
                                   Mackay
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Transposable elements and fitness in Drosophila melanogaster
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  284
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  295
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1989
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D284%26rft.epage%253D295%26rft.atitle%253DTransposable%2Belements%2Band%2Bfitness%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F2556325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2556325&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.17" data-doi="10.1016/j.tig.2007.02.006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ryan E
                                  </span>
                                  <span class="cit-name-surname">
                                   Mills
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E Andrew
                                  </span>
                                  <span class="cit-name-surname">
                                   Bennett
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rebecca C
                                  </span>
                                  <span class="cit-name-surname">
                                   Iskow
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Scott E
                                  </span>
                                  <span class="cit-name-surname">
                                   Devine
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Which transposable elements are active in the human genome?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  183
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  191
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMills%26rft.auinit1%253DR.%2BE.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D183%26rft.epage%253D191%26rft.atitle%253DWhich%2Btransposable%2Belements%2Bare%2Bactive%2Bin%2Bthe%2Bhuman%2Bgenome%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.02.006%26rft_id%253Dinfo%253Apmid%252F17331616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.02.006&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17331616&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000246002700007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.18" data-doi="10.1093/jhered/esh050">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   EG
                                  </span>
                                  <span class="cit-name-surname">
                                   Pasyukova
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   SV
                                  </span>
                                  <span class="cit-name-surname">
                                   Nuzhdin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   TV
                                  </span>
                                  <span class="cit-name-surname">
                                   Morozova
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   TFC
                                  </span>
                                  <span class="cit-name-surname">
                                   Mackay
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Accumulation of transposable elements in the genome of Drosophila melanogaster is associated with a decrease in fitness
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Heredity
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  95
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  284
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  290
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DPasyukova%26rft.auinit1%253DE.%2BG.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D284%26rft.epage%253D290%26rft.atitle%253DAccumulation%2Bof%2BTransposable%2BElements%2Bin%2Bthe%2BGenome%2Bof%2BDrosophila%2Bmelanogaster%2Bis%2BAssociated%2Bwith%2Ba%2BDecrease%2Bin%2BFitness%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesh050%26rft_id%253Dinfo%253Apmid%252F15247307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/jhered/esh050&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15247307&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000222729700002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nathan M
                                  </span>
                                  <span class="cit-name-surname">
                                   Springer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kai
                                  </span>
                                  <span class="cit-name-surname">
                                   Ying
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yan
                                  </span>
                                  <span class="cit-name-surname">
                                   Fu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tieming
                                  </span>
                                  <span class="cit-name-surname">
                                   Ji
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Cheng-Ting
                                  </span>
                                  <span class="cit-name-surname">
                                   Yeh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yi
                                  </span>
                                  <span class="cit-name-surname">
                                   Jia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wei
                                  </span>
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Todd
                                  </span>
                                  <span class="cit-name-surname">
                                   Richmond
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jacob
                                  </span>
                                  <span class="cit-name-surname">
                                   Kitzman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Heidi
                                  </span>
                                  <span class="cit-name-surname">
                                   Rosenbaum
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Leonardo Iniguez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brad Barbazuk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeffrey A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jeddeloh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dan
                                  </span>
                                  <span class="cit-name-surname">
                                   Nettleton
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Patrick S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schnable
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1000734
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D5%26rft.spage%253D1000734e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.20" data-doi="10.1126/science.aat5760">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ashley
                                  </span>
                                  <span class="cit-name-surname">
                                   Troth
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua R
                                  </span>
                                  <span class="cit-name-surname">
                                   Puzey
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rebecca S
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John H
                                  </span>
                                  <span class="cit-name-surname">
                                   Willis
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John K
                                  </span>
                                  <span class="cit-name-surname">
                                   Kelly
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Selective trade-offs maintain alleles underpinning complex trait variation in plants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  361
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6401
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  475
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  478
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTroth%26rft.auinit1%253DA.%26rft.volume%253D361%26rft.issue%253D6401%26rft.spage%253D475%26rft.epage%253D478%26rft.atitle%253DSelective%2Btrade-offs%2Bmaintain%2Balleles%2Bunderpinning%2Bcomplex%2Btrait%2Bvariation%2Bin%2Bplants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat5760%26rft_id%253Dinfo%253Apmid%252F30072534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjEvNjQwMS80NzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wNC8yMDIwLjExLjE2LjM4Mjk0NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.21" data-doi="10.3732/ajb.1500471">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mario
                                  </span>
                                  <span class="cit-name-surname">
                                   Vallejo-Marín
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Arielle M
                                  </span>
                                  <span class="cit-name-surname">
                                   Cooley
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michelle Yuequi
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Madison
                                  </span>
                                  <span class="cit-name-surname">
                                   Folmer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael R
                                  </span>
                                  <span class="cit-name-surname">
                                   McKain
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua R
                                  </span>
                                  <span class="cit-name-surname">
                                   Puzey
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Strongly asymmetric hybridization barriers shape the origin of a new polyploid species and its hybrid ancestor
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  American Journal of Botany
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  103
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1272
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1288
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1500471%26rft_id%253Dinfo%253Apmid%252F27221281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDMvNy8xMjcyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDQvMjAyMC4xMS4xNi4zODI5NDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-book no-rev-xref" id="cit-2020.11.16.382945v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicolaas Godfried
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Kampen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Stochastic Processes in Physics and Chemistry
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  North Holland
                                 </span>
                                 ,
                                 <span class="cit-edition">
                                  3rd
                                 </span>
                                 edition,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.23" data-doi="10.1038/nrg2165">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Wicker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   François
                                  </span>
                                  <span class="cit-name-surname">
                                   Sabot
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aurélie
                                  </span>
                                  <span class="cit-name-surname">
                                   Hua-Van
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeffrey L
                                  </span>
                                  <span class="cit-name-surname">
                                   Bennetzen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pierre
                                  </span>
                                  <span class="cit-name-surname">
                                   Capy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Boulos
                                  </span>
                                  <span class="cit-name-surname">
                                   Chalhoub
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrew
                                  </span>
                                  <span class="cit-name-surname">
                                   Flavell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philippe
                                  </span>
                                  <span class="cit-name-surname">
                                   Leroy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michele
                                  </span>
                                  <span class="cit-name-surname">
                                   Morgante
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Olivier
                                  </span>
                                  <span class="cit-name-surname">
                                   Panaud
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Etienne
                                  </span>
                                  <span class="cit-name-surname">
                                   Paux
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Phillip
                                  </span>
                                  <span class="cit-name-surname">
                                   SanMiguel
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alan H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schulman
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A unified classification system for eukaryotic transposable elements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  973
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  982
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMathew%26rft.auinit1%253DC.%2BG.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D973%26rft.epage%253D982%26rft.atitle%253DA%2Bunified%2Bclassification%2Bsystem%2Bfor%2Beukaryotic%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165%26rft_id%253Dinfo%253Apmid%252F17984973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg2165&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17984973&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F04%2F2020.11.16.382945.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <div class="section ref-list" id="ref-list-2">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.16.382945v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Deborah
                                  </span>
                                  <span class="cit-name-surname">
                                   Charlesworth
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The population dynamics of transposable elements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1983
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D42%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.16.382945v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Crispin
                                  </span>
                                  <span class="cit-name-surname">
                                   Gardiner
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Stochastic Methods: A Handbook for the Natural and Social Sciences
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-edition">
                                  4th
                                 </span>
                                 edition,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.16.382945v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicolaas Godfried
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Kampen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  S’tochastic Processes in Physics and Chemistry
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  North Holland
                                 </span>
                                 ,
                                 <span class="cit-edition">
                                  3rd
                                 </span>
                                 edition,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.11.16.382945v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.01.19.427347v2" title="LINE-1 Retrotransposon expression in cancerous, epithelial and neuronal cells revealed by 5’ single-cell RNA-Seq" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.07.12.452029v3" title="Sex-specific association between urinary kisspeptin and pubertal development" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted January 04, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.11.16.382945v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.11.16.382945v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.11.16.382945v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Population genetics of transposable element load: a mechanistic account of observed overdispersion
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2335923"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-0jhbYIfQ_VGR87S91eXDtPcIrDhQyScM3-Xucihnn5o"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484486054"/>
                                   <input type="hidden" name="captcha_token" value="e0ac96b024b1240e3f3a4e7e3d0abb39"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2335923--3" data-node-nid="2335923" id="node-2335923--436900811" data-pisa="biorxiv;2020.11.16.382945v3" data-pisa-master="biorxiv;2020.11.16.382945" data-seqnum="2335923" data-apath="/biorxiv/early/2022/01/04/2020.11.16.382945.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Population genetics of transposable element load: a mechanistic account of observed overdispersion
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Ronald D.
                                  </span>
                                  <span class="nlm-surname">
                                   Smith
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Joshua R.
                                  </span>
                                  <span class="nlm-surname">
                                   Puzey
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Gregory D.
                                  </span>
                                  <span class="nlm-surname">
                                   Conradi Smith
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.11.16.382945;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.11.16.382945
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.11.16.382945v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&title=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&title=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&text=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&t=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&title=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&title=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&title=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2335923--5" data-node-nid="2335923" id="citation-node-2335923--6478628584" data-pisa="biorxiv;2020.11.16.382945v3" data-pisa-master="biorxiv;2020.11.16.382945" data-seqnum="2335923" data-apath="/biorxiv/early/2022/01/04/2020.11.16.382945.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Population genetics of transposable element load: a mechanistic account of observed overdispersion
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Ronald D.
                                      </span>
                                      <span class="nlm-surname">
                                       Smith
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Joshua R.
                                      </span>
                                      <span class="nlm-surname">
                                       Puzey
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Gregory D.
                                      </span>
                                      <span class="nlm-surname">
                                       Conradi Smith
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.11.16.382945;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.11.16.382945
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Smith%2C+Ronald+D.&rft.au=Puzey%2C+Joshua+R.&rft.au=Conradi+Smith%2C+Gregory+D.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2335923/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2335923/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2335923/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2335923/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2335923/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2335923/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2335923/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2335923/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2335923/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2335923/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2335923/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2335923/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&count=horizontal&via=&text=Population%20genetics%20of%20transposable%20element%20load%3A%20a%20mechanistic%20account%20of%20observed%20overdispersion&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.11.16.382945v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.11.16.382945v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genomics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
