<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.29.926022v5.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.29.926022v5.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.29.926022v5.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2023/01/03/2020.01.29.926022.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.01.29.926022v5"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference"/>
  <meta name="DC.Identifier" content="10.1101/2020.01.29.926022"/>
  <meta name="DC.Date" content="2023-01-03"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2023, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Investigating tumor heterogeneity using single-cell sequencing technologies is imperative to understand how tumors evolve since each cell subpopulation harbors a unique set of genomic features that yields a unique phenotype, which is bound to have clinical relevance. Clustering of cells based on copy number data obtained from single-cell DNA sequencing provides an opportunity to identify different tumor cell subpopulations. Accordingly, computational methods have emerged for single-cell copy number profiling and clustering; however, these two tasks have, up to now, been handled sequentially and included various ad-hoc preprocessing steps; hence, a procedure vulnerable to introducing clustering artifacts. We propose CopyMix, a mixture-model-based method using Variational Inference, to simultaneously cluster cells and infer a single copy number profile for each cluster, revealing the underlying copy number variation pattern for each cluster. In addition, an extension of CopyMix, SNV-CopyMix, allows the incorporation of single-nucleotide variation (SNV) data into the proposed model to further inform cluster inference. We evaluate CopyMix using simulated data and published biological data from ovarian cancer. The results reveal high clustering performance and low errors in copy number profiling. These favorable results indicate a considerable potential to obtain clinical impact by using CopyMix in studies of cancer tumor heterogeneity.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Negar Safinianaini"/>
  <meta name="DC.Contributor" content="Camila P. E. de Souza"/>
  <meta name="DC.Contributor" content="Andrew Roth"/>
  <meta name="DC.Contributor" content="Hazal Koptagel"/>
  <meta name="DC.Contributor" content="Hosein Toosi"/>
  <meta name="DC.Contributor" content="Jens Lagergren"/>
  <meta name="article:published_time" content="2023-01-03"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Investigating tumor heterogeneity using single-cell sequencing technologies is imperative to understand how tumors evolve since each cell subpopulation harbors a unique set of genomic features that yields a unique phenotype, which is bound to have clinical relevance. Clustering of cells based on copy number data obtained from single-cell DNA sequencing provides an opportunity to identify different tumor cell subpopulations. Accordingly, computational methods have emerged for single-cell copy number profiling and clustering; however, these two tasks have, up to now, been handled sequentially and included various ad-hoc preprocessing steps; hence, a procedure vulnerable to introducing clustering artifacts. We propose CopyMix, a mixture-model-based method using Variational Inference, to simultaneously cluster cells and infer a single copy number profile for each cluster, revealing the underlying copy number variation pattern for each cluster. In addition, an extension of CopyMix, SNV-CopyMix, allows the incorporation of single-nucleotide variation (SNV) data into the proposed model to further inform cluster inference. We evaluate CopyMix using simulated data and published biological data from ovarian cancer. The results reveal high clustering performance and low errors in copy number profiling. These favorable results indicate a considerable potential to obtain clinical impact by using CopyMix in studies of cancer tumor heterogeneity.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2023/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.01.29.926022v5"/>
  <meta name="citation_id" content="2020.01.29.926022v5"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.926022v5"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.926022v5.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.926022v5.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.01.29.926022"/>
  <meta name="citation_num_pages" content="42"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.01.29.926022"/>
  <meta name="citation_author" content="Negar Safinianaini"/>
  <meta name="citation_author_institution" content="School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology"/>
  <meta name="citation_author_email" content="negar7918@gmail.com"/>
  <meta name="citation_author" content="Camila P. E. de Souza"/>
  <meta name="citation_author_institution" content="Department of Statistical and Actuarial Sciences, University of Western Ontario"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4465-5792"/>
  <meta name="citation_author" content="Andrew Roth"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of British Columbia"/>
  <meta name="citation_author_institution" content="British Columbia Cancer Agency"/>
  <meta name="citation_author" content="Hazal Koptagel"/>
  <meta name="citation_author_institution" content="School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology"/>
  <meta name="citation_author_institution" content="Science for Life Laboratory"/>
  <meta name="citation_author" content="Hosein Toosi"/>
  <meta name="citation_author_institution" content="School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology"/>
  <meta name="citation_author_institution" content="Science for Life Laboratory"/>
  <meta name="citation_author" content="Jens Lagergren"/>
  <meta name="citation_author_institution" content="School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology"/>
  <meta name="citation_author_institution" content="Science for Life Laboratory"/>
  <meta name="citation_reference" content="Baslan, T., Kendall, J., Rodgers, L., Cox, H., Riggs, M., Stepansky, A., ., ., Navin, N., and Hicks, J. (2012), “Genome-wide copy number analysis of single cells,” Nature Protocols, 7, 1024–10241."/>
  <meta name="citation_reference" content="Bishop, C. (2006), “Pattern recognition and machine learning,” Information science and statistics, New York, NY: Springer."/>
  <meta name="citation_reference" content="Blei, D., Kucukelbir, A., and Mcauliffe, J. (2017), “Variational Inference: A Review for Statisticians,” Journal of the American Statistical Association, 112, 859–877."/>
  <meta name="citation_reference" content="Blocker, A. W. and Meng, X.-L. (2013), “The potential and perils of preprocessing: Building new foundations,” Bernoulli, 19, 1176–1211."/>
  <meta name="citation_reference" content="Calinski, T. and Harabasz, J. (1974), “A dendrite method for cluster analysis,” Communications in Statistics, 1–27."/>
  <meta name="citation_reference" content="de Souza, C. P., Andronescu, M., Masud, T., Kabeer, F., Biele, J., Laks, E., Lai, D., Brimhall, J., Wang, B., Su, E., et al. (2020), “Epiclomal: probabilistic clustering of sparse single-cell DNA methylation data,” PLOS Computational Biology."/>
  <meta name="citation_reference" content="Eirew, P., Steif, A., Khattra, J., Ha, G., Yap, D., Farahani, H., Gelmon, K., Chia, S., Mar, C., Wan, A., Laks, E., Biele, J., Shumansky, K., Rosner, J., McPherson, A., Nielsen, C., Roth, A., Lefebvre, C., Bashashati, A., de Souza, C., Siu, C., Aniba, R., et al. (2015), “Dynamics of genomic clones in breast cancer patient xenografts at single cell resolution,” Nature, 518, 422."/>
  <meta name="citation_reference" content="Ewing, B., Hillier, L., Wendl, M., and Green, P. (1998), “Base-calling of automated sequencer traces using phred. I. Accuracy assessment,” Genome Research, 8, 175–185."/>
  <meta name="citation_reference" content="Gao, R., Bai, S., Henderson, Y. C., …, and Navin, N. E. (2021), “Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes,” Nat Biotechnology."/>
  <meta name="citation_reference" content="Garvin, T., Aboukhalil, R., and Kendall, J. e. a. (2015), “Interactive analysis and assessment of single-cell copy-number variations,” Nat Methods, 1058–11060."/>
  <meta name="citation_reference" content="Gawad, C., Koh, W., and Quake, S. R. (2016), “Single-cell genome sequencing: current state of the science,” Nature Reviews Genetics, 17, 175."/>
  <meta name="citation_reference" content="Johnson, R. A. and Wichern, D. W. (2007), Applied multivariate statistical analysis, 6th Ed., Pearson Prentice Hall."/>
  <meta name="citation_reference" content="Jordan, M. I., Ghahramani, Z., Jaakkola, T. S., and Saul, L. K. (1999), “An introduction to variational methods for graphical models,” Machine learning, 37, 183–233."/>
  <meta name="citation_reference" content="Kapourani, C.-A. and Sanguinetti, G. (2019), “Melissa: Bayesian clustering and imputation of single-cell methylomes,” Genome biology, 20, 61."/>
  <meta name="citation_reference" content="Koptagel, H., Jun, S., and Lagergren, J. (2018), “SCuPhr: A Probabilistic Framework for Cell Lineage Tree Reconstruction,” https://www.biorxiv.org/content/10.1101/357442v1."/>
  <meta name="citation_reference" content="Laks, E. and McPherson, A. e. a. (2019), “Clonal Decomposition and DNA Replication States Defined by Scaled Single-Cell Genome Sequencing,” Cell."/>
  <meta name="citation_reference" content="Lawson, D., Kessenbrock, K., Davis, R., Pervolarakis, N., and Werb, Z. (2018), “Tumour heterogeneity and metastasis at single-cell resolution,” Nature Cell Biology, 20, 1349–1360."/>
  <meta name="citation_reference" content="Leung, M., Davis, A., Gao, R., Casasent, A., Wang, Y., Sei, E., ., ., and Navin, N. (2017), “Single-cell DNA sequencing reveals a late-dissemination model in metastatic colorectal cancer,” Genome Research, 27, 1287–1299."/>
  <meta name="citation_reference" content="MacKay, D. J. (1997), “Ensemble Learning for Hidden Markov Models,” Tech. rep."/>
  <meta name="citation_reference" content="Mallory, X., Edrisi, M., Navin, N., and Nakhleh, L. (2020), “Methods for copy number aberration detection from single-cell DNA-sequencing data,” Genome Biology."/>
  <meta name="citation_reference" content="McGrory, C. A. and Titterington, D. (2009), “Variational Bayesian analysis for hidden Markov models,” Australian & New Zealand Journal of Statistics, 51, 227–244."/>
  <meta name="citation_reference" content="Navin, N., Kendall, J., Troge, J., Andrews, P., Rodgers, L., McIndoo, J., Cook, K., Stepansky, A., Levy, D., Esposito, D., et al. (2011), “Tumour evolution inferred by single-cell sequencing,” Nature, 472, 90."/>
  <meta name="citation_reference" content="Nowell, P. C. (1976), “The clonal evolution of tumor cell populations,” Science, 194, 23–28."/>
  <meta name="citation_reference" content="Olshen, A. B., Venkatraman, E. S., Lucito, R., and Wigler, M. (2004), “Circular binary segmentation for the analysis of array-based dna copy number data,” Biostatistics."/>
  <meta name="citation_reference" content="Rosenberg, A. and Hirschberg, J. (2007), “V-measure: A conditional entropy-based external cluster evaluation measure,” in Proceedings of the 2007 joint conference on empirical methods in natural language processing and computational natural language learning (EMNLP-CoNLL)."/>
  <meta name="citation_reference" content="Roth, A., McPherson, A., Laks, E., Biele, J., Yap, D., Wan, A., Smith, M. A., Nielsen, C. B., McAlpine, J. N., Aparicio, S., et al. (2016), “Clonal genotype and population structure inference from single-cell tumor sequencing,” Nature methods, 13, 573."/>
  <meta name="citation_reference" content="Rousseeuw, P. (1987), “Silhouettes: A Graphical Aid to the Interpretation and Validation of Cluster Analysis,” Comput. Appl. Math., 20, 53–65."/>
  <meta name="citation_reference" content="Shah, S. P., Xuan, X., DeLeeuw, R. J., Khojasteh, M., Lam, W. L., Ng, R., and Murphy, K. P. (2006), “Integrating copy number polymorphisms into array CGH analysis using a robust HMM,” Bioinformatics, 22, e431–e439."/>
  <meta name="citation_reference" content="Shapiro, E., Biezuner, T., and Linnarsson, S. (2013), “Single-cell sequencing-based technologies will revolutionize whole-organism science,” Nature Reviews Genetics, 14, 618."/>
  <meta name="citation_reference" content="Spiegelhalter, D., Best, N., Carlin, B., and Van Der Linde, A. (2002), “Bayesian measures of model complexity and fit,” Journal of the Royal Statistical Society: Series B (Statistical Methodology), 64, 583–639."/>
  <meta name="citation_reference" content="Vitak, S., Torkenczy, K., Rosenkrantz, J., Fields, A., Christiansen, L., …, and Adey, A. (2017), “Sequencing thousands of single-cell genomes with combinatorial indexing,” Nature methods, 14."/>
  <meta name="citation_reference" content="Witten, D. (2011), “Classification and clustering of sequencing data using a Poisson model,” Ann. Appl. Stat., 5, 2493–2518."/>
  <meta name="citation_reference" content="Zafar, H., Wang, Y., Nakhleh, L., Navin, N., and Chen, K. (2014), “Monovar: single-nucleotide variant detection in single cells,” Nature Methods."/>
  <meta name="citation_reference" content="Zahn, H., Steif, A., Laks, E., Eirew, P., VanInsberghe, M., Shah, S. P., Aparicio, S., and Hansen, C. L. (2017), “Scalable whole-genome single-cell library preparation without preamplification,” Nature methods, 14, 167."/>
  <meta name="citation_reference" content="Zhang, A. and Campbell, K. (2020), “Computational modelling in single-cell cancer genomics: methods and future directions,” Phys. Biol."/>
  <meta name="citation_reference" content="Zuo, C., Chen, K., Hewitt, K., Bresnick, E., and Keleş, S. (2016), “A hierarchical framework for state-space matrix inference and clustering,” Ann. Appl. Stat., 10, 1348–1372."/>
  <meta name="twitter:title" content="CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Investigating tumor heterogeneity using single-cell sequencing technologies is imperative to understand how tumors evolve since each cell subpopulation harbors a unique set of genomic features that yields a unique phenotype, which is bound to have clinical relevance. Clustering of cells based on copy number data obtained from single-cell DNA sequencing provides an opportunity to identify different tumor cell subpopulations. Accordingly, computational methods have emerged for single-cell copy number profiling and clustering; however, these two tasks have, up to now, been handled sequentially and included various ad-hoc preprocessing steps; hence, a procedure vulnerable to introducing clustering artifacts. We propose CopyMix, a mixture-model-based method using Variational Inference, to simultaneously cluster cells and infer a single copy number profile for each cluster, revealing the underlying copy number variation pattern for each cluster. In addition, an extension of CopyMix, SNV-CopyMix, allows the incorporation of single-nucleotide variation (SNV) data into the proposed model to further inform cluster inference. We evaluate CopyMix using simulated data and published biological data from ovarian cancer. The results reveal high clustering performance and low errors in copy number profiling. These favorable results indicate a considerable potential to obtain clinical impact by using CopyMix in studies of cancer tumor heterogeneity.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.29.926022v5"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Investigating tumor heterogeneity using single-cell sequencing technologies is imperative to understand how tumors evolve since each cell subpopulation harbors a unique set of genomic features that yields a unique phenotype, which is bound to have clinical relevance. Clustering of cells based on copy number data obtained from single-cell DNA sequencing provides an opportunity to identify different tumor cell subpopulations. Accordingly, computational methods have emerged for single-cell copy number profiling and clustering; however, these two tasks have, up to now, been handled sequentially and included various ad-hoc preprocessing steps; hence, a procedure vulnerable to introducing clustering artifacts. We propose CopyMix, a mixture-model-based method using Variational Inference, to simultaneously cluster cells and infer a single copy number profile for each cluster, revealing the underlying copy number variation pattern for each cluster. In addition, an extension of CopyMix, SNV-CopyMix, allows the incorporation of single-nucleotide variation (SNV) data into the proposed model to further inform cluster inference. We evaluate CopyMix using simulated data and published biological data from ovarian cancer. The results reveal high clustering performance and low errors in copy number profiling. These favorable results indicate a considerable potential to obtain clinical impact by using CopyMix in studies of cancer tumor heterogeneity.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2023-01-03"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.29.926022v5"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2927000"/>
  <title>
   CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Hw7aYMQqvU8Kz_OVIiY-vvhIJlhPiKDWbTEuU3MuIu0"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2927000","apath":"\/biorxiv\/early\/2023\/01\/03\/2020.01.29.926022.atom","pisa":"biorxiv;2020.01.29.926022v5","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.29.926022v5"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.29.926022v5","title":"CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference","identifier":"node\/2927000"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TIdxx4NDX8fK76UYY0MdtKTW2MoCsOjJcG1AB+CaXHDkvOkvJugasuoCm88W7ACE2pDUhG5DCRJrKFbdzdqbFpXNLkHPxQYwm9dtJ+yT8fD2b4Yw+6qG0yRV5xuRj8AEKmXqDwn+02eLek7aOj\\\\\\\/X9c7e3k8ce9LF02Y9lq7kc8UkgFcMMqFkpGRYqmdHkmIBBqf+fW+GCf5MsBIcraRZU5J5Qb4N0IrcQI2l31CtSZkZjQNQHMuiqYmDzYAzllKzQfhGNcl2FR16lCsti0txPlquBT9pTbRSRiW86VYQkfE8LYNnZGXquWPgvkpDOE2pLVfqFFlEdipfYBP71kxZwa4f0ZpxkkvN2RHon8LtlHkmmg+8zx56tyK+KOcoFl15tj7thF6C0Uq2aWpS1jz6kc5QgLznBhKpKmprED0IrFy5etToJVYCBpI34KRpeIPMWMrVHwnyVxtLqjPTuIUKTo09eri4ykeuEoJhsZYSgdyLINkvuVmO0uBZ+47U\\\\\\\/t\\\\\\\/rUYH\\\\\\\/xuUnbm7+4xNCTpmi3WT5H+L\\\\\\\/jy2URIbV59TDjSxYTghCMNnQq52\\\\\\\/6qAeYQ914puQS9WyY+Mv2nbiaIMitziJS29+bvfbegTloaTHdLyz96gpq4kgS5mXXPJqSL0h8tMVIdzkXEztef82ZLSsKM\\\\\\\/o+yUl3vwHVN7nIMNLAavJtmJKbFmJhqRQ+dih+V2gmidjZLZPUsBa8z45q\\\\\\\/nACyrL3B0LL4AY\\\\\\\/cGVoJ1SuldYcEljpAPCs8WyKIl10O7QYaxFggoy45Cr+zPWGwkUNoKStgpoYa+mOq+JZAfTlfQkYtPmR5bkrQMEPVzrT9r23hgunG\\\\\\\/pLtZ5k04vJIyjzq7GLIqIOX7jHHVicX7Da+0iT9S68hBpoH1rNviG4pF4oL9s807A1uKuCFByqyiaJnlg4+h6h\\\\\\\/UHejc1qGIsgkIDJQV3IgUbGwMIN9+6bpLbU3w4fQxDVrZRSXBc498lpsMF6bu6hcNO4suicknaIUbtpuNsJKDI+XPZZSnVSZ1tIDiOoavJr5aahJ4etVB4BZ0Qehxeuql6MF4CUpkXP\\\\\\\/0oJgn4GJ8apcIP3vx3flJOXjpDE+RKEpzigUE06AUu8txLquBVw9c3KBczJBMnJTiqkwc0yuUNj9ofQG9TPDP0kVyK6bN2oKu\\\\\\\/4PkxcotXuj+Wdw+ZmRoO1QoD3afY3xyip9gC\\\\\\\/eVXQCFPyTgsjezyMRskqjUZSfH0fovbzhe9qA2ORh6\\\\\\\/bnj2HZhbuS+xw2s4h39U0fK8xsbYtSac0246vRBuzRW605tpyI0dJdu2cEBvUj7UezNPvz1hyaq6BL0+r\\\\\\\/6N\\\\\\\/tPSB6N3RAad6\\\\\\\/fLx9Hq89A8SRlmy8khw9VKpDuC0apA606KW8nes5oEn7U=\\\u0022,\\\u0022iv\\\u0022:\\\u00224h66QAOXGfqeUv8XU2dm0w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e71d4734982815af972939ee3870d4d332cf2a4843d33701605fb9bed796aecd\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.29.926022v5.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2927000 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.29.926022v5.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1473197325">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1473197325" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_84815121" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-7DVC3H7ybMH9DsTDIvEL0_4d2-5eK6eW-MwlACW0s54"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2927000" data-node-nid="2927000" id="node-2927000--21929611126" data-pisa="biorxiv;2020.01.29.926022v5" data-pisa-master="biorxiv;2020.01.29.926022" data-apath="/biorxiv/early/2023/01/03/2020.01.29.926022.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Negar
                      </span>
                      <span class="nlm-surname">
                       Safinianaini
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-4465-5792" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Camila P. E.
                      </span>
                      <span class="nlm-surname">
                       de Souza
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Andrew
                      </span>
                      <span class="nlm-surname">
                       Roth
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Hazal
                      </span>
                      <span class="nlm-surname">
                       Koptagel
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Hosein
                      </span>
                      <span class="nlm-surname">
                       Toosi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Jens
                      </span>
                      <span class="nlm-surname">
                       Lagergren
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.01.29.926022
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2927000--21929611126" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Negar Safinianaini
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology
                       </span>
                       , Stockholm,
                       <span class="nlm-country">
                        Sweden
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Negar%2BSafinianaini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Safinianaini%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANegar%2BSafinianaini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:negar7918@gmail.com" class="" data-icon-position="" data-hide-link-title="0">
                        negar7918@gmail.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Camila P. E. de Souza
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Statistical and Actuarial Sciences, University of Western Ontario
                       </span>
                       , London, ON,
                       <span class="nlm-country">
                        Canada
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Camila%2BP.%2BE.%2Bde%2BSouza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=de%20Souza%20CP&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACamila%2BP.%2BE.%2Bde%2BSouza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4465-5792" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Camila P. E. de Souza
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Andrew Roth
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of British Columbia
                       </span>
                       , Vancouver, BC,
                       <span class="nlm-country">
                        Canada
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        British Columbia Cancer Agency
                       </span>
                       , Vancouver, BC,
                       <span class="nlm-country">
                        Canada
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrew%2BRoth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Roth%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAndrew%2BRoth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Hazal Koptagel
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology
                       </span>
                       , Stockholm,
                       <span class="nlm-country">
                        Sweden
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Science for Life Laboratory
                       </span>
                       , Solna,
                       <span class="nlm-country">
                        Sweden
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Hazal%2BKoptagel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Koptagel%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AHazal%2BKoptagel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Hosein Toosi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology
                       </span>
                       , Stockholm,
                       <span class="nlm-country">
                        Sweden
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Science for Life Laboratory
                       </span>
                       , Solna,
                       <span class="nlm-country">
                        Sweden
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Hosein%2BToosi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Toosi%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AHosein%2BToosi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Jens Lagergren
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology
                       </span>
                       , Stockholm,
                       <span class="nlm-country">
                        Sweden
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Science for Life Laboratory
                       </span>
                       , Solna,
                       <span class="nlm-country">
                        Sweden
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jens%2BLagergren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lagergren%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJens%2BLagergren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.01.29.926022v5" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2927000" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2927000/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.29.926022v5.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2927000" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2927000/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.29.926022v5.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2927000" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2927000/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.29.926022v5.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2927000" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2927000/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.29.926022v5.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2927000" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2927000/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.29.926022v5.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2927000" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2927000/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.01.29.926022v5.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2927000" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2927000/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Investigating tumor heterogeneity using single-cell sequencing technologies is imperative to understand how tumors evolve since each cell subpopulation harbors a unique set of genomic features that yields a unique phenotype, which is bound to have clinical relevance. Clustering of cells based on copy number data obtained from single-cell DNA sequencing provides an opportunity to identify different tumor cell subpopulations. Accordingly, computational methods have emerged for single-cell copy number profiling and clustering; however, these two tasks have, up to now, been handled sequentially and included various ad-hoc preprocessing steps; hence, a procedure vulnerable to introducing clustering artifacts. We propose CopyMix, a mixture-model-based method using Variational Inference, to simultaneously cluster cells and infer a single copy number profile for each cluster, revealing the underlying copy number variation pattern for each cluster. In addition, an extension of CopyMix, SNV-CopyMix, allows the incorporation of single-nucleotide variation (SNV) data into the proposed model to further inform cluster inference. We evaluate CopyMix using simulated data and published biological data from ovarian cancer. The results reveal high clustering performance and low errors in copy number profiling. These favorable results indicate a considerable potential to obtain clinical impact by using CopyMix in studies of cancer tumor heterogeneity.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-6">
                             A tumor typically consists of a collection of heterogeneous cell populations, each having distinct genetic and phenotypic properties, in particular, concerning the capacity to promote cancer progression, metastasis, and therapy resistance
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              Eirew et al. (2015)
                             </a>
                             ;
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Nowell (1976)
                             </a>
                             . Single-cell sequencing technologies
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Gawad et al. (2016)
                             </a>
                             ;
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              Navin et al. (2011)
                             </a>
                             ;
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              Shapiro et al. (2013)
                             </a>
                             ;
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              Zahn et al. (2017)
                             </a>
                             provide an opportunity to investigate the genomic profile of individual cells regarding both single nucleotide variation (SNV) and copy number variation (CNV). CNVs and SNVs are essential contributors to phenotypic variation relating to health, and disease
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              Baslan et al. (2012)
                             </a>
                             ;
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Lawson et al. (2018)
                             </a>
                             . Moreover, although singlecell SNV profiling is hampered by experimental imperfections such as drop-outs, copy number profiling, i.e., detecting single-cell CNVs, is feasible at least at coarser resolutions. Clustering of cells based on their copy number profiles provides opportunities to better understand tumor subpopulations and tumor heterogeneity, issues that are bound to have clinical relevance.
                            </p>
                            <p id="p-7">
                             Current single-cell datasets pose a wealth of computational challenges. As answers to some of those, methods have emerged for single-cell copy number profiling as well as clustering of cells based on their already inferred copy number profiles, e.g.,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              Garvin et al. (2015)
                             </a>
                             ;
                             <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                              Zahn et al. (2017)
                             </a>
                             ;
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              Leung et al. (2017)
                             </a>
                             ;
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              Vitak et al. (2017)
                             </a>
                             ; however, these two tasks have up to now been handled sequentially with various ad-hoc preprocessing steps. The sequential approach is vulnerable to clustering artifacts since preprocessing decisions, typically irreversible, constrain all later analyses. Even if each task has optimal performance, the final result may still fall short of the best possible performance
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              Blocker and Meng (2013)
                             </a>
                             . Regarding copy number profiling, HMMcopy
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              Shah et al. (2006
                             </a>
                             )—a widely used hidden-Markov-model-based method for copy number profiling—has limitations such as requiring manual calibration of more than ten parameters
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              Mallory et al. (2020)
                             </a>
                             .
                            </p>
                            <p id="p-8">
                             As a single-phase mathematically stable solution, also motivated by future directions of computational modeling in single-cell cancer genomics
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              Zhang and Campbell (2020)
                             </a>
                             , we propose a novel framework, referred to as CopyMix. CopyMix alleviates two problems: (1) the sequential treatment of copy number profiling and clustering, and (2) the labor of HMMcopy parameter tuning. CopyMix performs the copy number profiling and clustering simultaneously, and as to the best of our knowledge, no earlier work has simultaneously performed these two tasks. Simultaneous and model-based inference have been considered for single-cell DNA methylation
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              Kapourani and Sanguinetti (2019)
                             </a>
                             ;
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              de Souza et al. (2020)
                             </a>
                             , single-cell SNV data
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              Roth et al. (2016)
                             </a>
                             , and bulk chIP-seq data from several replicates
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              Zuo et al. (2016)
                             </a>
                             ; CopyMix similarly performs simultaneous inference using a model-based approach. Due to the model-based treatment, CopyMix enjoys the advantages of a fully probabilistic framework such as transparency, uncertainty measurements, and modeling flexibility. CopyMix uses a mixture model with components, corresponding to clusters, each having a specific copy number profile modeled by a sequence of latent variables which follows a Markov structure similar to the state-of-the-art methods using Hidden Markov Models (HMMs)
                             <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">
                              Shah et al. (2006)
                             </a>
                             ;
                             <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                              Vitak et al. (2017)
                             </a>
                             ;
                             <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">
                              Zahn et al. (2017)
                             </a>
                             , revealing the copy number variation pattern behind each cluster. We deploy a Bayesian treatment and infer all quantities of interest using Variational Inference (VI)
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Jordan et al. (1999)
                             </a>
                             , which typically yields faster inference methods than methodologies like Markov chain Monte Carlo sampling
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              Blei et al. (2017)
                             </a>
                             . Compared to Expectation-Maximization (EM), VI has multiple advantages, for example, it estimates the posterior distributions rather than point estimates, protects against over-fitting, and allows for principled model selection, i.e., identifying the optimal number of mixture components
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              Bishop (2006)
                             </a>
                             .
                            </p>
                            <p id="p-9">
                             CopyMix provides a generic framework for single-cell copy number data; however, in this paper, we focus on its application in the newly developed high-resolution direct library preparation (DLP) technology
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              Laks and McPherson (2019)
                             </a>
                             . We perform experiments on biologically inspired simulated data. Moreover, we conduct experiments on the published DLP data from ovarian cancer
                             <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                              Laks and McPherson (2019)
                             </a>
                             . We evaluate CopyMix on the ovarian cancer data and compare our results to those obtained by the original paper
                             <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                              Laks and McPherson (2019)
                             </a>
                             , where HMMcopy is used as a copy number profiling tool proceeding by an ad-hoc combination of a series of clustering methods: K-means, Hierarchical Density-Based Spatial Clustering, and UMAP. We also compare CopyMix results with Ginkgo—a state-of-art framework for clustering of single-cells based on previously estimated copy number profiles obtained by circular binary segmentation (CBS) method
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              Olshen et al. (2004)
                             </a>
                             . Motivated by the non-sequential mentality, as aforementioned, we, instead of performing SNV verification as a post-process, as in
                             <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">
                              Laks and McPherson (2019)
                             </a>
                             , extend CopyMix to exploit the SNV signals by incorporating SNVs in the proposed model. We refer to this extension of CopyMix as SNV-CopyMix.
                            </p>
                            <p id="p-10">
                             A sequential version of CopyMix, resembling
                             <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">
                              Shah et al. (2006)
                             </a>
                             ;
                             <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">
                              Vitak et al. (2017)
                             </a>
                             but excluding their ad-hoc processes, is compared to CopyMix, confirming the hypothesis that single-phase inference framework is superior over a sequential version of the framework.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 CopyMix
                            </h2>
                            <p id="p-11">
                             CopyMix is a probabilistic clustering method based on a mixture of Markov chains. CopyMic is a mixture model with components corresponding to clusters, each having a specific copy number profile modeled by a sequence of latent variables. Similarly, as in
                             <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">
                              Shah et al. (2006)
                             </a>
                             ;
                             <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">
                              Vitak et al. (2017)
                             </a>
                             ;
                             <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">
                              Zahn et al. (2017)
                             </a>
                             , we assume that each latent copy number sequence is governed by a Markov chain (describing a sequence of possible copy numbers in which the probability of each copy number value, defined as a state, depends only on the previous state in the sequence). We provide two observational model for CopyMix. First one is based on Poisson emission and the second one is based on Gaussian emission. In the following sections we describe these two choices of emission modelling.
                            </p>
                            <p id="p-12">
                             After presenting our method CopyMix. Next, we extend CopyMix to incorporate single nucleotide data; we refer to this method as SNV-CopyMix. The purpose of designing SNV-CopyMix is to exploit CNV-independent signals, i.e., SNVs, when performing single-cell clustering.
                            </p>
                            <div id="sec-3" class="subsection">
                             <h3>
                              2.1 Poisson emission
                             </h3>
                             <p id="p-13">
                              We assume that the data (read counts) follow a Poisson distribution as in
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               Witten (2011)
                              </a>
                              ; we consider read counts per fixed equal size genomic bin as in
                              <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                               Leung et al. (2017)
                              </a>
                              . Moreover, we assume the read counts are emitted from a latent sequence of copy number states.
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              illustrates an overview of CopyMix; it analyzes the input cell sequences, containing read counts per predefined genomic bin, and produces clusters of cells with their corresponding copy number sequences (profiles) explaining the clusters.
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              illustrates CopyMix graphical model where Y denotes the observable variables, read counts per predefined bins, C the latent copy number states forming a Markov chain, and Z the latent cell-specific cluster assignment variables. We notice in
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              that each Y has two levels of dependencies, which are reflections of the assumption that the Poisson distribution over the read counts depends on a latent cell-specific cluster assignment, Z, and a corresponding latent copy number state, C. Intuitively, a higher copy number should correspond to a higher read count. We incorporate this assumption in our model by defining the rate of Poisson distribution as the product of copy number state and cell-specific rate, θ
                              <sub>
                               n
                              </sub>
                              , see
                              <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                               Eq. (1)
                              </a>
                              ; this also implies that θ
                              <sub>
                               n
                              </sub>
                              corresponds to the average sequencing coverage for a haploid genome. Due to this multiplicative structure for the Poisson rate, a copy number state of zero results in the lowest rate implying copy number deletion events; however, since the rate of Poisson cannot be zero, our implementation handles this by assigning a tiny number instead of zero. Our model is described in more detail in what follows, using conjugacy regarding the priors.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F1.large.jpg?width=800&height=600&carousel=1" title="The overview of CopyMix with: input y, cells’ reads; outputs q(Z) and q(C), posterior distributions resulting in clusters within y with their corresponding copy number profiles illustrated by heatmaps (this is an example of binary clustering)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The overview of CopyMix with: input <em>y</em>, cells’ reads; outputs <em>q</em>(<em>Z</em>) and <em>q</em>(<em>C</em>), posterior distributions resulting in clusters within <em>y</em> with their corresponding copy number profiles illustrated by heatmaps (this is an example of binary clustering).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F1.medium.gif" width="440" height="292"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F1.medium.gif" width="440" height="292"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <p id="p-14" class="first-child">
                                The overview of CopyMix with: input y, cells’ reads; outputs q(Z) and q(C), posterior distributions resulting in clusters within y with their corresponding copy number profiles illustrated by heatmaps (this is an example of binary clustering).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F2.large.jpg?width=800&height=600&carousel=1" title="Probabilistic graphical model representing our proposed model. Shaded nodes represent observed values, the unshaded ones are the latent variables and the squares are the hyperparameters of the model; a posterior distribution over the values of the unshaded nodes is approximated using Variational Inference. Ynm, observed read counts from cell n and bin m; Ckm, corresponding latent copy number state forming a Markov chain; θn, cell-specific rate; Zn, latent cell-specific cluster assignment variable. π and Ak, the priors over Zn and Ckm respectively; ρk, prior over the starting copy number state." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Probabilistic graphical model representing our proposed model. Shaded nodes represent observed values, the unshaded ones are the latent variables and the squares are the hyperparameters of the model; a posterior distribution over the values of the unshaded nodes is approximated using Variational Inference. <em>Y<sub>nm</sub></em>, observed read counts from cell n and bin m; <em>C<sub>km</sub></em>, corresponding latent copy number state forming a Markov chain; <em>θ<sub>n</sub></em>, cell-specific rate; <em>Z<sub>n</sub></em>, latent cell-specific cluster assignment variable. π and <em>A<sub>k</sub></em>, the priors over <em>Z<sub>n</sub></em> and <em>C<sub>km</sub></em> respectively; <em>ρ<sub>k</sub></em>, prior over the starting copy number state.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F2.medium.gif" width="440" height="411"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F2.medium.gif" width="440" height="411"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <p id="p-15" class="first-child">
                                Probabilistic graphical model representing our proposed model. Shaded nodes represent observed values, the unshaded ones are the latent variables and the squares are the hyperparameters of the model; a posterior distribution over the values of the unshaded nodes is approximated using Variational Inference. Y
                                <sub>
                                 nm
                                </sub>
                                , observed read counts from cell n and bin m; C
                                <sub>
                                 km
                                </sub>
                                , corresponding latent copy number state forming a Markov chain; θ
                                <sub>
                                 n
                                </sub>
                                , cell-specific rate; Z
                                <sub>
                                 n
                                </sub>
                                , latent cell-specific cluster assignment variable. π and A
                                <sub>
                                 k
                                </sub>
                                , the priors over Z
                                <sub>
                                 n
                                </sub>
                                and C
                                <sub>
                                 km
                                </sub>
                                respectively; ρ
                                <sub>
                                 k
                                </sub>
                                , prior over the starting copy number state.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-16">
                              The genome considered is partitioned into M equally sized segments of consecutive positions called bins. Let
                              <strong>
                               Y
                              </strong>
                              <sub>
                               n
                              </sub>
                              = (Y
                              <sub>
                               n1
                              </sub>
                              , …, Y
                              <sub>
                               nM
                              </sub>
                              ), where Y
                              <sub>
                               nm
                              </sub>
                              is the random but observable number of reads aligned to bin m for cell n taking values in {0, 1, 2, …} for m ∈ [M] = {1, …, M} and n ∈ [N] = {1, …, N}. We assume that
                              <strong>
                               Y
                              </strong>
                              <sub>
                               1
                              </sub>
                              , …,
                              <strong>
                               Y
                              </strong>
                              <sub>
                               N
                              </sub>
                              are independent and given a vector of true copy number states, Y
                              <sub>
                               n1
                              </sub>
                              , …, Y
                              <sub>
                               nM
                              </sub>
                              are also independent with the distribution of Y
                              <sub>
                               nm
                              </sub>
                              depending on the true copy number state at bin m. The cluster membership of cell n is indicated by the hidden variable Z
                              <sub>
                               n
                              </sub>
                              that takes values in [K] = {1, …, K}. We assume there are K << N vectors of true hidden copy number states, i.e., one for each cluster. The variables Z
                              <sub>
                               1
                              </sub>
                              , …, Z
                              <sub>
                               N
                              </sub>
                              are independent following a categorical distribution with P(Z
                              <sub>
                               n
                              </sub>
                              = k) = π
                              <sub>
                               k
                              </sub>
                              and
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . If Z
                              <sub>
                               n
                              </sub>
                              = k then the distribution of
                              <strong>
                               Y
                              </strong>
                              <sub>
                               n
                              </sub>
                              depends on the k-th vector of true hidden copy number states, defined as
                              <strong>
                               C
                              </strong>
                              <sub>
                               k
                              </sub>
                              = (C
                              <sub>
                               k1
                              </sub>
                              , …, C
                              <sub>
                               kM
                              </sub>
                              ), with each C
                              <sub>
                               km
                              </sub>
                              taking values in [J] = {1, …, J}. We assume that C
                              <sub>
                               k1
                              </sub>
                              , …, C
                              <sub>
                               kM
                              </sub>
                              follow a discrete-time homogeneous Markov chain with initial probabilities ρ
                              <sub>
                               kj
                              </sub>
                              = P(C
                              <sub>
                               k1
                              </sub>
                              = j), j ∈ [J] and transition probabilities
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Consequently, given the cluster assignment and the corresponding true hidden vector of copy number states, Y
                              <sub>
                               n1
                              </sub>
                              , …, Y
                              <sub>
                               nM
                              </sub>
                              are independent with Y
                              <sub>
                               nm
                              </sub>
                              following a distribution with parameters depending on the hidden true state at bin m for cluster k, that is,
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . As mentioned earlier, we assume
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to be a Poisson distribution with rate j × θ
                              <sub>
                               n
                              </sub>
                              , that is,
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-17">
                              We, w.l.o.g., consider the initial probabilities, ρ
                              <sub>
                               kj
                              </sub>
                              ’s, to be fixed and known. We let
                              <strong>
                               Ψ
                              </strong>
                              be the set containing all the unknown model parameters, i.e.,
                              <strong>
                               Ψ
                              </strong>
                              = {
                              <strong>
                               A
                              </strong>
                              ,
                              <strong>
                               θ
                              </strong>
                              ,
                              <strong>
                               π
                              </strong>
                              }, where
                              <strong>
                               A
                              </strong>
                              = {
                              <strong>
                               A
                              </strong>
                              <sub>
                               k
                              </sub>
                              : k ∈ [K]} with
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ; and
                              <strong>
                               π
                              </strong>
                              = (π
                              <sub>
                               1
                              </sub>
                              , …, π
                              <sub>
                               K
                              </sub>
                              ). In order to infer
                              <strong>
                               Ψ
                              </strong>
                              , the hidden states
                              <strong>
                               C
                              </strong>
                              = {
                              <strong>
                               C
                              </strong>
                              <sub>
                               1
                              </sub>
                              , …,
                              <strong>
                               C
                              </strong>
                              <sub>
                               K
                              </sub>
                              }, and
                              <strong>
                               Z
                              </strong>
                              = (Z
                              <sub>
                               1
                              </sub>
                              , …, Z
                              <sub>
                               n
                              </sub>
                              ) we apply VI; that is, we derive an algorithm that, for given data, approximates the posterior distribution of the parameters by finding the Variational Distribution (VD), q(
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               Ψ
                              </strong>
                              ), with smallest Kullback-Leibler divergence to the posterior distribution P(
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               Ψ
                              </strong>
                              |
                              <strong>
                               Y
                              </strong>
                              ), which is equivalent to maximizing the evidence lower bound (ELBO)
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               Blei et al. (2017)
                              </a>
                              given by
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p>
                              We consider the following prior distributions for the parameters in
                              <strong>
                               Ψ
                              </strong>
                              .
                             </p>
                             <ul class="list-unord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-19">
                                <span class="inline-formula" id="inline-formula-6">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-6.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-6.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-20">
                                <span class="inline-formula" id="inline-formula-7">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-7.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-7.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , with
                                <span class="inline-formula" id="inline-formula-8">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-8.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-8.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , shape, and
                                <span class="inline-formula" id="inline-formula-9">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-9.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-9.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , rate, hyperparameters.
                               </p>
                              </li>
                              <li id="list-item-3">
                               <p id="p-21">
                                <span class="inline-formula" id="inline-formula-10">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-10.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-10.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                             </ul>
                             <p id="p-22">
                              The main steps of our VI approach, for inferring
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              , and
                              <strong>
                               Ψ
                              </strong>
                              , are described below.
                             </p>
                             <div id="sec-4" class="subsection">
                              <h4>
                               Step 1. VD factorization
                              </h4>
                              <p id="p-23">
                               We assume the following factorization of the VD:
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-5" class="subsection">
                              <h4>
                               Step 2. Joint distribution
                              </h4>
                              <p id="p-24">
                               The logarithm of the joint distribution satisfies: log P(
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) = log P(
                               <strong>
                                Y
                               </strong>
                               |
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) + log P(
                               <strong>
                                C
                               </strong>
                               |
                               <strong>
                                Ψ
                               </strong>
                               ) + log P(
                               <strong>
                                Z
                               </strong>
                               |
                               <strong>
                                Ψ
                               </strong>
                               ) + log P(
                               <strong>
                                Ψ
                               </strong>
                               ). For details of the calculations see
                               <a id="xref-sec-29-1" class="xref-app" href="#sec-29">
                                Appendix A.1
                               </a>
                               .
                              </p>
                             </div>
                             <div id="sec-6" class="subsection">
                              <h4>
                               Step 3. VD computation
                              </h4>
                              <p id="p-25">
                               We now derive a coordinate ascent algorithm for the VD. That is, we derive an update equation for each term in the factorization,
                               <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                                Eq. (3)
                               </a>
                               , by calculating the expectation of log P(
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) over the VD of all random variables except the one currently being updated
                               <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                                Bishop (2006)
                               </a>
                               . For example, we obtain the update equation for
                               <strong>
                                π
                               </strong>
                               , q(
                               <strong>
                                π
                               </strong>
                               ), by calculating E
                               <sub>
                                −
                                <strong>
                                 π
                                </strong>
                               </sub>
                               (log P(
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               )), where −
                               <strong>
                                π
                               </strong>
                               indicates that the expectation is taken with respect to the VD of all other random variables than
                               <strong>
                                π
                               </strong>
                               , i.e.,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               and {
                               <strong>
                                A
                               </strong>
                               ,
                               <strong>
                                θ
                               </strong>
                               } except
                               <strong>
                                π
                               </strong>
                               . See
                               <a id="xref-sec-30-1" class="xref-app" href="#sec-30">
                                A.2
                               </a>
                               and
                               <a id="xref-sec-31-1" class="xref-app" href="#sec-31">
                                A.3
                               </a>
                               for the update equation of each term in
                               <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">
                                Eq. (3)
                               </a>
                               .
                              </p>
                             </div>
                             <div id="sec-7" class="subsection">
                              <h4>
                               Step 4. Summary of updates
                              </h4>
                              <p id="p-26">
                               We update the parameters of each posterior distribution presented in Step 3; for the summary of the update equations, see
                               <a id="xref-statement-2-1" class="xref-statement" href="#statement-2">
                                Alg. 2
                               </a>
                               , in
                               <a id="xref-sec-30-2" class="xref-app" href="#sec-30">
                                Appendix A.2
                               </a>
                               . A shorter version of
                               <a id="xref-statement-2-2" class="xref-statement" href="#statement-2">
                                Alg. 2
                               </a>
                               , providing an overview of CopyMix, is shown in
                               <a id="xref-statement-1-1" class="xref-statement" href="#statement-1">
                                Alg. 1
                               </a>
                               . As shown in lines 8 and 9 of
                               <a id="xref-statement-1-2" class="xref-statement" href="#statement-1">
                                Alg. 1
                               </a>
                               , we aim to find the posterior distributions of cluster assignments and copy number states. Note that the update equations in line 5 of
                               <a id="xref-statement-1-3" class="xref-statement" href="#statement-1">
                                Alg. 1
                               </a>
                               can be run in a parallel fashion, keeping the algorithm’s computational complexity low. The implementation of CopyMix is available at
                               <strong>
                                <a href="https://github.com/negar7918/CopyMix">
                                 CopyMix
                                </a>
                               </strong>
                               .
                              </p>
                              <div id="statement-1" class="statement">
                               <span class="statement-label">
                                Algorithm 1
                               </span>
                               <h3>
                                CopyMix: a Variational Inference Algorithm
                               </h3>
                               <div id="F3" class="fig pos-float type-figure odd">
                                <div class="highwire-figure">
                                 <div class="fig-inline-img-wrapper">
                                  <div class="fig-inline-img">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F3.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                    <span class="hw-responsive-img">
                                     <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F3.medium.gif" width="440" height="112"/>
                                     <noscript>
                                      <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F3.medium.gif" width="440" height="112"/>
                                     </noscript>
                                    </span>
                                   </a>
                                  </div>
                                 </div>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure3" data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <p id="p-27">
                               </p>
                              </div>
                             </div>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              2.2 Gaussian emission
                             </h3>
                             <p id="p-28">
                              In this version, we assume that the read counts are GC-corrected (a bias correction performed in advance), which means that the data values are real numbers. Thus, it is meaningful to assume that the data follow a Gaussian distribution; again, we assume fixed equal size genomic bin as in the Poisson version. The rest of the model is similar to the previous section. The new probabilistic graphical model is illustrated in
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 3
                              </a>
                              , where we model the average sequencing coverage as Gaussian with independent conjugate priors; this means that the mean of the Gaussian follows itself a Gaussian distribution and the precision of the Gaussian follows a Gamma distribution.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F4.large.jpg?width=800&height=600&carousel=1" title="In this version Ynm are observed corrected read ratios from cell n and bin m; Ckm, corresponding latent copy number state forming a Markov chain; μn, cell-specific rate; Zn, latent cell-specific cluster assignment variable. π and Ak, the priors over Zn and Ckm respectively; ρk, prior over the starting copy number state." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">In this version <em>Y<sub>nm</sub></em> are observed corrected read ratios from cell n and bin m; <em>C<sub>km</sub></em>, corresponding latent copy number state forming a Markov chain; <em>μ<sub>n</sub></em>, cell-specific rate; <em>Z<sub>n</sub></em>, latent cell-specific cluster assignment variable. π and <em>A<sub>k</sub></em>, the priors over <em>Z<sub>n</sub></em> and <em>C<sub>km</sub></em> respectively; <em>ρ<sub>k</sub></em>, prior over the starting copy number state.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F4.medium.gif" width="440" height="403"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F4.medium.gif" width="440" height="403"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <p id="p-29" class="first-child">
                                In this version Y
                                <sub>
                                 nm
                                </sub>
                                are observed corrected read ratios from cell n and bin m; C
                                <sub>
                                 km
                                </sub>
                                , corresponding latent copy number state forming a Markov chain; μ
                                <sub>
                                 n
                                </sub>
                                , cell-specific rate; Z
                                <sub>
                                 n
                                </sub>
                                , latent cell-specific cluster assignment variable. π and A
                                <sub>
                                 k
                                </sub>
                                , the priors over Z
                                <sub>
                                 n
                                </sub>
                                and C
                                <sub>
                                 km
                                </sub>
                                respectively; ρ
                                <sub>
                                 k
                                </sub>
                                , prior over the starting copy number state.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              Let Y
                              <sub>
                               nm
                              </sub>
                              be the observed gc-corrected reads ratio to bin m for cell n for n = 1, …, N and m = 1, …, M. Let
                              <strong>
                               Y
                              </strong>
                              <sub>
                               n
                              </sub>
                              = (Y
                              <sub>
                               n1
                              </sub>
                              , …, Y
                              <sub>
                               nM
                              </sub>
                              ) be vector with all observed data for cell n. Given the cluster assignment and the corresponding true hidden vector of copy number states, Y
                              <sub>
                               n1
                              </sub>
                              , …, Y
                              <sub>
                               nM
                              </sub>
                              are independent with Y
                              <sub>
                               nm
                              </sub>
                              following a distribution with parameters depending on the hidden true state at bin m for cluster population k, that is,
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-31">
                              We assume
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to be a Gaussian distribution with jμ
                              <sub>
                               n
                              </sub>
                              = j × μ
                              <sub>
                               n
                              </sub>
                              (we add noise .001 to j=0 for computational purposes), i.e.,
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p>
                              Let
                              <strong>
                               Ψ
                              </strong>
                              be the set containing all the unknown model parameters, i.e.,
                              <strong>
                               Ψ
                              </strong>
                              = {
                              <strong>
                               A
                              </strong>
                              ,
                              <strong>
                               μ
                              </strong>
                              , σ
                              <sup>
                               2
                              </sup>
                              ,
                              <strong>
                               π
                              </strong>
                              }, where
                             </p>
                             <ul class="list-unord" id="list-2">
                              <li id="list-item-4">
                               <p id="p-33">
                                <strong>
                                 A
                                </strong>
                                = {
                                <strong>
                                 A
                                </strong>
                                <sub>
                                 k
                                </sub>
                                for k = 1, …, K} with
                                <span class="inline-formula" id="inline-formula-12">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-12.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-12.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for i = 1, …, J and j = 1, …, J};
                               </p>
                              </li>
                              <li id="list-item-5">
                               <p id="p-34">
                                <strong>
                                 μ
                                </strong>
                                = {μ
                                <sub>
                                 n
                                </sub>
                                for n = 1, …, N};
                               </p>
                              </li>
                              <li id="list-item-6">
                               <p id="p-35">
                                σ
                                <sup>
                                 2
                                </sup>
                                ;
                               </p>
                              </li>
                              <li id="list-item-7">
                               <p id="p-36">
                                <strong>
                                 π
                                </strong>
                                = (π
                                <sub>
                                 1
                                </sub>
                                , … π
                                <sub>
                                 K
                                </sub>
                                ).
                               </p>
                              </li>
                             </ul>
                             <p>
                              We consider the following prior distributions for the parameters in
                              <strong>
                               Ψ
                              </strong>
                              .
                             </p>
                             <ul class="list-unord" id="list-3">
                              <li id="list-item-8">
                               <p id="p-38">
                                <span class="inline-formula" id="inline-formula-13">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-13.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-13.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                              <li id="list-item-9">
                               <p id="p-39">
                                <span class="inline-formula" id="inline-formula-14">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-14.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-14.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The conjugate prior concerning the mean of Normal distribution is Normal distribution.
                               </p>
                              </li>
                              <li id="list-item-10">
                               <p id="p-40">
                                <span class="inline-formula" id="inline-formula-15">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-15.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-15.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The conjugate prior concerning the precision of Normal distribution is Gamma distribution.
                               </p>
                              </li>
                              <li id="list-item-11">
                               <p id="p-41">
                                <strong>
                                 π
                                </strong>
                                ~ Dirichlet(
                                <strong>
                                 δ
                                </strong>
                                )
                               </p>
                              </li>
                             </ul>
                             <p id="p-42">
                              The main steps of our VI approach, for inferring
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              , and
                              <strong>
                               Ψ
                              </strong>
                              , are described below.
                             </p>
                             <div id="sec-9" class="subsection">
                              <h4>
                               Step 1. VD factorization
                              </h4>
                              <p id="p-43">
                               We assume the following factorization of the VD:
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-10" class="subsection">
                              <h4>
                               Step 2. Joint distribution
                              </h4>
                              <p id="p-44">
                               The logarithm of the joint distribution satisfies: log P(
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) = log P(
                               <strong>
                                Y
                               </strong>
                               |
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) + log P(
                               <strong>
                                C
                               </strong>
                               |
                               <strong>
                                Ψ
                               </strong>
                               ) + log P(
                               <strong>
                                Z
                               </strong>
                               |
                               <strong>
                                Ψ
                               </strong>
                               ) + log P(
                               <strong>
                                Ψ
                               </strong>
                               ). For details of the calculations see
                               <a id="xref-sec-32-1" class="xref-app" href="#sec-32">
                                Appendix A.4
                               </a>
                               .
                              </p>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               Step 3. VD computation
                              </h4>
                              <p id="p-45">
                               We now derive a coordinate ascent algorithm for the VD; as we described this for the Poisson version, we derive an update equation for each term in the factorization,
                               <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                                Eq. (5)
                               </a>
                               , by calculating the expectation of log P(
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) over the VD of all random variables except the one currently being updated
                               <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                                Bishop (2006)
                               </a>
                               . See
                               <a id="xref-sec-33-1" class="xref-app" href="#sec-33">
                                A.5
                               </a>
                               and
                               <a id="xref-sec-41-1" class="xref-app" href="#sec-41">
                                A.6
                               </a>
                               for the update equation of each term in
                               <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">
                                Eq. (5)
                               </a>
                               .
                              </p>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               Step 4. Summary of updates
                              </h4>
                              <p id="p-46">
                               We update the parameters of each posterior distribution presented in Step 3; for the details of the update equations, see
                               <a id="xref-sec-33-2" class="xref-app" href="#sec-33">
                                Appendix A.5
                               </a>
                               . We summarize the updates as shown in
                               <a id="xref-statement-1-4" class="xref-statement" href="#statement-1">
                                Alg. 1
                               </a>
                               .
                              </p>
                             </div>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              2.3 SNV-CopyMix
                             </h3>
                             <p id="p-47">
                              In this section, we incorporate SNV data into the model (see
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 4
                              </a>
                              ). We augment CopyMix graphical model by a copy-number-independent SNV process; we refer to this version of CopyMix as SNV-CopyMix. In
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 4
                              </a>
                              , the new components are colored in magenta. X
                              <sub>
                               nl
                              </sub>
                              denotes the observable variable, corresponding to a nucleotide in the genome, that we assume to be dependent on cluster-specific latent point mutation S
                              <sub>
                               kl
                              </sub>
                              . Naturally, each S
                              <sub>
                               kl
                              </sub>
                              is distributed as Bernoulli, where mutation corresponds to value of zero, and non-mutation corresponds to value of one. The prior distribution of S
                              <sub>
                               kl
                              </sub>
                              is ξ, a Beta distribution. Note that the SNV data contains L sites such that L >> M. In the following subsection, the calculation of the probability of X
                              <sub>
                               nl
                              </sub>
                              is described; for more detail see the work by
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               Koptagel et al. (2018)
                              </a>
                              . The VI update equations are calculated analogously to those in CopyMix; see
                              <a id="xref-sec-42-1" class="xref-app" href="#sec-42">
                               A.7
                              </a>
                              .
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F5.large.jpg?width=800&height=600&carousel=1" title="The graphical model with SNV process included." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">The graphical model with SNV process included.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F5.medium.gif" width="440" height="387"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F5.medium.gif" width="440" height="387"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <p id="p-48" class="first-child">
                                The graphical model with SNV process included.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="sec-14" class="subsection">
                              <h4>
                               2.3.1 The likelihood model for SNVs
                              </h4>
                              <p id="p-49">
                               Here, we briefly describe the computations related to the likelihood model for the SNVs, i.e., the probability of each X
                               <sub>
                                nl
                               </sub>
                               is calculated. For more details, see the earlier work
                               <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                                Koptagel et al. (2018)
                               </a>
                               . The genotype of a site l is denoted by
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the nucleotide of allele i. The site genotype consists of two copies of the reference nucleotide in the case of no mutation. It consists of one reference (denoted as ref) and one alternate (denoted as alt) nucleotide in the case of a heterozygous SNV, and two alternates in the case of homozygous SNV. Note that the sites are assumed to be diploid. Let t be an arbitrary nucleotide in the read. Also, assume that r is an enumeration of the number of reads; e.g., if a cell has five reads at a site, then r = 1,.., 5. Moreover, we define ϵ to be a the error probability. It is computed by processing the Phred scores in data—Phred score
                               <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                                Ewing et al. (1998)
                               </a>
                               is a measure of the quality of the identification of the nucleotides generated by DNA sequencing. The calculation of genotype likelihood is similar to Monovar
                               <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                Zafar et al. (2014)
                               </a>
                               , but it is a generalized version of Monovar, where we explicitly distinguish allele info for refref and altalt cases. To this end, the likelihood of a single read of cell n at site
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with the corresponding error probability
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-50">
                               The reads are assumed to be i.i.d., so the likelihood of the reads of cell n at site l is
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-51">
                               The likelihood for a non-mutation case is
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The likelihood for a heterozygous mutation case is
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and a homozygous mutation case is
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Note we allow for two versions of SNV-CopyMix, that is, (1) likelihood conditioned on non-mutation versus homozygous mutation, and (2) likelihood conditioned on non-mutation versus heterozygous mutation. In the experiment section, we test both of these versions when referring to SNV-CopyMix.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             3 Empirical investigation
                            </h2>
                            <div id="sec-16" class="subsection">
                             <h3>
                              3.1 Experimental setup
                             </h3>
                             <div id="sec-17" class="subsection">
                              <h4>
                               3.1.1 Dataset
                              </h4>
                              <p id="p-52">
                               We perform experiments on both simulated data and two published biological datasets: (1) Count data from
                               <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               (available from the NCBI Sequence Read Archive (
                               <a href="https://www.ncbi.nlm.nih.gov/sra">
                                SRA
                               </a>
                               ) under accession number SRP074289); (2) Real value data from
                               <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">
                                Laks and McPherson (2019)
                               </a>
                               accessed via
                               <a href="https://zenodo.org/record/3445364#.YZTwKdbMK_u">
                                DLP
                               </a>
                               .
                              </p>
                              <p id="p-53">
                               The first biological data stem from single-cell whole-genome sequencing from 18 primary colon cancer tumor cells and 18 metastatic cells from matched liver samples for one patient referred to as CRC2 in
                               <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . In this work, we mainly analyze CRC2 patient data (more complex than CRC1 according to
                               <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               ) considering chromosomes 18, 19, 20, and 21 comprising a total of 904 genomic bins of an approximate size of 200Kb each. Each data point corresponds to the number of reads aligned per bin per cell after GC correction
                               <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . These counts stemming from the tumor cell’s genome reflect the hidden copy numbers—copy number integers are conventionally ranged from 0 to 6, as in
                               <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . The higher the counts are, the higher the copy numbers are expected to be. Regarding clustering in
                               <a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               , it is reasonable that the primary tumor cells from the colon and the metastatic cells from the liver should cluster separately, as inter-tumor copy number differences are expected to dominate over minor intra-tumor copy number variations; therefore, we use the terms primary and metastatic clusters.
                              </p>
                              <p id="p-54">
                               The second biological data stem from single-cell whole-genome sequencing using DLP technology. The data is from three cell lines from ovarian cancer with total number of 891 cells
                               <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">
                                Laks and McPherson (2019)
                               </a>
                               . It is meaningful to infer that each cell line is a separate cluster, however, the goal is to find sub-clusters as well.
                              </p>
                              <p id="p-55">
                               For the simulated data, we examine both Poisson and Gaussian models. For the Poisson version, we randomly generate 30 datasets for each test case, following our graphical model and inspired by biological datasets from
                               <a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               ; for data simulation details, see
                               <a id="xref-sec-50-1" class="xref-app" href="#sec-50">
                                A.8
                               </a>
                               . We examine different test cases, orthogonal to each other, to show our approach’s robustness; each scenario, for both clustering and copy number state inference, is reproducible. Regarding the Gaussian model, we similarly show the robustness of CopyMix by varying the number of clusters and clustering scenarios.
                              </p>
                             </div>
                             <div id="sec-18" class="subsection">
                              <h4>
                               3.1.2 Experimental Protocol
                              </h4>
                              <p id="p-56">
                               For the Poisson version, we run VI for different numbers of clusters considering various random and informative initialization methods, e.g., k-means and Baum-Welch, as recommended by
                               <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                                Blei et al. (2017)
                               </a>
                               . For each given number of clusters, we choose the best VI run based on the highest log-likelihood. Next, we select the number of clusters by an elbow selection method
                               <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                                Johnson and Wichern (2007)
                               </a>
                               based on the Deviance Information Criterion
                               <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                                Spiegelhalter et al. (2002)
                               </a>
                               over different numbers of clusters. We assess the clustering performance of CopyMix on 30 simulated datasets for each test case via V-measure
                               <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                                Rosenberg and Hirschberg (2007)
                               </a>
                               . The V-measure, a value between 0 and 1, quantifies to which extent a cluster only contains data points that are members of a single class (the higher V-measure, the better clustering). Regarding inference of copy number profiles, we evaluate the performance of CopyMix on simulated data by calculating the proportion of discordant (unequal) position values between true and inferred vectors, as in
                               <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                                de Souza et al. (2020)
                               </a>
                               , of copy number states. For biological data evaluation, V-measure is used to assess the clustering performance where the true clusters are assumed to correspond to the primary and metastatic cells; note that these may not be ground truth, but we believe there is certain concordance with the unknown ground truth. Regarding state inference, we compare the copy number profiles between the results from CopyMix and the original paper
                               <a id="xref-ref-18-10" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               .
                              </p>
                              <p id="p-57">
                               For the Gaussian version, we assume we know the number of clusters. We run for different numbers of clusters and choose the result with the highest ELBO. Note that this is a simplified version of the above. In addition to merely comparing to the clustering result in the original paper
                               <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">
                                Laks and McPherson (2019)
                               </a>
                               , we examine if the original paper has inferred meaningful clusters; we obtain this by using metrics CH Index
                               <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                                Calinski and Harabasz (1974)
                               </a>
                               and Silhouette
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                Rousseeuw (1987)
                               </a>
                               . To compare to the original paper
                               <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">
                                Laks and McPherson (2019)
                               </a>
                               , we use likelihood ration test. Finally, we provide ELBO in the reports.
                              </p>
                              <p id="p-58">
                               CH Index is the ratio of the sum of between-clusters dispersion and of inter-cluster dispersion for all clusters, the higher the score, the better the performances. Silhouette is calculated using two scores a, b, as shown below. Note that a is the mean distance between a sample and all other points in the same class. This score measure the closeness of points in the same cluster. And, b is the mean distance between a sample and all other points in the next nearest cluster. This score measure the distance of points of different clusters.
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-59">
                               The s score is bounded between −1 for incorrect clustering and +1 for highly dense clustering. Scores around zero indicate overlapping clusters. The score is higher when clusters are dense and well separated, which relates to a standard concept of a cluster.
                              </p>
                              <p id="p-60">
                               Finally, the likelihood ration test assesses the goodness of fit of two competing statistical models based on the ratio of their likelihoods. In our experiments, we examine how statically significant the winning model is; by the winning model we mean the one with the higher likelihood. The significance is measured by p-value; a high p-value indicates that the null hypothesis is preferred over the winning hypothesis.
                              </p>
                             </div>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              3.2 Experimental Results for Poisson Version
                             </h3>
                             <p id="p-61">
                              We perform experiments on simulated data and biological data, and we evaluate a sequential version of CopyMix to verify the superiority of single-phase inference over the sequential framework.
                             </p>
                             <div id="sec-20" class="subsection">
                              <h4>
                               Simulated data
                              </h4>
                              <p id="p-62">
                               Here, we evaluate CopyMix, and compare it to the initialization methods used by VI on 30 simulated datasets for each test scenario. The comparison is to illustrate, firstly, that VI results are not already achieved by the initialization methods, and secondly, that VI can recover the poor initialization performance and achieve optimal performance. We select the number of cells, sequence lengths, rate parameters, transition probabilities based on
                               <a id="xref-ref-18-11" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . We assume equal amplification on the read counts across bins since the bin size is high enough (200kb as in
                               <a id="xref-ref-18-12" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               ), meaning that the effect of unequal amplification is negligible.
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Figures 5
                               </a>
                               ,
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                6
                               </a>
                               and
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                7
                               </a>
                               provide summaries of V-measures and proportions of discordant positions; note that the median values in the boxplots, highlighted in red for initialisation method and in blue for CopyMix, aim to facilitate the comparisons. The figures illustrate the performance comparison of CopyMix and the chosen initialization method, by CopyMix, for different settings: copy number transition matrices and number of clusters (
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Fig. 3
                               </a>
                               ); sequence length (
                               <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                Fig.4
                               </a>
                               ); and number of cells (
                               <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                Fig. 5
                               </a>
                               ). For all the tests, CopyMix results in V-measure of 1 except for one case (
                               <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                Fig. 5
                               </a>
                               ) where V-measure is in an interval of [.85, 1] with the median being 1; proportions of discordant positions are mostly 0 and otherwise are in the interval of [0, .2]. These results are optimal as the copy number profiling errors are near 0 and clustering near 1 (or 100%).
                              </p>
                              <div id="F6" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F6.large.jpg?width=800&height=600&carousel=1" title="Performance of initialization methods (brown bars with red median values) and CopyMix (cyan bars with blue median values) across different sets of transition matrices. LHS: discordant positions between true and inferred vectors of copy number states. RHS: V-measure between true and inferred cell clustering assignments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">Performance of initialization methods (brown bars with red median values) and CopyMix (cyan bars with blue median values) across different sets of transition matrices. LHS: discordant positions between true and inferred vectors of copy number states. RHS: V-measure between true and inferred cell clustering assignments.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F6.medium.gif" width="440" height="164"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F6.medium.gif" width="440" height="164"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 5:
                                </span>
                                <p id="p-63" class="first-child">
                                 Performance of initialization methods (brown bars with red median values) and CopyMix (cyan bars with blue median values) across different sets of transition matrices. LHS: discordant positions between true and inferred vectors of copy number states. RHS: V-measure between true and inferred cell clustering assignments.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F7" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F7.large.jpg?width=800&height=600&carousel=1" title="Performance of initialization methods and CopyMix, varying the sequence length. LHS and RHS figures, as well as color codes, are similar to Fig. 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">Performance of initialization methods and CopyMix, varying the sequence length. LHS and RHS figures, as well as color codes, are similar to Fig. 5.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F7.medium.gif" width="440" height="165"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F7.medium.gif" width="440" height="165"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 6:
                                </span>
                                <p id="p-64" class="first-child">
                                 Performance of initialization methods and CopyMix, varying the sequence length. LHS and RHS figures, as well as color codes, are similar to
                                 <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                                  Fig. 5
                                 </a>
                                 .
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F8" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F8.large.jpg?width=800&height=600&carousel=1" title="Performance of initialization methods and CopyMix varying the number of cells. LHS and RHS figures, as well as color codes, are similar to Fig. 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">Performance of initialization methods and CopyMix varying the number of cells. LHS and RHS figures, as well as color codes, are similar to Fig. 5.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F8.medium.gif" width="440" height="165"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F8.medium.gif" width="440" height="165"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 7:
                                </span>
                                <p id="p-65" class="first-child">
                                 Performance of initialization methods and CopyMix varying the number of cells. LHS and RHS figures, as well as color codes, are similar to
                                 <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                                  Fig. 5
                                 </a>
                                 .
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p>
                               The results in
                               <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                                Fig. 5
                               </a>
                               are obtained considering 45 cells—each with a sequence length of 800 bins and the mean value around 100—clustered into two and three clusters, wherein each clustering scenario, each cluster bears a different transition pattern than other clusters. The copy number transition patterns (forming the transition matrix of a Markov chain) vary across configurations A to E, which are orthogonal to each other; Configurations A to E are formed by combining the following state patterns.
                              </p>
                              <ul class="list-unord" id="list-4">
                               <li id="list-item-12">
                                <p id="p-67">
                                 <strong>
                                  single state
                                 </strong>
                                 , a copy number state is inclined to stay at one certain state;
                                </p>
                               </li>
                               <li id="list-item-13">
                                <p id="p-68">
                                 <strong>
                                  self-transition state
                                 </strong>
                                 , a copy number state is inclined to stay at its current state for awhile;
                                </p>
                               </li>
                               <li id="list-item-14">
                                <p id="p-69">
                                 <strong>
                                  oscillating state
                                 </strong>
                                 , a copy number state changes shortly after being at a state;
                                </p>
                               </li>
                               <li id="list-item-15">
                                <p id="p-70">
                                 <strong>
                                  spike state
                                 </strong>
                                 , a sudden change of copy number state arises for a specific short range of positions in the copy number sequence;
                                </p>
                               </li>
                               <li id="list-item-16">
                                <p id="p-71">
                                 <strong>
                                  scaled state
                                 </strong>
                                 , a copy number changes by a multiplicative or additive constant.
                                </p>
                               </li>
                              </ul>
                              <p id="p-72">
                               The configurations A to E are:
                               <strong>
                                A
                               </strong>
                               , similar to chromosomes 11 in CRC1 and 4 in CRC2, (one cluster follows (
                               <strong>
                                scaled state
                               </strong>
                               );
                               <strong>
                                B
                               </strong>
                               , similar to chromosome 11 in CRC1 (one cluster follows
                               <strong>
                                self-transition state
                               </strong>
                               and the other one
                               <strong>
                                single state
                               </strong>
                               );
                               <strong>
                                C
                               </strong>
                               , similar to chromosome 3 in CRC1 (two clusters have identical transition matrices but one arrive at
                               <strong>
                                spike state
                               </strong>
                               );
                               <strong>
                                D
                               </strong>
                               , similar to chromosomes 18 to 21 in CRC2 (clusters combine
                               <strong>
                                self-transition state
                               </strong>
                               ,
                               <strong>
                                oscillating state
                               </strong>
                               , and
                               <strong>
                                scaled state
                               </strong>
                               );
                               <strong>
                                E
                               </strong>
                               , two of the clusters have the transitions mentioned in case
                               <strong>
                                C
                               </strong>
                               , and, the third cluster has a combination of
                               <strong>
                                oscillating state
                               </strong>
                               and
                               <strong>
                                single state
                               </strong>
                               .
                              </p>
                              <p id="p-73">
                               We can observe that, in
                               <a id="xref-fig-6-7" class="xref-fig" href="#F6">
                                Fig. 5
                               </a>
                               , CopyMix has a perfect performance in terms of clustering and copy number inference for easier cases and also a high performance concerning the harder cases (
                               <strong>
                                D
                               </strong>
                               and
                               <strong>
                                E
                               </strong>
                               ). Next, however not shown here, we investigate decreasing the rate parameter, θ
                               <sub>
                                n
                               </sub>
                               , for configuration
                               <strong>
                                A
                               </strong>
                               to see how our model handles shallow data as this can be a challenge in copy number profiling
                               <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">
                                Zahn et al. (2017)
                               </a>
                               . Our model is robust towards lower rates up to θ
                               <sub>
                                n
                               </sub>
                               = 3, without reducing the quality of performance concerning V-measure and discordant positions. As the third set of tests, we rerun the test using configuration
                               <strong>
                                A
                               </strong>
                               , but we increase the size of the sequences up to 10000 (inspired by the total length of the genomic bins from biological data in
                               <a id="xref-ref-18-13" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               ), see
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                Fig. 6
                               </a>
                               . We can observe that our model handles these sequences without any decrease in performance. Finally,
                               <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                                Fig. 7
                               </a>
                               illustrates the perfect performance of CopyMix when varying the number of cells from 15 to 500 when, again, using configuration
                               <strong>
                                A
                               </strong>
                               .
                              </p>
                             </div>
                             <div id="sec-21" class="subsection">
                              <h4>
                               Biological data
                              </h4>
                              <p id="p-74">
                               We evaluate CopyMix on the biological data from
                               <a id="xref-ref-18-14" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . As aforemen-tioned, we do not compare CopyMix to the earlier state-of-the-art methods due to their inclusion of ad-hoc processes in their framework; however, to evaluate the CopyMix results on the biological data, we use biologically verifiable results from the method in
                               <a id="xref-ref-18-15" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . Using chromosomes 18 to 21, CopyMix clusters the cells into primary and metastatic groups, perfectly matching the corresponding primary and metastatic organs—V-measure is 1 w.r.t. the cells found in the two organs
                               <a id="xref-ref-18-16" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . Adding chromosomes 3 and 8 to our test, metastatic tumor cells are clustered into two major subclusters as in
                               <a id="xref-ref-18-17" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               . As mentioned in
                               <a id="xref-ref-18-18" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               , amplifications on chromosomes 3 and 8, where some known genes are present, contribute to the detection of the metastatic subclusters. Note that,
                               <a id="xref-ref-18-19" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               does not present a method to determine the threshold in hierarchical clustering, which results in the reported subclusters; moreover, there is no ground truth to verify these results.
                              </p>
                              <p id="p-75">
                               Regarding the copy number profiling,
                               <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                                Fig. 8
                               </a>
                               illustrates the copy number profiles inferred by CopyMix and the original work by
                               <a id="xref-ref-18-20" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               , here referred to as Navin. The heatmaps indicate a higher copy number state by a darker color. Note that in Navin, they perform ploidy scaling, which contributes to scaling differences compared to the results of CopyMix. Moreover, the Navin heatmap, here, illustrates an average of the heatmap in the original work across the cells since there is no single copy number profile per cluster. That is, Navin, unlike CopyMix, lacks a single copy number profile per cluster, revealing the copy number variation pattern behind each cluster. CopyMix detects the breakpoints (copy number changes) similar to Navin, with an accuracy of 76%, using the breakpoint margin (of a size corresponding to 5% of the sequence length). Again, there is no ground truth to verify the Navin results from
                               <a id="xref-ref-18-21" class="xref-bibr" href="#ref-18">
                                Leung et al. (2017)
                               </a>
                               .
                              </p>
                              <div id="F9" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F9.large.jpg?width=800&height=600&carousel=1" title="Copy number profiling comparison between the results from Navin Leung et al. (2017) (LHS) and CopyMix (RHS) for both primary and metastatic clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Copy number profiling comparison between the results from Navin Leung et al. (2017) (<em>LHS</em>) and CopyMix (<em>RHS</em>) for both primary and metastatic clusters.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F9.medium.gif" width="440" height="160"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F9.medium.gif" width="440" height="160"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 8:
                                </span>
                                <p id="p-76" class="first-child">
                                 Copy number profiling comparison between the results from Navin
                                 <a id="xref-ref-18-22" class="xref-bibr" href="#ref-18">
                                  Leung et al. (2017)
                                 </a>
                                 (LHS) and CopyMix (RHS) for both primary and metastatic clusters.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div id="sec-22" class="subsection">
                              <h4>
                               A sequential CopyMix
                              </h4>
                              <p id="p-77">
                               Applying a sequential version of CopyMix (1. performing copy number profiling, using EM for HMMs; 2. performing clustering over the inferred copy numbers, using EM for a mixture model), resembling
                               <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">
                                Shah et al. (2006)
                               </a>
                               ;
                               <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">
                                Vitak et al. (2017)
                               </a>
                               but excluding their ad-hoc processes, on the biological data, results in poor clustering results (V-measure of .24) while, as mentioned, CopyMix gains V-measure of 1. This result confirms that simultaneous detection of copy number profiles and clustering is superior to the sequential approach. In particular, we conclude that methods that use EM for HMMs, using HMMcopy
                               <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">
                                Shah et al. (2006)
                               </a>
                               framework, benefit from the multiple ad-hoc transformations within the HMMcopy framework, much more than the pure computational model, e.g., HMM. Adding multiple ad-hoc rules within such computational models lacks elegance, integrity, and potential modelbased approach benefits.
                              </p>
                             </div>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              3.3 Experimental Results for Gaussian Version
                             </h3>
                             <p id="p-78">
                              We perform experiments on simulated data and biological data. For the biological data we evaluate CopyMix against the original paper, referring to as DLP, using CH Index, Silhouette, and likelihood ratio test.
                             </p>
                             <div id="sec-24" class="subsection">
                              <h4>
                               Simulated data
                              </h4>
                              <p id="p-79">
                               The experiments are similar to those shown for the Poisson version. Here, we, moreover, compare the ELBO. For the sake of investigating the model behavior w.r.t. various quality of initializations, we report the results for three different initializations.
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               illustrates the performance of CopyMix and the chosen initialization method, by CopyMix, for 19 different settings. The plots in
                               <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                                Fig. 9
                               </a>
                               to
                               <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                10
                               </a>
                               shows these configurations of clustering scenarios. In
                               <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               , the highlighted ELBO values (the best ones per configuration) guide us which λ for each configuration is desirable. The three λ are basically in the hyperparameter value of the Dirichlet for the edge weight (λ
                               <sub>
                                1
                               </sub>
                               : 0.1, λ
                               <sub>
                                2
                               </sub>
                               : 0.5, λ
                               <sub>
                                3
                               </sub>
                               : 10). The overall V-measure concerning the Init method based on the three different hyperparameters is also reported.
                              </p>
                              <div id="T1" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2929911/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2929911/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2929911" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 Table 1:
                                </span>
                                <p id="p-80" class="first-child">
                                 Clustering result of CopyMix vs initialization method, for Conf 1 to Conf 17.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F10" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F10.large.jpg?width=800&height=600&carousel=1" title="Binary clustering scenarios, using different CNV (transition) patterns, which form CONF 1 to CONF 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">Binary clustering scenarios, using different CNV (transition) patterns, which form CONF 1 to CONF 6.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F10.medium.gif" width="440" height="221"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 9:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F10.medium.gif" width="440" height="221"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 9:
                                </span>
                                <p id="p-81" class="first-child">
                                 Binary clustering scenarios, using different CNV (transition) patterns, which form CONF 1 to CONF 6.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F11" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F11.large.jpg?width=800&height=600&carousel=1" title="Clustering scenarios with 3 components (CONF 7 to 10), with 4 components (CONF 11 to 14), and with 5 components (CONF 15 to 17)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">Clustering scenarios with 3 components (CONF 7 to 10), with 4 components (CONF 11 to 14), and with 5 components (CONF 15 to 17).</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F11.medium.gif" width="433" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 10:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F11.medium.gif" width="433" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 10:
                                </span>
                                <p id="p-82" class="first-child">
                                 Clustering scenarios with 3 components (CONF 7 to 10), with 4 components (CONF 11 to 14), and with 5 components (CONF 15 to 17).
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-83">
                               We can observe that CopyMix, in general, performs well w.r.t. clustering. There are transition scenarios which are challenging for CopyMix, but they are still clustered with good V-measures. Finally, there are scenarios in which CopyMix cannot handle as good. For example, CONF 5 is hard for CopyMix, where the requirement of the successful clustering is to distinguish two shifted monotone sequence patterns. Another challenge for CopyMix, like CONF 17, is when the data has more variance in the transition patterns across the sequences and the clusters are overlapping.
                              </p>
                             </div>
                             <div id="sec-25" class="subsection">
                              <h4>
                               Biological data
                              </h4>
                              <p id="p-84">
                               We evaluate CopyMix on the biological data from
                               <a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">
                                Laks and McPherson (2019)
                               </a>
                               . We execute CopyMix on the whole genome with length of 6206 bins. We run 50 initializations for 50 iterations, and pick the ones corresponding to the 10 highest ELBO; subsequently, we let the 10 ones run until convergence. We pick the winning initialization method which has the highest final ELBO. We detect 4 clusters in total; the three original cell lines are fully recovered and one of the cell lines is clustered into two sub-clones. The original DLP co-clustering results (between the cell lines and the 9 clones detected by DLP), and the CopyMix co-clustering results (between 4 CopyMix clones and 9 DLP clones) are shown in
                               <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                                Fig. 11
                               </a>
                               . Note that the CopyMix clones 0 and 1 corresponds to DLP clones E and F, respectively. That is, 145 cells from cell line OV2295(R2) belong to DLP clone E and 75 cells from the same cell line belong to DLP clone F. The rest of the cell line are also matched correctly, as it can be seen by comparing the plots. The resulting ELBO for CopyMix is −4957712 and the V-measure when comparing CopyMix to DLP is 67%. The initialization method provides only V-measure of 1%.
                               <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                                Fig 12
                               </a>
                               shows how some clones are too overlapping, which explains why CopyMix did not account for them as separate clusters. We now show the results of three metrics which confirm this observation, statistically speaking.
                               <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                                Fig. 13
                               </a>
                               shows that CopyMix outperforms DLP method w.r.t. both CH Index and Silhouette. However, regarding Silhouette, the score .49 is not the perfect score, i.e., 1. These metrics both show that DLP method has detected overlapping clusters which is not in consensus with the definition of clustering in machine learning literature; however, biologically speaking, DLP results can make sense. Finally, we show that performing a likelihood ration test confirms that CopyMix as a null hypothesis is favored over DLP. Assuming each clone follows an empirical Gaussian distribution (the mean and variance can be calculated by MLE), the DLP log likelihood is −8980284.42 and the CopyMix log likelihood is −9023905.94. The test results in p-value 1 which means that DLP likelihood is not significantly higher that CopyMix likelihood.
                              </p>
                              <div id="F12" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F12.large.jpg?width=800&height=600&carousel=1" title="The original three cell lines and the co-clustering of CopyMix and DLP." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">The original three cell lines and the co-clustering of CopyMix and DLP.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F12.medium.gif" width="440" height="158"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 11:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F12.medium.gif" width="440" height="158"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 11:
                                </span>
                                <p id="p-85" class="first-child">
                                 The original three cell lines and the co-clustering of CopyMix and DLP.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F13" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F13.large.jpg?width=800&height=600&carousel=1" title="The GC-corrected data presented in 9 clones detected by DLP." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">The GC-corrected data presented in 9 clones detected by DLP.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 12:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F13.medium.gif" width="440" height="369"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 12:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F13.medium.gif" width="440" height="369"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 12:
                                </span>
                                <p id="p-86" class="first-child">
                                 The GC-corrected data presented in 9 clones detected by DLP.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F14" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F14.large.jpg?width=800&height=600&carousel=1" title="The result of CH Index and Silhouette w.r.t CopyMix, DLP, and DLP with some major sub-clones." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">The result of CH Index and Silhouette w.r.t CopyMix, DLP, and DLP with some major sub-clones.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 13:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F14.medium.gif" width="440" height="283"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 13:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F14.medium.gif" width="440" height="283"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 13:
                                </span>
                                <p id="p-87" class="first-child">
                                 The result of CH Index and Silhouette w.r.t CopyMix, DLP, and DLP with some major sub-clones.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-88">
                               Regarding the copy number profiling,
                               <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                                Fig. 14
                               </a>
                               illustrates the copy number profiles inferred by CopyMix.
                              </p>
                              <div id="F15" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F15.large.jpg?width=800&height=600&carousel=1" title="The copy number profiles detected by CopyMix corresponding to 4 clones. The bottom figure shows a higher resolution version by showing bins 400 to 800." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">The copy number profiles detected by CopyMix corresponding to 4 clones. The bottom figure shows a higher resolution version by showing bins 400 to 800.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 14:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F15.medium.gif" width="440" height="163"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 14:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F15.medium.gif" width="440" height="163"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 14:
                                </span>
                                <p id="p-89" class="first-child">
                                 The copy number profiles detected by CopyMix corresponding to 4 clones. The bottom figure shows a higher resolution version by showing bins 400 to 800.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-90">
                               Finally, we compared CopyMix’s clustering results with those of Ginkgo
                               <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                                Garvin et al. (2015)
                               </a>
                               . Unlike CopyMix, Ginkgo applies multiple ad-hoc processes such as outlier removal from cells and bins and various corrections to the processed data. As shown in
                               <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                                Fig. 15
                               </a>
                               , Ginkgo, after having removed 717 cells out of 891 cells (80% of the cells), detects three major clusters. Ginkgo, similarly to CopyMix, obtained a V-measure of 98% when detecting the three cell lines. However, CopyMix does not remove any cell in contrast to Ginkgo, which removed 80% the DLP cells prior to the clustering. As shown in
                               <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                                Fig. 15
                               </a>
                               , Ginkgo detects subgroups within the cell lines; however, the corresponding branch lengths in their hierarchical clustering are minimal compared to the branch lengths corresponding to the major clusters detected by Ginkgo. That is, to include these branches (and consequently the sub-clusters), one needs to choose a shallow threshold when cutting the dendrogram produced by the hierarchical clustering. If we, based on shared events in dark green in
                               <a id="xref-fig-16-3" class="xref-fig" href="#F16">
                                Fig. 15
                               </a>
                               , assign a threshold such that we include the two subgroups depicted in the bottom and green part of the
                               <a id="xref-fig-16-4" class="xref-fig" href="#F16">
                                Fig. 15
                               </a>
                               then Ginkgo detects four clusters. Regarding the four clusters, CopyMix outperforms Ginkgo with V-measure 67% compared to V-measure 55%. The rest of the Ginkgo subgroups, containing magenta-colored copy numbers, require a lower threshold than those for four clusters. Finally, it is essential to mention that Ginkgo’s web application, one advantage of using Ginkgo
                               <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                                Mallory et al. (2020)
                               </a>
                               , was not able to handle the large DLP data. To run Ginkgo, we instead developed a bash tool that assembled the back-end parts of the Ginkgo into a new script.
                              </p>
                              <div id="F16" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F16.large.jpg?width=800&height=600&carousel=1" title="The heat map and the denrogram show three major clusters detected by Ginkgo. The copy number is color-coded by a scale from green (small) to magenta (large)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup">The heat map and the denrogram show three major clusters detected by Ginkgo. The copy number is color-coded by a scale from green (small) to magenta (large).</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 15:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F16.medium.gif" width="440" height="223"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 15:" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F16.medium.gif" width="440" height="223"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 15:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 15:
                                </span>
                                <p id="p-91" class="first-child">
                                 The heat map and the denrogram show three major clusters detected by Ginkgo. The copy number is color-coded by a scale from green (small) to magenta (large).
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-26">
                            <h2 class="">
                             4 Conclusion
                            </h2>
                            <p id="p-92">
                             The tasks of single-cell copy number profiling and clustering have so far been sequential, requiring multiple ad-hoc processing of data; hence prone to introduce clustering artifacts. We introduce a mixture modelbased framework, CopyMix, to allow for a fully probabilistic and simultaneous single-cell copy number profiling and clustering using VI. Our approach is evaluated on both simulated and biological data resulting in good performance w.r.t. both of the tasks.
                            </p>
                            <p id="p-93">
                             A future direction is to apply a biological indicator to detect clustering artifacts, where the contribution of non-sequential methods can be additionally illustrated. Overall, CopyMix provides various research opportunities concerning clustering and copy number profiling, such as improving clustering, computational efficiency, augmenting and refining the model, and modeling biallelic deletions and aneuploidy inspired by a recently proposed CopyKat
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              Gao et al. (2021)
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-28">
                            <h2 class="">
                             Author contributions statement
                            </h2>
                            <p id="p-94">
                             N.S. partially designed CopyMix, partially developed the mathematics, fully implemented CopyMix, fully wrote the manuscript, partially selected the evaluations metrics, and fully conducted experiments on CopyMix. C.P.E.S. partially designed CopyMix, partially developed the mathematics, helped with the design and evaluation of the study, partially wrote and reviewed the manuscript. A.R. provided useful suggestions for the evaluation of CopyMix and was involved in the model discussions. H.K. provided SNV emissions for the SNV-CopyMix and provided the main text for the section “
                             <a id="xref-sec-14-1" class="xref-sec" href="#sec-14">
                              The likelihood model for SNVs
                             </a>
                             ”. H.T. ran Ginkgo on the biological DLP data. J.L. fully supervised and reviewed the paper, was involved in the entire project, and partially wrote the manuscript.
                            </p>
                           </div>
                           <div class="section app" id="app-1">
                            <h2 class="">
                             A Appendix: Derivation details
                            </h2>
                            <div class="section" id="sec-29">
                             <h3>
                              A.1 Step 2 of
                              <a id="xref-sec-3-1" class="xref-sec" href="#sec-3">
                               Section 2.1
                              </a>
                             </h3>
                             <p id="p-95">
                              In this section, we provide the calculation of each decomposed term of log P(
                              <strong>
                               Y
                              </strong>
                              ,
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               Ψ
                              </strong>
                              ). Each term of log P(
                              <strong>
                               Y
                              </strong>
                              ,
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               Ψ
                              </strong>
                              ) presented in Step 2 of
                              <a id="xref-sec-2-1" class="xref-sec" href="#sec-2">
                               Section 2
                              </a>
                              is calculated as follows:
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where,
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div class="section" id="sec-30">
                             <h3>
                              A.2 Step 3 and 4 of
                              <a id="xref-sec-3-2" class="xref-sec" href="#sec-3">
                               Section 2.1
                              </a>
                             </h3>
                             <p id="p-96">
                              In this section we first present the details of the step 3 of
                              <a id="xref-sec-2-2" class="xref-sec" href="#sec-2">
                               section 2
                              </a>
                              , i.e., the derivation of each posterior update equation (VD computation). We then summarise them in the form of a pseudo code, covering the details of step 4 in
                              <a id="xref-sec-2-3" class="xref-sec" href="#sec-2">
                               Section 2
                              </a>
                              . For convenience, we use +≈ to denote equality up to a constant additive factor.
                             </p>
                             <p id="p-97">
                              Update equation for
                              <strong>
                               π
                              </strong>
                              :
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-98">
                              Let I(·) be the indicator function. q(π) is a Dirichlet distribution with parameters α = (α
                              <sub>
                               1
                              </sub>
                              , …, α
                              <sub>
                               K
                              </sub>
                              ), where
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-99">
                              Update equation for Z
                              <sub>
                               n
                              </sub>
                              :
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-100">
                              Let y
                              <sub>
                               nm
                              </sub>
                              be the observed number of reads aligned to bin m for cell n corresponding to the random variable Y
                              <sub>
                               nm
                              </sub>
                              and let
                              <span class="inline-formula" id="inline-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Note that log P(
                              <strong>
                               Y
                              </strong>
                              |
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               Ψ
                              </strong>
                              ) and log P(
                              <strong>
                               Z
                              </strong>
                              |
                              <strong>
                               Ψ
                              </strong>
                              ) can be written as the sum of two terms, one that depends on Z
                              <sub>
                               n
                              </sub>
                              and one that does not, i.e.,
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-101">
                              Consequently,
                              <span class="disp-formula" id="disp-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-102">
                              The distribution q(Z
                              <sub>
                               n
                              </sub>
                              ) is a Categorical distribution with parameters
                              <strong>
                               π
                              </strong>
                              <sub>
                               n
                              </sub>
                              = (π
                              <sub>
                               n1
                              </sub>
                              , …, π
                              <sub>
                               nK
                              </sub>
                              ), where
                              <span class="disp-formula" id="disp-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-103">
                              Update equation for θ
                              <sub>
                               n
                              </sub>
                              :
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-104">
                              We can write log P(
                              <strong>
                               Y
                              </strong>
                              |
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               Ψ
                              </strong>
                              ) as in Eq. A.7 and log P(
                              <strong>
                               θ
                              </strong>
                              ) which depends on θ
                              <sub>
                               n
                              </sub>
                              as,
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-105">
                              Therefore,
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-106">
                              Note that
                              <span class="inline-formula" id="inline-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-107">
                              The distribution q(θ
                              <sub>
                               n
                              </sub>
                              ) is a Gamma distribution with shape and rate parameters:
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-108">
                              Update equation for
                              <span class="inline-formula" id="inline-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              :
                              <span class="disp-formula" id="disp-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-109">
                              Disregarding the terms that do not depend on
                              <span class="inline-formula" id="inline-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in log P(
                              <strong>
                               C|Ψ
                              </strong>
                              ) and log P(
                              <strong>
                               A
                              </strong>
                              ), we obtain:
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-110">
                              The distribution
                              <span class="inline-formula" id="inline-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is Dirichlet with parameters
                              <span class="inline-formula" id="inline-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-111">
                              Update equation for
                              <strong>
                               C
                              </strong>
                              <sub>
                               k
                              </sub>
                              : In the calculation of q(
                              <strong>
                               C
                              </strong>
                              <sub>
                               k
                              </sub>
                              ), the resulting distribution resembles the posterior probability of the hidden variables in an HMM and only differs in the normalization constant. Similarly to the approach adopted in
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               MacKay (1997)
                              </a>
                              , we use a slightly different implementation of forward-backward algorithm. More specifically, we define a graph (one per cluster k) G = {V, E} with vertices
                              <span class="inline-formula" id="inline-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , having weights denoted as
                              <span class="inline-formula" id="inline-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , and edges
                              <span class="inline-formula" id="inline-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , having weights denoted as
                              <span class="inline-formula" id="inline-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Note that for the simplicity of notation we use C
                              <sub>
                               mj
                              </sub>
                              instead of C
                              <sub>
                               km
                              </sub>
                              = j for edges and vertices. We define forward
                              <span class="inline-formula" id="inline-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and backward
                              <span class="inline-formula" id="inline-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              quantities similar to HMMs
                              <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">
                               Bishop (2006)
                              </a>
                              . Instead of using the terminologies of transition and emission probabilities, we formulate a corresponding algorithm in terms of the weights of the graph,
                              <span class="inline-formula" id="inline-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The initial transition probability can be defined as w
                              <sub>
                               k
                              </sub>
                              (C
                              <sub>
                               0
                              </sub>
                              C
                              <sub>
                               1j
                              </sub>
                              ). Notice that k indicates the cluster which the graph belongs to. We calculate forward and backward quantities using the weights, and there is no need to normalize the graph weights since a normalization is performed later when calculating the VD. Having the weights, we compute the two posterior probabilities q(C
                              <sub>
                               km
                              </sub>
                              = j) and q(C
                              <sub>
                               km–1
                              </sub>
                              = i, C
                              <sub>
                               km
                              </sub>
                              = j); note that they are normalized by summing over all j ∈ [J].
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               McGrory and Titterington (2009)
                              </a>
                              derive the exact normalization of the graph weights, for a similar but less complex graphical model; however, this is unnecessary since the normalization can be performed later, as we do.
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-112">
                              We can write log P(
                              <strong>
                               Y
                              </strong>
                              |
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               Ψ
                              </strong>
                              ) as the sum of two terms, one that depends on
                              <strong>
                               C
                              </strong>
                              <sub>
                               k
                              </sub>
                              and another that does not, that is,
                              <span class="disp-formula" id="disp-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-113">
                              Let f(k) be a function of k as
                              <span class="inline-formula" id="inline-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , We can write
                              <span class="inline-formula" id="inline-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-114">
                              Thus, disregarding the terms that do not depend on
                              <strong>
                               C
                              </strong>
                              <sub>
                               k
                              </sub>
                              we obtain:
                              <span class="disp-formula" id="disp-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-115">
                              Calculations regarding directed graph are as follows:
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-116">
                              We define
                              <span class="inline-formula" id="inline-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-117">
                              The graph weights are (we assume w
                              <sub>
                               k
                              </sub>
                              (C
                              <sub>
                               0
                              </sub>
                              C
                              <sub>
                               1j
                              </sub>
                              ) to be fixed):
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-118">
                              Note that we skip writing i, j in the calculations to make them short and more readable:
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-119">
                              We now calculate the posteriors:
                              <span class="disp-formula" id="disp-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-54.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-54.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-120">
                              Finally, the following are the resulted update equations. Subsequently, the detailed algorithm used in CopyMix is shown in
                              <a id="xref-statement-2-3" class="xref-statement" href="#statement-2">
                               Algorithm 2
                              </a>
                              .
                              <span class="disp-formula" id="disp-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-55.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-55.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div class="section" id="sec-31">
                             <h3>
                              A.3 Calculating expectations for
                              <a id="xref-sec-3-3" class="xref-sec" href="#sec-3">
                               section 2.1
                              </a>
                             </h3>
                             <p>
                              In this section we list the calculations of the expected values that are used in the update equations in the previous Appendix. Let
                              <strong>
                               Φ
                              </strong>
                              be the digamma function defined as
                              <span class="inline-formula" id="inline-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which can be easily calculated via numerical approximation. The values of the expectations taken with respect to the approximated distributions are given as follows where
                              <strong>
                               Φ
                              </strong>
                              is used for some of them.
                             </p>
                             <ul class="list-unord" id="list-5">
                              <li id="list-item-17">
                               <p id="p-122">
                                E
                                <sub>
                                 q(Z
                                 <sub>
                                  n
                                 </sub>
                                 )
                                </sub>
                                (I(Z
                                <sub>
                                 n
                                </sub>
                                = k)) = π
                                <sub>
                                 nk
                                </sub>
                               </p>
                              </li>
                              <li id="list-item-18">
                               <p id="p-123">
                                Due to computational issues in calculating
                                <span class="inline-formula" id="inline-formula-41">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-41.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-41.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , we can instead compute it using the fact that
                                <span class="disp-formula" id="disp-formula-39">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-56.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-56.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-124">
                                Therefore, we obtain
                                <span class="inline-formula" id="inline-formula-42">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-42.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-42.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , with
                                <span class="inline-formula" id="inline-formula-43">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-43.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-43.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                being a Poisson p.m.f. with rate parameter
                                <span class="inline-formula" id="inline-formula-44">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-44.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-44.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                               </p>
                              </li>
                              <li id="list-item-19">
                               <p id="p-125">
                                <span class="inline-formula" id="inline-formula-45">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-45.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-45.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                              <li id="list-item-20">
                               <p id="p-126">
                                <span class="inline-formula" id="inline-formula-46">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-46.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-46.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                              <li id="list-item-21">
                               <p id="p-127">
                                <span class="inline-formula" id="inline-formula-47">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-47.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-47.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="inline-formula" id="inline-formula-48">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-48.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-48.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                which are calculated in Eq. A.2 and A.2.
                               </p>
                              </li>
                             </ul>
                             <div id="statement-2" class="statement">
                              <span class="statement-label">
                               Algorithm 2
                              </span>
                              <h3>
                               CopyMix: a Variational Inference Algorithm
                              </h3>
                              <div id="F17" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F17.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1129313690" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F17.medium.gif" width="440" height="239"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F17.medium.gif" width="440" height="239"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure17" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2023/01/03/2020.01.29.926022/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <p id="p-128">
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-32">
                             <h3>
                              A.4 Step 2 of
                              <a id="xref-sec-8-1" class="xref-sec" href="#sec-8">
                               Section 2.2
                              </a>
                             </h3>
                             <p id="p-129">
                              The logarithm of the joint distribution of
                              <strong>
                               Y
                              </strong>
                              ,
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              and
                              <strong>
                               Ψ
                              </strong>
                              satisfies
                              <span class="disp-formula" id="disp-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="disp-formula" id="disp-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-60.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-60.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-61.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-61.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-130">
                              Moreover,
                              <span class="disp-formula" id="disp-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-62.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-62.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where, if B is the multivariate Beta function, i.e.,
                              <span class="disp-formula" id="disp-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-63.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-63.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-64.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-64.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-65.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-65.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="disp-formula" id="disp-formula-48">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-66.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-66.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div class="section" id="sec-33">
                             <h3>
                              A.5 Step 3 and 4 of
                              <a id="xref-sec-8-2" class="xref-sec" href="#sec-8">
                               Section 2.2
                              </a>
                             </h3>
                             <div id="sec-34" class="subsection">
                              <h4>
                               Update equation for π
                              </h4>
                              <p id="p-131">
                               Since
                               <a id="xref-disp-formula-43-1" class="xref-disp-formula" href="#disp-formula-43">
                                (15)
                               </a>
                               and
                               <a id="xref-disp-formula-48-1" class="xref-disp-formula" href="#disp-formula-48">
                                (20)
                               </a>
                               are the only terms in
                               <a id="xref-disp-formula-40-1" class="xref-disp-formula" href="#disp-formula-40">
                                (12)
                               </a>
                               that depend on
                               <strong>
                                π
                               </strong>
                               , the update equation q*(
                               <strong>
                                π
                               </strong>
                               ) can be derived as follows.
                               <span class="disp-formula" id="disp-formula-49">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-67.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-67.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-132">
                               Therefore, q*(
                               <strong>
                                π
                               </strong>
                               ) is a Dirichlet distribution with parameters
                               <span class="inline-formula" id="inline-formula-49">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-49.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-49.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where
                               <span class="disp-formula" id="disp-formula-50">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-68.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-68.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-35" class="subsection">
                              <h4>
                               Update equation for Z
                               <sub>
                                n
                               </sub>
                              </h4>
                              <p id="p-133">
                               Since (
                               <a id="xref-sec-42-2" class="xref-app" href="#sec-42">
                                A.7
                               </a>
                               ) and
                               <a id="xref-disp-formula-43-2" class="xref-disp-formula" href="#disp-formula-43">
                                (15)
                               </a>
                               are the only terms in
                               <a id="xref-disp-formula-40-2" class="xref-disp-formula" href="#disp-formula-40">
                                (12)
                               </a>
                               that depend on Z
                               <sub>
                                n
                               </sub>
                               , q*(Z
                               <sub>
                                n
                               </sub>
                               ) can be obtained as follows.
                               <span class="disp-formula" id="disp-formula-51">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-69.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-69.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-134">
                               Note that log P(
                               <strong>
                                Y
                               </strong>
                               |
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) and log P(
                               <strong>
                                Z
                               </strong>
                               |
                               <strong>
                                Ψ
                               </strong>
                               ) can be written as the sum of two terms, one that depends on Z
                               <sub>
                                n
                               </sub>
                               and one that does not, i.e.,
                               <span class="disp-formula" id="disp-formula-52">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-70.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-70.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="disp-formula" id="disp-formula-53">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-71.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-71.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-135">
                               Consequently,
                               <span class="disp-formula" id="disp-formula-54">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-72.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-72.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-136">
                               Where:
                               <span class="disp-formula" id="disp-formula-55">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-73.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-73.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-137">
                               We conclude that
                               <span class="inline-formula" id="inline-formula-50">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-50.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-50.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with parameters
                               <span class="inline-formula" id="inline-formula-51">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-51.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-51.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="disp-formula" id="disp-formula-56">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-74.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-74.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="disp-formula" id="disp-formula-57">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-75.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-75.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-36" class="subsection">
                              <h4>
                               Update equation for μ
                              </h4>
                              <p id="p-138">
                               Since (
                               <a id="xref-sec-42-3" class="xref-app" href="#sec-42">
                                A.7
                               </a>
                               ) and
                               <a id="xref-disp-formula-45-1" class="xref-disp-formula" href="#disp-formula-45">
                                (17)
                               </a>
                               are the only terms in
                               <a id="xref-disp-formula-40-3" class="xref-disp-formula" href="#disp-formula-40">
                                (12)
                               </a>
                               that depend on μ
                               <sub>
                                n
                               </sub>
                               , q*(μ
                               <sub>
                                n
                               </sub>
                               ) can be obtained as follows.
                               <span class="disp-formula" id="disp-formula-58">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-76.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-76.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-139">
                               We can write log P(
                               <strong>
                                Y
                               </strong>
                               |
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) as in Eq. A.7, and E
                               <sub>
                                −μn
                               </sub>
                               (log P(
                               <strong>
                                μ
                               </strong>
                               )) = log P(
                               <strong>
                                μ
                               </strong>
                               ), which is:
                               <span class="disp-formula" id="disp-formula-59">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-77.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-77.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-140">
                               Therefore,
                               <span class="disp-formula" id="disp-formula-60">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-78.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-78.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-141">
                               Thus, q*(μ
                               <sub>
                                n
                               </sub>
                               ) is a Normal distribution with parameters:
                               <span class="disp-formula" id="disp-formula-61">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-79.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-79.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-62">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-80.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-80.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-37" class="subsection">
                              <h4>
                               Update equation for
                               <span class="inline-formula" id="inline-formula-52">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-52.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-52.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </h4>
                              <p id="p-142">
                               Similar to the previous calculations, we do as follows.
                               <span class="disp-formula" id="disp-formula-63">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-81.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-81.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-143">
                               We can write log P(
                               <strong>
                                Y
                               </strong>
                               |
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) and log P(σ) as they are in Eq. A.7,
                               <a id="xref-disp-formula-46-1" class="xref-disp-formula" href="#disp-formula-46">
                                18
                               </a>
                               . Putting them together, we achieve the following.
                               <span class="disp-formula" id="disp-formula-64">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-82.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-82.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-144">
                               Therefore,
                               <span class="inline-formula" id="inline-formula-53">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-53.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-53.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is a Gamma distribution with parameters:
                               <span class="disp-formula" id="disp-formula-65">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-83.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-83.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-66">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-84.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-84.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-38" class="subsection">
                              <h4>
                               Update equation for
                               <span class="inline-formula" id="inline-formula-54">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-54.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-54.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </h4>
                              <p id="p-145">
                               Because
                               <a id="xref-disp-formula-42-1" class="xref-disp-formula" href="#disp-formula-42">
                                (14)
                               </a>
                               and
                               <a id="xref-disp-formula-47-1" class="xref-disp-formula" href="#disp-formula-47">
                                (19)
                               </a>
                               are the only terms in
                               <a id="xref-disp-formula-40-4" class="xref-disp-formula" href="#disp-formula-40">
                                (12)
                               </a>
                               that depend on
                               <span class="inline-formula" id="inline-formula-55">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-55.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-55.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we calculate
                               <span class="inline-formula" id="inline-formula-56">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-56.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-56.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               as follows.
                               <span class="disp-formula" id="disp-formula-67">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-85.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-85.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-146">
                               Disregarding the terms that do not depend on
                               <span class="inline-formula" id="inline-formula-57">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-57.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-57.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               in log P(
                               <strong>
                                C
                               </strong>
                               |
                               <strong>
                                Ψ
                               </strong>
                               ) and log P(
                               <strong>
                                A
                               </strong>
                               ), we obtain:
                               <span class="disp-formula" id="disp-formula-68">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-86.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-86.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-147">
                               Therefore,
                               <span class="inline-formula" id="inline-formula-58">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-58.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-58.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is Dirichlet with parameters
                               <span class="inline-formula" id="inline-formula-59">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-59.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-59.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="disp-formula" id="disp-formula-69">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-87.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-87.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-39" class="subsection">
                              <h4>
                               Update equation for C
                               <sub>
                                k
                               </sub>
                              </h4>
                              <p id="p-148">
                               Since only (
                               <a id="xref-sec-42-4" class="xref-app" href="#sec-42">
                                A.7
                               </a>
                               ) and
                               <a id="xref-disp-formula-42-2" class="xref-disp-formula" href="#disp-formula-42">
                                (14)
                               </a>
                               depend on
                               <strong>
                                C
                               </strong>
                               <sub>
                                k
                               </sub>
                               , log q*(
                               <strong>
                                C
                               </strong>
                               <sub>
                                k
                               </sub>
                               ) can be calculated as follows.
                               <span class="disp-formula" id="disp-formula-70">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-88.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-88.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-149">
                               Similarly we can write log P(
                               <strong>
                                C
                               </strong>
                               |
                               <strong>
                                Ψ
                               </strong>
                               ) as
                               <span class="disp-formula" id="disp-formula-71">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-89.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-89.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-150">
                               Thus, disregarding the terms that do not depend on
                               <strong>
                                C
                               </strong>
                               <sub>
                                k
                               </sub>
                               we obtain:
                               <span class="disp-formula" id="disp-formula-72">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-90.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-90.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <div id="sec-40" class="subsection">
                               <h5>
                                Calculations regarding directed graph
                               </h5>
                               <p id="p-151">
                                We define graph G = {V, E} where we have each vertex as V = {C
                                <sub>
                                 mj
                                </sub>
                                : m ∈ M, j ∈ J} with weight w(C
                                <sub>
                                 mj
                                </sub>
                                ) and each edge as E = {C
                                <sub>
                                 mj
                                </sub>
                                C
                                <sub>
                                 m+1j
                                </sub>
                                : m ∈ M, j ∈ J} with weight w(C
                                <sub>
                                 mj
                                </sub>
                                C
                                <sub>
                                 m+1j
                                </sub>
                                ). We define quantities of forward, ϕ
                                <sub>
                                 mj
                                </sub>
                                , and backward, β
                                <sub>
                                 mj
                                </sub>
                                , similar to HMM as below (the calculations are based on log q*(
                                <strong>
                                 C
                                </strong>
                                <sub>
                                 k
                                </sub>
                                )):
                                <span class="disp-formula" id="disp-formula-73">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-91.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-91.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-152">
                                Instead of using terminologies of transition and emission probabilities we formulate those as weights of the graph; w
                                <sub>
                                 k
                                </sub>
                                (C
                                <sub>
                                 mj
                                </sub>
                                C
                                <sub>
                                 m+1j
                                </sub>
                                ) and w
                                <sub>
                                 k
                                </sub>
                                (C
                                <sub>
                                 mj
                                </sub>
                                ) respectively. The initial transition probability can be define as w
                                <sub>
                                 k
                                </sub>
                                (C
                                <sub>
                                 0
                                </sub>
                                C
                                <sub>
                                 1j
                                </sub>
                                ) assuming the source starts at 1. Notice that k shows the cluster which the graph belongs to. We can calculate forward and backwards using dynamic programming; having those values we can compute the two posterior probabilities of
                                <span class="inline-formula" id="inline-formula-60">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-60.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-60.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="inline-formula" id="inline-formula-61">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-61.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-61.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                which are the expectations of the indicator functions; we then normalized them by summing over all j ∈ J. The graph weights are:
                                <span class="inline-formula" id="inline-formula-62">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-62.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-62.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="inline-formula" id="inline-formula-63">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-63.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-63.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                               </p>
                               <p id="p-153">
                                Note that we skip writing i, j, k in the calculations to make them short and more readable:
                                <span class="disp-formula" id="disp-formula-74">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-92.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-92.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-154">
                                We now calculate the posteriors:
                                <span class="disp-formula" id="disp-formula-75">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-93.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-93.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-41">
                             <h3>
                              A.6 Calculating expectations and ELBO for
                              <a id="xref-sec-8-3" class="xref-sec" href="#sec-8">
                               Section 2.2
                              </a>
                             </h3>
                             <p id="p-155">
                              Let
                              <strong>
                               Ψ
                              </strong>
                              be the digamma function defined as
                              <span class="disp-formula" id="disp-formula-76">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-94.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-94.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which can be easily calculated via numerical approximation. The values of the expectations above taken with respect to the approximated distributions are given as follows.
                              <span class="disp-formula" id="disp-formula-77">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-95.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-95.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-78">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-96.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-96.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-156">
                              Using the results above regarding the expectations, we update the parameters of the approximated distributions iteratively.
                             </p>
                             <p id="p-157">
                              We then conduct many iterations until the convergence of the ELBO in
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Eq. 2
                              </a>
                              which is calculated as below. An assumption in the ELBO calculation is that we ignore all the constants contributing in the ELBO value.
                              <span class="disp-formula" id="disp-formula-79">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-97.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-97.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-158">
                              We calculate the E[log q(
                              <strong>
                               C
                              </strong>
                              <sub>
                               k
                              </sub>
                              )] by decomposing it into initial and transition components. Note that the initial probabilities, ρ
                              <sub>
                               kj
                              </sub>
                              s, are fixed and, therefore, the term corresponding to that cancels out the corresponding term in E[log P(
                              <strong>
                               C
                              </strong>
                              |
                              <strong>
                               Ψ
                              </strong>
                              )]. The remaining term, the transition component, is calculated by the following using
                              <span class="inline-formula" id="inline-formula-64">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-64.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-64.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , as below:
                              <span class="disp-formula" id="disp-formula-80">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-98.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-98.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-159">
                              We calculate the E[log p(Z
                              <sub>
                               n
                              </sub>
                              |Ψ)] as below:
                              <span class="disp-formula" id="disp-formula-81">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-99.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-99.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-160">
                              We calculate the E[log q(Z
                              <sub>
                               n
                              </sub>
                              )] as below. Note that the expectation of q(Z
                              <sub>
                               n
                              </sub>
                              ) w.r.t. q(Z
                              <sub>
                               n
                              </sub>
                              ) is equal to q(Z
                              <sub>
                               n
                              </sub>
                              ). Also, we know that
                              <span class="inline-formula" id="inline-formula-65">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-65.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-65.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                              <span class="disp-formula" id="disp-formula-82">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-100.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-100.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-161">
                              We calculate the E[log P(μ
                              <sub>
                               n
                              </sub>
                              )] as below:
                              <span class="disp-formula" id="disp-formula-83">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-101.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-101.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-162">
                              We calculate the E[log q(μ
                              <sub>
                               n
                              </sub>
                              )] as below:
                              <span class="disp-formula" id="disp-formula-84">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-102.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-102.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-163">
                              We calculate the E[log P(
                              <strong>
                               π
                              </strong>
                              )] as below:
                              <span class="disp-formula" id="disp-formula-85">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-103.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-103.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-164">
                              We calculate the E(log q(
                              <strong>
                               π
                              </strong>
                              )) as below:
                              <span class="disp-formula" id="disp-formula-86">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-104.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-104.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p>
                              We now approximate the term,
                              <span class="inline-formula" id="inline-formula-66">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-66.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-66.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in the above, by using approximations:
                             </p>
                             <ol class="list-ord" id="list-6">
                              <li id="list-item-22">
                               <p id="p-166">
                                Stirling’s: lnΓ(α*) = (α* – 1) ln(α* – 1) – α* + 1
                               </p>
                              </li>
                              <li id="list-item-23">
                               <p id="p-167">
                                digamma-approximation:
                                <span class="inline-formula" id="inline-formula-67">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-67.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-67.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                              <li id="list-item-24">
                               <p id="p-168">
                                Also:
                                <span class="inline-formula" id="inline-formula-68">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-68.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-68.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                             </ol>
                             <p id="p-169">
                              Putting 3 and 2 in the original formula we obtain:
                              <span class="disp-formula" id="disp-formula-87">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-105.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-105.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div class="section" id="sec-42">
                             <h3>
                              A.7 VI for SNV-CopyMix
                             </h3>
                             <p>
                              Let
                              <strong>
                               Ψ
                              </strong>
                              be the set containing all the model parameters, i.e.,
                              <strong>
                               Ψ
                              </strong>
                              = {
                              <strong>
                               A
                              </strong>
                              ,
                              <strong>
                               μ
                              </strong>
                              , σ,
                              <strong>
                               π
                              </strong>
                              , ξ}, where
                             </p>
                             <ul class="list-unord" id="list-7">
                              <li id="list-item-25">
                               <p id="p-171">
                                <strong>
                                 A
                                </strong>
                                = {
                                <strong>
                                 A
                                </strong>
                                <sub>
                                 k
                                </sub>
                                for k = 1, …, K} with
                                <span class="inline-formula" id="inline-formula-69">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-69.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-69.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for i = 1, …, J and j = 1, …, J};
                               </p>
                              </li>
                              <li id="list-item-26">
                               <p id="p-172">
                                <strong>
                                 μ
                                </strong>
                                = {μ
                                <sub>
                                 n
                                </sub>
                                for n = 1, …, N};
                               </p>
                              </li>
                              <li id="list-item-27">
                               <p id="p-173">
                                σ;
                               </p>
                              </li>
                              <li id="list-item-28">
                               <p id="p-174">
                                <strong>
                                 π
                                </strong>
                                = (π
                                <sub>
                                 1
                                </sub>
                                , …, π
                                <sub>
                                 K
                                </sub>
                                );
                               </p>
                              </li>
                              <li id="list-item-29">
                               <p id="p-175">
                                ξ.
                               </p>
                              </li>
                             </ul>
                             <p id="p-176">
                              In order to infer
                              <strong>
                               Ψ
                              </strong>
                              and the hidden states
                              <strong>
                               Z
                              </strong>
                              = (Z
                              <sub>
                               1
                              </sub>
                              , …, Z
                              <sub>
                               n
                              </sub>
                              ),
                              <strong>
                               C
                              </strong>
                              = {
                              <strong>
                               C
                              </strong>
                              <sub>
                               1
                              </sub>
                              , …,
                              <strong>
                               C
                              </strong>
                              <sub>
                               K
                              </sub>
                              }, and
                              <strong>
                               S
                              </strong>
                              = {S
                              <sub>
                               1
                              </sub>
                              , …, S
                              <sub>
                               K
                              </sub>
                              } we apply the Variational Inference (VI) methodology; that is, we derive an algorithm that, for given data, approximates the posterior distribution by finding the Variational Distribution (VD),
                              <span class="disp-formula" id="disp-formula-88">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-106.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-106.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with smallest Kullback-Leibler divergence to the posterior distribution
                              <span class="disp-formula" id="disp-formula-89">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-107.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-107.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which is equivalent to maximizing the evidence lower bound (ELBO) given by
                              <span class="disp-formula" id="disp-formula-90">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-108.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-108.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p>
                              We consider the following prior distributions for the parameters in
                              <strong>
                               Ψ
                              </strong>
                              .
                             </p>
                             <ul class="list-unord" id="list-8">
                              <li id="list-item-30">
                               <p id="p-178">
                                <span class="inline-formula" id="inline-formula-70">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-70.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-70.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                              <li id="list-item-31">
                               <p id="p-179">
                                <span class="inline-formula" id="inline-formula-71">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-71.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-71.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The conjugate prior concerning the mean of Normal distribution is Normal distribution.
                               </p>
                              </li>
                              <li id="list-item-32">
                               <p id="p-180">
                                <span class="inline-formula" id="inline-formula-72">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-72.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-72.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . The conjugate prior concerning the precision of Normal distribution is Gamma distribution.
                               </p>
                              </li>
                              <li id="list-item-33">
                               <p id="p-181">
                                <strong>
                                 π
                                </strong>
                                ~ Dirichlet(
                                <strong>
                                 δ
                                </strong>
                                ).
                               </p>
                              </li>
                              <li id="list-item-34">
                               <p id="p-182">
                                ξ
                                <sub>
                                 k
                                </sub>
                                ~ Beta(γ
                                <sub>
                                 k
                                </sub>
                                , η
                                <sub>
                                 k
                                </sub>
                                ).
                               </p>
                              </li>
                             </ul>
                             <p id="p-183">
                              In what follows we describe the main steps of the VI algorithm for inferring
                              <strong>
                               Z
                              </strong>
                              ,
                              <strong>
                               C
                              </strong>
                              ,
                              <strong>
                               S
                              </strong>
                              and
                              <strong>
                               Ψ
                              </strong>
                              .
                             </p>
                             <div id="sec-43" class="subsection">
                              <h4>
                               Step 1. VD factorization
                              </h4>
                              <p id="p-184">
                               We assume the following factorization of the VD:
                               <span class="disp-formula" id="disp-formula-91">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-109.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-109.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-44" class="subsection">
                              <h4>
                               Step 2. Joint distribution of observed data, hidden variables, and parameters
                              </h4>
                              <p id="p-185">
                               The logarithm of the joint distribution of
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               and
                               <strong>
                                Ψ
                               </strong>
                               satisfies
                               <span class="disp-formula" id="disp-formula-92">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-110.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-110.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="disp-formula" id="disp-formula-93">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-111.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-111.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-186">
                               We assume we have these probabilities given the genotype g being either mutation or non mutation.
                               <span class="disp-formula" id="disp-formula-94">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-112.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-112.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-95">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-113.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-113.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-187">
                               Note that ρ
                               <sub>
                                kj
                               </sub>
                               is the initial probability in the MC which we fix.
                               <span class="disp-formula" id="disp-formula-96">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-114.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-114.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="disp-formula" id="disp-formula-97">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-115.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-115.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-188">
                               Moreover,
                               <span class="disp-formula" id="disp-formula-98">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-116.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-116.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-189">
                               Defining,
                               <span class="inline-formula" id="inline-formula-73">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-73.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-73.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               the multivariate Beta function, i.e.,
                               <span class="inline-formula" id="inline-formula-74">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-74.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-74.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and
                               <span class="inline-formula" id="inline-formula-75">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-75.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-75.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we have:
                               <span class="disp-formula" id="disp-formula-99">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-117.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-117.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-100">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-118.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-118.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-101">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-119.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-119.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-102">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-120.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-120.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="disp-formula" id="disp-formula-103">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-121.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-121.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-45" class="subsection">
                              <h4>
                               Step 3. VD computation by coordinate ascent
                              </h4>
                              <p id="p-190">
                               We now derive a coordinate ascent algorithm for the VD. That is, we derive an update equation for each term in the factorization (in step 1) by calculating the expectation of log P(
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                X
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                S
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) over the VD of all random variables except the one of interest. Below, the update equation is derived for each random variable. For convenience, we use +≈ to denote equality up to a constant additive factor. Note that
                               <strong>
                                X
                               </strong>
                               can have missing values at certain sites. Therefore, including this into the model, results in splitting
                               <strong>
                                X
                               </strong>
                               into
                               <strong>
                                X
                                <sub>
                                 miss
                                </sub>
                               </strong>
                               and
                               <strong>
                                X
                                <sub>
                                 obs
                                </sub>
                               </strong>
                               . Moreover, we would need a Bernoulli variable, ω, which shows which of the two group is the case in the joint distribution. The joint distribution then takes the following form.
                               <span class="disp-formula" id="disp-formula-104">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-122.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-122.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-191">
                               In the above, MCAR is referring to missing completely at random which is our assumption here, hence its probability is independent from all variational parameters. This means that this probability is a constant w.r.t. the variational parameters. We assume that P(
                               <strong>
                                X
                                <sub>
                                 miss
                                </sub>
                               </strong>
                               |ω,
                               <strong>
                                S
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ) is constant and, therefore, consider X = X
                               <sub>
                                o
                               </sub>
                               bs in our applications. However, for the ease of notation, in our derivations we assume X is complete. Namely, we can, for the sake of readability, simplify the joint distribution as P(
                               <strong>
                                Y
                               </strong>
                               ,
                               <strong>
                                X
                               </strong>
                               ,
                               <strong>
                                S
                               </strong>
                               ,
                               <strong>
                                Z
                               </strong>
                               ,
                               <strong>
                                C
                               </strong>
                               ,
                               <strong>
                                Ψ
                               </strong>
                               ), where
                               <strong>
                                X
                               </strong>
                               refers to
                               <strong>
                                X
                                <sub>
                                 obs
                                </sub>
                               </strong>
                               .
                              </p>
                              <div id="sec-46" class="subsection">
                               <h5>
                                Update equation for π
                               </h5>
                               <p id="p-192">
                                The update equation q(
                                <strong>
                                 π
                                </strong>
                                ) can be derived as follows.
                                <span class="disp-formula" id="disp-formula-105">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-123.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-123.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-193">
                                Therefore, q(
                                <strong>
                                 π
                                </strong>
                                ) is a Dirichlet distribution with parameters
                                <span class="inline-formula" id="inline-formula-76">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-76.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-76.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , where
                                <span class="disp-formula" id="disp-formula-106">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-124.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-124.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </div>
                              <div id="sec-47" class="subsection">
                               <h5>
                                Update equation for Z
                                <sub>
                                 n
                                </sub>
                               </h5>
                               <p id="p-194">
                                The update equation q(Z
                                <sub>
                                 n
                                </sub>
                                ) can be obtained as follows.
                                <span class="disp-formula" id="disp-formula-107">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-125.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-125.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-195">
                                Note that, each of the logarithms above can be written as the sum of two terms, one that depends on Z
                                <sub>
                                 n
                                </sub>
                                and one that does not; since we want to form a function of Z
                                <sub>
                                 n
                                </sub>
                                , we discard all other terms w.r.t. Z
                                <sub>
                                 c
                                </sub>
                                : c ∈ [N] \ n.
                                <span class="disp-formula" id="disp-formula-108">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-126.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-126.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-196">
                                Taking the expectation, we have:
                                <span class="disp-formula" id="disp-formula-109">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-127.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-127.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-197">
                                Combining the above parts from log P(
                                <strong>
                                 Y
                                </strong>
                                |
                                <strong>
                                 Z
                                </strong>
                                ,
                                <strong>
                                 C
                                </strong>
                                ,
                                <strong>
                                 Ψ
                                </strong>
                                ) and log P(
                                <strong>
                                 Z
                                </strong>
                                |
                                <strong>
                                 Ψ
                                </strong>
                                ), we obtain:
                                <span class="disp-formula" id="disp-formula-110">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-128.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-128.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-198">
                                Where:
                                <span class="disp-formula" id="disp-formula-111">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-129.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-129.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-199">
                                Reformulating log q(Z
                                <sub>
                                 n
                                </sub>
                                ), we achieve the below. Note that when processing the input data, for the missing values we merely calculate Q
                                <sub>
                                 Y
                                </sub>
                                .
                                <span class="disp-formula" id="disp-formula-112">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-130.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-130.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-200">
                                We conclude that q(Z
                                <sub>
                                 n
                                </sub>
                                ) ~ Categorical(
                                <strong>
                                 p
                                </strong>
                                <sub>
                                 n
                                </sub>
                                ) with parameters
                                <strong>
                                 p
                                </strong>
                                <sub>
                                 n
                                </sub>
                                = (p
                                <sub>
                                 n1
                                </sub>
                                , …, p
                                <sub>
                                 nK
                                </sub>
                                ), where
                                <span class="disp-formula" id="disp-formula-113">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-131.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-131.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="disp-formula" id="disp-formula-114">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-132.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-132.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </div>
                              <div id="sec-48" class="subsection">
                               <h5>
                                Update equation for ξ
                                <sub>
                                 k
                                </sub>
                               </h5>
                               <p id="p-201">
                                We calculate q(ξ
                                <sub>
                                 k
                                </sub>
                                ) as follows.
                                <span class="disp-formula" id="disp-formula-115">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-133.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-133.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-202">
                                Disregarding the terms that do not depend on ξ
                                <sub>
                                 k
                                </sub>
                                in log P(
                                <strong>
                                 S
                                </strong>
                                |
                                <strong>
                                 Ψ
                                </strong>
                                ) and log P(
                                <strong>
                                 ξ
                                </strong>
                                ), we obtain:
                                <span class="disp-formula" id="disp-formula-116">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-134.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-134.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-203">
                                Therefore, q(ξ
                                <sub>
                                 k
                                </sub>
                                ) is distributed by Beta with parameters:
                                <span class="disp-formula" id="disp-formula-117">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-135.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-135.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                <span class="disp-formula" id="disp-formula-118">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-136.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-136.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </div>
                              <div id="sec-49" class="subsection">
                               <h5>
                                Update equation for S
                                <sub>
                                 kl
                                </sub>
                               </h5>
                               <p id="p-204">
                                We calculate q(S
                                <sub>
                                 kl
                                </sub>
                                ) as follows.
                                <span class="disp-formula" id="disp-formula-119">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-137.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-137.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-205">
                                Disregarding the terms that do not depend on S
                                <sub>
                                 kl
                                </sub>
                                in the above, we obtain:
                                <span class="disp-formula" id="disp-formula-120">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-138.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-138.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                where
                                <span class="inline-formula" id="inline-formula-77">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-77.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-77.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , and
                                <span class="inline-formula" id="inline-formula-78">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-78.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/inline-graphic-78.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                               </p>
                               <p id="p-206">
                                Therefore, q(S
                                <sub>
                                 kl
                                </sub>
                                ) is distributed by Bernoulli up to an additive constant. The Bernoulli parameter, corresponding to S
                                <sub>
                                 kl
                                </sub>
                                = 1 is as the following.
                                <span class="disp-formula" id="disp-formula-121">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-139.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-139.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-207">
                                The rest of update equations are the same as those in CopyMix.
                               </p>
                               <p id="p-208">
                                The new expectations that are used in the above update equations are the following.
                                <span class="disp-formula" id="disp-formula-122">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-140.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2023/01/03/2020.01.29.926022/embed/graphic-140.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-50">
                             <h3>
                              A.8 Simulation procedure
                             </h3>
                             <p>
                              Here, we provide the details of the simulations. For each simulation, the following steps are performed.
                             </p>
                             <ol class="list-ord" id="list-9">
                              <li id="list-item-35">
                               <p id="p-210">
                                Set the random seed.
                               </p>
                              </li>
                              <li id="list-item-36">
                               <p id="p-211">
                                Generate cells belonging to clusters by sampling from a multinomial distribution given a vector of clusterassignment probabilities.
                               </p>
                              </li>
                              <li id="list-item-37">
                               <p id="p-212">
                                Generate rates for all cells. Rates are sampled from a Gaussian distribution with a mean of 10 and a standard deviation of 1.
                               </p>
                              </li>
                              <li id="list-item-38">
                               <p id="p-213">
                                Distribute the rates among the cells for different clusters; this is done based on step 2.
                               </p>
                              </li>
                              <li id="list-item-39">
                               <p id="p-214">
                                Set values to the number of HMM’s hidden states, transition matrix of each cluster, and sequence length.
                               </p>
                              </li>
                              <li id="list-item-40">
                               <p id="p-215">
                                Generate a Gaussian HMM for each cluster using cell rates of that cluster, number of hidden states, transition matrix of the cluster and sequence length; this results in one copy number hidden sequence and different cell ratios emitted from that copy number sequence (the copy number is multiplied by the rate, representing the emission’s mean).
                               </p>
                              </li>
                              <li id="list-item-41">
                               <p id="p-216">
                                Accumulate the cells from the previous step, and insert into a dataset. Similarly accumulate their cluster labels, and then accumulate the hidden copy number sequences.
                               </p>
                              </li>
                             </ol>
                            </div>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               The SNV-based model and the comparison to Ginkgo are added.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="https://github.com/negar7918/CopyMix">
                                https://github.com/negar7918/CopyMix
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baslan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kendall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodgers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cox
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Riggs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stepansky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 , ., ..,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hicks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Genome-wide copy number analysis of single cells
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Protocols
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1024
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10241
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BProtocols%26rft.volume%253D7%26rft.spage%253D1024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.01.29.926022v5.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bishop
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ), “
                                 <span class="cit-chapter-title">
                                  Pattern recognition and machine learning
                                 </span>
                                 ,”
                                 <span class="cit-source">
                                  Information science and statistics
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  New York, NY
                                 </span>
                                 :
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kucukelbir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mcauliffe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Variational Inference: A Review for Statisticians
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the American Statistical Association
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  859
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  877
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BStatistical%2BAssociation%26rft.volume%253D112%26rft.spage%253D859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blocker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. W.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.-L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  The potential and perils of preprocessing: Building new foundations
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Bernoulli
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1176
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1211
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBernoulli%26rft.volume%253D19%26rft.spage%253D1176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Calinski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harabasz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1974
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  A dendrite method for cluster analysis
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Communications in Statistics
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  27
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Souza
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andronescu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Masud
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kabeer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Biele
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brimhall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Su
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Epiclomal: probabilistic clustering of sparse single-cell DNA methylation data
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.7" data-doi="10.1038/nature13952">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eirew
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steif
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Khattra
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ha
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yap
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Farahani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gelmon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Biele
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shumansky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McPherson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nielsen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lefebvre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bashashati
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Souza
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Siu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aniba
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Dynamics of genomic clones in breast cancer patient xenografts at single cell resolution
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  518
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  422
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D518%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13952%26rft_id%253Dinfo%253Apmid%252F25470049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature13952&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25470049&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2023%2F01%2F03%2F2020.01.29.926022.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.8" data-doi="10.1101/gr.8.3.175">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ewing
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hillier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wendl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Green
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Base-calling of automated sequencer traces using phred. I. Accuracy assessment
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  175
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  185
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DEwing%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D185%26rft.atitle%253DBase-Calling%2Bof%2BAutomated%2BSequencer%2BTraces%2BUsing%2BPhred.%2B%2BI.%2BAccuracy%2BAssessment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.175%26rft_id%253Dinfo%253Apmid%252F9521921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvMy8xNzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMy8wMS8wMy8yMDIwLjAxLjI5LjkyNjAyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Henderson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. C.
                                  </span>
                                 </span>
                                 , …, and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnology
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garvin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aboukhalil
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kendall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. e. a.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Interactive analysis and assessment of single-cell copy-number variations
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  1058
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11060
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.11" data-doi="10.1038/nrg.2015.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gawad
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quake
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Single-cell genome sequencing: current state of the science
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  175
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D17%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2015.16%26rft_id%253Dinfo%253Apmid%252F26806412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg.2015.16&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26806412&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2023%2F01%2F03%2F2020.01.29.926022.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.01.29.926022v5.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wichern
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ),
                                 <span class="cit-source">
                                  Applied multivariate statistical analysis
                                 </span>
                                 ,
                                 <span class="cit-edition">
                                  6th
                                 </span>
                                 Ed.,
                                 <span class="cit-publ-name">
                                  Pearson Prentice Hall
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jordan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghahramani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jaakkola
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saul
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. K.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  An introduction to variational methods for graphical models
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Machine learning
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  183
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  233
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMachine%2Blearning%26rft.volume%253D37%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kapourani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.-A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanguinetti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Melissa: Bayesian clustering and imputation of single-cell methylomes
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  61
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D20%26rft.spage%253D61%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.01.29.926022v5.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koptagel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lagergren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  SCuPhr: A Probabilistic Framework for Cell Lineage Tree Reconstruction
                                 </span>
                                 ,”
                                 <a href="https://www.biorxiv.org/content/10.1101/357442v1">
                                  https://www.biorxiv.org/content/10.1101/357442v1
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McPherson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. e. a.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Clonal Decomposition and DNA Replication States Defined by Scaled Single-Cell Genome Sequencing
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.17" data-doi="10.1038/s41556-018-0236-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lawson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kessenbrock
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pervolarakis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Werb
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Tumour heterogeneity and metastasis at single-cell resolution
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Cell Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1349
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1360
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.volume%253D20%26rft.spage%253D1349%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41556-018-0236-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41556-018-0236-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.18" data-doi="10.1101/gr.209973.116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Casasent
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , ., .., and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Single-cell DNA sequencing reveals a late-dissemination model in metastatic colorectal cancer
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1287
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1299
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.209973.116%26rft_id%253Dinfo%253Apmid%252F28546418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTI4NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIzLzAxLzAzLzIwMjAuMDEuMjkuOTI2MDIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MacKay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Ensemble Learning for Hidden Markov Models
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Tech. rep
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mallory
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edrisi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nakhleh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Methods for copy number aberration detection from single-cell DNA-sequencing data
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.21" data-doi="10.1111/j.1467-842X.2009.00543.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGrory
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Titterington
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Variational Bayesian analysis for hidden Markov models
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Australian & New Zealand Journal of Statistics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  51
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  227
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  244
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAustralian%2B%2526%2BNew%2BZealand%2BJournal%2Bof%2BStatistics%26rft.volume%253D51%26rft.spage%253D227%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-842X.2009.00543.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1467-842X.2009.00543.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.22" data-doi="10.1038/nature09807">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kendall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Troge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrews
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodgers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McIndoo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cook
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stepansky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Esposito
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Tumour evolution inferred by single-cell sequencing
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  472
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  90
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNavin%26rft.auinit1%253DN.%26rft.volume%253D472%26rft.issue%253D7341%26rft.spage%253D90%26rft.epage%253D94%26rft.atitle%253DTumour%2Bevolution%2Binferred%2Bby%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09807%26rft_id%253Dinfo%253Apmid%252F21399628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature09807&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21399628&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2023%2F01%2F03%2F2020.01.29.926022.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000289199400043&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.23" data-doi="10.1126/science.959840">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nowell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1976
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  The clonal evolution of tumor cell populations
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  194
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  23
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  28
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNowell%26rft.auinit1%253DP.%26rft.volume%253D194%26rft.issue%253D4260%26rft.spage%253D23%26rft.epage%253D28%26rft.atitle%253DThe%2Bclonal%2Bevolution%2Bof%2Btumor%2Bcell%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.959840%26rft_id%253Dinfo%253Apmid%252F959840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIxOTQvNDI2MC8yMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIzLzAxLzAzLzIwMjAuMDEuMjkuOTI2MDIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olshen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Venkatraman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lucito
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wigler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Circular binary segmentation for the analysis of array-based dna copy number data
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Biostatistics
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hirschberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  V-measure: A conditional entropy-based external cluster evaluation measure
                                 </span>
                                 ,” in
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the 2007 joint conference on empirical methods in natural language processing and computational natural language learning (EMNLP-CoNLL)
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McPherson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Biele
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yap
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nielsen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McAlpine
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aparicio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Clonal genotype and population structure inference from single-cell tumor sequencing
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  573
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D13%26rft.spage%253D573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.27" data-doi="10.1016/0377-0427(87)90125-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rousseeuw
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1987
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Silhouettes: A Graphical Aid to the Interpretation and Validation of Cluster Analysis
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Comput. Appl. Math
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  53
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  65
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComput.%2BAppl.%2BMath%26rft.volume%253D20%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0377-0427%252887%252990125-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0377-0427(87)90125-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.28" data-doi="10.1093/bioinformatics/btl238">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xuan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   DeLeeuw
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Khojasteh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Murphy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Integrating copy number polymorphisms into array CGH analysis using a robust HMM
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e431
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  e439
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DShah%26rft.auinit1%253DS.%2BP.%26rft.volume%253D22%26rft.issue%253D14%26rft.spage%253De431%26rft.epage%253De439%26rft.atitle%253DIntegrating%2Bcopy%2Bnumber%2Bpolymorphisms%2Binto%2Barray%2BCGH%2Banalysis%2Busing%2Ba%2Brobust%2BHMM%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl238%26rft_id%253Dinfo%253Apmid%252F16873504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl238&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16873504&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2023%2F01%2F03%2F2020.01.29.926022.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000250005000053&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.29" data-doi="10.1038/nrg3542">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shapiro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Biezuner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Linnarsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Single-cell sequencing-based technologies will revolutionize whole-organism science
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  618
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D14%26rft.spage%253D618%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3542%26rft_id%253Dinfo%253Apmid%252F23897237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg3542&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23897237&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2023%2F01%2F03%2F2020.01.29.926022.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.30" data-doi="10.1111/1467-9868.00353">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spiegelhalter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Best
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carlin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Der Linde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Bayesian measures of model complexity and fit
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the Royal Statistical Society: Series B (Statistical Methodology)
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  64
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  583
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  639
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety%253A%2BSeries%2BB%2B%2528Statistical%2BMethodology%2529%26rft.volume%253D64%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00353%26rft_id%253Dinfo%253Apmid%252F617541238146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/1467-9868.00353&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=617541238146&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2023%2F01%2F03%2F2020.01.29.926022.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000179221100001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vitak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Torkenczy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenkrantz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fields
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Christiansen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , …, and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Adey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Sequencing thousands of single-cell genomes with combinatorial indexing
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  14
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.32" data-doi="10.1214/11-AOAS493">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Witten
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Classification and clustering of sequencing data using a Poisson model
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Ann. Appl. Stat
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2493
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2518
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat%26rft.volume%253D5%26rft.spage%253D2493%26rft_id%253Dinfo%253Adoi%252F10.1214%252F11-AOAS493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1214/11-AOAS493&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zafar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nakhleh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Monovar: single-nucleotide variant detection in single cells
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steif
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eirew
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   VanInsberghe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aparicio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hansen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Scalable whole-genome single-cell library preparation without preamplification
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  167
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D14%26rft.spage%253D167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Campbell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  Computational modelling in single-cell cancer genomics: methods and future directions
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Phys. Biol
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.29.926022v5.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zuo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hewitt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bresnick
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keleş
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), “
                                 <span class="cit-article-title">
                                  A hierarchical framework for state-space matrix inference and clustering
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Ann. Appl. Stat
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1348
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1372
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat%26rft.volume%253D10%26rft.spage%253D1348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.29.926022v5" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.03.11.483910v2" title="Blocking abundant RNA transcripts by high-affinity oligonucleotides during transcriptome library preparation" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2023.01.02.522257v2" title="HSV-1 Oncolytic Virus Targeting CEACAM6-Expressing Tumors Using a Bispecific T-Cell Engager" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted January 03, 2023.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.01.29.926022v5.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.01.29.926022v5.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2023/01/03/2020.01.29.926022.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.01.29.926022v5.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.01.29.926022v5" class="active" data-icon-position="" data-hide-link-title="0">
                                   CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2927000"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-gobs1N14JcTakuMIYpg0QLOIyc_zsZ2J-d5Ft6bfToE"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484466794"/>
                                   <input type="hidden" name="captcha_token" value="b5f8bce477cbf3a5f317ec0f60a13c18"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2927000--3" data-node-nid="2927000" id="node-2927000--41926923025" data-pisa="biorxiv;2020.01.29.926022v5" data-pisa-master="biorxiv;2020.01.29.926022" data-seqnum="2927000" data-apath="/biorxiv/early/2023/01/03/2020.01.29.926022.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Negar
                                  </span>
                                  <span class="nlm-surname">
                                   Safinianaini
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Camila P. E.
                                  </span>
                                  <span class="nlm-surname">
                                   de Souza
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Andrew
                                  </span>
                                  <span class="nlm-surname">
                                   Roth
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Hazal
                                  </span>
                                  <span class="nlm-surname">
                                   Koptagel
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Hosein
                                  </span>
                                  <span class="nlm-surname">
                                   Toosi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Jens
                                  </span>
                                  <span class="nlm-surname">
                                   Lagergren
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.01.29.926022;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.01.29.926022
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.29.926022v5" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&title=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&title=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&text=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&t=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&title=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&title=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&title=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2927000--5" data-node-nid="2927000" id="citation-node-2927000--62080287465" data-pisa="biorxiv;2020.01.29.926022v5" data-pisa-master="biorxiv;2020.01.29.926022" data-seqnum="2927000" data-apath="/biorxiv/early/2023/01/03/2020.01.29.926022.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     CopyMix: Mixture Model Based Single-Cell Clustering and Copy Number Profiling using Variational Inference
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Negar
                                      </span>
                                      <span class="nlm-surname">
                                       Safinianaini
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Camila P. E.
                                      </span>
                                      <span class="nlm-surname">
                                       de Souza
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Andrew
                                      </span>
                                      <span class="nlm-surname">
                                       Roth
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Hazal
                                      </span>
                                      <span class="nlm-surname">
                                       Koptagel
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Hosein
                                      </span>
                                      <span class="nlm-surname">
                                       Toosi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Jens
                                      </span>
                                      <span class="nlm-surname">
                                       Lagergren
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.01.29.926022;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.01.29.926022
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference&rft.volume&rft.issue&rft.date=2023-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Safinianaini%2C+Negar&rft.au=de+Souza%2C+Camila+P.+E.&rft.au=Roth%2C+Andrew&rft.au=Koptagel%2C+Hazal&rft.au=Toosi%2C+Hosein&rft.au=Lagergren%2C+Jens">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2927000/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2927000/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2927000/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2927000/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2927000/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2927000/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2927000/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2927000/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2927000/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2927000/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2927000/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2927000/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&count=horizontal&via=&text=CopyMix%3A%20Mixture%20Model%20Based%20Single-Cell%20Clustering%20and%20Copy%20Number%20Profiling%20using%20Variational%20Inference&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.29.926022v5&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.01.29.926022v5" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
