<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2021.06.10.21258550v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2021.06.10.21258550v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2021.06.10.21258550v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/medrxiv/early/2022/02/14/2021.06.10.21258550.atom"/>
  <meta name="HW.pisa" content="medrxiv;2021.06.10.21258550v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2"/>
  <meta name="DC.Identifier" content="10.1101/2021.06.10.21258550"/>
  <meta name="DC.Date" content="2022-02-14"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The COVID-19 pandemic has initiated an unprecedented worldwide effort to characterize its evolution through the mapping of mutations of the coronavirus SARS-CoV-2. The early identification of mutations that could confer adaptive advantages to the virus, such as higher infectivity or immune evasion, is of paramount importance. However, the large number of currently available genomes precludes the efficient use of phylogeny-based methods. Here we establish a fast and scalable early warning system based on Topological Data Analysis for the identification and surveillance of emerging adaptive mutations in large genomic datasets. Analyzing millions of SARS-CoV-2 genomes from GISAID, we demonstrate that topologically salient mutations are linked with an increase in infectivity or immune escape. We report on emerging potentially adaptive mutations as of January 2022, and pinpoint mutations in Variants of Concern that are likely due to convergent evolution. Our approach can improve the surveillance of mutations of concern, guide experimental studies, and aid vaccine development.

### Competing Interest Statement

Raul Rabadan is a founder of Genotwin, he is member of the Scientific Advisory Board of AimedBio and consults for Arquimea Research.

### Funding Statement

The authors acknowledge the use of de.NBI Cloud and the support by the High Performance and Cloud Computing Group at the Zentrum für Datenverarbeitung of the University of Tübingen and the Federal Ministry of Education and Research (BMBF) through grant no 031 A535A. This research was supported by the DFG Collaborative Research Center SFB/TRR 109 ''Discretization in Geometry and Dynamics''. M.B. was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy EXC 2181/1 - 390900948 (the Heidelberg STRUCTURES Excellence Cluster). L.H. thanks the Evangelisches Studienwerk Villigst for their support. A.O. acknowledges funding by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) -- 281869850 (RTG 2229).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

No specific IRB protocol as data can be found in public repositories (GISAID).

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

SARS-CoV-2 genome data used in this work are available from the GISAID EpiCov Database at www.gisaid.org.

<https://www.gisaid.org/>"/>
  <meta name="DC.Contributor" content="Michael Bleher"/>
  <meta name="DC.Contributor" content="Lukas Hahn"/>
  <meta name="DC.Contributor" content="Juan Ángel Patiño-Galindo"/>
  <meta name="DC.Contributor" content="Mathieu Carrière"/>
  <meta name="DC.Contributor" content="Ulrich Bauer"/>
  <meta name="DC.Contributor" content="Raúl Rabadán"/>
  <meta name="DC.Contributor" content="Andreas Ott"/>
  <meta name="article:published_time" content="2022-02-14"/>
  <meta name="citation_title" content="Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>The COVID-19 pandemic has initiated an unprecedented worldwide effort to characterize its evolution through the mapping of mutations of the coronavirus SARS-CoV-2. The early identification of mutations that could confer adaptive advantages to the virus, such as higher infectivity or immune evasion, is of paramount importance. However, the large number of currently available genomes precludes the efficient use of phylogeny-based methods. Here we establish a fast and scalable early warning system based on Topological Data Analysis for the identification and surveillance of emerging adaptive mutations in large genomic datasets. Analyzing millions of SARS-CoV-2 genomes from GISAID, we demonstrate that topologically salient mutations are linked with an increase in infectivity or immune escape. We report on emerging potentially adaptive mutations as of January 2022, and pinpoint mutations in Variants of Concern that are likely due to convergent evolution. Our approach can improve the surveillance of mutations of concern, guide experimental studies, and aid vaccine development.</p>"/>
  <meta name="citation_journal_title" content="medRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="medrxiv;2021.06.10.21258550v2"/>
  <meta name="citation_id" content="2021.06.10.21258550v2"/>
  <meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2021.06.10.21258550v2"/>
  <meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2021.06.10.21258550v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2021.06.10.21258550v2.full"/>
  <meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2021.06.10.21258550"/>
  <meta name="citation_num_pages" content="35"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_firstpage" content="2021.06.10.21258550"/>
  <meta name="citation_author" content="Michael Bleher"/>
  <meta name="citation_author_institution" content="Mathematical Institute, Heidelberg University"/>
  <meta name="citation_author_email" content="mbleher@mathi.uni-heidelberg.de"/>
  <meta name="citation_author_email" content="lhahn@mathi.uni-heidelberg.de"/>
  <meta name="citation_author_email" content="andreas.ott@kit.edu"/>
  <meta name="citation_author" content="Lukas Hahn"/>
  <meta name="citation_author_institution" content="Mathematical Institute, Heidelberg University"/>
  <meta name="citation_author_email" content="mbleher@mathi.uni-heidelberg.de"/>
  <meta name="citation_author_email" content="lhahn@mathi.uni-heidelberg.de"/>
  <meta name="citation_author_email" content="andreas.ott@kit.edu"/>
  <meta name="citation_author" content="Juan Ángel Patiño-Galindo"/>
  <meta name="citation_author_institution" content="Program for Mathematical Genomics, Department of Systems Biology, Columbia University"/>
  <meta name="citation_author" content="Mathieu Carrière"/>
  <meta name="citation_author_institution" content="DataShape, Inria Sophia-Antipolis"/>
  <meta name="citation_author" content="Ulrich Bauer"/>
  <meta name="citation_author_institution" content="TUM Department of Mathematics and Munich Data Science Institute"/>
  <meta name="citation_author" content="Raúl Rabadán"/>
  <meta name="citation_author_institution" content="Program for Mathematical Genomics, Department of Systems Biology, Columbia University"/>
  <meta name="citation_author" content="Andreas Ott"/>
  <meta name="citation_author_institution" content="Mathematics Department, Karlsruhe Institute of Technology"/>
  <meta name="citation_author_institution" content="Mathematical Institute, Heidelberg University"/>
  <meta name="citation_author_email" content="mbleher@mathi.uni-heidelberg.de"/>
  <meta name="citation_author_email" content="lhahn@mathi.uni-heidelberg.de"/>
  <meta name="citation_author_email" content="andreas.ott@kit.edu"/>
  <meta name="citation_reference" content="Hodcroft, E. B., De Maio, N., Lanfear, R., et al. Want to Track Pandemic Variants Faster? Fix the Bioinformatics Bottleneck. Nature 591, 30–33 (2021). doi:10.1038/d41586-021-00525-x."/>
  <meta name="citation_reference" content="citation_journal_title=PLOS ONE;citation_author=B. Schrörs;citation_author=P. Riesgo-Ferreiro;citation_author=P. Sorn;citation_title=Large-scale analysis of SARS-CoV-2 spike-glycoprotein mutants demonstrates the need for continuous screening of virus isolates;citation_pages=e0249254;citation_volume=16;citation_year=2021;citation_doi=10.1371/journal.pone.0249254"/>
  <meta name="citation_reference" content="Starr, T. N., Greaney, A. J., Hilton, S. K., et al. Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding. Cell 182, 1295–1310.e20 (2020). doi:10.1016/j.cell.2020.08.012."/>
  <meta name="citation_reference" content="Greaney, A. J., Loes, A. N., Crawford, K. H., et al. Comprehensive Mapping of Mutations in the SARS-CoV-2 Receptor-Binding Domain That Affect Recognition by Polyclonal Human Plasma Antibodies. Cell Host & Microbe 29, 463–476.e6 (2021). doi:10.1016/j.chom.2021.02.003."/>
  <meta name="citation_reference" content="Zahradník, J., Marciano, S., Shemesh, M., et al. SARS-CoV-2 variant prediction and antiviral drug design are enabled by RBD in vitro evolution. Nature Microbiology 6, 1188–1198 (2021). doi:10.1038/s41564-021-00954-4."/>
  <meta name="citation_reference" content="Starr, T. N., Greaney, A. J., Addetia, A., et al. Prospective Mapping of Viral Mutations That Escape Antibodies Used to Treat COVID-19. Science 371, 850–854 (2021). doi:10.1126/science.abf9302."/>
  <meta name="citation_reference" content="Sagulenko, P., Puller, V. & Neher, R. A. TreeTime: Maximum-Likelihood Phylodynamic Analysis. Virus Evolution 4 (2018). doi:10.1093/ve/vex042."/>
  <meta name="citation_reference" content="Crispell, J., Balaz, D. & Gordon, S. V. HomoplasyFinder: A Simple Tool to Identify Homoplasies on a Phylogeny. Microbial Genomics 5 (2019). doi:10.1099/mgen.0.000245."/>
  <meta name="citation_reference" content="Van Dorp, L., Richard, D., Tan, C. C. S., et al. No Evidence for Increased Transmissibility from Recurrent Mutations in SARS-CoV-2. Nature Communications 11, 5986 (2020). doi:10.1038/s41467-020-19818-2."/>
  <meta name="citation_reference" content="Van Dorp, L., Acman, M., Richard, D., et al. Emergence of Genomic Diversity and Recurrent Mutations in SARS-CoV-2. Infection, Genetics and Evolution 83, 104351 (2020). doi:10.1016/j.meegid.2020.104351."/>
  <meta name="citation_reference" content="Zahradník, J., Nunvar, J. & Schreiber, G. SARS-CoV-2 Convergent Evolution as a Guide to Explore Adaptive Advantage. bioRxiv (2021). doi:10.1101/2021.05.24.445534."/>
  <meta name="citation_reference" content="Rochman, N. D., Wolf, Y. I., Faure, G., et al. Ongoing Global and Regional Adaptive Evolution of SARS-CoV-2. Proceedings of the National Academy of Sciences 118, e2104241118 (2021). doi:10.1073/pnas.2104241118."/>
  <meta name="citation_reference" content="Korber, B., Fischer, W. M., Gnanakaran, S., et al. Tracking Changes in SARS-CoV-2 Spike: Evidence That D614G Increases Infectivity of the COVID-19 Virus. Cell 182, 812–827.e19 (2020). doi:10.1016/j.cell.2020.06.043."/>
  <meta name="citation_reference" content="Khare, S., Gurry, C., Freitas, L., et al. GISAID’s Role in Pandemic Response. China CDC Weekly 3, 1049–1051 (2021). doi:10.46234/ccdcw2021.255."/>
  <meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_author=B. Morel;citation_author=P. Barbera;citation_author=L. Czech;citation_title=Phylogenetic Analysis of SARS-CoV-2 Data Is Difficult;citation_year=2020;citation_pmid=33316067;citation_doi=10.1093/molbev/msaa314"/>
  <meta name="citation_reference" content="Hadfield, J., Megill, C., Bell, S. M., et al. Nextstrain: Real-Time Tracking of Pathogen Evolution. Bioinformatics 34, 4121–4123 (2018). doi:10.1093/bioinformatics/bty407."/>
  <meta name="citation_reference" content="Turakhia, Y., De Maio, N., Thornlow, B., et al. Stability of SARS-CoV-2 Phylogenies. PLOS Genetics 16, e1009175 (2020). doi:10.1371/journal.pgen.1009175."/>
  <meta name="citation_reference" content="Shu, Y. & McCauley, J. GISAID: Global Initiative on Sharing All Influenza Data – from Vision to Reality. Eurosurveillance 22 (2017). doi:10.2807/1560-7917.ES.2017.22.13.30494."/>
  <meta name="citation_reference" content="Chan, J. M., Carlsson, G. & Rabadan, R. Topology of Viral Evolution. Proceedings of the National Academy of Sciences 110, 18566–18571 (2013). doi:10.1073/pnas.1313480110."/>
  <meta name="citation_reference" content="Bauer, U. Ripser: efficient computation of Vietoris-Rips persistence barcodes. Journal of Applied and Computational Topology (2021). doi:10.1007/s41468-021-00071-5."/>
  <meta name="citation_reference" content="Minh, B. Q., Schmidt, H. A., Chernomor, O., et al. IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era. Molecular Biology and Evolution 37, 1530–1534 (2020). doi:10.1093/molbev/msaa015."/>
  <meta name="citation_reference" content="Keegan, L. & Kempf, D. Hammingdist: A Fast Tool to Calculate Hamming Distances version 0.13.0. 2021. https://github.com/ssciwr/hammingdist visited on 2021-12-01."/>
  <meta name="citation_reference" content="De Maio, N., Walker, C., Borges, R., et al. Issues with SARS-CoV-2 Sequencing Data - SARS-CoV-2 Coronavirus / nCoV-2019 Genomic Epidemiology Virological. https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473."/>
  <meta name="citation_reference" content="Huang, Y., Yang, C., Xu, X.-f., et al. Structural and Functional Properties of SARS-CoV-2 Spike Protein: Potential Antivirus Drug Development for COVID-19. Acta Pharmacologica Sinica 41, 1141–1149 (2020). doi:10.1038/s41401-020-0485-4."/>
  <meta name="citation_reference" content="Johnson, B. A., Xie, X., Bailey, A. L., et al. Loss of furin cleavage site attenuates SARS-CoV-2 pathogenesis. Nature 591, 293–299 (2021). doi:10.1038/s41586-021-03237-4."/>
  <meta name="citation_reference" content="Li, Q., Wu, J., Nie, J., et al. The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity. Cell 182, 1284–1294.e9 (2020). doi:10.1016/j.cell.2020.07.012."/>
  <meta name="citation_reference" content="Plante, J. A., Liu, Y., Liu, J., et al. Spike Mutation D614G Alters SARS-CoV-2 Fitness. Nature 592, 116–121 (2021). doi:10.1038/s41586-020-2895-3."/>
  <meta name="citation_reference" content="Hou, Y. J., Chiba, S., Halfmann, P., et al. SARS-CoV-2 D614G Variant Exhibits Efficient Replication Ex Vivo and Transmission in Vivo. Science, eabe8499 (2020). doi:10.1126/science.abe8499."/>
  <meta name="citation_reference" content="Yurkovetskiy, L., Wang, X., Pascal, K. E., et al. Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant. Cell 183, 739–751.e8 (2020). doi:10.1016/j.cell.2020.09.032."/>
  <meta name="citation_reference" content="Tracking SARS-CoV-2 Variants World Health Organization. https://www.who.int/activities/tracking-SARS-CoV-2-variants visited on 2022-01-25."/>
  <meta name="citation_reference" content="Singh, A., Steinkellner, G., Köchl, K., et al. Serine 477 Plays a Crucial Role in the Interaction of the SARS-CoV-2 Spike Protein with the Human Receptor ACE2. Scientific Reports 11, 4320 (2021). doi:10.1038/s41598-021-83761-5."/>
  <meta name="citation_reference" content="Hodcroft, E. B., Zuber, M., Nadeau, S., et al. Spread of a SARS-CoV-2 Variant through Europe in the Summer of 2020. Nature, 1–9 (2021). doi:10.1038/s41586-021-03677-y."/>
  <meta name="citation_reference" content="Welkers, M. R. A., Han, A. X., Reusken, C. B. E. M. & Eggink, D. Possible Host-Adaptation of SARS-CoV-2 Due to Improved ACE2 Receptor Binding in Mink. Virus Evolution (2020). doi:10.1093/ve/veaa094."/>
  <meta name="citation_reference" content="Oude Munnink, B. B., Sikkema, R. S., Nieuwenhuijse, D. F., et al. Transmission of SARS-CoV-2 on Mink Farms between Humans and Mink and Back to Humans. Science 371, 172–177 (2021). doi:10.1126/science.abe5901."/>
  <meta name="citation_reference" content="Van Dorp, L., Tan, C. C., Lam, S. D., et al. Recurrent Mutations in SARS-CoV-2 Genomes Isolated from Mink Point to Rapid Host-Adaptation. bioRxiv (2020). doi:10.1101/2020.11.16.384743."/>
  <meta name="citation_reference" content="McCallum, M., De Marco, A., Lempp, F. A., et al. N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2. Cell 184, 2332–2347.e16 (2021). doi:10.1016/j.cell.2021.03.028."/>
  <meta name="citation_reference" content="Liu, Z., VanBlargan, L. A., Bloyet, L.-M., et al. Identification of SARS-CoV-2 spike mutations that attenuate monoclonal and serum antibody neutralization. Cell Host & Microbe 29, 477–488.e4 (2021). doi:10.1016/j.chom.2021.01.014."/>
  <meta name="citation_reference" content="Haynes, W. A., Kamath, K., Lucas, C., et al. Impact of B.1.1.7 Variant Mutations on Antibody Recognition of Linear SARS-CoV-2 Epitopes. medRxiv (2021). doi:10.1101/2021.01.06.20248960."/>
  <meta name="citation_reference" content="Lista, M. J., Winstone, H., Wilson, H. D., et al. The P681H mutation in the Spike glycoprotein confers Type I interferon resistance in the SARS-CoV-2 alpha (B.1.1.7) variant. bioRxiv (2021). doi:10.1101/2021.11.09.467693."/>
  <meta name="citation_reference" content="Liu, L., Iketani, S., Guo, Y., et al. Striking Antibody Evasion Manifested by the Omicron Variant of SARS-CoV-2. bioRxiv (2021). doi:10.1101/2021.12.14.472719."/>
  <meta name="citation_reference" content="citation_journal_title=Microbiology Spectrum;citation_author=P. E. Romero;citation_author=A. Dávila-Barclay;citation_author=G. Salvatierra;citation_title=The Emergence of Sars-CoV-2 Variant Lambda (C.37) in South America;citation_pages=00789e-21;citation_volume=9;citation_year=2021;citation_doi=10.1128/Spectrum.00789-21"/>
  <meta name="citation_reference" content="Kimura, I., Kosugi, Y., Wu, J., et al. The SARS-CoV-2 Lambda variant exhibits enhanced infectivity and immune resistance. Cell Reports 38, 110218 (2022). doi:10.1016/j.celrep.2021.110218."/>
  <meta name="citation_reference" content="Viana, R., Moyo, S., Amoako, D. G., et al. Rapid epidemic expansion of the SARS-CoV-2 Omicron variant in southern Africa. Nature (2022). doi:10.1038/s41586-022-04411-y."/>
  <meta name="citation_reference" content="Rambaut, A., Holmes, E. C., O’Toole, Á., et al. A Dynamic Nomenclature Proposal for SARS-CoV-2 Lineages to Assist Genomic Epidemiology. Nature Microbiology 5, 1403–1407 (2020). doi:10.1038/s41564-020-0770-5."/>
  <meta name="citation_reference" content="COVID-19 Weekly Epidemiological Update, Edition 76 World Health Organization. https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---25-january-2022 visited on 2022-01-25."/>
  <meta name="citation_reference" content="Willett, B. J., Grove, J., MacLean, O., et al. The hyper-transmissible SARS-CoV-2 Omicron variant exhibits significant antigenic change, vaccine escape and a switch in cell entry mechanism. medRxiv (2022). doi:10.1101/2022.01.03.21268111."/>
  <meta name="citation_reference" content="Where did ‘weird’ Omicron come from? Science 374, 1179 (2021). doi:10.1126/science.acx9754."/>
  <meta name="citation_reference" content="Callaway, E. Beyond Omicron: what’s next for COVID’s viral evolution. Nature 600, 204–207 (2021). doi:10.1038/d41586-021-03619-8."/>
  <meta name="citation_reference" content="Cohen-Steiner, D., Edelsbrunner, H. & Harer, J. Stability of Persistence Diagrams. Discrete & Computational Geometry 37, 103–120 (2007). doi:10.1007/s00454-006-1276-5."/>
  <meta name="citation_reference" content="Lythgoe, K. A., Hall, M., Ferretti, L., et al. Shared SARS-CoV-2 Diversity Suggests Localised Transmission of Minority Variants. bioRxiv (2020). doi:10.1101/2020.05.28.118992."/>
  <meta name="citation_reference" content="Lythgoe, K. A., Hall, M., Ferretti, L., et al. SARS-CoV-2 within-host diversity and transmission. Science 372, eabg0821 (2021). doi:10.1126/science.abg0821."/>
  <meta name="citation_reference" content="Noh, J. Y., Jeong, H. W. & Shin, E.-C. SARS-CoV-2 Mutations, Vaccines, and Immunity: Implication of Variants of Concern. Signal Transduction and Targeted Therapy 6, 203 (2021). doi:10.1038/s41392-021-00623-2."/>
  <meta name="citation_reference" content="Edgar, R. C. MUSCLE: Multiple Sequence Alignment with High Accuracy and High Throughput. Nucleic Acids Research 32, 1792–1797 (2004). doi:10.1093/nar/gkh340."/>
  <meta name="citation_reference" content="Katoh, K. MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. Nucleic Acids Research 30, 3059–3066 (2002). doi:10.1093/nar/gkf436."/>
  <meta name="citation_reference" content="citation_title=in Essays in Group Theory (ed;citation_author=M. Gromov;citation_pages=75-263;citation_volume=8;citation_year=1987;citation_doi=10.1007/978-1-4613-9586-7_3"/>
  <meta name="citation_reference" content="Edelsbrunner, H. & Ö lsböck, K. Holes and Dependences in an Ordered Complex. Computer Aided Geometric Design 73, 1–15 (2019). doi:10.1016/j.cagd.2019.06.003."/>
  <meta name="citation_reference" content="Zomorodian, A. & Carlsson, G. Computing Persistent Homology. Discrete & Computational Geometry 33, 249–274 (2005). doi:10.1007/s00454-004-1146-y."/>
  <meta name="citation_reference" content="Jariani, A., Warth, C., Deforche, K., et al. SANTA-SIM: Simulating Viral Sequence Evolution Dynamics under Selection and Recombination. Virus Evolution 5 (2019). doi:10.1093/ve/vez003."/>
  <meta name="citation_reference" content="Panjer, H. H. Recursive Evaluation of a Family of Compound Distributions. ASTIN Bulletin 12, 22–26 (1981). doi:10.1017/S0515036100006796."/>
  <meta name="citation_reference" content="Sundt, B. & Jewell, W. S. Further Results on Recursive Evaluation of Compound Distributions. ASTIN Bulletin: The Journal of the IAA 12, 27–39 (1981). doi:10.1017/S0515036100006802."/>
  <meta name="citation_reference" content="Maddison, W. P. & Maddison, D. Mesquite: A Modular System for Evolutionary Analysis.version 3.61. 2019. http://www.mesquiteproject.org visited on 2021-06-01."/>
  <meta name="citation_reference" content="Hanussek, M. VALET 2021. https://github.com/MaximilianHanussek/VALET visited on 2021-06-01."/>
  <meta name="twitter:title" content="Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2"/>
  <meta name="twitter:site" content="@medrxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="twitter:description" content="The COVID-19 pandemic has initiated an unprecedented worldwide effort to characterize its evolution through the mapping of mutations of the coronavirus SARS-CoV-2. The early identification of mutations that could confer adaptive advantages to the virus, such as higher infectivity or immune evasion, is of paramount importance. However, the large number of currently available genomes precludes the efficient use of phylogeny-based methods. Here we establish a fast and scalable early warning system based on Topological Data Analysis for the identification and surveillance of emerging adaptive mutations in large genomic datasets. Analyzing millions of SARS-CoV-2 genomes from GISAID, we demonstrate that topologically salient mutations are linked with an increase in infectivity or immune escape. We report on emerging potentially adaptive mutations as of January 2022, and pinpoint mutations in Variants of Concern that are likely due to convergent evolution. Our approach can improve the surveillance of mutations of concern, guide experimental studies, and aid vaccine development.

### Competing Interest Statement

Raul Rabadan is a founder of Genotwin, he is member of the Scientific Advisory Board of AimedBio and consults for Arquimea Research.

### Funding Statement

The authors acknowledge the use of de.NBI Cloud and the support by the High Performance and Cloud Computing Group at the Zentrum für Datenverarbeitung of the University of Tübingen and the Federal Ministry of Education and Research (BMBF) through grant no 031 A535A. This research was supported by the DFG Collaborative Research Center SFB/TRR 109 ''Discretization in Geometry and Dynamics''. M.B. was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy EXC 2181/1 - 390900948 (the Heidelberg STRUCTURES Excellence Cluster). L.H. thanks the Evangelisches Studienwerk Villigst for their support. A.O. acknowledges funding by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) -- 281869850 (RTG 2229).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

No specific IRB protocol as data can be found in public repositories (GISAID).

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

SARS-CoV-2 genome data used in this work are available from the GISAID EpiCov Database at www.gisaid.org.

<https://www.gisaid.org/>"/>
  <meta name="og-title" property="og:title" content="Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2"/>
  <meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2021.06.10.21258550v2"/>
  <meta name="og-site-name" property="og:site_name" content="medRxiv"/>
  <meta name="og-description" property="og:description" content="The COVID-19 pandemic has initiated an unprecedented worldwide effort to characterize its evolution through the mapping of mutations of the coronavirus SARS-CoV-2. The early identification of mutations that could confer adaptive advantages to the virus, such as higher infectivity or immune evasion, is of paramount importance. However, the large number of currently available genomes precludes the efficient use of phylogeny-based methods. Here we establish a fast and scalable early warning system based on Topological Data Analysis for the identification and surveillance of emerging adaptive mutations in large genomic datasets. Analyzing millions of SARS-CoV-2 genomes from GISAID, we demonstrate that topologically salient mutations are linked with an increase in infectivity or immune escape. We report on emerging potentially adaptive mutations as of January 2022, and pinpoint mutations in Variants of Concern that are likely due to convergent evolution. Our approach can improve the surveillance of mutations of concern, guide experimental studies, and aid vaccine development.

### Competing Interest Statement

Raul Rabadan is a founder of Genotwin, he is member of the Scientific Advisory Board of AimedBio and consults for Arquimea Research.

### Funding Statement

The authors acknowledge the use of de.NBI Cloud and the support by the High Performance and Cloud Computing Group at the Zentrum für Datenverarbeitung of the University of Tübingen and the Federal Ministry of Education and Research (BMBF) through grant no 031 A535A. This research was supported by the DFG Collaborative Research Center SFB/TRR 109 ''Discretization in Geometry and Dynamics''. M.B. was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy EXC 2181/1 - 390900948 (the Heidelberg STRUCTURES Excellence Cluster). L.H. thanks the Evangelisches Studienwerk Villigst for their support. A.O. acknowledges funding by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) -- 281869850 (RTG 2229).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

No specific IRB protocol as data can be found in public repositories (GISAID).

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

SARS-CoV-2 genome data used in this work are available from the GISAID EpiCov Database at www.gisaid.org.

<https://www.gisaid.org/>"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="citation_date" content="2022-02-14"/>
  <meta name="description" content="medRxiv - The Preprint Server for Health Sciences"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2021.06.10.21258550v2"/>
  <link rel="shortlink" href="https://www.medrxiv.org/node/476466"/>
  <title>
   Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2 | medRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__MF4P_i8lKWP7DQR7GmjDAVTRBKLnzxvs1Dvton-Awi0__yi5OflfckG_G8-Li4dDQQ3wO9cGIcdPx_YQ7ZZDuj5w__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__5mxrBK4DuyvC2MR5irx_TzVn9c8vr4vIjmBXjunHfxE__rtQnXv8GOc-AsynbhzSlQBD_2lVxAkB6XEFUCK72qXc__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all"/>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UnV_rmQ8-Lf-opLEz6knyRJc5TincyrORopk6A4qglM"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"476466","apath":"\/medrxiv\/early\/2022\/02\/14\/2021.06.10.21258550.atom","pisa":"medrxiv;2021.06.10.21258550v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"medrxiv;2021.06.10.21258550v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2021.06.10.21258550v2","title":"Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2","identifier":"node\/476466"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002256ozgXOs\\\\\\\/CBg5ooZd\\\\\\\/ggraFsjWec4cEWIHCB7+IR96ex4IaTtvOaHgRO371OXo9xn0DuyZuQszjPezG0eg0+2c+3kXHt3kVZqjnyc0yki5Pajeu1hYWKcjupn1Mb8MVq7OhoTrOW1dPhAnUQl\\\\\\\/85DoY1Td0klpZlWGBDiNeLXQJCqHcIQLuYR0OhuBZofaaXHvzSZilq81WMiPISwdo5PAm4FMNcWHFuomh15Pw2OAskWMpTm+prbxM5PTz1e58EnFyEyPxKq16X1sQVie1MhPCvqQpPoC7rnZPpNl2KrSn9G+oGqqCBVzFmR0XBhFJTPlfYhz8FN7Glsr8GNRHcyWrCCKKQXTtWnIh87nP9YswN6tb6PVMsmFKyS7urQVxKJMacnnYk2RMgYev2Z2E8S9Bbm\\\\\\\/ttVN+X\\\\\\\/KI62zzYbF4xPQi5UXiXvtrWI\\\\\\\/qoI7Y5\\\\\\\/RfAirVcQMjikUWIq8YLqQzxk1EF11xxzrSD85d\\\\\\\/fKyC9TrlWRYeDXC9D7wvZ1kU3ZSd3vn8NVWqftOkTgPRe7uTX0hzPP6qFl8mooWcguu8YMiLmBJVlv+bYxpJ\\\\\\\/fjb0EumCHasP6u2okmZxCsLL1EDtaMeobliSRsF2pyAJxGDzzFUn2WKiMc0PVUzyH0KqLQrJ13Hd73NojrJNPMaQ7eHfdJcX61UrTEc7xufzTeydJ33\\\\\\\/\\\\\\\/xQ1S6ZFiUOnU5\\\\\\\/p7kLO5hhEV3SKUt8kmM62ZhWwMzFTRl0eGCfasUr+C+70U7MftcFynJ1Muu65D6RrByGZ3P3HHUCmWkitcpJ8a6a74l+sWvOCJW47JvrTAcQ2B0CTIK4ihbD97Z92hm8cEZ9m2k+hSGpv3Dw6i44IoXV09dUHtd160kkZfb5f7NWk6wQAs71fF7b3yEDbD7jbRhhJohErczTtrb4hKGAQQpCyMI\\\\\\\/UbgKTr13YOVbQWtGT2u1gW3tlKCqcPIWZtaOgLRALHFWtZ8ihcXak6bcLRPMtJoPhkq4qADYMtzZfA9PbOlV5bb7IO8e6lQDzGRhXtPqyj4awROiPE8a0shuDrlmKdFOCQ3QISvJiC+KcPNRj856dJhzUDU64TEkygTmOUltWEa3Aj7SVODUndYzUC7ArprdyJctP7uiALDKWEeZ\\\\\\\/HcJt\\\\\\\/3ZNBGJYbKpL23p1XFUQlKnIGPjC99\\\\\\\/DF8RMjW0dbp\\\\\\\/JxOv8uWEooe5GgYzsf4rcZUvpemU29IR78zhxKcq2CS0wrQmcJacSkcL+wJ8nRuhz\\\\\\\/sU8yjm88HV1x200zV+ihlLx1oKwxXdsIloXmepUdBJ1ZjU8dP7KVRkVJ3ohOK3EMFGu+6unOr2M0vVARDaaQl7zevO0U5aCtLdhoSqiDxQkM0prrL95NIeUK2Dm1XdhHv+0Qe2xfxssps=\\\u0022,\\\u0022iv\\\u0022:\\\u0022pMbJHCXuNmza3UrFu6n6HA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022dcd2cefc07b8986449604a533aa62fc5f2dcec018276a1ecd431ff62b2feda79\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2021.06.10.21258550v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-476466 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2021.06.10.21258550v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_800270183">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_800270183" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1643003139" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-6izN4vzBBnFo3JGrle_8vmlzlu8in7EJxQQlWfxkH00"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node476466" data-node-nid="476466" id="node-476466--2826764347" data-pisa="medrxiv;2021.06.10.21258550v2" data-pisa-master="medrxiv;2021.06.10.21258550" data-apath="/medrxiv/early/2022/02/14/2021.06.10.21258550.atom" data-hw-author-tooltip-instance="">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip">
                   <h1 class="highwire-cite-title" id="page-title">
                    Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Michael
                      </span>
                      <span class="nlm-surname">
                       Bleher
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Lukas
                      </span>
                      <span class="nlm-surname">
                       Hahn
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Juan Ángel
                      </span>
                      <span class="nlm-surname">
                       Patiño-Galindo
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Mathieu
                      </span>
                      <span class="nlm-surname">
                       Carrière
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Ulrich
                      </span>
                      <span class="nlm-surname">
                       Bauer
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Raúl
                      </span>
                      <span class="nlm-surname">
                       Rabadán
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Andreas
                      </span>
                      <span class="nlm-surname">
                       Ott
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2021.06.10.21258550
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-476466--2826764347" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Michael Bleher
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Mathematical Institute, Heidelberg University
                       </span>
                       , Heidelberg,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Michael%2BBleher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bleher%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMichael%2BBleher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:mbleher@mathi.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">
                        mbleher@mathi.uni-heidelberg.de
                       </a>
                       <a href="mailto:lhahn@mathi.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">
                        lhahn@mathi.uni-heidelberg.de
                       </a>
                       <a href="mailto:andreas.ott@kit.edu" class="" data-icon-position="" data-hide-link-title="0">
                        andreas.ott@kit.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Lukas Hahn
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Mathematical Institute, Heidelberg University
                       </span>
                       , Heidelberg,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Lukas%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hahn%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALukas%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:mbleher@mathi.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">
                        mbleher@mathi.uni-heidelberg.de
                       </a>
                       <a href="mailto:lhahn@mathi.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">
                        lhahn@mathi.uni-heidelberg.de
                       </a>
                       <a href="mailto:andreas.ott@kit.edu" class="" data-icon-position="" data-hide-link-title="0">
                        andreas.ott@kit.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Juan Ángel Patiño-Galindo
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Program for Mathematical Genomics, Department of Systems Biology, Columbia University
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Juan%2B%C3%81ngel%2BPati%C3%B1o-Galindo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Pati%C3%B1o-Galindo%20JN&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJuan%2B%25C3%2581ngel%2BPati%25C3%25B1o-Galindo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Mathieu Carrière
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        DataShape, Inria Sophia-Antipolis
                       </span>
                       , Biot,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mathieu%2BCarri%C3%A8re%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Carri%C3%A8re%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMathieu%2BCarri%25C3%25A8re%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Ulrich Bauer
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        TUM Department of Mathematics and Munich Data Science Institute
                       </span>
                       , Munich,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ulrich%2BBauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bauer%20U&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AUlrich%2BBauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Raúl Rabadán
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Program for Mathematical Genomics, Department of Systems Biology, Columbia University
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ra%C3%BAl%2BRabad%C3%A1n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Rabad%C3%A1n%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ARa%25C3%25BAl%2BRabad%25C3%25A1n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Andreas Ott
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Mathematics Department, Karlsruhe Institute of Technology
                       </span>
                       , Karlsruhe,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Mathematical Institute, Heidelberg University
                       </span>
                       , Heidelberg,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andreas%2BOtt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ott%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAndreas%2BOtt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:mbleher@mathi.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">
                        mbleher@mathi.uni-heidelberg.de
                       </a>
                       <a href="mailto:lhahn@mathi.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">
                        lhahn@mathi.uni-heidelberg.de
                       </a>
                       <a href="mailto:andreas.ott@kit.edu" class="" data-icon-position="" data-hide-link-title="0">
                        andreas.ott@kit.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2021.06.10.21258550v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:476466" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:476466/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.06.10.21258550v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:476466" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:476466/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.06.10.21258550v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:476466" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:476466/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.06.10.21258550v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:476466" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:476466/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.06.10.21258550v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:476466" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:476466/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.06.10.21258550v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:476466" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:476466/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2021.06.10.21258550v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:476466" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:476466/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             The COVID-19 pandemic has initiated an unprecedented worldwide effort to characterize its evolution through the mapping of mutations of the coronavirus SARS-CoV-2. The early identification of mutations that could confer adaptive advantages to the virus, such as higher infectivity or immune evasion, is of paramount importance. However, the large number of currently available genomes precludes the efficient use of phylogeny-based methods. Here we establish a fast and scalable early warning system based on Topological Data Analysis for the identification and surveillance of emerging adaptive mutations in large genomic datasets. Analyzing millions of SARS-CoV-2 genomes from GISAID, we demonstrate that topologically salient mutations are linked with an increase in infectivity or immune escape. We report on emerging potentially adaptive mutations as of January 2022, and pinpoint mutations in Variants of Concern that are likely due to convergent evolution. Our approach can improve the surveillance of mutations of concern, guide experimental studies, and aid vaccine development.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-16">
                             The COVID-19 pandemic, caused by the coronavirus SARS-CoV-2, has led to millions of lost human lives and devastating economic impact worldwide. As the virus continues to spread through the world, it is acquiring new mutations in its genome, and although most mutations will be deleterious or neutral, a few of them could be advantageous for the virus, for instance, by increasing its infectivity or by helping it to avoid the immune system. As more people develop immune protection by previous viral infections or through vaccination, it is important to rapidly and effectively identify mutations that could confer the virus some adaptive advantage [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ].
                            </p>
                            <p id="p-17">
                             One approach to identify potential adaptive mutations consists of experimentally mutating many positions and testing the effect on certain phenotypes, like binding to the human receptor or immune evasion [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             –
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. However, experimental approaches are limited by the vast number of possible variations.
                            </p>
                            <p id="p-18">
                             A more data-driven approach, which solely relies on the genomic information of the virus, is to look for mutations in a particular genomic locus that occur multiple times. If a mutation gives some sort of advantage to the virus, we expect it to occur in several places independently and its frequency to increase with time. In this approach, one usually reconstructs an estimated phylogenetic tree and identifies mutations that occur in independent branches [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             –
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ]. For instance, the
                             <kbd>
                              D614G
                             </kbd>
                             mutation in the Spike gene was identified early in the pandemic and is now found in virtually all virus isolates [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ].
                            </p>
                            <p id="p-19">
                             In the COVID-19 pandemic, an unprecedented worldwide effort to sequence viruses resulted in a growing number of millions of genomes available to the scientific community [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ]. Ideally, one would like to leverage all this genomic information at real-time to rapidly report the emergence of potential mutations of concern [
                             <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ]. Phylogenetic approaches, however, become daunting as the number of sequences increases, and are computationally prohibitive when the number of genomes exceeds the tens of thousands [
                             <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ]. In addition, the independent reemergence of mutations gives rise to homoplasies that confuse the generation of phylogenetic trees [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ]. Moreover it has been observed that constructing a single optimal phylogeny for SARS-CoV-2 is generally problematic, as the number of sequences is large while the genetic diversity is low [
                             <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ].
                            </p>
                            <p id="p-20">
                             Here we establish a novel method based on Topological Data Analysis (TDA) that can efficiently identify emerging adaptive mutations without the need to choose a single optimal phylogenetic tree in a vast range of equally plausible tree reconstructions. It systematically detects convergent events in viral evolution merely by their topological footprint, overcoming limitations of current phylogenetic inference techniques. Thanks to our use of highly optimized algorithms, it easily scales to hundreds of thousands of distinct genomes.
                            </p>
                            <p id="p-21">
                             Analyzing millions of genomic sequences shared via GISAID, the global data science initiative [
                             <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], we first characterize the convergent evolution of the coronavirus SARS-CoV-2 in the early phase of the pandemic from December 2019 until February 2021. We demonstrate that our method can detect adaptive mutations at an early stage, often several weeks before they become recognizable by their prevalence in the population, and explain how this can serve as an early warning and surveillance system. We further report on potentially adaptive mutations in the current phase of the pandemic as of January 2022, and identify mutations in Variants of Concern, most notably in the Omicron variant, that are likely due to convergent evolution.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Quantification of topological recurrence
                             </h3>
                             <p id="p-22">
                              Chan et al. [
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ] initiated the use of persistent homology, a method from Topological Data Analysis, to extract global evolutionary features from large genomic datasets. This method detects topological cycles in the dataset, which correspond to reticulate events in the phylogeny and may cause phylogenetic inference methods to produce ambiguous tree topologies. All information is compiled into a stable and unbiased descriptor known as a persistence barcode (see
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              ,
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              and Methods).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F1.large.jpg?width=800&height=600&carousel=1" title="Reticulate events in molecular evolution create topological cycles. (A) and (B) show possible evolutionary histories on the level of individuals in the example of a genome with only two nucleotides. The coloring of the edges corresponds to the acquisition of a specific mutation, while the coloring of the nodes represents individuals carrying this mutation. Convergent evolution (A) or recombination (B) leads to the presence of four alleles for which there is no single consistent phylogeny (four-gamete test). On the genomic level, genetically identical individuals cannot be distinguished and incompatible phylogenies are represented by a topological cycle in the corresponding phylogenetic network (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Reticulate events in molecular evolution create topological cycles.</span> (A) and (B) show possible evolutionary histories on the level of individuals in the example of a genome with only two nucleotides. The coloring of the edges corresponds to the acquisition of a specific mutation, while the coloring of the nodes represents individuals carrying this mutation. Convergent evolution (A) or recombination (B) leads to the presence of four alleles for which there is no single consistent phylogeny (four-gamete test). On the genomic level, genetically identical individuals cannot be distinguished and incompatible phylogenies are represented by a topological cycle in the corresponding phylogenetic network (C).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F1.medium.gif" width="440" height="165"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F1.medium.gif" width="440" height="165"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Reticulate events in molecular evolution create topological cycles.
                               </span>
                               <p id="p-23" class="first-child">
                                (A) and (B) show possible evolutionary histories on the level of individuals in the example of a genome with only two nucleotides. The coloring of the edges corresponds to the acquisition of a specific mutation, while the coloring of the nodes represents individuals carrying this mutation. Convergent evolution (A) or recombination (B) leads to the presence of four alleles for which there is no single consistent phylogeny (four-gamete test). On the genomic level, genetically identical individuals cannot be distinguished and incompatible phylogenies are represented by a topological cycle in the corresponding phylogenetic network (C).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F2.large.jpg?width=800&height=600&carousel=1" title="Topological data analysis quantifies convergent evolution. (A) Persistent homology detects reticulate events in viral evolution by means of a persistence barcode. Each bar in the barcode (A2) corresponds to a topological cycle in the reticulate phylogeny (A1). Bars at small genetic distance scales are expected to correspond mainly to homoplasies, while recombination events typically produce topological features at larger scales. (B) SNV cycles are topological cycles in the reticulate phylogeny for which adjacent sequences differ by single nucleotide variations (SNV) only. Under the assumption of single substitutions per site, any SNV in each such cycle appears twice, independently of each other, and distributed across both possible lineages. The topological recurrence index (tRI) of a specific mutation is the total number of SNV cycles in the reticulate phylogeny in which this mutation is acquired. In the displayed example phylogeny, the red edges indicate the acquisition of a specific mutation, while the red nodes represent viruses carrying this mutation. The mutation is acquired in two SNV cycles (shaded in blue) and therefore has a tRI of 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Topological data analysis quantifies convergent evolution.</span> (A) Persistent homology detects reticulate events in viral evolution by means of a persistence barcode. Each bar in the barcode (A2) corresponds to a topological cycle in the reticulate phylogeny (A1). Bars at small genetic distance scales are expected to correspond mainly to homoplasies, while recombination events typically produce topological features at larger scales. (B) SNV cycles are topological cycles in the reticulate phylogeny for which adjacent sequences differ by single nucleotide variations (SNV) only. Under the assumption of single substitutions per site, any SNV in each such cycle appears twice, independently of each other, and distributed across both possible lineages. The topological recurrence index (tRI) of a specific mutation is the total number of SNV cycles in the reticulate phylogeny in which this mutation is acquired. In the displayed example phylogeny, the red edges indicate the acquisition of a specific mutation, while the red nodes represent viruses carrying this mutation. The mutation is acquired in two SNV cycles (shaded in blue) and therefore has a tRI of 2.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F2.medium.gif" width="440" height="89"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F2.medium.gif" width="440" height="89"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Topological data analysis quantifies convergent evolution.
                               </span>
                               <p id="p-24" class="first-child">
                                (A) Persistent homology detects reticulate events in viral evolution by means of a persistence barcode. Each bar in the barcode (A2) corresponds to a topological cycle in the reticulate phylogeny (A1). Bars at small genetic distance scales are expected to correspond mainly to homoplasies, while recombination events typically produce topological features at larger scales. (B) SNV cycles are topological cycles in the reticulate phylogeny for which adjacent sequences differ by single nucleotide variations (SNV) only. Under the assumption of single substitutions per site, any SNV in each such cycle appears twice, independently of each other, and distributed across both possible lineages. The topological recurrence index (tRI) of a specific mutation is the total number of SNV cycles in the reticulate phylogeny in which this mutation is acquired. In the displayed example phylogeny, the red edges indicate the acquisition of a specific mutation, while the red nodes represent viruses carrying this mutation. The mutation is acquired in two SNV cycles (shaded in blue) and therefore has a tRI of 2.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              Here we define a novel index of recurrence that is based on persistent homology and does not rely on a possibly ambiguous tree reconstruction. We use a specifically designed algorithm, implemented in
                              <kbd>
                               Ripser
                              </kbd>
                              [
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ], that associates to relevant bars in the persistence barcode explicit SNV cycles, given by a series of isolates that approximates all evolutionary steps as faithfully as possible in terms of single nucleotide variations (SNV). We define the topological recurrence index (tRI) of a specific mutation as the total number of SNV cycles in the reticulate phylogeny that contain an edge corresponding to this mutation. This index provides a lower bound for the number of independent occurrences of the mutation in the phylogeny and is therefore a measure for convergent evolution (see
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              and Methods).
                             </p>
                             <p id="p-26">
                              Standard phylogenetic methods for the detection of convergent evolution are based on the reconstruction of a phylogenetic tree and therefore have an unfavorable scaling, due to the rapid growth of the number of trees representing evolutionary histories that are compatible with the observations [
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ]. Persistent homology provides a new approach by measuring convergent evolution purely in terms of topological cycles, without the need to construct any phylogenetic trees. It enables a rapid and scalable analysis of large datasets with hundreds of thousands of sequences. A performance analysis showed that when applied to sample alignments of up to 5,000 SARS-CoV-2 sequences, the topological method was at least an order of magnitude faster than phylogeny-based methods [
                              <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ,
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ,
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ] (see
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              ). Moreover, the topological recurrence analysis of sequence alignments with millions of SARS-CoV-2 genomes was accomplished in less than a day (see Methods).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F3.large.jpg?width=800&height=600&carousel=1" title="Performance analysis and comparison with phylogeny-based methods. (A) Basic runtime comparison between Topological Data Analysis (TDA)-based methods and standard phylogeny-based methods for random samples of up to 5,000 SARS-CoV-2 genomes. We used IQTree [21] to reconstruct phylogenetic trees. The subsequent homoplasy analysis was performed with TreeTime [7] and HomoplasyFinder [8]. For the TDA-approach we used Hammingdist [22] to generate genetic distance matrices and Ripser [20] for the subsequent computation of persistence barcodes (see Methods). (B) Runtime and memory usage for the computation of persistence barcodes with Ripser [20] for monthly sub-alignments of the GISAID alignment with 161,024 genetically distinct whole genome sequences covering the first year of the pandemic (see Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance analysis and comparison with phylogeny-based methods.</span> (A) Basic runtime comparison between Topological Data Analysis (TDA)-based methods and standard phylogeny-based methods for random samples of up to 5,000 SARS-CoV-2 genomes. We used <kbd>IQTree</kbd> [21] to reconstruct phylogenetic trees. The subsequent homoplasy analysis was performed with <kbd>TreeTime</kbd> [7] and <kbd>HomoplasyFinder</kbd> [8]. For the TDA-approach we used <kbd>Hammingdist</kbd> [22] to generate genetic distance matrices and <kbd>Ripser</kbd> [20] for the subsequent computation of persistence barcodes (see Methods). (B) Runtime and memory usage for the computation of persistence barcodes with <kbd>Ripser</kbd> [20] for monthly sub-alignments of the GISAID alignment with 161,024 genetically distinct whole genome sequences covering the first year of the pandemic (see Methods).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F3.medium.gif" width="440" height="224"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F3.medium.gif" width="440" height="224"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Performance analysis and comparison with phylogeny-based methods.
                               </span>
                               <p id="p-27" class="first-child">
                                (A) Basic runtime comparison between Topological Data Analysis (TDA)-based methods and standard phylogeny-based methods for random samples of up to 5,000 SARS-CoV-2 genomes. We used
                                <kbd>
                                 IQTree
                                </kbd>
                                [
                                <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">
                                 21
                                </a>
                                ] to reconstruct phylogenetic trees. The subsequent homoplasy analysis was performed with
                                <kbd>
                                 TreeTime
                                </kbd>
                                [
                                <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">
                                 7
                                </a>
                                ] and
                                <kbd>
                                 HomoplasyFinder
                                </kbd>
                                [
                                <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                                 8
                                </a>
                                ]. For the TDA-approach we used
                                <kbd>
                                 Hammingdist
                                </kbd>
                                [
                                <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                 22
                                </a>
                                ] to generate genetic distance matrices and
                                <kbd>
                                 Ripser
                                </kbd>
                                [
                                <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                                 20
                                </a>
                                ] for the subsequent computation of persistence barcodes (see Methods). (B) Runtime and memory usage for the computation of persistence barcodes with
                                <kbd>
                                 Ripser
                                </kbd>
                                [
                                <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">
                                 20
                                </a>
                                ] for monthly sub-alignments of the GISAID alignment with 161,024 genetically distinct whole genome sequences covering the first year of the pandemic (see Methods).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Topological analysis of the first year of the pandemic
                             </h3>
                             <p id="p-28">
                              We analyzed topological signals for convergent evolution of the coronavirus SARS-CoV-2, both across the whole genome and on the Spike gene, during the first year of the pandemic from its beginning in December 2019 until February 2021. To that end, we performed a topological recurrence analysis for a curated alignment of 303,651 high-quality SARS-CoV-2 whole genome sequences from GISAID [
                              <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ,
                              <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ] (see Methods). The resulting persistence barcode features 2,899 bars, 58% of which (corresponding to 1,684 SNV cycles) concentrate at small genetic distance scales ≤ 2 and are therefore expected to be associated mainly with homoplasic events (see
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Supplementary Information Figure 1
                              </a>
                              ). In large genomic datasets, homoplasies may arise randomly, causing a certain amount of noise in persistent homology. However, from simulations we inferred that at least 60% of the topological cycles found in the GISAID dataset must be real features due to increased mutation probabilities, selection, recombination, or sequencing errors (see
                              <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                               Supplementary Information Figure 6
                              </a>
                              and Methods). We detected 401 non-synonymous and 299 synonymous mutations across the whole genome showing significant tRI signal (see
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              and Supplementary Information Table 7). Here signals with tRI ≥ 2 are statistically significant (p < 0.05; see Methods). Most of the mutations with strong tRI signal, such as
                              <kbd>
                               ORF1ab:G11083T
                              </kbd>
                              (tRI = 141; see
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              ), are known to be highly homoplasic and cause stability issues in the construction of phylogenetic trees [
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                              ,
                              <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ,
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ,
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ].
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F4.large.jpg?width=800&height=600&carousel=1" title="Topological signals in the first year of the pandemic (December 2019 until February 2021). (A) Topological signals of recurrent mutations across the whole genome. The table in (A1) lists the topological recurrence index (tRI) and the prevalence of mutations with strongest topological signal. Many of these have previously been reported to be highly homoplasic in Turakhia et al.1 [17] and van Dorp et al.2 [10]. For a complete list see Supplementary Information Table 7. (A2) Histogram showing the distribution of topological signals across the whole genome. In every region of the genome, reticulate events play a crucial role in the evolution of the virus. (B) Topological signals of recurrent mutations across the Spike gene. The table in (B1) lists the topological recurrence index (tRI) and the prevalence of Spike mutations with strongest topological signal. For a complete list see Supplementary Information Table 8. (B2) Values of the tRI for highly variable Spike amino acid site positions. For a complete list see Supplementary Information Table 9. (B3) Histogram showing the distribution of topological signals across the Spike gene. There is an accumulation of topologically recurrent mutations in the S1 subunit and the S1-S2 cleavage region, as well as in the signal peptide at the beginning of the Spike gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Topological signals in the first year of the pandemic (December 2019 until February 2021).</span> (A) Topological signals of recurrent mutations across the whole genome. The table in (A1) lists the topological recurrence index (tRI) and the prevalence of mutations with strongest topological signal. Many of these have previously been reported to be highly homoplasic in Turakhia <em>et al</em>.<sup>1</sup> [17] and van Dorp <em>et al</em>.<sup>2</sup> [10]. For a complete list see Supplementary Information Table 7. (A2) Histogram showing the distribution of topological signals across the whole genome. In every region of the genome, reticulate events play a crucial role in the evolution of the virus. (B) Topological signals of recurrent mutations across the Spike gene. The table in (B1) lists the topological recurrence index (tRI) and the prevalence of Spike mutations with strongest topological signal. For a complete list see Supplementary Information Table 8. (B2) Values of the tRI for highly variable Spike amino acid site positions. For a complete list see Supplementary Information Table 9. (B3) Histogram showing the distribution of topological signals across the Spike gene. There is an accumulation of topologically recurrent mutations in the S1 subunit and the S1-S2 cleavage region, as well as in the signal peptide at the beginning of the Spike gene.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F4.medium.gif" width="440" height="213"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F4.medium.gif" width="440" height="213"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Topological signals in the first year of the pandemic (December 2019 until February 2021).
                               </span>
                               <p id="p-29" class="first-child">
                                (A) Topological signals of recurrent mutations across the whole genome. The table in (A1) lists the topological recurrence index (tRI) and the prevalence of mutations with strongest topological signal. Many of these have previously been reported to be highly homoplasic in Turakhia et al.
                                <sup>
                                 1
                                </sup>
                                [
                                <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                                 17
                                </a>
                                ] and van Dorp et al.
                                <sup>
                                 2
                                </sup>
                                [
                                <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                                 10
                                </a>
                                ]. For a complete list see Supplementary Information Table 7. (A2) Histogram showing the distribution of topological signals across the whole genome. In every region of the genome, reticulate events play a crucial role in the evolution of the virus. (B) Topological signals of recurrent mutations across the Spike gene. The table in (B1) lists the topological recurrence index (tRI) and the prevalence of Spike mutations with strongest topological signal. For a complete list see Supplementary Information Table 8. (B2) Values of the tRI for highly variable Spike amino acid site positions. For a complete list see Supplementary Information Table 9. (B3) Histogram showing the distribution of topological signals across the Spike gene. There is an accumulation of topologically recurrent mutations in the S1 subunit and the S1-S2 cleavage region, as well as in the signal peptide at the beginning of the Spike gene.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              Performing the topological recurrence analysis for the Spike gene alignment, we found a total of 322 non-synonymous and 196 synonymous Spike mutations with significant tRI signal (see
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              , Supplementary Information Table 8 and Methods). Here signals with tRI ≥ 8 are statistically significant (p < 0.05; see Methods). We observed a distinct accumulation of topological signals in the S1 subunit, which is associated with host receptor recognition and contains epitopes for antibody binding [
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ], and in the Spike protein signal peptide as well as in the S1-S2 cleavage region [
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ] (see
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              ). We found a strong signal (tRI = 400) for
                              <kbd>
                               S:D614G
                              </kbd>
                              , which indicates that this mutation appeared independently at least 400 times during the pandemic. In fact, the mutation increases transmissibility [
                              <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ,
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ] and in vitro infectiousness [
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              –
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ]. We performed a comparative time series analysis (tRI vs. prevalence) which revealed a steady increase in both tRI and prevalence until the new variant eventually superseded the wild type in the population (see
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              and Methods).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F5.large.jpg?width=800&height=600&carousel=1" title="Comparative time series analysis and ancestral state reconstruction analysis for mutations with strongest topological signal in the first year of the pandemic (December 2019 until February 2021). (A) Topological footprint of a highly recurrent mutation in the example of the mutation ORF1ab:G11083T. A time series analysis shows a monthly increase of the tRI, while the prevalence stays low. This indicates that the mutation has been reemerging frequently and steadily since the beginning of the pandemic. The consistently low prevalence suggests that the mutation is neutral or deleterious, as a beneficial substitution would be expected to establish itself in larger subpopulations. (B, C) Recurrence of the mutations S:A222V and S:S477N persists after its initial surge in prevalence in mid-2020. (D) The mutation S:D614G shows a pattern typical for an adaptive mutation—after a rapid increase in tRI and prevalence the tRI reaches a plateau once the mutation has become dominant, superseding the wild type in the early phase of the pandemic. All ancestral state reconstructions are based on the Nextstrain tree [16] of a curated subsample of 3,507 sequences from the GISAID dataset as of 3 March 2021 [14, 18]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparative time series analysis and ancestral state reconstruction analysis for mutations with strongest topological signal in the first year of the pandemic (December 2019 until February 2021).</span> (A) Topological footprint of a highly recurrent mutation in the example of the mutation <kbd>ORF1ab:G11083T</kbd>. A time series analysis shows a monthly increase of the tRI, while the prevalence stays low. This indicates that the mutation has been reemerging frequently and steadily since the beginning of the pandemic. The consistently low prevalence suggests that the mutation is neutral or deleterious, as a beneficial substitution would be expected to establish itself in larger subpopulations. (B, C) Recurrence of the mutations <kbd>S:A222V</kbd> and <kbd>S:S477N</kbd> persists after its initial surge in prevalence in mid-2020. (D) The mutation <kbd>S:D614G</kbd> shows a pattern typical for an adaptive mutation—after a rapid increase in tRI and prevalence the tRI reaches a plateau once the mutation has become dominant, superseding the wild type in the early phase of the pandemic. All ancestral state reconstructions are based on the Nextstrain tree [16] of a curated subsample of 3,507 sequences from the GISAID dataset as of 3 March 2021 [14, 18].</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F5.medium.gif" width="440" height="170"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F5.medium.gif" width="440" height="170"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Comparative time series analysis and ancestral state reconstruction analysis for mutations with strongest topological signal in the first year of the pandemic (December 2019 until February 2021).
                               </span>
                               <p id="p-31" class="first-child">
                                (A) Topological footprint of a highly recurrent mutation in the example of the mutation
                                <kbd>
                                 ORF1ab:G11083T
                                </kbd>
                                . A time series analysis shows a monthly increase of the tRI, while the prevalence stays low. This indicates that the mutation has been reemerging frequently and steadily since the beginning of the pandemic. The consistently low prevalence suggests that the mutation is neutral or deleterious, as a beneficial substitution would be expected to establish itself in larger subpopulations. (B, C) Recurrence of the mutations
                                <kbd>
                                 S:A222V
                                </kbd>
                                and
                                <kbd>
                                 S:S477N
                                </kbd>
                                persists after its initial surge in prevalence in mid-2020. (D) The mutation
                                <kbd>
                                 S:D614G
                                </kbd>
                                shows a pattern typical for an adaptive mutation—after a rapid increase in tRI and prevalence the tRI reaches a plateau once the mutation has become dominant, superseding the wild type in the early phase of the pandemic. All ancestral state reconstructions are based on the Nextstrain tree [
                                <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                                 16
                                </a>
                                ] of a curated subsample of 3,507 sequences from the GISAID dataset as of 3 March 2021 [
                                <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">
                                 14
                                </a>
                                ,
                                <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">
                                 18
                                </a>
                                ].
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-32">
                              The mutations
                              <kbd>
                               S:A222V
                              </kbd>
                              and
                              <kbd>
                               S:S477N
                              </kbd>
                              are among those with strongest topological signal for recurrence (see
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              ). Both are associated with lineage B.1.177 / 20E (EU1) which emerged in Europe in mid-2020, and
                              <kbd>
                               S:S477N
                              </kbd>
                              is now also seen in the Omicron variant B.1.1.529 [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ]. While
                              <kbd>
                               S:S477N
                              </kbd>
                              is known to affect the binding affinity to the ACE2 receptor [
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ] leading to a slight increase in fitness, there is no conclusive evidence yet whether or not
                              <kbd>
                               S:A222V
                              </kbd>
                              also results in higher transmissibility [
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ]. Our time series analysis for the latter shows that the particularly strong tRI signal is still rising after the initial surge in prevalence in the European summer of 2020 (see
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              ). This suggests that
                              <kbd>
                               S:A222V
                              </kbd>
                              is notably recurrent, independently of its impact on viral fitness.
                             </p>
                             <p id="p-33">
                              We noticed that in particular on the receptor-binding domain (RBD), several of the topologically significant amino acid changes (tRI ≥ 8) are found in Variants of Interest (VOI) or Variants of Concern (VOC) [
                              <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ], with a distinct accumulation in the receptor-binding motif (see
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ,
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Supplementary Information Figure 2
                              </a>
                              and
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Supplementary Information Figure 3
                              </a>
                              ). Specifically, the substitutions
                              <kbd>
                               S:N501Y
                              </kbd>
                              ,
                              <kbd>
                               S:E484K
                              </kbd>
                              ,
                              <kbd>
                               S:L452R
                              </kbd>
                              ,
                              <kbd>
                               S:K417N
                              </kbd>
                              ,
                              <kbd>
                               S:F490S
                              </kbd>
                              and
                              <kbd>
                               S:S494P
                              </kbd>
                              all result in reduced binding of polyclonal convalescent plasma [
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ] and exhibit a distinct increase in tRI and prevalence starting in late 2020 (see
                              <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                               Supplementary Information Figure 3
                              </a>
                              ). This pattern is likely due to selective pressures induced by immune evasion in a host population with rising immunity. While both
                              <kbd>
                               S:N501Y
                              </kbd>
                              and
                              <kbd>
                               S:N501T
                              </kbd>
                              produce comparable tRI signals and induce a slight antibody escape, the fact that
                              <kbd>
                               S:N501Y
                              </kbd>
                              has a comparatively high prevalence of 19%, and is seen in several VOCs, is probably due to the additional increase in ACE2-binding affinity caused by the asparagine-to-tyrosine substitution. Topological signals for the mutations
                              <kbd>
                               S:Y453F
                              </kbd>
                              and
                              <kbd>
                               S:F486L
                              </kbd>
                              exclusively originated from a small subpopulation in minks [
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              –
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ]. The fact that tRI signals remain low (tRI = 8) despite becoming significant already in June/May 2020 suggests that both mutations have an adaptive effect in minks but do not confer a significant fitness advantage in humans.
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/477267/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/477267/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/477267" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <span class="caption-title">
                                Amino acid changes on the receptor-binding domain with strong topological signal of convergence as of February 2021.
                               </span>
                               <p id="p-34" class="first-child">
                                The table displays the top ten amino acid substitutions on the receptor-binding domain with significant topological recurrence index (tRI ≥ 8), plus a few more selected mutations, together with the tendency of the tRI and the date of initial acquisition of a significant tRI signal. Several of the topologically salient mutations occur in lineages designated as VOI/VOC [
                                <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">
                                 30
                                </a>
                                ]. For the full table see
                                <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                 Supplementary Information Figure 2
                                </a>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Correlation with host adaptation
                             </h3>
                             <p id="p-35">
                              Our analysis of the first year of the pandemic revealed that on the receptor-binding domain, there is a strong correlation between significant topological signals (tRI ≥ 8) and an increase in ACE2-binding affinity [
                              <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ] compared to the wild type (Fisher’s exact test, p < 0.01;
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 6
                              </a>
                              ). We did not find a similar correlation between significant tRI and an increase in plasma antibody escape [
                              <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ], which is plausible as immune evasion has become a relevant factor for the evolution of the virus only towards the end of 2020. However, among those mutations with strong topological signal of convergence (tRI > 12) we found a correlation between increased mean plasma antibody escape > 0.025 and the initial acquisition of a significant tRI signal after October 2020 (Fisher’s exact test, p < 0.05;
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Figure 6
                              </a>
                              ). This provides evidence for a shift, beginning in late 2020, towards immune escape as the driving force behind adaptation during the first year of the pandemic.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F6.large.jpg?width=800&height=600&carousel=1" title="Topological recurrence correlates with increased binding affinity and immune evasion. There is a strong correlation between a significant topological recurrence index (tRI ≥ 8) and an increase in ACE2-binding affinity in the year from December 2019 until February 2021. Among highly topologically recurrent amino acid changes, the more recent ones show increased mean plasma antibody escape. Experimental data taken from Starr et al. [3, Table S2] and Greaney et al. [4, Table S3]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Topological recurrence correlates with increased binding affinity and immune evasion.</span> There is a strong correlation between a significant topological recurrence index (tRI ≥ 8) and an increase in ACE2-binding affinity in the year from December 2019 until February 2021. Among highly topologically recurrent amino acid changes, the more recent ones show increased mean plasma antibody escape. Experimental data taken from Starr <em>et al</em>. [3, Table S2] and Greaney <em>et al</em>. [4, Table S3].</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F6.medium.gif" width="421" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F6.medium.gif" width="421" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Topological recurrence correlates with increased binding affinity and immune evasion.
                               </span>
                               <p id="p-36" class="first-child">
                                There is a strong correlation between a significant topological recurrence index (tRI ≥ 8) and an increase in ACE2-binding affinity in the year from December 2019 until February 2021. Among highly topologically recurrent amino acid changes, the more recent ones show increased mean plasma antibody escape. Experimental data taken from Starr et al. [3, Table S2] and Greaney et al. [4, Table S3].
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              An early warning and surveillance tool
                             </h3>
                             <p id="p-37">
                              We found that the sensitivity of the topological recurrence index (tRI) is sufficient to detect signals of convergent evolution already at very low mutation frequencies, often several weeks before the respective mutation becomes recognizable by its prevalence in the population. More specifically, we observed that the onset of significant tRI signals precedes the actual rise in prevalence by several weeks for the adaptive mutations
                              <kbd>
                               S:L18F
                              </kbd>
                              [
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ],
                              <kbd>
                               K417N
                              </kbd>
                              [
                              <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ],
                              <kbd>
                               L452R
                              </kbd>
                              [
                              <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ],
                              <kbd>
                               S:E484K
                              </kbd>
                              [
                              <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ],
                              <kbd>
                               S:N501Y
                              </kbd>
                              [
                              <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ] and
                              <kbd>
                               S:P681H
                              </kbd>
                              [
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ,
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ] seen in the VOCs Alpha, Beta, Gamma, Delta and Omicron [
                              <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ] (see
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Figure 7
                              </a>
                              ). This demonstrates that the topological recurrence index serves as an early indicator for emerging adaptive mutations.
                             </p>
                             <p id="p-38">
                              In order to assess the reliability of this indicator, we retrospectively compared mutations that had been identified as potentially adaptive in February 2021 with the actual evolution of the virus, and the emergence of VOIs/VOCs in particular, during the later year 2021. Focusing on the RBD, we found that the amino acid changes
                              <kbd>
                               S:A348S
                              </kbd>
                              ,
                              <kbd>
                               S:N354D
                              </kbd>
                              ,
                              <kbd>
                               S:P384L
                              </kbd>
                              ,
                              <kbd>
                               S:N439K
                              </kbd>
                              ,
                              <kbd>
                               S:G446V
                              </kbd>
                              ,
                              <kbd>
                               S:A475V
                              </kbd>
                              ,
                              <kbd>
                               S:E484G
                              </kbd>
                              ,
                              <kbd>
                               S:F490S
                              </kbd>
                              ,
                              <kbd>
                               S:N501T
                              </kbd>
                              and
                              <kbd>
                               S:Y508H
                              </kbd>
                              developed a significant tRI signal over the course of the first year of the pandemic, with a rising tendency in February 2021, while they are associated with an increased mean plasma antibody escape > 0.01 [
                              <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ], but had low prevalence < 5% and had not been seen in any VOI/VOC as of February 2021 (see
                              <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ,
                              <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                               Supplementary Information Figure 2
                              </a>
                              and
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Supplementary Information Figure 4
                              </a>
                              ). Mutations at these RBD residues are likely to confer a fitness advantage to the virus and might therefore appear in future variants. In fact, several months after our analysis was completed, the immune escape mutations
                              <kbd>
                               S:F490S
                              </kbd>
                              [
                              <a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ] and
                              <kbd>
                               S:G446S
                              </kbd>
                              [
                              <a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ], which had not been observed in notable variants before, occurred in the Lambda [
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ,
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                              ] and Omicron [
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ] variants, which were designated as VOI/VOC in June/November 2021 [
                              <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ]. Similarly, the Spike substitutions
                              <kbd>
                               S:T95I
                              </kbd>
                              ,
                              <kbd>
                               S:L452R
                              </kbd>
                              ,
                              <kbd>
                               S:S477N
                              </kbd>
                              ,
                              <kbd>
                               S:T478K
                              </kbd>
                              ,
                              <kbd>
                               S:N679K
                              </kbd>
                              ,
                              <kbd>
                               S:P681R
                              </kbd>
                              and
                              <kbd>
                               S:D796Y
                              </kbd>
                              later occurred in the Delta, Kappa and Omicron variants, which were designated as VOC/VOI/VOC not until May/April/November 2021 [
                              <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ]. While
                              <kbd>
                               S:T478K
                              </kbd>
                              had not yet reached a statistically significant tRI, the tRI of all of these mutations showed a rising tendency in February 2021 (see
                              <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              and
                              <a id="xref-fig-11-3" class="xref-fig" href="#F11">
                               Supplementary Information Figure 3
                              </a>
                              ).
                             </p>
                             <p id="p-39">
                              By investigating explicit representatives of topological cycles in the genomic dataset, we were able to extract geographic and temporal information about the independent acquisition of topologically salient recurrent mutations during the pandemic (see
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Figure 7
                              </a>
                              ).
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F7.large.jpg?width=800&height=600&carousel=1" title="Early warning and surveillance for emerging adaptive Spike mutations. (A) Typical comparative time series pattern (tRI vs. prevalence) of an emerging adaptive mutation as observed in the escape mutation S:E484K. The tRI rises to a significant level (tRI = 5) in September 2020, while the prevalence stays very low < 0.02% and shows a visible increase to 0.06% by a factor of 3 for the first time several weeks later in October 2020. Both tRI and prevalence show a rising tendency in February 2021. (B) Cycle localization in the genome alignment yields geographic and temporal information about the independent acquisition of S:E484K. The diagram shows SNV cycles created by reticulate events involving the Zeta variant (P.2) in Brazil in October 2020, and within the Alpha variant (B.1.1.7) in England in January 2021. (C) Typical comparative time series pattern (tRI vs. prevalence) observed in emerging adaptive Spike mutations, with the onset of significant tRI signals preceding the actual rise in prevalence by several weeks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Early warning and surveillance for emerging adaptive Spike mutations.</span> (A) Typical comparative time series pattern (tRI vs. prevalence) of an emerging adaptive mutation as observed in the escape mutation <kbd>S:E484K</kbd>. The tRI rises to a significant level (tRI = 5) in September 2020, while the prevalence stays very low < 0.02% and shows a visible increase to 0.06% by a factor of 3 for the first time several weeks later in October 2020. Both tRI and prevalence show a rising tendency in February 2021. (B) Cycle localization in the genome alignment yields geographic and temporal information about the independent acquisition of <kbd>S:E484K</kbd>. The diagram shows SNV cycles created by reticulate events involving the Zeta variant (P.2) in Brazil in October 2020, and within the Alpha variant (B.1.1.7) in England in January 2021. (C) Typical comparative time series pattern (tRI vs. prevalence) observed in emerging adaptive Spike mutations, with the onset of significant tRI signals preceding the actual rise in prevalence by several weeks.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F7.medium.gif" width="440" height="222"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F7.medium.gif" width="440" height="222"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <span class="caption-title">
                                Early warning and surveillance for emerging adaptive Spike mutations.
                               </span>
                               <p id="p-40" class="first-child">
                                (A) Typical comparative time series pattern (tRI vs. prevalence) of an emerging adaptive mutation as observed in the escape mutation
                                <kbd>
                                 S:E484K
                                </kbd>
                                . The tRI rises to a significant level (tRI = 5) in September 2020, while the prevalence stays very low < 0.02% and shows a visible increase to 0.06% by a factor of 3 for the first time several weeks later in October 2020. Both tRI and prevalence show a rising tendency in February 2021. (B) Cycle localization in the genome alignment yields geographic and temporal information about the independent acquisition of
                                <kbd>
                                 S:E484K
                                </kbd>
                                . The diagram shows SNV cycles created by reticulate events involving the Zeta variant (P.2) in Brazil in October 2020, and within the Alpha variant (B.1.1.7) in England in January 2021. (C) Typical comparative time series pattern (tRI vs. prevalence) observed in emerging adaptive Spike mutations, with the onset of significant tRI signals preceding the actual rise in prevalence by several weeks.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-41">
                              Updated analyses are available at
                              <a href="https://tdalife.github.io/covtrec">
                               https://tdalife.github.io/covtrec
                              </a>
                              .
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Ongoing convergent evolution and the Omicron variant
                             </h3>
                             <p id="p-42">
                              We analyzed the ongoing evolution of the coronavirus by applying our early warning and surveillance pipeline to a curated alignment of 3,928,116 high-quality SARS-CoV-2 Spike gene sequences from GISAID [
                              <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ,
                              <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ] that have been collected during the year from January 2021 until January 2022 (see Methods).
                             </p>
                             <p id="p-43">
                              A total of 343 non-synonymous and 189 synonymous Spike mutations with significant tRI signal were found (see
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Figure 8
                              </a>
                              , Supplementary Information Table 11 and Methods). These mutations are potentially adaptive in the current phase of the pandemic and might therefore appear in future variants. Here signals with tRI ≥ 88 are statistically significant (p < 0.05; see Methods). We observed that on the receptor-binding domain, amino acid changes with significant topological signal accumulate in the receptor-binding motif (see
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               Figure 8
                              </a>
                              ).
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F8.large.jpg?width=800&height=600&carousel=1" title="Spike gene mutations with strong topological signal of convergence as of January 2022 and the Omicron variant. (A) The table displays the top 14 amino acid substitutions on the Spike gene with significant topological recurrence index (tRI ≥ 88), plus a few more selected mutations. Among those 14 substitutions, 50% occur in lineages designated as VOI/VOC [30]. For the full table see Supplementary Information Table 11. (B) Heatmap of all amino acid variations across the RBD showing topological signals of convergence. Significant signals (tRI ≥ 88; shown in red) accumulate in the receptor-binding motif. (C) Topological signals of convergence for the defining Spike mutations in the Omicron variant (B.1.1.529; BA.1, BA.2 and BA.3) [30, 44]. 47% of these mutations (53%/56% in the sublineage BA.1/BA.2) are significantly topologically recurrent with tRI ≥ 88 (displayed in boldface) and are likely due to convergent evolution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Spike gene mutations with strong topological signal of convergence as of January 2022 and the Omicron variant.</span> (A) The table displays the top 14 amino acid substitutions on the Spike gene with significant topological recurrence index (tRI ≥ 88), plus a few more selected mutations. Among those 14 substitutions, 50% occur in lineages designated as VOI/VOC [30]. For the full table see Supplementary Information Table 11. (B) Heatmap of all amino acid variations across the RBD showing topological signals of convergence. Significant signals (tRI ≥ 88; shown in red) accumulate in the receptor-binding motif. (C) Topological signals of convergence for the defining Spike mutations in the Omicron variant (B.1.1.529; BA.1, BA.2 and BA.3) [30, 44]. 47% of these mutations (53%/56% in the sublineage BA.1/BA.2) are significantly topologically recurrent with tRI ≥ 88 (displayed in boldface) and are likely due to convergent evolution.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F8.medium.gif" width="311" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F8.medium.gif" width="311" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8.
                               </span>
                               <span class="caption-title">
                                Spike gene mutations with strong topological signal of convergence as of January 2022 and the Omicron variant.
                               </span>
                               <p id="p-44" class="first-child">
                                (A) The table displays the top 14 amino acid substitutions on the Spike gene with significant topological recurrence index (tRI ≥ 88), plus a few more selected mutations. Among those 14 substitutions, 50% occur in lineages designated as VOI/VOC [
                                <a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">
                                 30
                                </a>
                                ]. For the full table see Supplementary Information Table 11. (B) Heatmap of all amino acid variations across the RBD showing topological signals of convergence. Significant signals (tRI ≥ 88; shown in red) accumulate in the receptor-binding motif. (C) Topological signals of convergence for the defining Spike mutations in the Omicron variant (B.1.1.529; BA.1, BA.2 and BA.3) [
                                <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">
                                 30
                                </a>
                                ,
                                <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                 44
                                </a>
                                ]. 47% of these mutations (53%/56% in the sublineage BA.1/BA.2) are significantly topologically recurrent with tRI ≥ 88 (displayed in boldface) and are likely due to convergent evolution.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-45">
                              Of particular interest are the defining amino acid substitutions in the Omicron variant B.1.1.529 [
                              <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ,
                              <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">
                               44
                              </a>
                              ], which was designated as VOC in November 2021 [
                              <a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ] and is rapidly spreading all over the world in January 2022 [
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ]. Our analysis revealed that 47% (18 out of 38) of the Spike amino acid substitutions seen in B.1.1.529—with 53% (16 out of 30) in the sublineage BA.1 and 56% (15 out of 27) in the sublineage BA.2—show a significant topological signal of recurrence (see
                              <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                               Figure 8
                              </a>
                              ). This includes, among others, the following Spike amino acid substitutions:
                              <kbd>
                               S:G339D
                              </kbd>
                              ,
                              <kbd>
                               S:N440K
                              </kbd>
                              ,
                              <kbd>
                               S:S477N
                              </kbd>
                              ,
                              <kbd>
                               S:T478K
                              </kbd>
                              and
                              <kbd>
                               S:N501Y
                              </kbd>
                              , which are known to enhance binding to the human ACE2 receptor;
                              <kbd>
                               S:K417N
                              </kbd>
                              ,
                              <kbd>
                               S:G446S
                              </kbd>
                              and
                              <kbd>
                               S:E484A
                              </kbd>
                              , which may reduce polyclonal antibody binding;
                              <kbd>
                               S:N679
                              </kbd>
                              and
                              <kbd>
                               S:P681H
                              </kbd>
                              at the S1-S2 cleavage region [
                              <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ,
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ]. Our findings provide further insight into the possible origins of the Spike mutations observed in the Omicron variant [
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ]. In fact, all 18 substitutions with significant topological recurrence index tRI ≥ 88 likely arose from convergent evolution, while the emergence of the remaining substitutions showing weak topological signals may be due to other reasons.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-46">
                             In the current COVID-19 pandemic, the early identification of emerging adaptive mutations in large SARS-CoV-2 genomic datasets is of paramount importance. Such mutations could be associated with vaccine resistance or higher transmissibility, among other concerning attributes [
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              48
                             </a>
                             ]. We present here an effective and unbiased method, based on a technique from Topological Data Analysis known as persistent homology, that can rapidly identify the presence of these mutations and is able to efficiently deal with the ever-increasing wealth of genomic sequencing data created by global public health surveillance. Our method does not rely on the prior reconstruction of an optimal phylogenetic tree and therefore (i) outperforms phylogeny-based methods [
                             <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ,
                             <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ] by at least an order of magnitude, and (ii) accomplishes the analysis of current datasets with millions of SARS-CoV-2 genomes within just one day. However, this gain in performance comes at the cost that, in contrast to phylogeny-based methods, our method will resolve homoplasies only on small genetic distance scales. We further remark that although persistent homology is robust with respect to noise in the genomic dataset [
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              49
                             </a>
                             ], systematic sequencing errors [
                             <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                              50
                             </a>
                             ,
                             <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                              51
                             </a>
                             ] might tamper topological inference to some extent.
                            </p>
                            <p id="p-47">
                             Drawing from data from GISAID [
                             <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], we applied our analysis pipeline to monitor the adaptive evolution of the coronavirus SARS-CoV-2 in two phases of the pandemic—the early phase from December 2019 until February 2021, and the current phase as of January 2022. The topological analysis of the early phase revealed a total of 700 (518) topologically recurrent mutations distributed across the whole genome (Spike gene), which shows that convergence plays a significant role in the evolution of SARS-CoV-2. We found that our method reliably identifies highly homoplasic sites across the whole genome, making it a useful tool in the design of unbiased masking schemes in phylogenetic inference. Including experimental data obtained by Starr et al. [
                             <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ] and Greaney et al. [
                             <a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ] into our analysis, we inferred that on the receptor-binding domain topological recurrence correlates with host adaptation. Specifically, we observed a beginning shift in late 2020 from host receptor binding towards immune evasion as the main selective pressure. The topological analysis of the current phase of the pandemic revealed a total of 532 topologically recurrent mutations on the Spike gene.
                            </p>
                            <p id="p-48">
                             We demonstrated that our method (i) is sufficiently sensitive to detect emerging adaptive mutations already at an early stage when mutation frequencies are still very low, and (ii) provides geographic and temporal information about virus isolates involved in concrete reticulate events. We identified potentially adaptive mutations on the Spike gene that showed significant topological signals of recurrence with a rising tendency in February 2021. These mutations are likely to appear in future variants, and in fact, several of these candidate mutations have later received special attention as features of the Delta, Lambda and Omicron Variants of Interest/Concern [
                             <a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ], not until several months after our analysis was completed. We explained how our results can aid the understanding of possible reasons for the occurrence of certain mutations in Variants of Concern such as the Omicron variant. We found that as of January 2022, at least half of the Spike gene mutations observed in the Omicron variant are likely due to convergent evolution. Updated analyses are available at
                             <a href="https://tdalife.github.io/covtrec">
                              https://tdalife.github.io/covtrec
                             </a>
                             .
                            </p>
                            <p id="p-49">
                             Based on these insights, we propose persistent homology as an early warning system for the emergence of new adaptive mutations in the ongoing COVID-19 pandemic and foresee this capability also in future pandemics of various pathogens. Basically, our method allows for a targeted mapping of recurrent mutations in any region of the genome, which can guide and motivate the experimental study of adaptive effects of specific mutations also in genes other than the Spike gene. Our method could in particular aid the development and rapid adaptation of vaccines for new emerging variants [
                             <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                              52
                             </a>
                             ]. We envision a combined effort between public health organizations with extensive sequencing of viral genomes, the computational characterization of potentially adaptive variants, and the experimental phenotypic characterization of these variants.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Data acquisition and data preparation
                             </h3>
                             <p id="p-50">
                              We use two separate datasets of SARS-CoV-2 genome sequences provided by the GISAID Epi-CoV Database [
                              <a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ,
                              <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ]. The first dataset was obtained by downloading all available SARS-CoV-2 whole genome sequences as of 28 February 2021, isolated from human and animal hosts, that carried the following attributes: “complete”, “high coverage”, “low coverage excluded”, “collection date complete”. This dataset comprised 450,473 sequences. We aligned all sequences with
                              <kbd>
                               MUSCLE (Version 3.8.31)
                              </kbd>
                              [
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ], using as reference genome the sequence Wuhan/Hu-1 with accession number
                              <kbd>
                               EPI_ISL_402125
                              </kbd>
                              , truncated at the start codon of
                              <kbd>
                               ORF1ab
                              </kbd>
                              (reference site position 266) and the stop codon of
                              <kbd>
                               ORF10
                              </kbd>
                              (reference site position 29,674). Subsequently all sequences containing at least one ambiguous site (character “N”) were removed. This resulted in an alignment comprising 303,651 complete SARS-CoV-2 genomes of length 30,130nt. The second dataset is the alignment
                              <kbd>
                               msa 0117.fasta
                              </kbd>
                              which we downloaded from GISAID on 19 January 2022. It comprises 6,475,061 SARS-CoV-2 whole genome sequences that have been aligned to the reference sequence Wuhan/WIV04 with GISAID accession number
                              <kbd>
                               EPI_ISL_402124
                              </kbd>
                              using
                              <kbd>
                               MAFFT (Version 7.497)
                              </kbd>
                              [
                              <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ]. Sequences with collection date between 1 January 2021 and 17 January 2022 were selected, then truncated to the Spike gene (reference site positions 21,563 to 25,384), and finally sequences containing any characters other than A, C, T or G were removed. This resulted in an alignment comprising 3,928,116 complete SARS-CoV-2 Spike genes of length 4,669nt. A list of accession numbers of all sequences in these two alignments, along with an acknowledgement of the contributions of both the submitting and the originating laboratories, can be retrieved through the Data Acknowledgement Locator at
                              <a href="https://www.gisaid.org">
                               https://www.gisaid.org
                              </a>
                              with IDs
                              <kbd>
                               EPI_SET_20220127bo
                              </kbd>
                              and
                              <kbd>
                               EPI_SET_20220124he
                              </kbd>
                              .
                             </p>
                             <p id="p-51">
                              The experimental data on viral phenotypes by Starr et al. and Greaney et al. was downloaded from [3, Table S2] and [4, Table S3]. The values for the mean plasma antibody escape used in
                              <a id="xref-fig-10-4" class="xref-fig" href="#F10">
                               Supplementary Information Figure 2
                              </a>
                              were computed by averaging mutation escape values for every substitution in [4, Table S3] over all subjects.
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Distance matrices
                             </h3>
                             <p id="p-52">
                              We used
                              <kbd>
                               Hammingdist (Version 0.13.0)
                              </kbd>
                              [
                              <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ] to compute the genetic distance matrix of a given alignment of genome sequences. For any pair of sequences in the alignment, this matrix gives the Hamming distance between the two sequences, which is the number of site positions at which the nucleotides in the two aligned sequences differ. Noteworthy, our convention in this work is that insertions and deletions (dashes “-” in aligned sequences) do not contribute to the genetic distance.
                             </p>
                             <p id="p-53">
                              From the whole genome alignment covering the first year of the pandemic we created 15 time buckets, each ranging from December 2019 to one of the months between December 2019 and February 2021. For each time bucket, a time bucket sub-alignment of all genetically distinct sequences whose collection dates belong to the given time bucket was created by selecting isolates by their date stamp and removing genetically identical sequences (Hamming distance = 0). The largest time bucket sub-alignment ranging from December 2019 to February 2021 contained 161,024 genetically distinct sequences. Then for each such sub-alignment the corresponding genetic distance matrix, which is a sub-matrix of the distance matrix of the whole alignment, was derived. We obtained 15 distance matrices of whole genome time bucket sub-alignments. This process was repeated for all sub-alignments after truncating sequences to the Spike gene (reference site positions 21,563 to 25,384). We obtained 15 distance matrices of Spike gene time bucket sub-alignments. The Spike gene alignment covering the year from January 2021 until January 2022 contained 291,141 genetically distinct sequences. We computed the genetic distance matrix for this alignment.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Topological Data Analysis and viral evolution
                             </h3>
                             <p id="p-54">
                              Topological Data Analysis (TDA) is a field of data science that aims to study the shape of large datasets, by extracting topological structures and patterns. Such topological structures have associated dimensions: structures of dimension zero can be thought of as the connected components, and structures of dimension one are essentially the loops, or topological cycles, of the dataset. Structures of higher dimensions can also be defined, but are also more difficult to interpret. Here we are interested in reticulate evolutionary processes, thus we choose to focus on topological structures in dimension one, since topological cycles can be interpreted as signals of divergence from phylogenetic trees (see
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              ).
                             </p>
                             <p id="p-55">
                              Datasets often come as point clouds: in our setting, each point corresponds to a virus genome sample, and lies in a high-dimensional space where each nucleotide of the genome is a dimension. A common way to extract the phylogenetic network from this point cloud simply amounts to connecting samples as soon as their genetic distance is less than a given threshold r > 0. This results in a (neighborhood) graph, whose set of cycles provides candidates for the topological structures in dimension one of the true underlying network. However, a main limitation of this approach comes from the fact that relevant topological structure typically appears at multiple scales (see
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Supplementary Information Figure 5
                              </a>
                              ).
                             </p>
                             <p id="p-56">
                              The most common way to handle this issue in Topological Data Analysis is to actually compute and track the cycles for all possible values of r ranging from 0 to + ∞. As r increases, some cycles can appear, and some already existing cycles can disappear, or get filled in. The whole point of Topological Data Analysis is to record, for each cycle, its radius of appearance, or birth time, and radius of disappearance, or death time. This is called the persistent homology of the point cloud. The construction, based on a scale parameter r, can be summarized as follows. The input is a distance matrix describing the dataset, considered as a finite metric space. First, consider the geometric graph at scale r, whose vertices are the data points, with any two points connected by an edge whenever their distance is at most r. Generalizing this construction, the Vietoris–Rips complex at scale r connects any subset of the data points with a simplex (an edge, a triangle, a tetrahedron, or a higher-dimensional generalization thereof) whenever all pairwise distances in the subset are at most r. A Vietoris–Rips complex is a particular type of simplicial complex, a higher-dimensional generalization of graphs which is of crucial interest in algebraic topology, in particular in homology theory. The family of Vietoris–Rips complexes for all parameters r is called the Vietoris–Rips filtration. It provides a multiscale method to extract cycles of various sizes, and to encode them in a so-called persistence barcode: each bar, or interval, in this barcode corresponds to a cycle representing a topological feature (a reticulate evolutionary process in our case), and the bar endpoints correspond to its radii of birth and death (the maximum genetic distance between consecutive samples forming the cycle, and, roughly, the maximum pairwise genetic distance between samples forming the cycle).
                             </p>
                             <p id="p-57">
                              Each bar indicates the presence of a reticulate event, implying that the evolutionary history cannot be fully explained in terms of a single phylogenetic tree. The mathematical background of this phenomenon is a classical theorem due to Rips, which asserts that trees have trivial persistent Vietoris–Rips homology [
                              <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              ]. The corresponding cycle in the associated reticulate phylogeny can then be localized in the sequence alignment by tracing it back to the isolates that constitute the reticulate event. Moreover, the length of the bar represents the cycle size. In our case, this corresponds to the length of the reticulate evolutionary process, which allows to distinguish, for instance, between homoplasies and recombinations (see
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              ). There are several scenarios that can lead to reticulate events. For instance, if a genome of an organism imports genetic material from a different genome, in lateral gene transfer for instance, we will observe that parts of the newly generated genome resemble the parent, while others resemble the genome of the organism that exported the material. Recombination and reassortments are common phenomena observed in viruses where two parental strains co-infect the same host cell generating a new virus containing genetic material from both parental strains. But similarity between genomes can also be generated at smaller scales, when the same mutation occurs independently twice, making the two strains more similar than expected. Persistent homology captures all these events, and also the scale of the events. Although in general it requires care to infer the biological origin of a given topological cycle, in viral evolution one expects bars at small genetic distance scales to correspond mostly to homoplasies, while well-supported recombination signals typically produce topological features at larger scales, as entire blocks of genetic material are exchanged in the process.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Computation of persistent homology
                             </h3>
                             <p id="p-58">
                              <kbd>
                               Ripser
                              </kbd>
                              is a state-of-the-art software for the computation of persistent homology based on the topological construction of Vietoris–Rips complexes, developed by one of the authors [
                              <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. For the computation of the persistence barcode,
                              <kbd>
                               Ripser
                              </kbd>
                              resorts to various optimizations, which are crucial when handling datasets of the size considered here. Notably,
                              <kbd>
                               Ripser
                              </kbd>
                              computes persistent cohomology, which is not based on cycles but instead on cocycles, often described intuitively as cuts that tear open a hole. In order to obtain the requisite cycles representing the features in persistent homology, we used a custom version of
                              <kbd>
                               Ripser
                              </kbd>
                              that subsequently carries out a second computation, this time based on cycles instead of cocycles. While a naive computation based on homology would be prohibitively expensive, the previous computation of the persistence barcode based on cocycles makes the subsequent computation of representative cycles feasible.
                             </p>
                             <p id="p-59">
                              For our computations, we used a customized version of
                              <kbd>
                               Ripser
                              </kbd>
                              to compute the representative cycles for the persistent homology of the Vietoris–Rips filtration associated to the genetic distance matrix for each time bucket sub-alignment (whole genome and Spike gene). As we are only interested in SNV cycles, the computation of persistence barcodes for the time bucket sub-alignments was restricted to small genetic distance scales (
                              <kbd>
                               Ripser
                              </kbd>
                              scale parameter threshold set to 2), which greatly increases the speed of the computation.
                             </p>
                             <p id="p-60">
                              The homological features identified by persistent homology admit different representative cycles. In order to obtain cycles that fit tightly to the data points, our customized version of Ripser uses a method called exhaustive reduction [
                              <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                               56
                              </a>
                              ,
                              <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                               57
                              </a>
                              ], which can be roughly summarized as follows. Whenever a representative cycle contains an edge that also appears in another cycle as the longest edge, a tighter representative can be obtained by replacing the edge with the remaining edges from the other cycles, which all have shorter length.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Topological features are statistically significant
                             </h3>
                             <p id="p-61">
                              We estimated the expected number of topological cycles in persistent homology that are created by random homoplasic events in the GISAID dataset covering the first year of the pandemic with 161,024 genetically distinct whole genome sequences collected from December 2019 until February 2021. To this end, we simulated several evolutionary scenarios under the following assumptions: uniform probability distribution for substitutions across the genome, no variations in fitness, and zero recombination rate.
                             </p>
                             <p id="p-62">
                              We generated forward simulations of viral evolution based on a Wright-Fisher model using
                              <kbd>
                               SANTA-SIM (Version 1.0)
                              </kbd>
                              [
                              <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ] with fixed parameters: number of generations (N = 10, 000), number of sequences sampled from the population per time step (n = 15), recombination rate (ρ = 0), and variable parameters: mutation rate per site per generation, effective initial population, carrying capacity, population growth rate per generation. We considered five scenarios: In scenarios I-III we varied the mutation rate under the assumption of fixed population size, while in scenarios IV and V we investigated the effects of logistic growth of the viral population (see
                              <a id="xref-fig-14-2" class="xref-fig" href="#F14">
                               Supplementary Information Figure 6
                              </a>
                              ). The range of mutation rates in scenarios I-III were chosen such that the diversity in the simulated phylogenies are in close correspondence to the observed diversity in the GISAID dataset. While a mutation rate of μ = 0.75E − 7 substitutions per generation per site systematically underestimates the maximal distances to the root, the highest value of μ = 1.25E − 7 produces slightly larger maximal values. In fact, scenario II with μ = 1.00E − 7 reproduced the observed maximal distance accurately and provides a good approximation of the GISAID dataset (see
                              <a id="xref-fig-14-3" class="xref-fig" href="#F14">
                               Supplementary Information Figure 6
                              </a>
                              ). Major differences between simulations and the GISAID dataset are likely due to epidemiological phenomena in the ongoing pandemic such as variable population size and sequencing rate, and the spread of certain variants.
                             </p>
                             <p id="p-63">
                              For each of the simulated datasets, we computed its persistent homology in dimension one using
                              <kbd>
                               Hammingdist
                              </kbd>
                              [
                              <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ] and
                              <kbd>
                               Ripser
                              </kbd>
                              [
                              <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. In order to keep overall computational expenses at a reasonable level, we resorted to extrapolations from smaller simulated datasets to the size of the GISAID dataset with 161,024 sequences. For all scenarios we produced 100 simulations for each of the following values of the effective population p (resp. carrying capacity c): 100, 500, 1000, 2500, 5000, 7500, 10
                              <sup>
                               4
                              </sup>
                              , 10
                              <sup>
                               5
                              </sup>
                              . Additionally, we included five simulations for p = 10
                              <sup>
                               6
                              </sup>
                              to achieve a better support of the extrapolation fit. For each value of p we randomly chose 60% of the simulations as training data, used to determine the parameters of different models in a non-linear least squares fit, while the remaining 40% were reserved for later validation and comparison of the models.
                             </p>
                             <p id="p-64">
                              For each scenario we considered a quadratic, cubic, powerlaw and exponential model for the observed points (x
                              <sub>
                               i
                              </sub>
                              , y
                              <sub>
                               i
                              </sub>
                              ), and linear and powerlaw fits for the squared residuals (y
                              <sub>
                               i
                              </sub>
                              − y
                              <sub>
                               fit
                              </sub>
                              (x
                              <sub>
                               i
                              </sub>
                              ))
                              <sup>
                               2
                              </sup>
                              in the training data (see
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Supplementary Information Figure 13
                              </a>
                              ). In each model, we then used the resulting fits mean(x) and var(x) as estimators for the mean and variance of an underlying Panjer (a, b, 0)-class distribution [
                              <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                               59
                              </a>
                              ,
                              <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                               60
                              </a>
                              ]. The quantiles of the observed number of cycles in the training data fit the quantiles of the Panjer distribution with corresponding mean and variance remarkably well (see
                              <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                               Supplementary Information Figure 12
                              </a>
                              ). We then determined the likelihood L = Π
                              <sub>
                               i
                              </sub>
                              P
                              <sub>
                               Panjer
                              </sub>
                              (y = y
                              <sub>
                               i
                              </sub>
                              |mean(x
                              <sub>
                               i
                              </sub>
                              ), var(x
                              <sub>
                               i
                              </sub>
                              )) to observe the validation data {(x
                              <sub>
                               i
                              </sub>
                              , y
                              <sub>
                               i
                              </sub>
                              )}. For each model, the corresponding log-likelihoods are listed alongside the corresponding fits in
                              <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                               Supplementary Information Figure 13
                              </a>
                              . According to the log-likelihoods, the variance of the Panjer distribution is generally best described by a powerlaw behaviour. An exception is scenario II, for which the small sample of 5 simulations at p = 10
                              <sup>
                               6
                              </sup>
                              has an uncharacteristically small variance that skews the fits and corresponding likelihoods. Among the models that assume a powerlaw dependence of the variance, again with exception of scenario II, the cubic-powerlaw model yields maximum likelihoods. Finally, we determined the 95% prediction intervals for the expected numbers of random cycles by use of the cubic-powerlaw extrapolation of mean and variance of a Panjer distribution (see
                              <a id="xref-fig-14-4" class="xref-fig" href="#F14">
                               Supplementary Information Figure 6
                              </a>
                              ). The validation data of scenarios I, IV and V, which were all based on the same mutation rate, are well described by the prediction intervals of scenario I. The prediction intervals of scenario V differ significantly from the other two scenarios only at high numbers of distinct sequences. This difference arises because simulations in scenario V generally produce fewer distinct sequences than scenario I and IV, such that a steeper extrapolation is not sufficiently penalized. Hence, the prediction intervals of scenario V illustrate the error margins of the extrapolations, but are not likely to faithfully represent the expected number of one-dimensional cycles. We also observe that higher mutation rates in scenarios II and III lead to smaller numbers of one-dimensional cycles in the dataset.
                             </p>
                             <p id="p-65">
                              In conclusion, the 95% prediction interval of scenario V yields an upper bound between 1023 and 1171 expected random cycles in a dataset comparable to the GISAID dataset with 161,024 distinct sequences. Moreover, since the diversity of the GISAID dataset is better approximated by scenario II than by scenarios I, IV or V, it is reasonable to rely on the prediction interval of scenario II, which predicts that in 95% of the cases we expect between 362 and 408 random cycles (see
                              <a id="xref-fig-14-5" class="xref-fig" href="#F14">
                               Supplementary Information Figure 6
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Topological recurrence analysis
                             </h3>
                             <p id="p-66">
                              We performed topological recurrence analyses for the whole genome and for the Spike gene. Notably, the analysis can also be carried out for single genes, based on alignments of appropriately truncated genome sequences. In this case, evolutionary processes outside the specific gene are ignored, leading to the creation of more topological cycles and hence to a more detailed picture of the ongoing convergent evolution in the respective gene.
                             </p>
                             <p id="p-67">
                              Regarding the alignments covering the first year of the pandemic, we proceeded as follows: For each time bucket sub-alignment (whole genome and Spike gene) a complete list of SNV cycles (topological cycles all of whose edges correspond to single nucleotide variations) in this alignment was generated from the corresponding
                              <kbd>
                               Ripser
                              </kbd>
                              output. For each edge in an SNV cycle the endpoints of the edge correspond to a pair of uniquely determined sequences in the alignment that differ in exactly one nucleotide site position and hence determine an SNV. Then for each such SNV, its topological recurrence index (tRI) is by definition the total number of all SNV cycles containing an edge that gives rise to the given SNV. We restricted our analysis to SNVs with the following two properties: (i) one of the two nucleotides involved in the SNV agrees with the nucleotide in the reference sequence
                              <kbd>
                               EPI_ISL_402125
                              </kbd>
                              at that site position, and (ii) the SNV is isolated in the sense that at the two preceding and following site positions the nucleotides are the same as in the reference sequence. These two conditions ensure that the corresponding SAAV is uniquely determined by the SNV. We used custom code implemented in Python to compute the tRI of each such SNV for every time bucket sub-alignment (whole genome and Spike gene). Moreover, for every whole genome time bucket sub-alignment the prevalence of every SNV was computed as the quotient of the number of all sequences carrying that SNV by the number of all sequences in that sub-alignment. Note that the sub-alignments entering into this computation consisted of genetically distinct sequences. Finally, for every SNV the measurements of both tRI (whole genome and Spike gene) and prevalence for all time buckets were combined into a time series analysis chart.
                             </p>
                             <p id="p-68">
                              Even if all SNV cycles arose through random processes, it is expected that the resulting tRIs are distributed uniformly among all observed mutations. So the probability for a given mutation to have tRI ≥ k is given by a binomial distribution where the number of trials corresponds to the number of mutations in SNV cycles, and the probability for success is the inverse of the number of mutations that are realized in the dataset. From this we deduce that in the whole genome analysis a tRI ≥ 2 is highly significant (p < 0.01), while for the Spike gene analysis any signal with tRI ≥ 8 is significant (p < 0.05).
                             </p>
                             <p id="p-69">
                              An analogous analysis was carried out for the Spike gene alignment covering the year from January 2021 until January 2022. In that case, a tRI ≥ 88 is significant (p < 0.05).
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Performance analysis
                             </h3>
                             <p id="p-70">
                              We performed a basic runtime comparison between Topological Data Analysis (TDA)-based methods and standard phylogeny-based methods for random samples of up to 5,000 SARS-CoV-2 genomes drawn from the GISAID alignment covering the first year of the pandemic. We used
                              <kbd>
                               IQTree
                              </kbd>
                              [
                              <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ] to reconstruct phylogenetic trees (with default settings and fast search option). The subsequent homoplasy analysis was performed with
                              <kbd>
                               TreeTime
                              </kbd>
                              [
                              <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ] and
                              <kbd>
                               HomoplasyFinder
                              </kbd>
                              [
                              <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ] (with default settings). For the TDA-approach we used
                              <kbd>
                               Hammingdist
                              </kbd>
                              [
                              <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ] (with OpenMP multithreading disabled) to generate genetic distance matrices, and
                              <kbd>
                               Ripser
                              </kbd>
                              [
                              <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ] (with scale parameter threshold set to 2) for the subsequent computation of persistence barcodes. All computations were carried out on one kernel of an Intel Xeon E7-4870 processor. The resulting runtimes for each sample are shown in
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              .
                             </p>
                             <p id="p-71">
                              The computation of the genetic distance matrix for the whole genome alignment covering the first year of the pandemic with 303,651 sequences was carried out with
                              <kbd>
                               Hammingdist
                              </kbd>
                              [
                              <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ] (with OpenMP multithreading enabled) on a virtual machine with Intel Xeon Gold 6230R processors and 52 kernels. The runtime was 57 minutes and the memory usage was 36 gigabytes for the whole genome analysis (49 seconds and 2 gigabytes for the Spike gene analysis).
                             </p>
                             <p id="p-72">
                              The computation of the persistence barcodes for all monthly sub-alignments of the corresponding alignment with 161,024 genetically distinct sequences was carried out with
                              <kbd>
                               Ripser
                              </kbd>
                              [
                              <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ] (with scale parameter threshold set to 2) on an Intel Xeon Gold 6230R processor. Runtime and memory usage for each sub-alignment are shown in
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              . For the largest time bucket sub-alignment ranging from December 2019 to February 2021 with 161,024 genetically distinct sequences, the runtime was 45 minutes and the memory usage was 49 gigabytes for the whole genome analysis (59 seconds and 2.1 gigabytes for the Spike gene analysis).
                             </p>
                             <p id="p-73">
                              Similarly, for the Spike gene alignment with 3,928,116 sequences covering the year from Janaury 2021 until January 2022, the computation of the genetic distance matrix for 291,141 genetically distinct sequences took 4 hours, and the computation of the corresponding persistence barcodes was completed within 2.2 hours.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Ancestral state reconstruction analysis
                             </h3>
                             <p id="p-74">
                              For the study of the evolutionary histories of topologically highly recurrent mutations (see
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              ) we performed ancestral state reconstruction analyses using Mesquite (Version 3.61) [
                              <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                               61
                              </a>
                              ]. As input we used a curated alignment of 3,507 genome sequences and its corresponding Maximum-Likelihood tree, downloaded from Nextstrain [
                              <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ] on 3 March 2021. The tree was rooted using the oldest sequence available (
                              <kbd>
                               EPI_ISL_406798
                              </kbd>
                              , collected on 26 December 2019). We inferred the evolution of each amino acid of interest along this SARS-CoV-2 tree using a parsimony approach.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             Data availability
                            </h2>
                            <p id="p-75">
                             The SARS-CoV-2 genome data used in this work are available from the GISAID EpiCov Database [
                             <a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ] at
                             <a href="https://www.gisaid.org">
                              https://www.gisaid.org
                             </a>
                             and can be retrieved through the Data Acknowledgement Locator with IDs
                             <kbd>
                              EPI_SET_20220127bo
                             </kbd>
                             and
                             <kbd>
                              EPI_SET_20220124he
                             </kbd>
                             . Experimental data on viral phenotypes by Starr et al. and Greaney et al. is available from [3, Table S2] and [4, Table S3].
                            </p>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Code availability
                            </h2>
                            <p id="p-76">
                             Code used for the analyses is available at
                             <a href="https://github.com/ssciwr/hammingdist">
                              https://github.com/ssciwr/hammingdist
                             </a>
                             and
                             <a href="https://github.com/Ripser/ripser/tree/tight-representative-cycles">
                              https://github.com/Ripser/ripser/tree/tight-representative-cycles
                             </a>
                             . All other code is available from the corresponding authors upon request.
                            </p>
                           </div>
                           <div class="section data-availability" id="sec-21">
                            <h2 class="">
                             Data Availability
                            </h2>
                            <p id="p-77">
                             SARS-CoV-2 genome data used in this work are available from the GISAID EpiCov Database at www.gisaid.org.
                            </p>
                            <p id="p-78">
                             <a href="https://www.gisaid.org/">
                              https://www.gisaid.org/
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-80">
                             M.B., L.H., A.O., J.P.G., R.R. designed the study; M.B., L.H., A.O. curated data; M.B., M.C., L.H., A.O., J.P.G. performed computational analyses; U.B., M.B., L.H., A.O. developed and implemented software; M.B., L.H., A.O. acquired computing resources; M.B., L.H., A.O. drafted the manuscript; all authors contributed to the final version of the paper.
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Competing interests
                            </h2>
                            <p id="p-81">
                             R.R. is a founder of Genotwin, he is member of the Scientific Advisory Board of AimedBio and consults for Arquimea Research. The other authors declare no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Additional information
                            </h2>
                            <p id="p-82">
                             <strong>
                              Supplementary Information
                             </strong>
                             is available for this paper.
                            </p>
                            <p id="p-83">
                             <strong>
                              Correspondence and requests for materials
                             </strong>
                             should be addressed to M.B., L.H. or A.O.
                            </p>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             Supplementary Information
                            </h2>
                            <div id="F9" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F9.large.jpg?width=800&height=600&carousel=1" title="Persistent homology of the GISAID dataset. Persistence barcode representing the persistent homology in dimension one of the GISAID [14, 18] dataset covering the first year of the pandemic from December 2019 until February 2021, comprising 161,024 genetically distinct high-quality SARS-CoV-2 genomes (see Methods). Each of the 2,899 bars in the barcode corresponds to a topological cycle in the reticulate phylogeny. The rich topology of the dataset indicates a multitude of reticulate events that shaped the evolution of the virus in the course of the pandemic. A total of 1,684 bars, which is 58% of all bars, concentrate at small genetic distance scales ≤ 2 and are therefore expected to be associated mainly with homoplasic events." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Persistent homology of the GISAID dataset.</span> Persistence barcode representing the persistent homology in dimension one of the GISAID [14, 18] dataset covering the first year of the pandemic from December 2019 until February 2021, comprising 161,024 genetically distinct high-quality SARS-CoV-2 genomes (see Methods). Each of the 2,899 bars in the barcode corresponds to a topological cycle in the reticulate phylogeny. The rich topology of the dataset indicates a multitude of reticulate events that shaped the evolution of the virus in the course of the pandemic. A total of 1,684 bars, which is 58% of all bars, concentrate at small genetic distance scales ≤ 2 and are therefore expected to be associated mainly with homoplasic events.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F9.medium.gif" width="440" height="337"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 1." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F9.medium.gif" width="440" height="337"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 1.
                              </span>
                              <span class="caption-title">
                               Persistent homology of the GISAID dataset.
                              </span>
                              <p id="p-84" class="first-child">
                               Persistence barcode representing the persistent homology in dimension one of the GISAID [
                               <a id="xref-ref-14-10" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                               ,
                               <a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">
                                18
                               </a>
                               ] dataset covering the first year of the pandemic from December 2019 until February 2021, comprising 161,024 genetically distinct high-quality SARS-CoV-2 genomes (see Methods). Each of the 2,899 bars in the barcode corresponds to a topological cycle in the reticulate phylogeny. The rich topology of the dataset indicates a multitude of reticulate events that shaped the evolution of the virus in the course of the pandemic. A total of 1,684 bars, which is 58% of all bars, concentrate at small genetic distance scales ≤ 2 and are therefore expected to be associated mainly with homoplasic events.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F10.large.jpg?width=800&height=600&carousel=1" title="Topological signals of amino acid changes on the receptor-binding domain during the first year of the pandemic (December 2019 until February 2021). (A) Table of all amino acid changes with statistically significant topological recurrence index (tRI ≥ 8), plus two more selected mutations. The table provides the tRI together with its tendency, the initial acquisition date of a significant tRI signal (p < 0.05), the prevalence together with its tendency, notable Pango lineages containing the mutation [44], ACE2-binding affinity as in Starr et al. [3, Table S2], and mean plasma antibody escape as in Greaney et al. [4, Table S3] (see Methods). Mutations with rising tRI signal and mean plasma antibody escape > 0.01 (shaded) potentially confer a fitness advantage to the virus and are therefore candidates that might appear in future new variants. (B) Heatmap of all amino acid variations across the RBD showing any topological signal of convergence. There is a distinct accumulation of signals in the receptor-binding motif, while other regions on the RBD, notably residues 390-435, show only few signals of convergence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Topological signals of amino acid changes on the receptor-binding domain during the first year of the pandemic (December 2019 until February 2021).</span> (A) Table of all amino acid changes with statistically significant topological recurrence index (tRI ≥ 8), plus two more selected mutations. The table provides the tRI together with its tendency, the initial acquisition date of a significant tRI signal (<em>p</em> < 0.05), the prevalence together with its tendency, notable Pango lineages containing the mutation [44], ACE2-binding affinity as in Starr <em>et al</em>. [3, Table S2], and mean plasma antibody escape as in Greaney <em>et al</em>. [4, Table S3] (see Methods). Mutations with rising tRI signal and mean plasma antibody escape > 0.01 (shaded) potentially confer a fitness advantage to the virus and are therefore candidates that might appear in future new variants. (B) Heatmap of all amino acid variations across the RBD showing any topological signal of convergence. There is a distinct accumulation of signals in the receptor-binding motif, while other regions on the RBD, notably residues 390-435, show only few signals of convergence.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F10.medium.gif" width="230" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 2." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F10.medium.gif" width="230" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 2.
                              </span>
                              <span class="caption-title">
                               Topological signals of amino acid changes on the receptor-binding domain during the first year of the pandemic (December 2019 until February 2021).
                              </span>
                              <p id="p-85" class="first-child">
                               (A) Table of all amino acid changes with statistically significant topological recurrence index (tRI ≥ 8), plus two more selected mutations. The table provides the tRI together with its tendency, the initial acquisition date of a significant tRI signal (p < 0.05), the prevalence together with its tendency, notable Pango lineages containing the mutation [
                               <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                               ], ACE2-binding affinity as in Starr et al. [3, Table S2], and mean plasma antibody escape as in Greaney et al. [4, Table S3] (see Methods). Mutations with rising tRI signal and mean plasma antibody escape > 0.01 (shaded) potentially confer a fitness advantage to the virus and are therefore candidates that might appear in future new variants. (B) Heatmap of all amino acid variations across the RBD showing any topological signal of convergence. There is a distinct accumulation of signals in the receptor-binding motif, while other regions on the RBD, notably residues 390-435, show only few signals of convergence.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F11.large.jpg?width=800&height=600&carousel=1" title="Topological signals of Spike mutations in Variants of Interest/Concern. Comparative time series analysis charts (tRI vs. prevalence) as of February 2021 for amino acid changes on the Spike gene seen in notable lineages that have been designated as VOIs/VOCs [30]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Topological signals of Spike mutations in Variants of Interest/Concern.</span> Comparative time series analysis charts (tRI vs. prevalence) as of February 2021 for amino acid changes on the Spike gene seen in notable lineages that have been designated as VOIs/VOCs [30].</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F11.medium.gif" width="324" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 3." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F11.medium.gif" width="324" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 3.
                              </span>
                              <span class="caption-title">
                               Topological signals of Spike mutations in Variants of Interest/Concern.
                              </span>
                              <p id="p-86" class="first-child">
                               Comparative time series analysis charts (tRI vs. prevalence) as of February 2021 for amino acid changes on the Spike gene seen in notable lineages that have been designated as VOIs/VOCs [
                               <a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                               ].
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F12.large.jpg?width=800&height=600&carousel=1" title="Early warning for emerging escape mutations on the receptor-binding domain. Comparative time series analysis charts (tRI vs. prevalence) for mutations on the receptor-binding domain that showed a significant tRI signal with rising tendency in February 2021 and are associated with an increased mean plasma antibody escape > 0.01 [4]. These mutations had low prevalence < 5% and were not seen in any VOI/VOC as of February 2021 (see Supplementary Information Figure 2), but mutations at corresponding residues are likely to confer a fitness advantage to the virus and might therefore appear in future variants. In fact, the immune escape mutations S:G446S [4, 40] and S:F490S [4, 37] later appeared in the Lambda [41, 42] and Omicron [43] variants, which were designated as VOI/VOC in June/November 2021 [30]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Early warning for emerging escape mutations on the receptor-binding domain.</span> Comparative time series analysis charts (tRI vs. prevalence) for mutations on the receptor-binding domain that showed a significant tRI signal with rising tendency in February 2021 and are associated with an increased mean plasma antibody escape > 0.01 [4]. These mutations had low prevalence < 5% and were not seen in any VOI/VOC as of February 2021 (see Supplementary Information Figure 2), but mutations at corresponding residues are likely to confer a fitness advantage to the virus and might therefore appear in future variants. In fact, the immune escape mutations <kbd>S:G446S</kbd> [4, 40] and <kbd>S:F490S</kbd> [4, 37] later appeared in the Lambda [41, 42] and Omicron [43] variants, which were designated as VOI/VOC in June/November 2021 [30].</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F12.medium.gif" width="370" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 4." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F12.medium.gif" width="370" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 4.
                              </span>
                              <span class="caption-title">
                               Early warning for emerging escape mutations on the receptor-binding domain.
                              </span>
                              <p id="p-87" class="first-child">
                               Comparative time series analysis charts (tRI vs. prevalence) for mutations on the receptor-binding domain that showed a significant tRI signal with rising tendency in February 2021 and are associated with an increased mean plasma antibody escape > 0.01 [
                               <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                               ]. These mutations had low prevalence < 5% and were not seen in any VOI/VOC as of February 2021 (see
                               <a id="xref-fig-10-5" class="xref-fig" href="#F10">
                                Supplementary Information Figure 2
                               </a>
                               ), but mutations at corresponding residues are likely to confer a fitness advantage to the virus and might therefore appear in future variants. In fact, the immune escape mutations
                               <kbd>
                                S:G446S
                               </kbd>
                               [
                               <a id="xref-ref-4-12" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                               ,
                               <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                               ] and
                               <kbd>
                                S:F490S
                               </kbd>
                               [
                               <a id="xref-ref-4-13" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                               ,
                               <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                               ] later appeared in the Lambda [
                               <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ,
                               <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                               ] and Omicron [
                               <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                               ] variants, which were designated as VOI/VOC in June/November 2021 [
                               <a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                               ].
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F13" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F13.large.jpg?width=800&height=600&carousel=1" title="The Vietoris–Rips filtration of a point cloud. Each point represents a sample, and we display the geometric graphs (A), the resulting Vietoris–Rips complexes at different scales (B) and the persistence barcode in dimension one (C). If one only chooses one scale, one might either see nothing, or detect the small red cycle but miss the large yellow one, or vice versa. A solution to handle this issue is to characterize each cycle with its scale of appearance and disappearance: the red cycle induces a red bar in the barcode, and similarly for the yellow cycle." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">The Vietoris–Rips filtration of a point cloud.</span> Each point represents a sample, and we display the geometric graphs (A), the resulting Vietoris–Rips complexes at different scales (B) and the persistence barcode in dimension one (C). If one only chooses one scale, one might either see nothing, or detect the small red cycle but miss the large yellow one, or vice versa. A solution to handle this issue is to characterize each cycle with its scale of appearance and disappearance: the red cycle induces a red bar in the barcode, and similarly for the yellow cycle.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F13.medium.gif" width="440" height="265"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 5." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F13.medium.gif" width="440" height="265"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 5.
                              </span>
                              <span class="caption-title">
                               The Vietoris–Rips filtration of a point cloud.
                              </span>
                              <p id="p-88" class="first-child">
                               Each point represents a sample, and we display the geometric graphs (A), the resulting Vietoris–Rips complexes at different scales (B) and the persistence barcode in dimension one (C). If one only chooses one scale, one might either see nothing, or detect the small red cycle but miss the large yellow one, or vice versa. A solution to handle this issue is to characterize each cycle with its scale of appearance and disappearance: the red cycle induces a red bar in the barcode, and similarly for the yellow cycle.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F14" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F14.large.jpg?width=800&height=600&carousel=1" title="The number of topological features in the GISAID dataset covering the first year of the pandemic (December 2019 until February 2021) is statistically significant. (A) Comparison of genetic distances to the root in simulated data vs. distances to the Wuhan/Hu-1 reference sequence EPI_ISL_402125 in the GISAID dataset. Scenarios I, IV and V with low mutation rate systematically underestimate the maximal distance, while the highest mutation rate in scenario III yields larger distances. The mutation rate of scenario II describes the maximal distance and overall diversity well. Differences to the GISAID data are expected to be due to real-world effects like variation of population growth, belated up-take in sequencing efforts, and enhanced spread of certain variants. (B) Simulations were generated with SANTA-SIM [58] for five distinct scenarios with varying growth rate g and mutation rate μ. The 95% prediction intervals for the number of one-dimensional cycles in each scenario are based on the extrapolation of a Panjer distribution for an increasing number of distinct sequences in the simulated phylogenies (see Methods). For each scenario, the validation dataset shown in the plot is well-described by the corresponding prediction intervals. The extrapolation in the worst case scenario V predicts that less than 40% of all topological cycles (1,171 topological cycles) appear randomly in the phylogeny. (C) Parameters and prediction intervals in scenarios I-V. Scenarios I-III vary over a range of mutation rates that roughly capture the diversity of the GISAID dataset. Scenarios IV and V probe the influence of logistic population growth. For all scenarios we produced 100 simulations for each of the values of the carrying population c ≤ 105, and five simulations for c = 106." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The number of topological features in the GISAID dataset covering the first year of the pandemic (December 2019 until February 2021) is statistically significant.</span> (A) Comparison of genetic distances to the root in simulated data vs. distances to the Wuhan/Hu-1 reference sequence <kbd>EPI_ISL_402125</kbd> in the GISAID dataset. Scenarios I, IV and V with low mutation rate systematically underestimate the maximal distance, while the highest mutation rate in scenario III yields larger distances. The mutation rate of scenario II describes the maximal distance and overall diversity well. Differences to the GISAID data are expected to be due to real-world effects like variation of population growth, belated up-take in sequencing efforts, and enhanced spread of certain variants. (B) Simulations were generated with <kbd>SANTA-SIM</kbd> [58] for five distinct scenarios with varying growth rate <em>g</em> and mutation rate <em>μ</em>. The 95% prediction intervals for the number of one-dimensional cycles in each scenario are based on the extrapolation of a Panjer distribution for an increasing number of distinct sequences in the simulated phylogenies (see Methods). For each scenario, the validation dataset shown in the plot is well-described by the corresponding prediction intervals. The extrapolation in the worst case scenario V predicts that less than 40% of all topological cycles (1,171 topological cycles) appear randomly in the phylogeny. (C) Parameters and prediction intervals in scenarios I-V. Scenarios I-III vary over a range of mutation rates that roughly capture the diversity of the GISAID dataset. Scenarios IV and V probe the influence of logistic population growth. For all scenarios we produced 100 simulations for each of the values of the carrying population <em>c</em> ≤ 10<sup>5</sup>, and five simulations for <em>c</em> = 10<sup>6</sup>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F14.medium.gif" width="263" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 6." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F14.medium.gif" width="263" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 6." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 6.
                              </span>
                              <span class="caption-title">
                               The number of topological features in the GISAID dataset covering the first year of the pandemic (December 2019 until February 2021) is statistically significant.
                              </span>
                              <p id="p-89" class="first-child">
                               (A) Comparison of genetic distances to the root in simulated data vs. distances to the Wuhan/Hu-1 reference sequence
                               <kbd>
                                EPI_ISL_402125
                               </kbd>
                               in the GISAID dataset. Scenarios I, IV and V with low mutation rate systematically underestimate the maximal distance, while the highest mutation rate in scenario III yields larger distances. The mutation rate of scenario II describes the maximal distance and overall diversity well. Differences to the GISAID data are expected to be due to real-world effects like variation of population growth, belated up-take in sequencing efforts, and enhanced spread of certain variants. (B) Simulations were generated with
                               <kbd>
                                SANTA-SIM
                               </kbd>
                               [
                               <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">
                                58
                               </a>
                               ] for five distinct scenarios with varying growth rate g and mutation rate μ. The 95% prediction intervals for the number of one-dimensional cycles in each scenario are based on the extrapolation of a Panjer distribution for an increasing number of distinct sequences in the simulated phylogenies (see Methods). For each scenario, the validation dataset shown in the plot is well-described by the corresponding prediction intervals. The extrapolation in the worst case scenario V predicts that less than 40% of all topological cycles (1,171 topological cycles) appear randomly in the phylogeny. (C) Parameters and prediction intervals in scenarios I-V. Scenarios I-III vary over a range of mutation rates that roughly capture the diversity of the GISAID dataset. Scenarios IV and V probe the influence of logistic population growth. For all scenarios we produced 100 simulations for each of the values of the carrying population c ≤ 10
                               <sup>
                                5
                               </sup>
                               , and five simulations for c = 10
                               <sup>
                                6
                               </sup>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-90">
                             <strong>
                              Supplementary Information Table 7
                             </strong>
                             . External spreadsheet containing full results of the topological recurrence analysis for the whole genome in the first year of the pandemic (December 2019 until February 2021). The table lists mutations together with their topological recurrence index (tRI) and prevalence. All mutations with statistically significant tRI ≥ 2 are included.
                            </p>
                            <p id="p-91">
                             <strong>
                              Supplementary Information Table 8
                             </strong>
                             . External spreadsheet containing full results of the topological recurrence analysis for the Spike gene in the first year of the pandemic (December 2019 until February 2021). The table lists mutations together with their topological recurrence index (tRI) and prevalence. All mutations with tRI ≥ 2 are included, but only a tRI ≥ 8 is statistically significant.
                            </p>
                            <p id="p-92">
                             <strong>
                              Supplementary Information Table 9
                             </strong>
                             . External spreadsheet containing a complete list of the topological recurrence index (tRI) for all variable amino acid site positions on the Spike gene in the first year of the pandemic (December 2019 until February 2021). All variable site positions with tRI ≥ 2 are included, but only a tRI ≥ 8 is statistically significant.
                            </p>
                            <p id="p-93">
                             <strong>
                              Supplementary Information Table 10
                             </strong>
                             . External spreadsheet containing a sublist of the list in Supplementary Information Table 8 featuring all mutations on the receptor-binding domain together with their topological recurrence index (tRI) and prevalence.
                            </p>
                            <p id="p-94">
                             <strong>
                              Supplementary Information Table 11
                             </strong>
                             . External spreadsheet containing full results of the topological recurrence analysis for the Spike gene as of January 2022. The table lists mutations together with their topological recurrence index (tRI). All mutations with tRI ≥ 2 are included, but only a tRI ≥ 88 is statistically significant.
                            </p>
                            <div id="F15" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-16.large.jpg?width=800&height=600&carousel=1" title="Quantile-quantile analysis of Panjer distribution versus observed number of one-dimensional cycles in simulated phylogenies. For each value of carrying population c we determined the mean and variation of the observations and used these as parameters for the Panjer distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Quantile-quantile analysis of Panjer distribution versus observed number of one-dimensional cycles in simulated phylogenies. For each value of carrying population <em>c</em> we determined the mean and variation of the observations and used these as parameters for the Panjer distribution.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-16.medium.gif" width="370" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 12." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-16.medium.gif" width="370" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-17.large.jpg?width=800&height=600&carousel=1" title="Quantile-quantile analysis of Panjer distribution versus observed number of one-dimensional cycles in simulated phylogenies. For each value of carrying population c we determined the mean and variation of the observations and used these as parameters for the Panjer distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Quantile-quantile analysis of Panjer distribution versus observed number of one-dimensional cycles in simulated phylogenies. For each value of carrying population <em>c</em> we determined the mean and variation of the observations and used these as parameters for the Panjer distribution.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-17.medium.gif" width="326" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 12." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-17.medium.gif" width="326" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-18.large.jpg?width=800&height=600&carousel=1" title="Quantile-quantile analysis of Panjer distribution versus observed number of one-dimensional cycles in simulated phylogenies. For each value of carrying population c we determined the mean and variation of the observations and used these as parameters for the Panjer distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Quantile-quantile analysis of Panjer distribution versus observed number of one-dimensional cycles in simulated phylogenies. For each value of carrying population <em>c</em> we determined the mean and variation of the observations and used these as parameters for the Panjer distribution.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-18.medium.gif" width="440" height="247"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 12." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-18.medium.gif" width="440" height="247"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F15/graphic-18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 12.
                              </span>
                              <p id="p-95" class="first-child">
                               Quantile-quantile analysis of Panjer distribution versus observed number of one-dimensional cycles in simulated phylogenies. For each value of carrying population c we determined the mean and variation of the observations and used these as parameters for the Panjer distribution.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F16" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-19.large.jpg?width=800&height=600&carousel=1" title="Analysis of models that extrapolate the simulated data. For each scenario we fit quadratic, cubic, powerlaw, and exponential models to the observed number of one-dimensional cycles in simulations. Then we fit a linear and powerlaw model to the corresponding residuals as an estimate for the variance of the data. The quality of each model is evaluated through the log-likelihood to observe the validation dataset given a certain model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup">Analysis of models that extrapolate the simulated data. For each scenario we fit quadratic, cubic, powerlaw, and exponential models to the observed number of one-dimensional cycles in simulations. Then we fit a linear and powerlaw model to the corresponding residuals as an estimate for the variance of the data. The quality of each model is evaluated through the log-likelihood to observe the validation dataset given a certain model.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-19.medium.gif" width="384" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 13." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-19.medium.gif" width="384" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-20.large.jpg?width=800&height=600&carousel=1" title="Analysis of models that extrapolate the simulated data. For each scenario we fit quadratic, cubic, powerlaw, and exponential models to the observed number of one-dimensional cycles in simulations. Then we fit a linear and powerlaw model to the corresponding residuals as an estimate for the variance of the data. The quality of each model is evaluated through the log-likelihood to observe the validation dataset given a certain model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup">Analysis of models that extrapolate the simulated data. For each scenario we fit quadratic, cubic, powerlaw, and exponential models to the observed number of one-dimensional cycles in simulations. Then we fit a linear and powerlaw model to the corresponding residuals as an estimate for the variance of the data. The quality of each model is evaluated through the log-likelihood to observe the validation dataset given a certain model.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-20.medium.gif" width="332" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 13." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-20.medium.gif" width="332" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-21.large.jpg?width=800&height=600&carousel=1" title="Analysis of models that extrapolate the simulated data. For each scenario we fit quadratic, cubic, powerlaw, and exponential models to the observed number of one-dimensional cycles in simulations. Then we fit a linear and powerlaw model to the corresponding residuals as an estimate for the variance of the data. The quality of each model is evaluated through the log-likelihood to observe the validation dataset given a certain model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-849951000" data-figure-caption='<div class="highwire-markup">Analysis of models that extrapolate the simulated data. For each scenario we fit quadratic, cubic, powerlaw, and exponential models to the observed number of one-dimensional cycles in simulations. Then we fit a linear and powerlaw model to the corresponding residuals as an estimate for the variance of the data. The quality of each model is evaluated through the log-likelihood to observe the validation dataset given a certain model.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-21.medium.gif" width="440" height="242"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 13." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-21.medium.gif" width="440" height="242"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/14/2021.06.10.21258550/F16/graphic-21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 13.
                              </span>
                              <p id="p-96" class="first-child">
                               Analysis of models that extrapolate the simulated data. For each scenario we fit quadratic, cubic, powerlaw, and exponential models to the observed number of one-dimensional cycles in simulations. Then we fit a linear and powerlaw model to the corresponding residuals as an estimate for the variance of the data. The quality of each model is evaluated through the log-likelihood to observe the validation dataset given a certain model.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-79">
                             The authors gratefully acknowledge all data contributors, i.e. the Authors and their Originating laboratories responsible for obtaining the specimens, and their Submitting laboratories for generating the genetic sequence and metadata and sharing via the GISAID Initiative [
                             <a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], on which this research is based. Acknowledgement tables can be retrieved through the Data Acknowledgement Locator at
                             <a href="https://www.gisaid.org">
                              https://www.gisaid.org
                             </a>
                             with IDs
                             <kbd>
                              EPI_SET_20220127bo
                             </kbd>
                             and
                             <kbd>
                              EPI_SET_20220124he
                             </kbd>
                             . The authors acknowledge the use of de.NBI Cloud and the support by the High Performance and Cloud Computing Group at the Zentrum für Datenverarbeitung of the University of Tübingen and the German Federal Ministry of Education and Research (BMBF) through grant no 031 A535A. They thank M. Hanussek for IT support and early access to VALET [
                             <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                              62
                             </a>
                             ]. The authors further acknowledge support from the Interdisciplinary Center for Scientific Computing at Heidelberg University and the development work of the Scientific Software Center of Heidelberg University carried out by L. Keegan and D. Kempf. This research was supported by the DFG Collaborative Research Center SFB/TRR 109 “Discretization in Geometry and Dynamics”. M.B. was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany’s Excellence Strategy EXC 2181/1 - 390900948 (the Heidelberg STRUCTURES Excellence Cluster). L.H. thanks the Evangelisches Studienwerk Villigst for their support. A.O. acknowledges funding by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) – 281869850 (RTG 2229).
                            </p>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.1" data-doi="10.1038/d41586-021-00525-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hodcroft
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lanfear
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Want to Track Pandemic Variants Faster? Fix the Bioinformatics Bottleneck
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  591
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  30
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  33
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/d41586-021-00525-x
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D591%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fd41586-021-00525-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/d41586-021-00525-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.2" data-doi="10.1371/journal.pone.0249254">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Khudyakov
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    Y. E.
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schrörs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Riesgo-Ferreiro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sorn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Large-scale analysis of SARS-CoV-2 spike-glycoprotein mutants demonstrates the need for continuous screening of virus isolates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS ONE
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (ed
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Khudyakov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. E.
                                  </span>
                                 </span>
                                 )
                                 <span class="cit-fpage">
                                  e0249254
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pone.0249254
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D16%26rft.spage%253De0249254%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0249254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0249254&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.3" data-doi="10.1016/j.cell.2020.08.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greaney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hilton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  182
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1295
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1310
                                 </span>
                                 .e20 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2020.08.012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.08.012%26rft_id%253Dinfo%253Apmid%252F32841599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.08.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32841599&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.4" data-doi="10.1016/j.chom.2021.02.003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greaney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crawford
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Comprehensive Mapping of Mutations in the SARS-CoV-2 Receptor-Binding Domain That Affect Recognition by Polyclonal Human Plasma Antibodies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Host & Microbe
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  463
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  476
                                 </span>
                                 .e6 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.chom.2021.02.003
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2021.02.003%26rft_id%253Dinfo%253Apmid%252F33592168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.chom.2021.02.003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33592168&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.06.10.21258550v2.5" data-doi="10.1038/s41564-021-00954-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zahradník
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marciano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shemesh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SARS-CoV-2 variant prediction and antiviral drug design are enabled by RBD in vitro evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Microbiology
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1188
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1198
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41564-021-00954-4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft.volume%253D6%26rft.spage%253D1188%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-021-00954-4%26rft_id%253Dinfo%253Apmid%252F34400835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41564-021-00954-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=34400835&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.6" data-doi="10.1126/science.abf9302">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greaney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Addetia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Prospective Mapping of Viral Mutations That Escape Antibodies Used to Treat COVID-19
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  371
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  850
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  854
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.abf9302
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abf9302%26rft_id%253Dinfo%253Apmid%252F33495308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNzEvNjUzMS84NTAiO3M6NDoiYXRvbSI7czo1MDoiL21lZHJ4aXYvZWFybHkvMjAyMi8wMi8xNC8yMDIxLjA2LjEwLjIxMjU4NTUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.7" data-doi="10.1093/ve/vex042">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sagulenko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Puller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  TreeTime: Maximum-Likelihood Phylodynamic Analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Virus Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/ve/vex042
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DVirus%2BEvolution%26rft.volume%253D4%26rft.spage%253Dvex042%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvex042%26rft_id%253Dinfo%253Apmid%252F29340210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/ve/vex042&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29340210&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.8" data-doi="10.1099/mgen.0.000245">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crispell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Balaz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gordon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  HomoplasyFinder: A Simple Tool to Identify Homoplasies on a Phylogeny
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microbial Genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1099/mgen.0.000245
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrobial%2BGenomics%26rft.volume%253D5%26rft.spage%253D000245e%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmgen.0.000245%26rft_id%253Dinfo%253Apmid%252F30663960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1099/mgen.0.000245&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30663960&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.06.10.21258550v2.9" data-doi="10.1038/s41467-020-19818-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Dorp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  No Evidence for Increased Transmissibility from Recurrent Mutations in SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5986
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41467-020-19818-2
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D11%26rft.spage%253D5986%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-020-19818-2%26rft_id%253Dinfo%253Apmid%252F33239633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-020-19818-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33239633&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.10" data-doi="10.1016/j.meegid.2020.104351">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Dorp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Acman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Emergence of Genomic Diversity and Recurrent Mutations in SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Infection, Genetics and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  83
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  104351
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.meegid.2020.104351
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2020.104351%26rft_id%253Dinfo%253Apmid%252F32387564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.meegid.2020.104351&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32387564&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.06.10.21258550v2.11" data-doi="10.1101/2021.05.24.445534">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zahradník
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nunvar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schreiber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  SARS-CoV-2 Convergent Evolution as a Guide to Explore Adaptive Advantage
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.05.24.445534
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.05.24.445534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4wNS4yNC40NDU1MzR2MSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE0LzIwMjEuMDYuMTAuMjEyNTg1NTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.12" data-doi="10.1073/pnas.2104241118">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rochman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wolf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faure
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Ongoing Global and Regional Adaptive Evolution of SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e2104241118
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1073/pnas.2104241118
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2104241118%26rft_id%253Dinfo%253Apmid%252F34292871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxODoiMTE4LzI5L2UyMTA0MjQxMTE4IjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMS4wNi4xMC4yMTI1ODU1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.13" data-doi="10.1016/j.cell.2020.06.043">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Korber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fischer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gnanakaran
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Tracking Changes in SARS-CoV-2 Spike: Evidence That D614G Increases Infectivity of the COVID-19 Virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  182
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  812
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  827
                                 </span>
                                 .e19 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2020.06.043
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.06.043%26rft_id%253Dinfo%253Apmid%252F32697968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.06.043&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32697968&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.14" data-doi="10.46234/ccdcw2021.255">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Khare
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gurry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Freitas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  GISAID’s Role in Pandemic Response
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  China CDC Weekly
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1049
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1051
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.46234/ccdcw2021.255
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DChina%2BCDC%2BWeekly%26rft.volume%253D3%26rft.spage%253D1049%26rft_id%253Dinfo%253Adoi%252F10.46234%252Fccdcw2021.255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.46234/ccdcw2021.255&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.15" data-doi="10.1093/molbev/msaa314">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Malik
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    H.
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barbera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Czech
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Phylogenetic Analysis of SARS-CoV-2 Data Is Difficult
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 (ed
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Malik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ) msaa314 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/molbev/msaa314
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsaa314%26rft_id%253Dinfo%253Apmid%252F33316067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msaa314&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33316067&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.16" data-doi="10.1093/bioinformatics/bty407">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hadfield
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Megill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Nextstrain: Real-Time Tracking of Pathogen Evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4121
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4123
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/bioinformatics/bty407
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty407%26rft_id%253Dinfo%253Apmid%252F29790939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty407&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29790939&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.17" data-doi="10.1371/journal.pgen.1009175">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Turakhia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thornlow
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Stability of SARS-CoV-2 Phylogenies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1009175
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pgen.1009175
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenetics%26rft.volume%253D16%26rft.spage%253De1009175%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1009175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1009175&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.18" data-doi="10.2807/1560-7917.ES.2017.22.13.30494">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCauley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  GISAID: Global Initiative on Sharing All Influenza Data – from Vision to Reality
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Eurosurveillance
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.2807/1560-7917.ES.2017.22.13.30494
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEurosurveillance%26rft.volume%253D22%26rft.spage%253Dpii30494%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2017.22.13.30494%26rft_id%253Dinfo%253Apmid%252F28382917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2017.22.13.30494&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28382917&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.19" data-doi="10.1073/pnas.1313480110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carlsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rabadan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Topology of Viral Evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  18566
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18571
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1073/pnas.1313480110
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1313480110%26rft_id%253Dinfo%253Apmid%252F24170857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ2LzE4NTY2IjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMS4wNi4xMC4yMTI1ODU1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.20" data-doi="10.1007/s41468-021-00071-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bauer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Ripser: efficient computation of Vietoris-Rips persistence barcodes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Applied and Computational Topology
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1007/s41468-021-00071-5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2Band%2BComputational%2BTopology%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs41468-021-00071-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s41468-021-00071-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.21" data-doi="10.1093/molbev/msaa015">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Minh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schmidt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chernomor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1530
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1534
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/molbev/msaa015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsaa015%26rft_id%253Dinfo%253Apmid%252F32011700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msaa015&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32011700&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.06.10.21258550v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keegan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kempf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Hammingdist: A Fast Tool to Calculate Hamming Distances version 0.13.0
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                 <a href="https://github.com/ssciwr/hammingdist">
                                  https://github.com/ssciwr/hammingdist
                                 </a>
                                 visited on 2021-12-01.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.06.10.21258550v2.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Borges
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-source">
                                  Issues with SARS-CoV-2 Sequencing Data - SARS-CoV-2 Coronavirus / nCoV-2019 Genomic Epidemiology Virological
                                 </span>
                                 .
                                 <a href="https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473">
                                  https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.24" data-doi="10.1038/s41401-020-0485-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.-f.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Structural and Functional Properties of SARS-CoV-2 Spike Protein: Potential Antivirus Drug Development for COVID-19
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Acta Pharmacologica Sinica
                                 </abbr>
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1141
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1149
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41401-020-0485-4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DActa%2BPharmacologica%2BSinica%26rft.volume%253D41%26rft.spage%253D1141%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41401-020-0485-4%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41401-020-0485-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.25" data-doi="10.1038/s41586-021-03237-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bailey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Loss of furin cleavage site attenuates SARS-CoV-2 pathogenesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  591
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  293
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  299
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41586-021-03237-4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-021-03237-4%26rft_id%253Dinfo%253Apmid%252F33494095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-021-03237-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33494095&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.26" data-doi="10.1016/j.cell.2020.07.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  182
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1284
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1294
                                 </span>
                                 .e9 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2020.07.012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.07.012%26rft_id%253Dinfo%253Apmid%252F32730807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.07.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32730807&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.27" data-doi="10.1038/s41586-020-2895-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Plante
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Spike Mutation D614G Alters SARS-CoV-2 Fitness
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  592
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  116
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  121
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41586-020-2895-3
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2895-3%26rft_id%253Dinfo%253Apmid%252F33106671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2895-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33106671&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.06.10.21258550v2.28" data-doi="10.1126/science.abe8499">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chiba
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Halfmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SARS-CoV-2 D614G Variant Exhibits Efficient Replication Ex Vivo and Transmission in Vivo
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  eabe8499
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.abe8499
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abe8499%26rft_id%253Dinfo%253Apmid%252F33184236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNzAvNjUyMy8xNDY0IjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMS4wNi4xMC4yMTI1ODU1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.29" data-doi="10.1016/j.cell.2020.09.032">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yurkovetskiy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pascal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  183
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  739
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  751
                                 </span>
                                 .e8 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2020.09.032
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.09.032%26rft_id%253Dinfo%253Apmid%252F32991842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.09.032&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32991842&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.06.10.21258550v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-source">
                                  Tracking SARS-CoV-2 Variants World Health Organization
                                 </span>
                                 .
                                 <a href="https://www.who.int/activities/tracking-SARS-CoV-2-variants">
                                  https://www.who.int/activities/tracking-SARS-CoV-2-variants
                                 </a>
                                 visited on 2022-01-25.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.31" data-doi="10.1038/s41598-021-83761-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steinkellner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Köchl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Serine 477 Plays a Crucial Role in the Interaction of the SARS-CoV-2 Spike Protein with the Human Receptor ACE2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4320
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41598-021-83761-5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D11%26rft.spage%253D4320%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-021-83761-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-021-83761-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.32" data-doi="10.1038/s41586-021-03677-y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hodcroft
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zuber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nadeau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Spread of a SARS-CoV-2 Variant through Europe in the Summer of 2020
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41586-021-03677-y
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-021-03677-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-021-03677-y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.33" data-doi="10.1093/ve/veaa094">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Welkers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. R. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reusken
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. B. E. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eggink
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Possible Host-Adaptation of SARS-CoV-2 Due to Improved ACE2 Receptor Binding in Mink
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Virus Evolution
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/ve/veaa094
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DVirus%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fveaa094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/ve/veaa094&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.06.10.21258550v2.34" data-doi="10.1126/science.abe5901">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oude Munnink
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sikkema
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nieuwenhuijse
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Transmission of SARS-CoV-2 on Mink Farms between Humans and Mink and Back to Humans
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  371
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  172
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  177
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.abe5901
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abe5901%26rft_id%253Dinfo%253Apmid%252F33172935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNzEvNjUyNS8xNzIiO3M6NDoiYXRvbSI7czo1MDoiL21lZHJ4aXYvZWFybHkvMjAyMi8wMi8xNC8yMDIxLjA2LjEwLjIxMjU4NTUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.35" data-doi="10.1101/2020.11.16.384743">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Dorp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Recurrent Mutations in SARS-CoV-2 Genomes Isolated from Mink Point to Rapid Host-Adaptation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2020.11.16.384743
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.11.16.384743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4xMS4xNi4zODQ3NDN2MiI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE0LzIwMjEuMDYuMTAuMjEyNTg1NTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.36" data-doi="10.1016/j.cell.2021.03.028">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCallum
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Marco
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lempp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  184
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2332
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2347
                                 </span>
                                 .e16 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2021.03.028
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2021.03.028%26rft_id%253Dinfo%253Apmid%252F33761326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2021.03.028&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33761326&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.37" data-doi="10.1016/j.chom.2021.01.014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   VanBlargan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bloyet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.-M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Identification of SARS-CoV-2 spike mutations that attenuate monoclonal and serum antibody neutralization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Host & Microbe
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  477
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  488
                                 </span>
                                 .e4 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.chom.2021.01.014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2021.01.014%26rft_id%253Dinfo%253Apmid%252F33535027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.chom.2021.01.014&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33535027&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.38" data-doi="10.1101/2021.01.06.20248960">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haynes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamath
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lucas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Impact of B.1.1.7 Variant Mutations on Antibody Recognition of Linear SARS-CoV-2 Epitopes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  medRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.01.06.20248960
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DmedRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.01.06.20248960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoibWVkcnhpdiI7czo1OiJyZXNpZCI7czoyMToiMjAyMS4wMS4wNi4yMDI0ODk2MHYxIjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMS4wNi4xMC4yMTI1ODU1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.39" data-doi="10.1101/2021.11.09.467693">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lista
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Winstone
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The P681H mutation in the Spike glycoprotein confers Type I interferon resistance in the SARS-CoV-2 alpha (B.1.1.7) variant
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.11.09.467693
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.11.09.467693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4xMS4wOS40Njc2OTN2MSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE0LzIwMjEuMDYuMTAuMjEyNTg1NTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.40" data-doi="10.1101/2021.12.14.472719">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iketani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Striking Antibody Evasion Manifested by the Omicron Variant of SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.12.14.472719
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.12.14.472719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4xMi4xNC40NzI3MTl2MyI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE0LzIwMjEuMDYuMTAuMjEyNTg1NTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.41" data-doi="10.1128/Spectrum.00789-21">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  ed
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Mostafa
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    H. H.
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Romero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dávila-Barclay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salvatierra
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Emergence of Sars-CoV-2 Variant Lambda (C.37) in South America
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Microbiology Spectrum
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (ed
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Mostafa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. H.
                                  </span>
                                 </span>
                                 )
                                 <span class="cit-fpage">
                                  e00789
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  21
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1128/Spectrum.00789-21
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMicrobiology%2BSpectrum%26rft.volume%253D9%26rft.spage%253D00789e%26rft_id%253Dinfo%253Adoi%252F10.1128%252FSpectrum.00789-21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1128/Spectrum.00789-21&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.42" data-doi="10.1016/j.celrep.2021.110218">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kimura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kosugi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The SARS-CoV-2 Lambda variant exhibits enhanced infectivity and immune resistance
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Reports
                                 </abbr>
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  110218
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.celrep.2021.110218
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D38%26rft.spage%253D110218%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2021.110218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.celrep.2021.110218&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.43" data-doi="10.1038/s41586-022-04411-y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Viana
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moyo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amoako
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Rapid epidemic expansion of the SARS-CoV-2 Omicron variant in southern Africa
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41586-022-04411-y
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-022-04411-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-022-04411-y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.44" data-doi="10.1038/s41564-020-0770-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rambaut
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’Toole
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Á.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A Dynamic Nomenclature Proposal for SARS-CoV-2 Lineages to Assist Genomic Epidemiology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Microbiology
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1403
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1407
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41564-020-0770-5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-020-0770-5%26rft_id%253Dinfo%253Apmid%252F32669681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41564-020-0770-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32669681&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.06.10.21258550v2.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-source">
                                  COVID-19 Weekly Epidemiological Update, Edition 76 World Health Organization
                                 </span>
                                 .
                                 <a href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---25-january-2022">
                                  https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---25-january-2022
                                 </a>
                                 visited on 2022-01-25.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.46" data-doi="10.1101/2022.01.03.21268111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Willett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grove
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MacLean
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The hyper-transmissible SARS-CoV-2 Omicron variant exhibits significant antigenic change, vaccine escape and a switch in cell entry mechanism
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  medRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2022.01.03.21268111
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DmedRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2022.01.03.21268111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoibWVkcnhpdiI7czo1OiJyZXNpZCI7czoyMToiMjAyMi4wMS4wMy4yMTI2ODExMXYyIjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMS4wNi4xMC4yMTI1ODU1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.47" data-doi="10.1126/science.acx9738">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Where did ‘weird’ Omicron come from?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  374
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1179
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.acx9754
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D374%26rft.spage%253D1179%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.acx9738%26rft_id%253Dinfo%253Apmid%252F34855502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1126/science.acx9738&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=34855502&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.48" data-doi="10.1038/d41586-021-03619-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Callaway
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Beyond Omicron: what’s next for COVID’s viral evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  600
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  204
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  207
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/d41586-021-03619-8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D600%26rft.spage%253D204%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fd41586-021-03619-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/d41586-021-03619-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.49" data-doi="10.1007/s00454-006-1276-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen-Steiner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edelsbrunner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Stability of Persistence Diagrams
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Discrete & Computational Geometry
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  103
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  120
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1007/s00454-006-1276-5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDiscrete%2B%2526%2BComputational%2BGeometry%26rft.volume%253D37%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00454-006-1276-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00454-006-1276-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000243074400007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.50" data-doi="10.1101/2020.05.28.118992">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lythgoe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferretti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Shared SARS-CoV-2 Diversity Suggests Localised Transmission of Minority Variants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2020.05.28.118992
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.05.28.118992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wNS4yOC4xMTg5OTJ2NCI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE0LzIwMjEuMDYuMTAuMjEyNTg1NTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.51" data-doi="10.1126/science.abg0821">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lythgoe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferretti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SARS-CoV-2 within-host diversity and transmission
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  372
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  eabg0821
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.abg0821
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abg0821%26rft_id%253Dinfo%253Apmid%252F33688063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNzIvNjUzOS9lYWJnMDgyMSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE0LzIwMjEuMDYuMTAuMjEyNTg1NTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.52" data-doi="10.1038/s41392-021-00623-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jeong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.-C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  SARS-CoV-2 Mutations, Vaccines, and Immunity: Implication of Variants of Concern
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Signal Transduction and Targeted Therapy
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  203
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41392-021-00623-2
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSignal%2BTransduction%2Band%2BTargeted%2BTherapy%26rft.volume%253D6%26rft.spage%253D203%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41392-021-00623-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41392-021-00623-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.53" data-doi="10.1093/nar/gkh340">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edgar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  MUSCLE: Multiple Sequence Alignment with High Accuracy and High Throughput
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1792
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1797
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/nar/gkh340
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15034147&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000220487200025&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.54" data-doi="10.1093/nar/gkf436">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katoh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3059
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3066
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/nar/gkf436
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3059%26rft.epage%253D3066%26rft.atitle%253DMAFFT%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Brapid%2Bmultiple%2Bsequence%2Balignment%2Bbased%2Bon%2Bfast%2BFourier%2Btransform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf436%26rft_id%253Dinfo%253Apmid%252F12136088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkf436&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12136088&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F14%2F2021.06.10.21258550.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000177154300016&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2021.06.10.21258550v2.55" data-doi="10.1007/978-1-4613-9586-7_3">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Gersten
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    S. M.
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gromov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  in Essays in Group Theory (ed
                                 </span>
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Gersten
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. M.
                                  </span>
                                 </span>
                                 )
                                 <span class="cit-fpage">
                                  75
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  263
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  New York, NY
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1987
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1007/978-1-4613-9586-7_3
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253Din%2BEssays%2Bin%2BGroup%2BTheory%2B%2528ed%26rft.volume%253D8%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4613-9586-7_3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/978-1-4613-9586-7_3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               56.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.56" data-doi="10.1016/j.cagd.2019.06.003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edelsbrunner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 & Ö
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   lsböck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Holes and Dependences in an Ordered Complex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Computer Aided Geometric Design
                                 </abbr>
                                 <span class="cit-vol">
                                  73
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cagd.2019.06.003
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComputer%2BAided%2BGeometric%2BDesign%26rft.volume%253D73%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cagd.2019.06.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cagd.2019.06.003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               57.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.57" data-doi="10.1007/s00454-004-1146-y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zomorodian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carlsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Computing Persistent Homology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Discrete & Computational Geometry
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  249
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  274
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1007/s00454-004-1146-y
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDiscrete%2B%2526%2BComputational%2BGeometry%26rft.volume%253D33%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00454-004-1146-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00454-004-1146-y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               58.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.58" data-doi="10.1093/ve/vez003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jariani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Warth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deforche
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  SANTA-SIM: Simulating Viral Sequence Evolution Dynamics under Selection and Recombination
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Virus Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/ve/vez003
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DVirus%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvez003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/ve/vez003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               59.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.59" data-doi="10.1017/S0515036100006796">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Panjer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Recursive Evaluation of a Family of Compound Distributions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ASTIN Bulletin
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  22
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  26
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1981
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1017/S0515036100006796
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DASTIN%2BBulletin%26rft.volume%253D12%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0515036100006796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1017/S0515036100006796&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               60.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.06.10.21258550v2.60" data-doi="10.1017/S0515036100006802">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sundt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jewell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Further Results on Recursive Evaluation of Compound Distributions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ASTIN Bulletin: The Journal of the IAA
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  27
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  39
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1981
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1017/S0515036100006802
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DASTIN%2BBulletin%253A%2BThe%2BJournal%2Bof%2Bthe%2BIAA%26rft.volume%253D12%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0515036100006802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1017/S0515036100006802&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               61.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.06.10.21258550v2.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddison
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddison
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Mesquite: A Modular System for Evolutionary Analysis.version 3.61
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                 <a href="http://www.mesquiteproject.org">
                                  http://www.mesquiteproject.org
                                 </a>
                                 visited on 2021-06-01.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               62.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.06.10.21258550v2.62">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanussek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  VALET
                                 </span>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                 <a href="https://github.com/MaximilianHanussek/VALET">
                                  https://github.com/MaximilianHanussek/VALET
                                 </a>
                                 visited on 2021-06-01.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2021.06.10.21258550v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.11.27.21266822v2" title="KRAS mutations impact clinical outcome in metastatic non-small cell lung cancer" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.02.08.22269319v2" title="Estimation of COVID-19 cases prevented by vaccination in California" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted February 14, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2021.06.10.21258550v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2021.06.10.21258550v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/02/14/2021.06.10.21258550.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2021.06.10.21258550v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about medRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2021.06.10.21258550v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from medRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the medRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/476466"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-oRw_4TyXKVMCDepuC68IxIkBNhBYFFRknQW-KPzbzOA"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="277157509"/>
                                   <input type="hidden" name="captcha_token" value="c427d0bd0c1c95e4abfa7b21e80c51de"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node476466--3" data-node-nid="476466" id="node-476466--4297658959" data-pisa="medrxiv;2021.06.10.21258550v2" data-pisa-master="medrxiv;2021.06.10.21258550" data-seqnum="476466" data-apath="/medrxiv/early/2022/02/14/2021.06.10.21258550.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Michael
                                  </span>
                                  <span class="nlm-surname">
                                   Bleher
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Lukas
                                  </span>
                                  <span class="nlm-surname">
                                   Hahn
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Juan Ángel
                                  </span>
                                  <span class="nlm-surname">
                                   Patiño-Galindo
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Mathieu
                                  </span>
                                  <span class="nlm-surname">
                                   Carrière
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Ulrich
                                  </span>
                                  <span class="nlm-surname">
                                   Bauer
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Raúl
                                  </span>
                                  <span class="nlm-surname">
                                   Rabadán
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Andreas
                                  </span>
                                  <span class="nlm-surname">
                                   Ott
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 medRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2021.06.10.21258550;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2021.06.10.21258550
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2021.06.10.21258550v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&title=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&title=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&text=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&t=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&title=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&title=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2&summary=&source=medRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&title=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node476466--5" data-node-nid="476466" id="citation-node-476466--62060241120" data-pisa="medrxiv;2021.06.10.21258550v2" data-pisa-master="medrxiv;2021.06.10.21258550" data-seqnum="476466" data-apath="/medrxiv/early/2022/02/14/2021.06.10.21258550.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Topological data analysis identifies emerging adaptive mutations in SARS-CoV-2
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Michael
                                      </span>
                                      <span class="nlm-surname">
                                       Bleher
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Lukas
                                      </span>
                                      <span class="nlm-surname">
                                       Hahn
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Juan Ángel
                                      </span>
                                      <span class="nlm-surname">
                                       Patiño-Galindo
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Mathieu
                                      </span>
                                      <span class="nlm-surname">
                                       Carrière
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Ulrich
                                      </span>
                                      <span class="nlm-surname">
                                       Bauer
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Raúl
                                      </span>
                                      <span class="nlm-surname">
                                       Rabadán
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Andreas
                                      </span>
                                      <span class="nlm-surname">
                                       Ott
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     medRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2021.06.10.21258550;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2021.06.10.21258550
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=medRxiv&rft.au=Bleher%2C+Michael&rft.au=Hahn%2C+Lukas&rft.au=Pati%C3%B1o-Galindo%2C+Juan+%C3%81ngel&rft.au=Carri%C3%A8re%2C+Mathieu&rft.au=Bauer%2C+Ulrich&rft.au=Rabad%C3%A1n%2C+Ra%C3%BAl&rft.au=Ott%2C+Andreas">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/476466/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/476466/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/476466/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/476466/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/476466/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/476466/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/476466/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/476466/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/476466/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/476466/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/476466/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/476466/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&count=horizontal&via=&text=Topological%20data%20analysis%20identifies%20emerging%20adaptive%20mutations%20in%20SARS-CoV-2&counturl=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2021.06.10.21258550v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.medrxiv.org/content/10.1101/2021.06.10.21258550v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/epidemiology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Epidemiology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Addiction Medicine
                          </a>
                          <span class="article-count">
                           (218)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Allergy and Immunology
                          </a>
                          <span class="article-count">
                           (496)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">
                           Anesthesia
                          </a>
                          <span class="article-count">
                           (106)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Cardiovascular Medicine
                          </a>
                          <span class="article-count">
                           (1121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Dentistry and Oral Medicine
                          </a>
                          <span class="article-count">
                           (198)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">
                           Dermatology
                          </a>
                          <span class="article-count">
                           (141)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Emergency Medicine
                          </a>
                          <span class="article-count">
                           (275)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">
                           Endocrinology (including Diabetes Mellitus and Metabolic Disease)
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (9829)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Forensic Medicine
                          </a>
                          <span class="article-count">
                           (5)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">
                           Gastroenterology
                          </a>
                          <span class="article-count">
                           (484)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Genetic and Genomic Medicine
                          </a>
                          <span class="article-count">
                           (2344)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Geriatric Medicine
                          </a>
                          <span class="article-count">
                           (225)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Economics
                          </a>
                          <span class="article-count">
                           (465)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Informatics
                          </a>
                          <span class="article-count">
                           (1574)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">
                           Health Policy
                          </a>
                          <span class="article-count">
                           (738)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">
                           Health Systems and Quality Improvement
                          </a>
                          <span class="article-count">
                           (609)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">
                           Hematology
                          </a>
                          <span class="article-count">
                           (239)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">
                           HIV/AIDS
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">
                           Infectious Diseases (except HIV/AIDS)
                          </a>
                          <span class="article-count">
                           (11686)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Intensive Care and Critical Care Medicine
                          </a>
                          <span class="article-count">
                           (617)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Education
                          </a>
                          <span class="article-count">
                           (242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Ethics
                          </a>
                          <span class="article-count">
                           (67)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">
                           Nephrology
                          </a>
                          <span class="article-count">
                           (259)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">
                           Neurology
                          </a>
                          <span class="article-count">
                           (2170)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">
                           Nursing
                          </a>
                          <span class="article-count">
                           (134)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">
                           Nutrition
                          </a>
                          <span class="article-count">
                           (340)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">
                           Obstetrics and Gynecology
                          </a>
                          <span class="article-count">
                           (427)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">
                           Occupational and Environmental Health
                          </a>
                          <span class="article-count">
                           (521)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">
                           Oncology
                          </a>
                          <span class="article-count">
                           (1188)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">
                           Ophthalmology
                          </a>
                          <span class="article-count">
                           (368)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">
                           Orthopedics
                          </a>
                          <span class="article-count">
                           (129)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">
                           Otolaryngology
                          </a>
                          <span class="article-count">
                           (221)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Pain Medicine
                          </a>
                          <span class="article-count">
                           (148)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Palliative Medicine
                          </a>
                          <span class="article-count">
                           (50)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (315)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">
                           Pediatrics
                          </a>
                          <span class="article-count">
                           (702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Therapeutics
                          </a>
                          <span class="article-count">
                           (304)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">
                           Primary Care Research
                          </a>
                          <span class="article-count">
                           (269)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">
                           Psychiatry and Clinical Psychology
                          </a>
                          <span class="article-count">
                           (2201)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">
                           Public and Global Health
                          </a>
                          <span class="article-count">
                           (4705)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">
                           Radiology and Imaging
                          </a>
                          <span class="article-count">
                           (786)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">
                           Rehabilitation Medicine and Physical Therapy
                          </a>
                          <span class="article-count">
                           (462)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Respiratory Medicine
                          </a>
                          <span class="article-count">
                           (626)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">
                           Rheumatology
                          </a>
                          <span class="article-count">
                           (276)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">
                           Sexual and Reproductive Health
                          </a>
                          <span class="article-count">
                           (227)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Sports Medicine
                          </a>
                          <span class="article-count">
                           (214)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">
                           Surgery
                          </a>
                          <span class="article-count">
                           (254)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Toxicology
                          </a>
                          <span class="article-count">
                           (43)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">
                           Transplantation
                          </a>
                          <span class="article-count">
                           (121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">
                           Urology
                          </a>
                          <span class="article-count">
                           (94)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
