<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.05.23.112508v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.05.23.112508v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.05.23.112508v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/13/2020.05.23.112508.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.05.23.112508v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations"/>
  <meta name="DC.Identifier" content="10.1101/2020.05.23.112508"/>
  <meta name="DC.Date" content="2022-03-13"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="A general method to infer both positive and purifying selection during the real-time evolution of hypermutator pathogens would be broadly useful. To this end, we introduce a simple test to infer mode of selection (STIMS) from metagenomic time series of evolving microbial populations. We test STIMS on metagenomic data generated by simulations of bacterial evolution, and on metagenomic data spanning 62,750 generations of Lenski’s long-term evolution experiment with Escherichia coli (LTEE). In both cases, STIMS recovers signals of positive and purifying selection on gold standard sets of genes. Using STIMS, we find strong evidence of ongoing positive selection on key regulators of the E. coli gene regulatory network, even in some hypermutator populations. STIMS also detects positive selection on regulatory genes in hypermutator populations of Pseudomonas aeruginosa that adapted to subinhibitory concentrations of colistin – an antibiotic of last resort – for just twenty-six days of laboratory evolution. Our results show that the fine-tuning of gene regulatory networks is a general mechanism for rapid and ongoing adaptation. The simplicity of STIMS, together with its intuitive visual interpretation, make it a useful test for positive and purifying selection in metagenomic data sets that track the evolution of hypermutator populations in real-time.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Rohan Maddamsetti"/>
  <meta name="DC.Contributor" content="Nkrumah A. Grant"/>
  <meta name="article:published_time" content="2022-03-13"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations"/>
  <meta name="citation_abstract" lang="en" content="<h3>ABSTRACT</h3>
<p>A general method to infer both positive and purifying selection during the real-time evolution of hypermutator pathogens would be broadly useful. To this end, we introduce a simple test to infer mode of selection (STIMS) from metagenomic time series of evolving microbial populations. We test STIMS on metagenomic data generated by simulations of bacterial evolution, and on metagenomic data spanning 62,750 generations of Lenski’s long-term evolution experiment with <i>Escherichia coli</i> (LTEE). In both cases, STIMS recovers signals of positive and purifying selection on gold standard sets of genes. Using STIMS, we find strong evidence of ongoing positive selection on key regulators of the <i>E. coli</i> gene regulatory network, even in some hypermutator populations. STIMS also detects positive selection on regulatory genes in hypermutator populations of <i>Pseudomonas aeruginosa</i> that adapted to subinhibitory concentrations of colistin – an antibiotic of last resort – for just twenty-six days of laboratory evolution. Our results show that the fine-tuning of gene regulatory networks is a general mechanism for rapid and ongoing adaptation. The simplicity of STIMS, together with its intuitive visual interpretation, make it a useful test for positive and purifying selection in metagenomic data sets that track the evolution of hypermutator populations in real-time.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.05.23.112508v2"/>
  <meta name="citation_id" content="2020.05.23.112508v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.05.23.112508v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.23.112508v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.23.112508v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.05.23.112508"/>
  <meta name="citation_num_pages" content="56"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.05.23.112508"/>
  <meta name="citation_author" content="Rohan Maddamsetti"/>
  <meta name="citation_author_institution" content="Department of Biomedical Engineering, Duke University"/>
  <meta name="citation_author_email" content="rohan.maddamsetti@gmail.com"/>
  <meta name="citation_author_email" content="nkrumah.grant88@gmail.com"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3370-092X"/>
  <meta name="citation_author" content="Nkrumah A. Grant"/>
  <meta name="citation_author_institution" content="BEACON Center for the Study of Evolution in Action, Michigan State University"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, Michigan State University"/>
  <meta name="citation_author_institution" content="Department of Biological Sciences, University of Idaho"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4555-5283"/>
  <meta name="citation_reference" content="Jeggo PA, Pearl LH, Carr AM. DNA repair, genome stability and cancer: a historical perspective. Nature Reviews Cancer. 2016;16(1):35."/>
  <meta name="citation_reference" content="Loeb LA. A mutator phenotype in cancer. Cancer research. 2001;61(8):3230–9."/>
  <meta name="citation_reference" content="Modrich P. Mismatch repair, genetic stability, and cancer. Science. 1994;266(5193):1959-61."/>
  <meta name="citation_reference" content="Feliziani S, Marvig RL, Lujan AM, Moyano AJ, Di Rienzo JA, Johansen HK, et al. Coexistence and within-host evolution of diversified lineages of hypermutable Pseudomonas aeruginosa in long-term cystic fibrosis infections. PLoS Genet. 2014;10(10):e1004651."/>
  <meta name="citation_reference" content="Ford CB, Shah RR, Maeda MK, Gagneux S, Murray MB, Cohen T, et al. Mycobacterium tuberculosis mutation rate estimates from different lineages predict substantial differences in the emergence of drug-resistant tuberculosis. Nature genetics. 2013;45(7):784–90."/>
  <meta name="citation_reference" content="Couce A, Caudwell LV, Feinauer C, Hindré T, Feugeas J-P, Weigt M, et al. Mutator genomes decay, despite sustained fitness gains, in a long-term experiment with bacteria. Proceedings of the National Academy of Sciences. 2017;114(43):E9026–E35."/>
  <meta name="citation_reference" content="Schiffels S, Szöllősi GJ, Mustonen V, Lässig M. Emergent neutrality in adaptive asexual evolution. Genetics. 2011;189(4):1361–75."/>
  <meta name="citation_reference" content="Mehta HH, Prater AG, Beabout K, Elworth RA, Karavis M, Gibbons HS, et al. The essential role of hypermutation in rapid adaptation to antibiotic stress. Antimicrobial agents and chemotherapy. 2019;63(7):e00744–19."/>
  <meta name="citation_reference" content="Hammerstrom TG, Beabout K, Clements TP, Saxer G, Shamoo Y. Acinetobacter baumannii repeatedly evolves a hypermutator phenotype in response to tigecycline that effectively surveys evolutionary trajectories to resistance. PloS one. 2015;10(10):e0140489."/>
  <meta name="citation_reference" content="Lenski RE, Rose MR, Simpson SC, Tadler SC. Long-term experimental evolution in Escherichia coli. I. Adaptation and divergence during 2,000 generations. The American Naturalist. 1991;138(6):1315-41."/>
  <meta name="citation_reference" content="Lenski RE. Experimental evolution and the dynamics of adaptation and genome evolution in microbial populations. The ISME journal. 2017;11(10):2181–94."/>
  <meta name="citation_reference" content="Barrick JE, Lenski RE. Genome-wide mutational diversity in an evolving population of Escherichia coli. Cold Spring Harbor symposia on quantitative biology. 2009;74:119–29."/>
  <meta name="citation_reference" content="Barrick JE, Yu DS, Yoon SH, Jeong H, Oh TK, Schneider D, et al. Genome evolution and adaptation in a long-term experiment with Escherichia coli. Nature. 2009;461(7268):1243-7."/>
  <meta name="citation_reference" content="Maddamsetti R, Lenski RE, Barrick JE. Adaptation, clonal interference, and frequency-dependent interactions in a long-term evolution experiment with Escherichia coli. Genetics. 2015;200(2):619–31."/>
  <meta name="citation_reference" content="Tenaillon O, Barrick JE, Ribeck N, Deatherage DE, Blanchard JL, Dasgupta A, et al. Tempo and mode of genome evolution in a 50,000-generation experiment. Nature. 2016;536(7615):165-70."/>
  <meta name="citation_reference" content="Good BH, McDonald MJ, Barrick JE, Lenski RE, Desai MM. The dynamics of molecular evolution over 60,000 generations. Nature. 2017;551(7678):45-50."/>
  <meta name="citation_reference" content="Vasi F, Travisano M, Lenski RE. Long-term experimental evolution in Escherichia coli. II. Changes in life-history traits during adaptation to a seasonal environment. The american naturalist. 1994;144(3):432–56."/>
  <meta name="citation_reference" content="Lenski RE, Mongold JA, Sniegowski PD, Travisano M, Vasi F, Gerrish PJ, et al. Evolution of competitive fitness in experimental populations of E. coli: what makes one genotype a better competitor than another? Antonie van Leeuwenhoek. 1998;73(1):35–47."/>
  <meta name="citation_reference" content="Elena SF, Cooper VS, Lenski RE. Punctuated evolution caused by selection of rare beneficial mutations. Science. 1996;272(5269):1802-4."/>
  <meta name="citation_reference" content="Wiser MJ, Ribeck N, Lenski RE. Long-term dynamics of adaptation in asexual populations. Science. 2013;342(6164):1364-7."/>
  <meta name="citation_reference" content="Grant NA, Maddamsetti R, Lenski RE. Maintenance of metabolic plasticity despite relaxed selection in a long-term evolution experiment with Escherichia coli. The American Naturalist. 2021;0(ja):null. doi: 10.1086/714530."/>
  <meta name="citation_reference" content="Grant NA, Magid AA, Franklin J, Dufour Y, Lenski RE. Changes in Cell Size and Shape During 50,000 Generations of Experimental Evolution with Escherichia coli. Journal of Bacteriology. 2021;203(10)."/>
  <meta name="citation_reference" content="Maddamsetti R, Hatcher PJ, Green AG, Williams BL, Marks DS, Lenski RE. Core genes evolve rapidly in the long-term evolution experiment with Escherichia coli. Genome biology and evolution. 2017;9(4):1072–83."/>
  <meta name="citation_reference" content="Maddamsetti R. Universal constraints on protein evolution in the long-term evolution experiment with Escherichia coli. Genome Biology and Evolution. 2021. doi: 10.1093/gbe/evab070."/>
  <meta name="citation_reference" content="Maddamsetti R. Selection maintains protein interactome resilience in the long-term evolution experiment with Escherichia coli. Genome Biology and Evolution. 2021. doi: 10.1093/gbe/evab074."/>
  <meta name="citation_reference" content="Hui S, Silverman JM, Chen SS, Erickson DW, Basan M, Wang J, et al. Quantitative proteomic analysis reveals a simple strategy of global resource allocation in bacteria. Molecular systems biology. 2015;11(2)."/>
  <meta name="citation_reference" content="Sastry AV, Gao Y, Szubin R, Hefner Y, Xu S, Kim D, et al. The Escherichia coli transcriptome mostly consists of independently regulated modules. Nature communications. 2019;10(1):1–14."/>
  <meta name="citation_reference" content="Wytock TP, Motter AE. Predicting growth rate from gene expression. Proceedings of the National Academy of Sciences. 2019;116(2):367–72."/>
  <meta name="citation_reference" content="Izutsu M, Lake DM, Matson ZW, Dodson JP, Lenski RE. Effects of periodic bottlenecks on the dynamics of adaptive evolution in microbial populations. bioRxiv. 2021."/>
  <meta name="citation_reference" content="Papadopoulos D, Schneider D, Meier-Eiss J, Arber W, Lenski RE, Blot M. Genomic evolution during a 10,000-generation experiment with bacteria. Proceedings of the National Academy of Sciences. 1999;96(7):3807–12."/>
  <meta name="citation_reference" content="Maddamsetti R, Grant NA. Divergent evolution of mutation rates and biases in the long-term evolution experiment with Escherichia coli. Genome Biology and Evolution. 2020. doi: 10.1093/gbe/evaa178."/>
  <meta name="citation_reference" content="Consuegra J, Gaffé J, Lenski RE, Hindré T, Barrick JE, Tenaillon O, et al. Insertion-sequence-mediated mutations both promote and constrain evolvability during a long-term experiment with bacteria. Nature communications. 2021;12(1):1–12."/>
  <meta name="citation_reference" content="Turner CB, Blount ZD, Mitchell DH, Lenski RE. Evolution and coexistence in response to a key innovation in a long-term evolution experiment with Escherichia coli. bioRxiv. 2015:020958."/>
  <meta name="citation_reference" content="Kinnersley M, Schwartz K, Yang D-D, Sherlock G, Rosenzweig F. Evolutionary dynamics and structural consequences of de novo beneficial mutations and mutant lineages arising in a constant environment. BMC biology. 2021;19(1):1–21."/>
  <meta name="citation_reference" content="Lenski RE, Wiser MJ, Ribeck N, Blount ZD, Nahum JR, Morris JJ, et al. Sustained fitness gains and variability in fitness trajectories in the long-term evolution experiment with Escherichia coli. Proceedings of the Royal Society B: Biological Sciences. 2015;282(1821):20152292."/>
  <meta name="citation_reference" content="Wielgoss S, Barrick JE, Tenaillon O, Wiser MJ, Dittmar WJ, Cruveiller S, et al. Mutation rate dynamics in a bacterial population reflect tension between adaptation and genetic load. Proceedings of the National Academy of Sciences. 2013;110(1):222–7."/>
  <meta name="citation_reference" content="Rudan M, Schneider D, Warnecke T, Krisko A. RNA chaperones buffer deleterious mutations in E. coli. Elife. 2015;4:e04745."/>
  <meta name="citation_reference" content="Bailey SF, Guo Q, Bataillon T. Identifying drivers of parallel evolution: A regression model approach. Genome biology and evolution. 2018;10(10):2801–12."/>
  <meta name="citation_reference" content="Foster PL, Hanson AJ, Lee H, Popodi EM, Tang H. On the mutational topology of the bacterial genome. G3: Genes, Genomes, Genetics. 2013;3(3):399-407."/>
  <meta name="citation_reference" content="Niccum BA, Lee H, MohammedIsmail W, Tang H, Foster PL. The Symmetrical Wave Pattern of Base-Pair Substitution Rates across the Escherichia coli Chromosome Has Multiple Causes. mBio. 2019;10(4)."/>
  <meta name="citation_reference" content="Quandt EM, Gollihar J, Blount ZD, Ellington AD, Georgiou G, Barrick JE. Fine-tuning citrate synthase flux potentiates and refines metabolic innovation in the Lenski evolution experiment. Elife. 2015;4:e09696."/>
  <meta name="citation_reference" content="Zee PC, Mendes-Soares H, Yu YTN, Kraemer SA, Keller H, Ossowski S, et al. A shift from magnitude to sign epistasis during adaptive evolution of a bacterial social trait. Evolution. 2014;68(9):2701–8."/>
  <meta name="citation_reference" content="Venkataram S, Monasky R, Sikaroodi SH, Kryazhimskiy S, Kacar B. Evolutionary stalling and a limit on the power of natural selection to improve a cellular module. Proceedings of the National Academy of Sciences. 2020;117(31):18582–90."/>
  <meta name="citation_reference" content="Rodrigues JV, Shakhnovich EI. Adaptation to mutational inactivation of an essential gene converges to an accessible suboptimal fitness peak. Elife. 2019;8."/>
  <meta name="citation_reference" content="Kaznatcheev A. Computational complexity as an ultimate constraint on evolution. Genetics. 2019;212(1):245–65."/>
  <meta name="citation_reference" content="Andersson DI, Slechta ES, Roth JR. Evidence that gene amplification underlies adaptive mutability of the bacterial lac operon. Science. 1998;282(5391):1133-5."/>
  <meta name="citation_reference" content="Hendrickson H, Slechta ES, Bergthorsson U, Andersson DI, Roth JR. Amplification– mutagenesis: evidence that “directed” adaptive mutation and general hypermutability result from growth with a selected gene amplification. Proceedings of the National Academy of Sciences. 2002;99(4):2164–9."/>
  <meta name="citation_reference" content="Good BH, Desai MM. Evolution of mutation rates in rapidly adapting asexual populations. Genetics. 2016;204(3):1249–66."/>
  <meta name="citation_reference" content="Philippe N, Crozat E, Lenski RE, Schneider D. Evolution of global regulatory networks during a long-term experiment with Escherichia coli. Bioessays. 2007;29(9):846–60."/>
  <meta name="citation_reference" content="Chen H, Wu C-I, He X. The genotype–phenotype relationships in the light of natural selection. Molecular biology and evolution. 2018;35(3):525–42."/>
  <meta name="citation_reference" content="Scott M, Gunderson CW, Mateescu EM, Zhang Z, Hwa T. Interdependence of cell growth and gene expression: origins and consequences. Science. 2010;330(6007):1099-102."/>
  <meta name="citation_reference" content="Erickson DW, Schink SJ, Patsalo V, Williamson JR, Gerland U, Hwa T. A global resource allocation strategy governs growth transition kinetics of Escherichia coli. Nature. 2017;551(7678):119-23."/>
  <meta name="citation_reference" content="Klumpp S, Hwa T. Bacterial growth: global effects on gene expression, growth feedback and proteome partition. Current opinion in biotechnology. 2014;28:96–102."/>
  <meta name="citation_reference" content="Basan M, Hui S, Okano H, Zhang Z, Shen Y, Williamson JR, et al. Overflow metabolism in Escherichia coli results from efficient proteome allocation. Nature. 2015;528(7580):99-104."/>
  <meta name="citation_reference" content="Mathis RA, Sokol ES, Gupta PB. Identification of Genes under Purifying Selection in Human Cancers. BioRxiv. 2017:129205."/>
  <meta name="citation_reference" content="Weghorn D, Sunyaev S. Bayesian inference of negative and positive selection in human cancers. Nature genetics. 2017;49(12):1785–8."/>
  <meta name="citation_reference" content="Patel SJ, Sanjana NE, Kishton RJ, Eidizadeh A, Vodnala SK, Cam M, et al. Identification of essential genes for cancer immunotherapy. Nature. 2017;548(7669):537-42."/>
  <meta name="citation_reference" content="Li X, Giorgi EE, Marichannegowda MH, Foley B, Xiao C, Kong X-P, et al. Emergence of SARS-CoV-2 through recombination and strong purifying selection. Science Advances. 2020;6(27):eabb9153."/>
  <meta name="citation_reference" content="Lieberman TD, Michel J-B, Aingaran M, Potter-Bynoe G, Roux D, Davis MR, et al. Parallel bacterial evolution within multiple patients identifies candidate pathogenicity genes. Nature genetics. 2011;43(12):1275–80."/>
  <meta name="citation_reference" content="Caballero JD, Clark ST, Coburn B, Zhang Y, Wang PW, Donaldson SL, et al. Selective sweeps and parallel pathoadaptation drive Pseudomonas aeruginosa evolution in the cystic fibrosis lung. MBio. 2015;6(5)."/>
  <meta name="citation_reference" content="Chung H, Lieberman TD, Vargas SO, Flett KB, McAdam AJ, Priebe GP, et al. Global and local selection acting on the pathogen Stenotrophomonas maltophilia in the human lung. Nature communications. 2017;8:14078."/>
  <meta name="citation_reference" content="Cooper VS, Honsa E, Rowe H, Deitrick C, Iverson AR, Whittall JJ, et al. Experimental Evolution in vivo to Identify Selective Pressures During Pneumococcal Colonization. Msystems. 2020;5(3)."/>
  <meta name="citation_reference" content="Deatherage DE, Traverse CC, Wolf LN, Barrick JE. Detecting rare structural variation in evolving microbial populations from new sequence junctions using breseq. Frontiers in genetics. 2015;5:468."/>
  <meta name="citation_reference" content="Haller BC, Messer PW. SLiM 3: forward genetic simulations beyond the Wright–Fisher model. Molecular biology and evolution. 2019;36(3):632–7."/>
  <meta name="citation_reference" content="Robert L, Ollion J, Robert J, Song X, Matic I, Elez M. Mutation dynamics and fitness effects followed in single cells. Science. 2018;359(6381):1283-6."/>
  <meta name="citation_reference" content="Perfeito L, Fernandes L, Mota C, Gordo I. Adaptive mutations in bacteria: high rate and small effects. science. 2007;317(5839):813-5."/>
  <meta name="twitter:title" content="Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="A general method to infer both positive and purifying selection during the real-time evolution of hypermutator pathogens would be broadly useful. To this end, we introduce a simple test to infer mode of selection (STIMS) from metagenomic time series of evolving microbial populations. We test STIMS on metagenomic data generated by simulations of bacterial evolution, and on metagenomic data spanning 62,750 generations of Lenski’s long-term evolution experiment with Escherichia coli (LTEE). In both cases, STIMS recovers signals of positive and purifying selection on gold standard sets of genes. Using STIMS, we find strong evidence of ongoing positive selection on key regulators of the E. coli gene regulatory network, even in some hypermutator populations. STIMS also detects positive selection on regulatory genes in hypermutator populations of Pseudomonas aeruginosa that adapted to subinhibitory concentrations of colistin – an antibiotic of last resort – for just twenty-six days of laboratory evolution. Our results show that the fine-tuning of gene regulatory networks is a general mechanism for rapid and ongoing adaptation. The simplicity of STIMS, together with its intuitive visual interpretation, make it a useful test for positive and purifying selection in metagenomic data sets that track the evolution of hypermutator populations in real-time.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.05.23.112508v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="A general method to infer both positive and purifying selection during the real-time evolution of hypermutator pathogens would be broadly useful. To this end, we introduce a simple test to infer mode of selection (STIMS) from metagenomic time series of evolving microbial populations. We test STIMS on metagenomic data generated by simulations of bacterial evolution, and on metagenomic data spanning 62,750 generations of Lenski’s long-term evolution experiment with Escherichia coli (LTEE). In both cases, STIMS recovers signals of positive and purifying selection on gold standard sets of genes. Using STIMS, we find strong evidence of ongoing positive selection on key regulators of the E. coli gene regulatory network, even in some hypermutator populations. STIMS also detects positive selection on regulatory genes in hypermutator populations of Pseudomonas aeruginosa that adapted to subinhibitory concentrations of colistin – an antibiotic of last resort – for just twenty-six days of laboratory evolution. Our results show that the fine-tuning of gene regulatory networks is a general mechanism for rapid and ongoing adaptation. The simplicity of STIMS, together with its intuitive visual interpretation, make it a useful test for positive and purifying selection in metagenomic data sets that track the evolution of hypermutator populations in real-time.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-13"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.05.23.112508v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2444657"/>
  <title>
   Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZHyllngP5no9MzuPjyt_XA7ASzc2ugKjJX2xRXQ9j_c"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2444657","apath":"\/biorxiv\/early\/2022\/03\/13\/2020.05.23.112508.atom","pisa":"biorxiv;2020.05.23.112508v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.05.23.112508v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.05.23.112508v2","title":"Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations","identifier":"node\/2444657"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ESoW5no5\\\\\\\/v3CjfUAirRYOaBE5dfRwcr0sHCSJoWknr4Z\\\\\\\/885gBdoDbxiwIcuiS4VdpI5Q9BrJIyn0C+E5czNlqR0SVRc0sniK+pbUSVKX1hUyzg4SIFrk8IFHP9CxjVLDA3hFlCNLuHH35guVoZ2PEA\\\\\\\/\\\\\\\/D7HQsctKvY9Q+nr\\\\\\\/taq792dC0A4j4XCXDWnJWAn\\\\\\\/CJWoMF+Q80O\\\\\\\/eHKqcr2gjXxw7yKfI4rkBySaxnZBq+ypOgC8qta\\\\\\\/fh6iqcbU9AiUjynMrZPkJHphMthPjEz\\\\\\\/uuki38DSi74tM17+CgL0X6tPHqTd+sH5gbfXjT3OUwYrS6znvyRhcLNvlp\\\\\\\/Uta43vVKejdVDE57XqBjZA4POGfxUdp9vQM\\\\\\\/cPVJ4Wq0LuYVsD+nZJyzGYGYUEAxU+gnPatq2Tzxc0rshjpFVkGT4K0L3I2t0eJkGDfXYzAw\\\\\\\/cGytg7l2Eb6ihEPCGfVDtSLa6KIG0XbsAe5QOjd1dUSSZd4aTI24g+HpUAn6h41V634lGpM9Q9kpCtgJI28Fi8N9fqcseVH5KBK0AfgwxEDoZg7ekaGIVdI0cCc\\\\\\\/nc7pDEajoumk2yS6BX2bwoUsxcXjGsh1HLCB+b\\\\\\\/MOH3SxWcCG5WRugR2ojxniODAClM9Pv4\\\\\\\/avwnMhsEnEMqec1PLExYfCSt7ouFnpe8kTF9+bwnavH21awgnR68MPVZKwGQxoMPE2JKefEn21pe121O+1SecbJWNAp5JiouHOD8ZpxCsyF89tW6PfAIjjy9P11Z6Fvt7hEQyQtAEbnjGrWIiBn5gnmA8ieox0F4UrKNpUvlpqeuZfOKDZjzyCPvug98mln3tcEHNpwLey9pdW5Tr2JWV6dE3ojRCOJhoD\\\\\\\/bLjUkDaBf1YgInsgbPQKT0II5cgfoDMqZ03O3IOSjywkwQLwqdhK1c7TtVyisO+howBQswtIS\\\\\\\/Wzn2ttY2GTBujsDBvKDxlvDMjkgr42z4DA+TRGg22StVfpIhyHoLtWBZ6NKISMBXBj8+phUlrw5IaulAySZOpT+QRSbkLTwxmsN3I6kEsG43nxGOeiEqXuC0VIxCmsPhqAxphNLmIeLx+AQ8pTwvj2mQH1bi7wSUskZh6I23jFO2u2IvF8NzJRpnIHvCQfydS1JICCcyZeN6Kwul7uakx+pRXXV6HjmjgGY4RoMfBf9A\\\\\\\/hhSBAN0\\\\\\\/oC2uukyQAGovZRIrfJ9P7wRgzeFUBS+Uct+sVO9vAproSlRYqRZjVHPnz7hImIMCCFQgoRgrmdwoaZFyk0Wc\\\\\\\/UbANj3RGZMGDg4JSKp+QfT9HeTMBFaA4kKI4JXXoTE8fHL7MrM3Ql6Enx2NSv5yHD8Jv1S9FNY5pCDnkgTOOkFUs\\\\\\\/f5THkqVKtFlKQ+C9Rwu4vqo400=\\\u0022,\\\u0022iv\\\u0022:\\\u0022n3shd3bT2b1EhLTtvqJvng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002234d38f932a1abbfafc10c7552223449a81b868e4293bd02eb02697babf4b349f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.05.23.112508v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2444657 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.05.23.112508v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_691690441">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_691690441" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_820320469" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-FaA1xpr7fJOTAQOVLj_KBc-owdPYZSPCasgmu8liOuE"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2444657" data-node-nid="2444657" id="node-2444657--21792500058" data-pisa="biorxiv;2020.05.23.112508v2" data-pisa-master="biorxiv;2020.05.23.112508" data-apath="/biorxiv/early/2022/03/13/2020.05.23.112508.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-3370-092X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Rohan
                      </span>
                      <span class="nlm-surname">
                       Maddamsetti
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-4555-5283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nkrumah A.
                      </span>
                      <span class="nlm-surname">
                       Grant
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.05.23.112508
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2444657--21792500058" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Rohan Maddamsetti
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Biomedical Engineering, Duke University
                       </span>
                       , Durham, NC
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Rohan%2BMaddamsetti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Maddamsetti%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARohan%2BMaddamsetti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-3370-092X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Rohan Maddamsetti
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:rohan.maddamsetti@gmail.com" class="" data-icon-position="" data-hide-link-title="0">
                        rohan.maddamsetti@gmail.com
                       </a>
                       <a href="mailto:nkrumah.grant88@gmail.com" class="" data-icon-position="" data-hide-link-title="0">
                        nkrumah.grant88@gmail.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Nkrumah A. Grant
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        BEACON Center for the Study of Evolution in Action, Michigan State University
                       </span>
                       , East Lansing, MI
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Microbiology and Molecular Genetics, Michigan State University
                       </span>
                       , East Lansing, MI
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Biological Sciences, University of Idaho
                       </span>
                       , Moscow, ID
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nkrumah%2BA.%2BGrant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Grant%20NA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANkrumah%2BA.%2BGrant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-4555-5283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nkrumah A. Grant
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.05.23.112508v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2444657" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2444657/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.23.112508v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2444657" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2444657/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.23.112508v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2444657" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2444657/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.23.112508v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2444657" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2444657/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.23.112508v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2444657" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2444657/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.05.23.112508v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2444657" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2444657/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             ABSTRACT
                            </h2>
                            <p id="p-3">
                             A general method to infer both positive and purifying selection during the real-time evolution of hypermutator pathogens would be broadly useful. To this end, we introduce a simple test to infer mode of selection (STIMS) from metagenomic time series of evolving microbial populations. We test STIMS on metagenomic data generated by simulations of bacterial evolution, and on metagenomic data spanning 62,750 generations of Lenski’s long-term evolution experiment with Escherichia coli (LTEE). In both cases, STIMS recovers signals of positive and purifying selection on gold standard sets of genes. Using STIMS, we find strong evidence of ongoing positive selection on key regulators of the E. coli gene regulatory network, even in some hypermutator populations. STIMS also detects positive selection on regulatory genes in hypermutator populations of Pseudomonas aeruginosa that adapted to subinhibitory concentrations of colistin – an antibiotic of last resort – for just twenty-six days of laboratory evolution. Our results show that the fine-tuning of gene regulatory networks is a general mechanism for rapid and ongoing adaptation. The simplicity of STIMS, together with its intuitive visual interpretation, make it a useful test for positive and purifying selection in metagenomic data sets that track the evolution of hypermutator populations in real-time.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             INTRODUCTION
                            </h2>
                            <p id="p-6">
                             Organisms often evolve defects in DNA repair and recombination pathways that cause very high mutation rates. Hypermutability is readily observed in cancer [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ] and in opportunistic pathogens attacking immunocompromised individuals [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. Hypermutability is also associated with the evolution of antibiotic resistance, including multi-drug-resistant tuberculosis [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ]. Hypermutability can increase the rate of beneficial mutations, but it can also obscure the genomic basis of adaptation [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ], because vast numbers of nearly-neutral mutations hitchhike to high frequencies with the beneficial mutations that drive the selection dynamics. In this regime, called “emergent neutrality” [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ], it is challenging to identify selection on particular genes, unless those genes are under very strong selection [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ]. As such, there is a need for methods that can resolve positive and purifying selection in hypermutator populations, due to their importance in basic research, genetic engineering, and medicine. To address this research gap, we present a simple test to infer mode of selection (STIMS) from metagenomic time series of large asexual hypermutator populations, and use this method to study positive and purifying selection in bacterial populations that evolved hypermutability during long-term experimental evolution.
                            </p>
                            <p id="p-7">
                             We developed STIMS in the context of Lenski’s long-term evolution experiment with Escherichia coli (LTEE) [
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. The LTEE has become an important test bed for many fundamental questions in evolutionary biology, due to its simplicity (daily serial transfer of twelve populations) and comprehensive record of frozen bacterial samples. Previous studies have used the LTEE as a model system to study the tempo and mode of both genomic [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             –
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ] and phenotypic evolution [
                             <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ,
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             –
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ]. Many of those previous studies focused on the evolutionary dynamics underlying adaptive evolution. The evolution of hypermutability in six of the LTEE populations has obscured the genomic signatures of adaptation [
                             <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ,
                             <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ,
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ], again due to emergent neutrality. STIMS gains statistical power to infer selection despite emergent neutrality, by aggregating mutations within a focal set of genes, counting their occurrence, then comparing the counts to a background distribution of the number of mutations per gene set.
                            </p>
                            <p id="p-8">
                             This work expands on previous observations suggesting the action of purifying selection on hypermutator populations of the LTEE [
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ]. In that work, we found that aerobic-specific genes were depleted in mutations compared to randomized sets of genes in three hypermutator populations: Ara−2, Ara+3, and Ara+6. That finding suggests that aerobic-specific genes may have transitioned from positive selection to purifying selection in those LTEE populations [
                             <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ]. Our work also complements recent findings of purifying selection affecting protein evolution [
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ] and protein-protein-interaction network evolution in the LTEE [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ].
                            </p>
                            <p id="p-9">
                             Here, we demonstrate that STIMS recovers signals of positive and purifying selection in evolutionary simulations of haploid populations under a Wright-Fisher model, and show that STIMS recovers signals of positive and purifying selection on gold-standard sets of genes with a priori evidence of those selection pressures in the LTEE. We then use STIMS to examine the tempo of molecular evolution in modules of co-regulated genes [
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             –
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ], and to generate testable hypotheses about the mode (i.e. purifying or positive selection) of evolution on those gene modules in the LTEE. Finally, we use STIMS to test for positive selection on regulatory genes in a 26-day evolution experiment involving adaptation of hypermutator Pseudomonas aeruginosa to subinhibitory concentrations of colistin, an antibiotic of last resort. These results indicate that STIMS is a useful and general test for selection, even over relatively short timescales of molecular evolution.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             RESULTS
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Simulations demonstrate the conditions in which STIMS is an effective test for positive and purifying selection
                             </h3>
                             <p id="p-10">
                              The key premise of STIMS is that the tempo of molecular evolution differs across different sets of genes, due to variation in the underlying selection pressures affecting those sets of genes. As input, STIMS requires a list of mutations that have been observed in metagenomic sequencing of a haploid population, sampled over time (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              ). STIMS then calculates a simple summary statistic of the evolutionary dynamics: the number of observed mutations in a query gene set, normalized by total gene length. A p-value is then calculated using the non-parametric bootstrap, which is simply a count of the times that a random gene set accumulates more mutations than the query gene set (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              ). The same logic applies when measuring whether a gene set is depleted of mutations: in this case, STIMS counts the number of times that a random selection of genes accumulates fewer mutations than the query gene set. Importantly, the bootstrapped background distribution does not, in general, correspond to a neutral null model of evolution: genome-wide positive and purifying selection may affect the background distribution. Accordingly, STIMS is interpreted in relation to the overall tempo at which mutations are observed across the genome in a given population, which is driven by population-genetic parameters such as mutation rate, population size, and the genomic distribution of mutation fitness effects (DFE).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F1.large.jpg?width=800&height=600&carousel=1" title="Data analysis workflow. 12 E. coli populations have evolved for more than 60,000 generations in Lenski’s long-term evolution experiment (LTEE). We reanalyzed metagenomic time series of the LTEE, described in Good et al. (2017). We also evolved 20 asexual haploid populations in silico for 5,000 generations. We identified variants segregating during the simulations to model metagenomic sequencing and variant calling. We designed a simple test to infer mode of selection (STIMS) to test pre-defined query gene sets for positive or purifying selection in the metagenomic time series. See the Materials and Methods for further details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Data analysis workflow.</span> 12 <em>E. coli</em> populations have evolved for more than 60,000 generations in Lenski’s long-term evolution experiment (LTEE). We reanalyzed metagenomic time series of the LTEE, described in Good et al. (2017). We also evolved 20 asexual haploid populations <em>in silico</em> for 5,000 generations. We identified variants segregating during the simulations to model metagenomic sequencing and variant calling. We designed a simple test to infer mode of selection (STIMS) to test pre-defined query gene sets for positive or purifying selection in the metagenomic time series. See the Materials and Methods for further details.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F1.medium.gif" width="338" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F1.medium.gif" width="338" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Data analysis workflow.
                               </span>
                               <p id="p-11" class="first-child">
                                12 E. coli populations have evolved for more than 60,000 generations in Lenski’s long-term evolution experiment (LTEE). We reanalyzed metagenomic time series of the LTEE, described in Good et al. (2017). We also evolved 20 asexual haploid populations in silico for 5,000 generations. We identified variants segregating during the simulations to model metagenomic sequencing and variant calling. We designed a simple test to infer mode of selection (STIMS) to test pre-defined query gene sets for positive or purifying selection in the metagenomic time series. See the Materials and Methods for further details.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F2.large.jpg?width=800&height=600&carousel=1" title="The STIMS algorithm. When examining the evolutionary dynamics of a gene set of interest, one often wants to know whether the set is enriched or depleted of observed mutations. To test for depletion, we use the non-parametric bootstrap to estimate the fraction of randomized gene sets that have fewer mutations than the query gene set. For example, if 11% of randomized gene sets have fewer mutations than the query gene set, then the query gene set is not depleted of mutations (one-tailed p-value = 0.11). A similar derivation holds for tests of enrichment. This worked example uses the allele frequency dynamics of ribosome-associated proteins in the Ara−1 population of Lenski’s long term evolution experiment with Escherichia coli (LTEE) to illustrate how the STIMS algorithm works. This population evolved a hypermutator phenotype around the 26,250-generation time point." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The STIMS algorithm.</span> When examining the evolutionary dynamics of a gene set of interest, one often wants to know whether the set is enriched or depleted of observed mutations. To test for depletion, we use the non-parametric bootstrap to estimate the fraction of randomized gene sets that have fewer mutations than the query gene set. For example, if 11% of randomized gene sets have fewer mutations than the query gene set, then the query gene set is not depleted of mutations (one-tailed <em>p</em>-value = 0.11). A similar derivation holds for tests of enrichment. This worked example uses the allele frequency dynamics of ribosome-associated proteins in the Ara−1 population of Lenski’s long term evolution experiment with <em>Escherichia coli</em> (LTEE) to illustrate how the STIMS algorithm works. This population evolved a hypermutator phenotype around the 26,250-generation time point.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F2.medium.gif" width="440" height="353"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F2.medium.gif" width="440" height="353"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                The STIMS algorithm.
                               </span>
                               <p id="p-12" class="first-child">
                                When examining the evolutionary dynamics of a gene set of interest, one often wants to know whether the set is enriched or depleted of observed mutations. To test for depletion, we use the non-parametric bootstrap to estimate the fraction of randomized gene sets that have fewer mutations than the query gene set. For example, if 11% of randomized gene sets have fewer mutations than the query gene set, then the query gene set is not depleted of mutations (one-tailed p-value = 0.11). A similar derivation holds for tests of enrichment. This worked example uses the allele frequency dynamics of ribosome-associated proteins in the Ara−1 population of Lenski’s long term evolution experiment with Escherichia coli (LTEE) to illustrate how the STIMS algorithm works. This population evolved a hypermutator phenotype around the 26,250-generation time point.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-13">
                              For this reason, we evolved haploid populations in silico using a Wright-Fisher model to examine the conditions under which STIMS can detect positive and purifying selection in idealized populations, in which the strength of selection on various regions of the genome is known a priori. We ran 10 replicate simulations of nonmutator haploid populations and 10 replicate simulations of hypermutator haploid populations, using large population sizes, mutation rates parameterized from LTEE measurements, and an idealized genome and DFE based on E. coli (Materials and Methods). The results of these simulations, and our analysis of these simulations using STIMS, are shown in
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              for the nonmutator case and in
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              for the hypermutator case.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F3.large.jpg?width=800&height=600&carousel=1" title="In silico evolution experiments demonstrate that STIMS can detect positive but not purifying selection in nonmutator populations. Bacterial evolution was simulated using a Wright-Fisher model. Each simulation lasted 5,000 generations. The effective population size was set to 1 million individuals. Nonmutator bacteria were modeled using a mutation rate of 10−10 per base-pair per generation. Each individual contains an idealized genome of 4,000 genes, each 1,000 base-pairs in length, for a genome of 4 million base-pairs. 100 genes are targets of positive selection, 100 genes are evolving completely neutrally (relaxed selection), and 100 genes are evolving under purifying selection. The remaining 3,700 genes evolve nearly neutrally, based on published estimates of the genomic distribution of mutation fitness effects in Escherichia coli. See the Materials and Methods for further details. A) Typical allele frequency dynamics of a nonmutator population, showing selective sweeps and clonal interference. B) STIMS shows high sensitivity and specificity for detecting positive selection in nonmutator populations. C) STIMS is unable to detect purifying selection (low sensitivity), but shows no false positives (high specificity). D) Typical result of running STIMS on genes evolving under positive selection in the nonmutator simulations. E) Typical result of running STIMS on genes evolving neutrally in the nonmutator simulations. F) Typical result of running STIMS on genes evolving under purifying selection in the nonmutator simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><em>In silico</em> evolution experiments demonstrate that STIMS can detect positive but not purifying selection in nonmutator populations. Bacterial evolution was simulated using a Wright-Fisher model. Each simulation lasted 5,000 generations. The effective population size was set to 1 million individuals. Nonmutator bacteria were modeled using a mutation rate of 10<sup>−10</sup> per base-pair per generation. Each individual contains an idealized genome of 4,000 genes, each 1,000 base-pairs in length, for a genome of 4 million base-pairs. 100 genes are targets of positive selection, 100 genes are evolving completely neutrally (relaxed selection), and 100 genes are evolving under purifying selection. The remaining 3,700 genes evolve nearly neutrally, based on published estimates of the genomic distribution of mutation fitness effects in <em>Escherichia coli</em>. See the Materials and Methods for further details. A) Typical allele frequency dynamics of a nonmutator population, showing selective sweeps and clonal interference. B) STIMS shows high sensitivity and specificity for detecting positive selection in nonmutator populations. C) STIMS is unable to detect purifying selection (low sensitivity), but shows no false positives (high specificity). D) Typical result of running STIMS on genes evolving under positive selection in the nonmutator simulations. E) Typical result of running STIMS on genes evolving neutrally in the nonmutator simulations. F) Typical result of running STIMS on genes evolving under purifying selection in the nonmutator simulations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F3.medium.gif" width="426" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F3.medium.gif" width="426" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <p id="p-14" class="first-child">
                                In silico evolution experiments demonstrate that STIMS can detect positive but not purifying selection in nonmutator populations. Bacterial evolution was simulated using a Wright-Fisher model. Each simulation lasted 5,000 generations. The effective population size was set to 1 million individuals. Nonmutator bacteria were modeled using a mutation rate of 10
                                <sup>
                                 −10
                                </sup>
                                per base-pair per generation. Each individual contains an idealized genome of 4,000 genes, each 1,000 base-pairs in length, for a genome of 4 million base-pairs. 100 genes are targets of positive selection, 100 genes are evolving completely neutrally (relaxed selection), and 100 genes are evolving under purifying selection. The remaining 3,700 genes evolve nearly neutrally, based on published estimates of the genomic distribution of mutation fitness effects in Escherichia coli. See the Materials and Methods for further details. A) Typical allele frequency dynamics of a nonmutator population, showing selective sweeps and clonal interference. B) STIMS shows high sensitivity and specificity for detecting positive selection in nonmutator populations. C) STIMS is unable to detect purifying selection (low sensitivity), but shows no false positives (high specificity). D) Typical result of running STIMS on genes evolving under positive selection in the nonmutator simulations. E) Typical result of running STIMS on genes evolving neutrally in the nonmutator simulations. F) Typical result of running STIMS on genes evolving under purifying selection in the nonmutator simulations.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F4.large.jpg?width=800&height=600&carousel=1" title="In silico evolution experiments demonstrate that STIMS can detect both positive and purifying selection in hypermutator populations. Bacterial evolution was simulated using a Wright-Fisher model. Each simulation lasted 5,000 generations. The effective population size was set to 1 million individuals. Hypermutator bacteria were modeled using a mutation rate of 10−8 per base-pair per generation. Each individual contains an idealized genome of 4,000 genes, each 1,000 base-pairs in length, for a genome of 4 million base-pairs. 100 genes are targets of positive selection, 100 genes are evolving completely neutrally (relaxed selection), and 100 genes are evolving under purifying selection. The remaining 3,700 genes evolve nearly neutrally, based on published estimates of the genomic distribution of mutation fitness effects in Escherichia coli. See the Materials and Methods for further details. A) Typical allele frequency dynamics of a hypermutator population, showing selective sweeps and clonal interference. B) STIMS shows high sensitivity and specificity for detecting positive selection in hypermutator populations. C) STIMS detects purifying selection with increasing sensitivity at lower allele frequency mutation detection thresholds, more genes sampled from the set of genes evolving under purifying selection, the length of the time series, and shorter intervals between metagenomic sampling, and shows high specificity across all tested parameters. D) Typical result of running STIMS on genes evolving under positive selection in the hypermutator simulations. E) Typical result of running STIMS on genes evolving neutrally in the hypermutator simulations. F) Typical result of running STIMS on genes evolving under purifying selection in the hypermutator simulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><em>In silico</em> evolution experiments demonstrate that STIMS can detect both positive and purifying selection in hypermutator populations. Bacterial evolution was simulated using a Wright-Fisher model. Each simulation lasted 5,000 generations. The effective population size was set to 1 million individuals. Hypermutator bacteria were modeled using a mutation rate of 10<sup>−8</sup> per base-pair per generation. Each individual contains an idealized genome of 4,000 genes, each 1,000 base-pairs in length, for a genome of 4 million base-pairs. 100 genes are targets of positive selection, 100 genes are evolving completely neutrally (relaxed selection), and 100 genes are evolving under purifying selection. The remaining 3,700 genes evolve nearly neutrally, based on published estimates of the genomic distribution of mutation fitness effects in <em>Escherichia coli</em>. See the Materials and Methods for further details. A) Typical allele frequency dynamics of a hypermutator population, showing selective sweeps and clonal interference. B) STIMS shows high sensitivity and specificity for detecting positive selection in hypermutator populations. C) STIMS detects purifying selection with increasing sensitivity at lower allele frequency mutation detection thresholds, more genes sampled from the set of genes evolving under purifying selection, the length of the time series, and shorter intervals between metagenomic sampling, and shows high specificity across all tested parameters. D) Typical result of running STIMS on genes evolving under positive selection in the hypermutator simulations. E) Typical result of running STIMS on genes evolving neutrally in the hypermutator simulations. F) Typical result of running STIMS on genes evolving under purifying selection in the hypermutator simulations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F4.medium.gif" width="28" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F4.medium.gif" width="28" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <p id="p-15" class="first-child">
                                In silico evolution experiments demonstrate that STIMS can detect both positive and purifying selection in hypermutator populations. Bacterial evolution was simulated using a Wright-Fisher model. Each simulation lasted 5,000 generations. The effective population size was set to 1 million individuals. Hypermutator bacteria were modeled using a mutation rate of 10
                                <sup>
                                 −8
                                </sup>
                                per base-pair per generation. Each individual contains an idealized genome of 4,000 genes, each 1,000 base-pairs in length, for a genome of 4 million base-pairs. 100 genes are targets of positive selection, 100 genes are evolving completely neutrally (relaxed selection), and 100 genes are evolving under purifying selection. The remaining 3,700 genes evolve nearly neutrally, based on published estimates of the genomic distribution of mutation fitness effects in Escherichia coli. See the Materials and Methods for further details. A) Typical allele frequency dynamics of a hypermutator population, showing selective sweeps and clonal interference. B) STIMS shows high sensitivity and specificity for detecting positive selection in hypermutator populations. C) STIMS detects purifying selection with increasing sensitivity at lower allele frequency mutation detection thresholds, more genes sampled from the set of genes evolving under purifying selection, the length of the time series, and shorter intervals between metagenomic sampling, and shows high specificity across all tested parameters. D) Typical result of running STIMS on genes evolving under positive selection in the hypermutator simulations. E) Typical result of running STIMS on genes evolving neutrally in the hypermutator simulations. F) Typical result of running STIMS on genes evolving under purifying selection in the hypermutator simulations.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-16">
                              The dynamics of adaptation in our simulations are strikingly similar to the dynamics observed in the LTEE: both the nonmutator populations (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 3A
                              </a>
                              ) and the hypermutator populations (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Figure 4A
                              </a>
                              ) show periodic selective sweeps, cohorts of mutations rising to fixation simultaneously, and clonal interference. These patterns are characteristic of the evolution of large asexual populations in which the supply of beneficial mutations does not limit adaptation [
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ,
                              <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ,
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ]. Thus, our in silico simulations are representative of the biology we aim to describe.
                             </p>
                             <p id="p-17">
                              We designed the genomes of our simulated haploid populations such that we knew a priori the mode of selection acting on particular genomic regions. Therefore, we used our simulated data to evaluate how well STIMS could detect selection given that it exists (sensitivity), and avoid false positives (specificity). Our simulations demonstrate that STIMS has both high sensitivity and specificity in detecting positive selection in both nonmutator and hypermutator populations (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figures 3B
                              </a>
                              and
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               4B
                              </a>
                              ). Most mutations observed in the nonmutator populations are beneficial. So, the background distribution for the nonmutator populations reflects positive selection on beneficial driver mutations and the dynamics of passenger mutations hitchhiking with those driver mutations. In the nonmutator populations, not enough mutations are observed across the genome to see which genes are depleted of passenger mutations. Consequently, STIMS cannot detect purifying selection in this context. Nevertheless, STIMS is highly specific. It does not call any false positives in our tests for positive and purifying selection in the nonmutator populations (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Figure 3C
                              </a>
                              ). Representative visualizations of STIMS when run on nonmutator populations are shown for genes evolving under positive selection in
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Figure 3D
                              </a>
                              , for genes evolving completely neutrally in
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Figure 3E
                              </a>
                              , and for genes evolving under purifying selection in
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Figure 3F
                              </a>
                              . These visualizations show that for nonmutator populations, gene modules with trajectories below the background distribution could be evolving under purifying selection, or could be evolving completely neutrally due to relaxed selection.
                             </p>
                             <p id="p-18">
                              This apparent difference from the background distribution is never statistically significant, because most random gene sets have no mutations at all, and these empty sets are not plotted. Gene modules with trajectories above the background distribution are evolving under strong positive selection.
                             </p>
                             <p id="p-19">
                              In contrast to the nonmutator populations, our simulations show that STIMS has high sensitivity and specificity in detecting purifying selection in hypermutator populations (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Figure 4C
                              </a>
                              ). In this case, mutation rates are high enough that passenger mutations arise densely across the genome, allowing for the detection of genes that are depleted of observed passenger mutations due to purifying selection. The sensitivity with which STIMS detects purifying selection increases when 1) mutations can be reliably detected at lower frequencies; 2) the number of genes in the gene set increases; 3) the interval between metagenomic sampling time is smaller; and 4) the length of the time series increases (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Figure 4C
                              </a>
                              ). All of these experimental variables increase the resolution of the data and increase the ability of STIMS to detect purifying selection in hypermutator populations. Representative visualizations of STIMS when run on hypermutator populations are shown for genes evolving under positive selection in
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Figure 4D
                              </a>
                              , for genes evolving completely neutrally in
                              <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                               Figure 4E
                              </a>
                              , and for genes evolving under purifying selection in
                              <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                               Figure 4F
                              </a>
                              .
                             </p>
                             <p id="p-20">
                              These results show that STIMS is an effective test for selection, given sufficient data. However, the realized DFE for microbial populations evolving in the laboratory or in nature may deviate substantially from the idealized DFE in the simulations that we have presented. For this reason, we validated STIMS on published metagenomic data tracking the allele frequency dynamics in the LTEE, using three gold standard sets of genes with empirical evidence of relaxed, purifying, and positive selection in the LTEE. These positive controls provide hard evidence that STIMS is effective in practice.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Control 1: Resolving relaxed selection in the LTEE
                             </h3>
                             <p id="p-21">
                              As a first positive control, we examined 63 genes that have empirical evidence for neutral fitness effects in REL606 (A. Couce, personal communication).
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              shows the results of applying STIMS on these genes, which we suppose are evolving under relaxed selection in the LTEE. In 5 out of 6 nonmutator populations (top two rows), the trajectory of this gene set falls within the null distribution. The remaining nonmutator population, Ara+5, has no mutations in this gene set, and so falls significantly below the null distribution (two-tailed bootstrap: Bonferroni-corrected p < 0.005). In 5 out of 6 hypermutator populations (bottom two rows), the trajectory of this gene set falls within the null distribution. The remaining hypermutator population, Ara+6 lies at the margin of the null distribution (two-tailed bootstrap: Bonferroni-corrected p = 0.06).
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Supplementary Figure S1
                              </a>
                              shows the result of applying STIMS on these genes, summed across all 12 LTEE populations. Overall, the global tempo of observed mutations in the nonmutator populations is best explained by the occurrence and fixation of beneficial mutations with small numbers of hitchhikers. By contrast, the global tempo of observed mutations in the hypermutator populations is best explained by large numbers of nearly neutral mutations hitchhiking to high frequency with a much smaller number of beneficial mutations [
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ].
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F5.large.jpg?width=800&height=600&carousel=1" title="Gold-standard genes under relaxed selection. Each panel shows the cumulative number of mutations in the gold-standard neutral genes (black) in the 12 LTEE populations. For comparison, random sets of 57 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray, in order to show a two-tailed statistical comparison of the cumulative number of mutations in the gold-standard neutral gene set to the null distribution at α = 0.05. The top six panels are populations with the ancestral mutation rate (nonmutator populations) and the bottom six panels are those that evolved elevated mutation rates (hypermutator populations)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Gold-standard genes under relaxed selection.</span> Each panel shows the cumulative number of mutations in the gold-standard neutral genes (black) in the 12 LTEE populations. For comparison, random sets of 57 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray, in order to show a two-tailed statistical comparison of the cumulative number of mutations in the gold-standard neutral gene set to the null distribution at <em>α</em> = 0.05. The top six panels are populations with the ancestral mutation rate (nonmutator populations) and the bottom six panels are those that evolved elevated mutation rates (hypermutator populations).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F5.medium.gif" width="440" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F5.medium.gif" width="440" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Gold-standard genes under relaxed selection.
                               </span>
                               <p id="p-22" class="first-child">
                                Each panel shows the cumulative number of mutations in the gold-standard neutral genes (black) in the 12 LTEE populations. For comparison, random sets of 57 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray, in order to show a two-tailed statistical comparison of the cumulative number of mutations in the gold-standard neutral gene set to the null distribution at α = 0.05. The top six panels are populations with the ancestral mutation rate (nonmutator populations) and the bottom six panels are those that evolved elevated mutation rates (hypermutator populations).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Control 2: Detecting purifying selection in the LTEE
                             </h3>
                             <p id="p-23">
                              As a second control, we examined genes that were experimentally shown to be essential or nearly essential in REL606 under LTEE conditions [
                              <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ]. Many of the genes under strongest positive selection in the LTEE are also essential genes [
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ]. Thus, we excluded 24 essential genes with evidence of parallel evolution in the LTEE (Materials and Methods). We hypothesized that the remaining 491 essential genes would show evidence of purifying selection in the hypermutator populations, such that those loci would be depleted in mutations in comparison to the null distribution (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 6
                              </a>
                              and
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Supplementary Figure S2
                              </a>
                              ). Indeed, these 491 loci show significant depletion of observed mutations in all 6 hypermutator populations (one-tailed bootstrap: p < 0.0001 for Ara−1; p < 0.01 for Ara−2; p < 0.01 for Ara−3; p < 0.005 for Ara−4; p < 0.02 for Ara+3; p < 0.0001 for Ara+6). Additionally, one of the six nonmutator populations, Ara+1 shows a marginally significant depletion of mutations at these loci (one-tailed bootstrap: p < 0.05). Ara+1 evolved a insertion-sequence hypermutator phenotype early in its evolutionary history [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              –
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ], so it is plausible that signatures of purifying selection can be seen in Ara+1 [
                              <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ], even though it did not evolve an elevated point-mutation rate [
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ].
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F6.large.jpg?width=800&height=600&carousel=1" title="Gold-standard genes under purifying selection. Each panel shows the cumulative number of mutations in the gold standard set of genes under purifying selection (black) in the 12 LTEE populations. For comparison, random sets of 490 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Gold-standard genes under purifying selection.</span> Each panel shows the cumulative number of mutations in the gold standard set of genes under purifying selection (black) in the 12 LTEE populations. For comparison, random sets of 490 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F6.medium.gif" width="440" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F6.medium.gif" width="440" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Gold-standard genes under purifying selection.
                               </span>
                               <p id="p-24" class="first-child">
                                Each panel shows the cumulative number of mutations in the gold standard set of genes under purifying selection (black) in the 12 LTEE populations. For comparison, random sets of 490 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              Despite the global signal of purifying selection on these 491 essential genes, it is not clear which specific ones are under purifying selection. The best candidates are the ones that have no mutations whatsoever in the LTEE, while the second-best are those that were only affected by synonymous mutations. We found 65 genes with no observed mutations in the metagenomics data, 18 of which are essential. 105 genes have only synonymous mutations, 30 of which are essential. We examined 491 essential genes, out of 3948 genes in the genome that passed our filters (Materials and Methods). Based on these numbers, there is a significant association between essentiality and having no observed mutations (Fisher’s exact test: p = 0.0007) and between essentiality and only having synonymous mutations (Fisher’s exact test: p < 10
                              <sup>
                               −5
                              </sup>
                              ). The first set of candidates for purifying selection in the LTEE are reported in
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Supplementary Table S1
                              </a>
                              , and the second set of candidates are reported in
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Supplementary Table S2
                              </a>
                              . As one would expect, many of these genes encode proteins that catalyze key biological reactions, such as ATP synthase and ribosomal proteins.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Control 3: Verifying positive selection in the LTEE
                             </h3>
                             <p id="p-26">
                              As a final control, we examined genes that have previously been shown to be evolving under positive selection in the LTEE. We examined the genes with the strongest signal of parallel evolution in genomes sampled over the first 50,000 generations of the LTEE [
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ].
                             </p>
                             <p id="p-27">
                              We first considered the 50 genes with the most parallelism in the nonmutator populations (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Figure 7
                              </a>
                              and
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Supplementary Figure S3
                              </a>
                              ). STIMS recovered the expected signal of positive selection on these genes in the nonmutator populations (one-tailed bootstrap: p < 0.0001 in all cases). STIMS also found an overall signal of positive selection on these genes in the hypermutator populations (one-tailed bootstrap: p < 0.05 in all hypermutator populations).
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F7.large.jpg?width=800&height=600&carousel=1" title="Gold-standard genes under positive selection. Each panel shows the cumulative number of mutations in the gold standard set of genes under positive selection (black) in the 12 LTEE populations. These genes are the 50 genes with the strongest signal of parallel evolution in clones isolated from the six nonmutator populations, as scored by Tenaillon et al. (2016). For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Gold-standard genes under positive selection.</span> Each panel shows the cumulative number of mutations in the gold standard set of genes under positive selection (black) in the 12 LTEE populations. These genes are the 50 genes with the strongest signal of parallel evolution in clones isolated from the six nonmutator populations, as scored by Tenaillon et al. (2016). For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F7.medium.gif" width="440" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F7.medium.gif" width="440" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <span class="caption-title">
                                Gold-standard genes under positive selection.
                               </span>
                               <p id="p-28" class="first-child">
                                Each panel shows the cumulative number of mutations in the gold standard set of genes under positive selection (black) in the 12 LTEE populations. These genes are the 50 genes with the strongest signal of parallel evolution in clones isolated from the six nonmutator populations, as scored by Tenaillon et al. (2016). For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-29">
                              In the same fashion, we examined the genes with the strongest signal of parallel evolution in the hypermutator genomes (
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Supplementary Figure S4
                              </a>
                              and
                              <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                               Supplementary Figure S5
                              </a>
                              ). The nonmutator populations accumulate mutations in this gene set at a rate similar to background. As expected, 5 out 6 hypermutator populations show a strong excess of mutations above background (one-tailed bootstrap: p < 0.02 in Ara−1, Ara−2, Ara−4, Ara+3, Ara+6). In the remaining population, Ara−3, mutations in this gene set accumulate at the background rate. It is unclear why this population is an outlier, but this finding may be related to the evolution of a citrate metabolic innovation that modified the ecology of this population [
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ].
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Comparison of STIMS to a test for selection based on the Poisson distribution
                             </h3>
                             <p id="p-30">
                              We also benchmarked STIMS against a statistical test for selection that uses the Poisson distribution to model the expected distribution of mutations per genomic site per unit time under neutral evolution [
                              <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ,
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. In brief, STIMS trades off sensitivity for higher specificity (lower false positive rate) in comparison to the Poisson method, and also has the advantage of showing how the tempo of molecular evolution varies over time in a gene set of interest. These findings, and additional considerations, are described in detail in the Supplementary Text.
                             </p>
                             <div id="sec-8" class="subsection">
                              <h4>
                               Application to module decompositions of the E. coli genome
                              </h4>
                              <p id="p-31">
                               Our control experiments demonstrate that STIMS is capable of recovering signals of purifying and positive selection. In this section, we use STIMS to examine how different modules of genes in the E. coli genome are evolving in the LTEE. Our goal is to develop testable hypotheses for which genetic modules and biochemical pathways underlie the ongoing fitness gains observed in the LTEE [
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                               ]. We examined three different module decompositions of the E. coli genome. The first partitioned the E. coli proteome into sectors based on quantitative mass spectrometry [
                               <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ]; the second reported sets of genes (“eigengenes”) whose expression best predicted E. coli growth rates [
                               <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">
                                28
                               </a>
                               ]; while the third partitioned the gene regulatory network into independent components based on gene expression [
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                               ].
                              </p>
                             </div>
                             <div id="sec-9" class="subsection">
                              <h4>
                               Two proteome sectors show evidence of purifying selection
                              </h4>
                              <p id="p-32">
                               We examined proteome sectors that were identified by quantitative mass spectroscopy during carbon-, nitrogen-, and ribosome-limiting growth conditions [
                               <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ].
                               <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                                Supplementary Figure S6
                               </a>
                               shows our findings. One proteome sector was significantly depleted in mutations in Ara−1, Ara+3, and Ara+6, suggesting purifying selection, and also shows evidence of positive selection in Ara+5. This one, called the U-sector (green), contains genes that were not upregulated by any of the growth-limitation treatments in [
                               <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ]. Rather, the expression of the proteins in the U-sector show a generic positive correlation with growth rate across all of the conditions tested by Hui et al. [
                               <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ]. When STIMS is applied to these modules, considering mutations summed across all LTEE populations, we also see an overall depletion of mutations in proteins in the R-sector (yellow;
                               <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                                Supplementary Figure S7
                               </a>
                               ). Proteins in the R-sector are upregulated under ribosome-limiting growth conditions, and are associated with translation [
                               <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ].
                              </p>
                             </div>
                             <div id="sec-10" class="subsection">
                              <h4>
                               No evidence of selection on E. coli eigengenes
                              </h4>
                              <p id="p-33">
                               Finally, we examined the sets of genes (called eigengenes), that were most predictive of E. coli’s growth rate (and presumably fitness under LTEE conditions), using different E. coli strains and growth environments [
                               <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">
                                28
                               </a>
                               ]. No eigengenes showed any evidence of selection in the LTEE (
                               <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                                Supplementary Figure S8
                               </a>
                               and S9).
                              </p>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               Regulators of I-modulons are under positive selection
                              </h4>
                              <p id="p-34">
                               Sastry et al. (2019) used independent component analysis on the E. coli transcriptome, sampled across diverse conditions and strains, to infer 92 independent modules (called I-modulons) in the E. coli gene regulatory network (GRN). 61 of the 92 I-modulons correspond to known transcription factors and regulators in the GRN. We asked two questions: first, is there a difference between how the regulators and regulated genes in the I-modulons evolve in the LTEE? Second, are any I-modulons enriched with or depleted of mutations in the LTEE?
                              </p>
                              <p id="p-35">
                               We find that transcriptional regulators of the I-modulons are under strong positive selection in the LTEE, while the genes within I-modulons are under much weaker selection (
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                Figure 8
                               </a>
                               ). In all 12 populations, regulators of I-modulons are under stronger selection than the genes in the I-modulons themselves. I-modulon regulators are under very strong selection in all 6 nonmutator populations (one-tailed bootstrap: p < 0.001 in all nonmutator populations), and show evidence of ongoing positive selection in some hypermutator populations as well (one-tailed bootstrap: p = 0.03 in Ara−3, p ≤ 0.0001 in Ara+3 and Ara+6). By contrast, the evolution of genes within I-modulons fits the null distribution in most populations, although Ara−4 and Ara+3 show evidence of purifying selection. When mutations across all LTEE populations are considered, genes regulated within I-modulons show an overall signal of purifying selection (
                               <a id="xref-fig-19-1" class="xref-fig" href="#F19">
                                Supplementary Figure S10
                               </a>
                               ). Together, these results indicate that the upper levels of the regulatory hierarchy of the E. coli GRN are under strong selection in the LTEE, while the effector genes at the lower levels of the gene regulatory hierarchy are under weaker selection.
                              </p>
                              <div id="F8" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F8.large.jpg?width=800&height=600&carousel=1" title="I-modulon regulators evolve under strong positive selection, while genes regulated within I-modulon evolve under weaker selection. Each panel shows the cumulative number of mutations in 70 I-modulon regulators (black) and in 1,394 genes regulated within I-modulons (red) in the 12 LTEE populations. For comparison to the I-modulon regulators, random sets of 70 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of the null distribution for I-modulon regulators is shown in gray. A similar procedure was used with random sets of 1,394 genes to make a null distribution in pink to compare to the genes regulated within I-modulons. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup">I-modulon regulators evolve under strong positive selection, while genes regulated within I-modulon evolve under weaker selection. Each panel shows the cumulative number of mutations in 70 I-modulon regulators (black) and in 1,394 genes regulated within I-modulons (red) in the 12 LTEE populations. For comparison to the I-modulon regulators, random sets of 70 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of the null distribution for I-modulon regulators is shown in gray. A similar procedure was used with random sets of 1,394 genes to make a null distribution in pink to compare to the genes regulated within I-modulons. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F8.medium.gif" width="440" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F8.medium.gif" width="440" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 8.
                                </span>
                                <p id="p-36" class="first-child">
                                 I-modulon regulators evolve under strong positive selection, while genes regulated within I-modulon evolve under weaker selection. Each panel shows the cumulative number of mutations in 70 I-modulon regulators (black) and in 1,394 genes regulated within I-modulons (red) in the 12 LTEE populations. For comparison to the I-modulon regulators, random sets of 70 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of the null distribution for I-modulon regulators is shown in gray. A similar procedure was used with random sets of 1,394 genes to make a null distribution in pink to compare to the genes regulated within I-modulons. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-37">
                               <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                                Figure 8
                               </a>
                               also shows that the tempo of I-modulon regulator evolution is quite variable across populations. Consider the number of mutations in I-modulon regulators in the nonmutator populations. After 40,000 generations, Ara+1 has 5, Ara+2 has 4, Ara+4 has 2, and Ara+5 has 1, while Ara−5 and Ara−6 have none. Variation in the tempo of evolution of I-modulon regulators is also seen in the hypermutator populations, in particular the strong signal of ongoing positive selection found in Ara+3 and Ara+6, in contrast with the complete lack of signal in Ara−4.
                              </p>
                              <p id="p-38">
                               While I-modulons tend to evolve at background rates within individual populations (
                               <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                                Figure 8
                               </a>
                               ), some specific I-modulons show evidence of idiosyncratic selection in the hypermutator populations. We hypothesize that some of these cases represent historical contingency and epistasis in the LTEE; these hypotheses could be tested in future studies. The results for all I-modulons are provided in Supplementary Files 1 and 2, while particular I-modulons of interest are listed in Supplementary Table 3.
                              </p>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               Summary of module decomposition results
                              </h4>
                              <p id="p-39">
                               Altogether, we find that the protein-coding genes with the strongest transcriptional response to changing conditions show little to no evidence of selection in the LTEE. By contrast, top-level transcriptional regulators showed strong signatures of positive selection in all nonmutator populations, as well as in some hypermutator populations. Based on these findings, we hypothesize that the genes that show the strongest transcriptional response to changing conditions tend to be at the lower levels of the gene regulatory network, while natural selection in the LTEE is largely operating on regulators of those genes.
                              </p>
                             </div>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Cis-regulatory regions of key regulators show significantly more parallel evolution than the cis-regulatory regions of downstream targets
                             </h3>
                             <p id="p-40">
                              If upper levels of the gene regulatory network are under stronger selection in the LTEE, then cis-regulatory regions of I-modulon regulators should also show more evidence of positive selection than the genes within I-modulons. We tested this prediction by examining non-coding mutations associated with I-modulon regulators and their downstream targets. These non-coding mutations occurred within the promoter regions (up to 100 bp upstream) of their annotated gene [
                              <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ]. There are 48 non-coding mutations associated with 70 I-modulon regulators, and 542 non-coding mutations associated with 1394 genes regulated within I-modulons. These data are consistent with our prediction (Binomial test with 48 successes out of 590 trials and expected probability of success = 70/1464: p = 0.0005).
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Application of STIMS to hypermutator populations of Pseudomonas aeruginosa evolving under antibiotic selection
                             </h3>
                             <p id="p-41">
                              We wanted to know whether STIMS could be applied beyond the LTEE. Unlike the LTEE, however, most evolution experiments do not have genomic, let alone metagenomic, data spanning 30+ years of evolutionary history. We sourced the metagenomic time series data reported by Mehta et al. [
                              <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ], in which replicate populations of Pseudomonas aeruginosa adapted to subinhibitory concentrations of colistin in continuous chemostat culture for one month. We reasoned that the Mehta dataset would be ideal for applying STIMS since both replicate populations in that experiment rapidly evolved hypermutator phenotypes. We asked whether annotated regulatory genes in the P. aeruginosa PAO11 genome showed evidence of positive selection in this evolution experiment (
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Figure 9
                              </a>
                              ,
                              <a id="xref-fig-20-1" class="xref-fig" href="#F20">
                               Supplementary Figure S11
                              </a>
                              ); indeed, STIMS reveals a clear signal of positive selection on the pre-specified set of 424 regulatory genes (one-tailed bootstrap: p < 0.001). This finding reveals that STIMS has the power to detect gene sets under strong positive or purifying selection, even when applied to metagenomic time series which span relatively short time periods.
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F9.large.jpg?width=800&height=600&carousel=1" title="Regulatory genes are under positive selection in hypermutator Pseudomonas aeruginosa populations evolving under subinhibitory concentrations of colistin. The cumulative number of mutations in 424 annotated regulatory genes in two replicate populations of hypermutator P. aeruginosa is shown in black. For comparison, random sets of 424 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Regulatory genes are under positive selection in hypermutator <em>Pseudomonas aeruginosa</em> populations evolving under subinhibitory concentrations of colistin. The cumulative number of mutations in 424 annotated regulatory genes in two replicate populations of hypermutator <em>P. aeruginosa</em> is shown in black. For comparison, random sets of 424 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F9.medium.gif" width="440" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 9." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F9.medium.gif" width="440" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 9.
                               </span>
                               <p id="p-42" class="first-child">
                                Regulatory genes are under positive selection in hypermutator Pseudomonas aeruginosa populations evolving under subinhibitory concentrations of colistin. The cumulative number of mutations in 424 annotated regulatory genes in two replicate populations of hypermutator P. aeruginosa is shown in black. For comparison, random sets of 424 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             DISCUSSION
                            </h2>
                            <p id="p-43">
                             We show that signals of both positive and purifying selection can be detected in metagenomic time series of large asexual haploid populations, including Escherichia coli from Lenski’s long-term evolution experiment (LTEE) and a clinically relevant pathogen, Pseudomonas aeruginosa, in a short-term evolution experiment under antibiotic selection.
                            </p>
                            <p id="p-44">
                             In contrast to previous studies, which examined global patterns in the tempo and mode of evolution in the LTEE [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ], our analysis focuses on functional modules encoded by the E. coli genome. We find that the tempo of evolution in particular molecular subsystems gives us insight into the mode of evolution acting on those modules (i.e., relaxed, purifying, or positive selection). We ran simulations to demonstrate that STIMS works in an idealized system in which selection pressures and the genomic DFE can be pre-specified a priori, and we ran computational positive control experiments to confirm that STIMS works on genes with prior evidence of relaxed (
                             <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                              Figure 5
                             </a>
                             ), purifying (
                             <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                              Figure 6
                             </a>
                             ), and positive selection (
                             <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                              Figure 7
                             </a>
                             ).
                            </p>
                            <p id="p-45">
                             Our findings indicate that the accelerated pace of genomic evolution in the hypermutators, combined with the detailed record of molecular evolution provided by metagenomic time series, may open new opportunities for understanding the genomic basis of adaptive evolution, even though the signal of selection is obscured in hypermutator genomes due to genomic draft [
                             <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. First, the vast number of nearly neutral hitchhikers that are observed in hypermutator populations provides insight into how mutations in modifier alleles affect genome-wide and local mutation rates and biases [
                             <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ]. Second, regions of the genome that are highly depleted in mutations in the hypermutator populations are strong candidates for purifying selection. Third and most importantly— metagenomic sequencing gives deep sampling of genetic variation off the (eventual) line of descent.
                            </p>
                            <p id="p-46">
                             We found compelling evidence of purifying selection in the hypermutator populations of the LTEE. Many of the strongest candidate genes for purifying selection are deeply conserved over evolutionary time, such as those encoding ribosomal subunits. The action of purifying selection on the hypermutator populations is also consistent with the observation that antimutator alleles have fixed in several populations of the LTEE [
                             <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ,
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              36
                             </a>
                             ], and with the experimental finding that overexpressing RNA chaperones in some hypermutator LTEE strains reduces the mutation load in those strains [
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              37
                             </a>
                             ]. Together, these observations suggest that hypermutator lineages are accumulating some deleterious passenger mutations, even as their fitness continues to increase. In any case, our analysis shows that the depletion of mutations in particular genes is not due to the evolution of antimutator alleles: by bootstrapping a null distribution of background rates, STIMS controls for the genome- and population-wide effects of antimutator alleles over time.
                            </p>
                            <p id="p-47">
                             The resampling approach used by STIMS can also take the effects of local mutation biases into account [
                             <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                              38
                             </a>
                             ]. For instance, one can control for local mutation biases by constructing null distributions that directly model chromosomal variation in mutation rates and biases. This can be somewhat complicated, given that mutational biases [
                             <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ] and regional mutation rates over the chromosome have evolved idiosyncratically across the replicate LTEE populations [
                             <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ]. Our implementation of STIMS samples gene sets uniformly over the E. coli genome: we also implemented a sampling procedure that take the wave pattern of mutation rate variation over the E. coli chromosome into account [
                             <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ,
                             <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                              39
                             </a>
                             ,
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             ], but this more complicated procedure produced the same results as sampling genes uniformly over the chromosome (Materials and Methods).
                            </p>
                            <p id="p-48">
                             When we applied STIMS to different module decompositions of the E. coli genome, we found compelling evidence of strong positive selection on key global regulators of the E. coli gene regulatory network, especially in comparison to the genes that they regulate. Furthermore, we found an excess of mutations in the cis-regulatory regions of those regulators in comparison to the genes that they regulate. One explanation could be that mutations that affect the cis-regulation and structure of global regulators at higher levels of the GRN cause a cascade of effects on downstream targets, and so are more effective targets for fine-tuning the E. coli GRN. Strikingly, we found evidence of continued strong positive selection on key regulatory genes in the two populations with the most mutations in the LTEE: Ara+3 and Ara+6 (
                             <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                              Figure 8
                             </a>
                             ). By contrast, two nonmutator populations, Ara−5 and Ara−6, showed no mutations at all in I-modulon regulators in the last 20,000 generations of the time course. This suggests that the GRN may initially evolve close to some local fitness maximum (subject to pleiotropic constraints), but then evolve further to compensate for the effects of mutations elsewhere in the genome.
                            </p>
                            <p id="p-49">
                             Further work will be needed to test the hypothesis that ongoing evolution of I-modulon regulators is related to compensatory evolution. It is possible that the hypermutator populations are evolving to compensate for an increasing mutation load of deleterious hitchhiker mutations [
                             <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ]. The appearance of multiple antimutator alleles in the LTEE suggests that that hypermutability has a hidden cost [
                             <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ,
                             <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                              36
                             </a>
                             ], but the magnitude of any mutation load in the LTEE populations remains unknown. Ongoing positive selection on the E. coli GRN could also be due to compensatory evolution that is not specifically for hitchhiking deleterious mutations. For instance, it is possible that early beneficial mutations become deleterious due to further mutations [
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              41
                             </a>
                             ,
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              42
                             </a>
                             ]. Furthermore, natural selection may greedily favor mutationally accessible but suboptimal trajectories [
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              43
                             </a>
                             ,
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              44
                             </a>
                             ] that then open new, idiosyncratic paths for further refinement [
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ].
                            </p>
                            <p id="p-50">
                             Our work has some limitations. By using STIMS to examine the tempo of GRN evolution in the LTEE using the same dataset, we find a number of patterns with an unknown rate of false positives. Therefore, future work is needed to test the specific hypotheses that we have generated, either by using additional time-course data from the LTEE, by analyzing related evolution experiments, or by experimental validation. An important caveat is that deletions that fix in the LTEE can lead to spurious inferences of purifying selection, if they are not taken into account, since deleted genes cannot accumulate mutations. By that same token, gene duplications, amplifications, or other forms of copy number variation could lead to spurious inferences of positive selection, or elevated mutation rates [
                             <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                              46
                             </a>
                             ,
                             <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                              47
                             </a>
                             ]. Those types of mutations appear to be rare in these data, in comparison to the point mutations, indels, and transposon insertions that we count. The complications induced by copy number variation may need to be considered (or safely ignored), depending on the context in which STIMS is used. Finally, STIMS is computationally intensive due to its use of bootstrapping. Often equivalent statistical results can be derived much faster, using exact tests like the binomial test or Fisher’s exact test [
                             <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ], or by using tests based on the Poisson distribution [
                             <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                              34
                             </a>
                             ,
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              48
                             </a>
                             ]. The advantage of STIMS is that it provides greater biological insight, by visualizing how statistical signatures of positive and purifying selection change in an evolving population over time.
                            </p>
                            <p id="p-51">
                             Overall, our work provides greater insight into the mechanistic connection between genotypic and phenotypic evolution in evolution experiments like the LTEE. The genetic architecture underlying fitness improvements in the LTEE likely involves a relatively small number of loci that control global aspects of cellular physiology [
                             <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ,
                             <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              49
                             </a>
                             ]. These factors may control many downstream pathways, which are being modulated in response to selection in the LTEE. Hence, we hypothesize that the regulation of these pathways is being rewired during the LTEE, often without significantly changing their downstream effectors. This would explain why the modules that show the strongest changes in expression, and best predict growth rate and fitness in response to environmental conditions [
                             <a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             –
                             <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ] show little evidence of adaptive evolution in the LTEE. Overall, the genotype-phenotype map of E. coli, at least in the context of the LTEE, resembles the hierarchical “supervisor-worker” gene architecture proposed by Chen et al. [
                             <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                              50
                             </a>
                             ] to explain the genetic architecture of quantitative traits in Saccharomyces cerevisiae.
                            </p>
                            <p id="p-52">
                             As the costs of protein production exerts a strong contrast on cellular energetics, the proper allocation of proteome resources in order to maximize growth, rather the evolution of particular genetic modules, may be the cellular phenotype under strongest selection in the LTEE [
                             <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                              51
                             </a>
                             –
                             <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                              54
                             </a>
                             ]. Future work could test this hypothesis by trying to mimic evolutionary changes in the LTEE by directly perturbing the balance of global physiological and metabolic state variables (chromosome conformation, ppGpp, cAMP levels and redox potential) to maximize growth in the ancestral REL606 strain. Coevolution between the function of those proteome resources, pleiotropic constraints on optimal allocation, as well as feedback with the environment could all play a part in causing the open-ended increases in fitness seen in the LTEE [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ,
                             <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ].
                            </p>
                            <p id="p-53">
                             Finally, we expect our methodology will be broadly useful, especially for workers in the experimental evolution field. To demonstrate the generality of STIMS, we reanalyzed data from a second evolution experiment in which two replicate hypermutator populations of P. aeruginosa adapted to subinhibitory concentrations of colistin [
                             <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ]. Our finding of positive selection on regulatory genes in this experiment validates STIMS as a general method and indicates that the rapid evolution of bacterial gene regulatory networks may be a general mechanism for adaptation during experimental evolution, and perhaps during adaptation to novel environments in general. We expect that our basic idea could be more rigorously justified by deeper theoretical work, and further extended to study evolving populations and communities in both the laboratory and in the wild. Purifying selection is of particular interest in the study of viruses and cancers, for the sake of finding conserved and effective drug targets [
                             <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                              55
                             </a>
                             –
                             <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                              58
                             </a>
                             ]. In particular, we anticipate that STIMS could be applied to clinical time series, such as genomic sampling from cystic fibrosis patients [
                             <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                              59
                             </a>
                             ,
                             <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                              60
                             </a>
                             ], in order to discover gene modules that are under selection in pathogens as they adapt to their host [
                             <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                              61
                             </a>
                             ,
                             <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                              62
                             </a>
                             ].
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             MATERIALS AND METHODS
                            </h2>
                            <div id="sec-17" class="subsection">
                             <h3>
                              The STIMS algorithm
                             </h3>
                             <p id="p-54">
                              In brief, STIMS counts the cumulative number of mutations occurring over time in a module of interest, and compares that number to a null distribution that is constructed by subsampling random sets of genes over the genome, in which the cardinality of these random gene sets is fixed to the cardinality of the module of interest. We normalize the number of observed mutations in a gene set by the total length of that gene set in base-pairs.
                             </p>
                             <p id="p-55">
                              In this work, we specifically counted point mutations, small insertions and deletions (indels), and transposon insertions (structural variants) that were detected in metagenomic time series using the breseq pipeline [
                              <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ,
                              <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ,
                              <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                               63
                              </a>
                              ]. Nonetheless, STIMS does not depend on any specific variant calling software, nor does it specifically require metagenomic time series (genomic time series would suffice). STIMS only requires a list of observed de novo mutations over time (i.e. information about the location of each mutation, and the time at which each was first observed in the population).
                             </p>
                             <p id="p-56">
                              By bootstrapping a null distribution based on random gene sets, STIMS implicitly controls for genome-wide variation in the mutation rate (since this will affect all genes). In order to control for local variations in mutation rate [
                              <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ,
                              <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ,
                              <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ], we subsampled random modules from the same chromosomal neighborhood as the module of interest (dividing the REL606 genome into 46 bins, roughly ∼100,000 bp each), under the assumption that nearby genes have similar mutation rates. This approach gave the same results as sampling genes uniformly across the genome, despite a higher computational cost. For this reason, the results reported here use the simplest method to construct the null distribution, in which genes are sampled without considering their position on the chromosome.
                             </p>
                             <p id="p-57">
                              Bootstrapped p-values were calculated separately for each population. p-values for one-sided tests are exact estimates of the relevant tail of the null distribution, where p is the upper-tail (or lower-tail) probability of the event of sampling a normalized cumulative number of mutations under the null distribution that is greater than (or less than) the normalized cumulative number of mutations of the module of interest. Two-tailed tests against the bootstrapped null distribution were calculated with a false-positive (type I error) probability α = 0.025 to account for both tails; this is equivalent to multiplying p-values by a factor of 2, in a Bonferroni correction for two statistical tests (i.e., on the upper tail and on the lower tail of the null distribution).
                             </p>
                             <p id="p-58">
                              In the visualizations shown in the figures, the top 2.5% and bottom 2.5% of points in the null distribution are omitted, such that each panel can be directly interpreted as a randomization test with a false-positive (type I error) probability α = 0.05.
                             </p>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Filtering of genes in REL606 genome
                             </h3>
                             <p id="p-59">
                              We analyzed the pre-processed LTEE metagenomics data published by [
                              <a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ], and so we applied filters to exclude regions of the REL606 genome that were omitted from that analysis. In particular, pseudogenes and genes in repetitive regions of the genome were excluded. A site was marked as repetitive if (1) it was annotated as a repeat region in the REL606 reference, (2) it was present in the set of masked regions compiled by [
                              <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ], or (3) it fell within the putative prophage at REL606 genome coordinates 880528–904682.
                             </p>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              Essential genes excluded from purifying selection control experiment
                             </h3>
                             <p id="p-60">
                              We excluded essential genes that showed parallel evolution in Tenaillon et al. [
                              <a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ]. In that work, a G-score was calculated to measure parallel evolution. G-scores were calculated separately for the nonmutator and hypermutator populations. We ranked all genes in REL606 that passed our filters by their nonmutator and hypermutator G-scores. Essential genes that were in the top 50 of either list were excluded from the purifying selection control experiment. 21 out of the top 50 G-scoring nonmutator genes were essential in REL606, based on the transposon mutagenesis and sequencing experiments carried out by Couce et al. [
                              <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ]. These genes were: arcA, arcB, crp, fabF, ftsI, hflB, infB, infC, mrdA, mreB, mreC, mreD, nusA, rne, rplF, rpoB, rpsD, sapF, spoT, topA, and yabB. 3 out of the top 50 G-scoring hypermutator genes were essential in REL606. These genes were: sapA, tilS, and yhbH.
                             </p>
                            </div>
                            <div id="sec-20" class="subsection">
                             <h3>
                              Analysis of regulatory genes in the month-long hypermutator P. aeruginosa evolution experiment
                             </h3>
                             <p id="p-61">
                              We analyzed the pre-processed metagenomics data from the hypermutator P. aeruginosa evolution experiment published by Mehta et al. [
                              <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ]. We excluded the Pf4 bacteriophage in the P. aeruginosa PAO11 reference genome used by Mehta et al. from the STIMS analysis; this region was also omitted from the statistical analysis in the original report [
                              <a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ].
                             </p>
                             <p id="p-62">
                              We used the keywords “regulator”, “regulatory”, and “regulation” to search for regulatory genes in the PAO11 genome to identify de novo mutations arising in their evolution experiment. This search resulted in a set of 424 regulatory genes, which we then analyzed using STIMS.
                             </p>
                            </div>
                            <div id="sec-21" class="subsection">
                             <h3>
                              Implementation of a test for selection based on the Poisson distribution
                             </h3>
                             <p id="p-63">
                              We implemented the Poisson method described by Kinnersley et al. [
                              <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. We summed all mutations across the LTEE associated with genes that passed our filters, and divided by the total length of those genes to calculate the background density of mutations per site, λ. For each gene g in the genome, we scale λ by the length of g, to calculate the expected number of mutations at gene g: λ
                              <sub>
                               g
                              </sub>
                              . The same calculation holds for a set of genes: by multiplying λ by the total length of genes in the set, we can calculate the expected number of mutations in the gene set, λ
                              <sub>
                               set
                              </sub>
                              . Suppose x mutations were counted in the gene set of interest. Then, a one-tailed test for purifying selection in this gene set can be done by calculating a p-value from the lower-tail of the Poisson distribution:
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/13/2020.05.23.112508/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/13/2020.05.23.112508/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Similarly, a one-tailed test for positive selection in the gene set can be done by directly calculating a p-value from the upper-tail of the Poisson distribution:
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/13/2020.05.23.112508/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/13/2020.05.23.112508/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-64">
                              When multiple genes or gene sets were tested for positive or purifying selection, we applied a Benjamini-Hochberg correction for multiple-testing, and report those false-discovery-rate (FDR) corrected p-values.
                             </p>
                            </div>
                            <div id="sec-22" class="subsection">
                             <h3>
                              Wright-Fisher simulations of microbial evolution
                             </h3>
                             <p id="p-65">
                              We implemented Wright-Fisher evolutionary simulations of asexual haploid populations in SLiM [
                              <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                               64
                              </a>
                              ]. We simulated populations of 1 million cells, each containing a genome of 1,000 contiguous genes. We tested STIMS in two scenarios: the evolution of a nonmutator populations with a point-mutation rate of 10
                              <sup>
                               −10
                              </sup>
                              per base-pair per generation, and the evolution of a hypermutator populations with a 100-fold increase in mutation rate: 10
                              <sup>
                               −8
                              </sup>
                              per base-pair per generation. These population genetic parameters are comparable to the effective population size (3.3 × 10
                              <sup>
                               7
                              </sup>
                              ) and the ancestral mutation rate (8.9 ×10
                              <sup>
                               −11
                              </sup>
                              ) of Lenski’s LTEE [
                              <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ], and the 100-fold evolved increases in mutation rates observed in the LTEE [
                              <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ]. Our simulations assume a background genomic distribution of mutation fitness effects (DFE) in which the vast majority of mutations are nearly neutral, with a long tail of deleterious mutations, based on high-throughput measurements of the DFE in E. coli in microfluidic mutation accumulation experiments [
                              <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">
                               65
                              </a>
                              ]. We assume that 100 genes are evolving under positive selection, such that 10% of mutations occurring in those genes are beneficial. This assumption implies that 0.0025 of mutations are beneficial in our model, which is comparable to the experimental estimate that 1 out of 150 newly arising mutations in E. coli are beneficial in laboratory conditions [
                              <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                               66
                              </a>
                              ]. Following recent simulations of adaptive evolution in the LTEE, we draw beneficial mutations from an exponential DFE with mean 0.01578 [
                              <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ]. We further assume that 100 genes are evolving completely neutrally such that mutations in these genes have no fitness effect, and assume that another set of 100 genes are evolving under purifying selection, such that 40% of mutations in those genes cause a 30% decrease in fitness. This last assumption matches the finding that 1% of new mutations in E. coli are lethal under laboratory conditions [
                              <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">
                               65
                              </a>
                              ].
                             </p>
                             <p id="p-66">
                              Ten replicate simulations were run for both the nonmutator and hypermutator populations, and all mutations in each simulation were saved at 100-generation intervals. Mutations were filtered based on a 1% allele frequency threshold, to simulate metagenomic sequencing: this is equivalent to uniform 100× short-read sequencing coverage across the genome per sequencing sample, with no sequencing errors. We then ran STIMS on the 100 genes under positive selection, the 100 genes under relaxed selection (those evolving completely neutrally), and the 100 genes under purifying selection. We examined the sensitivity and specificity of STIMS by varying the allele frequency detection threshold, the number of genes sampled per module, the sampling interval in the time series, and the length of the time series, holding all other setting to their default values. Four values of the allele frequency detection limit were examined: 0.01%, 1%, 5%, 10%. The number of genes sampled per module was varied between 1, 25, 50, and 100 genes. The sampling interval was either 100- or 500-generation intervals. The length of the time series was 500, 1000, 2000, or 5000 generations. True positives and false negatives were counted by the number of STIMS tests on the genes evolving under positive or purifying selection that passed a one-sided p-value significance threshold of 0.05.
                             </p>
                             <p id="p-67">
                              False positives and true negatives were counted by the number of STIMS tests on the genes evolving neutrally that nevertheless passed the same p-value significance threshold. Sensitivity (also known as the true positive rate, or as statistical power) was calculated as the number of true positives divided by the sum of true positives and false negatives, out of the 10 tests on the 10 replicate datasets. Specificity (also known as the true negative rate) was calculated as the number of true negatives divided by the sum of true negatives and false positives, out of the 10 tests on the 10 replicate datasets.
                             </p>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              Data and analysis code availability
                             </h3>
                             <p id="p-68">
                              All analysis codes for this project are available at:
                              <a href="http://www.github.com/rohanmaddamsetti/LTEE-purifying-selection">
                               www.github.com/rohanmaddamsetti/LTEE-purifying-selection
                              </a>
                              . A stand-alone implementation of STIMS in the Julia programming language is available at:
                              <a href="http://www.github.com/rohanmaddamsetti/STIMS">
                               www.github.com/rohanmaddamsetti/STIMS
                              </a>
                              . The sequence data used in our study have already been deposited into a public database. The other data and analysis scripts in this manuscript have been archived on the Zenodo Digital Repository (DOI:
                              <a href="http://doi.org/10.5281/zenodo.4783368">
                               http://doi.org/10.5281/zenodo.4783368
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             SUPPLEMENTARY INFORMATION
                            </h2>
                            <p id="p-70">
                             Supplementary File 1. Results of running STIMS, over all LTEE populations, on each I-modulon in Sastry et al. (2019).
                            </p>
                            <p id="p-71">
                             Supplementary File 2. Results of running STIMS, individually on each each LTEE population, on each I-modulon in Sastry et al. (2019).
                            </p>
                            <p id="p-72">
                             Supplementary File 3. Results of running Poisson method as a genome-wide screen for positive and purifying selection, over all LTEE populations, and individually on each LTEE population.
                            </p>
                           </div>
                           <div class="section" id="sec-26">
                            <h2 class="">
                             SUPPLEMENTARY TEXT
                            </h2>
                            <div class="section" id="sec-27">
                             <h3>
                              Comparison of STIMS to a test for selection based on the Poisson distribution
                             </h3>
                             <p id="p-73">
                              One class of statistical tests for selection in evolution experiments uses the Poisson distribution to model the expected distribution of mutations per genomic site per unit time under neutral evolution [
                              <a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ,
                              <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. Using this framework, significant deviations from the Poisson null expectation can indicate positive or purifying selection. In principle, Poisson methods should be simpler to implement and computationally cheaper than STIMS. However, methods based on the Poisson distribution may be sensitive to model misspecification, due to local mutation biases that can cause deviation from the Poisson expectation in the absence of selection [
                              <a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ]. Variations of the bootstrapping approach used by STIMS, on the other hand, can account for such mutational biases through randomization (Materials and Methods).
                             </p>
                             <p id="p-74">
                              To better understand the relative strengths and weaknesses of STIMS in comparison to Poisson methods, we implemented the Poisson method used by Kinnersley et al. [
                              <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ], and re-ran all of our positive control experiments on gold-standard sets of genes (Materials and Methods). Overall, both STIMS and the Poisson method give similar results. The Poisson method successfully recovers signals of purifying selection (p < 10
                              <sup>
                               −14
                              </sup>
                              ) and positive selection (p < 10
                              <sup>
                               −50
                              </sup>
                              ) on the respective sets of gold-standard genes. However, the Poisson method finds a weak signal of positive selection on the gold-standard set of genes under relaxed selection (p = 0.0397). The Poisson method finds a signal of purifying selection on most proteome sectors (FDR-corrected p < 10
                              <sup>
                               −11
                              </sup>
                              for U-sector; FDR-corrected p < 0.00001 for A-sector; FDR-corrected p = 0.000162 for R-sector; FDR-corrected p = 0.00182 for S-sector; FDR-corrected p = 0.00217 for O-sector; FDR-corrected p = 0.391 for C-sector). Eigengenes show little to no evidence of selection under the Poisson method (FDR-corrected p = 0.0327 for Eigengene 2; the rest are nonsignificant). I-modulon regulators show evidence of very strong positive selection (p < 10
                              <sup>
                               −13
                              </sup>
                              ), while 16 I-modulons show evidence of either positive or purifying selection at a Benjamini-Hochberg corrected p-value threshold of 0.01 (Supplementary Table 4). 14 out of these 16 I-modulons are significant by STIMS, while the other two, “flu-yeeRS” and “Pyruvate”, trend toward significance (Supplementary File S1). Finally, we used the Poisson method as a genome-wide screen for positive and purifying selection, both over all LTEE populations, and on each population individually. We report these results at a Benjamini-Hochberg corrected p-value threshold of 0.01 in Supplementary File S3. Most of the genes showing evidence of positive selection have been reported before [
                              <a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ,
                              <a id="xref-ref-16-11" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ,
                              <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ]. However, this screen finds 7 genes with a significant signal of purifying selection across the LTEE, none of which have been reported before: acnB (aconitate hydratase), ybjL (hypothetical protein), yhdP (conserved membrane protein predicted transporter), recG (ATP-dependent DNA helicase), cyoB (cytochrome o ubiquinol oxidase subunit I), wcaJ (predicted UDP-glucose lipid carrier transferase), and atpD (F0F1 ATP synthase subunit beta). All of these genes are longer than 1,300 bp, and three of them (ybjL, wcaJ, and atpD) have no observed mutations at all.
                             </p>
                             <p id="p-75">
                              Altogether, this detailed comparison indicates the Poisson method is faster and is more practical for genome-wide scans for positive and purifying selection on single genes, but that its sensitivity comes at the price of more false positives. STIMS is more conservative than the Poisson method, and gives results that are more consistent with ground truth, as defined by the sets of gold-standard genes that we used for empirical validation. STIMS, unlike the Poisson method, also provides a picture of the tempo of evolution in a gene set over time, as well as changes in statistical significance over time. These visualizations are especially useful for observing how the tempo of evolutionary change in a gene set of interest changes over time, and for generating further hypotheses and predictions for empirical validation (Supplementary Files 1 and 2).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-28">
                            <h2 class="">
                             SUPPLEMENTARY TABLES AND FIGURES
                            </h2>
                            <div id="T1" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2447891/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2447891/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2447891" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table S1.
                              </span>
                              <p id="p-76" class="first-child">
                               Essential genes in REL606 with no observed mutations in the LTEE metagenomics data. encoded by prophage CP-933O
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T2" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2447897/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2447897/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2447897" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table S2.
                              </span>
                              <p id="p-77" class="first-child">
                               Essential genes in REL606 with only synonymous mutations in the LTEE metagenomics data.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T3" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2447907/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2447907/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2447907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table S3.
                              </span>
                              <p id="p-78" class="first-child">
                               I-modulons showing evidence of selection, historical contingency and epistasis in the LTEE, using STIMS.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T4" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2447902/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2447902/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2447902" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table S4.
                              </span>
                              <p id="p-79" class="first-child">
                               I-modulons showing evidence of selection across all LTEE populations based on the Poisson method, at a FDR-corrected p-value threshold of 0.01.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F10.large.jpg?width=800&height=600&carousel=1" title="Gold-standard genes under relaxed selection. Each panel shows the cumulative number of mutations in the gold-standard neutral genes (black) in all 12 LTEE populations. For comparison, random sets of 57 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray, in order to show a two-tailed statistical comparison of the cumulative number of mutations in the gold-standard neutral gene set to the null distribution at α= 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Gold-standard genes under relaxed selection.</span> Each panel shows the cumulative number of mutations in the gold-standard neutral genes (black) in all 12 LTEE populations. For comparison, random sets of 57 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray, in order to show a two-tailed statistical comparison of the cumulative number of mutations in the gold-standard neutral gene set to the null distribution at <em>α</em>= 0.05.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F10.medium.gif" width="440" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F10.medium.gif" width="440" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S1.
                              </span>
                              <span class="caption-title">
                               Gold-standard genes under relaxed selection.
                              </span>
                              <p id="p-80" class="first-child">
                               Each panel shows the cumulative number of mutations in the gold-standard neutral genes (black) in all 12 LTEE populations. For comparison, random sets of 57 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray, in order to show a two-tailed statistical comparison of the cumulative number of mutations in the gold-standard neutral gene set to the null distribution at α= 0.05.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F11.large.jpg?width=800&height=600&carousel=1" title="Gold-standard genes under purifying selection. Each panel shows the cumulative number of mutations in the gold standard set of genes under purifying selection (black) in all 12 LTEE populations. For comparison, random sets of 490 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Gold-standard genes under purifying selection.</span> Each panel shows the cumulative number of mutations in the gold standard set of genes under purifying selection (black) in all 12 LTEE populations. For comparison, random sets of 490 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F11.medium.gif" width="440" height="437"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F11.medium.gif" width="440" height="437"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S2.
                              </span>
                              <span class="caption-title">
                               Gold-standard genes under purifying selection.
                              </span>
                              <p id="p-81" class="first-child">
                               Each panel shows the cumulative number of mutations in the gold standard set of genes under purifying selection (black) in all 12 LTEE populations. For comparison, random sets of 490 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F12.large.jpg?width=800&height=600&carousel=1" title="Gold-standard genes under positive selection. Each panel shows the cumulative number of mutations in the gold standard set of genes under positive selection (black) in all 12 LTEE populations. These genes are the 50 genes with the strongest signal of parallel evolution in clones isolated from the six nonmutator populations, as scored by Tenaillon et al. (2016). For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Gold-standard genes under positive selection.</span> Each panel shows the cumulative number of mutations in the gold standard set of genes under positive selection (black) in all 12 LTEE populations. These genes are the 50 genes with the strongest signal of parallel evolution in clones isolated from the six nonmutator populations, as scored by Tenaillon et al. (2016). For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F12.medium.gif" width="440" height="438"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F12.medium.gif" width="440" height="438"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S3.
                              </span>
                              <span class="caption-title">
                               Gold-standard genes under positive selection.
                              </span>
                              <p id="p-82" class="first-child">
                               Each panel shows the cumulative number of mutations in the gold standard set of genes under positive selection (black) in all 12 LTEE populations. These genes are the 50 genes with the strongest signal of parallel evolution in clones isolated from the six nonmutator populations, as scored by Tenaillon et al. (2016). For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F13" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F13.large.jpg?width=800&height=600&carousel=1" title="Genes evolving in parallel in hypermutator populations. Each panel shows the cumulative number of mutations in the 50 genes with the strongest signal of parallel evolution in clones isolated from the six hypermutator populations, as scored by Tenaillon et al. (2016), in the 12 LTEE populations. These curves are drawn in black. For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Genes evolving in parallel in hypermutator populations.</span> Each panel shows the cumulative number of mutations in the 50 genes with the strongest signal of parallel evolution in clones isolated from the six hypermutator populations, as scored by Tenaillon et al. (2016), in the 12 LTEE populations. These curves are drawn in black. For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F13.medium.gif" width="440" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F13.medium.gif" width="440" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S4.
                              </span>
                              <span class="caption-title">
                               Genes evolving in parallel in hypermutator populations.
                              </span>
                              <p id="p-83" class="first-child">
                               Each panel shows the cumulative number of mutations in the 50 genes with the strongest signal of parallel evolution in clones isolated from the six hypermutator populations, as scored by Tenaillon et al. (2016), in the 12 LTEE populations. These curves are drawn in black. For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F14" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F14.large.jpg?width=800&height=600&carousel=1" title="Genes evolving in parallel in hypermutator populations. Each panel shows the cumulative number of mutations in the 50 genes with the strongest signal of parallel evolution in clones isolated from the six hypermutator populations, as scored by Tenaillon et al. (2016), in all 12 LTEE populations. These curves are drawn in black. For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Genes evolving in parallel in hypermutator populations.</span> Each panel shows the cumulative number of mutations in the 50 genes with the strongest signal of parallel evolution in clones isolated from the six hypermutator populations, as scored by Tenaillon et al. (2016), in all 12 LTEE populations. These curves are drawn in black. For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F14.medium.gif" width="440" height="439"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F14.medium.gif" width="440" height="439"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S5.
                              </span>
                              <span class="caption-title">
                               Genes evolving in parallel in hypermutator populations.
                              </span>
                              <p id="p-84" class="first-child">
                               Each panel shows the cumulative number of mutations in the 50 genes with the strongest signal of parallel evolution in clones isolated from the six hypermutator populations, as scored by Tenaillon et al. (2016), in all 12 LTEE populations. These curves are drawn in black. For comparison, random sets of 50 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F15" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F15.large.jpg?width=800&height=600&carousel=1" title="Purifying selection on the proteome U-sector. Each panel shows the cumulative number of mutations in the A-sector (black), S-sector (red), O-sector (blue), U-sector (green), R-sector (yellow), and C-sector (orange) in the 12 LTEE populations. For comparison, random sets of 92 genes (i.e. the smallest proteome sector cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Purifying selection on the proteome U-sector.</span> Each panel shows the cumulative number of mutations in the A-sector (black), S-sector (red), O-sector (blue), U-sector (green), R-sector (yellow), and C-sector (orange) in the 12 LTEE populations. For comparison, random sets of 92 genes (i.e. the smallest proteome sector cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F15.medium.gif" width="440" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F15.medium.gif" width="440" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S6." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S6.
                              </span>
                              <span class="caption-title">
                               Purifying selection on the proteome U-sector.
                              </span>
                              <p id="p-85" class="first-child">
                               Each panel shows the cumulative number of mutations in the A-sector (black), S-sector (red), O-sector (blue), U-sector (green), R-sector (yellow), and C-sector (orange) in the 12 LTEE populations. For comparison, random sets of 92 genes (i.e. the smallest proteome sector cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F16" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F16.large.jpg?width=800&height=600&carousel=1" title="Purifying selection on the proteome U-sector and R-sector. Each panel shows the cumulative number of mutations in the A-sector (black), S-sector (red), O-sector (blue), U-sector (green), R-sector (yellow), and C-sector (orange) in all 12 LTEE populations. For comparison, random sets of 92 genes (i.e. the smallest proteome sector cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Purifying selection on the proteome U-sector and R-sector.</span> Each panel shows the cumulative number of mutations in the A-sector (black), S-sector (red), O-sector (blue), U-sector (green), R-sector (yellow), and C-sector (orange) in all 12 LTEE populations. For comparison, random sets of 92 genes (i.e. the smallest proteome sector cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F16.medium.gif" width="440" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F16.medium.gif" width="440" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S7." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S7.
                              </span>
                              <span class="caption-title">
                               Purifying selection on the proteome U-sector and R-sector.
                              </span>
                              <p id="p-86" class="first-child">
                               Each panel shows the cumulative number of mutations in the A-sector (black), S-sector (red), O-sector (blue), U-sector (green), R-sector (yellow), and C-sector (orange) in all 12 LTEE populations. For comparison, random sets of 92 genes (i.e. the smallest proteome sector cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F17" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F17.large.jpg?width=800&height=600&carousel=1" title="No evidence of selection on E. coli eigengenes. Each panel shows the cumulative number of mutations in eigengene 1 (red), eigengene 2 (orange), eigengene 3 (yellow), eigengene 4 (green), eigengene 5 (cyan), eigengene 6 (blue), eigengene 7 (violet), eigengene 8 (pink), and eigengene 9 (black) in the 12 LTEE populations. For comparison, random sets of 15 genes (i.e. the smallest eigengene cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">No evidence of selection on <em>E. coli</em> eigengenes.</span> Each panel shows the cumulative number of mutations in eigengene 1 (red), eigengene 2 (orange), eigengene 3 (yellow), eigengene 4 (green), eigengene 5 (cyan), eigengene 6 (blue), eigengene 7 (violet), eigengene 8 (pink), and eigengene 9 (black) in the 12 LTEE populations. For comparison, random sets of 15 genes (i.e. the smallest eigengene cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F17.medium.gif" width="440" height="439"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F17.medium.gif" width="440" height="439"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S8." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S8.
                              </span>
                              <span class="caption-title">
                               No evidence of selection on E. coli eigengenes.
                              </span>
                              <p id="p-87" class="first-child">
                               Each panel shows the cumulative number of mutations in eigengene 1 (red), eigengene 2 (orange), eigengene 3 (yellow), eigengene 4 (green), eigengene 5 (cyan), eigengene 6 (blue), eigengene 7 (violet), eigengene 8 (pink), and eigengene 9 (black) in the 12 LTEE populations. For comparison, random sets of 15 genes (i.e. the smallest eigengene cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. The top six panels are the nonmutator populations and the bottom six panels are the hypermutator populations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F18" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F18.large.jpg?width=800&height=600&carousel=1" title="No evidence of selection on E. coli eigengenes. Each panel shows the cumulative number of mutations in eigengene 1 (red), eigengene 2 (orange), eigengene 3 (yellow), eigengene 4 (green), eigengene 5 (cyan), eigengene 6 (blue), eigengene 7 (violet), eigengene 8 (pink), and eigengene 9 (black) in the 12 LTEE populations. For comparison, random sets of 15 genes (i.e. the smallest eigengene cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">No evidence of selection on <em>E. coli</em> eigengenes.</span> Each panel shows the cumulative number of mutations in eigengene 1 (red), eigengene 2 (orange), eigengene 3 (yellow), eigengene 4 (green), eigengene 5 (cyan), eigengene 6 (blue), eigengene 7 (violet), eigengene 8 (pink), and eigengene 9 (black) in the 12 LTEE populations. For comparison, random sets of 15 genes (i.e. the smallest eigengene cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F18.medium.gif" width="440" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S9." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F18.medium.gif" width="440" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S9." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S9.
                              </span>
                              <span class="caption-title">
                               No evidence of selection on E. coli eigengenes.
                              </span>
                              <p id="p-88" class="first-child">
                               Each panel shows the cumulative number of mutations in eigengene 1 (red), eigengene 2 (orange), eigengene 3 (yellow), eigengene 4 (green), eigengene 5 (cyan), eigengene 6 (blue), eigengene 7 (violet), eigengene 8 (pink), and eigengene 9 (black) in the 12 LTEE populations. For comparison, random sets of 15 genes (i.e. the smallest eigengene cardinality) were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F19" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F19.large.jpg?width=800&height=600&carousel=1" title="I-modulon regulators evolve under strong positive selection, while genes regulated within I-modulon evolve under purifying selection. Each panel shows the cumulative number of mutations in 70 I-modulon regulators (black) and in 1,394 genes regulated by I-modulon regulators (red) in the 12 LTEE populations. For comparison to the I-modulon regulators, random sets of 70 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. A similar procedure was used with random sets of 1,394 genes to make a comparable distribution in pink to compare to the genes regulated by I-modulon regulators." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup">I-modulon regulators evolve under strong positive selection, while genes regulated within I-modulon evolve under purifying selection. Each panel shows the cumulative number of mutations in 70 I-modulon regulators (black) and in 1,394 genes regulated by I-modulon regulators (red) in the 12 LTEE populations. For comparison to the I-modulon regulators, random sets of 70 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. A similar procedure was used with random sets of 1,394 genes to make a comparable distribution in pink to compare to the genes regulated by I-modulon regulators.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F19.medium.gif" width="440" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S10." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F19.medium.gif" width="440" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S10." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S10.
                              </span>
                              <p id="p-89" class="first-child">
                               I-modulon regulators evolve under strong positive selection, while genes regulated within I-modulon evolve under purifying selection. Each panel shows the cumulative number of mutations in 70 I-modulon regulators (black) and in 1,394 genes regulated by I-modulon regulators (red) in the 12 LTEE populations. For comparison to the I-modulon regulators, random sets of 70 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray. A similar procedure was used with random sets of 1,394 genes to make a comparable distribution in pink to compare to the genes regulated by I-modulon regulators.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F20" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F20.large.jpg?width=800&height=600&carousel=1" title="Regulatory genes are under positive selection in hypermutator Pseudomonas aeruginosa populations evolving under subinhibitory concentrations of colistin. Each panel shows the cumulative number of mutations in 424 annotated regulatory genes (black), in two replicate populations of hypermutator P. aeruginosa. For comparison, random sets of 424 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-561057806" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Regulatory genes are under positive selection in hypermutator <em>Pseudomonas aeruginosa</em> populations evolving under subinhibitory concentrations of colistin. Each panel shows the cumulative number of mutations in 424 annotated regulatory genes (black), in two replicate populations of hypermutator <em>P. aeruginosa</em>. For comparison, random sets of 424 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F20.medium.gif" width="440" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure S11." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F20.medium.gif" width="440" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S11." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/13/2020.05.23.112508/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure S11.
                              </span>
                              <p id="p-90" class="first-child">
                               Regulatory genes are under positive selection in hypermutator Pseudomonas aeruginosa populations evolving under subinhibitory concentrations of colistin. Each panel shows the cumulative number of mutations in 424 annotated regulatory genes (black), in two replicate populations of hypermutator P. aeruginosa. For comparison, random sets of 424 genes were sampled 1,000 times, and the cumulative number of mutations in those random gene sets, normalized by gene length, were calculated. The middle 95% of this null distribution is shown in gray.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             ACKNOWLEDGEMENTS
                            </h2>
                            <p id="p-69">
                             We thank Alejandro Couce for sharing the list of gold-standard neutral genes in E. coli B strain REL606. We thank Richard Lenski for valuable discussions and advice. We thank Jeffrey Barrick for valuable discussions, advice, and comments on an earlier version of our manuscript. We also thank Benjamin Good for making pre-processed LTEE metagenomic data and analysis scripts accessible for the research community. The LTEE that generated the bacteria we used in this study is supported by a grant from the National Science Foundation (currently DEB-1951307) to Richard Lenski and Jeffrey Barrick. N.A.G. was supported in part by the BEACON Center for the Study of Evolution in Action (NSF cooperative agreement DBI-0939454) and by Michigan State University.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn" id="fn-1">
                              <p id="p-1">
                               <a class="rev-xref" href="#xref-fn-1-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                †
                               </span>
                               Joint first author
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-5">
                               This version of the manuscript has been revised to include simulations of bacterial evolution. In addition, we perform additional analyses and have heavily revised the text.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             REFERENCES
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.1" data-doi="10.1038/nrc.2015.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jeggo
                                  </span>
                                  <span class="cit-name-given-names">
                                   PA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pearl
                                  </span>
                                  <span class="cit-name-given-names">
                                   LH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carr
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  DNA repair, genome stability and cancer: a historical perspective
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Cancer
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  35
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BCancer%26rft.volume%253D16%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc.2015.4%26rft_id%253Dinfo%253Apmid%252F26667849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrc.2015.4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26667849&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.23.112508v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Loeb
                                  </span>
                                  <span class="cit-name-given-names">
                                   LA
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A mutator phenotype in cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer research
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  61
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  3230
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DLoeb%26rft.auinit1%253DL.%2BA.%26rft.volume%253D61%26rft.issue%253D8%26rft.spage%253D3230%26rft.epage%253D3239%26rft.atitle%253DA%2BMutator%2BPhenotype%2Bin%2BCancer%26rft_id%253Dinfo%253Apmid%252F11309271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjk6IjYxLzgvMzIzMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzEzLzIwMjAuMDUuMjMuMTEyNTA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.3" data-doi="10.1126/science.7801122">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Modrich
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mismatch repair, genetic stability, and cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1994
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  266
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5193
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1959
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  61
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DModrich%26rft.auinit1%253DP%26rft.volume%253D266%26rft.issue%253D5193%26rft.spage%253D1959%26rft.epage%253D1960%26rft.atitle%253DMismatch%2Brepair%252C%2Bgenetic%2Bstability%252C%2Band%2Bcancer%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7801122%26rft_id%253Dinfo%253Apmid%252F7801122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTM6IjI2Ni81MTkzLzE5NTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.4" data-doi="10.1371/journal.pgen.1004651">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feliziani
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marvig
                                  </span>
                                  <span class="cit-name-given-names">
                                   RL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lujan
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moyano
                                  </span>
                                  <span class="cit-name-given-names">
                                   AJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Di Rienzo
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johansen
                                  </span>
                                  <span class="cit-name-given-names">
                                   HK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Coexistence and within-host evolution of diversified lineages of hypermutable Pseudomonas aeruginosa in long-term cystic fibrosis infections
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genet
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1004651
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D10%26rft.spage%253De1004651%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004651%26rft_id%253Dinfo%253Apmid%252F25330091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004651&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25330091&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.5" data-doi="10.1038/ng.2656">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ford
                                  </span>
                                  <span class="cit-name-given-names">
                                   CB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shah
                                  </span>
                                  <span class="cit-name-given-names">
                                   RR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maeda
                                  </span>
                                  <span class="cit-name-given-names">
                                   MK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gagneux
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                  <span class="cit-name-given-names">
                                   MB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Mycobacterium tuberculosis mutation rate estimates from different lineages predict substantial differences in the emergence of drug-resistant tuberculosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  45
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  784
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  90
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.volume%253D45%26rft.spage%253D784%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2656%26rft_id%253Dinfo%253Apmid%252F23749189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.2656&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23749189&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.6" data-doi="10.1073/pnas.1705887114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Couce
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caudwell
                                  </span>
                                  <span class="cit-name-given-names">
                                   LV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feinauer
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hindré
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feugeas
                                  </span>
                                  <span class="cit-name-given-names">
                                   J-P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weigt
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Mutator genomes decay, despite sustained fitness gains, in a long-term experiment with bacteria
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  43
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  E9026
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E35
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1705887114%26rft_id%253Dinfo%253Apmid%252F29073099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQzL0U5MDI2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.7" data-doi="10.1534/genetics.111.132027">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schiffels
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szöllősi
                                  </span>
                                  <span class="cit-name-given-names">
                                   GJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mustonen
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lässig
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Emergent neutrality in adaptive asexual evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  189
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1361
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  75
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132027%26rft_id%253Dinfo%253Apmid%252F21926305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS80LzEzNjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mehta
                                  </span>
                                  <span class="cit-name-given-names">
                                   HH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prater
                                  </span>
                                  <span class="cit-name-given-names">
                                   AG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beabout
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elworth
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Karavis
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gibbons
                                  </span>
                                  <span class="cit-name-given-names">
                                   HS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The essential role of hypermutation in rapid adaptation to antibiotic stress
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Antimicrobial agents and chemotherapy
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  63
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e00744
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  19
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2Bagents%2Band%2Bchemotherapy%26rft.volume%253D63%26rft.spage%253De00744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.9" data-doi="10.1371/journal.pone.0140489">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hammerstrom
                                  </span>
                                  <span class="cit-name-given-names">
                                   TG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beabout
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clements
                                  </span>
                                  <span class="cit-name-given-names">
                                   TP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saxer
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shamoo
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Acinetobacter baumannii repeatedly evolves a hypermutator phenotype in response to tigecycline that effectively surveys evolutionary trajectories to resistance
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PloS one
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e0140489
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D10%26rft.spage%253De0140489%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0140489%26rft_id%253Dinfo%253Apmid%252F26488727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0140489&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26488727&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.10" data-doi="10.1086/285289">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rose
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simpson
                                  </span>
                                  <span class="cit-name-given-names">
                                   SC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tadler
                                  </span>
                                  <span class="cit-name-given-names">
                                   SC
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long-term experimental evolution in Escherichia coli. I. Adaptation and divergence during 2,000 generations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The American Naturalist
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  138
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1315
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  41
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D138%26rft.spage%253D1315%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1086/285289&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1991HB55000001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Experimental evolution and the dynamics of adaptation and genome evolution in microbial populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The ISME journal
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2181
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  94
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BISME%2Bjournal%26rft.volume%253D11%26rft.spage%253D2181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.12" data-doi="10.1101/sqb.2009.74.018">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Genome-wide mutational diversity in an evolving population of Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cold Spring Harbor symposia on quantitative biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  74
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  119
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  29
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2Bsymposia%2Bon%2Bquantitative%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fsqb.2009.74.018%26rft_id%253Dinfo%253Apmid%252F19776167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6Ijc0LzAvMTE5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.13" data-doi="10.1038/nature08480">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  <span class="cit-name-given-names">
                                   DS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yoon
                                  </span>
                                  <span class="cit-name-given-names">
                                   SH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jeong
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oh
                                  </span>
                                  <span class="cit-name-given-names">
                                   TK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schneider
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genome evolution and adaptation in a long-term experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  461
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7268
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1243
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrick%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D461%26rft.issue%253D7268%26rft.spage%253D1243%26rft.epage%253D1247%26rft.atitle%253DGenome%2Bevolution%2Band%2Badaptation%2Bin%2Ba%2Blong-term%2Bexperiment%2Bwith%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08480%26rft_id%253Dinfo%253Apmid%252F19838166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature08480&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19838166&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000271190800040&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.14" data-doi="10.1534/genetics.115.176677">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddamsetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Adaptation, clonal interference, and frequency-dependent interactions in a long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  200
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  619
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  31
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.176677%26rft_id%253Dinfo%253Apmid%252F25911659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzIvNjE5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.15" data-doi="10.1038/nature18959">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tenaillon
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ribeck
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deatherage
                                  </span>
                                  <span class="cit-name-given-names">
                                   DE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blanchard
                                  </span>
                                  <span class="cit-name-given-names">
                                   JL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dasgupta
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Tempo and mode of genome evolution in a 50,000-generation experiment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  536
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7615
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  165
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  70
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18959%26rft_id%253Dinfo%253Apmid%252F27479321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature18959&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27479321&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.16" data-doi="10.1038/nature24287">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Good
                                  </span>
                                  <span class="cit-name-given-names">
                                   BH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McDonald
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Desai
                                  </span>
                                  <span class="cit-name-given-names">
                                   MM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The dynamics of molecular evolution over 60,000 generations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  551
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7678
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  45
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  50
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24287%26rft_id%253Dinfo%253Apmid%252F29045390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature24287&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29045390&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vasi
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Travisano
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long-term experimental evolution in Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  II. Changes in life-history traits during adaptation to a seasonal environment. The american naturalist
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1994
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  144
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  432
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  56
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DII.%2BChanges%2Bin%2Blife-history%2Btraits%2Bduring%2Badaptation%2Bto%2Ba%2Bseasonal%2Benvironment.%2BThe%2Bamerican%2Bnaturalist%26rft.volume%253D144%26rft.spage%253D432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.23.112508v2.18" data-doi="10.1023/A:1000675521611">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mongold
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sniegowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   PD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Travisano
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vasi
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gerrish
                                  </span>
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Evolution of competitive fitness in experimental populations of E. coli: what makes one genotype a better competitor than another?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Antonie van Leeuwenhoek
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  73
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  35
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  47
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.stitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.aulast%253DLenski%26rft.auinit1%253DR.%2BE.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D47%26rft.atitle%253DEvolution%2Bof%2Bcompetitive%2Bfitness%2Bin%2Bexperimental%2Bpopulations%2Bof%2BE.%2Bcoli%253A%2Bwhat%2Bmakes%2Bone%2Bgenotype%2Ba%2Bbetter%2Bcompetitor%2Bthan%2Banother%253F%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1000675521611%26rft_id%253Dinfo%253Apmid%252F9602277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1023/A:1000675521611&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9602277&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000073459600006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.23.112508v2.19" data-doi="10.1126/science.272.5269.1802">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elena
                                  </span>
                                  <span class="cit-name-given-names">
                                   SF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cooper
                                  </span>
                                  <span class="cit-name-given-names">
                                   VS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Punctuated evolution caused by selection of rare beneficial mutations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  272
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5269
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1802
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DElena%26rft.auinit1%253DS.%2BF.%26rft.volume%253D272%26rft.issue%253D5269%26rft.spage%253D1802%26rft.epage%253D1804%26rft.atitle%253DPunctuated%2BEvolution%2BCaused%2Bby%2BSelection%2Bof%2BRare%2BBeneficial%2BMutations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.272.5269.1802%26rft_id%253Dinfo%253Apmid%252F8650581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzIvNTI2OS8xODAyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.20" data-doi="10.1126/science.1243357">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiser
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ribeck
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long-term dynamics of adaptation in asexual populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  342
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6164
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1364
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1243357%26rft_id%253Dinfo%253Apmid%252F24231808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDIvNjE2NC8xMzY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.21" data-doi="10.1086/714530">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grant
                                  </span>
                                  <span class="cit-name-given-names">
                                   NA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddamsetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Maintenance of metabolic plasticity despite relaxed selection in a long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The American Naturalist
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  0
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  ja
                                 </span>
                                 ):null.
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1086/714530
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft_id%253Dinfo%253Adoi%252F10.1086%252F714530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1086/714530&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grant
                                  </span>
                                  <span class="cit-name-given-names">
                                   NA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Magid
                                  </span>
                                  <span class="cit-name-given-names">
                                   AA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Franklin
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dufour
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Changes in Cell Size and Shape During 50,000 Generations of Experimental Evolution with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Bacteriology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  203
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.23" data-doi="10.1093/GBE/EVX064">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddamsetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hatcher
                                  </span>
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Green
                                  </span>
                                  <span class="cit-name-given-names">
                                   AG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                  <span class="cit-name-given-names">
                                   BL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marks
                                  </span>
                                  <span class="cit-name-given-names">
                                   DS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Core genes evolve rapidly in the long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology and evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1072
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  83
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%2Band%2Bevolution%26rft.volume%253D9%26rft.spage%253D1072%26rft_id%253Dinfo%253Adoi%252F10.1093%252FGBE%252FEVX064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/GBE/EVX064&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.24" data-doi="10.1093/gbe/evab070">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddamsetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Universal constraints on protein evolution in the long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology and Evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/gbe/evab070
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevab070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/gbe/evab070&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.25" data-doi="10.1093/gbe/evab074">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddamsetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Selection maintains protein interactome resilience in the long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology and Evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/gbe/evab074
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevab074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/gbe/evab074&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hui
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Silverman
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   SS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Erickson
                                  </span>
                                  <span class="cit-name-given-names">
                                   DW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Basan
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Quantitative proteomic analysis reveals a simple strategy of global resource allocation in bacteria
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular systems biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sastry
                                  </span>
                                  <span class="cit-name-given-names">
                                   AV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szubin
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hefner
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Escherichia coli transcriptome mostly consists of independently regulated modules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.28" data-doi="10.1073/pnas.1808080116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wytock
                                  </span>
                                  <span class="cit-name-given-names">
                                   TP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Motter
                                  </span>
                                  <span class="cit-name-given-names">
                                   AE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Predicting growth rate from gene expression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  116
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  367
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  72
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1808080116%26rft_id%253Dinfo%253Apmid%252F30578321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTYvMi8zNjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Izutsu
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lake
                                  </span>
                                  <span class="cit-name-given-names">
                                   DM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Matson
                                  </span>
                                  <span class="cit-name-given-names">
                                   ZW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dodson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE.
                                  </span>
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Effects of periodic bottlenecks on the dynamics of adaptive evolution in microbial populations
                                 </abbr>
                                 . bioRxiv.
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.30" data-doi="10.1073/pnas.96.7.3807">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Papadopoulos
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schneider
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meier-Eiss
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arber
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blot
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Genomic evolution during a 10,000-generation experiment with bacteria
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  96
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  3807
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPapadopoulos%26rft.auinit1%253DD.%26rft.volume%253D96%26rft.issue%253D7%26rft.spage%253D3807%26rft.epage%253D3812%26rft.atitle%253DGenomic%2Bevolution%2Bduring%2Ba%2B10%252C000-generation%2Bexperiment%2Bwith%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.7.3807%26rft_id%253Dinfo%253Apmid%252F10097119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni83LzM4MDciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.31" data-doi="10.1093/gbe/evaa178">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddamsetti
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grant
                                  </span>
                                  <span class="cit-name-given-names">
                                   NA
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Divergent evolution of mutation rates and biases in the long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology and Evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/gbe/evaa178
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevaa178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/gbe/evaa178&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.32" data-doi="10.1038/s41467-021-22630-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Consuegra
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaffé
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hindré
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tenaillon
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Insertion-sequence-mediated mutations both promote and constrain evolvability during a long-term experiment with bacteria
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-021-22630-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-021-22630-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Turner
                                  </span>
                                  <span class="cit-name-given-names">
                                   CB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blount
                                  </span>
                                  <span class="cit-name-given-names">
                                   ZD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitchell
                                  </span>
                                  <span class="cit-name-given-names">
                                   DH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Evolution and coexistence in response to a key innovation in a long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 :
                                 <span class="cit-vol">
                                  020958
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kinnersley
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schwartz
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  <span class="cit-name-given-names">
                                   D-D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sherlock
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenzweig
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Evolutionary dynamics and structural consequences of de novo beneficial mutations and mutant lineages arising in a constant environment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  21
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbiology%26rft.volume%253D19%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.35" data-doi="10.1098/rspb.2015.2292">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiser
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ribeck
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blount
                                  </span>
                                  <span class="cit-name-given-names">
                                   ZD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nahum
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morris
                                  </span>
                                  <span class="cit-name-given-names">
                                   JJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Sustained fitness gains and variability in fitness trajectories in the long-term evolution experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the Royal Society B: Biological Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  282
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1821
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  20152292
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2015.2292%26rft_id%253Dinfo%253Apmid%252F26674951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1098/rspb.2015.2292&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26674951&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.36" data-doi="10.1073/pnas.1219574110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wielgoss
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tenaillon
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiser
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dittmar
                                  </span>
                                  <span class="cit-name-given-names">
                                   WJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cruveiller
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Mutation rate dynamics in a bacterial population reflect tension between adaptation and genetic load
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  222
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1219574110%26rft_id%253Dinfo%253Apmid%252F23248287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMS8yMjIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rudan
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schneider
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Warnecke
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krisko
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  RNA chaperones buffer deleterious mutations in E. coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 :
                                 <span class="cit-issue">
                                  e04745
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bailey
                                  </span>
                                  <span class="cit-name-given-names">
                                   SF
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guo
                                  </span>
                                  <span class="cit-name-given-names">
                                   Q
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bataillon
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Identifying drivers of parallel evolution: A regression model approach
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology and evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2801
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%2Band%2Bevolution%26rft.volume%253D10%26rft.spage%253D2801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foster
                                  </span>
                                  <span class="cit-name-given-names">
                                   PL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanson
                                  </span>
                                  <span class="cit-name-given-names">
                                   AJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Popodi
                                  </span>
                                  <span class="cit-name-given-names">
                                   EM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  On the mutational topology of the bacterial genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  G3: Genes, Genomes, Genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  399
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  407
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D3%26rft.spage%253D399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niccum
                                  </span>
                                  <span class="cit-name-given-names">
                                   BA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MohammedIsmail
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foster
                                  </span>
                                  <span class="cit-name-given-names">
                                   PL
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The Symmetrical Wave Pattern of Base-Pair Substitution Rates across the Escherichia coli Chromosome Has Multiple Causes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  mBio
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quandt
                                  </span>
                                  <span class="cit-name-given-names">
                                   EM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gollihar
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blount
                                  </span>
                                  <span class="cit-name-given-names">
                                   ZD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ellington
                                  </span>
                                  <span class="cit-name-given-names">
                                   AD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Georgiou
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Fine-tuning citrate synthase flux potentiates and refines metabolic innovation in the Lenski evolution experiment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 :
                                 <span class="cit-issue">
                                  e09696
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.42" data-doi="10.1111/evo.12467">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zee
                                  </span>
                                  <span class="cit-name-given-names">
                                   PC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mendes-Soares
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  <span class="cit-name-given-names">
                                   YTN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kraemer
                                  </span>
                                  <span class="cit-name-given-names">
                                   SA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keller
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ossowski
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A shift from magnitude to sign epistasis during adaptive evolution of a bacterial social trait
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  68
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2701
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D2701%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12467%26rft_id%253Dinfo%253Apmid%252F24909926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/evo.12467&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24909926&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.43" data-doi="10.1073/pnas.1921881117">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Venkataram
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Monasky
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sikaroodi
                                  </span>
                                  <span class="cit-name-given-names">
                                   SH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kryazhimskiy
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kacar
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Evolutionary stalling and a limit on the power of natural selection to improve a cellular module
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  117
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  31
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  18582
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  90
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1921881117%26rft_id%253Dinfo%253Apmid%252F32680961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE3LzMxLzE4NTgyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodrigues
                                  </span>
                                  <span class="cit-name-given-names">
                                   JV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shakhnovich
                                  </span>
                                  <span class="cit-name-given-names">
                                   EI
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Adaptation to mutational inactivation of an essential gene converges to an accessible suboptimal fitness peak
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.45" data-doi="10.1534/genetics.119.302000">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaznatcheev
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Computational complexity as an ultimate constraint on evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  212
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  245
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  65
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.119.302000%26rft_id%253Dinfo%253Apmid%252F30833289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEyLzEvMjQ1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.46" data-doi="10.1126/science.282.5391.1133">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andersson
                                  </span>
                                  <span class="cit-name-given-names">
                                   DI
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Slechta
                                  </span>
                                  <span class="cit-name-given-names">
                                   ES
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roth
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Evidence that gene amplification underlies adaptive mutability of the bacterial lac operon
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  282
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5391
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1133
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAndersson%26rft.auinit1%253DD.%2BI.%26rft.volume%253D282%26rft.issue%253D5391%26rft.spage%253D1133%26rft.epage%253D1135%26rft.atitle%253DEvidence%2BThat%2BGene%2BAmplification%2BUnderlies%2BAdaptive%2BMutability%2Bof%2Bthe%2BBacterial%2Blac%2BOperon%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5391.1133%26rft_id%253Dinfo%253Apmid%252F9804552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5MS8xMTMzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.47" data-doi="10.1073/pnas.032680899">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hendrickson
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Slechta
                                  </span>
                                  <span class="cit-name-given-names">
                                   ES
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bergthorsson
                                  </span>
                                  <span class="cit-name-given-names">
                                   U
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andersson
                                  </span>
                                  <span class="cit-name-given-names">
                                   DI
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roth
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Amplification– mutagenesis: evidence that “directed” adaptive mutation and general hypermutability result from growth with a selected gene amplification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  99
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2164
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.032680899%26rft_id%253Dinfo%253Apmid%252F11830643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS80LzIxNjQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.48" data-doi="10.1534/genetics.116.193565">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Good
                                  </span>
                                  <span class="cit-name-given-names">
                                   BH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Desai
                                  </span>
                                  <span class="cit-name-given-names">
                                   MM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Evolution of mutation rates in rapidly adapting asexual populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  204
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1249
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  66
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.193565%26rft_id%253Dinfo%253Apmid%252F27646140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC8zLzEyNDkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.49" data-doi="10.1002/bies.20629">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Philippe
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crozat
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenski
                                  </span>
                                  <span class="cit-name-given-names">
                                   RE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schneider
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Evolution of global regulatory networks during a long-term experiment with Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioessays
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  846
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  60
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DPhilippe%26rft.auinit1%253DN.%26rft.volume%253D29%26rft.issue%253D9%26rft.spage%253D846%26rft.epage%253D860%26rft.atitle%253DEvolution%2Bof%2Bglobal%2Bregulatory%2Bnetworks%2Bduring%2Ba%2Blong-term%2Bexperiment%2Bwith%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20629%26rft_id%253Dinfo%253Apmid%252F17691099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/bies.20629&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17691099&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000249285800004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  <span class="cit-name-given-names">
                                   C-I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   He
                                  </span>
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The genotype–phenotype relationships in the light of natural selection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular biology and evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  525
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  42
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft.volume%253D35%26rft.spage%253D525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.51" data-doi="10.1126/science.1192588">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Scott
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gunderson
                                  </span>
                                  <span class="cit-name-given-names">
                                   CW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mateescu
                                  </span>
                                  <span class="cit-name-given-names">
                                   EM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hwa
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Interdependence of cell growth and gene expression: origins and consequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  330
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6007
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1099
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  102
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DScott%26rft.auinit1%253DM.%26rft.volume%253D330%26rft.issue%253D6007%26rft.spage%253D1099%26rft.epage%253D1102%26rft.atitle%253DInterdependence%2Bof%2BCell%2BGrowth%2Band%2BGene%2BExpression%253A%2BOrigins%2Band%2BConsequences%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1192588%26rft_id%253Dinfo%253Apmid%252F21097934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAwNy8xMDk5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.23.112508v2.52" data-doi="10.1038/nature24299">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Erickson
                                  </span>
                                  <span class="cit-name-given-names">
                                   DW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schink
                                  </span>
                                  <span class="cit-name-given-names">
                                   SJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patsalo
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williamson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gerland
                                  </span>
                                  <span class="cit-name-given-names">
                                   U
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hwa
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A global resource allocation strategy governs growth transition kinetics of Escherichia coli
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  551
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7678
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  119
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature24299&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.23.112508v2.53" data-doi="10.1016/j.copbio.2014.01.001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klumpp
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hwa
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Bacterial growth: global effects on gene expression, growth feedback and proteome partition
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current opinion in biotechnology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  96
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  102
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bbiotechnology%26rft.volume%253D28%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2014.01.001%26rft_id%253Dinfo%253Apmid%252F24495512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.copbio.2014.01.001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24495512&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.54" data-doi="10.1038/nature15765">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Basan
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hui
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Okano
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williamson
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Overflow metabolism in Escherichia coli results from efficient proteome allocation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  528
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7580
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  99
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  104
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D528%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15765%26rft_id%253Dinfo%253Apmid%252F26632588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature15765&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26632588&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.55">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mathis
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sokol
                                  </span>
                                  <span class="cit-name-given-names">
                                   ES
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gupta
                                  </span>
                                  <span class="cit-name-given-names">
                                   PB
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Identification of Genes under Purifying Selection in Human Cancers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BioRxiv
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 :
                                 <span class="cit-vol">
                                  129205
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               56.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.23.112508v2.56" data-doi="10.1038/ng.3987">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weghorn
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sunyaev
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Bayesian inference of negative and positive selection in human cancers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1785
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.volume%253D49%26rft.spage%253D1785%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3987%26rft_id%253Dinfo%253Apmid%252F29106416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.3987&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29106416&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               57.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.23.112508v2.57" data-doi="10.1038/nature23477">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patel
                                  </span>
                                  <span class="cit-name-given-names">
                                   SJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanjana
                                  </span>
                                  <span class="cit-name-given-names">
                                   NE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kishton
                                  </span>
                                  <span class="cit-name-given-names">
                                   RJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eidizadeh
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vodnala
                                  </span>
                                  <span class="cit-name-given-names">
                                   SK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cam
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Identification of essential genes for cancer immunotherapy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  548
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7669
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  537
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  42
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D548%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature23477%26rft_id%253Dinfo%253Apmid%252F28783722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature23477&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28783722&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               58.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.58" data-doi="10.1126/sciadv.abb9153">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Giorgi
                                  </span>
                                  <span class="cit-name-given-names">
                                   EE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marichannegowda
                                  </span>
                                  <span class="cit-name-given-names">
                                   MH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foley
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xiao
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kong
                                  </span>
                                  <span class="cit-name-given-names">
                                   X-P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Emergence of SARS-CoV-2 through recombination and strong purifying selection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science Advances
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  27
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  eabb9153
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.abb9153%26rft_id%253Dinfo%253Apmid%252F32937441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiNi8yNy9lYWJiOTE1MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzEzLzIwMjAuMDUuMjMuMTEyNTA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               59.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.59" data-doi="10.1038/ng.997">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lieberman
                                  </span>
                                  <span class="cit-name-given-names">
                                   TD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Michel
                                  </span>
                                  <span class="cit-name-given-names">
                                   J-B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aingaran
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Potter-Bynoe
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roux
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Parallel bacterial evolution within multiple patients identifies candidate pathogenicity genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  43
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1275
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  80
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLieberman%26rft.auinit1%253DT.%2BD.%26rft.volume%253D43%26rft.issue%253D12%26rft.spage%253D1275%26rft.epage%253D1280%26rft.atitle%253DParallel%2Bbacterial%2Bevolution%2Bwithin%2Bmultiple%2Bpatients%2Bidentifies%2Bcandidate%2Bpathogenicity%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.997%26rft_id%253Dinfo%253Apmid%252F22081229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.997&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22081229&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F13%2F2020.05.23.112508.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               60.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caballero
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  <span class="cit-name-given-names">
                                   ST
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coburn
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   PW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Donaldson
                                  </span>
                                  <span class="cit-name-given-names">
                                   SL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Selective sweeps and parallel pathoadaptation drive Pseudomonas aeruginosa evolution in the cystic fibrosis lung
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  MBio
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               61.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chung
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lieberman
                                  </span>
                                  <span class="cit-name-given-names">
                                   TD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vargas
                                  </span>
                                  <span class="cit-name-given-names">
                                   SO
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flett
                                  </span>
                                  <span class="cit-name-given-names">
                                   KB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McAdam
                                  </span>
                                  <span class="cit-name-given-names">
                                   AJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Priebe
                                  </span>
                                  <span class="cit-name-given-names">
                                   GP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Global and local selection acting on the pathogen Stenotrophomonas maltophilia in the human lung
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 :
                                 <span class="cit-issue">
                                  14078
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               62.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.62">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cooper
                                  </span>
                                  <span class="cit-name-given-names">
                                   VS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Honsa
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rowe
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deitrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iverson
                                  </span>
                                  <span class="cit-name-given-names">
                                   AR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whittall
                                  </span>
                                  <span class="cit-name-given-names">
                                   JJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Experimental Evolution in vivo to Identify Selective Pressures During Pneumococcal Colonization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Msystems
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               63.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.63">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Deatherage
                                  </span>
                                  <span class="cit-name-given-names">
                                   DE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Traverse
                                  </span>
                                  <span class="cit-name-given-names">
                                   CC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wolf
                                  </span>
                                  <span class="cit-name-given-names">
                                   LN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barrick
                                  </span>
                                  <span class="cit-name-given-names">
                                   JE
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Detecting rare structural variation in evolving microbial populations from new sequence junctions using breseq
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-issue">
                                  468
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               64.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.64" data-doi="10.1101/418657">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haller
                                  </span>
                                  <span class="cit-name-given-names">
                                   BC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Messer
                                  </span>
                                  <span class="cit-name-given-names">
                                   PW
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  SLiM 3: forward genetic simulations beyond the Wright–Fisher model
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular biology and evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  632
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft.volume%253D36%26rft.spage%253D632%26rft_id%253Dinfo%253Adoi%252F10.1101%252F418657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1101/418657&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               65.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.65" data-doi="10.1126/science.aan0797">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robert
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ollion
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robert
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Song
                                  </span>
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Matic
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elez
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mutation dynamics and fitness effects followed in single cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  359
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6381
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1283
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRobert%26rft.auinit1%253DL.%26rft.volume%253D359%26rft.issue%253D6381%26rft.spage%253D1283%26rft.epage%253D1286%26rft.atitle%253DMutation%2Bdynamics%2Band%2Bfitness%2Beffects%2Bfollowed%2Bin%2Bsingle%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aan0797%26rft_id%253Dinfo%253Apmid%252F29590079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTkvNjM4MS8xMjgzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTMvMjAyMC4wNS4yMy4xMTI1MDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               66.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.23.112508v2.66" data-doi="10.1126/science.1142284">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Perfeito
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fernandes
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mota
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Gordo I. Adaptive mutations in bacteria: high rate and small effects
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  317
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5839
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  813
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPerfeito%26rft.auinit1%253DL.%26rft.volume%253D317%26rft.issue%253D5839%26rft.spage%253D813%26rft.epage%253D815%26rft.atitle%253DAdaptive%2BMutations%2Bin%2BBacteria%253A%2BHigh%2BRate%2Band%2BSmall%2BEffects%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142284%26rft_id%253Dinfo%253Apmid%252F17690297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTgzOS84MTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xMy8yMDIwLjA1LjIzLjExMjUwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.05.23.112508v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.06.30.450478v2" title="miRNAFinder: A Comprehensive Web Resource for Plant Pre-microRNA Classification" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2020.08.23.263608v2" title="Focal Electrical Stimulation of Human Retinal Ganglion Cells for Vision Restoration" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 13, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.05.23.112508v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.05.23.112508v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.05.23.112508v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.05.23.112508v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2444657"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-wckqgmtHK0E-4Gk8NMmEMDfr8vq97oxkXqLR5WTQTTY"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484471170"/>
                                   <input type="hidden" name="captcha_token" value="e9831dc9d1b860cf8fb15c5f5a1e27cc"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2444657--3" data-node-nid="2444657" id="node-2444657--4483933679" data-pisa="biorxiv;2020.05.23.112508v2" data-pisa-master="biorxiv;2020.05.23.112508" data-seqnum="2444657" data-apath="/biorxiv/early/2022/03/13/2020.05.23.112508.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Rohan
                                  </span>
                                  <span class="nlm-surname">
                                   Maddamsetti
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Nkrumah A.
                                  </span>
                                  <span class="nlm-surname">
                                   Grant
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.05.23.112508;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.05.23.112508
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.05.23.112508v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&title=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&title=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&text=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&t=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&title=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&title=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&title=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2444657--5" data-node-nid="2444657" id="citation-node-2444657--61640012497" data-pisa="biorxiv;2020.05.23.112508v2" data-pisa-master="biorxiv;2020.05.23.112508" data-seqnum="2444657" data-apath="/biorxiv/early/2022/03/13/2020.05.23.112508.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Discovery of positive and purifying selection in metagenomic time series of hypermutator microbial populations
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Rohan
                                      </span>
                                      <span class="nlm-surname">
                                       Maddamsetti
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Nkrumah A.
                                      </span>
                                      <span class="nlm-surname">
                                       Grant
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.05.23.112508;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.05.23.112508
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Maddamsetti%2C+Rohan&rft.au=Grant%2C+Nkrumah+A.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2444657/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2444657/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2444657/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2444657/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2444657/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2444657/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2444657/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2444657/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2444657/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2444657/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2444657/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2444657/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&count=horizontal&via=&text=Discovery%20of%20positive%20and%20purifying%20selection%20in%20metagenomic%20time%20series%20of%20hypermutator%20microbial%20populations&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.05.23.112508v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.05.23.112508v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Evolutionary Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
