<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.30.891457v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.30.891457v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.30.891457v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/01/07/2019.12.30.891457.atom"/>
  <meta name="HW.pisa" content="biorxiv;2019.12.30.891457v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)"/>
  <meta name="DC.Identifier" content="10.1101/2019.12.30.891457"/>
  <meta name="DC.Date" content="2022-01-07"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Ancestral sequence reconstruction is a technique that is gaining widespread use in molecular evolution studies and protein engineering. Accurate reconstruction requires the ability to handle appropriately large numbers of sequences, as well as insertion and deletion (“indel”) events, but available approaches exhibit limitations. To address these limitations, we developed Graphical Representation of Ancestral Sequence Predictions (GRASP), which efficiently implements maximum likelihood methods to enable the inference of ancestors of families with more than 10,000 members. GRASP implements partial order graphs (POGs) to represent and infer insertion and deletion events across ancestors, enabling the identification of building blocks for protein engineering.

To validate the capacity to engineer novel proteins from realistic data, we predicted ancestor sequences across three distinct enzyme families: glucose-methanol-choline (GMC) oxidoreductases, cytochromes P450, and dihydroxy/sugar acid dehydratases (DHAD). All tested ancestors demonstrated enzymatic activity. Our study demonstrates the ability of GRASP (1) to support large data sets over 10,000 sequences and (2) to employ insertions and deletions to identify building blocks for engineering biologically active ancestors, by exploring variation over evolutionary time.

Author summary Massive sequencing projects expose the extent of natural, genetic diversity. Here, we describe a method with capacity to perform ancestor sequence reconstruction from data sets in excess of 10,000 sequences, poised to recover ancestral diversity, including the evolutionary events that determine present-time biological function and structure.

We introduce a novel strategy for suggesting “indel variants” that are distinct from, but can be explored alongside, substitution variants for creating ancestral libraries. We demonstrate how indels can be used as building blocks to form “hybrid ancestors”; based on this strategy, we synthesise ancestor variants, with varying enzymatic activities, for wide-ranging applications in the biotechnology sector.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Gabriel Foley"/>
  <meta name="DC.Contributor" content="Ariane Mora"/>
  <meta name="DC.Contributor" content="Connie M. Ross"/>
  <meta name="DC.Contributor" content="Scott Bottoms"/>
  <meta name="DC.Contributor" content="Leander Sützl"/>
  <meta name="DC.Contributor" content="Marnie L. Lamprecht"/>
  <meta name="DC.Contributor" content="Julian Zaugg"/>
  <meta name="DC.Contributor" content="Alexandra Essebier"/>
  <meta name="DC.Contributor" content="Brad Balderson"/>
  <meta name="DC.Contributor" content="Rhys Newell"/>
  <meta name="DC.Contributor" content="Raine E. S. Thomson"/>
  <meta name="DC.Contributor" content="Bostjan Kobe"/>
  <meta name="DC.Contributor" content="Ross T. Barnard"/>
  <meta name="DC.Contributor" content="Luke Guddat"/>
  <meta name="DC.Contributor" content="Gerhard Schenk"/>
  <meta name="DC.Contributor" content="Jörg Carsten"/>
  <meta name="DC.Contributor" content="Yosephine Gumulya"/>
  <meta name="DC.Contributor" content="Burkhard Rost"/>
  <meta name="DC.Contributor" content="Dietmar Haltrich"/>
  <meta name="DC.Contributor" content="Volker Sieber"/>
  <meta name="DC.Contributor" content="Elizabeth M. J. Gillam"/>
  <meta name="DC.Contributor" content="Mikael Bodén"/>
  <meta name="article:published_time" content="2022-01-07"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Ancestral sequence reconstruction is a technique that is gaining widespread use in molecular evolution studies and protein engineering. Accurate reconstruction requires the ability to handle appropriately large numbers of sequences, as well as insertion and deletion (“indel”) events, but available approaches exhibit limitations. To address these limitations, we developed Graphical Representation of Ancestral Sequence Predictions (GRASP), which efficiently implements maximum likelihood methods to enable the inference of ancestors of families with more than 10,000 members. GRASP implements partial order graphs (POGs) to represent and infer insertion and deletion events across ancestors, enabling the identification of building blocks for protein engineering.</p><p>To validate the capacity to engineer novel proteins from realistic data, we predicted ancestor sequences across three distinct enzyme families: glucose-methanol-choline (GMC) oxidoreductases, cytochromes P450, and dihydroxy/sugar acid dehydratases (DHAD). All tested ancestors demonstrated enzymatic activity. Our study demonstrates the ability of GRASP (1) to support large data sets over 10,000 sequences and (2) to employ insertions and deletions to identify building blocks for engineering biologically active ancestors, by exploring variation over evolutionary time.</p><h3>Author summary</h3>
<p>Massive sequencing projects expose the extent of natural, genetic diversity. Here, we describe a method with capacity to perform ancestor sequence reconstruction from data sets in excess of 10,000 sequences, poised to recover <i>ancestral</i> diversity, including the evolutionary events that determine present-time biological function and structure.</p><p>We introduce a novel strategy for suggesting “indel variants” that are distinct from, but can be explored alongside, substitution variants for creating ancestral libraries. We demonstrate how indels can be used as building blocks to form “hybrid ancestors”; based on this strategy, we synthesise ancestor variants, with varying enzymatic activities, for wide-ranging applications in the biotechnology sector.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2019.12.30.891457v3"/>
  <meta name="citation_id" content="2019.12.30.891457v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.30.891457v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.30.891457v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.30.891457v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2019.12.30.891457"/>
  <meta name="citation_num_pages" content="39"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2019.12.30.891457"/>
  <meta name="citation_author" content="Gabriel Foley"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0487-2629"/>
  <meta name="citation_author" content="Ariane Mora"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Connie M. Ross"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Scott Bottoms"/>
  <meta name="citation_author_institution" content="Campus Straubing for Biotechnology and Sustainability, Technische Universität München"/>
  <meta name="citation_author" content="Leander Sützl"/>
  <meta name="citation_author_institution" content="Institut für Lebensmitteltechnologie, Universität für Bodenkultur Wien"/>
  <meta name="citation_author" content="Marnie L. Lamprecht"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Julian Zaugg"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Alexandra Essebier"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Brad Balderson"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Rhys Newell"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Raine E. S. Thomson"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Bostjan Kobe"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author_institution" content="Institute for Molecular Bioscience and Australian Infectious Diseases Research Centre, The University of Queensland"/>
  <meta name="citation_author" content="Ross T. Barnard"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5685-4828"/>
  <meta name="citation_author" content="Luke Guddat"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Gerhard Schenk"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author_institution" content="Sustainable Minerals Institute, The University of Queensland"/>
  <meta name="citation_author" content="Jörg Carsten"/>
  <meta name="citation_author_institution" content="Zentralinstitut für Katalyseforschung, Technische Universität München"/>
  <meta name="citation_author" content="Yosephine Gumulya"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Burkhard Rost"/>
  <meta name="citation_author_institution" content="Fakultät für Informatik, Technische Universität München"/>
  <meta name="citation_author" content="Dietmar Haltrich"/>
  <meta name="citation_author_institution" content="Institut für Lebensmitteltechnologie, Universität für Bodenkultur Wien"/>
  <meta name="citation_author" content="Volker Sieber"/>
  <meta name="citation_author_institution" content="Campus Straubing for Biotechnology and Sustainability, Technische Universität München"/>
  <meta name="citation_author_institution" content="Zentralinstitut für Katalyseforschung, Technische Universität München"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author" content="Elizabeth M. J. Gillam"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author_email" content="m.boden@uq.edu.au"/>
  <meta name="citation_author_email" content="e.gillam@uq.edu.au"/>
  <meta name="citation_author" content="Mikael Bodén"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, The University of Queensland"/>
  <meta name="citation_author_email" content="m.boden@uq.edu.au"/>
  <meta name="citation_author_email" content="e.gillam@uq.edu.au"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3548-268X"/>
  <meta name="citation_reference" content="Land, M. et al. Insights from 20 years of bacterial genome sequencing. Functional & Integrative Genomics 15, 141–161 (2015)."/>
  <meta name="citation_reference" content="Lewin, H. A. et al. Earth BioGenome Project: Sequencing life for the future of life. Proceedings of the National Academy of Sciences 115, 4325–4333 (2018)."/>
  <meta name="citation_reference" content="Gregory, A. C. et al. Marine DNA viral macro- and microdiversity from pole to pole. Cell 177, 1109–1123 (2019)."/>
  <meta name="citation_reference" content="Hochberg, G. K. A. & Thornton, J. W. Reconstructing ancient proteins to understand the causes of structure and function. Annual Review of Biophysics 46, 247–269 (2017)."/>
  <meta name="citation_reference" content="Selberg, A. G. A., Gaucher, E. A. & Liberles, D. A. Ancestral sequence reconstruction: From chemical paleogenetics to maximum likelihood algorithms and beyond. Journal of Molecular Evolution 89, 157–164 (2021)."/>
  <meta name="citation_reference" content="Bar-Rogovsky, H. et al. Assessing the prediction fidelity of ancestral reconstruction by a library approach. Protein Engineering, Design & Selection 28, 507–518 (2015)."/>
  <meta name="citation_reference" content="Sugrue, E., Scott, C. & Jackson, C. J. Constrained evolution of a bispecific enzyme: Lessons for biocatalyst design. Organic & Biomolecular Chemistry 15, 937–946 (2017)."/>
  <meta name="citation_reference" content="Gumulya, Y. et al. Engineering highly functional thermostable proteins using ancestral sequence reconstruction. Nature Catalysis 1, 878–888 (2018)."/>
  <meta name="citation_reference" content="Trudeau, D. L. & Tawfik, D. S. Protein engineers turned evolutionists—the quest for the optimal starting point. Current Opinion in Biotechnology 60, 46–52 (2019)."/>
  <meta name="citation_reference" content="Spence, M. A., Kaczmarski, J. A., Saunders, J. W. & Jackson, C. J. Ancestral sequence reconstruction for protein engineers. Current Opinion in Structural Biology 69, 131–141 (2021)."/>
  <meta name="citation_reference" content="Garcia, A. K. & Kaçar, B. How to resurrect ancestral proteins as proxies for ancient biogeochemistry. Free Radical Biology and Medicine 140, 260–269 (2019)."/>
  <meta name="citation_reference" content="Vialle, R. A., Tamuri, A. U. & Goldman, N. Alignment modulates ancestral sequence reconstruction accuracy. Molecular Biology and Evolution 35, 1783–1797 (2018)."/>
  <meta name="citation_reference" content="Aadland, K. & Kolaczkowski, B. Alignment-integrated reconstruction of ancestral sequences improves accuracy. Genome Biology and Evolution 12, 1549–1565 (2020)."/>
  <meta name="citation_reference" content="Lee, C., Grasso, C. & Sharlow, M. F. Multiple sequence alignment using partial order graphs. Bioinformatics 18, 452–464 (2002)."/>
  <meta name="citation_reference" content="Emond, S. et al. Accessing unexplored regions of sequence space in directed enzyme evolution via insertion/deletion mutagenesis. Nature Communications 11, 3469 (2020)."/>
  <meta name="citation_reference" content="Sützl, L., Foley, G., Gillam, E. M. J., Bodén, M. & Haltrich, D. The GMC superfamily of oxidoreductases revisited: Analysis and evolution of fungal GMC oxidoreductases. Biotechnology for Biofuels 12, 118 (2019)."/>
  <meta name="citation_reference" content="Kirischian, N., McArthur, A. G., Jesuthasan, C., Krattenmacher, B. & Wilson, J. Y. Phylogenetic and functional analysis of the vertebrate cytochrome P450 2 family. Journal of Molecular Evolution 72, 56–71 (2011)."/>
  <meta name="citation_reference" content="Gao, H. et al. Function and maturation of the Fe-S center in dihydroxyacid dehydratase from Arabidopsis. The Journal of Biological Chemistry 293, 4422–4433 (2018)."/>
  <meta name="citation_reference" content="Baldi, P. & Brunak, S. Bioinformatics: The Machine Learning Approach (The MIT Press, 2001)."/>
  <meta name="citation_reference" content="Koller, D. & Friedman, N. Probabilistic Graphical Models (The MIT Press, 2009)."/>
  <meta name="citation_reference" content="Hanson-Smith, V., Kolaczkowski, B. & Thornton, J. W. Robustness of ancestral sequence reconstruction to phylogenetic uncertainty. Molecular Biology and Evolution 27, 1988–1999 (2010)."/>
  <meta name="citation_reference" content="Edwards, R. J. & Shields, D. C. GASP: Gapped Ancestral Sequence Prediction for proteins. BMC Bioinformatics 5, 1–10 (2004)."/>
  <meta name="citation_reference" content="Musil, M. et al. FireProtASR: A web server for fully automated ancestral sequence reconstruction. Briefings in Bioinformatics 22, bbaa337 (2021)."/>
  <meta name="citation_reference" content="Simmons, M. P. & Ochoterena, H. Gaps as characters in sequence-based phylogenetic analyses. Systematic Biology 49, 369–381 (2000)."/>
  <meta name="citation_reference" content="Pupko, T., Pe’er, I., Hasegawa, M., Graur, D. & Friedman, N. A branch-and- bound algorithm for the inference of ancestral amino-acid sequences when the replacement rate varies among sites: Application to the evolution of five gene families. Bioinformatics 18, 1116–1123 (2002)."/>
  <meta name="citation_reference" content="Koshi, J. M. & Goldstein, R. A. Probabilistic reconstruction of ancestral protein sequences. Journal of Molecular Evolution 42, 313–320 (1996)."/>
  <meta name="citation_reference" content="Ashkenazy, H. et al. FastML: A web server for probabilistic reconstruction of ancestral sequences. Nucleic Acids Research 40, W580–W584 (2012)."/>
  <meta name="citation_reference" content="Yang, Z. PAML 4: Phylogenetic analysis by maximum likelihood. Molecular Biology and Evolution 24, 1586–1591 (2007)."/>
  <meta name="citation_reference" content="Afriat-Jurnou, L., Jackson, C. J. & Tawfik, D. S. Reconstructing a missing link in the evolution of a recently diverged phosphotriesterase by active-site loop remodeling. Biochemistry 51, 6047–6055 (2012)."/>
  <meta name="citation_reference" content="Grasso, C. & Lee, C. Combining partial order alignment and progressive multiple sequence alignment increases alignment speed and scalability to very large alignment problems. Bioinformatics 20, 1546–1556 (2004)."/>
  <meta name="citation_reference" content="Löytynoja, A. & Goldman, N. An algorithm for progressive multiple alignment of sequences with insertions. Proceedings of the National Academy of Sciences of the United States of America 102, 10557–10562 (2005)."/>
  <meta name="citation_reference" content="Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: Improvements in performance and usability. Molecular Biology and Evolution 30, 772–780 (2013)."/>
  <meta name="citation_reference" content="Capella-Gutiérrez, S., Silla-Martínez, J. M. & Gabaldón, T. trimAl: A tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics 25, 1972–1973 (2009)."/>
  <meta name="citation_reference" content="Talavera, G. & Castresana, J. Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Systematic Biology 56, 564–577 (2007)."/>
  <meta name="citation_reference" content="Guindon, S. et al. New algorithms and methods to estimate maximumlikelihood phylogenies: Assessing the performance of PhyML 3.0. Systematic Biology 59, 307–321 (2010)."/>
  <meta name="citation_reference" content="Le, S. Q. & Gascuel, O. An improved general amino acid replacement matrix. Molecular Biology and Evolution 25, 1307–1320 (2008)."/>
  <meta name="citation_reference" content="Spadiut, O. et al. Improving thermostability and catalytic activity of pyranose 2-oxidase from Trametes multicolor by rational and semi-rational design. The FEBS Journal 276, 776–792 (2009)."/>
  <meta name="citation_reference" content="Rahman, M. M. et al. The crystal structure of a bacterial l-arabinonate dehydratase contains a [2Fe-2S] cluster. ACS Chemical Biology 12, 1919–1927 (2017)."/>
  <meta name="citation_reference" content="Sievers, F. & Higgins, D. G. Clustal Omega for making accurate alignments of many protein sequences. Protein Science: A Publication of the Protein Society 27, 135–145 (2018)."/>
  <meta name="citation_reference" content="Price, M. N., Dehal, P. S. & Arkin, A. P. FastTree: Computing large minimum evolution trees with profiles instead of a distance matrix. Molecular Biology and Evolution 26, 1641–1650 (2009)."/>
  <meta name="citation_reference" content="Jones, D. T., Taylor, W. R. & Thornton, J. M. The rapid generation of mutation data matrices from protein sequences. Computer Applications in the Biosciences 8, 275–282 (1992)."/>
  <meta name="citation_reference" content="Geertsma, E. R. & Dutzler, R. A versatile and efficient high-throughput cloning tool for structural biology. Biochemistry 50, 3272–3278 (2011)."/>
  <meta name="citation_reference" content="Studier, F. W. Protein production by auto-induction in high density shaking cultures. Protein Expression and Purification 41, 207–234 (2005)."/>
  <meta name="citation_reference" content="Stamatakis, A. RAxML version 8: A tool for phylogenetic analysis and postanalysis of large phylogenies. Bioinformatics 30, 1312–1313 (2014)."/>
  <meta name="citation_reference" content="Amemiya, C. T. et al. Analysis of the African coelacanth genome sheds light on tetrapod evolution. Nature 496, 311–316 (2013)."/>
  <meta name="citation_reference" content="von Wachenfeldt, C., Richardson, T. H., Cosme, J. & Johnson, E. F. Microsomal P450 2C3 is expressed as a soluble dimer in Escherichia coli following modifications of its N-terminus. Archives of Biochemistry and Biophysics 339, 107–114 (1997)."/>
  <meta name="citation_reference" content="Siller, M. et al. Oxidation of endogenous N-arachidonoylserotonin by human cytochrome P450 2U1. The Journal of Biological Chemistry 289, 10476–10487 (2014)."/>
  <meta name="citation_reference" content="Gaspar, P., Moura, G., Santos, M. A. S. & Oliveira, J. L. mRNA secondary structure optimization using a correlated stem-loop prediction. Nucleic Acids Research 41, e73 (2013)."/>
  <meta name="citation_reference" content="Johnston, W. A., Huang, W., De Voss, J. J., Hayes, M. A. & Gillam, E. M. J. Quantitative whole-cell cytochrome P450 measurement suitable for high-throughput application. Journal of Biomolecular Screening 13, 135–141 (2008)."/>
  <meta name="citation_reference" content="Huang, Y., Niu, B., Gao, Y., Fu, L. & Li, W. CD-HIT Suite: A web server for clustering and comparing biological sequences. Bioinformatics 26, 680–682 (2010)."/>
  <meta name="citation_reference" content="Felsenstein, J. Evolutionary trees from DNA sequences: A maximum likelihood approach. Journal of Molecular Evolution 17, 368–376 (1981)."/>
  <meta name="citation_reference" content="Löytynoja, A., Vilella, A. J. & Goldman, N. Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm. Bioinformatics 28, 1684–1691 (2012)."/>
  <meta name="citation_reference" content="Yang, Z. Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: Approximate methods. Journal of Molecular Evolution 39, 306–314 (1994)."/>
  <meta name="citation_reference" content="Felsenstein, J. Taking variation of evolutionary rates between sites into account in inferring phylogenies. Journal of Molecular Evolution 53, 447–455 (2001)."/>
  <meta name="citation_reference" content="Dayhoff, M. O., Schwartz, R. M. & Orcutt, B. A model of evolutionary change in proteins. In Atlas of Protein Sequence and Structure, 345–352 (National Biomedical Research Foundation, 1978)."/>
  <meta name="citation_reference" content="Whelan, S. & Goldman, N. A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach. Molecular Biology and Evolution 18, 691–699 (2001)."/>
  <meta name="citation_reference" content="Dechter, R. Bucket elimination: A unifying framework for reasoning. Artificial Intelligence 113, 41–85 (1999)."/>
  <meta name="citation_reference" content="Hart, P. E., Nilsson, N. J. & Raphael, B. A formal basis for the heuristic determination of minimum cost paths. IEEE Transactions on Systems Science and Cybernetics 4, 100–107 (1968)."/>
  <meta name="citation_reference" content="Benner, S. A., Cohen, M. A. & Gonnet, G. H. Empirical and structural models for insertions and deletions in the divergent evolution of proteins. Journal of Molecular Biology 229, 1065–1082 (1993)."/>
  <meta name="twitter:title" content="Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Ancestral sequence reconstruction is a technique that is gaining widespread use in molecular evolution studies and protein engineering. Accurate reconstruction requires the ability to handle appropriately large numbers of sequences, as well as insertion and deletion (“indel”) events, but available approaches exhibit limitations. To address these limitations, we developed Graphical Representation of Ancestral Sequence Predictions (GRASP), which efficiently implements maximum likelihood methods to enable the inference of ancestors of families with more than 10,000 members. GRASP implements partial order graphs (POGs) to represent and infer insertion and deletion events across ancestors, enabling the identification of building blocks for protein engineering.

To validate the capacity to engineer novel proteins from realistic data, we predicted ancestor sequences across three distinct enzyme families: glucose-methanol-choline (GMC) oxidoreductases, cytochromes P450, and dihydroxy/sugar acid dehydratases (DHAD). All tested ancestors demonstrated enzymatic activity. Our study demonstrates the ability of GRASP (1) to support large data sets over 10,000 sequences and (2) to employ insertions and deletions to identify building blocks for engineering biologically active ancestors, by exploring variation over evolutionary time.

Author summary Massive sequencing projects expose the extent of natural, genetic diversity. Here, we describe a method with capacity to perform ancestor sequence reconstruction from data sets in excess of 10,000 sequences, poised to recover ancestral diversity, including the evolutionary events that determine present-time biological function and structure.

We introduce a novel strategy for suggesting “indel variants” that are distinct from, but can be explored alongside, substitution variants for creating ancestral libraries. We demonstrate how indels can be used as building blocks to form “hybrid ancestors”; based on this strategy, we synthesise ancestor variants, with varying enzymatic activities, for wide-ranging applications in the biotechnology sector.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.30.891457v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Ancestral sequence reconstruction is a technique that is gaining widespread use in molecular evolution studies and protein engineering. Accurate reconstruction requires the ability to handle appropriately large numbers of sequences, as well as insertion and deletion (“indel”) events, but available approaches exhibit limitations. To address these limitations, we developed Graphical Representation of Ancestral Sequence Predictions (GRASP), which efficiently implements maximum likelihood methods to enable the inference of ancestors of families with more than 10,000 members. GRASP implements partial order graphs (POGs) to represent and infer insertion and deletion events across ancestors, enabling the identification of building blocks for protein engineering.

To validate the capacity to engineer novel proteins from realistic data, we predicted ancestor sequences across three distinct enzyme families: glucose-methanol-choline (GMC) oxidoreductases, cytochromes P450, and dihydroxy/sugar acid dehydratases (DHAD). All tested ancestors demonstrated enzymatic activity. Our study demonstrates the ability of GRASP (1) to support large data sets over 10,000 sequences and (2) to employ insertions and deletions to identify building blocks for engineering biologically active ancestors, by exploring variation over evolutionary time.

Author summary Massive sequencing projects expose the extent of natural, genetic diversity. Here, we describe a method with capacity to perform ancestor sequence reconstruction from data sets in excess of 10,000 sequences, poised to recover ancestral diversity, including the evolutionary events that determine present-time biological function and structure.

We introduce a novel strategy for suggesting “indel variants” that are distinct from, but can be explored alongside, substitution variants for creating ancestral libraries. We demonstrate how indels can be used as building blocks to form “hybrid ancestors”; based on this strategy, we synthesise ancestor variants, with varying enzymatic activities, for wide-ranging applications in the biotechnology sector.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-01-07"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.30.891457v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2340580"/>
  <title>
   Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP) | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_pCktql-ly43KU6Xo7Ijr2fXYIyPrsIIoKRYtlgvo9k"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2340580","apath":"\/biorxiv\/early\/2022\/01\/07\/2019.12.30.891457.atom","pisa":"biorxiv;2019.12.30.891457v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.30.891457v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.30.891457v3","title":"Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)","identifier":"node\/2340580"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JMzrwga+cjxOVMsTA2nMHPaWRhXXqF5Vw1\\\\\\\/LqKSiJCe9Ds9lBYGLI7YDGK7xoi0L5YsyK8Rdh4zIzpuPObu+I+KuFSkQGQmbzTaX+BbpWBQXx1CIqDEsvV4Nn5wVOusBw2BWkzaUd83wBw7VAviRRVOiHH+ayJTKl2a8IKJtMoyROhtd1tzkl9ojrbBJVAWCx4bEdCj7pwwH7s77cwwQecP4SsLUfhUXm4CC\\\\\\\/wrTAB6CrEPZ5vS3Yfj8mYcgcethO9XyQuIY3we5cw+U6gZbAbeN\\\\\\\/L7L39NXe0LWfyJBRxQmwriyCVse2ghOxCxgabR4yRfSqYwtM8bSbZiiz61CperXBoquOBrI6T7a52noCXUg03ZVJbOrNyFtg6esgApzWQ4CbXJ2IjeDGdOz+ga4Bjxz4xFxwQRXfjEsWP9GaSdQgal+5t+wc+rsCktX8eOVkySAbDFRdd0QufERWQ9qc7+oo8rlxx9xZEPxmSaUYUjIPmPMMwEgFnOnnH+FsYMUNqeYpKNa3G9+PWEJAkJAC5EpgAIpNIF3UyBjZLHYPF1suuXM69loKBasvGBc31yo4elDWskwEzhSq+L2WDuRV9QoKjvAGr5I8TdFB4yY6H5wZAOT1yPVCafMBE0rCQ24146FHTJ8xOHzqg2bnKu2NhhBs+hJ5Ern1AP\\\\\\\/ILGTdwfbA4Jr5moDSXe5SQgpIkUgVIM8V6dJWkYwNQJvHaKzRNtAxOZM9B+wthErjEqnp\\\\\\\/RKx3dcvCxJzm52ubWiFnVpXHTLJDvLBAhOPDiiTAm\\\\\\\/emToBedUt3H3WwyvsujaSTiIbzllLPjvrqROIEl4gPCoakPXOlsO60CifC+wuW8\\\\\\\/0pirang1w0S382ERFifIi4v+v+Dq7LzREbfYNWwNwzavBjhx1J4YwhNviKg72NEoen7Uapjwt64CizSd9ZrKdpsAfnBXHUpw3vWOAWxUnbK9TDbXtTCRYOn5beUW0mLA1H5d34QC7vfSW0fPvbV65qs2PCz303bu1HZ0v7c+QlHlnV3eBunNgo63vsel8J9Gx1BBwNiiCgu5maHmmySBvPrsqBzI99bWTsdNKCkNmBDyl7Tht6JKvMkapGK99YD0LpdKNSWwqmEG4cAbIlWZJdnyAmy2LoPiY\\\\\\\/g5guTlPgOBCap8m+7asMZvy3S5blRhpYTnXVyH7ZWbxEkaHvY4lt0JDqCd8e7Yu7rHMUtLi48Wt7Gz76+989pIZqcf8cmBMGMNKZ8MuXRopz370xkqYeBp3zR+HtymAXPbDPcHOJ9hWrU\\\\\\\/u979xGzubq4Qzi3PUgY8Au\\\\\\\/6L8YJPcG7m+Mrw7iJZwlBe4AkYJ8IygrSdgSbD5cq8ZcW4nlU8UKKFQp\\\\\\\/5Q9A4OK4n\\\\\\\/wbKPNfk+Z0XTA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022CJMFCar7sWduYsttjinHJA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b539a36b5650527e0045834f7c5e0cce1697bcc5a4c32ccdff0b627f4915659f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.30.891457v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2340580 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.30.891457v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1788746411">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1788746411" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1068637764" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-7QLxkl_l5v29A_lLf06IWcKPb7y4htHBkx9TRezBLuw"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2340580" data-node-nid="2340580" id="node-2340580--21442988191" data-pisa="biorxiv;2019.12.30.891457v3" data-pisa-master="biorxiv;2019.12.30.891457" data-apath="/biorxiv/early/2022/01/07/2019.12.30.891457.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-0487-2629" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Gabriel
                      </span>
                      <span class="nlm-surname">
                       Foley
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Ariane
                      </span>
                      <span class="nlm-surname">
                       Mora
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Connie M.
                      </span>
                      <span class="nlm-surname">
                       Ross
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Scott
                      </span>
                      <span class="nlm-surname">
                       Bottoms
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Leander
                      </span>
                      <span class="nlm-surname">
                       Sützl
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Marnie L.
                      </span>
                      <span class="nlm-surname">
                       Lamprecht
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Julian
                      </span>
                      <span class="nlm-surname">
                       Zaugg
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="7">
                      <span class="nlm-given-names">
                       Alexandra
                      </span>
                      <span class="nlm-surname">
                       Essebier
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="8">
                      <span class="nlm-given-names">
                       Brad
                      </span>
                      <span class="nlm-surname">
                       Balderson
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="9">
                      <span class="nlm-given-names">
                       Rhys
                      </span>
                      <span class="nlm-surname">
                       Newell
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="10">
                      <span class="nlm-given-names">
                       Raine E. S.
                      </span>
                      <span class="nlm-surname">
                       Thomson
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="11">
                      <span class="nlm-given-names">
                       Bostjan
                      </span>
                      <span class="nlm-surname">
                       Kobe
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12">
                      <a href="http://orcid.org/0000-0002-5685-4828" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ross T.
                      </span>
                      <span class="nlm-surname">
                       Barnard
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="13">
                      <span class="nlm-given-names">
                       Luke
                      </span>
                      <span class="nlm-surname">
                       Guddat
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="14">
                      <span class="nlm-given-names">
                       Gerhard
                      </span>
                      <span class="nlm-surname">
                       Schenk
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="15">
                      <span class="nlm-given-names">
                       Jörg
                      </span>
                      <span class="nlm-surname">
                       Carsten
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="16">
                      <span class="nlm-given-names">
                       Yosephine
                      </span>
                      <span class="nlm-surname">
                       Gumulya
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="17">
                      <span class="nlm-given-names">
                       Burkhard
                      </span>
                      <span class="nlm-surname">
                       Rost
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="18">
                      <span class="nlm-given-names">
                       Dietmar
                      </span>
                      <span class="nlm-surname">
                       Haltrich
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="19">
                      <span class="nlm-given-names">
                       Volker
                      </span>
                      <span class="nlm-surname">
                       Sieber
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="20">
                      <span class="nlm-given-names">
                       Elizabeth M. J.
                      </span>
                      <span class="nlm-surname">
                       Gillam
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="21">
                      <a href="http://orcid.org/0000-0003-3548-268X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Mikael
                      </span>
                      <span class="nlm-surname">
                       Bodén
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2019.12.30.891457
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2340580--21442988191" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Gabriel Foley
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Gabriel%2BFoley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Foley%20G&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AGabriel%2BFoley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-0487-2629" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Gabriel Foley
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Ariane Mora
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ariane%2BMora%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mora%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAriane%2BMora%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Connie M. Ross
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Connie%2BM.%2BRoss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ross%20CM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AConnie%2BM.%2BRoss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Scott Bottoms
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Campus Straubing for Biotechnology and Sustainability, Technische Universität München
                       </span>
                       , Straubing,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Scott%2BBottoms%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bottoms%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AScott%2BBottoms%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Leander Sützl
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Institut für Lebensmitteltechnologie, Universität für Bodenkultur Wien
                       </span>
                       , Vienna,
                       <span class="nlm-country">
                        Austria
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Leander%2BS%C3%BCtzl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=S%C3%BCtzl%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ALeander%2BS%25C3%25BCtzl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Marnie L. Lamprecht
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Marnie%2BL.%2BLamprecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lamprecht%20ML&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMarnie%2BL.%2BLamprecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Julian Zaugg
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Julian%2BZaugg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zaugg%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJulian%2BZaugg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Alexandra Essebier
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alexandra%2BEssebier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Essebier%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAlexandra%2BEssebier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Brad Balderson
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Brad%2BBalderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Balderson%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ABrad%2BBalderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Rhys Newell
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Rhys%2BNewell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Newell%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ARhys%2BNewell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">
                     Raine E. S. Thomson
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Raine%2BE.%2BS.%2BThomson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Thomson%20RE&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ARaine%2BE.%2BS.%2BThomson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-11">
                    <div class="author-tooltip-name">
                     Bostjan Kobe
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Institute for Molecular Bioscience and Australian Infectious Diseases Research Centre, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Bostjan%2BKobe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kobe%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ABostjan%2BKobe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-12">
                    <div class="author-tooltip-name">
                     Ross T. Barnard
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ross%2BT.%2BBarnard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Barnard%20RT&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARoss%2BT.%2BBarnard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-5685-4828" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ross T. Barnard
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-13">
                    <div class="author-tooltip-name">
                     Luke Guddat
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Luke%2BGuddat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Guddat%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ALuke%2BGuddat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-14">
                    <div class="author-tooltip-name">
                     Gerhard Schenk
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Sustainable Minerals Institute, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Gerhard%2BSchenk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Schenk%20G&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AGerhard%2BSchenk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-15">
                    <div class="author-tooltip-name">
                     Jörg Carsten
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Zentralinstitut für Katalyseforschung, Technische Universität München
                       </span>
                       , Munich,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=J%C3%B6rg%2BCarsten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Carsten%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJ%25C3%25B6rg%2BCarsten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-16">
                    <div class="author-tooltip-name">
                     Yosephine Gumulya
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Yosephine%2BGumulya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gumulya%20Y&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AYosephine%2BGumulya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-17">
                    <div class="author-tooltip-name">
                     Burkhard Rost
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Fakultät für Informatik, Technische Universität München
                       </span>
                       , Munich,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Burkhard%2BRost%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Rost%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ABurkhard%2BRost%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-18">
                    <div class="author-tooltip-name">
                     Dietmar Haltrich
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Institut für Lebensmitteltechnologie, Universität für Bodenkultur Wien
                       </span>
                       , Vienna,
                       <span class="nlm-country">
                        Austria
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Dietmar%2BHaltrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Haltrich%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADietmar%2BHaltrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-19">
                    <div class="author-tooltip-name">
                     Volker Sieber
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Campus Straubing for Biotechnology and Sustainability, Technische Universität München
                       </span>
                       , Straubing,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Zentralinstitut für Katalyseforschung, Technische Universität München
                       </span>
                       , Munich,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Volker%2BSieber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sieber%20V&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AVolker%2BSieber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-20">
                    <div class="author-tooltip-name">
                     Elizabeth M. J. Gillam
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Elizabeth%2BM.%2BJ.%2BGillam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gillam%20EM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AElizabeth%2BM.%2BJ.%2BGillam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:m.boden@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">
                        m.boden@uq.edu.au
                       </a>
                       <a href="mailto:e.gillam@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">
                        e.gillam@uq.edu.au
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-21">
                    <div class="author-tooltip-name">
                     Mikael Bodén
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Chemistry and Molecular Biosciences, The University of Queensland
                       </span>
                       ,
                       <span class="nlm-country">
                        Australia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mikael%2BBod%C3%A9n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bod%C3%A9n%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMikael%2BBod%25C3%25A9n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-3548-268X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Mikael Bodén
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:m.boden@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">
                        m.boden@uq.edu.au
                       </a>
                       <a href="mailto:e.gillam@uq.edu.au" class="" data-icon-position="" data-hide-link-title="0">
                        e.gillam@uq.edu.au
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2019.12.30.891457v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2340580" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2340580/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2019.12.30.891457v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2340580" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2340580/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2019.12.30.891457v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2340580" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2340580/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2019.12.30.891457v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2340580" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2340580/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2019.12.30.891457v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2340580" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2340580/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2019.12.30.891457v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2340580" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2340580/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2019.12.30.891457v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2340580" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2340580/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             Ancestral sequence reconstruction is a technique that is gaining widespread use in molecular evolution studies and protein engineering. Accurate reconstruction requires the ability to handle appropriately large numbers of sequences, as well as insertion and deletion (“indel”) events, but available approaches exhibit limitations. To address these limitations, we developed Graphical Representation of Ancestral Sequence Predictions (GRASP), which efficiently implements maximum likelihood methods to enable the inference of ancestors of families with more than 10,000 members. GRASP implements partial order graphs (POGs) to represent and infer insertion and deletion events across ancestors, enabling the identification of building blocks for protein engineering.
                            </p>
                            <p id="p-4">
                             To validate the capacity to engineer novel proteins from realistic data, we predicted ancestor sequences across three distinct enzyme families: glucose-methanol-choline (GMC) oxidoreductases, cytochromes P450, and dihydroxy/sugar acid dehydratases (DHAD). All tested ancestors demonstrated enzymatic activity. Our study demonstrates the ability of GRASP (1) to support large data sets over 10,000 sequences and (2) to employ insertions and deletions to identify building blocks for engineering biologically active ancestors, by exploring variation over evolutionary time.
                            </p>
                            <div id="sec-1" class="subsection">
                             <p id="p-5">
                              <strong>
                               Author summary
                              </strong>
                              Massive sequencing projects expose the extent of natural, genetic diversity. Here, we describe a method with capacity to perform ancestor sequence reconstruction from data sets in excess of 10,000 sequences, poised to recover ancestral diversity, including the evolutionary events that determine present-time biological function and structure.
                             </p>
                             <p id="p-6">
                              We introduce a novel strategy for suggesting “indel variants” that are distinct from, but can be explored alongside, substitution variants for creating ancestral libraries. We demonstrate how indels can be used as building blocks to form “hybrid ancestors”; based on this strategy, we synthesise ancestor variants, with varying enzymatic activities, for wide-ranging applications in the biotechnology sector.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-10">
                             Sequencing technology is driving the identification of the extant (modern) portion of the universe of biological sequences
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              ,
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              ,
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             . With this increased coverage of natural diversity we are now better placed than ever before to leverage ancestral sequence reconstruction (ASR) to recover the ancestral portion and trace the evolutionary events that have determined biological function and structure
                             <sup>
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             . This is especially useful for protein engineering; the evolutionary record reveals essential cues for the discovery and engineering of new enzymes. The resurrection of ancestral enzymes can be used to generate enzymes with novel properties that can be exploited in biocatalysis
                             <sup>
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ,
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ,
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ,
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-11">
                             The ability to perform ASR on large-scale data has been limited by the available methodology and accompanying technology. A recent review highlighted 12 studies from the past decade that each sought to evaluate sources of ambiguity in ancestral inferences
                             <sup>
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             . Data set sizes within these studies ranged from 21 to 456 sequences, with an average of 168 sequences. Current methods for performing ASR have reached practical upper limits on data set size which constrains our ability to adequately represent and accurately analyse enzymes that have been evolving for billions of years. We have therefore developed the tool Graphical Representation of Ancestral Sequence Predictions (GRASP) to take advantage of the rapidly expanding databases of known protein sequences and the information from biological diversity that can be mined from large protein families.
                            </p>
                            <p id="p-12">
                             However, the challenge does not end with scaling-up processing capacity. Remote homologs that form part of large protein families are likely to have resulted from numerous evolutionary events, the order of which is often difficult to infer, and confounds current phylogenetic analysis techniques; the presence of insertion and deletion (indel) events has been shown to decrease alignment accuracy unless appropriately handled
                             <sup>
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                             </sup>
                             . Lee et al.
                             <sup>
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                             </sup>
                             demonstrated how a partial order graph (POG) can be used to represent and support the alignment of widely different sequences. The risk of aligning sequence fragments with different evolutionary origins motivated us to use the POG data structure to enable the separation of distinct sources of sequence diversity at evolutionary branch points.
                            </p>
                            <p id="p-13">
                             The premise of our study was that a substantial increase in the scope of ASR will provide (a) a rich resource for evolutionary studies, and (b) valuable guidance for protein engineering, given the demonstrated usefulness of ancestral enzymes as robust templates for directed evolution
                             <sup>
                              <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             . Accordingly, our method was designed with a view to identifying substitutions, insertions, and deletions that may be combined to form sequence configurations inspired by, but not necessarily present in, either extant or inferred ancestral sequences. Recent advances in protein engineering have shown this to be an effective way of surveying sequence space and here we applied similar approaches explicitly to ancestors
                             <sup>
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             . In particular, we sought to evaluate if inferred sequences and positions of indels can be used as building blocks to engineer biologically active ancestors. This, in turn, demands that we first identify a practical approach for their inference, in parallel with substitutions.
                            </p>
                            <p id="p-14">
                             In brief, this paper demonstrates the capacity of an approach based on POGs and maximum likelihood inference to perform ASR of large protein families and assist in the design of novel biocatalysts. Specifically, we explored the impact of quantity, diversity, and taxonomic context of input sequences on predicted sequences as well as resurrected structures and functions. Critically, we evaluated the prospect of re-purposing indel events inferred from the evolutionary record to compose and resurrect “hybrid” ancestors.
                            </p>
                            <p id="p-15">
                             Experimentally, we resurrected ancestral proteins from three families (exemplifying various degrees of sequence number, functional diversity, and sequence similarity) and evaluated these in terms of their structure and function. First, the glucose-methanol-choline (GMC) oxidoreductases represent a super-family of enzymes with varying biological functions and industrial applications; we focused on the enzymes glucose dehydrogenase (GDH, EC 1.1.5.9) and glucose oxidase (GOx, EC 1.1.2.4)
                             <sup>
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                             </sup>
                             . Second, members of cytochrome P450 subfamily 2 (CYP2) play a key role in drug and xenobiotic metabolism in metazoans
                             <sup>
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                             </sup>
                             . Here we concentrated on the CYP2U subfamily and two closely-related subfamilies, CYP2R, and CYP2D. Thirdly, the IlvD/ED dehydratase family includes dihydroxy-acid dehydratase (EC 4.2.1.9) and several sugar acid dehydratases all containing iron-sulfur-clusters across a broad taxonomic scope
                             <sup>
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                             </sup>
                             . We refer to this family as DHAD. It is present in bacteria, archaea, fungi, algae, and in some plants. We also evaluated the large-scale inference of, but did not resurrect, the ketol-acid reductoiso-merase (KARI) family, which includes enzymes in the branched-chain amino acid biosynthetic pathway (similar to DHAD) present in bacteria, fungi, and plants. We focused on KARI class I for a large-scale inference, and class II for a comparison between existing tools, having previously successfully resurrected ancestors of class II enzymes
                             <sup>
                              <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-4" class="subsection">
                             <h3>
                              GRASP infers partially ordered ancestor graphs, representing substitutions, insertions, and deletions
                             </h3>
                             <p id="p-16">
                              GRASP implements maximum likelihood for inference of ancestor states within trees, which in turn are graphical models
                              <sup>
                               <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                                19
                               </a>
                              </sup>
                              . It uses efficient algorithms developed for Bayesian networks
                              <sup>
                               <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                              </sup>
                              to process volumes of data that are unprecedented in the field of ASR.
                             </p>
                             <p id="p-17">
                              Unlike other reconstruction methods, GRASP uses POGs to represent sequence content, including insertions and deletions, to allow for 1) inference and visualisation of indel events across ancestors and 2) representation of alternative ancestors as alternative paths through an ancestor POG.
                             </p>
                             <p>
                              GRASP infers ancestor POGs from an input POG that represents a set of aligned homologous sequences and an input phylogenetic tree describing their evolutionary relationships. It does this in three stages that are designed to deconvolute sources of sequence variation.
                             </p>
                             <ol class="list-ord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-19">
                                A history of indel events is inferred by either maximum likelihood or maximum parsimony and mapped onto the tree to determine which positions contain actual sequence content (
                                <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                 Fig. 1a
                                </a>
                                ).
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-20">
                                For each ancestral position, the most probable character (to explain those observed at the leaves) is assigned to each phylogenetic branch point when performing a joint reconstruction (
                                <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                 Fig. 1b
                                </a>
                                ). Alternatively, for each position at a nominated branch point, the probability distribution over all possible characters is inferred when performing a marginal reconstruction (
                                <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                 Fig. 1c
                                </a>
                                ).
                               </p>
                              </li>
                              <li id="list-item-3">
                               <p id="p-21">
                                Edges are drawn to represent all inferred combinations of indels to form an ancestor POG with nodes that can form a valid sequence with inferred content; a preferred path through the POG is then inferred, nominating a single, best supported sequence.
                               </p>
                              </li>
                             </ol>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F1.large.jpg?width=800&height=600&carousel=1" title="Phylogenetic tree showing a reconstruction of fungal GDH and GOx sequences decorated with illustrations of key concepts used within GRASP. a, Two extant POGs (j indicates extant sequence number) mapped to an ancestral POG. Each extant POG has a single path through strictly ordered sequence positions (i indicates position). Ancestral states are influenced by all sequences, which explains why i = 608 is inferred as glycine, despite glycine not appearing in either sequence j = 359 or 360. b, Three ancestor POGs showing most probable assignments from a joint reconstruction at positions i ∈ {315,…, 327} for nodes N7, N8, and N9. GRASP supports the simultaneous viewing of multiple ancestors from a joint reconstruction, enabling a direct comparison at different time points. c, A single ancestor POG showing inferred marginal distributions at positions i ∈ {243,…, 254} for node N320. For marginal reconstructions, nodes are coloured according to their posterior probabilities and can be queried to view histograms of these underlying distributions, as is done for position i = 244. The marginal reconstruction from (c) was used to reconstruct the inferred ancestor (N320) as well as an alternative ancestor in which a single amino acid (N320_Y244E) was altered based on posterior probabilities from the marginal distribution that resulted in increased thermal stability (Supplementary Table 1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-120415564" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Phylogenetic tree showing a reconstruction of fungal GDH and GOx sequences decorated with illustrations of key concepts used within GRASP. <strong>a</strong>, Two extant POGs (<em>j</em> indicates extant sequence number) mapped to an ancestral POG. Each extant POG has a single path through strictly ordered sequence positions (<em>i</em> indicates position). Ancestral states are influenced by all sequences, which explains why <em>i</em> = 608 is inferred as glycine, despite glycine not appearing in either sequence <em>j</em> = 359 or 360. <strong>b</strong>, Three ancestor POGs showing most probable assignments from a joint reconstruction at positions <em>i</em> ∈ {315,…, 327} for nodes N7, N8, and N9. GRASP supports the simultaneous viewing of multiple ancestors from a joint reconstruction, enabling a direct comparison at different time points. <strong>c</strong>, A single ancestor POG showing inferred marginal distributions at positions <em>i</em> ∈ {243,…, 254} for node N320. For marginal reconstructions, nodes are coloured according to their posterior probabilities and can be queried to view histograms of these underlying distributions, as is done for position <em>i</em> = 244. The marginal reconstruction from (c) was used to reconstruct the inferred ancestor (N320) as well as an alternative ancestor in which a single amino acid (N320_Y244E) was altered based on posterior probabilities from the marginal distribution that resulted in increased thermal stability (Supplementary Table 1).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F1.medium.gif" width="440" height="290"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F1.medium.gif" width="440" height="290"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <p id="p-22" class="first-child">
                                Phylogenetic tree showing a reconstruction of fungal GDH and GOx sequences decorated with illustrations of key concepts used within GRASP.
                                <strong>
                                 a
                                </strong>
                                , Two extant POGs (j indicates extant sequence number) mapped to an ancestral POG. Each extant POG has a single path through strictly ordered sequence positions (i indicates position). Ancestral states are influenced by all sequences, which explains why i = 608 is inferred as glycine, despite glycine not appearing in either sequence j = 359 or 360.
                                <strong>
                                 b
                                </strong>
                                , Three ancestor POGs showing most probable assignments from a joint reconstruction at positions i ∈ {315,…, 327} for nodes N7, N8, and N9. GRASP supports the simultaneous viewing of multiple ancestors from a joint reconstruction, enabling a direct comparison at different time points.
                                <strong>
                                 c
                                </strong>
                                , A single ancestor POG showing inferred marginal distributions at positions i ∈ {243,…, 254} for node N320. For marginal reconstructions, nodes are coloured according to their posterior probabilities and can be queried to view histograms of these underlying distributions, as is done for position i = 244. The marginal reconstruction from (c) was used to reconstruct the inferred ancestor (N320) as well as an alternative ancestor in which a single amino acid (N320_Y244E) was altered based on posterior probabilities from the marginal distribution that resulted in increased thermal stability (Supplementary Table 1).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-23">
                              For GDH and GOx, we used GRASP to identify potential substitution variants through analysis of inferred distributions via marginal reconstruction. Marginal reconstruction is frequently performed to account for uncertainties in reconstructed sequences, suggest alternative or variant ancestors, and explore properties such as thermal stability or substrate preference in inferred variants (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1c
                              </a>
                              , Supplementary Table 1).
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              GRASP implements novel options for inferring indel histories
                             </h3>
                             <p id="p-24">
                              Insertions and deletions at branch points introduce a major source of variation in sequence. Not all reconstruction tools model indels, but several consider each sequence position independently and infer “gaps” in ancestors through parsimony (PAML/LAZARUS
                              <sup>
                               <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                                21
                               </a>
                              </sup>
                              ) or by probability based on the presence of gaps in descendants (GASP
                              <sup>
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                              </sup>
                              , FireProt
                              <sup>
                               ASR
                              </sup>
                              <sup>
                               <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              ). Extended gaps are accounted for by simple indel coding (SIC) that regards deletions in aligned sequences as equivalent if they start and end at the same positions
                              <sup>
                               <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                24
                               </a>
                              </sup>
                              . This in turn means that gaps have a binary state; they are either present or absent in a sequence. When gaps are considered relative to another, a third option “inapplicable” applies to a gap that is “contained” by another, thereby allowing preferences to be followed when multiple gap states are possible. It is therefore possible to encode a set of deletions by SIC and infer the optimal presence of each deletion independently at evolutionary branch points by parsimony or maximum likelihood (FastML
                              <sup>
                               <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                                25
                               </a>
                              </sup>
                              ).
                             </p>
                             <p id="p-25">
                              GRASP implements three methods for encoding patterns of insertions and deletions from the set of input sequences: position-specific (PS) encoding, SIC, and a novel method that considers indels as edges in a POG, which we refer to as bi-directional edge (BE) encoding. Similar to SIC, BE encodes indels by their start and end positions. However, BE does not treat indels with either shared start or shared end positions independently. Indeed, BE allows the inference to differentiate between deletions that start at the same position but end at different positions (forward direction) as well as to differentiate deletions that end at the same position but start at different positions (backward direction). Every indel event can then be designated as absent from the ancestor or present, and deemed to be optimal in one, both, or neither direction.
                             </p>
                             <p id="p-26">
                              GRASP implements both parsimony and maximum likelihood to infer indel histories based on any of the three encodings, giving six indel inference options (PS-P, PS-ML, SIC-P, SIC-ML, BE-P, and BE-ML). For all options, GRASP represents ancestors as POGs, which can be used to identify the most supported path, which in turn is a contiguous sequence.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Bi-directional edge methods return fewer unique indel events
                             </h3>
                             <p id="p-27">
                              It is not obvious to what extent predicted indel variants differ at ancestral branch points in a given evolutionary tree, based on the choice of encoding and inference method. Moreover, given the unavailability of historical data, we are limited to evaluating the accuracy of predicted indel events on simulated data. We used INDELible to generate indel histories at four indel rates (rate of indel events relative to substitution events) and for four taxon sizes (number of extant sequences within each data set), which resulted in phylogenetic trees and sequences with identified gaps at all branch points and leaves. In each run, these gapped sequences form an alignment where all positions are guaranteed to be homologous regardless of the number of indel events (Supplementary Fig. 1). The combination of tree and alignment from INDELible allows indel events (defined by start and end position) to be tracked and retrieved from branch points to measure the accuracy of the reconstruction at different rates and taxon sizes. We defined accuracy as the percentage of correctly recovered indel events (by position and branch point). The simulation of each setting was repeated to render different data sets to determine the variance explained by random factors.
                             </p>
                             <p id="p-28">
                              All six indel inference options recovered the correct length of ancestor sequence at the root, for each taxon size and indel rate, across every replicate. Accuracy was consistent and variance decreased as taxon size increased for a given indel rate, and accuracy decreased as the indel rate increased (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2a
                              </a>
                              ). The number of indels that were consistently identified or consistently missed by all methods stayed constant as taxon size increased for a given indel rate. As the indel rate increased, the number of consistently identified and consistently missed indels decreased and increased, respectively (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2b
                              </a>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F2.large.jpg?width=800&height=600&carousel=1" title="Results of indel evaluation of each of GRASP’s indel methods across 1) simulated data where the correct alignment and correct tree are supplied and 2) simulated data where a realigned alignment and correct tree are supplied across four indel rates (0.001, 0.005, 0.01, 0.03) and four taxon sizes (100, 250, 500, 750). a, Number of correct indels identified by each method for each taxon size for each indel rate (N = 5). b, Number of indels uniformly identified by all methods or uniformly missed by all methods for each taxon size for each indel rate (N = 5). c, Number of indels uniquely identified by each indel method at four taxon sizes at indel rate 0.03, organised by indel type and size (N = 5)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-120415564" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Results of indel evaluation of each of GRASP’s indel methods across 1) simulated data where the correct alignment and correct tree are supplied and 2) simulated data where a realigned alignment and correct tree are supplied across four indel rates (0.001, 0.005, 0.01, 0.03) and four taxon sizes (100, 250, 500, 750). <strong>a</strong>, Number of correct indels identified by each method for each taxon size for each indel rate (<em>N</em> = 5). <strong>b</strong>, Number of indels uniformly identified by all methods or uniformly missed by all methods for each taxon size for each indel rate (<em>N</em> = 5). <strong>c</strong>, Number of indels uniquely identified by each indel method at four taxon sizes at indel rate 0.03, organised by indel type and size (<em>N</em> = 5).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F2.medium.gif" width="309" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F2.medium.gif" width="309" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <p id="p-29" class="first-child">
                                Results of indel evaluation of each of GRASP’s indel methods across 1) simulated data where the correct alignment and correct tree are supplied and 2) simulated data where a realigned alignment and correct tree are supplied across four indel rates (0.001, 0.005, 0.01, 0.03) and four taxon sizes (100, 250, 500, 750).
                                <strong>
                                 a
                                </strong>
                                , Number of correct indels identified by each method for each taxon size for each indel rate (N = 5).
                                <strong>
                                 b
                                </strong>
                                , Number of indels uniformly identified by all methods or uniformly missed by all methods for each taxon size for each indel rate (N = 5).
                                <strong>
                                 c
                                </strong>
                                , Number of indels uniquely identified by each indel method at four taxon sizes at indel rate 0.03, organised by indel type and size (N = 5).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              As all methods performed comparatively and to simulate more challenging and realistic scenarios, we realigned the sequences generated by INDELible with MAFFT; we then used this realigned data with the original trees to generate additional sets of ancestors. In this scenario, we could no longer directly measure accuracy because the re-aligned sequences probably implied indels with different start and end positions. We therefore evaluated the effect of indel rate and taxon size on the sequence length at root and qualitative differences in the sets of indels identified by each indel inference method. This in turn would enable us to understand which indel inference method offers the broadest support in the pursuit of indel variants for engineering applications.
                             </p>
                             <p id="p-31">
                              Under the realigned conditions at the highest indel rate, SIC returned root ancestors with longer sequence length regardless of the inference method or taxon size (Supplementary Fig. 2a). There was substantial overlap in indels predicted by each method. That said, BE methods identified fewer unique indels than PS or SIC methods (see Supplementary Fig. 2b for an example replicate from the largest taxa size with the highest indel rate). The number of unique events predicted by all methods increased with both indel rate and taxon size, but BE methods tended to identify fewer. Moreover, BE was more often in agreement with at least one other method, compared to either SIC or PS. PS methods and SIC-P returned higher numbers of unique one-residue indels and SIC methods returned higher numbers of unique indels of length greater than three (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2c
                              </a>
                              , Supplementary Fig. 3). Unlike BE, SIC focuses on the representation of gaps rather than the edges that make up a sequence. SIC assumes that the starting point is a linear sequence, on which gaps are superimposed. SIC does not inherently resolve conflicts between multiple, partially overlapping indels, which are independently predicted at each branch point. POGs do not explicitly represent gaps, allowing predicted, conflicted indels to be separated. The conversion to a linear sequence involves choices that benefit from considering the evolutionary context and can be ascertained across multiple branch points in the phylogenetic tree.
                             </p>
                             <p id="p-32">
                              FastML implements SIC and offers both maximum parsimony and maximum likelihood inference, the latter with a gene loss/gain model. We used parsimony inference to ensure that the implementations were comparable, noting that FastML only presents a single parsimonious indel history even when several are optimal, while GRASP presents all. While FastML infers sequences and GRASP infers POGs, we tested at each branch point if the POG contained a valid path for the sequence predicted by FastML. For INDELible trees with 50 extants and 50 positions, even at the highest indel rates, SIC-predicted POGs contained the corresponding FastML prediction. At 100 extants and longer sequences, a small number of deviations occurred, which we expect are due to restrictions imposed by mapping inferred encodings to a valid POG.
                             </p>
                             <p id="p-33">
                              Unexpectedly, the indel tests with known alignments where ancestral indel events could be tracked precisely showed little difference between encoding and inference method. With non-curated alignments, there is a greater difference between methods. To next evaluate if insertions and deletions can be used as building blocks to engineer biologically active ancestors, we opted to use the BE encoding, which was better at capturing what the others also captured while containing less outlier indels; we opted for parsimony inference, which seemed to perform comparably to maximum likelihood but was generally quicker.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Indel variation can be used to create hybrid ancestors
                             </h3>
                             <p id="p-34">
                              The insertion or deletion of sequence blocks is well established as a means by which natural proteins evolve new functions. However, previous attempts to use insertions and deletions in protein engineering have been confounded by the fact that indels frequently compromise protein integrity. The location of the insertion or deletion determines its tolerability, but suitable locations are not necessarily evident from the structure and even less so from the sequence alone.
                             </p>
                             <p id="p-35">
                              We hypothesised that indel events suggest plausible blocks of sequence content that could be included or excluded in identified ancestors as a novel approach to creating ancestral variants, orthogonal to substitution. GRASP utilises the history of indel events to predict modular blocks of content capable of being removed from ancestors in which they occur or inserted into ancestors that never contained these modules. In doing this, GRASP fundamentally extends the nature and practical application of modulating variation within ancestors and is capable of identifying modular insertions that are well tolerated and can affect properties such as thermal stability and substrate specificity. The ability to manipulate both of these properties is desirable for protein engineering.
                             </p>
                             <p id="p-36">
                              We used GRASP to identify two distinct lineage-specific insertions within the CYP2U/CYP2R/CYP2D data set (LSEE, LLSPP), occurring at nodes N5 and N51 (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              ). We synthesised the inferred ancestors N5 and N51, as well as a more ancient ancestor, N2, that did not contain either insertion. We removed the insertion LSEE from N5 at sequence position 153 (N5_153dLSEE) and removed the insertion LLSPP from N51 at sequence position 27 (N51_27dLLSPP). We then separately inserted them into N2 at the equivalent sequence positions 152 (N2_152iLSEE) and 27 (N2_27iLLSPP). Similarly, we tested two variants of the N1 CYP2U ancestor. One form contained a CYP2U-specific insertion of 19 amino acids (N1), and in the other this insertion was removed to resemble the CYP2R and CYP2D sequences (N1_18dIPP…RR).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F3.large.jpg?width=800&height=600&carousel=1" title="a, Phylogenetic tree showing positions of CYP2U/CYP2R/CYP2D ancestors chosen for synthesis and evaluation. Coloured boxes indicate the content that was removed from correspondingly coloured ancestral nodes (N51, N5, and N1) and, in the case of the N51 and N5 insertions, preemptively inserted into N2. b, Amino acid sequences surrounding and including the insertion or deletion of the content at each ancestor. Numbers under sequences indicate the position numbers of the start and end columns represented in the alignment. c, Thermal stability assays for each ancestor with and without inserted content. Data are means +/- SEM, N = 2. d, e, Activity assays for the substrates luciferin CEE and luciferin ME-EGE for each ancestor with and without inserted content. Different symbols indicate different experimental repeats, performed in triplicate, lines indicate mean and standard deviation, and p-values were determined by a two-tailed Student’s t-test. No data points were excluded." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-120415564" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>a</strong>, Phylogenetic tree showing positions of CYP2U/CYP2R/CYP2D ancestors chosen for synthesis and evaluation. Coloured boxes indicate the content that was removed from correspondingly coloured ancestral nodes (N51, N5, and N1) and, in the case of the N51 and N5 insertions, preemptively inserted into N2. <strong>b</strong>, Amino acid sequences surrounding and including the insertion or deletion of the content at each ancestor. Numbers under sequences indicate the position numbers of the start and end columns represented in the alignment. <strong>c</strong>, Thermal stability assays for each ancestor with and without inserted content. Data are means +/- SEM, <em>N</em> = 2. <strong>d, e</strong>, Activity assays for the substrates luciferin CEE and luciferin ME-EGE for each ancestor with and without inserted content. Different symbols indicate different experimental repeats, performed in triplicate, lines indicate mean and standard deviation, and p-values were determined by a two-tailed Student’s <em>t</em>-test. No data points were excluded.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F3.medium.gif" width="306" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F3.medium.gif" width="306" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <p id="p-37" class="first-child">
                                <strong>
                                 a
                                </strong>
                                , Phylogenetic tree showing positions of CYP2U/CYP2R/CYP2D ancestors chosen for synthesis and evaluation. Coloured boxes indicate the content that was removed from correspondingly coloured ancestral nodes (N51, N5, and N1) and, in the case of the N51 and N5 insertions, preemptively inserted into N2.
                                <strong>
                                 b
                                </strong>
                                , Amino acid sequences surrounding and including the insertion or deletion of the content at each ancestor. Numbers under sequences indicate the position numbers of the start and end columns represented in the alignment.
                                <strong>
                                 c
                                </strong>
                                , Thermal stability assays for each ancestor with and without inserted content. Data are means +/- SEM, N = 2.
                                <strong>
                                 d, e
                                </strong>
                                , Activity assays for the substrates luciferin CEE and luciferin ME-EGE for each ancestor with and without inserted content. Different symbols indicate different experimental repeats, performed in triplicate, lines indicate mean and standard deviation, and p-values were determined by a two-tailed Student’s t-test. No data points were excluded.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-38">
                              All ancestral proteins inferred via this process were heterologously expressed in Escherichia coli and characterised. They were shown to express at similar levels and form intact haem-thiolate linkages as indicated by the characteristic spectral peak at 450 nm in the Fe(II).CO vs. Fe(II) difference spectrum (Supplementary Fig. 4), indicative of a properly folded cytochrome P450 enzyme. All were catalytically active towards at least one substrate, when tested with three different P450-Glo™ luminogenic probe substrates, luciferins MultiCYP, ME-EGE, and CEE. Therefore, the presence or absence of these lineage-specific insertions was not essential for the protein folding, cofactor binding, or interaction with the cytochrome P450 reductase. However, it was observed that the lineage-specific insertions did alter the substrate selectivity of the otherwise identical ancestors.
                             </p>
                             <p id="p-39">
                              Both N5 and N51 were active towards both luciferins CEE and ME-EGE, while N2 was only active towards luciferin ME-EGE. Loss of the insertion LLSPP from N51 reduced its activity towards luciferin CEE, and the corresponding gain of the LLSPP insertion in the N2 ancestor increased its activity towards luciferin CEE.
                             </p>
                             <p id="p-40">
                              Neither loss of the insertion LSEE from N5 nor gain of the insertion LSEE in N2 had an effect on luciferin CEE dealkylase activity. The presence of the LSEE insertion in the N2 and N5 ancestors increased both ancestors’ activity towards luciferin ME-EGE. Inclusion of the LLSPP insertion did not have a consistent effect on activity towards luciferin ME-EGE, whereas inclusion increased activity towards ME-EGE in N2, but not in N51. The N1 ancestor was only active towards luciferin MultiCYP, but N1_18dIPP…RR was slightly active towards all three proluciferin substrates, suggesting this insertion may also alter the selectivity of the ancestor.
                             </p>
                             <p id="p-41">
                              The LLSPP insertion also modulated the thermal stability of the ancestors; the insertion produced a small but statistically significant increase in the thermal stability in both the N2 and N51 ancestors, compared to their variants lacking this insertion (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3c
                              </a>
                              ). This effect was not seen for the LSEE insertion (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3c
                              </a>
                              ), suggesting that these effects are protein and sequence specific.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Using more sequences reduces variance in inferred ancestral sequences
                             </h3>
                             <p id="p-42">
                              GRASP is able to process large numbers of sequences (i.e., greater than 10,000), which is necessary to capture the true diversity of the sequence space. Intuitively, more data equates to better coverage (and resolution) of the biological sequence space. If this saturation of data is based on true homology, the level of variance is reduced in inferred ancestral states relative to reconstructions from less data. This intuition has been shown to be supported in experiments on smaller data sets
                              <sup>
                               <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                              </sup>
                              . Here, we generalise this assertion for a much larger data set that contains the added complexity of obscured homology, due to a greater presence of indel events.
                             </p>
                             <p id="p-43">
                              Indel handling is critical for ASR, yet routinely problematic, and the accurate management of indel events is essential to decide on the extent of sequence content to include for any particular ancestor
                              <sup>
                               <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                              </sup>
                              . As data set sizes grow, the number of columns in a sequence alignment, or positions in a POG, increases substantially and the indel histories may appear more complicated. Therefore, increasing the number of sequences does not necessarily lead to inferences of increasingly invariant ancestral sequences.
                             </p>
                             <p id="p-44">
                              Evidence that ancestors are constrained to a specific invariant form as the number of sequences increases must be combined with evidence that this form is viable and not a sequence that has been converged upon arbitrarily. To test the effect of increasing data set size on ancestral inference, we assembled sequence data sets for the DHAD and CYP2U protein families via increments of sequence data (see Methods) and compared the ancestral inferences for each data set size (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4a-d
                              </a>
                              ), ranging from 1,612 to 9,112 sequences for DHAD and between 165 and 595 sequences for CYP2U. The DHAD data sets were increased by adding sequences from across the DHAD taxonomic space, while the CYP2U data sets were increased by adding sequences from sister groups CYP2R and CYP2D while retaining the same number of CYP2U sequences at each point. For the DHAD data set we also performed a sparse reconstruction of 585 sequences, containing primarily reviewed Swiss-Prot sequences.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F4.large.jpg?width=800&height=600&carousel=1" title="a, Phylogenetic trees of the smallest and largest DHAD data sets after producing 14 randomly sampled data sets in 500 sequence increments, added to our base data set of 1,612 sequences and reaching a maximum size of 9,112 sequences. b, Heat maps of the fractional distances between ancestor sequences generated from different DHAD data set sizes, representing the same (principal) three branch points. c, Phylogenetic trees of the smallest and largest data sets after increasing CYP2U sequences via addition of homologous subfamilies, starting with 165 CYP2U sequences then growing to 359 sequences and reaching a maximum of 595 sequences via addition of sequences from CYP2R and CYP2D, respectively. d, Heat maps of the fractional distances between ancestor sequences resulting from different CYP2U data set sizes representing the same two branch points. Ancestors from the N4/N5 equivalent branch points across the three data set sizes are not shown but had 98% identity. e, Heat map of the average fractional distance of 50 randomly selected ancestors between the KARI I data sets, ranging from 1,176 to 11,756 sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-120415564" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>a</strong>, Phylogenetic trees of the smallest and largest DHAD data sets after producing 14 randomly sampled data sets in 500 sequence increments, added to our base data set of 1,612 sequences and reaching a maximum size of 9,112 sequences. <strong>b</strong>, Heat maps of the fractional distances between ancestor sequences generated from different DHAD data set sizes, representing the same (principal) three branch points. <strong>c</strong>, Phylogenetic trees of the smallest and largest data sets after increasing CYP2U sequences via addition of homologous subfamilies, starting with 165 CYP2U sequences then growing to 359 sequences and reaching a maximum of 595 sequences via addition of sequences from CYP2R and CYP2D, respectively. <strong>d</strong>, Heat maps of the fractional distances between ancestor sequences resulting from different CYP2U data set sizes representing the same two branch points. Ancestors from the N4/N5 equivalent branch points across the three data set sizes are not shown but had 98% identity. <strong>e</strong>, Heat map of the average fractional distance of 50 randomly selected ancestors between the KARI I data sets, ranging from 1,176 to 11,756 sequences.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F4.medium.gif" width="311" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F4.medium.gif" width="311" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <p id="p-45" class="first-child">
                                <strong>
                                 a
                                </strong>
                                , Phylogenetic trees of the smallest and largest DHAD data sets after producing 14 randomly sampled data sets in 500 sequence increments, added to our base data set of 1,612 sequences and reaching a maximum size of 9,112 sequences.
                                <strong>
                                 b
                                </strong>
                                , Heat maps of the fractional distances between ancestor sequences generated from different DHAD data set sizes, representing the same (principal) three branch points.
                                <strong>
                                 c
                                </strong>
                                , Phylogenetic trees of the smallest and largest data sets after increasing CYP2U sequences via addition of homologous subfamilies, starting with 165 CYP2U sequences then growing to 359 sequences and reaching a maximum of 595 sequences via addition of sequences from CYP2R and CYP2D, respectively.
                                <strong>
                                 d
                                </strong>
                                , Heat maps of the fractional distances between ancestor sequences resulting from different CYP2U data set sizes representing the same two branch points. Ancestors from the N4/N5 equivalent branch points across the three data set sizes are not shown but had 98% identity.
                                <strong>
                                 e
                                </strong>
                                , Heat map of the average fractional distance of 50 randomly selected ancestors between the KARI I data sets, ranging from 1,176 to 11,756 sequences.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-46">
                              With GRASP, we observed that as data set size increased, the predicted ancestor sequences approached invariant forms in terms of amino acid sequence at equivalent phylogenetic nodes between different tree sizes. To further illustrate these trends, we inferred KARI ancestors in regular increments ranging from 1,176 sequences to 11,756 sequences. These ancestors also converged towards invariant forms with the addition of more sequences (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4e
                              </a>
                              ). While the number of positions in the input sequence alignment generally increased with coverage, the length of the ancestor sequences was not correlated with the number of input sequences (Supplementary Fig. 5).
                             </p>
                             <p id="p-47">
                              GRASP was able to complete the reconstruction of the largest data sets in this study within 7 hours for DHAD (9,112 sequences, 1,381 positions in alignment) and within 6 hours for KARI (11,756 sequences, 667 positions) (Supplementary Fig. 6).
                             </p>
                             <p id="p-48">
                              Ancestral proteins inferred from the smallest and largest data sets for both DHAD and CYP2U were all active towards expected substrates, despite differences in ancestral sequence identity between the two extremes of data set size (DHAD 75%, CYP2U 80% sequence identity). All DHAD ancestors displayed enzymatic activity to D-gluconate. We observed that three DHAD ancestral proteins from the smallest data set had thermal shift profiles comparable to those of the three ancestors that were located in equivalent tree positions in the largest data set (Supplementary Fig. 7). For two of the three DHAD reference ancestors, the melting points in the proteins from the larger reconstruction were increased by approximately 5 °C relative to their counterparts from the smaller data set (Supplementary Fig. 7). Likewise, the inclusion of the sister clades for the CYP2U reconstruction increased the thermal stability of the CYP2U whole-subfamily ancestors inferred from each set (165, 359, and 595 sequences) and these ancestors were all shown to be active towards the substrate, luciferin MultiCYP (Supplementary Fig. 8).
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              On smaller data sets, GRASP’s predictions are consistent with the predictions of existing methods
                             </h3>
                             <p id="p-49">
                              We compared GRASP against two alternative ASR tools, selected due to their dominant use in the literature: FastML
                              <sup>
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                              </sup>
                              and the aaml program from the Phylogenetic Analysis by Maximum Likelihood (PAML) package
                              <sup>
                               <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                                28
                               </a>
                              </sup>
                              . We were able to produce ancestral proteins from reconstructions produced by GRASP, FastML, and PAML on a CYP2U/CYP2R data set comprising 359 sequences. To compare between indel inference methods again, we noted that GRASP (BE) and FastML (SIC) returned the same indel composition at their root ancestors and this pattern was overlaid onto the prediction from PAML, which was run without gap inference. The oldest CYP2U ancestors had ~95% sequence identity and, regardless of the tool used, ancestral proteins expressed at similar levels in E. coli, displayed characteristic cytochrome P450 spectra and activities towards the luciferin MultiCYP substrate, and had similar thermal stabilities (Supplementary Fig. 9).
                             </p>
                             <p id="p-50">
                              To make statements about the accuracy of ancestral predictions is problematic as the historically correct and complete evolutionary record is unavailable. To sidestep this issue, we first applied each tool and configuration to generate multiple predictions of the same principal ancestor branch point based on successively smaller subsets of a given sequence family. Secondly, we performed two tests asking: (a) between tools, how similar is the prediction of one tool to those of the others; and (b) how similar is the prediction of one tool from the sub-sampled data to a better-sampled ancestor, predicted from the complete family? We reasoned that a better method would be one which tended to agree with the majority of others, and one that with less data tended to agree with a prediction based on more data (assuming that more data help to improve a prediction).
                             </p>
                             <p id="p-51">
                              A large alignment of 1,682 sequences (KARI class II, adapted from Gumulya et al.
                              <sup>
                               <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">
                                8
                               </a>
                              </sup>
                              ) and the corresponding phylogenetic tree were divided into sub-groups and used to assess the effect of tool, data set size, and reconstruction parameters on ancestral inference (see Methods for details). We sought to corroborate any trends using a second independent data set (CYP2 with 975 sequences).
                             </p>
                             <p id="p-52">
                              Test (a) measured similarity between ancestors inferred using a given set of tool parameters and group size (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5a
                              </a>
                              and Supplementary Fig. 10a); specifically, we observed fractional distances D/L (where D is the number of substitutions, of L non-gapped, homologous positions) between sequences predicted for each condition tested. Test (b) measured similarity in terms of fractional distances between ancestor predictions of an individual tool (with a given set of parameters and group sample size) and a better-sampled ancestor using all available data (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5b
                              </a>
                              and Supplementary Fig. 10b). The better-sampled ancestor for the comparison in (b) was predicted by GRASP, because a data set of this size could not be completed by FastML or PAML. A series of statistical tests were performed. First, ANOVA was used to evaluate whether the choice of tool, data set size, and rate parameter setting were factors in determining how similar a predicted ancestor sequence (grouped by tool/size/rate setting) was to those predicted by alternative tools with the same setting (test a) and to those generated from the complete data set (test b). A t-test was then used to identify the pairs of labels (on groups) that best explained the observed differences (Supplementary Figs. 11 and 12).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F5.large.jpg?width=800&height=600&carousel=1" title="Tool comparison on KARI data. a, Average fractional distance between tools, calculated as pairwise fractional distances for each ancestral prediction for a given tool against all other ancestral predictions of other tools using 5 groups of 336 or 337 sequences, 10 groups of 168 or 169 sequences, or 20 groups of 84 or 85 sequences. Parameter choices are joint vs. marginal reconstruction and fixed vs. variable evolutionary rates (FastML and PAML only). b, Average fractional distance between a better-sampled ancestor inferred by GRASP using 1,682 sequences and each tool / parameter combination at 5, 10, and 20 groups. c, Run times of GRASP and FastML at 5 and 10 groups; PAML was omitted due to long run times. Run times for all tools at 10 and 20 groups are shown in Supplementary Fig. 13." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-120415564" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Tool comparison on KARI data. <strong>a</strong>, Average fractional distance between tools, calculated as pairwise fractional distances for each ancestral prediction for a given tool against all other ancestral predictions of other tools using 5 groups of 336 or 337 sequences, 10 groups of 168 or 169 sequences, or 20 groups of 84 or 85 sequences. Parameter choices are joint vs. marginal reconstruction and fixed vs. variable evolutionary rates (FastML and PAML only). <strong>b</strong>, Average fractional distance between a better-sampled ancestor inferred by GRASP using 1,682 sequences and each tool / parameter combination at 5, 10, and 20 groups. <strong>c</strong>, Run times of GRASP and FastML at 5 and 10 groups; PAML was omitted due to long run times. Run times for all tools at 10 and 20 groups are shown in Supplementary Fig. 13.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F5.medium.gif" width="310" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F5.medium.gif" width="310" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/07/2019.12.30.891457/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <p id="p-53" class="first-child">
                                Tool comparison on KARI data.
                                <strong>
                                 a
                                </strong>
                                , Average fractional distance between tools, calculated as pairwise fractional distances for each ancestral prediction for a given tool against all other ancestral predictions of other tools using 5 groups of 336 or 337 sequences, 10 groups of 168 or 169 sequences, or 20 groups of 84 or 85 sequences. Parameter choices are joint vs. marginal reconstruction and fixed vs. variable evolutionary rates (FastML and PAML only).
                                <strong>
                                 b
                                </strong>
                                , Average fractional distance between a better-sampled ancestor inferred by GRASP using 1,682 sequences and each tool / parameter combination at 5, 10, and 20 groups.
                                <strong>
                                 c
                                </strong>
                                , Run times of GRASP and FastML at 5 and 10 groups; PAML was omitted due to long run times. Run times for all tools at 10 and 20 groups are shown in Supplementary Fig. 13.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-54">
                              While GRASP can perform character inference via variable evolutionary rates when estimated externally, we opted to use the default uniform rate. Both FastML and PAML estimate position-specific rates internally, so we set out to understand the impact of rate setting. When comparing predictions between tools (test a) or between tool and better-sampled ancestor (test b), both the choice of tool and data set size separately and consistently explained the observed differences in distances; however the rate setting did not.
                             </p>
                             <p id="p-55">
                              The choice of tool mattered for both types of comparisons across the two data sets. GRASP and FastML predictions were generally more similar in terms of mean pairwise fractional distance, both relative to the better-sampled ancestor and relative to the ancestors predicted by PAML. Indeed, greater sequence numbers generally reduced distances between tools and the distances between a tool’s predictions and the ancestor based on the complete data set.
                             </p>
                             <p id="p-56">
                              We also calculated the time taken for all tools to complete the reconstructions with a run time cut-off of 48 hours (Supplementary Fig. 13) and highlighted the time taken for GRASP and FastML to complete the two larger data set sizes (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5c
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-57">
                             Increasing the scale at which ASR can be performed means that greater sequence and functional diversity can be explored and more complicated phylogenetic relationships can be evaluated
                             <sup>
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             . Incorporating data from sister clades and remote homologs can allow ancestors to be inferred with greater robustness at greater evolutionary distances. We substantiated these points computationally and experimentally by resurrecting twenty ancestral variants across three enzyme families, all of which were shown to be catalytically active.
                            </p>
                            <p id="p-58">
                             At greater scale, ASR will increasingly require judgements to be made as to whether ancestors contain an insertion or deletion present in only some of the multiple descendent branches of a phylogenetic tree. The presence of these events may pinpoint loop remodelling events
                             <sup>
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                             </sup>
                             or other conformational diversity in the family. The evolution of conformational diversity may promote entirely new functions and provide a source of variation that protein engineers can exploit
                             <sup>
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-59">
                             At the core of our approach is the POG data structure, originally proposed to facilitate multiple sequence alignment
                             <sup>
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ,
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                             </sup>
                             . We developed inference methods to directly use the POG data structure to pinpoint likely phylogenetic positions for indel events from homologs placed in an alignment. It effectively delineates sequence content at all internal nodes of a given phylogenetic tree, and collectively enables the evolutionary relationships between all branch points to be traced. As a consequence, evolutionary events that are isolated to specific clades, and alignment ambiguities that are difficult to resolve at a single branch point, can be disentangled across time. In contrast to current approaches based on gapped sequence representations, POGs enable the identification of all supported indel histories across a reconstructed family. In GRASP, edges that are partially supported but are not chosen to form the preferred ancestral sequence still appear as alternative paths through the POG; multiple branch points can be inspected and contrasted at once. GRASP thus provides a framework to delete, reintroduce, or preemptively include indel variation that supports both exploring sequence space and creating new function.
                            </p>
                            <p id="p-60">
                             Hybrid ancestors represent a novel class of variant that can be identified readily and resurrected through the partitioning of indel events onto individual edges within a POG. The identification of alternative indels, each compatible with a given ancestor (or nearby ancestors) can be used to test alternative hypotheses about the progression of evolutionary events where there is uncertainty in the inferred evolutionary history. In addition, combinations of compatible indels can be sampled in order to engineer novel sequences by including or removing blocks of sequence content. Strikingly, given the substantial impact that indel events are likely to have on any protein sequence, coupled with the divergence between chosen ancestral sequences, all hybrid ancestors we reconstructed were functional.
                            </p>
                            <p id="p-61">
                             With CYP2U as an example, we showed that hybrid ancestors folded to form holoenzymes that are catalytically active when tested in vitro and are capable of interacting with the native human reductase. Additionally, with CYP2U, we showed that inclusion of these modular blocks allowed for increased thermal stability and altered substrate preference. We stress that we are not attributing the increased stability or interaction with a specific substrate solely to the identified insertion. Rather, GRASP identified blocks of content that are likely to be tolerated at positions at which such changes have occurred during natural evolution, despite any effect they have on folding and function of these ancestral proteins. Due to the complex nature of protein folding and epistasis, these blocks will not always behave in predictable ways and effects will depend on the ancestral sequence and sequence context into which they are being inserted.
                            </p>
                            <p id="p-62">
                             The identification of modular insertions altered the substrate selectivity, not unlike how substitution variants identified by marginal reconstruction have in previous studies
                             <sup>
                              <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             . This study provides a proof-of-concept that indel histories can suggest a form of variation that protein engineers can use that is orthogonal to varying specific amino acids. We foresee this as being of practical use for (1) altering function through the addition and removal of discrete, evolutionarily-defined building blocks to engineer variants with altered catalytic and physical properties (e.g. thermal stability) and (2) exploring alternative ancestors where there is ambiguity in the true phylogenetic position of an indel.
                            </p>
                            <p id="p-63">
                             There are multiple options available for predicting indel events, and on simulated data they perform equally well. Bi-directional edge methods, which consider indels in the context of other possible choices, allow more opportunities to decide between alternative pathways and provide better ways to represent ambiguity when equally parsimonious or equally likely scenarios exist. This may prove beneficial to the protein engineer, who may have access to secondary data to guide their selection for applied purposes, including knowledge of structure or residue interaction.
                            </p>
                            <p id="p-64">
                             ASR has been used extensively in recent years
                             <sup>
                              <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ,
                              <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                             </sup>
                             and it is important to understand the relative performance of different tools, and to recognise proven principles that underpin successful methods. We note that FastML is in broad agreement with GRASP, with predictions made by FastML showing closer evolutionary distances to predictions made by GRASP than to those generated by PAML. We also demonstrated that incorporating more sequence data resulted in smaller fractional distances between inferred ancestors, regardless of the tool used.
                            </p>
                            <p id="p-65">
                             Based on the analyses of KARI and DHAD, we demonstrated that despite an increase in diversity, ancestral sequences converge towards invariant forms when using data sets with almost 10,000 sequences. Ancestral sequences generated from smaller data sets exhibit greater variation in ancestral sequence identity relative to the ancestral sequences from larger data sets. This supports the notion that greater representation of a family provides a constraint for the ancestor, i.e. that robust reconstructions are best achieved when available sequence data is exploited to the fullest extent.
                            </p>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              GDH-GOx experimental methods
                             </h3>
                             <div id="sec-13" class="subsection">
                              <h4>
                               GDH-GOx ancestral inference
                              </h4>
                              <p id="p-66">
                               Starting from an aligned data set and phylogenetic tree previously established by Sützl et al.
                               <sup>
                                <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                                 16
                                </a>
                               </sup>
                               for the GDH-GOx cluster, only the four major clades (GOx, GDH I, GDH II, and GDH III) were selected, together with the second small GDH III clade. All sequences with >800 amino acids as well as manually selected sequences showing large insertions were removed from the selection, resulting in 399 sequences. This sequence selection was aligned by MAFFT v7.271 G-INS-i
                               <sup>
                                <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                 32
                                </a>
                               </sup>
                               , the alignment trimmed for positions with >99% gaps by trimAl v1.2
                               <sup>
                                <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                 33
                                </a>
                               </sup>
                               , and pruned using Gblocks 0.91b
                               <sup>
                                <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                 34
                                </a>
                               </sup>
                               with a less stringent block selection. The phylogenetic tree was inferred by PhyML
                               <sup>
                                <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                 35
                                </a>
                               </sup>
                               using default settings, with subtree pruning and regrafting to optimise tree topology, smart model selection, and aLRT SH-like branch support calculated. The tree was rooted on the midpoint. Marginal reconstruction of ancestral nodes was performed with the LG evolutionary rate model
                               <sup>
                                <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                 36
                                </a>
                               </sup>
                               , with indels inferred using bi-directional edge parsimony, after the N- and C-termini of the alignment had been trimmed.
                              </p>
                             </div>
                             <div id="sec-14" class="subsection">
                              <h4>
                               GDH-GOx synthesis and cloning
                              </h4>
                              <p id="p-67">
                               The N- and C-terminal sequences not present in the ancestral sequences were replaced by the equivalent amino acid sequences of GOx from Aspergillus niger, ‘MQTLLVSSLVVSLAAALPHYIRSNGIEASLLTDPKDVSGRT’ and ‘ASMQ’, respectively. Resulting ancestral genes were codon-optimised for Komagataella phaffii (formerly Pichia pastoris) expression, ordered from BioCat GmbH, and cloned into the expression vector pPICZ A together with an added polyhistidine tag (6 x His). Constructs were linearized with PvuII and transformed into K. phaffiii via electroporation.
                              </p>
                             </div>
                             <div id="sec-15" class="subsection">
                              <h4>
                               GDH-GOx expression
                              </h4>
                              <p id="p-68">
                               Ancestral and extant GOx and GDH genes were expressed in K. phaffiii under the AOX1 promoter with methanol induction. Routine cultivations and selection of the transformed cells were done in liquid YPD medium supplemented with zeocin (100 mg/L) at 30 °C and 130 rpm. Expression was done in shake flasks at 30 °C and 130 rpm on modified BMMY medium (20 g/L peptone from casein, 10 g/L yeast extract, 100 mM potassium phosphate buffer pH 6.0, 10 g/L (NH
                               <sub>
                                4
                               </sub>
                               )
                               <sub>
                                2
                               </sub>
                               SO
                               <sub>
                                4
                               </sub>
                               , 3.4 g/L yeast nitrogen base (without amino acids and (NH
                               <sub>
                                4
                               </sub>
                               )
                               <sub>
                                2
                               </sub>
                               SO
                               <sub>
                                4
                               </sub>
                               ), and 0.4 mg/L biotin) containing 12 g/L sorbitol and 2% methanol. After centrifugation at 6,000 x g and 4 °C for 30 minutes, supernatants were loaded onto a 5 mL HisTrap column (GE Healthcare) equilibrated with binding buffer (50 mM potassium phosphate buffer pH 6.5, 500 mM NaCl, and 20 mM imidazole) and washed for at least 10 column volumes. Proteins were eluted using a linear gradient from 20 to 500 mM imidazole (50 mM potassium phosphate buffer pH 6.5, 500 mM NaCl and 500 mM imidazole). Manually collected fractions were concentrated and desalted (50 mM phosphate buffer pH 6.5) in Vivaspin 20 tubes (Sartorius) with 30,000 Da molecular mass cut-off.
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               GDH-GOx activity assays
                              </h4>
                              <p id="p-69">
                               Both GDH and GOx activity were measured spectrophotometrically at 30 °C on a UV/Vis spectrophotometer (Lambda 35, Perkin Elmer), using appropriately diluted enzyme solution, 20 mM D-glucose, and the respective electron acceptor in 50 mM potassium phosphate buffer pH 6.5. The electron acceptors 1,4-benzoquinone (BQ) and ferrocenium-hexafluorophosphate (FcPF6) were used at 0.5 and 0.2 mM, and their reduction was followed at 290 and 300 nm, respectively. Reduction of the electron acceptor oxygen was measured using the peroxidase-coupled 2,2’-azino-bis(3-ethylbenzothiazoline-6-sulphonic acid) (ABTS) assay
                               <sup>
                                <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                 37
                                </a>
                               </sup>
                               , following the reduction of 0.1 mM ABTS at 420 nm.
                              </p>
                             </div>
                             <div id="sec-17" class="subsection">
                              <h4>
                               GDH-GOx thermal stability assays
                              </h4>
                              <p id="p-70">
                               The thermal stability of GDH-GOx enzymes was assessed by differential scanning calorimetry conducted on a PEAQ-DSC automated instrument (Malvern Panalytical). All enzyme samples were diluted to 5 μM (~ 0.33 mg/ml) in 50 mM potassium phosphate buffer pH 6.5, and scanned from 20-90 °C with a scan rate of 60 °C/h and feedback set to high. Instrument blanks were recorded using buffer only and rescans were measured for all samples. Data analysis was performed using the MicroCal PEAQ-DSC software V.1.22. The background signal was subtracted using rescans whenever applicable or buffer blanks otherwise, the baseline was fitted using the spline method, and peaks were fitted with a non-two-state model.
                              </p>
                             </div>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              DHAD experimental methods
                             </h3>
                             <div id="sec-19" class="subsection">
                              <h4>
                               DHAD ancestral inference
                              </h4>
                              <p id="p-71">
                               A minimal 585-sequence set was created, consisting of members annotated with family “Ilvd/Edd” in UniProt, and excluding sequence fragments (as defined by UniProt). Most sequences were from Swiss-Prot, with several TrEMBL entries added due to function and structural data being available. A baseline data set of 1,612 sequences was created from the minimal data set, ensuring that 19 nominated enzymes with experimental data (functional and/or structural) were included, as well as members of their UniRef90 clusters. A background data set of 8,221 sequences included annotated members from the broadest assortment of species, only filtered to be non-redundant at 90% identity (using UniRef90). All three data sets were checked for the aligned location of two motifs (CDK and PCN/PGH/SAH with provision for a substitution) that are associated with the active site
                               <sup>
                                <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                 38
                                </a>
                               </sup>
                               ; sequences that did not exhibit these motifs were removed.
                              </p>
                              <p id="p-72">
                               The background data set was then repeatedly and independently sampled to extend the baseline data set to up to 9,112 sequences. At each size increment of 500 sequences an alignment was created using Clustal Omega
                               <sup>
                                <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                 39
                                </a>
                               </sup>
                               , and a phylogenetic tree was inferred using FastTree
                               <sup>
                                <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                 40
                                </a>
                               </sup>
                               and rooted using phosphogluconate dehydratase as an outgroup. Despite differences in alignments and phylogenetic trees at each data size increment, we were able to map any ancestor in a smaller tree to an ancestor in a larger tree by maximising shared inclusions and exclusions of member proteins of the ancestral subtrees. Joint reconstruction was performed with indels inferred with bi-directional edge parsimony and the JTT evolutionary model
                               <sup>
                                <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                 41
                                </a>
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-20" class="subsection">
                              <h4>
                               DHAD synthesis and cloning
                              </h4>
                              <p id="p-73">
                               The inferred DHAD ancestral genes N1, N423, and N560 from the 585 data set, and the equivalent nodes N9, N1442, and N1443 from the 9,112 data set were optimised for E. coli expression and synthesised by Twist Bioscience and ATG:biosynthetics GmbH, respectively. After amplification, the purified DNA fragments were digested with SapI, then ligated into a modified pET26 vector (p7XNH3)
                               <sup>
                                <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                 42
                                </a>
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-21" class="subsection">
                              <h4>
                               DHAD expression
                              </h4>
                              <p id="p-74">
                               Expression of the DHAD genes was performed in shaking flasks. E. coli BL21 (DE3) cells transformed with the p7XNH3 plasmid and the appropriate inserted gene fragment were grown as an overnight pre-culture in lysogeny broth supplemented with kanamycin (100 μg/ml), and then inoculated 1:50 into auto-induction ZP-5052 medium
                               <sup>
                                <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                 43
                                </a>
                               </sup>
                               supplemented with kanamycin (100 μg/ml). These cultures were incubated at 90 rpm and 37 °C for 3 hours and then overnight at 18 °C in a horizontal orbital shaking incubator. Cells were disrupted by sonication in binding buffer (50 mM potassium phosphate buffer, 500 mM NaCl, 10% glycerol, and 20 mM imidazole) at pH 8.0. Cell debris was pelleted by centrifugation. Proteins were purified using an ÄKTA Purifier FPLC system and a HisTrap HP Nickel column (GE Healthcare). Filtered samples were loaded onto the column and washed with binding buffer. The His-tagged proteins were then eluted with elution buffer (50 mM potassium phosphate buffer, 500 mM NaCl, 10% glycerol, and 500 mM imidazole) at pH 8.0. Desalting of the enzymes was carried out using HEPES buffer pH 7.0.
                              </p>
                             </div>
                             <div id="sec-22" class="subsection">
                              <h4>
                               DHAD activity assays
                              </h4>
                              <p id="p-75">
                               DHAD activity was analysed by HPLC of an assay mixture containing the respective DHAD, 25 mM HEPES buffer pH 7.0, 5 mM MgSO
                               <sub>
                                4
                               </sub>
                               , and 25 mM of sodium D-gluconate, and incubated at 30 °C. Samples were taken every few hours for 3 days. The enzyme was removed by ultrafiltration (PES 10 kDa MWCO, VWR) and the samples were stored at −20 °C until analysed by HPLC. HPLC measurements were performed on an Ultimate-3000 HPLC system (Dionex), equipped with an auto-sampler and diode-array detector. D-gluconate and products were separated by using a Metrosep A supp10-250/40 column (250 mm, particle size 4.6 μm, Metrohm) at 65 °C by isocratic elution with 12 mM ammonium bicarbonate at pH 10, followed by a washing step with 30 mM sodium carbonate at pH 10.4 and a flow rate of 0.2 ml/min. Each sample injection volume was 10 μl. System peak calibration was performed using external standards of the known compounds.
                              </p>
                             </div>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              CYP2U experimental methods
                             </h3>
                             <div id="sec-24" class="subsection">
                              <h4>
                               CYP2U ancestral inference
                              </h4>
                              <p id="p-76">
                               Five candidate CYP2U proteins were chosen, one each from Andrias davidianus, Python bivittatus, Marmota marmota, Poecilia reticulata, and Amazona aestiva. A BLASTp search of each of the candidates was conducted, excluding hits from plants (taxonomic id:3193) or fungi (taxonomic id:4751) (E-Value = 0.00001). Sequences from the BLASTp search were retained if they had at least 55% sequence identity to the original candidate sequence. This procedure was also repeated retaining sequences with at least 50% sequence identity, however, the additional sequences from this lower bound were all removed at later stages of curation, indicating that 55% was an appropriate level of identity to identify CYP2U subfamily sequences. Sequences from the BLASTp searches were collated and duplicate, identical sequences were removed. Manual curation was performed to remove sequences below 400 amino acids in length or containing unidentified amino acids. Sequences were aligned using MAFFT (L-INS-i) with default parameters
                               <sup>
                                <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                                 32
                                </a>
                               </sup>
                               . Removal of sequences with indel events over 20 amino acids (suggestive of incorrect annotation of splice sites) was completed in an iterative manner by first identifying which sequence had the longest indel over 20 amino acids, removing it and realigning the remaining sequences, and then continuing until no sequence had an indel over 20 amino acids. Sequences were manually inspected and any sequences with apparent frameshift mutations were removed. Sequences were mapped back to their exon structure and removed if they had more than two exons difference to the accepted number of five exons for CYP2U sequences. Sequences missing the conserved cysteine residue characteristic of cytochrome P450 enzymes were removed. Similar procedures were used to generate the CYP2R and CYP2D subfamily alignments, then the full set of all three subfamilies was combined and realigned in MAFFT to generate the largest data set. Phylogenetic trees were inferred using RAxML
                               <sup>
                                <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                 44
                                </a>
                               </sup>
                               . A CYP2R Latimeria chalumnae sequence (XP_005989762.1) was manually shifted on the phylogenetic tree to better represent the known phylogeny
                               <sup>
                                <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                 45
                                </a>
                               </sup>
                               , while retaining each sequence’s overall evolutionary distance to the root. Joint reconstruction was performed with indels inferred with bi-directional edge parsimony and the JTT evolutionary rate model
                               <sup>
                                <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                                 41
                                </a>
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-25" class="subsection">
                              <h4>
                               CYP2U synthesis and cloning
                              </h4>
                              <p id="p-77">
                               The amino acid sequences of CYP2U ancestors were inferred starting from the conserved PPGP motif, which signifies the junction between the transmembrane anchor and the catalytic domain. For expression of the resurrected ancestors in bacteria, this region was replaced with an N-terminal sequence (MAKKTSSKGKL), which is known to improve expression yields of microsomal P450s in bacteria
                               <sup>
                                <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                 46
                                </a>
                               </sup>
                               and had been used to express human CYP2U1 in E. coli
                               <sup>
                                <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                 47
                                </a>
                               </sup>
                               . To enable purification, a flexible ST linker followed by a polyhistidine tag (6 x His) was added to the C-termini of the sequences. All ancestor sequences were codon-optimised for E. coli expression, and the N-termini were optimised initially using mRNA optimiser
                               <sup>
                                <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                 48
                                </a>
                               </sup>
                               and subsequently manually until the predicted free energy of secondary structure formation was greater than −15 kJ/mol. The genes were synthesised as GeneStrings (GeneArt, Invitrogen) designed with 60 bp 5’ and 3’ end extensions complementary to the pCW vector, cloned by Gibson assembly, and then sequence-verified by dideoxy sequencing (Australian Genome Research Facility). Correct inserts were subcloned into a bicistronic pCW vector upstream of the open reading frame for the human cytochrome P450 reductase (hCPR) using the NdeI and XbaI sites.
                              </p>
                             </div>
                             <div id="sec-26" class="subsection">
                              <h4>
                               CYP2U expression
                              </h4>
                              <p id="p-78">
                               DH5α F’ IQ™ E. coli cells carrying the pGro7 plasmid were transformed with pCW vectors containing the relevant P450 and CPR genes or the empty vector (“pCW controls”), and selected using chloramphenicol (20 μg/ml) and ampicillin (100 μg/ml). Single colonies were used to inoculate overnight cultures in lysogeny broth with antibiotics, as above. Batch cultures were grown at 25 °C, 180 rpm in 500 ml flasks containing 50 ml terrific broth supplemented with trace elements, 1 mM thiamine, and antibiotics. Cultures were induced after 5 hours with 1 mM IPTG and 4 mg/ml L-arabinose, and supplemented with 500 mM delta-aminolaevulinic acid. Cultures were grown for a further 43 hours before harvesting by centrifugation at 6,000 x g for 10 minutes. E. coli pellets were weighed and resuspended in 2 ml/g (wet weight) sonication buffer (100 mM potassium phosphate buffer pH 7.4, containing 20% (w/v) glycerol, 6 mM magnesium acetate, 1 mM PMSF, and a protease inhibitor cocktail (Sigma-Aldrich)). Cells were lysed using a Constant Systems OneShot cell disruptor followed by centrifugation at 10,000 x g for 20 minutes. The supernatant was centrifuged at 180,000 x g for 1 hour and the pellet was resuspended in TES buffer (100 mM Tris acetate, 500 mM sucrose, and 0.5 mM EDTA pH 7.6) using a Potter-Elvehjem homogeniser. The P450 concentration was determined in intact cells and membranes using Fe(II).CO vs. Fe(II) difference spectroscopy
                               <sup>
                                <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                 49
                                </a>
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-27" class="subsection">
                              <h4>
                               CYP2U activity assays
                              </h4>
                              <p id="p-79">
                               P450 (0.02 μM), added in membranes prepared from bacteria coexpressing hCPR, was premixed with 50 μM luciferin CEE or luciferin ME-EGE (Promega) in 100 mM potassium phosphate pH 7.4, and incubated at 37 °C for 10 minutes. Reactions were initiated by addition of the NADPH-regenerating system (NGS; 0.25 mM NADP
                               <sup>
                                +
                               </sup>
                               , 10 mM glucose-6-phosphate, and 0.5 U/ml glucose-6-phosphate dehydrogenase), and incubated with gentle shaking at 37 °C for 30 minutes. An equal volume of the luciferin detection reagent was added, and reactions were incubated for a further 20 minutes at room temperature. Luminescence was measured using a CLARIOstar multimodal plate reader (BMG Labtech).
                              </p>
                             </div>
                             <div id="sec-28" class="subsection">
                              <h4>
                               CYP2U thermal stability assays
                              </h4>
                              <p id="p-80">
                               Ancestors were expressed in E. coli as described above and cell pellets were resuspended in whole cell spectral assay buffer (WCAB; 100 mM potassium phosphate, 20 mM D-glucose, and 6 mM magnesium acetate pH 7.4) to one eighth of the original culture volume. The resuspended cultures, distributed into tubes in 200 μL volumes, were incubated at a range of temperatures (25-80 °C, in 5 °C increments) for 60 minutes, followed by a 5 minute recovery at 4 °C and equilibration at 25 °C. The remaining P450 content was measured in intact cells using the method of Johnston et al.
                               <sup>
                                <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">
                                 49
                                </a>
                               </sup>
                               . The proportion of total P450 content compared to the unheated control (25 °C) was plotted against temperature and the
                               <sup>
                                60
                               </sup>
                               T
                               <sub>
                                50
                               </sub>
                               value was calculated by fitting the data to a variable slope (4-parameter) dose response curve in GraphPad Prism 8.0.
                              </p>
                             </div>
                            </div>
                            <div id="sec-29" class="subsection">
                             <h3>
                              KARI experimental methods
                             </h3>
                             <div id="sec-30" class="subsection">
                              <h4>
                               KARI ancestral inference
                              </h4>
                              <p id="p-81">
                               We created two separate data sets representing KARI class I and class II, respectively. The sequence alignment for class II was taken directly from Gumulya et al.
                               <sup>
                                <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">
                                 8
                                </a>
                               </sup>
                               and used to compare tools. Class I sequences were compiled by searching for both reviewed and unreviewed proteins in UniProt, designated as bacterial and belonging to the family (26,485 sequences). We removed all fragments and sequences above the length of 400 to exclude obvious cases of class II enzymes. The sequence set was redundancy-reduced with CD-HIT at 99%
                               <sup>
                                <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                 50
                                </a>
                               </sup>
                               , resulting in 11,920 sequences, from which 57 sequences were manually removed by observing a C-terminal knotted domain, indicative of class II. After aligning all sequences with Clustal Omega
                               <sup>
                                <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">
                                 39
                                </a>
                               </sup>
                               , the Shannon entropy of gap vs. character content was determined for all columns and sequences with high entropy over consecutive columns were removed, resulting in a final set of 11,756 sequences. Phylogenetic tree inference was carried out using FastTree
                               <sup>
                                <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                                 40
                                </a>
                               </sup>
                               .
                              </p>
                              <p id="p-82">
                               The KARI class I data sets were created by decreasing their size from 11,756 via 10 regular decrements reaching a minimum representation of 1,176 sequences. For each subset, sequences were randomly removed and the alignment was recalculated independently. For each alignment, a new tree was calculated, and rooted using KARI sequences in Aquificae and Thermotogae as an outgroup. For each sub-set, we computed reconstructions for 50 randomly chosen ancestor nodes (mapped across each tree as described for the DHAD data sets). Joint reconstruction was performed with indels inferred with bi-directional edge parsimony and the JTT evolutionary model
                               <sup>
                                <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">
                                 41
                                </a>
                               </sup>
                               .
                              </p>
                             </div>
                            </div>
                            <div id="sec-31" class="subsection">
                             <h3>
                              GRASP implementation
                             </h3>
                             <p id="p-83">
                              The three main stages of GRASP are (1) to construct an indel history for every position in the alignment, (2) to infer character states for all positions relevant to each ancestor, and (3) to form a POG by linking positions inferred for each ancestor.
                             </p>
                             <p id="p-84">
                              GRASP infers ancestor character states from a set of M input sequences
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              has N aligned positions, indexed with
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              In classical sequence alignments, positions without sequence content are padded, often shown as ‘–’; we use
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to index positions in sequence j with actual sequence content, S
                              <sub>
                               ji
                              </sub>
                              = x where
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              when
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the set of amino acids. Later, it will be convenient to refer to the transpose of
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , namely
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , which indexes all sequences with content at position i.
                             </p>
                             <p id="p-85">
                              Inference is based on a given phylogenetic tree T with a nominated root, that has M – 1 branch points (if bifurcating, fewer when multifurcating) indexed by
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ; we designate the index k = M + 1 for the root of the tree. The superset of extant and ancestor sequences (matched to POGs) is indexed by
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The topology of T defines parent-child relationships,
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              indexes the ancestral descendants of an ancestor k; conversely, we define a function κ(k′) = k″ to indicate that k″ is the direct ancestor of k′, where
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-86">
                              Character states are inferred with an evolutionary model (in the form of an instantaneous rate matrix, indexed by
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ) and maximum likelihood
                              <sup>
                               <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                                51
                               </a>
                              </sup>
                              . Inference is implemented via a Bayesian network that shares the topology of a positionspecific tree, which is T minus the ancestors without content. The position-specific trees are constructed by an indel inference method (in turn informed by an indel encoding). While GRASP implements a number of different encodings, our focus below is on bi-directional edge encoding, which is new.
                             </p>
                             <p id="p-87">
                              Below, we first define key data structures, then we distinguish between: (a) the handling of where homologous positions are placed relative to one another in the trace of ancestral sequences via POGs; and (b) the principles by which homologous positions in extant sequences are used to determine ancestral character states at branch points in the phylogenetic tree. The principles under (b) are unremarkable in themselves, but key benefits are achieved by using them in the ancestor POG from (a). For succinctness, we describe this procedure as it applies to a bifurcating tree; however, the same principles extend seamlessly to multifurcating trees.
                             </p>
                             <div id="sec-32" class="subsection">
                              <h4>
                               Representing sequence content as a partial order graph (POG)
                              </h4>
                              <p id="p-88">
                               A POG is a directed acyclic graph whose elements are ordered relative to other elements; a strict ordering is enforced within a subset of elements, but not always between subsets. When an order is imposed among elements, the relationship must be reflexive, anti-symmetric, and transitive
                               <sup>
                                <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                 14
                                </a>
                               </sup>
                               . A growing body of work in sequence alignment has demonstrated the flexibility that POGs offer for detecting and representing homologous sequence elements during alignment
                               <sup>
                                <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                                 30
                                </a>
                                ,
                                <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                                 31
                                </a>
                                ,
                                <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                                 52
                                </a>
                               </sup>
                               . We take advantage of the flexibility of POGs when projecting homologous elements back in time; they represent deletions and insertions by edges that exclude or include alternative character subsets allowing for optional histories by offering multiple paths at ancestral branch points.
                              </p>
                              <p id="p-89">
                               Formally, a POG is defined by a set of (up to) N nodes that are indexed by
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The indices are determined by performing a topological sort on the input POG (see below); this gives at least one linear and complete ordering (out of several possible). Nodes are connected by a set of directed edges, which is conveniently represented by a matrix
                               <strong>
                                E
                               </strong>
                               , where
                               <strong>
                                E
                               </strong>
                               (a, b) is set to 1 if there is an edge from a to b, and otherwise 0. We introduce an extended index-set
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for rows and columns in
                               <strong>
                                E
                               </strong>
                               , with 0 and N + 1 to start and terminate the POG, so
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . We define next(
                               <strong>
                                E
                               </strong>
                               ,a) = {b:
                               <strong>
                                E
                               </strong>
                               (a, b) > 0} and prev(
                               <strong>
                                E
                               </strong>
                               ,b) = {a:
                               <strong>
                                E
                               </strong>
                               (a, b) > 0} to refer to sets of nodes that occur after and before a node with a given index, respectively. next(
                               <strong>
                                E
                               </strong>
                               , 0) would thus give all possible start indices, and prev(
                               <strong>
                                E
                               </strong>
                               , N + 1) all terminating indices. Moreover, we define path(
                               <strong>
                                E
                               </strong>
                               ) to return all indices in
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               that can be accessed from 0, and N + 1, via recursive application of next and prev.
                              </p>
                              <p>
                               We distinguish between three types of POGs, the first two are determined directly from
                               <strong>
                                S
                               </strong>
                               , and the third by inference. All POGs share the node index
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , which allows character states to be mapped across extant sequences and ancestors (illustrated in
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Fig. 1
                               </a>
                               ).
                              </p>
                              <ul class="list-unord" id="list-2">
                               <li id="list-item-4">
                                <p id="p-91">
                                 An “extant POG”, is defined by a set of edges
                                 <strong>
                                  E
                                 </strong>
                                 <sup>
                                  (j)
                                 </sup>
                                 specific to an extant sequence S
                                 <sub>
                                  j
                                 </sub>
                                 , where
                                 <span class="inline-formula" id="inline-formula-22">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-22.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-22.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 . path(
                                 <strong>
                                  E
                                 </strong>
                                 <sup>
                                  (j)
                                 </sup>
                                 ) recovers the indices in
                                 <span class="inline-formula" id="inline-formula-23">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-23.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-23.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 ;
                                 <strong>
                                  E
                                 </strong>
                                 <sup>
                                  (j)
                                 </sup>
                                 forms a single path of “character” nodes X
                                 <sub>
                                  ji
                                 </sub>
                                 = S
                                 <sub>
                                  ji
                                 </sub>
                                 where
                                 <span class="inline-formula" id="inline-formula-24">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-24.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-24.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 .
                                </p>
                               </li>
                               <li id="list-item-5">
                                <p id="p-92">
                                 An “input POG”, denoted
                                 <span class="inline-formula" id="inline-formula-25">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-25.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-25.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 represents the joint set of edges collected from extant sequences. The presence of an edge between a and b is indicated by
                                 <strong>
                                  E
                                 </strong>
                                 *(a, b) > 0.
                                </p>
                               </li>
                               <li id="list-item-6">
                                <p id="p-93">
                                 An “ancestor POG”, is inferred to have a set of edges
                                 <strong>
                                  E
                                 </strong>
                                 <sup>
                                  (k)
                                 </sup>
                                 where
                                 <span class="inline-formula" id="inline-formula-26">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-26.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-26.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 . It links a series of nodes Y
                                 <sub>
                                  ki
                                 </sub>
                                 where
                                 <span class="inline-formula" id="inline-formula-27">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-27.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-27.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 ; each node either identifies a character state, or defines a probability distribution over character states. The latter is referred to as a “distribution” node. Once the POG for ancestor k is inferred, path(
                                 <strong>
                                  E
                                 </strong>
                                 <sup>
                                  (k)
                                 </sup>
                                 ) recovers its valid indices
                                 <span class="inline-formula" id="inline-formula-28">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-28.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-28.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 .
                                </p>
                               </li>
                              </ul>
                             </div>
                             <div id="sec-33" class="subsection">
                              <h4>
                               Inference of ancestral states, insertions, and deletions
                              </h4>
                              <p>
                               The phylogenetic tree with a nominated root and the collection of extant POGs serve as input to inference. GRASP supports two types of inference:
                              </p>
                              <ul class="list-unord" id="list-3">
                               <li id="list-item-7">
                                <p id="p-95">
                                 marginal reconstruction at a specified ancestral branch point in the phylogenetic tree; as a result of inference, the nominated ancestor POG will contain distribution nodes that represent the marginal distributions of character states.
                                </p>
                               </li>
                               <li id="list-item-8">
                                <p id="p-96">
                                 joint reconstruction of all ancestral branch points; all ancestor POGs will contain character nodes that represent the most probable character state.
                                </p>
                               </li>
                              </ul>
                              <div id="sec-34" class="subsection">
                               <h5>
                                Inferring insertions and deletions at ancestor branch points
                               </h5>
                               <p id="p-97">
                                POG
                                <strong>
                                 E
                                </strong>
                                <sup>
                                 (k)
                                </sup>
                                at an ancestor k defines all possible paths that can form a valid sequence and therefore determines if a character state needs to be inferred at any given position. This section describes how
                                <strong>
                                 E
                                </strong>
                                <sup>
                                 (k)
                                </sup>
                                is determined by using bi-directional edge encoding, and
                                <span class="inline-formula" id="inline-formula-29">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-29.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-29.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                by implication.
                               </p>
                               <p id="p-98">
                                GRASP considers all edges in
                                <strong>
                                 E
                                </strong>
                                * and seeks to jointly identify a set of edges across all branches in the tree by either maximum parsimony or maximum likelihood. To decompose this problem, GRASP scores edges leaving (δ = OUT) and edges entering (δ = IN) for a single position at a time.
                               </p>
                               <p id="p-99">
                                Maximum parsimony finds the edges at all ancestors that imply the smallest cost across the tree.
                                <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                                 Eq. 1
                                </a>
                                defines a score for each edge (between a and b, with direction δ) at a given ancestor k. The parsimony score of that edge depends on which edges are selected in its descendants: staying with the same edge is cost-free, whereas changing to any other edge will cost 1. Except for the base case when the descendant is an extant sequence, the costs from the descendants are propagated recursively, and (all) edge choices that ultimately lead to the best parsimony score at the top-most branch point are recorded. The edge with the best score (i′ in
                                <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">
                                 Eq. 1
                                </a>
                                , relative to either a or b depending on δ) is assigned a score of 1 to indicate it was most parsimonious in that direction:
                                <strong>
                                 E
                                </strong>
                                <sup>
                                 (k)
                                </sup>
                                (i, i′) = 1 for both δ =
                                <kbd>
                                 OUT
                                </kbd>
                                and δ =
                                <kbd>
                                 IN
                                </kbd>
                                , at any position
                                <span class="inline-formula" id="inline-formula-30">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-30.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-30.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . Note that δ =
                                <kbd>
                                 OUT
                                </kbd>
                                references only one half of the matrix (i < i′) and δ =
                                <kbd>
                                 IN
                                </kbd>
                                references the other (i > i′).
                                <span class="disp-formula" id="disp-formula-1">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-6.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-6.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-100">
                                For an ancestor k, an edge (a, b) is included if
                                <strong>
                                 E
                                </strong>
                                <sup>
                                 (k)
                                </sup>
                                (a, b) +
                                <strong>
                                 E
                                </strong>
                                <sup>
                                 (k)
                                </sup>
                                (b, a) > 0; if the sum is 2, it is bi-directionally parsimonious, which implies it is preferred when identifying ancestor sequences.
                               </p>
                               <p id="p-101">
                                For maximum likelihood, like parsimony, each edge competes twice: once among edges sharing the same source, once among those with the same target. Again like parsimony, this is framed as two inferences for each position i: a choice between all edges leading
                                <kbd>
                                 IN
                                </kbd>
                                to i, and a choice between all edges going
                                <kbd>
                                 OUT
                                </kbd>
                                from i. Given that the number of edges that are considered jointly varies, we construct a uniform evolutionary model with a congruent number of states. Inference determines the combination of states, across all ancestors, that has the greatest joint probability, following the approach described in the next subsection.
                               </p>
                               <p id="p-102">
                                Both maximum parsimony and maximum likelihood optimise the selection of edges that have, independently, the same source and the same target, so there is no guarantee that there is a complete path through the POG where all edges are bidirectionally supported; however, in practice this turns out to be mostly the case.
                               </p>
                              </div>
                              <div id="sec-35" class="subsection">
                               <h5>
                                Inferring the character state of ancestor nodes
                               </h5>
                               <p id="p-103">
                                For GRASP to infer character states and operate efficiently, we make several standard assumptions. First, we assume that each sequence position, i, can be modelled independently
                                <sup>
                                 <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">
                                  51
                                 </a>
                                </sup>
                                . Second, we assume that character substitutions depend only on the state of the immediate ancestor
                                <sup>
                                 <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">
                                  51
                                 </a>
                                </sup>
                                . Third, we assume that each position mutates at the same rate. Modelling variable rates across positions
                                <sup>
                                 <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                                  53
                                 </a>
                                 ,
                                 <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                                  54
                                 </a>
                                </sup>
                                compromises our ability to efficiently process large data sets and presently we opt not to do this. Instead, we leverage efficient procedures of graphical models for inference
                                <sup>
                                 <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                                  20
                                 </a>
                                </sup>
                                .
                               </p>
                               <p id="p-104">
                                The topology of the phylogenetic tree maps to a character tree for each position, subject to the position
                                <span class="inline-formula" id="inline-formula-31">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-31.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-31.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                in an ancestor k forming part of a valid sequence; for later, we define the transpose of that mapping as
                                <span class="inline-formula" id="inline-formula-32">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-32.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-32.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , i.e. the subset of ancestors that have character content for a position i.
                               </p>
                               <p id="p-105">
                                Each position-specific character tree maps to a directed Bayesian network, which is parameterised to reflect evolutionary distances (additively) at each branch, from the provided phylogenetic tree. The network is created with “observable” variables, instantiated to the characters in extant sequences X
                                <sub>
                                 ji
                                </sub>
                                = x. “Nonobservable” variables in the Bayesian network correspond to the ancestors Y
                                <sub>
                                 ki
                                </sub>
                                where
                                <span class="inline-formula" id="inline-formula-33">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-33.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-33.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                ; how the character state or distribution for Y
                                <sub>
                                 ki
                                </sub>
                                is inferred is described below.
                               </p>
                               <p id="p-106">
                                A Bayesian network node is a conditional probability P(X
                                <sub>
                                 ji
                                </sub>
                                |Y
                                <sub>
                                 κ(j)i
                                </sub>
                                ,d
                                <sub>
                                 j
                                </sub>
                                ) or P(Y
                                <sub>
                                 ki
                                </sub>
                                |Y
                                <sub>
                                 κ(k)i
                                </sub>
                                , d
                                <sub>
                                 k
                                </sub>
                                ), for
                                <span class="inline-formula" id="inline-formula-34">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-34.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-34.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="inline-formula" id="inline-formula-35">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-35.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-35.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and is parameterised by their respective distances (d
                                <sub>
                                 j
                                </sub>
                                or d
                                <sub>
                                 k
                                </sub>
                                ; which refer to their closest ancestor branch point, κ(j) or κ(k), respectively).
                               </p>
                               <p id="p-107">
                                The matrix of conditional probabilities is e
                                <sup>
                                 Q(d)
                                </sup>
                                where Q is the instantaneous rate matrix given by the evolutionary model. GRASP supports all popular models
                                <sup>
                                 <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                                  55
                                 </a>
                                 ,
                                 <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">
                                  41
                                 </a>
                                 ,
                                 <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                                  56
                                 </a>
                                 ,
                                 <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                                  36
                                 </a>
                                </sup>
                                . Inference of the joint ancestral character state at a position i is then defined by:
                                <span class="disp-formula" id="disp-formula-2">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-7.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-7.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                where T is the tree with distances for all branches. The implementation uses an adaptation of variable elimination
                                <sup>
                                 <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">
                                  20
                                 </a>
                                 ,
                                 <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                                  57
                                 </a>
                                </sup>
                                , which decomposes the inference into an efficient series of products, given the hierarchical topology of the tree. Ancestral states are determined by the highest joint probability across all non-observed variables (all ancestors, all positions). From the above, GRASP is also capable of inferring the marginal probability distribution for each position in a given ancestor, by summing out all other non-observed variables. All inferences are exact (not approximated). GRASP can make use of position-specific, relative evolutionary rates, which are estimated by external tools.
                               </p>
                               <p id="p-108">
                                Finally, GRASP facilitates an approach of exploring plausible alternative amino acids at sites with high Shannon entropy, by selecting residues that show a relatively high posterior probability in a marginal reconstruction
                                <sup>
                                 <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">
                                  8
                                 </a>
                                </sup>
                                . Mutations can be introduced at these positions to test the robustness of predictions and to create alternative ancestors. GRASP extends this method, providing the ability to prioritise mutations that best capture inferred probability distributions by minimising the expected relative entropy.
                               </p>
                              </div>
                             </div>
                             <div id="sec-36" class="subsection">
                              <h4>
                               Identifying a single, preferred ancestor sequence
                              </h4>
                              <p id="p-109">
                               Not uncommonly, multiple indel histories are equally parsimonious or the preferred edge in one direction is not preferred in the other, implying that several ancestor candidate sequences can be identified by traversing an ancestor POG; however, in some applications it is necessary to nominate a single sequence.
                              </p>
                              <p id="p-110">
                               To determine a “preferred” path through an ancestor POG, we first define a subset of extant sequences
                               <span class="inline-formula" id="inline-formula-36">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-36.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/inline-graphic-36.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               that are in the subtree under a given ancestor, k. To express preference between multiple edges, we calculate the proportion of extant sequences that contain a particular edge (see
                               <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                                Eq. 3
                               </a>
                               ).
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <div id="sec-37" class="subsection">
                               <h5>
                                Identifying the preferred path
                               </h5>
                               <p id="p-111">
                                Like a Markov chain, GRASP uses weights (
                                <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">
                                 Eq. 3
                                </a>
                                ) to represent transition probabilities and can use these to find the most probable path, enforcing a preference for bi-directional edges.
                               </p>
                               <p id="p-112">
                                As an alternate option, GRASP also implements the A* algorithm
                                <sup>
                                 <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                                  58
                                 </a>
                                </sup>
                                to determine the selection of edges in a POG that jointly minimises the cost, travelling from the N-to the C-terminus. Here, the cost assigned to an edge is given by
                                <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
                                 Eq. 4
                                </a>
                                .
                                <span class="disp-formula" id="disp-formula-4">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-9.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-9.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-113">
                                η is defined in
                                <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                                 Eq. 5
                                </a>
                                and imposes a preference for bi-directional edges; a unidirectional edge is only chosen in the absence of bi-directional edges to complete the traversal. The exception is the edge to the first node, and the edge from the last node, where bi-directionality is disregarded. The impact of the weight is normalised by the number of positions skipped by a given edge, b – a. This ensures that each complete ancestral sequence is scored evenly, regardless of the number of edges it takes to form.
                                <span class="disp-formula" id="disp-formula-5">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-10.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/01/07/2019.12.30.891457/embed/graphic-10.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </div>
                             </div>
                             <div id="sec-38" class="subsection">
                              <h4>
                               Access to tools and data
                              </h4>
                              <p id="p-114">
                               GRASP is freely accessible via a web server at
                               <a href="http://grasp.scmb.uq.edu.au">
                                http://grasp.scmb.uq.edu.au
                               </a>
                               . The online service allows users to upload their own data sets and predict ancestors. The results are presented to allow exploration of ancestral POGs and their states via an interactive phylogenetic tree. Numerous other functions are available including annotation of trees with taxonomy and user specified terms, inspection of probability distributions for the identification of mutations for alternative ancestors, and sharing of entire reconstructions. A tutorial, user guide, and several example reconstructions are also available from the web site.
                              </p>
                              <p id="p-115">
                               Data sets reported in the manuscript and a suite of tools to assist in the application of GRASP are available at
                               <a href="https://bodenlab.github.io/GRASP-suite">
                                https://bodenlab.github.io/GRASP-suite
                               </a>
                               . The bi-directional edge parsimony indel inference method is available through the web server and all indel inference methods are available through a separate command line version that lacks visualisation features.
                              </p>
                              <p id="p-116">
                               The implementation in Java and a web application are available from the same site. The software is available under the GNU General Public License v3.0.
                              </p>
                             </div>
                            </div>
                            <div id="sec-39" class="subsection">
                             <h3>
                              GRASP, FastML, and PAML comparison method
                             </h3>
                             <p id="p-117">
                              The following procedure was used to evaluate each of the tools: (1) the input multiple sequence alignment was randomly divided into G groups of alignments with approximately equal numbers of sequences, where G ∈ {5, 10, 20}; (2) for each sub-alignment, the input phylogenetic tree constructed from the full alignment was pruned to remove sequences not in the sub-alignment. To represent the same principal ancestor across all groups, as well as to maintain a valid tree, branches in the original phylogenetic tree with removed sequences were collapsed and branch distances added together; (3) sub-alignments and corresponding pruned trees were therefore pared-down representatives of the same family and used as input to each of the ASR tools; (4) the process was repeated until 20 ancestral sequences had been generated for each configuration, e.g., when G = 5 the process is repeated four times. This procedure therefore results in sequences that belong to multiple groups across replicates for G = 5 and G = 10.
                             </p>
                             <p id="p-118">
                              The JTT evolutionary rate model
                              <sup>
                               <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                              </sup>
                              was used for all inferences and variable rates were calculated from a discrete gamma distribution with eight categories. To remove confounding effects of different strategies for dealing with gaps, we removed any column that contained a deletion, leaving 455 and 242 columns in the KARI and CYP2 multiple sequence alignments, respectively.
                             </p>
                            </div>
                            <div id="sec-40" class="subsection">
                             <h3>
                              Indel inference comparison method
                             </h3>
                             <p id="p-119">
                              INDELible was used to generate indel histories that followed a Zipfian distribution with an alpha value of 1.7
                              <sup>
                               <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                                59
                               </a>
                              </sup>
                              , maximum indel length of 10, LG substitution model, with all trees set to have an evolutionary depth of 1 and an original ancestral sequence length of 300 amino across four indel rates (relative to substitution rate) of 0.001, 0.005, 0.01, and 0.03 across four taxon sizes of 100, 250, 500, and 750 extant sequences. Any entirely empty columns in the alignment of extants (i.e., where there was character content in an ancestor that was completely absent across all extant sequences) were removed from both the extant and ancestral alignments.
                             </p>
                             <p id="p-120">
                              The known alignments and trees were provided to GRASP and each of the indel inference methods used to predict a set of ancestral sequences. The ancestral alignments from each method as well as the true history as generated by INDELible were used to generate sets of indels defined by the specific branch and start and end position of the indel event.
                             </p>
                             <p id="p-121">
                              The unaligned extant sequences were then also aligned by MAFFT (FFT-NS-2) and the analysis re-performed with realigned alignments and known trees provided to GRASP.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-42">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-123">
                             Study design, MB, EMJG, GF; experimental design and work, EMJG, VS, DH, CMR, LS, SB, JC; software and algorithm development, MB, AM, MLL, GF, JZ, AE, BB, RN; supporting scientific advice, BR, BK, REST, RTB, LG, GS, JC, YG; manuscript, GF, MB, AM, CMR, EMJG, LS, SB, BK, RTB, LG, GS, BR, DH, VS.
                            </p>
                           </div>
                           <div class="section" id="sec-43">
                            <h2 class="">
                             Competing interests Statement
                            </h2>
                            <p id="p-124">
                             The authors declare that they have no competing financial interests.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-122">
                             We thank Broder Rühmann who helped prepare data for the manuscript. This work has been supported by the Australian Research Council (ARC) Discovery Project grants 210101802, 160100865 and 120101772. BK is an ARC Laureate Fellow (FL180100109).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-8">
                               Major changes, including a new section describing and benchmarking reconstruction of insertions and deletions
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-3">
                              <p id="p-9">
                               <a href="http://grasp.scmb.uq.edu.au">
                                http://grasp.scmb.uq.edu.au
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Land
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Insights from 20 years of bacterial genome sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Functional & Integrative Genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  141
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  161
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2BIntegrative%2BGenomics%26rft.volume%253D15%26rft.spage%253D141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.2" data-doi="10.1073/pnas.1720115115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lewin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Earth BioGenome Project: Sequencing life for the future of life
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4325
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4333
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1720115115%26rft_id%253Dinfo%253Apmid%252F29686065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzE3LzQzMjUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wNy8yMDE5LjEyLjMwLjg5MTQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gregory
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Marine DNA viral macro- and microdiversity from pole to pole
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  177
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1109
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1123
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hochberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. K. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thornton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Reconstructing ancient proteins to understand the causes of structure and function
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual Review of Biophysics
                                 </abbr>
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  247
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  269
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BBiophysics%26rft.volume%253D46%26rft.spage%253D247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Selberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. G. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaucher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liberles
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Ancestral sequence reconstruction: From chemical paleogenetics to maximum likelihood algorithms and beyond
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  89
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  157
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  164
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.volume%253D89%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.6" data-doi="10.1093/protein/gzv038">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bar-Rogovsky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Assessing the prediction fidelity of ancestral reconstruction by a library approach
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Engineering, Design & Selection
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  507
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  518
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%252C%2BDesign%2B%2526%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252Fgzv038%26rft_id%253Dinfo%253Apmid%252F26275856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/protein/gzv038&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26275856&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sugrue
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Scott
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jackson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Constrained evolution of a bispecific enzyme: Lessons for biocatalyst design
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Organic & Biomolecular Chemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  937
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  946
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DOrganic%2B%2526%2BBiomolecular%2BChemistry%26rft.volume%253D15%26rft.spage%253D937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gumulya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Engineering highly functional thermostable proteins using ancestral sequence reconstruction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Catalysis
                                 </abbr>
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  878
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  888
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCatalysis%26rft.volume%253D1%26rft.spage%253D878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trudeau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tawfik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Protein engineers turned evolutionists—the quest for the optimal starting point
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Opinion in Biotechnology
                                 </abbr>
                                 <span class="cit-vol">
                                  60
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  46
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  52
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.volume%253D60%26rft.spage%253D46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spence
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaczmarski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saunders
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jackson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Ancestral sequence reconstruction for protein engineers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Opinion in Structural Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  69
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  131
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  141
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BStructural%2BBiology%26rft.volume%253D69%26rft.spage%253D131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garcia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaçar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  How to resurrect ancestral proteins as proxies for ancient biogeochemistry
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Free Radical Biology and Medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  140
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  260
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  269
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFree%2BRadical%2BBiology%2Band%2BMedicine%26rft.volume%253D140%26rft.spage%253D260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.12" data-doi="10.1093/molbev/msy055">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vialle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tamuri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. U.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Alignment modulates ancestral sequence reconstruction accuracy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1783
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1797
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D35%26rft.spage%253D1783%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msy055&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aadland
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolaczkowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Alignment-integrated reconstruction of ancestral sequences improves accuracy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1549
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1565
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft.volume%253D12%26rft.spage%253D1549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.14" data-doi="10.1093/bioinformatics/18.3.452">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grasso
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharlow
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Multiple sequence alignment using partial order graphs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  452
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  464
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DLee%26rft.auinit1%253DC.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D452%26rft.epage%253D464%26rft.atitle%253DMultiple%2Bsequence%2Balignment%2Busing%2Bpartial%2Border%2Bgraphs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.3.452%26rft_id%253Dinfo%253Apmid%252F11934745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.3.452&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11934745&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000174708500011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Emond
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Accessing unexplored regions of sequence space in directed enzyme evolution via insertion/deletion mutagenesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3469
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D11%26rft.spage%253D3469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sützl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. M. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bodén
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haltrich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The GMC superfamily of oxidoreductases revisited: Analysis and evolution of fungal GMC oxidoreductases
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biotechnology for Biofuels
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  118
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Bfor%2BBiofuels%26rft.volume%253D12%26rft.spage%253D118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.17" data-doi="10.1007/s00239-010-9402-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kirischian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McArthur
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jesuthasan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krattenmacher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. Y.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Phylogenetic and functional analysis of the vertebrate cytochrome P450 2 family
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  72
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  56
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  71
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKirischian%26rft.auinit1%253DN.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D56%26rft.epage%253D71%26rft.atitle%253DPhylogenetic%2Band%2Bfunctional%2Banalysis%2Bof%2Bthe%2Bvertebrate%2Bcytochrome%2Bp450%2B2%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9402-7%26rft_id%253Dinfo%253Apmid%252F21116621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00239-010-9402-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21116621&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.18" data-doi="10.1074/jbc.RA117.001592">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Function and maturation of the Fe-S center in dihydroxyacid dehydratase from Arabidopsis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Biological Chemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  293
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4422
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4433
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.RA117.001592%26rft_id%253Dinfo%253Apmid%252F29425096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyOTMvMTIvNDQyMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzA3LzIwMTkuMTIuMzAuODkxNDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2019.12.30.891457v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baldi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brunak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Bioinformatics: The Machine Learning Approach
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  The MIT Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2019.12.30.891457v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Probabilistic Graphical Models
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  The MIT Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.21" data-doi="10.1093/molbev/msq081">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hanson-Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolaczkowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thornton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Robustness of ancestral sequence reconstruction to phylogenetic uncertainty
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1988
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1999
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq081%26rft_id%253Dinfo%253Apmid%252F20368266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msq081&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20368266&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000281184100003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.22" data-doi="10.1186/1471-2105-5-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edwards
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shields
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  GASP: Gapped Ancestral Sequence Prediction for proteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DButterfield%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DPyEvolve%253A%2Ba%2Btoolkit%2Bfor%2Bstatistical%2Bmodelling%2Bof%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-1%26rft_id%253Dinfo%253Apmid%252F14706121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14706121&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000223920500001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Musil
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  FireProtASR: A web server for fully automated ancestral sequence reconstruction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Briefings in Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  bbaa337
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BBioinformatics%26rft.volume%253D22%26rft.spage%253Dbbaa337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.24" data-doi="10.1080/10635159950173889">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simmons
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ochoterena
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Gaps as characters in sequence-based phylogenetic analyses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  369
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  381
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DSimmons%26rft.auinit1%253DM.%2BP.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D369%26rft.epage%253D381%26rft.atitle%253DGaps%2Bas%2Bcharacters%2Bin%2Bsequence-based%2Bphylogenetic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635159950173889%26rft_id%253Dinfo%253Apmid%252F12118412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1080/10635159950173889&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12118412&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000087494200009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.25" data-doi="10.1093/bioinformatics/18.8.1116">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pupko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pe’er
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hasegawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Graur
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A branch-and- bound algorithm for the inference of ancestral amino-acid sequences when the replacement rate varies among sites: Application to the evolution of five gene families
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1116
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1123
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DPupko%26rft.auinit1%253DT.%26rft.volume%253D18%26rft.issue%253D8%26rft.spage%253D1116%26rft.epage%253D1123%26rft.atitle%253DA%2Bbranch-and-bound%2Balgorithm%2Bfor%2Bthe%2Binference%2Bof%2Bancestral%2Bamino-acid%2Bsequences%2Bwhen%2Bthe%2Breplacement%2Brate%2Bvaries%2Bamong%2Bsites%253A%2BApplication%2Bto%2Bthe%2Bevolution%2Bof%2Bfive%2Bgene%2Bfamilies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.8.1116%26rft_id%253Dinfo%253Apmid%252F12176835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.8.1116&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12176835&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000177504400011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.26" data-doi="10.1007/BF02198858">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koshi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Probabilistic reconstruction of ancestral protein sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  313
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  320
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKoshi%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D320%26rft.atitle%253DProbabilistic%2Breconstruction%2Bof%2Bancestral%2Bprotein%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02198858%26rft_id%253Dinfo%253Apmid%252F8919883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/BF02198858&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8919883&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996UC69800029&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.27" data-doi="10.1093/nar/gks498">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashkenazy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  FastML: A web server for probabilistic reconstruction of ancestral sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  W580
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W584
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks498%26rft_id%253Dinfo%253Apmid%252F22661579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gks498&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22661579&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000306670900095&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.28" data-doi="10.1093/molbev/msm088">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  PAML 4: Phylogenetic analysis by maximum likelihood
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1586
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1591
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17483113&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000248848400003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.29" data-doi="10.1021/bi300694t">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Afriat-Jurnou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jackson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tawfik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Reconstructing a missing link in the evolution of a recently diverged phosphotriesterase by active-site loop remodeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  51
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  6047
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6055
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DAfriat-Jurnou%26rft.auinit1%253DL.%26rft.volume%253D51%26rft.issue%253D31%26rft.spage%253D6047%26rft.epage%253D6055%26rft.atitle%253DReconstructing%2Ba%2Bmissing%2Blink%2Bin%2Bthe%2Bevolution%2Bof%2Ba%2Brecently%2Bdiverged%2Bphosphotriesterase%2Bby%2Bactive-site%2Bloop%2Bremodeling.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi300694t%26rft_id%253Dinfo%253Apmid%252F22809311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/bi300694t&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22809311&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000307159400002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.30" data-doi="10.1093/bioinformatics/bth126">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grasso
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Combining partial order alignment and progressive multiple sequence alignment increases alignment speed and scalability to very large alignment problems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1546
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1556
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth126%26rft_id%253Dinfo%253Apmid%252F14962922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth126&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14962922&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000222402400013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.31" data-doi="10.1073/pnas.0409137102">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Löytynoja
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  An algorithm for progressive multiple alignment of sequences with insertions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 <span class="cit-vol">
                                  102
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  10557
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10562
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409137102%26rft_id%253Dinfo%253Apmid%252F16000407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzMwLzEwNTU3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMDcvMjAxOS4xMi4zMC44OTE0NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.32" data-doi="10.1093/molbev/mst010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katoh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Standley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  MAFFT multiple sequence alignment software version 7: Improvements in performance and usability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  772
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  780
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23329690&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000317002300004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.33" data-doi="10.1093/bioinformatics/btp348">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Capella-Gutiérrez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Silla-Martínez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gabaldón
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  trimAl: A tool for automated alignment trimming in large-scale phylogenetic analyses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1972
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1973
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp348&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19505945&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000268107100022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.34" data-doi="10.1080/10635150701472164">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Talavera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Castresana
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  56
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  564
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  577
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DTalavera%26rft.auinit1%253DG.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D564%26rft.epage%253D577%26rft.atitle%253DImprovement%2Bof%2Bphylogenies%2Bafter%2Bremoving%2Bdivergent%2Band%2Bambiguously%2Baligned%2Bblocks%2Bfrom%2Bprotein%2Bsequence%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701472164%26rft_id%253Dinfo%253Apmid%252F17654362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1080/10635150701472164&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17654362&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000248359900002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.35" data-doi="10.1093/sysbio/syq010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guindon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  New algorithms and methods to estimate maximumlikelihood phylogenies: Assessing the performance of PhyML 3.0
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  59
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  307
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  321
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20525638&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000276528300006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.36" data-doi="10.1093/molbev/msn067">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Le
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. Q.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gascuel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  An improved general amino acid replacement matrix
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1307
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1320
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn067%26rft_id%253Dinfo%253Apmid%252F18367465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msn067&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18367465&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000256979100007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.37" data-doi="10.1111/j.1742-4658.2008.06823.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spadiut
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Improving thermostability and catalytic activity of pyranose 2-oxidase from Trametes multicolor by rational and semi-rational design
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The FEBS Journal
                                 </abbr>
                                 <span class="cit-vol">
                                  276
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  776
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  792
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DSpadiut%26rft.auinit1%253DO.%26rft.volume%253D276%26rft.issue%253D3%26rft.spage%253D776%26rft.epage%253D792%26rft.atitle%253DImproving%2Bthermostability%2Band%2Bcatalytic%2Bactivity%2Bof%2Bpyranose%2B2-oxidase%2Bfrom%2BTrametes%2Bmulticolor%2Bby%2Brational%2Band%2Bsemi-rational%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2008.06823.x%26rft_id%253Dinfo%253Apmid%252F19143837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2008.06823.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19143837&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rahman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The crystal structure of a bacterial l-arabinonate dehydratase contains a [2Fe-2S] cluster
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ACS Chemical Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1919
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1927
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DACS%2BChemical%2BBiology%26rft.volume%253D12%26rft.spage%253D1919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sievers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Higgins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Clustal Omega for making accurate alignments of many protein sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Science: A Publication of the Protein Society
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  135
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  145
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%253A%2BA%2BPublication%2Bof%2Bthe%2BProtein%2BSociety%26rft.volume%253D27%26rft.spage%253D135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.40" data-doi="10.1093/molbev/msp077">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Price
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dehal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arkin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  FastTree: Computing large minimum evolution trees with profiles instead of a distance matrix
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1641
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1650
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp077%26rft_id%253Dinfo%253Apmid%252F19377059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msp077&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19377059&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000266966200020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.41" data-doi="10.1093/bioinformatics/8.3.275">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taylor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thornton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The rapid generation of mutation data matrices from protein sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Computer Applications in the Biosciences
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  275
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  282
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1992
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DJones%26rft.auinit1%253DD.%2BT.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D275%26rft.epage%253D282%26rft.atitle%253DThe%2Brapid%2Bgeneration%2Bof%2Bmutation%2Bdata%2Bmatrices%2Bfrom%2Bprotein%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F8.3.275%26rft_id%253Dinfo%253Apmid%252F1633570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/8.3.275&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=1633570&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.42" data-doi="10.1021/bi200178z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Geertsma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dutzler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A versatile and efficient high-throughput cloning tool for structural biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3272
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3278
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DBiochemistry%26rft.aulast%253DGeertsma%26rft.auinit1%253DE.%2BR.%26rft.volume%253D50%26rft.issue%253D15%26rft.spage%253D3272%26rft.epage%253D3278%26rft.atitle%253DA%2Bversatile%2Band%2Befficient%2Bhigh-throughput%2Bcloning%2Btool%2Bfor%2Bstructural%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi200178z%26rft_id%253Dinfo%253Apmid%252F21410291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/bi200178z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21410291&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000289319700023&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.43" data-doi="10.1016/j.pep.2005.01.016">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Studier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F. W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Protein production by auto-induction in high density shaking cultures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Protein Expression and Purification
                                 </abbr>
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  207
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  234
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProtein%2Bexpression%2Band%2Bpurification%26rft.stitle%253DProtein%2BExpr%2BPurif%26rft.aulast%253DStudier%26rft.auinit1%253DF.%2BW.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D234%26rft.atitle%253DProtein%2Bproduction%2Bby%2Bauto-induction%2Bin%2Bhigh%2Bdensity%2Bshaking%2Bcultures.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pep.2005.01.016%26rft_id%253Dinfo%253Apmid%252F15915565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.pep.2005.01.016&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15915565&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000228564900025&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.44" data-doi="10.1093/bioinformatics/btu033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stamatakis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  RAxML version 8: A tool for phylogenetic analysis and postanalysis of large phylogenies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1312
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1313
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24451623&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000336095100024&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.45" data-doi="10.1038/nature12027">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amemiya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Analysis of the African coelacanth genome sheds light on tetrapod evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  496
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  311
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  316
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAmemiya%26rft.auinit1%253DC.%2BT.%26rft.volume%253D496%26rft.issue%253D7445%26rft.spage%253D311%26rft.epage%253D316%26rft.atitle%253DThe%2BAfrican%2Bcoelacanth%2Bgenome%2Bprovides%2Binsights%2Binto%2Btetrapod%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12027%26rft_id%253Dinfo%253Apmid%252F23598338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12027&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2013044335&link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef">
                                 <span>
                                  GeoRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23598338&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000317599200027&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [46].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.46" data-doi="10.1006/abbi.1996.9859">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   von Wachenfeldt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richardson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cosme
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Microsomal P450 2C3 is expressed as a soluble dimer in Escherichia coli following modifications of its N-terminus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Archives of Biochemistry and Biophysics
                                 </abbr>
                                 <span class="cit-vol">
                                  339
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  107
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  114
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BBiochemistry%2Band%2BBiophysics%26rft.stitle%253DArchives%2Bof%2BBiochemistry%2Band%2BBiophysics%26rft.aulast%253Dvon%2BWachenfeldt%26rft.auinit1%253DC.%26rft.volume%253D339%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D114%26rft.atitle%253DMicrosomal%2BP450%2B2C3%2Bis%2Bexpressed%2Bas%2Ba%2Bsoluble%2Bdimer%2Bin%2BEscherichia%2Bcoli%2Bfollowing%2Bmodification%2Bof%2Bits%2BN-terminus.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fabbi.1996.9859%26rft_id%253Dinfo%253Apmid%252F9056240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1006/abbi.1996.9859&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9056240&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1997WL10900015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [47].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.47" data-doi="10.1074/jbc.M114.550004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Siller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Oxidation of endogenous N-arachidonoylserotonin by human cytochrome P450 2U1
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Biological Chemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  289
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  10476
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10487
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.550004%26rft_id%253Dinfo%253Apmid%252F24563460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvMTUvMTA0NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8wNy8yMDE5LjEyLjMwLjg5MTQ1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [48].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.48" data-doi="10.1093/nar/gks1473">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaspar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Santos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oliveira
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  mRNA secondary structure optimization using a correlated stem-loop prediction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e73
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1473%26rft_id%253Dinfo%253Apmid%252F23325845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gks1473&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23325845&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [49].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.49" data-doi="10.1177/1087057107312780">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnston
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Voss
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hayes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. M. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Quantitative whole-cell cytochrome P450 measurement suitable for high-throughput application
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Biomolecular Screening
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  135
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  141
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiomolecular%2BScreening%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1087057107312780%26rft_id%253Dinfo%253Apmid%252F18216397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1177/1087057107312780&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18216397&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [50].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.50" data-doi="10.1093/bioinformatics/btq003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  CD-HIT Suite: A web server for clustering and comparing biological sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  680
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  682
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq003%26rft_id%253Dinfo%253Apmid%252F20053844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20053844&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000274973800015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [51].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.51" data-doi="10.1007/BF01734359">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Felsenstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Evolutionary trees from DNA sequences: A maximum likelihood approach
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  368
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  376
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1981
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D368%26rft.epage%253D376%26rft.atitle%253DEvolutionary%2Btrees%2Bfrom%2BDNA%2Bsequences%253A%2Ba%2Bmaximum%2Blikelihood%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01734359%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/BF01734359&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7288891&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1981MG91100007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [52].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.52" data-doi="10.1093/bioinformatics/bts198">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Löytynoja
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vilella
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1684
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1691
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts198%26rft_id%253Dinfo%253Apmid%252F22531217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts198&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22531217&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000305825600003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [53].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.53" data-doi="10.1007/BF00160154">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: Approximate methods
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  306
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  314
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1994
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D306%26rft.epage%253D314%26rft.atitle%253DMaximum%2Blikelihood%2Bphylogenetic%2Bestimation%2Bfrom%2BDNA%2Bsequences%2Bwith%2Bvariable%2Brates%2Bover%2Bsites%253A%2Bapproximate%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00160154%26rft_id%253Dinfo%253Apmid%252F7932792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/BF00160154&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7932792&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1994PB53900008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [54].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.54" data-doi="10.1007/s002390010234">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Felsenstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Taking variation of evolutionary rates between sites into account in inferring phylogenies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  53
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  447
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  455
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D53%26rft.issue%253D4-5%26rft.spage%253D447%26rft.epage%253D455%26rft.atitle%253DTaking%2Bvariation%2Bof%2Bevolutionary%2Brates%2Bbetween%2Bsites%2Binto%2Baccount%2Bin%2Binferring%2Bphylogenies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002390010234%26rft_id%253Dinfo%253Apmid%252F11675604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s002390010234&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11675604&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000171169600020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [55].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2019.12.30.891457v3.55">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dayhoff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schwartz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Orcutt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  A model of evolutionary change in proteins
                                 </span>
                                 . In
                                 <span class="cit-source">
                                  Atlas of Protein Sequence and Structure
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  345
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  352
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  National Biomedical Research Foundation
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1978
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [56].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.56" data-doi="10.1093/oxfordjournals.molbev.a003851">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whelan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  691
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  699
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DWhelan%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D691%26rft.epage%253D699%26rft.atitle%253DA%2BGeneral%2BEmpirical%2BModel%2Bof%2BProtein%2BEvolution%2BDerived%2Bfrom%2BMultiple%2BProtein%2BFamilies%2BUsing%2Ba%2BMaximum-Likelihood%2BApproach%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003851%26rft_id%253Dinfo%253Apmid%252F11319253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003851&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11319253&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000168427400001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [57].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.57">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dechter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Bucket elimination: A unifying framework for reasoning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Artificial Intelligence
                                 </abbr>
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  41
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  85
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DArtificial%2BIntelligence%26rft.volume%253D113%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [58].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.58" data-doi="10.1109/TSSC.1968.300136">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nilsson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raphael
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A formal basis for the heuristic determination of minimum cost paths
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Transactions on Systems Science and Cybernetics
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  100
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  107
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1968
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTransactions%2Bon%2BSystems%2BScience%2Band%2BCybernetics%26rft.volume%253D4%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTSSC.1968.300136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/TSSC.1968.300136&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [59].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2019.12.30.891457v3.59" data-doi="10.1006/jmbi.1993.1105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Benner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gonnet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Empirical and structural models for insertions and deletions in the divergent evolution of proteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  229
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1065
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1082
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1993
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBenner%26rft.auinit1%253DS.%2BA.%26rft.volume%253D229%26rft.issue%253D4%26rft.spage%253D1065%26rft.epage%253D1082%26rft.atitle%253DEmpirical%2Band%2Bstructural%2Bmodels%2Bfor%2Binsertions%2Band%2Bdeletions%2Bin%2Bthe%2Bdivergent%2Bevolution%2Bof%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1993.1105%26rft_id%253Dinfo%253Apmid%252F8445636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1006/jmbi.1993.1105&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8445636&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F07%2F2019.12.30.891457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1993KP93900020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.30.891457v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.01.04.474895v2" title="Comparative whole plastome and low copy number phylogenetics of the core Saccharinae and Sorghinae" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.12.26.474221v3" title="Dose-response relationship of pulmonary disorders by inhalation exposure to cross-linked water-soluble acrylic acid polymers in F344 rats" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted January 07, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2019.12.30.891457v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2019.12.30.891457v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/01/07/2019.12.30.891457.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2019.12.30.891457v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2019.12.30.891457v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2340580"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-BCYfMgi6E0dXh78y_oeu8EnBRh1mn4QczbJPXu7ah60"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484466158"/>
                                   <input type="hidden" name="captcha_token" value="2fc132da3814d51388aeaf5b0f38a60d"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2340580--3" data-node-nid="2340580" id="node-2340580--41744919005" data-pisa="biorxiv;2019.12.30.891457v3" data-pisa-master="biorxiv;2019.12.30.891457" data-seqnum="2340580" data-apath="/biorxiv/early/2022/01/07/2019.12.30.891457.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Gabriel
                                  </span>
                                  <span class="nlm-surname">
                                   Foley
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Ariane
                                  </span>
                                  <span class="nlm-surname">
                                   Mora
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Connie M.
                                  </span>
                                  <span class="nlm-surname">
                                   Ross
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Scott
                                  </span>
                                  <span class="nlm-surname">
                                   Bottoms
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Leander
                                  </span>
                                  <span class="nlm-surname">
                                   Sützl
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Marnie L.
                                  </span>
                                  <span class="nlm-surname">
                                   Lamprecht
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Julian
                                  </span>
                                  <span class="nlm-surname">
                                   Zaugg
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Alexandra
                                  </span>
                                  <span class="nlm-surname">
                                   Essebier
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Brad
                                  </span>
                                  <span class="nlm-surname">
                                   Balderson
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Rhys
                                  </span>
                                  <span class="nlm-surname">
                                   Newell
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">
                                   Raine E. S.
                                  </span>
                                  <span class="nlm-surname">
                                   Thomson
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="11">
                                  <span class="nlm-given-names">
                                   Bostjan
                                  </span>
                                  <span class="nlm-surname">
                                   Kobe
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="12">
                                  <span class="nlm-given-names">
                                   Ross T.
                                  </span>
                                  <span class="nlm-surname">
                                   Barnard
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="13">
                                  <span class="nlm-given-names">
                                   Luke
                                  </span>
                                  <span class="nlm-surname">
                                   Guddat
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="14">
                                  <span class="nlm-given-names">
                                   Gerhard
                                  </span>
                                  <span class="nlm-surname">
                                   Schenk
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="15">
                                  <span class="nlm-given-names">
                                   Jörg
                                  </span>
                                  <span class="nlm-surname">
                                   Carsten
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="16">
                                  <span class="nlm-given-names">
                                   Yosephine
                                  </span>
                                  <span class="nlm-surname">
                                   Gumulya
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="17">
                                  <span class="nlm-given-names">
                                   Burkhard
                                  </span>
                                  <span class="nlm-surname">
                                   Rost
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="18">
                                  <span class="nlm-given-names">
                                   Dietmar
                                  </span>
                                  <span class="nlm-surname">
                                   Haltrich
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="19">
                                  <span class="nlm-given-names">
                                   Volker
                                  </span>
                                  <span class="nlm-surname">
                                   Sieber
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="20">
                                  <span class="nlm-given-names">
                                   Elizabeth M. J.
                                  </span>
                                  <span class="nlm-surname">
                                   Gillam
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="21">
                                  <span class="nlm-given-names">
                                   Mikael
                                  </span>
                                  <span class="nlm-surname">
                                   Bodén
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2019.12.30.891457;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2019.12.30.891457
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.30.891457v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&title=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&title=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&text=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&t=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&title=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&title=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&title=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2340580--5" data-node-nid="2340580" id="citation-node-2340580--61236921140" data-pisa="biorxiv;2019.12.30.891457v3" data-pisa-master="biorxiv;2019.12.30.891457" data-seqnum="2340580" data-apath="/biorxiv/early/2022/01/07/2019.12.30.891457.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Engineering indel and substitution variants of diverse and ancient enzymes using Graphical Representation of Ancestral Sequence Predictions (GRASP)
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Gabriel
                                      </span>
                                      <span class="nlm-surname">
                                       Foley
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Ariane
                                      </span>
                                      <span class="nlm-surname">
                                       Mora
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Connie M.
                                      </span>
                                      <span class="nlm-surname">
                                       Ross
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Scott
                                      </span>
                                      <span class="nlm-surname">
                                       Bottoms
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Leander
                                      </span>
                                      <span class="nlm-surname">
                                       Sützl
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Marnie L.
                                      </span>
                                      <span class="nlm-surname">
                                       Lamprecht
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Julian
                                      </span>
                                      <span class="nlm-surname">
                                       Zaugg
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Alexandra
                                      </span>
                                      <span class="nlm-surname">
                                       Essebier
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Brad
                                      </span>
                                      <span class="nlm-surname">
                                       Balderson
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Rhys
                                      </span>
                                      <span class="nlm-surname">
                                       Newell
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">
                                       Raine E. S.
                                      </span>
                                      <span class="nlm-surname">
                                       Thomson
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="11">
                                      <span class="nlm-given-names">
                                       Bostjan
                                      </span>
                                      <span class="nlm-surname">
                                       Kobe
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="12">
                                      <span class="nlm-given-names">
                                       Ross T.
                                      </span>
                                      <span class="nlm-surname">
                                       Barnard
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="13">
                                      <span class="nlm-given-names">
                                       Luke
                                      </span>
                                      <span class="nlm-surname">
                                       Guddat
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="14">
                                      <span class="nlm-given-names">
                                       Gerhard
                                      </span>
                                      <span class="nlm-surname">
                                       Schenk
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="15">
                                      <span class="nlm-given-names">
                                       Jörg
                                      </span>
                                      <span class="nlm-surname">
                                       Carsten
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="16">
                                      <span class="nlm-given-names">
                                       Yosephine
                                      </span>
                                      <span class="nlm-surname">
                                       Gumulya
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="17">
                                      <span class="nlm-given-names">
                                       Burkhard
                                      </span>
                                      <span class="nlm-surname">
                                       Rost
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="18">
                                      <span class="nlm-given-names">
                                       Dietmar
                                      </span>
                                      <span class="nlm-surname">
                                       Haltrich
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="19">
                                      <span class="nlm-given-names">
                                       Volker
                                      </span>
                                      <span class="nlm-surname">
                                       Sieber
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="20">
                                      <span class="nlm-given-names">
                                       Elizabeth M. J.
                                      </span>
                                      <span class="nlm-surname">
                                       Gillam
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="21">
                                      <span class="nlm-given-names">
                                       Mikael
                                      </span>
                                      <span class="nlm-surname">
                                       Bodén
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2019.12.30.891457;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2019.12.30.891457
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Foley%2C+Gabriel&rft.au=Mora%2C+Ariane&rft.au=Ross%2C+Connie+M.&rft.au=Bottoms%2C+Scott&rft.au=S%C3%BCtzl%2C+Leander&rft.au=Lamprecht%2C+Marnie+L.&rft.au=Zaugg%2C+Julian&rft.au=Essebier%2C+Alexandra&rft.au=Balderson%2C+Brad&rft.au=Newell%2C+Rhys&rft.au=Thomson%2C+Raine+E.+S.&rft.au=Kobe%2C+Bostjan&rft.au=Barnard%2C+Ross+T.&rft.au=Guddat%2C+Luke&rft.au=Schenk%2C+Gerhard&rft.au=Carsten%2C+J%C3%B6rg&rft.au=Gumulya%2C+Yosephine&rft.au=Rost%2C+Burkhard&rft.au=Haltrich%2C+Dietmar&rft.au=Sieber%2C+Volker&rft.au=Gillam%2C+Elizabeth+M.+J.&rft.au=Bod%C3%A9n%2C+Mikael">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2340580/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2340580/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2340580/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2340580/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2340580/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2340580/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2340580/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2340580/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2340580/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2340580/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2340580/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2340580/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&count=horizontal&via=&text=Engineering%20indel%20and%20substitution%20variants%20of%20diverse%20and%20ancient%20enzymes%20using%20Graphical%20Representation%20of%20Ancestral%20Sequence%20Predictions%20%28GRASP%29&counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.30.891457v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2019.12.30.891457v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
