<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.09.17.301788v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.09.17.301788v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.09.17.301788v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/06/26/2020.09.17.301788.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.09.17.301788v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A framework to efficiently smooth L1 penalties for linear regression"/>
  <meta name="DC.Identifier" content="10.1101/2020.09.17.301788"/>
  <meta name="DC.Date" content="2022-06-26"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Penalized linear regression approaches that include an L 1 term have become an important tool in statistical data analysis. One prominent example is the least absolute shrinkage and selection operator (Lasso), though the class of L 1 penalized regression operators also includes the fused and graphical Lasso, the elastic net, etc. Although the L 1 penalty makes their objective function convex, it is not differentiable everywhere, motivating the development of proximal gradient algorithms such as Fista, the current gold standard in the literature. In this work, we take a different approach based on smoothing in a fixed parameter setting (the problem size n and number of parameters p are fixed). The methodological contribution of our article is threefold: (1) We introduce a unified framework to compute closed-form smooth surrogates of a whole class of L 1 penalized regression problems using Nesterov smoothing. The surrogates preserve the convexity of the original (unsmoothed) objective functions, are uniformly close to them, and have closed-form derivatives everywhere for efficient minimization via gradient descent; (2) We prove that the estimates obtained with the smooth surrogates can be made arbitrarily close to the ones of the original (unsmoothed) objective functions, and provide explicitly computable a priori error bounds on the accuracy of our estimates; (3) We propose an iterative algorithm to progressively smooth the L 1 penalty which increases accuracy and is virtually free of tuning parameters. The proposed methodology is applicable to a large class of L 1 penalized regression operators, including all the operators mentioned above. Although the resulting estimates are typically dense, sparseness can be enforced again via thresholding. Using simulation studies, we compare our framework to current gold standards such as Fista, glmnet, gLasso, etc. Our results suggest that our proposed smoothing framework provides predictions of equal or higher accuracy than the gold standards while keeping the aforementioned theoretical guarantees and having roughly the same asymptotic runtime scaling.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Georg Hahn"/>
  <meta name="DC.Contributor" content="Sharon M. Lutz"/>
  <meta name="DC.Contributor" content="Nilanjana Laha"/>
  <meta name="DC.Contributor" content="Christoph Lange"/>
  <meta name="article:published_time" content="2022-06-26"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A framework to efficiently smooth L1 penalties for linear regression"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Penalized linear regression approaches that include an <i>L</i><sub>1</sub> term have become an important tool in statistical data analysis. One prominent example is the least absolute shrinkage and selection operator (Lasso), though the class of <i>L</i><sub>1</sub> penalized regression operators also includes the fused and graphical Lasso, the elastic net, etc. Although the <i>L</i><sub>1</sub> penalty makes their objective function convex, it is not differentiable everywhere, motivating the development of proximal gradient algorithms such as Fista, the current gold standard in the literature. In this work, we take a different approach based on smoothing in a fixed parameter setting (the problem size <i>n</i> and number of parameters <i>p</i> are fixed). The methodological contribution of our article is threefold: (1) We introduce a unified framework to compute closed-form smooth surrogates of a whole class of <i>L</i><sub>1</sub> penalized regression problems using Nesterov smoothing. The surrogates preserve the convexity of the original (unsmoothed) objective functions, are uniformly close to them, and have closed-form derivatives everywhere for efficient minimization via gradient descent; (2) We prove that the estimates obtained with the smooth surrogates can be made arbitrarily close to the ones of the original (unsmoothed) objective functions, and provide explicitly computable a priori error bounds on the accuracy of our estimates; (3) We propose an iterative algorithm to progressively smooth the <i>L</i><sub>1</sub> penalty which increases accuracy and is virtually free of tuning parameters. The proposed methodology is applicable to a large class of <i>L</i><sub>1</sub> penalized regression operators, including all the operators mentioned above. Although the resulting estimates are typically dense, sparseness can be enforced again via thresholding. Using simulation studies, we compare our framework to current gold standards such as Fista, glmnet, gLasso, etc. Our results suggest that our proposed smoothing framework provides predictions of equal or higher accuracy than the gold standards while keeping the aforementioned theoretical guarantees and having roughly the same asymptotic runtime scaling.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.09.17.301788v3"/>
  <meta name="citation_id" content="2020.09.17.301788v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.09.17.301788v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.17.301788v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.17.301788v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.09.17.301788"/>
  <meta name="citation_num_pages" content="36"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.09.17.301788"/>
  <meta name="citation_author" content="Georg Hahn"/>
  <meta name="citation_author_institution" content="Department of Biostatistics, T.H. Chan School of Public Health, Harvard University"/>
  <meta name="citation_author_email" content="ghahn@hsph.harvard.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6008-2720"/>
  <meta name="citation_author" content="Sharon M. Lutz"/>
  <meta name="citation_author_institution" content="Harvard T.H. Chan School of Public Health"/>
  <meta name="citation_author" content="Nilanjana Laha"/>
  <meta name="citation_author_institution" content="Harvard T.H. Chan School of Public Health"/>
  <meta name="citation_author" content="Christoph Lange"/>
  <meta name="citation_author_institution" content="Harvard T.H. Chan School of Public Health"/>
  <meta name="citation_reference" content="Arnold, T. B. & Tibshirani, R. J. (2020). genlasso: Path Algorithm for Generalized Lasso Problems. R-package version 1.5: https://cran.r-project.org/package=genlasso."/>
  <meta name="citation_reference" content="Banerjee, O., Ghaoui, L. E. & d’Aspremont, A. (2008). Model Selection Through Sparse Maximum Likelihood Estimation for Multivariate Gaussian or Binary Data. Journal of Machine Learning Research 9, 485–516."/>
  <meta name="citation_reference" content="Beck, A. & Teboulle, M. (2009). A Fast Iterative Shrinkage-Thresholding Algorithm for Linear Inverse Problems. SIAM J Imaging Sciences 2, 183–202."/>
  <meta name="citation_reference" content="Beck, A. & Teboulle, M. (2012). Smoothing And First Order Methods: A Unified Framework. Siam J Optim 22, 557–580."/>
  <meta name="citation_reference" content="Becker, S. R. & Candès, E. J. (2011). Templates for convex cone problems with applications to sparse signal recovery. Math Prog Comp 3, 165–218."/>
  <meta name="citation_reference" content="Belloni, A., Chernozhukov, V. & Wang, L. (2011). Square-root lasso: pivotal recovery of sparse signals via conic programming. Biometrika 98, 791–806."/>
  <meta name="citation_reference" content="Chen, X., Lin, Q., Kim, S., Carbonell, J. G. & Xing, E. P. (2010b). An efficient proximal gradient method for general structured sparse learning. Journal of Machine Learning Research 11."/>
  <meta name="citation_reference" content="Chen, X., Lin, Q., Kim, S., Carbonell, J. G. & Xing, E. P. (2012). Smoothing proximal gradient method for general structured sparse regression. Ann Appl Stat 6, 719–752."/>
  <meta name="citation_reference" content="Chi, E., Goldstein, T., Studer, C. & Baraniuk, R. (2018). fasta: Fast Adaptive Shrinkage/Thresholding Algorithm. R-package version 0.1.0: https://cran.r-project.org/package=fasta."/>
  <meta name="citation_reference" content="Daubechies, I., Defrise, M. & Mol, C. (2004). An iterative thresholding algorithm for linear inverse problems with a sparsity constraint. Comm. Pure Appl. Math. 57, 1413–1457."/>
  <meta name="citation_reference" content="Dondelinger, F. & Mukherjee, S. (2020). The joint lasso: high-dimensional regression for group structured data. Biostatistics 21, 219–235."/>
  <meta name="citation_reference" content="Efron, B., Hastie, T., Johnstone, I. & Tibshirani, R. (2004). Least angle regression. Ann Stat 32, 407–499."/>
  <meta name="citation_reference" content="Fan, J. & Li, R. (2001). Variable Selection via Nonconcave Penalized Likelihood and its Oracle Properties. J Am Stat Assoc 96, 1348–1360."/>
  <meta name="citation_reference" content="Friedman, J., Hastie, T. & Tibshirani, R. (2008). Sparse inverse covariance estimation with the graphical lasso. Biostatistics 9, 432–441."/>
  <meta name="citation_reference" content="Friedman, J., Hastie, T. & Tibshirani, R. (2010a). Regularization Paths for Generalized Linear Models via Coordinate Descent. Journal of Statistical Software 33, 1–22."/>
  <meta name="citation_reference" content="Friedman, J., Hastie, T. & Tibshirani, R. (2010b). Regularization Paths for Generalized Linear Modelsvia Coordinate Descent. J Stat Softw 33, 1–22."/>
  <meta name="citation_reference" content="Friedman, J., Hastie, T. & Tibshirani, R. (2019). glasso: Graphical Lasso: Estimation of Gaussian Graphical Models. R-package version 1.11: https://cran.r-project.org/package=glasso."/>
  <meta name="citation_reference" content="Friedman, J., Hastie, T., Tibshirani, R., Narasimhan, B., Tay, K., Simon, N. & Qian, J. (2020). glmnet: Lasso and Elastic-Net Regularized Generalized Linear Models. R-package version 4.0: https://cran.r-project.org/package=glmnet."/>
  <meta name="citation_reference" content="Hahn, G., Banerjee, M. & Sen, B. (2017). Parameter Estimation and Inference in a Contin-uous Piecewise Linear Regression Model. http://www.cantab.net/users/ghahn/preprints/PhaseRegMultiDim.pdf."/>
  <meta name="citation_reference" content="Hahn, G., Lutz, S., Laha, N., Cho, M., Silverman, E. & Lange, C. (2021). A fast and efficient smoothing approach to LASSO regression and an application in statistical genetics: polygenic risk scores for Chronic obstructive pulmonary disease (COPD). Stat Comput 31, 1–11."/>
  <meta name="citation_reference" content="Hahn, G., Lutz, S. M., Laha, N. & Lange, C. (2020). smoothedLasso: Smoothed LASSO Regres-sion via Nesterov Smoothing. R-package version 1.4: https://cran.r-project.org/package=smoothedLasso."/>
  <meta name="citation_reference" content="Hastie, T. & Efron, B. (2013). lars: Least Angle Regression, Lasso and Forward Stagewise. R-package version 1.2: https://cran.r-project.org/package=lars."/>
  <meta name="citation_reference" content="Massias, M., Fercoq, O., Gramfort, A. & Salmon, J. (2018). Generalized Concomitant Multi-Task Lasso for Sparse Multimodal Regression. In Proceedings of the 21st International Conference on Artificial Intelligence and Statistics (AISTATS) 2018, Lanzarote, Spain, vol. 84, PMLR."/>
  <meta name="citation_reference" content="Mazumder, R., Choudhury, A., Iyengar, G. & Sen, B. (2019). A Computational Framework for Multivariate Convex Regression and Its Variants. J Am Stat Assoc 114, 318–331."/>
  <meta name="citation_reference" content="Michelot, C. (1986). A finite algorithm for finding the projection of a point onto the canonical simplex of Rn. J Optimiz Theory App 50, 195–200."/>
  <meta name="citation_reference" content="Ndiaye, E., Fercoq, O., Gramfort, A., Leclère, V. & Salmon, J. (2017). Efficient Smoothed Con-comitant Lasso Estimation for High Dimensional Regression. In 7th International Conference on New Computational Methods for Inverse Problems."/>
  <meta name="citation_reference" content="Nesterov, Y. (1983). A method of solving a convex programming problem with convergence rate O(1/k2). Dokl Akad Nauk SSSR 269, 543–547."/>
  <meta name="citation_reference" content="Nesterov, Y. (2005). Smooth minimization of non-smooth functions. Math. Program. Ser. A 103, 127–152."/>
  <meta name="citation_reference" content="R Core Team (2014). R: A Language and Environment for Statistical Computing. R Foundation for Stat Comp, Vienna, Austria."/>
  <meta name="citation_reference" content="Seijo, E. & Sen, B. (2011). A continuous mapping theorem for the smallest argmax functional. Electron J Stat 5, 421–439."/>
  <meta name="citation_reference" content="Tibshirani, R. (1996). Regression Shrinkage and Selection Via the Lasso. J Roy Stat Soc B Met 58, 267–288."/>
  <meta name="citation_reference" content="Tibshirani, R., Saunders, M., Rosset, S., Zhu, J. & Knight, K. (2005). Sparsity and Smoothness via the Fused Lasso. J Roy Stat Soc B Met 67, 91–108."/>
  <meta name="citation_reference" content="Yuan, M. & Lin, Y. (2005). Model selection and estimation in regression with grouped variables. J Roy Stat Soc B Met 68, 49–67."/>
  <meta name="citation_reference" content="Zou, H. & Hastie, T. (2005). Regularization and variable selection via the elastic net. J Roy Stat Soc B Met 67, 301–320."/>
  <meta name="twitter:title" content="A framework to efficiently smooth L1 penalties for linear regression"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Penalized linear regression approaches that include an L 1 term have become an important tool in statistical data analysis. One prominent example is the least absolute shrinkage and selection operator (Lasso), though the class of L 1 penalized regression operators also includes the fused and graphical Lasso, the elastic net, etc. Although the L 1 penalty makes their objective function convex, it is not differentiable everywhere, motivating the development of proximal gradient algorithms such as Fista, the current gold standard in the literature. In this work, we take a different approach based on smoothing in a fixed parameter setting (the problem size n and number of parameters p are fixed). The methodological contribution of our article is threefold: (1) We introduce a unified framework to compute closed-form smooth surrogates of a whole class of L 1 penalized regression problems using Nesterov smoothing. The surrogates preserve the convexity of the original (unsmoothed) objective functions, are uniformly close to them, and have closed-form derivatives everywhere for efficient minimization via gradient descent; (2) We prove that the estimates obtained with the smooth surrogates can be made arbitrarily close to the ones of the original (unsmoothed) objective functions, and provide explicitly computable a priori error bounds on the accuracy of our estimates; (3) We propose an iterative algorithm to progressively smooth the L 1 penalty which increases accuracy and is virtually free of tuning parameters. The proposed methodology is applicable to a large class of L 1 penalized regression operators, including all the operators mentioned above. Although the resulting estimates are typically dense, sparseness can be enforced again via thresholding. Using simulation studies, we compare our framework to current gold standards such as Fista, glmnet, gLasso, etc. Our results suggest that our proposed smoothing framework provides predictions of equal or higher accuracy than the gold standards while keeping the aforementioned theoretical guarantees and having roughly the same asymptotic runtime scaling.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="A framework to efficiently smooth L1 penalties for linear regression"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.09.17.301788v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Penalized linear regression approaches that include an L 1 term have become an important tool in statistical data analysis. One prominent example is the least absolute shrinkage and selection operator (Lasso), though the class of L 1 penalized regression operators also includes the fused and graphical Lasso, the elastic net, etc. Although the L 1 penalty makes their objective function convex, it is not differentiable everywhere, motivating the development of proximal gradient algorithms such as Fista, the current gold standard in the literature. In this work, we take a different approach based on smoothing in a fixed parameter setting (the problem size n and number of parameters p are fixed). The methodological contribution of our article is threefold: (1) We introduce a unified framework to compute closed-form smooth surrogates of a whole class of L 1 penalized regression problems using Nesterov smoothing. The surrogates preserve the convexity of the original (unsmoothed) objective functions, are uniformly close to them, and have closed-form derivatives everywhere for efficient minimization via gradient descent; (2) We prove that the estimates obtained with the smooth surrogates can be made arbitrarily close to the ones of the original (unsmoothed) objective functions, and provide explicitly computable a priori error bounds on the accuracy of our estimates; (3) We propose an iterative algorithm to progressively smooth the L 1 penalty which increases accuracy and is virtually free of tuning parameters. The proposed methodology is applicable to a large class of L 1 penalized regression operators, including all the operators mentioned above. Although the resulting estimates are typically dense, sparseness can be enforced again via thresholding. Using simulation studies, we compare our framework to current gold standards such as Fista, glmnet, gLasso, etc. Our results suggest that our proposed smoothing framework provides predictions of equal or higher accuracy than the gold standards while keeping the aforementioned theoretical guarantees and having roughly the same asymptotic runtime scaling.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-06-26"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.09.17.301788v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2613581"/>
  <title>
   A framework to efficiently smooth L1 penalties for linear regression | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FCoFcAdp21HwcAkwglSmUk7Mt6mBbvSMo98H5x1BdSs"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.09.17.301788v3"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.09.17.301788v3"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.09.17.301788v3"}],"nid":"2613581","apath":"\/biorxiv\/early\/2022\/06\/26\/2020.09.17.301788.atom","pisa":"biorxiv;2020.09.17.301788v3","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.09.17.301788v3","title":"A framework to efficiently smooth L1 penalties for linear regression","identifier":"node\/2613581"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R6k4BBwPq1h6dW10BxaT+Q+E2BHdyeLdtwy50y76iUDWeDWTgCwx9wrVmdQB2pWXOnIq\\\\\\\/UbXMt9VY1lvvJu88XzXKHTNAcZsgH4OFYArYx8WnNbHnxSuQ8BskdhqZPB0UbhtYQQengOn8GA1hR8JwcuAbyd8UIDnCPmZ+NHj0cTXRVBY7nuMBqaGVDQcMUvLYg4UGiep\\\\\\\/z2rOWAVmFeHGKIJVCVDRTCTov1nNe\\\\\\\/E0UUOKKP6QUCcOiv4TX4dedbKXwz+6nFpoRPg7P046PkdP53\\\\\\\/MaW4KEAL0xCsIv7S59LHv00wwff2ubW8onOJNzVPLKQDYMxnLtDRuQR8dVdnXH5gq0areJg\\\\\\\/m8QC3bedkHTAVTA7Uqbqrg9ac47uL26\\\\\\\/h2H5bjK+hhGbrrxFGxp6B3I3vXRV7mjF6jEG0lneMgJ9Mdi2+HIAQlqAZcYW+QO3H8cseP0FQGF4I38bMhH8MyCwtVKDagvOrpyn9eD74O58Gas8WkxXd7\\\\\\\/uP5wpDVIiYE88BKfViRtBtaIi54bCqPDRIHMrrwfbQCE3TDNhGfhZP\\\\\\\/SuRT5VgHPVNICls8XCXM7j6v1VbYooqpEsMtqqm6XEKhmKN4T59dmIoB4Hd5HxNVQnJ5pe7xkwK\\\\\\\/7HqazlaH5lN3DW4HPKUkPkU1vctPom\\\\\\\/a6dbuQjxomEKXXBNRaqgDppI\\\\\\\/DIf03yzh3QABvsa\\\\\\\/7Xb00xSNViwB8fs2AwMARlfauMmrN6NSVb+lj+aVPKJUlacexuAeqoseG3dql0bh7ByQCKKbtTir1GB4606Lq06zPErSHev1zLMgZ8f+jLBYCq3EUGnWkmBdUN+ee644uRtke2SquT+b24SEKHaBcOCCKc7pJaRp+lZYyUmIEXOZC6\\\\\\\/xpaiphGinCES0fUYq4qU7ruMMLBaZBtD2m+0LxvT16NQTF2uEIKdbZ+A\\\\\\\/9lTY\\\\\\\/iWxmYAhKKucM0BeeKiGExutMw2LXaVdEpgBU5VzXGDfoghOxO674pnHfvC23Okx8nNqow1t\\\\\\\/+vjkJ080dv20XTy7B5yugQYOOxSflRX279D5DnFhZV4c3qzNAefScKb77tyeLUm+H7m+WCwczw7auFJQvFRUzuQzQzl\\\\\\\/apGkZf6UYnugApnmC\\\\\\\/oIf5SAOpevE9C2y3BJNYmPhIHHq6+4bhqXgp36jpmUKykwrurKX\\\\\\\/\\\\\\\/nTptXuBKLcTi6o+wQGaA2ge6hBQ9dYnqb4Mcy+7Rl4ZnYI6pCTGmlmcHgDfT\\\\\\\/I1yzF0qWmBpgtVb21A2Od6Z6tGvYzNH5rG+++GfPfBdATPlhH\\\\\\\/ccD321\\\\\\\/VGIqL6+4DLtlrsHwb0AW83yYd2On7dUR3RhD6OesMU3yPMkJzUwSbPXnbyhavebfg7eqJgAxUq\\\\\\\/jZ+hxbIrbhyE204M=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZvlyNkFvrBZRw7DIh6+sbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226e7d176d4dbe5ee23e7549bce755b24e4a88456720c9b608dce460ce1675a17c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.09.17.301788v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2613581 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.09.17.301788v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2050193981">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2050193981" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_710309085" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-Mxk5F7dJiBDWuVSgzV3z8O5LXquarTMVxhkGl3Ze0B0"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2613581" data-node-nid="2613581" id="node-2613581--2238620539" data-pisa="biorxiv;2020.09.17.301788v3" data-pisa-master="biorxiv;2020.09.17.301788" data-apath="/biorxiv/early/2022/06/26/2020.09.17.301788.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    A framework to efficiently smooth L
                    <sub>
                     1
                    </sub>
                    penalties for linear regression
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-6008-2720" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Georg
                      </span>
                      <span class="nlm-surname">
                       Hahn
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Sharon M.
                      </span>
                      <span class="nlm-surname">
                       Lutz
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Nilanjana
                      </span>
                      <span class="nlm-surname">
                       Laha
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Christoph
                      </span>
                      <span class="nlm-surname">
                       Lange
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.09.17.301788
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2613581--2238620539" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Georg Hahn
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Biostatistics, T.H. Chan School of Public Health, Harvard University
                       </span>
                       , Boston, MA 02115
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Georg%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hahn%20G&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AGeorg%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-6008-2720" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Georg Hahn
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:ghahn@hsph.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">
                        ghahn@hsph.harvard.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Sharon M. Lutz
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Harvard T.H. Chan School of Public Health
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Sharon%2BM.%2BLutz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lutz%20SM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ASharon%2BM.%2BLutz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Nilanjana Laha
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Harvard T.H. Chan School of Public Health
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nilanjana%2BLaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Laha%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ANilanjana%2BLaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Christoph Lange
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Harvard T.H. Chan School of Public Health
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Christoph%2BLange%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lange%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AChristoph%2BLange%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.09.17.301788v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2613581" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2613581/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.17.301788v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2613581" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2613581/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.17.301788v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2613581" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2613581/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.17.301788v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2613581" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2613581/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.09.17.301788v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2613581" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2613581/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Penalized linear regression approaches that include an L
                             <sub>
                              1
                             </sub>
                             term have become an important tool in statistical data analysis. One prominent example is the least absolute shrinkage and selection operator (Lasso), though the class of L
                             <sub>
                              1
                             </sub>
                             penalized regression operators also includes the fused and graphical Lasso, the elastic net, etc. Although the L
                             <sub>
                              1
                             </sub>
                             penalty makes their objective function convex, it is not differentiable everywhere, motivating the development of proximal gradient algorithms such as Fista, the current gold standard in the literature. In this work, we take a different approach based on smoothing in a fixed parameter setting (the problem size n and number of parameters p are fixed). The methodological contribution of our article is threefold: (1) We introduce a unified framework to compute closed-form smooth surrogates of a whole class of L
                             <sub>
                              1
                             </sub>
                             penalized regression problems using Nesterov smoothing. The surrogates preserve the convexity of the original (unsmoothed) objective functions, are uniformly close to them, and have closed-form derivatives everywhere for efficient minimization via gradient descent; (2) We prove that the estimates obtained with the smooth surrogates can be made arbitrarily close to the ones of the original (unsmoothed) objective functions, and provide explicitly computable a priori error bounds on the accuracy of our estimates; (3) We propose an iterative algorithm to progressively smooth the L
                             <sub>
                              1
                             </sub>
                             penalty which increases accuracy and is virtually free of tuning parameters. The proposed methodology is applicable to a large class of L
                             <sub>
                              1
                             </sub>
                             penalized regression operators, including all the operators mentioned above. Although the resulting estimates are typically dense, sparseness can be enforced again via thresholding. Using simulation studies, we compare our framework to current gold standards such as Fista, glmnet, gLasso, etc. Our results suggest that our proposed smoothing framework provides predictions of equal or higher accuracy than the gold standards while keeping the aforementioned theoretical guarantees and having roughly the same asymptotic runtime scaling.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-5">
                             In this communication, we aim to efficiently solve L
                             <sub>
                              1
                             </sub>
                             penalized regression problems. Since first considered around 1800 by Adrien-Marie Legendre and Carl Friedrich Gauss, the least squares approach has traditionally been used to solve linear regression problems. Nevertheless, least squares estimates have two major drawbacks, a lack of robustness and a lack of sparsity in high dimensional settings, i.e. settings characterized by p ≫ n, where p ∈ ℕ denotes the number of covariates and n ∈ ℕ is the sample size. Those two limitations have motivated the development of new estimation operators that are more suitable for such settings, e.g. the least absolute shrinkage and selection operator (Lasso) (
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              Tibshirani, 1996
                             </a>
                             ), or the least-angle regression (LARS) (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Efron et al., 2004
                             </a>
                             ). Many important extensions of the standard Lasso approach have been proposed, such as the elastic net (
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              Zou & Hastie, 2005
                             </a>
                             ), the fused Lasso (
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              Tibshirani et al., 2005
                             </a>
                             ), or the graphical Lasso (
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              Friedman et al., 2008
                             </a>
                             ). All Lasso approaches involve an L
                             <sub>
                              1
                             </sub>
                             penalty on the parameters which are being estimated to enforce sparseness.
                            </p>
                            <p id="p-6">
                             This article focuses on penalized L
                             <sub>
                              1
                             </sub>
                             regression, where we model the relationship of a given design matrix X ∈ ℝ
                             <sup>
                              n×p
                             </sup>
                             to an observed response y ∈ ℝ
                             <sup>
                              n
                             </sup>
                             with the help of some unknown parameters β ∈ ℝ
                             <sup>
                              p
                             </sup>
                             . In the most general case, we consider any regression operator which can be written as
                             <span class="disp-formula" id="disp-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-1.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-1.gif"/>
                               </noscript>
                              </span>
                             </span>
                             where u : ℝ
                             <sup>
                              p
                             </sup>
                             → ℝ implicitly depends on X and y, v : ℝ
                             <sup>
                              m
                             </sup>
                             → ℝ, and w
                             <sub>
                              j
                             </sub>
                             : ℝ
                             <sup>
                              p
                             </sup>
                             → ℝ for m ∈ ℕ and j ∈ {1, …, m}. The regression estimate is computed accordingly as arg min
                             <sub>
                              β
                             </sub>
                             Φ(β). Throughout the article, we consider a fixed parameter setting (n, m, p are fixed).
                            </p>
                            <p id="p-7">
                             Amongst others, all the previously mentioned estimation operators can be casted in the form of
                             <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             . For instance, for a fixed design matrix X ∈ ℝ
                             <sup>
                              n×p
                             </sup>
                             and response y ∈ ℝ
                             <sup>
                              n
                             </sup>
                             , the objective function of the standard Lasso (
                             <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                              Tibshirani, 1996
                             </a>
                             ) is obtained from
                             <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             by choosing
                             <span class="inline-formula" id="inline-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-1.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-1.gif"/>
                               </noscript>
                              </span>
                             </span>
                             , as well as
                             <span class="inline-formula" id="inline-formula-2">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-2.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-2.gif"/>
                               </noscript>
                              </span>
                             </span>
                             for z ∈ ℝ
                             <sup>
                              p
                             </sup>
                             , and w
                             <sub>
                              j
                             </sub>
                             (β) = β
                             <sub>
                              j
                             </sub>
                             for j ∈ {1, …, p}.
                            </p>
                            <p id="p-8">
                             If the objective function in
                             <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             is convex, steepest descent (quasi-Newton) methods can be employed to compute arg min
                             <sub>
                              β
                             </sub>
                             Φ(β). Nevertheless, the non-differentiability of the L
                             <sub>
                              1
                             </sub>
                             penalty may cause a loss of accuracy in conventional gradient(-free) solvers.
                            </p>
                            <p id="p-9">
                             We address this issue by providing a smooth surrogate Φ
                             <sup>
                              μ
                             </sup>
                             of
                             <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             , which depends on a smoothing parameter μ > 0. The smooth surrogate is derived by applying Nesterov smoothing (
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              Nesterov, 2005
                             </a>
                             ) to the absolute values in
                             <a id="xref-disp-formula-1-5" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             . Under the condition that u is differentiable and convex, v is differentiable and Lipschitz continuous, v preserves strict convexity, and w
                             <sub>
                              j
                             </sub>
                             are differentiable for j ∈ {1, …, p}, our framework provides the following guarantees:
                            </p>
                            <ol class="list-ord" id="list-1">
                             <li id="list-item-1">
                              <p id="p-10">
                               The surrogate Φ
                               <sup>
                                μ
                               </sup>
                               has explicit gradients everywhere.
                              </p>
                             </li>
                             <li id="list-item-2">
                              <p id="p-11">
                               The surrogate Φ
                               <sup>
                                μ
                               </sup>
                               is strictly convex.
                              </p>
                             </li>
                             <li id="list-item-3">
                              <p id="p-12">
                               The surrogate Φ
                               <sup>
                                μ
                               </sup>
                               is uniformly close to Φ, in the sense that it satisfies
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               |Φ(β) − Φ
                               <sup>
                                μ
                               </sup>
                               (β)| ≤C
                               <sub>
                                μ
                               </sub>
                               = O(μ) for fixed m, where C
                               <sub>
                                μ
                               </sub>
                               is a constant which does not depend on β.
                              </p>
                             </li>
                             <li id="list-item-4">
                              <p id="p-13">
                               We prove explicit bounds on ∥ arg min
                               <sub>
                                β
                               </sub>
                               Φ(β) − arg min
                               <sub>
                                β
                               </sub>
                               Φ
                               <sup>
                                μ
                               </sup>
                               (β) ∥
                               <sub>
                                2
                               </sub>
                               , that is on the distance between the minimizers of the unsmoothed objective function and its smooth surrogate.
                              </p>
                             </li>
                             <li id="list-item-5">
                              <p id="p-14">
                               We prove that ∥ arg min
                               <sub>
                                β
                               </sub>
                               Φ(β) − arg min
                               <sub>
                                β
                               </sub>
                               Φ
                               <sup>
                                μ
                               </sup>
                               (β) ∥ → 0 in the supremum norm as μ → 0.
                              </p>
                             </li>
                            </ol>
                            <p id="p-15">
                             In particular, all these properties hold true for the aforementioned regression operators, thus making it possible to obtain a unified surrogate with the theoretical guarantees enumerated above for a class of common regression operators. Although the estimates obtained by minimizing Φ
                             <sup>
                              μ
                             </sup>
                             are typically dense, sparseness can be enforced again via thresholding.
                            </p>
                            <p id="p-16">
                             To summarize, the contribution of our article is threefold: (1) We introduce the closed-form surrogate Φ
                             <sup>
                              μ
                             </sup>
                             ; (2) We prove the theoretical properties of Φ
                             <sup>
                              μ
                             </sup>
                             enumerated above; (3) Starting with a high degree of smoothness (i.e., a large value of μ), an iterative algorithm is proposed to progressively smooth the surrogate in order to facilitate minimization. The choice of the smoothing parameter does not have a major effect on the performance of our progressive smoothing algorithm, thus making it virtually free of tuning parameters.
                            </p>
                            <p id="p-17">
                             We evaluate our proposed algorithms with respect to accuracy and runtime using simulation studies. Our aim is to compare them to the current gold standards in the literature. First, we use as a benchmark the implementation of the Fista algorithm (
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              Beck & Teboulle, 2009
                             </a>
                             ) that is included in the fasta R-package on CRAN (
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              Chi et al., 2018
                             </a>
                             ). Fista combines the basic iterations of the Iterative Shrinkage-Thresholding Algorithm (
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Daubechies et al., 2004
                             </a>
                             ) with a Nesterov acceleration step. The algorithm can be regarded as a proximal gradient version of the Nesterov method (
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              Nesterov, 1983
                             </a>
                             ). Analogously to
                             <a id="xref-disp-formula-1-6" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             , Fista requires the separate specification of the smooth and non-smooth parts of the objective function and their explicit gradients. For the application of our smoothing algorithms to the elastic net, we benchmark against the glmnet algorithm (
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              Friedman et al., 2010a
                             </a>
                             ) that is implemented in the R-package glmnet (
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              Friedman et al., 2020
                             </a>
                             ). The glmnet algorithm is a variant of Fista which performs a cyclic update of all coordinates, whereas Fista updates all coordinates per iteration. For the smoothed fused Lasso, we compare with the fused Lasso methodology in the R-package genlasso (
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              Arnold & Tibshirani, 2020
                             </a>
                             ). Finally, we benchmark our smoothed graphical Lasso against the R-package glasso (
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              Friedman et al., 2019
                             </a>
                             ).
                            </p>
                            <p id="p-18">
                             Since the least-angle regression (LARS) (
                             <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                              Efron et al., 2004
                             </a>
                             ) or the group Lasso (
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              Yuan & Lin, 2005
                             </a>
                             ) do not involve an L
                             <sub>
                              1
                             </sub>
                             penalty on the regression estimates, our smoothing framework does not apply to them. We provide a detailed literature review in
                             <a id="xref-sec-2-1" class="xref-sec" href="#sec-2">
                              Section 1.1
                             </a>
                             to highlight previous work, distinguish it from ours, and emphasize the contribution of our article.
                            </p>
                            <p id="p-19">
                             This article is structured as follows.
                             <a id="xref-sec-3-1" class="xref-sec" href="#sec-3">
                              Section 2
                             </a>
                             starts by deriving the smooth surrogate of the objective function. We also derive the aforementioned theoretical guarantees of our smoothing framework, and state precise conditions on the quantities involved in
                             <a id="xref-disp-formula-1-7" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             which ensure that the theoretical guarantees hold true. Moreover, we demonstrate how all the aforementioned L
                             <sub>
                              1
                             </sub>
                             penalized regression operators fit in our framework, and we refine our approach by proposing a iterative and virtually tuning-free smoothing procedure. We benchmark our proposed methodology on simulated data against state-of-the-art algorithms in
                             <a id="xref-sec-9-1" class="xref-sec" href="#sec-9">
                              Section 3
                             </a>
                             . The article concludes with a discussion in
                             <a id="xref-sec-13-1" class="xref-sec" href="#sec-13">
                              Section 4
                             </a>
                             . A detailed overview of Nesterov smoothing (Section A), all proofs (Section B), and additional simulations (Section C) are provided in the Supporting Information.
                            </p>
                            <p id="p-20">
                             This article extends a previous publication (
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              Hahn et al., 2021
                             </a>
                             ) in which Nesterov smoothing is applied to the Lasso objective function only, meaning without any theoretical results. In contrast to the previous work (
                             <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">
                              Hahn et al., 2021
                             </a>
                             ),
                             <a id="xref-sec-3-2" class="xref-sec" href="#sec-3">
                              Section 2
                             </a>
                             presents a framework to smooth an entire class of L
                             <sub>
                              1
                             </sub>
                             penalized regression operators, and the theoretical results of
                             <a id="xref-sec-6-1" class="xref-sec" href="#sec-6">
                              Sections 2.3
                             </a>
                             and
                             <a id="xref-sec-7-1" class="xref-sec" href="#sec-7">
                              2.4
                             </a>
                             , the progressive smoothing algorithm of
                             <a id="xref-sec-8-1" class="xref-sec" href="#sec-8">
                              Section 2.5
                             </a>
                             , as well as all simulations of
                             <a id="xref-sec-9-2" class="xref-sec" href="#sec-9">
                              Section 3
                             </a>
                             are new. The methodology of this article is implemented in the R-package smoothedLasso (
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              Hahn et al., 2020
                             </a>
                             ), available on The Comprehensive R Archive Network (CRAN) (
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              R Core Team, 2014
                             </a>
                             ).
                            </p>
                            <p id="p-21">
                             Throughout the article, the ith entry of a vector v is denoted as v
                             <sub>
                              i
                             </sub>
                             . The entry (i, j) of a matrix X is denoted as X
                             <sub>
                              ij
                             </sub>
                             , while the ith row of X is denoted as X
                             <sub>
                              i,·
                             </sub>
                             and the jth column is denoted as X
                             <sub>
                              ·,j
                             </sub>
                             . The absolute value, the L
                             <sub>
                              1
                             </sub>
                             norm, as well as the Euclidean norm are written as | · |, ∥ · ∥
                             <sub>
                              1
                             </sub>
                             , and ∥ · ∥
                             <sub>
                              2
                             </sub>
                             , respectively. The Kronecker delta function is written as δ
                             <sub>
                              ij
                             </sub>
                             , defined as δ
                             <sub>
                              ij
                             </sub>
                             = 1 if i = j and δ
                             <sub>
                              ij
                             </sub>
                             = 0 if i ≠ j. The size of a set S is denoted as |S|.
                            </p>
                            <div id="sec-2" class="subsection">
                             <h3>
                              1.1 Literature review
                             </h3>
                             <p id="p-22">
                              Since the seminal publication of the Lasso (
                              <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">
                               Tibshirani, 1996
                              </a>
                              ), numerous approaches have focused on (smoothing) approaches to facilitate the minimization of the Lasso objective function, or other related objective functions. Nevertheless, the following publications differ from our work in that they do not consider a unified framework to smooth L
                              <sub>
                               1
                              </sub>
                              regularized regression operators, and if they provide a tailored solution to a particular regression operator, the bounds provided differ from the ones we establish on the accuracy of the unsmoothed objective and surrogate. Most importantly, none of the publications we are aware of quantifies the distance between the minimizer of the unsmoothed objective and the minimizer of the surrogate a priori (as we do with our explicitly computable bound), and no progressive smoothing procedure yielding stable regression estimates is derived.
                             </p>
                             <p id="p-23">
                              Smoothing approaches for the Lasso L
                              <sub>
                               1
                              </sub>
                              penalty, the SCAD (Smoothly Clipped Absolute Deviation) penalty, and hard thresholding penalties have been considered (
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               Fan & Li, 2001
                              </a>
                              ). However, these smoothing approaches are not based on the Nesterov framework (
                              <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ). Instead, the authors employ a quadratic approximation at the singularity of the penalties to achieve a smoothing effect, and they propose a one-step shooting algorithm for minimization. However, their main focus is on root-n consistency results of the resulting estimators and asymptotic normality results for the SCAD penalty, results which the authors state do not all apply to the Lasso.
                             </p>
                             <p id="p-24">
                              Some smoothing approaches (
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               Belloni et al., 2011
                              </a>
                              ;
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               Chen et al., 2010a
                              </a>
                              ;
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               Banerjee et al., 2008
                              </a>
                              ) build upon Nesterov’s first-order accelerated gradient descent algorithm (
                              <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ). Those variants of Nesterov’s algorithm are iterative methods which are unrelated to our adaptive smoothing procedure. A detailed overview of several variants of the first-order accelerated gradient descent algorithm can be found in the literature (
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               Becker & Candès, 2011
                              </a>
                              ).
                             </p>
                             <p id="p-25">
                              Publications which extend the work of Nesterov (
                              <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ) are available (
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               Beck & Teboulle, 2012
                              </a>
                              ). In the latter, the authors consider a more general smoothing framework which, as a special case, includes the same smoothing we establish for the absolute value in the L
                              <sub>
                               1
                              </sub>
                              penalty of the Lasso. However, in contrast to our work, the authors do not consider a smooth surrogate of a general form that applies to a wide range of regression operators, and the theoretical guarantees on the surrogate that we derive are unconsidered.
                             </p>
                             <p id="p-26">
                              In another publication (
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               Haselimashhadi & Vinciotti, 2016
                              </a>
                              ), the authors likewise smooth the absolute value in the L
                              <sub>
                               1
                              </sub>
                              penalty of the Lasso using Nesterov’s technique, and they state the same bound on the difference between the unsmoothed and smoothed Lasso objective functions taken from Nesterov’s results. However, their results focus on the Lasso only, the general surrogate we provide is unconsidered, and no explicit a priori bounds on the accuracy of the obtained minimizers are given. Importantly, these works (
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               Haselimashhadi & Vinciotti, 2016
                              </a>
                              ) differ from our work in that the authors enforce that the smoothed Lasso penalty passes through zero, leading the focus of their article to be on another smoothed Lasso approach which is based on the error function of the normal distribution.
                             </p>
                             <p id="p-27">
                              Further work available in the literature employs Nesterov’s smoothing techniques for a variety of specialized Lasso objective functions. For instance, some authors (
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               Chen et al., 2010b
                              </a>
                              ) consider the group Lasso and employ Nesterov’s formalism to smooth the Lasso penalty using the squared error proximity function which we also consider. Nevertheless, the authors focus on adapting Nesterov’s first-order accelerated gradient descent algorithm in order to compute the Lasso regression estimate, whereas we focus on deriving theoretical bounds on the smooth surrogate and our progressive smoothing algorithm. In our work, we do not propose a modification of Nesterov’s first-order accelerated gradient descent algorithm. The group Lasso is also considered in the literature by some authors (
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               Chen et al., 2012
                              </a>
                              ), who separate out the simple nonsmooth L
                              <sub>
                               1
                              </sub>
                              penalty from the more complex structure-inducing penalties, and only smooth the latter. This leaves the L
                              <sub>
                               1
                              </sub>
                              norm on the parameters unchanged, thus still enforcing individual feature level sparsity. In contrast to this published work (
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               Chen et al., 2012
                              </a>
                              ), we always smooth all L
                              <sub>
                               1
                              </sub>
                              penalties which are present in the objective function.
                             </p>
                             <p id="p-28">
                              The joint Lasso is also considered in the literature (
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               Dondelinger & Mukherjee, 2020
                              </a>
                              ), who state an iterative minimization procedure which smoothes the Lasso penalty using Nesterov’s techniques. The authors state closed form derivatives for minimization (as we do as well in the form of the more general derivatives of the smooth surrogate), but no other theoretical results are given.
                             </p>
                             <p id="p-29">
                              One variant of the original Lasso which has recently gained attention is the concomitant Lasso. The concomitant Lasso augments the original Lasso with a term σ/2 for which a second regularization parameter σ is introduced (
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               Ndiaye et al., 2017
                              </a>
                              ). The parameter σ is meant to be decreased to zero. Smoothing the concomitant Lasso has the advantage that Nesterov’s techniques do not need to be applied to the L
                              <sub>
                               1
                              </sub>
                              penalty. Instead, the smooth concomitant Lasso has a closed form expression which is different from the smoothed Lasso approaches we consider (
                              <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">
                               Ndiaye et al., 2017
                              </a>
                              ), and results in the literature (
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               Massias et al., 2018
                              </a>
                              ) are only named in analogy to the smoothing terminology introduced in Nesterov’s original paper (
                              <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ). Since the concomitant Lasso does not contain an L
                              <sub>
                               1
                              </sub>
                              penalty, our results to not apply.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             2 The smooth surrogate
                            </h2>
                            <p id="p-30">
                             This section states the precise conditions and results of our smoothing framework for regression operators with an objective function of the form of
                             <a id="xref-disp-formula-1-8" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             . The smoothing of
                             <a id="xref-disp-formula-1-9" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             will be achieved with the help of Nesterov smoothing, which is briefly summarized in
                             <a id="xref-sec-4-1" class="xref-sec" href="#sec-4">
                              Section 2.1
                             </a>
                             . In
                             <a id="xref-sec-5-1" class="xref-sec" href="#sec-5">
                              Section 2.2
                             </a>
                             we smooth the L
                             <sub>
                              1
                             </sub>
                             contribution in
                             <a id="xref-disp-formula-1-10" class="xref-disp-formula" href="#disp-formula-1">
                              eq. (1)
                             </a>
                             , leading to the smooth surrogate objective function.
                             <a id="xref-sec-6-2" class="xref-sec" href="#sec-6">
                              Section 2.3
                             </a>
                             states the theoretical guarantees provided for the smooth surrogate, and the precise conditions required for the guarantees to hold true.
                             <a id="xref-sec-7-2" class="xref-sec" href="#sec-7">
                              Section 2.4
                             </a>
                             gives examples of how common regression operators fit into our framework. Finally,
                             <a id="xref-sec-8-2" class="xref-sec" href="#sec-8">
                              Section 2.5
                             </a>
                             refines our approach by proposing a iterative and virtually tuning-free smoothing procedure.
                            </p>
                            <div id="sec-4" class="subsection">
                             <h3>
                              2.1 Summary of Nesterov smoothing
                             </h3>
                             <p id="p-31">
                              The smoothing of a piecewise affine and convex function f : ℝ
                              <sup>
                               q
                              </sup>
                              → ℝ, where q ∈ ℕ, has been established in the literature (
                              <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ). Since f is piecewise affine, it can be represented as f (z) = max
                              <sub>
                               i=1,…,k
                              </sub>
                              (A[z, 1]
                              <sup>
                               T
                              </sup>
                              )
                              <sub>
                               i
                              </sub>
                              , where k ∈ ℕ is the number of linear pieces (components). Here, the rows of the matrix A ∈ ℝ
                              <sup>
                               k×(q+1)
                              </sup>
                              contain the coefficients of each linear piece, with column q + 1 containing all constant coefficients. Moreover, [z, 1] ∈ ℝ
                              <sup>
                               q+1
                              </sup>
                              denotes the vector obtained by concatenating z ∈ ℝ
                              <sup>
                               q
                              </sup>
                              and the scalar 1.
                             </p>
                             <p id="p-32">
                              The goal is to derive an approximation f
                              <sup>
                               μ
                              </sup>
                              of f which is both smooth and uniformly close to f (
                              <a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ). The approximation f
                              <sup>
                               μ
                              </sup>
                              depends on a so-called proximity (or prox) function, see
                              <a id="xref-sec-18-1" class="xref-sec" href="#sec-18">
                               Section A.1
                              </a>
                              , which is parameterized by a smoothing parameter μ > 0 controlling the degree of smoothness. We consider two choices of the prox function, presented in
                              <a id="xref-sec-19-1" class="xref-sec" href="#sec-19">
                               Section A.2
                              </a>
                              . The so-called entropy prox function yields a closed-form expression of the smooth approximation
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              given by
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which satisfies the uniform approximation bound
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-33">
                              Another choice, introduced in
                              <a id="xref-sec-19-2" class="xref-sec" href="#sec-19">
                               Section A.2
                              </a>
                              , is the squared error prox function, given by
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for τ ∈ ℝ
                              <sup>
                               k
                              </sup>
                              . Let the vector c(z) = (c
                              <sub>
                               1
                              </sub>
                              (z), …, c
                              <sub>
                               k
                              </sub>
                              (z)) ∈ ℝ
                              <sup>
                               k
                              </sup>
                              be defined componentwise by c
                              <sub>
                               i
                              </sub>
                              (z) = 1/μ · (A[z, 1]
                              <sup>
                               T
                              </sup>
                              )
                              <sub>
                               i
                              </sub>
                              − 1/k for i ∈ {1, …, k}. Let ĉ(z) ∈ ℝ be the Michelot projection (
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               Michelot, 1986
                              </a>
                              ) of c(z) onto the k-dimensional unit simplex Q
                              <sub>
                               k
                              </sub>
                              (see
                              <a id="xref-sec-21-1" class="xref-sec" href="#sec-21">
                               Section A.2.2
                              </a>
                              ). In the original publication (
                              <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ), it is shown that
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is a smooth approximation of f satisfying the uniform bound
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-34">
                              Further details on the above results can be found in Section A of the Supplementary Materials.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              2.2 The smooth surrogate
                             </h3>
                             <p id="p-35">
                              We employ the results of
                              <a id="xref-sec-4-2" class="xref-sec" href="#sec-4">
                               Section 2.1
                              </a>
                              to smooth the absolute value in
                              <a id="xref-disp-formula-1-11" class="xref-disp-formula" href="#disp-formula-1">
                               eq. (1)
                              </a>
                              . In the following subsections, we always consider k = 2, z ∈ ℝ, and the specific choice of the matrix A ∈ ℝ
                              <sup>
                               2×2
                              </sup>
                              given by
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-36">
                              The specific choice of A in
                              <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                               eq. (6)
                              </a>
                              allows us to express the (one dimensional) absolute value as a piecewise affine and convex function f (z) = max{−z, z} = max
                              <sub>
                               i=1,2
                              </sub>
                              (A[z, 1]
                              <sup>
                               T
                              </sup>
                              )
                              <sub>
                               i
                              </sub>
                              .
                             </p>
                             <p id="p-37">
                              Simplifying
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               eq. (2)
                              </a>
                              with the specific choice of A of
                              <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">
                               eq. (6)
                              </a>
                              results in the entropy prox approximation of the absolute value and its explicit derivative, given by
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-38">
                              The approximation bound of
                              <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                               eq. (3)
                              </a>
                              simplifies to
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-39">
                              In a similar fashion, the squared error prox smoothing of
                              <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
                               eq. (4)
                              </a>
                              with A as in
                              <a id="xref-disp-formula-6-3" class="xref-disp-formula" href="#disp-formula-6">
                               eq. (6)
                              </a>
                              leads to the following approximation of the absolute value and its explicit derivative:
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where ĉ(z) ∈ ℝ
                              <sup>
                               2
                              </sup>
                              denotes the Michelot projection of c(z) = 1/μ · [−z, z] − 1/2 onto the twodimensional unit simplex. A derivation of the derivative of
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              can be found in Lemma 4 (
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               Hahn et al., 2017
                              </a>
                              ). The approximation bound of
                              <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                               eq. (5)
                              </a>
                              simplifies to
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-40">
                              With the smooth approximation of the absolute value in place, we define the surrogate of Φ as
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where in the remainder of
                              <a id="xref-sec-3-3" class="xref-sec" href="#sec-3">
                               Section 2
                              </a>
                              , f
                              <sup>
                               μ
                              </sup>
                              always denotes either the entropy prox smoothed absolute value
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , or the squared error prox smoothed absolute value
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              2.3 Theoretical guarantees
                             </h3>
                             <p id="p-41">
                              Depending on the conditions imposed upon u, v, and w
                              <sub>
                               j
                              </sub>
                              for j ∈ {1, …, m} in
                              <a id="xref-disp-formula-1-12" class="xref-disp-formula" href="#disp-formula-1">
                               eq. (1)
                              </a>
                              , the surrogate Φ
                              <sup>
                               μ
                              </sup>
                              of
                              <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">
                               eq. (11)
                              </a>
                              has a variety of properties. The proofs of all results presented in this section can be found in Section B of the Supplementary Materials.
                             </p>
                             <div id="statement-1" class="statement">
                              <span class="statement-label">
                               Condition 1.
                              </span>
                              <p id="p-42">
                               The functions u, v and w
                               <sub>
                                j
                               </sub>
                               of
                               <a id="xref-disp-formula-1-13" class="xref-disp-formula" href="#disp-formula-1">
                                eq. (1)
                               </a>
                               are differentiable everywhere for all j ∈ {1, …, m}.
                              </p>
                             </div>
                             <div id="statement-2" class="statement">
                              <span class="statement-label">
                               Proposition 1.
                              </span>
                              <p id="p-43">
                               Under
                               <a id="xref-statement-1-1" class="xref-statement" href="#statement-1">
                                Condition 1
                               </a>
                               , the surrogate Φ
                               <sup>
                                μ
                               </sup>
                               is differentiable everywhere with explicit gradient
                               <span class="disp-formula" id="disp-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where g
                               <sup>
                                μ
                               </sup>
                               is the derivative of f
                               <sup>
                                μ
                               </sup>
                               as given in
                               <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                                eq. (7)
                               </a>
                               and
                               <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                                eq. (9)
                               </a>
                               , D
                               <sub>
                                j
                               </sub>
                               v is the derivative of v with respect to its jth argument, and θ is given by the vector θ = (f
                               <sup>
                                μ
                               </sup>
                               (w
                               <sub>
                                1
                               </sub>
                               (β)), …, f
                               <sup>
                                μ
                               </sup>
                               (w
                               <sub>
                                m
                               </sub>
                               (β))) ∈ ℝ
                               <sup>
                                m
                               </sup>
                               .
                              </p>
                             </div>
                             <p id="p-44">
                              The proof of
                              <a id="xref-statement-2-1" class="xref-statement" href="#statement-2">
                               Proposition 1
                              </a>
                              follows from a direct calculation. Subject to the following commonly satisfied condition, the surrogate Φ
                              <sup>
                               μ
                              </sup>
                              is strictly convex.
                             </p>
                             <div id="statement-3" class="statement">
                              <span class="statement-label">
                               Condition 2.
                              </span>
                              <p id="p-45">
                               In
                               <a id="xref-disp-formula-1-14" class="xref-disp-formula" href="#disp-formula-1">
                                eq. (1)
                               </a>
                               , the function u is convex and v preserves strict convexity of its input arguments.
                              </p>
                             </div>
                             <div id="statement-4" class="statement">
                              <span class="statement-label">
                               Proposition 2.
                              </span>
                              <p id="p-46">
                               Under
                               <a id="xref-statement-3-1" class="xref-statement" href="#statement-3">
                                Condition 2
                               </a>
                               , Φ
                               <sup>
                                μ
                               </sup>
                               is strictly convex.
                              </p>
                             </div>
                             <p id="p-47">
                              The proof of
                              <a id="xref-statement-4-1" class="xref-statement" href="#statement-4">
                               Proposition 2
                              </a>
                              follows from the fact that according to
                              <a id="xref-statement-10-1" class="xref-statement" href="#statement-10">
                               Proposition 6
                              </a>
                              in Section B,
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are strictly convex. Thus if u is convex and v preserves strict convexity, Φ
                              <sup>
                               μ
                              </sup>
                              is itself strictly convex as the sum of a convex function and a strictly convex function. Alternatively, if v only preserves convexity, Φ
                              <sup>
                               μ
                              </sup>
                              will still be convex (given u is convex). However, the results of Propositions 4 and 5 below do not hold true any more as they require strict convexity.
                             </p>
                             <p id="p-48">
                              Next, we consider an approximation bound of Φ
                              <sup>
                               μ
                              </sup>
                              to Φ. Under the following condition, the bounds on
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in
                              <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                               eq. (8)
                              </a>
                              and
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in
                              <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                               eq. (10)
                              </a>
                              translate to a guaranteed error bound on the surrogate Φ
                              <sup>
                               μ
                              </sup>
                              .
                             </p>
                             <div id="statement-5" class="statement">
                              <span class="statement-label">
                               Condition 3.
                              </span>
                              <p id="p-49">
                               The function v of
                               <a id="xref-disp-formula-1-15" class="xref-disp-formula" href="#disp-formula-1">
                                eq. (1)
                               </a>
                               is Lipschitz continuous with Lipschitz constant L
                               <sub>
                                v
                               </sub>
                               .
                              </p>
                             </div>
                             <p id="p-50">
                              <a id="xref-statement-5-1" class="xref-statement" href="#statement-5">
                               Condition 3
                              </a>
                              can easily be verified. Given
                              <a id="xref-statement-1-2" class="xref-statement" href="#statement-1">
                               Condition 1
                              </a>
                              is already satisfied, and v is thus differentiable, then a bounded derivative (in L
                              <sub>
                               1
                              </sub>
                              norm) of v will make v Lipschitz continuous with
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-51">
                              <a id="xref-statement-5-2" class="xref-statement" href="#statement-5">
                               Condition 3
                              </a>
                              is used in the following proposition to establish error bounds for Φ
                              <sup>
                               μ
                              </sup>
                              with respect to Φ.
                             </p>
                             <div id="statement-6" class="statement">
                              <span class="statement-label">
                               Proposition 3.
                              </span>
                              <p id="p-52">
                               Let C
                               <sup>
                                μ
                               </sup>
                               denote either
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               of
                               <a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">
                                eq. (8)
                               </a>
                               or
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               of
                               <a id="xref-disp-formula-10-2" class="xref-disp-formula" href="#disp-formula-10">
                                eq. (10)
                               </a>
                               depending on whether
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               or
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are employed in Φ
                               <sup>
                                μ
                               </sup>
                               . Under
                               <a id="xref-statement-5-3" class="xref-statement" href="#statement-5">
                                Condition 3
                               </a>
                               ,
                               <span class="disp-formula" id="disp-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <p id="p-53">
                              In
                              <a id="xref-statement-6-1" class="xref-statement" href="#statement-6">
                               Proposition 3
                              </a>
                              , the dimension m (which oftentimes is taken to be p, see
                              <a id="xref-sec-7-3" class="xref-sec" href="#sec-7">
                               Section 2.4
                              </a>
                              ) and the Lipschitz constant L
                              <sub>
                               v
                              </sub>
                              are fixed for a particular estimation problem, thus allowing us to make the approximation error arbitrarily small as the smoothing parameter μ → 0.
                             </p>
                             <p id="p-54">
                              Importantly, we are not only interested in the error bound between the objective function Φ and its surrogate Φ
                              <sup>
                               μ
                              </sup>
                              , but in how the minimizer
                              <span class="inline-formula" id="inline-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              compares to the one obtained had we computed
                              <span class="inline-formula" id="inline-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-55">
                              Following Lemma 2.9 (
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               Seijo & Sen, 2011
                              </a>
                              ), the following proposition shows that continuity, strict convexity and a vanishing approximation error of the surrogate implies that the global minimizers of Φ and Φ
                              <sup>
                               μ
                              </sup>
                              converge to each other in the supremum norm metric (again assuming m is kept fixed).
                             </p>
                             <div id="statement-7" class="statement">
                              <span class="statement-label">
                               Proposition 4.
                              </span>
                              <p id="p-56">
                               Let F
                               <sub>
                                1
                               </sub>
                               : ℝ
                               <sup>
                                s
                               </sup>
                               → ℝ be continuous and strictly convex for s ∈ ℕ. Then
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is continuous at F
                               <sub>
                                1
                               </sub>
                               with respect to the supremum norm.
                              </p>
                             </div>
                             <p id="p-57">
                              Under Conditions 1–3, Φ
                              <sup>
                               μ
                              </sup>
                              satisfies the requirements of
                              <a id="xref-statement-7-1" class="xref-statement" href="#statement-7">
                               Proposition 4
                              </a>
                              . Using the result of
                              <a id="xref-statement-6-2" class="xref-statement" href="#statement-6">
                               Proposition 3
                              </a>
                              ,
                              <span class="inline-formula" id="inline-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for μ → 0 implies that the minimizers of Φ
                              <sup>
                               μ
                              </sup>
                              and Φ converge to each other in the supremum norm. This result is stronger than the one of Theorem 4.4 in the original publication of the FISTA algorithm(
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               Beck & Teboulle, 2009
                              </a>
                              ), where it is proven that the FISTA method finds a minimizer which is of similar quality than the true minimizer.
                             </p>
                             <p id="p-58">
                              Although
                              <a id="xref-statement-7-2" class="xref-statement" href="#statement-7">
                               Proposition 4
                              </a>
                              shows convergence, it does not give an explicit error bound on the distance between the two minimizers. This is done in the next result.
                             </p>
                             <div id="statement-8" class="statement">
                              <span class="statement-label">
                               Proposition 5.
                              </span>
                              <p id="p-59">
                               Let s ∈ ℕ and ϵ > 0. Let F
                               <sub>
                                1
                               </sub>
                               : ℝ
                               <sup>
                                s
                               </sup>
                               → ℝ be differentiable and strictly convex. Let F
                               <sub>
                                2
                               </sub>
                               : ℝ
                               <sup>
                                s
                               </sup>
                               → ℝ be such that
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Let
                               <span class="inline-formula" id="inline-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               be the two minimizers for i ∈ {1, 2}. Then for any δ > 0, there exist two constants C
                               <sub>
                                δ
                               </sub>
                               > 0 and L
                               <sub>
                                δ
                               </sub>
                               > 0 independent of x
                               <sub>
                                2
                               </sub>
                               such that
                               <span class="disp-formula" id="disp-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where S
                               <sub>
                                δ
                               </sub>
                               (x) = {y : ∥x − y ∥
                               <sub>
                                2
                               </sub>
                               = δ} ⊂ ℝ
                               <sup>
                                s
                               </sup>
                               denotes the sphere of radius δ around x ∈ ℝ
                               <sup>
                                s
                               </sup>
                               with respect to the Euclidean norm.
                              </p>
                             </div>
                             <p id="p-60">
                              Note that
                              <a id="xref-statement-8-1" class="xref-statement" href="#statement-8">
                               Proposition 5
                              </a>
                              does not generalize to non strictly convex functions. Importantly,
                              <span class="inline-formula" id="inline-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in
                              <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              is well-defined since it suffices to take the supremum over the sphere of radius δ around x
                              <sub>
                               1
                              </sub>
                              , not the ball of radius δ around x
                              <sub>
                               1
                              </sub>
                              , thus avoiding the case that ∥∇F
                              <sub>
                               1
                              </sub>
                              (y
                              <sub>
                               1
                              </sub>
                              ) ∥
                              <sub>
                               2
                              </sub>
                              = 0 at the minimizer y
                              <sub>
                               1
                              </sub>
                              = x
                              <sub>
                               1
                              </sub>
                              .
                             </p>
                             <p id="p-61">
                              Under Conditions 1–3, applying
                              <a id="xref-statement-8-2" class="xref-statement" href="#statement-8">
                               Proposition 5
                              </a>
                              with F
                              <sub>
                               1
                              </sub>
                              taken to be the differentiable and strictly convex Φ
                              <sup>
                               μ
                              </sup>
                              and F
                              <sub>
                               2
                              </sub>
                              taken to be Φ immediately gives an explicit bound on the distance between the two minimizers
                              <span class="inline-formula" id="inline-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . This bound can be computed using the minimizer of Φ
                              <sup>
                               μ
                              </sup>
                              only. We demonstrate the quality of the bound in the simulations of
                              <a id="xref-sec-12-1" class="xref-sec" href="#sec-12">
                               Section 3.3
                              </a>
                              .
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              2.4 Applications to popular L1 penalized regression operators
                             </h3>
                             <p id="p-62">
                              Let X ∈ ℝ
                              <sup>
                               n×p
                              </sup>
                              , y ∈ ℝ
                              <sup>
                               n
                              </sup>
                              , and β ∈ ℝ
                              <sup>
                               p
                              </sup>
                              as introduced in
                              <a id="xref-sec-1-1" class="xref-sec" href="#sec-1">
                               Section 1
                              </a>
                              . Additionally, let λ, α, γ ≥ 0. A variety of popular regression operators can be written in the form of
                              <a id="xref-disp-formula-1-16" class="xref-disp-formula" href="#disp-formula-1">
                               eq. (1)
                              </a>
                              and satisfy the conditions of
                              <a id="xref-sec-6-3" class="xref-sec" href="#sec-6">
                               Section 2.3
                              </a>
                              .
                             </p>
                             <ol class="list-ord" id="list-2">
                              <li id="list-item-6">
                               <p id="p-63">
                                The least absolute shrinkage and selection operator (Lasso) (
                                <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">
                                 Tibshirani, 1996
                                </a>
                                ) is defined as
                                <span class="disp-formula" id="disp-formula-15">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-15.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-15.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-64">
                                It can be expressed in the form of
                                <a id="xref-disp-formula-1-17" class="xref-disp-formula" href="#disp-formula-1">
                                 eq. (1)
                                </a>
                                with
                                <span class="inline-formula" id="inline-formula-27">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-27.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-27.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , as well as
                                <span class="inline-formula" id="inline-formula-28">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-28.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-28.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for z ∈ ℝ
                                <sup>
                                 p
                                </sup>
                                , and w
                                <sub>
                                 j
                                </sub>
                                (β) = β
                                <sub>
                                 j
                                </sub>
                                for j ∈ {1, …, p}. Here,
                                <span class="inline-formula" id="inline-formula-29">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-29.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-29.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , and D
                                <sub>
                                 j
                                </sub>
                                v = λ for i, j ∈ {1, …, p}.
                               </p>
                              </li>
                              <li id="list-item-7">
                               <p id="p-65">
                                The elastic net (
                                <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                                 Zou & Hastie, 2005
                                </a>
                                ) augments the Lasso with an L
                                <sub>
                                 2
                                </sub>
                                penalty on the parameters β, resulting in
                                <span class="disp-formula" id="disp-formula-16">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-16.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-16.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                using a published parameterization (
                                <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                                 Friedman et al., 2010b
                                </a>
                                ) which is also used in the specification of the R-package glmnet (
                                <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                                 Friedman et al., 2020
                                </a>
                                ). It can be expressed in the form of
                                <a id="xref-disp-formula-1-18" class="xref-disp-formula" href="#disp-formula-1">
                                 eq. (1)
                                </a>
                                with
                                <span class="inline-formula" id="inline-formula-30">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-30.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-30.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , as well as
                                <span class="inline-formula" id="inline-formula-31">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-31.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-31.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for z ∈ ℝ
                                <sup>
                                 p
                                </sup>
                                and w
                                <sub>
                                 j
                                </sub>
                                as in the previous case of the standard Lasso, where j ∈ {1, …, p}. The derivative of u is
                                <span class="inline-formula" id="inline-formula-32">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-32.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-32.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                              </li>
                              <li id="list-item-8">
                               <p id="p-66">
                                The fused Lasso (
                                <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                                 Tibshirani et al., 2005
                                </a>
                                ) is defined as
                                <span class="disp-formula" id="disp-formula-17">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-17.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-17.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                ,where E is the edge set of some underlying graph (
                                <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                                 Arnold & Tibshirani, 2020
                                </a>
                                ). Let E = {e
                                <sub>
                                 1
                                </sub>
                                , …, e |
                                <sub>
                                 E
                                </sub>
                                |) in some arbitrary ordering, and
                                <span class="inline-formula" id="inline-formula-33">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-33.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-33.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for all i ∈ {1, …, |E|}. The fused Lasso can be casted in the form of
                                <a id="xref-disp-formula-1-19" class="xref-disp-formula" href="#disp-formula-1">
                                 eq. (1)
                                </a>
                                with u(β) defined similarly to the standard Lasso, as well as
                                <span class="inline-formula" id="inline-formula-34">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-34.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-34.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for m = p + |E| and z ∈ ℝ
                                <sup>
                                 m
                                </sup>
                                . Accordingly, w
                                <sub>
                                 j
                                </sub>
                                (β) = β
                                <sub>
                                 j
                                </sub>
                                for j ∈ {1, …, p} and
                                <span class="inline-formula" id="inline-formula-35">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-35.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-35.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for j ∈ {p + 1, …, m}. Here, D
                                <sub>
                                 j
                                </sub>
                                v = λ for j ∈ {1, …, p} and D
                                <sub>
                                 j
                                </sub>
                                v = λγ for j ∈ {p + 1, …, m}. Moreover, ∂w
                                <sub>
                                 j
                                </sub>
                                /∂β
                                <sub>
                                 i
                                </sub>
                                = δ
                                <sub>
                                 ij
                                </sub>
                                for j ∈ {1, …, p} and
                                <span class="inline-formula" id="inline-formula-36">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-36.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-36.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for j ∈ {p + 1, …, m}.
                               </p>
                              </li>
                              <li id="list-item-9">
                               <p id="p-67">
                                The graphical Lasso (
                                <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                                 Friedman et al., 2008
                                </a>
                                ,
                                <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                                 2019
                                </a>
                                ) considers observations X
                                <sub>
                                 1
                                </sub>
                                , …, X
                                <sub>
                                 n
                                </sub>
                                ∼N (0, Σ) from a multivariate Gaussian distribution and estimates the precision matrix Θ = Σ
                                <sup>
                                 −1
                                </sup>
                                ∈ ℝ
                                <sup>
                                 p×p
                                </sup>
                                as
                                <span class="disp-formula" id="disp-formula-18">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-18.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-18.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                where S is the sample covariance matrix, tr denotes the trace of a matrix, and the arg min is taken over all positive definite matrices Θ. Define
                                <span class="inline-formula" id="inline-formula-37">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-37.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-37.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                as the vector containing all parameters of Θ stacked together by row. As before, we define the differentiable part of
                                <a id="xref-disp-formula-1-20" class="xref-disp-formula" href="#disp-formula-1">
                                 eq. (1)
                                </a>
                                to include the trace and log terms, thus u(Θ) = tr(SΘ) − log det(Θ) with derivative ∇u = S
                                <sup>
                                 T
                                </sup>
                                − Θ
                                <sup>
                                 −1
                                </sup>
                                , and v : ℝ
                                <sup>
                                 m
                                </sup>
                                → ℝ and
                                <span class="inline-formula" id="inline-formula-38">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-38.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-38.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                with m = p
                                <sup>
                                 2
                                </sup>
                                as in the case of the standard Lasso.
                               </p>
                              </li>
                             </ol>
                             <div id="statement-9" class="statement">
                              <span class="statement-label">
                               Algorithm 1:
                              </span>
                              <h3>
                               Progressive smoothing
                              </h3>
                              <div id="F1" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F1.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F1.medium.gif" width="440" height="102"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F1.medium.gif" width="440" height="102"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure1" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <p id="p-68">
                              </p>
                             </div>
                             <p id="p-69">
                              In all above cases, the functions u, v, and w
                              <sub>
                               i
                              </sub>
                              are differentiable for i ∈ {1, …, m}, thus satisfying
                              <a id="xref-statement-1-3" class="xref-statement" href="#statement-1">
                               Condition 1
                              </a>
                              . Their derivatives are given explicitly for convenience.
                             </p>
                             <p id="p-70">
                              Moreover, all functions u above, including the one of the graphical Lasso, are convex as they can be written as the linear combinations of (Euclidean) norms. Importantly, the functions v are always linear combinations of their inputs with positive coefficients and thus preserve strict convexity. Therefore,
                              <a id="xref-statement-3-2" class="xref-statement" href="#statement-3">
                               Condition 2
                              </a>
                              is satisfied.
                             </p>
                             <p id="p-71">
                              Finally, the functions v are all differentiable and have a bounded gradient (in L
                              <sub>
                               1
                              </sub>
                              norm), thus making them Lipschitz continuous as required in
                              <a id="xref-statement-5-4" class="xref-statement" href="#statement-5">
                               Condition 3
                              </a>
                              . For instance,
                              <span class="inline-formula" id="inline-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for z ∈ ℝ
                              <sup>
                               p
                              </sup>
                              in case of the standard Lasso, thus we obtain ∇v = λ[1, …, 1] and
                              <span class="inline-formula" id="inline-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              using the fact that λ is non-negative.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              2.5 Progressive smoothing algorithm
                             </h3>
                             <p id="p-72">
                              This section proposes an adaptive smoothing technique for the surrogate Φ
                              <sup>
                               μ
                              </sup>
                              . Simulations in
                              <a id="xref-sec-9-3" class="xref-sec" href="#sec-9">
                               Section 3
                              </a>
                              show that the progressive smoothing algorithm yields stable estimators for linear regression.
                             </p>
                             <p id="p-73">
                              Instead of solving the smoothed problem
                              <span class="inline-formula" id="inline-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              directly for some μ > 0, we employ a progressive smoothing procedure along the following rationale: We start with a large value of the smoothing parameter μ to facilitate the minimization. After computing
                              <span class="inline-formula" id="inline-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we decrease the smoothing parameter and repeat the minimization using the previously found minimizer as the new starting value. This approach is based on the heuristic idea that as μ decreases and the smoothed surrogate Φ
                              <sup>
                               μ
                              </sup>
                              approaches Φ, the found minimizers in each iteration remain close to each other and converge to the minimizer of Φ.
                             </p>
                             <p id="p-74">
                              <a id="xref-statement-9-1" class="xref-statement" href="#statement-9">
                               Algorithm 1
                              </a>
                              formalizes our approach. The input of the algorithm is the function Φ
                              <sup>
                               μ
                              </sup>
                              , a target smoothing parameter μ
                              <sub>
                               0
                              </sub>
                              > 0, and a number of smoothing steps N ∈ ℕ. After initializing a random starting value
                              <span class="inline-formula" id="inline-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for the first minimization, we gradually decrease the degree of smoothness according to μ = 2
                              <sup>
                               i
                              </sup>
                              μ
                              <sub>
                               0
                              </sub>
                              from i = N to the target level μ
                              <sub>
                               0
                              </sub>
                              at i = 0. In each iteration i, we compute a new estimate
                              <span class="inline-formula" id="inline-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              using the current smoothing level μ and the previous estimate
                              <span class="inline-formula" id="inline-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as the starting value. The output of the algorithm is
                              <span class="inline-formula" id="inline-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the parameter estimate corresponding to the target smoothing degree μ
                              <sub>
                               0
                              </sub>
                              .
                             </p>
                             <p id="p-75">
                              Importantly, the advantage of
                              <a id="xref-statement-9-2" class="xref-statement" href="#statement-9">
                               Algorithm 1
                              </a>
                              consists in the fact that the precise specification of the smoothing parameter does not play a major role. It suffices to start with any sufficiently large value (that is, 2
                              <sup>
                               N
                              </sup>
                              μ
                              <sub>
                               0
                              </sub>
                              ≫ 1) and to end the iteration with any sufficiently small value μ
                              <sub>
                               0
                              </sub>
                              , for instance of the order of the machine precision or of the square root of the machine precision. This effectively makes
                              <a id="xref-statement-9-3" class="xref-statement" href="#statement-9">
                               Algorithm 1
                              </a>
                              free of tuning parameters.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             3 Experimental results
                            </h2>
                            <p id="p-76">
                             In this section, we evaluate the performance of the regression estimates computed with our proposed smooth surrogate Φ
                             <sup>
                              μ
                             </sup>
                             of
                             <a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">
                              eq. (11)
                             </a>
                             . We benchmark against the four approaches considered in
                             <a id="xref-sec-7-4" class="xref-sec" href="#sec-7">
                              Section 2.4
                             </a>
                             . We start with results for the standard Lasso on simulated data (
                             <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">
                              Section 3.1
                             </a>
                             ), followed by the elastic net (
                             <a id="xref-sec-11-1" class="xref-sec" href="#sec-11">
                              Section 3.2
                             </a>
                             ). Further experiments on simulated data with the fused Lasso and the graphical Lasso are reported in Section C of the Supporting Information. The quality of the theoretical bound stated in
                             <a id="xref-statement-8-3" class="xref-statement" href="#statement-8">
                              Proposition 5
                             </a>
                             is assessed in
                             <a id="xref-sec-12-2" class="xref-sec" href="#sec-12">
                              Section 3.3
                             </a>
                             .
                            </p>
                            <p id="p-77">
                             We implement the three methodologies we developed in this paper:
                            </p>
                            <ol class="list-ord" id="list-3">
                             <li id="list-item-10">
                              <p id="p-78">
                               We carry out the minimization of the unsmoothed Φ of
                               <a id="xref-disp-formula-1-21" class="xref-disp-formula" href="#disp-formula-1">
                                eq. (1)
                               </a>
                               using R’s function optim. Within the optim function, we select the quasi-Newton method BFGS for which we supply its explicit (though non-smooth) gradient
                               <span class="disp-formula" id="disp-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the sign function is defined as sign(z) = 1 for z > 0, sign(z) = −1 for z < 1, and zero otherwise. This approach will be referred to as unsmoothed Phi.
                              </p>
                             </li>
                             <li id="list-item-11">
                              <p id="p-79">
                               We minimize the smooth surrogate Φ
                               <sup>
                                μ
                               </sup>
                               of
                               <a id="xref-disp-formula-11-3" class="xref-disp-formula" href="#disp-formula-11">
                                eq. (11)
                               </a>
                               using its explicit gradient given in
                               <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">
                                eq. (12)
                               </a>
                               based on the entropy prox function of
                               <a id="xref-sec-5-2" class="xref-sec" href="#sec-5">
                                Section 2.2
                               </a>
                               . The smoothing parameter of the smooth surrogate Φ
                               <sup>
                                μ
                               </sup>
                               was fixed at μ = 0.1. This approach will be referred to as smoothed Phi.
                              </p>
                             </li>
                             <li id="list-item-12">
                              <p id="p-80">
                               We employ the progressive smoothing approach of
                               <a id="xref-sec-8-3" class="xref-sec" href="#sec-8">
                                Section 2.5
                               </a>
                               . As suggested at the end of
                               <a id="xref-sec-8-4" class="xref-sec" href="#sec-8">
                                Section 2.5
                               </a>
                               , we set the target smoothing parameter to μ
                               <sub>
                                0
                               </sub>
                               = 2
                               <sup>
                                −6
                               </sup>
                               (roughly the square root of the machine precision) and employ N = 9 smoothing steps (thus implying an initial value of the smoothing parameter of μ = 2
                               <sup>
                                −6
                               </sup>
                               · 2
                               <sup>
                                9
                               </sup>
                               = 2
                               <sup>
                                3
                               </sup>
                               ).
                              </p>
                             </li>
                            </ol>
                            <p id="p-81">
                             We benchmark our methods against the gold standards available in the literature for the four problems we consider:
                            </p>
                            <ol class="list-ord" id="list-4">
                             <li id="list-item-13">
                              <p id="p-82">
                               As a general method, we use the Fista algorithm to minimize Φ as implemented in the fasta R-package (
                               <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                                Chi et al., 2018
                               </a>
                               ). The main function of the fasta package which implements the Fista algorithm, also called fasta, requires the separate specification of the smooth and non-smooth parts of the objective function including their explicit gradients. We follow Example 2 in the vignette of the fasta R-package (
                               <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">
                                Chi et al., 2018
                               </a>
                               ) and supply both as specified in
                               <a id="xref-disp-formula-1-22" class="xref-disp-formula" href="#disp-formula-1">
                                eq. (1)
                               </a>
                               . Moreover, in contrast to our approach, a variety of tuning parameters need to be selected by the user, e.g. an initial starting value, an initial stepsize, parameters determining the lookback window for non-monotone line search and a shrinkage parameter for the stepsize. The initial stepsize for Fista is set to τ = 10 as in Example 2 of the package vignette (
                               <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">
                                Chi et al., 2018
                               </a>
                               ). The lookback window for non-monotone line search and the shrinkage parameter for the stepsize are left at their default values. We employ a uniformly random starting value for β ∈ ℝ
                               <sup>
                                p
                               </sup>
                               (each vector entry is drawn independently from U [0, 1]) as done for our own approaches.
                              </p>
                             </li>
                             <li id="list-item-14">
                              <p id="p-83">
                               For the elastic net comparison, we benchmark against the glmnet algorithm (
                               <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                                Friedman et al., 2010a
                               </a>
                               ), available in the R-package glmnet (
                               <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">
                                Friedman et al., 2020
                               </a>
                               ) on CRAN. The glmnet algorithm is a variant of Fista which performs a cyclic update of all coordinates, whereas Fista updates all coordinates per iteration. To solve an elastic net problem with glmnet, we set its parameter alpha to 0.5 as specified on page 24 of the package vignette (
                               <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">
                                Friedman et al., 2020
                               </a>
                               ).
                              </p>
                             </li>
                             <li id="list-item-15">
                              <p id="p-84">
                               For the fused Lasso, we employ the R-package genlasso (
                               <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">
                                Arnold & Tibshirani, 2020
                               </a>
                               ) on CRAN with default parameters. The fused Lasso is implemented in the function fusedlasso.
                              </p>
                             </li>
                             <li id="list-item-16">
                              <p id="p-85">
                               For the graphical Lasso, we employ the R-package glasso (
                               <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">
                                Friedman et al., 2019
                               </a>
                               ) on CRAN with default parameters. The function to run the graphical Lasso is likewise called glasso.
                              </p>
                             </li>
                            </ol>
                            <p id="p-86">
                             Two algorithms are unconsidered in this article for the following reasons. Since the R-package SIS (
                             <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                              Fan & Li, 2001
                             </a>
                             ) does itself rely on glmnet for computing regression estimates, we omit it in this section. The LARS algorithm (
                             <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                              Efron et al., 2004
                             </a>
                             ) is implemented in the R-package lars (
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              Hastie & Efron, 2013
                             </a>
                             ) on CRAN. As remarked in the literature (
                             <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                              Friedman et al., 2010a
                             </a>
                             ), LARS is slower than glmnet or Fista. Additionally, since the LARS implementation (
                             <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                              Hastie & Efron, 2013
                             </a>
                             ) always computes a full LASSO path, it is considerably slower than the aforementioned methods.
                            </p>
                            <p id="p-87">
                             All results are averages over 100 repetitions. The regularization parameters of
                             <a id="xref-sec-7-5" class="xref-sec" href="#sec-7">
                              Section 2.4
                             </a>
                             were fixed at λ = 1, α = 0.5, and γ = 0.5. We evaluate the accuracy of the aforementioned algorithms using two metrics. First, we report the L
                             <sub>
                              2
                             </sub>
                             norm of the true parameters β that we generate to the estimated ones returned by each method. Second, we report the standard vector correlation between true and estimated parameters. To assess computational efficiency and runtime scaling, we report log-log plots of the empirical runtimes we measure.
                            </p>
                            <div id="sec-10" class="subsection">
                             <h3>
                              3.1 Standard Lasso on simulated data
                             </h3>
                             <p id="p-88">
                              We start by applying our smoothing framework to the standard Lasso (
                              <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">
                               Tibshirani, 1996
                              </a>
                              ). To this end, we employ the choices of u, v, and w
                              <sub>
                               j
                              </sub>
                              , j ∈ {1, …, p}, for the standard Lasso given in
                              <a id="xref-sec-7-6" class="xref-sec" href="#sec-7">
                               Section 2.4
                              </a>
                              .
                             </p>
                             <p id="p-89">
                              We simulate a regression model of the type y = Xβ by drawing the rows of the design matrix X ∈ ℝ
                              <sup>
                               n×p
                              </sup>
                              from a multidimensional normal distribution with the following mean vector and covariance matrix. The entries of the p dimensional mean vector of the multidimensional normal distribution are drawn independently from a uniform distribution in [0, 1]. To ensure positive definiteness, the p × p dimensional covariance matrix of the multidimensional normal distribution is drawn from a Wishart distribution with sample size 1, p degrees of freedom, and scale matrix set to the p × p dimensional identity matrix.
                             </p>
                             <p id="p-90">
                              After generating X, we draw the true parameters β independently from a standard normal distribution. To ensure sparseness, we set all but nz ∈ {0, …, p} out of the p entries to zero. We fix nz = 0.2p in the entire simulation section. Finally, we calculate y ∈ ℝ
                              <sup>
                               n
                              </sup>
                              as y = Xβ + ϵ for some noise vector ϵ ∈ ℝ
                              <sup>
                               n
                              </sup>
                              . The entries of ϵ are generated independently from a Normal distribution with mean zero and some variance σ
                              <sup>
                               2
                              </sup>
                              . The smaller we choose σ
                              <sup>
                               2
                              </sup>
                              , the easier the recovery of β will be. We fix σ
                              <sup>
                               2
                              </sup>
                              = 0.1.
                             </p>
                             <p id="p-91">
                              We benchmark our three approaches (unsmoothed Φ, smoothed surrogate Φ
                              <sup>
                               μ
                              </sup>
                              , and progressive smoothing) against the Fista algorithm. We are interested in accuracy and runtime of all algorithms as a function of n (while keeping p = 100 fixed) or p (while keeping n = 100 fixed).
                             </p>
                             <p id="p-92">
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 1
                              </a>
                              shows results for the scaling in n while keeping p = 100 fixed. We observe that, as expected, estimation becomes easier as n becomes larger. Fista yields the best estimates when measuring the error in the L
                              <sub>
                               2
                              </sub>
                              norm, followed by the unsmoothed Lasso objective (unsmoothed Phi) and the progressive smoothing algorithm. Minimizing the smoothed Lasso objective (smoothed Phi) does not seem to have an advantage in this scenario. The correlation of the truth to the estimate seems equally high for almost all approaches. Moreover, we observe that all approaches have roughly the same runtime scaling in n. Since the progressive smoothing approach works by repeatedly minimizing the smoothed surrogate, its runtime is roughly a constant factor higher than the one for minimizing the smoothed surrogate Φ
                              <sup>
                               μ
                              </sup>
                              .
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F2.large.jpg?width=800&height=600&carousel=1" title="Standard Lasso: L2 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of n while p = 100. Runtime in seconds (right) has a log scale on both axes. A color version of this figure can be found in the electronic version of the article." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Standard Lasso: <em>L</em><sub>2</sub> norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of <em>n</em> while <em>p</em> = 100. Runtime in seconds (right) has a log scale on both axes. A color version of this figure can be found in the electronic version of the article.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F2.medium.gif" width="440" height="116"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F2.medium.gif" width="440" height="116"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <p id="p-93" class="first-child">
                                Standard Lasso: L
                                <sub>
                                 2
                                </sub>
                                norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of n while p = 100. Runtime in seconds (right) has a log scale on both axes. A color version of this figure can be found in the electronic version of the article.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-94">
                              The more important case is the scenario in which p ≫ n. This scenario, depicted in
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 2
                              </a>
                              , shows a different picture. Fista does not seem to approximate the truth well when measuring the error with the L
                              <sub>
                               2
                              </sub>
                              norm, whereas minimizing the smoothed surrogate (smoothed Phi) and progressive smoothing yield much more accurate results. The picture is confirmed when measuring accuracy with the correlation between truth and estimate. Here, using Fista and minimizing the unsmoothed objective function results in estimates with a considerably lower correlation to the true parameter vector than using both our smoothing approaches (smoothed Phi or progressive smoothing). As can be seen from the left and middle panels in
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 2
                              </a>
                              , minimizing the unsmoothed Lasso objective Φ is more prone to instabilities due to its non-differentiability. This is to be expected and motivates our smoothing approach. Not surprisingly, while keeping the data size n fixed, the estimation becomes more challenging as p increases. The runtime scaling of our three methods is again similar, though Fista seems to have a slightly more favorable runtime scaling in p.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F3.large.jpg?width=800&height=600&carousel=1" title="Standard Lasso: L2 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of p while n = 100. Runtime in seconds (right) has a log scale on both axes. A color version of this figure can be found in the electronic version of the article." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Standard Lasso: <em>L</em><sub>2</sub> norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of <em>p</em> while <em>n</em> = 100. Runtime in seconds (right) has a log scale on both axes. A color version of this figure can be found in the electronic version of the article.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F3.medium.gif" width="440" height="116"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F3.medium.gif" width="440" height="116"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <p id="p-95" class="first-child">
                                Standard Lasso: L
                                <sub>
                                 2
                                </sub>
                                norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of p while n = 100. Runtime in seconds (right) has a log scale on both axes. A color version of this figure can be found in the electronic version of the article.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              3.2 Elastic net on simulated data
                             </h3>
                             <p id="p-96">
                              We repeat the experiments of
                              <a id="xref-sec-10-2" class="xref-sec" href="#sec-10">
                               Section 3.1
                              </a>
                              for the elastic net (
                              <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">
                               Zou & Hastie, 2005
                              </a>
                              ) using the R-package glmnet (
                              <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">
                               Friedman et al., 2010a
                              </a>
                              ). For this, we employ the choices of u, v, and w
                              <sub>
                               j
                              </sub>
                              , j ∈ {1, …, p}, for the elastic net given in
                              <a id="xref-sec-7-7" class="xref-sec" href="#sec-7">
                               Section 2.4
                              </a>
                              . We generate models y = Xβ as described in
                              <a id="xref-sec-10-3" class="xref-sec" href="#sec-10">
                               Section 3.1
                              </a>
                              , and evaluate Fista, glmnet, unsmoothed Φ, smoothed surrogate Φ
                              <sup>
                               μ
                              </sup>
                              , and progressive smoothing with respect to their accuracy and scaling behavior in n and p.
                             </p>
                             <p id="p-97">
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 3
                              </a>
                              shows scaling results for n while keeping p = 100 fixed. We observe that glmnet and the simple smoothing approach (smoothed Phi) do not seem to perform well in this experiment, while Fista as well as the minimization of the unsmoothed objective Φ and progressive smoothing perform notably better. When looking at the correlation between truth and estimate, all methods with the exception of glmnet and the smoothed surrogate Φ
                              <sup>
                               μ
                              </sup>
                              (smoothed Phi) again perform similarly. The runtime of all methods does not seem very dependent on n.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F4.large.jpg?width=800&height=600&carousel=1" title="Elastic net: L2 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of n while p = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes Fista, glmnet, unsmoothed Φ, smoothed surrogate Φμ, and progressive smoothing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Elastic net: <em>L</em><sub>2</sub> norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of <em>n</em> while <em>p</em> = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes Fista, glmnet, unsmoothed Φ, smoothed surrogate Φ<sup><em>μ</em></sup>, and progressive smoothing.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F4.medium.gif" width="440" height="116"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F4.medium.gif" width="440" height="116"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <p id="p-98" class="first-child">
                                Elastic net: L
                                <sub>
                                 2
                                </sub>
                                norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of n while p = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes Fista, glmnet, unsmoothed Φ, smoothed surrogate Φ
                                <sup>
                                 μ
                                </sup>
                                , and progressive smoothing.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-99">
                              Analogously,
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 4
                              </a>
                              presents scaling results for the more interesting case that p ≫ n while keeping n = 100 fixed. These results confirm the observations made for the case of the standard Lasso in
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 2
                              </a>
                              . As expected, the estimation of the parameters becomes more challenging for all methods as p increases while n stays fixed.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F5.large.jpg?width=800&height=600&carousel=1" title="Elastic net: L2 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of p while n = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes Fista, glmnet, unsmoothed Φ, smoothed surrogate Φμ, and progressive smoothing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Elastic net: <em>L</em><sub>2</sub> norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of <em>p</em> while <em>n</em> = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes Fista, glmnet, unsmoothed Φ, smoothed surrogate Φ<sup><em>μ</em></sup>, and progressive smoothing.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F5.medium.gif" width="440" height="118"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F5.medium.gif" width="440" height="118"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <p id="p-100" class="first-child">
                                Elastic net: L
                                <sub>
                                 2
                                </sub>
                                norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of p while n = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes Fista, glmnet, unsmoothed Φ, smoothed surrogate Φ
                                <sup>
                                 μ
                                </sup>
                                , and progressive smoothing.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-101">
                              Fista seems to yield estimates of poorest quality when assessing accuracy with the L
                              <sub>
                               2
                              </sub>
                              distance between truth and estimate, while the other approaches perform more favorably. Regarding the correlation between truth and estimate, Fista again performs suboptimally, while glmnet and the minimization of the unsmoothed Φ (unsmoothed Phi) perform better. Using our smoothed surrogate (smoothed Phi) and the progressive smoothing algorithm perform best in that they retain the largest correlation between their estimates to the true parameters. As expected, the runtime scaling in p of Fista and glmnet is roughly similar, and the asymptotic runtimes of both seem to be lower than for our methods. Minimizing the unsmoothed objective function and the smooth surrogate seems to have an almost identical asymptotic runtime, while progressive smoothing is a constant factor slower as expected.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              3.3 Verification of theoretical bounds
                             </h3>
                             <p id="p-102">
                              Finally, we verify the theoretical bounds derived in
                              <a id="xref-statement-8-4" class="xref-statement" href="#statement-8">
                               Proposition 5
                              </a>
                              for the case of the standard Lasso using simulated data. In this experiment, we therefore again employ the choices of u, v, and w
                              <sub>
                               j
                              </sub>
                              , j ∈ {1, …, p}, for the standard Lasso given in
                              <a id="xref-sec-7-8" class="xref-sec" href="#sec-7">
                               Section 2.4
                              </a>
                              .
                             </p>
                             <p id="p-103">
                              First, we compute regression estimates with the unsmoothed and smoothed Lasso objective functions as done in
                              <a id="xref-sec-10-4" class="xref-sec" href="#sec-10">
                               Section 3.1
                              </a>
                              . We employ a regularization parameter of λ = 1, and a smoothing parameter of μ = 1. The generation of the model y = Xβ was done as described in
                              <a id="xref-sec-10-5" class="xref-sec" href="#sec-10">
                               Section 3.1
                              </a>
                              .
                             </p>
                             <p id="p-104">
                              After performing the minimization we record ∥ x
                              <sub>
                               1
                              </sub>
                              − x
                              <sub>
                               2
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              , the left hand side of
                              <a id="xref-disp-formula-14-2" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              . This quantity is assumed to be unknown as we aim to avoid the computation of the minimizer x
                              <sub>
                               2
                              </sub>
                              of the unsmoothed objective and solely focus on the minimizer x
                              <sub>
                               1
                              </sub>
                              of the smoothed surrogate instead.
                             </p>
                             <p id="p-105">
                              Moreover, we compute the right hand side of
                              <a id="xref-disp-formula-14-3" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              , given by the expression
                              <span class="inline-formula" id="inline-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . C
                              <sub>
                               δ
                              </sub>
                              (δL
                              <sub>
                               δ
                              </sub>
                              + 2ϵ) + C
                              <sub>
                               δ
                              </sub>
                              δ In the case of the standard Lasso considered here, F
                              <sub>
                               1
                              </sub>
                              is the differentiable and strictly convex function Φ
                              <sup>
                               μ
                              </sup>
                              , that is the smooth surrogate of the standard Lasso (see the discussion at the end of
                              <a id="xref-sec-6-4" class="xref-sec" href="#sec-6">
                               Section 2.3
                              </a>
                              ). The parameter δ is chosen by the user: for more stable results, we compute the bound of
                              <a id="xref-disp-formula-14-4" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              on the grid δ ∈ {0.1, 0.2, …, 1} and take the minimal value. The value of is given by
                              <a id="xref-disp-formula-8-3" class="xref-disp-formula" href="#disp-formula-8">
                               eq. (8)
                              </a>
                              . Taking the supremum over the parameter y
                              <sub>
                               1
                              </sub>
                              on the sphere S
                              <sub>
                               δ
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ) is performed in a Monte Carlo fashion by computing the bound of
                              <a id="xref-disp-formula-14-5" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              for random points generated uniformly on S
                              <sub>
                               δ
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ) and taking the maximum. Last, C
                              <sub>
                               δ
                              </sub>
                              can be computed explicitly using a scalar product between two known quantities as shown in the proof of
                              <a id="xref-statement-8-5" class="xref-statement" href="#statement-8">
                               Proposition 5
                              </a>
                              , and L
                              <sub>
                               δ
                              </sub>
                              is taken to be the local Lipschitz constant of Φ
                              <sup>
                               μ
                              </sup>
                              at its minimizer x
                              <sub>
                               1
                              </sub>
                              . This makes the right hand side of
                              <a id="xref-disp-formula-14-6" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              explicitly computable without knowledge of the minimizer x
                              <sub>
                               2
                              </sub>
                              of the unsmoothed (original) Lasso objective.
                             </p>
                             <p id="p-106">
                              Results for the left and right hand sides of
                              <a id="xref-disp-formula-14-7" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              are shown in
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 5
                              </a>
                              , once for the scaling in n while p = 100 is kept fixed, and once for the scaling in p while n = 100 is kept fixed. As expected, the estimation becomes easier as n increases for a fixed p, and more challenging as p increases for a fixed n.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F6.large.jpg?width=800&height=600&carousel=1" title="True L2 norm between minimizers of unsmoothed and smoothed Lasso (left hand side of eq. (13)) in black, and theoretical upper bound (right hand side of eq. (13)) in red. Scaling in n while p = 100 is fixed (left) and scaling in p while n = 100 is fixed (right). Log scale on the y-axis. A color version of this figure can be found in the electronic version of the article." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">True <em>L</em><sub>2</sub> norm between minimizers of unsmoothed and smoothed Lasso (left hand side of eq. (13)) in black, and theoretical upper bound (right hand side of eq. (13)) in red. Scaling in <em>n</em> while <em>p</em> = 100 is fixed (left) and scaling in <em>p</em> while <em>n</em> = 100 is fixed (right). Log scale on the y-axis. A color version of this figure can be found in the electronic version of the article.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F6.medium.gif" width="440" height="191"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F6.medium.gif" width="440" height="191"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <p id="p-107" class="first-child">
                                True L
                                <sub>
                                 2
                                </sub>
                                norm between minimizers of unsmoothed and smoothed Lasso (left hand side of
                                <a id="xref-disp-formula-14-8" class="xref-disp-formula" href="#disp-formula-14">
                                 eq. (13)
                                </a>
                                ) in black, and theoretical upper bound (right hand side of
                                <a id="xref-disp-formula-14-9" class="xref-disp-formula" href="#disp-formula-14">
                                 eq. (13)
                                </a>
                                ) in red. Scaling in n while p = 100 is fixed (left) and scaling in p while n = 100 is fixed (right). Log scale on the y-axis. A color version of this figure can be found in the electronic version of the article.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-108">
                              Overall, we observe that
                              <a id="xref-disp-formula-14-10" class="xref-disp-formula" href="#disp-formula-14">
                               eq. (13)
                              </a>
                              indeed provides a valid, explicit upper bound on the error between the unsmoothed and smoothed minimizers for each fixed choice of μ. For the scaling in n, the bound is very conservative, but in the more challenging scenario of the scaling in p, the bound seems to accurately follow the error we make by using the smoothed surrogate. Note that by Propositions 3 and 5, this error will eventually go to zero as the smoothing parameter μ → 0.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             4 Discussion
                            </h2>
                            <p id="p-109">
                             In this paper, we propose a smoothing framework for a general class of L
                             <sub>
                              1
                             </sub>
                             penalized regression operators that includes most common approaches, e.g. the Lasso, elastic net, fused and graphical Lasso, etc. Throughout the article, we consider a fixed parameter setting (parameters n, m, p are fixed, see
                             <a id="xref-sec-1-2" class="xref-sec" href="#sec-1">
                              Section 1
                             </a>
                             ). The framework is based on a closed-form smooth surrogate and its closed-form gradient. Since the aforementioned regression operators are convex but non-smooth, due to the non-differentiability of their L
                             <sub>
                              1
                             </sub>
                             norm, minimizing the smooth surrogates instead allows for a fast and efficient computation of regression estimates.
                            </p>
                            <p id="p-110">
                             Most importantly, we prove a series of theoretical results which apply to the entire class of regression operators belonging to our framework. Under regularity conditions, we show that the smooth surrogate is both strictly convex and uniformly close (in supremum norm) to the original (unsmoothed) objective function. This has the important implication that the regression estimates obtained by minimizing the smooth surrogate can be made arbitrarily close to the ones of the original (unsmoothed) objective function. Additionally, we provide explicitly computable error bounds on the accuracy of the estimates obtained with the smooth surrogate. Since we can efficiently carry out the minimization of the smooth surrogate, our approach yields easily computable regression estimates that are both guaranteed to be close to the estimates of the original objective function, and allow for a priori error estimation. Furthermore, we develop a progressive smoothing algorithm for iterative smoothing. This approach is virtually free of tuning parameters. Its runtime is only a constant factor higher than that of the other algorithms (due to repeated optimization calls).
                            </p>
                            <p id="p-111">
                             Smoothing the L
                             <sub>
                              1
                             </sub>
                             penalty of the aforementioned regression operators typically causes the resulting regression estimates to be dense. Although the regression estimates for those variables which are shrunk to zero by the original regression operators (with L
                             <sub>
                              1
                             </sub>
                             penalty) will typically be very small, smoothing the L
                             <sub>
                              1
                             </sub>
                             penalty in our framework thus somewhat leads to a loss of the variable selection property. However, an easy fix to re-establish sparse estimates and enable variable selection is to apply some form of thresholding.
                            </p>
                            <p id="p-112">
                             Our experimental results support our theoretical conclusions that minimizing the proposed smooth surrogate yields estimates of equal or better quality than many of the gold standard approaches available in the literature (the Fista, glmnet, gLasso algorithms, etc.). At the same time, our approach has roughly the same runtime scaling as the established approaches we include in our comparisons while keeping all aforementioned theoretical guarantees.
                            </p>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             Funding
                            </h2>
                            <p id="p-113">
                             The initial methodology work for this paper was funded by Cure Alzheimer’s Fund; Funding for this research was provided through the National Heart, Lung, and Blood Institute [U01HL089856, U01HL089897, P01HL120839, P01HL132825, 2U01HG008685] and National Human Genome Research Institute [R01HG008976].
                            </p>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             Supporting Information
                            </h2>
                            <p id="p-114">
                             The Supporting Information contains a detailed overview of Nesterov smoothing (Section A), all proofs (Section B), as well as additional simulations (Section C).
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             Supporting Information
                            </h2>
                            <div class="section" id="sec-17">
                             <h3>
                              A Nesterov smoothing
                             </h3>
                             <p id="p-115">
                              This section follows
                              <a id="xref-sec-3-4" class="xref-sec" href="#sec-3">
                               Sections 2
                              </a>
                              and
                              <a id="xref-sec-13-2" class="xref-sec" href="#sec-13">
                               4
                              </a>
                              of Nesterov’s published work (
                              <a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">
                               Nesterov, 2005
                              </a>
                              ). It introduces the basic formalism of Nesterov smoothing in
                              <a id="xref-sec-18-2" class="xref-sec" href="#sec-18">
                               Section A.1
                              </a>
                              and concretizes the approach in
                              <a id="xref-sec-19-3" class="xref-sec" href="#sec-19">
                               Section A.2
                              </a>
                              .
                             </p>
                             <div class="section" id="sec-18">
                              <h4>
                               A.1 Description of Nesterov smoothing
                              </h4>
                              <p id="p-116">
                               We are given a piecewise affine and convex function f : ℝ
                               <sup>
                                q
                               </sup>
                               → ℝ which we aim to smooth, where q ∈ ℕ. We assume that f is composed of k ∈ ℕ linear pieces (components). The function f can be expressed as
                               <span class="disp-formula" id="disp-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where A ∈ ℝ
                               <sup>
                                k×(q+1)
                               </sup>
                               is a matrix whose rows contain the linear coefficients for each of the k pieces (with the constant coefficients being in column q + 1), z ∈ ℝ
                               <sup>
                                q
                               </sup>
                               , and [z, 1] ∈ ℝ
                               <sup>
                                q+1
                               </sup>
                               denotes the vector obtained by concatenating z and the scalar 1 (see
                               <a id="xref-sec-3-5" class="xref-sec" href="#sec-3">
                                Section 2
                               </a>
                               ).
                              </p>
                              <p id="p-117">
                               Let ∥ · ∥
                               <sup>
                                k
                               </sup>
                               be a norm on ℝ
                               <sup>
                                k
                               </sup>
                               and ⟨ ·, · ⟩ be the Euclidean inner product. Define the unit simplex Q
                               <sub>
                                k
                               </sub>
                               ⊆ ℝ
                               <sup>
                                k
                               </sup>
                               <span class="disp-formula" id="disp-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-118">
                               To introduce the smoothing procedure, a proximity function (or prox function) is defined first on Q
                               <sub>
                                k
                               </sub>
                               (
                               <a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">
                                Nesterov, 2005
                               </a>
                               ). A prox function ρ is any nonnegative, continuously differentiable, and strongly convex function (with parameter σ > 0 and with respect to the norm ∥ · ∥
                               <sub>
                                k
                               </sub>
                               ). The latter means that ρ satisfies
                               <span class="disp-formula" id="disp-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for all s, t ∈ Q
                               <sub>
                                k
                               </sub>
                               .
                              </p>
                              <p id="p-119">
                               For any μ > 0, consider the function
                               <span class="disp-formula" id="disp-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-120">
                               According to Theorem 1 in Nesterov’s original publication (
                               <a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">
                                Nesterov, 2005
                               </a>
                               ), the function f
                               <sup>
                                μ
                               </sup>
                               defined in
                               <a id="xref-disp-formula-23-1" class="xref-disp-formula" href="#disp-formula-23">
                                eq. (15)
                               </a>
                               is convex and everywhere differentiable in z for any μ > 0. The function f
                               <sup>
                                μ
                               </sup>
                               depends only on the parameter μ controlling the degree of smoothness. For μ = 0, we recover the original unsmoothed function since f (z). The gradient
                               <span class="inline-formula" id="inline-formula-48">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-48.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-48.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The gradient
                               <span class="inline-formula" id="inline-formula-49">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-49.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-49.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is Lipschitz continuous with a Lipschitz constant that is proportional to μ
                               <sup>
                                −1
                               </sup>
                               . A closed form expression of both the gradient and the Lipschitz constant are given in Theorem 1 in Nesterov’s original publication (
                               <a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">
                                Nesterov, 2005
                               </a>
                               ).
                              </p>
                              <p id="p-121">
                               Importantly, the function f
                               <sup>
                                μ
                               </sup>
                               is a uniform smooth approximation of f = f
                               <sup>
                                0
                               </sup>
                               since
                               <span class="disp-formula" id="disp-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for all z ∈ ℝ
                               <sup>
                                q
                               </sup>
                               , meaning that the approximation error is uniformly upper bounded by
                               <span class="disp-formula" id="disp-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-122">
                               Indeed,
                               <a id="xref-disp-formula-24-1" class="xref-disp-formula" href="#disp-formula-24">
                                eq. (16)
                               </a>
                               holds true since for all z ∈ ℝ
                               <sup>
                                q
                               </sup>
                               ,
                               <span class="disp-formula" id="disp-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-32.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-32.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where it was used that both the function ρ and the parameter μ are nonnegative.
                              </p>
                             </div>
                             <div class="section" id="sec-19">
                              <h4>
                               A.2 Two choices for the proximity function
                              </h4>
                              <p id="p-123">
                               We consider two choices of the prox function ρ.
                              </p>
                              <div class="section" id="sec-20">
                               <h5>
                                A2.1. Entropy prox function
                               </h5>
                               <p id="p-124">
                                The entropy prox function ρ
                                <sub>
                                 e
                                </sub>
                                : ℝ
                                <sup>
                                 k
                                </sup>
                                → ℝ is given by
                                <span class="disp-formula" id="disp-formula-27">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-33.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-33.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for w ∈ ℝ
                                <sup>
                                 k
                                </sup>
                                .
                               </p>
                               <p id="p-125">
                                Setting the norm ∥ · ∥
                                <sub>
                                 k
                                </sub>
                                as the L
                                <sub>
                                 1
                                </sub>
                                norm in ℝ
                                <sup>
                                 k
                                </sup>
                                , Lemma 3 of Nesterov’s original publication (
                                <a id="xref-ref-30-13" class="xref-bibr" href="#ref-30">
                                 Nesterov, 2005
                                </a>
                                ) shows that ρ
                                <sub>
                                 e
                                </sub>
                                is strongly convex with respect to the L
                                <sub>
                                 1
                                </sub>
                                norm and satisfies
                                <span class="inline-formula" id="inline-formula-50">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-50.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-50.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                Using
                                <a id="xref-disp-formula-25-1" class="xref-disp-formula" href="#disp-formula-25">
                                 eq. (17)
                                </a>
                                , we obtain the uniform bound
                                <span class="disp-formula" id="disp-formula-28">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-34.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-34.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for the entropy smoothed function
                                <span class="inline-formula" id="inline-formula-51">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-51.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-51.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                obtained by using ρ
                                <sub>
                                 e
                                </sub>
                                in
                                <a id="xref-disp-formula-23-2" class="xref-disp-formula" href="#disp-formula-23">
                                 eq. (15)
                                </a>
                                . Interestingly, smoothing with the entropy prox function admits a closed-form expression of
                                <span class="inline-formula" id="inline-formula-52">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-52.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-52.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                given by
                                <span class="disp-formula" id="disp-formula-29">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-35.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-35.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                see Lemma 4 of Nesterov’s original publication (
                                <a id="xref-ref-30-14" class="xref-bibr" href="#ref-30">
                                 Nesterov, 2005
                                </a>
                                ).
                               </p>
                              </div>
                              <div class="section" id="sec-21">
                               <h5>
                                A.2.2 Squared error prox function
                               </h5>
                               <p id="p-126">
                                The squared error prox function is given by
                                <span class="disp-formula" id="disp-formula-30">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-36.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-36.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-127">
                                It has been shown (
                                <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                 Mazumder et al., 2019
                                </a>
                                ) that the optimization in
                                <a id="xref-disp-formula-23-3" class="xref-disp-formula" href="#disp-formula-23">
                                 eq. (15)
                                </a>
                                with squared error prox function is equivalent to the convex program
                                <span class="disp-formula" id="disp-formula-31">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-37.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-37.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                where c
                                <sub>
                                 i
                                </sub>
                                (z) = 1/μ · (A[z, 1]
                                <sup>
                                 T
                                </sup>
                                )
                                <sub>
                                 i
                                </sub>
                                − 1/k depends on A and μ and is defined for any i ∈ {1, …, k}. The problem in
                                <a id="xref-disp-formula-31-1" class="xref-disp-formula" href="#disp-formula-31">
                                 eq. (18)
                                </a>
                                is equivalent to finding the Euclidean projection of the vector c(z) = (c
                                <sub>
                                 1
                                </sub>
                                (z), …, c
                                <sub>
                                 k
                                </sub>
                                (z)) ∈ ℝ
                                <sup>
                                 k
                                </sup>
                                onto the k-dimensional unit simplex Q
                                <sub>
                                 k
                                </sub>
                                . This projection can be carried out efficiently using the Michelot algorithm (
                                <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                                 Michelot, 1986
                                </a>
                                ), for which a computationally more efficient version was proposed in the literature (
                                <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                 Wang & Carreira-Perpiñán, 2013
                                </a>
                                ) that we use in our implementations. Denoting the Euclidean projection of the vector c(z) onto Q
                                <sub>
                                 k
                                </sub>
                                as vector ĉ(z), the squared error prox approximation of f can be written as
                                <span class="disp-formula" id="disp-formula-32">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-38.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-38.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-128">
                                As
                                <span class="inline-formula" id="inline-formula-53">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-53.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-53.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                (see Section 4.1 in Nesterov’s original publication
                                <a id="xref-ref-30-15" class="xref-bibr" href="#ref-30">
                                 Nesterov (2005)
                                </a>
                                ), we obtain the uniform bound
                                <span class="disp-formula" id="disp-formula-33">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-39.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-39.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for the squared error smoothing approach.
                               </p>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              B Proofs
                             </h3>
                             <div id="statement-10" class="statement">
                              <span class="statement-label">
                               Proposition 6.
                              </span>
                              <p id="p-129">
                               The entropy prox smoothed function
                               <span class="inline-formula" id="inline-formula-54">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-54.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-54.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               of
                               <a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">
                                eq. (7)
                               </a>
                               and the squared error prox smoothed
                               <span class="inline-formula" id="inline-formula-55">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-55.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-55.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               of
                               <a id="xref-disp-formula-9-2" class="xref-disp-formula" href="#disp-formula-9">
                                eq. (9)
                               </a>
                               are strictly convex functions.
                              </p>
                             </div>
                             <div id="statement-11" class="statement">
                              <span class="statement-label">
                               Proof of Proposition 6.
                              </span>
                              <p id="p-130">
                               The second derivative of
                               <span class="inline-formula" id="inline-formula-56">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-56.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-56.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is given by
                               <span class="disp-formula" id="disp-formula-34">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-40.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-40.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and hence always positive, thus making
                               <span class="inline-formula" id="inline-formula-57">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-57.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-57.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               strictly convex. Similar arguments show that
                               <span class="inline-formula" id="inline-formula-58">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-58.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-58.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is strictly convex. □
                              </p>
                             </div>
                             <div id="statement-12" class="statement">
                              <span class="statement-label">
                               Proof of Proposition 3.
                              </span>
                              <p id="p-131">
                               Using the fact that according to
                               <a id="xref-statement-5-5" class="xref-statement" href="#statement-5">
                                Condition 3
                               </a>
                               , v is Lipschitz continuous with Lipschitz constant L
                               <sub>
                                v
                               </sub>
                               , we have
                               <span class="disp-formula" id="disp-formula-35">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-41.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-41.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               thus establishing the claimed bound. □
                              </p>
                             </div>
                             <div id="statement-13" class="statement">
                              <span class="statement-label">
                               Proof of Proposition 4.
                              </span>
                              <p id="p-132">
                               Since F
                               <sub>
                                1
                               </sub>
                               is continuous and strictly convex, it lays in the Skorohod topology 𝒟
                               <sub>
                                K
                               </sub>
                               , see Definition 2.2 (
                               <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                                Seijo & Sen, 2011
                               </a>
                               ). According to Lemma 2.9 (
                               <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">
                                Seijo & Sen, 2011
                               </a>
                               ), the argmax functional is continuous at F
                               <sub>
                                1
                               </sub>
                               with respect to the supremum norm metric. □
                              </p>
                             </div>
                             <div id="statement-14" class="statement">
                              <span class="statement-label">
                               Proof of Proposition 5.
                              </span>
                              <p id="p-133">
                               Let y
                               <sub>
                                1
                               </sub>
                               ∈ S
                               <sub>
                                δ
                               </sub>
                               (x
                               <sub>
                                1
                               </sub>
                               ), y
                               <sub>
                                1
                               </sub>
                               ≠ x
                               <sub>
                                1
                               </sub>
                               , be such that x
                               <sub>
                                1
                               </sub>
                               , x
                               <sub>
                                2
                               </sub>
                               and y
                               <sub>
                                1
                               </sub>
                               are collinear.
                              </p>
                              <p id="p-134">
                               Since F
                               <sub>
                                1
                               </sub>
                               is differentiable, we know that ∇F
                               <sub>
                                1
                               </sub>
                               exists. Since F
                               <sub>
                                1
                               </sub>
                               is strictly convex, the minimum x
                               <sub>
                                1
                               </sub>
                               is unique and ∇F
                               <sub>
                                1
                               </sub>
                               (y
                               <sub>
                                1
                               </sub>
                               ) ≠ 0 as y
                               <sub>
                                1
                               </sub>
                               ≠ x
                               <sub>
                                1
                               </sub>
                               . Since F
                               <sub>
                                1
                               </sub>
                               is convex, the tangent at every point stays below the function. Thus considering the tangent at y
                               <sub>
                                1
                               </sub>
                               , we have for all z that
                               <span class="disp-formula" id="disp-formula-36">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-42.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-42.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and thus we can bound F
                               <sub>
                                1
                               </sub>
                               − ϵ from below as
                               <span class="disp-formula" id="disp-formula-37">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-43.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-43.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <p id="p-135">
                              Observe that since
                              <span class="inline-formula" id="inline-formula-59">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , and since F
                              <sub>
                               1
                              </sub>
                              is strictly convex and F
                              <sub>
                               2
                              </sub>
                              is convex, the minimum x
                              <sub>
                               2
                              </sub>
                              of F
                              <sub>
                               2
                              </sub>
                              must satisfy F
                              <sub>
                               2
                              </sub>
                              (x
                              <sub>
                               2
                              </sub>
                              ) ≤ F
                              <sub>
                               1
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ) + ϵ. Moreover, since at x
                              <sub>
                               2
                              </sub>
                              we have F
                              <sub>
                               2
                              </sub>
                              (x
                              <sub>
                               2
                              </sub>
                              ) ∈ [F
                              <sub>
                               1
                              </sub>
                              (x
                              <sub>
                               2
                              </sub>
                              ) − ϵ, F
                              <sub>
                               1
                              </sub>
                              (x
                              <sub>
                               2
                              </sub>
                              ) + ϵ], since the tangent of
                              <a id="xref-disp-formula-37-1" class="xref-disp-formula" href="#disp-formula-37">
                               eq. (19)
                              </a>
                              stays below F
                              <sub>
                               1
                              </sub>
                              (x
                              <sub>
                               2
                              </sub>
                              ) − ϵ, and since x
                              <sub>
                               1
                              </sub>
                              , x
                              <sub>
                               2
                              </sub>
                              and y
                              <sub>
                               1
                              </sub>
                              are collinear, the solution z
                              <sub>
                               0
                              </sub>
                              satisfying
                              <span class="disp-formula" id="disp-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              has the property that x
                              <sub>
                               2
                              </sub>
                              cannot be further away from x
                              <sub>
                               1
                              </sub>
                              than z
                              <sub>
                               0
                              </sub>
                              , thus ∥x
                              <sub>
                               1
                              </sub>
                              − x
                              <sub>
                               2
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              ≤ ∥ x
                              <sub>
                               1
                              </sub>
                              − z
                              <sub>
                               0
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              . Rewriting z
                              <sub>
                               0
                              </sub>
                              − y
                              <sub>
                               1
                              </sub>
                              in
                              <a id="xref-disp-formula-38-1" class="xref-disp-formula" href="#disp-formula-38">
                               eq. (20)
                              </a>
                              as z
                              <sub>
                               0
                              </sub>
                              − x
                              <sub>
                               1
                              </sub>
                              + x
                              <sub>
                               1
                              </sub>
                              − y
                              <sub>
                               1
                              </sub>
                              and rearranging terms yields
                              <span class="disp-formula" id="disp-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-136">
                              Rewriting ∇F
                              <sub>
                               1
                              </sub>
                              (y
                              <sub>
                               1
                              </sub>
                              )
                              <sup>
                               T
                              </sup>
                              (z
                              <sub>
                               0
                              </sub>
                              − x
                              <sub>
                               1
                              </sub>
                              ) as ∥∇F
                              <sub>
                               1
                              </sub>
                              (y
                              <sub>
                               1
                              </sub>
                              ) ∥
                              <sub>
                               2
                              </sub>
                              ·∥z
                              <sub>
                               0
                              </sub>
                              − x
                              <sub>
                               1
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              · cos(θ) for some θ ∈ [0, π/2) and applying the L
                              <sub>
                               2
                              </sub>
                              norm on both sides yields, after applying the triangle inequality on the right hand side,
                              <span class="disp-formula" id="disp-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which after rearranging yields
                              <span class="disp-formula" id="disp-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-137">
                              We write
                              <span class="inline-formula" id="inline-formula-60">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-60.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-60.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and note that θ is determined by y
                              <sub>
                               1
                              </sub>
                              (and z) but independent of x
                              <sub>
                               2
                              </sub>
                              . Since x
                              <sub>
                               1
                              </sub>
                              is fixed, and F
                              <sub>
                               1
                              </sub>
                              is differentiable, it is also locally Lipschitz in a ball around x
                              <sub>
                               1
                              </sub>
                              (note that the Lipschitz constant is independent of x
                              <sub>
                               2
                              </sub>
                              ). Thus there exists
                              <span class="inline-formula" id="inline-formula-61">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-61.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-61.gif"/>
                                </noscript>
                               </span>
                              </span>
                              > 0 such that ∥ F
                              <sub>
                               1
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ) − F
                              <sub>
                               1
                              </sub>
                              (y
                              <sub>
                               1
                              </sub>
                              ) ∥
                              <sub>
                               2
                              </sub>
                              ≤
                              <span class="inline-formula" id="inline-formula-62">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-62.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/inline-graphic-62.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . This is true for any y
                              <sub>
                               1
                              </sub>
                              ∈ S
                              <sub>
                               δ
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ), thus by taking the supremum over y
                              <sub>
                               1
                              </sub>
                              ∈ S
                              <sub>
                               δ
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ) we arrive at two constants C
                              <sub>
                               δ
                              </sub>
                              > 0 and L
                              <sub>
                               δ
                              </sub>
                              > 0 (independent of y
                              <sub>
                               1
                              </sub>
                              ) which satisfy
                              <span class="disp-formula" id="disp-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where it was used that ∥ x
                              <sub>
                               1
                              </sub>
                              − y
                              <sub>
                               1
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              = δ since y
                              <sub>
                               1
                              </sub>
                              ∈ S
                              <sub>
                               δ
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ), and that ∥ x
                              <sub>
                               1
                              </sub>
                              − x
                              <sub>
                               2
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              ≤ ∥x
                              <sub>
                               1
                              </sub>
                              − z
                              <sub>
                               0
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              .
                             </p>
                             <p id="p-138">
                              The bound ∥ x
                              <sub>
                               1
                              </sub>
                              − x
                              <sub>
                               2
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              ≤ ∥ z
                              <sub>
                               0
                              </sub>
                              − x
                              <sub>
                               1
                              </sub>
                              ∥
                              <sub>
                               2
                              </sub>
                              is conditional on y
                              <sub>
                               1
                              </sub>
                              and x
                              <sub>
                               1
                              </sub>
                              , x
                              <sub>
                               2
                              </sub>
                              being collinear. Since x
                              <sub>
                               2
                              </sub>
                              is unknown, we take the supremum over all y
                              <sub>
                               1
                              </sub>
                              ∈ S
                              <sub>
                               δ
                              </sub>
                              (x
                              <sub>
                               1
                              </sub>
                              ), resulting in
                              <span class="disp-formula" id="disp-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/26/2020.09.17.301788/embed/graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              thus establishing the claimed upper bound. □
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              C Additional results on simulated data
                             </h3>
                             <p id="p-139">
                              This section presents additional simulations using the simulation setup of
                              <a id="xref-sec-10-6" class="xref-sec" href="#sec-10">
                               Section 3.1
                              </a>
                              .
                             </p>
                             <div class="section" id="sec-24">
                              <h4>
                               C.1 Fused Lasso on simulated data
                              </h4>
                              <p id="p-140">
                               The fused Lasso (
                               <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">
                                Tibshirani et al., 2005
                               </a>
                               ) is provided in the fusedlasso function of the R-package genlasso (
                               <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">
                                Arnold & Tibshirani, 2020
                               </a>
                               ).
                              </p>
                              <p id="p-141">
                               For the fused Lasso, a dependency structure among the entries of the parameter vector β ∈ ℝ
                               <sup>
                                p
                               </sup>
                               has to be generated. This dependency structure is given by the adjacency matrix E, see
                               <a id="xref-sec-7-9" class="xref-sec" href="#sec-7">
                                Section 2.4
                               </a>
                               . We generate E at random with edge probability 0.5 in each repetition. In this experiment, we employ the choices of u, v, and w
                               <sub>
                                j
                               </sub>
                               , j ∈ {1, …, p}, for the fused Lasso given in
                               <a id="xref-sec-7-10" class="xref-sec" href="#sec-7">
                                Section 2.4
                               </a>
                               .
                              </p>
                              <p id="p-142">
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Figure 6
                               </a>
                               shows scaling results in n while keeping p = 100 fixed. We observe that the functionfusedlasso of the R-package genlasso seems to have troubles finding good estimates, while optimizing the objective function of the fused Lasso, given in
                               <a id="xref-sec-7-11" class="xref-sec" href="#sec-7">
                                Section 2.4
                               </a>
                               , with our progressive smoothing algorithm works much better and yields stable regression estimates having a low L
                               <sub>
                                2
                               </sub>
                               norm and a high correlation with the truth. The fusedlasso function and minimizing the unsmoothed objective yield fastest results, followed by our smoothing approach. Progressive smoothing is a constant factor slower as expected. Overall, there only seems to be a weak runtime dependence on n.
                              </p>
                              <div id="F7" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F7.large.jpg?width=800&height=600&carousel=1" title="Fused Lasso: L2 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of n while p = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes fusedlasso (R-package genlasso), unsmoothed Φ, smoothed surrogate Φμ, and progressive smoothing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Fused Lasso: <em>L</em><sub>2</sub> norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of <em>n</em> while <em>p</em> = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes fusedlasso (R-package genlasso), unsmoothed Φ, smoothed surrogate Φ<sup><em>μ</em></sup>, and progressive smoothing.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F7.medium.gif" width="440" height="116"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F7.medium.gif" width="440" height="116"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 6:
                                </span>
                                <p id="p-143" class="first-child">
                                 Fused Lasso: L
                                 <sub>
                                  2
                                 </sub>
                                 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of n while p = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes fusedlasso (R-package genlasso), unsmoothed Φ, smoothed surrogate Φ
                                 <sup>
                                  μ
                                 </sup>
                                 , and progressive smoothing.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-144">
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                Figure 7
                               </a>
                               shows similar results for the scaling in p. Here, the fusedlasso function again yields estimates with the largest deviation from the generated truth in the L
                               <sub>
                                2
                               </sub>
                               norm, and also the lowest correlation with the true parameters. Progressive smoothing yields the most accurate and stable results with respect to both L
                               <sub>
                                2
                               </sub>
                               norm and correlation, while the other two approaches (unsmoothed objective and smooth surrogate, denoted as unsmoothed Phi and smoothed Phi) exhibit a more unstable behavior. The runtime scaling of all methods is roughly similar in p.
                              </p>
                              <div id="F8" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F8.large.jpg?width=800&height=600&carousel=1" title="Fused Lasso: L2 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of p while n = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes fusedlasso (R-package genlasso), unsmoothed Φ, smoothed surrogate Φμ, and progressive smoothing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Fused Lasso: <em>L</em><sub>2</sub> norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of <em>p</em> while <em>n</em> = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes fusedlasso (R-package genlasso), unsmoothed Φ, smoothed surrogate Φ<sup><em>μ</em></sup>, and progressive smoothing.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F8.medium.gif" width="440" height="116"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F8.medium.gif" width="440" height="116"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 7:
                                </span>
                                <p id="p-145" class="first-child">
                                 Fused Lasso: L
                                 <sub>
                                  2
                                 </sub>
                                 norm (left) and correlation (middle) between simulated truth and parameter estimates as a function of p while n = 100. Runtime in seconds (right) has a log scale on both axes. The comparison includes fusedlasso (R-package genlasso), unsmoothed Φ, smoothed surrogate Φ
                                 <sup>
                                  μ
                                 </sup>
                                 , and progressive smoothing.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              C.2 Graphical Lasso on simulated data
                             </h3>
                             <p id="p-146">
                              The graphical Lasso (
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               Friedman et al., 2008
                              </a>
                              ) is provided in the glasso function of the R-package glasso (
                              <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">
                               Friedman et al., 2019
                              </a>
                              ).
                             </p>
                             <p id="p-147">
                              For the graphical Lasso, we generate a sample covariance matrix S from a Wishart distribution with sample size 1, p ∈ {5, …, 30} degrees of freedom, and scale matrix set to the p × p dimensional identity matrix, see
                              <a id="xref-sec-7-12" class="xref-sec" href="#sec-7">
                               Section 2.4
                              </a>
                              . To define Φ and its smooth surrogate Φ
                              <sup>
                               μ
                              </sup>
                              , we employ the functions u, v, and w
                              <sub>
                               j
                              </sub>
                              , j ∈ {1, …, m} with m = p(p + 1)/2, for the graphical Lasso given in
                              <a id="xref-sec-7-13" class="xref-sec" href="#sec-7">
                               Section 2.4
                              </a>
                              .
                             </p>
                             <p id="p-148">
                              We then compute estimates for the unsmoothed graphical Lasso objective function using the R-package glasso. Additionally, we minimize the unsmoothed Φ for the graphical Lasso as well as our smooth surrogate Φ
                              <sup>
                               μ
                              </sup>
                              using the BFGS algorithm in R’s function optim, and employ our progressive smoothing algorithm. To optimize over all positive definite matrices Θ in the objective function of the graphical Lasso (see
                              <a id="xref-sec-7-14" class="xref-sec" href="#sec-7">
                               Section 2.4
                              </a>
                              ), we parametrize Θ with its Cholesky decomposition. To be precise, we write Θ = CC
                              <sup>
                               T
                              </sup>
                              , where C is a lower triangular matrix. This ensures that Θ will always be positive definite. As Θ is fully parametrized via C, and C is lower diagonal, we actually only optimize over p(p + 1)/2 parameters.
                             </p>
                             <p id="p-149">
                              We report the correlation between the true and estimated precision matrices (for this the matrices are unfolded as vectors), as well as the runtime.
                             </p>
                             <p id="p-150">
                              Results are given in
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Figure 8
                              </a>
                              . The results show that estimates of the precision matrix found with the glasso R-package and by minimizing the unsmoothed objective function have a lower correlation with the generated truth than the estimates obtained with our two smoothing approaches. The implementation of the graphical Lasso in the R-package glasso seems highly optimized, resulting in fast runtimes. Our smoothing approaches are not as highly optimized and thus slower than glasso, though all three exhibit roughly a similar runtime scaling.
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F9.large.jpg?width=800&height=600&carousel=1" title="Graphical Lasso: correlation (left) between simulated truth and parameter estimates as a function of n. Runtime in seconds (right) has a log scale on both axes. The comparison includes glasso (R-package glasso), unsmoothed Φ, smoothed surrogate Φμ, and progressive smoothing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1586784865" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Graphical Lasso: correlation (left) between simulated truth and parameter estimates as a function of <em>n</em>. Runtime in seconds (right) has a log scale on both axes. The comparison includes glasso (R-package glasso), unsmoothed Φ, smoothed surrogate Φ<sup><em>μ</em></sup>, and progressive smoothing.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F9.medium.gif" width="440" height="177"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F9.medium.gif" width="440" height="177"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/26/2020.09.17.301788/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8:
                               </span>
                               <p id="p-151" class="first-child">
                                Graphical Lasso: correlation (left) between simulated truth and parameter estimates as a function of n. Runtime in seconds (right) has a log scale on both axes. The comparison includes glasso (R-package glasso), unsmoothed Φ, smoothed surrogate Φ
                                <sup>
                                 μ
                                </sup>
                                , and progressive smoothing.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               Simulations are updated.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.17.301788v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arnold
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  genlasso: Path Algorithm for Generalized Lasso Problems
                                 </span>
                                 .
                                 <span class="cit-source">
                                  R-package version 1.5
                                 </span>
                                 :
                                 <a href="https://cran.r-project.org/package=genlasso">
                                  https://cran.r-project.org/package=genlasso
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Banerjee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghaoui
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   d’Aspremont
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Model Selection Through Sparse Maximum Likelihood Estimation for Multivariate Gaussian or Binary Data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Machine Learning Research
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  485
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  516
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMachine%2BLearning%2BResearch%26rft.volume%253D9%26rft.spage%253D485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.3" data-doi="10.1137/080716542">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teboulle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A Fast Iterative Shrinkage-Thresholding Algorithm for Linear Inverse Problems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  SIAM J Imaging Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  183
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  202
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSIAM%2BJ%2BImaging%2BSciences%26rft.volume%253D2%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1137%252F080716542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1137/080716542&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000278101000009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teboulle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Smoothing And First Order Methods: A Unified Framework
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Siam J Optim
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  557
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  580
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSiam%2BJ%2BOptim%26rft.volume%253D22%26rft.spage%253D557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Becker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Candès
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Templates for convex cone problems with applications to sparse signal recovery
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Math Prog Comp
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  165
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  218
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMath%2BProg%2BComp%26rft.volume%253D3%26rft.spage%253D165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.6" data-doi="10.1093/biomet/asr043">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Belloni
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chernozhukov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Square-root lasso: pivotal recovery of sparse signals via conic programming
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biometrika
                                 </abbr>
                                 <span class="cit-vol">
                                  98
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  791
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  806
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252Fasr043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biomet/asr043&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000297366000003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carbonell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xing
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010a
                                 </span>
                                 ).
                                 <abbr class="cit-jnl-abbrev">
                                  Graph-Structured Multi-task Regression and an Efficient Optimization Method for General Fused Lasso
                                 </abbr>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1005.3579
                                 </span>
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  21
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carbonell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xing
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010b
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  An efficient proximal gradient method for general structured sparse learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Machine Learning Research
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carbonell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xing
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Smoothing proximal gradient method for general structured sparse regression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ann Appl Stat
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  719
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  752
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn%2BAppl%2BStat%26rft.volume%253D6%26rft.spage%253D719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.17.301788v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Studer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baraniuk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  fasta: Fast Adaptive Shrinkage/Thresholding Algorithm
                                 </span>
                                 .
                                 <span class="cit-source">
                                  R-package version 0.1.0
                                 </span>
                                 :
                                 <a href="https://cran.r-project.org/package=fasta">
                                  https://cran.r-project.org/package=fasta
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Daubechies
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Defrise
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  An iterative thresholding algorithm for linear inverse problems with a sparsity constraint
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Comm. Pure Appl. Math
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  57
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1413
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1457
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComm.%2BPure%2BAppl.%2BMath%26rft.volume%253D57%26rft.spage%253D1413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dondelinger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mukherjee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The joint lasso: high-dimensional regression for group structured data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biostatistics
                                 </abbr>
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  219
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  235
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.volume%253D21%26rft.spage%253D219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.13" data-doi="10.1214/009053604000000067">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Efron
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnstone
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Least angle regression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ann Stat
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  407
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  499
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn%2BStat%26rft.volume%253D32%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1214%252F009053604000000067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1214/009053604000000067&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000221411000001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.14" data-doi="10.1198/016214501753382273">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Variable Selection via Nonconcave Penalized Likelihood and its Oracle Properties
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Am Stat Assoc
                                 </abbr>
                                 <span class="cit-vol">
                                  96
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1348
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1360
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BAm%2BStat%2BAssoc%26rft.volume%253D96%26rft.spage%253D1348%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214501753382273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1198/016214501753382273&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000172728000028&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.15" data-doi="10.1093/biostatistics/kxm045">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Sparse inverse covariance estimation with the graphical lasso
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biostatistics
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  432
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  441
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxm045%26rft_id%253Dinfo%253Apmid%252F18079126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxm045&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18079126&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F26%2F2020.09.17.301788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000256977000005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010a
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Regularization Paths for Generalized Linear Models via Coordinate Descent
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Statistical Software
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D33%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.17" data-doi="10.1111/j.1467-9868.2005.00503.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010b
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Regularization Paths for Generalized Linear Modelsvia Coordinate Descent
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Stat Softw
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20808728&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F26%2F2020.09.17.301788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000275203200001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.17.301788v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  glasso: Graphical Lasso: Estimation of Gaussian Graphical Models
                                 </span>
                                 .
                                 <span class="cit-source">
                                  R-package version 1.11
                                 </span>
                                 :
                                 <a href="https://cran.r-project.org/package=glasso">
                                  https://cran.r-project.org/package=glasso
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.17.301788v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Narasimhan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  glmnet: Lasso and Elastic-Net Regularized Generalized Linear Models
                                 </span>
                                 .
                                 <span class="cit-source">
                                  R-package version 4.0
                                 </span>
                                 :
                                 <a href="https://cran.r-project.org/package=glmnet">
                                  https://cran.r-project.org/package=glmnet
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.17.301788v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Banerjee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Parameter Estimation and Inference in a Contin-uous Piecewise Linear Regression Model
                                 </span>
                                 .
                                 <a href="http://www.cantab.net/users/ghahn/preprints/PhaseRegMultiDim.pdf">
                                  http://www.cantab.net/users/ghahn/preprints/PhaseRegMultiDim.pdf
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lutz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laha
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cho
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Silverman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lange
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A fast and efficient smoothing approach to LASSO regression and an application in statistical genetics: polygenic risk scores for Chronic obstructive pulmonary disease (COPD)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stat Comput
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStat%2BComput%26rft.volume%253D31%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.17.301788v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lutz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laha
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lange
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  smoothedLasso: Smoothed LASSO Regres-sion via Nesterov Smoothing
                                 </span>
                                 .
                                 <span class="cit-source">
                                  R-package version 1.4
                                 </span>
                                 :
                                 <a href="https://cran.r-project.org/package=smoothedLasso">
                                  https://cran.r-project.org/package=smoothedLasso
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haselimashhadi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vinciotti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <abbr class="cit-jnl-abbrev">
                                  A Differentiable Alternative to the Lasso Penalty
                                 </abbr>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1609.04985
                                 </span>
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.09.17.301788v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Efron
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  lars: Least Angle Regression, Lasso and Forward Stagewise
                                 </span>
                                 .
                                 <span class="cit-source">
                                  R-package version 1.2
                                 </span>
                                 :
                                 <a href="https://cran.r-project.org/package=lars">
                                  https://cran.r-project.org/package=lars
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.09.17.301788v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Massias
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fercoq
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gramfort
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salmon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  Generalized Concomitant Multi-Task Lasso for Sparse Multimodal Regression
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In Proceedings of the 21st International Conference on Artificial Intelligence and Statistics (AISTATS) 2018, Lanzarote, Spain
                                 </span>
                                 , vol.
                                 <span class="cit-vol">
                                  84
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  PMLR
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mazumder
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Choudhury
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iyengar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A Computational Framework for Multivariate Convex Regression and Its Variants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Am Stat Assoc
                                 </abbr>
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  318
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  331
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BAm%2BStat%2BAssoc%26rft.volume%253D114%26rft.spage%253D318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Michelot
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A finite algorithm for finding the projection of a point onto the canonical simplex of Rn
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Optimiz Theory App
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  195
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  200
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BOptimiz%2BTheory%2BApp%26rft.volume%253D50%26rft.spage%253D195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ndiaye
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fercoq
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gramfort
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leclère
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salmon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Efficient Smoothed Con-comitant Lasso Estimation for High Dimensional Regression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In 7th International Conference on New Computational Methods for Inverse Problems
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nesterov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1983
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A method of solving a convex programming problem with convergence rate O(1/k
                                  <sup>
                                   2
                                  </sup>
                                  )
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Dokl Akad Nauk SSSR
                                 </abbr>
                                 <span class="cit-vol">
                                  269
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  543
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  547
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDokl%2BAkad%2BNauk%2BSSSR%26rft.volume%253D269%26rft.spage%253D543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nesterov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Smooth minimization of non-smooth functions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Math. Program. Ser. A
                                 </abbr>
                                 <span class="cit-vol">
                                  103
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  127
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  152
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMath.%2BProgram.%2BSer.%2BA%26rft.volume%253D103%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.09.17.301788v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  R Core Team
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  R: A Language and Environment for Statistical Computing
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  R Foundation for Stat Comp
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  Vienna, Austria
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seijo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A continuous mapping theorem for the smallest argmax functional
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Electron J Stat
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  421
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  439
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElectron%2BJ%2BStat%26rft.volume%253D5%26rft.spage%253D421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Regression Shrinkage and Selection Via the Lasso
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Roy Stat Soc B Met
                                 </abbr>
                                 <span class="cit-vol">
                                  58
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  267
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  288
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BRoy%2BStat%2BSoc%2BB%2BMet%26rft.volume%253D58%26rft.spage%253D267%26rft_id%253Dinfo%253Apmid%252FWOS%253AA1996TU3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=WOS:A1996TU3&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F26%2F2020.09.17.301788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saunders
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosset
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Knight
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Sparsity and Smoothness via the Fused Lasso
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Roy Stat Soc B Met
                                 </abbr>
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  91
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  108
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BRoy%2BStat%2BSoc%2BB%2BMet%26rft.volume%253D67%26rft.spage%253D91%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carreira-Perpiñán
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <abbr class="cit-jnl-abbrev">
                                  Projection onto the probability simplex: An efficient algorithm with a simple proof, and an application
                                 </abbr>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1309.1541
                                 </span>
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yuan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Model selection and estimation in regression with grouped variables
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Roy Stat Soc B Met
                                 </abbr>
                                 <span class="cit-vol">
                                  68
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  49
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  67
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BRoy%2BStat%2BSoc%2BB%2BMet%26rft.volume%253D68%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.17.301788v3.37" data-doi="10.1111/j.1467-9868.2005.00503.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Regularization and variable selection via the elastic net
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Roy Stat Soc B Met
                                 </abbr>
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  301
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  320
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BRoy%2BStat%2BSoc%2BB%2BMet%26rft.volume%253D67%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.09.17.301788v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.06.22.497031v1" title="Dam Site and Vegetation Selection by Reintroduced Beaver (Castor canadensis) on the San Pedro Riparian National Conservation Area, Arizona" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.10.483885v1" title="Cancer related subarachnoid hemorrhage: a multicenter retrospective study using propensity score matching analysis" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted June 26, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.09.17.301788v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.09.17.301788v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.09.17.301788v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   A framework to efficiently smooth L1 penalties for linear regression
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2613581"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-HzOYafikpVbpqZyW7cAX6nfFTVounfCneA8Vl322r14"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484479936"/>
                                   <input type="hidden" name="captcha_token" value="42875a7fb8ef930f15887ed0a6216683"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2613581--3" data-node-nid="2613581" id="node-2613581--4132994979" data-pisa="biorxiv;2020.09.17.301788v3" data-pisa-master="biorxiv;2020.09.17.301788" data-seqnum="2613581" data-apath="/biorxiv/early/2022/06/26/2020.09.17.301788.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 A framework to efficiently smooth L
                                 <sub>
                                  1
                                 </sub>
                                 penalties for linear regression
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Georg
                                  </span>
                                  <span class="nlm-surname">
                                   Hahn
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Sharon M.
                                  </span>
                                  <span class="nlm-surname">
                                   Lutz
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Nilanjana
                                  </span>
                                  <span class="nlm-surname">
                                   Laha
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Christoph
                                  </span>
                                  <span class="nlm-surname">
                                   Lange
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.09.17.301788;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.09.17.301788
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.09.17.301788v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&title=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&title=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&text=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&t=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&title=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&title=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&title=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2613581--5" data-node-nid="2613581" id="citation-node-2613581--6182072931" data-pisa="biorxiv;2020.09.17.301788v3" data-pisa-master="biorxiv;2020.09.17.301788" data-seqnum="2613581" data-apath="/biorxiv/early/2022/06/26/2020.09.17.301788.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     A framework to efficiently smooth L
                                     <sub>
                                      1
                                     </sub>
                                     penalties for linear regression
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Georg
                                      </span>
                                      <span class="nlm-surname">
                                       Hahn
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Sharon M.
                                      </span>
                                      <span class="nlm-surname">
                                       Lutz
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Nilanjana
                                      </span>
                                      <span class="nlm-surname">
                                       Laha
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Christoph
                                      </span>
                                      <span class="nlm-surname">
                                       Lange
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.09.17.301788;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.09.17.301788
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Hahn%2C+Georg&rft.au=Lutz%2C+Sharon+M.&rft.au=Laha%2C+Nilanjana&rft.au=Lange%2C+Christoph">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2613581/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2613581/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2613581/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2613581/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2613581/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2613581/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2613581/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2613581/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2613581/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2613581/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2613581/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2613581/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&count=horizontal&via=&text=A%20framework%20to%20efficiently%20smooth%20L1%20penalties%20for%20linear%20regression&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.09.17.301788v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.09.17.301788v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
