<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.08.11.245944v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.08.11.245944v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.08.11.245944v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/02/08/2020.08.11.245944.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.08.11.245944v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A flexible Bayesian framework for unbiased estimation of timescales"/>
  <meta name="DC.Identifier" content="10.1101/2020.08.11.245944"/>
  <meta name="DC.Date" content="2022-02-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Timescales characterize the pace of change for many dynamic processes in nature. Timescales are usually estimated by fitting the exponential decay of data autocorrelation in the time or frequency domain. We show that this standard procedure often fails to recover the correct timescales due to a statistical bias arising from the finite sample size. We develop an alternative approach to estimating timescales by fitting the sample autocorrelation or power spectrum with a generative model based on a mixture of Ornstein-Uhlenbeck processes using adaptive Approximate Bayesian Computations. Our method accounts for finite sample size and noise in data and returns a posterior distribution of timescales that quantifies the estimation uncertainty and can be used for model selection. We demonstrate the accuracy of our method on synthetic data and illustrate its application to recordings from the primate cortex. We provide a customizable Python package implementing our framework with different generative models suitable for diverse applications.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Roxana Zeraati"/>
  <meta name="DC.Contributor" content="Tatiana A. Engel"/>
  <meta name="DC.Contributor" content="Anna Levina"/>
  <meta name="article:published_time" content="2022-02-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A flexible Bayesian framework for unbiased estimation of timescales"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Timescales characterize the pace of change for many dynamic processes in nature. Timescales are usually estimated by fitting the exponential decay of data autocorrelation in the time or frequency domain. We show that this standard procedure often fails to recover the correct timescales due to a statistical bias arising from the finite sample size. We develop an alternative approach to estimating timescales by fitting the sample autocorrelation or power spectrum with a generative model based on a mixture of Ornstein-Uhlenbeck processes using adaptive Approximate Bayesian Computations. Our method accounts for finite sample size and noise in data and returns a posterior distribution of timescales that quantifies the estimation uncertainty and can be used for model selection. We demonstrate the accuracy of our method on synthetic data and illustrate its application to recordings from the primate cortex. We provide a customizable Python package implementing our framework with different generative models suitable for diverse applications.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.08.11.245944v3"/>
  <meta name="citation_id" content="2020.08.11.245944v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.08.11.245944v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.08.11.245944v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.08.11.245944v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.08.11.245944"/>
  <meta name="citation_num_pages" content="44"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.08.11.245944"/>
  <meta name="citation_author" content="Roxana Zeraati"/>
  <meta name="citation_author_institution" content="International Max Planck Research School for the Mechanisms of Mental Function and Dysfunction, University of Tübingen"/>
  <meta name="citation_author_institution" content="Max Planck Institute for Biological Cybernetics"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7946-1464"/>
  <meta name="citation_author" content="Tatiana A. Engel"/>
  <meta name="citation_author_institution" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_author_email" content="engel@cshl.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5842-9406"/>
  <meta name="citation_author" content="Anna Levina"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of Tübingen"/>
  <meta name="citation_author_institution" content="Max Planck Institute for Biological Cybernetics"/>
  <meta name="citation_author_institution" content="Bernstein Center for Computational Neuroscience Tübingen"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1355-6617"/>
  <meta name="citation_reference" content="Thomas Veitshans, Dmitri Klimov, and Devarajan Thirumalai. Protein folding kinetics: timescales, pathways and energy landscapes in terms of sequence-dependent properties. Folding and Design, 2(1):1–22, 1997."/>
  <meta name="citation_reference" content="Jan Kubelka, James Hofrichter, and William A Eaton. The protein folding ‘speed limit’. Current Opinion in Structural Biology, 14(1):76–88, 2004."/>
  <meta name="citation_reference" content="John D Murray, Alberto Bernacchia, David J Freedman, Ranulfo Romo, Jonathan D Wallis, Xinying Cai, Camillo Padoa-Schioppa, Tatiana Pasternak, Hyojung Seo, Daeyeol Lee, et al. A hierarchy of intrinsic timescales across primate cortex. Nature Neuroscience, 17(12):1661, 2014."/>
  <meta name="citation_reference" content="Yang Yang, Michael DeWeese, Gonzalo Otazu, and Anthony Zador. Millisecond-scale differences in neural activity in auditory cortex can drive decisions. Nature, pages 1–1, 2008."/>
  <meta name="citation_reference" content="Kamran Diba, Asohan Amarasingham, Kenji Mizuseki, and György Buzsáki. Millisecond timescale synchrony among hippocampal neurons. Journal of Neuroscience, 34(45):14984–14994, 2014."/>
  <meta name="citation_reference" content="Jarnail Singh and Richard A Padgett. Rates of in situ transcription and splicing in large human genes. Nature Structural & Molecular Biology, 16(11):1128, 2009."/>
  <meta name="citation_reference" content="Agnes Audibert, Dominique Weil, and Francois Dautry. In vivo kinetics of mrna splicing and transport in mammalian cells. Molecular and Cellular Biology, 22(19):6706–6718, 2002."/>
  <meta name="citation_reference" content="Maya Shamir, Yinon Bar-On, Rob Phillips, and Ron Milo. Snapshot: timescales in cell biology. Cell, 164(6):1302–1302, 2016."/>
  <meta name="citation_reference" content="Henry Markram, Joachim Lübke, Michael Frotscher, and Bert Sakmann. Regulation of synaptic efficacy by coincidence of postsynaptic aps and epsps. Science, 275(5297):213–215, 1997."/>
  <meta name="citation_reference" content="Guo-qiang Bi and Mu-ming Poo. Synaptic modifications in cultured hippocampal neurons: dependence on spike timing, synaptic strength, and postsynaptic cell type. Journal of Neuroscience, 18(24):10464–10472, 1998."/>
  <meta name="citation_reference" content="Dominique Debanne, Beat H Gähwiler, and Scott M Thompson. Long-term synaptic plasticity between pairs of individual ca3 pyramidal cells in rat hippocampal slice cultures. The Journal of Physiology, 507(Pt 1):237, 1998."/>
  <meta name="citation_reference" content="Krishnagopal Dharani. The biology of thought: A neuronal mechanism in the generation of thought-A new molecular model. Academic Press, 2014."/>
  <meta name="citation_reference" content="Denis Mollison and Mollison Denis. Epidemic models: their structure and relation to data, volume 5. Cambridge University Press, 1995."/>
  <meta name="citation_reference" content="Denis Mollison, Valerie Isham, and Bryan Grenfell. Epidemics: models and data. Journal of the Royal Statistical Society: Series A (Statistics in Society), 157(1):115–129, 1994."/>
  <meta name="citation_reference" content="Sang Woo Park, Daniel M Cornforth, Jonathan Dushoff, and Joshua S Weitz. The time scale of asymptomatic transmission affects estimates of epidemic potential in the covid-19 outbreak. Epidemics, 31:100392, 2020."/>
  <meta name="citation_reference" content="Richard P Cincotta, Jennifer Wisnewski, and Robert Engelman. Human population in the biodiversity hotspots. Nature, 404(6781):990–992, 2000."/>
  <meta name="citation_reference" content="Miikka Tallavaara, Miska Luoto, Natalia Korhonen, Heikki Järvinen, and Heikki Seppä. Human population dynamics in europe over the last glacial maximum. Proceedings of the National Academy of Sciences, 112(27):8232–8237, 2015."/>
  <meta name="citation_reference" content="Ulrike Rippke, Julian Ramirez-Villegas, Andy Jarvis, Sonja J Vermeulen, Louis Parker, Flora Mer, Bernd Diekkrüger, Andrew J Challinor, and Mark Howden. Timescales of transformational climate change adaptation in sub-saharan african agriculture. Nature Climate Change, 6(6):605–609, 2016."/>
  <meta name="citation_reference" content="Paul A Baker, Catherine A Rigsby, Geoffrey O Seltzer, Sherilyn C Fritz, Tim K Lowenstein, Niklas P Bacher, and Carlos Veliz. Tropical climate changes at millennial and orbital timescales on the bolivian altiplano. Nature, 409(6821):698–701, 2001."/>
  <meta name="citation_reference" content="Ryan V Raut, Abraham Z Snyder, and Marcus E Raichle. Hierarchical dynamics as a macroscopic organizing principle of the human brain. Proceedings of the National Academy of Sciences, 2020."/>
  <meta name="citation_reference" content="Mehran Moradi Spitmaan, Hyojung Seo, Daeyeol Lee, and Alireza Soltani. Multiple timescales of neural dynamics and integration of task-relevant signals across cortex. Proceedings of the National Academy of Sciences, 2020."/>
  <meta name="citation_reference" content="Richard Gao, Ruud L van den Brink, Thomas Pfeffer, and Bradley Voytek. Neuronal timescales are functionally dynamic and shaped by cortical microarchitecture. eLife, 9:e61277, 2020."/>
  <meta name="citation_reference" content="Takamitsu Watanabe, Geraint Rees, and Naoki Masuda. Atypical intrinsic neural timescale in autism. eLife, 8:e42256, 2019."/>
  <meta name="citation_reference" content="Roxana Zeraati, Yan-Liang Shi, Nicholas A. Steinmetz, Marc A. Gieselmann, Alexander Thiele, Tirin Moore, Anna Levina, and Tatiana A. Engel. Attentional modulation of intrinsic timescales in visual cortex and spatial networks. bioRxiv, May 2021."/>
  <meta name="citation_reference" content="Jens Wilting and Viola Priesemann. Inferring collective dynamical states from widely unobserved systems. Nature Communications, 9(1):1–7, 2018."/>
  <meta name="citation_reference" content="William Bialek. Biophysics: searching for principles. Princeton University Press, 2012."/>
  <meta name="citation_reference" content="William Bialek, Andrea Cavagna, Irene Giardina, Thierry Mora, Edmondo Silvestri, Massimiliano Viale, and Aleksandra M Walczak. Statistical mechanics for natural flocks of birds. Proceedings of the National Academy of Sciences, 109(13):4786–4791, 2012."/>
  <meta name="citation_reference" content="Gašper Tkačik, Thierry Mora, Olivier Marre, Dario Amodei, Stephanie E Palmer, Michael J Berry, and William Bialek. Thermodynamics and signatures of criticality in a network of neurons. Proceedings of the National Academy of Sciences, 112(37):11508–11513, 2015."/>
  <meta name="citation_reference" content="Sean E Cavanagh, Joni D Wallis, Steven W Kennerley, and Laurence T Hunt. Autocorrelation structure at rest predicts value correlates of single neurons during reward-guided choice. eLife, 5:e18937, 2016."/>
  <meta name="citation_reference" content="Rishidev Chaudhuri, Kenneth Knoblauch, Marie-Alice Gariel, Henry Kennedy, and Xiao-Jing Wang. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron, 88(2):419–431, 2015."/>
  <meta name="citation_reference" content="Carsen Stringer, Marius Pachitariu, Nicholas A Steinmetz, Michael Okun, Peter Bartho, Kenneth D Harris, Maneesh Sahani, and Nicholas A Lesica. Inhibitory control of correlated intrinsic variability in cortical networks. eLife, 5:e19695, 2016."/>
  <meta name="citation_reference" content="A Sarracino, O Arviv, O Shriki, and L de Arcangelis. Predicting brain evoked response to external stimuli from temporal correlations of spontaneous activity. Physical Review Research, 2(3):033355, 2020."/>
  <meta name="citation_reference" content="Joshua H Siegle, Xiaoxuan Jia, Séverine Durand, Sam Gale, Corbett Bennett, Nile Graddis, Greggory Heller, Tamina K Ramirez, Hannah Choi, Jennifer A Luviano, et al. Survey of spiking in the mouse visual system reveals functional hierarchy. Nature, 592(7852):86–92, 2021."/>
  <meta name="citation_reference" content="Valeria Fascianelli, Satoshi Tsujimoto, Encarni Marcos, and Aldo Genovesio. Autocorrelation structure in the macaque dorsolateral, but not orbital or polar, prefrontal cortex predicts response-coding strength in a visually cued strategy task. Cerebral Cortex, 29(1):230–241, 2019."/>
  <meta name="citation_reference" content="Camden J MacDowell and Timothy J Buschman. Low-dimensional spatiotemporal dynamics underlie cortex-wide neural activity. Current Biology, 2020."/>
  <meta name="citation_reference" content="Robert Kim and Terrence J Sejnowski. Strong inhibitory signaling underlies stable temporal dynamics and working memory in spiking neural networks. Nature Neuroscience, 24(1):129–139, 2021."/>
  <meta name="citation_reference" content="Takuya Ito, Luke J Hearne, and Michael W Cole. A cortical hierarchy of localized and distributed processes revealed via dissociation of task activations, connectivity changes, and intrinsic timescales. NeuroImage, 221:117141, 2020."/>
  <meta name="citation_reference" content="H Strey, M Peterson, and E Sackmann. Measurement of erythrocyte membrane elasticity by flicker eigenmode decomposition. Biophysical Journal, 69(2):478–488, 1995."/>
  <meta name="citation_reference" content="Aleander Rohrbach, Tim Meyer, Ernst HK Stelzer, and Holger Kress. Measuring stepwise binding of thermally fluctuating particles to cell membranes without fluorescence. Biophysical Journal, 2020."/>
  <meta name="citation_reference" content="Kai Liu, Brian Chu, Jay Newby, Elizabeth L Read, John Lowengrub, and Jun Allard. Hydrodynamics of transient cell-cell contact: The role of membrane permeability and active protrusion length. PLoS Computational Biology, 15(4):e1006352, 2019."/>
  <meta name="citation_reference" content="Christian Meisel. Antiepileptic drugs induce subcritical dynamics in human cortical networks. Proceedings of the National Academy of Sciences, 117(20):11118–11125, 2020."/>
  <meta name="citation_reference" content="Thomas Donoghue, Matar Haller, Erik J Peterson, Paroma Varma, Priyadarshini Sebastian, Richard Gao, Torben Noto, Antonio H Lara, Joni D Wallis, Robert T Knight, et al. Parameterizing neural power spectra into periodic and aperiodic components. Nature Neuroscience, 23(12):1655–1665, 2020."/>
  <meta name="citation_reference" content="Alexander Khintchine. Korrelationstheorie der stationären stochastischen prozesse. Math-ematische Annalen, 109(1):604–615, 1934."/>
  <meta name="citation_reference" content="Carlos D Brody. Slow covariations in neuronal resting potentials can lead to artefactually fast cross-correlations in their spike trains. Journal of Neurophysiology, 80(6):3345–3351, 1998."/>
  <meta name="citation_reference" content="Valérie Ventura, Can Cai, and Robert E Kass. Trial-to-trial variability and its effect on time-varying dependency between two neurons. Journal of Neurophysiology, 94(4):2928–2939, 2005."/>
  <meta name="citation_reference" content="Asohan Amarasingham, Matthew T Harrison, Nicholas G Hatsopoulos, and Stuart Geman. Conditional modeling and the jitter method of spike resampling. Journal of Neurophysiology, 107(2):517–531, 2012."/>
  <meta name="citation_reference" content="Marlene R Cohen and Adam Kohn. Measuring and interpreting neuronal correlations. Nature Neuroscience, 14(7):811, 2011."/>
  <meta name="citation_reference" content="FHC Marriott and JA Pope. Bias in the estimation of autocorrelations. Biometrika, 41(3/4):390–402, 1954."/>
  <meta name="citation_reference" content="A Sree Rama Sastry. Bias in estimation of serial correlation coefficients. Sankhyā: The Indian Journal of Statistics, pages 281–296, 1951."/>
  <meta name="citation_reference" content="Bradley E Huitema and Joseph W McKean. Autocorrelation estimation and inference with small samples. Psychological Bulletin, 110(2):291, 1991."/>
  <meta name="citation_reference" content="John S White. Asymptotic expansions for the mean and variance of the serial correlation coefficient. Biometrika, 48(1-2):85–94, 1961."/>
  <meta name="citation_reference" content="ZA Lomnicki and SK Zaremba. On the estimation of autocorrelation in time series. The Annals of Mathematical Statistics, 28(1):140–158, 1957."/>
  <meta name="citation_reference" content="Maurice G Kendall. Note on bias in the estimation of autocorrelation. Biometrika, 41(3-4):403–404, 1954."/>
  <meta name="citation_reference" content="Soroosh Afyouni, Stephen M Smith, and Thomas E Nichols. Effective degrees of freedom of the pearson’s correlation coefficient under autocorrelation. NeuroImage, 199:609–625, 2019."/>
  <meta name="citation_reference" content="Maurice S Bartlett. On the theoretical specification and sampling properties of autocorrelated time-series. Supplement to the Journal of the Royal Statistical Society, 8(1):27–41, 1946."/>
  <meta name="citation_reference" content="Helmut H Strey. Estimation of parameters from time traces originating from an ornstein-uhlenbeck process. Physical Review E, 100(6):062142, 2019."/>
  <meta name="citation_reference" content="Mikael Sunnåker, Alberto Giovanni Busetto, Elina Numminen, Jukka Corander, Matthieu Foll, and Christophe Dessimoz. Approximate bayesian computation. PLoS Computational Biology, 9(1), 2013."/>
  <meta name="citation_reference" content="Mark A Beaumont, Jean-Marie Cornuet, Jean-Michel Marin, and Christian P Robert. Adaptive approximate bayesian computation. Biometrika, 96(4):983–990, 2009."/>
  <meta name="citation_reference" content="George E Uhlenbeck and Leonard S Ornstein. On the theory of the brownian motion. Physical Review, 36(5):823, 1930."/>
  <meta name="citation_reference" content="Hannes Risken. Fokker-planck equation. In The Fokker-Planck Equation, pages 63–95. Springer, 1996."/>
  <meta name="citation_reference" content="Benjamin Lindner. A brief introduction to some simple stochastic processes. Stochastic Methods in Neuroscience, 1, 2009."/>
  <meta name="citation_reference" content="Ryan C Kelly, Matthew A Smith, Robert E Kass, and Tai Sing Lee. Local field potentials indicate network state and account for neuronal response variability. Journal of Computational Neuroscience, 29(3):567–579, 2010."/>
  <meta name="citation_reference" content="Alexander S Ecker, Philipp Berens, R James Cotton, Manivannan Subramaniyan, George H Denfield, Cathryn R Cadwell, Stelios M Smirnakis, Matthias Bethge, and Andreas S Tolias. State dependence of noise correlations in macaque primary visual cortex. Neuron, 82(1):235–248, 2014."/>
  <meta name="citation_reference" content="Mikhail Genkin and Tatiana A Engel. Moving beyond generalization to accurate interpretation of flexible models. Nature Machine Intelligence, 2(11):674–683, 2020."/>
  <meta name="citation_reference" content="Anne K Churchland, Roozbeh Kiani, Rishidev Chaudhuri, Xiao-Jing Wang, Alexandre Pouget, and Michael N Shadlen. Variance as a signature of neural computations during decision making. Neuron, 69(4):818–831, 2011."/>
  <meta name="citation_reference" content="Demetrios Neophytou, Diego Arribas, Robert Levy, Il Memming Park, and Hysell V Oviedo. Recurrent connectivity underlies lateralized temporal processing differences in auditory cortex. bioRxiv, 2021."/>
  <meta name="citation_reference" content="Behtash Babadi and Emery N Brown. A review of multitaper spectral analysis. IEEE Transactions on Biomedical Engineering, 61(5):1555–1564, 2014."/>
  <meta name="citation_reference" content="Clayton Haldeman and John M Beggs. Critical branching captures activity in living neural networks and maximizes the number of metastable states. Physical Review Letters, 94(5):058101, 2005."/>
  <meta name="citation_reference" content="Johannes Zierenberg, Jens Wilting, Viola Priesemann, and Anna Levina. Description of spreading dynamics by microscopic network models and macroscopic branching processes can differ due to coalescence. Physical Review E, 101(2):022301, 2020."/>
  <meta name="citation_reference" content="Johannes Zierenberg, Jens Wilting, Viola Priesemann, and Anna Levina. Tailored ensembles of neural networks optimize sensitivity to stimulus statistics. Physical Review Research, 2(1):013115, 2020."/>
  <meta name="citation_reference" content="Aude Grelaud, Christian P Robert, Jean-Michel Marin, François Rodolphe, Jean-François Taly, et al. Abc likelihood-free methods for model choice in gibbs random fields. Bayesian Analysis, 4(2):317–335, 2009."/>
  <meta name="citation_reference" content="Xavier Didelot, Richard G Everitt, Adam M Johansen, Daniel J Lawson, et al. Likelihood-free estimation of model evidence. Bayesian Analysis, 6(1):49–76, 2011."/>
  <meta name="citation_reference" content="Jean-Michel Marin, Natesh S Pillai, Christian P Robert, and Judith Rousseau. Relevant statistics for bayesian model choice. Journal of the Royal Statistical Society: Series B: Statistical Methodology, pages 833–859, 2014."/>
  <meta name="citation_reference" content="Christian P Robert, Jean-Marie Cornuet, Jean-Michel Marin, and Natesh S Pillai. Lack of confidence in approximate bayesian computation model choice. Proceedings of the National Academy of Sciences, 108(37):15112–15117, 2011."/>
  <meta name="citation_reference" content="Christopher M Bishop. Pattern recognition and machine learning. Springer, 2006."/>
  <meta name="citation_reference" content="Tina Toni and Michael PH Stumpf. Simulation-based model selection for dynamical systems in systems and population biology. Bioinformatics, 26(1):104–110, 2010."/>
  <meta name="citation_reference" content="Tatiana A Engel, Nicholas A Steinmetz, Marc A Gieselmann, Alexander Thiele, Tirin Moore, and Kwabena Boahen. Selective modulation of cortical state during spatial attention. Science, 354(6316):1140–1144, 2016."/>
  <meta name="citation_reference" content="Robbe LT Goris, J Anthony Movshon, and Eero P Simoncelli. Partitioning neuronal variability. Nature Neuroscience, 17(6):858, 2014."/>
  <meta name="citation_reference" content="Carlos D Brody. Correlations without synchrony. Neural Computation, 11(7):1537–1551, 1999."/>
  <meta name="citation_reference" content="Richard Gao, Erik J Peterson, and Bradley Voytek. Inferring synaptic excitation/inhibition balance from field potentials. Neuroimage, 158:70–78, 2017."/>
  <meta name="citation_reference" content="Eric-Jan Wagenmakers, Simon Farrell, and Roger Ratcliff. Estimation and interpretation of 1/f α noise in human cognition. Psychonomic bulletin & review, 11(4):579–615, 2004."/>
  <meta name="citation_reference" content="Robert C. Morehead and Alex Hagen. A python package for approximate bayesian computation. https://github.com/rcmorehead/simpleabc, 2014."/>
  <meta name="citation_reference" content="Ryan N Gutenkunst, Joshua J Waterfall, Fergal P Casey, Kevin S Brown, Christopher R Myers, and James P Sethna. Universally sloppy parameter sensitivities in systems biology models. PLoS computational biology, 3(10):e189, 2007."/>
  <meta name="twitter:title" content="A flexible Bayesian framework for unbiased estimation of timescales"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Timescales characterize the pace of change for many dynamic processes in nature. Timescales are usually estimated by fitting the exponential decay of data autocorrelation in the time or frequency domain. We show that this standard procedure often fails to recover the correct timescales due to a statistical bias arising from the finite sample size. We develop an alternative approach to estimating timescales by fitting the sample autocorrelation or power spectrum with a generative model based on a mixture of Ornstein-Uhlenbeck processes using adaptive Approximate Bayesian Computations. Our method accounts for finite sample size and noise in data and returns a posterior distribution of timescales that quantifies the estimation uncertainty and can be used for model selection. We demonstrate the accuracy of our method on synthetic data and illustrate its application to recordings from the primate cortex. We provide a customizable Python package implementing our framework with different generative models suitable for diverse applications.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="A flexible Bayesian framework for unbiased estimation of timescales"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.08.11.245944v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Timescales characterize the pace of change for many dynamic processes in nature. Timescales are usually estimated by fitting the exponential decay of data autocorrelation in the time or frequency domain. We show that this standard procedure often fails to recover the correct timescales due to a statistical bias arising from the finite sample size. We develop an alternative approach to estimating timescales by fitting the sample autocorrelation or power spectrum with a generative model based on a mixture of Ornstein-Uhlenbeck processes using adaptive Approximate Bayesian Computations. Our method accounts for finite sample size and noise in data and returns a posterior distribution of timescales that quantifies the estimation uncertainty and can be used for model selection. We demonstrate the accuracy of our method on synthetic data and illustrate its application to recordings from the primate cortex. We provide a customizable Python package implementing our framework with different generative models suitable for diverse applications.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-02-08"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.08.11.245944v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2390714"/>
  <title>
   A flexible Bayesian framework for unbiased estimation of timescales | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"kox_gqMwdhFnALuFK3flcSgUN7TFeYWmBFCgljjVuZQ"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2390714","apath":"\/biorxiv\/early\/2022\/02\/08\/2020.08.11.245944.atom","pisa":"biorxiv;2020.08.11.245944v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.08.11.245944v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.08.11.245944v3","title":"A flexible Bayesian framework for unbiased estimation of timescales","identifier":"node\/2390714"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dzpPSb9UlhWWsHS+izG4BH9j\\\\\\\/8ptI5dRJrFdvzc\\\\\\\/wXUjVPVDz1mY99mT6EO2rBt3CKvrtf10mD0Mq5N0gSTHF8tfbcdYUCpAxu8CKxTBbBw2jyHuE6AfF9BX1tuyOoVdoYyrptwIyS3ANuMRQnkLDJ\\\\\\\/Qrep587\\\\\\\/iZPY1PiJxOXyYh5hsu9YDnAVnOOs9mG2pLzux+3yHYT4n\\\\\\\/FrCuOKBB4AYAq0rEigpMmmbFYJ5Immx4M1gwZGtVp4EpxdomppxQhwp4M3BXw30vKwJjTjjWCgeGx+SbZ8AiMKMVRlLSEhgJw9\\\\\\\/QTYx48vGv9OirKczkHHxJxvXlfOogL4mcTHAT7w1ew5WR4mGaR+KJAZMJFx37KaSV8tSJaVGG82BQnexPiCTrUPAfDvFy9pouw0TvAOX5X+uFgiP7VtxC2IWVCW2eatmzl39Ko2qJ8W22RY7pDnBNTu2vgh1IETX5N\\\\\\\/\\\\\\\/aEjCFJd8qDpuFtBmwm2FHeNXr3gm6MQbtFSEQklfjkD6K5PjVgSk8jCP67BrAIDn+ltMukBnkf+\\\\\\\/W\\\\\\\/IKTmO5+ILPNo3K0GUGo\\\\\\\/39L93qk0pQP84yflV++1KCz6x+yy6EPbzKbp9UBVfJtTMKkbKUJ3Sn\\\\\\\/lZNuMLUv40aJ\\\\\\\/+mhoTI0UmwPC7C8jJZwFxKye9KTfHu7U\\\\\\\/wbflE16G1UrzUYktmbnLh7X7gzhH9p2AiIqLTmBBn\\\\\\\/69lBES87BYVvhWq\\\\\\\/b5TcpH6ssKW9IC6ecMVGREd7oc27n42f4onkl3Lc\\\\\\\/jdWoVJdkSzb1e1o1i5T821urblycjV0Z2eV3KqpCZ5i4eO+NG6OeMeIpUxKyDKlQDvlhSAddec4r\\\\\\\/B\\\\\\\/cCGeRln0qo1dl+Da1n5tkwSiC\\\\\\\/d\\\\\\\/pume6K3g40TogwDRUWxFuHUth5a93RJlNKd64amYQ131dLODyhp\\\\\\\/ZthcuYsWsfMGfO0ZoVRheZ+TvUQhYScAuoD2ZJ7MG\\\\\\\/d2RbO3LnqOOTP\\\\\\\/zhFTVdphedd0Iiw\\\\\\\/PsOeScTSFaYHF1b9IX4NCXOihEZWfD9LtRys4bYtxECtoqMPZ7jqqPAa0TMkpbSwvajuu7xD0wOm4BXYnmChtZ8ClIA3oCSm+WVpAahvoBWNt+\\\\\\\/R7wrcQ2fxl5556wMsFlZ74Nc\\\\\\\/dEImcD2DcvpwUPwrJSpfm\\\\\\\/3h0KoMHXK8fZT8n+ebP9+uBXktAROySzhm5Yetw\\\\\\\/aT3spkU3B00nMN\\\\\\\/WlDwzn\\\\\\\/10fYDRukVn5BcYG1O3bq4h\\\\\\\/W2B2G1G3lvYJbCrW7DfxfYY2h5Lr5rCEbKGh9lbM9okY\\\\\\\/sq4dbLMUwDRyGZS1W5xiZMrDhyZSqwVrSWRlSIhwglsK1Znb5TZ5S354DXM7SvbhtN6xyaWjETy2chCpx98NgVidVsZFlOKxerotTk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022zgCac9dKTFTFwIvaP3jIOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00223925101bddef10dc5694e1a0d19b4799157ea94880398b5ab96a4307ac94af3e\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.08.11.245944v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2390714 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.08.11.245944v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1683415164">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1683415164" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1984121396" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-F1Lk-Yf7VMyQWXOl_V5Cf4SUkbWr_E9hn3LmcfjdsN4"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2390714" data-node-nid="2390714" id="node-2390714--21490586488" data-pisa="biorxiv;2020.08.11.245944v3" data-pisa-master="biorxiv;2020.08.11.245944" data-apath="/biorxiv/early/2022/02/08/2020.08.11.245944.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    A flexible Bayesian framework for unbiased estimation of timescales
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-7946-1464" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Roxana
                      </span>
                      <span class="nlm-surname">
                       Zeraati
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0001-5842-9406" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Tatiana A.
                      </span>
                      <span class="nlm-surname">
                       Engel
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0003-1355-6617" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Anna
                      </span>
                      <span class="nlm-surname">
                       Levina
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.08.11.245944
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2390714--21490586488" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Roxana Zeraati
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        International Max Planck Research School for the Mechanisms of Mental Function and Dysfunction, University of Tübingen
                       </span>
                       ,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Max Planck Institute for Biological Cybernetics
                       </span>
                       , Tübingen,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Roxana%2BZeraati%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zeraati%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARoxana%2BZeraati%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-7946-1464" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Roxana Zeraati
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Tatiana A. Engel
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Cold Spring Harbor Laboratory
                       </span>
                       , Cold Spring Harbor, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Tatiana%2BA.%2BEngel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Engel%20TA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ATatiana%2BA.%2BEngel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-5842-9406" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Tatiana A. Engel
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:engel@cshl.edu" class="" data-icon-position="" data-hide-link-title="0">
                        engel@cshl.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Anna Levina
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of Tübingen
                       </span>
                       ,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Max Planck Institute for Biological Cybernetics
                       </span>
                       , Tübingen,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Bernstein Center for Computational Neuroscience Tübingen
                       </span>
                       , Tübingen,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Anna%2BLevina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Levina%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAnna%2BLevina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-1355-6617" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Anna Levina
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.08.11.245944v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2390714" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2390714/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.11.245944v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2390714" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2390714/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.11.245944v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2390714" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2390714/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.11.245944v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2390714" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2390714/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.08.11.245944v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2390714" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2390714/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.08.11.245944v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2390714" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2390714/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-5">
                             Timescales characterize the pace of change for many dynamic processes in nature. Timescales are usually estimated by fitting the exponential decay of data autocorrelation in the time or frequency domain. We show that this standard procedure often fails to recover the correct timescales due to a statistical bias arising from the finite sample size. We develop an alternative approach to estimating timescales by fitting the sample autocorrelation or power spectrum with a generative model based on a mixture of Ornstein-Uhlenbeck processes using adaptive Approximate Bayesian Computations. Our method accounts for finite sample size and noise in data and returns a posterior distribution of timescales that quantifies the estimation uncertainty and can be used for model selection. We demonstrate the accuracy of our method on synthetic data and illustrate its application to recordings from the primate cortex. We provide a customizable Python package implementing our framework with different generative models suitable for diverse applications.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-9">
                             Dynamic changes in many stochastic processes occur over typical periods known as timescales. Timescales of different processes in nature range broadly from milliseconds in protein folding [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ] and neural signalling [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             –
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ], to minutes in gene splicing [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             –
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ] and synaptic plasticity [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             –
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ], to days in spreading of infectious diseases [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             –
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ], to years in demographic changes of populations [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ,
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ], and up to millennia in the climate change [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ,
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ]. Accurate measurements of timescales from experimental data are necessary to uncover mechanisms controlling the dynamics of underlying processes and reveal their function. The variation of timescales across brain areas reflects the functional hierarchy in the neocortex and areal differences in temporal integration of information [
                             <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             –
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ], while aberrant timescales were associated with autism [
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ]. Precise estimation methods are necessary to detect changes in timescales during cognitive processes such as attention [
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ]. Estimated timescales are also used to determine the operating regime of the dynamics, e.g., how close the system is to a critical point [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ], which can reveal general principles organizing the collective behavior of complex systems such as the brain [
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             –
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ]. Thus, many problems require estimating timescales from experimental data to constrain theoretical models and enable accurate predictions in practical applications.
                            </p>
                            <p id="p-10">
                             The timescales of a stochastic process are defined by the exponential decay rates of its autocorrelation function. Accordingly, timescales are usually estimated by fitting the autocorrelation of a sample time-series with exponential decay functions [
                             <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ,
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             –
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             ] or using the autocorrelation half-width [
                             <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ,
                             <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              41
                             </a>
                             ]. Equivalently, timescales can be estimated in frequency domain by fitting the shape of the sample power spectral density (PSD) with a Lorentzian function which is the PSD of a process with an exponentially decaying autocorrelation [
                             <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ,
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              42
                             </a>
                             ]. However, many factors unrelated to the dynamics of the processes under study can alter the shape of autocorrelations and, according to the Wiener–Khinchin theorem [
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              43
                             </a>
                             ], the PSD shape. For example, autocorrelations of in vivo neural activity can contain components arising from a specific trial structure of a behavioral task or slow drifts in the average activity. To correct for these irrelevant factors, several techniques were developed based on data resampling, in particular, trial shuffling and spike jittering methods [
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              44
                             </a>
                             –
                             <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                              47
                             </a>
                             ]. These methods remove from the autocorrelation the average autocorrelation of surrogate data, which are designed to match the irrelevant factors in the real data but are otherwise random. The success of these methods critically depends on the ability to construct the appropriate surrogate data that exactly reproduce the irrelevant factors.
                            </p>
                            <p id="p-11">
                             The values of sample autocorrelation computed from a finite time series systematically deviate from the true autocorrelation [
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              48
                             </a>
                             –
                             <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                              53
                             </a>
                             ]. This bias is exacerbated by the lack of independence between the samples, which is particularly harmful when the timescales are large and timeseries are short [
                             <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                              54
                             </a>
                             ,
                             <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                              55
                             </a>
                             ]. Generally, the magnitude of bias depends on the length of the sample time-series, but also on the value of the true autocorrelation at each time-lag. The expected value and variance of the autocorrelation bias can be derived analytically in some simple cases, such as a Markov process with a single timescale [
                             <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">
                              48
                             </a>
                             ,
                             <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                              51
                             </a>
                             ]. However, the analytical derivation is not tractable for more general processes that involve multiple timescales or have additional temporal structure. Moreover, since the bias depends on the true autocorrelation itself, which is unknown, it cannot be corrected by constructing appropriate surrogate data as in shuffling or jittering methods.
                            </p>
                            <p id="p-12">
                             The statistical bias deforms the shape of empirical autocorrelations and PSDs and hence can affect the timescales estimated by direct fitting of the shapes. Indeed, it was noticed that fitting the sample autocorrelation of an Ornstein-Uhlenbeck (OU) process with an exponential decay function results in systematic errors in the estimated timescale and the confidence interval [
                             <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ]. To avoid these errors, it is possible to fit the time-series directly with an autoregressive model, without using autocorrelation or PSD [
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ,
                             <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ]. For example, the parameters of an OU process can be obtained with a maximum-likelihood estimator, which directly fits time-series and thus evades the autocorrelation bias [
                             <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ]. However, the advantage of autocorrelation is that irrelevant factors (e.g., slow activity drifts) can be efficiently removed with resampling methods (e.g., spike jittering), and any additional dynamics not locked to trial onset (e.g., oscillations) can be easily incorporated in the autocorrelation shape. In contrast, accounting for irrelevant factors and dynamics in the raw time-series is generally difficult as it requires adding components to the autoregressive model matching the temporal structure of these factors on single trials, all of which need to be fitted to the data. Thus, fitting the summary statistic such as autocorrelation or PSD is attractive, but how the statistical bias affects the estimated timescales was not studied systematically. Moreover, rather than maximum-likelihood estimates, inference of a full posterior distribution is often necessary, for example, for model selection, which has not been addressed for complex processes with multiple timescales, additional temporal structure and noise.
                            </p>
                            <p id="p-13">
                             We show that large systematic errors in estimated timescales arise from the statistical bias due to a finite sample size, which is evident in different processes with various number of timescales and different trial durations. These estimation errors may result in misleading interpretations of the underlying biological processes. To correct for the bias, we develop a flexible computational framework based on adaptive Approximate Bayesian Computations (aABC) that estimates timescales by fitting the autocorrelation or PSD with a generative model. ABC is a family of likelihood-free inference algorithms for estimating model parameters when the likelihood function cannot be calculated analytically [
                             <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                              57
                             </a>
                             ]. The aABC algorithm approximates the multivariate posterior distribution of parameters of a generative model using population Monte-Carlo sampling [
                             <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                              58
                             </a>
                             ]. Our generative model is based on a mixture of Ornstein-Uhlenbeck processes—one for each estimated timescale—which have exponentially decaying autocorrelations. The generative model can be further augmented with the desired noise model (e.g., a spike generation process) and additional temporal structure to match the statistics of the data. Our method accurately recovers the correct timescales from finite data samples for various synthetic processes with known ground-truth dynamics. The inferred posterior distributions quantify the uncertainty of estimates and can be used for model selection to compare alternative hypotheses about the dynamics of the underlying process. To illustrate an application of our method, we estimate timescales of ongoing spiking activity in the primate visual cortex during a behavioral task. The computational framework presented here can be adapted to various types of data and can find broad applications in neuroscience, cellular biology, epidemiology, physics, and other fields. To allow for an easy adoption and further development of our framework, we provide a Python package called abcTau that supports different types of data and generative models, and includes the model comparison functionality and a parallel processing option for analyzing large-scale datasets.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              2.1 Bias in timescales estimated by direct fitting
                             </h3>
                             <p id="p-14">
                              Timescales of a stochastic process A(t′) are defined by the exponential decay rates of its autocorrelation function. The autocorrelation is the correlation between the values of the process at two time points separated by a time lag t. For stationary processes, the autocorrelation function only depends on the time lag:
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-15">
                              Here μ and σ
                              <sup>
                               2
                              </sup>
                              are, respectively, the mean and variance of the process, which are constant in time, and E[·]
                              <sub>
                               t′
                              </sub>
                              is the expectation over t′. Different normalizations of autocorrelation are used in literature, but our results do not depend on a specific choice of normalization.
                             </p>
                             <p id="p-16">
                              In experiments or simulations, the autocorrelation needs to be estimated from a finite sample of empirical data. A data sample from the process A(t′) constitutes a finite time-series measured at discrete times
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (i = 1… N, where N is the length of the time-series). For example, the sample time-series can be spike-counts of a neuron in discrete time bins, or a continuous voltage signal measured at a specific sampling rate. Accordingly, the sample autocorrelation is defined for a discrete set of time lags t
                              <sub>
                               j
                              </sub>
                              . For empirical data, the true values of μ and σ are unknown. Hence, several estimators of the sample autocorrelation were proposed, which use different estimators for the sample mean
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and sample variance
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              [
                              <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              ,
                              <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                               49
                              </a>
                              ]. One possible choice is:
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with the sample variance
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and two different sample means
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The sample autocorrelation can also be computed as the inverse Fourier transform of PSD, based on the Wiener–Khinchin theorem [
                              <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ]. However, for any of these methods, the sample autocorrelation is a biased estimator: for a finite length time-series the values of the sample autocorrelation systematically deviate from the ground-truth autocorrelation [
                              <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              –
                              <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ] (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ,
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Supplementary Fig. 1
                              </a>
                              ). This statistical bias deforms the shape of the sample autocorrelation or PSD and therefore may affect the estimation of timescales by direct fitting of the shape.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F1.large.jpg?width=800&height=600&carousel=1" title="Bias in sample autocorrelations and timescales estimated by direct exponential fitting. (a) Data are generated from an OU process with the timescale τ = 20 ms. Left: Sample autocorrelation (colored dots) systematically deviates from the ground-truth autocorrelation (gray line). The shape of sample autocorrelation depends on the time-series duration (T) approaching the ground truth with increasing T (inset: close up in logarithmic-linear scale). Middle: Direct fitting of the analytical autocorrelation function (cyan) to sample autocorrelation (brown, T = 1 s) does not recover the ground-truth autocorrelation shape (gray). Right: The ground-truth timescales largely deviate from the distribution of timescales estimated by direct exponential fitting across 500 independent realizations of the same process. (b) Same as a for data from a linear mixture of an OU process with the timescale τ = 60 ms and an oscillation with frequency f = 2 Hz (Eq. 5). (c) Same as a for data from an inhomogeneous Poisson process with the instantaneous rate generated from a linear mixture of two OU processes with timescales τ1 = 5 ms and τ2 = 80 ms. All simulation parameters are provided in Methods 4.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Bias in sample autocorrelations and timescales estimated by direct exponential fitting.</span> <strong>(a)</strong> Data are generated from an OU process with the timescale <em>τ</em> = 20 ms. <strong>Left</strong>: Sample autocorrelation (colored dots) systematically deviates from the ground-truth autocorrelation (gray line). The shape of sample autocorrelation depends on the time-series duration (<em>T</em>) approaching the ground truth with increasing <em>T</em> (inset: close up in logarithmic-linear scale). <strong>Middle:</strong> Direct fitting of the analytical autocorrelation function (cyan) to sample autocorrelation (brown, <em>T</em> = 1 s) does not recover the ground-truth autocorrelation shape (gray). <strong>Right:</strong> The ground-truth timescales largely deviate from the distribution of timescales estimated by direct exponential fitting across 500 independent realizations of the same process. <strong>(b)</strong> Same as a for data from a linear mixture of an OU process with the timescale <em>τ</em> = 60 ms and an oscillation with frequency <em>f</em> = 2 Hz (Eq. 5). <strong>(c)</strong> Same as a for data from an inhomogeneous Poisson process with the instantaneous rate generated from a linear mixture of two OU processes with timescales <em>τ</em><sub>1</sub> = 5 ms and <em>τ</em><sub>2</sub> = 80 ms. All simulation parameters are provided in Methods 4.5.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F1.medium.gif" width="440" height="259"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F1.medium.gif" width="440" height="259"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Fig. 1.
                               </span>
                               <span class="caption-title">
                                Bias in sample autocorrelations and timescales estimated by direct exponential fitting.
                               </span>
                               <p id="p-17" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Data are generated from an OU process with the timescale τ = 20 ms.
                                <strong>
                                 Left
                                </strong>
                                : Sample autocorrelation (colored dots) systematically deviates from the ground-truth autocorrelation (gray line). The shape of sample autocorrelation depends on the time-series duration (T) approaching the ground truth with increasing T (inset: close up in logarithmic-linear scale).
                                <strong>
                                 Middle:
                                </strong>
                                Direct fitting of the analytical autocorrelation function (cyan) to sample autocorrelation (brown, T = 1 s) does not recover the ground-truth autocorrelation shape (gray).
                                <strong>
                                 Right:
                                </strong>
                                The ground-truth timescales largely deviate from the distribution of timescales estimated by direct exponential fitting across 500 independent realizations of the same process.
                                <strong>
                                 (b)
                                </strong>
                                Same as a for data from a linear mixture of an OU process with the timescale τ = 60 ms and an oscillation with frequency f = 2 Hz (
                                <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                                 Eq. 5
                                </a>
                                ).
                                <strong>
                                 (c)
                                </strong>
                                Same as a for data from an inhomogeneous Poisson process with the instantaneous rate generated from a linear mixture of two OU processes with timescales τ
                                <sub>
                                 1
                                </sub>
                                = 5 ms and τ
                                <sub>
                                 2
                                </sub>
                                = 80 ms. All simulation parameters are provided in
                                <a id="xref-sec-19-1" class="xref-sec" href="#sec-19">
                                 Methods 4.5
                                </a>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-18">
                              To investigate how the autocorrelation bias affects the timescales estimated by direct exponential fitting, we tested how accurately this procedure recovers the correct timescales on synthetic data with a known ground truth. We generated synthetic data from several stochastic processes for which the autocorrelation function can be computed analytically. The exponential decay rates of the analytical autocorrelation provide the ground-truth timescales. Each synthetic dataset consisted of 500 independent realizations of the process (i.e. trials) with fixed duration. Such trial-based data are typical in neuroscience but usually with a smaller number of trials. We computed the sample autocorrelation for each trial using
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Eq. 2
                              </a>
                              , averaged them to reduce the noise, and then fitted the average autocorrelation with the correct analytical functional form to estimate the timescale parameters. We repeated the entire procedure 500 times to obtain a distribution with multiple independent samples of timescales estimated by direct fit (i.e. we simulated 500 × 500 trials for each distribution).
                             </p>
                             <p id="p-19">
                              We considered three ground-truth processes which differed in the number of timescales, additional temporal structure and noise. First, we used an Ornstein–Uhlenbeck (OU) process (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1A
                              </a>
                              ):
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where ξ(t′) is a Gaussian white noise with zero mean, and the diffusion parameter D sets the variance Var[A
                              <sub>
                               OU
                              </sub>
                              (t′)] = D
                              <sub>
                               τ
                              </sub>
                              [
                              <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                               59
                              </a>
                              ,
                              <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                               60
                              </a>
                              ]. The autocorrelation of the OU process is an exponential decay function [
                              <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                               61
                              </a>
                              ]
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-20">
                              Accordingly, the parameter τ provides the ground-truth timescale.
                             </p>
                             <p id="p-21">
                              Second, we used a linear mixture of an OU process and an additional oscillatory component (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              ):
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-22">
                              Here the coefficient c
                              <sub>
                               1
                              </sub>
                              ∈ [0, 1] sets the relative weights of two components without changing the total variance of the process, and the phase ϕ is drawn on each trial from a uniform distribution on [0, 2π]. This process resembles the firing rate of a neuron modulated by a slow oscillation [
                              <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                               62
                              </a>
                              ,
                              <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                               63
                              </a>
                              ]. The autocorrelation of this process is given by
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              hence, the ground-truth timescale is defined by the OU parameter τ.
                             </p>
                             <p id="p-23">
                              Finally, we used an inhomogeneous Poisson process with the instantaneous rate modeled as a linear mixture of two OU processes with different timescales (
                              <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">
                               Eq. 13
                              </a>
                              ,
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1C
                              </a>
                              ). The OU mixture was shifted, scaled and rectified to produce the Poisson rate with the desired mean and variance (
                              <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">
                               Eq. 17
                              </a>
                              ). Similar doubly-stochastic processes are often used to model spiking activity of neurons [
                              <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                               64
                              </a>
                              ]. The total variance of this process consists of two contributions: the rate variance and the Poisson process variance (
                              <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">
                               Eq. 19
                              </a>
                              ) [
                              <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">
                               65
                              </a>
                              ]. We simulated this process in discrete time bins
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              by sampling event-counts from a Poisson distribution with the given instantaneous rate (
                              <a id="xref-sec-13-1" class="xref-sec" href="#sec-13">
                               Methods 4.1.3
                              </a>
                              ). The autocorrelation at all lags t
                              <sub>
                               j
                              </sub>
                              (j > 0) arises only from the autocorrelation of the rate, since the Poisson process has no temporal correlations. The drop of autocorrelation between t
                              <sub>
                               0
                              </sub>
                              and t
                              <sub>
                               1
                              </sub>
                              mainly reflects the Poisson-process variance, and for all non-zero lags t
                              <sub>
                               j
                              </sub>
                              (j > 0) the autocorrelation is given by
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-24">
                              Here τ
                              <sub>
                               1
                              </sub>
                              and τ
                              <sub>
                               2
                              </sub>
                              are the ground-truth timescales defined by the parameters of two OU processes, c
                              <sub>
                               1
                              </sub>
                              is the mixing coefficient, and μ and σ
                              <sup>
                               2
                              </sup>
                              are the mean and variance of the event-counts, respectively.
                             </p>
                             <p id="p-25">
                              For all three processes, the sample autocorrelation exhibits a negative bias: the values of the sample autocorrelation are systematically below the ground-truth autocorrelation function (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              , left). This bias is clearly visible in the logarithmic-linear scale, where the ground-truth exponential decay turns into a straight line. The sample autocorrelation deviates from the straight line and even becomes systematically negative at intermediate lags (hence not visible on the logarithmic scale) for processes with a strictly positive ground-truth autocorrelation (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1a,c
                              </a>
                              ). The deviations from the ground truth are larger when the timescales are longer or when multiple timescales are involved. The negative bias decreases for longer trial durations (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              , left inset), but it is still substantial for realistic trial durations such as in neuroscience data.
                             </p>
                             <p id="p-26">
                              Due to the negative bias, a direct fit of the sample autocorrelation with the correct analytical function cannot recover the ground-truth timescales (
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              , middle, right,
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Supplementary Fig. 2
                              </a>
                              ,
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               3
                              </a>
                              ). When increasing the duration of each trial, the timescales obtained from the direct fits become closer to the ground-truth values (
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Supplementary Fig. 4
                              </a>
                              ). This observation indicates that timescales estimated from datasets with different trial durations cannot be directly compared, as differences in the estimation bias may result in misleading interpretations. Thus, direct fitting of a sample autocorrelation, even when the correct analytical form of autocorrelation is known, is not a reliable method for measuring timescales in experimental data. Bias-correction methods based on parametric bootstrapping can mitigate the estimation bias of timescales from direct fits [
                              <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                               66
                              </a>
                              ]. However, since the amount of bias depends on the ground-truth timescales, such corrections cannot guarantee accurate estimates in all cases (
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Supplementary Fig. 5
                              </a>
                              ).
                             </p>
                             <p id="p-27">
                              Alternatively, we can estimate the timescales by fitting the PSD shape in the frequency domain. In this approach, the PSD is fitted with a Lorentzian function [
                              <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ,
                              <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                              ] which is the ground-truth PSD for a stochastic process with an exponentially decaying autocorrelation (e.g., OU process)
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-28">
                              Here f is the frequency and
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the normalization constant. From the knee frequency f
                              <sub>
                               k
                              </sub>
                              , we can estimate the timescale as τ = (2πf
                              <sub>
                               k
                              </sub>
                              )
                              <sup>
                               -1
                              </sup>
                              . Comparison of the ground-truth PSD with the sample PSD of an OU process with a finite duration reveals that a statistical bias observed in the time domain also persists in the frequency domain (
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Supplementary Fig. 6A
                              </a>
                              ). Due to this bias, the estimated knee frequency deviates from the ground-truth knee frequency depending on the fitted frequency range, which results in biased estimation of the timescale. Although careful choice of the fitted frequency range can improve the accuracy of estimates (
                              <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                               Supplementary Fig. 6B
                              </a>
                              ), without knowing the ground-truth timescale, there is no principled way to choose the correct frequency range that will work in all cases. Slightly changing the fitted frequency range can produce large errors in estimated timescale, especially in the presence of additional noise e.g., in spiking activity.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              2.2 Estimating timescales by fitting generative models with Approximate Bayesian Computations
                             </h3>
                             <p id="p-29">
                              Since direct fitting of the sample autocorrelation (or PSD) cannot estimate timescales reliably, we developed an alternative computational framework based on fitting the sample autocorrelation (or PSD) with a generative model. Using a generative model with known ground-truth timescales, we can generate synthetic data matching the essential statistics of the observed data, i.e. with the same duration and number of trials, mean and variance. Hence the sample autocorrelations (or PSDs) of the synthetic and observed data will be affected by a similar statistical bias when their shapes match. As a generative model, we chose a linear mixture of OU processes—one for each estimated timescale—which if necessary can be augmented with additional temporal structure (e.g., oscillations) and noise. The advantage of using a mixture of OU processes for modelling the observed data is that the analytical autocorrelation function of this mixture explicitly defines the timescales. We set the number of components in the generative model in accordance with our hypothesis about the shape of the autocorrelation in the data, e.g., the number of timescales, additional temporal structure and noise (
                              <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">
                               Methods 4.1
                              </a>
                              ). Then, we optimize parameters of the generative model to match the shape of the autocorrelation (or PSD) between the synthetic and observed data. The timescales of the optimized generative model provide an approximation for the timescales in the observed data, without the bias corruption.
                             </p>
                             <p id="p-30">
                              For complex generative models, calculating the likelihood can be computationally expensive or even intractable. Therefore, we optimize the generative model parameters using adaptive Approximate Bayesian Computations (aABC) [
                              <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ] (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). aABC is an iterative algorithm that minimizes the distance between the summary statistic of synthetic and observed data. Depending on the application, we can choose a different summary statistic in time or frequency domain. For example, we can use autocorrelations as the summary statistic and define the suitable distance d between the autocorrelations of synthetic and observed data, e.g., as
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where t
                              <sub>
                               m
                              </sub>
                              is the maximum time-lag considered in computing the distance. Alternatively, we can compute distances between the PSDs of synthetic and observed data:
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where f
                              <sub>
                               n
                              </sub>
                              and f
                              <sub>
                               n+m
                              </sub>
                              define the frequency range for computing the distance.
                             </p>
                             <p id="p-31">
                              Due to stochasticity of the observed data, point estimates (i.e. assigning a single value to a timescale) are not reliable as different realizations of the same stochastic process lead to slightly different autocorrelation shapes (
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Supplementary Fig. 7
                              </a>
                              ). aABC overcomes this problem by estimating the joint posterior distribution of the generative model parameters, which quantifies the estimation uncertainty.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F2.large.jpg?width=800&height=600&carousel=1" title="Estimation of timescales with adaptive Approximate Bayesian Computations. aABC estimates timescales by fitting the sample autocorrelation of observed data with a generative model. At the first iteration of the algorithm, parameters of the generative model are drawn from the multivariate prior distribution, e.g., a uniform distribution (upper left). Synthetic data are generated from the generative model with these parameters. If the distance d between the autocorrelations of synthetic and observed data is smaller than a specific error threshold ε, these parameters are added to the multivariate posterior distribution (lower right). In the subsequent iterations, new parameters are drawn from a proposal distribution which is computed based on the posterior distribution of the previous iteration and the initial prior distribution (upper right, see Methods 4.2 and 4.3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Estimation of timescales with adaptive Approximate Bayesian Computations.</span> aABC estimates timescales by fitting the sample autocorrelation of observed data with a generative model. At the first iteration of the algorithm, parameters of the generative model are drawn from the multivariate prior distribution, e.g., a uniform distribution (upper left). Synthetic data are generated from the generative model with these parameters. If the distance <em>d</em> between the autocorrelations of synthetic and observed data is smaller than a specific error threshold <em>ε</em>, these parameters are added to the multivariate posterior distribution (lower right). In the subsequent iterations, new parameters are drawn from a proposal distribution which is computed based on the posterior distribution of the previous iteration and the initial prior distribution (upper right, see Methods 4.2 and 4.3).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F2.medium.gif" width="440" height="196"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F2.medium.gif" width="440" height="196"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 2.
                               </span>
                               <span class="caption-title">
                                Estimation of timescales with adaptive Approximate Bayesian Computations.
                               </span>
                               <p id="p-32" class="first-child">
                                aABC estimates timescales by fitting the sample autocorrelation of observed data with a generative model. At the first iteration of the algorithm, parameters of the generative model are drawn from the multivariate prior distribution, e.g., a uniform distribution (upper left). Synthetic data are generated from the generative model with these parameters. If the distance d between the autocorrelations of synthetic and observed data is smaller than a specific error threshold ε, these parameters are added to the multivariate posterior distribution (lower right). In the subsequent iterations, new parameters are drawn from a proposal distribution which is computed based on the posterior distribution of the previous iteration and the initial prior distribution (upper right, see
                                <a id="xref-sec-15-1" class="xref-sec" href="#sec-15">
                                 Methods 4.2
                                </a>
                                and
                                <a id="xref-sec-16-1" class="xref-sec" href="#sec-16">
                                 4.3
                                </a>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-33">
                              The aABC algorithm operates iteratively (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). First, we choose a multivariate prior distribution over the parameters of the generative model and set an initial error threshold ε at a rather large value. On each iteration, we draw samples of the parameter-vector θ. On the first iteration, θ is drawn directly from the prior distribution. On subsequent iterations, θ is drawn from a proposal distribution defined based on the prior distribution and the parameter samples accepted on the previous iteration. We use the generative model with the sample parameters θ to generate synthetic data. If the distance d between the summary statistic (autocorrelation or PSD) of the observed and synthetic data (
                              <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                               Eq. 9
                              </a>
                              ,
                              <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                               10
                              </a>
                              ) is smaller than the error threshold, the sample θ is accepted. We then repeatedly draw new parameter samples and evaluate d until a fixed number of parameter samples are accepted. For each iteration, the fraction of accepted samples out of all drawn parameter samples is recorded as the acceptance rate accR. Next, the proposal distribution is updated using the samples accepted on the current iteration, and the new error threshold is set at the first quartile of the distance d for the accepted samples. The iterations continue until the acceptance rate reaches a specified value. The last set of accepted parameter samples is treated as an approximation for the posterior distribution (
                              <a id="xref-sec-15-2" class="xref-sec" href="#sec-15">
                               Methods 4.2
                              </a>
                              ). In the regular ABC algorithm, the error threshold is fixed and parameters are sampled from the prior distribution (same as in the first step of aABC). Updating the error threshold and proposal distribution in successive iterations of aABC allows for a more efficient fitting procedure especially when setting wide prior distributions [
                              <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ]. We implemented this algorithm in the abcTau Python package, which includes different types of summary statistics and generative models that can be flexibly adjusted to account for various types of dynamics and noise in the biological data (
                              <a id="xref-sec-16-2" class="xref-sec" href="#sec-16">
                               Methods 4.3
                              </a>
                              ). To visualize the posterior distribution of a parameter (e.g., a timescale), we marginalize the obtained multivariate posterior distribution over all other parameters of the generative model.
                             </p>
                             <p id="p-34">
                              We tested our method on the same synthetic data from the processes with known groundtruth timescales that we used to demonstrate the bias of direct fitting (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              cf.
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ). In these examples, we use the autocorrelation as the summary statistic and compute distances in linear scale. For all three processes, the shape of the sample autocorrelation of the observed data is accurately reproduced by the autocorrelation of synthetic data generated using the maximum a posteriori (MAP) estimate of parameters from the joint multivariate posterior distribution (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              , left). The posterior distributions inferred by the aABC algorithm include the ground-truth timescales (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              , middle). The variance of posterior distributions quantifies the uncertainty of estimates. In our simulations, the number of trials controls the signal to noise ratio in sample autocorrelation, and consequently the width of the posterior distributions (
                              <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                               Supplementary Fig. 8
                              </a>
                              ).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F3.large.jpg?width=800&height=600&carousel=1" title="The aABC algorithm accurately recovers the ground-truth timescales and quantifies the estimation uncertainty. The same synthetic data as in Fig. 1 for T = 1 s. (a) Data are generated from an OU process with the timescale τ = 20 ms. Left: The shape of the data autocorrelation (brown) is accurately reproduced by the autocorrelation of synthetic data from the generative model with the MAP estimate parameters (orange, τMAP = 20.2 ms), but it cannot be captured by the direct exponential fit (cyan). Middle: The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) include the ground-truth timescales, while direct exponential fits (cyan) underestimate the ground-truth timescales. The width of the posteriors indicates the estimation uncertainty. Right: The convergence of the aABC algorithm is defined based on the acceptance rate accR (purple), which decreases together with the error threshold ε (green) over the iterations of the algorithm. Data are plotted from the second iteration. Initial error threshold for all fits was set to 1. (b) Same as a for data from a linear mixture of an OU process with the timescale τ = 20 ms and an oscillation with frequency f = 2 Hz (Eq. 5). τMAP = 60.5 ms, (c) Same as a for data from an inhomogeneous Poisson process with two timescales τ1 = 5 ms and τ2 = 80 ms. τ1,MAP = 4.7 ms, τ2,MAP = 80 ms. See Methods 4.5 for other parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The aABC algorithm accurately recovers the ground-truth timescales and quantifies the estimation uncertainty.</span> The same synthetic data as in Fig. 1 for <em>T</em> = 1 s. (<strong>a</strong>) Data are generated from an OU process with the timescale <em>τ</em> = 20 ms. <strong>Left:</strong> The shape of the data autocorrelation (brown) is accurately reproduced by the autocorrelation of synthetic data from the generative model with the MAP estimate parameters (orange, <em>τ</em><sub>MAP</sub> = 20.2 ms), but it cannot be captured by the direct exponential fit (cyan). <strong>Middle:</strong> The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) include the ground-truth timescales, while direct exponential fits (cyan) underestimate the ground-truth timescales. The width of the posteriors indicates the estimation uncertainty. <strong>Right:</strong> The convergence of the aABC algorithm is defined based on the acceptance rate <em>accR</em> (purple), which decreases together with the error threshold <em>ε</em> (green) over the iterations of the algorithm. Data are plotted from the second iteration. Initial error threshold for all fits was set to 1. <strong>(b)</strong> Same as a for data from a linear mixture of an OU process with the timescale <em>τ</em> = 20 ms and an oscillation with frequency <em>f</em> = 2 Hz (Eq. 5). <em>τ</em><sub>MAP</sub> = 60.5 ms, <strong>(c)</strong> Same as a for data from an inhomogeneous Poisson process with two timescales <em>τ</em><sub>1</sub> = 5 ms and <em>τ</em><sub>2</sub> = 80 ms. <em>τ</em><sub>1,MAP</sub> = 4.7 ms, <em>τ</em><sub>2,MAP</sub> = 80 ms. See Methods 4.5 for other parameters.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F3.medium.gif" width="440" height="260"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F3.medium.gif" width="440" height="260"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 3.
                               </span>
                               <span class="caption-title">
                                The aABC algorithm accurately recovers the ground-truth timescales and quantifies the estimation uncertainty.
                               </span>
                               <p id="p-35" class="first-child">
                                The same synthetic data as in
                                <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                                 Fig. 1
                                </a>
                                for T = 1 s. (
                                <strong>
                                 a
                                </strong>
                                ) Data are generated from an OU process with the timescale τ = 20 ms.
                                <strong>
                                 Left:
                                </strong>
                                The shape of the data autocorrelation (brown) is accurately reproduced by the autocorrelation of synthetic data from the generative model with the MAP estimate parameters (orange, τ
                                <sub>
                                 MAP
                                </sub>
                                = 20.2 ms), but it cannot be captured by the direct exponential fit (cyan).
                                <strong>
                                 Middle:
                                </strong>
                                The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) include the ground-truth timescales, while direct exponential fits (cyan) underestimate the ground-truth timescales. The width of the posteriors indicates the estimation uncertainty.
                                <strong>
                                 Right:
                                </strong>
                                The convergence of the aABC algorithm is defined based on the acceptance rate accR (purple), which decreases together with the error threshold ε (green) over the iterations of the algorithm. Data are plotted from the second iteration. Initial error threshold for all fits was set to 1.
                                <strong>
                                 (b)
                                </strong>
                                Same as a for data from a linear mixture of an OU process with the timescale τ = 20 ms and an oscillation with frequency f = 2 Hz (
                                <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">
                                 Eq. 5
                                </a>
                                ). τ
                                <sub>
                                 MAP
                                </sub>
                                = 60.5 ms,
                                <strong>
                                 (c)
                                </strong>
                                Same as a for data from an inhomogeneous Poisson process with two timescales τ
                                <sub>
                                 1
                                </sub>
                                = 5 ms and τ
                                <sub>
                                 2
                                </sub>
                                = 80 ms. τ
                                <sub>
                                 1,MAP
                                </sub>
                                = 4.7 ms, τ
                                <sub>
                                 2,MAP
                                </sub>
                                = 80 ms. See
                                <a id="xref-sec-19-2" class="xref-sec" href="#sec-19">
                                 Methods 4.5
                                </a>
                                for other parameters.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-36">
                              Our method can also estimate timescales by fitting the sample PSD in the frequency domain. The aABC method recovers the ground-truth timescales by fitting the whole shape of the PSD without the need to tune the fitted frequency range, even in the presence of multiple timescales and additional noise (e.g., spiking noise) (
                              <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                               Supplementary Fig. 9
                              </a>
                              ) or multiple oscillatory components (
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Supplementary Fig. 10
                              </a>
                              ). Moreover, aABC method can uncover slow oscillations in signals that do not exhibit clear peaks in PSD due to the short duration of time-series and the low frequency resolution (
                              <a id="xref-fig-15-2" class="xref-fig" href="#F15">
                               Supplementary Fig. 9C
                              </a>
                              ). The aABC method can be used in combination with any method for computing the PSD (e.g., any window functions for removing the spectral leakage), since the exact same method applied to synthetic data would alter its sample PSD in the same way.
                             </p>
                             <p id="p-37">
                              Different summary statistics and fitting ranges may be preferred depending on the application. For example, autocorrelations allow for using additional correction methods such as jittering [
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ], whereas PSD estimation can be improved with filtering or multitapers [
                              <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">
                               67
                              </a>
                              ]. Further, selecting a smaller t
                              <sub>
                               m
                              </sub>
                              when fitting autocorrelations prevents overfitting to noise in the autocorrelation tail. Distances between summary statistics can also be computed on the logarithmic scale (e.g., log(AC) or log(PSD)). The choice of summary statistic (metric and fitting range) can influence the shape of the approximated posterior (e.g., posterior width), but the posteriors peak close to the ground-truth timescales as long as the same summary statistic is used for observed and synthetic data (
                              <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                               Supplementary Fig. 11
                              </a>
                              ). The advantage of such behavior is particularly visible when changing the range over which we compute the summary statistics (e.g., minimum and maximum frequency in PSD). The direct fitting is sensitive to the selected range and can lead to wrong estimates of timescales, whereas aABC recovers correct timescales for all ranges (
                              <a id="xref-fig-12-3" class="xref-fig" href="#F12">
                               Supplementary Fig. 6
                              </a>
                              ).
                             </p>
                             <p id="p-38">
                              In the aABC algorithm, we set a multivariate uniform prior distribution over the parameters of the generative model. The ranges of uniform prior should be broad enough to include the ground-truth values. Selecting broader priors does not affect the shape of posterior distributions obtained with the aABC and only slows down the fitting procedure (
                              <a id="xref-fig-18-1" class="xref-fig" href="#F18">
                               Supplementary Fig. 12
                              </a>
                              ). Hence, we can set wide prior distributions when a reasonable range of parameters is unknown. Timescales estimated from the direct exponential fits can be used as a potential lower bound for the priors.
                             </p>
                             <p id="p-39">
                              We evaluated the reliability of the aABC method on a wide range of timescales and trial durations and compared the results to direct fitting (
                              <a id="xref-fig-19-1" class="xref-fig" href="#F19">
                               Supplementary Fig. 13
                              </a>
                              ). The estimation error of direct fitting increases when trial durations become short relative to the timescale, whereas the aABC method always returns reliable estimates. However, the estimation of the full posterior distribution comes with a price of higher computational costs (
                              <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                               Supplementary Table 1
                              </a>
                              ) compared to point estimates. Thus, the direct fit of the sample autocorrelation may be preferred when the long time-series data is available so that statistical bias does not corrupt the results. To empirically verify whether this is the case, we implemented a pre-processing algorithm in our Python package that uses parametric bootstrapping to return an approximate error bound of the direct fit estimates (
                              <a id="xref-sec-17-1" class="xref-sec" href="#sec-17">
                               Methods 4.3.1
                              </a>
                              ,
                              <a id="xref-fig-20-1" class="xref-fig" href="#F20">
                               Supplementary Fig. 14
                              </a>
                              ).
                             </p>
                             <p id="p-40">
                              Fitting generative models with the aABC method provides a principled framework for estimating timescales that can be used with different metrics in time or frequency domain. Furthermore, joint posterior distribution of the inferred parameters allows us to examine correlations between different parameters, find manifolds of possible solutions, and identify a potential degeneracy in the parameter space (
                              <a id="xref-fig-21-1" class="xref-fig" href="#F21">
                               Supplementary Fig. 15
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              2.3 Estimating the timescale of activity in a branching network model
                             </h3>
                             <p id="p-41">
                              So far, we demonstrated that our aABC method accurately recovers the ground-truth timescales within the same model class, i.e. when the generative model and the process that produced the observed data are the same. However, the timescale inference based on OU processes is broadly applicable when the mechanism that generated the exponential decay of autocorrelation in the data is not an OU process.
                             </p>
                             <p id="p-42">
                              Here, we tested our inference method on discrete-time data from an integer-valued autoregressive model with a known ground-truth autocorrelation function. Specifically, we applied our method to estimate the timescale of the global activity in a branching network model [
                              <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">
                               68
                              </a>
                              –
                              <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">
                               70
                              </a>
                              ]. This model is often used to study the operating regime of dynamics in neuronal networks, e.g., how close the dynamics is to the critical state. A branching network consists of k interconnected binary neurons, each described by the state variable x
                              <sub>
                               i
                              </sub>
                              ∈ {0, 1}, where x
                              <sub>
                               i
                              </sub>
                              = 1 indicates that neuron i is active, and 0 that it is silent (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ). We considered a fully-connected network. Each active neuron can activate other neurons with the probability p = m/k and then, if not activated by other neurons, it becomes inactive again in the next time-step. Additionally, at every time-step, each neuron can be activated with a probability h by an external input. For a small input strength h, the state of the network’s dynamics is governed by a branching parameter m (m = 1 corresponds to the critical state). The autocorrelation function of the global activity A(t′) = Σ
                              <sub>
                               i
                              </sub>
                              x
                              <sub>
                               i
                              </sub>
                              (t′) in this network is known analytically AC(t
                              <sub>
                               j
                              </sub>
                              ) = exp(t
                              <sub>
                               j
                              </sub>
                              ln(m)) [
                              <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ]. Thus the ground-truth timescale of this activity is given by τ = – 1/ ln(m).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F4.large.jpg?width=800&height=600&carousel=1" title="The aABC inference with a generative model based on the OU process recovers the ground-truth timescale of activity in a branching network. (a) A schematic of a fully connected branching network with binary neurons. Branching parameter (m = 0.96) and number of neurons (k = 104) define the probability of activity propagation. Each neuron receives an external input with probability h = 10-3. (b) The shape of the sample autocorrelation of simulated activity in the branching network (brown) deviates from the ground truth (gray), but is accurately reproduced by the autocorrelation of synthetic data generated from a one-timescale OU process with the MAP estimate timescale from the aABC method. τMAP = 24.9. The data contained 100 trials of 500 timesteps. (c) The aABC posterior distribution includes the ground-truth timescale. τground truth = 24.5, τaABC = 24.9 ± 0.9 (mean ± std). Fitting parameters are provided in Methods 4.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The aABC inference with a generative model based on the OU process recovers the ground-truth timescale of activity in a branching network.</span> (<strong>a</strong>) A schematic of a fully connected branching network with binary neurons. Branching parameter (<em>m</em> = 0.96) and number of neurons (<em>k</em> = 10<sup>4</sup>) define the probability of activity propagation. Each neuron receives an external input with probability <em>h</em> = 10<sup>-3</sup>. (<strong>b</strong>) The shape of the sample autocorrelation of simulated activity in the branching network (brown) deviates from the ground truth (gray), but is accurately reproduced by the autocorrelation of synthetic data generated from a one-timescale OU process with the MAP estimate timescale from the aABC method. <em>τ</em><sub>MAP</sub> = 24.9. The data contained 100 trials of 500 timesteps. (<strong>c</strong>) The aABC posterior distribution includes the ground-truth timescale. <em>τ</em><sub>ground truth</sub> = 24.5, <em>τ</em><sub>aABC</sub> = 24.9 ± 0.9 (mean ± std). Fitting parameters are provided in Methods 4.5.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F4.medium.gif" width="440" height="110"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F4.medium.gif" width="440" height="110"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 4.
                               </span>
                               <span class="caption-title">
                                The aABC inference with a generative model based on the OU process recovers the ground-truth timescale of activity in a branching network.
                               </span>
                               <p id="p-43" class="first-child">
                                (
                                <strong>
                                 a
                                </strong>
                                ) A schematic of a fully connected branching network with binary neurons. Branching parameter (m = 0.96) and number of neurons (k = 10
                                <sup>
                                 4
                                </sup>
                                ) define the probability of activity propagation. Each neuron receives an external input with probability h = 10
                                <sup>
                                 -3
                                </sup>
                                . (
                                <strong>
                                 b
                                </strong>
                                ) The shape of the sample autocorrelation of simulated activity in the branching network (brown) deviates from the ground truth (gray), but is accurately reproduced by the autocorrelation of synthetic data generated from a one-timescale OU process with the MAP estimate timescale from the aABC method. τ
                                <sub>
                                 MAP
                                </sub>
                                = 24.9. The data contained 100 trials of 500 timesteps. (
                                <strong>
                                 c
                                </strong>
                                ) The aABC posterior distribution includes the ground-truth timescale. τ
                                <sub>
                                 ground truth
                                </sub>
                                = 24.5, τ
                                <sub>
                                 aABC
                                </sub>
                                = 24.9 ± 0.9 (mean ± std). Fitting parameters are provided in
                                <a id="xref-sec-19-3" class="xref-sec" href="#sec-19">
                                 Methods 4.5
                                </a>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-44">
                              We simulated the binary network model to generate the time-series data of the global activity. We then used aABC to fit the sample autocorrelation of these data with a one-timescale OU process as the generative model. The inferred posterior distribution is centered on the theoretically predicted timescale of the branching network, and the MAP estimate parameter accurately reproduces the shape of the sample autocorrelation of the network activity (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4B,C
                              </a>
                              ). These results show that our framework can be used to estimate timescales in diverse types of data with different underlying mechanisms.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              2.4 Model selection with Approximate Bayesian Computations
                             </h3>
                             <p id="p-45">
                              For experimental data, the correct generative model is usually unknown. For example, it may not be obvious a priori how many timescales should the generative model contain. Several alternative hypotheses may be plausible, and we need a procedure to select which one is more likely to explain the observed data. Assuming that the autocorrelation function or PSD is a sufficient summary statistic for estimating timescales, we can use aABC to approximate the Bayes factor for selecting between alternative models [
                              <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">
                               71
                              </a>
                              –
                              <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">
                               73
                              </a>
                              ]. Model selection based on ABC can produce inconsistent results when the summary statistic is insufficient [
                              <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">
                               74
                              </a>
                              ], but whether this is likely the case can be verified empirically [
                              <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">
                               73
                              </a>
                              ]. Specifically, the summary statistic can be used for model selection with ABC if their mathematical expectation is significantly different for the two models.
                             </p>
                             <p id="p-46">
                              Based on this empirical procedure [
                              <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">
                               73
                              </a>
                              ], we developed a method for selecting between two alternative models M
                              <sub>
                               1
                              </sub>
                              and M
                              <sub>
                               2
                              </sub>
                              using their aABC fits. Models are compared using a goodness of fit measure that describes how well each model fits the data. The goodness of fit can be measured by the distance d between the summary statistic (e.g., autocorrelation or PSD) of synthetic and observed data (i.e. residual errors,
                              <a id="xref-disp-formula-9-2" class="xref-disp-formula" href="#disp-formula-9">
                               Eq. 9
                              </a>
                              ). For fair comparison, the same summary-statistics and fitting range should be used for fitting both models. Since d is a noisy measure because of the finite sample size and uncertainty in the model parameters, we compare the distributions of distances generated by two models with parameters sampled from their posterior distributions. To approximate the distributions of distances, we generate multiple samples of synthetic data from each model with parameters drawn from its posterior distribution and compute the distance d for each sample. If the distributions of distances are significantly different (i.e. expectations of the summary statistic for two models are significantly different [
                              <a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">
                               73
                              </a>
                              ]), then we continue with the model selection, otherwise the summary statistic is insufficient to distinguish these models.
                             </p>
                             <p id="p-47">
                              For selecting between M
                              <sub>
                               1
                              </sub>
                              and M
                              <sub>
                               2
                              </sub>
                              , we use the distributions of distances to estimate the Bayes factor, which is the ratio of marginal likelihoods of the two models and accounts for the model complexity [
                              <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">
                               75
                              </a>
                              ]. Assuming both models are a priori equally probable (p(M
                              <sub>
                               1
                              </sub>
                              ) = p(M
                              <sub>
                               2
                              </sub>
                              )), the Bayes factor can be approximated using the models’ acceptance rates for a specific error threshold ε [
                              <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">
                               71
                              </a>
                              ,
                              <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">
                               74
                              </a>
                              ,
                              <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">
                               76
                              </a>
                              ]
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-48">
                              B
                              <sub>
                               21
                              </sub>
                              (ε) > 1 indicates that the model M
                              <sub>
                               2
                              </sub>
                              is more likely to explain the observed data and vice versa. To eliminate the dependence on a specific error threshold, we compute the acceptance rates and the Bayes factor with a varying error threshold. For each error threshold ε, the acceptance rate is given by the cumulative distribution function of the distances CDF
                              <sub>
                               M
                               <sub>
                                i
                               </sub>
                              </sub>
                              (ε) = p
                              <sub>
                               M
                               <sub>
                                i
                               </sub>
                              </sub>
                              (d < ε) = accR
                              <sub>
                               M
                               <sub>
                                i
                               </sub>
                              </sub>
                              (ε) (i = 1, 2). Hence, the ratio between CDFs of two models gives the value of the Bayes factor for every error threshold B
                              <sub>
                               21
                              </sub>
                              (ε) = CDF
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              (ε)/CDF
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              (ε). We select the model M
                              <sub>
                               2
                              </sub>
                              if B
                              <sub>
                               21
                              </sub>
                              (ε) > 1, i.e. if CDF
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              (ε) > CDF
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              (ε) for all ε, and vice versa.
                             </p>
                             <p id="p-49">
                              We evaluated our model selection method using synthetic data from three example processes with known ground truth, so that the correct number of timescales is known. Specifically, we used an OU process with a single timescale (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              ) and two different examples of an inhomogeneous Poisson process with two timescales. In the first example of an inhomogeneous Poisson process, the ground-truth timescales were well separated, so that the shape of the data autocorrelation suggested that the underlying process had multiple timescales (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5D
                              </a>
                              ). In the second example of an inhomogeneous Poisson process, we chose the ground-truth timescales to be more similar, so that a simple visual inspection of the data autocorrelation could not clearly suggest the number of timescales in the underlying process (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5G
                              </a>
                              ). For all three example processes, we fitted the data with one-timescale (M
                              <sub>
                               1
                              </sub>
                              ) and two-timescale (M
                              <sub>
                               2
                              </sub>
                              ) generative models using aABC and selected between these models by computing the Bayes factors. The one- and two-timescale models were based on a single OU process or a linear mixture of two OU processes, respectively. For the data from inhomogeneous Poisson processes, the generative model also incorporated an inhomogeneous Poisson noise.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F5.large.jpg?width=800&height=600&carousel=1" title="Model selection with ABC. (a-c) Data are generated from a one-timescale OU process with τ = 20 ms. (a) The data autocorrelation (brown) is fitted with a one-timescale (olive, M1) and two-timescale (orange, M2) generative models (sample autocorrelations for MAP estimate parameters are shown). (b) Marginal posterior distribution of the timescale estimated using the one-timescale model includes the ground truth and has small variance (left). Marginal posterior distributions of two timescales estimated using the two-timescale model heavily overlap and have large variance (right). (c) Cumulative distributions of distances d for two models, CDFMi(ε). Since CDFM2 (ε) < CDFM1 (ε) for all ε, the one-timescale model is selected (green box in B). (d-f) Data are generated from an inhomogeneous Poisson process with two timescales: τ1 = 5 ms, τ2 = 80 ms. (d) Same format as A. (e) Marginal posterior distributions for two timescales estimated by the two-timescale model include the ground truth (right). Marginal posterior distributions for the timescale estimated by the one-timescale model falls in between the ground-truth timescales (left). (f) Cumulative distributions of distances d for two models. Since CDFM2 (ε) > CDFM1 (ε) for all ε, the two-timescale model is selected (green box in E). (g-i) Same as d-f, but for τ1 = 20 ms, τ2 = 80 ms. For all fits, CDFMi (ε) are computed from 1000 samples. Simulation and fitting parameters are provided in 4.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Model selection with ABC.</span> (<strong>a-c</strong>) Data are generated from a one-timescale OU process with <em>τ</em> = 20 ms. (<strong>a</strong>) The data autocorrelation (brown) is fitted with a one-timescale (olive, <em>M</em><sub>1</sub>) and two-timescale (orange, <em>M</em><sub>2</sub>) generative models (sample autocorrelations for MAP estimate parameters are shown). (<strong>b</strong>) Marginal posterior distribution of the timescale estimated using the one-timescale model includes the ground truth and has small variance (<strong>left</strong>). Marginal posterior distributions of two timescales estimated using the two-timescale model heavily overlap and have large variance (<strong>right</strong>). (<strong>c</strong>) Cumulative distributions of distances <em>d</em> for two models, CDF<em><sub>M<sub>i</sub></sub></em>(<em>ε</em>). Since CDF<sub><em>M</em><sub>2</sub></sub> (<em>ε</em>) < CDF<sub><em>M</em><sub>1</sub></sub> (<em>ε</em>) for all <em>ε</em>, the one-timescale model is selected (green box in B). (<strong>d-f</strong>) Data are generated from an inhomogeneous Poisson process with two timescales: <em>τ</em><sub>1</sub> = 5 ms, <em>τ</em><sub>2</sub> = 80 ms. (<strong>d</strong>) Same format as A. (<strong>e</strong>) Marginal posterior distributions for two timescales estimated by the two-timescale model include the ground truth (<strong>right</strong>). Marginal posterior distributions for the timescale estimated by the one-timescale model falls in between the ground-truth timescales (<strong>left</strong>). (<strong>f</strong>) Cumulative distributions of distances <em>d</em> for two models. Since CDF<sub><em>M</em><sub>2</sub></sub> (<em>ε</em>) > CDF<sub><em>M</em><sub>1</sub></sub> (<em>ε</em>) for all <em>ε</em>, the two-timescale model is selected (green box in E). (<strong>g-i</strong>) Same as d-f, but for <em>τ</em><sub>1</sub> = 20 ms, <em>τ</em><sub>2</sub> = 80 ms. For all fits, CDF<em><sub>M<sub>i</sub></sub></em> (<em>ε</em>) are computed from 1000 samples. Simulation and fitting parameters are provided in 4.5.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F5.medium.gif" width="440" height="248"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F5.medium.gif" width="440" height="248"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 5.
                               </span>
                               <span class="caption-title">
                                Model selection with ABC.
                               </span>
                               <p id="p-50" class="first-child">
                                (
                                <strong>
                                 a-c
                                </strong>
                                ) Data are generated from a one-timescale OU process with τ = 20 ms. (
                                <strong>
                                 a
                                </strong>
                                ) The data autocorrelation (brown) is fitted with a one-timescale (olive, M
                                <sub>
                                 1
                                </sub>
                                ) and two-timescale (orange, M
                                <sub>
                                 2
                                </sub>
                                ) generative models (sample autocorrelations for MAP estimate parameters are shown). (
                                <strong>
                                 b
                                </strong>
                                ) Marginal posterior distribution of the timescale estimated using the one-timescale model includes the ground truth and has small variance (
                                <strong>
                                 left
                                </strong>
                                ). Marginal posterior distributions of two timescales estimated using the two-timescale model heavily overlap and have large variance (
                                <strong>
                                 right
                                </strong>
                                ). (
                                <strong>
                                 c
                                </strong>
                                ) Cumulative distributions of distances d for two models, CDF
                                <sub>
                                 M
                                 <sub>
                                  i
                                 </sub>
                                </sub>
                                (ε). Since CDF
                                <sub>
                                 M
                                 <sub>
                                  2
                                 </sub>
                                </sub>
                                (ε) < CDF
                                <sub>
                                 M
                                 <sub>
                                  1
                                 </sub>
                                </sub>
                                (ε) for all ε, the one-timescale model is selected (green box in B). (
                                <strong>
                                 d-f
                                </strong>
                                ) Data are generated from an inhomogeneous Poisson process with two timescales: τ
                                <sub>
                                 1
                                </sub>
                                = 5 ms, τ
                                <sub>
                                 2
                                </sub>
                                = 80 ms. (
                                <strong>
                                 d
                                </strong>
                                ) Same format as A. (
                                <strong>
                                 e
                                </strong>
                                ) Marginal posterior distributions for two timescales estimated by the two-timescale model include the ground truth (
                                <strong>
                                 right
                                </strong>
                                ). Marginal posterior distributions for the timescale estimated by the one-timescale model falls in between the ground-truth timescales (
                                <strong>
                                 left
                                </strong>
                                ). (
                                <strong>
                                 f
                                </strong>
                                ) Cumulative distributions of distances d for two models. Since CDF
                                <sub>
                                 M
                                 <sub>
                                  2
                                 </sub>
                                </sub>
                                (ε) > CDF
                                <sub>
                                 M
                                 <sub>
                                  1
                                 </sub>
                                </sub>
                                (ε) for all ε, the two-timescale model is selected (green box in E). (
                                <strong>
                                 g-i
                                </strong>
                                ) Same as d-f, but for τ
                                <sub>
                                 1
                                </sub>
                                = 20 ms, τ
                                <sub>
                                 2
                                </sub>
                                = 80 ms. For all fits, CDF
                                <sub>
                                 M
                                 <sub>
                                  i
                                 </sub>
                                </sub>
                                (ε) are computed from 1000 samples. Simulation and fitting parameters are provided in 4.5.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-51">
                              For the example OU process with a single timescale, the one- and two-timescale models fitted the shape of the data autocorrelation almost equally well (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              ). The marginal posterior distributions of two timescales estimated by the two-timescale model heavily overlap around their peak values (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig. 5B
                              </a>
                              ), which indicates that the one-timescale model possibly better describes the data. For the two-timescale model we enforce the ordering τ
                              <sub>
                               1
                              </sub>
                              < τ
                              <sub>
                               2
                              </sub>
                              , which generates the difference between their distributions. To select between two models, we compare the cumulative distributions of distances CDF
                              <sub>
                               M
                               <sub>
                                i
                               </sub>
                              </sub>
                              (ε) (
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Fig. 5C
                              </a>
                              ). Although the two-timescale model has more parameters, it has significantly larger distances than the one-timescale model (Wilcoxon rank-sum test, P = 0.002, mean d
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              = 6 × 10
                              <sup>
                               -5
                              </sup>
                              , mean d
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              = 8 × 10
                              <sup>
                               -5
                              </sup>
                              ). The two-timescale model has a larger average distance because its posterior distribution has larger variance (larger uncertainty), which leads to a higher possibility to sample a combination of parameters with a larger distance. Since CDF
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              (ε) < CDF
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              (ε) (i.e. B
                              <sub>
                               21
                              </sub>
                              (ε) < 1), the one-timescale model is preferred over the two-timescale model, in agreement with the ground-truth generative process.
                             </p>
                             <p id="p-52">
                              For both example inhomogeneous Poisson processes with two timescales, the shape of the data autocorrelation is better matched by the two-timescale than by the one-timescale model (the difference is subtle for the second example,
                              <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                               Fig. 5D,G
                              </a>
                              ). The marginal posterior distributions of two timescales estimated by the two-timescale model are clearly separated and include the ground-truth values, whereas the timescale estimated by the one-timescale model is in between the two ground-truth values (
                              <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                               Fig. 5E,H
                              </a>
                              ). The two-timescale model has significantly smaller distances (Wilcoxon rank-sum test,
                              <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                               Fig. 5F
                              </a>
                              : P < 10
                              <sup>
                               -10
                              </sup>
                              , mean d
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              = 6 × 10
                              <sup>
                               -4
                              </sup>
                              , mean d
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              = 1.5 × 10
                              <sup>
                               -5
                              </sup>
                              ;
                              <a id="xref-fig-5-10" class="xref-fig" href="#F5">
                               Fig. 5I
                              </a>
                              : P < 10
                              <sup>
                               -10
                              </sup>
                              , mean d
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              = 10
                              <sup>
                               -6
                              </sup>
                              , mean d
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              = 7 × 10
                              <sup>
                               -7
                              </sup>
                              ). Since CDF
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              (ε) > CDF
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              (ε) (i.e. B
                              <sub>
                               21d
                              </sub>
                              (ε) > 1) for all error thresholds, the two-timescale model provides a better description of the data for both examples, in agreement with the ground truth. Thus, our method selects the correct generative model even for a challenging case where the shape of the data autocorrelation does not suggest the existence of multiple timescales. Our method can be used to discriminate between a broad class of models, e.g., slow periodic (e.g., slow oscillation) versus aperiodic (e.g., slow exponential decay) components in the time-series (
                              <a id="xref-fig-22-1" class="xref-fig" href="#F22">
                               Supplementary Fig. 16
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              2.5 Estimating timescales of ongoing neural activity
                             </h3>
                             <p id="p-53">
                              To illustrate an application of our computational framework to experimental data, we inferred the timescales of ongoing spiking activity in the primate visual cortex. The spiking activity was recorded from the visual area V4 with a 16-channels micro-electrode array [
                              <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">
                               77
                              </a>
                              ]. During recordings, a monkey was fixating a central dot on a blank screen for 3 s on each trial. To estimate the timescales, we pooled the spiking activity recorded across all channels and computed the spike-count autocorrelation of the pooled activity with a bin-size of 1 ms.
                             </p>
                             <p id="p-54">
                              Previously, the autocorrelation of neural activity in several brain regions was modeled as an exponential decay with a single timescale [
                              <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ]. To determine whether a single timescale is sufficient to describe the temporal dynamics of neural activity in our data, we compared the one-timescale (M
                              <sub>
                               1
                              </sub>
                              ) and two-timescale (M
                              <sub>
                               2
                              </sub>
                              ) models and selected the model that better described the data. As a generative model for neural spike-counts, we used a doubly-stochastic process [
                              <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">
                               64
                              </a>
                              ,
                              <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">
                               65
                              </a>
                              ], where spike-counts are generated from an instantaneous firing rate modelled as one OU process (M
                              <sub>
                               1
                              </sub>
                              ) or a mixture of two OU processes (M
                              <sub>
                               2
                              </sub>
                              ). To account for the non-Poisson statistics of the spike-generation process, we sampled spike-counts from a gamma distribution [
                              <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">
                               78
                              </a>
                              ] (
                              <a id="xref-sec-13-2" class="xref-sec" href="#sec-13">
                               Methods 4.1.3
                              </a>
                              ). We fitted both models with the aABC algorithm and selected between the models using Bayes factors (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 6A-C
                              </a>
                              ). The two-timescale model provided a better description for the data, since it had smaller distances and CDF
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              (ε) > CDF
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              (ε) for all error thresholds (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig. 6C
                              </a>
                              , Wilcoxon rank-sum test, P < 10
                              <sup>
                               -10
                              </sup>
                              , mean d
                              <sub>
                               M
                               <sub>
                                1
                               </sub>
                              </sub>
                              = 8 × 10
                              <sup>
                               -4
                              </sup>
                              , mean d
                              <sub>
                               M
                               <sub>
                                2
                               </sub>
                              </sub>
                              = 2 × 10
                              <sup>
                               -4
                              </sup>
                              ).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F6.large.jpg?width=800&height=600&carousel=1" title="Estimating timescales of ongoing neural activity and comparing hypotheses about their number with aABC. (A) Left: The autocorrelation of neural spike-counts (brown) is fitted with a one-timescale doubly-stochastic model using aABC (olive, sample autocorrelation for MAP estimate parameters). Right: Posterior distribution of the timescale. τMAP = 58 ms. (B) Left: Same data as in A is fitted directly with a double exponential function (cyan) and with a two-timescale doubly-stochastic model using aABC (orange, sample autocorrelation for MAP estimate parameters). Right: Timescales estimated by the direct exponential fit (τ1 - blue, τ2 -cyan) and marginal posterior distributions of timescales inferred with aABC (τ1 - red, τ2 - orange). τ1,exp = 5 ms, τ2,exp = 57 ms, τ1,MAP = 8 ms, τ2,MAP = 70 ms. (C) Cumulative distribution of distances d for one-timescale (M1) and two-timescale (M2) models. Since CDFM2 (ε) > CDFM1 (ε) for all ε, the two-timescale model is selected. (D) Cumulative distributions of distances d between the autocorrelation of neural data and synthetic data from the two-timescale doubly-stochastic model with parameters either from the direct fit (cyan) or MAP estimate with aABC (red). The MAP parameters have smaller distances, i.e. describe the autocorrelation of neural data more accurately than the direct fit. Fitting parameters are provided in Methods 4.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Estimating timescales of ongoing neural activity and comparing hypotheses about their number with aABC.</span> (<strong>A</strong>) <strong>Left:</strong> The autocorrelation of neural spike-counts (brown) is fitted with a one-timescale doubly-stochastic model using aABC (olive, sample autocorrelation for MAP estimate parameters). <strong>Right:</strong> Posterior distribution of the timescale. <em>τ</em><sub>MAP</sub> = 58 ms. (<strong>B</strong>) <strong>Left:</strong> Same data as in A is fitted directly with a double exponential function (cyan) and with a two-timescale doubly-stochastic model using aABC (orange, sample autocorrelation for MAP estimate parameters). <strong>Right:</strong> Timescales estimated by the direct exponential fit (<em>τ</em><sub>1</sub> - blue, <em>τ</em><sub>2</sub> -cyan) and marginal posterior distributions of timescales inferred with aABC (<em>τ</em><sub>1</sub> - red, <em>τ</em><sub>2</sub> - orange). <em>τ</em><sub>1,exp</sub> = 5 ms, <em>τ</em><sub>2,exp</sub> = 57 ms, <em>τ</em><sub>1,MAP</sub> = 8 ms, <em>τ</em><sub>2,MAP</sub> = 70 ms. (<strong>C</strong>) Cumulative distribution of distances <em>d</em> for one-timescale (<em>M</em><sub>1</sub>) and two-timescale (<em>M</em><sub>2</sub>) models. Since CDF<sub><em>M</em><sub>2</sub></sub> (<em>ε</em>) > CDF<sub><em>M</em><sub>1</sub></sub> (<em>ε</em>) for all <em>ε</em>, the two-timescale model is selected. (<strong>D</strong>) Cumulative distributions of distances <em>d</em> between the autocorrelation of neural data and synthetic data from the two-timescale doubly-stochastic model with parameters either from the direct fit (cyan) or MAP estimate with aABC (red). The MAP parameters have smaller distances, i.e. describe the autocorrelation of neural data more accurately than the direct fit. Fitting parameters are provided in Methods 4.5.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F6.medium.gif" width="440" height="177"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F6.medium.gif" width="440" height="177"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 6.
                               </span>
                               <span class="caption-title">
                                Estimating timescales of ongoing neural activity and comparing hypotheses about their number with aABC.
                               </span>
                               <p id="p-55" class="first-child">
                                (
                                <strong>
                                 A
                                </strong>
                                )
                                <strong>
                                 Left:
                                </strong>
                                The autocorrelation of neural spike-counts (brown) is fitted with a one-timescale doubly-stochastic model using aABC (olive, sample autocorrelation for MAP estimate parameters).
                                <strong>
                                 Right:
                                </strong>
                                Posterior distribution of the timescale. τ
                                <sub>
                                 MAP
                                </sub>
                                = 58 ms. (
                                <strong>
                                 B
                                </strong>
                                )
                                <strong>
                                 Left:
                                </strong>
                                Same data as in A is fitted directly with a double exponential function (cyan) and with a two-timescale doubly-stochastic model using aABC (orange, sample autocorrelation for MAP estimate parameters).
                                <strong>
                                 Right:
                                </strong>
                                Timescales estimated by the direct exponential fit (τ
                                <sub>
                                 1
                                </sub>
                                - blue, τ
                                <sub>
                                 2
                                </sub>
                                -cyan) and marginal posterior distributions of timescales inferred with aABC (τ
                                <sub>
                                 1
                                </sub>
                                - red, τ
                                <sub>
                                 2
                                </sub>
                                - orange). τ
                                <sub>
                                 1,exp
                                </sub>
                                = 5 ms, τ
                                <sub>
                                 2,exp
                                </sub>
                                = 57 ms, τ
                                <sub>
                                 1,MAP
                                </sub>
                                = 8 ms, τ
                                <sub>
                                 2,MAP
                                </sub>
                                = 70 ms. (
                                <strong>
                                 C
                                </strong>
                                ) Cumulative distribution of distances d for one-timescale (M
                                <sub>
                                 1
                                </sub>
                                ) and two-timescale (M
                                <sub>
                                 2
                                </sub>
                                ) models. Since CDF
                                <sub>
                                 M
                                 <sub>
                                  2
                                 </sub>
                                </sub>
                                (ε) > CDF
                                <sub>
                                 M
                                 <sub>
                                  1
                                 </sub>
                                </sub>
                                (ε) for all ε, the two-timescale model is selected. (
                                <strong>
                                 D
                                </strong>
                                ) Cumulative distributions of distances d between the autocorrelation of neural data and synthetic data from the two-timescale doubly-stochastic model with parameters either from the direct fit (cyan) or MAP estimate with aABC (red). The MAP parameters have smaller distances, i.e. describe the autocorrelation of neural data more accurately than the direct fit. Fitting parameters are provided in
                                <a id="xref-sec-19-4" class="xref-sec" href="#sec-19">
                                 Methods 4.5
                                </a>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-56">
                              We further compared our method with a direct exponential fit of the sample autocorrelation which is usually employed to infer the timescales of neural activity [
                              <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ,
                              <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ,
                              <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              –
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ]. We fitted the sample autocorrelation with a double exponential function
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              up to the same t
                              <sub>
                               m
                              </sub>
                              = 150 ms as in the aABC method and compared the result with the two-timescale aABC fit (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Fig. 6B
                              </a>
                              ). Similar to what we observed with synthetic data, the direct exponential fit produced timescales that were systematically smaller than the MAP estimates with the aABC fit. Including all the time lags in exponential fitting results in even larger bias. Since the ground-truth timescales are not available for biological data, we used a sampling procedure to evaluate whether the data is better described by the timescales from the direct fit or from the MAP estimates with the aABC fit. We generated multiple samples of synthetic data using the two-timescale doubly-stochastic generative model with the parameters from either the direct fit or MAP estimates from aABC. For each sample, we measured the distance d between the autocorrelation of synthetic and neural data to obtain the distribution of distances for both types of fits (
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               Fig. 6D
                              </a>
                              ). The distances were significantly smaller for synthetic data generated with the parameters from the MAP estimate with aABC than from the direct exponential fit (Wilcoxon rank-sum test, P < 10
                              <sup>
                               -10
                              </sup>
                              , mean distance of MAP parameters 10
                              <sup>
                               -4
                              </sup>
                              , mean distance of exponential fit parameters 3× 10
                              <sup>
                               -4
                              </sup>
                              ). Our method performs better because it accounts for the bias in the autocorrelation shape which is ignored by the direct fit. These results suggest that our method estimates the timescales of neural activity more accurately than a direct exponential fit and, moreover, allows for comparing alternative hypotheses about the underlying dynamics.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             3 Discussion
                            </h2>
                            <p id="p-57">
                             We demonstrated that direct fitting the shape of sample autocorrelation or PSD often fails to recover the correct timescales due to a statistical bias arising from finite sample size. Since the bias depends on the duration of time-series (
                             <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                              Fig. 1
                             </a>
                             , left), comparing timescales estimated from direct fits of autocorrelations or PSDs of experimental data with different duration can lead to misleading interpretations.
                            </p>
                            <p id="p-58">
                             To overcome this problem, we developed a flexible computational framework based on adaptive Approximate Bayesian Computations that is broadly applicable to different types of data. Our framework fits the sample autocorrelation or PSD of the data with a generative model which is based on a mixture of Ornstein-Uhlenbeck processes (one for each estimated timescale) and can incorporate additional temporal structure and noise. Our method infers a posterior distribution of timescales consistent with the observed data. The width of the posterior distribution depends on the noise and amount of data and quantifies the estimation uncertainty, e.g., it provides a confidence interval for estimated timescales. The posterior distributions can be used for model selection to compare alternative hypotheses about the dynamics of the underlying process. This framework is suitable for comparing timescales between datasets with different durations and between processes that exhibit different temporal dynamics (e.g., oscillations). Our approach is not sensitive to the specific method for computing autocorrelations or PSDs and does not require fine-tuning of the fitting ranges, since the same method applies to observed and synthetic data.
                            </p>
                            <p id="p-59">
                             While previous work [
                             <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ] attributed errors in the estimated timescales to fitting noise in the tail of autocorrelation, we find that the main reason is the statistical bias in the sample autocorrelation. This bias arises primarily due to the deviation of the sample mean from the ground-truth mean. If the ground-truth mean of the process is known, then using the true mean instead of the sample mean for computing the autocorrelation largely eliminates the bias. In this case, direct fitting can produce satisfactory estimates, but our method additionally provides the measure of estimation uncertainty. When the true mean is unknown but it is known that it is the same across all trials, it is beneficial to estimate a single sample mean from the whole dataset instead of estimating the mean for trials individually. However, this assumption does not always hold. For example, the mean of spiking neural activity can change across trials because of changes in animal’s behavioral state. If the assumption of a constant mean is violated in the data, estimating the sample mean from the whole dataset leads to strong distortions of the autocorrelation shape introducing additional slow timescales [
                             <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">
                              79
                             </a>
                             ].
                            </p>
                            <p id="p-60">
                             We validated our method using data from synthetic processes, where the exponential decay rate of autocorrelation is known analytically by design. We further illustrated the application of our method to spiking neural activity data, where the underlying ground-truth process is unknown. In this case, the posterior distributions estimated by our method can be used to approximate the Bayes factor for selecting between different generative models that represent alternative hypotheses about neural dynamics. Since the Bayes factor takes the model complexity into account, it can be used for comparing models with different number of parameters, e.g., to infer the number of timescales in the data.
                            </p>
                            <p id="p-61">
                             Here, we focused on exponentially decaying autocorrelation shapes (i.e. Lorentzian PSD shape with the 1/f power-law exponent of 2) since the definition of timescales based on the exponential decay rates of the autocorrelation is widely used in the literature [
                             <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ,
                             <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             –
                             <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                              40
                             </a>
                             ] and has a clear interpretation as a timescale of a generative dynamical system. While it is true that many types of data exhibit PSD with 1/f exponents deviating from 2 [
                             <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">
                              42
                             </a>
                             ,
                             <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">
                              80
                             </a>
                             ], there is no universally accepted answer to what would be the definition of timescale for these types of data and what is the nature of processes generating this behavior in PSD. Indeed, a prominent hypothesis is that the 1/f PSD arises from a mixture of processes with exponential autocorrelations and different timescales [
                             <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ,
                             <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">
                              42
                             </a>
                             ,
                             <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">
                              80
                             </a>
                             ,
                             <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">
                              81
                             </a>
                             ] that can be modeled as a mixture of OU processes. For example, in neuroscience, a combination of excitatory (AMPA) and inhibitory (GABA) synaptic currents with distinct timescales was suggested as a potential mechanism for creating different 1/f exponents [
                             <a id="xref-ref-80-3" class="xref-bibr" href="#ref-80">
                              80
                             </a>
                             ]. However, sometimes we are interested in separating a process with a well-defined timescale from 1/f background. To handle such cases, we introduced an augmented generative model (
                             <a id="xref-sec-14-1" class="xref-sec" href="#sec-14">
                              Methods 4.1.4
                             </a>
                             ) which can be used to estimate timescales in the presence of a background process with an arbitrary shape of PSD (e.g., 1/f exponents other than 2). This generative model is agnostic to the nature of the process generating the background activity and directly models the desired PSD shape. We can use this model to simultaneously estimate the 1/f exponent and exponential decay timescales (
                             <a id="xref-fig-23-1" class="xref-fig" href="#F23">
                              Supplementary Fig. 17
                             </a>
                             ).
                            </p>
                            <p id="p-62">
                             The general framework of inferring timescales with aABC based on OU processes can be adapted to various types of data, different generative models and summary statistics using our Python package. We implemented a set of generative models, summary statistic computation methods and distance functions in the abcTau package. Moreover, the modular implementation of our Python package allows users to easily incorporate additional types of dynamics and non-stationarities into customized generative models, or use other types of summary statistic which can be added directly to the package. Thus, the package can be extended to a wide range of applications in different fields including neuroscience, cellular biology, epidemiology, and physics. Our framework provides a principled method for estimating timescales using summary statistics in time or frequency domain. This approach is particularly favorable for data organized in short trials or trials of different durations, when standard direct fitting is unreliable, and it allows for comparing timescales between datasets with different durations and data amount.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             4 Methods
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              4.1 Generative models
                             </h3>
                             <p id="p-63">
                              We used several generative models based on a linear mixture of OU processes—one for each estimated timescale—sometimes augmented with additional temporal structure (e.g., oscillations) and noise.
                             </p>
                             <div id="sec-11" class="subsection">
                              <h4>
                               4.1.1 Ornstein–Uhlenbeck process with multiple timescales
                              </h4>
                              <p id="p-64">
                               We define an OU process with multiple timescales A(t′) as a linear mixture of OU processes A
                               <sub>
                                k
                               </sub>
                               (t′) with timescales τ
                               <sub>
                                k
                               </sub>
                               , k ∈ {1,…, n}, zero mean and unit variance:
                               <span class="disp-formula" id="disp-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-65">
                               Here n is the number of timescales in the mixture and c
                               <sub>
                                k
                               </sub>
                               are the mixing coefficients. We simulate each OU process A
                               <sub>
                                k
                               </sub>
                               by iterating its time-discrete version using the Euler scheme [
                               <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">
                                61
                               </a>
                               ]
                               <span class="disp-formula" id="disp-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the discretization time-step and
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is a random number generated from a normal distribution. We set the unit variance for each OU process Var(A
                               <sub>
                                k
                               </sub>
                               ) = D
                               <sub>
                                k
                               </sub>
                               τ
                               <sub>
                                k
                               </sub>
                               = 1 by fixing D
                               <sub>
                                k
                               </sub>
                               = 1/τ
                               <sub>
                                k
                               </sub>
                               . The parameter-vector θ for a linear mixture of n OU processes consists of 2n – 1 values: n timescales τ
                               <sub>
                                k
                               </sub>
                               and n – 1 coefficients c
                               <sub>
                                k
                               </sub>
                               (see
                               <a id="xref-disp-formula-13-2" class="xref-disp-formula" href="#disp-formula-13">
                                Eq. 13
                               </a>
                               ).
                              </p>
                              <p id="p-66">
                               We match the mean and variance of the multi-timescale OU process to the sample mean
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and sample variance
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               of the observed data using a linear transformation:
                               <span class="disp-formula" id="disp-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-67">
                               We use the process Atrans(t′) as a generative model for data fitting and hypothesis testing (
                               <a id="xref-sec-15-3" class="xref-sec" href="#sec-15">
                                Methods 4.2
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               4.1.2 Multi-timescale Ornstein–Uhlenbeck process with an oscillation
                              </h4>
                              <p id="p-68">
                               To obtain a generative model with an oscillation, we add to a multi-timescale OU process (
                               <a id="xref-disp-formula-13-3" class="xref-disp-formula" href="#disp-formula-13">
                                Eq. 13
                               </a>
                               ) an oscillatory component with the weight c
                               <sub>
                                k+1
                               </sub>
                               :
                               <span class="disp-formula" id="disp-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-69">
                               For the analysis in
                               <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                                Fig. 1B
                               </a>
                               and in
                               <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                Fig. 3B
                               </a>
                               , we assumed that the frequency f is known and only fitted the weight c
                               <sub>
                                k+1
                               </sub>
                               , showing that the bias problem for the direct fit persists even if we know the correct frequency of oscillation. The frequency f can be fitted with aABC as an additional free parameter (
                               <a id="xref-fig-15-3" class="xref-fig" href="#F15">
                                Supplementary Fig. 9
                               </a>
                               and
                               <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                                10
                               </a>
                               ). For each trial, we draw the phase φ independently from a uniform distribution on [0, 2π]. We use the linear transformation in
                               <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">
                                Eq. 15
                               </a>
                               to match the mean and variance of this generative process to the observed data.
                              </p>
                             </div>
                             <div id="sec-13" class="subsection">
                              <h4>
                               4.1.3 Doubly stochastic process with multiple timescales
                              </h4>
                              <p id="p-70">
                               The doubly stochastic process with multiple timescales is generated in two steps: first generating time-varying rate and then generating event-counts from this rate. To generate the time-varying rate, we scale, shift and rectify a multi-timescale OU process (
                               <a id="xref-disp-formula-13-4" class="xref-disp-formula" href="#disp-formula-13">
                                Eq. 13
                               </a>
                               ) using the transformation
                               <span class="disp-formula" id="disp-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-71">
                               The resulting rate A
                               <sub>
                                trans
                               </sub>
                               (t′) is non-negative and for μ′ ≫ σ it has the mean
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and variance
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . We then draw event-counts s for each time-bin
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               from an event-count distribution
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the mean event-count and
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the bin size (in our simulations Δt′ = 1 ms). A frequent choice of
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is a Poisson distribution
                               <span class="disp-formula" id="disp-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               which results in an inhomogeneous Poisson process.
                              </p>
                              <p id="p-72">
                               To match the mean and variance of the doubly stochastic process to the observed data, we need to estimate the mean rate μ′ and both the variance of the rate σ′
                               <sup>
                                2
                               </sup>
                               and the variance of the event-count distribution
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . According to the law of total expectation, the mean rate
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the sample mean of the observed event-counts. According to the law of total variance [
                               <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">
                                65
                               </a>
                               ], the total variance of event-counts σ
                               <sup>
                                2
                               </sup>
                               arises from two contributions: the variance of the rate and the variance of the event-count distribution:
                               <span class="disp-formula" id="disp-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-73">
                               For the Poisson distribution, the variance of the event-count distribution is equal to its mean:
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . However, the condition of equal mean and variance does not always hold in experimental data [
                               <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">
                                78
                               </a>
                               ]. Therefore, we also use other event-count distributions, in particular a Gaussian and gamma distribution. We define α as the variance over mean ratio of the eventcount distribution
                               <span class="inline-formula" id="inline-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . For the Poisson distribution, α = 1 always holds. For other distributions, we assume that α is constant (i.e. does not depend on the rate). With this assumption, the law of total variance
                               <a id="xref-disp-formula-19-2" class="xref-disp-formula" href="#disp-formula-19">
                                Eq. 19
                               </a>
                               becomes
                               <span class="disp-formula" id="disp-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the mean rate. From
                               <a id="xref-disp-formula-20-1" class="xref-disp-formula" href="#disp-formula-20">
                                Eq. 20
                               </a>
                               we find the rate variance
                               <span class="disp-formula" id="disp-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the sample variance of event-counts in the observed data. We find that with the correct values of μ′, σ′
                               <sup>
                                2
                               </sup>
                               and α, both the Gaussian and gamma distributions of event-counts produce comparable estimates of timescales (
                               <a id="xref-fig-24-1" class="xref-fig" href="#F24">
                                Supplementary Fig. 18
                               </a>
                               ).
                              </p>
                              <p id="p-74">
                               To calculate σ′
                               <sup>
                                2
                               </sup>
                               with
                               <a id="xref-disp-formula-21-1" class="xref-disp-formula" href="#disp-formula-21">
                                Eq. 21
                               </a>
                               , we first need to estimate a from the observed data. We estimate a from the drop of autocorrelation of event-counts between the time-lags t
                               <sub>
                                0
                               </sub>
                               and t
                               <sub>
                                1
                               </sub>
                               . Since eventcounts in different time-bins are drawn independently, this drop mainly reflects the difference between the total variance of event-counts and variance of the rate (
                               <a id="xref-disp-formula-19-3" class="xref-disp-formula" href="#disp-formula-19">
                                Eq. 19
                               </a>
                               , we neglect a small decrease of the rate autocorrelation between t
                               <sub>
                                0
                               </sub>
                               and t
                               <sub>
                                1
                               </sub>
                               ) and does not depend on timescales. Thus, we find a with a grid search that minimizes the distance between the autocorrelation at t
                               <sub>
                                1
                               </sub>
                               of the observed and synthetic data from the generative model with fixed timescales. Alternatively, a can be fitted together with all other parameters of the generative model using aABC. We find that since a is almost independent from other parameters, aABC finds the correct value of a first and then fits the rest of parameters. The MAP estimate of a converges to the same value as estimated by the grid search, but aABC requires more iterations to get posterior distributions for estimated timescales with a similar variance (
                               <a id="xref-fig-25-1" class="xref-fig" href="#F25">
                                Supplementary Fig. 19
                               </a>
                               ). Therefore, gridsearch method is preferred when moderate errors in a are acceptable and approximate range of ground-truth timescales are known, but for more accurate results it is better to fit the a by aABC together with other parameters.
                              </p>
                              <p id="p-75">
                               To estimate the timescales of the two-timescale inhomogeneous Poisson process with exponential fits (
                               <a id="xref-fig-1-13" class="xref-fig" href="#F1">
                                Fig. 1C
                               </a>
                               ), we assumed that the mean and variance of the event-counts are known and only estimated τ
                               <sub>
                                1
                               </sub>
                               , τ
                               <sub>
                                2
                               </sub>
                               and the coefficient c
                               <sub>
                                1
                               </sub>
                               by fitting
                               <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                                Eq. 7
                               </a>
                               to the sample autocorrelation starting from the lag t
                               <sub>
                                1
                               </sub>
                               .
                              </p>
                             </div>
                             <div id="sec-14" class="subsection">
                              <h4>
                               4.1.4 Modelling background processes with an arbitrary PSD shape
                              </h4>
                              <p id="p-76">
                               We can generate random time-series with any desired shape of the PSD. First, we convert the power spectrum PSD(f) to amplitudes
                               <span class="inline-formula" id="inline-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Then, we draw random phases ϕ(f) from a uniform distribution on [0, 2π] and construct a frequency domain signal Z(f) = A(f)e
                               <sup>
                                iϕ(f)
                               </sup>
                               . We transform this signal to the time domain by applying the inverse Fast Fourier Transform (iFFT) and z-score the time-series to obtain zero mean and unit variance.
                              </p>
                              <p id="p-77">
                               For example, a common class of background processes exhibit power-law decay in PSD [
                               <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                               ,
                               <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                               ,
                               <a id="xref-ref-80-4" class="xref-bibr" href="#ref-80">
                                80
                               </a>
                               ,
                               <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">
                                81
                               </a>
                               ]. We model this PSD shape as PSD(f) = f
                               <sup>
                                -χ
                               </sup>
                               , where f
                               <sub>
                                min
                               </sub>
                               < f < f
                               <sub>
                                max
                               </sub>
                               is the frequency range and χ is the power-law exponent. We can set the lower frequency cut-off at f
                               <sub>
                                min
                               </sub>
                               = 1 Hz and the upper frequency cut-off f
                               <sub>
                                max
                               </sub>
                               is defined by the Nyquist frequency (i.e. half of the desired sampling rate). We can combine this process with time-series generated from an OU process with the timescale τ to obtain a PSD shape with both 1/f and Lorentzian components (
                               <a id="xref-fig-23-2" class="xref-fig" href="#F23">
                                Supplementary Fig. 17
                               </a>
                               ). We sum the two time-series with the coefficients c and 1 – c and rescale using the linear transformation
                               <a id="xref-disp-formula-15-2" class="xref-disp-formula" href="#disp-formula-15">
                                Eq. 15
                               </a>
                               to match the mean and variance to the observed data. This generative process can be used when the 1/f exponent in the data PSD deviates from 2 (i.e. Lorentzian shape). This method can be applied to generate background processes with arbitrary desired PSD shapes and estimate the relevant parameters.
                              </p>
                             </div>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              4.2 Optimizing generative model parameters with adaptive Approximate Bayesian Computations
                             </h3>
                             <p id="p-78">
                              We optimize parameters of generative models with adaptive Approximate Bayesian Computations (aABC) following the algorithm from Ref. [
                              <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">
                               58
                              </a>
                              ]. aABC is an iterative algorithm to approximate the multivariate posterior distribution of model parameters. It uses population Monte-Carlo sampling to minimize the distance between the summary statistic of the observed and synthetic data from the generative model. We use sample autocorrelation or PSD as the summary statistic and define the distance d as the mean squared deviation between the observed and synthetic data summary statistics (
                              <a id="xref-disp-formula-9-3" class="xref-disp-formula" href="#disp-formula-9">
                               Eq. 9
                              </a>
                              ,
                              <a id="xref-disp-formula-10-2" class="xref-disp-formula" href="#disp-formula-10">
                               Eq. 10
                              </a>
                              ).
                             </p>
                             <p id="p-79">
                              On the first iteration of the algorithm, the parameters of the generative model
                              <span class="inline-formula" id="inline-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are drawn from the prior distribution. We use a multidimensional uniform prior distribution π(θ) over fitted parameters (e.g., timescales and their weights). The domain of prior distribution for the timescales is chosen to include a broad range below and above the timescales estimated by the direct exponential fits of data autocorrelation (
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). For the weights of timescales c
                              <sub>
                               k
                              </sub>
                              , we use uniform prior distributions on [0, 1]. The model with parameters
                              <span class="inline-formula" id="inline-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is used to generate synthetic time-series A(t′) with the same duration and number of trials as in the observed data. Next, we compute the distance d between the summary statistics of the synthetic and observed data. If d is smaller than the error threshold ε (initially set to 1), the parameters are accepted and added to the multivariate posterior distribution. Each iteration of the algorithm is repeated until 500 parameters samples are accepted.
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2395263/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2395263/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2395263" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <p id="p-80" class="first-child">
                                Simulation parameters for data autocorrelations.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2395253/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2395253/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2395253" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2.
                               </span>
                               <p id="p-81" class="first-child">
                                aABC fit parameters. U[a, b] denotes a uniform distribution on the interval [a, b].
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-82">
                              On subsequent iterations, the same steps are repeated but with parameters drawn from a proposal distribution and with an updated error threshold. On each iteration, the error threshold is set at the first quartile of the accepted sample distances from the previous iteration. The proposal distribution is computed for each iteration ξ as a mixture of Gaussian distributions based on the prior distribution and the accepted samples θ
                              <sub>
                               r
                              </sub>
                              , r = 1,…, N from the previous iteration:
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-83">
                              Here
                              <span class="inline-formula" id="inline-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the importance weight of the accepted sample
                              <span class="inline-formula" id="inline-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              from the previous iteration
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-84">
                              K
                              <sub>
                               ξ
                              </sub>
                              is the random walk kernel for the population Monte Carlo algorithm, which is a multivariate Gaussian with the mean
                              <span class="inline-formula" id="inline-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/inline-graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and the covariance equal to twice the covariance of all accepted samples from the previous iteration Σ = 2Cov[θ
                              <sup>
                               (ξ–1)
                              </sup>
                              ]:
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/08/2020.08.11.245944/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-85">
                              Here κ is the number of fitted parameters, and |Σ| is the determinant of Σ.
                             </p>
                             <p id="p-86">
                              The convergence of the algorithm is defined based on the acceptance rate accR, which is the number of accepted samples divided by the total number of drawn samples on each iteration. The algorithm terminates when the acceptance rate reaches accRmin, which is set to accR
                              <sub>
                               min
                              </sub>
                              = 0.003 in our simulations. Smaller accR
                              <sub>
                               min
                              </sub>
                              leads to a smaller final error threshold (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              , right) and a better approximation of the posterior distributions, but requires longer fitting time. To find the MAP estimates, we smooth the final joint posterior distribution with a multivariate Gaussian kernel and find its maximum with a grid search.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              4.3 abcTau: A Python package for timescales estimation and model comparison with aABC
                             </h3>
                             <p id="p-87">
                              We developed the abcTau Python package implementing our aABC framework for estimation of timescales from autocorrelations or PSDs of various types of experimental data, and the Bayesian model comparison between different hypotheses. We also provided tutorials as Jupyter Notebooks and example Python scripts to make our framework easily accessible for researchers in different fields.
                             </p>
                             <p id="p-88">
                              The minimal requirements for using this package are Python 3.7.1, Numpy 1.15.4 and Scipy 1.1.0. For visualization, Matplotlib >= 3.0.2 and Seaborn >= 0.9.0 are required. The basis of aABC algorithm in the package is adopted from a previous implementation originally developed in Python 2 [
                              <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">
                               82
                              </a>
                              ]. Since all parameters of our generative models are positive and sometimes subject to additional conditions (e.g., τ
                              <sub>
                               2
                              </sub>
                              > τ
                              <sub>
                               1
                              </sub>
                              ), we introduced constraints on sampling from proposal distributions. Moreover, we enhanced the algorithm for parallel processing required for analyzing large datasets.
                             </p>
                             <p id="p-89">
                              The abcTau package includes various types of generative models that are relevant for different types data and various methods for computing the autocorrelation or PSD. Using this functionality, users can apply our framework to their time-series data, supplied in a Numpy array structured as trials times time-points. The object oriented implementation of the package allows users to easily replace any function, including generative models, summary statistic computations, distance functions, etc., with their customized functions to better describe statistics of the data. Users can also add their customized generative models directly to the package to create a larger database of generative models available for different applications.
                             </p>
                             <p id="p-90">
                              The package also includes a module for Bayesian model selection. This module computes the cumulative distribution of distances from estimated posterior distributions (i.e., Bayes factor for different error thresholds), runs the statistical tests and suggests the best hypothesis describing the underlying processes in data.
                             </p>
                             <div id="sec-17" class="subsection">
                              <h4>
                               4.3.1 Pre-processing for evaluation of direct fit quality
                              </h4>
                              <p id="p-91">
                               Since Bayesian inference of a full posterior distribution can be computationally expensive, we implemented a fast pre-processing function that uses parametric bootstrapping to determine whether the direct exponential fit provides satisfactory estimates of timescales (
                               <a id="xref-fig-20-2" class="xref-fig" href="#F20">
                                Supplementary Fig. 14
                               </a>
                               ). In this function, a generative model (e.g., based on a mixture of OU processes) with parameters obtained from the direct exponential fit can be used to generate multiple synthetic datasets, each with the same amount of data as in the original data. For each synthetic dataset, timescales are estimated by direct exponential fitting. The obtained distribution of timescales from this bootstrapping procedure can be compared to the initial direct-fit estimate from the original data. The error between the mean of bootstrapping distribution and the initial direct fit is used to approximately evaluate the direct fit quality. If the error is small enough, the direct exponential fit may be sufficiently accurate. The accuracy of timescale estimates with the direct fit can be further improved by empirical bias-correction using the measured deviation between the mean of the parametric bootstrap distribution and the direct-fit estimates (
                               <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                                Supplementary Fig. 5
                               </a>
                               ). However, this method does not guarantee an accurate bias correction since the deviation of the direct fit from the ground truth can be larger than the observed deviation between the bootstrap and direct fit. Hence, we recommend users to be conservative with the decision to rely on the direct-fit estimates if accurate estimates of timescales are desired.
                              </p>
                             </div>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              4.4 Description of neural recordings
                             </h3>
                             <p id="p-92">
                              Experimental procedures and data pre-processing were described previously [
                              <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">
                               77
                              </a>
                              ]. In brief, a monkey was trained to fixate a central dot on a blank screen for 3 s on each trial. Spiking activity was recorded with a 16-channel micro-electrode array inserted perpendicularly to the cortical surface to record from all layers in the visual area V4. For fitting, we used a recording session with 81 trials. We pooled the activity across all channels and calculated the population spike-counts in bins of 1 ms. First, we subtracted the trail-averaged activity (PSTH) from spike-counts to remove the slow trends locked to the trial onset [
                              <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ]. Then, we computed the autocorrelation of spike-counts using
                              <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">
                               Eq. 2
                              </a>
                              for each trial and averaged over trials’ autocorrelations.
                             </p>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              4.5 Parameters of simulations and aABC fits in figures
                             </h3>
                             <p id="p-93">
                              For all fits, the initial error threshold was set to ε = 1. The aABC iterations continued until accR ⩽ 0.003 was reached. All datasets (except for the branching network) consisted of 500 trials each of 1 s duration. The dataset for the branching network (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              ) consisted of 100 trials with 500 time-steps. The parameters for simulations and aABC fits are given in
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              and
                              <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              , respectively.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-20">
                            <h2 class="">
                             Data and Code availability
                            </h2>
                            <p id="p-94">
                             The abcTau Python package with example data and tutorials are available on GitHub at:
                             <a href="https://github.com/roxana-zeraati/abcTau">
                              https://github.com/roxana-zeraati/abcTau
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-96">
                             RZ, TAE and AL designed the research, discussed the results and wrote the paper. RZ wrote the computer code, performed simulations, and analyzed the data.
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             5 Supplementary tables
                            </h2>
                            <div id="T3" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2395259/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2395259/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2395259" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 1.
                              </span>
                              <p id="p-97" class="first-child">
                               Running time of the aABC algorithm using the PSD as the summary statistic and 20 parallel processors (Intel(R) Xeon(R) Gold 6152 CPU @ 2.10GHz). Trial duration is 1 sec.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             6 Supplementary figures
                            </h2>
                            <div id="F7" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F7.large.jpg?width=800&height=600&carousel=1" title="Different methods of estimating the sample autocorrelation from finite data produce biased results. Computing autocorrelations in the time domain (Eq. 2, blue) or via the inverse Fourier transform of the PSD (orange) both produce biased results. Data are generated from an OU process with the timescale τ = 100 ms and duration T = 1 s. Autocorrelations are averaged over 500 trials (realizations). PSDs are computed after applying a Hamming window to each trial." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Different methods of estimating the sample autocorrelation from finite data produce biased results.</span> Computing autocorrelations in the time domain (Eq. 2, blue) or via the inverse Fourier transform of the PSD (orange) both produce biased results. Data are generated from an OU process with the timescale <em>τ</em> = 100 ms and duration <em>T</em> = 1 s. Autocorrelations are averaged over 500 trials (realizations). PSDs are computed after applying a Hamming window to each trial.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F7.medium.gif" width="440" height="270"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F7.medium.gif" width="440" height="270"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 1.
                              </span>
                              <span class="caption-title">
                               Different methods of estimating the sample autocorrelation from finite data produce biased results.
                              </span>
                              <p id="p-98" class="first-child">
                               Computing autocorrelations in the time domain (
                               <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">
                                Eq. 2
                               </a>
                               , blue) or via the inverse Fourier transform of the PSD (orange) both produce biased results. Data are generated from an OU process with the timescale τ = 100 ms and duration T = 1 s. Autocorrelations are averaged over 500 trials (realizations). PSDs are computed after applying a Hamming window to each trial.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F8" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F8.large.jpg?width=800&height=600&carousel=1" title="Autocorrelation of synthetic data (cyan) generated with parameters estimated by direct fit deviates from the autocorrelation of the original fitted data (brown). The same synthetic data as in Fig. 1, middle, from A-C respectively. The synthetic data are generated from the correct functional form of generative models with the same statistics (e.g., duration and number of trials) as in the observed data, using parameters obtained from direct exponential fits to observed data autocorrelations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Autocorrelation of synthetic data (cyan) generated with parameters estimated by direct fit deviates from the autocorrelation of the original fitted data (brown).</span> The same synthetic data as in Fig. 1, middle, from A-C respectively. The synthetic data are generated from the correct functional form of generative models with the same statistics (e.g., duration and number of trials) as in the observed data, using parameters obtained from direct exponential fits to observed data autocorrelations.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F8.medium.gif" width="440" height="116"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F8.medium.gif" width="440" height="116"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 2.
                              </span>
                              <span class="caption-title">
                               Autocorrelation of synthetic data (cyan) generated with parameters estimated by direct fit deviates from the autocorrelation of the original fitted data (brown).
                              </span>
                              <p id="p-99" class="first-child">
                               The same synthetic data as in
                               <a id="xref-fig-1-14" class="xref-fig" href="#F1">
                                Fig. 1
                               </a>
                               , middle, from A-C respectively. The synthetic data are generated from the correct functional form of generative models with the same statistics (e.g., duration and number of trials) as in the observed data, using parameters obtained from direct exponential fits to observed data autocorrelations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F9" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F9.large.jpg?width=800&height=600&carousel=1" title="Directly fitting the autocorrelation shape in logarithmic-linear space leads to larger estimation errors than in linear space. The estimated timescale from fitting the sample autocorrelation shape with an exponential decay function is more biased when using logarithmic distances (blue line) than using linear distances (orange line), since it puts more emphasis on the autocorrelation tail which is more affected by the bias (larger deviations between the autocorrelations of data and the ground truth). For a fair comparison between logarithmic and linear fits, only positive correlation values are included in the fitting. Data consists of 500 trials that are generated from an OU process with the timescale τ = 80 ms and duration T = 1 s." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Directly fitting the autocorrelation shape in logarithmic-linear space leads to larger estimation errors than in linear space.</span> The estimated timescale from fitting the sample autocorrelation shape with an exponential decay function is more biased when using logarithmic distances (blue line) than using linear distances (orange line), since it puts more emphasis on the autocorrelation tail which is more affected by the bias (larger deviations between the autocorrelations of data and the ground truth). For a fair comparison between logarithmic and linear fits, only positive correlation values are included in the fitting. Data consists of 500 trials that are generated from an OU process with the timescale <em>τ</em> = 80 ms and duration <em>T</em> = 1 s.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F9.medium.gif" width="440" height="350"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F9.medium.gif" width="440" height="350"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 3.
                              </span>
                              <span class="caption-title">
                               Directly fitting the autocorrelation shape in logarithmic-linear space leads to larger estimation errors than in linear space.
                              </span>
                              <p id="p-100" class="first-child">
                               The estimated timescale from fitting the sample autocorrelation shape with an exponential decay function is more biased when using logarithmic distances (blue line) than using linear distances (orange line), since it puts more emphasis on the autocorrelation tail which is more affected by the bias (larger deviations between the autocorrelations of data and the ground truth). For a fair comparison between logarithmic and linear fits, only positive correlation values are included in the fitting. Data consists of 500 trials that are generated from an OU process with the timescale τ = 80 ms and duration T = 1 s.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F10.large.jpg?width=800&height=600&carousel=1" title="Direct exponential fit produces different estimated timescale for datasets from the same generative process but with different trial duration. Each dataset is generated from an OU process with τ = 80 ms and 500 trials each with duration T. Dots - sample autocorrelation, dashed lines - direct exponential fit, solid line - ground truth." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Direct exponential fit produces different estimated timescale for datasets from the same generative process but with different trial duration.</span> Each dataset is generated from an OU process with <em>τ</em> = 80 ms and 500 trials each with duration T. Dots - sample autocorrelation, dashed lines - direct exponential fit, solid line - ground truth.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F10.medium.gif" width="440" height="290"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F10.medium.gif" width="440" height="290"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 4.
                              </span>
                              <span class="caption-title">
                               Direct exponential fit produces different estimated timescale for datasets from the same generative process but with different trial duration.
                              </span>
                              <p id="p-101" class="first-child">
                               Each dataset is generated from an OU process with τ = 80 ms and 500 trials each with duration T. Dots - sample autocorrelation, dashed lines - direct exponential fit, solid line - ground truth.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F11.large.jpg?width=800&height=600&carousel=1" title="Bootstrapping for the uncertainty estimation and bias correction. (A-B) Using bootstrapping for the bias correction does not guarantee unbiased estimation of timescales, since the amount of bias depends on the ground-truth timescale. Data are generated from an OU process with a given ground-truth timescale (dashed line, 500 trials with T = 1 s duration). Directly fitting the average autocorrelation shape with an exponential decay function underestimates the timescale (blue line). For parametric bootstrapping, we generate 100 synthetic datasets from an OU process with the timescale estimated by the direct fit and other statistics (e.g., duration and number of trials) as in the original data. We estimate the timescale from each synthetic dataset again with a direct exponential fit (magenta filled bars distribution). The difference between the mean of bootstrapping distribution and the original direct fit estimate can be used as an offset to correct the bias (magenta empty bars distribution). The variance of bootstrapping distribution provides a measure of uncertainty. However, this method only works for some cases (A) and does not guarantee an unbiased estimate for every case (B). For non-parametric bootstrapping, we resample trials from the original data (500 trails with replacement) and estimate the timescale from the average autocorrelation of resampled data with a direct exponential fit (blue distribution). (C) Average values of timescales estimated by direct fit with non-parametric bootstrap (blue) and parametric bootstrap with bias correction (magenta) deviate from the ground-truth timescale (dashed lines) and the uncertainty intervals (5 and 95 percentiles, error bars) do not contain these ground-truth values for every example. In contrast, the aABC method provides a reliable mean estimate with uncertainty intervals including the ground-truth timescale (orange). Data are generated from an OU process with the given groundtruth timescales and duration T = 1 s (500 trials). Bootstrapping and aABC distributions contain 100 samples. For bootstrapping, the error bars are generally smaller than symbol for the average value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Bootstrapping for the uncertainty estimation and bias correction.</span> <strong>(A-B)</strong> Using bootstrapping for the bias correction does not guarantee unbiased estimation of timescales, since the amount of bias depends on the ground-truth timescale. Data are generated from an OU process with a given ground-truth timescale (dashed line, 500 trials with <em>T</em> = 1 s duration). Directly fitting the average autocorrelation shape with an exponential decay function underestimates the timescale (blue line). For parametric bootstrapping, we generate 100 synthetic datasets from an OU process with the timescale estimated by the direct fit and other statistics (e.g., duration and number of trials) as in the original data. We estimate the timescale from each synthetic dataset again with a direct exponential fit (magenta filled bars distribution). The difference between the mean of bootstrapping distribution and the original direct fit estimate can be used as an offset to correct the bias (magenta empty bars distribution). The variance of bootstrapping distribution provides a measure of uncertainty. However, this method only works for some cases (A) and does not guarantee an unbiased estimate for every case (B). For non-parametric bootstrapping, we resample trials from the original data (500 trails with replacement) and estimate the timescale from the average autocorrelation of resampled data with a direct exponential fit (blue distribution). <strong>(C)</strong> Average values of timescales estimated by direct fit with non-parametric bootstrap (blue) and parametric bootstrap with bias correction (magenta) deviate from the ground-truth timescale (dashed lines) and the uncertainty intervals (5 and 95 percentiles, error bars) do not contain these ground-truth values for every example. In contrast, the aABC method provides a reliable mean estimate with uncertainty intervals including the ground-truth timescale (orange). Data are generated from an OU process with the given groundtruth timescales and duration <em>T</em> = 1 s (500 trials). Bootstrapping and aABC distributions contain 100 samples. For bootstrapping, the error bars are generally smaller than symbol for the average value.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F11.medium.gif" width="440" height="126"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F11.medium.gif" width="440" height="126"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 5.
                              </span>
                              <span class="caption-title">
                               Bootstrapping for the uncertainty estimation and bias correction.
                              </span>
                              <p id="p-102" class="first-child">
                               <strong>
                                (A-B)
                               </strong>
                               Using bootstrapping for the bias correction does not guarantee unbiased estimation of timescales, since the amount of bias depends on the ground-truth timescale. Data are generated from an OU process with a given ground-truth timescale (dashed line, 500 trials with T = 1 s duration). Directly fitting the average autocorrelation shape with an exponential decay function underestimates the timescale (blue line). For parametric bootstrapping, we generate 100 synthetic datasets from an OU process with the timescale estimated by the direct fit and other statistics (e.g., duration and number of trials) as in the original data. We estimate the timescale from each synthetic dataset again with a direct exponential fit (magenta filled bars distribution). The difference between the mean of bootstrapping distribution and the original direct fit estimate can be used as an offset to correct the bias (magenta empty bars distribution). The variance of bootstrapping distribution provides a measure of uncertainty. However, this method only works for some cases (A) and does not guarantee an unbiased estimate for every case (B). For non-parametric bootstrapping, we resample trials from the original data (500 trails with replacement) and estimate the timescale from the average autocorrelation of resampled data with a direct exponential fit (blue distribution).
                               <strong>
                                (C)
                               </strong>
                               Average values of timescales estimated by direct fit with non-parametric bootstrap (blue) and parametric bootstrap with bias correction (magenta) deviate from the ground-truth timescale (dashed lines) and the uncertainty intervals (5 and 95 percentiles, error bars) do not contain these ground-truth values for every example. In contrast, the aABC method provides a reliable mean estimate with uncertainty intervals including the ground-truth timescale (orange). Data are generated from an OU process with the given groundtruth timescales and duration T = 1 s (500 trials). Bootstrapping and aABC distributions contain 100 samples. For bootstrapping, the error bars are generally smaller than symbol for the average value.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F12.large.jpg?width=800&height=600&carousel=1" title="Direct fitting of sample PSD results in biased estimates of timescales, whereas aABC produces accurate estimates. (A) The shape of sample PSD (brown) deviates from the ground-truth PSD (gray). Knee frequencies estimated by directly fitting the sample PSD with a Lorentzian function (pink) or using the FOOOF toolbox [42] (magenta) are similar and overestimate the ground-truth knee frequency (i.e. underestimate the ground-truth timescale). FOOOF estimates the timescale by fitting the shape of log(PSD(f)) with the function b – log(k + fχ), where f is the frequency and b, k and χ are free parameters. For an exponential decay function χ = 2, but FOOOF allows for χ ≠ 2. The sample PSD is fitted between 1 Hz and 100 Hz frequencies. (B) Timescales estimated in the frequency domain strongly depend on the fitted frequency range and can overestimate, underestimate or produce accurate results depending on the selected frequency range. Distribution of point estimates for the timescales estimated with the FOOOF fit of PSD for 100 independent datasets of the OU process. For each dataset, we averaged the PSD over 500 trials with 1 s duration and the sampling frequency of 1000 Hz. The PSDs are computed using a Hamming window applied to each trial. FOOOF is fitted on the average PSD for each dataset over the frequency ranges of [1, 100] Hz, [1, 200] Hz and [1, 400] Hz from left to right. (C) The aABC method estimates the correct timescales (orange posterior distributions) independent of the selected frequency range. Data are the same as in A and the frequency ranges for fitting are the same as B. The direct-fit estimates (magenta vertical lines) are computed using FOOOF." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Direct fitting of sample PSD results in biased estimates of timescales, whereas aABC produces accurate estimates.</span> <strong>(A)</strong> The shape of sample PSD (brown) deviates from the ground-truth PSD (gray). Knee frequencies estimated by directly fitting the sample PSD with a Lorentzian function (pink) or using the FOOOF toolbox [42] (magenta) are similar and overestimate the ground-truth knee frequency (i.e. underestimate the ground-truth timescale). FOOOF estimates the timescale by fitting the shape of log(PSD(f)) with the function <em>b</em> – log(<em>k</em> + <em>f<sup>χ</sup></em>), where <em>f</em> is the frequency and <em>b, k</em> and <em>χ</em> are free parameters. For an exponential decay function <em>χ</em> = 2, but FOOOF allows for <em>χ</em> ≠ 2. The sample PSD is fitted between 1 Hz and 100 Hz frequencies. <strong>(B)</strong> Timescales estimated in the frequency domain strongly depend on the fitted frequency range and can overestimate, underestimate or produce accurate results depending on the selected frequency range. Distribution of point estimates for the timescales estimated with the FOOOF fit of PSD for 100 independent datasets of the OU process. For each dataset, we averaged the PSD over 500 trials with 1 s duration and the sampling frequency of 1000 Hz. The PSDs are computed using a Hamming window applied to each trial. FOOOF is fitted on the average PSD for each dataset over the frequency ranges of [1, 100] Hz, [1, 200] Hz and [1, 400] Hz from left to right. <strong>(C)</strong> The aABC method estimates the correct timescales (orange posterior distributions) independent of the selected frequency range. Data are the same as in A and the frequency ranges for fitting are the same as B. The direct-fit estimates (magenta vertical lines) are computed using FOOOF.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F12.medium.gif" width="440" height="396"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F12.medium.gif" width="440" height="396"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 6." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 6.
                              </span>
                              <span class="caption-title">
                               Direct fitting of sample PSD results in biased estimates of timescales, whereas aABC produces accurate estimates.
                              </span>
                              <p id="p-103" class="first-child">
                               <strong>
                                (A)
                               </strong>
                               The shape of sample PSD (brown) deviates from the ground-truth PSD (gray). Knee frequencies estimated by directly fitting the sample PSD with a Lorentzian function (pink) or using the FOOOF toolbox [42] (magenta) are similar and overestimate the ground-truth knee frequency (i.e. underestimate the ground-truth timescale). FOOOF estimates the timescale by fitting the shape of log(PSD(f)) with the function b – log(k + f
                               <sup>
                                χ
                               </sup>
                               ), where f is the frequency and b, k and χ are free parameters. For an exponential decay function χ = 2, but FOOOF allows for χ ≠ 2. The sample PSD is fitted between 1 Hz and 100 Hz frequencies.
                               <strong>
                                (B)
                               </strong>
                               Timescales estimated in the frequency domain strongly depend on the fitted frequency range and can overestimate, underestimate or produce accurate results depending on the selected frequency range. Distribution of point estimates for the timescales estimated with the FOOOF fit of PSD for 100 independent datasets of the OU process. For each dataset, we averaged the PSD over 500 trials with 1 s duration and the sampling frequency of 1000 Hz. The PSDs are computed using a Hamming window applied to each trial. FOOOF is fitted on the average PSD for each dataset over the frequency ranges of [1, 100] Hz, [1, 200] Hz and [1, 400] Hz from left to right.
                               <strong>
                                (C)
                               </strong>
                               The aABC method estimates the correct timescales (orange posterior distributions) independent of the selected frequency range. Data are the same as in A and the frequency ranges for fitting are the same as B. The direct-fit estimates (magenta vertical lines) are computed using FOOOF.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F13" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F13.large.jpg?width=800&height=600&carousel=1" title="The shape of sample autocorrelation is slightly different across realizations of the same generative process due to noise in the data. Each trace is the average autocorrelation of 500 trials with T = 1 s generated from an OU process with τ = 20 ms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The shape of sample autocorrelation is slightly different across realizations of the same generative process due to noise in the data.</span> Each trace is the average autocorrelation of 500 trials with <em>T</em> = 1 s generated from an OU process with <em>τ</em> = 20 ms.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F13.medium.gif" width="440" height="291"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F13.medium.gif" width="440" height="291"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 7." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 7.
                              </span>
                              <span class="caption-title">
                               The shape of sample autocorrelation is slightly different across realizations of the same generative process due to noise in the data.
                              </span>
                              <p id="p-104" class="first-child">
                               Each trace is the average autocorrelation of 500 trials with T = 1 s generated from an OU process with τ = 20 ms.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F14" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F14.large.jpg?width=800&height=600&carousel=1" title="Variance of posterior distributions estimated with aABC captures the estimation uncertainty and depends on the signal to noise ratio. Increasing the number of trials in observed data reduces the noise in sample autocorrelation (left) and the estimation uncertainty (narrower posterior distribution) (right). Estimated τ: mean ± std. The data was generated from an OU process with τ = 20 ms and consisted of trials with T = 1 s." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Variance of posterior distributions estimated with aABC captures the estimation uncertainty and depends on the signal to noise ratio.</span> Increasing the number of trials in observed data reduces the noise in sample autocorrelation <strong>(left)</strong> and the estimation uncertainty (narrower posterior distribution) <strong>(right)</strong>. Estimated <em>τ</em>: mean ± std. The data was generated from an OU process with <em>τ</em> = 20 ms and consisted of trials with <em>T</em> = 1 s.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F14.medium.gif" width="440" height="143"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 8." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F14.medium.gif" width="440" height="143"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 8." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 8.
                              </span>
                              <span class="caption-title">
                               Variance of posterior distributions estimated with aABC captures the estimation uncertainty and depends on the signal to noise ratio.
                              </span>
                              <p id="p-105" class="first-child">
                               Increasing the number of trials in observed data reduces the noise in sample autocorrelation
                               <strong>
                                (left)
                               </strong>
                               and the estimation uncertainty (narrower posterior distribution)
                               <strong>
                                (right)
                               </strong>
                               . Estimated τ: mean ± std. The data was generated from an OU process with τ = 20 ms and consisted of trials with T = 1 s.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F15" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F15.large.jpg?width=800&height=600&carousel=1" title="Using PSD in the aABC algorithm accurately recovers the groundtruth parameters. (A) Data are generated from an OU process with the timescale τ = 100 ms. Left: The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the generative model with the MAP estimate parameters (orange). Right: The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters. (B) Same as A for data from a linear mixture of an OU process with the timescale τ = 100 ms and an oscillation with frequency f = 10 Hz. (C) Same as B with τ = 100 ms and f = 2 Hz. The slow oscillation does not manifest in any clear peak in PSD due to short duration of trials (T = 1 s) and the low frequency resolution. However, aABC algorithm can still uncover the oscillatory component with the correct parameters. (D) Same as A for data from an inhomogeneous Poisson process with two timescales τ1 = 5 ms and τ2 = 80 ms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Using PSD in the aABC algorithm accurately recovers the groundtruth parameters.</span> <strong>(A)</strong> Data are generated from an OU process with the timescale <em>τ</em> = 100 ms. <strong>Left:</strong> The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the generative model with the MAP estimate parameters (orange). <strong>Right:</strong> The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters. <strong>(B)</strong> Same as A for data from a linear mixture of an OU process with the timescale <em>τ</em> = 100 ms and an oscillation with frequency <em>f</em> = 10 Hz. <strong>(C)</strong> Same as B with <em>τ</em> = 100 ms and <em>f</em> = 2 Hz. The slow oscillation does not manifest in any clear peak in PSD due to short duration of trials (T = 1 s) and the low frequency resolution. However, aABC algorithm can still uncover the oscillatory component with the correct parameters. <strong>(D)</strong> Same as A for data from an inhomogeneous Poisson process with two timescales <em>τ</em><sub>1</sub> = 5 ms and <em>τ</em><sub>2</sub> = 80 ms.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F15.medium.gif" width="440" height="422"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 9." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F15.medium.gif" width="440" height="422"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 9." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 9.
                              </span>
                              <span class="caption-title">
                               Using PSD in the aABC algorithm accurately recovers the groundtruth parameters.
                              </span>
                              <p id="p-106" class="first-child">
                               <strong>
                                (A)
                               </strong>
                               Data are generated from an OU process with the timescale τ = 100 ms.
                               <strong>
                                Left:
                               </strong>
                               The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the generative model with the MAP estimate parameters (orange).
                               <strong>
                                Right:
                               </strong>
                               The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters.
                               <strong>
                                (B)
                               </strong>
                               Same as A for data from a linear mixture of an OU process with the timescale τ = 100 ms and an oscillation with frequency f = 10 Hz.
                               <strong>
                                (C)
                               </strong>
                               Same as B with τ = 100 ms and f = 2 Hz. The slow oscillation does not manifest in any clear peak in PSD due to short duration of trials (T = 1 s) and the low frequency resolution. However, aABC algorithm can still uncover the oscillatory component with the correct parameters.
                               <strong>
                                (D)
                               </strong>
                               Same as A for data from an inhomogeneous Poisson process with two timescales τ
                               <sub>
                                1
                               </sub>
                               = 5 ms and τ
                               <sub>
                                2
                               </sub>
                               = 80 ms.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F16" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F16.large.jpg?width=800&height=600&carousel=1" title="The aABC algorithm can accurately recover the ground-truth parameters even in presence of multiple oscillatory components. (A-B) Data are generated from a linear mixture of an OU process with the timescale τ = 100 ms and two oscillations with frequencies f1 = 10 Hz and f2 = 20 Hz with coefficients c1 = c2 = 0.3. (A) The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the generative model with the MAP estimate parameters (orange). (B) The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters (vertical dashed lines). (C-D) Same as A-B with τ = 50 ms, f1 = 10 Hz, f2 = 50 Hz, c1 = 0.3 and c2 = 0.1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The aABC algorithm can accurately recover the ground-truth parameters even in presence of multiple oscillatory components.</span> <strong>(A-B)</strong> Data are generated from a linear mixture of an OU process with the timescale <em>τ</em> = 100 ms and two oscillations with frequencies <em>f</em><sub>1</sub> = 10 Hz and <em>f</em><sub>2</sub> = 20 Hz with coefficients <em>c</em><sub>1</sub> = <em>c</em><sub>2</sub> = 0.3. <strong>(A)</strong> The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the generative model with the MAP estimate parameters (orange). <strong>(B)</strong> The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters (vertical dashed lines). <strong>(C-D)</strong> Same as A-B with <em>τ</em> = 50 ms, <em>f</em><sub>1</sub> = 10 Hz, <em>f</em><sub>2</sub> = 50 Hz, <em>c</em><sub>1</sub> = 0.3 and <em>c</em><sub>2</sub> = 0.1.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F16.medium.gif" width="440" height="157"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 10." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F16.medium.gif" width="440" height="157"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 10." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 10.
                              </span>
                              <span class="caption-title">
                               The aABC algorithm can accurately recover the ground-truth parameters even in presence of multiple oscillatory components.
                              </span>
                              <p id="p-107" class="first-child">
                               <strong>
                                (A-B)
                               </strong>
                               Data are generated from a linear mixture of an OU process with the timescale τ = 100 ms and two oscillations with frequencies f
                               <sub>
                                1
                               </sub>
                               = 10 Hz and f
                               <sub>
                                2
                               </sub>
                               = 20 Hz with coefficients c
                               <sub>
                                1
                               </sub>
                               = c
                               <sub>
                                2
                               </sub>
                               = 0.3.
                               <strong>
                                (A)
                               </strong>
                               The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the generative model with the MAP estimate parameters (orange).
                               <strong>
                                (B)
                               </strong>
                               The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters (vertical dashed lines).
                               <strong>
                                (C-D)
                               </strong>
                               Same as A-B with τ = 50 ms, f
                               <sub>
                                1
                               </sub>
                               = 10 Hz, f
                               <sub>
                                2
                               </sub>
                               = 50 Hz, c
                               <sub>
                                1
                               </sub>
                               = 0.3 and c
                               <sub>
                                2
                               </sub>
                               = 0.1.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F17" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F17.large.jpg?width=800&height=600&carousel=1" title="Posterior distributions approximated by the aABC method recover the ground-truth timescales independent of the selected summary statistic and fitting range. Data are generated from an OU process with the timescale τ = 100 ms. (A) The aABC algorithm fitted the data autocorrelation up to the time-lag tm (indicated in the figure title) using distances computed in linear (lin) or logarithmic (log) scale. (B) The aABC algorithm fitted the data PSD from f = 1 Hz up to the frequency fm (indicated in the figure title) using distances computed in linear (lin) or logarithmic (log) scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Posterior distributions approximated by the aABC method recover the ground-truth timescales independent of the selected summary statistic and fitting range.</span> Data are generated from an OU process with the timescale <em>τ</em> = 100 ms. <strong>(A)</strong> The aABC algorithm fitted the data autocorrelation up to the time-lag <em>t<sub>m</sub></em> (indicated in the figure title) using distances computed in linear (lin) or logarithmic (log) scale. <strong>(B)</strong> The aABC algorithm fitted the data PSD from <em>f</em> = 1 Hz up to the frequency <em>f<sub>m</sub></em> (indicated in the figure title) using distances computed in linear (lin) or logarithmic (log) scale.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F17.medium.gif" width="440" height="175"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 11." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F17.medium.gif" width="440" height="175"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 11." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 11.
                              </span>
                              <span class="caption-title">
                               Posterior distributions approximated by the aABC method recover the ground-truth timescales independent of the selected summary statistic and fitting range.
                              </span>
                              <p id="p-108" class="first-child">
                               Data are generated from an OU process with the timescale τ = 100 ms.
                               <strong>
                                (A)
                               </strong>
                               The aABC algorithm fitted the data autocorrelation up to the time-lag t
                               <sub>
                                m
                               </sub>
                               (indicated in the figure title) using distances computed in linear (lin) or logarithmic (log) scale.
                               <strong>
                                (B)
                               </strong>
                               The aABC algorithm fitted the data PSD from f = 1 Hz up to the frequency f
                               <sub>
                                m
                               </sub>
                               (indicated in the figure title) using distances computed in linear (lin) or logarithmic (log) scale.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F18" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F18.large.jpg?width=800&height=600&carousel=1" title="Wide prior distributions for generative model parameters do not affect the shape of posterior distributions and only slow down the fitting procedure. (A) Wide and narrow prior distributions (left) give rise to similar posterior distributions (middle), but aABC fit with a narrower prior converges in fewer iterations (right). Data are generated from an OU process with the timescale τ = 20 ms and fitted with aABC algorithm and PSD as a summary statistic using two different prior distributions: narrow prior τ: U[0, 60] and wide prior τ: U[0, 500] ms, where U[·, ·] denotes a uniform distribution. The vertical dashed line indicates the ground-truth timescale. Error thresholds (ε) are plotted starting from the second iteration. Initial error threshold for all fits was set to 1. (B) Same as A for data generated from an inhomogeneous Poisson process with two timescales τ1 = 5 ms and τ2 = 80 ms and prior distributions: narrow prior τ1: U[0, 60] ms, τ2: U[20, 140] ms; wide prior τ1: U[0, 200] ms, τ2: U[20, 500] ms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Wide prior distributions for generative model parameters do not affect the shape of posterior distributions and only slow down the fitting procedure.</span> <strong>(A)</strong> Wide and narrow prior distributions (left) give rise to similar posterior distributions (middle), but aABC fit with a narrower prior converges in fewer iterations (right). Data are generated from an OU process with the timescale <em>τ</em> = 20 ms and fitted with aABC algorithm and PSD as a summary statistic using two different prior distributions: narrow prior <em>τ</em>: <em>U</em>[0, 60] and wide prior <em>τ</em>: <em>U</em>[0, 500] ms, where <em>U</em>[·, ·] denotes a uniform distribution. The vertical dashed line indicates the ground-truth timescale. Error thresholds (<em>ε</em>) are plotted starting from the second iteration. Initial error threshold for all fits was set to 1. <strong>(B)</strong> Same as A for data generated from an inhomogeneous Poisson process with two timescales <em>τ</em><sub>1</sub> = 5 ms and <em>τ</em><sub>2</sub> = 80 ms and prior distributions: narrow prior <em>τ</em><sub>1</sub>: <em>U</em>[0, 60] ms, <em>τ</em><sub>2</sub>: <em>U</em>[20, 140] ms; wide prior <em>τ</em><sub>1</sub>: <em>U</em>[0, 200] ms, <em>τ</em><sub>2</sub>: <em>U</em>[20, 500] ms.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F18.medium.gif" width="440" height="215"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 12." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F18.medium.gif" width="440" height="215"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 12." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 12.
                              </span>
                              <span class="caption-title">
                               Wide prior distributions for generative model parameters do not affect the shape of posterior distributions and only slow down the fitting procedure.
                              </span>
                              <p id="p-109" class="first-child">
                               <strong>
                                (A)
                               </strong>
                               Wide and narrow prior distributions (left) give rise to similar posterior distributions (middle), but aABC fit with a narrower prior converges in fewer iterations (right). Data are generated from an OU process with the timescale τ = 20 ms and fitted with aABC algorithm and PSD as a summary statistic using two different prior distributions: narrow prior τ: U[0, 60] and wide prior τ: U[0, 500] ms, where U[·, ·] denotes a uniform distribution. The vertical dashed line indicates the ground-truth timescale. Error thresholds (ε) are plotted starting from the second iteration. Initial error threshold for all fits was set to 1.
                               <strong>
                                (B)
                               </strong>
                               Same as A for data generated from an inhomogeneous Poisson process with two timescales τ
                               <sub>
                                1
                               </sub>
                               = 5 ms and τ
                               <sub>
                                2
                               </sub>
                               = 80 ms and prior distributions: narrow prior τ
                               <sub>
                                1
                               </sub>
                               : U[0, 60] ms, τ
                               <sub>
                                2
                               </sub>
                               : U[20, 140] ms; wide prior τ
                               <sub>
                                1
                               </sub>
                               : U[0, 200] ms, τ
                               <sub>
                                2
                               </sub>
                               : U[20, 500] ms.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F19" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F19.large.jpg?width=800&height=600&carousel=1" title="Comparison of goodness of fit for the aABC and direct fitting. (A) The estimation error of the MAP estimates from the aABC fit is always small (upper panel), whereas the estimation error of the direct fit depends on the trial duration relative to the ground-truth timescale (lower panel). Data for each fit are generated from an OU process with the given timescales τ and the trial duration T (500 trials). Estimates of direct fits are computed from fitting the autocorrelation shape with an exponential decay function. Errors are computed as |ground-truth timescale—estimated timescale|/(ground-truth timescale). (B) Same as A, for data generated from two-timescale OU processes with the given τ1 and τ2 timescales, coefficients c1 = c2 = 0.5 and trial duration of T = 1000 ms. MAP estimates for both timescales from the joint posterior distribution are always more accurate than direct fitting with the double-exponential function. Here, the relationship between the values of two timescales and accuracy is non-monotonous. In addition, the aABC method returns a full posterior for each estimate (not only the MAP estimate) and the ground-truth timescales are within the bounds of posterior." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparison of goodness of fit for the aABC and direct fitting.</span> <strong>(A)</strong> The estimation error of the MAP estimates from the aABC fit is always small (upper panel), whereas the estimation error of the direct fit depends on the trial duration relative to the ground-truth timescale (lower panel). Data for each fit are generated from an OU process with the given timescales <em>τ</em> and the trial duration <em>T</em> (500 trials). Estimates of direct fits are computed from fitting the autocorrelation shape with an exponential decay function. Errors are computed as |ground-truth timescale—estimated timescale|/(ground-truth timescale). <strong>(B)</strong> Same as A, for data generated from two-timescale OU processes with the given <em>τ</em><sub>1</sub> and <em>τ</em><sub>2</sub> timescales, coefficients <em>c</em><sub>1</sub> = <em>c</em><sub>2</sub> = 0.5 and trial duration of <em>T</em> = 1000 ms. MAP estimates for both timescales from the joint posterior distribution are always more accurate than direct fitting with the double-exponential function. Here, the relationship between the values of two timescales and accuracy is non-monotonous. In addition, the aABC method returns a full posterior for each estimate (not only the MAP estimate) and the ground-truth timescales are within the bounds of posterior.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F19.medium.gif" width="440" height="288"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 13." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F19.medium.gif" width="440" height="288"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 13." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 13.
                              </span>
                              <span class="caption-title">
                               Comparison of goodness of fit for the aABC and direct fitting.
                              </span>
                              <p id="p-110" class="first-child">
                               <strong>
                                (A)
                               </strong>
                               The estimation error of the MAP estimates from the aABC fit is always small (upper panel), whereas the estimation error of the direct fit depends on the trial duration relative to the ground-truth timescale (lower panel). Data for each fit are generated from an OU process with the given timescales τ and the trial duration T (500 trials). Estimates of direct fits are computed from fitting the autocorrelation shape with an exponential decay function. Errors are computed as |ground-truth timescale—estimated timescale|/(ground-truth timescale).
                               <strong>
                                (B)
                               </strong>
                               Same as A, for data generated from two-timescale OU processes with the given τ
                               <sub>
                                1
                               </sub>
                               and τ
                               <sub>
                                2
                               </sub>
                               timescales, coefficients c
                               <sub>
                                1
                               </sub>
                               = c
                               <sub>
                                2
                               </sub>
                               = 0.5 and trial duration of T = 1000 ms. MAP estimates for both timescales from the joint posterior distribution are always more accurate than direct fitting with the double-exponential function. Here, the relationship between the values of two timescales and accuracy is non-monotonous. In addition, the aABC method returns a full posterior for each estimate (not only the MAP estimate) and the ground-truth timescales are within the bounds of posterior.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F20" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F20.large.jpg?width=800&height=600&carousel=1" title="Evaluating the quality of direct fit using abcTau package preprocessing functionality. We use the parametric bootstrapping method to evaluate the quality of direct fit. We generate multiple synthetic datasets from a generative model with parameters estimated by the direct fit and estimate the timescale from each of these datasets again by direct fitting. If the error between the mean of bootstrapping estimates and original direct fit is small enough (based on a user-defined tolerance) the direct fit estimate can be used. Since the bootstrapping error can be smaller than the true error (error between the ground-truth timescale and the direct fit estimate), users should consider setting a stricter tolerance than required. Data in each panel are generated from an OU process with timescale τ = 20 ms and the indicated trial duration T (500 trials). Increasing the trial duration leads to smaller errors for the direct fit estimates. Errors (black arrows) are computed as |direct-fit estimate – mean(bootstrapping distribution)|/(direct-fit estimate)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Evaluating the quality of direct fit using abcTau package preprocessing functionality.</span> We use the parametric bootstrapping method to evaluate the quality of direct fit. We generate multiple synthetic datasets from a generative model with parameters estimated by the direct fit and estimate the timescale from each of these datasets again by direct fitting. If the error between the mean of bootstrapping estimates and original direct fit is small enough (based on a user-defined tolerance) the direct fit estimate can be used. Since the bootstrapping error can be smaller than the true error (error between the ground-truth timescale and the direct fit estimate), users should consider setting a stricter tolerance than required. Data in each panel are generated from an OU process with timescale <em>τ</em> = 20 ms and the indicated trial duration <em>T</em> (500 trials). Increasing the trial duration leads to smaller errors for the direct fit estimates. Errors (black arrows) are computed as |direct-fit estimate – mean(bootstrapping distribution)|/(direct-fit estimate).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 14." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F20.medium.gif" width="440" height="115"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 14." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F20.medium.gif" width="440" height="115"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 14." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 14.
                              </span>
                              <span class="caption-title">
                               Evaluating the quality of direct fit using abcTau package preprocessing functionality.
                              </span>
                              <p id="p-111" class="first-child">
                               We use the parametric bootstrapping method to evaluate the quality of direct fit. We generate multiple synthetic datasets from a generative model with parameters estimated by the direct fit and estimate the timescale from each of these datasets again by direct fitting. If the error between the mean of bootstrapping estimates and original direct fit is small enough (based on a user-defined tolerance) the direct fit estimate can be used. Since the bootstrapping error can be smaller than the true error (error between the ground-truth timescale and the direct fit estimate), users should consider setting a stricter tolerance than required. Data in each panel are generated from an OU process with timescale τ = 20 ms and the indicated trial duration T (500 trials). Increasing the trial duration leads to smaller errors for the direct fit estimates. Errors (black arrows) are computed as |direct-fit estimate – mean(bootstrapping distribution)|/(direct-fit estimate).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F21" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F21.large.jpg?width=800&height=600&carousel=1" title="Evaluating the relation between parameters in the joint posterior distributions from the aABC method. (A-C) Bivariate marginal distributions obtained from the three-dimensional posterior distribution of parameters from fitting a two-timescale model on different example data (same data as in Fig. 5). Isolines are computed by fitting bivariate Gaussian kernel density on the posterior samples and indicate the parameter combinations with equal probability in the joint posterior (isoline levels are set at p = 0.75, 0.5, 0.25 from the inner to outer contours, i.e. for 0.25 isoline 25% of the probability mass will lie within the contour). The blue star shows the MAP estimate from the joint posterior distribution and the gray dots show the samples of the posterior. (A) When data has two clear distinct timescales (τ1 = 5 ms, τ2 = 80 ms) the joint marginals have a clear peak (black isoline with p = 0.75) around the MAP estimate and parameters are more independent from each other (i.e. circular shape of bivariate distributions). (B) When the timescales are closer to each other (τ1 = 20 ms, τ2 = 80 ms) and the distinction between the timescales is less clear in the autocorrelation shape (Fig. 5C), the parameters become more correlated in the posterior. (C) When data has only one timescale (τ = 20 ms), there is a degeneracy in parameters of the two-timescale model and fit provides different possible solutions: (i) both timescales are close to the ground truth and the coefficient takes a broad range of values with the same probability, indicating that the coefficient is a sloppy parameter [83], (b) one of the timescales is away from the ground truth and its coefficient is close to zero, suggesting that this additional timescale is a sloppy parameter. This degeneracy indicates that the model is overparametrized. Note that we enforce τ1 < τ2 which generates asymmetry in the posterior." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Evaluating the relation between parameters in the joint posterior distributions from the aABC method.</span> <strong>(A-C)</strong> Bivariate marginal distributions obtained from the three-dimensional posterior distribution of parameters from fitting a two-timescale model on different example data (same data as in Fig. 5). Isolines are computed by fitting bivariate Gaussian kernel density on the posterior samples and indicate the parameter combinations with equal probability in the joint posterior (isoline levels are set at <em>p</em> = 0.75, 0.5, 0.25 from the inner to outer contours, i.e. for 0.25 isoline 25% of the probability mass will lie within the contour). The blue star shows the MAP estimate from the joint posterior distribution and the gray dots show the samples of the posterior. <strong>(A)</strong> When data has two clear distinct timescales (<em>τ</em><sub>1</sub> = 5 ms, <em>τ</em><sub>2</sub> = 80 ms) the joint marginals have a clear peak (black isoline with <em>p</em> = 0.75) around the MAP estimate and parameters are more independent from each other (i.e. circular shape of bivariate distributions). <strong>(B)</strong> When the timescales are closer to each other (<em>τ</em><sub>1</sub> = 20 ms, <em>τ</em><sub>2</sub> = 80 ms) and the distinction between the timescales is less clear in the autocorrelation shape (Fig. 5C), the parameters become more correlated in the posterior. <strong>(C)</strong> When data has only one timescale (<em>τ</em> = 20 ms), there is a degeneracy in parameters of the two-timescale model and fit provides different possible solutions: (i) both timescales are close to the ground truth and the coefficient takes a broad range of values with the same probability, indicating that the coefficient is a sloppy parameter [83], (b) one of the timescales is away from the ground truth and its coefficient is close to zero, suggesting that this additional timescale is a sloppy parameter. This degeneracy indicates that the model is overparametrized. Note that we enforce <em>τ</em><sub>1</sub> < <em>τ</em><sub>2</sub> which generates asymmetry in the posterior.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F21.medium.gif" width="440" height="399"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 15." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F21.medium.gif" width="440" height="399"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 15." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 15.
                              </span>
                              <span class="caption-title">
                               Evaluating the relation between parameters in the joint posterior distributions from the aABC method.
                              </span>
                              <p id="p-112" class="first-child">
                               <strong>
                                (A-C)
                               </strong>
                               Bivariate marginal distributions obtained from the three-dimensional posterior distribution of parameters from fitting a two-timescale model on different example data (same data as in
                               <a id="xref-fig-5-11" class="xref-fig" href="#F5">
                                Fig. 5
                               </a>
                               ). Isolines are computed by fitting bivariate Gaussian kernel density on the posterior samples and indicate the parameter combinations with equal probability in the joint posterior (isoline levels are set at p = 0.75, 0.5, 0.25 from the inner to outer contours, i.e. for 0.25 isoline 25% of the probability mass will lie within the contour). The blue star shows the MAP estimate from the joint posterior distribution and the gray dots show the samples of the posterior.
                               <strong>
                                (A)
                               </strong>
                               When data has two clear distinct timescales (τ
                               <sub>
                                1
                               </sub>
                               = 5 ms, τ
                               <sub>
                                2
                               </sub>
                               = 80 ms) the joint marginals have a clear peak (black isoline with p = 0.75) around the MAP estimate and parameters are more independent from each other (i.e. circular shape of bivariate distributions).
                               <strong>
                                (B)
                               </strong>
                               When the timescales are closer to each other (τ
                               <sub>
                                1
                               </sub>
                               = 20 ms, τ
                               <sub>
                                2
                               </sub>
                               = 80 ms) and the distinction between the timescales is less clear in the autocorrelation shape (
                               <a id="xref-fig-5-12" class="xref-fig" href="#F5">
                                Fig. 5C
                               </a>
                               ), the parameters become more correlated in the posterior.
                               <strong>
                                (C)
                               </strong>
                               When data has only one timescale (τ = 20 ms), there is a degeneracy in parameters of the two-timescale model and fit provides different possible solutions: (i) both timescales are close to the ground truth and the coefficient takes a broad range of values with the same probability, indicating that the coefficient is a sloppy parameter [83], (b) one of the timescales is away from the ground truth and its coefficient is close to zero, suggesting that this additional timescale is a sloppy parameter. This degeneracy indicates that the model is overparametrized. Note that we enforce τ
                               <sub>
                                1
                               </sub>
                               < τ
                               <sub>
                                2
                               </sub>
                               which generates asymmetry in the posterior.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F22" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F22.large.jpg?width=800&height=600&carousel=1" title="Model selection with ABC discriminates between periodic (oscillatory) and aperiodic (exponential decay) components. (A-C) Data are generated from a linear mixture of an OU process with the timescale τ = 60 ms and an oscillation with frequency f = 2 Hz (same as Supplementary Fig. 9C). Using the ABC model comparison method, we tested whether the shape of the PSD from these data can be equally well described with a two-timescales OU process without any oscillatory components. (A) The data PSD is fitted with two models: 1) a linear mixture of an OU process and oscillation (blue), 2) a two-timescale OU process (purple) (left). Since cumulative distribution of distances for the first model is larger than for the second model for all ε, the model with the linear mixture of an OU process and oscillation is selected (right). (B) The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) for the linear mixture of an OU process and oscillation. Vertical lines indicate the ground-truth values. (C) The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) for the two-timescale OU process. Vertical lines indicate the ground-truth value of the single ground-truth timescale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Model selection with ABC discriminates between periodic (oscillatory) and aperiodic (exponential decay) components.</span> <strong>(A-C)</strong> Data are generated from a linear mixture of an OU process with the timescale <em>τ</em> = 60 ms and an oscillation with frequency <em>f</em> = 2 Hz (same as Supplementary Fig. 9C). Using the ABC model comparison method, we tested whether the shape of the PSD from these data can be equally well described with a two-timescales OU process without any oscillatory components. <strong>(A)</strong> The data PSD is fitted with two models: 1) a linear mixture of an OU process and oscillation (blue), 2) a two-timescale OU process (purple) (left). Since cumulative distribution of distances for the first model is larger than for the second model for all <em>ε</em>, the model with the linear mixture of an OU process and oscillation is selected (right). <strong>(B)</strong> The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) for the linear mixture of an OU process and oscillation. Vertical lines indicate the ground-truth values. <strong>(C)</strong> The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) for the two-timescale OU process. Vertical lines indicate the ground-truth value of the single ground-truth timescale.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 16." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F22.medium.gif" width="440" height="375"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 16." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F22.medium.gif" width="440" height="375"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 16." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 16.
                              </span>
                              <span class="caption-title">
                               Model selection with ABC discriminates between periodic (oscillatory) and aperiodic (exponential decay) components.
                              </span>
                              <p id="p-113" class="first-child">
                               <strong>
                                (A-C)
                               </strong>
                               Data are generated from a linear mixture of an OU process with the timescale τ = 60 ms and an oscillation with frequency f = 2 Hz (same as
                               <a id="xref-fig-15-4" class="xref-fig" href="#F15">
                                Supplementary Fig. 9C
                               </a>
                               ). Using the ABC model comparison method, we tested whether the shape of the PSD from these data can be equally well described with a two-timescales OU process without any oscillatory components.
                               <strong>
                                (A)
                               </strong>
                               The data PSD is fitted with two models: 1) a linear mixture of an OU process and oscillation (blue), 2) a two-timescale OU process (purple) (left). Since cumulative distribution of distances for the first model is larger than for the second model for all ε, the model with the linear mixture of an OU process and oscillation is selected (right).
                               <strong>
                                (B)
                               </strong>
                               The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) for the linear mixture of an OU process and oscillation. Vertical lines indicate the ground-truth values.
                               <strong>
                                (C)
                               </strong>
                               The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) for the two-timescale OU process. Vertical lines indicate the ground-truth value of the single ground-truth timescale.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F23" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F23.large.jpg?width=800&height=600&carousel=1" title="Estimating timescales with aABC in the presence of background processes. (A-B) Data are generated from a linear mixture of an OU process with the timescale τ = 50 ms (coefficient c = 0.5) and random time-series generated from a 1/f PSD shape with exponent χ = 1 over the frequency range of [1, 500] Hz (corresponding to the time-domain signals with T = 1 s duration). (A) The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the fitted generative model with the MAP estimate parameters (orange). The generative model is the same as for the original data (i.e. a linear mixture of an OU process and random time-series generated from a 1/f PSD shape). (B) The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters (vertical dashed lines). (C-D) Same as C-D with τ = 20 ms, χ = 0.75, c = 0.3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Estimating timescales with aABC in the presence of background processes.</span> <strong>(A-B)</strong> Data are generated from a linear mixture of an OU process with the timescale <em>τ</em> = 50 ms (coefficient <em>c</em> = 0.5) and random time-series generated from a 1/<em>f</em> PSD shape with exponent <em>χ</em> = 1 over the frequency range of [1, 500] Hz (corresponding to the time-domain signals with <em>T</em> = 1 s duration). <strong>(A)</strong> The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the fitted generative model with the MAP estimate parameters (orange). The generative model is the same as for the original data (i.e. a linear mixture of an OU process and random time-series generated from a 1/<em>f</em> PSD shape). <strong>(B)</strong> The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters (vertical dashed lines). <strong>(C-D)</strong> Same as C-D with <em>τ</em> = 20 ms, <em>χ</em> = 0.75, <em>c</em> = 0.3.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 17." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F23.medium.gif" width="440" height="170"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 17." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F23.medium.gif" width="440" height="170"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 17." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 17.
                              </span>
                              <span class="caption-title">
                               Estimating timescales with aABC in the presence of background processes.
                              </span>
                              <p id="p-114" class="first-child">
                               <strong>
                                (A-B)
                               </strong>
                               Data are generated from a linear mixture of an OU process with the timescale τ = 50 ms (coefficient c = 0.5) and random time-series generated from a 1/f PSD shape with exponent χ = 1 over the frequency range of [1, 500] Hz (corresponding to the time-domain signals with T = 1 s duration).
                               <strong>
                                (A)
                               </strong>
                               The shape of the data PSD (brown) is accurately reproduced by the PSD of synthetic data from the fitted generative model with the MAP estimate parameters (orange). The generative model is the same as for the original data (i.e. a linear mixture of an OU process and random time-series generated from a 1/f PSD shape).
                               <strong>
                                (B)
                               </strong>
                               The marginal posterior distributions (histograms overlaid with Gaussian kernel smoothing) recover the ground-truth parameters (vertical dashed lines).
                               <strong>
                                (C-D)
                               </strong>
                               Same as C-D with τ = 20 ms, χ = 0.75, c = 0.3.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F24" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F24.large.jpg?width=800&height=600&carousel=1" title="Doubly stochastic generative models with the Gaussian and gamma distributions of event-counts produce similar aABC fits. We fitted the inhomogeneous Poisson process example from Fig. 1C using aABC with generative models which had a Gaussian or gamma event-count distributions with α = 1. Both generative models recover the ground-truth timescales and produce posterior distributioins that are not significantly different (Wilcoxon rank-sum test: Pτ1 = 0.23, Pτ2 = 0.37)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Doubly stochastic generative models with the Gaussian and gamma distributions of event-counts produce similar aABC fits.</span> We fitted the inhomogeneous Poisson process example from Fig. 1C using aABC with generative models which had a Gaussian or gamma event-count distributions with <em>α</em> = 1. Both generative models recover the ground-truth timescales and produce posterior distributioins that are not significantly different (Wilcoxon rank-sum test: <em>P</em><sub><em>τ</em><sub>1</sub></sub> = 0.23, <em>P</em><sub><em>τ</em><sub>2</sub></sub> = 0.37).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 18." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F24.medium.gif" width="440" height="138"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 18." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F24.medium.gif" width="440" height="138"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 18." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 18.
                              </span>
                              <span class="caption-title">
                               Doubly stochastic generative models with the Gaussian and gamma distributions of event-counts produce similar aABC fits.
                              </span>
                              <p id="p-115" class="first-child">
                               We fitted the inhomogeneous Poisson process example from
                               <a id="xref-fig-1-15" class="xref-fig" href="#F1">
                                Fig. 1C
                               </a>
                               using aABC with generative models which had a Gaussian or gamma event-count distributions with α = 1. Both generative models recover the ground-truth timescales and produce posterior distributioins that are not significantly different (Wilcoxon rank-sum test: P
                               <sub>
                                τ
                                <sub>
                                 1
                                </sub>
                               </sub>
                               = 0.23, P
                               <sub>
                                τ
                                <sub>
                                 2
                                </sub>
                               </sub>
                               = 0.37).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F25" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F25.large.jpg?width=800&height=600&carousel=1" title="Estimating the parameter α of the doubly stochastic generative model with aABC and with the grid search. We fitted the neural data autocorrelation in Fig. 6 considering α as an additional parameter of the generative model estimated with aABC (complete aABC). We compared the results of complete aABC with first estimating α with a grid search and then fitting the rest of parameters with aABC (grid search + aABC). (A) Starting from the same error threshold ε = 1, after the same number of iterations (23), the complete aABC has a larger variance of marginal posterior distributions than the grid search + aABC. This result indicates that the grid-search method reaches narrower posteriors in fewer iterations. Moreover, for the complete aABC, the posterior of α is relatively narrow, while posteriors of all other parameters are wider at iteration 23. This observation shows that the complete aABC fits α first and almost independently from the other parameters of the generative model. (B) After running both methods until they reach accR ≤ 0.003, MAP estimates with complete aABC converge toward the estimations from the grid search + aABC." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-439438014" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Estimating the parameter <em>α</em> of the doubly stochastic generative model with aABC and with the grid search.</span> We fitted the neural data autocorrelation in Fig. 6 considering α as an additional parameter of the generative model estimated with aABC (complete aABC). We compared the results of complete aABC with first estimating α with a grid search and then fitting the rest of parameters with aABC (grid search + aABC). <strong>(A)</strong> Starting from the same error threshold <em>ε</em> = 1, after the same number of iterations (23), the complete aABC has a larger variance of marginal posterior distributions than the grid search + aABC. This result indicates that the grid-search method reaches narrower posteriors in fewer iterations. Moreover, for the complete aABC, the posterior of <em>α</em> is relatively narrow, while posteriors of all other parameters are wider at iteration 23. This observation shows that the complete aABC fits α first and almost independently from the other parameters of the generative model. <strong>(B)</strong> After running both methods until they reach <em>accR</em> ≤ 0.003, MAP estimates with complete aABC converge toward the estimations from the grid search + aABC.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Fig. 19." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F25.medium.gif" width="440" height="209"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Fig. 19." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F25.medium.gif" width="440" height="209"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Fig. 19." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/08/2020.08.11.245944/F25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Fig. 19.
                              </span>
                              <span class="caption-title">
                               Estimating the parameter α of the doubly stochastic generative model with aABC and with the grid search.
                              </span>
                              <p id="p-116" class="first-child">
                               We fitted the neural data autocorrelation in
                               <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                                Fig. 6
                               </a>
                               considering α as an additional parameter of the generative model estimated with aABC (complete aABC). We compared the results of complete aABC with first estimating α with a grid search and then fitting the rest of parameters with aABC (grid search + aABC).
                               <strong>
                                (A)
                               </strong>
                               Starting from the same error threshold ε = 1, after the same number of iterations (23), the complete aABC has a larger variance of marginal posterior distributions than the grid search + aABC. This result indicates that the grid-search method reaches narrower posteriors in fewer iterations. Moreover, for the complete aABC, the posterior of α is relatively narrow, while posteriors of all other parameters are wider at iteration 23. This observation shows that the complete aABC fits α first and almost independently from the other parameters of the generative model.
                               <strong>
                                (B)
                               </strong>
                               After running both methods until they reach accR ≤ 0.003, MAP estimates with complete aABC converge toward the estimations from the grid search + aABC.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-95">
                             This work was supported by a Sofja Kovalevskaja Award from the Alexander von Humboldt Foundation, endowed by the Federal Ministry of Education and Research (RZ, AL),
                             <strong>
                              SMART
                             </strong>
                             START2 program provided by Bernstein Center for Computational Neuroscience and Volkswagen Foundation (RZ), NIH grant R01 EB026949 (TAE), and the Pershing Square Foundation (TAE). We acknowledge the support from the BMBF through the Tübingen AI Center (FKZ: 01IS18039B) and International Max Planck Research School for the Mechanisms of Mental Function and Dysfunction (IMPRS-MMFD). We thank N. A. Steinmetz and T. Moore for sharing the electrophysiological data, which are presented in Ref. [
                             <a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">
                              77
                             </a>
                             ] and are archived at the Stanford Neuroscience Institute server at Stanford University.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-2">
                              <p id="p-2">
                               <a class="rev-xref" href="#xref-fn-2-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                †
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 engel{at}cshl.edu
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-others" id="fn-3">
                              <p id="p-3">
                               <a class="rev-xref" href="#xref-fn-3-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                ‡
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 anna.levina{at}uni-tuebingen.de
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-4">
                              <p id="p-7">
                               added more generative model flexibility; extended supplementary materials
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-5">
                              <p id="p-8">
                               <a href="https://github.com/roxana-zeraati/abcTau">
                                https://github.com/roxana-zeraati/abcTau
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.1" data-doi="10.1016/S1359-0278(97)00002-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Veitshans
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dmitri
                                  </span>
                                  <span class="cit-name-surname">
                                   Klimov
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Devarajan
                                  </span>
                                  <span class="cit-name-surname">
                                   Thirumalai
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Protein folding kinetics: timescales, pathways and energy landscapes in terms of sequence-dependent properties
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Folding and Design
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFolding%2B%2526%2Bdesign%26rft.stitle%253DFold%2BDes%26rft.aulast%253DVeitshans%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DProtein%2Bfolding%2Bkinetics%253A%2Btimescales%252C%2Bpathways%2Band%2Benergy%2Blandscapes%2Bin%2Bterms%2Bof%2Bsequence-dependent%2Bproperties.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1359-0278%252897%252900002-3%26rft_id%253Dinfo%253Apmid%252F9080195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S1359-0278(97)00002-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9080195&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1997WK22100002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.2" data-doi="10.1016/j.sbi.2004.01.013">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jan
                                  </span>
                                  <span class="cit-name-surname">
                                   Kubelka
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James
                                  </span>
                                  <span class="cit-name-surname">
                                   Hofrichter
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William A
                                  </span>
                                  <span class="cit-name-surname">
                                   Eaton
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The protein folding ‘speed limit’
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Opinion in Structural Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  76
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  88
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DKubelka%26rft.auinit1%253DJ.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D88%26rft.atitle%253DThe%2Bprotein%2Bfolding%2B%2527speed%2Blimit%2527.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2004.01.013%26rft_id%253Dinfo%253Apmid%252F15102453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.sbi.2004.01.013&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15102453&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000220197300012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.3" data-doi="10.1038/nn.3862">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John D
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alberto
                                  </span>
                                  <span class="cit-name-surname">
                                   Bernacchia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David J
                                  </span>
                                  <span class="cit-name-surname">
                                   Freedman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ranulfo
                                  </span>
                                  <span class="cit-name-surname">
                                   Romo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan D
                                  </span>
                                  <span class="cit-name-surname">
                                   Wallis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xinying
                                  </span>
                                  <span class="cit-name-surname">
                                   Cai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Camillo
                                  </span>
                                  <span class="cit-name-surname">
                                   Padoa-Schioppa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tatiana
                                  </span>
                                  <span class="cit-name-surname">
                                   Pasternak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hyojung
                                  </span>
                                  <span class="cit-name-surname">
                                   Seo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daeyeol
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A hierarchy of intrinsic timescales across primate cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1661
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D17%26rft.spage%253D1661%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.3862%26rft_id%253Dinfo%253Apmid%252F25383900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.3862&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25383900&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yang
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   DeWeese
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gonzalo
                                  </span>
                                  <span class="cit-name-surname">
                                   Otazu
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anthony
                                  </span>
                                  <span class="cit-name-surname">
                                   Zador
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Millisecond-scale differences in neural activity in auditory cortex can drive decisions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.5" data-doi="10.1523/JNEUROSCI.1091-14.2014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kamran
                                  </span>
                                  <span class="cit-name-surname">
                                   Diba
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Asohan
                                  </span>
                                  <span class="cit-name-surname">
                                   Amarasingham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kenji
                                  </span>
                                  <span class="cit-name-surname">
                                   Mizuseki
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   György
                                  </span>
                                  <span class="cit-name-surname">
                                   Buzsáki
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Millisecond timescale synchrony among hippocampal neurons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  45
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  14984
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14994
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DDiba%26rft.auinit1%253DK.%26rft.volume%253D34%26rft.issue%253D45%26rft.spage%253D14984%26rft.epage%253D14994%26rft.atitle%253DMillisecond%2BTimescale%2BSynchrony%2Bamong%2BHippocampal%2BNeurons%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.1091-14.2014%26rft_id%253Dinfo%253Apmid%252F25378164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjExOiIzNC80NS8xNDk4NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzA4LzIwMjAuMDguMTEuMjQ1OTQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jarnail
                                  </span>
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard A
                                  </span>
                                  <span class="cit-name-surname">
                                   Padgett
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Rates of in situ transcription and splicing in large human genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Structural & Molecular Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1128
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BStructural%2B%2526%2BMolecular%2BBiology%26rft.volume%253D16%26rft.spage%253D1128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.7" data-doi="10.1128/MCB.22.19.6706-6718.2002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Agnes
                                  </span>
                                  <span class="cit-name-surname">
                                   Audibert
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dominique
                                  </span>
                                  <span class="cit-name-surname">
                                   Weil
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Francois
                                  </span>
                                  <span class="cit-name-surname">
                                   Dautry
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  In vivo kinetics of mrna splicing and transport in mammalian cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular and Cellular Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  19
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  6706
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6718
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAudibert%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D19%26rft.spage%253D6706%26rft.epage%253D6718%26rft.atitle%253DIn%2BVivo%2BKinetics%2Bof%2BmRNA%2BSplicing%2Band%2BTransport%2Bin%2BMammalian%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.19.6706-6718.2002%26rft_id%253Dinfo%253Apmid%252F12215528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xOS82NzA2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMDgvMjAyMC4wOC4xMS4yNDU5NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.8" data-doi="10.1016/j.cell.2016.02.058">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maya
                                  </span>
                                  <span class="cit-name-surname">
                                   Shamir
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yinon
                                  </span>
                                  <span class="cit-name-surname">
                                   Bar-On
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rob
                                  </span>
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ron
                                  </span>
                                  <span class="cit-name-surname">
                                   Milo
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Snapshot: timescales in cell biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  164
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1302
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1302
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D164%26rft.spage%253D1302%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.02.058%26rft_id%253Dinfo%253Apmid%252F26967295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.02.058&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26967295&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.9" data-doi="10.1126/science.275.5297.213">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Henry
                                  </span>
                                  <span class="cit-name-surname">
                                   Markram
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joachim
                                  </span>
                                  <span class="cit-name-surname">
                                   Lübke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Frotscher
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bert
                                  </span>
                                  <span class="cit-name-surname">
                                   Sakmann
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Regulation of synaptic efficacy by coincidence of postsynaptic aps and epsps
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  275
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5297
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  213
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  215
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMarkram%26rft.auinit1%253DH.%26rft.volume%253D275%26rft.issue%253D5297%26rft.spage%253D213%26rft.epage%253D215%26rft.atitle%253DRegulation%2Bof%2BSynaptic%2BEfficacy%2Bby%2BCoincidence%2Bof%2BPostsynaptic%2BAPs%2Band%2BEPSPs%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.275.5297.213%26rft_id%253Dinfo%253Apmid%252F8985014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvNTI5Ny8yMTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8wOC8yMDIwLjA4LjExLjI0NTk0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.10" data-doi="10.1523/JNEUROSCI.18-24-10464.1998">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Guo-qiang
                                  </span>
                                  <span class="cit-name-surname">
                                   Bi
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mu-ming
                                  </span>
                                  <span class="cit-name-surname">
                                   Poo
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Synaptic modifications in cultured hippocampal neurons: dependence on spike timing, synaptic strength, and postsynaptic cell type
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  24
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  10464
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10472
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DBi%26rft.auinit1%253DG.-q.%26rft.volume%253D18%26rft.issue%253D24%26rft.spage%253D10464%26rft.epage%253D10472%26rft.atitle%253DSynaptic%2BModifications%2Bin%2BCultured%2BHippocampal%2BNeurons%253A%2BDependence%2Bon%2BSpike%2BTiming%252C%2BSynaptic%2BStrength%252C%2Band%2BPostsynaptic%2BCell%2BType%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.18-24-10464.1998%26rft_id%253Dinfo%253Apmid%252F9852584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjExOiIxOC8yNC8xMDQ2NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAyLzA4LzIwMjAuMDguMTEuMjQ1OTQ0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.11" data-doi="10.1111/j.1469-7793.1998.237bu.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dominique
                                  </span>
                                  <span class="cit-name-surname">
                                   Debanne
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Beat H
                                  </span>
                                  <span class="cit-name-surname">
                                   Gähwiler
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Scott M
                                  </span>
                                  <span class="cit-name-surname">
                                   Thompson
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long-term synaptic plasticity between pairs of individual ca3 pyramidal cells in rat hippocampal slice cultures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Physiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  507
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  Pt 1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  237
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPhysiology%26rft.stitle%253DJ.%2BPhysiol.%26rft.aulast%253DDebanne%26rft.auinit1%253DD.%26rft.volume%253D507%26rft.issue%253D1%26rft.spage%253D237%26rft.epage%253D247%26rft.atitle%253DLong-term%2Bsynaptic%2Bplasticity%2Bbetween%2Bpairs%2Bof%2Bindividual%2BCA3%2Bpyramidal%2Bcells%2Bin%2Brat%2Bhippocampal%2Bslice%2Bcultures%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-7793.1998.237bu.x%26rft_id%253Dinfo%253Apmid%252F9490845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1469-7793.1998.237bu.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9490845&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000072401700018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.08.11.245944v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Krishnagopal
                                  </span>
                                  <span class="cit-name-surname">
                                   Dharani
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  The biology of thought: A neuronal mechanism in the generation of thought-A new molecular model
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Academic Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.08.11.245944v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Denis
                                  </span>
                                  <span class="cit-name-surname">
                                   Mollison
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mollison
                                  </span>
                                  <span class="cit-name-surname">
                                   Denis
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Epidemic models: their structure and relation to data
                                 </span>
                                 , volume
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Cambridge University Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Denis
                                  </span>
                                  <span class="cit-name-surname">
                                   Mollison
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Valerie
                                  </span>
                                  <span class="cit-name-surname">
                                   Isham
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bryan
                                  </span>
                                  <span class="cit-name-surname">
                                   Grenfell
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Epidemics: models and data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the Royal Statistical Society: Series A (Statistics in Society)
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  157
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  115
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  129
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1994
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety%252C%2BSeries%2BA%253A%2BStatistics%2Bin%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety%252C%2BSeries%2BA%253A%2BStatistics%2Bin%2BSociety%26rft.aulast%253DMollison%26rft.auinit1%253DD.%26rft.volume%253D157%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D149%26rft.atitle%253DEpidemics%253A%2B%2Bmodels%2Band%2Bdata.%26rft_id%253Dinfo%253Apmid%252F12159126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12159126&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sang Woo
                                  </span>
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel M
                                  </span>
                                  <span class="cit-name-surname">
                                   Cornforth
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan
                                  </span>
                                  <span class="cit-name-surname">
                                   Dushoff
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua S
                                  </span>
                                  <span class="cit-name-surname">
                                   Weitz
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The time scale of asymptomatic transmission affects estimates of epidemic potential in the covid-19 outbreak
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Epidemics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  100392
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEpidemics%26rft.volume%253D31%26rft.spage%253D100392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.16" data-doi="10.1038/35010105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard P
                                  </span>
                                  <span class="cit-name-surname">
                                   Cincotta
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jennifer
                                  </span>
                                  <span class="cit-name-surname">
                                   Wisnewski
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert
                                  </span>
                                  <span class="cit-name-surname">
                                   Engelman
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Human population in the biodiversity hotspots
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  404
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6781
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  990
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  992
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCincotta%26rft.auinit1%253DR.%2BP.%26rft.volume%253D404%26rft.issue%253D6781%26rft.spage%253D990%26rft.epage%253D992%26rft.atitle%253DHuman%2Bpopulation%2Bin%2Bthe%2Bbiodiversity%2Bhotspots.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35010105%26rft_id%253Dinfo%253Apmid%252F10801126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/35010105&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10801126&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000086762000054&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.17" data-doi="10.1073/pnas.1503784112">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Miikka
                                  </span>
                                  <span class="cit-name-surname">
                                   Tallavaara
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Miska
                                  </span>
                                  <span class="cit-name-surname">
                                   Luoto
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Natalia
                                  </span>
                                  <span class="cit-name-surname">
                                   Korhonen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Heikki
                                  </span>
                                  <span class="cit-name-surname">
                                   Järvinen
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Heikki
                                  </span>
                                  <span class="cit-name-surname">
                                   Seppä
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Human population dynamics in europe over the last glacial maximum
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  27
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  8232
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8237
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503784112%26rft_id%253Dinfo%253Apmid%252F26100880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzI3LzgyMzIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8wOC8yMDIwLjA4LjExLjI0NTk0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ulrike
                                  </span>
                                  <span class="cit-name-surname">
                                   Rippke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Julian
                                  </span>
                                  <span class="cit-name-surname">
                                   Ramirez-Villegas
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andy
                                  </span>
                                  <span class="cit-name-surname">
                                   Jarvis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sonja J
                                  </span>
                                  <span class="cit-name-surname">
                                   Vermeulen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Louis
                                  </span>
                                  <span class="cit-name-surname">
                                   Parker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Flora
                                  </span>
                                  <span class="cit-name-surname">
                                   Mer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bernd
                                  </span>
                                  <span class="cit-name-surname">
                                   Diekkrüger
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrew J
                                  </span>
                                  <span class="cit-name-surname">
                                   Challinor
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mark
                                  </span>
                                  <span class="cit-name-surname">
                                   Howden
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Timescales of transformational climate change adaptation in sub-saharan african agriculture
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Climate Change
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  605
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  609
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BClimate%2BChange%26rft.volume%253D6%26rft.spage%253D605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.19" data-doi="10.1038/35055524">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Paul A
                                  </span>
                                  <span class="cit-name-surname">
                                   Baker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Catherine A
                                  </span>
                                  <span class="cit-name-surname">
                                   Rigsby
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geoffrey O
                                  </span>
                                  <span class="cit-name-surname">
                                   Seltzer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sherilyn C
                                  </span>
                                  <span class="cit-name-surname">
                                   Fritz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tim K
                                  </span>
                                  <span class="cit-name-surname">
                                   Lowenstein
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Niklas P
                                  </span>
                                  <span class="cit-name-surname">
                                   Bacher
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carlos
                                  </span>
                                  <span class="cit-name-surname">
                                   Veliz
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Tropical climate changes at millennial and orbital timescales on the bolivian altiplano
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  409
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6821
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  698
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  701
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBaker%26rft.auinit1%253DP.%2BA.%26rft.volume%253D409%26rft.issue%253D6821%26rft.spage%253D698%26rft.epage%253D701%26rft.atitle%253DTropical%2Bclimate%2Bchanges%2Bat%2Bmillennial%2Band%2Borbital%2Btimescales%2Bon%2Bthe%2BBolivian%2BAltiplano.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35055524%26rft_id%253Dinfo%253Apmid%252F11217855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/35055524&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11217855&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000166816400038&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ryan V
                                  </span>
                                  <span class="cit-name-surname">
                                   Raut
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Abraham Z
                                  </span>
                                  <span class="cit-name-surname">
                                   Snyder
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marcus E
                                  </span>
                                  <span class="cit-name-surname">
                                   Raichle
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Hierarchical dynamics as a macroscopic organizing principle of the human brain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mehran Moradi
                                  </span>
                                  <span class="cit-name-surname">
                                   Spitmaan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hyojung
                                  </span>
                                  <span class="cit-name-surname">
                                   Seo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daeyeol
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alireza
                                  </span>
                                  <span class="cit-name-surname">
                                   Soltani
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Multiple timescales of neural dynamics and integration of task-relevant signals across cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard
                                  </span>
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ruud L
                                  </span>
                                  <span class="cit-name-surname">
                                   van den Brink
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Pfeffer
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bradley
                                  </span>
                                  <span class="cit-name-surname">
                                   Voytek
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Neuronal timescales are functionally dynamic and shaped by cortical microarchitecture
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e61277
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D9%26rft.spage%253De61277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.23" data-doi="10.7554/eLife.42256">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Takamitsu
                                  </span>
                                  <span class="cit-name-surname">
                                   Watanabe
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Geraint
                                  </span>
                                  <span class="cit-name-surname">
                                   Rees
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Naoki
                                  </span>
                                  <span class="cit-name-surname">
                                   Masuda
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Atypical intrinsic neural timescale in autism
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e42256
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D8%26rft.spage%253De42256%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.42256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.42256&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Roxana
                                  </span>
                                  <span class="cit-name-surname">
                                   Zeraati
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yan-Liang
                                  </span>
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marc A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gieselmann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander
                                  </span>
                                  <span class="cit-name-surname">
                                   Thiele
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tirin
                                  </span>
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anna
                                  </span>
                                  <span class="cit-name-surname">
                                   Levina
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tatiana A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Engel
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Attentional modulation of intrinsic timescales in visual cortex and spatial networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 ,
                                 <span class="cit-month">
                                  May
                                 </span>
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jens
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilting
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Viola
                                  </span>
                                  <span class="cit-name-surname">
                                   Priesemann
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Inferring collective dynamical states from widely unobserved systems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.08.11.245944v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William
                                  </span>
                                  <span class="cit-name-surname">
                                   Bialek
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Biophysics: searching for principles
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Princeton University Press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.27" data-doi="10.1073/pnas.1118633109">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William
                                  </span>
                                  <span class="cit-name-surname">
                                   Bialek
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrea
                                  </span>
                                  <span class="cit-name-surname">
                                   Cavagna
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Irene
                                  </span>
                                  <span class="cit-name-surname">
                                   Giardina
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thierry
                                  </span>
                                  <span class="cit-name-surname">
                                   Mora
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edmondo
                                  </span>
                                  <span class="cit-name-surname">
                                   Silvestri
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Massimiliano
                                  </span>
                                  <span class="cit-name-surname">
                                   Viale
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aleksandra M
                                  </span>
                                  <span class="cit-name-surname">
                                   Walczak
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Statistical mechanics for natural flocks of birds
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  13
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  4786
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4791
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1118633109%26rft_id%253Dinfo%253Apmid%252F22427355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzEzLzQ3ODYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8wOC8yMDIwLjA4LjExLjI0NTk0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.28" data-doi="10.1073/pnas.1514188112">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gašper
                                  </span>
                                  <span class="cit-name-surname">
                                   Tkačik
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thierry
                                  </span>
                                  <span class="cit-name-surname">
                                   Mora
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Olivier
                                  </span>
                                  <span class="cit-name-surname">
                                   Marre
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dario
                                  </span>
                                  <span class="cit-name-surname">
                                   Amodei
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephanie E
                                  </span>
                                  <span class="cit-name-surname">
                                   Palmer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael J
                                  </span>
                                  <span class="cit-name-surname">
                                   Berry
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William
                                  </span>
                                  <span class="cit-name-surname">
                                   Bialek
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Thermodynamics and signatures of criticality in a network of neurons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  37
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  11508
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11513
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1514188112%26rft_id%253Dinfo%253Apmid%252F26330611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzM3LzExNTA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMDgvMjAyMC4wOC4xMS4yNDU5NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sean E
                                  </span>
                                  <span class="cit-name-surname">
                                   Cavanagh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joni D
                                  </span>
                                  <span class="cit-name-surname">
                                   Wallis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Steven W
                                  </span>
                                  <span class="cit-name-surname">
                                   Kennerley
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Laurence T
                                  </span>
                                  <span class="cit-name-surname">
                                   Hunt
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Autocorrelation structure at rest predicts value correlates of single neurons during reward-guided choice
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e18937
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De18937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.30" data-doi="10.1016/j.neuron.2015.09.008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rishidev
                                  </span>
                                  <span class="cit-name-surname">
                                   Chaudhuri
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kenneth
                                  </span>
                                  <span class="cit-name-surname">
                                   Knoblauch
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marie-Alice
                                  </span>
                                  <span class="cit-name-surname">
                                   Gariel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Henry
                                  </span>
                                  <span class="cit-name-surname">
                                   Kennedy
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xiao-Jing
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  88
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  419
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  431
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D88%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2015.09.008%26rft_id%253Dinfo%253Apmid%252F26439530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2015.09.008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26439530&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.31" data-doi="10.7554/eLife.19695">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carsen
                                  </span>
                                  <span class="cit-name-surname">
                                   Stringer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marius
                                  </span>
                                  <span class="cit-name-surname">
                                   Pachitariu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas A
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael
                                  </span>
                                  <span class="cit-name-surname">
                                   Okun
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                  <span class="cit-name-surname">
                                   Bartho
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kenneth D
                                  </span>
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maneesh
                                  </span>
                                  <span class="cit-name-surname">
                                   Sahani
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas A
                                  </span>
                                  <span class="cit-name-surname">
                                   Lesica
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Inhibitory control of correlated intrinsic variability in cortical networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  e19695
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D5%26rft.spage%253De19695%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.19695%26rft_id%253Dinfo%253Apmid%252F27926356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.19695&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27926356&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                  <span class="cit-name-surname">
                                   Sarracino
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                  <span class="cit-name-surname">
                                   Arviv
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                  <span class="cit-name-surname">
                                   Shriki
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                  <span class="cit-name-surname">
                                   de Arcangelis
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Predicting brain evoked response to external stimuli from temporal correlations of spontaneous activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  033355
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BResearch%26rft.volume%253D2%26rft.spage%253D033355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.33" data-doi="10.1038/s41586-020-03171-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua H
                                  </span>
                                  <span class="cit-name-surname">
                                   Siegle
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xiaoxuan
                                  </span>
                                  <span class="cit-name-surname">
                                   Jia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Séverine
                                  </span>
                                  <span class="cit-name-surname">
                                   Durand
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sam
                                  </span>
                                  <span class="cit-name-surname">
                                   Gale
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Corbett
                                  </span>
                                  <span class="cit-name-surname">
                                   Bennett
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nile
                                  </span>
                                  <span class="cit-name-surname">
                                   Graddis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Greggory
                                  </span>
                                  <span class="cit-name-surname">
                                   Heller
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tamina K
                                  </span>
                                  <span class="cit-name-surname">
                                   Ramirez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hannah
                                  </span>
                                  <span class="cit-name-surname">
                                   Choi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jennifer A
                                  </span>
                                  <span class="cit-name-surname">
                                   Luviano
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Survey of spiking in the mouse visual system reveals functional hierarchy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  592
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7852
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  86
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  92
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D592%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-03171-x%26rft_id%253Dinfo%253Apmid%252F33473216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-020-03171-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33473216&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.34" data-doi="10.1093/cercor/bhx321">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Valeria
                                  </span>
                                  <span class="cit-name-surname">
                                   Fascianelli
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Satoshi
                                  </span>
                                  <span class="cit-name-surname">
                                   Tsujimoto
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Encarni
                                  </span>
                                  <span class="cit-name-surname">
                                   Marcos
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aldo
                                  </span>
                                  <span class="cit-name-surname">
                                   Genovesio
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Autocorrelation structure in the macaque dorsolateral, but not orbital or polar, prefrontal cortex predicts response-coding strength in a visually cued strategy task
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cerebral Cortex
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  230
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  241
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCerebral%2BCortex%26rft.volume%253D29%26rft.spage%253D230%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcercor%252Fbhx321%26rft_id%253Dinfo%253Apmid%252F29228110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/cercor/bhx321&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29228110&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Camden J
                                  </span>
                                  <span class="cit-name-surname">
                                   MacDowell
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Timothy J
                                  </span>
                                  <span class="cit-name-surname">
                                   Buschman
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Low-dimensional spatiotemporal dynamics underlie cortex-wide neural activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Biology
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.36" data-doi="10.1038/s41593-020-00753-w">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Terrence J
                                  </span>
                                  <span class="cit-name-surname">
                                   Sejnowski
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Strong inhibitory signaling underlies stable temporal dynamics and working memory in spiking neural networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  129
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  139
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D24%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-020-00753-w%26rft_id%253Dinfo%253Apmid%252F33288909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41593-020-00753-w&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33288909&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Takuya
                                  </span>
                                  <span class="cit-name-surname">
                                   Ito
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Luke J
                                  </span>
                                  <span class="cit-name-surname">
                                   Hearne
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael W
                                  </span>
                                  <span class="cit-name-surname">
                                   Cole
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A cortical hierarchy of localized and distributed processes revealed via dissociation of task activations, connectivity changes, and intrinsic timescales
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  NeuroImage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  221
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  117141
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroImage%26rft.volume%253D221%26rft.spage%253D117141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                  <span class="cit-name-surname">
                                   Strey
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                  <span class="cit-name-surname">
                                   Sackmann
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Measurement of erythrocyte membrane elasticity by flicker eigenmode decomposition
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical Journal
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  69
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  478
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  488
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DStrey%26rft.auinit1%253DH%26rft.volume%253D69%26rft.issue%253D2%26rft.spage%253D478%26rft.epage%253D488%26rft.atitle%253DMeasurement%2Bof%2Berythrocyte%2Bmembrane%2Belasticity%2Bby%2Bflicker%2Beigenmode%2Bdecomposition.%26rft_id%253Dinfo%253Apmid%252F8527662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8527662&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1995RX25100020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aleander
                                  </span>
                                  <span class="cit-name-surname">
                                   Rohrbach
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tim
                                  </span>
                                  <span class="cit-name-surname">
                                   Meyer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ernst HK
                                  </span>
                                  <span class="cit-name-surname">
                                   Stelzer
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Holger
                                  </span>
                                  <span class="cit-name-surname">
                                   Kress
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Measuring stepwise binding of thermally fluctuating particles to cell membranes without fluorescence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical Journal
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kai
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Chu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jay
                                  </span>
                                  <span class="cit-name-surname">
                                   Newby
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elizabeth L
                                  </span>
                                  <span class="cit-name-surname">
                                   Read
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John
                                  </span>
                                  <span class="cit-name-surname">
                                   Lowengrub
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jun
                                  </span>
                                  <span class="cit-name-surname">
                                   Allard
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Hydrodynamics of transient cell-cell contact: The role of membrane permeability and active protrusion length
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Computational Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1006352
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D15%26rft.spage%253D1006352e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.41" data-doi="10.1073/pnas.1911461117">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian
                                  </span>
                                  <span class="cit-name-surname">
                                   Meisel
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Antiepileptic drugs induce subcritical dynamics in human cortical networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  117
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  11118
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11125
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1911461117%26rft_id%253Dinfo%253Apmid%252F32358198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE3LzIwLzExMTE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMDgvMjAyMC4wOC4xMS4yNDU5NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.42" data-doi="10.1038/s41593-020-00744-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Donoghue
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matar
                                  </span>
                                  <span class="cit-name-surname">
                                   Haller
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Erik J
                                  </span>
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Paroma
                                  </span>
                                  <span class="cit-name-surname">
                                   Varma
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Priyadarshini
                                  </span>
                                  <span class="cit-name-surname">
                                   Sebastian
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard
                                  </span>
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Torben
                                  </span>
                                  <span class="cit-name-surname">
                                   Noto
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Antonio H
                                  </span>
                                  <span class="cit-name-surname">
                                   Lara
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joni D
                                  </span>
                                  <span class="cit-name-surname">
                                   Wallis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert T
                                  </span>
                                  <span class="cit-name-surname">
                                   Knight
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Parameterizing neural power spectra into periodic and aperiodic components
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1655
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1665
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D23%26rft.spage%253D1655%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-020-00744-x%26rft_id%253Dinfo%253Apmid%252F33230329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41593-020-00744-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33230329&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.43" data-doi="10.1007/BF01449156">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander
                                  </span>
                                  <span class="cit-name-surname">
                                   Khintchine
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Korrelationstheorie der stationären stochastischen prozesse
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Math-ematische Annalen
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  604
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  615
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1934
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMath-ematische%2BAnnalen%26rft.volume%253D109%26rft.spage%253D604%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01449156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/BF01449156&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000201316200036&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.44" data-doi="10.1152/jn.1998.80.6.3345">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carlos D
                                  </span>
                                  <span class="cit-name-surname">
                                   Brody
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Slow covariations in neuronal resting potentials can lead to artefactually fast cross-correlations in their spike trains
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  80
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  3345
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3351
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeurophysiology%26rft.stitle%253DJ.%2BNeurophysiol.%26rft.aulast%253DBrody%26rft.auinit1%253DC.%2BD.%26rft.volume%253D80%26rft.issue%253D6%26rft.spage%253D3345%26rft.epage%253D3351%26rft.atitle%253DSlow%2BCovariations%2Bin%2BNeuronal%2BResting%2BPotentials%2BCan%2BLead%2Bto%2BArtefactually%2BFast%2BCross-Correlations%2Bin%2BTheir%2BSpike%2BTrains%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.1998.80.6.3345%26rft_id%253Dinfo%253Apmid%252F9862930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.1998.80.6.3345&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9862930&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000077835000046&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.45" data-doi="10.1152/jn.00644.2004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Valérie
                                  </span>
                                  <span class="cit-name-surname">
                                   Ventura
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Can
                                  </span>
                                  <span class="cit-name-surname">
                                   Cai
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert E
                                  </span>
                                  <span class="cit-name-surname">
                                   Kass
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Trial-to-trial variability and its effect on time-varying dependency between two neurons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  94
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2928
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2939
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeurophysiology%26rft.stitle%253DJ.%2BNeurophysiol.%26rft.aulast%253DVentura%26rft.auinit1%253DV.%26rft.volume%253D94%26rft.issue%253D4%26rft.spage%253D2928%26rft.epage%253D2939%26rft.atitle%253DTrial-to-Trial%2BVariability%2Band%2BIts%2BEffect%2Bon%2BTime-Varying%2BDependency%2BBetween%2BTwo%2BNeurons%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.00644.2004%26rft_id%253Dinfo%253Apmid%252F16160096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.00644.2004&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16160096&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000231834200055&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [46].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.46" data-doi="10.1152/jn.00633.2011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Asohan
                                  </span>
                                  <span class="cit-name-surname">
                                   Amarasingham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthew T
                                  </span>
                                  <span class="cit-name-surname">
                                   Harrison
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas G
                                  </span>
                                  <span class="cit-name-surname">
                                   Hatsopoulos
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stuart
                                  </span>
                                  <span class="cit-name-surname">
                                   Geman
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Conditional modeling and the jitter method of spike resampling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  517
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  531
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeurophysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.00633.2011%26rft_id%253Dinfo%253Apmid%252F22031767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.00633.2011&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22031767&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000299167000001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [47].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.47" data-doi="10.1038/nn.2842">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marlene R
                                  </span>
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Adam
                                  </span>
                                  <span class="cit-name-surname">
                                   Kohn
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Measuring and interpreting neuronal correlations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  811
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DCohen%26rft.auinit1%253DM.%2BR.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D811%26rft.epage%253D819%26rft.atitle%253DMeasuring%2Band%2Binterpreting%2Bneuronal%2Bcorrelations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.2842%26rft_id%253Dinfo%253Apmid%252F21709677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.2842&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21709677&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [48].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.48" data-doi="10.1093/biomet/41.3-4.390">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   FHC
                                  </span>
                                  <span class="cit-name-surname">
                                   Marriott
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                  <span class="cit-name-surname">
                                   Pope
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Bias in the estimation of autocorrelations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biometrika
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3/4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  390
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  402
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1954
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DMARRIOTT%26rft.auinit1%253DF.%2BH.%2BC.%26rft.volume%253D41%26rft.issue%253D3-4%26rft.spage%253D390%26rft.epage%253D402%26rft.atitle%253DBIAS%2BIN%2BTHE%2BESTIMATION%2BOF%2BAUTOCORRELATIONS%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F41.3-4.390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biomet/41.3-4.390&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1954UZ96600013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [49].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A Sree Rama
                                  </span>
                                  <span class="cit-name-surname">
                                   Sastry
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Bias in estimation of serial correlation coefficients
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sankhyā: The Indian Journal of Statistics
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  281
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  296
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1951
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [50].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.50" data-doi="10.1037//0033-2909.110.2.291">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bradley E
                                  </span>
                                  <span class="cit-name-surname">
                                   Huitema
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joseph W
                                  </span>
                                  <span class="cit-name-surname">
                                   McKean
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Autocorrelation estimation and inference with small samples
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Psychological Bulletin
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  291
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPsychological%2BBulletin%26rft.volume%253D110%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1037%252F%252F0033-2909.110.2.291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1037//0033-2909.110.2.291&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1991GE03600008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [51].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.51" data-doi="10.1093/biomet/48.1-2.85">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John S
                                  </span>
                                  <span class="cit-name-surname">
                                   White
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Asymptotic expansions for the mean and variance of the serial correlation coefficient
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biometrika
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1-2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  85
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  94
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1961
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DWHITE%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D48%26rft.issue%253D1-2%26rft.spage%253D85%26rft.epage%253D94%26rft.atitle%253DAsymptotic%2Bexpansions%2Bfor%2Bthe%2Bmean%2Band%2Bvariance%2Bof%2Bthe%2Bserial%2Bcorrelation%2Bcoefficient%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F48.1-2.85%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biomet/48.1-2.85&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [52].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.52">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   ZA
                                  </span>
                                  <span class="cit-name-surname">
                                   Lomnicki
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   SK
                                  </span>
                                  <span class="cit-name-surname">
                                   Zaremba
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  On the estimation of autocorrelation in time series
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Annals of Mathematical Statistics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  140
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  158
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1957
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BMathematical%2BStatistics%26rft.volume%253D28%26rft.spage%253D140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [53].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.53" data-doi="10.1093/biomet/41.3-4.403">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maurice G
                                  </span>
                                  <span class="cit-name-surname">
                                   Kendall
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Note on bias in the estimation of autocorrelation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biometrika
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3-4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  403
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  404
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1954
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DKENDALL%26rft.auinit1%253DM.%2BG.%26rft.volume%253D41%26rft.issue%253D3-4%26rft.spage%253D403%26rft.epage%253D404%26rft.atitle%253DNOTE%2BON%2BBIAS%2BIN%2BTHE%2BESTIMATION%2BOF%2BAUTOCORRELATION%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F41.3-4.403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biomet/41.3-4.403&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1954UZ96600014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [54].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.54">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Soroosh
                                  </span>
                                  <span class="cit-name-surname">
                                   Afyouni
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephen M
                                  </span>
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas E
                                  </span>
                                  <span class="cit-name-surname">
                                   Nichols
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Effective degrees of freedom of the pearson’s correlation coefficient under autocorrelation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  NeuroImage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  199
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  609
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  625
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroImage%26rft.volume%253D199%26rft.spage%253D609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [55].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.55" data-doi="10.2307/2983611">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maurice S
                                  </span>
                                  <span class="cit-name-surname">
                                   Bartlett
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  On the theoretical specification and sampling properties of autocorrelated time-series
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Supplement to the Journal of the Royal Statistical Society
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  27
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1946
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSupplement%2Bto%2Bthe%2BJournal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety%26rft.volume%253D8%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2983611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/2983611&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1946YH05600002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [56].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.56">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Helmut H
                                  </span>
                                  <span class="cit-name-surname">
                                   Strey
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Estimation of parameters from time traces originating from an ornstein-uhlenbeck process
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review E
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  100
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  062142
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BE%26rft.volume%253D100%26rft.spage%253D062142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [57].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.57">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mikael
                                  </span>
                                  <span class="cit-name-surname">
                                   Sunnåker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alberto Giovanni
                                  </span>
                                  <span class="cit-name-surname">
                                   Busetto
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elina
                                  </span>
                                  <span class="cit-name-surname">
                                   Numminen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jukka
                                  </span>
                                  <span class="cit-name-surname">
                                   Corander
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthieu
                                  </span>
                                  <span class="cit-name-surname">
                                   Foll
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christophe
                                  </span>
                                  <span class="cit-name-surname">
                                   Dessimoz
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Approximate bayesian computation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Computational Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [58].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.58" data-doi="10.1093/biomet/asp052">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mark A
                                  </span>
                                  <span class="cit-name-surname">
                                   Beaumont
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Marie
                                  </span>
                                  <span class="cit-name-surname">
                                   Cornuet
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Michel
                                  </span>
                                  <span class="cit-name-surname">
                                   Marin
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian P
                                  </span>
                                  <span class="cit-name-surname">
                                   Robert
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Adaptive approximate bayesian computation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biometrika
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  96
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  983
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  990
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252Fasp052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biomet/asp052&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000272179100018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [59].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.59" data-doi="10.1103/PhysRev.36.823">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   George E
                                  </span>
                                  <span class="cit-name-surname">
                                   Uhlenbeck
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Leonard S
                                  </span>
                                  <span class="cit-name-surname">
                                   Ornstein
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  On the theory of the brownian motion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  823
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1930
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%26rft.volume%253D36%26rft.spage%253D823%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRev.36.823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1103/PhysRev.36.823&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [60].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.08.11.245944v3.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hannes
                                  </span>
                                  <span class="cit-name-surname">
                                   Risken
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Fokker-planck equation
                                 </span>
                                 . In
                                 <span class="cit-source">
                                  The Fokker-Planck Equation
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  63
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  95
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [61].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Benjamin
                                  </span>
                                  <span class="cit-name-surname">
                                   Lindner
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A brief introduction to some simple stochastic processes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stochastic Methods in Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [62].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.62" data-doi="10.1007/s10827-009-0208-9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ryan C
                                  </span>
                                  <span class="cit-name-surname">
                                   Kelly
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthew A
                                  </span>
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert E
                                  </span>
                                  <span class="cit-name-surname">
                                   Kass
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tai Sing
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Local field potentials indicate network state and account for neuronal response variability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Computational Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  567
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  579
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bneuroscience%26rft.stitle%253DJ%2BComput%2BNeurosci%26rft.aulast%253DKelly%26rft.auinit1%253DR.%2BC.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D567%26rft.epage%253D579%26rft.atitle%253DLocal%2Bfield%2Bpotentials%2Bindicate%2Bnetwork%2Bstate%2Band%2Baccount%2Bfor%2Bneuronal%2Bresponse%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10827-009-0208-9%26rft_id%253Dinfo%253Apmid%252F20094906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10827-009-0208-9&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20094906&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000284162900015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [63].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.63" data-doi="10.1016/j.neuron.2014.02.006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander S
                                  </span>
                                  <span class="cit-name-surname">
                                   Ecker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philipp
                                  </span>
                                  <span class="cit-name-surname">
                                   Berens
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R James
                                  </span>
                                  <span class="cit-name-surname">
                                   Cotton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Manivannan
                                  </span>
                                  <span class="cit-name-surname">
                                   Subramaniyan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   George H
                                  </span>
                                  <span class="cit-name-surname">
                                   Denfield
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Cathryn R
                                  </span>
                                  <span class="cit-name-surname">
                                   Cadwell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stelios M
                                  </span>
                                  <span class="cit-name-surname">
                                   Smirnakis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthias
                                  </span>
                                  <span class="cit-name-surname">
                                   Bethge
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andreas S
                                  </span>
                                  <span class="cit-name-surname">
                                   Tolias
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  State dependence of noise correlations in macaque primary visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  82
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  235
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  248
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D82%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2014.02.006%26rft_id%253Dinfo%253Apmid%252F24698278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2014.02.006&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24698278&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000333804800021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [64].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.64">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mikhail
                                  </span>
                                  <span class="cit-name-surname">
                                   Genkin
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tatiana A
                                  </span>
                                  <span class="cit-name-surname">
                                   Engel
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Moving beyond generalization to accurate interpretation of flexible models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Machine Intelligence
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  674
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  683
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMachine%2BIntelligence%26rft.volume%253D2%26rft.spage%253D674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [65].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference [65] in text" id="ref-65">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.65" data-doi="10.1016/j.neuron.2010.12.037">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anne K
                                  </span>
                                  <span class="cit-name-surname">
                                   Churchland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Roozbeh
                                  </span>
                                  <span class="cit-name-surname">
                                   Kiani
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rishidev
                                  </span>
                                  <span class="cit-name-surname">
                                   Chaudhuri
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xiao-Jing
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexandre
                                  </span>
                                  <span class="cit-name-surname">
                                   Pouget
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael N
                                  </span>
                                  <span class="cit-name-surname">
                                   Shadlen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Variance as a signature of neural computations during decision making
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  69
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  818
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  831
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D69%26rft.spage%253D818%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2010.12.037%26rft_id%253Dinfo%253Apmid%252F21338889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2010.12.037&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21338889&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000288413200021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [66].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference [66] in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.66">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Demetrios
                                  </span>
                                  <span class="cit-name-surname">
                                   Neophytou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Diego
                                  </span>
                                  <span class="cit-name-surname">
                                   Arribas
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert
                                  </span>
                                  <span class="cit-name-surname">
                                   Levy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Il Memming
                                  </span>
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Hysell V
                                  </span>
                                  <span class="cit-name-surname">
                                   Oviedo
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Recurrent connectivity underlies lateralized temporal processing differences in auditory cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [67].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference [67] in text" id="ref-67">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.67" data-doi="10.1109/TBME.2014.2311996">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Behtash
                                  </span>
                                  <span class="cit-name-surname">
                                   Babadi
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Emery N
                                  </span>
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A review of multitaper spectral analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Transactions on Biomedical Engineering
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  61
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1555
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1564
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTransactions%2Bon%2BBiomedical%2BEngineering%26rft.volume%253D61%26rft.spage%253D1555%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTBME.2014.2311996%26rft_id%253Dinfo%253Apmid%252F24759284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/TBME.2014.2311996&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24759284&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [68].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference [68] in text" id="ref-68">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.68" data-doi="10.1103/PhysRevLett.94.058101">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Clayton
                                  </span>
                                  <span class="cit-name-surname">
                                   Haldeman
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John M
                                  </span>
                                  <span class="cit-name-surname">
                                   Beggs
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Critical branching captures activity in living neural networks and maximizes the number of metastable states
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review Letters
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  94
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  058101
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BLetters%26rft.stitle%253DPhysical%2BReview%2BLetters%26rft.aulast%253DHaldeman%26rft.auinit1%253DC.%26rft.volume%253D94%26rft.issue%253D5%26rft.spage%253D058101%26rft.epage%253D058101%26rft.atitle%253DCritical%2Bbranching%2Bcaptures%2Bactivity%2Bin%2Bliving%2Bneural%2Bnetworks%2Band%2Bmaximizes%2Bthe%2Bnumber%2Bof%2Bmetastable%2BStates.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevLett.94.058101%26rft_id%253Dinfo%253Apmid%252F15783702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1103/PhysRevLett.94.058101&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15783702&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [69].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.69">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Johannes
                                  </span>
                                  <span class="cit-name-surname">
                                   Zierenberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jens
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilting
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Viola
                                  </span>
                                  <span class="cit-name-surname">
                                   Priesemann
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anna
                                  </span>
                                  <span class="cit-name-surname">
                                   Levina
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Description of spreading dynamics by microscopic network models and macroscopic branching processes can differ due to coalescence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review E
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  101
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  022301
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BE%26rft.volume%253D101%26rft.spage%253D022301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [70].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference [70] in text" id="ref-70">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.70">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Johannes
                                  </span>
                                  <span class="cit-name-surname">
                                   Zierenberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jens
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilting
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Viola
                                  </span>
                                  <span class="cit-name-surname">
                                   Priesemann
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anna
                                  </span>
                                  <span class="cit-name-surname">
                                   Levina
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Tailored ensembles of neural networks optimize sensitivity to stimulus statistics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physical Review Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  013115
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysical%2BReview%2BResearch%26rft.volume%253D2%26rft.spage%253D013115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [71].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference [71] in text" id="ref-71">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.71">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Aude
                                  </span>
                                  <span class="cit-name-surname">
                                   Grelaud
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian P
                                  </span>
                                  <span class="cit-name-surname">
                                   Robert
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Michel
                                  </span>
                                  <span class="cit-name-surname">
                                   Marin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   François
                                  </span>
                                  <span class="cit-name-surname">
                                   Rodolphe
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-François
                                  </span>
                                  <span class="cit-name-surname">
                                   Taly
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Abc likelihood-free methods for model choice in gibbs random fields
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bayesian Analysis
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  317
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  335
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnalysis%26rft.volume%253D4%26rft.spage%253D317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [72].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.72">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xavier
                                  </span>
                                  <span class="cit-name-surname">
                                   Didelot
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard G
                                  </span>
                                  <span class="cit-name-surname">
                                   Everitt
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Adam M
                                  </span>
                                  <span class="cit-name-surname">
                                   Johansen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel J
                                  </span>
                                  <span class="cit-name-surname">
                                   Lawson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Likelihood-free estimation of model evidence
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bayesian Analysis
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  49
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  76
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnalysis%26rft.volume%253D6%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [73].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference [73] in text" id="ref-73">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.73">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Michel
                                  </span>
                                  <span class="cit-name-surname">
                                   Marin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Natesh S
                                  </span>
                                  <span class="cit-name-surname">
                                   Pillai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian P
                                  </span>
                                  <span class="cit-name-surname">
                                   Robert
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Judith
                                  </span>
                                  <span class="cit-name-surname">
                                   Rousseau
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Relevant statistics for bayesian model choice
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the Royal Statistical Society: Series B: Statistical Methodology
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  833
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  859
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [74].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference [74] in text" id="ref-74">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.74" data-doi="10.1073/pnas.1102900108">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian P
                                  </span>
                                  <span class="cit-name-surname">
                                   Robert
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Marie
                                  </span>
                                  <span class="cit-name-surname">
                                   Cornuet
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Michel
                                  </span>
                                  <span class="cit-name-surname">
                                   Marin
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Natesh S
                                  </span>
                                  <span class="cit-name-surname">
                                   Pillai
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Lack of confidence in approximate bayesian computation model choice
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  108
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  37
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  15112
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15117
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102900108%26rft_id%253Dinfo%253Apmid%252F21876135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM3LzE1MTEyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMDgvMjAyMC4wOC4xMS4yNDU5NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [75].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference [75] in text" id="ref-75">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.08.11.245944v3.75">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher M
                                  </span>
                                  <span class="cit-name-surname">
                                   Bishop
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Pattern recognition and machine learning
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [76].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference [76] in text" id="ref-76">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.76" data-doi="10.1093/bioinformatics/btp619">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tina
                                  </span>
                                  <span class="cit-name-surname">
                                   Toni
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael PH
                                  </span>
                                  <span class="cit-name-surname">
                                   Stumpf
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Simulation-based model selection for dynamical systems in systems and population biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  104
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  110
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp619%26rft_id%253Dinfo%253Apmid%252F19880371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp619&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19880371&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000273116100015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [77].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference [77] in text" id="ref-77">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.77" data-doi="10.1126/science.aag1420">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tatiana A
                                  </span>
                                  <span class="cit-name-surname">
                                   Engel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nicholas A
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marc A
                                  </span>
                                  <span class="cit-name-surname">
                                   Gieselmann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexander
                                  </span>
                                  <span class="cit-name-surname">
                                   Thiele
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tirin
                                  </span>
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kwabena
                                  </span>
                                  <span class="cit-name-surname">
                                   Boahen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Selective modulation of cortical state during spatial attention
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  354
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6316
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1140
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1144
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DEngel%26rft.auinit1%253DT.%2BA.%26rft.volume%253D354%26rft.issue%253D6316%26rft.spage%253D1140%26rft.epage%253D1144%26rft.atitle%253DSelective%2Bmodulation%2Bof%2Bcortical%2Bstate%2Bduring%2Bspatial%2Battention%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aag1420%26rft_id%253Dinfo%253Apmid%252F27934763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTQvNjMxNi8xMTQwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMDgvMjAyMC4wOC4xMS4yNDU5NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [78].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference [78] in text" id="ref-78">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.78" data-doi="10.1038/nn.3711">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robbe LT
                                  </span>
                                  <span class="cit-name-surname">
                                   Goris
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J Anthony
                                  </span>
                                  <span class="cit-name-surname">
                                   Movshon
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eero P
                                  </span>
                                  <span class="cit-name-surname">
                                   Simoncelli
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Partitioning neuronal variability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  858
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D17%26rft.spage%253D858%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.3711%26rft_id%253Dinfo%253Apmid%252F24777419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.3711&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24777419&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [79].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference [79] in text" id="ref-79">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.79" data-doi="10.1162/089976699300016133">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Carlos D
                                  </span>
                                  <span class="cit-name-surname">
                                   Brody
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Correlations without synchrony
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural Computation
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1537
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1551
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BComputation%26rft.stitle%253DNeural%2BComput.%26rft.aulast%253DBrody%26rft.auinit1%253DC.%2BD.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D1537%26rft.epage%253D1551%26rft.atitle%253DCorrelations%2Bwithout%2Bsynchrony%26rft_id%253Dinfo%253Adoi%252F10.1162%252F089976699300016133%26rft_id%253Dinfo%253Apmid%252F10490937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/089976699300016133&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10490937&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F08%2F2020.08.11.245944.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000083169000004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [80].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference [80] in text" id="ref-80">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.80">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Richard
                                  </span>
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Erik J
                                  </span>
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bradley
                                  </span>
                                  <span class="cit-name-surname">
                                   Voytek
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Inferring synaptic excitation/inhibition balance from field potentials
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuroimage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  158
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  70
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  78
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D158%26rft.spage%253D70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [81].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference [81] in text" id="ref-81">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.08.11.245944v3.81">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eric-Jan
                                  </span>
                                  <span class="cit-name-surname">
                                   Wagenmakers
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Simon
                                  </span>
                                  <span class="cit-name-surname">
                                   Farrell
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Roger
                                  </span>
                                  <span class="cit-name-surname">
                                   Ratcliff
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Estimation and interpretation of 1/f α noise in human cognition
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Psychonomic bulletin & review
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  579
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  615
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPsychonomic%2Bbulletin%2B%2526%2Breview%26rft.volume%253D11%26rft.spage%253D579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [82].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference [82] in text" id="ref-82">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.08.11.245944v3.82">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Morehead
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alex
                                  </span>
                                  <span class="cit-name-surname">
                                   Hagen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  A python package for approximate bayesian computation
                                 </span>
                                 .
                                 <a href="https://github.com/rcmorehead/simpleabc">
                                  https://github.com/rcmorehead/simpleabc
                                 </a>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [83].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.08.11.245944v3.83">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ryan N
                                  </span>
                                  <span class="cit-name-surname">
                                   Gutenkunst
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joshua J
                                  </span>
                                  <span class="cit-name-surname">
                                   Waterfall
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fergal P
                                  </span>
                                  <span class="cit-name-surname">
                                   Casey
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kevin S
                                  </span>
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher R
                                  </span>
                                  <span class="cit-name-surname">
                                   Myers
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James P
                                  </span>
                                  <span class="cit-name-surname">
                                   Sethna
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Universally sloppy parameter sensitivities in systems biology models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e189
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D3%26rft.spage%253De189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.08.11.245944v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.01.10.475695v2" title="unc-37/Groucho and lsy-22/AES repress Wnt target genes in C. elegans asymmetric cell divisions" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.02.08.479500v1" title="Proteomics profiling reveals regulation of immune response to Salmonella Typhimurium infection in mice" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted February 08, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.08.11.245944v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/02/08/2020.08.11.245944.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.08.11.245944v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.08.11.245944v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   A flexible Bayesian framework for unbiased estimation of timescales
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2390714"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-8_UnCS7TF7XUKD9RGyPFD6so9gKESlQZL4WvBv7wX0I"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484475932"/>
                                   <input type="hidden" name="captcha_token" value="226b964ee65cf4e72c1b91b1bfd05000"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2390714--3" data-node-nid="2390714" id="node-2390714--41418961595" data-pisa="biorxiv;2020.08.11.245944v3" data-pisa-master="biorxiv;2020.08.11.245944" data-seqnum="2390714" data-apath="/biorxiv/early/2022/02/08/2020.08.11.245944.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 A flexible Bayesian framework for unbiased estimation of timescales
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Roxana
                                  </span>
                                  <span class="nlm-surname">
                                   Zeraati
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Tatiana A.
                                  </span>
                                  <span class="nlm-surname">
                                   Engel
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Anna
                                  </span>
                                  <span class="nlm-surname">
                                   Levina
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.08.11.245944;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.08.11.245944
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.08.11.245944v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&title=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&title=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&text=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&t=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&title=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&title=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&title=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2390714--5" data-node-nid="2390714" id="citation-node-2390714--6746859600" data-pisa="biorxiv;2020.08.11.245944v3" data-pisa-master="biorxiv;2020.08.11.245944" data-seqnum="2390714" data-apath="/biorxiv/early/2022/02/08/2020.08.11.245944.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     A flexible Bayesian framework for unbiased estimation of timescales
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Roxana
                                      </span>
                                      <span class="nlm-surname">
                                       Zeraati
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Tatiana A.
                                      </span>
                                      <span class="nlm-surname">
                                       Engel
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Anna
                                      </span>
                                      <span class="nlm-surname">
                                       Levina
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.08.11.245944;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.08.11.245944
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Zeraati%2C+Roxana&rft.au=Engel%2C+Tatiana+A.&rft.au=Levina%2C+Anna">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2390714/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2390714/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2390714/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2390714/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2390714/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2390714/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2390714/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2390714/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2390714/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2390714/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2390714/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2390714/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&count=horizontal&via=&text=A%20flexible%20Bayesian%20framework%20for%20unbiased%20estimation%20of%20timescales&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.08.11.245944v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.08.11.245944v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
