<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2021.09.26.21263968v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2021.09.26.21263968v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2021.09.26.21263968v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.medrxiv.org/content/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/medrxiv/early/2022/07/26/2021.09.26.21263968.atom"/>
  <meta name="HW.pisa" content="medrxiv;2021.09.26.21263968v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Generation of realistic synthetic data using multimodal neural ordinary differential equations"/>
  <meta name="DC.Identifier" content="10.1101/2021.09.26.21263968"/>
  <meta name="DC.Date" content="2022-07-26"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Individual organizations, such as hospitals, pharmaceutical companies and health insurance providers are currently limited in their ability to collect data that is fully representative of a disease population. This can in turn negatively impact the generalization ability of statistical models and scientific insights. However, sharing data across different organizations is highly restricted by legal regulations. While federated data access concepts exist, they are technically and organizationally difficult to realize. An alternative approach would be to exchange synthetic patient data instead. In this work, we introduce the Multimodal Neural Ordinary Differential Equations (MultiNODEs), a hybrid, multimodal AI approach, which allows for generating highly realistic synthetic patient trajectories on a continuous time scale, hence enabling smooth interpolation and extrapolation of clinical studies. Our proposed method can integrate both static and longitudinal data, and implicitly handles missing values. We demonstrate the capabilities of MultiNODEs by applying them to real patient-level data from two independent clinical studies and simulated epidemiological data of an infectious disease.

### Competing Interest Statement

The authors have declared no competing interest.

### Clinical Trial

We use data from two clinical studies: Parkinson Progression Marker Initiative [NCT01141023][1] We use data of the national alzheimers coordinating center of this paper <https://pubmed.ncbi.nlm.nih.gov/30376508/>

### Funding Statement

This project has received funding from the European Unions Horizon 2020 research and innovation programme under grant agreement No. 826421, TheVirtualBrain-Cloud and from the Deutsche Forschungsgemeinschaft (DFG) funded project NFDI4Health (project number 442326535).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Our research is in the area of health informatics. In our work we use freely available data for the purpose of research and the ethics committee of the Fraunhofer-Gesellschaft allows us to work with patient data

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data used in our manuscript are freely available for the purpose of (non-profit) research

<https://www.ppmi-info.org/access-data-specimens/download-data/> 

<https://naccdata.org/>

 [1]: /lookup/external-ref?link_type=CLINTRIALGOV&access_num=NCT01141023&atom=%2Fmedrxiv%2Fearly%2F2022%2F07%2F26%2F2021.09.26.21263968.atom"/>
  <meta name="DC.Contributor" content="Philipp Wendland"/>
  <meta name="DC.Contributor" content="Colin Birkenbihl"/>
  <meta name="DC.Contributor" content="Marc Gomez-Freixa"/>
  <meta name="DC.Contributor" content="Meemansa Sood"/>
  <meta name="DC.Contributor" content="Maik Kschischo"/>
  <meta name="DC.Contributor" content="Holger Fröhlich"/>
  <meta name="article:published_time" content="2022-07-26"/>
  <meta name="citation_title" content="Generation of realistic synthetic data using multimodal neural ordinary differential equations"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Individual organizations, such as hospitals, pharmaceutical companies and health insurance providers are currently limited in their ability to collect data that is fully representative of a disease population. This can in turn negatively impact the generalization ability of statistical models and scientific insights. However, sharing data across different organizations is highly restricted by legal regulations. While federated data access concepts exist, they are technically and organizationally difficult to realize. An alternative approach would be to exchange synthetic patient data instead. In this work, we introduce the Multimodal Neural Ordinary Differential Equations (MultiNODEs), a hybrid, multimodal AI approach, which allows for generating highly realistic synthetic patient trajectories on a continuous time scale, hence enabling smooth interpolation and extrapolation of clinical studies. Our proposed method can integrate both static and longitudinal data, and implicitly handles missing values. We demonstrate the capabilities of MultiNODEs by applying them to real patient-level data from two independent clinical studies and simulated epidemiological data of an infectious disease.</p>"/>
  <meta name="citation_journal_title" content="medRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="medrxiv;2021.09.26.21263968v3"/>
  <meta name="citation_id" content="2021.09.26.21263968v3"/>
  <meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2021.09.26.21263968v3"/>
  <meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2021.09.26.21263968v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2021.09.26.21263968v3.full"/>
  <meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2022/07/26/2021.09.26.21263968.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2021.09.26.21263968"/>
  <meta name="citation_num_pages" content="32"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_firstpage" content="2021.09.26.21263968"/>
  <meta name="citation_author" content="Philipp Wendland"/>
  <meta name="citation_author_institution" content="Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)"/>
  <meta name="citation_author_institution" content="Department of Mathematics and Technology, University of Applied Sciences Koblenz"/>
  <meta name="citation_author" content="Colin Birkenbihl"/>
  <meta name="citation_author_institution" content="Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)"/>
  <meta name="citation_author_institution" content="Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn"/>
  <meta name="citation_author" content="Marc Gomez-Freixa"/>
  <meta name="citation_author_institution" content="Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn"/>
  <meta name="citation_author" content="Meemansa Sood"/>
  <meta name="citation_author_institution" content="Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)"/>
  <meta name="citation_author_institution" content="Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn"/>
  <meta name="citation_author" content="Maik Kschischo"/>
  <meta name="citation_author_institution" content="Department of Mathematics and Technology, University of Applied Sciences Koblenz"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3977-9982"/>
  <meta name="citation_author" content="Holger Fröhlich"/>
  <meta name="citation_author_institution" content="Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)"/>
  <meta name="citation_author_institution" content="Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn"/>
  <meta name="citation_author_email" content="holger.froehlich@scai.fraunhofer.de"/>
  <meta name="citation_reference" content="Fröhlich, H. et al. From hype to reality: data science enabling personalized medicine. BMC Med. 16, 150 (2018)."/>
  <meta name="citation_reference" content="Birkenbihl, C., Salimi, Y., Fröhlich, H., for the Japanese Alzheimer’s Disease Neuroimaging Initiative, & the Alzheimer’s Disease Neuroimaging Initiative. Unraveling the heterogeneity in Alzheimer’s disease progression across multiple cohorts and the implications for data-driven disease modeling. Alzheimers Dement. 18, 251–261 (2022)."/>
  <meta name="citation_reference" content="Birkenbihl, C. et al. Differences in cohort study data affect external validation of artificial intelligence models for predictive diagnostics of dementia - lessons for translation into clinical practice. EPMA J. 11, 367–376 (2020)."/>
  <meta name="citation_reference" content="Gootjes-Dreesbach, L., Sood, M., Sahay, A., Hofmann-Apitius, M. & Fröhlich, H. Variational Autoencoder Modular Bayesian Networks for Simulation of Heterogeneous Clinical Study Data. Front. Big Data 3, 16 (2020)."/>
  <meta name="citation_reference" content="Sood, M. et al. Realistic simulation of virtual multi-scale, multi-modal patient trajectories using Bayesian networks and sparse auto-encoders. Sci. Rep. 10, 10971 (2020)."/>
  <meta name="citation_reference" content="Chen, R. J., Lu, M. Y., Chen, T. Y., Williamson, D. F. K. & Mahmood, F. Synthetic data in machine learning for medicine and healthcare. Nat. Biomed. Eng. 5, 493–497 (2021)."/>
  <meta name="citation_reference" content="Thorlund, K., Dron, L., Park, J. J. & Mills, E. J. Synthetic and External Controls in Clinical Trials – A Primer for Researchers. Clin. Epidemiol. Volume 12, 457–467 (2020)."/>
  <meta name="citation_reference" content="Lei, Y. et al. MRI-only based synthetic CT generation using dense cycle consistent generative adversarial networks. Med. Phys. 46, 3565–3581 (2019)."/>
  <meta name="citation_reference" content="Yang, G. et al. DAGAN: Deep De-Aliasing Generative Adversarial Networks for Fast Compressed Sensing MRI Reconstruction. IEEE Trans. Med. Imaging 37, 1310–1321 (2018)."/>
  <meta name="citation_reference" content="citation_title=in Advances in Neural Information Processing Systems;citation_volume=31;citation_year=2018"/>
  <meta name="citation_reference" content="Marek, K. et al. The Parkinson Progression Marker Initiative (PPMI). Prog. Neurobiol. 95, 629–635 (2011)."/>
  <meta name="citation_reference" content="Besser, L. et al. Version 3 of the National Alzheimer’s Coordinating Center’s Uniform Data Set. Alzheimer Dis. Assoc. Disord. 32, 351–358 (2018)."/>
  <meta name="citation_reference" content="Nazabal, A., Olmos, P. M., Ghahramani, Z. & Valera, I. Handling Incomplete Heterogeneous Data using VAEs. ArXiv180703653 Cs Stat (2020)."/>
  <meta name="citation_reference" content="de Jong, J. et al. Deep learning for clustering of multivariate clinical patient trajectories with missing values. GigaScience 8, giz134 (2019)."/>
  <meta name="citation_reference" content="Kingma, D. P. & Welling, M. Auto-Encoding Variational Bayes. at http://arxiv.org/abs/1312.6114 (2014)."/>
  <meta name="citation_reference" content="He, K., Zhang, X., Ren, S. & Sun, J. Deep Residual Learning for Image Recognition. in 2016 IEEE Conference on Computer Vision and Pattern Recognition (CVPR) 770–778 (IEEE, 2016). doi:10.1109/CVPR.2016.90."/>
  <meta name="citation_reference" content="Hochreiter, S. & Schmidhuber, J. Long Short-Term Memory. Neural Comput. 9, 1735–1780 (1997)."/>
  <meta name="citation_reference" content="citation_title=in Advances in Neural Information Processing Systems;citation_volume=32;citation_year=2019"/>
  <meta name="citation_reference" content="Lin, Z., Jain, A., Wang, C., Fanti, G. & Sekar, V. Using GANs for Sharing Networked Time Series Data: Challenges, Initial Promise, and Open Questions. in Proceedings of the ACM Internet Measurement Conference 464–483 (ACM, 2020). doi:10.1145/3419394.3423643."/>
  <meta name="citation_reference" content="Bae, H., Jung, D., Choi, H.-S. & Yoon, S. AnomiGAN: Generative Adversarial Networks for Anonymizing Private Medical Data. Pac. Symp. Biocomput. Pac. Symp. Biocomput. 25, 563–574 (2020)."/>
  <meta name="citation_reference" content="Jordon, J. & Yoon, J. PATE-GAN: Generating synthetic data with differential privacy guarantees. in International Conference on Learning Representations 21 (2019)."/>
  <meta name="citation_reference" content="Beaulieu-Jones, B. K. et al. Privacy-Preserving Generative Deep Neural Networks Support Clinical Data Sharing. Circ. Cardiovasc. Qual. Outcomes 12, e005122 (2019)."/>
  <meta name="citation_reference" content="Goncalves, A. et al. Generation and evaluation of synthetic patient data. BMC Med. Res. Methodol. 20, 108 (2020)."/>
  <meta name="citation_reference" content="Park, N. et al. Data Synthesis based on Generative Adversarial Networks. Proc. VLDB Endow. 11, 1071–1083 (2018)."/>
  <meta name="twitter:title" content="Generation of realistic synthetic data using multimodal neural ordinary differential equations"/>
  <meta name="twitter:site" content="@medrxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="twitter:description" content="Individual organizations, such as hospitals, pharmaceutical companies and health insurance providers are currently limited in their ability to collect data that is fully representative of a disease population. This can in turn negatively impact the generalization ability of statistical models and scientific insights. However, sharing data across different organizations is highly restricted by legal regulations. While federated data access concepts exist, they are technically and organizationally difficult to realize. An alternative approach would be to exchange synthetic patient data instead. In this work, we introduce the Multimodal Neural Ordinary Differential Equations (MultiNODEs), a hybrid, multimodal AI approach, which allows for generating highly realistic synthetic patient trajectories on a continuous time scale, hence enabling smooth interpolation and extrapolation of clinical studies. Our proposed method can integrate both static and longitudinal data, and implicitly handles missing values. We demonstrate the capabilities of MultiNODEs by applying them to real patient-level data from two independent clinical studies and simulated epidemiological data of an infectious disease.

### Competing Interest Statement

The authors have declared no competing interest.

### Clinical Trial

We use data from two clinical studies: Parkinson Progression Marker Initiative [NCT01141023][1] We use data of the national alzheimers coordinating center of this paper <https://pubmed.ncbi.nlm.nih.gov/30376508/>

### Funding Statement

This project has received funding from the European Unions Horizon 2020 research and innovation programme under grant agreement No. 826421, TheVirtualBrain-Cloud and from the Deutsche Forschungsgemeinschaft (DFG) funded project NFDI4Health (project number 442326535).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Our research is in the area of health informatics. In our work we use freely available data for the purpose of research and the ethics committee of the Fraunhofer-Gesellschaft allows us to work with patient data

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data used in our manuscript are freely available for the purpose of (non-profit) research

<https://www.ppmi-info.org/access-data-specimens/download-data/> 

<https://naccdata.org/>

 [1]: /lookup/external-ref?link_type=CLINTRIALGOV&access_num=NCT01141023&atom=%2Fmedrxiv%2Fearly%2F2022%2F07%2F26%2F2021.09.26.21263968.atom"/>
  <meta name="og-title" property="og:title" content="Generation of realistic synthetic data using multimodal neural ordinary differential equations"/>
  <meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2021.09.26.21263968v3"/>
  <meta name="og-site-name" property="og:site_name" content="medRxiv"/>
  <meta name="og-description" property="og:description" content="Individual organizations, such as hospitals, pharmaceutical companies and health insurance providers are currently limited in their ability to collect data that is fully representative of a disease population. This can in turn negatively impact the generalization ability of statistical models and scientific insights. However, sharing data across different organizations is highly restricted by legal regulations. While federated data access concepts exist, they are technically and organizationally difficult to realize. An alternative approach would be to exchange synthetic patient data instead. In this work, we introduce the Multimodal Neural Ordinary Differential Equations (MultiNODEs), a hybrid, multimodal AI approach, which allows for generating highly realistic synthetic patient trajectories on a continuous time scale, hence enabling smooth interpolation and extrapolation of clinical studies. Our proposed method can integrate both static and longitudinal data, and implicitly handles missing values. We demonstrate the capabilities of MultiNODEs by applying them to real patient-level data from two independent clinical studies and simulated epidemiological data of an infectious disease.

### Competing Interest Statement

The authors have declared no competing interest.

### Clinical Trial

We use data from two clinical studies: Parkinson Progression Marker Initiative [NCT01141023][1] We use data of the national alzheimers coordinating center of this paper <https://pubmed.ncbi.nlm.nih.gov/30376508/>

### Funding Statement

This project has received funding from the European Unions Horizon 2020 research and innovation programme under grant agreement No. 826421, TheVirtualBrain-Cloud and from the Deutsche Forschungsgemeinschaft (DFG) funded project NFDI4Health (project number 442326535).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Our research is in the area of health informatics. In our work we use freely available data for the purpose of research and the ethics committee of the Fraunhofer-Gesellschaft allows us to work with patient data

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data used in our manuscript are freely available for the purpose of (non-profit) research

<https://www.ppmi-info.org/access-data-specimens/download-data/> 

<https://naccdata.org/>

 [1]: /lookup/external-ref?link_type=CLINTRIALGOV&access_num=NCT01141023&atom=%2Fmedrxiv%2Fearly%2F2022%2F07%2F26%2F2021.09.26.21263968.atom"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="citation_date" content="2022-07-26"/>
  <meta name="description" content="medRxiv - The Preprint Server for Health Sciences"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2021.09.26.21263968v3"/>
  <link rel="shortlink" href="https://www.medrxiv.org/node/542641"/>
  <title>
   Generation of realistic synthetic data using multimodal neural ordinary differential equations | medRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__MF4P_i8lKWP7DQR7GmjDAVTRBKLnzxvs1Dvton-Awi0__yi5OflfckG_G8-Li4dDQQ3wO9cGIcdPx_YQ7ZZDuj5w__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__5mxrBK4DuyvC2MR5irx_TzVn9c8vr4vIjmBXjunHfxE__rtQnXv8GOc-AsynbhzSlQBD_2lVxAkB6XEFUCK72qXc__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all"/>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lStvwpoWQr9IEeKPn0rs38W_ArymdWEBbpa7AXn-XZ0"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"medrxiv;2021.09.26.21263968v3"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"medrxiv;2021.09.26.21263968v3"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"medrxiv;2021.09.26.21263968v3"}],"nid":"542641","apath":"\/medrxiv\/early\/2022\/07\/26\/2021.09.26.21263968.atom","pisa":"medrxiv;2021.09.26.21263968v3","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2021.09.26.21263968v3","title":"Generation of realistic synthetic data using multimodal neural ordinary differential equations","identifier":"node\/542641"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MiVwfdgpC6J2cowqpB2BfRWtnLkgoVRUO\\\\\\\/pAjeBnX+ZTZYi6u3ITn\\\\\\\/6oFT1KuGfUXJtbOFkbv5TN+AXky\\\\\\\/iRDRz3micPETumfGXuPQtr6XFjmFBKEWuVuBjZNK+2aILg7ilOtcd8XoLNpxzsyqk+QbnbLTphN5iqHj3oeITT9XMOOA+0kkU6q38UALfpNRoeMwpGqDHYyMMbmcVHVYba8bJ0Cp0qPeVzsYccVS4jVkf\\\\\\\/Ev27ruPz4aYYqa4uyoDl+7OFJtjKlemBk1q5GKMnuEy4qCNE8DhmH\\\\\\\/p0PbdcVkMUazvevsXFL1UBukgmcdo5+kn8QTKPzr+8+XqFrv1Yt8yEx1jrQsQbN+4IskhSP4AZbFs8\\\\\\\/C9jLnBHO2OSB4bRsgKIjv9EVNocFoIMxssp7QsVwCI8fzfe47ejuaiCtCZc1vKCSrVAM14E8cnhT3fKSq8b+6MDZBDXN0oP5pbDXN0kYLyue\\\\\\\/qSke2esGTXnjQWPnBAPuP+jtA9XELnKAF+W5yCR8lE270H2T20w3o6hoiaRQyrH2v0da8aUUnULIKyW2jrVMaYBmTXp8\\\\\\\/MPAKJdvEOPIoC8ROpY9T1xRjQR6CEO9KlPG3X1wWUfEtaAolIWsTlTGUl++5ADeTQWNbbqKUi21+mLM9bhtuwFSlRm1042U+IZeqO4iNKlcVFdE+S4f60\\\\\\\/IZSUR0wLLr51PQvp8283EW\\\\\\\/GljESlN3I9\\\\\\\/k9LKd5R7HA0uizmurj4tOhD9ARN5Ix\\\\\\\/WkrN+sO1+LIZ\\\\\\\/DqzNwygD\\\\\\\/3nsLsAiMsRmp22WfZRmQT2qoQG17EEa\\\\\\\/udA6Y80li49DPQLNSEmH\\\\\\\/8e9hBBH5LyRivEbptJdRRAMkAfIa2wGXcXSloLhZonxKtWllD93ev23r1no+8RSDEDrWhh3oxFhcdBSw5Mw9\\\\\\\/spHWRUmrRkFb5HkGBKdZ8B6rrznLSSB3zbHGr1fPuLfp5dvzd1M5YZ+1u6Esr3M8uT1rIl0t4jGQ0OJZDwjf90PR7g4wdZyB4UQ\\\\\\\/Bs\\\\\\\/BVeDQGkkDsmz1d7jVPjSrlj9DKtri2W+TClw9Vy4Vlj1qgNTkyXNfnqGwSLXo6hcXogNzqv1+aSEmylEr8xUJ6CdocuVDgdj\\\\\\\/7myGM+GEn86Y5MH\\\\\\\/Zapvpn\\\\\\\/0FYIIgAXMj553WEaz+mA6Z9Ww3EvkRUoWcQFGo8yO9oek0r2PX7KhTYzdFSIvKod9DdFpQTG84SMvJj0AzFNKQgSm+\\\\\\\/ZyhIhcCZTFEndvXXxA3gNawpUDxHvwBG\\\\\\\/uK9WbzIbnV0++RMl52jCgdyXj\\\\\\\/XK711KRu5xe5B9QrIH4frMtaA2euM2OsGrXGPko3+7oJ9Mg5MN0GafzfLZM+eIA2Fh\\\\\\\/v+BKg66ja\\\\\\\/gXbRIeLD2v2KCG0e4pFZTdA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022xY0mvsFVcCUIRe0hkUWl3w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002294b864d73ad5bc68afab43d07e9b2d471db541c0fc176a575f64c7a0007720bb\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2021.09.26.21263968v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-542641 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2021.09.26.21263968v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_887581285">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_887581285" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_696031892" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-sxXKyxAwksSsmzdTTnDHbZQT9ISUn3qx9xBiaP8QxQs"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node542641" data-node-nid="542641" id="node-542641--2860888483" data-pisa="medrxiv;2021.09.26.21263968v3" data-pisa-master="medrxiv;2021.09.26.21263968" data-apath="/medrxiv/early/2022/07/26/2021.09.26.21263968.atom" data-hw-author-tooltip-instance="">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip">
                   <h1 class="highwire-cite-title" id="page-title">
                    Generation of realistic synthetic data using multimodal neural ordinary differential equations
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Philipp
                      </span>
                      <span class="nlm-surname">
                       Wendland
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Colin
                      </span>
                      <span class="nlm-surname">
                       Birkenbihl
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Marc
                      </span>
                      <span class="nlm-surname">
                       Gomez-Freixa
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Meemansa
                      </span>
                      <span class="nlm-surname">
                       Sood
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0003-3977-9982" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Maik
                      </span>
                      <span class="nlm-surname">
                       Kschischo
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Holger
                      </span>
                      <span class="nlm-surname">
                       Fröhlich
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2021.09.26.21263968
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-542641--2860888483" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Philipp Wendland
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)
                       </span>
                       , Sankt Augustin 53754,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Mathematics and Technology, University of Applied Sciences Koblenz
                       </span>
                       , Remagen 53424,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Philipp%2BWendland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wendland%20P&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3APhilipp%2BWendland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Colin Birkenbihl
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)
                       </span>
                       , Sankt Augustin 53754,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn
                       </span>
                       , Bonn 53115,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Colin%2BBirkenbihl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Birkenbihl%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AColin%2BBirkenbihl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Marc Gomez-Freixa
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn
                       </span>
                       , Bonn 53115,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Marc%2BGomez-Freixa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gomez-Freixa%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMarc%2BGomez-Freixa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Meemansa Sood
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)
                       </span>
                       , Sankt Augustin 53754,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn
                       </span>
                       , Bonn 53115,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Meemansa%2BSood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sood%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMeemansa%2BSood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Maik Kschischo
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Mathematics and Technology, University of Applied Sciences Koblenz
                       </span>
                       , Remagen 53424,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Maik%2BKschischo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kschischo%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMaik%2BKschischo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3977-9982" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Maik Kschischo
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Holger Fröhlich
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Bioinformatics, Fraunhofer Institute for Algorithms and Scientific Computing (SCAI)
                       </span>
                       , Sankt Augustin 53754,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Bonn-Aachen International Center for IT, Rheinische Friedrich-Wilhelms-Universität Bonn
                       </span>
                       , Bonn 53115,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Holger%2BFr%C3%B6hlich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Fr%C3%B6hlich%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AHolger%2BFr%25C3%25B6hlich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:holger.froehlich@scai.fraunhofer.de" class="" data-icon-position="" data-hide-link-title="0">
                        holger.froehlich@scai.fraunhofer.de
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2021.09.26.21263968v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:542641" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:542641/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.26.21263968v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:542641" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:542641/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.26.21263968v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:542641" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:542641/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.26.21263968v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:542641" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:542641/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.26.21263968v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:542641" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:542641/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.26.21263968v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:542641" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:542641/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2021.09.26.21263968v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:542641" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:542641/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             Individual organizations, such as hospitals, pharmaceutical companies and health insurance providers are currently limited in their ability to collect data that is fully representative of a disease population. This can in turn negatively impact the generalization ability of statistical models and scientific insights. However, sharing data across different organizations is highly restricted by legal regulations. While federated data access concepts exist, they are technically and organizationally difficult to realize. An alternative approach would be to exchange synthetic patient data instead. In this work, we introduce the Multimodal Neural Ordinary Differential Equations (MultiNODEs), a hybrid, multimodal AI approach, which allows for generating highly realistic synthetic patient trajectories on a continuous time scale, hence enabling smooth interpolation and extrapolation of clinical studies. Our proposed method can integrate both static and longitudinal data, and implicitly handles missing values. We demonstrate the capabilities of MultiNODEs by applying them to real patient-level data from two independent clinical studies and simulated epidemiological data of an infectious disease.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-18">
                             Patient-level data build the foundation for a plethora of healthcare research endeavors such as drug discovery, clinical trials, biomarker discovery, and precision medicine [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ]. Collecting such data is extremely time-consuming and cost-intensive, and additionally access-restricted by ethical and legal regulations in most countries. Individual organizations, such as hospitals, pharmaceutical companies and health insurance providers are currently limited in their ability to collect data that is fully representative of a disease population. This issue is especially pronounced in clinical studies, where patients are usually recruited based on predefined inclusion and exclusion criteria which introduce cohort-specific statistical biases [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ]. These biases, in turn, can negatively impact the generalization ability of machine learning models, since the usual i.i.d. assumption is violated [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. A naïve idea to counteract this issue might be to build up large data repositories pooling diverse clinical studies from several organizations. However, here, a major obstacle is that sharing patient-level data across different organizations is exceedingly difficult due to legal restrictions, as formulated, for example, in the General Data Protection Rule (GDPR) of the European Union.
                            </p>
                            <p id="p-19">
                             The idea we propagate in this paper is to learn a continuous-time generative machine learning model from clinical study data. Given the distribution of the real training data was appropriately learned by such a model, the generated synthetic datasets maintain the real data signals, such as variable interdependencies and time-dependent trajectories. Furthermore, these synthetic datasets can overcome crucial limitations of their real counterparts like missing values or irregular assessment intervals, hence opening the opportunity to make at least subsets of variables from different studies statistically comparable. A further strong motivation for generating synthetic datasets is the aim to use the generated data as an anonymized version of its real-world counterpart and thereby mitigate the increased restrictions for sharing human data [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ,
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. However, synthetic patient-level datasets open opportunities that reach far beyond data sharing. For example, trained generative models could be used for synthesizing control arms for clinical trials based on data from previously conducted trials, or from real-world clinical routine data [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ]. This helps addressing major ethical concerns in disease areas, such as cancer, where it is impossible to leave patients untreated. Both, the American Food and Drug Administration (FDA) and the European Medicines Agency (EMA) have recognized this issue and taken initiatives to allow for synthetic control arms [
                             <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ].
                            </p>
                            <p id="p-20">
                             Over the last years, generative models (mostly Generative Adversarial Networks - GANs) have found notable success, mostly in the medical imaging domain [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             -
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ]. However, GANs are often found to show a collapse to the statistical mode of a distribution, which raises concerns regarding coverage of the real patient distribution by synthetic data. Moreover, these methods are not necessarily suited to cope with the complex nature of clinical data collected in observational, longitudinal cohort studies, which is the main focus of our work: In addition to the previously mentioned issue of irregular measurement frequencies and missing values not at random (for example due to participant drop-out), clinical studies often comprise several modalities combining time-dependent variables (e.g., measures of disease severity) and static information (e.g., biological sex). One approach specifically designed for the joint modeling and generation of multimodal, time-dependent and static patient-level data containing missing values are the recently introduced Variational Autoencoder Modular Bayesian networks (VAMBN) [
                             <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. However, VAMBN only operates on a discrete time scale while relevant clinical indicators such as, for example, disease progression expressed through cognitive decline or rising inflammatory markers, are intrinsically time continuous. Recently, Neural Ordinary Differential Equations (NODEs) have been introduced as a hybrid approach fusing neural networks and ordinary differential equations (ODE) [
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ]. While NODEs are time continuous and thus enable smooth interpolation between observed data points and extrapolation beyond the observations in the data, they are not able to integrate static variables.
                            </p>
                            <p id="p-21">
                             In this work, we present the Multimodal Neural Ordinary Differential Equations (MultiNODEs) as an extension of the NODEs. MultiNODEs allow learning a generative model from multimodal longitudinal and static data that may contain missing values not at random. To demonstrate MultiNODEs’ generative capabilities, we applied the model to clinical, patient-level data from an observational Parkinson’s disease (PD) cohort study (the Parkinson’s Progression Markers Initiative, PPMI [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]) and, additionally, a longitudinal Alzheimer’s disease (AD) data collection (National Alzheimer’s Coordination Center, NACC [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ]). We compared the generated trajectories and correlation structure with the real counterpart. In this context, we additionally evaluated MultiNODEs’ performance against the previously published VAMBN approach. Furthermore, we assessed MultiNODEs’ interpolation and extrapolation performance. Finally, we investigated the influence of sample size, noisiness of the data, and longitudinal assessment density on the training of MultiNODEs in a systematic benchmark on data simulated from a mathematical model well-known in the epidemiology field.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Conceptual Introduction of the MultiNODEs
                             </h3>
                             <p id="p-22">
                              MultiNODEs represent an extension of the original NODEs framework [
                              <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                              ] that overcomes the limitations of its predecessor such that an application to incomplete datasets consisting of both static and time-dependent variables becomes feasible. Conceptually, MultiNODEs build on three key components
                              <strong>
                               (
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                               )
                              </strong>
                              : 1) latent NODEs, 2) a variational autoencoder (more specifically a Heterogenous Incomplete Variational Autoencoder - HI-VAE - designed to handle multimodal data with missing values [
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ]), and 3) an implicit imputation layer [
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ]. The latent NODEs enable the learning and subsequent generation of continuous longitudinal variable trajectories. The longitudinal properties of the initial condition (i.e., the starting point for the ODE system solver of the latent NODEs) are defined by the output of a recurrent variational encoder which embeds the longitudinal input data into a latent space
                              <strong>
                               (
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                               orange box)
                              </strong>
                              . To allow for an additional influence of static variables on the estimation of the longitudinal variable trajectories, the second component, a HI-VAE, is introduced
                              <strong>
                               (
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                               blue box)
                              </strong>
                              . This component transforms the static information into a distinct latent space and the resulting embedding is used to augment the latent starting condition of the NODEs by concatenating the static variable embedding and the latent representation of the longitudinal variables
                              <strong>
                               (
                               <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                               ‘augmentation’)
                              </strong>
                              . The HI-VAE component itself holds generative properties and conducts the synthesis of the static variables when MultiNODEs are applied in a generative setting. Conclusively, MultiNODEs integrate static variables (e.g., biological sex or genotype information) both to inform the learning of longitudinal trajectories, and in the generative process. Finally, to mitigate the original NODEs’ incapability of dealing with missing values, we introduced the imputation layer which implicitly replaces missing values during model training with learned estimates
                              <strong>
                               (
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                               green box)
                              </strong>
                              . For further details on the model architecture, training, and hyperparameter optimization, we refer to the Method section and Supplements, respectively.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <span class="caption-title">
                                Conceptual framework of MultiNODEs.
                               </span>
                               <p id="p-23" class="first-child">
                                Blue box: HI-VAE for the encoding and generation of static variables. Orange box: NODEs that learn and generate longitudinal trajectories. Green box: The imputation layer which can handle missing data implicitly during model training.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Synthetic Data Generation using MultiNODEs
                             </h3>
                             <p id="p-24">
                              Generating synthetic data using MultiNODEs starts by randomly sampling a latent representation for both the static and longitudinal variables, respectively. The longitudinal variables in data space are then generated by first constructing the initial conditions of the latent ODE system (i.e., concatenating the static latent representation to the longitudinal one), followed by solving the ODE system given these initial conditions, and finally by decoding the result into data space. The static variables are generated by directly transforming their sampled latent representation into data space using the HI-VAE decoder.
                             </p>
                             <p id="p-25">
                              MultiNODEs support two different approaches for the initial sampling of the latent representations, namely sampling from the prior distribution employed during model training and sampling from the learned posterior distribution of the input data.
                             </p>
                             <p id="p-26">
                              During the posterior sampling procedure, the reparameterization trick [
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ] is applied to draw a latent representation from the posterior distribution learned from the training data. The amount of noise added in this process can be tuned, whereas greater noise will lead to a wider spread of the generated marginal distributions of the synthetic data. Alternatively, the latent representations can be sampled from the prior distributions imposed on the latent space during variational model training. We ensure statistical dependence between static and longitudinal variables by drawing their values from a Bayesian network that connects both latent representations such that the longitudinal variables are conditionally dependent on the static variables. More detailed descriptions of both generation procedures are provided in the Method section.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Application cases: Parkinson’s disease and Alzheimer’s disease
                             </h3>
                             <p id="p-27">
                              We applied MultiNODEs to longitudinal, multimodal data from two independent clinical datasets with the goal of generating realistic synthetic datasets that maintain the real data properties. Details about the data pre-processing steps are described in the Supplementary Material.
                             </p>
                             <p id="p-28">
                              The first dataset was the Parkinson’s Progression Markers Initiative (PPMI), an observational clinical study containing 354 de-novo PD patients who participated in a range of clinical, neurological, and demographic assessments which form the variables of the dataset. In total, a set of 25 longitudinal and 43 static variables was investigated.
                             </p>
                             <p id="p-29">
                              Furthermore, as a second example, we applied MultiNODEs to longitudinal, multimodal data from the National Alzheimer’s Coordinating Center (NACC). NACC is a database storing patient-level AD data collected across multiple memory clinics. After preprocessing, the dataset used in this study contained 2284 patients and a set of 3 longitudinal and 4 static variables was investigated.
                             </p>
                             <p id="p-30">
                              In the following sections, we will focus on the results achieved on the PPMI data and refer to the equivalent experiments based on the NACC data which are presented in the Supplementary Material.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              MultiNODEs generate realistic synthetic patient-level datasets
                             </h3>
                             <p id="p-31">
                              We applied prior as well as posterior sampling for comparison purposes. With each method, we generated the same number of synthetic patients as encountered in the real dataset to allow for a fair comparison. To assess whether the generated data followed the real data characteristics, we conducted thorough comparisons of the marginal distributions using qualitative, visual assessments and further, quantitatively compared the Jensen-Shannon divergence (JS-divergence) between the generated data and real distributions. The JS-divergence is bound between 0 and 1 with 0 indicating equal distributions. Additionally, we investigated the underlying correlation structure of the measured variables. Finally, we trained a machine learning classifier (Random Forest) that evaluated whether real and synthetic patients showed similar clinical characteristics when compared to real healthy control individuals from their respective studies. Across all these aspects, we evaluated MultiNODEs’ performance in comparison to the previously published VAMBN approach [
                              <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ].
                             </p>
                             <p id="p-32">
                              The synthetic data generated using MultiNODE generally exhibited marginal distributions which bore high similarity to their corresponding real counterparts
                              <strong>
                               (
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Figure 2
                               </a>
                               , Supplementary Table 1, Supplementary Figure 1
                              </strong>
                              ; equivalent figures for the NACC data are presented in
                              <strong>
                               Supplementary Figure 4.)
                              </strong>
                              . The average JS-divergences between the real and synthetic distributions calculated across all variables and timepoints amounted to 0.018 ± 0.015 and 0.011 ± 0.009 for the PPMI data generated from the prior and posterior, respectively. For NACC the average JS-divergence was 0.071 ± 0.055 and 0.029 ± 0.031 for prior and posterior sampling, respectively. With respect to PPMI, data generation from the posterior distribution resulted in synthetic data that resembled the real data significantly closer than those generated from the prior distribution (Mann-Whitney-U-test, p < 0.02).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <span class="caption-title">
                                Marginal distributions of real and synthesized data for multiple variables.
                               </span>
                               <p id="p-33" class="first-child">
                                Mean, standard deviation and KL-Divergence for the displayed variables can be found in
                                <strong>
                                 Supplementary Table 1
                                </strong>
                                . Equivalent results for the NACC data are presented in
                                <strong>
                                 Supplementary Figure 5. A
                                </strong>
                                , time dependent variable ‘SCOPA’ at month 12.
                                <strong>
                                 B
                                </strong>
                                , time dependent variable ‘UPDRS2’ at month 24.
                                <strong>
                                 C
                                </strong>
                                , static variable ‘Aβ.42’.
                                <strong>
                                 D
                                </strong>
                                , categorical static variable ‘Handedness’.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-34">
                              Compared to VAMBN, the prior sampling method seemed to be inferior with respect to the average JS-divergence when using NACC (U-test, p = 0.038). However, no statistically significant difference in the performance of VAMBN compared to MultiNODE’s posterior sampling could be observed (U-test, p = 0.80). For PPMI, no significant differences were found between VAMBN and any of MultiNODEs’ generation approaches (U-test, p = 0.31 for the prior approach; U-test, p = 0.24 for the posterior).
                             </p>
                             <p id="p-35">
                              In order to evaluate whether MultiNODEs learned not only to reproduce univariate distributions but actually captured their interdependencies accurately, we compared the correlation structure of the generated data to that of the real variables. Visualizations of the Spearman rank correlation coefficients showed that both the prior and posterior sampling generated synthetic data which successfully reproduced the real variables’ interdependencies
                              <strong>
                               (
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Figure 3
                               </a>
                               )
                              </strong>
                              . Comparing the results against VAMBN generated data revealed that both generation procedures of MultiNODEs were significantly better at reproducing the real data characteristics: the Frobenius norm of real data correlation matrix resulted in 45.3, and with a Frobenius norm of 25.66 the VAMBN generated data placed substantially further from the real data than the MultiNODEs approaches with 62.63 and 56.47 for the prior and posterior sampling, respectively. This shows that MultiNODEs slightly overestimated the present correlations, while VAMBN underestimated them. Concordantly, the relative error (i.e., the deviation of the respective synthetic dataset’s correlation matrix from the real one normalized by the norm of the real correlation matrix), was 0.81, 0.62, and 0.46 respectively for VAMBN and MultiNODEs’ prior and posterior sampling, leaving MultiNODEs with a substantially lower error than the VAMBN approach.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <span class="caption-title">
                                Correlation structure of real and synthetic data expressed as spearman rank correlation coefficients.
                               </span>
                               <p id="p-36" class="first-child">
                                Equivalent results for the NACC data are shown in
                                <strong>
                                 Supplementary Figure 6. A
                                </strong>
                                , real data.
                                <strong>
                                 B
                                </strong>
                                , posterior sampling from MultiNODEs.
                                <strong>
                                 C
                                </strong>
                                , prior sampling from MultiNODEs.
                                <strong>
                                 D
                                </strong>
                                , VAMBN generated data.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Assessment of the utility of generated synthetic patients for machine learning
                             </h3>
                             <p id="p-37">
                              To evaluate whether the generated synthetic patients could be reliably used in a machine learning context, we built a Random Forest classifier that aimed to distinguish between healthy individuals and diseased patients. The classifier was trained within a five-fold cross-validation scheme once using real and once using synthetic diseased patients. Additionally, we trained a classifier on each respective synthetic dataset (comprising synthetically generated diseased and healthy subjects) and evaluated their performance on the real data
                              <strong>
                               (
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               )
                              </strong>
                              . As predictors, we used clinical symptoms and genetic markers that are characteristic for the disease in question. For PD (PPMI), these were the UPDRS scores which describe a series of motor and non-motor symptoms commonly encountered in PD patients, for AD (NACC), we predominantly used cognitive assessments and a genetic risk factor. Technical details about the classifiers can be found in the Supplementary Material. Distinguishing real patients from healthy control subjects was possible with a 10 times repeated five-fold cross validated performance of 0.97 ± 0.02 area under the receiver operator curve (AUC) and 0.90 ± 0.01 AUC for PPMI and NACC, respectively. On PPMI, all evaluated generative methods achieved almost equal performance, indicating that clinical characteristics of synthetic patients followed the same patterns as in real patients. In addition, the most relevant features were the same across the real and all synthetic data-trained classifiers
                              <strong>
                               (Supplementary Figure 13)
                              </strong>
                              .
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/543251/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup last">
                                  <a href="/highwire/markup/543251/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1:
                               </span>
                               <span class="caption-title">
                                Performance (AUC) of machine learning classifiers differentiating between real healthy control subject and real as well as synthetic patients, respectively.
                               </span>
                               <p id="p-38" class="first-child">
                                Values represent the average and standard deviation across a 10-times repeated 5-fold cross-validation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-39">
                              For NACC, some deviations were found between a classifier’s cross-validated performance on real data and the synthetic-data-based performances. Here, MultiNODEs’ posterior and VAMBN showed similar deviations in opposite directions, with the posterior slightly overperforming and VAMBN slightly underperforming. The performance on the data generated via MultiNODE’s prior sampling method deviated the most
                              <strong>
                               (
                               <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               )
                              </strong>
                              . When trained on synthetic data and evaluated on real data, all trained classifiers underperformed compared to classifiers trained on real data. The feature importances of predictors were highly similar between the real data-trained and the respective synthetic data-trained classifiers.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Generating data in continuous time through smooth interpolation and extrapolation
                             </h3>
                             <p id="p-40">
                              One particular strength of MultiNODEs, that sets it apart from alternative approaches such as VAMBN, is its ability to model variable trajectories in continuous time. The latent ODE system allows for estimation of variable trajectories at any arbitrary time point and thereby opens possibilities for 1) the generation of smooth trajectories, 2) overcoming panel-data limitations through interpolation, and finally 3) extrapolation beyond the time span covered in training data themselves. Again, we evaluated these capabilities based on the PPMI and NACC datasets (for brevity, NACC results are presented in the Supplementary Material). For the following we only focused on the MultiNODE posterior sampling approach to generate synthetic subjects.
                             </p>
                             <p id="p-41">
                              Comparing the median trajectories of variables from the real data to those generated using MultiNODEs revealed that MultiNODEs accurately learned and reproduced the longitudinal dynamics exhibited in the real data
                              <strong>
                               (
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Figure 4
                               </a>
                               )
                              </strong>
                              . Generation from both the prior and posterior distribution led to synthesized median trajectories that closely resembled the real median trajectories. Equivalently, also the 97.5% and 2.5% quantiles of the synthetic data approximated the corresponding real quantiles closely, indicating a realistic distribution of the synthetic data across the observed time points. This observation held true for most of the time-dependent variables (plots for all variables are linked in the Supplementary Material).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <span class="caption-title">
                                Comparison of median trajectories including the 2.5% / 97.5% quantiles of longitudinal variables from synthetic and real PPMI data.
                               </span>
                               <p id="p-42" class="first-child">
                                Additional examples are provided in
                                <strong>
                                 Supplementary Figure 2
                                </strong>
                                . A corresponding example for the NACC dataset is shown in
                                <strong>
                                 Supplementary Figure 7. A, B, C, D
                                </strong>
                                , depict different longitudinal variables from the PPMI dataset.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-43">
                              We further assessed the interpolation and extrapolation capabilities of MultiNODEs. For interpolation, one time point was excluded from model training and subsequently data was generated for all time points including the one left out. Contrasting the interpolated / imputed values against the corresponding real values showed that MultiNODEs accurately reproduced the longitudinal dynamics of a variable, even for unobserved time points
                              <strong>
                               (
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Figure 5 A, C
                               </a>
                               )
                              </strong>
                              . In this context, we further compared the interpolated values against synthetic data that was generated based on the complete real data trajectory. We observed that the mean JS-divergence calculated across all variables between the interpolated data and the real data was slightly higher (0.025 ± 0.011) than that of the real data and the synthetic data generated after training MultiNODEs on the complete trajectory (0.016 ± 0.011). Similarly, the relative error between the interpolated correlation matrix and the real data was again only marginally higher than between the complete data and the real data (0.48 and 0.46, respectively;
                              <strong>
                               Supplementary Figure 4
                              </strong>
                              ).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <span class="caption-title">
                                Time-continuous interpolation and extrapolation of exemplary PPMI variables.
                               </span>
                               <p id="p-44" class="first-child">
                                The black box indicates the interpolated and extrapolated sections. Plots for additional variables are presented in
                                <strong>
                                 Supplementary Figure 3
                                </strong>
                                . A corresponding example for the NACC dataset is shown in
                                <strong>
                                 Supplementary Figure 8. A
                                </strong>
                                , interpolation of the UPDRS1 variable at month 24.
                                <strong>
                                 B
                                </strong>
                                , extrapolation of the last five assessments of the UPDRS1 variable.
                                <strong>
                                 C
                                </strong>
                                , distribution of the interpolated values for UPDRS1 at visit 24.
                                <strong>
                                 D
                                </strong>
                                , distribution of the extrapolated values for UPDRS1 at month 42.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-45">
                              In order to test MultiNODEs’ extrapolation capabilities, only the first 24 months of assessment follow-up and the static variables were used during model training. The trained model was then applied to generate data for the remaining, left out time points of the longitudinal variables. In this course, 77 values were extrapolated while not every variable had the same number of follow-up assessments after month 24. Comparing the extrapolated synthetic data to the left out real data demonstrated reliable extrapolation beyond the training data
                              <strong>
                               (
                               <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                Figure 5 B, D
                               </a>
                               )
                              </strong>
                              . As in the interpolation setting, we also compared the average JS-divergence between the extrapolated data and the real data with that between the real data and synthetic data that were generated after training MultiNODEs on the complete trajectory. As expected, we could see a larger difference between the JS-divergences compared to the interpolation setting with 0.037 ± 0.024 for the extrapolated data and 0.016 ± 0.009 for the synthetic data based on the complete trajectory. The correlation structure in the extrapolation culminated in a relative error of 0.64 compared to 0.46 when using the complete trajectory for training MultiNODEs (
                              <strong>
                               Supplementary Figure 4
                              </strong>
                              ).
                             </p>
                             <p id="p-46">
                              In addition, the marginal distributions at both the interpolated and extrapolated time points also followed those of the real data
                              <strong>
                               (
                               <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                Figure 5 C, D
                               </a>
                               )
                              </strong>
                              .
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Systematic model benchmarking on simulated data
                             </h3>
                             <p id="p-47">
                              To explore the learning properties of MultiNODEs more systematically, we investigated how alternating training conditions with respect to measurement frequency, sample size, and noisiness of the data influence MultiNODEs’ generative performance.
                             </p>
                             <p id="p-48">
                              The benchmarking data was simulated via the well-established Susceptible-Infected-Removed (SIR) model which is often used to describe the spread of infectious diseases and follows a highly non-linear structure: Let S (t) be the number of susceptible individuals at a timepoint t, I (t) be the number of infectious individuals at a timepoint t and R (t) be the number of removed or recovered individuals at a timepoint t. With β as transmission rate, γ as mean recovery / death rate and N =S (t)+ I (t)+ R (t) as fixed population size the SIR model can be defined by the ODE system presented in
                              <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                               Equation (1)
                              </a>
                              :
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Details about the SIR parameter settings are described in the supplementary material.
                             </p>
                             <p id="p-49">
                              As baseline settings for each investigation, we simulated 1000 data points with 10 equidistant assessment time points each, distributed over a span of 40 time intervals and added 5% Gaussian noise to each measurement. That means we added a normally distributed variable with the standard deviation set to 5% of the theoretical range of each of the variables S(t), I(t) and R(t). During the benchmarking, we individually alternated the sample size, time points and the noise level. For the time point investigation, we compared MultiNODEs’ trained on 5, 10 and 100 equidistant assessments; for the sample size we considered 100, 1000, and 5000 samples; and for the noise level we tested 50%, 75% and 100% of the maximum encountered value added as noise.
                             </p>
                             <p id="p-50">
                              Alternating the amount of equidistant, longitudinal time points exposed a strong dependency of MultiNODEs on the longitudinal coverage of the time dependent process
                              <strong>
                               (
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Figure 6 A
                               </a>
                               )
                              </strong>
                              . While the general trends in the data were appropriately learned for all explored assessment frequencies, the position of the observations in time influenced how close the learned function approximated the true data underlying process. Especially the peak of the ‘Infected’-function represented a challenge for MultiNODEs if no data point was located close to it
                              <strong>
                               (
                               <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                Figure 6 A
                               </a>
                               ‘Infected’)
                              </strong>
                              . Similarly, the start of the decline in the ‘Susceptible’-function and the incline in the ‘Removed’-function were shifted, depending on the positioning of measurements. In conclusion, and as expected, a higher observation frequency of the data underlying time-dependent process significantly increased the fit of MultiNODEs to the process, although, general trends could already be approximated for lower assessment frequencies.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <span class="caption-title">
                                Model benchmarking on simulated data from the SIR-model.
                               </span>
                               <p id="p-51" class="first-child">
                                Each panel A, B, C represents the evaluation of another parameter (assessment frequency, sample size, noise level).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-52">
                              Investigating the effect of the sample size on training MultiNODEs, we observed that an increase of the sample size led to an expected improvement of the model fit to the SIR dynamics
                              <strong>
                               (
                               <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                Figure 6 B
                               </a>
                               )
                              </strong>
                              . While the general trends could again be learned from limited data (n = 100), sample sizes of 1000 or 5000 substantially reduced the model’s deviation from the true SIR model. With 1000 samples, the learned dynamic is less stable than when trained on 5000 samples, where a smooth dynamic was learned that closely resembled the true underlying process. In conclusion, MultiNODEs can already learn longitudinal dynamics based on only a few data points, however, they tend to underfit under these circumstances and benefit from larger sample sizes.
                             </p>
                             <p id="p-53">
                              Adding an increasing noise level to the SIR training data revealed that MultiNODEs remain very robust
                              <strong>
                               (
                               <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                                Figure 6 C
                               </a>
                               )
                              </strong>
                              . Only when introducing 100% of the maximal encountered value as additional noise, a clear deviation from the underlying true model could be observed.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-54">
                             In this work, we presented MultiNODEs, a hybrid AI approach to generate synthetic patient-level datasets. MultiNODEs are specifically designed to consider the characteristics of clinical studies, as extends its predecessor, the Neural ODEs, and enables the application of the latent ODE system to multimodal datasets comprising both time-dependent and static variables with values missing not at random. MultiNODEs learn a latent, continuous time trajectory from observed data. This concept fits well to processes like disease progression, where relevant observations (e.g., biomarkers and disease symptoms) only indirectly mimic the true, underlying disease mechanism. Consequently, MultiNODEs are well suited for an application to heterogeneous datasets holding complex signals as encountered, for example, in biomedical research.
                            </p>
                            <p id="p-55">
                             Our evaluations showed that MultiNODEs successfully generated complex, synthetic medical datasets that accurately reproduced the characteristics of their real-world counterparts. In a direct comparison MultiNODEs’ outperformed the state-of-the-art VAMBN approach, most notably with respect to the integrity of the correlation structure. This finding implies that the single data instances generated using MultiNODEs exhibit more realistic properties and that the real data characteristics are not only reproduced on population-level. Out of MultiNODEs two generative methods, the posterior sampling expectedly led to more realistic synthetic patients, however, generating from the prior distribution comes with the benefit that the model itself can be shared and used for data generation without needing any real datapoints in the process.
                            </p>
                            <p id="p-56">
                             Machine learning classifiers that discriminated between real healthy controls and diseased subjects showed almost equal performance when trained on data from synthetic and real diseased subjects, respectively. Here, we only observed small deviations from the performance on real data for the NACC dataset, where classifiers trained and tested on synthetic patients and real healthy controls within a cross-validation setting showed a slightly increased performance to those trained on real data. Interestingly, at the same time we found a lower prediction performance compared to real data when we trained on synthetic subjects and evaluated on the real data. A possible explanation is that synthetic data can contain noise that is introduced during the generation of synthetic data points (for example through overestimated correlations between variables). Therefore, synthetically generated diseased patients are better discriminated against real healthy controls than real diseased patients. At the same time this situation leads to the fact that a classifier trained on synthetic data (synthetic patients as well as healthy controls) shows a slightly lower prediction performance on real data compared to a classifier trained entirely on real data. Altogether our results demonstrate that synthetically generated subjects share patterns of real patients, but they are not completely identical.
                            </p>
                            <p id="p-57">
                             Besides the reproduction of marginal distributions and synthesis of realistic data instances, MultiNODEs most prominent strength lies in the generation of smooth longitudinal data. The latent ODE system allows MultiNODEs to learn dynamics which are continuous in time and cover the unobserved time intervals of real-world data. Here, both the prior and posterior sampling approach resulted in synthetic trajectories that obey real variables’ dynamics.
                            </p>
                            <p id="p-58">
                             Furthermore, the time-continuous generative capabilities of MultiNODEs create opportunities to fill gaps in the real data through interpolation and go beyond the observation time by extrapolating the longitudinal dynamics. Hence, MultiNODEs could be used to support the design of longitudinal clinical studies, in which the maximum observation period as well as visit frequency are always crucial decisions to make. Here, the question of how patients might develop between two visits or after the last one determines the optimal follow-up time, to demonstrate, for example, the most significant treatment effect. Furthermore, synthetic disease trajectories generated based on data from one clinical study can be compared to those generated based on other studies, even if the visit intervals employed in the real studies were not identical.
                            </p>
                            <p id="p-59">
                             Our benchmark experiments on the simulated SIR model data demonstrated that MultiNODEs are applicable under a variety of different data settings. While the general trends of a data underlying process could already be learned from a relatively limited dataset, similar to any machine learning task, the accuracy and trustworthiness of the model critically depends on the available data. Especially for complex, nonlinear processes, a sufficiently high observation frequency should be considered. Here, the position of the observation time-points relative to the true underlying process is crucial for MultiNODEs to accurately learn nonlinear dynamics. The sample size of the training data mainly impacts how well MultiNODEs fitted the data dynamics and we observed that lower sample sizes can lead to underfitting and rather rigid ODE systems. On the other hand, only severe noise levels led to a model deviation from the true data-underlying process and thus, with respect to noise, MultiNODEs proved to be highly robust. In conclusion, MultiNODEs’ requirements towards the training data ultimately depend on the complexity of the data underlying process, whereas the learning of more complex processes requires more frequent observations and larger sample size, while more linear systems can already be learned from rather limited datasets.
                            </p>
                            <p id="p-60">
                             One limitation of MultiNODEs in their current form only allow static categorical variables. This is, because the variational encoder for longitudinal data maps trajectories to a latent Gaussian distribution. Sampling from this distribution (even, if conditioned on the distribution of the static data) and decoding will result into real valued features rather than categorical ones. In future work, we will thus explore whether a recurrent version of the HI-VAE encoder can be used instead of a recurrent variational LSTM encoder.
                            </p>
                            <p id="p-61">
                             Additionally, MultiNODEs are sensitive to several hyperparameters that should be optimized for optimal performance. The training process and all relevant hyperparameters are explained in the Method section.
                            </p>
                            <p id="p-62">
                             Synthetic data generated using models trained on sensitive personal information, can bear a risk of information disclosure (e.g. attribute disclosure or dataset membership disclosure), if an attacker has information about properties of real patients that are similar to a synthetic subject. Therefore, before synthetic data are distributed, it must be assured that the probability of private information disclosure remains within task-appropriate boundaries [
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ]. Disclosure risk often stands in a direct trade-off with data utility and a sensible compromise should be taken balancing the two according to the application in question. Several approaches are described in the literature that can reduce the risk of information disclosure [
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ], one of which is based on the concept of differential privacy [
                             <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. MultiNODEs themselves provide a way to tune the deviation from the real data when sampling from the posterior distribution by changing the amount of noise injected in the latent space.
                            </p>
                            <p id="p-63">
                             We would like to mention that a rigorous quantification of the re-identification risk is a non-trivial and challenging task for its own requiring several assumptions and is thus beyond the scope of this paper.
                            </p>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Application case datasets
                             </h3>
                             <p id="p-64">
                              Both datasets, namely PPMI and NACC, are well known staples in their respective fields and can be accessed after successful data access applications. For PPMI see
                              <a href="https://www.ppmi-info.org/">
                               https://www.ppmi-info.org/
                              </a>
                              . For NACC we refer to
                              <a href="https://naccdata.org/">
                               https://naccdata.org/
                              </a>
                              . More details on the investigated variables are presented in the Supplementary Material.
                             </p>
                             <p id="p-65">
                              Both studies retrieved informed consent from their participants for data collection and sharing and followed the declaration of Helsinki to ensure ethical data collection. Both studies got ethical approval by their respective review boards. We followed their employed regulations and thus did not seek further ethical approval, as we did not work with human participants ourselves.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Neural ODEs (NODEs)
                             </h3>
                             <p id="p-66">
                              NODEs are a hybrid of neural networks and ODEs [
                              <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                              ]. They can be seen as an extension of a ResNet [
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ], which does not rely on a discrete sequence of hidden layers, but on a continuous hidden dynamical system defined by an ordinary differential equation.
                             </p>
                             <p id="p-67">
                              For 0 <t < M and z
                              <sub>
                               0
                              </sub>
                              ∈ R
                              <sup>
                               D
                              </sup>
                              the dynamics of the hidden layer of a NODE are given as
                              <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                               Equation (2)
                              </a>
                              .
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where z (0) may be interpreted as the first hidden layer and z (T) as the solution to the initial value problem at time point T. Importantly, f is a feed-forward neural network parameterized by θ.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              NODEs as generative latent time series models
                             </h3>
                             <p id="p-68">
                              As demonstrated by the authors in their publication, NODEs can be trained as a continuous time Variational Autoencoder. The basic idea is to learn the initial conditions z
                              <sub>
                               0
                              </sub>
                              of the dynamical system in
                              <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">
                               Equation (2)
                              </a>
                              from observed time series data using a variational long-short term memory (LSTM) recurrent encoder [
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ]. Hence,
                              <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">
                               Equation (2)
                              </a>
                              now describes the dynamics of a latent system, resulting into a classical state-observation model. Accordingly, a feed-forward neural network decoder is required to project the solution of
                              <a id="xref-disp-formula-2-4" class="xref-disp-formula" href="#disp-formula-2">
                               Equation (2)
                              </a>
                              back to observed data at defined time points (
                              <strong>
                               Supplementary Figure 10
                              </strong>
                              ).
                             </p>
                             <p id="p-69">
                              Overall NODEs are trained at once by maximizing the Evidence Lower Variational Bound (ELBO): Let the training data be
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where N is the number of patients and
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              the observed time points / patient visits. That means
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the p-dimensional vector of measurements taken for the n-th patient at visit t
                              <sub>
                               i
                              </sub>
                              . The ELBO for NODEs is then given as
                              <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                               Equation (3)
                              </a>
                              .
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , as usual. For details we refer to Chen et al. [
                              <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                              ]
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Multi Modal Neural NODEs (MultiNODEs)
                             </h3>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Handling missing values
                              </h4>
                              <p id="p-70">
                               To handle missing values (potentially not at random) in longitudinal clinical data we build on our previously published work, in which we introduced an imputation layer to implicitly estimate missing values during neural network training [
                               <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                               ]: Let
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , 1
                               <sub>
                                A
                               </sub>
                               be the indicator function on set A with cardinality |A|. The imputation layer can be defined as a data transformation
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where parameters
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are trainable weights. That means missing values in a patient’s data vector
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are replaced by
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The accordingly completed data is subsequently mapped through a recurrent neural network encoder to a static, lower dimensional vector, which is interpreted as the initial condition of the latent ODE system (
                               <strong>
                                Supplementary Figure 11
                               </strong>
                               ).
                              </p>
                              <p id="p-71">
                               To learn parameters
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               the NODEs’ loss function needs to be adapted. More specifically, we use the modified ELBO criterion presented in
                               <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
                                Equation (4)
                               </a>
                               .
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               denotes the reconstructed data. Note that we only aim for reconstructing the observed data, but not the imputed one. Due to the layer-wise architecture of a neural network
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               implicitly depends on
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-72">
                               In practice we initialize
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for neural network training as
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-17" class="subsection">
                              <h4>
                               Dealing with multimodal data
                              </h4>
                              <p id="p-73">
                               In addition to implicit missing value imputation, the second main idea of MultiNODEs is to complement NODEs with a HI-VAE encoder [
                               <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                               ] for static variables (
                               <strong>
                                Supplementary Figure 11
                               </strong>
                               ). A HI-VAE is an extension of a Variational Autoencoder which can implicitly impute missing values via an input drop-out model and handle heterogeneous multimodal data, including categorical data and count data, via an accordingly factorized generative model. In addition, a HI-VAE uses a Gaussian Mixture Model (GMM) as a prior distribution rather than a single Gaussian. We refer to Nazabal et al. [
                               <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                               ] for details.
                              </p>
                              <p id="p-74">
                               The HI-VAE results in a lower dimensional latent representation z
                               <sub>
                                stat
                               </sub>
                               of static variables, which can be used to augment the initial conditions
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               learned from time series data. Consequently, we arrive at the following formulation of the latent ODE system given in
                               <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                                Equation (5)
                               </a>
                               .
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               This approach resembles the Augmented Neural ODEs by Dupont et al. [
                               <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                                18
                               </a>
                               ]. The main difference to our work is that in their work no additional features are added by the augmentation step, i.e. z
                               <sub>
                                stat
                               </sub>
                               =0. According to Dupont et al. the purpose of Augmented Neural ODEs is to smoothen f, whereas we focus here on multimodal data integration.
                              </p>
                              <p id="p-75">
                               For training MultiNODEs, we have to jointly consider
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               as well as ELBO
                               <sup>
                                HI -VAE
                               </sup>
                               . After bringing both quantities on a comparable numerical scale, we use a weighted sum as our final training objective (see
                               <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                                Equation (6)
                               </a>
                               and
                               <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                                (7)
                               </a>
                               ):
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Whereβ is a tunable hyperparameter. Details about hyperparameter optimization are described in the Supplements.
                              </p>
                             </div>
                             <div id="sec-18" class="subsection">
                              <h4>
                               Generating synthetic subjects
                              </h4>
                              <p id="p-76">
                               We tested two methods to generate synthetic subjects with MultiNODEs:
                              </p>
                              <ol class="list-alphalower" id="list-1">
                               <li id="list-item-1">
                                <p id="p-77">
                                 The first option is drawing a sample of latent static and longitudinal representations from the respective prior distributions
                                 <span class="inline-formula" id="inline-formula-18">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-18.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-18.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 and z
                                 <sub>
                                  stat
                                 </sub>
                                 ∼ GMM (π). To assure that interdependencies between static and longitudinal variables are conserved, we model their joint distribution P (z
                                 <sub>
                                  t 0
                                 </sub>
                                 , z
                                 <sub>
                                  stat
                                 </sub>
                                 ) using a Bayesian network. This network contains three nodes (random variables) representing 1) the GMM mixture coefficients π for the static data used by the HI-VAE, 2) the latent static representations Z
                                 <sub>
                                  stat
                                 </sub>
                                 =GMM (π), and 3) the latent longitudinal representations Z
                                 <sub>
                                  t 0
                                 </sub>
                                 =N (0, I), respectively. The network is constraint such that directed edges can only go from the π to Z
                                 <sub>
                                  stat
                                 </sub>
                                 and from there to Z
                                 <sub>
                                  t 0
                                 </sub>
                                 . After randomly sampling a mixture component s
                                 <sub>
                                  i
                                 </sub>
                                 from a multinomial distribution multinom (π), we can conditionally sample z
                                 <sub>
                                  stat
                                 </sub>
                                 ∼ Z
                                 <sub>
                                  stat
                                 </sub>
                                 ∨ s
                                 <sub>
                                  i
                                 </sub>
                                 and finally z
                                 <sub>
                                  t 0
                                 </sub>
                                 ∼ Z
                                 <sub>
                                  t 0
                                 </sub>
                                 ∨ Z
                                 <sub>
                                  stat
                                 </sub>
                                 . Subsequently, we concatenate
                                 <span class="inline-formula" id="inline-formula-19">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-19.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-19.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 into a vector forming the initial conditions for the latent ODE system, solve the ODE system and decode the solution. We call this approach “prior sampling”.
                                </p>
                               </li>
                               <li id="list-item-2">
                                <p id="p-78">
                                 A second option is to draw
                                 <span class="inline-formula" id="inline-formula-20">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-20.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-20.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 for the longitudinal data and
                                 <span class="inline-formula" id="inline-formula-21">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-21.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-21.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 Categorical
                                 <span class="inline-formula" id="inline-formula-22">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-22.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/07/26/2021.09.26.21263968/embed/inline-graphic-22.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 for the static data. That means we generate a blurred / noisy version of the original n-th patient. We call this approach “posterior sampling” and recommend this sampling procedure for data generation. In our experiments, we doubled the posterior variance during sampling because we found the synthetic data otherwise to lie too close to the real data. Tuning the added noise can provide one option to balance identification risk versus data utility.
                                </p>
                               </li>
                               <li id="list-item-3">
                                <p id="p-79">
                                 Synthetic data can not only be generated for observed visits, but also for definable time points in between (interpolation) and after the end of study (extrapolation). This is possible because the latent ODE system is continuous in time.
                                </p>
                               </li>
                              </ol>
                             </div>
                             <div id="sec-19" class="subsection">
                              <h4>
                               Data preprocessing
                              </h4>
                              <p id="p-80">
                               Few steps are required to preprocess the clinical data before MultiNODEs can be applied. First, the data must be organized into a three-dimensional tensor of the shape samples × time points × variables for the longitudinal variables, and samples × variables for the static ones. Furthermore, the longitudinal variables are then transformed into a progression score by subtracting the baseline value and normalizing them by the standard deviation of this variable at baseline.
                              </p>
                             </div>
                             <div id="sec-20" class="subsection">
                              <h4>
                               Calculating the relative error for correlation matrices
                              </h4>
                              <p id="p-81">
                               The relative error between correlation matrices is calculated as the norm of the matrix describing the difference between the real correlation matrix and synthetic data correlation matrix divided by the norm of the real correlation matrix.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section data-availability" id="sec-22">
                            <h2 class="">
                             Data Availability
                            </h2>
                            <p id="p-82">
                             All data used in our manuscript are freely available for the purpose of (non-profit) research
                            </p>
                            <p id="p-83">
                             <a href="https://www.ppmi-info.org/access-data-specimens/download-data/">
                              https://www.ppmi-info.org/access-data-specimens/download-data/
                             </a>
                            </p>
                            <p id="p-84">
                             <a href="https://naccdata.org/">
                              https://naccdata.org/
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Code availability
                            </h2>
                            <p id="p-85">
                             The code for MultiNODEs is available at
                             <a href="https://github.com/philippwendland/MultiNODEs">
                              https://github.com/philippwendland/MultiNODEs
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Data availability
                            </h2>
                            <p id="p-86">
                             The PPMI dataset is available under:
                             <a href="https://www.ppmi-info.org/">
                              https://www.ppmi-info.org/
                             </a>
                             . The NACC data is available under:
                             <a href="https://naccdata.org/">
                              https://naccdata.org/
                             </a>
                             . The data is shared by the data owners after successful application.
                            </p>
                            <p id="p-87">
                             The data generated for this study can not be shared by the authors due to the signed data usage agreements with the data owners of the corresponding real data (ie PPMI and NACC).
                            </p>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             Competing Interests
                            </h2>
                            <p id="p-91">
                             The authors declare that there are no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-26">
                            <h2 class="">
                             Author Contribution
                            </h2>
                            <p id="p-92">
                             CB and PW contributed equally to this work. HF, CB, PW, and MK conceived the project. PW implemented the method. PW, MGF, and MS performed the experiments. CB and HF wrote the manuscript. PW, MK, and MS revised the manuscript. CB and HF supervised the work.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-88">
                             This project has received funding from the European Union’s Horizon 2020 research and innovation programme under grant agreement No. 826421, “TheVirtualBrain-Cloud” and from the Deutsche Forschungsgemeinschaft (DFG) funded project “NFDI4Health” (project number 442326535).
                            </p>
                            <p id="p-89">
                             Data used in the preparation of this article were obtained from the Parkinson’s Progression Markers Initiative (PPMI) database (
                             <a href="http://www.ppmi-info.org/data">
                              www.ppmi-info.org/data
                             </a>
                             ). For up-to-date information on the study, visit
                             <a href="http://www.ppmi-info.org">
                              www.ppmi-info.org
                             </a>
                             . PPMI – a public-private partnership – is funded by the Michael J. Fox Foundation for Parkinson’s Research and funding partners, including [list the full names of all of the PPMI funding partners found at
                             <a href="http://www.ppmi-info.org/fundingpartners">
                              www.ppmi-info.org/fundingpartners
                             </a>
                             ].
                            </p>
                            <p id="p-90">
                             The NACC database is funded by NIA/NIH Grant U01 AG016976. NACC data are contributed by the NIA-funded ADCs: P30 AG019610 (PI Eric Reiman, MD), P30 AG013846 (PI Neil Kowall, MD), P30 AG062428-01 (PI James Leverenz, MD) P50 AG008702 (PI Scott Small, MD), P50 AG025688 (PI Allan Levey, MD, PhD), P50 AG047266 (PI Todd Golde, MD, PhD), P30 AG010133 (PI Andrew Saykin, PsyD), P50 AG005146 (PI Marilyn Albert, PhD), P30 AG062421-01 (PI Bradley Hyman, MD, PhD), P30 AG062422-01 (PI Ronald Petersen, MD, PhD), P50 AG005138 (PI Mary Sano, PhD), P30 AG008051 (PI Thomas Wisniewski, MD), P30 AG013854 (PI Robert Vassar, PhD), P30 AG008017 (PI Jeffrey Kaye, MD), P30 AG010161 (PI David Bennett, MD), P50 AG047366 (PI Victor Henderson, MD, MS), P30 AG010129 (PI Charles DeCarli, MD), P50 AG016573 (PI Frank LaFerla, PhD), P30 AG062429-01(PI James Brewer, MD, PhD), P50 AG023501 (PI Bruce Miller, MD), P30 AG035982 (PI Russell Swerdlow, MD), P30 AG028383 (PI Linda Van Eldik, PhD), P30 AG053760 (PI Henry Paulson, MD, PhD), P30 AG010124 (PI John Trojanowski, MD, PhD), P50 AG005133 (PI Oscar Lopez, MD), P50 AG005142 (PI Helena Chui, MD), P30 AG012300 (PI Roger Rosenberg, MD), P30 AG049638 (PI Suzanne Craft, PhD), P50 AG005136 (PI Thomas Grabowski, MD), P30 AG062715-01 (PI Sanjay Asthana, MD, FRCP), P50 AG005681 (PI John Morris, MD), P50 AG047270 (PI Stephen Strittmatter, MD, PhD).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-17">
                               Minor updates, f.e. parts of references and classificators
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fröhlich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  From hype to reality: data science enabling personalized medicine
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  150
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BMed%26rft.volume%253D16%26rft.spage%253D150%26rft_id%253Dinfo%253Apmid%252F30145981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30145981&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F07%2F26%2F2021.09.26.21263968.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Birkenbihl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salimi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fröhlich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  for the Japanese Alzheimer’s Disease Neuroimaging Initiative, & the Alzheimer’s Disease Neuroimaging Initiative. Unraveling the heterogeneity in Alzheimer’s disease progression across multiple cohorts and the implications for data-driven disease modeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Alzheimers Dement
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  251
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  261
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAlzheimers%2BDement%26rft.volume%253D18%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Birkenbihl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Differences in cohort study data affect external validation of artificial intelligence models for predictive diagnostics of dementia - lessons for translation into clinical practice
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  EPMA J
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  367
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  376
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEPMA%2BJ%26rft.volume%253D11%26rft.spage%253D367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gootjes-Dreesbach
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sood
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sahay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hofmann-Apitius
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fröhlich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Variational Autoencoder Modular Bayesian Networks for Simulation of Heterogeneous Clinical Study Data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front. Big Data
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  16
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFront.%2BBig%2BData%26rft.volume%253D3%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sood
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Realistic simulation of virtual multi-scale, multi-modal patient trajectories using Bayesian networks and sparse auto-encoders
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  10971
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D10%26rft.spage%253D10971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Williamson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. F. K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahmood
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Synthetic data in machine learning for medicine and healthcare
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Biomed. Eng
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  493
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  497
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BBiomed.%2BEng%26rft.volume%253D5%26rft.spage%253D493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.7" data-doi="10.2147/CLEP.S242097">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thorlund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dron
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mills
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Synthetic and External Controls in Clinical Trials – A Primer for Researchers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Clin. Epidemiol
                                 </abbr>
                                 . Volume
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  457
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  467
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DClin.%2BEpidemiol%26rft.volume%253D12%26rft.spage%253D457%26rft_id%253Dinfo%253Adoi%252F10.2147%252FCLEP.S242097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2147/CLEP.S242097&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  MRI-only based synthetic CT generation using dense cycle consistent generative adversarial networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Med. Phys
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3565
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3581
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMed.%2BPhys%26rft.volume%253D46%26rft.spage%253D3565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.9" data-doi="10.1109/TMI.2017.2785879">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  DAGAN: Deep De-Aliasing Generative Adversarial Networks for Fast Compressed Sensing MRI Reconstruction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Trans. Med. Imaging
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1310
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1321
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BMed.%2BImaging%26rft.volume%253D37%26rft.spage%253D1310%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTMI.2017.2785879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/TMI.2017.2785879&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2021.09.26.21263968v3.10">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Bengio
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    S.
                                   </span>
                                  </span>
                                  ,
                                  <span class="cit-etal">
                                   et al.
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. T. Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rubanova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bettencourt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duvenaud
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. K.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  Neural Ordinary Differential Equations
                                 </span>
                                 .
                                 <span class="cit-source">
                                  in Advances in Neural Information Processing Systems
                                 </span>
                                 (eds.
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Bengio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ) vol.
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Curran Associates, Inc
                                 </span>
                                 .,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.11" data-doi="10.1016/j.pneurobio.2011.09.005">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Parkinson Progression Marker Initiative (PPMI)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Prog. Neurobiol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  95
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  629
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  635
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Bneurobiology%26rft.stitle%253DProg%2BNeurobiol%26rft.aulast%253DMarek%26rft.auinit1%253DK.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D635%26rft.atitle%253DThe%2BParkinson%2BProgression%2BMarker%2BInitiative%2B%2528PPMI%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pneurobio.2011.09.005%26rft_id%253Dinfo%253Apmid%252F21930184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.pneurobio.2011.09.005&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21930184&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F07%2F26%2F2021.09.26.21263968.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Besser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Version 3 of the National Alzheimer’s Coordinating Center’s Uniform Data Set
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Alzheimer Dis. Assoc. Disord
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  351
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  358
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAlzheimer%2BDis.%2BAssoc.%2BDisord%26rft.volume%253D32%26rft.spage%253D351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nazabal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Olmos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghahramani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Valera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Handling Incomplete Heterogeneous Data using VAEs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ArXiv180703653 Cs Stat
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Jong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Deep learning for clustering of multivariate clinical patient trajectories with missing values
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  GigaScience
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  giz134
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D8%26rft.spage%253Dgiz134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.09.26.21263968v3.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kingma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Welling
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Auto-Encoding Variational Bayes
                                 </span>
                                 . at
                                 <a href="http://arxiv.org/abs/1312.6114">
                                  http://arxiv.org/abs/1312.6114
                                 </a>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2021.09.26.21263968v3.16" data-doi="10.1109/CVPR.2016.90">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   He
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Deep Residual Learning for Image Recognition. in 2016 IEEE Conference on Computer Vision and Pattern Recognition (CVPR)
                                 </span>
                                 <span class="cit-fpage">
                                  770
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  778
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  IEEE
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1109/CVPR.2016.90
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDeep%2BResidual%2BLearning%2Bfor%2BImage%2BRecognition.%2Bin%2B2016%2BIEEE%2BConference%2Bon%2BComputer%2BVision%2Band%2BPattern%2BRecognition%2B%2528CVPR%2529%26rft_id%253Dinfo%253Adoi%252F10.1109%252FCVPR.2016.90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/CVPR.2016.90&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.17" data-doi="10.1162/neco.1997.9.8.1735">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hochreiter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schmidhuber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Long Short-Term Memory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural Comput
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1735
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1780
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BComputation%26rft.stitle%253DNeural%2BComput.%26rft.aulast%253DHochreiter%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D1735%26rft.epage%253D1780%26rft.atitle%253DLong%2BShort-Term%2BMemory%26rft_id%253Dinfo%253Adoi%252F10.1162%252Fneco.1997.9.8.1735%26rft_id%253Dinfo%253Apmid%252F9377276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/neco.1997.9.8.1735&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9377276&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F07%2F26%2F2021.09.26.21263968.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1997YA04500007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2021.09.26.21263968v3.18">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Wallach
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    H.
                                   </span>
                                  </span>
                                  ,
                                  <span class="cit-etal">
                                   et al.
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dupont
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Doucet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. W.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  Augmented Neural ODEs
                                 </span>
                                 .
                                 <span class="cit-source">
                                  in Advances in Neural Information Processing Systems
                                 </span>
                                 (eds.
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Wallach
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ) vol.
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Curran Associates, Inc
                                 </span>
                                 .,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2021.09.26.21263968v3.19" data-doi="10.1145/3419394.3423643">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jain
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fanti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sekar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  Using GANs for Sharing Networked Time Series Data: Challenges, Initial Promise, and Open Questions
                                 </span>
                                 .
                                 <span class="cit-source">
                                  in Proceedings of the ACM Internet Measurement Conference
                                 </span>
                                 <span class="cit-fpage">
                                  464
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  483
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  ACM
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1145/3419394.3423643
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253Din%2BProceedings%2Bof%2Bthe%2BACM%2BInternet%2BMeasurement%2BConference%26rft_id%253Dinfo%253Adoi%252F10.1145%252F3419394.3423643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1145/3419394.3423643&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.26.21263968v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bae
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Choi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yoon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  AnomiGAN: Generative Adversarial Networks for Anonymizing Private Medical Data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Pac. Symp. Biocomput. Pac. Symp. Biocomput
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  563
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  574
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPac.%2BSymp.%2BBiocomput.%2BPac.%2BSymp.%2BBiocomput%26rft.volume%253D25%26rft.spage%253D563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.26.21263968v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jordon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yoon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  PATE-GAN: Generating synthetic data with differential privacy guarantees
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  in International Conference on Learning Representations
                                 </abbr>
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.22" data-doi="10.1161/CIRCOUTCOMES.118.005122">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beaulieu-Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Privacy-Preserving Generative Deep Neural Networks Support Clinical Data Sharing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Circ. Cardiovasc. Qual. Outcomes
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e005122
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCirc.%2BCardiovasc.%2BQual.%2BOutcomes%26rft.volume%253D12%26rft.spage%253De005122%26rft_id%253Dinfo%253Adoi%252F10.1161%252FCIRCOUTCOMES.118.005122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1161/CIRCOUTCOMES.118.005122&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goncalves
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Generation and evaluation of synthetic patient data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Med. Res. Methodol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  108
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BMed.%2BRes.%2BMethodol%26rft.volume%253D20%26rft.spage%253D108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.26.21263968v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Data Synthesis based on Generative Adversarial Networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. VLDB Endow
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1071
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1083
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BVLDB%2BEndow%26rft.volume%253D11%26rft.spage%253D1071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2021.09.26.21263968v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.08.15.21262087v3" title="The relationships between ongoing COVID-19 lockdown and the financial and mental health experiences of Australian families" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.01.27.21250658v2" title="COVID-19 mortality: positive correlation with cloudiness but no correlation with sunlight and latitude in Europe" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted July 26, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2021.09.26.21263968v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2021.09.26.21263968v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/07/26/2021.09.26.21263968.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2021.09.26.21263968v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about medRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2021.09.26.21263968v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Generation of realistic synthetic data using multimodal neural ordinary differential equations
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from medRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the medRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/542641"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-D3ml2zTYxuQB4Bs6Qf4DYxVheaRZoHkhHBLPU9poRPM"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="277160046"/>
                                   <input type="hidden" name="captcha_token" value="45ef795b7a735e2cfdc35ae8e1115574"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node542641--3" data-node-nid="542641" id="node-542641--4998595887" data-pisa="medrxiv;2021.09.26.21263968v3" data-pisa-master="medrxiv;2021.09.26.21263968" data-seqnum="542641" data-apath="/medrxiv/early/2022/07/26/2021.09.26.21263968.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Generation of realistic synthetic data using multimodal neural ordinary differential equations
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Philipp
                                  </span>
                                  <span class="nlm-surname">
                                   Wendland
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Colin
                                  </span>
                                  <span class="nlm-surname">
                                   Birkenbihl
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Marc
                                  </span>
                                  <span class="nlm-surname">
                                   Gomez-Freixa
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Meemansa
                                  </span>
                                  <span class="nlm-surname">
                                   Sood
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Maik
                                  </span>
                                  <span class="nlm-surname">
                                   Kschischo
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Holger
                                  </span>
                                  <span class="nlm-surname">
                                   Fröhlich
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 medRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2021.09.26.21263968;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2021.09.26.21263968
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2021.09.26.21263968v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&title=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&title=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&text=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&t=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&title=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&title=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations&summary=&source=medRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&title=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node542641--5" data-node-nid="542641" id="citation-node-542641--61395077291" data-pisa="medrxiv;2021.09.26.21263968v3" data-pisa-master="medrxiv;2021.09.26.21263968" data-seqnum="542641" data-apath="/medrxiv/early/2022/07/26/2021.09.26.21263968.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Generation of realistic synthetic data using multimodal neural ordinary differential equations
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Philipp
                                      </span>
                                      <span class="nlm-surname">
                                       Wendland
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Colin
                                      </span>
                                      <span class="nlm-surname">
                                       Birkenbihl
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Marc
                                      </span>
                                      <span class="nlm-surname">
                                       Gomez-Freixa
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Meemansa
                                      </span>
                                      <span class="nlm-surname">
                                       Sood
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Maik
                                      </span>
                                      <span class="nlm-surname">
                                       Kschischo
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Holger
                                      </span>
                                      <span class="nlm-surname">
                                       Fröhlich
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     medRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2021.09.26.21263968;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2021.09.26.21263968
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=medRxiv&rft.au=Wendland%2C+Philipp&rft.au=Birkenbihl%2C+Colin&rft.au=Gomez-Freixa%2C+Marc&rft.au=Sood%2C+Meemansa&rft.au=Kschischo%2C+Maik&rft.au=Fr%C3%B6hlich%2C+Holger">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/542641/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/542641/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/542641/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/542641/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/542641/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/542641/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/542641/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/542641/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/542641/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/542641/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/542641/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/542641/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&count=horizontal&via=&text=Generation%20of%20realistic%20synthetic%20data%20using%20multimodal%20neural%20ordinary%20differential%20equations&counturl=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2021.09.26.21263968v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.medrxiv.org/content/10.1101/2021.09.26.21263968v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/health-informatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Health Informatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Addiction Medicine
                          </a>
                          <span class="article-count">
                           (218)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Allergy and Immunology
                          </a>
                          <span class="article-count">
                           (496)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">
                           Anesthesia
                          </a>
                          <span class="article-count">
                           (106)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Cardiovascular Medicine
                          </a>
                          <span class="article-count">
                           (1121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Dentistry and Oral Medicine
                          </a>
                          <span class="article-count">
                           (198)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">
                           Dermatology
                          </a>
                          <span class="article-count">
                           (141)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Emergency Medicine
                          </a>
                          <span class="article-count">
                           (275)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">
                           Endocrinology (including Diabetes Mellitus and Metabolic Disease)
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (9829)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Forensic Medicine
                          </a>
                          <span class="article-count">
                           (5)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">
                           Gastroenterology
                          </a>
                          <span class="article-count">
                           (484)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Genetic and Genomic Medicine
                          </a>
                          <span class="article-count">
                           (2344)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Geriatric Medicine
                          </a>
                          <span class="article-count">
                           (225)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Economics
                          </a>
                          <span class="article-count">
                           (465)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Informatics
                          </a>
                          <span class="article-count">
                           (1574)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">
                           Health Policy
                          </a>
                          <span class="article-count">
                           (738)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">
                           Health Systems and Quality Improvement
                          </a>
                          <span class="article-count">
                           (609)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">
                           Hematology
                          </a>
                          <span class="article-count">
                           (239)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">
                           HIV/AIDS
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">
                           Infectious Diseases (except HIV/AIDS)
                          </a>
                          <span class="article-count">
                           (11686)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Intensive Care and Critical Care Medicine
                          </a>
                          <span class="article-count">
                           (617)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Education
                          </a>
                          <span class="article-count">
                           (242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Ethics
                          </a>
                          <span class="article-count">
                           (67)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">
                           Nephrology
                          </a>
                          <span class="article-count">
                           (259)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">
                           Neurology
                          </a>
                          <span class="article-count">
                           (2170)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">
                           Nursing
                          </a>
                          <span class="article-count">
                           (134)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">
                           Nutrition
                          </a>
                          <span class="article-count">
                           (340)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">
                           Obstetrics and Gynecology
                          </a>
                          <span class="article-count">
                           (427)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">
                           Occupational and Environmental Health
                          </a>
                          <span class="article-count">
                           (521)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">
                           Oncology
                          </a>
                          <span class="article-count">
                           (1188)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">
                           Ophthalmology
                          </a>
                          <span class="article-count">
                           (368)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">
                           Orthopedics
                          </a>
                          <span class="article-count">
                           (129)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">
                           Otolaryngology
                          </a>
                          <span class="article-count">
                           (221)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Pain Medicine
                          </a>
                          <span class="article-count">
                           (148)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Palliative Medicine
                          </a>
                          <span class="article-count">
                           (50)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (315)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">
                           Pediatrics
                          </a>
                          <span class="article-count">
                           (702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Therapeutics
                          </a>
                          <span class="article-count">
                           (304)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">
                           Primary Care Research
                          </a>
                          <span class="article-count">
                           (269)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">
                           Psychiatry and Clinical Psychology
                          </a>
                          <span class="article-count">
                           (2201)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">
                           Public and Global Health
                          </a>
                          <span class="article-count">
                           (4705)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">
                           Radiology and Imaging
                          </a>
                          <span class="article-count">
                           (786)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">
                           Rehabilitation Medicine and Physical Therapy
                          </a>
                          <span class="article-count">
                           (462)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Respiratory Medicine
                          </a>
                          <span class="article-count">
                           (626)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">
                           Rheumatology
                          </a>
                          <span class="article-count">
                           (276)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">
                           Sexual and Reproductive Health
                          </a>
                          <span class="article-count">
                           (227)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Sports Medicine
                          </a>
                          <span class="article-count">
                           (214)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">
                           Surgery
                          </a>
                          <span class="article-count">
                           (254)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Toxicology
                          </a>
                          <span class="article-count">
                           (43)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">
                           Transplantation
                          </a>
                          <span class="article-count">
                           (121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">
                           Urology
                          </a>
                          <span class="article-count">
                           (94)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__jnN2NeYkYJiOsZIjmjzMyRUq-BPexbFCB6tt10YTdRs__NUy5LzWFINyoPmO8CpYVRZuK8HPdQ1G_L4n5lMB-Irw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
