<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2021.09.07.21263228v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2021.09.07.21263228v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2021.09.07.21263228v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-5.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/medrxiv/early/2022/02/16/2021.09.07.21263228.atom"/>
  <meta name="HW.pisa" content="medrxiv;2021.09.07.21263228v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness"/>
  <meta name="DC.Identifier" content="10.1101/2021.09.07.21263228"/>
  <meta name="DC.Date" content="2022-02-16"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="DC.Rights" content="Â© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Repeated emergence of SARS-CoV-2 variants with increased fitness necessitates rapid detection and characterization of new lineages. To address this need, we developed PyR, a hierarchical Bayesian multinomial logistic regression model that infers relative prevalence of all viral lineages across geographic regions, detects lineages increasing in prevalence, and identifies mutations relevant to fitness. Applying PyR to all publicly available SARS-CoV-2 genomes, we identify numerous substitutions that increase fitness, including previously identified spike mutations and many non-spike mutations within the nucleocapsid and nonstructural proteins. PyR forecasts growth of new lineages from their mutational profile, identifies viral lineages of concern as they emerge, and prioritizes mutations of biological and public health concern for functional characterization.

One Sentence summary A Bayesian hierarchical model of all SARS-CoV-2 viral genomes predicts lineage fitness and identifies associated mutations.

### Competing Interest Statement

The authors have declared no competing interest.

### Clinical Trial

Study is based on SARS-CoV-2 genetic sequences publicly available at GISAID.org.

### Clinical Protocols

<https://github.com/broadinstitute/pyro-cov> 

### Funding Statement

This work was sponsored by the U.S. Centers for Disease Control and Prevention (BAA), as well as support from the Doris Duke Charitable Foundation (J.E.L.), the Howard Hughes Medical Institute (P.C.S.), and the Evergrande COVID-19 Response Fund Award from the Massachusetts Consortium on Pathogen Readiness (J.L.).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

The study was conducted using data from a public database (GISAID). No IRB approval is necessary.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data was gathered from other public resources. Data preprocessing scripts are open source.

<https://gisaid.org> 

<https://github.com/CSSEGISandData/COVID-19> 

<https://cov-lineages.org/>"/>
  <meta name="DC.Contributor" content="Fritz Obermeyer"/>
  <meta name="DC.Contributor" content="Martin Jankowiak"/>
  <meta name="DC.Contributor" content="Nikolaos Barkas"/>
  <meta name="DC.Contributor" content="Stephen F. Schaffner"/>
  <meta name="DC.Contributor" content="Jesse D. Pyle"/>
  <meta name="DC.Contributor" content="Lonya Yurkovetskiy"/>
  <meta name="DC.Contributor" content="Matteo Bosso"/>
  <meta name="DC.Contributor" content="Daniel J. Park"/>
  <meta name="DC.Contributor" content="Mehrtash Babadi"/>
  <meta name="DC.Contributor" content="Bronwyn L. MacInnis"/>
  <meta name="DC.Contributor" content="Jeremy Luban"/>
  <meta name="DC.Contributor" content="Pardis C. Sabeti"/>
  <meta name="DC.Contributor" content="Jacob E. Lemieux"/>
  <meta name="article:published_time" content="2022-02-16"/>
  <meta name="citation_title" content="Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Repeated emergence of SARS-CoV-2 variants with increased fitness necessitates rapid detection and characterization of new lineages. To address this need, we developed PyR<sub>0</sub>, a hierarchical Bayesian multinomial logistic regression model that infers relative prevalence of all viral lineages across geographic regions, detects lineages increasing in prevalence, and identifies mutations relevant to fitness. Applying PyR<sub>0</sub> to all publicly available SARS-CoV-2 genomes, we identify numerous substitutions that increase fitness, including previously identified spike mutations and many non-spike mutations within the nucleocapsid and nonstructural proteins. PyR<sub>0</sub> forecasts growth of new lineages from their mutational profile, identifies viral lineages of concern as they emerge, and prioritizes mutations of biological and public health concern for functional characterization.</p><h3>One Sentence summary</h3>
<p>A Bayesian hierarchical model of all SARS-CoV-2 viral genomes predicts lineage fitness and identifies associated mutations.</p>"/>
  <meta name="citation_journal_title" content="medRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="medrxiv;2021.09.07.21263228v2"/>
  <meta name="citation_id" content="2021.09.07.21263228v2"/>
  <meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v2"/>
  <meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v2.full"/>
  <meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2021.09.07.21263228"/>
  <meta name="citation_num_pages" content="59"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_firstpage" content="2021.09.07.21263228"/>
  <meta name="citation_author" content="Fritz Obermeyer"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_institution" content="Pyro Committee, Linux AI & Data Foundation"/>
  <meta name="citation_author_email" content="lemieux@broadinstitute.org"/>
  <meta name="citation_author_email" content="fobermey@broadinstitute.org"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2850-4991"/>
  <meta name="citation_author" content="Martin Jankowiak"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_institution" content="Pyro Committee, Linux AI & Data Foundation"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1724-0436"/>
  <meta name="citation_author" content="Nikolaos Barkas"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author" content="Stephen F. Schaffner"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University"/>
  <meta name="citation_author_institution" content="Department of Immunology and Infectious Diseases, Harvard T. H. Chan School of Public Health, Harvard University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6699-3568"/>
  <meta name="citation_author" content="Jesse D. Pyle"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0115-9921"/>
  <meta name="citation_author" content="Lonya Yurkovetskiy"/>
  <meta name="citation_author_institution" content="Program in Molecular Medicine, University of Massachusetts Medical School"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5078-493X"/>
  <meta name="citation_author" content="Matteo Bosso"/>
  <meta name="citation_author_institution" content="Program in Molecular Medicine, University of Massachusetts Medical School"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9344-9963"/>
  <meta name="citation_author" content="Daniel J. Park"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5191-1371"/>
  <meta name="citation_author" content="Mehrtash Babadi"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1829-8397"/>
  <meta name="citation_author" content="Bronwyn L. MacInnis"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_institution" content="Department of Immunology and Infectious Diseases, Harvard T. H. Chan School of Public Health, Harvard University"/>
  <meta name="citation_author_institution" content="Massachusetts Consortium on Pathogen Readiness"/>
  <meta name="citation_author" content="Jeremy Luban"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_institution" content="Program in Molecular Medicine, University of Massachusetts Medical School"/>
  <meta name="citation_author_institution" content="Massachusetts Consortium on Pathogen Readiness"/>
  <meta name="citation_author_institution" content="Ragon Institute of MGH, MIT, and Harvard"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5650-4054"/>
  <meta name="citation_author" content="Pardis C. Sabeti"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University"/>
  <meta name="citation_author_institution" content="Department of Immunology and Infectious Diseases, Harvard T. H. Chan School of Public Health, Harvard University"/>
  <meta name="citation_author_institution" content="Massachusetts Consortium on Pathogen Readiness"/>
  <meta name="citation_author_institution" content="Howard Hughes Medical Institute"/>
  <meta name="citation_author" content="Jacob E. Lemieux"/>
  <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard"/>
  <meta name="citation_author_institution" content="Division of Infectious Diseases, Massachusetts General Hospital"/>
  <meta name="citation_author_email" content="lemieux@broadinstitute.org"/>
  <meta name="citation_author_email" content="fobermey@broadinstitute.org"/>
  <meta name="citation_reference" content="N. G. Davies, S. Abbott, R. C. Barnard, C. I. Jarvis, A. J. Kucharski, J. D. Munday, C. A. B. Pearson, T. W. Russell, D. C. Tully, A. D. Washburne, T. Wenseleers, A. Gimma, W. Waites, K. L. M. Wong, K. van Zandvoort, J. D. Silverman, CMMID COVID-19 Working Group, COVID-19 Genomics UK (COG-UK) Consortium, K. Diaz-Ordaz, R. Keogh, R. M. Eggo, S. Funk, M. Jit, K. E. Atkins, W. J. Edmunds, Estimated transmissibility and impact of SARS-CoV-2 lineage B.1.1.7 in England. Science. 372 (2021), doi:10.1126/science.abg3055."/>
  <meta name="citation_reference" content="E. Volz, S. Mishra, M. Chand, J. C. Barrett, R. Johnson, L. Geidelberg, W. R. Hinsley, D. J. Laydon, G. Dabrera, Ã. OâToole, Others, Assessing transmissibility of SARS-CoV-2 lineage B. 1.1. 7 in England. Nature, 1â17 (2021)."/>
  <meta name="citation_reference" content="P. Stefanelli, F. Trentini, G. Guzzetta, V. Marziano, A. Mammone, P. Poletti, C. M. GranÃ©, M. Manica, M. del Manso, X. Andrianou, Others, Co-circulation of SARS-CoV-2 variants B. 1.1. 7 and P. 1. medRxiv (2021) (available at https://www.medrxiv.org/content/10.1101/2021.04.06.21254923v1.abstract)."/>
  <meta name="citation_reference" content="H. S. VÃ¶hringer, T. Sanderson, M. Sinnott, N. De Maio, T. Nguyen, R. Goater, F. Schwach, I. Harrison, J. Hellewell, C. Ariani, S. GonÃ§alves, D. Jackson, I. Johnston, A. W. Jung, C. Saint, J. Sillitoe, M. Suciu, N. Goldman, E. Birney, S. Funk, E. Volz, D. Kwiatkowski, M. Chand, I. Martincorena, J. C. Barrett, M. Gerstung, The Wellcome Sanger Institute Covid-19 Surveillance Team, The COVID-19 Genomics UK (COG-UK) Consortium, Genomic reconstruction of the SARS-CoV-2 epidemic across England from September 2020 to May 2021. bioRxiv (2021),, doi:10.1101/2021.05.22.21257633."/>
  <meta name="citation_reference" content="B. Korber, W. M. Fischer, S. Gnanakaran, H. Yoon, J. Theiler, W. Abfalterer, N. Hengartner, E. E. Giorgi, T. Bhattacharya, B. Foley, K. M. Hastie, Parker, D. G. Partridge, C. M. Evans, T. M. Freeman, T. I. de Silva, C. McDanal, L. G. Perez, H. Tang, A. Moon-Walker, S. P. Whelan, C. C. LaBranche, E. O. Saphire, D. C. Montefiori, A. Angyal, R. L. Brown, L. Carrilero, L. R. Green, D. C. Groves, K. J. Johnson, A. J. Keeley, B. B. Lindsey, P. J. Parsons, M. Raza, S. Rowland-Jones, N. Smith, R. M. Tucker, D. Wang, M. D. Wyles, Tracking changes in SARS-CoV-2 Spike: evidence that D614G increases infectivity of the COVID-19 virus. Cell (2020), doi:10.1016/j.cell.2020.06.043."/>
  <meta name="citation_reference" content="L. Yurkovetskiy, X. Wang, K. E. Pascal, C. Tomkins-Tinch, T. P. Nyalile, Y. Wang, A. Baum, W. E. Diehl, A. Dauphin, C. Carbone, K. Veinotte, S. B. Egri, S. F. Schaffner, J. E. Lemieux, J. B. Munro, A. Rafique, A. Barve, P. C. Sabeti, C. A. Kyratsous, N. V. Dudkina, K. Shen, J. Luban, Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant. Cell. 183, 739â751.e8 (2020)."/>
  <meta name="citation_reference" content="X. Deng, M. A. Garcia-Knight, M. M. Khalid, V. Servellita, C. Wang, M. K. Morris, A. Sotomayor-GonzÃ¡lez, D. R. Glasner, K. R. Reyes, A. S. Gliwa, N. P. Reddy, C. Sanchez San Martin, S. Federman, J. Cheng, J. Balcerek, J. Taylor, J. A. Streithorst, S. Miller, B. Sreekumar, P.-Y. Chen, U. Schulze-Gahmen, T. Y. Taha, J. M. Hayashi, C. R. Simoneau, G. R. Kumar, S. McMahon, P. V. Lidsky, Y. Xiao, P. Hemarajata, N. M. Green, A. Espinosa, C. Kath, M. Haw, J. Bell, J. K. Hacker, C. Hanson, D. A. Wadford, C. Anaya, D. Ferguson, P. A. Frankino, H. Shivram, L. F. Lareau, S. K. Wyman, M. Ott, R. Andino, C. Y. Chiu, Transmission, infectivity, and neutralization of a spike L452R SARS-CoV-2 variant. Cell. 184, 3426â3437.e8 (2021)."/>
  <meta name="citation_reference" content="T. N. Starr, A. J. Greaney, S. K. Hilton, D. Ellis, K. H. D. Crawford, A. S. Dingens, M. J. Navarro, J. E. Bowen, M. A. Tortorici, A. C. Walls, N. P. King, D. Veesler, J. D. Bloom, Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding. Cell. 182, 1295â1310.e20 (2020)."/>
  <meta name="citation_reference" content="B. Choi, M. C. Choudhary, J. Regan, J. A. Sparks, R. F. Padera, X. Qiu, I. H. Solomon, H.-H. Kuo, J. Boucau, K. Bowman, U. D. Adhikari, M. L. Winkler, A. A. Mueller, T. Y.-T. Hsu, M. Desjardins, L. R. Baden, B. T. Chan, B. D. Walker, M. Lichterfeld, M. Brigl, D. S. Kwon, S. Kanjilal, E. T. Richardson, A. H. Jonsson, G. Alter, A. K. Barczak, W. P. Hanage, X. G. Yu, G. D. Gaiha, M. S. Seaman, M. Cernadas, J. Z. Li, Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host. N. Engl. J. Med. 383, 2291â2293 (2020)."/>
  <meta name="citation_reference" content="A. J. Greaney, T. N. Starr, P. Gilchuk, S. J. Zost, E. Binshtein, A. N. Loes, S. K. Hilton, J. Huddleston, R. Eguia, K. H. D. Crawford, A. S. Dingens, R. S. Nargi, R. E. Sutton, N. Suryadevara, P. W. Rothlauf, Z. Liu, S. P. J. Whelan, R. H. Carnahan, J. E. Crowe Jr, J. D. Bloom, Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition. Cell Host Microbe. 29, 44â57.e9 (2021)."/>
  <meta name="citation_reference" content="A. Rambaut, E. C. Holmes, Ã. OâToole, V. Hill, J. T. McCrone, C. Ruis, L. du Plessis, O. G. Pybus, A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology. Nat Microbiol. 5, 1403â1407 (2020)."/>
  <meta name="citation_reference" content="F. Campbell, B. Archer, H. Laurenson-Schafer, Y. Jinnai, F. Konings, N. Batra, B. Pavlin, K. Vandemaele, M. D. Van Kerkhove, T. Jombart, O. Morgan, O. le Polain de Waroux, Increased transmissibility and global spread of SARS-CoV-2 variants of concern as at June 2021. Euro Surveill. 26 (2021), doi:10.2807/1560-7917.ES.2021.26.24.2100509."/>
  <meta name="citation_reference" content="GISAID Initiative and global contributors, EpiCoV(TM) human coronavirus 2019 database. GISAID (2020), (available at https://gisaid.org)."/>
  <meta name="citation_reference" content="S. Elbe, G. Buckland-Merrett, Data, disease and diplomacy: GISAIDâs innovative contribution to global health. Glob Chall. 1, 33â46 (2017)."/>
  <meta name="citation_reference" content="E. Bingham, J. P. Chen, M. Jankowiak, F. Obermeyer, N. Pradhan, T. Karaletsos, R. Singh, P. Szerlip, P. Horsfall, N. D. Goodman, Pyro: Deep universal probabilistic programming. J. Mach. Learn. Res. 20, 973â978 (2019)."/>
  <meta name="citation_reference" content="Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations. Virological (2020), (available at https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563)."/>
  <meta name="citation_reference" content="A. J. Greaney, T. N. Starr, J. D. Bloom, An antibody-escape calculator for mutations to the SARS-CoV-2 receptor-binding domain. bioRxiv (2021), doi:10.1101/2021.12.04.471236."/>
  <meta name="citation_reference" content="A. M. Syed, T. Y. Taha, T. Tabata, I. P. Chen, A. Ciling, M. M. Khalid, B. Sreekumar, P.-Y. Chen, J. M. Hayashi, K. M. Soczek, M. Ott, J. A. Doudna, Rapid assessment of SARS-CoV-2âevolved variants using virus-like particles. Science. 374, 1626â1632 (2021)."/>
  <meta name="citation_reference" content="L. Ferretti, A. Ledda, C. Wymant, L. Zhao, V. Ledda, L. Abeler-DÃ¶rner, M. Kendall, A. Nurtay, H.-Y. Cheng, T.-C. Ng, H.-H. Lin, R. Hinch, J. Masel, A. M. Kilpatrick, C. Fraser, The timing of COVID-19 transmission. bioRxiv (2020),, doi:10.1101/2020.09.04.20188516."/>
  <meta name="citation_reference" content="J. McBroome, B. Thornlow, A. S. Hinrichs, A. Kramer, N. De Maio, N. Goldman, D. Haussler, R. Corbett-Detig, Y. Turakhia, A daily-updated database and tools for comprehensive SARS-CoV-2 mutation-annotated trees. Mol. Biol. Evol. (2021), doi:10.1093/molbev/msab264."/>
  <meta name="citation_reference" content="Y. Turakhia, B. Thornlow, A. S. Hinrichs, N. De Maio, L. Gozashti, R. Lanfear, D. Haussler, R. Corbett-Detig, Ultrafast Sample placement on Existing tRees (UShER) enables real-time phylogenetics for the SARS-CoV-2 pandemic. Nat. Genet. 53, 809â816 (2021)."/>
  <meta name="citation_reference" content="S. Nersisyan, A. Zhiyanov, M. Shkurnikov, A. Tonevitsky, T-CoV: a comprehensive portal of HLA-peptide interactions affected by SARS-CoV-2 mutations. bioRxiv (2021), p. 2021.07.06.451227."/>
  <meta name="citation_reference" content="T. A. Hopf, C. P. I. SchÃ¤rfe, J. P. G. L. M. Rodrigues, A. G. Green, O. Kohlbacher, C. Sander, A. M. J. J. Bonvin, D. S. Marks, Sequence co-evolution gives 3D contacts and structures of protein complexes. Elife. 3 (2014), doi:10.7554/eLife.03430."/>
  <meta name="citation_reference" content="J. Frazer, P. Notin, M. Dias, A. Gomez, J. K. Min, K. Brock, Y. Gal, D. S. Marks, Disease variant prediction with deep generative models of evolutionary data. Nature. 599, 91â95 (2021)."/>
  <meta name="citation_reference" content="A. Paszke, S. Gross, S. Chintala, G. Chanan, E. Yang, Z. DeVito, Z. Lin, A. Desmaison, L. Antiga, A. Lerer, Automatic differentiation in PyTorch (2017), (available at https://openreview.net/pdf?id=BJJsrmfCZ)."/>
  <meta name="citation_reference" content="M. Gorinova, D. Moore, M. Hoffman, in Proceedings of the 37th International Conference on Machine Learning, H. D. Iii, A. Singh, Eds. (PMLR, 2020), vol. 119 of Proceedings of Machine Learning Research, pp. 3648â3657."/>
  <meta name="citation_reference" content="R. M. Neal, Slice sampling. The Annals of Statistics. 31 (2003),, doi:10.1214/aos/1056562461."/>
  <meta name="citation_reference" content="D. P. Kingma, J. Ba, Adam: A Method for Stochastic Optimization. arXiv [cs.LG] (2014), (available at http://arxiv.org/abs/1412.6980)."/>
  <meta name="citation_reference" content="L. Cappello, J. Kim, S. Liu, J. A. Palacios, Statistical Challenges in Tracking the Evolution of SARS-CoV-2. arXiv [stat.AP] (2021), (available at http://arxiv.org/abs/2108.13362)."/>
  <meta name="citation_reference" content="A. E. Lin, W. E. Diehl, Y. Cai, C. L. Finch, C. Akusobi, R. N. Kirchdoerfer, L. Bollinger, S. F. Schaffner, E. A. Brown, E. O. Saphire, K. G. Andersen, J. H. Kuhn, J. Luban, P. C. Sabeti, Reporter Assays for Ebola Virus Nucleoprotein Oligomerization, Virion-Like Particle Budding, and Minigenome Activity Reveal the Importance of Nucleoprotein Amino Acid Position 111. Viruses. 12 (2020), doi:10.3390/v12010105."/>
  <meta name="citation_reference" content="A. M. Syed, T. Y. Taha, M. M. Khalid, T. Tabata, I. P. Chen, B. Sreekumar, P.-Y. Chen, J. M. Hayashi, K. M. Soczek, M. Ott, J. A. Doudna, Rapid assessment of SARS-CoV-2 evolved variants using virus-like particles. bioRxiv (2021), p. 2021.08.05.455082."/>
  <meta name="citation_reference" content="M. M. Angelini, M. Akhlaghpour, B. W. Neuman, M. J. Buchmeier, Severe Acute Respiratory Syndrome Coronavirus Nonstructural Proteins 3, 4, and 6 Induce Double-Membrane Vesicles. mBio. 4 (2013),, doi:10.1128/mbio.00524-13."/>
  <meta name="citation_reference" content="R. L. Graham, A. C. Sims, S. M. Brockway, R. S. Baric, M. R. Denison, The nsp2 replicase proteins of murine hepatitis virus and severe acute respiratory syndrome coronavirus are dispensable for viral replication. J. Virol. 79, 13399â13411 (2005)."/>
  <meta name="citation_reference" content="I. Jungreis, R. Sealfon, M. Kellis, SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 Sarbecovirus genomes. Nat. Commun. 12, 2642 (2021)."/>
  <meta name="citation_reference" content="M. R. Islam, M. N. Hoque, M. S. Rahman, A. S. M. R. U. Alam, M. Akther, J. A. Puspo, S. Akter, M. Sultana, K. A. Crandall, M. A. Hossain, Genome-wide analysis of SARS-CoV-2 virus strains circulating worldwide implicates heterogeneity. Sci. Rep. 10, 14004 (2020)."/>
  <meta name="citation_reference" content="C. T. Cornillez-Ty, L. Liao, J. R. Yates 3rd, P. Kuhn, M. J. Buchmeier, Severe acute respiratory syndrome coronavirus nonstructural protein 2 interacts with a host protein complex involved in mitochondrial biogenesis and intracellular signaling. J. Virol. 83, 10314â10318 (2009)."/>
  <meta name="citation_reference" content="M. Gupta, C. M. Azumaya, M. Moritz, S. Pourmal, A. Diallo, G. E. Merz, G. Jang, M. Bouhaddou, A. Fossati, A. F. Brilot, D. Diwanji, E. Hernandez, N. Herrera, H. T. Kratochvil, V. L. Lam, F. Li, Y. Li, H. C. Nguyen, C. Nowotny, T. W. Owens, J. K. Peters, A. N. Rizo, U. Schulze-Gahmen, A. M. Smith, I. D. Young, Z. Yu, D. Asarnow, C. BillesbÃ¸lle, M. G. Campbell, J. Chen, K.-H. Chen, U. S. Chio, M. S. Dickinson, L. Doan, M. Jin, K. Kim, J. Li, Y.-L. Li, E. Linossi, Y. Liu, M. Lo, J. Lopez, K. E. Lopez, A. Mancino, F. R. Moss, M. D. Paul, K. I. Pawar, A. Pelin, T. H. Pospiech, C. Puchades, S. G. Remesh, M. Safari, K. Schaefer, M. Sun, M. C. Tabios, A. C. Thwin, E. W. Titus, R. Trenker, E. Tse, T. K. M. Tsui, F. Wang, K. Zhang, Y. Zhang, J. Zhao, F. Zhou, Y. Zhou, L. Zuliani-Alvarez, QCRG Structural Biology Consortium, D. A. Agard, Y. Cheng, J. S. Fraser, N. Jura, T. Kortemme, A. Manglik, D. R. Southworth, R. M. Stroud, D. L. Swaney, N. J. Krogan, A. Frost, O. S. Rosenberg, K. A. Verba, CryoEM and AI reveal a structure of SARS-CoV-2 Nsp2, a multifunctional protein involved in key host processes. bioRxiv (2021), doi:10.1101/2021.05.10.443524."/>
  <meta name="citation_reference" content="Z. Jin, X. Du, Y. Xu, Y. Deng, M. Liu, Y. Zhao, B. Zhang, X. Li, L. Zhang, C. Peng, Y. Duan, J. Yu, L. Wang, K. Yang, F. Liu, R. Jiang, X. Yang, T. You, X. Liu, X. Yang, F. Bai, H. Liu, X. Liu, L. W. Guddat, W. Xu, G. Xiao, C. Qin, Z. Shi, H. Jiang, Z. Rao, H. Yang, Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors. Nature. 582, 289â293 (2020)."/>
  <meta name="citation_reference" content="J. Osipiuk, S.-A. Azizi, S. Dvorkin, M. Endres, R. Jedrzejczak, K. A. Jones, S. Kang, R. S. Kathayat, Y. Kim, V. G. Lisnyak, S. L. Maki, V. Nicolaescu, C. A. Taylor, C. Tesar, Y.-A. Zhang, Z. Zhou, G. Randall, K. Michalska, S. A. Snyder, B. C. Dickinson, A. Joachimiak, Structure of papain-like protease from SARS-CoV-2 and its complexes with non-covalent inhibitors. Nat. Commun. 12, 743 (2021)."/>
  <meta name="citation_reference" content="H. S. Hillen, G. Kokic, L. Farnung, C. Dienemann, D. Tegunov, P. Cramer, Structure of replicating SARS-CoV-2 polymerase. Nature. 584, 154â156 (2020)."/>
  <meta name="citation_reference" content="L. Yan, J. Ge, L. Zheng, Y. Zhang, Y. Gao, T. Wang, Y. Huang, Y. Yang, S. Gao, M. Li, Z. Liu, H. Wang, Y. Li, Y. Chen, L. W. Guddat, Q. Wang, Z. Rao, Z. Lou, Cryo-EM Structure of an Extended SARS-CoV-2 Replication and Transcription Complex Reveals an Intermediate State in Cap Synthesis. Cell. 184, 184â193.e10 (2021)."/>
  <meta name="citation_reference" content="J. Chen, B. Malone, E. Llewellyn, M. Grasso, P. M. M. Shelton, P. D. B. Olinares, K. Maruthi, E. T. Eng, H. Vatandaslar, B. T. Chait, T. M. Kapoor, S. A. Darst, E. A. Campbell, Structural Basis for Helicase-Polymerase Coupling in the SARS-CoV-2 Replication-Transcription Complex. Cell. 182, 1560â1573.e13 (2020)."/>
  <meta name="citation_reference" content="Y. Chen, H. Cai, J. âan Pan, N. Xiang, P. Tien, T. Ahola, D. Guo, Functional screen reveals SARS coronavirus nonstructural protein nsp14 as a novel cap N7 methyltransferase. Proc. Natl. Acad. Sci. U. S. A. 106, 3484â3489 (2009)."/>
  <meta name="citation_reference" content="C. Liu, W. Shi, S. T. Becker, D. G. Schatz, B. Liu, Y. Yang, Structural basis of mismatch recognition by a SARS-CoV-2 proofreading enzyme. Science (2021), doi:10.1126/science.abi9310."/>
  <meta name="citation_reference" content="Y. Huang, C. Yang, X.-F. Xu, W. Xu, S.-W. Liu, Structural and functional properties of SARS-CoV-2 spike protein: potential antivirus drug development for COVID-19. Acta Pharmacol. Sin. 41, 1141â1149 (2020)."/>
  <meta name="citation_reference" content="J. Cubuk, J. J. Alston, J. J. Incicco, S. Singh, M. D. Stuchell-Brereton, M. D. Ward, M. I. Zimmerman, N. Vithani, D. Griffith, J. A. Wagoner, G. R. Bowman, K. B. Hall, A. Soranno, A. S. Holehouse, The SARS-CoV-2 nucleocapsid protein is dynamic, disordered, and phase separates with RNA. Nat. Commun. 12, 1936 (2021)."/>
  <meta name="citation_reference" content="Z. Chen, D. Pei, L. Jiang, Y. Song, J. Wang, H. Wang, D. Zhou, J. Zhai, Z. Du, B. Li, M. Qiu, Y. Han, Z. Guo, R. Yang, Antigenicity analysis of different regions of the severe acute respiratory syndrome coronavirus nucleocapsid protein. Clin. Chem. 50, 988â995 (2004)."/>
  <meta name="twitter:title" content="Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness"/>
  <meta name="twitter:site" content="@medrxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="twitter:description" content="Repeated emergence of SARS-CoV-2 variants with increased fitness necessitates rapid detection and characterization of new lineages. To address this need, we developed PyR, a hierarchical Bayesian multinomial logistic regression model that infers relative prevalence of all viral lineages across geographic regions, detects lineages increasing in prevalence, and identifies mutations relevant to fitness. Applying PyR to all publicly available SARS-CoV-2 genomes, we identify numerous substitutions that increase fitness, including previously identified spike mutations and many non-spike mutations within the nucleocapsid and nonstructural proteins. PyR forecasts growth of new lineages from their mutational profile, identifies viral lineages of concern as they emerge, and prioritizes mutations of biological and public health concern for functional characterization.

One Sentence summary A Bayesian hierarchical model of all SARS-CoV-2 viral genomes predicts lineage fitness and identifies associated mutations.

### Competing Interest Statement

The authors have declared no competing interest.

### Clinical Trial

Study is based on SARS-CoV-2 genetic sequences publicly available at GISAID.org.

### Clinical Protocols

<https://github.com/broadinstitute/pyro-cov> 

### Funding Statement

This work was sponsored by the U.S. Centers for Disease Control and Prevention (BAA), as well as support from the Doris Duke Charitable Foundation (J.E.L.), the Howard Hughes Medical Institute (P.C.S.), and the Evergrande COVID-19 Response Fund Award from the Massachusetts Consortium on Pathogen Readiness (J.L.).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

The study was conducted using data from a public database (GISAID). No IRB approval is necessary.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data was gathered from other public resources. Data preprocessing scripts are open source.

<https://gisaid.org> 

<https://github.com/CSSEGISandData/COVID-19> 

<https://cov-lineages.org/>"/>
  <meta name="og-title" property="og:title" content="Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness"/>
  <meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v2"/>
  <meta name="og-site-name" property="og:site_name" content="medRxiv"/>
  <meta name="og-description" property="og:description" content="Repeated emergence of SARS-CoV-2 variants with increased fitness necessitates rapid detection and characterization of new lineages. To address this need, we developed PyR, a hierarchical Bayesian multinomial logistic regression model that infers relative prevalence of all viral lineages across geographic regions, detects lineages increasing in prevalence, and identifies mutations relevant to fitness. Applying PyR to all publicly available SARS-CoV-2 genomes, we identify numerous substitutions that increase fitness, including previously identified spike mutations and many non-spike mutations within the nucleocapsid and nonstructural proteins. PyR forecasts growth of new lineages from their mutational profile, identifies viral lineages of concern as they emerge, and prioritizes mutations of biological and public health concern for functional characterization.

One Sentence summary A Bayesian hierarchical model of all SARS-CoV-2 viral genomes predicts lineage fitness and identifies associated mutations.

### Competing Interest Statement

The authors have declared no competing interest.

### Clinical Trial

Study is based on SARS-CoV-2 genetic sequences publicly available at GISAID.org.

### Clinical Protocols

<https://github.com/broadinstitute/pyro-cov> 

### Funding Statement

This work was sponsored by the U.S. Centers for Disease Control and Prevention (BAA), as well as support from the Doris Duke Charitable Foundation (J.E.L.), the Howard Hughes Medical Institute (P.C.S.), and the Evergrande COVID-19 Response Fund Award from the Massachusetts Consortium on Pathogen Readiness (J.L.).

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

The study was conducted using data from a public database (GISAID). No IRB approval is necessary.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data was gathered from other public resources. Data preprocessing scripts are open source.

<https://gisaid.org> 

<https://github.com/CSSEGISandData/COVID-19> 

<https://cov-lineages.org/>"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="citation_date" content="2022-02-16"/>
  <meta name="description" content="medRxiv - The Preprint Server for Health Sciences"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v2"/>
  <link rel="shortlink" href="https://www.medrxiv.org/node/477231"/>
  <title>
   Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness | medRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__MF4P_i8lKWP7DQR7GmjDAVTRBKLnzxvs1Dvton-Awi0__yi5OflfckG_G8-Li4dDQQ3wO9cGIcdPx_YQ7ZZDuj5w__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__5mxrBK4DuyvC2MR5irx_TzVn9c8vr4vIjmBXjunHfxE__rtQnXv8GOc-AsynbhzSlQBD_2lVxAkB6XEFUCK72qXc__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all"/>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7y8urXFaW3UeBluIdimW78sZdJRGZ4aQffktTtisDa0"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"477231","apath":"\/medrxiv\/early\/2022\/02\/16\/2021.09.07.21263228.atom","pisa":"medrxiv;2021.09.07.21263228v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"medrxiv;2021.09.07.21263228v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2021.09.07.21263228v2","title":"Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness","identifier":"node\/477231"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DZyBh7pAucF\\\\\\\/Yj46C4uDmMWQ3w8QusVXRBgm+\\\\\\\/Lj1NkQ0YwPlYzgbLDAmB77EUY\\\\\\\/7KvPP2XkNVuLa+KaV9KKnSubb507VaT3wookqUKgksw5Zik9gs07NixVboD+\\\\\\\/hPkq02LWtlpeBWX\\\\\\\/hIs+idHnjevU9ifSekGR0uPbUZnj6yDeYNJQyUVm\\\\\\\/FgjIZgSYgqOFRb+bl7OG0D350d+2HRWb0yxUkHdngnBGu4dGKoZiA\\\\\\\/hyDZ\\\\\\\/tN+8o4n\\\\\\\/mQZIoJFdCcCWjqIFwUDmBTXmXv5nq+ngemjYBoXmqEMh3MAGJCnBTr4\\\\\\\/NP58qZ+pF++rY3qbFg+FlWbBNe9u8ISP4b7iIj8Ud6lfeSAZx8OUBOaAAzX6fVaJVstQueEJByl\\\\\\\/47+\\\\\\\/l\\\\\\\/mgEL1v29AWYbJJdnfwx0xq61jpfeDPPjZiNQpJu+BODzEe0GQQ607qgPQkVvxdS39YbVoTh0DwUvCO8MpI1t17+KTCAG2GoSFmORHFqRxefTlg\\\\\\\/brj1XiI3UAYPspHt+VEGscwtWDrY2c1wNxeHOQyfcElpARnDhsr7h\\\\\\\/ixBjgtHTqI0y9jJtJNvuqm\\\\\\\/s8tVY053cVAFmI6AJVUWlAQh+CDGbnPTwqNgVPmw+YH7gpwL7mtphDCNKJetszKu+WwbWcOl31MG2Ceakl7VK2GbaU2TgK4ly8diXUoeske5Q\\\\\\\/7EsXF1+Z2p6lmbsdX5I4Fy5C1L4vme9jHalLK\\\\\\\/gMWSyZ1ni7M66eVeApklj\\\\\\\/VBmKnQXJ8oL0G3wvJoAmCdZuQJY0Y6gzreh\\\\\\\/pDj1Ad96CkF4Yb6k9tClZXZZTV9cvaHWKI0BL+DZSXqH5PHCmLCwiNwm3JkFq634jAltlKCBDxFEkzVm7tGi6ClNVJ6Q6mZVxq8qUABYzWvrAIvu1TJll8VDG1HTJ+0pdM0AMFEEJLVxsnv7nOqWC0wvyWkQeqemXEBW9b2Lv0cYIswxQMRYhO+JLo3w6vWcT9BwbHL\\\\\\\/Zfl3FzsxWUYsbSRNKwsQGdJmGKGMin8yFEkivRo2ikx4p4iseijjYheZ5DtEXHqcvLe9BFjU3keMkry2qfgejY1uiO4ZRRvr6sdZESd+qeJQU7aGvPmvMzjpCWlGRJoonq0Sey00rIhPY2ctrENFySAVTLzIA9rbQvvzFyshOIu2wj7T12S2nQqDl4oc+3H1RmRpTo0QHrBRZMmxSBSG6EQiYk1BGeicS1qq3J4nsSw8ugHdjI7UrZqNll65q9SVp9bvrrVboDOWDF\\\\\\\/EJ\\\\\\\/J6cVBHlU0glYvKJlpqgpgtw24uIpgZ6qX2mfVAJWvgFz0YO90r6H\\\\\\\/93q9A1eY+E14Pmo+QuPooMJdr4nNkLToZhwD9p3yaW0haShssR63IzMSzZUszl5YUaYzZ7+XsukQGpA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/lj303uqrklkfitmmdCgQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e80092cbc6d78d85f1bc6c5f993b2cd50708b3c4551a4c73cd972e7539268253\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2021.09.07.21263228v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-477231 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2021.09.07.21263228v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1650446274">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1650446274" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1971598929" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-eMDKxqVA_V55x8fhDu7-45cl-6Ya9eIhCZT9cMjtHEk"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node477231" data-node-nid="477231" id="node-477231--2220177150" data-pisa="medrxiv;2021.09.07.21263228v2" data-pisa-master="medrxiv;2021.09.07.21263228" data-apath="/medrxiv/early/2022/02/16/2021.09.07.21263228.atom" data-hw-author-tooltip-instance="">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip">
                   <h1 class="highwire-cite-title" id="page-title">
                    Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-2850-4991" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Fritz
                      </span>
                      <span class="nlm-surname">
                       Obermeyer
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-1724-0436" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Martin
                      </span>
                      <span class="nlm-surname">
                       Jankowiak
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Nikolaos
                      </span>
                      <span class="nlm-surname">
                       Barkas
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0001-6699-3568" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Stephen F.
                      </span>
                      <span class="nlm-surname">
                       Schaffner
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0003-0115-9921" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jesse D.
                      </span>
                      <span class="nlm-surname">
                       Pyle
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0001-5078-493X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Lonya
                      </span>
                      <span class="nlm-surname">
                       Yurkovetskiy
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0001-9344-9963" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Matteo
                      </span>
                      <span class="nlm-surname">
                       Bosso
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0001-5191-1371" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Daniel J.
                      </span>
                      <span class="nlm-surname">
                       Park
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8">
                      <a href="http://orcid.org/0000-0003-1829-8397" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Mehrtash
                      </span>
                      <span class="nlm-surname">
                       Babadi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="9">
                      <span class="nlm-given-names">
                       Bronwyn L.
                      </span>
                      <span class="nlm-surname">
                       MacInnis
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10">
                      <a href="http://orcid.org/0000-0001-5650-4054" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jeremy
                      </span>
                      <span class="nlm-surname">
                       Luban
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="11">
                      <span class="nlm-given-names">
                       Pardis C.
                      </span>
                      <span class="nlm-surname">
                       Sabeti
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="12">
                      <span class="nlm-given-names">
                       Jacob E.
                      </span>
                      <span class="nlm-surname">
                       Lemieux
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2021.09.07.21263228
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-477231--2220177150" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Fritz Obermeyer
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Pyro Committee, Linux AI & Data Foundation
                       </span>
                       ; 548 Market St San Francisco, California 94104
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Fritz%2BObermeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Obermeyer%20F&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AFritz%2BObermeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-2850-4991" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Fritz Obermeyer
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:lemieux@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">
                        lemieux@broadinstitute.org
                       </a>
                       <a href="mailto:fobermey@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">
                        fobermey@broadinstitute.org
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Martin Jankowiak
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Pyro Committee, Linux AI & Data Foundation
                       </span>
                       ; 548 Market St San Francisco, California 94104
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Martin%2BJankowiak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Jankowiak%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMartin%2BJankowiak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-1724-0436" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Martin Jankowiak
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Nikolaos Barkas
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nikolaos%2BBarkas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Barkas%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ANikolaos%2BBarkas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Stephen F. Schaffner
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Organismic and Evolutionary Biology, Harvard University
                       </span>
                       ; Cambridge, MA 02138,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Immunology and Infectious Diseases, Harvard T. H. Chan School of Public Health, Harvard University
                       </span>
                       ; Boston, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Stephen%2BF.%2BSchaffner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Schaffner%20SF&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AStephen%2BF.%2BSchaffner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-6699-3568" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Stephen F. Schaffner
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Jesse D. Pyle
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jesse%2BD.%2BPyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Pyle%20JD&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJesse%2BD.%2BPyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0115-9921" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jesse D. Pyle
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Lonya Yurkovetskiy
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Program in Molecular Medicine, University of Massachusetts Medical School
                       </span>
                       ; Worcester, MA 01605,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Lonya%2BYurkovetskiy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Yurkovetskiy%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALonya%2BYurkovetskiy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5078-493X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Lonya Yurkovetskiy
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Matteo Bosso
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Program in Molecular Medicine, University of Massachusetts Medical School
                       </span>
                       ; Worcester, MA 01605,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Matteo%2BBosso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bosso%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMatteo%2BBosso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-9344-9963" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Matteo Bosso
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Daniel J. Park
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Daniel%2BJ.%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Park%20DJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ADaniel%2BJ.%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5191-1371" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Daniel J. Park
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Mehrtash Babadi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mehrtash%2BBabadi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Babadi%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMehrtash%2BBabadi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-1829-8397" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Mehrtash Babadi
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Bronwyn L. MacInnis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Immunology and Infectious Diseases, Harvard T. H. Chan School of Public Health, Harvard University
                       </span>
                       ; Boston, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Massachusetts Consortium on Pathogen Readiness
                       </span>
                       ; Boston, MA 02115,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Bronwyn%2BL.%2BMacInnis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=MacInnis%20BL&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ABronwyn%2BL.%2BMacInnis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">
                     Jeremy Luban
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Program in Molecular Medicine, University of Massachusetts Medical School
                       </span>
                       ; Worcester, MA 01605,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Massachusetts Consortium on Pathogen Readiness
                       </span>
                       ; Boston, MA 02115,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Ragon Institute of MGH, MIT, and Harvard
                       </span>
                       ; 400 Technology Square, Cambridge, MA 02139,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jeremy%2BLuban%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Luban%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJeremy%2BLuban%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5650-4054" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jeremy Luban
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-11">
                    <div class="author-tooltip-name">
                     Pardis C. Sabeti
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Organismic and Evolutionary Biology, Harvard University
                       </span>
                       ; Cambridge, MA 02138,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Immunology and Infectious Diseases, Harvard T. H. Chan School of Public Health, Harvard University
                       </span>
                       ; Boston, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Massachusetts Consortium on Pathogen Readiness
                       </span>
                       ; Boston, MA 02115,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        Howard Hughes Medical Institute
                       </span>
                       ; 4000 Jones Bridge Rd, Chevy Chase, MD 20815,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Pardis%2BC.%2BSabeti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sabeti%20PC&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3APardis%2BC.%2BSabeti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-12">
                    <div class="author-tooltip-name">
                     Jacob E. Lemieux
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Broad Institute of MIT and Harvard
                       </span>
                       ; 415 Main Street, Cambridge, MA 02142,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        9
                       </span>
                       <span class="nlm-institution">
                        Division of Infectious Diseases, Massachusetts General Hospital
                       </span>
                       ; Boston, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jacob%2BE.%2BLemieux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lemieux%20JE&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJacob%2BE.%2BLemieux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:lemieux@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">
                        lemieux@broadinstitute.org
                       </a>
                       <a href="mailto:fobermey@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">
                        fobermey@broadinstitute.org
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2021.09.07.21263228v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:477231" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:477231/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.07.21263228v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:477231" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:477231/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.07.21263228v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:477231" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:477231/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.07.21263228v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:477231" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:477231/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.07.21263228v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:477231" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:477231/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.07.21263228v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:477231" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:477231/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2021.09.07.21263228v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:477231" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:477231/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             Repeated emergence of SARS-CoV-2 variants with increased fitness necessitates rapid detection and characterization of new lineages. To address this need, we developed PyR
                             <sub>
                              0
                             </sub>
                             , a hierarchical Bayesian multinomial logistic regression model that infers relative prevalence of all viral lineages across geographic regions, detects lineages increasing in prevalence, and identifies mutations relevant to fitness. Applying PyR
                             <sub>
                              0
                             </sub>
                             to all publicly available SARS-CoV-2 genomes, we identify numerous substitutions that increase fitness, including previously identified spike mutations and many non-spike mutations within the nucleocapsid and nonstructural proteins. PyR
                             <sub>
                              0
                             </sub>
                             forecasts growth of new lineages from their mutational profile, identifies viral lineages of concern as they emerge, and prioritizes mutations of biological and public health concern for functional characterization.
                            </p>
                            <div id="sec-1" class="subsection">
                             <p id="p-4">
                              <strong>
                               One Sentence summary
                              </strong>
                              A Bayesian hierarchical model of all SARS-CoV-2 viral genomes predicts lineage fitness and identifies associated mutations.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Main Text
                            </h2>
                            <p id="p-20">
                             The SARS-CoV-2 pandemic has been characterized by repeated waves of cases driven by the emergence of new lineages with higher fitness, where fitness encompasses any trait that affects the lineageâs growth, including its basic reproduction number (R
                             <sub>
                              0
                             </sub>
                             ), ability to evade existing immunity, and generation time. Rapidly identifying such lineages as they emerge and accurately forecasting their dynamics is critical for guiding outbreak response. Doing so effectively would benefit from the ability to interrogate the entirety of the global SARS-CoV-2 genomic dataset. The large size (currently over 7.5 million virus genomes) and geographic and temporal variability of the available data present significant challenges that will only become greater as more viruses are sequenced. Current phylogenetic approaches are computationally inefficient on datasets with more than â¼5000 samples and take days to run at that scale. Ad hoc methods to estimate the relative fitness of particular SARS-CoV-2 lineages are a computationally efficient alternative (
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             â
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ), but have typically relied on models in which one or two lineages of interest are compared to all others and do not capture the complex dynamics of multiple co-circulating lineages.
                            </p>
                            <p id="p-21">
                             Furthermore, estimates of relative fitness based on lineage frequency data alone (
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             â
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ) do not take advantage of additional statistical power that can be gained from analyzing the independent appearance and growth of the same mutation in multiple lineages. Performing a mutation-based analysis of lineage prevalence has the additional advantage of identifying specific genetic determinants of a lineageâs phenotype, which is critically important both for understanding the biology of transmission and pathogenesis and for predicting the phenotype of new lineages. The SARS-CoV-2 pandemic has already been dominated by several genetic changes of functional and epidemiological importance, including the spike (S) D614G mutation that is associated with higher SARS-CoV-2 loads (
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ). In addition, mutations found in Variants of Concern (VoC), such as S:N439R, S:N501Y, and S:E484K, have been linked, respectively, to increased transmissibility (
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ), enhanced binding to ACE2 (
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ), and antibody escape (
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ). Despite these successes, identifying functionally important mutations in the context of a large background of genetic variants of little or no phenotypic consequence remains challenging.
                            </p>
                            <p id="p-22">
                             We set out to formulate a principled approach to modeling the relative fitness of SARS-CoV-2 lineages, estimating their growth as a linear combination of the effects of individual mutations. We developed PyR
                             <sub>
                              0
                             </sub>
                             , a hierarchical Bayesian regression model that enables scalable analysis of the complete set of publicly available SARS-CoV-2 genomes, and that could be applied to any viral genomic dataset and to other phenotypes. The model, which is summarized in
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Figure 1A
                             </a>
                             and described in detail in the supplemental note, avoids the complexity of full phylogenetic inference by first clustering genomes by genetic similarity (refining PANGO lineages (
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             )), and then estimating the incremental effect on growth rate of each of the most common amino acid changes on the lineages in which they appear. By regressing growth rate as a function of genome sequence, the model shares statistical strength among genetically similar lineages without explicitly relying on phylogeny. By modeling only the multinomial proportion of different lineages rather than the absolute number of samples for each lineage (
                             <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ,
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ), and by doing so within 14-day intervals in 1560 globally distributed geographic regions, the model achieves robustness to a number of sources of bias that affect all lineages, across regions, and over time, including differences in data collection and changes in transmission due to such factors as social behavior, public health policy, and vaccination.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F1.large.jpg?width=800&height=600&carousel=1" title="A. Overview of the PyR0 analysis pipeline. After clustering UShERâs mutation annotated tree, sequence data are used to construct spatio-temporal lineage prevalence counts ytpc and amino acid substitution covariates Xcf. Pyro is used to fit a Bayesian multivariate logistic multinomial regression model to ytpc and Xcf. B. Relative fitness versus date of lineage emergence. Circle size is proportional to cumulative case count inferred from lineage proportion estimates and confirmed case counts. Inset table lists the 10 fittest lineages inferred by the model. R/RA is the fold increase in relative fitness over the Wuhan (A) lineage, assuming a fixed generation time of 5.5 days." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A.</strong> Overview of the PyR<sub>0</sub> analysis pipeline. After clustering UShERâs mutation annotated tree, sequence data are used to construct spatio-temporal lineage prevalence counts y<sub>tpc</sub> and amino acid substitution covariates X<sub>cf</sub>. Pyro is used to fit a Bayesian multivariate logistic multinomial regression model to y<sub>tpc</sub> and X<sub>cf</sub>. <strong>B.</strong> Relative fitness versus date of lineage emergence. Circle size is proportional to cumulative case count inferred from lineage proportion estimates and confirmed case counts. Inset table lists the 10 fittest lineages inferred by the model. R/R<sub>A</sub> is the fold increase in relative fitness over the Wuhan (A) lineage, assuming a fixed generation time of 5.5 days.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F1.medium.gif" width="440" height="394"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F1.medium.gif" width="440" height="394"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1.
                              </span>
                              <p id="p-23" class="first-child">
                               <strong>
                                A.
                               </strong>
                               Overview of the PyR
                               <sub>
                                0
                               </sub>
                               analysis pipeline. After clustering UShERâs mutation annotated tree, sequence data are used to construct spatio-temporal lineage prevalence counts y
                               <sub>
                                tpc
                               </sub>
                               and amino acid substitution covariates X
                               <sub>
                                cf
                               </sub>
                               . Pyro is used to fit a Bayesian multivariate logistic multinomial regression model to y
                               <sub>
                                tpc
                               </sub>
                               and X
                               <sub>
                                cf
                               </sub>
                               .
                              </p>
                              <p id="p-24">
                               <strong>
                                B.
                               </strong>
                               Relative fitness versus date of lineage emergence. Circle size is proportional to cumulative case count inferred from lineage proportion estimates and confirmed case counts. Inset table lists the 10 fittest lineages inferred by the model. R/R
                               <sub>
                                A
                               </sub>
                               is the fold increase in relative fitness over the Wuhan (A) lineage, assuming a fixed generation time of 5.5 days.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-25">
                             We fit PyR
                             <sub>
                              0
                             </sub>
                             to 6,466,300 SARS-CoV-2 genomes available on GISAID (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ) as of January 20, 2022, in a model that contained 1544 PANGO lineages and 2904 nonsynonymous mutations. The output of the model is a posterior distribution for the relative fitness (exponential growth rate) of each lineage and for the contribution to the fitness from each mutation. Fitting this large model is computationally challenging, so we used stochastic variational inference, an approximate inference method that reduced our task to solving a 75-million-dimensional optimization problem on a GPU. Inference was implemented in the Pyro (
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ) probabilistic programming framework (see Supplemental Materials). The trained model can be used to infer lineage fitness, predict the fitness of completely new lineages, forecast future lineage proportions, and estimate the effects of individual mutations on fitness.
                            </p>
                            <p id="p-26">
                             The modelâs lineage fitness estimates (
                             <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                              Figure 1B
                             </a>
                             ) show a modest upward trend over time among all lineages, accompanied by numerous lineages with dramatically higher fitness. Sensitivity analyses revealed broad consistency of fitness estimates across spatial data subsets (
                             <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                              Figure S1
                             </a>
                             ). The upward trend may in part reflect an upward bias caused by the lineage assignment process, as can be seen in simulation studies (
                             <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                              Figure S2
                             </a>
                             ), but the high tail of the distribution exhibits elevated fitness values far in excess of this trend. The rate of increase in fitness was not constant between the emergence of the virus into human populations in late 2019 and early 2022. Rather, periods of rapid evolution in fitness occurred and heralded new waves of increase in case counts (
                             <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                              Figure 1B
                             </a>
                             and
                             <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                              Figure 2CDE
                             </a>
                             ). The model correctly inferred BA.2 to have the highest fitness to date, 8.9-fold (95% CI, 8.6-9.2) higher than the original A lineage (
                             <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                              Figure 1B
                             </a>
                             inset). Similar fitness was estimated for other Omicron sub-lineages BA.1 and BA.1.1 (
                             <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                              Figure 1B
                             </a>
                             ). These fitness estimates, obtained in mid January 2022, predict B.1.1.529 and sublineages (collectively called Omicron in the WHO classification) will continue to displace other lineages, including the previously dominant Delta (
                             <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                              Figure S3
                             </a>
                             ). While PANGO lineages facilitate communication by providing a stable nomenclature, we observed some PANGO lineages with multiple successive peaks in some regions, which could not be accounted for by a multivariate logistic growth model. We therefore algorithmically refined the 1544 PANGO lineages into 3000 finer clusters, and found our model identified significant heterogeneity within some PANGO lineages (
                             <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                              Figure S4
                             </a>
                             ). Notably, B.1.1 displayed the greatest variability among lineages, followed by B.1.
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F2.large.jpg?width=800&height=600&carousel=1" title="A. Infectivity relative to WT of lentiviral vectors pseudotyped with the indicated Spike mutants. Target cells were HEK293T cells expressing ACE2 and TMPRSS2 transgenes. The genetic background of the Spike was Wuhan-Hu-1 bearing D614G. Red bars were significantly different from WT (adjusted p values shown). Black bars were not significantly different from WT. B. For the 1701 SARS-CoV-2 clusters with at least one amino acid substitution in the RBD domain we compare: i) the PyR0 prediction for the contribution to Î log R from RBD substitutions only; to ii) antibody binding computed using the antibody-escape calculator in (17). The escape calculator is based on an intuitive non-linear model parameterized using deep mutational scanning data for 33 neutralizing antibodies elicited by SARS-CoV-2. PyR0 predictions exhibit high (Spearman) correlation with predictions from Greaney et al. C-E. We dissect PyR0 Î log R estimates into S-gene (C), RBD (D), and non-S-gene (E) contributions for 3000 SARS-CoV-2 clusters (blue dots). The horizontal axis corresponds to the date at which each cluster first emerged. Red squares denote the median Î log R within each monthly bin. The increased importance of S-gene mutations (notably in the RBD) over non-S-gene mutations starting around November 2021 is apparent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A.</strong> Infectivity relative to WT of lentiviral vectors pseudotyped with the indicated Spike mutants. Target cells were HEK293T cells expressing ACE2 and TMPRSS2 transgenes. The genetic background of the Spike was Wuhan-Hu-1 bearing D614G. Red bars were significantly different from WT (adjusted p values shown). Black bars were not significantly different from WT. <strong>B.</strong> For the 1701 SARS-CoV-2 clusters with at least one amino acid substitution in the RBD domain we compare: i) the PyR<sub>0</sub> prediction for the contribution to Î log R from RBD substitutions only; to ii) antibody binding computed using the antibody-escape calculator in (17). The escape calculator is based on an intuitive non-linear model parameterized using deep mutational scanning data for 33 neutralizing antibodies elicited by SARS-CoV-2. PyR<sub>0</sub> predictions exhibit high (Spearman) correlation with predictions from Greaney et al. <strong>C-E</strong>. We dissect PyR<sub>0</sub> Î log R estimates into S-gene (C), RBD (D), and non-S-gene (E) contributions for 3000 SARS-CoV-2 clusters (blue dots). The horizontal axis corresponds to the date at which each cluster first emerged. Red squares denote the median Î log R within each monthly bin. The increased importance of S-gene mutations (notably in the RBD) over non-S-gene mutations starting around November 2021 is apparent.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F2.medium.gif" width="440" height="348"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F2.medium.gif" width="440" height="348"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 2.
                              </span>
                              <p id="p-27" class="first-child">
                               <strong>
                                A.
                               </strong>
                               Infectivity relative to WT of lentiviral vectors pseudotyped with the indicated Spike mutants. Target cells were HEK293T cells expressing ACE2 and TMPRSS2 transgenes. The genetic background of the Spike was Wuhan-Hu-1 bearing D614G. Red bars were significantly different from WT (adjusted p values shown). Black bars were not significantly different from WT.
                               <strong>
                                B.
                               </strong>
                               For the 1701 SARS-CoV-2 clusters with at least one amino acid substitution in the RBD domain we compare: i) the PyR
                               <sub>
                                0
                               </sub>
                               prediction for the contribution to Î log R from RBD substitutions only; to ii) antibody binding computed using the antibody-escape calculator in (
                               <a href="https://paperpile.com/c/2OlJ78/O3NS">
                                17
                               </a>
                               ). The escape calculator is based on an intuitive non-linear model parameterized using deep mutational scanning data for 33 neutralizing antibodies elicited by SARS-CoV-2. PyR
                               <sub>
                                0
                               </sub>
                               predictions exhibit high (Spearman) correlation with predictions from Greaney et al.
                               <strong>
                                C-E
                               </strong>
                               . We dissect PyR
                               <sub>
                                0
                               </sub>
                               Î log R estimates into S-gene (C), RBD (D), and non-S-gene (E) contributions for 3000 SARS-CoV-2 clusters (blue dots). The horizontal axis corresponds to the date at which each cluster first emerged. Red squares denote the median Î log R within each monthly bin. The increased importance of S-gene mutations (notably in the RBD) over non-S-gene mutations starting around November 2021 is apparent.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-28">
                             We found that the model would have provided early warning of the rise of VoCs had it been routinely applied to SARS-CoV-2 samples, highlighting the benefit of timely publication of genomic data. For example, PyR
                             <sub>
                              0
                             </sub>
                             would have forecast the coming dominance of B.1.1.7 in late November 2020 (
                             <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                              Figure S5A
                             </a>
                             ), while the first models forecasting its rapid rise were published in mid December 2020 (
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ). Similar predictions would have been available for BA.1 by early December 2021 (
                             <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                              Figure S5B
                             </a>
                             ,
                             <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                              S6
                             </a>
                             ) and for AY.4 by May 2021 (
                             <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                              Figure S5C
                             </a>
                             ). Likewise the elevated fitness of BA.2 was identified by mid December 2021 on the basis of 76 observed sequences (
                             <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                              Figure S6
                             </a>
                             ). While variant-specific models were accurate and useful (
                             <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ) in predicting the rise of these lineages, each modeling effort was specific to a particular lineage and geographic region; by contrast, PyR
                             <sub>
                              0
                             </sub>
                             âs global approach provides similar early detection while also offering automated, rapid, and unbiased consideration of all variants and lineages, together with ranking based on relative fitness. When we tested the modelâs predictive ability (
                             <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                              Figure S5
                             </a>
                             ), we found that forecasts were reliable for 1-2 months into the future, when they tended to be disrupted by the emergence of a completely new strain (
                             <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                              Table S1
                             </a>
                             ,
                             <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                              Figure S7
                             </a>
                             ). Remarkably, the accuracy of forecasts stabilized typically within two weeks after the emergence of a new competitive lineage in a region (
                             <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                              Figure S7
                             </a>
                             ).
                            </p>
                            <p id="p-29">
                             By basing fitness estimates on the contributions of individual mutations, PyR
                             <sub>
                              0
                             </sub>
                             can forecast the fitness of novel or hypothetical lineages using their mutational profiles alone. This is possible with SARS-CoV-2 because of the high rate of convergent evolution (
                             <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                              Table 1
                             </a>
                             ,
                             <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                              Figure S8
                             </a>
                             ), which allows the model to infer the fitness of new constellations of mutations based on the trajectories of other lineages in which they have previously emerged. This predictive capability is highly desirable from a public health standpoint because forecasts are available as soon as sequences from new lineages appear. To test the reliability of this kind of estimate, we fit leave-one-out estimators on subsets of the dataset with entire PANGO lineages removed (
                             <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                              Figure S9
                             </a>
                             ). These estimators showed excellent agreement with estimators based on the observed behavior of the lineages, and they were also more accurate than naive phylogenetic estimators that assume the fitness of each new strain is equal to its parent lineageâs fitness (Pearsonâs Ï = 0.983, after correcting for parent fitness,
                             <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                              Figure S9
                             </a>
                             ). These results demonstrate the feasibility of this kind of estimate using the simplest possible linear-additive model, and provide a foundation for future research for more complex modeling that includes effects such as epistasis between mutations and migration across regions.
                            </p>
                            <div id="T1" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/478256/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/478256/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/478256" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Table 1:
                              </span>
                              <p id="p-30" class="first-child">
                               Amino acid substitutions most significantly associated with increased fitness. Significance is defined as posterior mean / posterior standard deviation. Fitness is per 5.5 days (estimated generation time of the Wuhan (A) lineage (
                               <a href="https://paperpile.com/c/2OlJ78/ojrOI%2BKFmwo">
                                1, 19
                               </a>
                               )). Final column: number of PANGO lineages in which each substitution emerged independently.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-31">
                             Unbiased, genome-wide estimates of the effect of SARS-CoV-2 mutations on fitness also provide a powerful tool for better understanding the biology of fitness. Our model allowed us to estimate the contribution of 2904 amino acid substitutions (
                             <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                              Figure 3a
                             </a>
                             ,
                             <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                              Table 1
                             </a>
                             ) to lineage fitness and to rank them by inferred statistical significance (
                             <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                              Figure S10
                             </a>
                             ). Cross-validation confirmed that these results replicate across different geographic regions (
                             <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                              Figure S11
                             </a>
                             ). The highest concentrations of fitness-associated mutations were found in the S, N, and the ORF1 polyprotein genes (ORF1a and ORF1b,
                             <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                              Figures 3A-B
                             </a>
                             ,
                             <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                              S12
                             </a>
                             -
                             <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                              S13
                             </a>
                             ). Using spatial autocorrelation as a measure of spatial structure, we found evidence of functional hotspots in the S, N, ORF7a, ORF3a, and ORF1a genes (
                             <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                              Table S2
                             </a>
                             ). Within S, there were three hotspots of fitness-enhancing mutations, each within a defined functional region: the N-terminal domain, the receptor-binding domain (RBD), and the furin-cleavage site (
                             <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                              Figure 3B
                             </a>
                             ). We assessed mutational enrichment in the top-ranked set of mutations and identified an enrichment for lysine to asparagine mutations in the S gene (
                             <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                              Figure S14C
                             </a>
                             ). We visualized top scoring mutations within atomic structures for the spike protein (
                             <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                              Figure 3D-E
                             </a>
                             ), the nucleocapsidâs N-terminal domain (
                             <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                              Figure 3F
                             </a>
                             ), the polymerase (
                             <a id="xref-fig-18-1" class="xref-fig" href="#F18">
                              Figure S15
                             </a>
                             ), and two proteases (
                             <a id="xref-fig-19-1" class="xref-fig" href="#F19">
                              Figure S16
                             </a>
                             ). Many of the top mutations in the S gene occurred in the receptor binding domain (RBD) making direct contacts with the ACE2 receptor, including K417N/T and E484K (
                             <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                              Figures 3D-E
                             </a>
                             ). Two top-ranked mutations, T478K and S477N, occur in a flexible loop adjacent to the S-ACE2 interface (
                             <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                              Figure 3E
                             </a>
                             ), suggesting that these mutations may affect the kinetics of receptor engagement and possibly viral entry. Other mutations occurred in regions proximal to essential enzymatic active sites of the viral replication (
                             <a id="xref-fig-18-2" class="xref-fig" href="#F18">
                              Figure S15
                             </a>
                             ) or protein processing (
                             <a id="xref-fig-19-2" class="xref-fig" href="#F19">
                              Figure S16
                             </a>
                             ) machinery.
                            </p>
                            <div id="F3" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F3.large.jpg?width=800&height=600&carousel=1" title="Manhattan plot of amino acid changes assessed in this study. A. Changes across the entire genome. B. Changes in the first 850 amino acids of S. In each of A-C the y axis shows effect size Î log R, the estimated change in log relative fitness due to each amino acid change. The bottom three axes show the background density of all observed amino acid changes, the density of those associated with growth (weighted by |Î log R|), and the ratio of the two. The top 55 amino acid changes are labeled. See Figure S13 for detailed views of S, N, ORF1a, and ORF1b. C. Changes in the first 250 amino acids of N. D. Structure of the spike-ACE2 complex (PDB: 7KNB). Spike subunits colored light blue, light orange, and gray. Top-ranked mutations are shown as red spheres. ACE2 is shown in magenta. E. Close-up view of the RBD interface. F. Top-ranked mutations in the N-terminal RNA-binding domain of N. Residues 44-180 of N (PDB: 7ACT) are shown in light blue. Amino acid positions corresponding to top mutations in this region are shown as red spheres. A 10-nt bound RNA is shown in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Manhattan plot of amino acid changes assessed in this study. <strong>A.</strong> Changes across the entire genome. <strong>B.</strong> Changes in the first 850 amino acids of S. In each of A-C the y axis shows effect size Î log R, the estimated change in log relative fitness due to each amino acid change. The bottom three axes show the background density of all observed amino acid changes, the density of those associated with growth (weighted by |Î log R|), and the ratio of the two. The top 55 amino acid changes are labeled. See Figure S13 for detailed views of S, N, ORF1a, and ORF1b. <strong>C.</strong> Changes in the first 250 amino acids of N. <strong>D.</strong> Structure of the spike-ACE2 complex (PDB: 7KNB). Spike subunits colored light blue, light orange, and gray. Top-ranked mutations are shown as red spheres. ACE2 is shown in magenta. <strong>E.</strong> Close-up view of the RBD interface. <strong>F.</strong> Top-ranked mutations in the N-terminal RNA-binding domain of N. Residues 44-180 of N (PDB: 7ACT) are shown in light blue. Amino acid positions corresponding to top mutations in this region are shown as red spheres. A 10-nt bound RNA is shown in gray.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F3.medium.gif" width="357" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F3.medium.gif" width="357" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 3.
                              </span>
                              <p id="p-32" class="first-child">
                               Manhattan plot of amino acid changes assessed in this study.
                               <strong>
                                A.
                               </strong>
                               Changes across the entire genome.
                               <strong>
                                B.
                               </strong>
                               Changes in the first 850 amino acids of S. In each of A-C the y axis shows effect size Î log R, the estimated change in log relative fitness due to each amino acid change. The bottom three axes show the background density of all observed amino acid changes, the density of those associated with growth (weighted by |Î log R|), and the ratio of the two. The top 55 amino acid changes are labeled. See
                               <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                                Figure S13
                               </a>
                               for detailed views of S, N, ORF1a, and ORF1b.
                               <strong>
                                C.
                               </strong>
                               Changes in the first 250 amino acids of N.
                               <strong>
                                D.
                               </strong>
                               Structure of the spike-ACE2 complex (PDB: 7KNB). Spike subunits colored light blue, light orange, and gray. Top-ranked mutations are shown as red spheres. ACE2 is shown in magenta.
                               <strong>
                                E.
                               </strong>
                               Close-up view of the RBD interface.
                               <strong>
                                F.
                               </strong>
                               Top-ranked mutations in the N-terminal RNA-binding domain of N. Residues 44-180 of N (PDB: 7ACT) are shown in light blue. Amino acid positions corresponding to top mutations in this region are shown as red spheres. A 10-nt bound RNA is shown in gray.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-33">
                             We tested several of the high-scoring mutations in single-cycle infectivity assays as done previously (
                             <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ), focusing on the RBD (
                             <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                              Figure 2A
                             </a>
                             ). We found that while some individual mutations increased infectivity, on average high-scoring RBD mutations did not promote infectivity per se. We considered an alternate possibility that fitness of Spike mutations is driven by immune escape. Using RBD-aggregated mutations as a proxy for immune escape, we found that the fitness effect of these Spike mutations correlates well with antibody escape estimates from Greaney et al.(
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ) (
                             <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                              Figure 2B
                             </a>
                             ). Together with the observed jump in fitness beginning in late 2021 (
                             <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                              Figure 2C
                             </a>
                             ) associated with Spike mutations, but not mutations elsewhere in the genome (
                             <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                              Figure 2E
                             </a>
                             ), these results suggest that immune escape is currently the dominant driver of fitness increases. In contrast to mutations in Spike, those in the serine-arginine rich region of N were linked to increased efficiency of SARS-CoV-2 genomic RNA packaging (
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ). Within ORF1, we found fitness-associated mutations across all viral enzymes, and clusters within additional non-structural proteins (nsps). The highest concentration of fitness-associated mutations is found in nsp4, nsp6, and nsp12â14 (
                             <a id="xref-fig-15-2" class="xref-fig" href="#F15">
                              Figure S12B
                             </a>
                             ,
                             <a id="xref-fig-16-3" class="xref-fig" href="#F16">
                              S13C-D
                             </a>
                             ), suggesting unexplored function at those sites. For example, nsp4 and nsp6 have roles in assembly of replication compartments, and substitutions in these regions may influence the kinetics of replication (see Supplemental Note 3). We note that while convergent evolution makes it possible to identify candidate functional mutations, observational data alone is insufficient to declare mutations as causal rather than merely correlated. For this reason hits identified by our study require functional followup, and can be prioritized by our uncertainty-ranked list of important mutations.
                            </p>
                            <p id="p-34">
                             In summary, PyR
                             <sub>
                              0
                             </sub>
                             provides an unbiased, automated approach for detecting viral lineages with increased fitness. By combining a model-based assessment of lineage fitness with absolute case counts, our model provides a global picture of the events of the first two years of the pandemic. Because it assesses the contribution of individual mutations and aggregates across all lineages and geographic regions, it can identify mutations and gene regions that likely increase fitness, and it can predict the relative fitness of new lineages based solely on viral sequence. Applied to the full set of publicly available SARS-CoV-2 genomes, it provides a principled, unbiased analysis of the mutations driving increased fitness of the virus, identifying experimentally established driver mutations in S and highlighting the key role of non-S mutations, particularly in N, ORF1b, and ORF1a, which have received relatively less research attention. By jointly estimating lineage and mutational fitness from millions of viral sequences across thousands of regions, PyR
                             <sub>
                              0
                             </sub>
                             shares statistical strength across regions and mutations to yield mechanistic insight into viral fitness and enhance public health by forecasting lineage dynamics.
                            </p>
                           </div>
                           <div class="section data-availability" id="sec-4">
                            <h2 class="">
                             Data Availability
                            </h2>
                            <p id="p-35">
                             All data was gathered from other public resources. Data preprocessing scripts are open source.
                            </p>
                            <p id="p-36">
                             <a href="https://gisaid.org">
                              https://gisaid.org
                             </a>
                            </p>
                            <p id="p-37">
                             <a href="https://github.com/CSSEGISandData/COVID-19">
                              https://github.com/CSSEGISandData/COVID-19
                             </a>
                            </p>
                            <p id="p-38">
                             <a href="https://cov-lineages.org/">
                              https://cov-lineages.org/
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-5">
                            <h2 class="">
                             Supplementary Materials
                            </h2>
                            <div class="section" id="sec-6">
                             <h3>
                              Materials and methods
                             </h3>
                             <div class="section" id="sec-7">
                              <h4>
                               Data and Code Availability
                              </h4>
                              <p id="p-39">
                               Source code for data preprocessing and modeling and available at
                               <a href="https://github.com/broadinstitute/pyro-cov">
                                https://github.com/broadinstitute/pyro-cov
                               </a>
                               . GISAID sequence data is publicly available at
                               <a href="https://gisaid.org">
                                https://gisaid.org
                               </a>
                               . PANGO lineage aliases are available at
                               <a href="https://cov-lineages.org">
                                https://cov-lineages.org
                               </a>
                               with source code at
                               <a href="https://github.com/cov-lineages/lineages-website">
                                https://github.com/cov-lineages/lineages-website
                               </a>
                               and lineage aliases available at
                               <a href="https://github.com/cov-lineages/pango-designation">
                                https://github.com/cov-lineages/pango-designation
                               </a>
                               . UShER phylogenies of public data are available at
                               <a href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2">
                                http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2
                               </a>
                               . The whole genome map is available as part of NextClade at
                               <a href="https://github.com/nextstrain/ncov/blob/50ceffa/defaults/annotation.gff">
                                https://github.com/nextstrain/ncov/blob/50ceffa/defaults/annotation.gff
                               </a>
                               . Structures of ORFab regions are available at
                               <a href="https://www.ncbi.nlm.nih.gov/protein">
                                https://www.ncbi.nlm.nih.gov/protein
                               </a>
                               .
                              </p>
                             </div>
                             <div class="section" id="sec-8">
                              <h4>
                               Regression model of relative fitness (PyR
                               <sub>
                                0
                               </sub>
                               model)
                              </h4>
                              <p id="p-40">
                               We fit a Bayesian, hierarchical multinomial logistic regression model to data from GISAID using Pyro. Details are provided in the supplemental note below.
                              </p>
                             </div>
                             <div class="section" id="sec-9">
                              <h4>
                               Simulation of lineages
                              </h4>
                              <p id="p-41">
                               We carried out a simulation study to determine whether the process of clustering genomes into named lineages could generate an artifactual increase in estimated fitness. The simulation was of a single neutrally evolving viral population with discrete generations and a stochastic population size generated by a highly overdispersed negative binomial distribution with the current fitness. (Overdispersion parameter = 0.11, which yields 10% of cases causing 80% of transmission.) The fitness is 2.5 for the first 10 generations; subsequently it drops to 1.5 until the viral population reaches 80,000 infections, whereupon it drops again to 0.8. When the population decreases to 10,000, the growth switches back to 1.5, and continues cycling when the high and low population thresholds are reached. (A model with a roughly constant-sized population yields similar results.) The population starts as a single named lineage. Each generation, the most successful nodes in that generation are determined by looking ahead four generations and counting descendants. New lineages are assigned to the nodes with the most descendants (minimum of 200 descendants), up to a maximum of 10 lineages per generation. 10% of all infections are randomly sampled and any lineage with fewer than 20 descendants is discarded. When all new lineages have been generated and all nodes assigned a lineage, a global multinomial logistic regression is performed, using the Python package sklearn.linear_model, yielding relative fitness estimates of all lineages.
                              </p>
                             </div>
                             <div class="section" id="sec-10">
                              <h4>
                               Spatial analysis of mutation coefficients
                              </h4>
                              <p id="p-42">
                               To assess the spatial structure of the inferred amino acid coefficients Î²
                               <sub>
                                f
                               </sub>
                               (described in Probabilistic Model below), we utilize the Moran I spatial autocorrelation statistic. We report (see
                               <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                                Table S1
                               </a>
                               ) one-sided p-values for Moran I computed using a permutation test with 999,999 random permutations. We use a gaussian weighting function of the form exp(-distance
                               <sup>
                                2
                               </sup>
                               /lengthscale
                               <sup>
                                2
                               </sup>
                               ), where distance is measured in units of nucleotides. We compute Moran I statistics both for individual genes and the entire genome. For larger genes whose extent is 1000+ nucleotides we use a length scale of 50 nucleotides. For smaller genes (e.g. ORF8) we set the length scale to one twentieth of the extent of the gene. When considering the entire set of amino acid changes, i.e. all 2,904 coefficients that make up Î²
                               <sub>
                                f
                               </sub>
                               , we compute the Moran I statistic for two different length scales. We note, however, that the Moran I statistic is somewhat simplistic, since it is designed to pick up spatial structure at a single length scale. In particular it can be insensitive to complex spatial structure that involves multiple overlapping substructures at different scales. Nevertheless it offers a simple quantitative metric for identifying spatial structure in the coefficients Î²
                               <sub>
                                f
                               </sub>
                               .
                              </p>
                             </div>
                             <div class="section" id="sec-11">
                              <h4>
                               Analysis of substitution statistics
                              </h4>
                              <p id="p-43">
                               To assess enrichment of amino acid changes we compared the event frequencies for the leading mutation sets (as determined by posterior mean/std ranking) against a background of all mutations used as features in the model using multiple testing corrected binomial tests. We performed this analysis for both the asymmetric case (where A->V and V->A are different events) and for the symmetric case.
                              </p>
                             </div>
                             <div class="section" id="sec-12">
                              <h4>
                               Comparison to other regression models
                              </h4>
                              <p id="p-44">
                               We fit logistic regression models in R version 4.0.3. The stats::glm() was used to fit binomial logistic regression models and the nnet::multinom function was used to fit multinomial logistic regression models. For multinomial logistic regressions, the data were filtered to contain sequences between January 1 2021 and December 31 2021 from the most common 25 pango lineages in the 10 countries with the most sequences available. The resulting dataset was downsampled to 10% of its initial size.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-13">
                             <h3>
                              Supplemental Note 1: Detailed description of PyR
                              <sub>
                               0
                              </sub>
                              model
                             </h3>
                             <div class="section" id="sec-14">
                              <h4>
                               Data Preparation
                              </h4>
                              <p id="p-45">
                               We downloaded 6,466,300 samples from GISAID (
                               <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                               ,
                               <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                               ) on January 20, 2021. Each sample record includes labels for time, location, PANGO lineage annotation (
                               <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                               ), and genetic sequence. We discard records with missing time, location, or lineage. We use UShER (
                               <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                               ) to build a mutation-annotated phylogenetic tree, discarding sequences whose alignment quality is not reported as âgoodâ. We bin time intervals into 14-day segments, choosing a multiple of 7 to minimize weekly seasonality, but binning coarser than a week so as to reduce memory requirements; this results in 56 time bins.
                              </p>
                              <p id="p-46">
                               Because sample counts vary widely across GISAID geographic region (by as much as five orders of magnitude), we aggregate regions into the following coarse partitions: each country counts as a region, and any first level subregion of a country counts as a region if it has at least 50 samples; otherwise it is aggregated into a whole-country bin. Note this means that e.g. a country may be split up into its larger regions, with smaller regions being subsumed into an aggregate country level bin. We then drop regions without samples in at least two different time intervals, resulting in 1560 regions in total.
                               <a id="xref-fig-20-1" class="xref-fig" href="#F20">
                                Figure S17
                               </a>
                               shows the distribution of samples among countries and GISAID regions.
                              </p>
                              <p id="p-47">
                               After preprocessing, the model input data are a T Ã P Ã C = 56 Ã 1560 Ã 3000 shaped array y
                               <sub>
                                tpc
                               </sub>
                               â â of counts (this array is sparse and our inference code uses a sparse representation), and an C Ã F = 3000 Ã 2904 shaped array X
                               <sub>
                                sf
                               </sub>
                               â {0, 1} of mutation features.
                              </p>
                              <p id="p-48">
                               Cases per day (see
                               <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                                Figure 3
                               </a>
                               inset) were estimated by multiplying confirmed case count data from Johns Hopkins University by the estimated proportion of each lineage within each (time, region) bin. We manually matched each GISAID region to the finest enclosing JHU region.
                              </p>
                             </div>
                             <div class="section" id="sec-15">
                              <h4>
                               Lineage Clustering
                              </h4>
                              <p id="p-49">
                               Our method relies on a partitioning of genetic samples into clusters, where we estimate the fitness of each cluster. We initially tried to use the 1544 PANGO lineages as clusters, but found that some PANGO lineages appeared to include multiple distinct viruses of different fitness, e.g. B.1.1. exhibits two peaks in relative abundance in England, contrary to our multivariate logistic growth model. We therefore refined the 1544 PANGO lineages into 3000 finer clusters, with rates estimated individually for each cluster. Indeed
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                Figure S4
                               </a>
                               shows that some PANGO lineages contain multiple distinct clusters of fitness estimates differing by more than a factor of two.
                              </p>
                              <p id="p-50">
                               To create genetic clusters finer than PANGO lineages we began with a complete 4,833,238 node phylogeny of all GISAID samples maintained by Angie Hinrichs (
                               <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                               ) (this phylogeny was created using UShER (
                               <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                                21
                               </a>
                               ), excluding private mutations, masking difficult-to-sequence regions, eliding deletions, parsimoniously imputing missing sequence data). To coarsen the 4,833,238-node phylogenetic tree down to 3000 nodes (treated as clusters) we greedily collapsed parent-child edges, minimizing the the following distance function TreeDistance(-,-) between two mutation annotated trees
                               <span class="disp-formula" id="disp-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where T is the true mutation annotated tree, Tâ is the collapsed tree whose nodes we treat as clusters, u and v are sample sequences, mrca(T,u,v) is the sequence of the most recent common ancestor of u,v in the mutation annotated tree T, and EditDistance(-,-) counts amino acid substitutions between two sequences. This objective function minimizes the mean edit distance between the true mrca sequence and its clusterâs sequence, for each pair of sequences. Changes in the objective function can be computed cheaply, and the O(n log(n)) time greedy algorithm can process the entire n=4,833,238 node phylogeny in under 5 minutes. Empirically this heuristic clustering produces trees that are approximately balanced in both cluster size and cluster-cluster edit distance, on both the true data and on synthetic datasets.
                               <a id="xref-fig-21-1" class="xref-fig" href="#F21">
                                Figure S18
                               </a>
                               shows the distribution of samples among both coarse PANGO lineages and the finer clusters.
                               <a id="xref-fig-22-1" class="xref-fig" href="#F22">
                                Figure S19
                               </a>
                               shows small example trees produced by clustering large synthetic trees.
                              </p>
                             </div>
                             <div class="section" id="sec-16">
                              <h4>
                               Probabilistic Model
                              </h4>
                              <p id="p-51">
                               We model relative lineage growth with a hierarchical Bayesian regression model with a multinomial likelihood. Arrays in the model index over one or more indices: T=56 time steps (increments of 14 days) t; C=3000 clusters c; P=1560 regions (âplacesâ) p; and F=2904 amino acid substitutions (âfeaturesâ) f. The model, shown below, regresses lineage counts y
                               <sub>
                                tpc
                               </sub>
                               â â in each time-region-lineage bin against amino acid mutation covariates X
                               <sub>
                                cf
                               </sub>
                               â {0,1}. The variables y and X are observed and all other variables in the model are latent. Each latent variable is governed by a prior distribution. The full model is specified as follows (visualized in
                               <a id="xref-fig-23-1" class="xref-fig" href="#F23">
                                Figure S20
                               </a>
                               ), where the observed counts y
                               <sub>
                                tpc
                               </sub>
                               are underlined:
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               The proportion of lineages in a single time-region bin is modeled as a Multinomial distribution whose probability parameter is a multivariate logistic growth function softmax(Î±
                               <sub>
                                p
                                <strong>
                                 Â·
                                </strong>
                               </sub>
                               + tÎ²
                               <sub>
                                p
                                <strong>
                                 Â·
                                </strong>
                               </sub>
                               /Ï) with intercept Î±
                               <sub>
                                pc
                               </sub>
                               and slope Î²
                               <sub>
                                pc
                               </sub>
                               in units of generation time Ï = 5.5 days (these units are for interpretability only; the model does not use the notion of generation, and thus is robust to changes in generation time). Here the dot subscripts Î±
                               <sub>
                                p
                                <strong>
                                 Â·
                                </strong>
                               </sub>
                               â â
                               <sup>
                                C
                               </sup>
                               , Î²
                               <sub>
                                p
                                <strong>
                                 Â·
                                </strong>
                               </sub>
                               â â
                               <sup>
                                C
                               </sup>
                               , and y
                               <sub>
                                tp.
                               </sub>
                               â â
                               <sup>
                                C
                               </sup>
                               denote vectors over cluster ids. The softmax function implements the multivariate generalization of logistic growth, inputting and outputting vectors, and is defined as
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               For a simple model of two lineages, each of the two components of the softmax function are sigmoid curves; however for more lineages, the functional forms may be more complex. Early iterations of the model used overdispersed likelihoods such as Dirichlet-Multinomial to account for additional variability not directly encoded in the generative process. However, we found that we can obtain much more accurate model predictions by using a Multinomial likelihood and accounting for model misfit by adding hierarchical structure elsewhere. The intercepts Î±
                               <sub>
                                pc
                               </sub>
                               denote initial relative log prevalence of cluster c in region p; these are modeled hierarchically around the global relative log prevalence Î±
                               <sub>
                                c
                               </sub>
                               of each cluster. The slopes Î²
                               <sub>
                                pc
                               </sub>
                               are modeled hierarchically around global per-cluster fitness Î£
                               <sub>
                                f
                               </sub>
                               Î²
                               <sub>
                                f
                               </sub>
                               Xc
                               <sub>
                                f
                               </sub>
                               that are linearly regressed against amino acid substitution features X
                               <sub>
                                cf
                               </sub>
                               . These linear coefficients Î²
                               <sub>
                                f
                               </sub>
                               can be directly interpreted as the effect of a mutation on a lineageâs fitness, all other variation being equal. In figures we plot posterior means ð¼[Î²
                               <sub>
                                f
                               </sub>
                               |data] =: Î log R as an estimate of effect size and plot the posterior z-score |ð¼[Î²
                               <sub>
                                f
                               </sub>
                               |data]| / ð [Î²
                               <sub>
                                f
                               </sub>
                               |data]
                               <sup>
                                1/2
                               </sup>
                               =: |Âµ|/Ï as a proxy for statistical significance.
                              </p>
                              <p id="p-52">
                               Note that by regressing against amino acid changes we obviate the need to directly incorporate phylogenetic information into the model: if two lineages are close together in a phylogeny, then their amino acid features are likely also similar, so their regressed fitness values will likely be similar. By sharing statistical strength in this way we are also able to make accurate predictions for emergent lineages with few observations. (Note phylogenetic information is still used in preprocessing, since our clustering is created from an UShER phylogenetic tree.) Both of the hierarchies in Î± and Î² empirically improve model fit in the presence of heavily skewed observations (e.g. most samples are from the UK, and there is a long tail of sparsely sampled regions). We chose these model structures based on extensive cross-validation and forecasting experiments.
                              </p>
                              <p id="p-53">
                               We place weak priors on scale parameters Ï
                               <sub>
                                1
                               </sub>
                               , Ï
                               <sub>
                                2
                               </sub>
                               , and Ï
                               <sub>
                                4
                               </sub>
                               (these denote standard deviations, the square roots of prior variance). The Ï
                               <sub>
                                1
                               </sub>
                               and Ï
                               <sub>
                                2
                               </sub>
                               priors are centered at large values to allow for wide variation in initial infection proportions across regions. The Ï
                               <sub>
                                4
                               </sub>
                               prior is centered around the smaller value e
                               <sup>
                                â4
                               </sup>
                               â 0.018 because we expect little variation of relative fitness across geographic regions a priori (some variation is expected, due to geographic variations in e.g. age distribution, behavior, or genetics as in binding affinity due HLA complex genotypes (
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                               )). We fix the linear regression scale parameter Ï
                               <sub>
                                3
                               </sub>
                               to a small value, forcing the regression problem towards a sparse solution (i.e. we assume a priori that most observed mutations have little effect on fitness). We choose a Laplace prior on regression coefficients because it is heavier-tailed than a Normal prior, but not so heavy-tailed that the regression problem becomes multimodal (as it would for e.g. a Cauchy or Studentâs t prior).
                              </p>
                              <p id="p-54">
                               This proportional growth model differs from many forecasting models in the literature that are formulated in terms of absolute sample counts. Whereas our Multinomial likelihood allows us to model only the relative portions of lineages in each (time,region) bin, a Poisson likelihood would force us to additionally model the total number of genome samples in each (time,place) bin, a task which is less related to viral dynamics and more related to local lab capacity, political dynamics, and local calendars. We choose to model relative proportions rather than absolute counts because the relative model is robust to a number of sources of bias, including: sampling bias across regions (e.g. one region samples 1000x more than another); sampling bias over time (e.g. change in sampling rate over time); and change in absolute fitness of all lineages, in any (time, region) bin (e.g. due to changes in local policies or weather, as long as those changes affect all lineages equally). However the model is susceptible to the following sources of bias: biased sampling in any (time, region) cell (e.g. sequencing only in case of S-gene target failure); and changes in sampling bias within a single region over time (e.g. a country has a lab in only one city, then spins up a second lab in another distant city with different lineage proportions).
                              </p>
                              <p id="p-55">
                               This model has several advantages over existing approaches. First, it provides a principled, agnostic approach that can be applied to a large dataset to identify lineages that demonstrate concerning epidemiological features. Second, by modeling the relative fitness of lineages separately across 1560 geographic regions, the model is robust to region-specific differences in non-pharmaceutical interventions and vaccination rates. Third, the hierarchical nature of the model which represents lineages as collections of mutations reflects the underlying biology and yields both strain- and lineage-specific coefficients from a single inferential approach. While the linear-additive model of mutation biology is a coarse approximation to true biology including epistasis, our hierarchical model serves as a framework to explore such models (
                               <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                               ,
                               <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                24
                               </a>
                               ) on SARS-CoV-2 genomic surveillance data.
                              </p>
                              <p id="p-56">
                               We interpret the regression coefficients as the relative fitness based on a well-known result in population genetics
                               <span class="underline">
                                (Crow and and Kimura 1970)
                               </span>
                               that the change in genotype frequency in a large haploid population under selection follows a logistic curve, where the logistic growth rate parameter defines the relative fitness of genotypes.
                              </p>
                             </div>
                             <div class="section" id="sec-17">
                              <h4>
                               Probabilistic Inference
                              </h4>
                              <p id="p-57">
                               The model is implemented in the Pyro probabilistic programming language (
                               <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               ) built on PyTorch (
                               <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                                25
                               </a>
                               ). To fit an approximate joint posterior distribution over all latent variables (a space of dimension 375,909), we train a flexible reparameterized variational distribution using stochastic variational inference. Our variational approach starts by reparameterizing the model via a sequence of learnable but distribution-preserving decentering transforms (
                               <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ) on the Î± and Î² latent variables. Reparameterizing is particularly helpful in avoiding Nealâs-funnel situations (
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                               ) by smoothing out the geometry of latent variables with Normal prior whose scale parameter is also a latent variable. After reparameterizing we model the posterior over all variables as a joint multivariate Normal distribution whose covariance matrix Î£ is parametrized by a rank-200 matrix plus a diagonal matrix
                               <strong>
                                D
                               </strong>
                               with positive entries:
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/02/16/2021.09.07.21263228/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where â§ is an unconstrained matrix of size 375,909 x 200. This low-rank multivariate Normal distribution allows the approximate posterior to capture correlated uncertainty among competing mutations each of which might explain increased fitness. This variational distribution has 75,936,525 parameters to be optimized (much larger than the number 375,909 of latent variables, but much smaller than the 375,909 â¨ (375,909 + 1) / 2 â 7 â¨ 10
                               <sup>
                                10
                               </sup>
                               parameters that would be required to represent a full-rank covariance matrix).
                              </p>
                              <p id="p-58">
                               Variational inference is performed for 10,000 iterations with the Adam optimizer (
                               <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                                28
                               </a>
                               ) with clipped gradients and an exponentially decreasing learning rate schedule and initial learning rates between 0.05 and 0.0025 for different parameter groups (see
                               <a id="xref-fig-24-1" class="xref-fig" href="#F24">
                                Figure S21
                               </a>
                               ). Optimization proceeds in batch-mode, i.e. without any data subsampling. We initialize model parameters to median prior values with a small amount of noise added to avoid scale parameters collapsing early in training. After inference we make predictions by drawing 1000 posterior samples. See source code for detailed optimizer and initialization configuration.
                              </p>
                              <p id="p-59">
                               Inference and prediction on a single GPU (NVIDIA Tesla A100 with 48GB of RAM) takes about 10 minutes (compared to 14.5 hours on an 8-core CPU), which is less than the amount of time required to download and preprocess each daily snapshot of data from GISAID. The cost of fitting the model is O((TP+F)C), dominated by pointwise mathematical operations, particularly computing the softmax function on a dense array of shape TÃPÃC. This cost does not depend directly on the number of genetic samples, since samples are aggregated into counts y of constant shape TÃPÃC.
                              </p>
                              <p id="p-60">
                               We emphasize that inference in this model is very challenging due to the large dimension of the latent space (namely 375,909), itself a consequence of the large number of regions, lineages, and mutations considered by the model (
                               <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                                29
                               </a>
                               ). While variational inference has a number of attractive features, especially computationally, like any approximate inference scheme it comes with disadvantages. In our case the most notable disadvantage of variational inference is its tendency to yield biased posterior uncertainty estimates. Typically posterior uncertainty is underestimated, leading to credible intervals (CI) that in some cases can be unrealistically narrow. The primary parameters of interest in the PyR
                               <sub>
                                0
                               </sub>
                               model are the mutation-level coefficients Î²
                               <sub>
                                f
                               </sub>
                               and the per-lineage fitness values Î£
                               <sub>
                                f
                               </sub>
                               Î²
                               <sub>
                                f
                               </sub>
                               Xc
                               <sub>
                                f
                               </sub>
                               . Since the latter quantity governs the prior over Î²
                               <sub>
                                pc
                               </sub>
                               , which in turn directly feeds into the multinomial likelihood, the per-lineage fitness estimates are more-or-less tightly constrained by the observed counts y
                               <sub>
                                tpc
                               </sub>
                               . Consequently the posterior uncertainty of per-lineage fitness is comparatively easy to estimate and we expect variational inference to yield reasonable credible intervals for these quantities. In contrast the mutation-level coefficients Î²
                               <sub>
                                f
                               </sub>
                               interact with correlated features X
                               <sub>
                                cf
                               </sub>
                               (leading to a multi-modal posterior) and are less directly constrained by the observed counts y
                               <sub>
                                tpc
                               </sub>
                               . Consequently it is significantly more challenging to estimate the corresponding posterior uncertainty. In practice we obtain implausibly narrow credible intervals for these quantities and the posterior uncertainty must be interpreted with caution. Importantly, while the uncertainty estimates for Î²
                               <sub>
                                f
                               </sub>
                               should not be taken at face value, we believe that they are still very useful for interpreting inferred model parameters, since they can be used to rank/prioritize different hits Î²
                               <sub>
                                f
                               </sub>
                               . In particular, while the absolute magnitudes of Î²
                               <sub>
                                f
                               </sub>
                               uncertainty estimates are implausible, their relative magnitudes are representative of the amount of supporting evidence, and thus are useful for ranking. Since we consider a large number of mutations (F=2904) this information is invaluable for designing experiments for functional characterization.
                              </p>
                             </div>
                             <div class="section" id="sec-18">
                              <h4>
                               Implementation
                              </h4>
                              <p id="p-61">
                               We implemented the PyR
                               <sub>
                                0
                               </sub>
                               model using the probabilistic programming language Pyro (
                               <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               ). The model leverages PyTorch and Pyro to scale efficiently to large data sets and can therefore be applied continuously as datasets grow, completing model training and prediction with millions of viral genomes in minutes on a single GPU. We chose the Pyro framework because it cleanly separates model specification from inference customization, and scales to large models and datasets by leveraging GPUs. This flexible modeling framework allowed us to experiment with different hierarchical structures. Additionally by relying on an open source and well-tested modeling and inference framework, we minimize the risk of introducing software bugs into our analysis. The speed of inferenceâwhich took about 10 minutes on a single GPU on the full dataset of >6 million genomesâallowed quick model iteration and thorough validation on subsets of the data, including both geographic cross-validation and temporal data truncation.
                              </p>
                             </div>
                             <div class="section" id="sec-19">
                              <h4>
                               Prediction
                              </h4>
                              <p id="p-62">
                               In
                               <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                                Figure 3
                               </a>
                               , the 95% confidence intervals in parentheses were estimated by drawing 1000 samples from the variational posterior distribution. Confirmed cases per day were estimated at the end of the training period (Jan 20 2021) by combining our modelâs relative lineage portions with confirmed case count data from Johns Hopkins university. Quantities defined over our 3000 fine clusters were aggregated up to coarser PANGO lineages for reporting. To facilitate downstream use of model predictions we have provided complete tables of lineage fitness estimates (Data S1) and mutation coefficients (Data S2). These predictions have been used e.g. by
                               <a href="http://Nextstrain.org">
                                Nextstrain.org
                               </a>
                               to visualize our predicted mutational fitness along a phylogenetic tree (
                               <a id="xref-fig-25-1" class="xref-fig" href="#F25">
                                Figure S22
                               </a>
                               ).
                              </p>
                             </div>
                             <div class="section" id="sec-20">
                              <h4>
                               Validation
                              </h4>
                              <p id="p-63">
                               We considered the possibility of biased submission to the GISAID database and compared results obtained from the full dataset with results obtained from disjoint subsets. For this purpose we divided the data into samples from the most heavily sampled region (Europe, with 3.3M samples) and those from the rest of the world (with 3.1M samples) (
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Figures S1
                               </a>
                               ,
                               <a id="xref-fig-14-2" class="xref-fig" href="#F14">
                                S11
                               </a>
                               ). This split is motivated by most samples originating from the UK: we widened the region around the UK until the region and its complement both had roughly equivalent statistical strength and narrow posterior estimates. We conducted two-fold cross-validation experiments for both lineages (
                               <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                                Figure S1
                               </a>
                               ) and mutations (
                               <a id="xref-fig-14-3" class="xref-fig" href="#F14">
                                Figure S11
                               </a>
                               ). Additionally, in
                               <a id="xref-fig-26-1" class="xref-fig" href="#F26">
                                Figure S23
                               </a>
                               , we show that PyR
                               <sub>
                                0
                               </sub>
                               lineage-level Î log R estimates are largely driven by regions with the largest numbers of samples and are thus robust to the manner in which under-sampled regions are organized into spatial units.
                              </p>
                              <p id="p-64">
                               We found the full GISAID dataset to be invaluable to making accurate predictions. Using data up to July 2021, we tried restricting to either all CDC data or CDCâs randomly sampled NS3 dataset and found those subsets to result in insufficient diversity and lead to unclear results (Pearson correlation 0.49, 0.28, respectively). Using data snapshots from mid January 2022, we tried restricting to open data available in GENBANK, but found the model made implausible estimates of Omicron fitness, due to a combination of lack of geographic diversity (GENBANK has only about 1/10 as many geographic regions as we were able to extract from GISAID data, and particularly has very few samples from South Africa) and data upload latency (GISAID appeared to have â¼1 week upload latency, versus â¼1 month for GENBANK).
                              </p>
                              <p id="p-65">
                               Our model assumes each single point mutation independently linearly contributes to change in fitness. A natural generalization is to search for groups of mutations that affect fitness. To explore this we fit a similar model of both single and pair mutations, considering only pairs that lie within the same gene. Fitting this model on data up to July 2021, we discovered no pairwise mutations stronger than the top 100 single mutations. While these experiments did not discover pairwise mutations, we believe that more sophisticated models would be able to measure epistasis, but sophistication in that area is beyond the scope of the present work.
                              </p>
                              <p id="p-66">
                               Finally, to compare our multinomial multivariate logistic growth model to simple binomial univariate logistic growth, we compared lineage fitness estimates (
                               <a id="xref-fig-27-1" class="xref-fig" href="#F27">
                                Figure S24
                               </a>
                               ) and logistic growth curves (S25) of all but one lineage at a time, showing good agreement on the narrow selection of lineages examined by each binomial logistic fit.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              Supplemental Note 2
                             </h3>
                             <div class="section" id="sec-22">
                              <h4>
                               Cell culture
                              </h4>
                              <p id="p-67">
                               Cells were cultured in humidified incubators with 5% CO
                               <sub>
                                2
                               </sub>
                               at 37Â° C, and monitored for mycoplasma contamination using the Mycoplasma Detection kit (Lonza LT07-318). HEK293 Homo sapiens, female, embryonic kidney cells (ATCC CRL-1573) were cultured in DMEM supplemented with 10% heat-inactivated FBS, 1 mM sodium pyruvate, 20 mM GlutaMAX, 1Ã MEM non-essential amino acids, and 25 mM HEPES, pH 7.2.
                              </p>
                             </div>
                             <div class="section" id="sec-23">
                              <h4>
                               Virus production
                              </h4>
                              <p id="p-68">
                               24 hrs prior to transfection, 6 Ã 10
                               <sup>
                                5
                               </sup>
                               HEK-293 cells were plated per well in 6 well plates. All transfections used 2.49 Âµg plasmid DNA with 6.25 ÂµL TransIT LT1 transfection reagent (Mirus, Madison, WI) in 250 ÂµL Opti-MEM (Gibco). Single-cycle HIV-1 vectors pseudotyped with SARS-CoV-2 Spike protein, either D614 or D614G, were produced by transfection of either HIV-1 pNL4-3 Îenv Îvpr luciferase reporter plasmid (pNL4-3.Luc.R-E-), or pUC57mini NL4-3 Îenv eGFP reporter plasmid, in combination with the indicated Spike expression plasmid, at a ratio of 4:1. ACE2 expression vectors were produced by transfecting cells with one of the pscALPSpuro-ACE2 plasmids, along with the HIV-1 gag-pol expression plasmid psPAX2, and the VSV glycoprotein expression plasmid pMD2.G (4:3:1 ratio of plasmids). 16 hrs post-transfection, culture media was changed. Viral supernatant was harvested 48 hours after media change, passed through a 0.45 Âµm filter, and stored at 4Â°C. TMPRSS2 expression transfer vector was produced similarly but with pscALPSblasti-TMPRSS2.
                              </p>
                             </div>
                             <div class="section" id="sec-24">
                              <h4>
                               Generation of cell lines expressing ACE2 and TMPRSS2
                              </h4>
                              <p id="p-69">
                               2.5 x 10
                               <sup>
                                5
                               </sup>
                               HEK-293 cells were plated per well in a 12 well plate. The next day cells were transduced with 250 uL of supernatant containing TMPRSS2-encoding lentivirus for 16 hr at 37Â°C, after which fresh media was added to cells. 48 hrs after transduction cells were replated and selected with blasticidin (InvivoGen, catalogue #ant-bl-1) at 10 ug/ml. After selection, cells were transduced similarly with supernatant containing ACE2-encoding lentivirus and selected with 1 ug/mL of puromycin (InvivoGen, San Diego, CA, catalogue #ant-pr-1).
                              </p>
                             </div>
                             <div class="section" id="sec-25">
                              <h4>
                               Virus Infectivity Assays
                              </h4>
                              <p id="p-70">
                               16 hours prior to transduction, adherent cells were seeded in 96 well plates. HEK-293 cells were plated at 5 x 10
                               <sup>
                                4
                               </sup>
                               cells per well. Cells were incubated in virus-containing media for 16 hrs at 37Â°C when fresh medium was added to cells. 48 to 72 hours after transduction cells were assessed for luciferase activity. Cells transduced with luciferase expressing virus were assessed using Promega Steady-Glo system (Promega Madison, WI). GraphPad Prism 8.4.3 was used to analyze the infectivity data using a ratio paired t test. In these experiments, all values shown are the mean with standard deviation, with the actual calculated two-tailed P value indicated.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              Supplemental Note 3
                             </h3>
                             <p id="p-71">
                              We include here an extended discussion of high-scoring mutations.
                             </p>
                             <div class="section" id="sec-27">
                              <h4>
                               Relation to other viruses
                              </h4>
                              <p id="p-72">
                               The concentration of putative transmission-promoting substitutions in N at positions 160-210 is remarkable, but is supported by a similar observation in Ebola virus(
                               <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                               ), and recent data for SARS-CoV-2 showing mutations in that region increase the efficiency of viral packaging(
                               <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                                31
                               </a>
                               ), validating some of the modelâs most unexpected predictions and supporting its ability to identify novel biology.
                              </p>
                             </div>
                             <div class="section" id="sec-28">
                              <h4>
                               Potential functional roles of mutations within ORF1
                              </h4>
                              <p id="p-73">
                               Our model highlighted mutations within the ORF1 non-structural proteins (nsps) whose functions are not fully understood (e.g.
                               <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">
                                Table S3
                               </a>
                               ). We found two predominant clusters within ORF1a: one in the C-terminal â¼120 amino acids of nsp4 and the other within the N-terminal â¼160 amino acids of nsp6 (
                               <a id="xref-fig-16-4" class="xref-fig" href="#F16">
                                Figure S13C
                               </a>
                               ). Nsp4 and nsp6 are both membrane-anchored proteins with roles in assembly and concentration of the viral replication and transcription complex (RTC) machinery within double-membrane vesicles (
                               <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                32
                               </a>
                               ). Amino acid substitutions in these regions, combined with transmission-associated mutations identified within additional RTC-associated nsps (e.g., nsp12-16,
                               <a id="xref-fig-16-5" class="xref-fig" href="#F16">
                                Figure S13D
                               </a>
                               ), may therefore affect the kinetics of replication and gene expression, resulting in higher virus yields from infected cells. Nsp2, a rapidly evolving accessory protein (
                               <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                33
                               </a>
                               )(
                               <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                34
                               </a>
                               )(
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                               ) whose proposed function in disrupting host cell signaling (
                               <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                               ) and viral mRNA translation initiation (
                               <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                               ) remains obscure, harbored many additional mutations associated with higher fitness (
                               <a id="xref-fig-16-6" class="xref-fig" href="#F16">
                                Figure S13C
                               </a>
                               ).
                              </p>
                              <p id="p-74">
                               The ORF1a-ORF1b polyprotein is processed into 16 non-structural proteins by two viral proteases: a papain-like protease (nsp3) and 3C-like protease (nsp5). Multiple transmission-associated mutations were found within the protease coding regions (e.g., ORF1a:V1750A, ORF1a:P3395H). Most of the amino acid substitutions identified by our model were outside of the domains containing catalytic residues for nsp3 (C1674, H1835, D1849) or nsp5 (H3304, C3408) (
                               <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                               )(
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                               ). However, the potential effects of these mutations on protease architecture and activity warrant further experimentation. A few of the top mutations from our model (e.g., ORF1a:T3255I, ORF1a:A3571V) are positioned adjacent to nsp cleavage sites, potentially influencing local structures and kinetics of polyprotein processing by nsp3 and nsp5 (
                               <a id="xref-fig-16-7" class="xref-fig" href="#F16">
                                Figure S13C-D
                               </a>
                               ).
                              </p>
                              <p id="p-75">
                               Multiple highly-ranked mutations are distributed across the replication and transcription-associated nsps in ORF1b (
                               <a id="xref-fig-16-8" class="xref-fig" href="#F16">
                                Figure S13D
                               </a>
                               ). The P314L (P323L) mutation in nsp12 â the viral RNA-dependent RNA polymerase (RdRP) â emerged early during the pandemic and became established in circulating lineages alongside S D614G (
                               <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                               ). A later variant at this site (P314F) was also highly ranked in our list. Additional mutations in nsp12 can be found within the canonical fingers (D445A, V631I, D514N, G662S), palm (M592I, H604Y, T701I, C721R, S763F), and thumb (L820F, L829I, D870N) subdomains of the RdRP conserved catalytic fold (
                               <a id="xref-fig-18-3" class="xref-fig" href="#F18">
                                Figure S15
                               </a>
                               ). The functional effects of these mutations on polymerase processivity and fidelity remain to be investigated. A structural model of the SARS-CoV-2 polymerase complex has been resolved (
                               <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                               )(
                               <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ), and contains a single subunit of nsp12, two subunits of the nsp13 helicase, and additional RdRP cofactor proteins (nsp7, 8, and 9). The ORF1b P314 residue is located at the interaction interface between nsp12 and a single subunit of nsp8. Moreover, several of the top mutations from our dataset ORF1b (e.g., P1000L, P1001S, Q1011H) are harbored within the nsp13 N-terminal zinc-binding domain that directly interacts with nsp8 (
                               <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                               ). These findings implicate transmission-associated mutations within the SARS-CoV-2 RNA synthesis machinery in altering the stability of the replication complex, possibly via interactions with nsp8.
                              </p>
                              <p id="p-76">
                               Nsp14 is a dual-functional enzyme with N-terminal 3â-to-5â exonuclease (ExoN) and C-terminal guanine-N7 methyltransferase (N7-MTase) activities (
                               <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                               )(
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                               ) and is a core component of the coronavirus RNA proofreading complex. Nsp14 is uniquely responsible for excision of mismatched bases from the nascent RNA and methylation of the viral mRNA cap structure. Two mutational hotspots in nsp14 map to discrete regions in the ExoN (e.g., T1540I, I1566V) and N7-MTase (e.g., D1848Y, P1936H) domains. The functional consequences of these clusters of transmission-associated mutations on mRNA synthesis and genome replication remain unknown.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-29">
                             <h3>
                              Supplementary Figures
                             </h3>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F4.large.jpg?width=800&height=600&carousel=1" title="A. Sensitivity of lineage fitness estimates to data subset. We depict the relative fitness of all lineages as estimated by either the full data or two disjoint geographic subsets (within Europe and outside Europe). High Pearson correlation (Ï) suggests estimates are largely insensitive to data subset. B. Estimates of fold increases in fitness for the top 20 lineages. Sensitivity analysis shows consistency across estimates from subsets of the data in different geographic regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A</strong>. Sensitivity of lineage fitness estimates to data subset. We depict the relative fitness of all lineages as estimated by either the full data or two disjoint geographic subsets (within Europe and outside Europe). High Pearson correlation (Ï) suggests estimates are largely insensitive to data subset. <strong>B</strong>. Estimates of fold increases in fitness for the top 20 lineages. Sensitivity analysis shows consistency across estimates from subsets of the data in different geographic regions.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F4.medium.gif" width="440" height="396"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F4.medium.gif" width="440" height="396"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S1.
                               </span>
                               <p id="p-77" class="first-child">
                                <strong>
                                 A
                                </strong>
                                . Sensitivity of lineage fitness estimates to data subset. We depict the relative fitness of all lineages as estimated by either the full data or two disjoint geographic subsets (within Europe and outside Europe). High Pearson correlation (Ï) suggests estimates are largely insensitive to data subset.
                                <strong>
                                 B
                                </strong>
                                . Estimates of fold increases in fitness for the top 20 lineages. Sensitivity analysis shows consistency across estimates from subsets of the data in different geographic regions.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F5.large.jpg?width=800&height=600&carousel=1" title="Simulation study assessing bias. Distribution of inferred fitness of new lineages as a function of time, for a simulated neutrally evolving viral population. The most successful subclades of each generation are designated as new lineages, leading to a trend toward higher estimated fitness even though all lineages are equally transmissible." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Simulation study assessing bias. Distribution of inferred fitness of new lineages as a function of time, for a simulated neutrally evolving viral population. The most successful subclades of each generation are designated as new lineages, leading to a trend toward higher estimated fitness even though all lineages are equally transmissible.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F5.medium.gif" width="440" height="331"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F5.medium.gif" width="440" height="331"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S2.
                               </span>
                               <p id="p-78" class="first-child">
                                Simulation study assessing bias. Distribution of inferred fitness of new lineages as a function of time, for a simulated neutrally evolving viral population. The most successful subclades of each generation are designated as new lineages, leading to a trend toward higher estimated fitness even though all lineages are equally transmissible.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-11.large.jpg?width=800&height=600&carousel=1" title="A. Regional fits and forecasts for USA, France, England, Brazil, Australia and Russia. Solid circles at the beginning of each two week time interval denote observed lineage proportions on a [0, 1] scale for the top 20 lineages. Solid curves and 95% confidence bands denote model predictions and three-month forecasts. Each of the six (aggregate) regions is made up of multiple subregions. The behavior of each SARS-Cov-2 cluster in each subregion is represented by only two numbers in the model: a slope and an intercept. The complex model fit results from the multivariate logistic function applied jointly to multiple competing trends, which are then aggregated over subregions and multiple clusters per lineage. England shows clear waves of dominance: B.1.1, B.1.177, B.1.1.7 (Alpha), AY.4 (Delta), and finally BA.1 (Omicron), with the latter currently being overtaken by BA.1.1 (also Omicron). Massachusetts and Brazil both start with very low sampling rates early in the pandemic. The legend reports the estimated fitness for the top 15 lineages. B. Region-specific fits for several regions in Asia, demonstrating better fits in regions with high sampling (Japan, India), and degraded fits in regions with low sampling (Myanmar, Pakistan)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A.</strong> Regional fits and forecasts for USA, France, England, Brazil, Australia and Russia. Solid circles at the beginning of each two week time interval denote observed lineage proportions on a [0, 1] scale for the top 20 lineages. Solid curves and 95% confidence bands denote model predictions and three-month forecasts. Each of the six (aggregate) regions is made up of multiple subregions. The behavior of each SARS-Cov-2 cluster in each subregion is represented by only two numbers in the model: a slope and an intercept. The complex model fit results from the multivariate logistic function applied jointly to multiple competing trends, which are then aggregated over subregions and multiple clusters per lineage. England shows clear waves of dominance: B.1.1, B.1.177, B.1.1.7 (Alpha), AY.4 (Delta), and finally BA.1 (Omicron), with the latter currently being overtaken by BA.1.1 (also Omicron). Massachusetts and Brazil both start with very low sampling rates early in the pandemic. The legend reports the estimated fitness for the top 15 lineages. <strong>B.</strong> Region-specific fits for several regions in Asia, demonstrating better fits in regions with high sampling (Japan, India), and degraded fits in regions with low sampling (Myanmar, Pakistan).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-11.medium.gif" width="370" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-11.medium.gif" width="370" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-12.large.jpg?width=800&height=600&carousel=1" title="A. Regional fits and forecasts for USA, France, England, Brazil, Australia and Russia. Solid circles at the beginning of each two week time interval denote observed lineage proportions on a [0, 1] scale for the top 20 lineages. Solid curves and 95% confidence bands denote model predictions and three-month forecasts. Each of the six (aggregate) regions is made up of multiple subregions. The behavior of each SARS-Cov-2 cluster in each subregion is represented by only two numbers in the model: a slope and an intercept. The complex model fit results from the multivariate logistic function applied jointly to multiple competing trends, which are then aggregated over subregions and multiple clusters per lineage. England shows clear waves of dominance: B.1.1, B.1.177, B.1.1.7 (Alpha), AY.4 (Delta), and finally BA.1 (Omicron), with the latter currently being overtaken by BA.1.1 (also Omicron). Massachusetts and Brazil both start with very low sampling rates early in the pandemic. The legend reports the estimated fitness for the top 15 lineages. B. Region-specific fits for several regions in Asia, demonstrating better fits in regions with high sampling (Japan, India), and degraded fits in regions with low sampling (Myanmar, Pakistan)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A.</strong> Regional fits and forecasts for USA, France, England, Brazil, Australia and Russia. Solid circles at the beginning of each two week time interval denote observed lineage proportions on a [0, 1] scale for the top 20 lineages. Solid curves and 95% confidence bands denote model predictions and three-month forecasts. Each of the six (aggregate) regions is made up of multiple subregions. The behavior of each SARS-Cov-2 cluster in each subregion is represented by only two numbers in the model: a slope and an intercept. The complex model fit results from the multivariate logistic function applied jointly to multiple competing trends, which are then aggregated over subregions and multiple clusters per lineage. England shows clear waves of dominance: B.1.1, B.1.177, B.1.1.7 (Alpha), AY.4 (Delta), and finally BA.1 (Omicron), with the latter currently being overtaken by BA.1.1 (also Omicron). Massachusetts and Brazil both start with very low sampling rates early in the pandemic. The legend reports the estimated fitness for the top 15 lineages. <strong>B.</strong> Region-specific fits for several regions in Asia, demonstrating better fits in regions with high sampling (Japan, India), and degraded fits in regions with low sampling (Myanmar, Pakistan).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-12.medium.gif" width="401" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-12.medium.gif" width="401" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F6/graphic-12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S3.
                               </span>
                               <p id="p-79" class="first-child">
                                <strong>
                                 A.
                                </strong>
                                Regional fits and forecasts for USA, France, England, Brazil, Australia and Russia. Solid circles at the beginning of each two week time interval denote observed lineage proportions on a [0, 1] scale for the top 20 lineages. Solid curves and 95% confidence bands denote model predictions and three-month forecasts. Each of the six (aggregate) regions is made up of multiple subregions. The behavior of each SARS-Cov-2 cluster in each subregion is represented by only two numbers in the model: a slope and an intercept. The complex model fit results from the multivariate logistic function applied jointly to multiple competing trends, which are then aggregated over subregions and multiple clusters per lineage. England shows clear waves of dominance: B.1.1, B.1.177, B.1.1.7 (Alpha), AY.4 (Delta), and finally BA.1 (Omicron), with the latter currently being overtaken by BA.1.1 (also Omicron). Massachusetts and Brazil both start with very low sampling rates early in the pandemic. The legend reports the estimated fitness for the top 15 lineages.
                                <strong>
                                 B.
                                </strong>
                                Region-specific fits for several regions in Asia, demonstrating better fits in regions with high sampling (Japan, India), and degraded fits in regions with low sampling (Myanmar, Pakistan).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F7.large.jpg?width=800&height=600&carousel=1" title="Heterogeneity of PANGO lineages. We hypothesized that the PANGO lineage clustering conflated viruses with distinct growth rate, e.g. B.1.1 exhibits two peaks in relative abundance in England, contrary to our multivariate logistic model. To test this hypothesis we refined 1544 PANGO lineages into 3000 finer clusters and estimated each clusterâs growth rate. As shown in the figure some PANGO lineages include clusters with estimated fitness differing by more than a factor of 6, including the B.1.1 lineage. This heterogeneity is also reflected in the temporal structure: for example, the three B.1.1 clusters with the largest growth rate emerged in December 2021 and January 2022, whereas the majority of B.1.1 clusters emerged in the twelve months leading up to April 2021. The top four clusters in B.1.1 and the top cluster in B.1 are labeled by their top 5 fitness-increasing mutations to the S gene, relative to the PANGO lineageâs basal sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Heterogeneity of PANGO lineages. We hypothesized that the PANGO lineage clustering conflated viruses with distinct growth rate, e.g. B.1.1 exhibits two peaks in relative abundance in England, contrary to our multivariate logistic model. To test this hypothesis we refined 1544 PANGO lineages into 3000 finer clusters and estimated each clusterâs growth rate. As shown in the figure some PANGO lineages include clusters with estimated fitness differing by more than a factor of 6, including the B.1.1 lineage. This heterogeneity is also reflected in the temporal structure: for example, the three B.1.1 clusters with the largest growth rate emerged in December 2021 and January 2022, whereas the majority of B.1.1 clusters emerged in the twelve months leading up to April 2021. The top four clusters in B.1.1 and the top cluster in B.1 are labeled by their top 5 fitness-increasing mutations to the S gene, relative to the PANGO lineageâs basal sequence.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F7.medium.gif" width="440" height="268"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F7.medium.gif" width="440" height="268"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S4.
                               </span>
                               <p id="p-80" class="first-child">
                                Heterogeneity of PANGO lineages. We hypothesized that the PANGO lineage clustering conflated viruses with distinct growth rate, e.g. B.1.1 exhibits two peaks in relative abundance in England, contrary to our multivariate logistic model. To test this hypothesis we refined 1544 PANGO lineages into 3000 finer clusters and estimated each clusterâs growth rate. As shown in the figure some PANGO lineages include clusters with estimated fitness differing by more than a factor of 6, including the B.1.1 lineage. This heterogeneity is also reflected in the temporal structure: for example, the three B.1.1 clusters with the largest growth rate emerged in December 2021 and January 2022, whereas the majority of B.1.1 clusters emerged in the twelve months leading up to April 2021. The top four clusters in B.1.1 and the top cluster in B.1 are labeled by their top 5 fitness-increasing mutations to the S gene, relative to the PANGO lineageâs basal sequence.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F8.large.jpg?width=800&height=600&carousel=1" title="Forecasts in England with time-truncated input data. (A) Prediction for rise of B.1.1.7 using data through late November 2020 (solid circles at the beginning of each two-week time interval). (B) Prediction for rise of AY.4 using data through early May 2021. (C) Prediction for rise of BA.1 using data through mid December 2021, and (D) late December 2021. Future data points, not used during the model training, are shown in crosses. The legend reports lineage fitness estimates based on all available data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Forecasts in England with time-truncated input data. (A) Prediction for rise of B.1.1.7 using data through late November 2020 (solid circles at the beginning of each two-week time interval). (B) Prediction for rise of AY.4 using data through early May 2021. (C) Prediction for rise of BA.1 using data through mid December 2021, and (D) late December 2021. Future data points, not used during the model training, are shown in crosses. The legend reports lineage fitness estimates based on all available data.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F8.medium.gif" width="440" height="345"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F8.medium.gif" width="440" height="345"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S5.
                               </span>
                               <p id="p-81" class="first-child">
                                Forecasts in England with time-truncated input data. (A) Prediction for rise of B.1.1.7 using data through late November 2020 (solid circles at the beginning of each two-week time interval). (B) Prediction for rise of AY.4 using data through early May 2021. (C) Prediction for rise of BA.1 using data through mid December 2021, and (D) late December 2021. Future data points, not used during the model training, are shown in crosses. The legend reports lineage fitness estimates based on all available data.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F9.large.jpg?width=800&height=600&carousel=1" title="We depict the ability of PyR0 to predict the fitness of Omicron sublineages BA.1 and BA.2 as the number of sequenced genomes increased throughout the last two months of 2021. PyR0 predicted that BA.1 (respectively, BA.2) was substantially more fit than Delta by December 1st (15th) 2021, by which time 2906 (76) genomic sequences had been collected. The substantial heterogeneity of the BA.1 sublineage is reflected in the uncertainty in BA.1 R estimates; this heterogeneity also helps explain why PyR0 required more sequences to identify the elevated fitness of BA.1 as compared to the case of BA.2. Left: The number of amino acid mutations that make up BA.1 and BA.2 that had been observed in at least 10/102/103/104 sequences by the given date. Middle: Estimates of R/RA using sequences collected by the given date. Right: The total number of BA.1 and BA.2 sequences collected by the given date." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">We depict the ability of PyR<sub>0</sub> to predict the fitness of Omicron sublineages BA.1 and BA.2 as the number of sequenced genomes increased throughout the last two months of 2021. PyR<sub>0</sub> predicted that BA.1 (respectively, BA.2) was substantially more fit than Delta by December 1<sup>st</sup> (15<sup>th</sup>) 2021, by which time 2906 (76) genomic sequences had been collected. The substantial heterogeneity of the BA.1 sublineage is reflected in the uncertainty in BA.1 R estimates; this heterogeneity also helps explain why PyR<sub>0</sub> required more sequences to identify the elevated fitness of BA.1 as compared to the case of BA.2. <strong>Left:</strong> The number of amino acid mutations that make up BA.1 and BA.2 that had been observed in at least 10/10<sup>2</sup>/10<sup>3</sup>/10<sup>4</sup> sequences by the given date. <strong>Middle:</strong> Estimates of R/R<sub>A</sub> using sequences collected by the given date. <strong>Right:</strong> The total number of BA.1 and BA.2 sequences collected by the given date.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F9.medium.gif" width="440" height="218"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F9.medium.gif" width="440" height="218"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S6.
                               </span>
                               <p id="p-82" class="first-child">
                                We depict the ability of PyR
                                <sub>
                                 0
                                </sub>
                                to predict the fitness of Omicron sublineages BA.1 and BA.2 as the number of sequenced genomes increased throughout the last two months of 2021. PyR
                                <sub>
                                 0
                                </sub>
                                predicted that BA.1 (respectively, BA.2) was substantially more fit than Delta by December 1
                                <sup>
                                 st
                                </sup>
                                (15
                                <sup>
                                 th
                                </sup>
                                ) 2021, by which time 2906 (76) genomic sequences had been collected. The substantial heterogeneity of the BA.1 sublineage is reflected in the uncertainty in BA.1 R estimates; this heterogeneity also helps explain why PyR
                                <sub>
                                 0
                                </sub>
                                required more sequences to identify the elevated fitness of BA.1 as compared to the case of BA.2.
                                <strong>
                                 Left:
                                </strong>
                                The number of amino acid mutations that make up BA.1 and BA.2 that had been observed in at least 10/10
                                <sup>
                                 2
                                </sup>
                                /10
                                <sup>
                                 3
                                </sup>
                                /10
                                <sup>
                                 4
                                </sup>
                                sequences by the given date.
                                <strong>
                                 Middle:
                                </strong>
                                Estimates of R/R
                                <sub>
                                 A
                                </sub>
                                using sequences collected by the given date.
                                <strong>
                                 Right:
                                </strong>
                                The total number of BA.1 and BA.2 sequences collected by the given date.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F10.large.jpg?width=800&height=600&carousel=1" title="Forecasting evaluation based on independently trained models at 45 time points during the pandemic, tcensor, and predicting at time tpredicted up to 12 two-week periods into the future. The results are shown for (A) England and (B) Massachusetts, USA. The top panels are as in Figure S3, heatmaps depict the prediction L1 error, and the inset bar plots depict the aggregated prediction errors over all periods. Note the rapid increase in error as new fit lineages emerge in a region (vertical dashed lines provided as a guide to the eye), followed by rapid recovery and stabilization of forecasting accuracy within only a single period, highlighting the predictive value of PyR0 for detecting variants of concern. Refer to Table S1 for tabulated forecasting accuracy figures in several other regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Forecasting evaluation based on independently trained models at 45 time points during the pandemic, t<sub>censor</sub>, and predicting at time t<sub>predicted</sub> up to 12 two-week periods into the future. The results are shown for (A) England and (B) Massachusetts, USA. The top panels are as in Figure S3, heatmaps depict the prediction L1 error, and the inset bar plots depict the aggregated prediction errors over all periods. Note the rapid increase in error as new fit lineages emerge in a region (vertical dashed lines provided as a guide to the eye), followed by rapid recovery and stabilization of forecasting accuracy within only a single period, highlighting the predictive value of PyR<sub>0</sub> for detecting variants of concern. Refer to Table S1 for tabulated forecasting accuracy figures in several other regions.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F10.medium.gif" width="440" height="234"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F10.medium.gif" width="440" height="234"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S7:
                               </span>
                               <p id="p-83" class="first-child">
                                Forecasting evaluation based on independently trained models at 45 time points during the pandemic, t
                                <sub>
                                 censor
                                </sub>
                                , and predicting at time t
                                <sub>
                                 predicted
                                </sub>
                                up to 12 two-week periods into the future. The results are shown for (A) England and (B) Massachusetts, USA. The top panels are as in
                                <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                 Figure S3
                                </a>
                                , heatmaps depict the prediction L1 error, and the inset bar plots depict the aggregated prediction errors over all periods. Note the rapid increase in error as new fit lineages emerge in a region (vertical dashed lines provided as a guide to the eye), followed by rapid recovery and stabilization of forecasting accuracy within only a single period, highlighting the predictive value of PyR
                                <sub>
                                 0
                                </sub>
                                for detecting variants of concern. Refer to
                                <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">
                                 Table S1
                                </a>
                                for tabulated forecasting accuracy figures in several other regions.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F11" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F11.large.jpg?width=800&height=600&carousel=1" title="A. Histogram of the number of independent emergences across all observed mutations. A mutation was considered to have emerged independently if it was present in a lineage but not in its parent. B. Scatterplot of the fold-change in fitness versus the number of independent emergences. The top 10 ranked mutations are labeled, colored by ORF. Linear regression with standard error for the slope given as shaded area. C. Violin plots of fold-change in fitness for mutation, grouped by gene. The top 10% most statistically significant mutations are shown (where significance is determined by z-score of the approximate variational posterior)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A.</strong> Histogram of the number of independent emergences across all observed mutations. A mutation was considered to have emerged independently if it was present in a lineage but not in its parent. <strong>B.</strong> Scatterplot of the fold-change in fitness versus the number of independent emergences. The top 10 ranked mutations are labeled, colored by ORF. Linear regression with standard error for the slope given as shaded area. <strong>C.</strong> Violin plots of fold-change in fitness for mutation, grouped by gene. The top 10% most statistically significant mutations are shown (where significance is determined by z-score of the approximate variational posterior).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F11.medium.gif" width="440" height="183"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F11.medium.gif" width="440" height="183"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S8.
                               </span>
                               <p id="p-84" class="first-child">
                                <strong>
                                 A.
                                </strong>
                                Histogram of the number of independent emergences across all observed mutations. A mutation was considered to have emerged independently if it was present in a lineage but not in its parent.
                                <strong>
                                 B.
                                </strong>
                                Scatterplot of the fold-change in fitness versus the number of independent emergences. The top 10 ranked mutations are labeled, colored by ORF. Linear regression with standard error for the slope given as shaded area.
                                <strong>
                                 C.
                                </strong>
                                Violin plots of fold-change in fitness for mutation, grouped by gene. The top 10% most statistically significant mutations are shown (where significance is determined by z-score of the approximate variational posterior).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F12" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F12.large.jpg?width=800&height=600&carousel=1" title="Accuracy of predicted fitness based solely on mutation content. Left: estimated log(R/RA) for each lineage based on the full set of GISAID samples (y axis), and on the leave-one-out subset with each lineageâs subclade removed and the fitness estimated from the mutations present in the lineage (x axis). Right: the same quantities but relative to a baseline estimator in which each child lineageâs fitness is the same as that of its parent lineage. If a mutation is entirely removed from the LOO dataset, then the corresponding mutation coefficient is estimated as zero. The evaluation metrics are Pearson correlation (â´) and mean absolute error (MAE). The MAE of the leave-one-out estimator is 0.001, more than 100x smaller than the MAE of the baseline estimator (0.129). Both panels highlight the CDCâs variants of concern and variants of interest. The lineages selected for testing are those with at least 100 samples and with the largest deviations from their parent, i.e. where the baseline estimator performs worst. Note that the fitness of child lineages can deviate substantially from that of the parent, e.g. BA.1 is surprisingly higher fitness than its parent B.1.1.529." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Accuracy of predicted fitness based solely on mutation content. <strong>Left:</strong> estimated log(R/R<sub>A</sub>) for each lineage based on the full set of GISAID samples (y axis), and on the leave-one-out subset with each lineageâs subclade removed and the fitness estimated from the mutations present in the lineage (x axis). <strong>Right:</strong> the same quantities but relative to a baseline estimator in which each child lineageâs fitness is the same as that of its parent lineage. If a mutation is entirely removed from the LOO dataset, then the corresponding mutation coefficient is estimated as zero. The evaluation metrics are Pearson correlation (â´) and mean absolute error (MAE). The MAE of the leave-one-out estimator is 0.001, more than 100x smaller than the MAE of the baseline estimator (0.129). Both panels highlight the CDCâs variants of concern and variants of interest. The lineages selected for testing are those with at least 100 samples and with the largest deviations from their parent, i.e. where the baseline estimator performs worst. Note that the fitness of child lineages can deviate substantially from that of the parent, e.g. BA.1 is surprisingly higher fitness than its parent B.1.1.529.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F12.medium.gif" width="440" height="210"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S9." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F12.medium.gif" width="440" height="210"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S9.
                               </span>
                               <p id="p-85" class="first-child">
                                Accuracy of predicted fitness based solely on mutation content.
                                <strong>
                                 Left:
                                </strong>
                                estimated log(R/R
                                <sub>
                                 A
                                </sub>
                                ) for each lineage based on the full set of GISAID samples (y axis), and on the leave-one-out subset with each lineageâs subclade removed and the fitness estimated from the mutations present in the lineage (x axis).
                                <strong>
                                 Right:
                                </strong>
                                the same quantities but relative to a baseline estimator in which each child lineageâs fitness is the same as that of its parent lineage. If a mutation is entirely removed from the LOO dataset, then the corresponding mutation coefficient is estimated as zero. The evaluation metrics are Pearson correlation (â´) and mean absolute error (MAE). The MAE of the leave-one-out estimator is 0.001, more than 100x smaller than the MAE of the baseline estimator (0.129). Both panels highlight the CDCâs variants of concern and variants of interest. The lineages selected for testing are those with at least 100 samples and with the largest deviations from their parent, i.e. where the baseline estimator performs worst. Note that the fitness of child lineages can deviate substantially from that of the parent, e.g. BA.1 is surprisingly higher fitness than its parent B.1.1.529.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F13" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F13.large.jpg?width=800&height=600&carousel=1" title="Volcano plot highlighting the most statistically significant mutations linked to increased fitness. The x-axis depicts the effect size as a ratio of estimated fitness of lineages with-versus-without each mutation. The y-axis depicts z-score from the approximate variational posterior as a proxy for statistical significance. The top 50 most statistically significant mutations are labeled, colored by gene. The 540 growth-associated mutations with z-score greater than 5 are shaded dark gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Volcano plot highlighting the most statistically significant mutations linked to increased fitness. The x-axis depicts the effect size as a ratio of estimated fitness of lineages with-versus-without each mutation. The y-axis depicts z-score from the approximate variational posterior as a proxy for statistical significance. The top 50 most statistically significant mutations are labeled, colored by gene. The 540 growth-associated mutations with z-score greater than 5 are shaded dark gray.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F13.medium.gif" width="440" height="313"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S10." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F13.medium.gif" width="440" height="313"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S10.
                               </span>
                               <p id="p-86" class="first-child">
                                Volcano plot highlighting the most statistically significant mutations linked to increased fitness. The x-axis depicts the effect size as a ratio of estimated fitness of lineages with-versus-without each mutation. The y-axis depicts z-score from the approximate variational posterior as a proxy for statistical significance. The top 50 most statistically significant mutations are labeled, colored by gene. The 540 growth-associated mutations with z-score greater than 5 are shaded dark gray.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F14" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-20.large.jpg?width=800&height=600&carousel=1" title="Sensitivity of mutation estimates. A. Scatterplot of the mutation coefficients on the full model and geographic subsets, with Pearson correlation (Ï) as shown. B. Box-and-whisker plot depicting estimated growth rates with corresponding uncertainties for the 20 lineages with highest growth rate (effect size) across geographic subsets. C. Same as B but with the top 20 lineages sorted by statistical significance (z-score). Note that in B, the World estimates (center) tend to be higher than subsets (left and right) only because the ranked selection is based on those estimates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Sensitivity of mutation estimates. <strong>A.</strong> Scatterplot of the mutation coefficients on the full model and geographic subsets, with Pearson correlation (Ï) as shown. <strong>B.</strong> Box-and-whisker plot depicting estimated growth rates with corresponding uncertainties for the 20 lineages with highest growth rate (effect size) across geographic subsets. <strong>C.</strong> Same as B but with the top 20 lineages sorted by statistical significance (z-score). Note that in B, the World estimates (center) tend to be higher than subsets (left and right) only because the ranked selection is based on those estimates.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-20.medium.gif" width="440" height="427"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S11." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-20.medium.gif" width="440" height="427"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-21.large.jpg?width=800&height=600&carousel=1" title="Sensitivity of mutation estimates. A. Scatterplot of the mutation coefficients on the full model and geographic subsets, with Pearson correlation (Ï) as shown. B. Box-and-whisker plot depicting estimated growth rates with corresponding uncertainties for the 20 lineages with highest growth rate (effect size) across geographic subsets. C. Same as B but with the top 20 lineages sorted by statistical significance (z-score). Note that in B, the World estimates (center) tend to be higher than subsets (left and right) only because the ranked selection is based on those estimates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Sensitivity of mutation estimates. <strong>A.</strong> Scatterplot of the mutation coefficients on the full model and geographic subsets, with Pearson correlation (Ï) as shown. <strong>B.</strong> Box-and-whisker plot depicting estimated growth rates with corresponding uncertainties for the 20 lineages with highest growth rate (effect size) across geographic subsets. <strong>C.</strong> Same as B but with the top 20 lineages sorted by statistical significance (z-score). Note that in B, the World estimates (center) tend to be higher than subsets (left and right) only because the ranked selection is based on those estimates.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-21.medium.gif" width="440" height="220"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S11." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-21.medium.gif" width="440" height="220"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F14/graphic-21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S11.
                               </span>
                               <p id="p-87" class="first-child">
                                Sensitivity of mutation estimates.
                                <strong>
                                 A.
                                </strong>
                                Scatterplot of the mutation coefficients on the full model and geographic subsets, with Pearson correlation (Ï) as shown.
                                <strong>
                                 B.
                                </strong>
                                Box-and-whisker plot depicting estimated growth rates with corresponding uncertainties for the 20 lineages with highest growth rate (effect size) across geographic subsets.
                                <strong>
                                 C.
                                </strong>
                                Same as B but with the top 20 lineages sorted by statistical significance (z-score). Note that in B, the World estimates (center) tend to be higher than subsets (left and right) only because the ranked selection is based on those estimates.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F15" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F15.large.jpg?width=800&height=600&carousel=1" title="Information content of different subsets of the SARS-CoV-2 genome in explaining fitness. The metric is the expected log likelihood. The dotted line at the top shows the performance of the full model that regresses against all genes (A) or against all of ORF1 (B). The circles show estimators based on only single genes (A) or single nonstructural proteins (B). The most informative genes are S, ORF1a, M, N, and ORF1b; within ORF1 the most informative nsps are nsp4, 3, 5, 6, 12 and 7. The bottom dashed lines show the performance of a naive estimator that ignores genetic information, effectively estimating each lineageâs growth rate in each region independent of growth rate estimates in other regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Information content of different subsets of the SARS-CoV-2 genome in explaining fitness. The metric is the expected log likelihood. The dotted line at the top shows the performance of the full model that regresses against all genes (A) or against all of ORF1 (B). The circles show estimators based on only single genes (A) or single nonstructural proteins (B). The most informative genes are S, ORF1a, M, N, and ORF1b; within ORF1 the most informative nsps are nsp4, 3, 5, 6, 12 and 7. The bottom dashed lines show the performance of a naive estimator that ignores genetic information, effectively estimating each lineageâs growth rate in each region independent of growth rate estimates in other regions.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F15.medium.gif" width="414" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S12." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F15.medium.gif" width="414" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S12." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S12.
                               </span>
                               <p id="p-88" class="first-child">
                                Information content of different subsets of the SARS-CoV-2 genome in explaining fitness. The metric is the expected log likelihood. The dotted line at the top shows the performance of the full model that regresses against all genes (A) or against all of ORF1 (B). The circles show estimators based on only single genes (A) or single nonstructural proteins (B). The most informative genes are S, ORF1a, M, N, and ORF1b; within ORF1 the most informative nsps are nsp4, 3, 5, 6, 12 and 7. The bottom dashed lines show the performance of a naive estimator that ignores genetic information, effectively estimating each lineageâs growth rate in each region independent of growth rate estimates in other regions.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F16" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-23.large.jpg?width=800&height=600&carousel=1" title="Manhattan plot details of the four most informative genes. See Figure 3 for a whole-genome view and Figure S12 for ranking by information. A. View of the 1237 amino acids of the S protein, annotated by structure (45); many mutations occur in the N-terminal domain (NTD), receptor-binding domain (RBD), and furin cleavage (FC) site. Regions containing the fusion peptide (FP), heptad repeat (HR) 1 and 2, transmembrane domain (TM), and C-terminal domain (CTD) are annotated. B. View of the 419 amino acids of the nucleocapsid (N) protein domains, annotated by structure (46); many mutations occur in the serineâarginine rich region (SR), identified by (47) as immunogenic. C. View of the ORF1a polyprotein, including 11 non-structural proteins (nsps). D. View of the ORF1b polyprotein, including nsp12-16; note the amino acid positions do not account for 9 additional residues at the N-terminus of nsp12 (RNA polymerase) resulting from the -1 ribosomal frameshift." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Manhattan plot details of the four most informative genes. See Figure 3 for a whole-genome view and Figure S12 for ranking by information. <strong>A.</strong> View of the 1237 amino acids of the S protein, annotated by structure (45); many mutations occur in the N-terminal domain (NTD), receptor-binding domain (RBD), and furin cleavage (FC) site. Regions containing the fusion peptide (FP), heptad repeat (HR) 1 and 2, transmembrane domain (TM), and C-terminal domain (CTD) are annotated. <strong>B.</strong> View of the 419 amino acids of the nucleocapsid (N) protein domains, annotated by structure (46); many mutations occur in the serineâarginine rich region (SR), identified by (47) as immunogenic. <strong>C.</strong> View of the ORF1a polyprotein, including 11 non-structural proteins (nsps). <strong>D.</strong> View of the ORF1b polyprotein, including nsp12-16; note the amino acid positions do not account for 9 additional residues at the N-terminus of nsp12 (RNA polymerase) resulting from the -1 ribosomal frameshift.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-23.medium.gif" width="381" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S13." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-23.medium.gif" width="381" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-24.large.jpg?width=800&height=600&carousel=1" title="Manhattan plot details of the four most informative genes. See Figure 3 for a whole-genome view and Figure S12 for ranking by information. A. View of the 1237 amino acids of the S protein, annotated by structure (45); many mutations occur in the N-terminal domain (NTD), receptor-binding domain (RBD), and furin cleavage (FC) site. Regions containing the fusion peptide (FP), heptad repeat (HR) 1 and 2, transmembrane domain (TM), and C-terminal domain (CTD) are annotated. B. View of the 419 amino acids of the nucleocapsid (N) protein domains, annotated by structure (46); many mutations occur in the serineâarginine rich region (SR), identified by (47) as immunogenic. C. View of the ORF1a polyprotein, including 11 non-structural proteins (nsps). D. View of the ORF1b polyprotein, including nsp12-16; note the amino acid positions do not account for 9 additional residues at the N-terminus of nsp12 (RNA polymerase) resulting from the -1 ribosomal frameshift." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Manhattan plot details of the four most informative genes. See Figure 3 for a whole-genome view and Figure S12 for ranking by information. <strong>A.</strong> View of the 1237 amino acids of the S protein, annotated by structure (45); many mutations occur in the N-terminal domain (NTD), receptor-binding domain (RBD), and furin cleavage (FC) site. Regions containing the fusion peptide (FP), heptad repeat (HR) 1 and 2, transmembrane domain (TM), and C-terminal domain (CTD) are annotated. <strong>B.</strong> View of the 419 amino acids of the nucleocapsid (N) protein domains, annotated by structure (46); many mutations occur in the serineâarginine rich region (SR), identified by (47) as immunogenic. <strong>C.</strong> View of the ORF1a polyprotein, including 11 non-structural proteins (nsps). <strong>D.</strong> View of the ORF1b polyprotein, including nsp12-16; note the amino acid positions do not account for 9 additional residues at the N-terminus of nsp12 (RNA polymerase) resulting from the -1 ribosomal frameshift.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-24.medium.gif" width="382" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S13." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-24.medium.gif" width="382" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F16/graphic-24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S13.
                               </span>
                               <p id="p-89" class="first-child">
                                Manhattan plot details of the four most informative genes. See
                                <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                 Figure 3
                                </a>
                                for a whole-genome view and
                                <a id="xref-fig-15-3" class="xref-fig" href="#F15">
                                 Figure S12
                                </a>
                                for ranking by information.
                                <strong>
                                 A.
                                </strong>
                                View of the 1237 amino acids of the S protein, annotated by structure (
                                <a href="https://paperpile.com/c/2OlJ78/wwiQc">
                                 45
                                </a>
                                ); many mutations occur in the N-terminal domain (NTD), receptor-binding domain (RBD), and furin cleavage (FC) site. Regions containing the fusion peptide (FP), heptad repeat (HR) 1 and 2, transmembrane domain (TM), and C-terminal domain (CTD) are annotated.
                                <strong>
                                 B.
                                </strong>
                                View of the 419 amino acids of the nucleocapsid (N) protein domains, annotated by structure (
                                <a href="https://paperpile.com/c/2OlJ78/4VCdI">
                                 46
                                </a>
                                ); many mutations occur in the serineâarginine rich region (SR), identified by (
                                <a href="https://paperpile.com/c/2OlJ78/bWrkf">
                                 47
                                </a>
                                ) as immunogenic.
                                <strong>
                                 C.
                                </strong>
                                View of the ORF1a polyprotein, including 11 non-structural proteins (nsps).
                                <strong>
                                 D.
                                </strong>
                                View of the ORF1b polyprotein, including nsp12-16; note the amino acid positions do not account for 9 additional residues at the N-terminus of nsp12 (RNA polymerase) resulting from the -1 ribosomal frameshift.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F17" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F17.large.jpg?width=800&height=600&carousel=1" title="Enrichment analysis of amino acid changes among top-ranked mutations A. Mutation frequency in top 1000 most significant mutations (as ranked by posterior mean/stddev) B. Leading set enrichment analysis of most significant mutations predicted by the model for non-symmetrical (e.g. A->V) (left) and symmetrical (e.g. A" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Enrichment analysis of amino acid changes among top-ranked mutations <strong>A.</strong> Mutation frequency in top 1000 most significant mutations (as ranked by posterior mean/stddev) <strong>B.</strong> Leading set enrichment analysis of most significant mutations predicted by the model for non-symmetrical (e.g. A->V) (left) and symmetrical (e.g. A<-<V) (right) amino acid changes. The blue curve depicts the most significant p-value obtained for different top-N mutation cutoff values across all amino acid changes, while the red curve depicts the mean p-value. <strong>C.</strong> Further examination reveals that top mutations are enriched in K to N changes in the S gene. <strong>D.</strong> No other genes (gray) other than S (in red) show significant enrichment.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S14:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F17.medium.gif" width="404" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S14:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F17.medium.gif" width="404" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S14:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S14:
                               </span>
                               <p id="p-90" class="first-child">
                                Enrichment analysis of amino acid changes among top-ranked mutations
                                <strong>
                                 A.
                                </strong>
                                Mutation frequency in top 1000 most significant mutations (as ranked by posterior mean/stddev)
                                <strong>
                                 B.
                                </strong>
                                Leading set enrichment analysis of most significant mutations predicted by the model for non-symmetrical (e.g. A->V) (left) and symmetrical (e.g. A<-<V) (right) amino acid changes. The blue curve depicts the most significant p-value obtained for different top-N mutation cutoff values across all amino acid changes, while the red curve depicts the mean p-value.
                                <strong>
                                 C.
                                </strong>
                                Further examination reveals that top mutations are enriched in K to N changes in the S gene.
                                <strong>
                                 D.
                                </strong>
                                No other genes (gray) other than S (in red) show significant enrichment.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F18" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F18.large.jpg?width=800&height=600&carousel=1" title="Top-ranked mutations in the viral RNA-dependent RNA polymerase (RdRP, nsp12, PDB: 7CYQ). Amino acid positions corresponding to top mutations are shown as red spheres. The catalytic site residues 750-SDD-752 are highlighted as yellow spheres. The coronavirus-specific domains (NiRAN, Interface) are shown as cartoon structures. The conserved RdRP domains (Fingers, Palm, Thumb) are shown as cartoon and surface filling structures" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Top-ranked mutations in the viral RNA-dependent RNA polymerase (RdRP, nsp12, PDB: 7CYQ). Amino acid positions corresponding to top mutations are shown as red spheres. The catalytic site residues 750-SDD-752 are highlighted as yellow spheres. The coronavirus-specific domains (NiRAN, Interface) are shown as cartoon structures. The conserved RdRP domains (Fingers, Palm, Thumb) are shown as cartoon and surface filling structures</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F18.medium.gif" width="440" height="178"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F18.medium.gif" width="440" height="178"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S15." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S15.
                               </span>
                               <p id="p-91" class="first-child">
                                Top-ranked mutations in the viral RNA-dependent RNA polymerase (RdRP, nsp12, PDB: 7CYQ). Amino acid positions corresponding to top mutations are shown as red spheres. The catalytic site residues 750-SDD-752 are highlighted as yellow spheres. The coronavirus-specific domains (NiRAN, Interface) are shown as cartoon structures. The conserved RdRP domains (Fingers, Palm, Thumb) are shown as cartoon and surface filling structures
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F19" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F19.large.jpg?width=800&height=600&carousel=1" title="Top-ranked mutations in the two viral proteases, PLpro (A) and Mpro (B). Both protease structures are shown in light blue. Amino acid positions corresponding to top mutations are shown as red spheres. The catalytic cysteine residues for each are shown as yellow spheres. Two active-site inhibitors, VIR250 and N3, are shown as orange spheres." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Top-ranked mutations in the two viral proteases, PLpro (A) and M<sup>pro</sup> (B). Both protease structures are shown in light blue. Amino acid positions corresponding to top mutations are shown as red spheres. The catalytic cysteine residues for each are shown as yellow spheres. Two active-site inhibitors, VIR250 and N3, are shown as orange spheres.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S16." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F19.medium.gif" width="440" height="142"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S16." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F19.medium.gif" width="440" height="142"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S16." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S16.
                               </span>
                               <p id="p-92" class="first-child">
                                Top-ranked mutations in the two viral proteases, PLpro (A) and M
                                <sup>
                                 pro
                                </sup>
                                (B). Both protease structures are shown in light blue. Amino acid positions corresponding to top mutations are shown as red spheres. The catalytic cysteine residues for each are shown as yellow spheres. Two active-site inhibitors, VIR250 and N3, are shown as orange spheres.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F20" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F20.large.jpg?width=800&height=600&carousel=1" title="Distribution of samples among regions. Each 2nd level GISAID region (country) is plotted as a curve, with the sizes of 3rd level GISAID regions (usually provinces or states) plotted as points along the curve. The 3rd level is dominated by a few countries with many small regions (e.g. Belgium with 1475 regions), so we merge regions smaller than a threshold (50 samples) into their respective countries." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Distribution of samples among regions. Each 2nd level GISAID region (country) is plotted as a curve, with the sizes of 3rd level GISAID regions (usually provinces or states) plotted as points along the curve. The 3rd level is dominated by a few countries with many small regions (e.g. Belgium with 1475 regions), so we merge regions smaller than a threshold (50 samples) into their respective countries.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S17." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F20.medium.gif" width="440" height="268"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S17." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F20.medium.gif" width="440" height="268"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S17." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S17.
                               </span>
                               <p id="p-93" class="first-child">
                                Distribution of samples among regions. Each 2nd level GISAID region (country) is plotted as a curve, with the sizes of 3rd level GISAID regions (usually provinces or states) plotted as points along the curve. The 3rd level is dominated by a few countries with many small regions (e.g. Belgium with 1475 regions), so we merge regions smaller than a threshold (50 samples) into their respective countries.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F21" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F21.large.jpg?width=800&height=600&carousel=1" title="Distribution of samples among PANGO lineages and refined clusters. PANGO lineage sizes are heavy-tailed and appear heterogeneous, so we split into a larger number of clusters (colored). We chose a final clustering of 3000 clusters (orange), balancing between a smaller number of clusters (which improves statistical efficiency) and a larger number of clusters (which better represents lineage heterogeneity)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Distribution of samples among PANGO lineages and refined clusters. PANGO lineage sizes are heavy-tailed and appear heterogeneous, so we split into a larger number of clusters (colored). We chose a final clustering of 3000 clusters (orange), balancing between a smaller number of clusters (which improves statistical efficiency) and a larger number of clusters (which better represents lineage heterogeneity).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S18." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F21.medium.gif" width="440" height="283"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S18." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F21.medium.gif" width="440" height="283"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S18." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S18.
                               </span>
                               <p id="p-94" class="first-child">
                                Distribution of samples among PANGO lineages and refined clusters. PANGO lineage sizes are heavy-tailed and appear heterogeneous, so we split into a larger number of clusters (colored). We chose a final clustering of 3000 clusters (orange), balancing between a smaller number of clusters (which improves statistical efficiency) and a larger number of clusters (which better represents lineage heterogeneity).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F22" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F22.large.jpg?width=800&height=600&carousel=1" title="Example algorithmic clustering of three synthetic trees. Example synthetic mutation annotated trees are clustered into smaller trees with only 10 nodes. Nodes are annotated by the number of sequences represented by each cluster. Edges are annotated by the edit distance between clusters. (A) clusters a balanced binary tree of 262,143 nodes, (B) clusters a single linear chain of 20,001 nodes, (C) clusters a random binary tree with Geometric(Â½)-many children at each node and 200,000 nodes. In all examples the clustered trees are approximately balanced insofar as they exhibit narrow distributions of edge distances (in SNPs) and the cluster sizes (in genomes sampled)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup">Example algorithmic clustering of three synthetic trees. Example synthetic mutation annotated trees are clustered into smaller trees with only 10 nodes. Nodes are annotated by the number of sequences represented by each cluster. Edges are annotated by the edit distance between clusters. (A) clusters a balanced binary tree of 262,143 nodes, (B) clusters a single linear chain of 20,001 nodes, (C) clusters a random binary tree with Geometric(Â½)-many children at each node and 200,000 nodes. In all examples the clustered trees are approximately balanced insofar as they exhibit narrow distributions of edge distances (in SNPs) and the cluster sizes (in genomes sampled).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S19:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F22.medium.gif" width="426" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S19:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F22.medium.gif" width="426" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S19:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S19:
                               </span>
                               <p id="p-95" class="first-child">
                                Example algorithmic clustering of three synthetic trees. Example synthetic mutation annotated trees are clustered into smaller trees with only 10 nodes. Nodes are annotated by the number of sequences represented by each cluster. Edges are annotated by the edit distance between clusters. (A) clusters a balanced binary tree of 262,143 nodes, (B) clusters a single linear chain of 20,001 nodes, (C) clusters a random binary tree with Geometric(Â½)-many children at each node and 200,000 nodes. In all examples the clustered trees are approximately balanced insofar as they exhibit narrow distributions of edge distances (in SNPs) and the cluster sizes (in genomes sampled).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F23" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F23.large.jpg?width=800&height=600&carousel=1" title="Probabilistic graphical model structure of the PyR0 model. Variables Ï are scale parameters of distributions. Variable Î²f is the per-amino-acid-substitution fitness coefficient. Variables Î²pc and Î±pc are respectively the per-region per-cluster slope and intercepts parametrizing multivariate logistic growth curves. The mean parameter of Î²pc is determined by Î²f via matrix multiplication by the feature matrix Xcf. The mean parameter of Î±pc is a per-cluster intercept Î±c shared across regions. The multinomial observations are vectors ytp. each of whose entries ytpc is the number of samples of cluster c in place p in time bucket t. Green boxes denote plates, i.e. conditionally independent replicas of random variables. Note the vector-valued observation ytp. is outside of the C plate because the multinomial distribution couples entries across the cluster coordinate c. Because the P x C plates are sparse (in most places most clusters never appear) the model omits Î±pc and Î²pc for pairs (p,c) with no observations in y." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Probabilistic graphical model structure of the PyR<sub>0</sub> model. Variables Ï are scale parameters of distributions. Variable Î²<sub>f</sub> is the per-amino-acid-substitution fitness coefficient. Variables Î²<sub>pc</sub> and Î±<sub>pc</sub> are respectively the per-region per-cluster slope and intercepts parametrizing multivariate logistic growth curves. The mean parameter of Î²<sub>pc</sub> is determined by Î²<sub>f</sub> via matrix multiplication by the feature matrix X<sub>cf</sub>. The mean parameter of Î±<sub>pc</sub> is a per-cluster intercept Î±<sub>c</sub> shared across regions. The multinomial observations are vectors y<sub>tp.</sub> each of whose entries y<sub>tpc</sub> is the number of samples of cluster c in place p in time bucket t. Green boxes denote plates, i.e. conditionally independent replicas of random variables. Note the vector-valued observation y<sub>tp.</sub> is outside of the C plate because the multinomial distribution couples entries across the cluster coordinate c. Because the P x C plates are sparse (in most places most clusters never appear) the model omits Î±<sub>pc</sub> and Î²<sub>pc</sub> for pairs (p,c) with no observations in y.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S20." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F23.medium.gif" width="440" height="242"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S20." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F23.medium.gif" width="440" height="242"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S20." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S20.
                               </span>
                               <p id="p-96" class="first-child">
                                Probabilistic graphical model structure of the PyR
                                <sub>
                                 0
                                </sub>
                                model. Variables Ï are scale parameters of distributions. Variable Î²
                                <sub>
                                 f
                                </sub>
                                is the per-amino-acid-substitution fitness coefficient. Variables Î²
                                <sub>
                                 pc
                                </sub>
                                and Î±
                                <sub>
                                 pc
                                </sub>
                                are respectively the per-region per-cluster slope and intercepts parametrizing multivariate logistic growth curves. The mean parameter of Î²
                                <sub>
                                 pc
                                </sub>
                                is determined by Î²
                                <sub>
                                 f
                                </sub>
                                via matrix multiplication by the feature matrix X
                                <sub>
                                 cf
                                </sub>
                                . The mean parameter of Î±
                                <sub>
                                 pc
                                </sub>
                                is a per-cluster intercept Î±
                                <sub>
                                 c
                                </sub>
                                shared across regions. The multinomial observations are vectors y
                                <sub>
                                 tp.
                                </sub>
                                each of whose entries y
                                <sub>
                                 tpc
                                </sub>
                                is the number of samples of cluster c in place p in time bucket t. Green boxes denote plates, i.e. conditionally independent replicas of random variables. Note the vector-valued observation y
                                <sub>
                                 tp.
                                </sub>
                                is outside of the C plate because the multinomial distribution couples entries across the cluster coordinate c. Because the P x C plates are sparse (in most places most clusters never appear) the model omits Î±
                                <sub>
                                 pc
                                </sub>
                                and Î²
                                <sub>
                                 pc
                                </sub>
                                for pairs (p,c) with no observations in y.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F24" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F24.large.jpg?width=800&height=600&carousel=1" title="Convergence of variational inference algorithm. A. Convergence of ELBO loss. B. Convergence of posterior medians of scale parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Convergence of variational inference algorithm. <strong>A.</strong> Convergence of ELBO loss. <strong>B.</strong> Convergence of posterior medians of scale parameters.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S21." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F24.medium.gif" width="398" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S21." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F24.medium.gif" width="398" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S21." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S21.
                               </span>
                               <p id="p-97" class="first-child">
                                Convergence of variational inference algorithm.
                                <strong>
                                 A.
                                </strong>
                                Convergence of ELBO loss.
                                <strong>
                                 B.
                                </strong>
                                Convergence of posterior medians of scale parameters.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F25" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F25.large.jpg?width=800&height=600&carousel=1" title="Screenshot of https://nextstrain.org displaying Nextstrainâs subsampled phylogeny with color and y-axis (mutational fitness) determined by our model predicted Î log R for each lineage (here using a slightly older version of our model). Although PyR0 does not explicitly rely on phylogenetic information, fitness estimates vary smoothly across the phylogeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Screenshot of https://nextstrain.org displaying Nextstrainâs subsampled phylogeny with color and y-axis (mutational fitness) determined by our model predicted Î log R for each lineage (here using a slightly older version of our model). Although PyR<sub>0</sub> does not explicitly rely on phylogenetic information, fitness estimates vary smoothly across the phylogeny.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S22:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F25.medium.gif" width="440" height="435"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S22:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F25.medium.gif" width="440" height="435"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S22:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S22:
                               </span>
                               <p id="p-98" class="first-child">
                                Screenshot of
                                <a href="https://nextstrain.org">
                                 https://nextstrain.org
                                </a>
                                displaying Nextstrainâs subsampled phylogeny with color and y-axis (mutational fitness) determined by our model predicted Î log R for each lineage (here using a slightly older version of our model). Although PyR
                                <sub>
                                 0
                                </sub>
                                does not explicitly rely on phylogenetic information, fitness estimates vary smoothly across the phylogeny.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F26" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F26.large.jpg?width=800&height=600&carousel=1" title="We show that PyR0 lineage-level Î log R estimates are largely driven by regions with the largest numbers of samples, as would be expected from a Bayesian hierarchical model. The vertical axis depicts Î log R estimates based on all 1560 regions, while the horizontal axis on the left (respectively, right) depicts Î log R estimates based on the 112 (60) regions with at least 104 (2 â¨ 104) samples. Collectively these regions contain 80.7% (69.8%) of the total number of SARS-CoV-2 sequences in our full dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">We show that PyR<sub>0</sub> lineage-level Î log R estimates are largely driven by regions with the largest numbers of samples, as would be expected from a Bayesian hierarchical model. The vertical axis depicts Î log R estimates based on all 1560 regions, while the horizontal axis on the left (respectively, right) depicts Î log R estimates based on the 112 (60) regions with at least 10<sup>4</sup> (2 â¨ 10<sup>4</sup>) samples. Collectively these regions contain 80.7% (69.8%) of the total number of SARS-CoV-2 sequences in our full dataset.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S23." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F26.medium.gif" width="440" height="214"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S23." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F26.medium.gif" width="440" height="214"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F26.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S23." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F26.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S23.
                               </span>
                               <p id="p-99" class="first-child">
                                We show that PyR
                                <sub>
                                 0
                                </sub>
                                lineage-level Î log R estimates are largely driven by regions with the largest numbers of samples, as would be expected from a Bayesian hierarchical model. The vertical axis depicts Î log R estimates based on all 1560 regions, while the horizontal axis on the left (respectively, right) depicts Î log R estimates based on the 112 (60) regions with at least 10
                                <sup>
                                 4
                                </sup>
                                (2 â¨ 10
                                <sup>
                                 4
                                </sup>
                                ) samples. Collectively these regions contain 80.7% (69.8%) of the total number of SARS-CoV-2 sequences in our full dataset.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F27" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F27.large.jpg?width=800&height=600&carousel=1" title="Comparison of mutation-level regression coefficients for growth rate among 50 most prevalent lineages using a standard multinomial logistic regression model with estimates of lineage growth rates from PyR0. Pearsonâs R = 0.95." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Comparison of mutation-level regression coefficients for growth rate among 50 most prevalent lineages using a standard multinomial logistic regression model with estimates of lineage growth rates from PyR<sub>0</sub>. Pearsonâs R = 0.95.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S24." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F27.medium.gif" width="440" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S24." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F27.medium.gif" width="440" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F27.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S24." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F27.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S24.
                               </span>
                               <p id="p-100" class="first-child">
                                Comparison of mutation-level regression coefficients for growth rate among 50 most prevalent lineages using a standard multinomial logistic regression model with estimates of lineage growth rates from PyR
                                <sub>
                                 0
                                </sub>
                                . Pearsonâs R = 0.95.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F28" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F28.large.jpg?width=800&height=600&carousel=1" title="A. Estimated growth rate per viral generation (5.5 days) in each state using binomial logistic regressions for the emergence of BA.1 against a background consisting of Delta (B.1.617.2 and sublineages) viruses in all 50 US states between November 1 2021 and January 7 2022. Fold increase in relative fitness is expressed as exp(Î²1), where time is measured in viral generations. Error bars show exp(Î²1 +/- SE(Î²1)).For all 50 US states, the median growth rate per viral generation of Omicron over Delta was 3.9. For all states, the confidence interval for the binomial logistic regression coefficient contained the estimate for the ratio of Omicron to Delta from the PyR0 model, which was 3.1 for BA.1.1 / B.1.617.2 and 2.8 for BA.1 / B.1.617.2. B. Estimated probability of BA.1 by state from the binomial logistic regression." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1490906533" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A.</strong> Estimated growth rate per viral generation (5.5 days) in each state using binomial logistic regressions for the emergence of BA.1 against a background consisting of Delta (B.1.617.2 and sublineages) viruses in all 50 US states between November 1 2021 and January 7 2022. Fold increase in relative fitness is expressed as exp(<em>Î²</em><sub>1</sub>), where time is measured in viral generations. Error bars show exp(<em>Î²</em><sub>1</sub> +/- SE(<em>Î²</em><sub>1</sub>)).For all 50 US states, the median growth rate per viral generation of Omicron over Delta was 3.9. For all states, the confidence interval for the binomial logistic regression coefficient contained the estimate for the ratio of Omicron to Delta from the PyR<sub>0</sub> model, which was 3.1 for BA.1.1 / B.1.617.2 and 2.8 for BA.1 / B.1.617.2. <strong>B.</strong> Estimated probability of BA.1 by state from the binomial logistic regression.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S25." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F28.medium.gif" width="440" height="423"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S25." src="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F28.medium.gif" width="440" height="423"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F28.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S25." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/16/2021.09.07.21263228/F28.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S25.
                               </span>
                               <p id="p-101" class="first-child">
                                <strong>
                                 A.
                                </strong>
                                Estimated growth rate per viral generation (5.5 days) in each state using binomial logistic regressions for the emergence of BA.1 against a background consisting of Delta (B.1.617.2 and sublineages) viruses in all 50 US states between November 1 2021 and January 7 2022. Fold increase in relative fitness is expressed as exp(Î²
                                <sub>
                                 1
                                </sub>
                                ), where time is measured in viral generations. Error bars show exp(Î²
                                <sub>
                                 1
                                </sub>
                                +/- SE(Î²
                                <sub>
                                 1
                                </sub>
                                )).For all 50 US states, the median growth rate per viral generation of Omicron over Delta was 3.9. For all states, the confidence interval for the binomial logistic regression coefficient contained the estimate for the ratio of Omicron to Delta from the PyR
                                <sub>
                                 0
                                </sub>
                                model, which was 3.1 for BA.1.1 / B.1.617.2 and 2.8 for BA.1 / B.1.617.2.
                                <strong>
                                 B.
                                </strong>
                                Estimated probability of BA.1 by state from the binomial logistic regression.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/478249/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/478249/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/478249" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S1.
                               </span>
                               <p id="p-102" class="first-child">
                                Regional evaluation of forecasts. We evaluate the ability of PyR
                                <sub>
                                 0
                                </sub>
                                to accurately forecast the dominant lineage 4- and 8-weeks into the future in six selected regions with a relatively large number of GISAID samples. Percentage accuracies are obtained by averaging over 45 training windows.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T3" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/478254/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/478254/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/478254" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S2.
                               </span>
                               <p id="p-103" class="first-child">
                                Spatial structure of the inferred amino acid coefficients Î²
                                <sub>
                                 f
                                </sub>
                                . We report one-sided p-values for the Moran I spatial autocorrelation statistic computed using a permutation test. We use a gaussian weighting function of the form exp(-distance
                                <sup>
                                 2
                                </sup>
                                /lengthscale
                                <sup>
                                 2
                                </sup>
                                ), where distance is measured in units of nucleotides. We find that there is significant evidence for spatial structure in S, N, ORF7a, ORF3a, and ORF1a as well as across the SARS-CoV-2 genome as a whole.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T4" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/478230/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/478230/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/478230" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S3.
                               </span>
                               <p id="p-104" class="first-child">
                                Table of the most statistically significant mutations in spike, ORF1b, and nucleocapsid. The top 9 mutations for each of the listed ORFs is shown. Mutations such as S:H655, S:T95I, and N:R203M have emerged independently in VoC lineages.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-105">
                              <strong>
                               Data S1.
                              </strong>
                              (separate file strains.tsv)
                             </p>
                             <p id="p-106">
                              Complete list of PANGO lineages with inferred relative fitness, ranked by fitness. Mirrored at
                              <a href="https://github.com/broadinstitute/pyro-cov/blob/v0.2/paper/strains.tsv">
                               https://github.com/broadinstitute/pyro-cov/blob/v0.2/paper/strains.tsv
                              </a>
                             </p>
                             <p id="p-107">
                              <strong>
                               Data S2.
                              </strong>
                              (separate file mutations.tsv)
                             </p>
                             <p id="p-108">
                              Complete list of amino acid changes with inferred effect size, ranked by z-score. Mirrored at
                              <a href="https://github.com/broadinstitute/pyro-cov/blob/v0.2/paper/mutations.tsv">
                               https://github.com/broadinstitute/pyro-cov/blob/v0.2/paper/mutations.tsv
                              </a>
                             </p>
                             <p id="p-109">
                              <strong>
                               Data S3.
                              </strong>
                              (separate file accession_ids.txt.xz)
                             </p>
                             <p id="p-110">
                              Complete list of GISAID accession numbers of viral genomes used in this study.
                             </p>
                             <p id="p-111">
                              Mirrored at
                              <a href="https://github.com/broadinstitute/pyro-cov/blob/v0.2/paper/accession_ids.txt.xz">
                               https://github.com/broadinstitute/pyro-cov/blob/v0.2/paper/accession_ids.txt.xz
                              </a>
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-30">
                            <h2 class="">
                             Funding
                            </h2>
                            <p id="p-113">
                             This work was sponsored by the U.S. Centers for Disease Control and Prevention (BAA), as well as support from the Doris Duke Charitable Foundation (J.E.L.), the Howard Hughes Medical Institute (P.C.S.), the National Institute of Allergy and Infectious Diseases R37AI147868 (J.L.), and the Evergrande COVID-19 Response Fund Award from the Massachusetts Consortium on Pathogen Readiness (J.L.).
                            </p>
                           </div>
                           <div class="section" id="sec-31">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-114">
                             Conceptualization: F.O., S.F.S., J.E.L., M.J.
                            </p>
                            <p id="p-115">
                             Data curation: F.O., N.B.
                            </p>
                            <p id="p-116">
                             Formal Analysis: F.O., S.F.S, M.J., N.B., J.E.L.
                            </p>
                            <p id="p-117">
                             Funding acquisition: B.M., P.C.S, J.L., J.E.L.
                            </p>
                            <p id="p-118">
                             Investigation: all authors
                            </p>
                            <p id="p-119">
                             Methodology: F.O., S.F.S, M.J., J.E.L., L.Y., M.B.
                            </p>
                            <p id="p-120">
                             Project administration: all authors Software: F.O., N.B., M.J.
                            </p>
                            <p id="p-121">
                             Supervision: D.J.P., B.M., J.L., P.C.S., J.E.L.
                            </p>
                            <p id="p-122">
                             Validation: F.O., N.B., M.J., S.F.S.
                            </p>
                            <p id="p-123">
                             Visualization: F.O., J.E.L., N.B., J.P., S.F.S.
                            </p>
                            <p id="p-124">
                             Writing â original draft: F.O., S.F.S., B.M., P.C.S, J.E.L.
                            </p>
                            <p id="p-125">
                             Writing â review & editing: all authors
                            </p>
                            <p id="p-126">
                             Authors have no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-32">
                            <h2 class="">
                             Data and materials availability
                            </h2>
                            <p id="p-127">
                             We gratefully acknowledge all data contributors, i.e. the Authors and their Originating laboratories responsible for obtaining the specimens, and their Submitting laboratories for generating the genetic sequence and metadata and sharing via the GISAID initiative (
                             <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ) on which this research is based. A total of 6,466,300 submissions are included in this study. A complete list of 6.4million accession numbers is included as Data (S3).
                            </p>
                           </div>
                           <div class="section" id="sec-33">
                            <h2 class="">
                             List of Supplementary materials
                            </h2>
                            <p id="p-128">
                             Materials and Methods
                            </p>
                            <p id="p-129">
                             Supplemental Note 1: Detailed description of PyR0 model
                            </p>
                            <p id="p-130">
                             Supplemental Note 2
                            </p>
                            <p id="p-131">
                             Supplemental Note 3
                            </p>
                            <p id="p-132">
                             Fig S1 â S25
                            </p>
                            <p id="p-133">
                             Table S1 â S3
                            </p>
                            <p id="p-134">
                             References (20â47)
                            </p>
                            <p id="p-135">
                             Data S1 (strains.tsv), S2 (mutations.tsv), S3 (accession_ids.txt.xz)
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-112">
                             We acknowledge crucial assistance in data preprocessing from Angie Hinrichs. We thank Trevor Bedford and Cornelius Roemer for visualizing the outputs of our model on nextstrain.org. We acknowledge helpful discussions and feedback from Du Phan, William Hanage, Christopher Tomkins-Tinch, Shira Weingarten-Gabbay, Katie Siddle, Sagar Gosai, Steven Reilly, Eli Bingham, Mehrtash Babadi, Holly Soutter, Debora Marks, Noor Youssef, Sarah Gurev, and Nicole Thadani. We gratefully acknowledge the authors from the originating laboratories and the submitting laboratories, who generated and shared via GISAID genetic sequence data on which this research is based.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-19">
                               1. We used our model to conduct a global analysis of variants and case counts during the entire first two years of the SARS-CoV-2 pandemic, extending the analysis to include data from the beginning of the outbreak through to January 2022 (rather than through July 2021).
2. Our data now includes 6.4 million genomes, over 3x times the size of our original study, and by far the largest genomic analysis for SARS-CoV-2 or any virus to date.
3. The updated model closely covers the emergence of Omicron and includes a detailed analysis of relative fitness of the Omicron sublineages, BA.1, BA.2, and BA.3, predicting BA.2 as the fittest lineage, a prediction that seems in line with current events.
3. We have completely rewritten the initial clustering method for the model to use a fine-grained, phylogenetic approach, improving on the previous approach that was based on PANGO lineages. The new clustering method recapitulates our earlier, PANGO-based findings, reveals heterogeneity within lineages, and captures these effects in the model to improve the inference.
4. We have added experimental data to probe and validate our findings, including assessment of several high-scoring mutations in cellular infectivity assays. We found that high-scoring spike RBD mutations do not consistently enhance infectivity; rather, they appear to confer immune escape.
5. We have further expanded our analysis of potential mechanisms driving fitness, by correlating immune escape predictions with our mutational fitness predictions.
6. On the basis of these new data and analyses, we have broadened the scope of our discussion to highlight the major forces dominating the pandemic during its first two years, and in particular the recent transition to an immune escape phase.
7. We have added new structural analyses and figures, showing that mutational fitness changes are highly concentrated in specific structural regions of proteins, especially spike, nucleocapsid, and ORF1a.
8. We have rewritten key parts of the manuscript to speak to the broader implications of our work for basic science and public health.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References and Notes
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.1" data-doi="10.1126/science.abg3055">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Davies
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Abbott
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Barnard
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jarvis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kucharski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Munday
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. A. B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pearson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Russell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tully
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Washburne
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wenseleers
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gimma
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Waites
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. L. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wong
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   van Zandvoort
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Silverman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth cit-collab">
                                  CMMID COVID-19 Working Group, COVID-19 Genomics UK (COG-UK) Consortium
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Diaz-Ordaz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Keogh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Eggo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Funk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jit
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Atkins
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Edmunds
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Estimated transmissibility and impact of SARS-CoV-2 lineage B.1.1.7 in England
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  372
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.abg3055
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abg3055%26rft_id%253Dinfo%253Apmid%252F33658326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNzIvNjUzOC9lYWJnMzA1NSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjEuMDkuMDcuMjEyNjMyMjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Volz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mishra
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chand
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Barrett
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Geidelberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinsley
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Laydon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dabrera
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ã.
                                  </span>
                                  <span class="cit-name-surname">
                                   OâToole
                                  </span>
                                 </span>
                                 , Others,
                                 <span class="cit-article-title">
                                  Assessing transmissibility of SARS-CoV-2 lineage B. 1.1. 7 in England
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  17
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.09.07.21263228v2.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Stefanelli
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Trentini
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Guzzetta
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Marziano
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mammone
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Poletti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   GranÃ©
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Manica
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   del Manso
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Andrianou
                                  </span>
                                 </span>
                                 , Others,
                                 <span class="cit-article-title">
                                  Co-circulation of SARS-CoV-2 variants B. 1.1. 7 and P. 1
                                 </span>
                                 .
                                 <span class="cit-source">
                                  medRxiv
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ) (available at
                                 <a href="https://www.medrxiv.org/content/10.1101/2021.04.06.21254923v1.abstract">
                                  https://www.medrxiv.org/content/10.1101/2021.04.06.21254923v1.abstract
                                 </a>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.4" data-doi="10.1101/2021.05.22.21257633">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   VÃ¶hringer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sanderson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sinnott
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nguyen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Goater
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schwach
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Harrison
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hellewell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ariani
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   GonÃ§alves
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jackson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Johnston
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jung
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Saint
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sillitoe
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Suciu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Goldman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Birney
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Funk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Volz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kwiatkowski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chand
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Martincorena
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Barrett
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gerstung
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth cit-collab">
                                  The Wellcome Sanger Institute Covid-19 Surveillance Team, The COVID-19 Genomics UK (COG-UK) Consortium
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Genomic reconstruction of the SARS-CoV-2 epidemic across England from September 2020 to May 2021
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.05.22.21257633
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.05.22.21257633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1101/2021.05.22.21257633&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.5" data-doi="10.1016/j.cell.2020.06.043">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Korber
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fischer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gnanakaran
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yoon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Theiler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Abfalterer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hengartner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Giorgi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bhattacharya
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Foley
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. G.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Partridge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. M.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Evans
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. M.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Freeman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. I.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Silva
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McDanal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. G.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Perez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moon-Walker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. P.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whelan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   LaBranche
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. O.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saphire
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. C.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Montefiori
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Angyal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. L.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carrilero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. R.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Green
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. C.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Groves
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. J.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keeley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. B.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lindsey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. J.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parsons
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raza
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rowland-Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tucker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                 </span>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wyles
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Tracking changes in SARS-CoV-2 Spike: evidence that D614G increases infectivity of the COVID-19 virus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2020.06.043
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.06.043%26rft_id%253Dinfo%253Apmid%252F32697968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.06.043&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32697968&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F16%2F2021.09.07.21263228.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.6" data-doi="10.1016/J.CELL.2020.09.032">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yurkovetskiy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pascal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tomkins-Tinch
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nyalile
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Baum
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Diehl
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dauphin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Carbone
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Veinotte
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Egri
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schaffner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lemieux
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Munro
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rafique
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Barve
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sabeti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kyratsous
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dudkina
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Luban
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  183
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  739
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  751
                                 </span>
                                 .e8 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D183%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.CELL.2020.09.032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/J.CELL.2020.09.032&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Deng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Garcia-Knight
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Khalid
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Servellita
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Morris
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sotomayor-GonzÃ¡lez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Glasner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Reyes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gliwa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sanchez San Martin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Federman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Balcerek
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Taylor
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Streithorst
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sreekumar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.-Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schulze-Gahmen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Taha
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hayashi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Simoneau
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kumar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   McMahon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lidsky
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xiao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hemarajata
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Green
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Espinosa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kath
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Haw
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hacker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hanson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wadford
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Anaya
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferguson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Frankino
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shivram
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lareau
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wyman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ott
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Andino
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chiu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Transmission, infectivity, and neutralization of a spike L452R SARS-CoV-2 variant
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  184
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3426
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  3437
                                 </span>
                                 .e8 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D184%26rft.spage%253D3426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Greaney
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hilton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ellis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. H. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Crawford
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dingens
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Navarro
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bowen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tortorici
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Walls
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   King
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Veesler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bloom
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  182
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1295
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  1310
                                 </span>
                                 .e20 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D182%26rft.spage%253D1295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.9" data-doi="10.1056/NEJMc2031364">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Choi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Choudhary
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Regan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sparks
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Padera
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Qiu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I. H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Solomon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.-H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kuo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Boucau
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bowman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   U. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Adhikari
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Winkler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mueller
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. Y.-T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hsu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Desjardins
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Baden
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Walker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lichterfeld
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brigl
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kwon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kanjilal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Richardson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jonsson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Alter
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Barczak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hanage
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gaiha
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Seaman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cernadas
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  N. Engl. J. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  383
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2291
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  2293
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft.volume%253D383%26rft.spage%253D2291%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMc2031364%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1056/NEJMc2031364&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F16%2F2021.09.07.21263228.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.10" data-doi="10.1016/J.CHOM.2020.11.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Greaney
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gilchuk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zost
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Binshtein
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Loes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hilton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Huddleston
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Eguia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. H. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Crawford
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dingens
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nargi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sutton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Suryadevara
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rothlauf
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. P. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Whelan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Carnahan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Crowe Jr
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bloom
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Host Microbe
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  44
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  57
                                 </span>
                                 .e9 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D29%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.CHOM.2020.11.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/J.CHOM.2020.11.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rambaut
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ã.
                                  </span>
                                  <span class="cit-name-surname">
                                   OâToole
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hill
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   McCrone
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ruis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   du Plessis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pybus
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Microbiol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1403
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  1407
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D5%26rft.spage%253D1403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.12" data-doi="10.2807/1560-7917.ES.2021.26.24.2100509">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Campbell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Archer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Laurenson-Schafer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jinnai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Konings
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Batra
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pavlin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vandemaele
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Kerkhove
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jombart
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                  <span class="cit-name-surname">
                                   Morgan
                                  </span>
                                 </span>
                                 , O. le Polain de Waroux,
                                 <span class="cit-article-title">
                                  Increased transmissibility and global spread of SARS-CoV-2 variants of concern as at June 2021
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Euro Surveill
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.2807/1560-7917.ES.2021.26.24.2100509
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill%26rft.volume%253D26%26rft.spage%253D2100509%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2021.26.24.2100509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2021.26.24.2100509&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  GISAID Initiative and global contributors
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  EpiCoV(TM) human coronavirus 2019 database
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  GISAID
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), (available at
                                 <a href="https://gisaid.org">
                                  https://gisaid.org
                                 </a>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.14" data-doi="10.1002/gch2.1018">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Elbe
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Buckland-Merrett
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Data, disease and diplomacy: GISAIDâs innovative contribution to global health
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Glob Chall
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  33
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  46
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGlob%2BChall%26rft.volume%253D1%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgch2.1018%26rft_id%253Dinfo%253Apmid%252F31565258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/gch2.1018&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31565258&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F16%2F2021.09.07.21263228.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bingham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jankowiak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Obermeyer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pradhan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Karaletsos
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Szerlip
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Horsfall
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Goodman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Pyro: Deep universal probabilistic programming
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Mach. Learn. Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  973
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  978
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes%26rft.volume%253D20%26rft.spage%253D973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.09.07.21263228v2.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-article-title">
                                  Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Virological
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), (available at
                                 <a href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">
                                  https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563
                                 </a>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.17" data-doi="10.1101/2021.12.04.471236">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Greaney
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Starr
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bloom
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  An antibody-escape calculator for mutations to the SARS-CoV-2 receptor-binding domain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.12.04.471236
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.12.04.471236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4xMi4wNC40NzEyMzZ2MSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjEuMDkuMDcuMjEyNjMyMjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Syed
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Taha
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tabata
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ciling
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Khalid
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sreekumar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.-Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hayashi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Soczek
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ott
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Doudna
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Rapid assessment of SARS-CoV-2âevolved variants using virus-like particles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  374
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1626
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  1632
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D374%26rft.spage%253D1626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.07.21263228v2.19" data-doi="10.1101/2020.09.04.20188516">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferretti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ledda
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wymant
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ledda
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Abeler-DÃ¶rner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kendall
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nurtay
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.-Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.-C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.-H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinch
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Masel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kilpatrick
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fraser
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  The timing of COVID-19 transmission
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2020.09.04.20188516
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.09.04.20188516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1101/2020.09.04.20188516&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.20" data-doi="10.1093/molbev/msab264">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   McBroome
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Thornlow
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinrichs
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kramer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Goldman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Haussler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Corbett-Detig
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Turakhia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  A daily-updated database and tools for comprehensive SARS-CoV-2 mutation-annotated trees
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Biol. Evol
                                 </abbr>
                                 . (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/molbev/msab264
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsab264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msab264&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.21" data-doi="10.1038/s41588-021-00862-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Turakhia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Thornlow
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinrichs
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gozashti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lanfear
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Haussler
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Corbett-Detig
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Ultrafast Sample placement on Existing tRees (UShER) enables real-time phylogenetics for the SARS-CoV-2 pandemic
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  53
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  809
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  816
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D53%26rft.spage%253D809%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-021-00862-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41588-021-00862-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nersisyan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhiyanov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shkurnikov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tonevitsky
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  T-CoV: a comprehensive portal of HLA-peptide interactions affected by SARS-CoV-2 mutations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  2021.07.06.451227
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.23" data-doi="10.7554/eLife.03430">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hopf
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. P. I.
                                  </span>
                                  <span class="cit-name-surname">
                                   SchÃ¤rfe
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. P. G. L. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rodrigues
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Green
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kohlbacher
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sander
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. M. J. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bonvin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Marks
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Sequence co-evolution gives 3D contacts and structures of protein complexes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.7554/eLife.03430
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.03430%26rft_id%253Dinfo%253Apmid%252F25255213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.03430&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25255213&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F16%2F2021.09.07.21263228.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Frazer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Notin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dias
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gomez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Min
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brock
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Marks
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Disease variant prediction with deep generative models of evolutionary data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  599
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  91
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  95
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D599%26rft.spage%253D91%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2021.09.07.21263228v2.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Paszke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gross
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chintala
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chanan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   DeVito
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Desmaison
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Antiga
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lerer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Automatic differentiation in PyTorch
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), (available at
                                 <a href="https://openreview.net/pdf?id=BJJsrmfCZ">
                                  https://openreview.net/pdf?id=BJJsrmfCZ
                                 </a>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2021.09.07.21263228v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gorinova
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hoffman
                                  </span>
                                 </span>
                                 , in
                                 <span class="cit-source">
                                  Proceedings of the 37th International Conference on Machine Learning
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Iii
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                 </span>
                                 , Eds. (
                                 <span class="cit-publ-name">
                                  PMLR
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), vol. 119 of Proceedings of Machine Learning Research, pp.
                                 <span class="cit-fpage">
                                  3648
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  3657
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.27" data-doi="10.1214/aos/1056562461">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Neal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Slice sampling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Annals of Statistics
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1214/aos/1056562461
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BStatistics%26rft.volume%253D31%26rft.spage%253D705%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1056562461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1214/aos/1056562461&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kingma
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ba
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Adam: A Method for Stochastic Optimization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv [cs.LG]
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ), (available at
                                 <a href="http://arxiv.org/abs/1412.6980">
                                  http://arxiv.org/abs/1412.6980
                                 </a>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cappello
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Palacios
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Statistical Challenges in Tracking the Evolution of SARS-CoV-2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv [stat.AP
                                 </abbr>
                                 ] (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), (available at
                                 <a href="http://arxiv.org/abs/2108.13362">
                                  http://arxiv.org/abs/2108.13362
                                 </a>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.30" data-doi="10.3390/v12010105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Diehl
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Finch
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Akusobi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kirchdoerfer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bollinger
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schaffner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. O.
                                  </span>
                                  <span class="cit-name-surname">
                                   Saphire
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Andersen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kuhn
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Luban
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sabeti
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Reporter Assays for Ebola Virus Nucleoprotein Oligomerization, Virion-Like Particle Budding, and Minigenome Activity Reveal the Importance of Nucleoprotein Amino Acid Position 111
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Viruses
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.3390/v12010105
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DViruses%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv12010105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.3390/v12010105&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Syed
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Taha
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Khalid
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tabata
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sreekumar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.-Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hayashi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Soczek
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ott
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Doudna
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Rapid assessment of SARS-CoV-2 evolved variants using virus-like particles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), p.
                                 <span class="cit-vol">
                                  2021
                                 </span>
                                 <span class="cit-fpage">
                                  .08.05.455082
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft.volume%253D2021%26rft.spage%253D0805455082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.32" data-doi="10.1128/mbio.00524-13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Angelini
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Akhlaghpour
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Neuman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Buchmeier
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Severe Acute Respiratory Syndrome Coronavirus Nonstructural Proteins 3, 4, and 6 Induce Double-Membrane Vesicles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  mBio
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1128/mbio.00524-13
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DmBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fmbio.00524-13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1128/mbio.00524-13&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.33" data-doi="10.1128/JVI.79.21.13399-13411.2005">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Graham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sims
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brockway
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Baric
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Denison
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  The nsp2 replicase proteins of murine hepatitis virus and severe acute respiratory syndrome coronavirus are dispensable for viral replication
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Virol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  79
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  13399
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  13411
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGraham%26rft.auinit1%253DR.%2BL.%26rft.volume%253D79%26rft.issue%253D21%26rft.spage%253D13399%26rft.epage%253D13411%26rft.atitle%253DThe%2Bnsp2%2BReplicase%2BProteins%2Bof%2BMurine%2BHepatitis%2BVirus%2Band%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BAre%2BDispensable%2Bfor%2BViral%2BReplication%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.21.13399-13411.2005%26rft_id%253Dinfo%253Apmid%252F16227261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OS8yMS8xMzM5OSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjEuMDkuMDcuMjEyNjMyMjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jungreis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sealfon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kellis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 Sarbecovirus genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2642
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D12%26rft.spage%253D2642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.35" data-doi="10.1038/s41598-020-70812-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Islam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hoque
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rahman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. S. M. R. U.
                                  </span>
                                  <span class="cit-name-surname">
                                   Alam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Akther
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Puspo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Akter
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sultana
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Crandall
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hossain
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Genome-wide analysis of SARS-CoV-2 virus strains circulating worldwide implicates heterogeneity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  14004
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D10%26rft.spage%253D14004%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-020-70812-6%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-020-70812-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F16%2F2021.09.07.21263228.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.36" data-doi="10.1128/JVI.00842-09">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cornillez-Ty
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yates
                                  </span>
                                  <span class="cit-name-suffix">
                                   3rd
                                  </span>
                                  .
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kuhn
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Buchmeier
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Severe acute respiratory syndrome coronavirus nonstructural protein 2 interacts with a host protein complex involved in mitochondrial biogenesis and intracellular signaling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Virol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  83
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  10314
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  10318
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00842-09%26rft_id%253Dinfo%253Apmid%252F19640993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8xOS8xMDMxNCI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjEuMDkuMDcuMjEyNjMyMjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.37" data-doi="10.1101/2021.05.10.443524">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gupta
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Azumaya
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Moritz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pourmal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Diallo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Merz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bouhaddou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fossati
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brilot
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Diwanji
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hernandez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Herrera
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kratochvil
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nguyen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nowotny
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Owens
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Peters
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rizo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schulze-Gahmen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Young
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Asarnow
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   BillesbÃ¸lle
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Campbell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.-H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   U. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dickinson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Doan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.-L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Linossi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lopez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lopez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mancino
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Moss
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Paul
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pawar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pelin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pospiech
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Puchades
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Remesh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Safari
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schaefer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tabios
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Thwin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Titus
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Trenker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tse
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. K. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tsui
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zuliani-Alvarez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth cit-collab">
                                  QCRG Structural Biology Consortium
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Agard
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fraser
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jura
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kortemme
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Manglik
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Southworth
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Stroud
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Swaney
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Krogan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Frost
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rosenberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Verba
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  CryoEM and AI reveal a structure of SARS-CoV-2 Nsp2, a multifunctional protein involved in key host processes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.05.10.443524
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.05.10.443524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMS4wNS4xMC40NDM1MjR2MSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjEuMDkuMDcuMjEyNjMyMjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Du
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Deng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Peng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Duan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   You
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Guddat
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xiao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Qin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  582
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  289
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  293
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D582%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Osipiuk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.-A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Azizi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dvorkin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Endres
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jedrzejczak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kathayat
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lisnyak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Maki
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nicolaescu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Taylor
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tesar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.-A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Randall
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Michalska
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Snyder
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dickinson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Joachimiak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structure of papain-like protease from SARS-CoV-2 and its complexes with non-covalent inhibitors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  743
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D12%26rft.spage%253D743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hillen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kokic
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Farnung
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dienemann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tegunov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cramer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structure of replicating SARS-CoV-2 polymerase
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  584
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  154
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  156
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D584%26rft.spage%253D154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ge
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zheng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Guddat
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Cryo-EM Structure of an Extended SARS-CoV-2 Replication and Transcription Complex Reveals an Intermediate State in Cap Synthesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  184
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  184
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  193
                                 </span>
                                 .e10 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D184%26rft.spage%253D184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Malone
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Llewellyn
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Grasso
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. M. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shelton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. D. B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Olinares
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Maruthi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Eng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vatandaslar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chait
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kapoor
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Darst
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Campbell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structural Basis for Helicase-Polymerase Coupling in the SARS-CoV-2 Replication-Transcription Complex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  182
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1560
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  1573
                                 </span>
                                 .e13 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D182%26rft.spage%253D1560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.43" data-doi="10.1073/pnas.0808790106">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   âan Pan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xiang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tien
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ahola
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Guo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Functional screen reveals SARS coronavirus nonstructural protein nsp14 as a novel cap N7 methyltransferase
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U. S. A
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  106
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3484
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  3489
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808790106%26rft_id%253Dinfo%253Apmid%252F19208801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzkvMzQ4NCI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjEuMDkuMDcuMjEyNjMyMjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               âµ
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.07.21263228v2.44" data-doi="10.1126/science.abi9310">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Becker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schatz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structural basis of mismatch recognition by a SARS-CoV-2 proofreading enzyme
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.abi9310
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abi9310%26rft_id%253Dinfo%253Apmid%252F34315827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiJzY2llbmNlLmFiaTkzMTB2MSI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAyLzE2LzIwMjEuMDkuMDcuMjEyNjMyMjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.07.21263228v2.45" data-doi="10.1038/s41401-020-0485-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.-F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.-W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Structural and functional properties of SARS-CoV-2 spike protein: potential antivirus drug development for COVID-19
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Acta Pharmacol. Sin
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1141
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  1149
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DActa%2BPharmacol.%2BSin%26rft.volume%253D41%26rft.spage%253D1141%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41401-020-0485-4%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41401-020-0485-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F16%2F2021.09.07.21263228.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.07.21263228v2.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cubuk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Alston
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Incicco
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Stuchell-Brereton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ward
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zimmerman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vithani
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Griffith
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wagoner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bowman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Soranno
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Holehouse
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  The SARS-CoV-2 nucleocapsid protein is dynamic, disordered, and phase separates with RNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1936
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D12%26rft.spage%253D1936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.07.21263228v2.47" data-doi="10.1373/clinchem.2004.031096">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pei
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Song
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhai
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Du
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Qiu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                  <span class="cit-name-surname">
                                   Guo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Antigenicity analysis of different regions of the severe acute respiratory syndrome coronavirus nucleocapsid protein
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Clin. Chem
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  988
                                 </span>
                                 â
                                 <span class="cit-lpage">
                                  995
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DClin.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2004.031096%26rft_id%253Dinfo%253Apmid%252F15054081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6ODoiNTAvNi85ODgiO3M6NDoiYXRvbSI7czo1MDoiL21lZHJ4aXYvZWFybHkvMjAyMi8wMi8xNi8yMDIxLjA5LjA3LjIxMjYzMjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2021.09.07.21263228v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.02.13.22270420v1" title="Intraocular water movement visualization using 1H-MRI with eye drops of O-17 labeled saline: a first-in-human study" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.11.18.21266536v2" title="Assessing the Use of Prescription Drugs in Obese Respondents in the National Health and Nutrition Examination Survey" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 PostedÂ February 16, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2021.09.07.21263228v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2021.09.07.21263228v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/02/16/2021.09.07.21263228.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2021.09.07.21263228v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about medRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2021.09.07.21263228v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from medRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the medRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/477231"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-JBoyrRG1fVfJcdz9bXvilJfl8vw_XW1Np6WeoQRRa5Q"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="277159504"/>
                                   <input type="hidden" name="captcha_token" value="a34de55923d9aa445dfefaaf7403251f"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node477231--3" data-node-nid="477231" id="node-477231--41500373670" data-pisa="medrxiv;2021.09.07.21263228v2" data-pisa-master="medrxiv;2021.09.07.21263228" data-seqnum="477231" data-apath="/medrxiv/early/2022/02/16/2021.09.07.21263228.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Fritz
                                  </span>
                                  <span class="nlm-surname">
                                   Obermeyer
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Martin
                                  </span>
                                  <span class="nlm-surname">
                                   Jankowiak
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Nikolaos
                                  </span>
                                  <span class="nlm-surname">
                                   Barkas
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Stephen F.
                                  </span>
                                  <span class="nlm-surname">
                                   Schaffner
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Jesse D.
                                  </span>
                                  <span class="nlm-surname">
                                   Pyle
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Lonya
                                  </span>
                                  <span class="nlm-surname">
                                   Yurkovetskiy
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Matteo
                                  </span>
                                  <span class="nlm-surname">
                                   Bosso
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Daniel J.
                                  </span>
                                  <span class="nlm-surname">
                                   Park
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Mehrtash
                                  </span>
                                  <span class="nlm-surname">
                                   Babadi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Bronwyn L.
                                  </span>
                                  <span class="nlm-surname">
                                   MacInnis
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">
                                   Jeremy
                                  </span>
                                  <span class="nlm-surname">
                                   Luban
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="11">
                                  <span class="nlm-given-names">
                                   Pardis C.
                                  </span>
                                  <span class="nlm-surname">
                                   Sabeti
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="12">
                                  <span class="nlm-given-names">
                                   Jacob E.
                                  </span>
                                  <span class="nlm-surname">
                                   Lemieux
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 medRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2021.09.07.21263228;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2021.09.07.21263228
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&title=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&title=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&text=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&t=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&title=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&title=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness&summary=&source=medRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&title=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node477231--5" data-node-nid="477231" id="citation-node-477231--61740713015" data-pisa="medrxiv;2021.09.07.21263228v2" data-pisa-master="medrxiv;2021.09.07.21263228" data-seqnum="477231" data-apath="/medrxiv/early/2022/02/16/2021.09.07.21263228.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Analysis of 6.4 million SARS-CoV-2 genomes identifies mutations associated with fitness
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Fritz
                                      </span>
                                      <span class="nlm-surname">
                                       Obermeyer
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Martin
                                      </span>
                                      <span class="nlm-surname">
                                       Jankowiak
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Nikolaos
                                      </span>
                                      <span class="nlm-surname">
                                       Barkas
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Stephen F.
                                      </span>
                                      <span class="nlm-surname">
                                       Schaffner
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Jesse D.
                                      </span>
                                      <span class="nlm-surname">
                                       Pyle
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Lonya
                                      </span>
                                      <span class="nlm-surname">
                                       Yurkovetskiy
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Matteo
                                      </span>
                                      <span class="nlm-surname">
                                       Bosso
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Daniel J.
                                      </span>
                                      <span class="nlm-surname">
                                       Park
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Mehrtash
                                      </span>
                                      <span class="nlm-surname">
                                       Babadi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Bronwyn L.
                                      </span>
                                      <span class="nlm-surname">
                                       MacInnis
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">
                                       Jeremy
                                      </span>
                                      <span class="nlm-surname">
                                       Luban
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="11">
                                      <span class="nlm-given-names">
                                       Pardis C.
                                      </span>
                                      <span class="nlm-surname">
                                       Sabeti
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="12">
                                      <span class="nlm-given-names">
                                       Jacob E.
                                      </span>
                                      <span class="nlm-surname">
                                       Lemieux
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     medRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2021.09.07.21263228;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2021.09.07.21263228
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=medRxiv&rft.au=Obermeyer%2C+Fritz&rft.au=Jankowiak%2C+Martin&rft.au=Barkas%2C+Nikolaos&rft.au=Schaffner%2C+Stephen+F.&rft.au=Pyle%2C+Jesse+D.&rft.au=Yurkovetskiy%2C+Lonya&rft.au=Bosso%2C+Matteo&rft.au=Park%2C+Daniel+J.&rft.au=Babadi%2C+Mehrtash&rft.au=MacInnis%2C+Bronwyn+L.&rft.au=Luban%2C+Jeremy&rft.au=Sabeti%2C+Pardis+C.&rft.au=Lemieux%2C+Jacob+E.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/477231/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/477231/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/477231/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/477231/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/477231/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/477231/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/477231/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/477231/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/477231/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/477231/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/477231/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/477231/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&count=horizontal&via=&text=Analysis%20of%206.4%20million%20SARS-CoV-2%20genomes%20identifies%20mutations%20associated%20with%20fitness&counturl=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2021.09.07.21263228v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/epidemiology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Epidemiology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Addiction Medicine
                          </a>
                          <span class="article-count">
                           (218)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Allergy and Immunology
                          </a>
                          <span class="article-count">
                           (496)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">
                           Anesthesia
                          </a>
                          <span class="article-count">
                           (106)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Cardiovascular Medicine
                          </a>
                          <span class="article-count">
                           (1121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Dentistry and Oral Medicine
                          </a>
                          <span class="article-count">
                           (198)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">
                           Dermatology
                          </a>
                          <span class="article-count">
                           (141)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Emergency Medicine
                          </a>
                          <span class="article-count">
                           (275)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">
                           Endocrinology (including Diabetes Mellitus and Metabolic Disease)
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (9829)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Forensic Medicine
                          </a>
                          <span class="article-count">
                           (5)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">
                           Gastroenterology
                          </a>
                          <span class="article-count">
                           (484)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Genetic and Genomic Medicine
                          </a>
                          <span class="article-count">
                           (2344)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Geriatric Medicine
                          </a>
                          <span class="article-count">
                           (225)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Economics
                          </a>
                          <span class="article-count">
                           (465)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Informatics
                          </a>
                          <span class="article-count">
                           (1574)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">
                           Health Policy
                          </a>
                          <span class="article-count">
                           (738)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">
                           Health Systems and Quality Improvement
                          </a>
                          <span class="article-count">
                           (609)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">
                           Hematology
                          </a>
                          <span class="article-count">
                           (239)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">
                           HIV/AIDS
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">
                           Infectious Diseases (except HIV/AIDS)
                          </a>
                          <span class="article-count">
                           (11686)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Intensive Care and Critical Care Medicine
                          </a>
                          <span class="article-count">
                           (617)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Education
                          </a>
                          <span class="article-count">
                           (242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Ethics
                          </a>
                          <span class="article-count">
                           (67)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">
                           Nephrology
                          </a>
                          <span class="article-count">
                           (259)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">
                           Neurology
                          </a>
                          <span class="article-count">
                           (2170)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">
                           Nursing
                          </a>
                          <span class="article-count">
                           (134)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">
                           Nutrition
                          </a>
                          <span class="article-count">
                           (340)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">
                           Obstetrics and Gynecology
                          </a>
                          <span class="article-count">
                           (427)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">
                           Occupational and Environmental Health
                          </a>
                          <span class="article-count">
                           (521)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">
                           Oncology
                          </a>
                          <span class="article-count">
                           (1188)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">
                           Ophthalmology
                          </a>
                          <span class="article-count">
                           (368)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">
                           Orthopedics
                          </a>
                          <span class="article-count">
                           (129)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">
                           Otolaryngology
                          </a>
                          <span class="article-count">
                           (221)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Pain Medicine
                          </a>
                          <span class="article-count">
                           (148)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Palliative Medicine
                          </a>
                          <span class="article-count">
                           (50)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (315)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">
                           Pediatrics
                          </a>
                          <span class="article-count">
                           (702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Therapeutics
                          </a>
                          <span class="article-count">
                           (304)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">
                           Primary Care Research
                          </a>
                          <span class="article-count">
                           (269)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">
                           Psychiatry and Clinical Psychology
                          </a>
                          <span class="article-count">
                           (2201)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">
                           Public and Global Health
                          </a>
                          <span class="article-count">
                           (4705)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">
                           Radiology and Imaging
                          </a>
                          <span class="article-count">
                           (786)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">
                           Rehabilitation Medicine and Physical Therapy
                          </a>
                          <span class="article-count">
                           (462)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Respiratory Medicine
                          </a>
                          <span class="article-count">
                           (626)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">
                           Rheumatology
                          </a>
                          <span class="article-count">
                           (276)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">
                           Sexual and Reproductive Health
                          </a>
                          <span class="article-count">
                           (227)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Sports Medicine
                          </a>
                          <span class="article-count">
                           (214)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">
                           Surgery
                          </a>
                          <span class="article-count">
                           (254)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Toxicology
                          </a>
                          <span class="article-count">
                           (43)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">
                           Transplantation
                          </a>
                          <span class="article-count">
                           (121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">
                           Urology
                          </a>
                          <span class="article-count">
                           (94)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
