<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2021.10.15.21265069v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2021.10.15.21265069v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2021.10.15.21265069v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/medrxiv/early/2022/02/15/2021.10.15.21265069.atom"/>
  <meta name="HW.pisa" content="medrxiv;2021.10.15.21265069v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="SVRare: discovering disease-causing structural variants in the 100K Genomes Project"/>
  <meta name="DC.Identifier" content="10.1101/2021.10.15.21265069"/>
  <meta name="DC.Date" content="2022-02-15"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Discovery of disease-causing structural variants (dcSV) from whole genome sequencing data is difficult due to high number of false positives and a lack of efficient way to estimate allele frequency. Here we introduce SVRare, an application that aggregates structural variants (SV) called by other tools, and efficiently annotates rare SVs to aid dcSVs discovery.

Applied in the Genomics England (GEL) research environment to data from the 100K Genomes Project, SVRare aggregated 554,060,126 SVs called by Manta and Canvas in all the 71,408 participants in the rare-disease arm. From a pilot study of 4313 families, SVRare identified 36 novel protein-coding disrupting SVs on diagnostic grade genes that may explain proband’s phenotype. It is estimated that SVRare can increase SV-based diagnosis yield by at least 4-fold. We also performed a genome-wide association study, and uncovered clusters of dcSVs in genes with known pathogenicity, such as PKD1/2 - cystic kidney diseases and LDLR - familial hypercholesterolaemia.

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

Jing Yu's salary was supported by UKIRDC (RetinaUK)

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

United Kingdom National Research Ethics Service Research Ethics Committee for East of England - Cambridge South Research Ethics Committee gave ethical approval for this work.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data produced in the present study are available in the manuscript and online at <https://github.com/Oxford-Eye/SVRare>

<https://github.com/Oxford-Eye/SVRare>"/>
  <meta name="DC.Contributor" content="Jing Yu"/>
  <meta name="DC.Contributor" content="Anita Szabo"/>
  <meta name="DC.Contributor" content="Alistair T Pagnamenta"/>
  <meta name="DC.Contributor" content="Ahmed Shalaby"/>
  <meta name="DC.Contributor" content="Edoardo Giacopuzzi"/>
  <meta name="DC.Contributor" content="Jenny Taylor"/>
  <meta name="DC.Contributor" content="Deborah Shears"/>
  <meta name="DC.Contributor" content="Nikolas Pontikos"/>
  <meta name="DC.Contributor" content="Genevieve Wright"/>
  <meta name="DC.Contributor" content="Michel Michaelides"/>
  <meta name="DC.Contributor" content="Stephanie Halford"/>
  <meta name="DC.Contributor" content="Susan Downes"/>
  <meta name="DC.Contributor" content="Genomics England Research Consortium"/>
  <meta name="article:published_time" content="2022-02-15"/>
  <meta name="citation_title" content="SVRare: discovering disease-causing structural variants in the 100K Genomes Project"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Discovery of disease-causing structural variants (dcSV) from whole genome sequencing data is difficult due to high number of false positives and a lack of efficient way to estimate allele frequency. Here we introduce SVRare, an application that aggregates structural variants (SV) called by other tools, and efficiently annotates rare SVs to aid dcSVs discovery.</p><p>Applied in the Genomics England (GEL) research environment to data from the 100K Genomes Project, SVRare aggregated 554,060,126 SVs called by Manta and Canvas in all the 71,408 participants in the rare-disease arm. From a pilot study of 4313 families, SVRare identified 36 novel protein-coding disrupting SVs on diagnostic grade genes that may explain proband’s phenotype. It is estimated that SVRare can increase SV-based diagnosis yield by at least 4-fold. We also performed a genome-wide association study, and uncovered clusters of dcSVs in genes with known pathogenicity, such as <i>PKD1/2</i> - cystic kidney diseases and <i>LDLR</i> - familial hypercholesterolaemia.</p>"/>
  <meta name="citation_journal_title" content="medRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="medrxiv;2021.10.15.21265069v2"/>
  <meta name="citation_id" content="2021.10.15.21265069v2"/>
  <meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2021.10.15.21265069v2"/>
  <meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2021.10.15.21265069v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2021.10.15.21265069v2.full"/>
  <meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2021.10.15.21265069"/>
  <meta name="citation_num_pages" content="18"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_firstpage" content="2021.10.15.21265069"/>
  <meta name="citation_author" content="Jing Yu"/>
  <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_email" content="jing.yu@ndcn.ox.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8001-8494"/>
  <meta name="citation_author" content="Anita Szabo"/>
  <meta name="citation_author_institution" content="Institute of Ophthalmology, University College London"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8912-897X"/>
  <meta name="citation_author" content="Alistair T Pagnamenta"/>
  <meta name="citation_author_institution" content="Wellcome Centre for Human Genetics, Oxford University"/>
  <meta name="citation_author_institution" content="NIHR Biomedical Research Centre"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7334-0602"/>
  <meta name="citation_author" content="Ahmed Shalaby"/>
  <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Oxford Eye Hospital, John Radcliffe Hospital"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0647-5862"/>
  <meta name="citation_author" content="Edoardo Giacopuzzi"/>
  <meta name="citation_author_institution" content="Wellcome Centre for Human Genetics, Oxford University"/>
  <meta name="citation_author_institution" content="NIHR Biomedical Research Centre"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5692-366X"/>
  <meta name="citation_author" content="Jenny Taylor"/>
  <meta name="citation_author_institution" content="Wellcome Centre for Human Genetics, Oxford University"/>
  <meta name="citation_author_institution" content="NIHR Biomedical Research Centre"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3602-5704"/>
  <meta name="citation_author" content="Deborah Shears"/>
  <meta name="citation_author_institution" content="Oxford Centre for Genomic Medicine, Nuffield Orthopaedic Centre, Oxford University Hospitals NHS Foundation Trust"/>
  <meta name="citation_author" content="Nikolas Pontikos"/>
  <meta name="citation_author_institution" content="Institute of Ophthalmology, University College London"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1782-4711"/>
  <meta name="citation_author" content="Genevieve Wright"/>
  <meta name="citation_author_institution" content="Moorfields Eye Hospital"/>
  <meta name="citation_author" content="Michel Michaelides"/>
  <meta name="citation_author_institution" content="UCL Institute of Ophthalmology, University College London"/>
  <meta name="citation_author_institution" content="Moorfields Eye Hospital NHS Foundation Trust"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1552-7046"/>
  <meta name="citation_author" content="Stephanie Halford"/>
  <meta name="citation_author_institution" content="Nuffield Laboratory of Ophthalmology, Nuffield Department of Clinical Neuroscience, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2179-907X"/>
  <meta name="citation_author" content="Susan Downes"/>
  <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7373-2665"/>
  <meta name="citation_author" content="Genomics England Research Consortium"/>
  <meta name="citation_reference" content="A, S.-J. et al. Complex structural variants in Mendelian disorders: identification and breakpoint resolution using short- and long-read genome sequencing. Genome medicine 10, (2018)."/>
  <meta name="citation_reference" content="Roller, E., Ivakhno, S., Lee, S., Royce, T. & Tanner, S. Canvas: versatile and scalable detection of copy number variants. Bioinformatics 32, 2375–2377 (2016)."/>
  <meta name="citation_reference" content="Chen, X. et al. Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. Bioinformatics 32, 1220–1222 (2016)."/>
  <meta name="citation_reference" content="P, R., M, S., J, S. & M, K. CADD-Splice-improving genome-wide variant effect prediction using deep learning-derived splice scores. Genome medicine 13, (2021)."/>
  <meta name="citation_reference" content="G, Y. & CB, B. Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals. Journal of computational biology : a journal of computational molecular cell biology 11, 377–394 (2004)."/>
  <meta name="citation_reference" content="Jaganathan, K. et al. Predicting Splicing from Primary Sequence with Deep Learning. Cell 176, 535-548.e24 (2019)."/>
  <meta name="citation_reference" content="Grati, M. et al. MYO3A causes human dominant deafness and interacts with protocadherin 15-CD2 isoform. Human mutation 37, 481 (2016)."/>
  <meta name="citation_reference" content="A, M. et al. Robust relationship inference in genome-wide association studies. Bioinformatics (Oxford, England) 26, 2867–2873 (2010)."/>
  <meta name="citation_reference" content="Henderson, R., O”Kane, M., McGilligan, V. & Watterson, S. The genetics and screening of familial hypercholesterolaemia. Journal of Biomedical Science 23, (2016)."/>
  <meta name="citation_reference" content="Cameron, D. L., di Stefano, L. & Papenfuss, A. T. Comprehensive evaluation and characterisation of short read general-purpose structural variant calling software. Nature Communications 10, 1–11 (2019)."/>
  <meta name="citation_reference" content="Fs, van D. et al. Interaction between KDELR2 and HSP47 as a Key Determinant in Osteogenesis Imperfecta Caused by Bi-allelic Variants in KDELR2. American journal of human genetics 107, 989–999 (2020)."/>
  <meta name="citation_reference" content="Efthymiou, S. et al. Two novel bi-allelic KDELR2 missense variants cause osteogenesis imperfecta with neurodevelopmental features. American Journal of Medical Genetics Part A 185, 2241–2249 (2021)."/>
  <meta name="twitter:title" content="SVRare: discovering disease-causing structural variants in the 100K Genomes Project"/>
  <meta name="twitter:site" content="@medrxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="twitter:description" content="Discovery of disease-causing structural variants (dcSV) from whole genome sequencing data is difficult due to high number of false positives and a lack of efficient way to estimate allele frequency. Here we introduce SVRare, an application that aggregates structural variants (SV) called by other tools, and efficiently annotates rare SVs to aid dcSVs discovery.

Applied in the Genomics England (GEL) research environment to data from the 100K Genomes Project, SVRare aggregated 554,060,126 SVs called by Manta and Canvas in all the 71,408 participants in the rare-disease arm. From a pilot study of 4313 families, SVRare identified 36 novel protein-coding disrupting SVs on diagnostic grade genes that may explain proband’s phenotype. It is estimated that SVRare can increase SV-based diagnosis yield by at least 4-fold. We also performed a genome-wide association study, and uncovered clusters of dcSVs in genes with known pathogenicity, such as PKD1/2 - cystic kidney diseases and LDLR - familial hypercholesterolaemia.

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

Jing Yu's salary was supported by UKIRDC (RetinaUK)

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

United Kingdom National Research Ethics Service Research Ethics Committee for East of England - Cambridge South Research Ethics Committee gave ethical approval for this work.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data produced in the present study are available in the manuscript and online at <https://github.com/Oxford-Eye/SVRare>

<https://github.com/Oxford-Eye/SVRare>"/>
  <meta name="og-title" property="og:title" content="SVRare: discovering disease-causing structural variants in the 100K Genomes Project"/>
  <meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2021.10.15.21265069v2"/>
  <meta name="og-site-name" property="og:site_name" content="medRxiv"/>
  <meta name="og-description" property="og:description" content="Discovery of disease-causing structural variants (dcSV) from whole genome sequencing data is difficult due to high number of false positives and a lack of efficient way to estimate allele frequency. Here we introduce SVRare, an application that aggregates structural variants (SV) called by other tools, and efficiently annotates rare SVs to aid dcSVs discovery.

Applied in the Genomics England (GEL) research environment to data from the 100K Genomes Project, SVRare aggregated 554,060,126 SVs called by Manta and Canvas in all the 71,408 participants in the rare-disease arm. From a pilot study of 4313 families, SVRare identified 36 novel protein-coding disrupting SVs on diagnostic grade genes that may explain proband’s phenotype. It is estimated that SVRare can increase SV-based diagnosis yield by at least 4-fold. We also performed a genome-wide association study, and uncovered clusters of dcSVs in genes with known pathogenicity, such as PKD1/2 - cystic kidney diseases and LDLR - familial hypercholesterolaemia.

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

Jing Yu's salary was supported by UKIRDC (RetinaUK)

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

United Kingdom National Research Ethics Service Research Ethics Committee for East of England - Cambridge South Research Ethics Committee gave ethical approval for this work.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

All data produced in the present study are available in the manuscript and online at <https://github.com/Oxford-Eye/SVRare>

<https://github.com/Oxford-Eye/SVRare>"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="citation_date" content="2022-02-15"/>
  <meta name="description" content="medRxiv - The Preprint Server for Health Sciences"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2021.10.15.21265069v2"/>
  <link rel="shortlink" href="https://www.medrxiv.org/node/476862"/>
  <title>
   SVRare: discovering disease-causing structural variants in the 100K Genomes Project | medRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__MF4P_i8lKWP7DQR7GmjDAVTRBKLnzxvs1Dvton-Awi0__yi5OflfckG_G8-Li4dDQQ3wO9cGIcdPx_YQ7ZZDuj5w__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__5mxrBK4DuyvC2MR5irx_TzVn9c8vr4vIjmBXjunHfxE__rtQnXv8GOc-AsynbhzSlQBD_2lVxAkB6XEFUCK72qXc__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all"/>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"t84X54F1zoCBaVYdR2uuWUqV9lyoO9_7OHonc9k9KyI"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"medrxiv;2021.10.15.21265069v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"medrxiv;2021.10.15.21265069v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"medrxiv;2021.10.15.21265069v2"}],"nid":"476862","apath":"\/medrxiv\/early\/2022\/02\/15\/2021.10.15.21265069.atom","pisa":"medrxiv;2021.10.15.21265069v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2021.10.15.21265069v2","title":"SVRare: discovering disease-causing structural variants in the 100K Genomes Project","identifier":"node\/476862"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022a8371v6NFFRR5GGVNRVsAoc89N4qXyhHZA5KxocQk21vhPEIWDkGu7KP7\\\\\\\/vnh50jgU7qclMnYcEEHPSQb10wgduPAfOe6K20oYaHGic\\\\\\\/d5FuTRrpZnOy9T7WctD\\\\\\\/eAxdQD8C2eDCdLegicqpLLUxeoR+1T3uQob7mSRXFtKdYEyGxrh9Yl3+k+ijzeXMQdThomRKjll1oASDR1zJZhXyBi+8FEpYvIGUdg666HAvTw2Q+d0CD4\\\\\\\/EFfkYAuhmdRjbAUcKCjNB52EprwkwAbAFlkL8q2NMSONxLalvm+CRIzrliALz1QSLp3NKa5+pFWrnjSFsbRixrsH+8+4KtdA16iB58hz5BodgnfIDuGgMjQhENk8aYCQwPxZMmeSpzuTKG\\\\\\\/x6FsdB7ZB03o2n0Z+s3MGWTqSGCYikz9rPB5mgvuVx4tXkzyYNw69LOJQZshKzmE8LEynnIK56To8y11A5A+INrJ\\\\\\\/nWKyOyoYxZrWgcS2dvGNLYShRZHYQAhVVj8672EWLZdqg9wsDy5y1nIqyRsHaLLLFhxF78\\\\\\\/F4NfzfG2KKCcvebXaOCJduE\\\\\\\/vlFxM5HSojE2DyeCiznGazxu\\\\\\\/f5Ua2vrVY5Xzk4OF4Tp3s4oiQIzBD\\\\\\\/DVXqqMTjOCuVuZu8lODRbs4LndlMa6us26klCfY6ZXZu3rf+Wqz7sMgdaZxUe421uFuwhhbYEcn7W5t2rfm5S8Ru8LJdZxZjqLJRSt5HJfXzpoYFGLhAudGDvnlQKbhwRmQX3O\\\\\\\/I4i45edd\\\\\\\/\\\\\\\/gT+LGvsk3eMzZnfWkZ4VNA391v+c1EVW1G4Mhoe0LUECH2PUx+QGq7qqleaaKbGlzJpjVhHiB0PTdROdQgh5SZJX+YFVXtZI5h5lHFOQSAzkM2EGTksT2y3o60YQKzxhhoRcGnR4ZPdcFDueE4Aw5aNWefrcgxeaoCKlw7ZSz6cOI4U7rChwSPpqf5Vdx8TPksMcuUGM7EGXNO7r00fAK5a7iPOQi8qktsmlJnkZDhH0CUNVhEysGYFCoNm2A4aB18Ec2hZRd5EmaOfplKKszD6U7c\\\\\\\/3DYT4JFxu5I0zwefvRgQfLuKcRc1bfRXanbh8op9YUgQa\\\\\\\/D92Xhefxb\\\\\\\/5XLdFXTfUsnNEpK23ZyzFvD5eWb9icWBoomNFfpUQOOGZGQ4aBVKDcIx4oUEB9xAdqahZm\\\\\\\/OuUWGBtLT7pYU17MbsOAHmQQuqWF6VLLI4PYNtUWhlQvUrvN6yaqaw20CwqSnro+XB\\\\\\\/TCXgD8lSwAULhgmKIR56fhKJ+pHEjaRwnvQPCjB1uKMlO0Fpy0PCaMM7e1VbXjTj2Km7WWNVlK3z4o9T+eedyUnCjvtJfNcqmnk141McpDSPBBuDroiQLG07NDRym95yh1HdUaGTmBys=\\\u0022,\\\u0022iv\\\u0022:\\\u0022aYMecNr44SZVzEMD1yY3pQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00225816af5d1bd6b9577d752a7b49d9af38a9a81192b2072e13f309a9be3381a065\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2021.10.15.21265069v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-476862 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2021.10.15.21265069v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_338175069">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_338175069" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_761450410" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-YveCfh0XN8Ki3EODOsqExR6KH78NKwpcx1d8JACmoak"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node476862" data-node-nid="476862" id="node-476862--21857554421" data-pisa="medrxiv;2021.10.15.21265069v2" data-pisa-master="medrxiv;2021.10.15.21265069" data-apath="/medrxiv/early/2022/02/15/2021.10.15.21265069.atom" data-hw-author-tooltip-instance="">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip">
                   <h1 class="highwire-cite-title" id="page-title">
                    SVRare: discovering disease-causing structural variants in the 100K Genomes Project
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-8001-8494" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jing
                      </span>
                      <span class="nlm-surname">
                       Yu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-8912-897X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Anita
                      </span>
                      <span class="nlm-surname">
                       Szabo
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0001-7334-0602" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Alistair T
                      </span>
                      <span class="nlm-surname">
                       Pagnamenta
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0002-0647-5862" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ahmed
                      </span>
                      <span class="nlm-surname">
                       Shalaby
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0002-5692-366X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Edoardo
                      </span>
                      <span class="nlm-surname">
                       Giacopuzzi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0003-3602-5704" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jenny
                      </span>
                      <span class="nlm-surname">
                       Taylor
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Deborah
                      </span>
                      <span class="nlm-surname">
                       Shears
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0003-1782-4711" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Nikolas
                      </span>
                      <span class="nlm-surname">
                       Pontikos
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="8">
                      <span class="nlm-given-names">
                       Genevieve
                      </span>
                      <span class="nlm-surname">
                       Wright
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9">
                      <a href="http://orcid.org/0000-0002-1552-7046" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Michel
                      </span>
                      <span class="nlm-surname">
                       Michaelides
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10">
                      <a href="http://orcid.org/0000-0003-2179-907X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Stephanie
                      </span>
                      <span class="nlm-surname">
                       Halford
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11">
                      <a href="http://orcid.org/0000-0001-7373-2665" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Susan
                      </span>
                      <span class="nlm-surname">
                       Downes
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="12">
                      <span class="nlm-collab" data-hwp-id="collab-1">
                       Genomics England Research Consortium
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2021.10.15.21265069
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-476862--21857554421" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Jing Yu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Clinical Neurosciences, University of Oxford, John Radcliffe Hospital
                       </span>
                       , Oxford OX3 9DU,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jing%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Yu%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJing%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-8001-8494" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jing Yu
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:jing.yu@ndcn.ox.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        jing.yu@ndcn.ox.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Anita Szabo
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute of Ophthalmology, University College London
                       </span>
                       ,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Anita%2BSzabo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Szabo%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAnita%2BSzabo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-8912-897X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Anita Szabo
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Alistair T Pagnamenta
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Wellcome Centre for Human Genetics, Oxford University
                       </span>
                       , Oxford, Oxfordshire,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        NIHR Biomedical Research Centre
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alistair%2BT%2BPagnamenta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Pagnamenta%20AT&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAlistair%2BT%2BPagnamenta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-7334-0602" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Alistair T Pagnamenta
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Ahmed Shalaby
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Clinical Neurosciences, University of Oxford, John Radcliffe Hospital
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Oxford Eye Hospital, John Radcliffe Hospital
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ahmed%2BShalaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Shalaby%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAhmed%2BShalaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-0647-5862" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ahmed Shalaby
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Edoardo Giacopuzzi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Wellcome Centre for Human Genetics, Oxford University
                       </span>
                       , Oxford, Oxfordshire,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        NIHR Biomedical Research Centre
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Edoardo%2BGiacopuzzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Giacopuzzi%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AEdoardo%2BGiacopuzzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-5692-366X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Edoardo Giacopuzzi
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Jenny Taylor
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Wellcome Centre for Human Genetics, Oxford University
                       </span>
                       , Oxford, Oxfordshire,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        NIHR Biomedical Research Centre
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jenny%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Taylor%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJenny%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3602-5704" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jenny Taylor
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Deborah Shears
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Oxford Centre for Genomic Medicine, Nuffield Orthopaedic Centre, Oxford University Hospitals NHS Foundation Trust
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Deborah%2BShears%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Shears%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADeborah%2BShears%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Nikolas Pontikos
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute of Ophthalmology, University College London
                       </span>
                       ,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nikolas%2BPontikos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Pontikos%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANikolas%2BPontikos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-1782-4711" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Nikolas Pontikos
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Genevieve Wright
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        Moorfields Eye Hospital
                       </span>
                       , 162 City Road, London EC1V 2PD,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Genevieve%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wright%20G&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AGenevieve%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Michel Michaelides
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        9
                       </span>
                       <span class="nlm-institution">
                        UCL Institute of Ophthalmology, University College London
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                       ;
                       <span class="nlm-institution">
                        Moorfields Eye Hospital NHS Foundation Trust
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Michel%2BMichaelides%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Michaelides%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMichel%2BMichaelides%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-1552-7046" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Michel Michaelides
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">
                     Stephanie Halford
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        10
                       </span>
                       <span class="nlm-institution">
                        Nuffield Laboratory of Ophthalmology, Nuffield Department of Clinical Neuroscience, University of Oxford, John Radcliffe Hospital
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Stephanie%2BHalford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Halford%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AStephanie%2BHalford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2179-907X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Stephanie Halford
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-11">
                    <div class="author-tooltip-name">
                     Susan Downes
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Nuffield Department of Clinical Neurosciences, University of Oxford, John Radcliffe Hospital
                       </span>
                       , Oxford,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Susan%2BDownes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Downes%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ASusan%2BDownes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-7373-2665" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Susan Downes
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2021.10.15.21265069v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:476862" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:476862/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.10.15.21265069v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:476862" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:476862/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.10.15.21265069v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:476862" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:476862/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.10.15.21265069v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:476862" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:476862/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.10.15.21265069v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:476862" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:476862/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2021.10.15.21265069v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:476862" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:476862/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Discovery of disease-causing structural variants (dcSV) from whole genome sequencing data is difficult due to high number of false positives and a lack of efficient way to estimate allele frequency. Here we introduce SVRare, an application that aggregates structural variants (SV) called by other tools, and efficiently annotates rare SVs to aid dcSVs discovery.
                            </p>
                            <p id="p-3">
                             Applied in the Genomics England (GEL) research environment to data from the 100K Genomes Project, SVRare aggregated 554,060,126 SVs called by Manta and Canvas in all the 71,408 participants in the rare-disease arm. From a pilot study of 4313 families, SVRare identified 36 novel protein-coding disrupting SVs on diagnostic grade genes that may explain proband’s phenotype. It is estimated that SVRare can increase SV-based diagnosis yield by at least 4-fold. We also performed a genome-wide association study, and uncovered clusters of dcSVs in genes with known pathogenicity, such as PKD1/2 - cystic kidney diseases and LDLR - familial hypercholesterolaemia.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-17">
                             Structural variants (SVs) are typically defined as DNA changes that extend to at least 50 nucleotides. A wide range of different types of SVs have been shown to cause rare inherited diseases, such as deletion, duplication, inversion, insertion and translocation. Complex SVs involving at least three breakpoint junctions have also been found to cause disease
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                             </sup>
                             . However, the full contribution of dcSVs to rare diseases is difficult to ascertain, primarily because of a lack of utilities to efficiently prioritise SVs that are potentially disease-causing. This issue is exacerbated for large cohorts of whole genome sequencing (WGS) data.
                            </p>
                            <p id="p-18">
                             For the 100K Genome Project (100KGP), a UK wide genome sequencing initiative run by Genomics England (GEL, The National Genomics Research and Healthcare Knowledgebase v5, Genomics England. doi:10.6084/m9.figshare.4530893.v5. 2019), there were 71,408 participants from 33,924 families in the rare disease arm (as of 20th of June, 2021). The diagnosis pipeline applied by GEL is as shown in
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Figure 1
                             </a>
                             , where each single nucleotide variant or short indel variant goes through a series of decision stages to be assigned a tier. An essential component of the process is PanelApp, which is a crowd-sourcing knowledgebase that allows virtual gene panels related to human disorders to be created, stored and queried. Each panel may have three categories of genes and regions, and they are coloured according to the evidence available to support aetiology:
                             <strong>
                              green
                             </strong>
                             genes are of diagnostic-grade with high level of evidence to support gene-disease association;
                             <strong>
                              amber
                             </strong>
                             genes have moderate evidence and should not yet be used for genome interpretation;
                             <strong>
                              red
                             </strong>
                             genes have not enough evidence and should not be used for genome interpretation. In GEL, Rare variants of Tier 1 and 2 (protein altering variant in a green gene, on a gene panel from PanelApp applied to the participant) will be further assessed by the NHS Genomics Medical Centres (GMCs) before reporting.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F1.large.jpg?width=800&height=600&carousel=1" title="Overview of the GEL”s tiering process for SNV or indel variant. A recreation of the tiering process flowchart from Genomics England “Rare Disease Results Guide”." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-114763281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Overview of the GEL”s tiering process for SNV or indel variant.</span> A recreation of the tiering process flowchart from Genomics England “Rare Disease Results Guide”.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F1.medium.gif" width="440" height="277"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F1.medium.gif" width="440" height="277"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1:
                              </span>
                              <span class="caption-title">
                               Overview of the GEL”s tiering process for SNV or indel variant.
                              </span>
                              <p id="p-19" class="first-child">
                               A recreation of the tiering process flowchart from Genomics England “Rare Disease Results Guide”.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-20">
                             GEL also features a copy number variant (CNV) reporting pipeline, which involves tiering of CNV calls produced by the Canvas software
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             . Although calls produced by the Manta software
                             <sup>
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             are also made available in the GEL research environment, they are not being considered in the GEL’s CNV reporting pipeline.
                            </p>
                            <p id="p-21">
                             Here we used SVRare to collate 554,060,126 deletions, duplications and inversions from all the 71,408 participants in the rare disease arm of GEL. From a pilot study of 4313 families, SVRare identified 36 potentially dcSVs; this was equivalent of a 4-fold increase in SV-based diagnostic yield.
                            </p>
                            <p id="p-22">
                             We also performed a genome-wide association study, and uncovered clusters of dcSVs in genes with known pathogenicity, such as PKD1/2 - cystic kidney diseases and LDLR - familial hypercholesterolaemia.
                            </p>
                            <p id="p-23">
                             All SVs of interest described in this study are detailed in Supplementary table 2. The codebase for SVRare can be found in
                             <a href="https://github.com/Oxford-Eye/SVRare">
                              https://github.com/Oxford-Eye/SVRare
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Method
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Database configuration
                             </h3>
                             <p id="p-24">
                              SVs had been called using Manta
                              <sup>
                               <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                                1
                               </a>
                              </sup>
                              and Canvas
                              <sup>
                               <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                              </sup>
                              . SV types called by Canvas included LOSS and GAIN, and those called by Manta included DEL, INV (inversion), INS (insertion), DUP (duplication of tandem repeats), and BND (breaking ends). For this work, Canvas LOSS, Canvas GAIN, Manta DEL, Manta INV were included for further analysis, since the similarity calculation is less complex for these types. For the purposes for the current study, Canvas LOSS and Manta DEL are treated as the same type: LOSS.
                             </p>
                             <p id="p-25">
                              The workflow is shown in
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              . Most samples on GEL were aligned against GRCh38. A small fraction that were aligned against GRCh37, the SV coordinates were first lifted to GRCh38 before being imported to the MySQL database with the other samples. Those that failed the liftover were discarded. Therefore, all the genomic coordinates mentioned in the texts use GRCh38 as the reference, unless otherwise stated. Structural variants were imported into the “Interval” table with the following features: chromosome, start, end and type. Type takes one of the three values: “LOSS” (either called from Manta DEL, or Canvas LOSS), “GAIN” (called from Canvas) and “INV” (called from Manta). Event specific data, such as “sv_id” (the ID value from the Manta or Canvas call), “filter”, source (either Manta or Canvas), genotype (either “HET” or “HOM”), CN (copy number from Canvas call) were imported into the “Participant_Interval”” relation table. Genes that are covered by the SV were recorded in the “Interval_Gene”” relation table. Participant”s HPO terms were imported into the “Participant_HPO” relation table. Details of all the panels in PanelApp, and which panels were applied to which participants, were also imported into the database.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F2.large.jpg?width=800&height=600&carousel=1" title="SVRare workflow on GEL. It used Labkey to locate the VCF file for each participant. The VCF file was a merged result from Manta and Canvas calls. If the genome build was GRCh37, it converted the coordinates of all SVs to GRCh38, when possible. The SVs of type LOSS (deletion), GAIN (duplication) and INV (inversion) were imported into a MySQL database, together with participant details, such as disease, HPO terms, family and gene panels that had been applied to the participant. Intervals were then clustered using DBSCAN (eps = 0.1, minimal cluster size = 2). When queried for rare SVs, it would filter for clusters with a small cluster size, then annotate each SV with estimated AF, genes covered, if genes were of interest (either based on HPO terms, or gene panels), if coding regions were disrupted, if the SV was also observed in other family members, and if the SV was also observed in other participants affected by the same disease. Note that some panels in the PanelApps in GEL included genomic regions instead of gene names; they were not imported in the database." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-114763281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">SVRare workflow on GEL.</span> It used Labkey to locate the VCF file for each participant. The VCF file was a merged result from Manta and Canvas calls. If the genome build was GRCh37, it converted the coordinates of all SVs to GRCh38, when possible. The SVs of type LOSS (deletion), GAIN (duplication) and INV (inversion) were imported into a MySQL database, together with participant details, such as disease, HPO terms, family and gene panels that had been applied to the participant. Intervals were then clustered using DBSCAN (eps = 0.1, minimal cluster size = 2). When queried for rare SVs, it would filter for clusters with a small cluster size, then annotate each SV with estimated AF, genes covered, if genes were of interest (either based on HPO terms, or gene panels), if coding regions were disrupted, if the SV was also observed in other family members, and if the SV was also observed in other participants affected by the same disease. Note that some panels in the PanelApps in GEL included genomic regions instead of gene names; they were not imported in the database.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F2.medium.gif" width="440" height="288"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F2.medium.gif" width="440" height="288"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <span class="caption-title">
                                SVRare workflow on GEL.
                               </span>
                               <p id="p-26" class="first-child">
                                It used Labkey to locate the VCF file for each participant. The VCF file was a merged result from Manta and Canvas calls. If the genome build was GRCh37, it converted the coordinates of all SVs to GRCh38, when possible. The SVs of type LOSS (deletion), GAIN (duplication) and INV (inversion) were imported into a MySQL database, together with participant details, such as disease, HPO terms, family and gene panels that had been applied to the participant. Intervals were then clustered using DBSCAN (eps = 0.1, minimal cluster size = 2). When queried for rare SVs, it would filter for clusters with a small cluster size, then annotate each SV with estimated AF, genes covered, if genes were of interest (either based on HPO terms, or gene panels), if coding regions were disrupted, if the SV was also observed in other family members, and if the SV was also observed in other participants affected by the same disease. Note that some panels in the PanelApps in GEL included genomic regions instead of gene names; they were not imported in the database.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-27">
                              Intervals of the same type were clustered using DBSCAN, with eps = 0.1and min_samples = 2. Similarity between two intervals was calculated as the fraction of the overlap (intersection) over the total length (union). Hence the similarity between any two intervals ranges between 0 and 1. The result of the clustering, together with the sample-size of each cluster, were imported into the tables “Interval_Group” and “Group”, respectively.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              GEL 4K families pilot study
                             </h3>
                             <p id="p-28">
                              To demonstrate the effectiveness of SVRare in finding disease-causing SVs, we applied it to 4,365 families from the rare disease arm in GEL (both solved and unsolved). Among them, 52 families had missing variant calling files for their proband and were removed from the downstream analyses. This left us with 4313 families. To maximise our chance of identifying pathogenic SVs, we applied the most stringent filters to all the rare SVs found in all the 4313 families: each SV from a family had to satisfy: 1) no other SV with a similarity score of >0.5 can be found carried by an individual who was not a family member and was not affected by the same disease; 2) for deletions, the SV had to cover at least one coding region of a green gene from a gene panel that had been applied to the proband of the family; 3) for inversions and duplications, the SV had to cover at least one coding region, and one of its two ends had to be within a green gene from a gene panel that had been applied to the proband of the family.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Gene-disease association study
                             </h3>
                             <p id="p-29">
                              We applied SVRare to all the 19,977 protein coding genes (coordinates extracted from Ensembl Homo_sapien GRCh38 version 98). For each gene, we used DBSCAN to cluster all the SVs, and clusters with a sample size over 50 were filtered. Fisher’s exact test was applied to SVs that are likely to disrupt a coding region of the gene. For this work, only deletions were considered.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-6">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Summary of SVs in GEL
                             </h3>
                             <p id="p-30">
                              There were 71,408 participants from 33,923 families in the rare disease arm of GEL (snapshot on the 20th of June, 2021). They included 221 normalised rare disease terms. From them, a total of 554,060,126 SVs were imported to the MySQL database (average 7,759 SVs per individual), among which 415,568,340 SVs had a ‘PASS’ filter inherited from Manta or Canvas (75% PASS rate). Of all the ∼554 million SV records, the majority were recurrences. For example, all of the 71,408 participants carry a chr2:g.186572387_186572581del. There were 20,773,308 distinct SVs, most of which were rare; and only a small fraction of those were carried by >1% of the participants (52,073 SVs, 0.2% of all distinct SVs). Strikingly, these 0.2% common SVs account for 85.5% of all SV calls. Around 72% of all the distinct SVs were deletions (14,853,528), 20% were duplications (4,239,974), and 8% were inversions (1,679,806).
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Pilot study of 4313 families
                             </h3>
                             <div id="sec-9" class="subsection">
                              <h4>
                               Overview
                              </h4>
                              <p id="p-31">
                               In order to determine the effectiveness of SVRare, we applied the method to 4313 families (we will refer to it as 4k families henceforth). They encompassed 179 distinct normalised rare disease terms (
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Supplementary table 1
                               </a>
                               shows the top 15 diseases). 686 of them had previously been solved or partially solved, of which 9 diagnoses were based on SVs (excluding 2 microsatellite expansions as they are out of the scope of this work); and SVRare correctly prioritised 6 of the 9 SVs. Of all the 3 SVs missed by SVRare, 2 were miss-called by both Manta and Canvas and thus were not present in the SVRare’s database. Both diagnoses were likely made outside of the 100KGP program. The third SV was a deletion in a region with ∼300 other deletions with a similarity score >0.8, and was considered as a ‘common’ SV by SVRare.
                              </p>
                              <p id="p-32">
                               A total of 280,285 rare SVs were pooled from the 4k families (mean of 65 SVs per family), of which 35,221 were private (mean of 8 SVs per family). Interestingly, WGS aligned against GRCh37 produced more than double rare SVs than GRCh38, in average (mean: 46 vs 141, median: 35 vs 71,
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Figure 3A
                               </a>
                               ), a likely result of build-specific artifacts. Among all the rare SVs, there were 247,322 deletions (88.2%), 23,822 duplications (8.5%) and 9,141 inversions (3.3%).
                              </p>
                              <div id="F3" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F3.large.jpg?width=800&height=600&carousel=1" title="A) Distribution of rare SV count per family against genome build from the 4k families. Y-axis is in log10 scale. B) Distribution of SV length against true/false positives.SV sizes of false positives are significantly longer than true positives for LOSS/deletions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-114763281" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A)</strong> Distribution of rare SV count per family against genome build from the 4k families. Y-axis is in log10 scale. <strong>B)</strong> Distribution of SV length against true/false positives.SV sizes of false positives are significantly longer than true positives for LOSS/deletions.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F3.medium.gif" width="389" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F3.medium.gif" width="389" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 3:
                                </span>
                                <p id="p-33" class="first-child">
                                 <strong>
                                  A)
                                 </strong>
                                 Distribution of rare SV count per family against genome build from the 4k families. Y-axis is in log10 scale.
                                 <strong>
                                  B)
                                 </strong>
                                 Distribution of SV length against true/false positives.SV sizes of false positives are significantly longer than true positives for LOSS/deletions.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-34">
                               In order to find disease-causing SVs, we chose those that were private, disrupt coding regions of a gene, and the gene was a green gene on a panel that had been applied to the proband of a family. This gave 223 SVs in 197 families as a result (one family had 8,832 rare SVs, and exceptionally more private SVs than the average, most of which were false positives. The family was removed from the list). Of them, 132 (59%) were deletions, 74 (33%) were duplications, and 17 (8%) were inversions. We manually inspected each SV and found 176 true positives (precision = 78.9%). As expected from the observation above, precision strongly correlates with genome builds (precision for GRCh38: 87.4%, precision for GRCh37: 59.6%, p = 6.8e
                               <sup>
                                −6
                               </sup>
                               of chi-square test of independence). A second variable correlated with precision was the SV size: the larger the size, the higher the likelihood of the SV being false positive, which was significant for deletions (
                               <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                Figure 3B
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-10" class="subsection">
                              <h4>
                               Complex SVs
                              </h4>
                              <p id="p-35">
                               Since Manta and Canvas were not able to identify cxSVs, they were usually called as multiple SVs next to each other. From the manual inspection of the 223 SVs, we found three pseudogenes, four inverted deletions (INVdel), one paired-deletion inversion (delINVdel), one paired-deletion translocation (deldelTrans), and one deletion inversion duplication (delINVdup) (
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Figure 4
                               </a>
                               ). The three pseudogenes were unlikely to be disease-causing: they typically appear as complete intronic deletions in the known genes, but they are often inactive elements inserted into regions with unknown clinical relevance.
                              </p>
                              <div id="F4" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F4.large.jpg?width=800&height=600&carousel=1" title="Complex SVs found in the 4k families study. They include pseudogene, inverted deletion (INVdel), paired-deletion inversion (delINVdel), deletion inversion duplication (delINVdup), and paired-duplication inversion (dupINVdup)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-114763281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Complex SVs found in the 4k families study.</span> They include pseudogene, inverted deletion (INVdel), paired-deletion inversion (delINVdel), deletion inversion duplication (delINVdup), and paired-duplication inversion (dupINVdup).</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F4.medium.gif" width="440" height="259"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F4.medium.gif" width="440" height="259"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 4:
                                </span>
                                <span class="caption-title">
                                 Complex SVs found in the 4k families study.
                                </span>
                                <p id="p-36" class="first-child">
                                 They include pseudogene, inverted deletion (INVdel), paired-deletion inversion (delINVdel), deletion inversion duplication (delINVdup), and paired-duplication inversion (dupINVdup).
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-37">
                               The four INVdel were observed in GRIN2B – Intellectual disability, LGI1 – Familial focal epilepsies (Genetic epilepsy syndrome panel), MEIS2 – Syndromic cleft lip and or cleft palate (Clefting panel, Intellectual disability panel), and PKD1 – Cystic kidney disease. Interestingly, all four INVdel SVs were associated with autosomal dominant (AD) inheritance. Considering that over all there were 31 rare SVs in genes mostly associated with AD and 87 SVs in genes largely associated autosomal recessive (AR) disorders, this enrichment of INVdel in AD genes associated is very unusual.
                              </p>
                              <p id="p-38">
                               The delINVdel was identified in the DSG2 gene of a participant affected by arrhythmogenic right ventricular cardiomyopathy. Since DSG2 is associated with autosomal dominant arrhythmogenic cardiomyopathy, this complex SV is likely the cause of the participant”s disease.
                              </p>
                              <p id="p-39">
                               The heterozygous delTrans was found in the NDUFS1 gene of a participant affected by intellectual disability. However, since NDUFS1 is associated with autosomal recessive intellectual disability, and we were unable to find a second pathogenic variant in the gene, its clinical relevance in this case is therefore uncertain.
                              </p>
                              <p id="p-40">
                               Similarly, the delINVdup SV was not considered as causal since the participant carrying the SV in the UROC1 gene had previously been solved on the NR2F1 gene for intellectual disability.
                              </p>
                              <p id="p-41">
                               A heterozygous paired-duplication inversion was found in the FAM20C gene. The 5’ and 3’ end of the inversion were in the 3
                               <sup>
                                rd
                               </sup>
                               intron of FAM20C and downstream of FAM20C, thus the inverted copy of FAM20C missed the first three exons and was unlikely to encode any protein. However since FAM20C is associated with AR disorders, and a second rare variant was not found in this study, the clinical relevance of this cxSV remains unknown (i.e. this individual may be a carrier).
                              </p>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               Interesting SVs
                              </h4>
                              <p id="p-42">
                               SVRare made the discovery of pathogenic SVs fast, and we found 97 ‘interesting’ SVs. To be counted as interesting, one SV had to be predicted as protein-coding disrupting, and was only found within the family, or unrelated individuals affected by the same disease. We considered SVs interesting if they were not pseudogenes, the individuals carrying the SVs were stated as unsolved in GMC, and the inheritance of the SVs did not conflict with the family’s pedigree (Table 1). We also merged SVs if they could collectively explain a cxSV event. Notably, four deletions were on genes associated with AD diseases, but were also found to be carried by an unaffected parent (CYFIP2 – Intellectual disability, RAC1/ACTB – Intellectual disability, ERF – Craniosynostosis, and TAOK1 – Intellectual disability). Therefore, the clinical relevance of the SVs was uncertain, and their ‘interesting’ status was assigned with ‘unsure’.
                              </p>
                              <p id="p-43">
                               There were 27 interesting SVs predicted to be disrupting protein-coding in genes associated with AD diseases, and an additional 3 chromosome X hemizygous SVs found in XY participants. Among them, 10 were de novo, 2 X-linked affected male probands inherited the SVs maternally, and 16 had insufficient information to determine SV hereditary. In the two remaining cases, TRIO (Intellectual disability) was inherited from the unaffected mother who was mosaic on the SV, and UBE3A (Intellectual disability, a paternally imprinted gene) was maternally inherited.
                              </p>
                              <p id="p-44">
                               For the 67 interesting SVs in AR genes, we went to search for a short nucleotide variant (SNV) segregating with the disease. To be considered as pathogenic, the variant must be rare (gnomAD v3.0 allele frequency <= 0.05 or GEL allele frequency <= 0.05, and homozygote/hemizygote count <= 1 in gnomAD) and predicted as likely disease-causing (CADD_phred
                               <sup>
                                <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                                 4
                                </a>
                               </sup>
                               >= 15 or MaxEntScan_diff
                               <sup>
                                <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                                 5
                                </a>
                               </sup>
                               > 2 or SpliceAI_DS
                               <sup>
                                <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                                 6
                                </a>
                               </sup>
                               > 0.5). This resulted in 3 SNVs compound-heterozygous with the SVs of interest. A further 2 SNVs were found in singleton families. It is likely that more biallelic disease-causing changes can be found with further inspections.
                              </p>
                              <p id="p-45">
                               A heterozygous SV knocking out exon 23 in MYO3A (aa.836-879, NM_017433.5), a gene usually linked to AR hearing loss, was found in three individuals of a family who all had hearing loss (proband, father, and a full sibling). However, recent research illustrated the molecular mechanism underlying an AD form of hearing loss caused by dominant negative mutations in MYO3A that encodes the motor domain (aa.338-1053) of the protein
                               <sup>
                                <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                                 7
                                </a>
                               </sup>
                               . Therefore, we surmise that the heterozygous deletion of exon 23 in MYO3A was disease-causing in this AD hearing loss family.
                              </p>
                             </div>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Gene - disease associations
                             </h3>
                             <p id="p-46">
                              We grouped rare protein-coding disrupting SVs on a gene level and performed disease associations. For this study, we performed association study on deletions only, as it was recognised as the predominant SV type for causing diseases. As shown in
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              , this uncovered many well-established gene-disease relationships, e.g. PKD1/2 – Cystic kidney disease, LDLR – Familial hypercholesterolaemia, and SPAST - Hereditary spastic paraplegia. However, majorities of the deletions identified had not been previously reported, and the carriers remained unsolved.
                             </p>
                             <p id="p-47">
                              A notable difference between SNP GWAS and SV GWAS is that an SV may span multiple genes, and only one or two of the genes are of clinical relevance. Therefore, it could give rise to false associations on neighbouring genes. A prime example of this was TSC1 – Classical tuberous sclerosis. Four affected participants from the same family carried the same 1.4Mb deletion that knocked out TSC1, GFI1B, GTF3C5, CEL, RALGDS, and GBGT1. Although perhaps only TSC1 was disease-causing, the other neighbouring genes also produced a strong association with Classical tuberous sclerosis.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F5.large.jpg?width=800&height=600&carousel=1" title="Disease associations. Top panel: Manhattan plot shows strong associations in known gene-disease relations. Known gene-disease relations below the significant level of p = e-5 are highlighted according to the colour scheme of GEL PanelApp. Annotated are PKD1 (with cystic kidney disease), LDLR (with Familial Hypercholesterolaemia), PKD2 (with cystic kidney disease), SPAST (with Hereditary spastic paraplegia), TSC1 (with classical tuberous sclerosis) and EP300 (with intellectual disability). Bottom panel: genome tracks to show SV positions. Green tracks are SVs previously identified. Teal tracks are novel SVs. The numbers on track depict the number of carriers / affected families. A) PKD1 – Cystic kidney disease. B) PKD2 – Cystic kidney disease. C) LDLR – familial hypercholersterolaemia. D) SPAST – Hereditary paraplegia. SV tracks highlighted in green had previously been confirmed as disease-causing. The numbers on each SV track are “number of carriers” / “number of carrier families”. There are two cases with incomplete penetrance and unmatched disease terms: one in B) where the SV is carried by two of a family affected by CAKUT (one is an unaffected relative); the other in C) where the SV was found in three families, and one family was registered with ‘Ultra-rare undescribed monogenic disorders’, and has an unaffected carrier." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-114763281" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Disease associations.</span> <strong>Top panel:</strong> Manhattan plot shows strong associations in known gene-disease relations. Known gene-disease relations below the significant level of p = e<sup>-5</sup> are highlighted according to the colour scheme of GEL PanelApp. Annotated are PKD1 (with cystic kidney disease), LDLR (with Familial Hypercholesterolaemia), PKD2 (with cystic kidney disease), SPAST (with Hereditary spastic paraplegia), TSC1 (with classical tuberous sclerosis) and EP300 (with intellectual disability). <strong>Bottom panel</strong>: genome tracks to show SV positions. Green tracks are SVs previously identified. Teal tracks are novel SVs. The numbers on track depict the number of carriers / affected families. A) PKD1 – Cystic kidney disease. B) PKD2 – Cystic kidney disease. C) LDLR – familial hypercholersterolaemia. D) SPAST – Hereditary paraplegia. SV tracks highlighted in green had previously been confirmed as disease-causing. The numbers on each SV track are “number of carriers” / “number of carrier families”. There are two cases with incomplete penetrance and unmatched disease terms: one in B) where the SV is carried by two of a family affected by CAKUT (one is an unaffected relative); the other in C) where the SV was found in three families, and one family was registered with ‘Ultra-rare undescribed monogenic disorders’, and has an unaffected carrier.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F5.medium.gif" width="422" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F5.medium.gif" width="422" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <span class="caption-title">
                                Disease associations.
                               </span>
                               <p id="p-48" class="first-child">
                                <strong>
                                 Top panel:
                                </strong>
                                Manhattan plot shows strong associations in known gene-disease relations. Known gene-disease relations below the significant level of p = e
                                <sup>
                                 -5
                                </sup>
                                are highlighted according to the colour scheme of GEL PanelApp. Annotated are PKD1 (with cystic kidney disease), LDLR (with Familial Hypercholesterolaemia), PKD2 (with cystic kidney disease), SPAST (with Hereditary spastic paraplegia), TSC1 (with classical tuberous sclerosis) and EP300 (with intellectual disability).
                                <strong>
                                 Bottom panel
                                </strong>
                                : genome tracks to show SV positions. Green tracks are SVs previously identified. Teal tracks are novel SVs. The numbers on track depict the number of carriers / affected families. A) PKD1 – Cystic kidney disease. B) PKD2 – Cystic kidney disease. C) LDLR – familial hypercholersterolaemia. D) SPAST – Hereditary paraplegia. SV tracks highlighted in green had previously been confirmed as disease-causing. The numbers on each SV track are “number of carriers” / “number of carrier families”. There are two cases with incomplete penetrance and unmatched disease terms: one in B) where the SV is carried by two of a family affected by CAKUT (one is an unaffected relative); the other in C) where the SV was found in three families, and one family was registered with ‘Ultra-rare undescribed monogenic disorders’, and has an unaffected carrier.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-49">
                              We found a 1.2Mb deletion (chr4:g.87164899_88268586del) carried by four participants that knocked out PKD2. Two had previously been solved based on the deletion, and the other two were from the same family, with the proband having Congenital Anomaly of the Kidneys and Urinary Tract (CAKUT) instead of Cystic kidney disease. More interestingly, the other carrier from the family was not affected, thus the penetrance of the deletion may not be complete. Although the coordinates were reported differently, a manual inspection of the IGV plots concluded that they were in fact deletions of the identical region. A kinship analysis revealed that the two solved participants were related to the second degree (kinship coefficient of 0.126383)
                              <sup>
                               <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                                8
                               </a>
                              </sup>
                              , but neither of them were related to the CAKUT family. Analysis of the surrounding regions reviewed rare SNPs shared only by the four participants, indicating the alleles carrying the deletion share a common ancestor. Further investigation will be required to confirm this hypothesis. Similarly, a 10kb deletion (chr19:g.11097784_11108089del) in LDLR that knocks out most of its ligand binding domain
                              <sup>
                               <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                              </sup>
                              was carried by four participants. Two unrelated individuals were affected by typical Familial hypercholesterolaemia; but the other two were from a family that had a normalised disease term of “Ultra-rare undescribed monogenic disorders”, with one of them being unaffected. These two cases demonstrated that same SVs may confer varying phenotypes.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Three solved cases
                             </h3>
                             <p id="p-50">
                              During the course of this study, three cases recruited in Oxford were solved based on the results produced by SVRare.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              REEP6, CNGB3 and retinal dystrophy
                             </h3>
                             <p id="p-51">
                              As a result of initial testing of SVRare in a group of participants recruited in the Oxford Eye Hospital, we identified a homozygous deletion in the REEP6 gene, and a heterozygous tandem duplication in the CNGB3 gene. The REEP6 deletion (chr19:g.1490941_1492690del) knocks out the first exon of the gene, and the CNGB3 SV (chr8:g.86652138_86663932dup) duplicates exon 7. We also identified a second variant in CNGB3 (chr8:g.86643780AG>A, NP_061971.3 p.Thr383IlefsTer13). Given the phenotype is supportive in both cases (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 5
                              </a>
                              ), we surmise that the SVs are highly likely to be disease-causing.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F6.large.jpg?width=800&height=600&carousel=1" title="retinal imaging of patients carrying a homozygous REEP6 deletion and a heterozygous CNGB3 tandem duplication. Panel A shows colour imaging of the REEP6 patient that displays bilateral peripheral bone-spicule pigmentation mainly located in the mid periphery, with attenuated blood vessel and pale optic discs; and of the CNGB3 patient that displays a central patch of atrophy involving both central maculae, with mild pallor of discs and subtle attenuation of vasculature. Panel B shows spectral domain optical coherence tomography (SD-OCT) of the REEP6 patient that displays bilateral cystoid macular oedema, atrophy of the outer retinal layers external to the fovea with thickening of the ellipsoid zone at the fovea; and of the CNGB3 patient that displays bilateral optical gaps with loss of the ellipsoid zone involving both foveae. Panel C shows short-wavelength autofluorescence imaging of the REEP6 patient that displays an annulus of increased signal around the fovea with widespread patchy loss of signal corresponding to areas of atrophy in the mid periphery; and of the CNGB3 patient that displays central gross reduction in signal involving both central maculae consistent with atrophy with an irregular edge and a small patch of retained AF signal at both foveae." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-114763281" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">retinal imaging of patients carrying a homozygous REEP6 deletion and a heterozygous CNGB3 tandem duplication.</span> Panel <strong>A</strong> shows colour imaging of the REEP6 patient that displays bilateral peripheral bone-spicule pigmentation mainly located in the mid periphery, with attenuated blood vessel and pale optic discs; and of the CNGB3 patient that displays a central patch of atrophy involving both central maculae, with mild pallor of discs and subtle attenuation of vasculature. Panel <strong>B</strong> shows spectral domain optical coherence tomography (SD-OCT) of the REEP6 patient that displays bilateral cystoid macular oedema, atrophy of the outer retinal layers external to the fovea with thickening of the ellipsoid zone at the fovea; and of the CNGB3 patient that displays bilateral optical gaps with loss of the ellipsoid zone involving both foveae. Panel <strong>C</strong> shows short-wavelength autofluorescence imaging of the REEP6 patient that displays an annulus of increased signal around the fovea with widespread patchy loss of signal corresponding to areas of atrophy in the mid periphery; and of the CNGB3 patient that displays central gross reduction in signal involving both central maculae consistent with atrophy with an irregular edge and a small patch of retained AF signal at both foveae.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F6.medium.gif" width="440" height="252"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F6.medium.gif" width="440" height="252"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <span class="caption-title">
                                retinal imaging of patients carrying a homozygous REEP6 deletion and a heterozygous CNGB3 tandem duplication.
                               </span>
                               <p id="p-52" class="first-child">
                                Panel
                                <strong>
                                 A
                                </strong>
                                shows colour imaging of the REEP6 patient that displays bilateral peripheral bone-spicule pigmentation mainly located in the mid periphery, with attenuated blood vessel and pale optic discs; and of the CNGB3 patient that displays a central patch of atrophy involving both central maculae, with mild pallor of discs and subtle attenuation of vasculature. Panel
                                <strong>
                                 B
                                </strong>
                                shows spectral domain optical coherence tomography (SD-OCT) of the REEP6 patient that displays bilateral cystoid macular oedema, atrophy of the outer retinal layers external to the fovea with thickening of the ellipsoid zone at the fovea; and of the CNGB3 patient that displays bilateral optical gaps with loss of the ellipsoid zone involving both foveae. Panel
                                <strong>
                                 C
                                </strong>
                                shows short-wavelength autofluorescence imaging of the REEP6 patient that displays an annulus of increased signal around the fovea with widespread patchy loss of signal corresponding to areas of atrophy in the mid periphery; and of the CNGB3 patient that displays central gross reduction in signal involving both central maculae consistent with atrophy with an irregular edge and a small patch of retained AF signal at both foveae.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              KDELR2 and Osteogenesis imperfecta
                             </h3>
                             <p id="p-53">
                              A family-oriented analysis was performed on a parent-proband-sibling family from 100KGP where both the proband and her brother presented with severe Osteogenesis Imperfecta (OI).
                             </p>
                             <p id="p-54">
                              In the proband, we identified two compound heterozygous intragenic deletions in KDELR2 (Supplementary Figure 6). The affected brother also harboured the same combination of deletions, with phasing confirmed by parental inheritance. This gene was only associated with OI in 2020, when four families with biallelic SNVs were identified
                              <sup>
                               <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                              </sup>
                              . A more recent follow up study identified two further families with OI and additional neurodevelopmental features
                              <sup>
                               <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                              </sup>
                              . This is also the first reported OI case that was caused by biallelic SVs. Detailed clinical and neurodevelopmental assessment of the family described here is underway and will be described elsewhere.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F7.large.jpg?width=800&height=600&carousel=1" title="An Integrative Genomics Viewer shows two exonic deletions disrupting KDELR2 in a family affected by Osteogenesis imperfecta. Top two tracks were affected siblings. Bottom two tracks were unaffected parents." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-114763281" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">An Integrative Genomics Viewer shows two exonic deletions disrupting KDELR2 in a family affected by Osteogenesis imperfecta.</span> Top two tracks were affected siblings. Bottom two tracks were unaffected parents.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F7.medium.gif" width="440" height="205"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F7.medium.gif" width="440" height="205"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/02/15/2021.10.15.21265069/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <span class="caption-title">
                                An Integrative Genomics Viewer shows two exonic deletions disrupting KDELR2 in a family affected by Osteogenesis imperfecta.
                               </span>
                               <p id="p-55" class="first-child">
                                Top two tracks were affected siblings. Bottom two tracks were unaffected parents.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-56">
                             Many tools have been developed to call SVs from short-read WGS
                             <sup>
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             . However, high false-positive rates and a lack of means to efficiently merge SVs from different individuals have been hindering dcSV discovery. Indeed, from the 686 existing diagnoses or partial diagnoses in the 4k families included in our pilot study (15.9%), SV contributed to only nine, four of which were likely made outside GEL’s diagnostic pipeline.
                            </p>
                            <p id="p-57">
                             In this study, we introduce SVRare and its implementation in 100KGP that enabled fast dcSV discovery. It aggregated 554 million SV events called from Manta and Canvas in 71 thousand individuals, or 34 thousand families. In the 4k families pilot study, we used SVRare to identify 97 interesting SVs, with 31 of them likely causing AD (or X-linked) disorders. We also found a second rare variant predicted to be pathogenic in five AR cases that segregates with an interesting SV. Suppose all the 36 SVs are causal, this is a four-fold increase in SV-based diagnostic yield. From the 9 SV-based diagnoses previously made, SVRare identified 6 of them. Excluding two that were miss called by Canvas and Manta, SVRare achieved a high recall (85.7%).
                            </p>
                            <p id="p-58">
                             Notably, all the 9 dcSVs identified prior to SVRare were deletions. In contrast, SVRare uncovered a range of different types of dcSVs in the pilot study, including 64 deletions, 23 duplications, three inversions, and seven cxSVs. In the 31 SVs associated with AD disorders, 18 were deletions, 6 were duplications, 2 were inversions and 5 were cxSVs. Interestingly, all the 5 cxSVs involving both inversions and deletions found in this study were associated with AD disorders. The reason for the unbalanced distribution of cxSVs underlying AD and AR disorders remains unknown. Further studies will be carried out to verify this finding.
                            </p>
                            <p id="p-59">
                             The 4k family pilot study was conducted to consider only private protein-coding disrupting SVs: an SVs was considered private if they were only found in families that had the same normalised disease term. This setting inevitably favoured the findings associated with AD disorders, since protein-coding disrupting SVs in genes associated with AR disorders tend to be more tolerated, and thus are more pervasive in the general population. It is therefore highly likely that a dedicated family-wise investigation will produce more findings.
                            </p>
                            <p id="p-60">
                             In this study, we performed GWAS for deletions only, since it was the predominant SV type in the database. As shown in
                             <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                              Figure 4
                             </a>
                             , many of the top ranked associations were well known, such as PKD1/2 and Cystic kidney disease. However, majority of the deletions underlying the strong associations had not been reported.
                            </p>
                            <p id="p-61">
                             An advantage of gene-wise analysis over family-wise analysis is that one can systematically investigate all dcSVs within a gene. For example, the heterozygous deletion that completely knocked out PKD2 was not reported in a carrier with CAKUT. We suspect that this was because the unaffected relative of the proband was also a carrier of the deletion. From our gene-wise analysis on the gene, we found that the same deletion had been confirmed disease-causing in two separated cases. We surmise that the whole gene deletion of PKD2 has incomplete penetrance. We also uncovered a list of rare SNVs shared exclusively among the four carriers, indicating a common haplotype carrying the deletion. The rare SNVs can then be used as a surrogate to probe for the existence of the deletion.
                            </p>
                            <p id="p-62">
                             The first step of common practices when investigating SVs is usually removing calls that do not pass the filter in variant calling tools such as Manta and Canvas. However, in the 4k families pilot study, we found that 31 out of the 97 interesting SVs failed either Manta or Canvas filters. Therefore, we suggest against removing SVs based on filters. Further investigation is required to study the distribution of different filters in the true and false positives.
                            </p>
                            <p id="p-63">
                             One problem with merging records aligned against GRCh37 and GRCh38 was that a proportion of SVs were dropped during the process (i.e. LiftOver tools fails for regions that do not map 1 to 1). Despite this, samples aligned against GRCh37 produced significantly more rare SVs in average (
                             <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                              Figure 3A
                             </a>
                             ). The reason for this is still unclear, but we suspect that it is due to genome-build related artifacts. The solution would be to realign all GRCh37 samples to GRCh38 and recall SVs.
                            </p>
                           </div>
                           <div class="section data-availability" id="sec-17">
                            <h2 class="">
                             Data Availability
                            </h2>
                            <p id="p-64">
                             All data produced in the present study are available in the manuscript and online at
                             <a href="https://github.com/Oxford-Eye/SVRare">
                              https://github.com/Oxford-Eye/SVRare
                             </a>
                            </p>
                            <p id="p-65">
                             <a href="https://github.com/Oxford-Eye/SVRare">
                              https://github.com/Oxford-Eye/SVRare
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             Supplemental materials
                            </h2>
                            <div id="T1" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/477573/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/477573/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/477573" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary table 1,
                              </span>
                              <p id="p-67" class="first-child">
                               Disease count of the top 15 diseases in the pilot 4k families
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-68">
                             <strong>
                              Supplementary table 2
                             </strong>
                             , SVs of interest discovered in the pilot 4k families
                            </p>
                            <p id="p-69">
                             <a href="https://github.com/Oxford-Eye/SVRare/raw/master/supplementaries/top_hits_for_publish.xlsx">
                              https://github.com/Oxford-Eye/SVRare/raw/master/supplementaries/top_hits_for_publish.xlsx
                             </a>
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgement
                            </h2>
                            <p id="p-66">
                             We would like to thank RetinaUK for supporting fundings for JY’s work on SVRare; John Fell Fund for setting up a local GEL user group which made this work possible. This research was also supported by the National Institute for Health Research (NIHR) Oxford Biomedical Research Centre Programme, and the Wellcome Trust (203141/Z/16/Z). This research was made possible through access to the data and findings generated by the 100,000 Genomes Project. The 100,000 Genomes Project is managed by Genomics England Limited (a wholly owned company of the Department of Health and Social Care). The 100,000 Genomes Project is funded by the National Institute for Health Research and NHS England. The Wellcome Trust, Cancer Research UK and the Medical Research Council have also funded research infrastructure. The 100,000 Genomes Project uses data provided by patients and collected by the National Health Service as part of their care and support
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-16">
                               Updated funding status
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  A, S.-J
                                 </span>
                                 .
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Complex structural variants in Mendelian disorders: identification and breakpoint resolution using short- and long-read genome sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.2" data-doi="10.1093/bioinformatics/btw163">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ivakhno
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Royce
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tanner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Canvas: versatile and scalable detection of copy number variants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2375
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2377
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw163%26rft_id%253Dinfo%253Apmid%252F27153601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw163&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27153601&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F15%2F2021.10.15.21265069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.3" data-doi="10.1093/bioinformatics/btv710">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1220
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1222
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv710%26rft_id%253Dinfo%253Apmid%252F26647377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv710&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26647377&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F15%2F2021.10.15.21265069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  P, R., M, S., J, S. & M, K
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  CADD-Splice-improving genome-wide variant effect prediction using deep learning-derived splice scores
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  G, Y. & CB, B
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of computational biology : a journal of computational molecular cell biology
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  377
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  394
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.volume%253D11%26rft.spage%253D377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.6" data-doi="10.1016/J.CELL.2018.12.015">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jaganathan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Predicting Splicing from Primary Sequence with Deep Learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  176
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  535
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  548
                                 </span>
                                 .e24 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.CELL.2018.12.015%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/J.CELL.2018.12.015&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F15%2F2021.10.15.21265069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.7" data-doi="10.1002/humu.22961">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grati
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  MYO3A causes human dominant deafness and interacts with protocadherin 15-CD2 isoform
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Human mutation
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  481
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.volume%253D37%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.22961%26rft_id%253Dinfo%253Apmid%252F26841241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/humu.22961&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26841241&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F15%2F2021.10.15.21265069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.8" data-doi="10.1093/bioinformatics/btq559">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  A, M.
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Robust relationship inference in genome-wide association studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics (Oxford, England)
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2867
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2873
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq559%26rft_id%253Dinfo%253Apmid%252F20926424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq559&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20926424&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F02%2F15%2F2021.10.15.21265069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000283919800010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Henderson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O”Kane
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGilligan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Watterson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The genetics and screening of familial hypercholesterolaemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Biomedical Science
                                 </abbr>
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cameron
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   di Stefano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Papenfuss
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Comprehensive evaluation and characterisation of short read general-purpose structural variant calling software
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   van D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Interaction between KDELR2 and HSP47 as a Key Determinant in Osteogenesis Imperfecta Caused by Bi-allelic Variants in KDELR2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  American journal of human genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  989
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  999
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.volume%253D107%26rft.spage%253D989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.10.15.21265069v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Efthymiou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Two novel bi-allelic KDELR2 missense variants cause osteogenesis imperfecta with neurodevelopmental features
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  American Journal of Medical Genetics Part A
                                 </abbr>
                                 <span class="cit-vol">
                                  185
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2241
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2249
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BMedical%2BGenetics%2BPart%2BA%26rft.volume%253D185%26rft.spage%253D2241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2021.10.15.21265069v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.01.07.22268886v2" title="Estimation of excess all-cause mortality due to COVID-19 in Thailand" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.09.09.21263013v3" title="Interpretable Brain Disease Classification and Relevance-Guided Deep Learning" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted February 15, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2021.10.15.21265069v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/02/15/2021.10.15.21265069.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2021.10.15.21265069v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about medRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2021.10.15.21265069v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   SVRare: discovering disease-causing structural variants in the 100K Genomes Project
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from medRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the medRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/476862"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-sHOdQSopD1YMAuZ5TEYoe00Rt2sqVAuOHrXzP2rihyk"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="277160564"/>
                                   <input type="hidden" name="captcha_token" value="b24c4b750a746d4bb6c1ab73f9f5ca2e"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node476862--3" data-node-nid="476862" id="node-476862--4297158069" data-pisa="medrxiv;2021.10.15.21265069v2" data-pisa-master="medrxiv;2021.10.15.21265069" data-seqnum="476862" data-apath="/medrxiv/early/2022/02/15/2021.10.15.21265069.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 SVRare: discovering disease-causing structural variants in the 100K Genomes Project
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Jing
                                  </span>
                                  <span class="nlm-surname">
                                   Yu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Anita
                                  </span>
                                  <span class="nlm-surname">
                                   Szabo
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Alistair T
                                  </span>
                                  <span class="nlm-surname">
                                   Pagnamenta
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Ahmed
                                  </span>
                                  <span class="nlm-surname">
                                   Shalaby
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Edoardo
                                  </span>
                                  <span class="nlm-surname">
                                   Giacopuzzi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Jenny
                                  </span>
                                  <span class="nlm-surname">
                                   Taylor
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Deborah
                                  </span>
                                  <span class="nlm-surname">
                                   Shears
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Nikolas
                                  </span>
                                  <span class="nlm-surname">
                                   Pontikos
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Genevieve
                                  </span>
                                  <span class="nlm-surname">
                                   Wright
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Michel
                                  </span>
                                  <span class="nlm-surname">
                                   Michaelides
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">
                                   Stephanie
                                  </span>
                                  <span class="nlm-surname">
                                   Halford
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="11">
                                  <span class="nlm-given-names">
                                   Susan
                                  </span>
                                  <span class="nlm-surname">
                                   Downes
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="12">
                                  <span class="nlm-collab" data-hwp-id="collab-1">
                                   Genomics England Research Consortium
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 medRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2021.10.15.21265069;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2021.10.15.21265069
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2021.10.15.21265069v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&title=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&title=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&text=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&t=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&title=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&title=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project&summary=&source=medRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&title=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node476862--5" data-node-nid="476862" id="citation-node-476862--61981222503" data-pisa="medrxiv;2021.10.15.21265069v2" data-pisa-master="medrxiv;2021.10.15.21265069" data-seqnum="476862" data-apath="/medrxiv/early/2022/02/15/2021.10.15.21265069.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     SVRare: discovering disease-causing structural variants in the 100K Genomes Project
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Jing
                                      </span>
                                      <span class="nlm-surname">
                                       Yu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Anita
                                      </span>
                                      <span class="nlm-surname">
                                       Szabo
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Alistair T
                                      </span>
                                      <span class="nlm-surname">
                                       Pagnamenta
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Ahmed
                                      </span>
                                      <span class="nlm-surname">
                                       Shalaby
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Edoardo
                                      </span>
                                      <span class="nlm-surname">
                                       Giacopuzzi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Jenny
                                      </span>
                                      <span class="nlm-surname">
                                       Taylor
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Deborah
                                      </span>
                                      <span class="nlm-surname">
                                       Shears
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Nikolas
                                      </span>
                                      <span class="nlm-surname">
                                       Pontikos
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Genevieve
                                      </span>
                                      <span class="nlm-surname">
                                       Wright
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Michel
                                      </span>
                                      <span class="nlm-surname">
                                       Michaelides
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">
                                       Stephanie
                                      </span>
                                      <span class="nlm-surname">
                                       Halford
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="11">
                                      <span class="nlm-given-names">
                                       Susan
                                      </span>
                                      <span class="nlm-surname">
                                       Downes
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="12">
                                      <span class="nlm-collab" data-hwp-id="collab-1">
                                       Genomics England Research Consortium
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     medRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2021.10.15.21265069;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2021.10.15.21265069
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=medRxiv&rft.au=Yu%2C+Jing&rft.au=Szabo%2C+Anita&rft.au=Pagnamenta%2C+Alistair+T&rft.au=Shalaby%2C+Ahmed&rft.au=Giacopuzzi%2C+Edoardo&rft.au=Taylor%2C+Jenny&rft.au=Shears%2C+Deborah&rft.au=Pontikos%2C+Nikolas&rft.au=Wright%2C+Genevieve&rft.au=Michaelides%2C+Michel&rft.au=Halford%2C+Stephanie&rft.au=Downes%2C+Susan&rft.au=%2C+">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/476862/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/476862/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/476862/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/476862/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/476862/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/476862/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/476862/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/476862/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/476862/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/476862/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/476862/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/476862/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&count=horizontal&via=&text=SVRare%3A%20discovering%20disease-causing%20structural%20variants%20in%20the%20100K%20Genomes%20Project&counturl=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2021.10.15.21265069v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.medrxiv.org/content/10.1101/2021.10.15.21265069v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genetic-and-genomic-medicine" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genetic and Genomic Medicine
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Addiction Medicine
                          </a>
                          <span class="article-count">
                           (218)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Allergy and Immunology
                          </a>
                          <span class="article-count">
                           (496)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">
                           Anesthesia
                          </a>
                          <span class="article-count">
                           (106)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Cardiovascular Medicine
                          </a>
                          <span class="article-count">
                           (1121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Dentistry and Oral Medicine
                          </a>
                          <span class="article-count">
                           (198)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">
                           Dermatology
                          </a>
                          <span class="article-count">
                           (141)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Emergency Medicine
                          </a>
                          <span class="article-count">
                           (275)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">
                           Endocrinology (including Diabetes Mellitus and Metabolic Disease)
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (9829)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Forensic Medicine
                          </a>
                          <span class="article-count">
                           (5)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">
                           Gastroenterology
                          </a>
                          <span class="article-count">
                           (484)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Genetic and Genomic Medicine
                          </a>
                          <span class="article-count">
                           (2344)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Geriatric Medicine
                          </a>
                          <span class="article-count">
                           (225)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Economics
                          </a>
                          <span class="article-count">
                           (465)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Informatics
                          </a>
                          <span class="article-count">
                           (1574)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">
                           Health Policy
                          </a>
                          <span class="article-count">
                           (738)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">
                           Health Systems and Quality Improvement
                          </a>
                          <span class="article-count">
                           (609)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">
                           Hematology
                          </a>
                          <span class="article-count">
                           (239)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">
                           HIV/AIDS
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">
                           Infectious Diseases (except HIV/AIDS)
                          </a>
                          <span class="article-count">
                           (11686)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Intensive Care and Critical Care Medicine
                          </a>
                          <span class="article-count">
                           (617)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Education
                          </a>
                          <span class="article-count">
                           (242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Ethics
                          </a>
                          <span class="article-count">
                           (67)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">
                           Nephrology
                          </a>
                          <span class="article-count">
                           (259)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">
                           Neurology
                          </a>
                          <span class="article-count">
                           (2170)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">
                           Nursing
                          </a>
                          <span class="article-count">
                           (134)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">
                           Nutrition
                          </a>
                          <span class="article-count">
                           (340)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">
                           Obstetrics and Gynecology
                          </a>
                          <span class="article-count">
                           (427)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">
                           Occupational and Environmental Health
                          </a>
                          <span class="article-count">
                           (521)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">
                           Oncology
                          </a>
                          <span class="article-count">
                           (1188)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">
                           Ophthalmology
                          </a>
                          <span class="article-count">
                           (368)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">
                           Orthopedics
                          </a>
                          <span class="article-count">
                           (129)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">
                           Otolaryngology
                          </a>
                          <span class="article-count">
                           (221)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Pain Medicine
                          </a>
                          <span class="article-count">
                           (148)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Palliative Medicine
                          </a>
                          <span class="article-count">
                           (50)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (315)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">
                           Pediatrics
                          </a>
                          <span class="article-count">
                           (702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Therapeutics
                          </a>
                          <span class="article-count">
                           (304)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">
                           Primary Care Research
                          </a>
                          <span class="article-count">
                           (269)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">
                           Psychiatry and Clinical Psychology
                          </a>
                          <span class="article-count">
                           (2201)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">
                           Public and Global Health
                          </a>
                          <span class="article-count">
                           (4705)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">
                           Radiology and Imaging
                          </a>
                          <span class="article-count">
                           (786)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">
                           Rehabilitation Medicine and Physical Therapy
                          </a>
                          <span class="article-count">
                           (462)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Respiratory Medicine
                          </a>
                          <span class="article-count">
                           (626)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">
                           Rheumatology
                          </a>
                          <span class="article-count">
                           (276)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">
                           Sexual and Reproductive Health
                          </a>
                          <span class="article-count">
                           (227)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Sports Medicine
                          </a>
                          <span class="article-count">
                           (214)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">
                           Surgery
                          </a>
                          <span class="article-count">
                           (254)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Toxicology
                          </a>
                          <span class="article-count">
                           (43)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">
                           Transplantation
                          </a>
                          <span class="article-count">
                           (121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">
                           Urology
                          </a>
                          <span class="article-count">
                           (94)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
