<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.05.11.078535v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.05.11.078535v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.05.11.078535v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/05/06/2020.05.11.078535.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.05.11.078535v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks"/>
  <meta name="DC.Identifier" content="10.1101/2020.05.11.078535"/>
  <meta name="DC.Date" content="2022-05-06"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Decoding the levels of consciousness from cortical activity recording is a major challenge in neuroscience. Using clustering algorithms, we previously demonstrated that resting-state functional MRI (rsfMRI) data can be split into several clusters also called “brain states” corresponding to “functional configurations” of the brain. Here, we propose to use a selfsupervised machine learning method based on artificial neural networks to predict functional brain states across levels of consciousness from rsfMRI. The Functional Connectivity (FC) matrices reflect the brain-state dynamic at a given time. Because it is key to consider the FC topologies, a specific graph-Convolutional Neural Network (gCNN), namely BrainNetCNN, is considered to predict the brain states in awake and anesthetized nonhuman primates. To avoid the circularity that remains in the training stage, where the target is composed of pseudo-labels, recent self-supervised techniques are implemented. Using a linear probe for the prediction, the network achieves a prediction accuracy consistent with state-of-the-art methods lying in [0.655, 0.759] depending on the experimental settings. To put forward the interest of such a representation, the transition probabilities and the set of connections found to be important for predicting a brain state are computed. This latter is directly linked with the level of consciousness. The results demonstrate that deep learning methods are not only able to predict brain states but also provide additional insight into cortical signatures of consciousness with potential clinical consequences for the monitoring of anesthesia and the diagnosis of disorders of consciousness.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Antoine Grigis"/>
  <meta name="DC.Contributor" content="Chloé Gomez"/>
  <meta name="DC.Contributor" content="Jordy Tasserie"/>
  <meta name="DC.Contributor" content="Corentin Ambroise"/>
  <meta name="DC.Contributor" content="Vincent Frouin"/>
  <meta name="DC.Contributor" content="Béchir Jarraya"/>
  <meta name="DC.Contributor" content="Lynn Uhrig"/>
  <meta name="article:published_time" content="2022-05-06"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Decoding the levels of consciousness from cortical activity recording is a major challenge in neuroscience. Using clustering algorithms, we previously demonstrated that resting-state functional MRI (rsfMRI) data can be split into several clusters also called “brain states” corresponding to “functional configurations” of the brain. Here, we propose to use a selfsupervised machine learning method based on artificial neural networks to predict functional brain states across levels of consciousness from rsfMRI. The Functional Connectivity (FC) matrices reflect the brain-state dynamic at a given time. Because it is key to consider the FC topologies, a specific graph-Convolutional Neural Network (gCNN), namely BrainNetCNN, is considered to predict the brain states in awake and anesthetized nonhuman primates. To avoid the circularity that remains in the training stage, where the target is composed of pseudo-labels, recent self-supervised techniques are implemented. Using a linear probe for the prediction, the network achieves a prediction accuracy consistent with state-of-the-art methods lying in [0.655, 0.759] depending on the experimental settings. To put forward the interest of such a representation, the transition probabilities and the set of connections found to be important for predicting a brain state are computed. This latter is directly linked with the level of consciousness. The results demonstrate that deep learning methods are not only able to predict brain states but also provide additional insight into cortical signatures of consciousness with potential clinical consequences for the monitoring of anesthesia and the diagnosis of disorders of consciousness.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.05.11.078535v2"/>
  <meta name="citation_id" content="2020.05.11.078535v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.05.11.078535v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.11.078535v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.11.078535v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.05.11.078535"/>
  <meta name="citation_num_pages" content="20"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.05.11.078535"/>
  <meta name="citation_author" content="Antoine Grigis"/>
  <meta name="citation_author_institution" content="Université Paris-Saclay, CEA, NeuroSpin"/>
  <meta name="citation_author_email" content="antoine.grigis@cea.fr"/>
  <meta name="citation_author" content="Chloé Gomez"/>
  <meta name="citation_author_institution" content="Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992"/>
  <meta name="citation_author" content="Jordy Tasserie"/>
  <meta name="citation_author_institution" content="Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992"/>
  <meta name="citation_author" content="Corentin Ambroise"/>
  <meta name="citation_author_institution" content="Université Paris-Saclay, CEA, NeuroSpin"/>
  <meta name="citation_author" content="Vincent Frouin"/>
  <meta name="citation_author_institution" content="Université Paris-Saclay, CEA, NeuroSpin"/>
  <meta name="citation_author" content="Béchir Jarraya"/>
  <meta name="citation_author_institution" content="Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992"/>
  <meta name="citation_author_institution" content="University of Versailles, Université Paris-Saclay, Neurosurgery Department, Foch Hospital"/>
  <meta name="citation_author" content="Lynn Uhrig"/>
  <meta name="citation_author_institution" content="Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992"/>
  <meta name="citation_reference" content="T. Caliński and J. Harabasz. “A dendrite method for cluster analysis”. In: Communications in Statistics 3.1 (1974), pp. 1–27. doi: 10.1080/03610927408827101."/>
  <meta name="citation_reference" content="Peter Rousseeuw. “Silhouettes: A Graphical Aid to the Interpretation and Validation of Cluster Analysis”. In: Journal of Computational and Applied Mathematics 20 (1987), pp. 53–65. doi: 10.1016/0377-0427(87)90125-7."/>
  <meta name="citation_reference" content="Sotiris Kotsiantis. “Supervised Machine Learning: A Review of Classification Techniques”. In: Informatica 31 (Oct. 2007)."/>
  <meta name="citation_reference" content="Rembrandt Bakker, Thomas Wachtler, and Markus Diesmann. “Co-CoMac 2.0 and the future of tract-tracing databases”. In: Frontiers in Neuroinformatics 6 (2012), p. 30. issn: 1662-5196. doi: 10.3389/fninf.2012.00030."/>
  <meta name="citation_reference" content="Trupti M. Kodinariya and Prashant R. Makwana. “Review on determining number of Cluster in K-Means Clustering”. In: Applied Mathematics and Information Sciences 10 (2013). doi: 10.18576/amis/100428."/>
  <meta name="citation_reference" content="Lynn Uhrig, Stanislas Dehaene, and Béchir Jarraya. “A Hierarchy of Responses to Auditory Regularities in the Macaque Brain”. In: Journal of Neuroscience 34.4 (2014), pp. 1127–1132. issn: 0270-6474. doi: 10.1523/JNEUROSCI.3165-13.2014."/>
  <meta name="citation_reference" content="Pablo Barttfeld et al. “Signature of consciousness in the dynamics of resting-state brain activity”. In: Proceedings of the National Academy of Sciences 112.3 (2015), pp. 887–892. issn: 0027-8424. doi: 10.1073/pnas.1418031112."/>
  <meta name="citation_reference" content="Etienne Combrisson and Karim Jerbi. “Exceeding chance level by chance: The caveat of theoretical chance levels in brain signal classification and statistical assessment of decoding accuracy”. In: Journal of Neuroscience Methods 250 (2015), pp. 126–136. issn: 0165-0270. doi: 10.1016/j.jneumeth.2015.01.010."/>
  <meta name="citation_reference" content="Jeremy Kawahara et al. “BrainNetCNN: Convolutional neural networks for brain networks; towards predicting neurodevelopment”. In: NeuroImage 146 (2017), pp. 1038–1049. issn: 1053-8119. doi: 10.1016/j.neuroimage.2016.09.046."/>
  <meta name="citation_reference" content="Mathilde Caron et al. “Deep Clustering for Unsupervised Learning of Visual Features”. In: ECCV. 2018, pp. 139–156."/>
  <meta name="citation_reference" content="Lynn Uhrig et al. “Resting-state Dynamics as a Cortical Signature of Anesthesia in Monkeys”. In: Anesthesiology 129.5 (Nov. 2018), pp. 942–958. issn: 0003-3022. doi: 10.1097/ALN.0000000000002336."/>
  <meta name="citation_reference" content="Mathilde Caron et al. “Unsupervised Learning of Visual Features by Contrasting Cluster Assignments”. en. In: NeurIPS 33 (2020), pp. 9912–9924."/>
  <meta name="citation_reference" content="Ting Chen et al. “A Simple Framework for Contrastive Learning of Visual Representations”. In: International Conference on Machine Learning. ISSN: 2640-3498. PMLR, Nov. 2020, pp. 1597–1607."/>
  <meta name="citation_reference" content="“Conscious Processing and the Global Neuronal Workspace Hypothesis”. In: Neuron 105.5 (2020), pp. 776–798. issn: 0896-6273. doi: 10.1016/j.neuron.2020.01.026."/>
  <meta name="citation_reference" content="“Pypreclin: An automatic pipeline for macaque functional MRI preprocessing”. In: NeuroImage 207 (2020), p. 116353. issn: 1053-8119. doi: 10.1016/j.neuroimage.2019.116353."/>
  <meta name="twitter:title" content="Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Decoding the levels of consciousness from cortical activity recording is a major challenge in neuroscience. Using clustering algorithms, we previously demonstrated that resting-state functional MRI (rsfMRI) data can be split into several clusters also called “brain states” corresponding to “functional configurations” of the brain. Here, we propose to use a selfsupervised machine learning method based on artificial neural networks to predict functional brain states across levels of consciousness from rsfMRI. The Functional Connectivity (FC) matrices reflect the brain-state dynamic at a given time. Because it is key to consider the FC topologies, a specific graph-Convolutional Neural Network (gCNN), namely BrainNetCNN, is considered to predict the brain states in awake and anesthetized nonhuman primates. To avoid the circularity that remains in the training stage, where the target is composed of pseudo-labels, recent self-supervised techniques are implemented. Using a linear probe for the prediction, the network achieves a prediction accuracy consistent with state-of-the-art methods lying in [0.655, 0.759] depending on the experimental settings. To put forward the interest of such a representation, the transition probabilities and the set of connections found to be important for predicting a brain state are computed. This latter is directly linked with the level of consciousness. The results demonstrate that deep learning methods are not only able to predict brain states but also provide additional insight into cortical signatures of consciousness with potential clinical consequences for the monitoring of anesthesia and the diagnosis of disorders of consciousness.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.05.11.078535v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Decoding the levels of consciousness from cortical activity recording is a major challenge in neuroscience. Using clustering algorithms, we previously demonstrated that resting-state functional MRI (rsfMRI) data can be split into several clusters also called “brain states” corresponding to “functional configurations” of the brain. Here, we propose to use a selfsupervised machine learning method based on artificial neural networks to predict functional brain states across levels of consciousness from rsfMRI. The Functional Connectivity (FC) matrices reflect the brain-state dynamic at a given time. Because it is key to consider the FC topologies, a specific graph-Convolutional Neural Network (gCNN), namely BrainNetCNN, is considered to predict the brain states in awake and anesthetized nonhuman primates. To avoid the circularity that remains in the training stage, where the target is composed of pseudo-labels, recent self-supervised techniques are implemented. Using a linear probe for the prediction, the network achieves a prediction accuracy consistent with state-of-the-art methods lying in [0.655, 0.759] depending on the experimental settings. To put forward the interest of such a representation, the transition probabilities and the set of connections found to be important for predicting a brain state are computed. This latter is directly linked with the level of consciousness. The results demonstrate that deep learning methods are not only able to predict brain states but also provide additional insight into cortical signatures of consciousness with potential clinical consequences for the monitoring of anesthesia and the diagnosis of disorders of consciousness.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-05-06"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.05.11.078535v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2530823"/>
  <title>
   Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9_GkyhccWs7fny8JziNwhT3gm6ys9RdqUgYclDEG9EY"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2530823","apath":"\/biorxiv\/early\/2022\/05\/06\/2020.05.11.078535.atom","pisa":"biorxiv;2020.05.11.078535v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.05.11.078535v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.05.11.078535v2","title":"Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks","identifier":"node\/2530823"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZEMLLghQBs1tuO6VUN8j47Mb8UbzEHF6+k\\\\\\\/pFSx8G5aWuXUFbfHkBO09Ulufez\\\\\\\/k+DnMn4+wO\\\\\\\/3ZVsJUL9qC1qiN3D86utDsfeNAtBgj2IF3PrzU57b7SUCjcrbhI4ES33D4HVvIDJ+iqsPeqvYYtjk\\\\\\\/4ipwr\\\\\\\/XdsAXoTk7jAGB\\\\\\\/zcqua4b\\\\\\\/YJUaRJZXhNzfP5SghaOHf\\\\\\\/tOWBF88EkXlwhJw6vZWQRcUi62mg04qdZ+hUic\\\\\\\/KbZ2qqB3YF1AzP0uklWfPK3ZUzIKkhI34860K2JMEvwZCB8vv7qqarC9kFI2JfRQG4Wj62ujcGQn3H1aJlWmXSRw4b\\\\\\\/JwIp6DIDOFKvM+qPbm4rQ6mlJx3UIT8laERkQtoDbOGYAEj05cKpNx3WWi5TxU+4wbeEtGPe4byDG\\\\\\\/bOp3sTnWhbJ2q1dbw9p7AK54PUFDws4hf41WczCrfLD4DelWgV6kEaXSWy7YF383eFdqj0tjuj3LY7t4QMPtY\\\\\\\/tASZQM8vVDtHiQBEg2092bo7hPfZPO5uLRwDPjKhzffnmLF8Sr7h9MFs2JexLGEMBoqwppe5rWDTFyF18c6wEtXRtporlN\\\\\\\/3F2iLWNUvNU6td79NMENkx6yF6o84h7T4zGCC39DhXzsPbj6QRyYawrDwwAjEDt4ym+pkpc4OAG5HfNDfHBj99H0idGyCcI+a6Q0C7o4oBqfQCs6KDJk4v\\\\\\\/JMkCfM3av\\\\\\\/5+Vf\\\\\\\/w9rbeUKp3RymQfSqfYoxaUvD4gpf0Mce\\\\\\\/9eH2+eIIYKaMOFQdyu9LLcRKXNmHSmxTnrvvSpynDOqJihsAGj5THJxDduELBlNJK+iHcDokXbJ0Ozwara9ekViWNlG0xsKVwCLtJEYo+zk1NQh6vTA77ywhb2s+Vsi9C7p7FGxAtZrZ9Exsi1fNm3X6Ngf12JaOeNd3pqLXI50hVRfkhrmWxjldKPoQTwPUdnQqGF4XHlDmhkFHbEMdx3uWgfXTceCjDCcetLpftUBz56qeKGwz9fnCrTf7Ex9H5ePw+9wzs54Q0rTnO5lacwiZsVEHxHciE1ruaxZzfdWUMMvnNA5sp9qMbuDwPZwl4B1cn\\\\\\\/vi7zUaKMaoxp9irlcF2s2Yvo8MnbcWslVDna4VRL4YS7Jkx5s\\\\\\\/tT5HhCh5NfSMCxxGyPTzviv6U5IFq+yvk3+vtyaRXGurLAgZ5N2t2DvC7s8IqZ1p4n53NxMKT9goCQdoZnGo451YlWi7WPyFIX6I1VDjkirC0GIqxXh8YA0HsECPaTw1I26sa8h3D4ATq7V5R5hDXvXIAuX1wHt8u8pJRbWRQIMQe6EubRQkcguQfV0+WTtJzYnylqirMtMB4u\\\\\\\/eLGRRZqSsAaSwRIEuWz+lF+iXsywvr8biLgmG+zxLoRG0U=\\\u0022,\\\u0022iv\\\u0022:\\\u0022q3Jddj0K3uR1RK7bIieJ3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022159159b2f6eeee6cf1f337b85f25e01b6333b3395909cc99a261ffc1750c21d3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.05.11.078535v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2530823 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.05.11.078535v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_527281912">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_527281912" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_790187954" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-WX16vwYhHtF84TCF3eJ6Q_v4EeAGmwp2iW5TFKyG9lg"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2530823" data-node-nid="2530823" id="node-2530823--21123056381" data-pisa="biorxiv;2020.05.11.078535v2" data-pisa-master="biorxiv;2020.05.11.078535" data-apath="/biorxiv/early/2022/05/06/2020.05.11.078535.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Antoine
                      </span>
                      <span class="nlm-surname">
                       Grigis
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Chloé
                      </span>
                      <span class="nlm-surname">
                       Gomez
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Jordy
                      </span>
                      <span class="nlm-surname">
                       Tasserie
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Corentin
                      </span>
                      <span class="nlm-surname">
                       Ambroise
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Vincent
                      </span>
                      <span class="nlm-surname">
                       Frouin
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Béchir
                      </span>
                      <span class="nlm-surname">
                       Jarraya
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Lynn
                      </span>
                      <span class="nlm-surname">
                       Uhrig
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.05.11.078535
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2530823--21123056381" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Antoine Grigis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Université Paris-Saclay, CEA, NeuroSpin
                       </span>
                       , 91191, Gif-sur-Yvette,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Antoine%2BGrigis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Grigis%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAntoine%2BGrigis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:antoine.grigis@cea.fr" class="" data-icon-position="" data-hide-link-title="0">
                        antoine.grigis@cea.fr
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Chloé Gomez
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992
                       </span>
                       , Gif-sur-Yvette,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Chlo%C3%A9%2BGomez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gomez%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AChlo%25C3%25A9%2BGomez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Jordy Tasserie
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992
                       </span>
                       , Gif-sur-Yvette,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jordy%2BTasserie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Tasserie%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJordy%2BTasserie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Corentin Ambroise
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Université Paris-Saclay, CEA, NeuroSpin
                       </span>
                       , 91191, Gif-sur-Yvette,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Corentin%2BAmbroise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ambroise%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ACorentin%2BAmbroise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Vincent Frouin
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Université Paris-Saclay, CEA, NeuroSpin
                       </span>
                       , 91191, Gif-sur-Yvette,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Vincent%2BFrouin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Frouin%20V&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AVincent%2BFrouin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Béchir Jarraya
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992
                       </span>
                       , Gif-sur-Yvette,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        University of Versailles, Université Paris-Saclay, Neurosurgery Department, Foch Hospital
                       </span>
                       , Suresnes,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=B%C3%A9chir%2BJarraya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Jarraya%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AB%25C3%25A9chir%2BJarraya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Lynn Uhrig
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Cognitive Neuroimaging Unit, Institut National de la Santé et de la Recherche Médicale U992
                       </span>
                       , Gif-sur-Yvette,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Lynn%2BUhrig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Uhrig%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ALynn%2BUhrig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.05.11.078535v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2530823" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2530823/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.11.078535v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2530823" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2530823/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.11.078535v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2530823" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2530823/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.11.078535v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2530823" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2530823/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.05.11.078535v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2530823" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2530823/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Decoding the levels of consciousness from cortical activity recording is a major challenge in neuroscience. Using clustering algorithms, we previously demonstrated that resting-state functional MRI (rsfMRI) data can be split into several clusters also called “brain states” corresponding to “functional configurations” of the brain. Here, we propose to use a selfsupervised machine learning method based on artificial neural networks to predict functional brain states across levels of consciousness from rsfMRI. The Functional Connectivity (FC) matrices reflect the brain-state dynamic at a given time. Because it is key to consider the FC topologies, a specific graph-Convolutional Neural Network (gCNN), namely BrainNetCNN, is considered to predict the brain states in awake and anesthetized nonhuman primates. To avoid the circularity that remains in the training stage, where the target is composed of pseudo-labels, recent self-supervised techniques are implemented. Using a linear probe for the prediction, the network achieves a prediction accuracy consistent with state-of-the-art methods lying in [0.655, 0.759] depending on the experimental settings. To put forward the interest of such a representation, the transition probabilities and the set of connections found to be important for predicting a brain state are computed. This latter is directly linked with the level of consciousness. The results demonstrate that deep learning methods are not only able to predict brain states but also provide additional insight into cortical signatures of consciousness with potential clinical consequences for the monitoring of anesthesia and the diagnosis of disorders of consciousness.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-4">
                             A challenge in fundamental and clinical neuroscience is to predict the level of consciousness from brain activity recordings such as EEG or functional MRI (fMRI). Decoding levels of consciousness from the neural activity is essential for understanding the mechanism by which anesthetics induce consciousness loss, objectively monitoring the depth of anesthesia, and helping diagnose disorders of consciousness with higher accuracy [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ]. Instead of a bottom-up investigation starting from molecules and cells, an information-processing analysis of brain functional imaging studies during the awake state or loss of consciousness induced by various anesthetics was proposed in non-human primates using dynamical resting-state [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ]. The resting-state fMRI (rsfMRI) data were divided into several clusters, also called “brain states”, corresponding to “functional configurations” of the brain and allowing decomposition of the consciousness conditions of a subject in terms of functional patterns. Briefly, a set of k Brain States {BS}
                             <sup>
                              k
                             </sup>
                             , corresponding to functional configurations, is obtained from nonhuman primates rsfMRI in the awake state and under different anesthetics. Then, each {BS}
                             <sup>
                              k
                             </sup>
                             is ranked according to its similarity with the anatomical connectivity matrix defined in [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. In [
                             <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ] the authors demonstrated that dynamic long-range functional correlation matrices reduce to the underlying structural anatomical connectivity matrix during anesthesia (
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Fig. 1
                             </a>
                             ). In other words, the brain states {BS}
                             <sup>
                              k
                             </sup>
                             are directly related to the level of consciousness.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F1.large.jpg?width=800&height=600&carousel=1" title="Probability distributions to observe in a rsfMRI time window, one of the ranked brain states {BS}k for the awake and anesthesia conditions, and k = 4 or k = 7. Each bar represents the within-condition probability of occurrence of a state." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Probability distributions to observe in a rsfMRI time window, one of the ranked brain states {<em>BS</em>}<sup><em>k</em></sup> for the awake and anesthesia conditions, and <em>k</em> = 4 or <em>k</em> = 7. Each bar represents the within-condition probability of occurrence of a state.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F1.medium.gif" width="329" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F1.medium.gif" width="329" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1:
                              </span>
                              <p id="p-5" class="first-child">
                               Probability distributions to observe in a rsfMRI time window, one of the ranked brain states {BS}
                               <sup>
                                k
                               </sup>
                               for the awake and anesthesia conditions, and k = 4 or k = 7. Each bar represents the within-condition probability of occurrence of a state.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-6">
                             The methodology described above includes the entire dataset for the extraction and annotation of {BS}
                             <sup>
                              k
                             </sup>
                             . Although the set of {BS}
                             <sup>
                              k
                             </sup>
                             is robust and stays stable when a new rsfMRI sequence is added to the database, the analysis process resumes the whole clustering, with potentially an appropriate initialization. Leveraging the increasing number of rsfMRI runs available, the objectives of this work are three-fold and will not require recomputing the {BS}
                             <sup>
                              k
                             </sup>
                             set. First, we aim to confirm the interest of {BS}
                             <sup>
                              k
                             </sup>
                             as labels to characterize the succession of states occurring under a given arousal condition. Using self-supervised machine learning methods, we will train a classifier that can predict one brain state from small rsfMRI temporal windows in completely new samples/runs [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. In this work, we will use a specific graph-Convolutional Neural Network (gCNN) classifier named BrainNetCNN [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. Standard CNNs can’t handle the spatial neighborhood of the Region Of Interests (ROIs) used to build the functional connectivity (FC) matrices. BrainNetCNN is composed of specific edge-to-edge, edge-to-node, and node-to-graph convolutional filters. These filters better reflect the topological locality of the ROIs used to generate the FC matrices. Another point in the training stage is to minimize the circularity between the targets composed of pseudo-labels and the predicted probabilities. For this purpose, recent contrastive learning techniques are implemented [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ]. Second, we will challenge the data and learning process to model the brain dynamic oscillating from one state to another. Finally, we will leverage the embedded automatic differentiation of the deep learning framework to uncover which connections were used by the classifying network to be predictive of brain states. These predictive connections maps, which are duals of the {BS}
                             <sup>
                              k
                             </sup>
                             , yield some insights into both the brain states and the cortical signature of consciousness and can be considered (to some extent) as proxies of cortical signatures of consciousness.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 Related works
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Unsupervised clustering
                             </h3>
                             <p id="p-7">
                              The objective of unsupervised clustering methods is to determine an internal grouping in a set of unlabeled data. In machine learning, many strategies can solve this task. Proposed methods can be grouped into four categories: 1) exclusive clustering, such as k-means, where each sample is assigned to one group, 2) overlapping clustering, such as fuzzy k-means, where each sample belongs to two or more clusters with different degrees of membership, 3) hierarchical clustering based on the fusion of nearest clusters starting from clusters containing only one sample, and 4) probabilistic clustering, such as mixtures of Gaussians, where each cluster is represented by a parametric distribution. These clustering machine learning techniques have been successfully applied in diverse applications such as the computation of the BS
                              <sup>
                               k
                              </sup>
                              set [
                              <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ].
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Deep self-supervised clustering
                             </h3>
                             <p id="p-8">
                              Recent works propose to uncover semantically relevant groups of samples using self-supervised deep learning techniques [
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ,
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ]. Notably, Deep Cluster [
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ] alternates between a clustering phase and a back-propagation through a classification encoder driven by pseudo-labels. More specifically, at each epoch, previous clustering assignments produced by k-means on the generated embeddings are used as pseudo-labels for minimizing the cross-entropy. Such self-supervised techniques have proven to produce semantically valid clusters with no manual annotation. However, the training of these models is paved with pitfalls. The switch between the generation of pseudo-labels and the network training leads to clustering degeneracy and inconsistency. While the degeneracy issue is often solved with heuristics (which reassigns out-of-bounds samples/clusters), the inconsistency issue highly disrupts the training. For instance, in Deep Cluster [
                              <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ], the clustering head is re-initialized at each epoch because k-means inevitably shuffles the assigned labels. In ODC [
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ], representations from a momentum encoder are stored, leading to a computationally costly nearest centroid problem at each batch. Such a training scheme is also sensitive to local minima because of the circular entanglement with the pseudo-labels.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Deep contrastive learning
                             </h3>
                             <p id="p-9">
                              Another branch of the literature explores contrastive methods [
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ,
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ,
                              <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. Distinctly, in SwAV [
                              <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ], prototypes/clusters are discovered while enforcing consistency between cluster assignments of contrasted samples. Generic contrastive methods can also tackle the downstream clustering problem by applying a linear classifier on top of the fixed learned representation. For example, in SimCLR [
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ], input-distortion invariance is used as a pretext task to learn an appropriate representation of the data. Specifically, it encourages two augmented samples to be close in the representation space. An alternative option, computationally efficient and stable, has been proposed in Barlow Twins [
                              <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. The learned representation is discovered by estimating the empirical cross-correlation matrix from contrasted samples. From a classification point of view, no constraint on class collision is enforced, thus allowing several samples sharing similar semantic content to be pulled apart. However, in practice, this problem is limited, and Barlow Twins produces clustering results as good or better than SwAV and SimCLR on ImageNet, Places-205, VOC07, and iNat18 [
                              <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ].
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-6">
                            <h2 class="">
                             3 Material and Methods
                            </h2>
                            <div id="sec-7" class="subsection">
                             <h3>
                              3.1 Material
                             </h3>
                             <p id="p-10">
                              This study uses the dataset described in [
                              <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ]. Data were acquired either in the awake state or under different anesthetic conditions (ketamine, propofol, or sevoflurane anesthesia) in five macaque monkeys. Anesthesia levels were defined by a clinical score (the monkey sedation scale) and continuous electroencephalography monitoring. 156 rsfMRI runs with corresponding structural MRI were acquired on a Siemens 3-Tesla with a customized single transmit-receiver surface coil. The spatial prepossessing was performed with the NSM pipeline described in [
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ,
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ], with specific time-series denoising operations [
                              <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ,
                              <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ]. Specifically, voxel time series were filtered with a low-pass (0.05-Hz cutoff) and high-pass (0.0025-Hz cutoff) filters and a zero-phase fast-Fourier notch filter (0.03 Hz) to remove a pure artefactual frequency present in all the data. The variance of each time series was normalized to transform covariance into correlation matrices. Sliding windows are used to compute dynamic FC matrices reflecting the brain state at a given time [
                              <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ]. Windowed segments of each time series were computed with a rectangular window (84 s width, which corresponds to 35 scans of 2.4 s each) and a sliding step of one scan, resulting in 464 windows per run. From the windowed rsfMRI, dynamic FC matrices were derived using the CoCoMac template composed of 82 cortical regions (41 cortical regions within each hemisphere) [
                              <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ,
                              <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ]. The windowed rsfMRI time series from each region was converted into a connectivity matrix by projection into tangent space, capturing connectivity aspects from the correlation and partial correlation. The corresponding dynamic connectivity matrices led to a data array of shape 156 × 464 × 82 × 82 which was Fisher transformed to form a set of 156 × 464 samples. Each sample is more than a list of independent numbers and encodes the functional connectivity patterns. Dominant recurrent patterns of brain correlations (the {BS}
                              <sup>
                               k
                              </sup>
                              set) were determined using an unsupervised k-means clustering with k states (or classes), k in [
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              ,
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ] as proposed in [
                              <a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ]. To avoid double-dipping (ie. the use of the same dataset for the training and the testing), the k-means was performed on a train set defined as a subset of the data. The labels associated with the remaining data were recovered by selecting the closest brain state {BS}
                              <sup>
                               k
                              </sup>
                              forming the test set. To this end, the correlation coefficients between the test set dynamic FC matrices and each {BS}
                              <sup>
                               k
                              </sup>
                              were computed, leading to a label array of shape 156 × 464 that will form the empirical ground-truth dataset.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              3.2 Data partitioning
                             </h3>
                             <p id="p-11">
                              The dataset contains 5 monkeys with 156 rsfMRI runs in different arousal conditions (31 runs for the awake state, 25 runs for ketamine anesthesia, 25 runs for moderate propofol sedation, 30 runs for deep propofol anesthesia, 25 runs for moderate sevoflurane sedation, and 20 runs for deep sevoflurane anesthesia). We propose to leave one monkey out as the test set (35 runs) and the four others as the train set (121 runs). We further checked that both train and test sets have matching proportions of arousal conditions and brain state labels. The test set was set aside and not used during training. The train set was further split into validation and training folds using a 3-folds stratified cross-validation scheme.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              3.3 Model architecture
                             </h3>
                             <p id="p-12">
                              The BrainNetCNN [
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ] works specifically with network data and will enforce the FC patterns. The model architecture is composed of convolutional layers followed by fully connected layers. Among the proposed configurations, we selected the E2Enet-sml, which consists in removing one edge-to-edge layer and two of the fully connected layers (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). This configuration has shown excellent performances with a restricted number of parameters. Precisely, E2Enet-sml has an edge-to-edge layer composed of 32 1 × 82 and 32 82 × 1 filters producing feature maps of size 32 × 82 × 82, followed by an edge-to-node layer with 64 1 × 82 × 32 filters yielding feature maps of size 64 × 82 × 1, a node-to-graph layer with feature maps of size 1 × 1 × 30, and a fully connected layer with an output of size k. Increasing the number of feature maps with each layer is a common strategy for CNNs to compensate for the reductions along the other dimensions. Every layer uses very leaky rectified linear units as an activation function with a negative slope of 1/3.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F2.large.jpg?width=800&height=600&carousel=1" title="Schematic representation of the proposed training. The BrainNetCNN E2Enet-sml layers are represented in blue. In the first row the Barlow Twin contrastive approach is described. The fully connected layer of the BrainNetCNN is replaced by a projection head (in orange), and empirical cross-correlation matrix (in green) is computed from augmented/distorted samples (in yellow). In the second row the connected layer of the BrainNetCNN is trained using the k-means pseudo-labels by fixing the other weights." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup">Schematic representation of the proposed training. The BrainNetCNN E2Enet-sml layers are represented in blue. In the first row the Barlow Twin contrastive approach is described. The fully connected layer of the BrainNetCNN is replaced by a projection head (in orange), and empirical cross-correlation matrix (in green) is computed from augmented/distorted samples (in yellow). In the second row the connected layer of the BrainNetCNN is trained using the k-means pseudo-labels by fixing the other weights.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F2.medium.gif" width="414" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F2.medium.gif" width="414" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <p id="p-13" class="first-child">
                                Schematic representation of the proposed training. The BrainNetCNN E2Enet-sml layers are represented in blue. In the first row the Barlow Twin contrastive approach is described. The fully connected layer of the BrainNetCNN is replaced by a projection head (in orange), and empirical cross-correlation matrix (in green) is computed from augmented/distorted samples (in yellow). In the second row the connected layer of the BrainNetCNN is trained using the k-means pseudo-labels by fixing the other weights.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-14">
                              Another objective is to minimize the impact of the pseudo-labels derived from the k-means during the training. This circularity issue is problematic, and without particular care, the weights of the CNN may more reflect the clustering algorithm than the variability of the data. To circumvent this limitation, we propose to use self-supervised contrastive learning techniques, more specifically, the Barlow Twins strategy [
                              <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. The BrainNetCNN fully connected layer is replaced by a two hidden layers projection head leading to an embeddings space of size 120. The objective function tries to make the empirical cross-correlation matrix computed from twin embeddings (ie., outputs of the network fed with augmented/distorted versions of a sample) as close to the identity matrix as possible (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). The FC matrices are distorted using random connections erasing scheme. The proportion of zeros is chosen randomly in the interval [0.3, 0.7]. The pseudo-labels from the k-means are now only used to train a linear classifier on the fixed representations learned by the Barlow Twins. All other weights are frozen to minimize the circularity between the targets and the predicted probabilities.
                             </p>
                             <p id="p-15">
                              For training, we employed a dropout of 0.5 before the node-to-graph layer as shown in
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              , and we follow the optimization protocol described in [
                              <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. We use a LARS optimizer, a mini-batch of size 1024, a weight decay of 1e-6, a learning rate of 0.2 for the weights and 0.0048 for the biases and batch normalization parameters, and train the model for 1000 epochs. We use a learning rate warm-up period of 10 epochs, after which we reduce the learning rate by a factor of 1000 with a cosine decay schedule. The biases and batch normalization parameters are excluded from LARS adaptation and weight decay. The linear classifier is trained from the fixed representation, optimizing the cross-entropy loss during 100 epochs with an SGD optimizer, a weight decay of 1e-6, a learning rate of 0.01, and a cosine annealing schedule.
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              3.4 Maps of Predictive Connections
                             </h3>
                             <p id="p-16">
                              We use saliency maps to uncover the connections learned by the BrainNetCNN to be predictive of consciousness. Saliency maps are a popular visualization tool for gaining insight into why a deep learning model made an individual decision, such as classifying an image or a FC matrix [
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ,
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                              ]. From a single backpropagation, the gradients of the target class w.r.t. the input FC matrix are generated from the first convolution layer. These gradients are used to visualize an FC-specific class saliency map, giving some intuition on connections within the input that contribute the most and least to the corresponding prediction. The badly predicted FC matrices are discarded, and resulting saliency maps are averaged brain-states-wise. In our application, they are helpful to extract proxies of the cortical signature of consciousness. These consciousness markers are rendered using a circular graph layout. The 41 cortical regions within each hemisphere are grouped, and the top p = 15% largest positive and negative connections are displayed. This threshold is computed at the brain-state level or across all brain states.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             4 Results
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              4.1 Optimal number of the brain states
                             </h3>
                             <p id="p-17">
                              Before the presentation of the results on the three objectives of our work, we detail our sound estimation of the optimal number of brain states in the k-means. The basic idea behind the k-means consists in defining k clusters such that the within-cluster variations are minimum. Different metrics can be employed to select the optimal number of brain states among which we have retained the Elbow [
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ], the Silhouette [
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ,
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              ], and the Calinski and Harabasz [
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              ] scores. The Elbow value is defined as the Within-Cluster Sum of Squared (WCSS) error for different values of k. The optimal k corresponds to the point of inflection on the WCSS versus k curve. Note that the Elbow score is more a decision rule than a metric. The Silhouette value measures how similar a sample is to its cluster compared to others and lies in [-1, 1]. The Calinski and Harabasz value is defined as the ratio between the within-cluster and the between-cluster dispersions. The last two scores are maximized with k. From the proposed experiment, we found no clear rule to select the optimal number of brain states for the k-means clustering (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              ). The Silhouette and the Calinski and Harabasz indicators would select the smallest k value. However, the choice of k is the choice of the number of brain states that needs to be driven by biomarkers.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F3.large.jpg?width=800&height=600&carousel=1" title="Determining the optimal number of brain states for the k-means clustering using the Elbow, the Silhouette, and the Calinski and Harabasz scores." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup">Determining the optimal number of brain states for the k-means clustering using the Elbow, the Silhouette, and the Calinski and Harabasz scores.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F3.medium.gif" width="440" height="310"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F3.medium.gif" width="440" height="310"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <p id="p-18" class="first-child">
                                Determining the optimal number of brain states for the k-means clustering using the Elbow, the Silhouette, and the Calinski and Harabasz scores.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              4.2 Performance of the classifier
                             </h3>
                             <p id="p-19">
                              Thus, we decided to monitor the BrainNetCNN prediction accuracy through different values of k (or targeted brain states {BS}
                              <sup>
                               k
                              </sup>
                              ) as shown in
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              . In a k-classes classification problem, the theoretical chance level is 1/k. As described in [
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ], this threshold holds for an infinite number of samples, and the smaller the sample size, the more likely chance performance differs from this theoretical chance level. In our application, the sample size is large (56,144 samples), and we assume that the chance level is close to its theoretical value. For all k values, we noticed that the accuracy is much higher than the theoretical chance level, demonstrating the interest of {BS}
                              <sup>
                               k
                              </sup>
                              as labels to characterize the succession of states occurring under a given arousal condition. Smaller k values give better prediction results regardless of the chance level. The overall predictions are satisfactory and lie in [0.655, 0.759]. We put these results in perspective by training a machine learning linear SVC classifier directly on the input upper triangular FC data (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              ). Both methods give similar results with a lower variability for the deep learning prediction when k increases. This result seems disappointing but reveals that the addressed downstream classification task is simple. Indeed, the annotations are generated using a k-means clustering, which consists of linear decision boundaries. While the proposed network uses a cascade of many layers of nonlinear processing units for feature extraction, the performance capabilities of the linear clustering task are likely limited by the nature of these empirical annotations. To go further, we rendered the machine learning and deep learning feature spaces using a Uniform Manifold Approximation and Projection (UMAP) [
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ]. The corresponding spaces have 3403 (the upper triangular elements of the FC matrices) and 30 (the latent space dimension of the BrainNetCNN) dimensions, respectively. We believe that the low dimensional features learned by the BrainNetCNN capture all the relevant information and will ease the interpretation as illustrated in
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              .
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F4.large.jpg?width=800&height=600&carousel=1" title="Determining the optimal number of brain states for the BrainNetCNN prediction using the accuracy as a reference metric: the mean and standard deviation computed across the training folds are displayed. A linear SVC prediction, as well as the theoretical chance levels are displayed as baselines. Note that results are always much higher than random chance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup">Determining the optimal number of brain states for the BrainNetCNN prediction using the accuracy as a reference metric: the mean and standard deviation computed across the training folds are displayed. A linear SVC prediction, as well as the theoretical chance levels are displayed as baselines. Note that results are always much higher than random chance.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F4.medium.gif" width="440" height="345"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F4.medium.gif" width="440" height="345"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <p id="p-20" class="first-child">
                                Determining the optimal number of brain states for the BrainNetCNN prediction using the accuracy as a reference metric: the mean and standard deviation computed across the training folds are displayed. A linear SVC prediction, as well as the theoretical chance levels are displayed as baselines. Note that results are always much higher than random chance.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F5.large.jpg?width=800&height=600&carousel=1" title="UMAP 2-d and 3-d projections of the machine learning (left) and deep learning (right) feature spaces: the upper triangular elements of the FC matrices (3403) vs the latent space dimension of the BrainNetCNN (30). The samples are colored using the k=7 brain states labels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup">UMAP 2-d and 3-d projections of the machine learning (left) and deep learning (right) feature spaces: the upper triangular elements of the FC matrices (3403) vs the latent space dimension of the BrainNetCNN (30). The samples are colored using the k=7 brain states labels.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F5.medium.gif" width="440" height="207"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F5.medium.gif" width="440" height="207"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <p id="p-21" class="first-child">
                                UMAP 2-d and 3-d projections of the machine learning (left) and deep learning (right) feature spaces: the upper triangular elements of the FC matrices (3403) vs the latent space dimension of the BrainNetCNN (30). The samples are colored using the k=7 brain states labels.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              For the remaining part of the paper, we retained a biologically credible number of brain states, ie. k=4 or k=7 [
                              <a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ]. We chose accuracy as the metric to select the best-trained model. Then, the Receiver Operating Characteristic (ROC) metric is used to evaluate the BrainNetCNN classifier output quality. ROC curves show the true positive rate on the y-axis and the false positive rate on the x-axis. An Area Under the Curve (AUC) of 1 corresponds to the ideal case where the false positive rate is zero, and the true positive rate is one. Multi-label classification requires the binarization of the predicted brain state outputs. First, one ROC curve is drawn per label, followed by micro and macro-averaging. We thus demonstrate that the BrainNetCNN predicts the brain states with high reproducibility (> 0.92 AUC) as shown in
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 6
                              </a>
                              .
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F6.large.jpg?width=800&height=600&carousel=1" title="Evaluation of the BrainNetCNN classifier outputs quality using Receiver Operating Characteristic (ROC) curves for k = 4 and k = 7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Evaluation of the BrainNetCNN classifier outputs quality using Receiver Operating Characteristic (ROC) curves for <em>k</em> = 4 and <em>k</em> = 7.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F6.medium.gif" width="289" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F6.medium.gif" width="289" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <p id="p-23" class="first-child">
                                Evaluation of the BrainNetCNN classifier outputs quality using Receiver Operating Characteristic (ROC) curves for k = 4 and k = 7.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              4.3 Towards modeling the brain states dynamic
                             </h3>
                             <p id="p-24">
                              Replaying the dynamic FC matrices movie for each run, the transitions probabilities to be in one state or another are directly accessible as the output of the trained network.
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Fig. 7
                              </a>
                              displays the predicted and the true labels as well as the network estimated probabilities for a specific run acquired in the awake condition. According to
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              , in the awake condition, we expect a prevalence of state 1. The network decisions differ at state transitions. These chronograms give an insight into the monitored brain configurations, and in a prospective way model the brain dynamic oscillating from one state to another.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F7.large.jpg?width=800&height=600&carousel=1" title="The BrainNetCNN predicted transitions with the associated probabilities for a test set run acquired in the awake condition for k = 4 and k = 7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The BrainNetCNN predicted transitions with the associated probabilities for a test set run acquired in the awake condition for <em>k</em> = 4 and <em>k</em> = 7.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F7.medium.gif" width="440" height="396"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F7.medium.gif" width="440" height="396"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7:
                               </span>
                               <p id="p-25" class="first-child">
                                The BrainNetCNN predicted transitions with the associated probabilities for a test set run acquired in the awake condition for k = 4 and k = 7.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              4.4 Maps of predictive connections
                             </h3>
                             <p id="p-26">
                              Proxies of the cortical signatures of consciousness are computed as described in
                              <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">
                               section 3.4
                              </a>
                              . The resulting mean saliency maps for each brain state are displayed in
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Fig. 8
                              </a>
                              . The top p = 15% positive and negative connections using a specific or global threshold across brain states are presented in
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Fig. 9
                              </a>
                              . The CoCoMac regions are grouped into seven locations comprising the cingulate, frontal, gustatory, insular, occipital, temporal, and parietal cortex. The brain states are characterized by the dynamical exploration of a rich, fluctuating set of functional configurations. These dynamical properties constitute a marker of consciousness. More specifically, the network uses shorter connections to identify a brain state far from the structural connectivity as depicted in
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Fig. 10
                              </a>
                              . Let’s focus more closely on the two dominant brain states characterizing wakefulness and anesthesia, BS
                              <sup>
                               1
                              </sup>
                              and BS
                              <sup>
                               7
                              </sup>
                              , respectively (cf.
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ). The connections contributing the most to the corresponding predictions have an average Euclidean distance of 24.27 and 33.15, while the connections contributing the least have an average Euclidean distance of 30.55 and 21.05. This finding seems in line with the hypothesis formulated in [
                              <a id="xref-ref-13-10" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ] stating that dynamic FC matrices during anesthesia reduce to the underlying structural anatomical connectivity (ie. long-range connections), while during wakefulness closer auxiliary links are involved.
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F8.large.jpg?width=800&height=600&carousel=1" title="Resulting mean saliency maps for each brain state." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup">Resulting mean saliency maps for each brain state.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F8.medium.gif" width="440" height="226"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F8.medium.gif" width="440" height="226"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8:
                               </span>
                               <p id="p-27" class="first-child">
                                Resulting mean saliency maps for each brain state.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F9.large.jpg?width=800&height=600&carousel=1" title="Within-brain states marker of consciousness for k=7, rendered using a circular graph where the top p = 15% positive and negative connections using a specific or global threshold across brain states are presented. Top to bottom rows contain the global-positive, global-negative, specific-positive, and specific-negative results, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Within-brain states marker of consciousness for k=7, rendered using a circular graph where the top <em>p</em> = 15% positive and negative connections using a specific or global threshold across brain states are presented. Top to bottom rows contain the global-positive, global-negative, specific-positive, and specific-negative results, respectively.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F9.medium.gif" width="440" height="291"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 9:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F9.medium.gif" width="440" height="291"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 9:
                               </span>
                               <p id="p-28" class="first-child">
                                Within-brain states marker of consciousness for k=7, rendered using a circular graph where the top p = 15% positive and negative connections using a specific or global threshold across brain states are presented. Top to bottom rows contain the global-positive, global-negative, specific-positive, and specific-negative results, respectively.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F10.large.jpg?width=800&height=600&carousel=1" title="Within-brain states marker of consciousness for k=7, rendered using the top p = 15% positive and negative connections lengths (a global threshold is used). The means of each distribution are highlighted with dashed red lines. Top to bottom rows contain the positive, and negative connections of interest, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1738586305" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Within-brain states marker of consciousness for k=7, rendered using the top <em>p</em> = 15% positive and negative connections lengths (a global threshold is used). The means of each distribution are highlighted with dashed red lines. Top to bottom rows contain the positive, and negative connections of interest, respectively.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F10.medium.gif" width="440" height="122"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 10:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F10.medium.gif" width="440" height="122"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/06/2020.05.11.078535/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 10:
                               </span>
                               <p id="p-29" class="first-child">
                                Within-brain states marker of consciousness for k=7, rendered using the top p = 15% positive and negative connections lengths (a global threshold is used). The means of each distribution are highlighted with dashed red lines. Top to bottom rows contain the positive, and negative connections of interest, respectively.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             5 Discussion and conclusion
                            </h2>
                            <p id="p-30">
                             The BrainNetCNN gCNN model can predict brain states with good reproducibility and accuracy. It gives similar performance to a linear SVC on the input dynamical FC data pointing out the simplicity/restriction induced by the downstream classification task driven by the k-means pseudolabels. Nonetheless, the proposed network can scale to more complex tasks by learning complex representations. Using a self-supervised contrastive learning strategy reduces the circularity with the pseudo-labels. Nevertheless, this suited framework may suffer from a too coarse augmentation strategy. We also believe that the learned low-dimensional representation eases further interpretation. Interestingly, the BrainNetCNN predictions differ from the k-means ones at brain state transitions. More than a simple prediction tool, the proposed network can model the brain dynamic oscillating from one state to another and generate state signatures as sets of dominant connections.
                            </p>
                            <p id="p-31">
                             As presented in [
                             <a id="xref-ref-13-11" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ], the entropy of the state transition matrix, the distance of each state to the underlying anatomical connectivity matrix, and the presence of negative correlations are helpful candidates when trying to find signatures of consciousness. We illustrate how the maps of predictive connections can disentangle which connections are useful to discriminate between different levels of wakefulness and bring significant insight into the brain states. These maps should help in understanding the brain states’ signature of consciousness. In future work, we want to apply this technique to anesthesia and disorders of consciousness clinical data.
                            </p>
                            <p id="p-32">
                             In future work, we also want to systematically compare the performance of different classifiers based on machine learning or deep learning models to predict brain states. We believe that gCNN such as the BrainNetCNN will outperform models that do not model the topological locality of the regions used to generate the FC matrices. We also want to investigate the impact of enforcing consistency between cluster assignments on the learned representations as in SwAV [
                             <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ].
                            </p>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.1" data-doi="10.1080/03610927408827101">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Caliński
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Harabasz
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  A dendrite method for cluster analysis
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Communications in Statistics
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1974
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  27
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1080/03610927408827101
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BCommunications%2Bin%2BStatistics%26rft_id%253Dinfo%253Adoi%252F10.1080%252F03610927408827101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1080/03610927408827101&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.2" data-doi="10.1016/0377-0427(87)90125-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                  <span class="cit-name-surname">
                                   Rousseeuw
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Silhouettes: A Graphical Aid to the Interpretation and Validation of Cluster Analysis
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Journal of Computational and Applied Mathematics
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1987
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  53
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  65
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/0377-0427(87)90125-7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BJournal%2Bof%2BComputational%2Band%2BApplied%2BMathematics%26rft.volume%253D20%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0377-0427%252887%252990125-7%26rft_id%253Dinfo%253Apmid%252FWOS%253AA1987L11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0377-0427(87)90125-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=WOS:A1987L11&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F06%2F2020.05.11.078535.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1987L111800005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sotiris
                                  </span>
                                  <span class="cit-name-surname">
                                   Kotsiantis
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Supervised Machine Learning: A Review of Classification Techniques
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Informatica
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-month">
                                  Oct
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.4" data-doi="10.3389/fninf.2012.00030">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rembrandt
                                  </span>
                                  <span class="cit-name-surname">
                                   Bakker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Wachtler
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Markus
                                  </span>
                                  <span class="cit-name-surname">
                                   Diesmann
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Co-CoMac 2.0 and the future of tract-tracing databases
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Frontiers in Neuroinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  30
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  1662-5196
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.3389/fninf.2012.00030
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BFrontiers%2Bin%2BNeuroinformatics%26rft.volume%253D6%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffninf.2012.00030%26rft_id%253Dinfo%253Apmid%252F23293600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.3389/fninf.2012.00030&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23293600&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F06%2F2020.05.11.078535.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.5" data-doi="10.18576/amis/100428">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Trupti M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kodinariya
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Prashant R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Makwana
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Review on determining number of Cluster in K-Means Clustering
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Applied Mathematics and Information Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.18576/amis/100428
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BApplied%2BMathematics%2Band%2BInformation%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.18576%252Famis%252F100428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.18576/amis/100428&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Karen
                                  </span>
                                  <span class="cit-name-surname">
                                   Simonyan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrea
                                  </span>
                                  <span class="cit-name-surname">
                                   Vedaldi
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrew
                                  </span>
                                  <span class="cit-name-surname">
                                   Zisserman
                                  </span>
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Deep Inside Convolutional Networks: Visualising Image Classification Models and Saliency Maps
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1312.6034
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.7" data-doi="10.1523/JNEUROSCI.3165-13.2014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lynn
                                  </span>
                                  <span class="cit-name-surname">
                                   Uhrig
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stanislas
                                  </span>
                                  <span class="cit-name-surname">
                                   Dehaene
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Béchir
                                  </span>
                                  <span class="cit-name-surname">
                                   Jarraya
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  A Hierarchy of Responses to Auditory Regularities in the Macaque Brain
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1127
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1132
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  0270-6474
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1523/JNEUROSCI.3165-13.2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DUhrig%26rft.auinit1%253DL.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D1127%26rft.epage%253D1132%26rft.atitle%253DA%2BHierarchy%2Bof%2BResponses%2Bto%2BAuditory%2BRegularities%2Bin%2Bthe%2BMacaque%2BBrain%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.3165-13.2014%26rft_id%253Dinfo%253Apmid%252F24453305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjk6IjM0LzQvMTEyNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA1LzA2LzIwMjAuMDUuMTEuMDc4NTM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.8" data-doi="10.1073/pnas.1418031112">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pablo
                                  </span>
                                  <span class="cit-name-surname">
                                   Barttfeld
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Signature of consciousness in the dynamics of resting-state brain activity
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  887
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  892
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  0027-8424
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1073/pnas.1418031112
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1418031112%26rft_id%253Dinfo%253Apmid%252F25561541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTIvMy84ODciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8wNi8yMDIwLjA1LjExLjA3ODUzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.9" data-doi="10.1016/j.jneumeth.2015.01.010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Etienne
                                  </span>
                                  <span class="cit-name-surname">
                                   Combrisson
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Karim
                                  </span>
                                  <span class="cit-name-surname">
                                   Jerbi
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Exceeding chance level by chance: The caveat of theoretical chance levels in brain signal classification and statistical assessment of decoding accuracy
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Journal of Neuroscience Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  250
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  126
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  136
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  0165-0270
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.jneumeth.2015.01.010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BJournal%2Bof%2BNeuroscience%2BMethods%26rft.volume%253D250%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jneumeth.2015.01.010%26rft_id%253Dinfo%253Apmid%252F25596422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jneumeth.2015.01.010&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25596422&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F06%2F2020.05.11.078535.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jost Tobias
                                  </span>
                                  <span class="cit-name-surname">
                                   Springenberg
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Striving for Simplicity: The All Convolutional Net
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1412.6806
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.11" data-doi="10.1016/j.neuroimage.2016.09.046">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeremy
                                  </span>
                                  <span class="cit-name-surname">
                                   Kawahara
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  BrainNetCNN: Convolutional neural networks for brain networks; towards predicting neurodevelopment
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: NeuroImage
                                 </abbr>
                                 <span class="cit-vol">
                                  146
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1038
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1049
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  1053-8119
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.neuroimage.2016.09.046
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNeuroImage%26rft.volume%253D146%26rft.spage%253D1038%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2016.09.046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2016.09.046&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mathilde
                                  </span>
                                  <span class="cit-name-surname">
                                   Caron
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Deep Clustering for Unsupervised Learning of Visual Features
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: ECCV
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  139
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  156
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.13" data-doi="10.1097/ALN.0000000000002336">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lynn
                                  </span>
                                  <span class="cit-name-surname">
                                   Uhrig
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Resting-state Dynamics as a Cortical Signature of Anesthesia in Monkeys
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Anesthesiology
                                 </abbr>
                                 <span class="cit-vol">
                                  129
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-month">
                                  Nov
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  942
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  958
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  0003-3022
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1097/ALN.0000000000002336
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BAnesthesiology%26rft.volume%253D129%26rft.spage%253D942%26rft_id%253Dinfo%253Adoi%252F10.1097%252FALN.0000000000002336%26rft_id%253Dinfo%253Apmid%252F30028727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1097/ALN.0000000000002336&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30028727&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F06%2F2020.05.11.078535.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mathilde
                                  </span>
                                  <span class="cit-name-surname">
                                   Caron
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  “Unsupervised Learning of Visual Features by Contrasting Cluster Assignments”. en
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In: NeurIPS
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  9912
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9924
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNeurIPS%26rft.volume%253D33%26rft.spage%253D9912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.11.078535v2.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ting
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-chapter-title">
                                  A Simple Framework for Contrastive Learning of Visual Representations
                                 </span>
                                 ”.
                                 <span class="cit-source">
                                  In: International Conference on Machine Learning
                                 </span>
                                 . ISSN:
                                 <span class="cit-issn">
                                  2640-3498
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  PMLR
                                 </span>
                                 ,
                                 <span class="cit-month">
                                  Nov
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1597
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1607
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.16" data-doi="10.1016/j.neuron.2020.01.026">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  “Conscious Processing and the Global Neuronal Workspace Hypothesis”
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In: Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  776
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  798
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  0896-6273
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.neuron.2020.01.026
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNeuron%26rft.volume%253D105%26rft.spage%253D776%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2020.01.026%26rft_id%253Dinfo%253Apmid%252F32135090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2020.01.026&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32135090&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F06%2F2020.05.11.078535.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Leland
                                  </span>
                                  <span class="cit-name-surname">
                                   McInnes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John
                                  </span>
                                  <span class="cit-name-surname">
                                   Healy
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James
                                  </span>
                                  <span class="cit-name-surname">
                                   Melville
                                  </span>
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1802.03426
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.18" data-doi="10.1016/j.neuroimage.2019.116353">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  “Pypreclin: An automatic pipeline for macaque functional MRI preprocessing”
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In: NeuroImage
                                 </abbr>
                                 <span class="cit-vol">
                                  207
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  116353
                                 </span>
                                 . issn:
                                 <span class="cit-issn">
                                  1053-8119
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.neuroimage.2019.116353
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNeuroImage%26rft.volume%253D207%26rft.spage%253D116353%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2019.116353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2019.116353&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xiaohang
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhan
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Online Deep Clustering for Unsupervised Representation Learning
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  2006.10645
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.11.078535v2.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jure
                                  </span>
                                  <span class="cit-name-surname">
                                   Zbontar
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Barlow Twins: Self-Supervised Learning via Redundancy Reduction
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  2103.03230
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.05.11.078535v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.05.04.490255v2" title="Expression and functional evaluation of recombinant human ABO blood group antigen cleaving glycoside hydrolases α-N-acetylgalactosaminidase, α-galactosidase, and endo-β-galactosidase produced in Escherichia coli" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.05.05.490722v1" title="The effect of dietary chitin on Atlantic salmon (Salmo salar) chitinase activity, gene expression, and microbial composition" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted May 06, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.05.11.078535v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.05.11.078535v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.05.11.078535v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2530823"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-eFHd1Ondj8EN2odcZMLdKupHKKJEV-2j8vo_OAa6S-4"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484470543"/>
                                   <input type="hidden" name="captcha_token" value="a29c7b28de06993402287f79fd159e7d"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2530823--3" data-node-nid="2530823" id="node-2530823--41998446422" data-pisa="biorxiv;2020.05.11.078535v2" data-pisa-master="biorxiv;2020.05.11.078535" data-seqnum="2530823" data-apath="/biorxiv/early/2022/05/06/2020.05.11.078535.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Antoine
                                  </span>
                                  <span class="nlm-surname">
                                   Grigis
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Chloé
                                  </span>
                                  <span class="nlm-surname">
                                   Gomez
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Jordy
                                  </span>
                                  <span class="nlm-surname">
                                   Tasserie
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Corentin
                                  </span>
                                  <span class="nlm-surname">
                                   Ambroise
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Vincent
                                  </span>
                                  <span class="nlm-surname">
                                   Frouin
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Béchir
                                  </span>
                                  <span class="nlm-surname">
                                   Jarraya
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Lynn
                                  </span>
                                  <span class="nlm-surname">
                                   Uhrig
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.05.11.078535;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.05.11.078535
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.05.11.078535v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&title=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&title=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&text=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&t=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&title=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&title=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&title=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2530823--5" data-node-nid="2530823" id="citation-node-2530823--62016477492" data-pisa="biorxiv;2020.05.11.078535v2" data-pisa-master="biorxiv;2020.05.11.078535" data-seqnum="2530823" data-apath="/biorxiv/early/2022/05/06/2020.05.11.078535.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Predicting Cortical Signatures of Consciousness using Dynamic Functional Connectivity Graph-Convolutional Neural Networks
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Antoine
                                      </span>
                                      <span class="nlm-surname">
                                       Grigis
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Chloé
                                      </span>
                                      <span class="nlm-surname">
                                       Gomez
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Jordy
                                      </span>
                                      <span class="nlm-surname">
                                       Tasserie
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Corentin
                                      </span>
                                      <span class="nlm-surname">
                                       Ambroise
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Vincent
                                      </span>
                                      <span class="nlm-surname">
                                       Frouin
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Béchir
                                      </span>
                                      <span class="nlm-surname">
                                       Jarraya
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Lynn
                                      </span>
                                      <span class="nlm-surname">
                                       Uhrig
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.05.11.078535;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.05.11.078535
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Grigis%2C+Antoine&rft.au=Gomez%2C+Chlo%C3%A9&rft.au=Tasserie%2C+Jordy&rft.au=Ambroise%2C+Corentin&rft.au=Frouin%2C+Vincent&rft.au=Jarraya%2C+B%C3%A9chir&rft.au=Uhrig%2C+Lynn">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2530823/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2530823/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2530823/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2530823/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2530823/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2530823/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2530823/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2530823/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2530823/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2530823/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2530823/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2530823/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&count=horizontal&via=&text=Predicting%20Cortical%20Signatures%20of%20Consciousness%20using%20Dynamic%20Functional%20Connectivity%20Graph-Convolutional%20Neural%20Networks&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.05.11.078535v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.05.11.078535v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
