<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.06.14.150748v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.06.14.150748v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.06.14.150748v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/12/08/2020.06.14.150748.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.06.14.150748v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells"/>
  <meta name="DC.Identifier" content="10.1101/2020.06.14.150748"/>
  <meta name="DC.Date" content="2022-12-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Cell cycle progression follows a precise sequence of events marked by different phases and checkpoints which are associated with specific chromatin organisation. Whilst these changes have been extensively studied, their consequences on transcriptional networks remain to be fully uncovered, especially in dynamic model systems such as stem cells. Here, we take advantage of the FUCCI reporter system to show that chromatin accessibility, gene expression and key transcription factors binding change during cell cycle progression in human Embryonic Stem Cells (hESCs). These analyses reveal that core pluripotency factors such as OCT4, NANOG and SOX2 but also chromatin remodelers such as CTCF and RING1B bind the genome at specific phases of the cell cycle. Importantly, this binding pattern allows differentiation in the G1 phase while preserving pluripotency in the S/G2/M. Our results highlight the importance of studying transcriptional and epigenetic regulations in the dynamic context of the cell cycle.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Anna Osnato"/>
  <meta name="DC.Contributor" content="Stephanie Brown"/>
  <meta name="DC.Contributor" content="Ludovic Vallier"/>
  <meta name="article:published_time" content="2022-12-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells"/>
  <meta name="citation_abstract" lang="en" content="<h3>Summary</h3>
<p>Cell cycle progression follows a precise sequence of events marked by different phases and checkpoints which are associated with specific chromatin organisation. Whilst these changes have been extensively studied, their consequences on transcriptional networks remain to be fully uncovered, especially in dynamic model systems such as stem cells. Here, we take advantage of the FUCCI reporter system to show that chromatin accessibility, gene expression and key transcription factors binding change during cell cycle progression in human Embryonic Stem Cells (hESCs). These analyses reveal that core pluripotency factors such as OCT4, NANOG and SOX2 but also chromatin remodelers such as CTCF and RING1B bind the genome at specific phases of the cell cycle. Importantly, this binding pattern allows differentiation in the G1 phase while preserving pluripotency in the S/G2/M. Our results highlight the importance of studying transcriptional and epigenetic regulations in the dynamic context of the cell cycle.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.06.14.150748v2"/>
  <meta name="citation_id" content="2020.06.14.150748v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.06.14.150748v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.06.14.150748v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.06.14.150748v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.06.14.150748"/>
  <meta name="citation_num_pages" content="46"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.06.14.150748"/>
  <meta name="citation_author" content="Anna Osnato"/>
  <meta name="citation_author_institution" content="Wellcome – MRC Cambridge Stem Cell Institute, Jeffrey Cheah Biomedical Centre, University of Cambridge"/>
  <meta name="citation_author_institution" content="Department of Surgery, University of Cambridge"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5241-1512"/>
  <meta name="citation_author" content="Stephanie Brown"/>
  <meta name="citation_author_institution" content="Wellcome – MRC Cambridge Stem Cell Institute, Jeffrey Cheah Biomedical Centre, University of Cambridge"/>
  <meta name="citation_author_institution" content="Department of Surgery, University of Cambridge"/>
  <meta name="citation_author" content="Ludovic Vallier"/>
  <meta name="citation_author_institution" content="Wellcome – MRC Cambridge Stem Cell Institute, Jeffrey Cheah Biomedical Centre, University of Cambridge"/>
  <meta name="citation_author_institution" content="Department of Surgery, University of Cambridge"/>
  <meta name="citation_author_email" content="lv225@cam.ac.uk"/>
  <meta name="citation_author_email" content="ludovic.vallier@bih-charite.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3848-2602"/>
  <meta name="citation_reference" content="Naumova, N. et al. Organization of the mitotic chromosome. Science 342, 948–953, doi:10.1126/science.1236083 (2013)."/>
  <meta name="citation_reference" content="Nagano, T. et al. Cell-cycle dynamics of chromosomal organization at single-cell resolution. Nature 547, 61–67, doi:10.1038/nature23001 (2017)."/>
  <meta name="citation_reference" content="Chen, X. et al. ATAC-see reveals the accessible genome by transposase-mediated imaging and sequencing. Nat Methods 13, 1013–1020, doi:10.1038/nmeth.4031 (2016)."/>
  <meta name="citation_reference" content="Pauklin, S. & Vallier, L. The cell-cycle state of stem cells determines cell fate propensity. Cell 155, 135–147, doi:10.1016/j.cell.2013.08.031 (2013)."/>
  <meta name="citation_reference" content="Sakaue-Sawano, A. et al. Visualizing spatiotemporal dynamics of multicellular cellcycle progression. Cell 132, 487–498, doi:10.1016/j.cell.2007.12.033 (2008)."/>
  <meta name="citation_reference" content="Landt, S. G. et al. ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia. Genome Res 22, 1813–1831, doi:10.1101/gr.136184.111 (2012)."/>
  <meta name="citation_reference" content="Madrigal, P. et al. Epigenetic regulations follow cell cycle progression during differentiation of human pluripotent stem cells. bioRxiv, 2020.2006.2026.173211, doi:10.1101/2020.06.26.173211 (2020)."/>
  <meta name="citation_reference" content="Shen, L. et al. diffReps: detecting differential chromatin modification sites from ChIP-seq data with biological replicates. PLoS One 8, e65598, doi:10.1371/journal.pone.0065598 (2013)."/>
  <meta name="citation_reference" content="Yardimci, G. G., Frank, C. L., Crawford, G. E. & Ohler, U. Explicit DNase sequence bias modeling enables high-resolution transcription factor footprint detection. Nucleic Acids Res 42, 11865–11878, doi:10.1093/nar/gku810 (2014)."/>
  <meta name="citation_reference" content="Piper, J. et al. Wellington: a novel method for the accurate identification of digital genomic footprints from DNase-seq data. Nucleic Acids Res 41, e201, doi:10.1093/nar/gkt850 (2013)."/>
  <meta name="citation_reference" content="Zaret, K. S. & Carroll, J. S. Pioneer transcription factors: establishing competence for gene expression. Genes Dev 25, 2227–2241, doi:10.1101/gad.176826.111 (2011)."/>
  <meta name="citation_reference" content="Singh, A. M. et al. Cell-cycle control of developmentally regulated transcription factors accounts for heterogeneity in human pluripotent cells. Stem Cell Reports 1, 532–544, doi:10.1016/j.stemcr.2013.10.009 (2013)."/>
  <meta name="citation_reference" content="Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol 15, 550, doi:10.1186/s13059-014-0550-8 (2014)."/>
  <meta name="citation_reference" content="Ji, X. et al. 3D Chromosome Regulatory Landscape of Human Pluripotent Cells. Cell Stem Cell 18, 262–275, doi:10.1016/j.stem.2015.11.007 (2016)."/>
  <meta name="citation_reference" content="Jin, C. et al. H3.3/H2A.Z double variant-containing nucleosomes mark ‘nucleosome-free regions’ of active promoters and other regulatory regions. Nat Genet 41, 941–945, doi:10.1038/ng.409 (2009)."/>
  <meta name="citation_reference" content="Jang, C. W., Shibata, Y., Starmer, J., Yee, D. & Magnuson, T. Histone H3.3 maintains genome integrity during mammalian development. Genes Dev 29, 1377–1392, doi:10.1101/gad.264150.115 (2015)."/>
  <meta name="citation_reference" content="Sekiya, T., Kato, K., Kawaguchi, A. & Nagata, K. Involvement of CTCF in transcription regulation of EGR1 at early G1 phase as an architecture factor. Sci Rep 9, 329, doi:10.1038/s41598-018-36753-x (2019)."/>
  <meta name="citation_reference" content="Canela, A. et al. Genome Organization Drives Chromosome Fragility. Cell 170, 507–521 e518, doi:10.1016/j.cell.2017.06.034 (2017)."/>
  <meta name="citation_reference" content="Endoh, M. et al. Polycomb group proteins Ring1A/B are functionally linked to the core transcriptional regulatory circuitry to maintain ES cell identity. Development 135, 1513–1524, doi:10.1242/dev.014340 (2008)."/>
  <meta name="citation_reference" content="Aranda, S., Mas, G. & Di Croce, L. Regulation of gene transcription by Polycomb proteins. Sci Adv 1, e1500737, doi:10.1126/sciadv.1500737 (2015)."/>
  <meta name="citation_reference" content="Swaney, D. L., Wenger, C. D., Thomson, J. A. & Coon, J. J. Human embryonic stem cell phosphoproteome revealed by electron transfer dissociation tandem mass spectrometry. Proc Natl Acad Sci U S A 106, 995–1000, doi:10.1073/pnas.0811964106 (2009)."/>
  <meta name="citation_reference" content="Saxe, J. P., Tomilin, A., Scholer, H. R., Plath, K. & Huang, J. Post-translational regulation of Oct4 transcriptional activity. PLoS One 4, e4467, doi:10.1371/journal.pone.0004467 (2009)."/>
  <meta name="citation_reference" content="Shin, J. et al. Aurkb/PP1-mediated resetting of Oct4 during the cell cycle determines the identity of embryonic stem cells. Elife 5, doi:10.7554/eLife.10877 (2016)."/>
  <meta name="citation_reference" content="Teo, A. K. K. et al. Pluripotency factors regulate definitive endoderm specification through eomesodermin. Gene Dev 25, 238–250, doi:10.1101/gad.607311 (2011)."/>
  <meta name="citation_reference" content="Vallier, L. et al. Signaling pathways controlling pluripotency and early cell fate decisions of human induced pluripotent stem cells. Stem Cells 27, 2655–2666, doi:10.1002/stem.199 (2009)."/>
  <meta name="citation_reference" content="Touboul, T. et al. Generation of functional hepatocytes from human embryonic stem cells under chemically defined conditions that recapitulate liver development. Hepatology 51, 1754–1765, doi:10.1002/hep.23506 (2010)."/>
  <meta name="citation_reference" content="Bertero, A. et al. Optimized inducible shRNA and CRISPR/Cas9 platforms for in vitro studies of human development using hPSCs. Development 143, 4405–4418, doi:10.1242/dev.138081 (2016)."/>
  <meta name="citation_reference" content="Xin, X. et al. miR-155 Accelerates the Growth of Human Liver Cancer Cells by Activating CDK2 via Targeting H3F3A. Mol Ther Oncolytics 17, 471–483, doi:10.1016/j.omto.2020.05.002 (2020)."/>
  <meta name="citation_reference" content="Kundu, S. et al. Polycomb Repressive Complex 1 Generates Discrete Compacted Domains that Change during Differentiation. Mol Cell 65, 432–446 e435, doi:10.1016/j.molcel.2017.01.009 (2017)."/>
  <meta name="citation_reference" content="Gonzales, K. A. et al. Deterministic Restriction on Pluripotent State Dissolution by Cell-Cycle Pathways. Cell 162, 564–579, doi:10.1016/j.cell.2015.07.001 (2015)."/>
  <meta name="citation_reference" content="Endoh, M. et al. Histone H2A mono-ubiquitination is a crucial step to mediate PRC1-dependent repression of developmental genes to maintain ES cell identity. PLoS Genet 8, e1002774, doi:10.1371/journal.pgen.1002774 (2012)."/>
  <meta name="citation_reference" content="Vallier, L. Serum-free and feeder-free culture conditions for human embryonic stem cells. Methods Mol Biol 690, 57–66, doi:10.1007/978-1-60761-962-8_3 (2011)."/>
  <meta name="citation_reference" content="Chen, G. et al. Chemically defined conditions for human iPSC derivation and culture. Nat Methods 8, 424–429, doi:10.1038/nmeth.1593 (2011)."/>
  <meta name="citation_reference" content="Vallier, L. et al. Early cell fate decisions of human embryonic stem cells and mouse epiblast stem cells are controlled by the same signalling pathways. PLoS One 4, e6082, doi:10.1371/journal.pone.0006082 (2009)."/>
  <meta name="citation_reference" content="Kumasaka, N., Knights, A. J. & Gaffney, D. J. Fine-mapping cellular QTLs with RASQUAL and ATAC-seq. Nat Genet 48, 206–213, doi:10.1038/ng.3467 (2016)."/>
  <meta name="citation_reference" content="Vallier, L. et al. Enhancing and diminishing gene function in human embryonic stem cells. Stem Cells 22, 2–11, doi:10.1634/stemcells.22-1-2 (2004)."/>
  <meta name="citation_reference" content="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079, doi:10.1093/bioinformatics/btp352 (2009)."/>
  <meta name="citation_reference" content="Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol 9, R137, doi:10.1186/gb-2008-9-9-r137 (2008)."/>
  <meta name="citation_reference" content="Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842, doi:10.1093/bioinformatics/btq033 (2010)."/>
  <meta name="citation_reference" content="Down, T. A., Piipari, M. & Hubbard, T. J. Dalliance: interactive genome viewing on the web. Bioinformatics 27, 889–890, doi:10.1093/bioinformatics/btr020 (2011)."/>
  <meta name="citation_reference" content="Ross-Innes, C. S. et al. Differential oestrogen receptor binding is associated with clinical outcome in breast cancer. Nature 481, 389–393, doi:10.1038/nature10730 (2012)."/>
  <meta name="citation_reference" content="Akalin, A., Franke, V., Vlahovicek, K., Mason, C. E. & Schubeler, D. Genomation: a toolkit to summarize, annotate and visualize genomic intervals. Bioinformatics 31, 1127–1129, doi:10.1093/bioinformatics/btu775 (2015)."/>
  <meta name="citation_reference" content="Zhu, L. J. et al. ChIPpeakAnno: a Bioconductor package to annotate ChIP-seq and ChIP-chip data. BMC Bioinformatics 11, 237, doi:10.1186/1471-2105-11-237 (2010)."/>
  <meta name="citation_reference" content="McLean, C. Y. et al. GREAT improves functional interpretation of cis-regulatory regions. Nat Biotechnol 28, 495–501, doi:10.1038/nbt.1630 (2010)."/>
  <meta name="citation_reference" content="Ou J, Y. J., Kelliher M, Castilla L, Lawson N and Zhu LJ. ATACseqQC: ATAC-seq Quality Control. R package version 1.0.5. (2017)."/>
  <meta name="citation_reference" content="Gusmao, E. G., Allhoff, M., Zenke, M. & Costa, I. G. Analysis of computational footprinting methods for DNase sequencing experiments. Nat Methods 13, 303–309, doi:10.1038/nmeth.3772 (2016)."/>
  <meta name="citation_reference" content="Schep, A. N. et al. Structured nucleosome fingerprints enable high-resolution mapping of chromatin architecture within regulatory regions. Genome Res 25, 1757–1770, doi:10.1101/gr.192294.115 (2015)."/>
  <meta name="citation_reference" content="Sickle: A sliding-window, adaptive, quality-based trimming tool for FastQ files Software. Available at https://github.com/najoshi/sickle. v. 1.33 (2011)."/>
  <meta name="citation_reference" content="Kim, D. et al. TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol 14, R36, doi:10.1186/gb-2013-14-4-r36 (2013)."/>
  <meta name="citation_reference" content="Langmead, B. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Nat Methods 9, 357–359, doi:10.1038/nmeth.1923 (2012)."/>
  <meta name="citation_reference" content="SeqMonk: A tool to visualise and analyse high throughput mapped sequence data. http://www.bioinformatics.babraham.ac.uk/projects/seqmonk/ (2007)."/>
  <meta name="citation_reference" content="Thomas, P. D. et al. PANTHER: a library of protein families and subfamilies indexed by function. Genome Res 13, 2129–2141, doi:10.1101/gr.772403 (2003)."/>
  <meta name="citation_reference" content="Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC Bioinformatics 14, 128, doi:10.1186/1471-2105-14-128 (2013)."/>
  <meta name="twitter:title" content="Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Cell cycle progression follows a precise sequence of events marked by different phases and checkpoints which are associated with specific chromatin organisation. Whilst these changes have been extensively studied, their consequences on transcriptional networks remain to be fully uncovered, especially in dynamic model systems such as stem cells. Here, we take advantage of the FUCCI reporter system to show that chromatin accessibility, gene expression and key transcription factors binding change during cell cycle progression in human Embryonic Stem Cells (hESCs). These analyses reveal that core pluripotency factors such as OCT4, NANOG and SOX2 but also chromatin remodelers such as CTCF and RING1B bind the genome at specific phases of the cell cycle. Importantly, this binding pattern allows differentiation in the G1 phase while preserving pluripotency in the S/G2/M. Our results highlight the importance of studying transcriptional and epigenetic regulations in the dynamic context of the cell cycle.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.06.14.150748v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Cell cycle progression follows a precise sequence of events marked by different phases and checkpoints which are associated with specific chromatin organisation. Whilst these changes have been extensively studied, their consequences on transcriptional networks remain to be fully uncovered, especially in dynamic model systems such as stem cells. Here, we take advantage of the FUCCI reporter system to show that chromatin accessibility, gene expression and key transcription factors binding change during cell cycle progression in human Embryonic Stem Cells (hESCs). These analyses reveal that core pluripotency factors such as OCT4, NANOG and SOX2 but also chromatin remodelers such as CTCF and RING1B bind the genome at specific phases of the cell cycle. Importantly, this binding pattern allows differentiation in the G1 phase while preserving pluripotency in the S/G2/M. Our results highlight the importance of studying transcriptional and epigenetic regulations in the dynamic context of the cell cycle.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-12-08"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.06.14.150748v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2886546"/>
  <title>
   Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"t8DfqRfxItoK3GDNS45ypsNF68KxX7fB8eME0sZzrgw"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2886546","apath":"\/biorxiv\/early\/2022\/12\/08\/2020.06.14.150748.atom","pisa":"biorxiv;2020.06.14.150748v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.06.14.150748v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.06.14.150748v2","title":"Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells","identifier":"node\/2886546"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RHLaCfut0vvvDc7TmyWDafvuy198+qkJYAjmTnPatKadmPKpR\\\\\\\/\\\\\\\/Kw1rG7xELc0qrHe8EJ8IzV1drJqS+ojDahBWPbTl602BD360eKa\\\\\\\/1jah14bQN9kb486jlO6\\\\\\\/GhruGSCcmDXX9j4E0PD6eB7gotirQomlVq9IQvbpCdtpko6LnCjudVWDewPjvhF5nD64hAnoF714jdLQEAEX2NRt1aPm+9Qroqs2R5bmHM8fOKNTf\\\\\\\/yGGR4NyohNVPFLpAqRX\\\\\\\/b2FUZSZPZ1rCayrFdiX7o2hXt8fnKLZ5UnUJi5FmxXTIWe53Rp4Jgkp9GH99RFsdVFGj69U3Cp6qoQU4MhVT+34aDblSOkSHzuQz7pUQAkSKvhM0LSI2B4oXsEIhEFJQkZBkBu72TpxnaGnRM0NUX+uO7P6CulRH4YLVEooSHDIvZJIYWS85R0wSNWd5jBTZwMM3w6DNTUh7pKaYq6eh3sBONv2faMc8CgHsdroWvzsWIU\\\\\\\/g\\\\\\\/MU\\\\\\\/8CcmkjYqeQquqi9OhG\\\\\\\/Si5ANcJXK3Cm8rm8u4ryueibnkuRB0N80wbPOFBZoA3CuIP5GFxp\\\\\\\/vAuAh4O2avbf5bVHVBlrXS2xEMOaobnNqDa4kwKklbV62jkaBdg6A34CmH162361AsufnCu6UfdoeFpAOGEbC+dY58GSFPADGN5RMSI0dOPu+gkyntYut6O2KMzWAySZ10vrDyubpCgN2HXmha2sGPKlQ8oHLWJQOVfBg\\\\\\\/0wsIZDf1qNDCySIUSrRf1LOmZGfdpM2Lxylrksf+0zBjRkYyXU2kmarm1wxxeiFLDFkEErHVW1LyfXYEVC8hsUiTTU7iT2P+QMQGWCqGSeDpOTgwpKAwfX8vkWl2MhwdLzUllm0PLCt193wJLM5Y9lJ39Ihk8sJW5bEs6JFr5r0r4iyc0TwkdZzVsq4TVVLxn6+xNhwVvZZV4vz4GTG6pFKkGO8C9F5yUniuaiRRakoziMhCqBM57iPYyiPn6C7uwxy6pVXGxuZk1BRTcADgg3CMqPcopx3WzokrdGGdpjyEJUVjrkJxkBOydecJkOt8aJzXbjgLZ4dm+qyfq\\\\\\\/cT0kReMy5CZ9s0LIcFpTbpF9Fz5+LB9Xr5iX7tVFXnwFHuQ5tiyMVhdDlxyeNvHf7mOJRHIBO9qhPB8tfMfxnVG3SxUTlBkogIZv1jTRaYHoXs8jS3SyMWxkSq4q5fujQTiwFYp1MeCktm3Jtkmc7ID1wP8l5xzwi537y+HbzTFOK3j3AhVJrZomuojYIxu8gaseTrYzuwL+pLaJTf7Sa+m41+ODs5UWIk+RjSF3hcY3\\\\\\\/fV60bMXq6abgQd1r5XgMDr4Jv2Fusrw+df5skW4MBTigraI5MTfMtp0FeQ6wR9ZhVlee8Iumo=\\\u0022,\\\u0022iv\\\u0022:\\\u00220KA6Tg8zHJBOwI1Z79VFCg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022eff6dc9638ee897fd26e825ff74ccb0778cbd5458513381dbe40c9c63ccc103a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.06.14.150748v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2886546 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.06.14.150748v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_888869303">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_888869303" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1925759917" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-iCuRvKZYNEL9YTKsPqnp26DF9kGbRUu6G_GON3B1yNE"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2886546" data-node-nid="2886546" id="node-2886546--21260361143" data-pisa="biorxiv;2020.06.14.150748v2" data-pisa-master="biorxiv;2020.06.14.150748" data-apath="/biorxiv/early/2022/12/08/2020.06.14.150748.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-5241-1512" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Anna
                      </span>
                      <span class="nlm-surname">
                       Osnato
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Stephanie
                      </span>
                      <span class="nlm-surname">
                       Brown
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-3848-2602" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ludovic
                      </span>
                      <span class="nlm-surname">
                       Vallier
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.06.14.150748
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2886546--21260361143" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Anna Osnato
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Wellcome – MRC Cambridge Stem Cell Institute, Jeffrey Cheah Biomedical Centre, University of Cambridge
                       </span>
                       , CB2 0AW, Cambridge,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Surgery, University of Cambridge
                       </span>
                       , Cambridge,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Anna%2BOsnato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Osnato%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAnna%2BOsnato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5241-1512" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Anna Osnato
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Stephanie Brown
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Wellcome – MRC Cambridge Stem Cell Institute, Jeffrey Cheah Biomedical Centre, University of Cambridge
                       </span>
                       , CB2 0AW, Cambridge,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Surgery, University of Cambridge
                       </span>
                       , Cambridge,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Stephanie%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Brown%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AStephanie%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Ludovic Vallier
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Wellcome – MRC Cambridge Stem Cell Institute, Jeffrey Cheah Biomedical Centre, University of Cambridge
                       </span>
                       , CB2 0AW, Cambridge,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Surgery, University of Cambridge
                       </span>
                       , Cambridge,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ludovic%2BVallier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Vallier%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALudovic%2BVallier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-3848-2602" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ludovic Vallier
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:lv225@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        lv225@cam.ac.uk
                       </a>
                       <a href="mailto:ludovic.vallier@bih-charite.de" class="" data-icon-position="" data-hide-link-title="0">
                        ludovic.vallier@bih-charite.de
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.06.14.150748v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2886546" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2886546/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.14.150748v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2886546" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2886546/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.14.150748v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2886546" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2886546/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.14.150748v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2886546" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2886546/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.06.14.150748v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2886546" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2886546/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Summary
                            </h2>
                            <p id="p-2">
                             Cell cycle progression follows a precise sequence of events marked by different phases and checkpoints which are associated with specific chromatin organisation. Whilst these changes have been extensively studied, their consequences on transcriptional networks remain to be fully uncovered, especially in dynamic model systems such as stem cells. Here, we take advantage of the FUCCI reporter system to show that chromatin accessibility, gene expression and key transcription factors binding change during cell cycle progression in human Embryonic Stem Cells (hESCs). These analyses reveal that core pluripotency factors such as OCT4, NANOG and SOX2 but also chromatin remodelers such as CTCF and RING1B bind the genome at specific phases of the cell cycle. Importantly, this binding pattern allows differentiation in the G1 phase while preserving pluripotency in the S/G2/M. Our results highlight the importance of studying transcriptional and epigenetic regulations in the dynamic context of the cell cycle.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-5">
                             Cell cycle regulation is essential to control the number of cells generated during development and organ homeostasis, while dysregulation of proliferation can result in disease. Importantly, recent studies have suggested that cell cycle regulation could go beyond its canonical function in controlling cell division initiation. Indeed, chromatin alternates between an invariant mitotic conformation and a cell-type specific interphase organisation during cell cycle progression
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                             </sup>
                             . Loops and Topological-Associated Domains (TADs) insulation emerge very early in the G1 phase, but A/B compartments reappear only during the S phase
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             . Global chromatin decondenses in the G1 phase, and refolds during S/G2/M
                             <sup>
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             . In addition, cell fate propensity has been shown to be determined by the cell cycle state of hESCs via the regulation of SMAD2/3, the main effector of the TGFβ signalling pathway
                             <sup>
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                             </sup>
                             . However, the genome wide impact of these changes on transcriptional networks and the link with chromatin structure are still unknown. To address these questions, we used a FUCCI-hESC reporter line
                             <sup>
                              <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             to perform ATAC-seq, RNA-seq and ChIP-seq on specific transcription factors (TFs) during cell cycle progression to look at the dynamic changes in chromatin accessibility and transcription factor binding. We observed that the genomic location of several pluripotency factors and epigenetic modifiers are cell cycle regulated, and this allows induction of differentiation in the G1 phase.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Chromatin accessibility changes during cell cycle progression in hESCs
                             </h3>
                             <p id="p-6">
                              FUCCI-hESCs
                              <sup>
                               <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                              </sup>
                              were grown in chemically defined conditions maintaining pluripotency and then sorted based on their fluorescence (Early G1 (EG1), Late G1 (LG1), G1/S and S/G2/M) (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1a
                              </a>
                              ,
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Supplementary Information Fig. 1a, b
                              </a>
                              ). ATAC-seq (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1b
                              </a>
                              ,
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Supplementary Information Fig. 1c
                              </a>
                              ) was performed and a conservative set of peaks was generated using the IDR pipeline
                              <sup>
                               <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                              </sup>
                              . 88,479 reproducible peaks were identified in EG1, 117,283 in LG1, 99,109 in G1/S, 80,398 in S/G2/M (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Supplementary Information Fig. 1d
                              </a>
                              ). This suggests that chromatin shows a high degree of accessibility in all cell cycle phases in hESCs especially when compared to definitive Endoderm cells (dEN
                              <sup>
                               <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                                7
                               </a>
                              </sup>
                              ). Interestingly, the number of open regions was higher in the early cell cycle while slightly decreasing during the S/G2/M phase (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1c
                              </a>
                              ), thereby confirming that chromatin could narrowly close during progression towards cell division, as shown in two key regions upstream the pluripotency genes POU5F1 and NANOG (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1d
                              </a>
                              ). In total, we could identify 153,007 unique open regions. However, only ~33% of these regions were opened in all the phases (50,461) whereas ~67% of these regions were accessible in either a single phase (~34%) or more than one, but not all (~33%) (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig. 1e
                              </a>
                              ). Conservative pairwise comparisons
                              <sup>
                               <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                                8
                               </a>
                              </sup>
                              detected about 1,000 highly dynamic sites (|FC| >1.5 and corrected p-value p-adj < 1E-6), mostly changing through the EG1 phase: 715 regions from EG1 to LG1, 195 from S/G2/M to EG1 (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1f, g
                              </a>
                              ). These observations suggest that chromatin accessibility dynamically changes during cell cycle progression. Interestingly, regions opening in the EG1 phase were enriched for genes associated with transcriptional regulation, whereas regions closing from S/G2/M to EG1 were enriched for genes related to cell cycle (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig. 1h
                              </a>
                              ,
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               Supplementary Information Fig. 1e
                              </a>
                              ). This suggests that rewiring of the transcriptional networks required to establish cell identity could occur in the early phase of cell cycle, whereas mechanisms controlling cell division are induced later. Interestingly, changes around promoter regions are most frequent in the EG1 phase, confirming that chromatin around promoters rapidly reorganises after division to regulate transcription (
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig. 1i
                              </a>
                              ) thereby suggesting that the EG1 phase could be the most dynamic in terms of transcriptional regulation. In order to understand which transcription factors are regulating such changes, we performed digital footprinting analyses on the regions showing dynamic chromatin accessibility during cell cycle progression. Combining FLR
                              <sup>
                               <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                              </sup>
                              and Wellington
                              <sup>
                               <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                                10
                               </a>
                              </sup>
                              pipelines, we generated a conservative list of 1,807 footprints. Of these, 74% (1,330) were found in EG1, 8% (147) in LG1, 11% (203) in G1/S, and 7% (127) in S/G2/M (
                              <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                               Supplementary Information Fig. 1f
                              </a>
                              ). In all the phases, apart from the G1/S transition, the top-ranking footprint corresponded to the transcription factor CTCF (
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Fig. 1j, k
                              </a>
                              ;
                              <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                               Supplementary Information Fig. 1g
                              </a>
                              , Supplementary Table 1). More specifically, CTCF represented ~50% of all the footprints in the EG1 phase, dropping to ~10% in the other phases. A wider distribution of factors’ footprints could be detected in the rest of the cell cycle, with a gradual increase in FOX-protein footprints (FOXO3, FOXO4) which were previously described as pioneering factors able to bind condensed chromatin
                              <sup>
                               <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                              </sup>
                              . These data suggest that transcriptional networks are dynamically regulated during cell cycle progression in hESCs and that CTCF could play a role in chromatin regulations especially at the start of the cell cycle.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F1.large.jpg?width=800&height=600&carousel=1" title="Chromatin accessibility changes during cell cycle progression in hESCs. (a) FUCCI hESCs colony showing reporters for the 4 different cell cycle phases: Early G1 (EG1, no fluorescence), Late G1 (LG1, red), G1/S transition (G1/S, yellow), and S/G2/M (green). Scale bar 50μm. (b) Experimental approach: FUCCI hESCs were sorted in the 4 cell cycle phases and ATAC-seq was performed on biological duplicates. (c) Normalised ATAC-seq signal ±1kb around nucleosome free regions in EG1, LG1, G1/S, S/G2/M, compared to definitive Endoderm (dEN). (d) Genome browser tracks reporting chromatin accessibility (ATAC-seq) in cell cycle sorted hESCs at the POU5F1 and NANOG loci in biological duplicates. (e) Venn diagram showing ATAC-seq peaks overlap in all 4 cell cycle phases. (f) Number of differentially accessible peaks during cell cycle progression obtained by pairwise comparison (|FC| > 1.5, corrected p-value (p-adj) < 1E-6) and their distribution up vs down. (g) Heat-maps displaying normalised accessibility signals of differentially accessible regions from (f) averaged using 50bp bins. Regions are centred on the merged peaks for the two replicates ±1kb. K-means clustering corresponds to k=2. (h) Gene ontology for Biological Process of differentially accessible peaks in S/G2/M vs EG1 down, S/G2/M vs EG1 up (diffReps, Fig. 1f). Combined score is calculated by multiplying the unadjusted p-values with the z-score. (i) Feature distribution (expressed as percentage) for the ATAC-seq peaks for differentially open regions (diffReps) in EG1vsLG1 down, EG1vsLG1 up, S/G2/M vs EG1 down, S/G2/M vs EG1 up. (j) Footprints frequency (expressed as percentage) related to single transcription factors in EG1, LG1, G1/S, S/G2/M. (k) Average profile of CTCF footprint in the different cell cycle phases generated from Tn5 insertion frequencies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Chromatin accessibility changes during cell cycle progression in hESCs.</span> (<strong>a</strong>) FUCCI hESCs colony showing reporters for the 4 different cell cycle phases: Early G1 (EG1, no fluorescence), Late G1 (LG1, red), G1/S transition (G1/S, yellow), and S/G2/M (green). Scale bar 50μm. (<strong>b</strong>) Experimental approach: FUCCI hESCs were sorted in the 4 cell cycle phases and ATAC-seq was performed on biological duplicates. <strong>(c)</strong> Normalised ATAC-seq signal ±1kb around nucleosome free regions in EG1, LG1, G1/S, S/G2/M, compared to definitive Endoderm (dEN). <strong>(d)</strong> Genome browser tracks reporting chromatin accessibility (ATAC-seq) in cell cycle sorted hESCs at the <em>POU5F1</em> and <em>NANOG</em> loci in biological duplicates. <strong>(e)</strong> Venn diagram showing ATAC-seq peaks overlap in all 4 cell cycle phases. <strong>(f)</strong> Number of differentially accessible peaks during cell cycle progression obtained by pairwise comparison (|FC| > 1.5, corrected p-value (p-adj) < 1E-6) and their distribution up vs down. <strong>(g)</strong> Heat-maps displaying normalised accessibility signals of differentially accessible regions from (f) averaged using 50bp bins. Regions are centred on the merged peaks for the two replicates ±1kb. K-means clustering corresponds to k=2. <strong>(h)</strong> Gene ontology for Biological Process of differentially accessible peaks in S/G2/M vs EG1 down, S/G2/M vs EG1 up (diffReps, Fig. 1f). Combined score is calculated by multiplying the unadjusted p-values with the z-score. <strong>(i)</strong> Feature distribution (expressed as percentage) for the ATAC-seq peaks for differentially open regions (diffReps) in EG1vsLG1 down, EG1vsLG1 up, S/G2/M vs EG1 down, S/G2/M vs EG1 up. <strong>(j)</strong> Footprints frequency (expressed as percentage) related to single transcription factors in EG1, LG1, G1/S, S/G2/M. <strong>(k)</strong> Average profile of CTCF footprint in the different cell cycle phases generated from Tn5 insertion frequencies.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F1.medium.gif" width="440" height="365"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F1.medium.gif" width="440" height="365"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Chromatin accessibility changes during cell cycle progression in hESCs.
                               </span>
                               <p id="p-7" class="first-child">
                                (
                                <strong>
                                 a
                                </strong>
                                ) FUCCI hESCs colony showing reporters for the 4 different cell cycle phases: Early G1 (EG1, no fluorescence), Late G1 (LG1, red), G1/S transition (G1/S, yellow), and S/G2/M (green). Scale bar 50μm. (
                                <strong>
                                 b
                                </strong>
                                ) Experimental approach: FUCCI hESCs were sorted in the 4 cell cycle phases and ATAC-seq was performed on biological duplicates.
                                <strong>
                                 (c)
                                </strong>
                                Normalised ATAC-seq signal ±1kb around nucleosome free regions in EG1, LG1, G1/S, S/G2/M, compared to definitive Endoderm (dEN).
                                <strong>
                                 (d)
                                </strong>
                                Genome browser tracks reporting chromatin accessibility (ATAC-seq) in cell cycle sorted hESCs at the POU5F1 and NANOG loci in biological duplicates.
                                <strong>
                                 (e)
                                </strong>
                                Venn diagram showing ATAC-seq peaks overlap in all 4 cell cycle phases.
                                <strong>
                                 (f)
                                </strong>
                                Number of differentially accessible peaks during cell cycle progression obtained by pairwise comparison (|FC| > 1.5, corrected p-value (p-adj) < 1E-6) and their distribution up vs down.
                                <strong>
                                 (g)
                                </strong>
                                Heat-maps displaying normalised accessibility signals of differentially accessible regions from (f) averaged using 50bp bins. Regions are centred on the merged peaks for the two replicates ±1kb. K-means clustering corresponds to k=2.
                                <strong>
                                 (h)
                                </strong>
                                Gene ontology for Biological Process of differentially accessible peaks in S/G2/M vs EG1 down, S/G2/M vs EG1 up (diffReps,
                                <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                                 Fig. 1f
                                </a>
                                ). Combined score is calculated by multiplying the unadjusted p-values with the z-score.
                                <strong>
                                 (i)
                                </strong>
                                Feature distribution (expressed as percentage) for the ATAC-seq peaks for differentially open regions (diffReps) in EG1vsLG1 down, EG1vsLG1 up, S/G2/M vs EG1 down, S/G2/M vs EG1 up.
                                <strong>
                                 (j)
                                </strong>
                                Footprints frequency (expressed as percentage) related to single transcription factors in EG1, LG1, G1/S, S/G2/M.
                                <strong>
                                 (k)
                                </strong>
                                Average profile of CTCF footprint in the different cell cycle phases generated from Tn5 insertion frequencies.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Changes in chromatin accessibility during cell cycle are not associated with transcriptional variations
                             </h3>
                             <p id="p-8">
                              We next decided to assess whether changes in chromatin accessibility correlate with variations in gene expression. Therefore, RNA-seq was performed on FUCCI-hESCs sorted in the different cell cycle phases (EG1, LG1, S/G2/M;
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2a
                              </a>
                              and
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Supplementary Information Fig. 2a, b
                              </a>
                              ). Key pluripotency markers such as POU5F1 and NANOG did not show cell phase specific expression, whereas several cell cycle regulators such as CDK1 showed the expected pattern on expression (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2b
                              </a>
                              ;
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Supplementary Information Fig. 2c
                              </a>
                              ). Interestingly, in agreement with previous reports
                              <sup>
                               <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                              </sup>
                              , some developmental regulators such as SOX17 and GATA6 showed higher background expression in the LG1 phase. Differential expression analysis (DESeq2
                              <sup>
                               <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                              </sup>
                              ) revealed 394 genes showing cell cycle regulated expression (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2c, d
                              </a>
                              ). Gene Ontology and pathway analyses showed enrichment for cell cycle terms especially regulators of cell division (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2e
                              </a>
                              ). Hierarchical clustering identified three distinct patterns of expression (
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig. 2b, f
                              </a>
                              ). Cluster 1 included genes mostly induced during the S/G2/M phase such as CDK1, CDC25C, AURKA (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Supplementary Information Fig. 2c
                              </a>
                              ). Cluster 3 included a list of metabolic genes downregulated from EG1 to S/G2/M, suggesting that metabolic activity might decrease during cell cycle progression. In agreement with our finding of highly dynamic chromatin around the EG1 phase, this cluster included several histone genes clusters, and pathway analysis showed enrichment for chromatin rearrangement terms such as promoter opening and histones acetylation/deacetylation (
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               Supplementary Information Fig. 2d
                              </a>
                              ). Cluster 2 includes genes upregulated in LG1, some of which are related to endoderm formation and developmental processes, including EOMES, MIXL1, SOX17, FOXA2, GATA4 and GATA6. In parallel, to make sure that steady-state transcriptomic analysis was representative of subtle changes in dynamic expression, we performed RT-qPCR for key genes using intronic primers in order to interrogate nascent transcription events. This confirmed a similar pattern for developmental regulators as identified by RNA-seq (
                              <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                               Fig. 2g
                              </a>
                              ;
                              <a id="xref-fig-7-5" class="xref-fig" href="#F7">
                               Supplementary Information Fig. 2e, f
                              </a>
                              ). However, their expression level remains low when compared to induction observed during endoderm differentiation, suggesting priming of expression rather than full induction. In order to assess whether this transcriptional background correlates with changes in chromatin accessibility, we overlapped all of the 13,478 expressed transcripts (log
                              <sub>
                               2
                              </sub>
                              RPM > 1) with all the associated features of the ATAC-seq peaks: ~85% of expressed genes overlap with accessible regions (
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               Fig. 2h
                              </a>
                              ), confirming that most transcribed genes are indeed located in regions of open chromatin. We then overlapped the list of differentially expressed genes with the list of dynamic ATAC regions (from
                              <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                               Fig. 1f
                              </a>
                              ). Strikingly, only ~3% of differentially expressed genes overlapped with dynamic regions thereby indicating that the majority of cell cycle regulated transcripts originate from constantly open chromatin regions (
                              <a id="xref-fig-2-8" class="xref-fig" href="#F2">
                               Fig. 2i
                              </a>
                              ;
                              <a id="xref-fig-7-6" class="xref-fig" href="#F7">
                               Supplementary Information Fig. 2g
                              </a>
                              ). Taken together, these results suggest that developmental regulators are expressed at low level in the G1 phase, potentially marking the priming of hESCs for differentiation. However, this transcriptional “leakiness” originates mostly in constantly open chromatin regions suggesting that it is not induced by changes in accessibility but rather other mechanisms such as differential binding of transcription factors.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F2.large.jpg?width=800&height=600&carousel=1" title="Changes in chromatin accessibility during cell cycle are not associated with transcriptional variations. (a) Experimental approach. FUCCI hESCs were sorted in 3 different cell cycle phases: Early G1 (EG1), Late G1 (LG1), G1/S transition (G1/S), and S/G2/M. RNA-seq was performed following sorting on biological triplicates. (b) Heatmap reporting log2RPM expression of selected genes (pluripotency markers, developmental regulators, cell cycle) in three biological replicates. (c) Hierarchical clustering of 394 differentially expressed genes during cell cycle progression (p-value < 0.05, per probe normalised). Three main clusters and relative representative genes are reported. (d) Genome browser tracks reporting cell cycle regulated expression (RNA-seq) in sorted hESCs of POU5F1, CDK1 and SOX17 in 3 biological replicates. (e) Gene Ontology (Biological Processes and Reactome) for the 394 differentially expressed genes ranked by combined score. Combined score is calculated by multiplying unadjusted p-value with z-score. (f) Average expression and related gene Ontology (Biological Processes) for the three main clusters in (c), ranked by combined score. Combined score is calculated by multiplying unadjusted p-value with z-score. (g) RT-qPCR expression analysis of cell cycle sorted FUCCI hESCs using standard primers spanning exon-exon junctions (exonic) and primers designed over introns (intronic) to measure pre-processed transcripts for pluripotency and genes selected from (c). Expression is relative to the housekeeping gene RPLP0. (h) Venn diagram summarising overlap of expressed genes (RNA-seq, log2RPM > 1) and accessible chromatin (ATAC-seq). 85% represents the percentage of expressed genes present in open chromatin regions (overlapping ATAC peaks). (i) Venn diagram reporting overlap of differentially expressed genes (RNA-seq DESeq2) and regions changing accessibility (ATAC-seq DiffRep). 3% represents the percentage of differentially expressed genes overlapping with changes in chromatin accessibility." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Changes in chromatin accessibility during cell cycle are not associated with transcriptional variations.</span> <strong>(a)</strong> Experimental approach. FUCCI hESCs were sorted in 3 different cell cycle phases: Early G1 (EG1), Late G1 (LG1), G1/S transition (G1/S), and S/G2/M. RNA-seq was performed following sorting on biological triplicates. <strong>(b)</strong> Heatmap reporting log2RPM expression of selected genes (pluripotency markers, developmental regulators, cell cycle) in three biological replicates. <strong>(c)</strong> Hierarchical clustering of 394 differentially expressed genes during cell cycle progression (p-value < 0.05, per probe normalised). Three main clusters and relative representative genes are reported. <strong>(d)</strong> Genome browser tracks reporting cell cycle regulated expression (RNA-seq) in sorted hESCs of <em>POU5F1, CDK1</em> and <em>SOX17</em> in 3 biological replicates. <strong>(e)</strong> Gene Ontology (Biological Processes and Reactome) for the 394 differentially expressed genes ranked by combined score. Combined score is calculated by multiplying unadjusted p-value with z-score. <strong>(f)</strong> Average expression and related gene Ontology (Biological Processes) for the three main clusters in (c), ranked by combined score. Combined score is calculated by multiplying unadjusted p-value with z-score. <strong>(g)</strong> RT-qPCR expression analysis of cell cycle sorted FUCCI hESCs using standard primers spanning exon-exon junctions (exonic) and primers designed over introns (intronic) to measure pre-processed transcripts for pluripotency and genes selected from (c). Expression is relative to the housekeeping gene <em>RPLP0</em>. <strong>(h)</strong> Venn diagram summarising overlap of expressed genes (RNA-seq, log<sub>2</sub>RPM > 1) and accessible chromatin (ATAC-seq). 85% represents the percentage of expressed genes present in open chromatin regions (overlapping ATAC peaks). <strong>(i)</strong> Venn diagram reporting overlap of differentially expressed genes (RNA-seq DESeq2) and regions changing accessibility (ATAC-seq DiffRep). 3% represents the percentage of differentially expressed genes overlapping with changes in chromatin accessibility.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F2.medium.gif" width="440" height="438"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F2.medium.gif" width="440" height="438"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Changes in chromatin accessibility during cell cycle are not associated with transcriptional variations.
                               </span>
                               <p id="p-9" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Experimental approach. FUCCI hESCs were sorted in 3 different cell cycle phases: Early G1 (EG1), Late G1 (LG1), G1/S transition (G1/S), and S/G2/M. RNA-seq was performed following sorting on biological triplicates.
                                <strong>
                                 (b)
                                </strong>
                                Heatmap reporting log2RPM expression of selected genes (pluripotency markers, developmental regulators, cell cycle) in three biological replicates.
                                <strong>
                                 (c)
                                </strong>
                                Hierarchical clustering of 394 differentially expressed genes during cell cycle progression (p-value < 0.05, per probe normalised). Three main clusters and relative representative genes are reported.
                                <strong>
                                 (d)
                                </strong>
                                Genome browser tracks reporting cell cycle regulated expression (RNA-seq) in sorted hESCs of POU5F1, CDK1 and SOX17 in 3 biological replicates.
                                <strong>
                                 (e)
                                </strong>
                                Gene Ontology (Biological Processes and Reactome) for the 394 differentially expressed genes ranked by combined score. Combined score is calculated by multiplying unadjusted p-value with z-score.
                                <strong>
                                 (f)
                                </strong>
                                Average expression and related gene Ontology (Biological Processes) for the three main clusters in (c), ranked by combined score. Combined score is calculated by multiplying unadjusted p-value with z-score.
                                <strong>
                                 (g)
                                </strong>
                                RT-qPCR expression analysis of cell cycle sorted FUCCI hESCs using standard primers spanning exon-exon junctions (exonic) and primers designed over introns (intronic) to measure pre-processed transcripts for pluripotency and genes selected from (c). Expression is relative to the housekeeping gene RPLP0.
                                <strong>
                                 (h)
                                </strong>
                                Venn diagram summarising overlap of expressed genes (RNA-seq, log
                                <sub>
                                 2
                                </sub>
                                RPM > 1) and accessible chromatin (ATAC-seq). 85% represents the percentage of expressed genes present in open chromatin regions (overlapping ATAC peaks).
                                <strong>
                                 (i)
                                </strong>
                                Venn diagram reporting overlap of differentially expressed genes (RNA-seq DESeq2) and regions changing accessibility (ATAC-seq DiffRep). 3% represents the percentage of differentially expressed genes overlapping with changes in chromatin accessibility.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Transcription factors binding changes during cell cycle progression in hESCs
                             </h3>
                             <p id="p-10">
                              In order to test whether transcription factors are driving these transcriptional changes, we performed ChIP-seq in sorted FUCCI-H9 for different TFs (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3a
                              </a>
                              ). We first looked at CTCF binding in order to validate footprinting predictions. Genome wide mapping revealed 5,606 unique binding sites, of which 4,401 were found in EG1 confirming its higher occupancy in the early cell cycle (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3b-d
                              </a>
                              ). This pattern was confirmed looking at previously identified key loci
                              <sup>
                               <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                              </sup>
                              such as the LEFTY/MIXL1, where CTCF anchor sites that regulate this super enhancer are strongly present in the EG1 phase, and gradually decrease during cell cycle progression (
                              <a id="xref-fig-2-9" class="xref-fig" href="#F2">
                               Fig. 2e
                              </a>
                              ). Interestingly, this locus also includes the MIXL1 and H3F3A genes (
                              <a id="xref-fig-2-10" class="xref-fig" href="#F2">
                               Fig. 2f
                              </a>
                              ), the latter being the top hit for CTCF differential binding analysis (
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3a
                              </a>
                              , Supplementary Table 2). H3F3A encodes for the histone H3.3, previously described as being enriched at the centre of the intergenic CTCF-binding sites
                              <sup>
                               <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                              </sup>
                              , necessary for correct cell cycle progression and to regulate heterochromatin structures
                              <sup>
                               <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              . Of the 708 CTCF footprints identified using ATAC-seq data ~60% overlapped with CTCF ChIP-seq confirming the robustness of our prediction (
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3b
                              </a>
                              ). Interestingly, 77% of CTCF binding sites were located in open chromatin regions, with a higher overlap in the EG1 phase (
                              <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3c
                              </a>
                              ), thereby suggesting a specific role for CTCF in regulating chromatin accessibility in this phase of the cell cycle. All together, these results show that CTCF genome wide occupancy changes during cell cycle progression. The increased binding of CTCF in EG1 supports the hypothesis that CTCF could drive chromatin restructuring to regulate transcription
                              <sup>
                               <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              and genome integrity
                              <sup>
                               <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                                18
                               </a>
                              </sup>
                              during the subsequent cell cycle.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F3.large.jpg?width=800&height=600&carousel=1" title="Transcription factors binding changes during cell cycle progression in hESCs. (a) Experimental approach. FUCCI hESCs were sorted in 3 different cell cycle phases: Early G1 (EG1), Late G1 (LG1), and S/G2/M. ChIP-seq was performed following sorting on biological duplicates for CTCF, OCT4, NANOG, and SOX2. (b) Heatmaps displaying CTCF binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. (c) Normalised ChIP-seq signal ±1kb around all peaks in EG1, LG1, S/G2/M, compared to INPUT. (d) Venn diagram reporting overlap of CTCF binding sites during cell cycle. (e) Genome browser tracks reporting CTCF binding in cell cycle sorted hESCs at the LEFTY locus. A selected CTCF-CTCF loop14 is depicted as a black line. A subset of genes present in this loop is shown for simplicity. (f) Genome browser tracks reporting CTCF binding in cell cycle sorted hESCs at the LEFTY/H3F3A/MIXL1 adjacent loci. Relevant genes are highlighted in red. (g) Heatmaps displaying OCT4 binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. (h) Normalised OCT4 ChIP-seq signal ±1kb around all peaks in EG1, LG1, S/G2/M, compared to INPUT. (i) Venn diagram showing peaks overlap for OCT4 ChIP-seq in EG1, LG1, S/G2/M. (j) Genome browser tracks reporting OCT4 binding in cell cycle sorted hESCs at the NANOG promoter. (k) Heatmaps displaying NANOG and SOX2 binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. (l) Venn diagram showing binding sites overlap for OCT4, NANOG and SOX2. (m) Heatmaps displaying RING1B binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. (n) Gene ontology for Encode ChEA consensus sequences of RING1B EG1 and S/G2/M consensus sequences. Combined score is calculated by multiplying the unadjusted p-values with the z-score. (o) Overlap of RING1B bound genes with highly expressed genes by RNA-seq (genes are ranked based on their expression, and the top 20% are selected). For all the heat-maps, regions are centred on the merged peaks for the two replicates ±1kb in all the phases and compared to INPUT." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Transcription factors binding changes during cell cycle progression in hESCs.</span> (<strong>a</strong>) Experimental approach. FUCCI hESCs were sorted in 3 different cell cycle phases: Early G1 (EG1), Late G1 (LG1), and S/G2/M. ChIP-seq was performed following sorting on biological duplicates for CTCF, OCT4, NANOG, and SOX2. <strong>(b)</strong> Heatmaps displaying CTCF binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. <strong>(c)</strong> Normalised ChIP-seq signal ±1kb around all peaks in EG1, LG1, S/G2/M, compared to INPUT. <strong>(d)</strong> Venn diagram reporting overlap of CTCF binding sites during cell cycle. <strong>(e)</strong> Genome browser tracks reporting CTCF binding in cell cycle sorted hESCs at the <em>LEFTY</em> locus. A selected CTCF-CTCF loop<sup>14</sup> is depicted as a black line. A subset of genes present in this loop is shown for simplicity. <strong>(f)</strong> Genome browser tracks reporting CTCF binding in cell cycle sorted hESCs at the <em>LEFTY/H3F3A/MIXL1</em> adjacent loci. Relevant genes are highlighted in red. <strong>(g)</strong> Heatmaps displaying OCT4 binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. <strong>(h)</strong> Normalised OCT4 ChIP-seq signal ±1kb around all peaks in EG1, LG1, S/G2/M, compared to INPUT. <strong>(i)</strong> Venn diagram showing peaks overlap for OCT4 ChIP-seq in EG1, LG1, S/G2/M. <strong>(j)</strong> Genome browser tracks reporting OCT4 binding in cell cycle sorted hESCs at the <em>NANOG</em> promoter. <strong>(k)</strong> Heatmaps displaying NANOG and SOX2 binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. <strong>(l)</strong> Venn diagram showing binding sites overlap for OCT4, NANOG and SOX2. <strong>(m)</strong> Heatmaps displaying RING1B binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT. <strong>(n)</strong> Gene ontology for Encode ChEA consensus sequences of RING1B EG1 and S/G2/M consensus sequences. Combined score is calculated by multiplying the unadjusted p-values with the z-score. <strong>(o)</strong> Overlap of RING1B bound genes with highly expressed genes by RNA-seq (genes are ranked based on their expression, and the top 20% are selected). For all the heat-maps, regions are centred on the merged peaks for the two replicates ±1kb in all the phases and compared to INPUT.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F3.medium.gif" width="440" height="423"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F3.medium.gif" width="440" height="423"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Transcription factors binding changes during cell cycle progression in hESCs.
                               </span>
                               <p id="p-11" class="first-child">
                                (
                                <strong>
                                 a
                                </strong>
                                ) Experimental approach. FUCCI hESCs were sorted in 3 different cell cycle phases: Early G1 (EG1), Late G1 (LG1), and S/G2/M. ChIP-seq was performed following sorting on biological duplicates for CTCF, OCT4, NANOG, and SOX2.
                                <strong>
                                 (b)
                                </strong>
                                Heatmaps displaying CTCF binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT.
                                <strong>
                                 (c)
                                </strong>
                                Normalised ChIP-seq signal ±1kb around all peaks in EG1, LG1, S/G2/M, compared to INPUT.
                                <strong>
                                 (d)
                                </strong>
                                Venn diagram reporting overlap of CTCF binding sites during cell cycle.
                                <strong>
                                 (e)
                                </strong>
                                Genome browser tracks reporting CTCF binding in cell cycle sorted hESCs at the LEFTY locus. A selected CTCF-CTCF loop
                                <sup>
                                 <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                  14
                                 </a>
                                </sup>
                                is depicted as a black line. A subset of genes present in this loop is shown for simplicity.
                                <strong>
                                 (f)
                                </strong>
                                Genome browser tracks reporting CTCF binding in cell cycle sorted hESCs at the LEFTY/H3F3A/MIXL1 adjacent loci. Relevant genes are highlighted in red.
                                <strong>
                                 (g)
                                </strong>
                                Heatmaps displaying OCT4 binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT.
                                <strong>
                                 (h)
                                </strong>
                                Normalised OCT4 ChIP-seq signal ±1kb around all peaks in EG1, LG1, S/G2/M, compared to INPUT.
                                <strong>
                                 (i)
                                </strong>
                                Venn diagram showing peaks overlap for OCT4 ChIP-seq in EG1, LG1, S/G2/M.
                                <strong>
                                 (j)
                                </strong>
                                Genome browser tracks reporting OCT4 binding in cell cycle sorted hESCs at the NANOG promoter.
                                <strong>
                                 (k)
                                </strong>
                                Heatmaps displaying NANOG and SOX2 binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT.
                                <strong>
                                 (l)
                                </strong>
                                Venn diagram showing binding sites overlap for OCT4, NANOG and SOX2.
                                <strong>
                                 (m)
                                </strong>
                                Heatmaps displaying RING1B binding in the different cell cycle phases EG1, LG1, S/G2/M, compared to INPUT.
                                <strong>
                                 (n)
                                </strong>
                                Gene ontology for Encode ChEA consensus sequences of RING1B EG1 and S/G2/M consensus sequences. Combined score is calculated by multiplying the unadjusted p-values with the z-score.
                                <strong>
                                 (o)
                                </strong>
                                Overlap of RING1B bound genes with highly expressed genes by RNA-seq (genes are ranked based on their expression, and the top 20% are selected). For all the heat-maps, regions are centred on the merged peaks for the two replicates ±1kb in all the phases and compared to INPUT.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-12">
                              In order to check whether this binding pattern could be shared with other TFs or it is uniquely related to the role of CTCF in the early cell cycle, we performed ChIP-seq on sorted FUCCI-H9 for the core pluripotency factors OCT4, NANOG and SOX2 (OSN). Interestingly, we found a different binding pattern shared between the three factors with higher genome occupancy in the late cell cycle. For OCT4, we found a total of 4,843 binding sites distributed as follows: 611 in EG1, 1,378 in LG1, and 2,854 in S/G2/M (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3g-i
                              </a>
                              ) showing that the number of genomic regions bound by this key pluripotency factor increases during cell cycle progression, as shown for the NANOG locus (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3j
                              </a>
                              ). The same binding pattern was observed for NANOG and SOX2 with both factors binding the genome more frequently in the S/G2/M phase (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig. 3k
                              </a>
                              ;
                              <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3d, e
                              </a>
                              ). OSN combinatorial analysis (
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig. 3l
                              </a>
                              ) showed that of the 5,904 unique peaks, 30% (1,767) were shared, with the majority involved in signalling pathways regulating pluripotency (
                              <a id="xref-fig-8-5" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3g
                              </a>
                              ). Single factors revealed specific features: OCT4 peaks were enriched for deubiquitination terms, NANOG for insulin receptor signalling pathway, and SOX2 involved in neuron differentiation.
                             </p>
                             <p id="p-13">
                              Intrigued by these findings, we asked whether additional factors known to be linked to the core transcriptional network might behave similarly. For that we performed ChIP-seq for RING1B, a chromatin modifier shown to partially bind chromatin in an OCT4-dependent manner
                              <sup>
                               <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                                19
                               </a>
                              </sup>
                              . These analyses revealed a unique binding pattern with high occupancy in the EG1 phase, a decrease in the LG1, and an increase in the S/G2/M (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig. 3m
                              </a>
                              ,
                              <a id="xref-fig-8-6" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3f
                              </a>
                              ). Interestingly, RING1B is known to have several roles from controlling chromatin compaction to regulating gene expression
                              <sup>
                               <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                              </sup>
                              . Thus, our observations could imply that these divergent roles might have a temporal distribution during cell cycle progression. Of all the bound sites, gene ontology analysis for Biological Process highlighted its role in regulating transcription (
                              <a id="xref-fig-8-7" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3h
                              </a>
                              ). Interestingly, regions bound by RING1B in EG1 were enriched for CTCF binding sites together with SMC3, a core subunit of the cohesin complex (
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Fig. 3n
                              </a>
                              ), suggesting that RING1B might have a structural role in opening chromatin at the start of the cell cycle. However, pluripotency factors such as OCT4 are enriched in S/G2/M (
                              <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                               Fig. 3n
                              </a>
                              ), validated by the stronger binding of OCT4 at binding sites shared with RING1B in S/G2/M (
                              <a id="xref-fig-8-8" class="xref-fig" href="#F8">
                               Supplementary Information Fig. 3i
                              </a>
                              ). To further validate this dual role at the transcriptional level, we then overlapped RING1B peaks with top 20% expressed genes, and found that ~50% of the genes bound by RING1B are highly expressed (
                              <a id="xref-fig-3-10" class="xref-fig" href="#F3">
                               Fig. 3o
                              </a>
                              ), reinforcing the hypothesis that RING1B does not only have a repressive role but it is also bound to highly transcribed genes. Taken together, these findings show that transcription factors have different binding patterns during cell cycle progression and that this regulation could define their activity.
                             </p>
                             <p id="p-14">
                              The dynamic binding of these transcription factors could be explained by a variety of mechanisms including variation in their expression during cell cycle progression. However, RNA-seq and RT-qPCR validations showed that the expression of these genes does not change between cell cycle phases (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4a, b
                              </a>
                              ). For this reason, we looked at the protein level of these TFs in sorted FUCCI-H9. Pluripotency factors’ expression levels were quite variable and did not show a clear expression pattern (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4c, d
                              </a>
                              ;
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Supplementary Information Fig. 4a
                              </a>
                              ). Therefore, variations in the protein levels cannot explain their differential binding. Interestingly, CTCF instead showed a trend of higher expression in the early cell cycle, in line with its chromatin binding (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4e, f
                              </a>
                              ), suggesting that different kinds of regulation are present during cell cycle progression. For pluripotency factors instead, we hypothesised that post-translational modifications could regulate their differential binding. Of particular interest is the phosphorylation at Serine 236 of OCT4 (pOCT4) which is known to regulate OCT4 binding to chromatin and its transcriptional activity in hESCs
                              <sup>
                               <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                                21
                               </a>
                              </sup>
                              . This modification is known to be the equivalent phosphorylation of mouse Serine 229, which sterically hinders DNA binding
                              <sup>
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                              </sup>
                              . To confirm that such a mechanism could control OCT4 binding in EG1, we performed immunostaining for pOCT4 (Ser236) in FUCCI-hESCs. These analyses showed that mitotic cells have enriched phosphorylation signal, thereby confirming findings in mouse Embryonic Stem Cell (mESCs)
                              <sup>
                               <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                              </sup>
                              . However, hESCs in EG1 still partially retain pOCT4 (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4g
                              </a>
                              ;
                              <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                               Supplementary Information Fig. 4b, c
                              </a>
                              ; white circles), suggesting that complete dephosphorylation only happens past the EG1 phase. This was further validated with flow cytometry, showing that a similar fraction of cells retains pOCT4 in S/G2/M and EG1 compared to unsorted cells and LG1 (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig. 4h, i
                              </a>
                              ;
                              <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                               Supplementary Information Fig. 4d
                              </a>
                              ). Taken together, these results suggest that phosphorylation could regulate the genomic binding of OCT4 during cell cycle progression also in hESCs. Importantly, EG1 has been shown to be a key phase during cell cycle for cell fate decision in hESCs
                              <sup>
                               <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                              </sup>
                              . For this reason, we hypothesised that the reduced capacity of pluripotency factors to bind the genome in EG1 could be necessary for induction of differentiation.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F4.large.jpg?width=800&height=600&carousel=1" title="Post-translational modification of OCT4 regulates its chromatin binding. (a) RT-qPCR for OCT4, NANOG and SOX2 in EG1, LG1, and S/G2/M relative to housekeeping gene PBGD (two-ways ANOVA, n=3). (b) RNA-seq data for OCT4, SOX2, and NANOG, in EG1, LG1, and S/G2/M, expressed in log2RPM. (c) Representative western-blot for OCT4, SOX and NANOG in sorted EG1, LG1, and S/G2/M, plus vinculin (Vin) as loading control. (d) Relative quantification of (c) performed using the software Fiji (ImageJ) reporting OCT4/SOX2/NANOG to vinculin ratio. Each dot represents a biological replicate (n=3). (e) Representative western-blot for CTCF in sorted EG1, LG1, and S/G2/M, plus vinculin (Vin) as loading control. (f) Relative quantification of (e) performed using the software Fiji (ImageJ) reporting CTCF to vinculin ratio. Each dot represents a biological replicate (n=3). (g) Immunofluorescence for phospho-OCT4 Ser236 (pOCT4, magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M cells in green-mAG, LG1 cells in red-mKO2, EG1 cells no colour). Bottom two panels report higher magnification of numbered circled cells in the EG1 phase. Scale bars: top panel 50μm, bottom panels 20μm. (h) Flow cytometry plots of FUCCI hESCs positive for pOCT4 in all cell cycle phases. Unstained cells and secondary (2nd only) reported as negative controls. (i) Quantification (expressed as percentage) of sorted FUCCI hESCs positive for pOCT4 divided by cell cycle phase compared to the unsorted control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Post-translational modification of OCT4 regulates its chromatin binding.</span> <strong>(a)</strong> RT-qPCR for <em>OCT4, NANOG</em> and <em>SOX2</em> in EG1, LG1, and S/G2/M relative to housekeeping gene <em>PBGD</em> (two-ways ANOVA, n=3). <strong>(b)</strong> RNA-seq data for <em>OCT4, SOX2</em>, and <em>NANOG</em>, in EG1, LG1, and S/G2/M, expressed in log<sub>2</sub>RPM. <strong>(c)</strong> Representative western-blot for OCT4, SOX and NANOG in sorted EG1, LG1, and S/G2/M, plus vinculin (Vin) as loading control. <strong>(d)</strong> Relative quantification of (c) performed using the software Fiji (ImageJ) reporting OCT4/SOX2/NANOG to vinculin ratio. Each dot represents a biological replicate (n=3). <strong>(e)</strong> Representative western-blot for CTCF in sorted EG1, LG1, and S/G2/M, plus vinculin (Vin) as loading control. <strong>(f)</strong> Relative quantification of (e) performed using the software Fiji (ImageJ) reporting CTCF to vinculin ratio. Each dot represents a biological replicate (n=3). <strong>(g)</strong> Immunofluorescence for phospho-OCT4 Ser236 (pOCT4, magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M cells in green-mAG, LG1 cells in red-mKO2, EG1 cells no colour). Bottom two panels report higher magnification of numbered circled cells in the EG1 phase. Scale bars: top panel 50μm, bottom panels 20μm. <strong>(h)</strong> Flow cytometry plots of FUCCI hESCs positive for pOCT4 in all cell cycle phases. Unstained cells and secondary (2nd only) reported as negative controls. <strong>(i)</strong> Quantification (expressed as percentage) of sorted FUCCI hESCs positive for pOCT4 divided by cell cycle phase compared to the unsorted control.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F4.medium.gif" width="440" height="324"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F4.medium.gif" width="440" height="324"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Post-translational modification of OCT4 regulates its chromatin binding.
                               </span>
                               <p id="p-15" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                RT-qPCR for OCT4, NANOG and SOX2 in EG1, LG1, and S/G2/M relative to housekeeping gene PBGD (two-ways ANOVA, n=3).
                                <strong>
                                 (b)
                                </strong>
                                RNA-seq data for OCT4, SOX2, and NANOG, in EG1, LG1, and S/G2/M, expressed in log
                                <sub>
                                 2
                                </sub>
                                RPM.
                                <strong>
                                 (c)
                                </strong>
                                Representative western-blot for OCT4, SOX and NANOG in sorted EG1, LG1, and S/G2/M, plus vinculin (Vin) as loading control.
                                <strong>
                                 (d)
                                </strong>
                                Relative quantification of (c) performed using the software Fiji (ImageJ) reporting OCT4/SOX2/NANOG to vinculin ratio. Each dot represents a biological replicate (n=3).
                                <strong>
                                 (e)
                                </strong>
                                Representative western-blot for CTCF in sorted EG1, LG1, and S/G2/M, plus vinculin (Vin) as loading control.
                                <strong>
                                 (f)
                                </strong>
                                Relative quantification of (e) performed using the software Fiji (ImageJ) reporting CTCF to vinculin ratio. Each dot represents a biological replicate (n=3).
                                <strong>
                                 (g)
                                </strong>
                                Immunofluorescence for phospho-OCT4 Ser236 (pOCT4, magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M cells in green-mAG, LG1 cells in red-mKO2, EG1 cells no colour). Bottom two panels report higher magnification of numbered circled cells in the EG1 phase. Scale bars: top panel 50μm, bottom panels 20μm.
                                <strong>
                                 (h)
                                </strong>
                                Flow cytometry plots of FUCCI hESCs positive for pOCT4 in all cell cycle phases. Unstained cells and secondary (2nd only) reported as negative controls.
                                <strong>
                                 (i)
                                </strong>
                                Quantification (expressed as percentage) of sorted FUCCI hESCs positive for pOCT4 divided by cell cycle phase compared to the unsorted control.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Cell cycle specific OCT4 binding is required for induction of differentiation
                             </h3>
                             <p id="p-16">
                              To test whether pluripotency factor binding regulates initiation of differentiation, we developed two FUCCI-hESC lines to respectively over-express or knock-down OCT4 in a cell cycle phase specific manner (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5a
                              </a>
                              ). For that, we used a tamoxifen (4OHT) inducible form of OCT4 (OCT4-ERT2), that relies on the fusion of OCT4 with a modified fragment of the Estrogen receptor
                              <sup>
                               <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                24
                               </a>
                              </sup>
                              (
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Supplementary Information Fig. 5a
                              </a>
                              ). Western blot confirmed correct integration (
                              <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                               Supplementary Information Fig. 5b
                              </a>
                              ) and immunostaining confirmed correct nuclear translocation of OCT4 upon addition of 4-OHT in as little as 4h in both pluripotent and neuroectoderm cells (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5b
                              </a>
                              ). We then sorted OCT4-ERT2-FUCCI-hESCs in EG1 in order to start the differentiation when OCT4 is not bound to chromatin, and re-plated them in culture condition inducing endoderm differentiation
                              <sup>
                               <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                                25
                               </a>
                               ,
                               <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                              </sup>
                              for 24h in the presence or absence of 4-OHT for the first 4h. Interestingly, OCT4 induction decreases the expression of the key early endoderm markers such as T, MIXL1 and GSC (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5c
                              </a>
                              ) suggesting a less efficient differentiation. In parallel, we generated an inducible hESCs for the knockdown of OCT4 expression (iKD-OCT4-FUCCI-hESCs), taking advantage of the Tetracycline (TET) based system OptiKD
                              <sup>
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                              </sup>
                              (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5d
                              </a>
                              ;
                              <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                               Supplementary Information Fig. 5c, d
                              </a>
                              ). iKD-OCT4-FUCCI-hESCs sorted in S/G2/M were re-plated in culture condition inducing endoderm differentiation for 24h in presence or absence of TET, in order to deplete OCT4 when its binding is at its peak during the cell cycle. Interestingly, decrease of OCT4 in the S/G2/M phase promoted the expression of late endoderm (SOX17) and mesoderm (HAND1) markers (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig. 5e
                              </a>
                              ) thereby suggesting a more efficient differentiation process. Taken together, these results show that variation in OCT4 binding during cell cycle is required for optimal cell fate decision in hESCs or more broadly, that cell cycle specific regulation of transcriptional networks could control capacity of differentiation in hESCs.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F5.large.jpg?width=800&height=600&carousel=1" title="Cell cycle specific OCT4 binding is required for induction of differentiation. (a) Experimental approach: FUCCI OCT4-ERT2 were sorted in Early G1 (EG1) when OCT4 binding is at its minimum and re-plated in Endoderm inducing conditions (dEN). OCT4 nuclear translocation was induced upon 4-Hydroxytamoxifen addition (4OHT). FUCCI iKD OCT4 were sorted in S/G2/M when OCT4 binding is at its max and re-plated in dEN conditions. OCT4 knock-down was induced upon tetracycline addition (TET). (b) Immunofluorescence for OCT4 (magenta) and nuclear staining DAPI (blue) in FUCCI OCT4-ERT2. In presence of 4-Hydroxytamoxifen (+4OHT), OCT4 is uniquely nuclear. Upon neuroectoderm induction (SB), endogenous OCT4 is strongly reduced and upon 4-Hydroxytamoxifen induction is translocated into the nucleus (SB +4OHT). Scale bar 50μm. (c) RT-qPCR expression of endoderm markers T, MIXL1, and GSC in presence or absence of 4 hours of 4-Hydroxytamoxifen (+4OHT) in EG1 sorted cells after 24h of differentiation. Expression reported as relative to housekeeping gene PBGD (d) RT-qPCR reporting OCT4 expression time-course (0-72 hours) during knock-down in presence of tetracycline (+TET) compared to -TET control. Expression relative to housekeeping gene PBGD. (e) RT-qPCR reporting expression of late endoderm markers SOX17 and HAND1 in presence or absence of tetracycline (TET) in S/G2/M sorted cells after 24h of differentiation. (f) Schematic summary of the proposed working model. In the EG1 phase, immediately after cell division, chromatin is highly dynamic and CTCF binding is proposed to regulate changes in its accessibility. Core pluripotency factors OCT4, SOX2 and NANOG (OSN) show low occupancy in the G1 phase, and this allows hESCs to respond to differentiation cues. In the S/G2/M phase, core pluripotency factors fully re-bind chromatin, re-establishing the pluripotency network while sustaining self-renewal and blocking differentiation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Cell cycle specific OCT4 binding is required for induction of differentiation.</span> <strong>(a)</strong> Experimental approach: FUCCI OCT4-ERT2 were sorted in Early G1 (EG1) when OCT4 binding is at its minimum and re-plated in Endoderm inducing conditions (dEN). OCT4 nuclear translocation was induced upon 4-Hydroxytamoxifen addition (4OHT). FUCCI iKD OCT4 were sorted in S/G2/M when OCT4 binding is at its max and re-plated in dEN conditions. OCT4 knock-down was induced upon tetracycline addition (TET). <strong>(b)</strong> Immunofluorescence for OCT4 (magenta) and nuclear staining DAPI (blue) in FUCCI OCT4-ERT2. In presence of 4-Hydroxytamoxifen (+4OHT), OCT4 is uniquely nuclear. Upon neuroectoderm induction (SB), endogenous OCT4 is strongly reduced and upon 4-Hydroxytamoxifen induction is translocated into the nucleus (SB +4OHT). Scale bar 50μm. <strong>(c)</strong> RT-qPCR expression of endoderm markers <em>T</em>, <em>MIXL1</em>, and <em>GSC</em> in presence or absence of 4 hours of 4-Hydroxytamoxifen (+4OHT) in EG1 sorted cells after 24h of differentiation. Expression reported as relative to housekeeping gene <em>PBGD</em> <strong>(d)</strong> RT-qPCR reporting OCT4 expression time-course (0-72 hours) during knock-down in presence of tetracycline (+TET) compared to -TET control. Expression relative to housekeeping gene <em>PBGD</em>. <strong>(e)</strong> RT-qPCR reporting expression of late endoderm markers <em>SOX17</em> and <em>HAND1</em> in presence or absence of tetracycline (TET) in S/G2/M sorted cells after 24h of differentiation. <strong>(f)</strong> Schematic summary of the proposed working model. In the EG1 phase, immediately after cell division, chromatin is highly dynamic and CTCF binding is proposed to regulate changes in its accessibility. Core pluripotency factors OCT4, SOX2 and NANOG (OSN) show low occupancy in the G1 phase, and this allows hESCs to respond to differentiation cues. In the S/G2/M phase, core pluripotency factors fully re-bind chromatin, re-establishing the pluripotency network while sustaining self-renewal and blocking differentiation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F5.medium.gif" width="440" height="182"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F5.medium.gif" width="440" height="182"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Cell cycle specific OCT4 binding is required for induction of differentiation.
                               </span>
                               <p id="p-17" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Experimental approach: FUCCI OCT4-ERT2 were sorted in Early G1 (EG1) when OCT4 binding is at its minimum and re-plated in Endoderm inducing conditions (dEN). OCT4 nuclear translocation was induced upon 4-Hydroxytamoxifen addition (4OHT). FUCCI iKD OCT4 were sorted in S/G2/M when OCT4 binding is at its max and re-plated in dEN conditions. OCT4 knock-down was induced upon tetracycline addition (TET).
                                <strong>
                                 (b)
                                </strong>
                                Immunofluorescence for OCT4 (magenta) and nuclear staining DAPI (blue) in FUCCI OCT4-ERT2. In presence of 4-Hydroxytamoxifen (+4OHT), OCT4 is uniquely nuclear. Upon neuroectoderm induction (SB), endogenous OCT4 is strongly reduced and upon 4-Hydroxytamoxifen induction is translocated into the nucleus (SB +4OHT). Scale bar 50μm.
                                <strong>
                                 (c)
                                </strong>
                                RT-qPCR expression of endoderm markers T, MIXL1, and GSC in presence or absence of 4 hours of 4-Hydroxytamoxifen (+4OHT) in EG1 sorted cells after 24h of differentiation. Expression reported as relative to housekeeping gene PBGD
                                <strong>
                                 (d)
                                </strong>
                                RT-qPCR reporting OCT4 expression time-course (0-72 hours) during knock-down in presence of tetracycline (+TET) compared to -TET control. Expression relative to housekeeping gene PBGD.
                                <strong>
                                 (e)
                                </strong>
                                RT-qPCR reporting expression of late endoderm markers SOX17 and HAND1 in presence or absence of tetracycline (TET) in S/G2/M sorted cells after 24h of differentiation.
                                <strong>
                                 (f)
                                </strong>
                                Schematic summary of the proposed working model. In the EG1 phase, immediately after cell division, chromatin is highly dynamic and CTCF binding is proposed to regulate changes in its accessibility. Core pluripotency factors OCT4, SOX2 and NANOG (OSN) show low occupancy in the G1 phase, and this allows hESCs to respond to differentiation cues. In the S/G2/M phase, core pluripotency factors fully re-bind chromatin, re-establishing the pluripotency network while sustaining self-renewal and blocking differentiation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-7">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-18">
                             Our study investigates for the first time the dynamic mechanisms that coordinate chromatin accessibility, gene expression, and transcriptional networks during cell cycle progression in hESCs. In the EG1 phase, immediately after cell division, chromatin is highly dynamic and CTCF appears to regulate changes in DNA accessibility. Indeed, CTCF might be the driver for reorganising chromatin after chromosome condensation during cell division
                             <sup>
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ,
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                             </sup>
                             in conjunction with other chromatin modifiers such as RING1B
                             <sup>
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                             </sup>
                             . In parallel, the core pluripotency factors OCT4, SOX2 and NANOG show low occupancy in the G1 phase allowing hESCs to rapidly respond to differentiation cues. In the S/G2/M phase, they fully bind chromatin to re-establish the pluripotency network, sustain self-renewal and block differentiation. These results reinforce previous studies showing that this phase actively promotes the pluripotent state
                             <sup>
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                             </sup>
                             . This change in binding might be regulated via post-translational modifications such as phosphorylation, preventing OCT4 from binding DNA in EG1
                             <sup>
                              <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                             </sup>
                             . Importantly, these changes could define the temporal function of RING1B which might help CTCF in restructuring chromatin in EG1 while acting as transcriptional regulator in S/G2/M to maintain pluripotency
                             <sup>
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                             </sup>
                             . Ultimately, these cell cycle specific interplays are necessary to coordinate induction of differentiation and exit of pluripotency with cell division.
                            </p>
                            <p id="p-19">
                             Our results also demonstrate that key transcription factors are cell cycle regulated and dynamically change their genomic location. This binding pattern suggests that transcription factor activity and specificity are coordinated with cell cycle progression. This process could represent a key mechanism that allows stem cells to not only maintain their capacity to change cellular identity but also for transcription factors to have different functions based on the cell cycle state of the cells. Importantly, these findings highlight the importance of studying transcriptional and epigenetic mechanisms in the context of the cell cycle. More generally, most ChIP-seq studies are currently capturing transcription factors’ binding on the most extended cell cycle phase which can vary from cell to cell. Thus, we would like to propose that genomic location of transcription factors should be investigated in a cell cycle dependent manner, since phase-specific analyses could reveal new molecular functions even for commonly studied regulators. This novel concept could apply to any cell type and thus have a major impact on our understanding of transcription factors’ functions.
                            </p>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-21">
                             A.O. conceived the study, performed all the experiments, analysed data and wrote the manuscript. S.B. helped with experimental planning, data analysis and reviewed the manuscript. L.V. conceived, supervised, and supported the study, wrote and provided final approval of the manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Declaration of interests
                            </h2>
                            <p id="p-22">
                             The authors declare no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Methods
                            </h2>
                            <div class="section" id="sec-11">
                             <h3>
                              hESCs culture and germ layers specification
                             </h3>
                             <p id="p-23">
                              FUCCI-hESCs (H9/WA09 line; WiCell) were grown feeder- and serum-free as previously described
                              <sup>
                               <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                32
                               </a>
                               ,
                               <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                33
                               </a>
                              </sup>
                              . Briefly, cells were grown in either in-house E6 supplemented with 25ng/ml Fibroblast growth factor 2 (FGF2) and 2ng/ml TGFβ (BioTechne) or in Chemically Defined Medium (CDM), supplemented with Bovine Serum Albumin (BSA), recombinant Activin-A (10 ng/ml) and 12 ng/ml FGF2 on 10μg/ml Vitronectin-XF (STEMCELL Technologies) or 0.1% Gelatin and MEF-medium coated plates, respectively (Supplementary Table 3). If not specified, growth factors were provided by Dr Marko Hyvonen (Department of Biochemistry, University of Cambridge). Cells were passaged weekly using either EDTA or Collagenase IV and plated as clumps of 50-100 cells dispensed at a density of 100-150 clumps/cm
                              <sup>
                               2
                              </sup>
                              . All the FUCCI-hESCs lines used were kept under selection with Puromycin (1mM) and Geneticin (G-418, 0.2 mg/ml), or alternatively G-418 (0.2 mg/ml) and Blasticidin (1mM). Definitive Endoderm (dEN) was induced for 3 days in CDM-Polyvinyl alcohol (PVA) without insulin supplemented with 20ng/ml FGF2, 10μM LY294002, 100ng/ml Activin A, and 10ng/ml BMP4, changing media daily, as previously described
                              <sup>
                               <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                                25
                               </a>
                               ,
                               <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                              </sup>
                              . Neuroectoderm was induced for 6 days in CDM-BSA supplemented with 12ng/ml FGF2 and 10μM SB431542 (Activin/Nodal/TGFβ inhibitor; Tocris), as previously described
                              <sup>
                               <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                34
                               </a>
                              </sup>
                              . hESCs were monthly monitored for mycoplasma infection.
                             </p>
                            </div>
                            <div class="section" id="sec-12">
                             <h3>
                              Immunofluorescence (IF)
                             </h3>
                             <p id="p-24">
                              Cells were grown on glass coverslips coated with either Vitronectin XF or Gelatine and fixed with 4% PFA for 10 min at RT, rinsed twice with PBS, and permeabilised for 20 min at RT using PBS/0.25% Triton X-100 (Sigma). Cells were blocked for 30 min at RT with blocking solution (PBS-0.25% Triton X-100 plus BSA 1%). Primary and secondary antibodies (listed in Supplementary Table 4) were diluted in blocking solution and incubated for 1 h at 37 °C. Cells were washed twice with blocking solution after each antibody staining, and stained with DAPI for 5 min at RT (0.1 mg/ml DAPI in PBS-0.1% Triton). Finally, coverslips were mounted on slides using ProLong Glass antifade reagent (ThermoFisher Scientific) and imaged using an LSM 700 confocal microscope (Zeiss). Images were processed using the software Fiji (ImageJ). At least four different fields from each experiment were imaged and representative ones are shown in the figures.
                             </p>
                            </div>
                            <div class="section" id="sec-13">
                             <h3>
                              Western Blotting
                             </h3>
                             <p id="p-25">
                              For whole cell lysates, cells were washed once in D-PBS and resuspended in ice cold RIPA buffer (150mM NaCl, 50mM Tris, pH8.0, 1% NP-40, 0.5% sodium deoxycholate, 0.1% sodium dodecyl sulfate) containing protease and phosphatase inhibitors for 10 min. Protein concentration was quantified by a BCA assay (Pierce) following the manufacturer’s instructions using a standard curve generated from BSA and read at 600 nm on an EnVision 2104 plate reader. Samples were prepared by adding 4x NuPAGE LDS sample buffer (ThermoFisher Scientific) plus 1% b-mercaptoethanol and heated at 95 °C for 5 min. 5–10mg of protein per sample was run on a 4–12% NuPAGE Bis-Tris Gel (ThermoFisher Scientific) and then transferred to PVDF membrane by liquid transfer using NuPAGE Transfer buffer (ThermoFisher Scientific). Membranes were blocked for 1 hr at RT in PBS 0.05% Tween-20 (PBST) supplemented with 4% non-fat dried milk and incubated overnight at 4 °C with primary antibodies diluted in the same blocking buffer (Supplementary Table 4). After three washes in PBST, membranes were incubated for 1 hr at RT with horseradish peroxidase (HRP)-conjugated secondary antibodies diluted in blocking buffer, then washed a further three times before being incubated with Pierce ECL2 Western Blotting Substrate (ThermoFisher Scientific) and exposed to X-Ray Film. Relative quantification was performed using Fiji (ImageJ). All western blots were performed at least in three biological replicates.
                             </p>
                            </div>
                            <div class="section" id="sec-14">
                             <h3>
                              RT-qPCR
                             </h3>
                             <p id="p-26">
                              RNA was extracted using the GenElute Mammalian Total RNA Miniprep Kit and the On-Column DNase I Digestion Set (both from Sigma-Aldrich) following manufacturer’s instructions. cDNA was synthesised using 500ng of RNA in a reaction with 250ng random primers, 0.5mM deoxynucleotides (dNTPs), 20U RNaseOUT, and 25U of SuperScript II (all Invitrogen) in a total of 20μl. cDNA was diluted 30-fold and 5μl were used for qPCR using SensiMix SYBR low-ROX (Bioline) with 150nM Forward and Reverse primers (Sigma-Aldrich; Supplementary Table 5 for primer sequences). Samples were run in technical duplicates on 96-well plates on a Stratagene Mx-3005P (Agilent) or 384-well plate on a QuantStudio Flex Real-Time PCR System, and results were analysed using the delta-delta cycle threshold (ΔΔCt) approach using PBGD as housekeeping gene. Primers were designed using PrimerBlast (
                              <a href="http://www.ncbi.nlm.nih.gov/tools/primer-blast/">
                               http://www.ncbi.nlm.nih.gov/tools/primer-blast/
                              </a>
                              ), and tested to produce a single PCR product, with an efficiency between 80% and 120%.
                             </p>
                            </div>
                            <div class="section" id="sec-15">
                             <h3>
                              Cell Sorting
                             </h3>
                             <p id="p-27">
                              FUCCI-hESCs were sorted as previously described
                              <sup>
                               <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                              </sup>
                              . Briefly, cells were washed with PBS and collected by incubating with CDB for 10 min at 37°C. When needed, crosslinking of proteins to DNA was performed prior to sorting, according to the specific experiment (see
                              <a id="xref-sec-16-1" class="xref-sec" href="#sec-16">
                               Chromatin Immunoprecipitation
                              </a>
                              ). Cells were washed with 1% PBS/BSA and incubated in 1% PBS/BSA with anti-TRA-1-60-antibody mouse monoclonal IgM (1:100, Santa Cruz Biotechnology, Inc.) for 20 min on ice, together with the secondary antibody, Alexa Fluor® 647 goat anti-mouse IgM (1:1,000, Thermo Fisher). Cells were washed with PBS, re-suspended in 6ml of PBS and sorted into the different cell cycle phases using BD FACS AriaTM III, BD Fusion or BD Influx sorters by the Cambridge NIHR BRC Cell Phenotyping Hub. Gating strategy set as following: cells were then size selected for Forward (FS) and Side Scatter (SS), and doublets were removed. TRA-1-60+ cells were then sorted in the four different cell cycle phases according to the FUCCI reporter: Early G1 (EG1) for double negative (mKO2- / mAG-), Late G1 (LG1) for red-mKO2+, G1/S transition for double positive (yellow-mKO2+ / mAG+), and S/G2/M for green-mAG+. Sorted cells were then used for ATAC-seq, ChIP-seq, and RNA-seq. All the experiments have been performed normalising for cell number, usually to the lowest sample, depending on the downstream application.
                             </p>
                            </div>
                            <div class="section" id="sec-16">
                             <h3>
                              Chromatin Immunoprecipitation (ChIP)
                             </h3>
                             <p id="p-28">
                              Crosslinking of protein to DNA was performed prior to sorting in 1% PBS/Formaldehyde, rocking for 10 min at room temperature. All the experiments were performed with single crosslinking and neutralised with Glycine (0.125M), for 5 min shaking at room temperature. ChIP was then carried out using LowCell# ChIP kit (C0101007, Diagenode) and DNA recovered using iPure kit (C03010012, Diagenode) according to manufacturer’s instructions, with the following modifications. 5μg of antibodies were used per IP, and the quantity of magnetic beads adjusted accordingly, based on their binding capacity (10μl of beads for ~3μg). Elution was made in Buffer C from the iPure kit, compatible with qPCR and next generation sequencing. For all the experiments, the same number of cells was used per IP after sorting, normalised for the less abundant cell cycle phase, usually EG1 ~1M cells.
                             </p>
                            </div>
                            <div class="section" id="sec-17">
                             <h3>
                              Assay for transposase-accessible chromatin (ATAC)
                             </h3>
                             <p id="p-29">
                              TRA-1-60+ hESCs were sorted in the different cell cycle phases (EG1, LG1, G1/S, S/G2/M). Samples were generated from two independent experiments, 100,000 cells per sample, as previously described
                              <sup>
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                              </sup>
                              with some modifications. In brief, cells were washed with ice-cold PBS and incubated with 500μl of ice-cold sucrose buffer (10mM Tris-Cl pH 7.5, 3mM CaCl2, 2mM MgCl2 and 0.32M sucrose) for 12 min on ice. To collect the nuclei, Triton X-100 was added to a final concentration of 0.5% and cells incubated for additional 6 min on ice after mixing. Samples were then resuspended in 1ml of sucrose buffer and transferred to a fresh 1.5 ml Eppendorf tube. The nuclei were centrifuged at 1500g for 3 min at 4°C and the supernatant discarded. Nuclei pellets resuspended in 50μl of Nextera DNA Sample Preparation Kit (Illumina FC-121-1030) for tagmentation, comprising 25μl 2x Tagment DNA buffer, 20μl nuclease-free water and 5μl Tagment DNA Enzyme 1. The tagmentation reaction mixture was transferred in a 1.5ml low-binding tube and incubated at 37°C for 30 min and stopped by the addition of 250μl Buffer PB (Qiagen). The tagmented chromatin was purified using the MinElute PCR purification kit (Qiagen 28004), according to the manufacturer’s instructions, eluting in 11.5μl of buffer EB (Qiagen). For PCR amplification, 10μl of the tagmented chromatin was mixed with 2.5μl of Nextera PCR primer cocktail and 7.5μl of Nextera PCR master-mix (Illumina FC-121-1030) in low-binding PCR tubes. Primers were from the Nextera Index kit (Illumina FC-121-1011), using 2.5μl of an i5 primer and 2.5μl of an i7 primer per PCR, totalling 25μl. PCR amplification was performed as follows: 72°C for 3 min and 98°C for 30 sec, followed by 12 cycles at 98°C for 10 sec, 63°C for 30 sec and 72°C for 3 min. The sample volumes were then increased to 50μl by adding Qiagen EB buffer. The PCR primers were removed with 1x 0.9:1 SPRI beads (Beckman Coulter, Cat no. A63880) according to manufacturer’s instructions and samples eluted in 20μl. Following amplification, excess of primers was removed running the samples of 50ml 1% agarose TAE gel at 90V for 25 min, and size selection obtained by cutting from 120bp to 1kb. MinElute Gel Extraction kit (Qiagen 28604) was used to extract DNA, eluted in 20μl of buffer EB.
                             </p>
                            </div>
                            <div class="section" id="sec-18">
                             <h3>
                              Transfection
                             </h3>
                             <p id="p-30">
                              The generation of the inducible lines was performed using lipofectamine transfection. Lipofection was performed as previously described
                              <sup>
                               <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                              </sup>
                              . In brief, FUCCI-hESCs cells were seeded into 6 well plates using medium without Penicillin-Streptomycin (Pen/Strep, Gibco), in low density. After 48-72h (depending on colony size), cells were transfected using Lipofectamine® 2000 Transfection Reagent (Thermo Fisher), according to manufacturer’s instructions. In sum, the medium was aspirated and replaced with 1mL/well of Opti-MEM® Reduced Serum Media (Thermo Fisher). 4μg of (each) plasmid was resuspended in 0.25ml of Opti-MEM medium/well. 10μl of Lipofectamine 2000 were dissolved in 0.25ml of Opti-MEM medium/well, and incubated for 5 min at room temperature. Lipofectamine 2000 and the plasmid were mixed together and incubated for 20 min at room temperature. 0.5ml of the DNA/Lipofectamine mix was added drop by drop to the well and incubated for 18-20h. Opti-MEM was then replaced with regular media without antibiotics. Antibiotic selection was started 4-5 days after the transfection (Puromycin 1 mg/ml, G-418 50μg/ml). After 6 days, clones were picked into a 24 wells plate. Each clone was passaged into 12 well plates, for analysis of transgene expression by qPCR, immunostaining or western blot.
                             </p>
                            </div>
                            <div class="section" id="sec-19">
                             <h3>
                              Bioinformatics analyses and pipelines
                             </h3>
                             <p id="p-31">
                              The following data processing pipelines have been used for all the bioinformatics analyses according to ENCODE guidelines
                              <sup>
                               <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                              </sup>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-20">
                             <h3>
                              ChIP-seq
                             </h3>
                             <p id="p-32">
                              TRA-1-60+ hESCs were sorted in the different cell cycle phases (EG1, LG1, S/G2/M). From two independent experiments, chromatin immunoprecipitation was performed as described above. Library preparation and sequencing were performed at the Wellcome Trust Sanger Institute next-generation sequencing facility. Size selection was applied, and fragments between 100bp and 400bp (average length of 200bp) were used to prepare barcoded sequencing libraries using NEBNext Sample Prep Kit1 (NEB) following manufacturer’s instructions. 10ng of input material (as a pool of the different phases) for each experiment was processed in parallel as control. To test whether pooled inputs were an acceptable control, one experiment was run with one input per cell cycle phase. No differences were reported with > 90% peaks shared with the pooled input (data not shown). Equimolar amounts of each library were pooled, and 10 samples/lane were multiplexed on Illumina HiSeq 2000, 2 X 75bp paired-end reads. On average, 400M reads per lane were generated, mapped to human genome GRCh38 reference assembly and stored as cram files. Cram were converted in bam keeping only reads with mapping score above 10 (q>10) using Samtools view
                              <sup>
                               <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              . From two independent replicates, consistency between replicates was measured using the Irreproducibility Discovery Rate (IDR) pipeline (
                              <a href="https://sites.google.com/site/anshulkundaje/projects/idr">
                               https://sites.google.com/site/anshulkundaje/projects/idr
                              </a>
                              ). This approach is based on an independent peak-calling for the two replicates with a relaxed cut-off (-p1e-1, --to-large) using MACS
                              <sup>
                               <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              . This allows to sample enough noise in the experiment for efficient statistical comparison of replicates in subsequent steps, meaning that many false positives are present in this peak set and they are not considered as final peaks for the separate replicates. Peaks are then ranked for p-value, and the same number is compared based on reproducibility (top 100,000 peaks). This is used to score rankings between peaks in each replicate to determine an optimal cut-off for significance. The final peak calls (bed and bigBed format) are the set of peak calls that pass IDR at a threshold of 5-10% max. This is the conservative peak set and it is used for further analyses. If the minimum number of recommended reads is not achieved, replicates are pooled. For visualisation purposes, biological replicates were combined. Normalised bedGraph format files were generated for each sample using BEDTools v2.24.0
                              <sup>
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                              </sup>
                              . The reads mapped at both DNA strands from 5’ to 3’ direction were extended to a length of 300bp, and the read-enrichment was normalised by million mapped reads and size of the library. bedGraph files were converted to bigWig using UCSC tool bedGraphToBigWig, and visualised on the Biodalliance genome viewer
                              <sup>
                               <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                              </sup>
                              . Differential binding analysis was performed using the R package Diffbind
                              <sup>
                               <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                              </sup>
                              . For the annotation of genomic intervals, Heatmaps and Metaplot generation, the R packages Genomation and ChIPpeakAnno were used
                              <sup>
                               <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                               ,
                               <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                              </sup>
                              . GREAT (Genomic Regions Enrichment of Annotations Tool) was used to associate genes with regulatory regions
                              <sup>
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                              </sup>
                              , combined with UCSC liftOver tool to convert bed coordinates from Homo Sapiens GRCh38/hg38 assembly to GRCh37/hg37 assembly (
                              <a href="https://genome.ucsc.edu/cgi-bin/hgLiftOver">
                               https://genome.ucsc.edu/cgi-bin/hgLiftOver
                              </a>
                              ).
                             </p>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              ATAC-seq
                             </h3>
                             <p id="p-33">
                              TRA-1-60+ hESCs were sorted in the different cell cycle phases (EG1, LG1, G1/S, S/G2/M). Samples generated from two independent experiments, 100,000 were used per sample, as described above. Library preparation and sequencing were performed at the Wellcome Trust Sanger Institute next-generation sequencing facility. 8 ATAC-seq libraries were prepared with one of i5 and i7 Nextera tags combination and pooled equimolarly. Sequencing was performed on Illumina HiSeq 2000, 2 X 75bp paired-end reads obtaining more than 60M mapped reads per library. Reads were mapped to the human genome GRCh38 reference assembly and stored as cram files. Duplicates were removed with Samtools rmdup
                              <sup>
                               <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              . Cram were converted in bam keeping only reads with mapping score above 5 (q>5) and from chromosome 1-22 and X (H9 is a female line) to remove mitochondrial contamination using Samtools view
                              <sup>
                               <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              . From two independent replicates, consistency between replicates was measured using the Irreproducibility Discovery Rate (IDR) pipeline, as described above. Conservative peak set was used for further analyses. Diagnostic plots of fragment size distribution were generated using the R package ATACseqQC
                              <sup>
                               <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                              </sup>
                              . For visualisation purposes, biological replicates were combined as described above to generate normalised bigWig.
                             </p>
                             <p id="p-34">
                              Differential accessibility was assessed using the G-test implemented in diffReps
                              <sup>
                               <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                                8
                               </a>
                              </sup>
                              by Dr Pedro Madrigal. This tool takes into account the biological variations within a group of samples and uses that information to enhance the statistical power. For this, we used a sliding window of 600bp, p < 1E-6. Regions with differential binding that did not overlap with previously detected peaks through the IDR pipeline were removed. Genomic regions were ranked by their fold change (FC), and reported as differential only if both the |FC| >1.5 and corrected p-value padj < 1E-6. Footprinting analyses were done by Dr Pedro Madrigal combining two methods: Footprint Log-likelihood Ratio (FLR)
                              <sup>
                               <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                              </sup>
                              and Wellington
                              <sup>
                               <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                                10
                               </a>
                              </sup>
                              , adjusted for ATAC-seq. To get high quality reproducible footprints, motif matches with FLR ≥ 10.0 in each biological replicate were classified as “bound”. Transcription factor footprints that were present in both replicates, for a motif with at least 50 footprints, were considered as final predicted transcription factor binding sites. FLR mean of the replicates was calculated, only based on unique genomic regions (for those palindromic sequences with a footprint reported in each strand, we obtained the mean FLR). The quality of footprint was assessed by calculating the Protection Score, removing scores below 0
                              <sup>
                               <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                46
                               </a>
                              </sup>
                              . Wellington instead uses a “non-motif centric” approach, which considers the imbalance in this strand-specific information to efficiently identify DNA footprints. For this, a relaxed cut-off has been used (-p32 -A -fdr 0.1 --FDR_limit -4 --pv_cutoffs −4 -fp 4,30,1 - fdriter 500—one_dimension) on merged replicates (to increase sequencing depth), in order to achieve a significant overlap with FLR, and a final list generated. A conservative list of footprints was generated by overlap between FLR and Wellington, removing not expressed genes according to RNA-seq data (Fragments Per Kilobase of transcript per Million mapped reads, FPKM > 1.0).
                             </p>
                             <p id="p-35">
                              For visualisation of footprints, bigWigs of ATAC-seq data containing Tn5 insertion frequencies have been generated using NucleoATAC
                              <sup>
                               <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                              </sup>
                              from the bam files (--bam $bamFile --out $outputBasename --cores 16).
                             </p>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              RNA-seq
                             </h3>
                             <p id="p-36">
                              TRA-1-60+ hESCs were sorted in the different cell cycle phases (EG1, LG1, S/G2/M). From three independent experiments, RNA was extracted using GenEluteTM Mammalian Total RNA Miniprep Kit and On-Column DNase I Digestion Set (both Sigma-Aldrich) and a total of 9 samples fulfilling the submission requirements (0.5-1.0μg total RNA in nuclease free dH2O with A260/A280 and A60/A230 ratios above 1.8) were sent for library preparation and nextgeneration sequencing at CGS (Cambridge Genomic Services). PolyA purified library preparation kit was used, and samples were multiplexed in one HIGH Output NextSeq run Cycle, 2 X 75bp, paired-end reads. A total of 462,569,268 reads evenly distributed across the lanes were obtained (50,547,439 on average per sample). Reads were trimmed using Sickle
                              <sup>
                               <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                48
                               </a>
                              </sup>
                              , an application for trimming FASTQ files based on quality values (Sanger), with a q=20 (quality threshold) and l=30 (length threshold). For the alignment, the transcriptome was built with TopHat2
                              <sup>
                               <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                49
                               </a>
                              </sup>
                              (v2.1.0), based on Bowtie2
                              <sup>
                               <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                50
                               </a>
                              </sup>
                              (v2.2.6) using as reference genome the human GRCh38.p6 and GTF annotation from Ensembl (
                              <a href="http://ftp.ensembl.org/pub/release-83/gtf/homo_sapiens/">
                               http://ftp.ensembl.org/pub/release-83/gtf/homo_sapiens/
                              </a>
                              ). Alignment was performed using TopHat2 with standard parameters. Using Samtools view
                              <sup>
                               <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              , reads with mapping quality > 10 were kept for further analyses. Data were quantitated and differential expression analysis was performed using SeqMonk
                              <sup>
                               <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                                51
                               </a>
                              </sup>
                              . In sum, data were quantitated using the RNA-seq pipeline, and differential expression analysis was performed using the Intensity Difference Test and the R package DESeq2
                              <sup>
                               <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                              </sup>
                              (for this, uncorrected raw read counts were used), generating a list of transcripts passing the filter, with a p-value cutoff < 0.05. Hierarchial clustering plots (r > 0.7). Gene Ontology analyses were performed using the Enrichment analysis tool by the Gene Ontology Consortium (Panther)
                              <sup>
                               <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                                52
                               </a>
                              </sup>
                              , and validated using Enrich
                              <sup>
                               <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                                53
                               </a>
                              </sup>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              Data availability
                             </h3>
                             <p id="p-37">
                              The datasets generated during this study are available at ArrayExpress under the following accession numbers: E-MTAB-8347 (RNA-seq), E-MTAB-8348 (ATAC-seq), E-MTAB-8358 (ChIP-seq). Login details for reviewers available.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Supplementary Information Figure legends
                            </h2>
                            <div id="F6" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F6.large.jpg?width=800&height=600&carousel=1" title="Chromatin accessibility and CTCF footprints during cell cycle progression. (a) Gating strategy for FUCCI hESCs: P1 cells are size selected for forward scatter area (FSC-A) and side scatter area (SSC-A); P2, doublets are removed (FSC-A, FSC-W, width); P3 TRA-1-60+ cells are selected and sorted in the four different cell cycle phases according to the FUCCI reporter: EG1 for double negative (mKO2-/mAG-), LG1 for red-mKO2+, G1/S transition for double positive (yellow-mKO2+/mAG+), and S/G2/M for green-mAG+. For all events, average and standard deviation are reported with n=10. (b) DNA content analysis by DAPI staining of the different cell cycle phases in FUCCI hESCs. To validate the reporters, EG1 and LG1 show 1n (in grey and red), S/G2/M 2n (in green). (c) ATAC-seq fragment size density plot for the 4 sequenced libraries, EG1, LG1, G1/S, and S/G2/M sorted cells. Inset histograms (log-transformed) show periodicity to four nucleosomes. (c) IDR (Irreproducible Discovery Rate) plots reporting reproducible ATAC-seq peaks between two replicates in EG1, LG1, G1/S, S/G2/M. IDR threshold < 5%. (e) Gene ontology for Biological Process analysis of differentially accessible peaks in EG1vsLG1 down, EG1vsLG1 up, (diffReps, Fig. 1f). Combined score is calculated by multiplying the unadjusted p-values with the z-score. (f) Number of predicted footprints overlapping with dynamic regions (ATAC-diffReps) in the different cell cycle phases. Final footprints were obtained combining FLR (Yardimici et al., 2014) and Wellington (Piper et al., 2013) pipelines (combining list with Footprint Likelihood Ratio (FLR) ≥10.0, Protection Score > 0). (g) Pairwise comparisons of all cell cycle phases, reporting percentages of footprints for specific transcription factors, highlighting CTCF abundance in all comparison. Log scale axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Chromatin accessibility and CTCF footprints during cell cycle progression.</span> <strong>(a)</strong> Gating strategy for FUCCI hESCs: P1 cells are size selected for forward scatter area (FSC-A) and side scatter area (SSC-A); P2, doublets are removed (FSC-A, FSC-W, width); P3 TRA-1-60+ cells are selected and sorted in the four different cell cycle phases according to the FUCCI reporter: EG1 for double negative (mKO2-/mAG-), LG1 for red-mKO2+, G1/S transition for double positive (yellow-mKO2+/mAG+), and S/G2/M for green-mAG+. For all events, average and standard deviation are reported with n=10. <strong>(b)</strong> DNA content analysis by DAPI staining of the different cell cycle phases in FUCCI hESCs. To validate the reporters, EG1 and LG1 show 1n (in grey and red), S/G2/M 2n (in green). <strong>(c)</strong> ATAC-seq fragment size density plot for the 4 sequenced libraries, EG1, LG1, G1/S, and S/G2/M sorted cells. Inset histograms (log-transformed) show periodicity to four nucleosomes. <strong>(c)</strong> IDR (Irreproducible Discovery Rate) plots reporting reproducible ATAC-seq peaks between two replicates in EG1, LG1, G1/S, S/G2/M. IDR threshold < 5%. <strong>(e)</strong> Gene ontology for Biological Process analysis of differentially accessible peaks in EG1vsLG1 down, EG1vsLG1 up, (diffReps, Fig. 1f). Combined score is calculated by multiplying the unadjusted p-values with the z-score. <strong>(f)</strong> Number of predicted footprints overlapping with dynamic regions (ATAC-diffReps) in the different cell cycle phases. Final footprints were obtained combining FLR (Yardimici et al., 2014) and Wellington (Piper et al., 2013) pipelines (combining list with Footprint Likelihood Ratio (FLR) ≥10.0, Protection Score > 0). <strong>(g)</strong> Pairwise comparisons of all cell cycle phases, reporting percentages of footprints for specific transcription factors, highlighting CTCF abundance in all comparison. Log scale axis.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F6.medium.gif" width="409" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F6.medium.gif" width="409" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 1.
                              </span>
                              <span class="caption-title">
                               Chromatin accessibility and CTCF footprints during cell cycle progression.
                              </span>
                              <p id="p-38" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Gating strategy for FUCCI hESCs: P1 cells are size selected for forward scatter area (FSC-A) and side scatter area (SSC-A); P2, doublets are removed (FSC-A, FSC-W, width); P3 TRA-1-60+ cells are selected and sorted in the four different cell cycle phases according to the FUCCI reporter: EG1 for double negative (mKO2-/mAG-), LG1 for red-mKO2+, G1/S transition for double positive (yellow-mKO2+/mAG+), and S/G2/M for green-mAG+. For all events, average and standard deviation are reported with n=10.
                               <strong>
                                (b)
                               </strong>
                               DNA content analysis by DAPI staining of the different cell cycle phases in FUCCI hESCs. To validate the reporters, EG1 and LG1 show 1n (in grey and red), S/G2/M 2n (in green).
                               <strong>
                                (c)
                               </strong>
                               ATAC-seq fragment size density plot for the 4 sequenced libraries, EG1, LG1, G1/S, and S/G2/M sorted cells. Inset histograms (log-transformed) show periodicity to four nucleosomes.
                               <strong>
                                (c)
                               </strong>
                               IDR (Irreproducible Discovery Rate) plots reporting reproducible ATAC-seq peaks between two replicates in EG1, LG1, G1/S, S/G2/M. IDR threshold < 5%.
                               <strong>
                                (e)
                               </strong>
                               Gene ontology for Biological Process analysis of differentially accessible peaks in EG1vsLG1 down, EG1vsLG1 up, (diffReps,
                               <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                                Fig. 1f
                               </a>
                               ). Combined score is calculated by multiplying the unadjusted p-values with the z-score.
                               <strong>
                                (f)
                               </strong>
                               Number of predicted footprints overlapping with dynamic regions (ATAC-diffReps) in the different cell cycle phases. Final footprints were obtained combining FLR (Yardimici et al., 2014) and Wellington (Piper et al., 2013) pipelines (combining list with Footprint Likelihood Ratio (FLR) ≥10.0, Protection Score > 0).
                               <strong>
                                (g)
                               </strong>
                               Pairwise comparisons of all cell cycle phases, reporting percentages of footprints for specific transcription factors, highlighting CTCF abundance in all comparison. Log scale axis.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F7" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F7.large.jpg?width=800&height=600&carousel=1" title="Transcriptome changes during cell cycle progression. (a) Datastore Tree clustering of different samples/replicates based on Pearson correlation. (b) PCA clustering of FUCCI hESCs sorted in EG1, LG1, S/G2/M. Each dot represents a biological replicate, Principal component contribution reported in brackets. (c) Examples of differentially expressed cell cycle regulators CDK1, CDC25C, AURKA, up-regulated in the S/G2/M phase (p-value cut-off < 0.05). (d) Gene ontology for Reactome Pathway analysis for Cluster 3 (Fig. 2c). Combined score is calculated by multiplying the unadjusted p-values with the z-score. (e) RNA-seq log2RPM values for the developmental regulators SOX17, EOMES, and MIXL1, validated by RT-qPCR (values relative to housekeeping gene PBGD). (f) Heatmap reporting RT-qPCR data of unsorted FUCCI hESCs including non-retrotranscribed RNAs (noSS) as a negative control for genomic contamination in exonic vs intronic data (Fig. 2g). Data expressed as ct values of 3 biological replicates, in technical duplicate. (g) Scatter plots reporting pairwise comparison of normalised gene expression by RNA-seq in the different cell cycle phases. Highlighted are differentially expressed genes (DESeq2, DE, in blue) and differentially accessible regions (Fig. 1f, DA, in red), each dot represents one gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Transcriptome changes during cell cycle progression.</span> <strong>(a)</strong> Datastore Tree clustering of different samples/replicates based on Pearson correlation. <strong>(b)</strong> PCA clustering of FUCCI hESCs sorted in EG1, LG1, S/G2/M. Each dot represents a biological replicate, Principal component contribution reported in brackets. <strong>(c)</strong> Examples of differentially expressed cell cycle regulators <em>CDK1, CDC25C, AURKA</em>, up-regulated in the S/G2/M phase (p-value cut-off < 0.05). <strong>(d)</strong> Gene ontology for Reactome Pathway analysis for Cluster 3 (Fig. 2c). Combined score is calculated by multiplying the unadjusted p-values with the z-score. <strong>(e)</strong> RNA-seq log<sub>2</sub>RPM values for the developmental regulators <em>SOX17, EOMES</em>, and <em>MIXL1</em>, validated by RT-qPCR (values relative to housekeeping gene <em>PBGD</em>). <strong>(f)</strong> Heatmap reporting RT-qPCR data of unsorted FUCCI hESCs including non-retrotranscribed RNAs (noSS) as a negative control for genomic contamination in exonic vs intronic data (Fig. 2g). Data expressed as ct values of 3 biological replicates, in technical duplicate. <strong>(g)</strong> Scatter plots reporting pairwise comparison of normalised gene expression by RNA-seq in the different cell cycle phases. Highlighted are differentially expressed genes (DESeq2, DE, in blue) and differentially accessible regions (Fig. 1f, DA, in red), each dot represents one gene.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F7.medium.gif" width="440" height="340"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F7.medium.gif" width="440" height="340"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 2.
                              </span>
                              <span class="caption-title">
                               Transcriptome changes during cell cycle progression.
                              </span>
                              <p id="p-39" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Datastore Tree clustering of different samples/replicates based on Pearson correlation.
                               <strong>
                                (b)
                               </strong>
                               PCA clustering of FUCCI hESCs sorted in EG1, LG1, S/G2/M. Each dot represents a biological replicate, Principal component contribution reported in brackets.
                               <strong>
                                (c)
                               </strong>
                               Examples of differentially expressed cell cycle regulators CDK1, CDC25C, AURKA, up-regulated in the S/G2/M phase (p-value cut-off < 0.05).
                               <strong>
                                (d)
                               </strong>
                               Gene ontology for Reactome Pathway analysis for Cluster 3 (
                               <a id="xref-fig-2-11" class="xref-fig" href="#F2">
                                Fig. 2c
                               </a>
                               ). Combined score is calculated by multiplying the unadjusted p-values with the z-score.
                               <strong>
                                (e)
                               </strong>
                               RNA-seq log
                               <sub>
                                2
                               </sub>
                               RPM values for the developmental regulators SOX17, EOMES, and MIXL1, validated by RT-qPCR (values relative to housekeeping gene PBGD).
                               <strong>
                                (f)
                               </strong>
                               Heatmap reporting RT-qPCR data of unsorted FUCCI hESCs including non-retrotranscribed RNAs (noSS) as a negative control for genomic contamination in exonic vs intronic data (
                               <a id="xref-fig-2-12" class="xref-fig" href="#F2">
                                Fig. 2g
                               </a>
                               ). Data expressed as ct values of 3 biological replicates, in technical duplicate.
                               <strong>
                                (g)
                               </strong>
                               Scatter plots reporting pairwise comparison of normalised gene expression by RNA-seq in the different cell cycle phases. Highlighted are differentially expressed genes (DESeq2, DE, in blue) and differentially accessible regions (
                               <a id="xref-fig-1-13" class="xref-fig" href="#F1">
                                Fig. 1f
                               </a>
                               , DA, in red), each dot represents one gene.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F8" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F8.large.jpg?width=800&height=600&carousel=1" title="Transcription factors binding dynamics during cell cycle. (a) MA plot of CTCF binding sites between EG1 and LG1 phases of the cell cycle in sorted FUCCI hESCs. Each point represents a binding site, in magenta are highlighted the differentially bound sites. (b) Venn diagram reporting the overlap of CTCF footprints predicted by ATAC-seq and CTCF ChIP-seq peaks. 60% represent the percentage of footprints overlapping with CTCF ChIP-seq peaks. (c) Venn diagram and cell cycle specific quantification of the overlap of CTCF ChIP-seq peaks and ATAC-seq peaks. 77% represent the overall percentage of CTCF ChIP-seq peaks with ATAC-seq peaks. (d) Venn diagrams showing peaks overlap for SOX2 and (e) NANOG ChIP-seq in EG1, LG1, S/G2/M. (e) Gene ontology for KEGG Pathways and Biological Process of OCT4/SOX2/NANOG shared binding sites (OSN) and sites bound by single factors. Values represent the combined score, which is calculated by multiplying the unadjusted p-values with the z-score. (f) Venn diagrams showing peaks overlap for RING1B ChIP-seq in EG1, LG1, S/G2/M. (h) Gene ontology for Biological Process of RING1B binding sites. Combined score is calculated by multiplying the unadjusted p-values with the z-score. (i) Heatmaps displaying normalised ChIP-seq signal ±1kb around all RING1B/OCT4 shared peaks in the different cell cycle phases for both RING1B and OCT4 ChIP-seq, compared to relative INPUTs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Transcription factors binding dynamics during cell cycle.</span> <strong>(a)</strong> MA plot of CTCF binding sites between EG1 and LG1 phases of the cell cycle in sorted FUCCI hESCs. Each point represents a binding site, in magenta are highlighted the differentially bound sites. <strong>(b)</strong> Venn diagram reporting the overlap of CTCF footprints predicted by ATAC-seq and CTCF ChIP-seq peaks. 60% represent the percentage of footprints overlapping with CTCF ChIP-seq peaks. <strong>(c)</strong> Venn diagram and cell cycle specific quantification of the overlap of CTCF ChIP-seq peaks and ATAC-seq peaks. 77% represent the overall percentage of CTCF ChIP-seq peaks with ATAC-seq peaks. <strong>(d)</strong> Venn diagrams showing peaks overlap for SOX2 and <strong>(e)</strong> NANOG ChIP-seq in EG1, LG1, S/G2/M. <strong>(e)</strong> Gene ontology for KEGG Pathways and Biological Process of OCT4/SOX2/NANOG shared binding sites (OSN) and sites bound by single factors. Values represent the combined score, which is calculated by multiplying the unadjusted p-values with the z-score. <strong>(f)</strong> Venn diagrams showing peaks overlap for RING1B ChIP-seq in EG1, LG1, S/G2/M. <strong>(h)</strong> Gene ontology for Biological Process of RING1B binding sites. Combined score is calculated by multiplying the unadjusted p-values with the z-score. <strong>(i)</strong> Heatmaps displaying normalised ChIP-seq signal ±1kb around all RING1B/OCT4 shared peaks in the different cell cycle phases for both RING1B and OCT4 ChIP-seq, compared to relative INPUTs.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F8.medium.gif" width="440" height="358"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F8.medium.gif" width="440" height="358"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 3.
                              </span>
                              <span class="caption-title">
                               Transcription factors binding dynamics during cell cycle.
                              </span>
                              <p id="p-40" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               MA plot of CTCF binding sites between EG1 and LG1 phases of the cell cycle in sorted FUCCI hESCs. Each point represents a binding site, in magenta are highlighted the differentially bound sites.
                               <strong>
                                (b)
                               </strong>
                               Venn diagram reporting the overlap of CTCF footprints predicted by ATAC-seq and CTCF ChIP-seq peaks. 60% represent the percentage of footprints overlapping with CTCF ChIP-seq peaks.
                               <strong>
                                (c)
                               </strong>
                               Venn diagram and cell cycle specific quantification of the overlap of CTCF ChIP-seq peaks and ATAC-seq peaks. 77% represent the overall percentage of CTCF ChIP-seq peaks with ATAC-seq peaks.
                               <strong>
                                (d)
                               </strong>
                               Venn diagrams showing peaks overlap for SOX2 and
                               <strong>
                                (e)
                               </strong>
                               NANOG ChIP-seq in EG1, LG1, S/G2/M.
                               <strong>
                                (e)
                               </strong>
                               Gene ontology for KEGG Pathways and Biological Process of OCT4/SOX2/NANOG shared binding sites (OSN) and sites bound by single factors. Values represent the combined score, which is calculated by multiplying the unadjusted p-values with the z-score.
                               <strong>
                                (f)
                               </strong>
                               Venn diagrams showing peaks overlap for RING1B ChIP-seq in EG1, LG1, S/G2/M.
                               <strong>
                                (h)
                               </strong>
                               Gene ontology for Biological Process of RING1B binding sites. Combined score is calculated by multiplying the unadjusted p-values with the z-score.
                               <strong>
                                (i)
                               </strong>
                               Heatmaps displaying normalised ChIP-seq signal ±1kb around all RING1B/OCT4 shared peaks in the different cell cycle phases for both RING1B and OCT4 ChIP-seq, compared to relative INPUTs.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F9" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F9.large.jpg?width=800&height=600&carousel=1" title="Post-translational phosphorylation of OCT4. (a) Additional western-blot analysis for OCT4, SOX and NANOG in EG1, LG1, and S/G2/M, plus unsorted (UNS) control (α-tubulin as loading control). (b) Control immunofluorescence for OCT4 (magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M in greenmAG, LG1 in red-mKO2, EG1 no fluorescence). Scale bar 50μm. (c) Additional immunofluorescence for phospho-OCT4 Ser236 (pOCT4, magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M in green-mAG, LG1 in red-mKO2, EG1 no fluorescence). Bottom panel reports zoom-ed in area from top panel, cells circled are in the EG1 phase. Scale bars: top panel 50μm, bottom panels 20μm. (d) Flow cytometry plots of FUCCI hESCs positive for pOCT4 divided by cell cycle phase (EG1 grey, LG1 red, S/G2/M green). Unstained cells and secondary (2nd only) reported for all as negative controls. Quantification in Fig. 4i." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Post-translational phosphorylation of OCT4.</span> <strong>(a)</strong> Additional western-blot analysis for OCT4, SOX and NANOG in EG1, LG1, and S/G2/M, plus unsorted (UNS) control (α-tubulin as loading control). <strong>(b)</strong> Control immunofluorescence for OCT4 (magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M in greenmAG, LG1 in red-mKO2, EG1 no fluorescence). Scale bar 50μm. <strong>(c)</strong> Additional immunofluorescence for phospho-OCT4 Ser236 (pOCT4, magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M in green-mAG, LG1 in red-mKO2, EG1 no fluorescence). Bottom panel reports zoom-ed in area from top panel, cells circled are in the EG1 phase. Scale bars: top panel 50μm, bottom panels 20μm. <strong>(d)</strong> Flow cytometry plots of FUCCI hESCs positive for pOCT4 divided by cell cycle phase (EG1 grey, LG1 red, S/G2/M green). Unstained cells and secondary (2nd only) reported for all as negative controls. Quantification in Fig. 4i.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F9.medium.gif" width="440" height="185"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F9.medium.gif" width="440" height="185"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 4.
                              </span>
                              <span class="caption-title">
                               Post-translational phosphorylation of OCT4.
                              </span>
                              <p id="p-41" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Additional western-blot analysis for OCT4, SOX and NANOG in EG1, LG1, and S/G2/M, plus unsorted (UNS) control (α-tubulin as loading control).
                               <strong>
                                (b)
                               </strong>
                               Control immunofluorescence for OCT4 (magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M in greenmAG, LG1 in red-mKO2, EG1 no fluorescence). Scale bar 50μm.
                               <strong>
                                (c)
                               </strong>
                               Additional immunofluorescence for phospho-OCT4 Ser236 (pOCT4, magenta) and nuclear staining (DAPI, blue) in FUCCI hESCs (S/G2/M in green-mAG, LG1 in red-mKO2, EG1 no fluorescence). Bottom panel reports zoom-ed in area from top panel, cells circled are in the EG1 phase. Scale bars: top panel 50μm, bottom panels 20μm.
                               <strong>
                                (d)
                               </strong>
                               Flow cytometry plots of FUCCI hESCs positive for pOCT4 divided by cell cycle phase (EG1 grey, LG1 red, S/G2/M green). Unstained cells and secondary (2nd only) reported for all as negative controls. Quantification in
                               <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                                Fig. 4i
                               </a>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F10.large.jpg?width=800&height=600&carousel=1" title="Cell cycle specific OCT4 overexpression and knock-down. (a) The modified fragment of the Estrogen receptor (ERT2) is fused to the OCT4 gene, which is retained in the cytoplasm. In presence of an Estrogen receptor antagonist, such as 4-Hydroxytamoxifen (4OHT), the fusion protein binds to it and relocates into the nucleus. CAAG: CMV early enhancer, chicken β-actin and rabbit β-globin hybrid promoter; IRES: internal ribosome entry site; PuroR: Puromycin resistance. (b) OCT4 WB for 2 control lines (lane 1-2) and 4 OCT4 ERT2 lines (lanes 3-6), wild-type OCT4 band at 39 kDa and the OCT4 ERT2 fusion-protein band at 74 kDa. α-tubulin as loading control. (c) Single step optimised inducible knock-down27. AAVS1 locus targeting, with OCT4 shRNA under the control of a tetracycline inducible promoter. ZFN: zinc-finger nucleases; 5’-HAR/3’-HAR: upstream/downstream homology arm; H1-TO: Tetracycline-inducible H1 Pol III promoter carrying one tet operon after the TATA box; CAAG: CMV early enhancer, chicken β-actin and rabbit β-globin hybrid promoter; TetR: Tetracycline-sensitive repressor protein. (d) OCT4 knock-down WB at 0h, 6h, 12h, 24h, 48h, 72h after TET induction (+TET), compared to -TET control. α-tubulin as loading control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1986666152" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Cell cycle specific OCT4 overexpression and knock-down.</span> <strong>(a)</strong> The modified fragment of the Estrogen receptor (ERT2) is fused to the <em>OCT4</em> gene, which is retained in the cytoplasm. In presence of an Estrogen receptor antagonist, such as 4-Hydroxytamoxifen (4OHT), the fusion protein binds to it and relocates into the nucleus. CAAG: CMV early enhancer, chicken β-actin and rabbit β-globin hybrid promoter; IRES: internal ribosome entry site; PuroR: Puromycin resistance. <strong>(b)</strong> OCT4 WB for 2 control lines (lane 1-2) and 4 OCT4 ERT2 lines (lanes 3-6), wild-type OCT4 band at 39 kDa and the OCT4 ERT2 fusion-protein band at 74 kDa. α-tubulin as loading control. <strong>(c)</strong> Single step optimised inducible knock-down<sup>27</sup>. AAVS1 locus targeting, with OCT4 shRNA under the control of a tetracycline inducible promoter. ZFN: zinc-finger nucleases; 5’-HAR/3’-HAR: upstream/downstream homology arm; H1-TO: Tetracycline-inducible H1 Pol III promoter carrying one tet operon after the TATA box; CAAG: CMV early enhancer, chicken β-actin and rabbit β-globin hybrid promoter; TetR: Tetracycline-sensitive repressor protein. <strong>(d)</strong> OCT4 knock-down WB at 0h, 6h, 12h, 24h, 48h, 72h after TET induction (+TET), compared to -TET control. α-tubulin as loading control.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Information Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F10.medium.gif" width="440" height="156"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Information Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F10.medium.gif" width="440" height="156"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Information Figure 5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/08/2020.06.14.150748/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Information Figure 5.
                              </span>
                              <span class="caption-title">
                               Cell cycle specific OCT4 overexpression and knock-down.
                              </span>
                              <p id="p-42" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               The modified fragment of the Estrogen receptor (ERT2) is fused to the OCT4 gene, which is retained in the cytoplasm. In presence of an Estrogen receptor antagonist, such as 4-Hydroxytamoxifen (4OHT), the fusion protein binds to it and relocates into the nucleus. CAAG: CMV early enhancer, chicken β-actin and rabbit β-globin hybrid promoter; IRES: internal ribosome entry site; PuroR: Puromycin resistance.
                               <strong>
                                (b)
                               </strong>
                               OCT4 WB for 2 control lines (lane 1-2) and 4 OCT4 ERT2 lines (lanes 3-6), wild-type OCT4 band at 39 kDa and the OCT4 ERT2 fusion-protein band at 74 kDa. α-tubulin as loading control.
                               <strong>
                                (c)
                               </strong>
                               Single step optimised inducible knock-down
                               <sup>
                                <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                                 27
                                </a>
                               </sup>
                               . AAVS1 locus targeting, with OCT4 shRNA under the control of a tetracycline inducible promoter. ZFN: zinc-finger nucleases; 5’-HAR/3’-HAR: upstream/downstream homology arm; H1-TO: Tetracycline-inducible H1 Pol III promoter carrying one tet operon after the TATA box; CAAG: CMV early enhancer, chicken β-actin and rabbit β-globin hybrid promoter; TetR: Tetracycline-sensitive repressor protein.
                               <strong>
                                (d)
                               </strong>
                               OCT4 knock-down WB at 0h, 6h, 12h, 24h, 48h, 72h after TET induction (+TET), compared to -TET control. α-tubulin as loading control.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-20">
                             The work was supported by the Wellcome PhD programme in Stem Cell Biology (A.O.), the European Research Council starting grant “Relieve IMDs” (L.V.); and a core support grant from the Wellcome and Medical Research Council to the Cambridge Stem Cell Institute. We thank Dr Siim Pauklin for providing the ATAC-seq samples. We thank the Cambridge NIHR BRC Cell Phenotyping Hub for help with cell sorting and the DNA Pipelines Operations at the Wellcome Sanger Institute for next generation sequencing. We also thank Professor Hong-Duk Youn and Jihoon Shin for kindly providing pOCT4 antibody.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               Author list and affiliations updated; revised text and figures; Figure 4 added.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list">
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.1" data-doi="10.1126/science.1236083">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Naumova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Organization of the mitotic chromosome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  342
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  948
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  953
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.1236083
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1236083%26rft_id%253Dinfo%253Apmid%252F24200812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE2MS85NDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOC8yMDIwLjA2LjE0LjE1MDc0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.2" data-doi="10.1038/nature23001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nagano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Cell-cycle dynamics of chromosomal organization at single-cell resolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  547
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  61
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  67
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nature23001
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D547%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature23001%26rft_id%253Dinfo%253Apmid%252F28682332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature23001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28682332&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.3" data-doi="10.1038/nmeth.4031">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  ATAC-see reveals the accessible genome by transposase-mediated imaging and sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1013
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1020
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nmeth.4031
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D13%26rft.spage%253D1013%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4031%26rft_id%253Dinfo%253Apmid%252F27749837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.4031&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27749837&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.4" data-doi="10.1016/j.cell.2013.08.031">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pauklin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vallier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The cell-cycle state of stem cells determines cell fate propensity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  155
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  135
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  147
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2013.08.031
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D155%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.08.031%26rft_id%253Dinfo%253Apmid%252F24074866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.08.031&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24074866&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000324916700015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.5" data-doi="10.1016/j.cell.2007.12.033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sakaue-Sawano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Visualizing spatiotemporal dynamics of multicellular cellcycle progression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  132
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  487
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  498
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2007.12.033
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSakaue-Sawano%26rft.auinit1%253DA.%26rft.volume%253D132%26rft.issue%253D3%26rft.spage%253D487%26rft.epage%253D498%26rft.atitle%253DVisualizing%2Bspatiotemporal%2Bdynamics%2Bof%2Bmulticellular%2Bcell-cycle%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.12.033%26rft_id%253Dinfo%253Apmid%252F18267078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.12.033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18267078&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000253180700022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.6" data-doi="10.1101/gr.136184.111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Landt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1813
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1831
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/gr.136184.111
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLandt%26rft.auinit1%253DS.%2BG.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1813%26rft.epage%253D1831%26rft.atitle%253DChIP-seq%2Bguidelines%2Band%2Bpractices%2Bof%2Bthe%2BENCODE%2Band%2BmodENCODE%2Bconsortia%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.136184.111%26rft_id%253Dinfo%253Apmid%252F22955991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTgxMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzEyLzA4LzIwMjAuMDYuMTQuMTUwNzQ4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.7" data-doi="10.1101/2020.06.26.173211">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Madrigal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Epigenetic regulations follow cell cycle progression during differentiation of human pluripotent stem cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , 2020.2006.2026.173211
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2020.06.26.173211
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.06.26.173211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wNi4yNi4xNzMyMTF2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzEyLzA4LzIwMjAuMDYuMTQuMTUwNzQ4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.8" data-doi="10.1371/journal.pone.0065598">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  diffReps: detecting differential chromatin modification sites from ChIP-seq data with biological replicates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e65598
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pone.0065598
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De65598%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0065598%26rft_id%253Dinfo%253Apmid%252F23762400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0065598&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23762400&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.9" data-doi="10.1093/nar/gku810">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yardimci
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frank
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crawford
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ohler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Explicit DNase sequence bias modeling enables high-resolution transcription factor footprint detection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  11865
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11878
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/nar/gku810
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku810%26rft_id%253Dinfo%253Apmid%252F25294828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gku810&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25294828&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.10" data-doi="10.1093/nar/gkt850">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Piper
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Wellington: a novel method for the accurate identification of digital genomic footprints from DNase-seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e201
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/nar/gkt850
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt850%26rft_id%253Dinfo%253Apmid%252F24071585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkt850&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24071585&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.11" data-doi="10.1101/gad.176826.111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zaret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carroll
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Pioneer transcription factors: establishing competence for gene expression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genes Dev
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2227
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2241
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/gad.176826.111
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZaret%26rft.auinit1%253DK.%2BS.%26rft.volume%253D25%26rft.issue%253D21%26rft.spage%253D2227%26rft.epage%253D2241%26rft.atitle%253DPioneer%2Btranscription%2Bfactors%253A%2Bestablishing%2Bcompetence%2Bfor%2Bgene%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.176826.111%26rft_id%253Dinfo%253Apmid%252F22056668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI1LzIxLzIyMjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOC8yMDIwLjA2LjE0LjE1MDc0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.12" data-doi="10.1016/j.stemcr.2013.10.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Cell-cycle control of developmentally regulated transcription factors accounts for heterogeneity in human pluripotent cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stem Cell Reports
                                 </abbr>
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  532
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  544
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.stemcr.2013.10.009
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStem%2BCell%2BReports%26rft.volume%253D1%26rft.spage%253D532%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stemcr.2013.10.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.stemcr.2013.10.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.13" data-doi="10.1186/s13059-014-0550-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Love
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anders
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  550
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1186/s13059-014-0550-8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25516281&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.14" data-doi="10.1016/j.stem.2015.11.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ji
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  3D Chromosome Regulatory Landscape of Human Pluripotent Cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Stem Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  262
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  275
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.stem.2015.11.007
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BStem%2BCell%26rft.volume%253D18%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2015.11.007%26rft_id%253Dinfo%253Apmid%252F26686465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.stem.2015.11.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26686465&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.15" data-doi="10.1038/ng.409">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  H3.3/H2A.Z double variant-containing nucleosomes mark ‘nucleosome-free regions’ of active promoters and other regulatory regions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  941
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  945
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/ng.409
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJin%26rft.auinit1%253DC.%26rft.volume%253D41%26rft.issue%253D8%26rft.spage%253D941%26rft.epage%253D945%26rft.atitle%253DH3.3%252FH2A.Z%2Bdouble%2Bvariant-containing%2Bnucleosomes%2Bmark%2B%2527nucleosome-free%2Bregions%2527%2Bof%2Bactive%2Bpromoters%2Band%2Bother%2Bregulatory%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.409%26rft_id%253Dinfo%253Apmid%252F19633671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.409&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19633671&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000268432900022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.16" data-doi="10.1101/gad.264150.115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shibata
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Starmer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Magnuson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Histone H3.3 maintains genome integrity during mammalian development
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genes Dev
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1377
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1392
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/gad.264150.115
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DJang%26rft.auinit1%253DC.-W.%26rft.volume%253D29%26rft.issue%253D13%26rft.spage%253D1377%26rft.epage%253D1392%26rft.atitle%253DHistone%2BH3.3%2Bmaintains%2Bgenome%2Bintegrity%2Bduring%2Bmammalian%2Bdevelopment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.264150.115%26rft_id%253Dinfo%253Apmid%252F26159997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI5LzEzLzEzNzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOC8yMDIwLjA2LjE0LjE1MDc0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.17" data-doi="10.1038/s41598-018-36753-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sekiya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kato
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kawaguchi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nagata
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Involvement of CTCF in transcription regulation of EGR1 at early G1 phase as an architecture factor
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  329
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/s41598-018-36753-x
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D329%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-36753-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-018-36753-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.18" data-doi="10.1016/j.cell.2017.06.034">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Canela
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genome Organization Drives Chromosome Fragility
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  170
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  507
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  521
                                 </span>
                                 <span class="cit-page-range">
                                  e518
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2017.06.034
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.06.034%26rft_id%253Dinfo%253Apmid%252F28735753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.06.034&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28735753&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.19" data-doi="10.1242/dev.014340">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Endoh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Polycomb group proteins Ring1A/B are functionally linked to the core transcriptional regulatory circuitry to maintain ES cell identity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Development
                                 </abbr>
                                 <span class="cit-vol">
                                  135
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1513
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1524
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1242/dev.014340
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.014340%26rft_id%253Dinfo%253Apmid%252F18339675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTM1LzgvMTUxMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzEyLzA4LzIwMjAuMDYuMTQuMTUwNzQ4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.20" data-doi="10.1126/sciadv.1500737">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aranda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Di Croce
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Regulation of gene transcription by Polycomb proteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Adv
                                 </abbr>
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1500737
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/sciadv.1500737
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DAranda%26rft.auinit1%253DS.%26rft.volume%253D1%26rft.issue%253D11%26rft.spage%253De1500737%26rft.epage%253De1500737%26rft.atitle%253DRegulation%2Bof%2Bgene%2Btranscription%2Bby%2BPolycomb%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1500737%26rft_id%253Dinfo%253Apmid%252F26665172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiMS8xMS9lMTUwMDczNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzEyLzA4LzIwMjAuMDYuMTQuMTUwNzQ4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.21" data-doi="10.1073/pnas.0811964106">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Swaney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wenger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thomson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Human embryonic stem cell phosphoproteome revealed by electron transfer dissociation tandem mass spectrometry
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  106
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  995
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1000
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1073/pnas.0811964106
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0811964106%26rft_id%253Dinfo%253Apmid%252F19144917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDYvNC85OTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOC8yMDIwLjA2LjE0LjE1MDc0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.22" data-doi="10.1371/journal.pone.0004467">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saxe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tomilin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Scholer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Plath
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Post-translational regulation of Oct4 transcriptional activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e4467
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pone.0004467
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSaxe%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De4467%26rft.epage%253De4467%26rft.atitle%253DPost-translational%2Bregulation%2Bof%2BOct4%2Btranscriptional%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004467%26rft_id%253Dinfo%253Apmid%252F19221599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004467&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19221599&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.23" data-doi="10.7554/eLife.10877">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Aurkb/PP1-mediated resetting of Oct4 during the cell cycle determines the identity of embryonic stem cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.7554/eLife.10877
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.10877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.10877&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.24" data-doi="10.1101/gad.607311">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. K. K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Pluripotency factors regulate definitive endoderm specification through eomesodermin
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gene Dev
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  238
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  250
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/gad.607311
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGene%2BDev%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.607311%26rft_id%253Dinfo%253Apmid%252F21245162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjUvMy8yMzgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOC8yMDIwLjA2LjE0LjE1MDc0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.25" data-doi="10.1002/stem.199">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vallier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Signaling pathways controlling pluripotency and early cell fate decisions of human induced pluripotent stem cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stem Cells
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2655
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2666
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/stem.199
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStem%2BCells%26rft.stitle%253DStem%2BCells%26rft.aulast%253DVallier%26rft.auinit1%253DL.%26rft.volume%253D27%26rft.issue%253D11%26rft.spage%253D2655%26rft.epage%253D2666%26rft.atitle%253DSignaling%2Bpathways%2Bcontrolling%2Bpluripotency%2Band%2Bearly%2Bcell%2Bfate%2Bdecisions%2Bof%2Bhuman%2Binduced%2Bpluripotent%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fstem.199%26rft_id%253Dinfo%253Apmid%252F19688839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/stem.199&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19688839&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000271830700003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.26" data-doi="10.1002/hep.23506">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Touboul
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Generation of functional hepatocytes from human embryonic stem cells under chemically defined conditions that recapitulate liver development
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Hepatology
                                 </abbr>
                                 <span class="cit-vol">
                                  51
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1754
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1765
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/hep.23506
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHepatology%2B%2528Baltimore%252C%2BMd.%2529%26rft.stitle%253DHepatology%26rft.aulast%253DTouboul%26rft.auinit1%253DT.%26rft.volume%253D51%26rft.issue%253D5%26rft.spage%253D1754%26rft.epage%253D1765%26rft.atitle%253DGeneration%2Bof%2Bfunctional%2Bhepatocytes%2Bfrom%2Bhuman%2Bembryonic%2Bstem%2Bcells%2Bunder%2Bchemically%2Bdefined%2Bconditions%2Bthat%2Brecapitulate%2Bliver%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhep.23506%26rft_id%253Dinfo%253Apmid%252F20301097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/hep.23506&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20301097&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000277261400033&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.27" data-doi="10.1242/dev.138081">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bertero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Optimized inducible shRNA and CRISPR/Cas9 platforms for in vitro studies of human development using hPSCs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Development
                                 </abbr>
                                 <span class="cit-vol">
                                  143
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4405
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4418
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1242/dev.138081
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DBertero%26rft.auinit1%253DA.%26rft.volume%253D143%26rft.issue%253D23%26rft.spage%253D4405%26rft.epage%253D4418%26rft.atitle%253DOptimized%2Binducible%2BshRNA%2Band%2BCRISPR%252FCas9%2Bplatforms%2Bfor%2Bin%2Bvitro%2Bstudies%2Bof%2Bhuman%2Bdevelopment%2Busing%2BhPSCs%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.138081%26rft_id%253Dinfo%253Apmid%252F27899508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTQzLzIzLzQ0MDUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOC8yMDIwLjA2LjE0LjE1MDc0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.28" data-doi="10.1016/j.omto.2020.05.002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  miR-155 Accelerates the Growth of Human Liver Cancer Cells by Activating CDK2 via Targeting H3F3A
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Ther Oncolytics
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  471
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  483
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.omto.2020.05.002
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BTher%2BOncolytics%26rft.volume%253D17%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.omto.2020.05.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.omto.2020.05.002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.29" data-doi="10.1016/j.molcel.2017.01.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kundu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Polycomb Repressive Complex 1 Generates Discrete Compacted Domains that Change during Differentiation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  432
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  446
                                 </span>
                                 <span class="cit-page-range">
                                  e435
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.molcel.2017.01.009
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BCell%26rft.volume%253D65%26rft.spage%253D432%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2017.01.009%26rft_id%253Dinfo%253Apmid%252F28157505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2017.01.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28157505&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.30" data-doi="10.1016/j.cell.2015.07.001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gonzales
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Deterministic Restriction on Pluripotent State Dissolution by Cell-Cycle Pathways
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  162
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  564
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  579
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.cell.2015.07.001
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D162%26rft.spage%253D564%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.07.001%26rft_id%253Dinfo%253Apmid%252F26232226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.07.001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26232226&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.31" data-doi="10.1371/journal.pgen.1002774">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Endoh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Histone H2A mono-ubiquitination is a crucial step to mediate PRC1-dependent repression of developmental genes to maintain ES cell identity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1002774
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pgen.1002774
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DEndoh%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002774%26rft.epage%253De1002774%26rft.atitle%253DHistone%2BH2A%2Bmono-ubiquitination%2Bis%2Ba%2Bcrucial%2Bstep%2Bto%2Bmediate%2BPRC1-dependent%2Brepression%2Bof%2Bdevelopmental%2Bgenes%2Bto%2Bmaintain%2BES%2Bcell%2Bidentity.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002774%26rft_id%253Dinfo%253Apmid%252F22844243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002774&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22844243&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.32" data-doi="10.1007/978-1-60761-962-8_3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vallier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Serum-free and feeder-free culture conditions for human embryonic stem cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Methods Mol Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  690
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  57
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  66
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1007/978-1-60761-962-8_3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DVallier%26rft.auinit1%253DL.%26rft.volume%253D690%26rft.spage%253D57%26rft.epage%253D66%26rft.atitle%253DSerum-free%2Band%2Bfeeder-free%2Bculture%2Bconditions%2Bfor%2Bhuman%2Bembryonic%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60761-962-8_3%26rft_id%253Dinfo%253Apmid%252F21042984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/978-1-60761-962-8_3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21042984&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.33" data-doi="10.1038/nmeth.1593">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Chemically defined conditions for human iPSC derivation and culture
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  424
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  429
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nmeth.1593
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DChen%26rft.auinit1%253DG.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D424%26rft.epage%253D429%26rft.atitle%253DChemically%2Bdefined%2Bconditions%2Bfor%2Bhuman%2BiPSC%2Bderivation%2Band%2Bculture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1593%26rft_id%253Dinfo%253Apmid%252F21478862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.1593&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21478862&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000289987100021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.34" data-doi="10.1371/journal.pone.0006082">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vallier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Early cell fate decisions of human embryonic stem cells and mouse epiblast stem cells are controlled by the same signalling pathways
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e6082
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1371/journal.pone.0006082
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DVallier%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De6082%26rft.epage%253De6082%26rft.atitle%253DEarly%2Bcell%2Bfate%2Bdecisions%2Bof%2Bhuman%2Bembryonic%2Bstem%2Bcells%2Band%2Bmouse%2Bepiblast%2Bstem%2Bcells%2Bare%2Bcontrolled%2Bby%2Bthe%2Bsame%2Bsignalling%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006082%26rft_id%253Dinfo%253Apmid%252F19564924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006082&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19564924&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.35" data-doi="10.1038/ng.3467">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kumasaka
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Knights
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaffney
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fine-mapping cellular QTLs with RASQUAL and ATAC-seq
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Genet
                                 </abbr>
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  206
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  213
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/ng.3467
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D48%26rft.spage%253D206%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3467%26rft_id%253Dinfo%253Apmid%252F26656845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ng.3467&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26656845&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.36" data-doi="10.1634/stemcells.22-1-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vallier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Enhancing and diminishing gene function in human embryonic stem cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stem Cells
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1634/stemcells.22-1-2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStem%2BCells%26rft.stitle%253DStem%2BCells%26rft.aulast%253DVallier%26rft.auinit1%253DL.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D2%26rft.epage%253D11%26rft.atitle%253DEnhancing%2Band%2BDiminishing%2BGene%2BFunction%2Bin%2BHuman%2BEmbryonic%2BStem%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1634%252Fstemcells.22-1-2%26rft_id%253Dinfo%253Apmid%252F14688386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1634/stemcells.22-1-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14688386&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000187497200001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.37" data-doi="10.1093/bioinformatics/btp352">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Sequence Alignment/Map format and SAMtools
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2078
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2079
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/bioinformatics/btp352
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19505943&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000268808600014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.38" data-doi="10.1186/gb-2008-9-9-r137">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Model-based analysis of ChIP-Seq (MACS)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  R137
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1186/gb-2008-9-9-r137
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253DR137%26rft.epage%253DR137%26rft.atitle%253DModel-based%2Banalysis%2Bof%2BChIP-Seq%2B%2528MACS%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-9-r137%26rft_id%253Dinfo%253Apmid%252F18798982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-9-r137&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18798982&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.39" data-doi="10.1093/bioinformatics/btq033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quinlan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  BEDTools: a flexible suite of utilities for comparing genomic features
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  841
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  842
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/bioinformatics/btq033
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20110278&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000275243500019&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.40" data-doi="10.1093/bioinformatics/btr020">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Down
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Piipari
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hubbard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Dalliance: interactive genome viewing on the web
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  889
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  890
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/bioinformatics/btr020
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr020%26rft_id%253Dinfo%253Apmid%252F21252075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr020&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21252075&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000288277300030&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.41" data-doi="10.1038/nature10730">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ross-Innes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Differential oestrogen receptor binding is associated with clinical outcome in breast cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  481
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  389
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  393
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nature10730
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRoss-Innes%26rft.auinit1%253DC.%2BS.%26rft.volume%253D481%26rft.issue%253D7381%26rft.spage%253D389%26rft.epage%253D393%26rft.atitle%253DDifferential%2Boestrogen%2Breceptor%2Bbinding%2Bis%2Bassociated%2Bwith%2Bclinical%2Boutcome%2Bin%2Bbreast%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10730%26rft_id%253Dinfo%253Apmid%252F22217937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature10730&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22217937&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000299210600048&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.42" data-doi="10.1093/bioinformatics/btu775">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Akalin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Franke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vlahovicek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mason
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schubeler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Genomation: a toolkit to summarize, annotate and visualize genomic intervals
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1127
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1129
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1093/bioinformatics/btu775
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu775%26rft_id%253Dinfo%253Apmid%252F25417204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu775&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25417204&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.43" data-doi="10.1186/1471-2105-11-237">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  ChIPpeakAnno: a Bioconductor package to annotate ChIP-seq and ChIP-chip data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  237
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1186/1471-2105-11-237
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DZhu%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D11%26rft.spage%253D237%26rft.epage%253D237%26rft.atitle%253DChIPpeakAnno%253A%2Ba%2BBioconductor%2Bpackage%2Bto%2Bannotate%2BChIP-seq%2Band%2BChIP-chip%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-237%26rft_id%253Dinfo%253Apmid%252F20459804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-237&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20459804&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.44" data-doi="10.1038/nbt.1630">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McLean
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  GREAT improves functional interpretation of cis-regulatory regions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnol
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  495
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  501
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nbt.1630
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMcLean%26rft.auinit1%253DC.%2BY.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D495%26rft.epage%253D501%26rft.atitle%253DGREAT%2Bimproves%2Bfunctional%2Binterpretation%2Bof%2Bcis-regulatory%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1630%26rft_id%253Dinfo%253Apmid%252F20436461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.1630&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20436461&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000277452700030&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ou J
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kelliher
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Castilla
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lawson
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  <span class="cit-name-given-names">
                                   LJ.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  ATACseqQC: ATAC-seq Quality Control
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  R package version 1.0.5
                                 </abbr>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.46" data-doi="10.1038/nmeth.3772">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gusmao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Allhoff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zenke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Costa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Analysis of computational footprinting methods for DNase sequencing experiments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  303
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  309
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nmeth.3772
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D13%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3772%26rft_id%253Dinfo%253Apmid%252F26901649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.3772&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26901649&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47 in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.47" data-doi="10.1101/gr.192294.115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schep
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. N.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Structured nucleosome fingerprints enable high-resolution mapping of chromatin architecture within regulatory regions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1757
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1770
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/gr.192294.115
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.192294.115%26rft_id%253Dinfo%253Apmid%252F26314830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8xNzU3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMTIvMDgvMjAyMC4wNi4xNC4xNTA3NDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48 in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.06.14.150748v2.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-source">
                                  Sickle: A sliding-window, adaptive, quality-based trimming tool for FastQ files Software
                                 </span>
                                 . Available at
                                 <a href="https://github.com/najoshi/sickle">
                                  https://github.com/najoshi/sickle
                                 </a>
                                 . v. 1.33 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49 in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.49" data-doi="10.1186/gb-2013-14-4-r36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  R36
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1186/gb-2013-14-4-r36
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23618408&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50 in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.50" data-doi="10.1038/nmeth.1923">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langmead
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fast gapped-read alignment with Bowtie 2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  357
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  359
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nmeth.1923
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22388286&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000302218500017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51 in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.06.14.150748v2.51">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-source">
                                  SeqMonk: A tool to visualise and analyse high throughput mapped sequence data
                                 </span>
                                 .
                                 <a href="http://www.bioinformatics.babraham.ac.uk/projects/seqmonk/">
                                  http://www.bioinformatics.babraham.ac.uk/projects/seqmonk/
                                 </a>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52 in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.52" data-doi="10.1101/gr.772403">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thomas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  PANTHER: a library of protein families and subfamilies indexed by function
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2129
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2141
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/gr.772403
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DThomas%26rft.auinit1%253DP.%2BD.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2129%26rft.epage%253D2141%26rft.atitle%253DPANTHER%253A%2BA%2BLibrary%2Bof%2BProtein%2BFamilies%2Band%2BSubfamilies%2BIndexed%2Bby%2BFunction%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.772403%26rft_id%253Dinfo%253Apmid%252F12952881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjEyOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzEyLzA4LzIwMjAuMDYuMTQuMTUwNzQ4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53 in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.14.150748v2.53" data-doi="10.1186/1471-2105-14-128">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  128
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1186/1471-2105-14-128
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D128%26rft.atitle%253DENRICHR%253A%2BINTERACTIVE%2BAND%2BCOLLABORATIVE%2BHTML5%2BGENE%2BLIST%2BENRICHMENT%2BANALYSIS%2BTOOL%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-128%26rft_id%253Dinfo%253Apmid%252F23586463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-128&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23586463&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F08%2F2020.06.14.150748.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.06.14.150748v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.11.20.517267v2" title="A freshwater zooplankton in the face to boat noise pollution" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.09.11.507506v2" title="Broad host tropism of ACE2-using MERS-related coronaviruses and determinants restricting viral recognition" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted December 08, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.06.14.150748v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.06.14.150748v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.06.14.150748v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2886546"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-PVmMNc92IRAeP1dVV_5JpIkxSotncfmLPI4Z9Axitq8"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484472466"/>
                                   <input type="hidden" name="captcha_token" value="8d9dc6d23c0e194115f287d270a9c93f"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2886546--3" data-node-nid="2886546" id="node-2886546--41649794573" data-pisa="biorxiv;2020.06.14.150748v2" data-pisa-master="biorxiv;2020.06.14.150748" data-seqnum="2886546" data-apath="/biorxiv/early/2022/12/08/2020.06.14.150748.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Anna
                                  </span>
                                  <span class="nlm-surname">
                                   Osnato
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Stephanie
                                  </span>
                                  <span class="nlm-surname">
                                   Brown
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Ludovic
                                  </span>
                                  <span class="nlm-surname">
                                   Vallier
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.06.14.150748;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.06.14.150748
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.06.14.150748v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&title=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&title=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&text=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&t=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&title=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&title=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&title=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2886546--5" data-node-nid="2886546" id="citation-node-2886546--61754107479" data-pisa="biorxiv;2020.06.14.150748v2" data-pisa-master="biorxiv;2020.06.14.150748" data-seqnum="2886546" data-apath="/biorxiv/early/2022/12/08/2020.06.14.150748.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Transcriptional networks are dynamically regulated during cell cycle progression in human Pluripotent Stem Cells
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Anna
                                      </span>
                                      <span class="nlm-surname">
                                       Osnato
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Stephanie
                                      </span>
                                      <span class="nlm-surname">
                                       Brown
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Ludovic
                                      </span>
                                      <span class="nlm-surname">
                                       Vallier
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.06.14.150748;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.06.14.150748
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Osnato%2C+Anna&rft.au=Brown%2C+Stephanie&rft.au=Vallier%2C+Ludovic">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2886546/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2886546/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2886546/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2886546/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2886546/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2886546/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2886546/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2886546/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2886546/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2886546/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2886546/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2886546/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&count=horizontal&via=&text=Transcriptional%20networks%20are%20dynamically%20regulated%20during%20cell%20cycle%20progression%20in%20human%20Pluripotent%20Stem%20Cells&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.06.14.150748v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.06.14.150748v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/cell-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Cell Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
