<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.06.03.131573v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.06.03.131573v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.06.03.131573v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/12/09/2020.06.03.131573.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.06.03.131573v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns"/>
  <meta name="DC.Identifier" content="10.1101/2020.06.03.131573"/>
  <meta name="DC.Date" content="2022-12-09"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Neural coding and memory formation depend on temporal spiking sequences that span high-dimensional neural ensembles. The unsupervised discovery and characterization of these spiking sequences requires a suitable dissimilarity measure to spiking patterns. Here, we present a new dissimilarity measure based on optimal transport theory called SpikeShip, which compares multi-neuron spiking patterns based on all the relative spike-timing relationships among neurons. SpikeShip computes the optimal transport cost to make all the relative spike-timing relationships (across neurons) identical between two spiking patterns. We show that this transport cost can be decomposed into a temporal rigid translation term and a vector of neuron-specific transport flows. SpikeShip can be effectively computed for high-dimensional neuronal ensembles, has a low (linear) computational cost, and is sensitive to higher-order correlations. We applied SpikeShip to large-scale Neuropixel recordings during spontaneous activity and visual encoding. We show that high-dimensional spiking sequences detected via SpikeShip reliably distinguish between different natural images and different behavioral states. These spiking sequences carried complementary information to conventional firing rate codes. SpikeShip opens new avenues for studying neural coding and memory consolidation by finding temporal patterns in high-dimensional neural ensembles.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Boris Sotomayor-Gómez"/>
  <meta name="DC.Contributor" content="Francesco P. Battaglia"/>
  <meta name="DC.Contributor" content="Martin Vinck"/>
  <meta name="article:published_time" content="2022-12-09"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Neural coding and memory formation depend on temporal spiking sequences that span high-dimensional neural ensembles. The unsupervised discovery and characterization of these spiking sequences requires a suitable dissimilarity measure to spiking patterns. Here, we present a new dissimilarity measure based on optimal transport theory called SpikeShip, which compares multi-neuron spiking patterns based on all the relative spike-timing relationships among neurons. SpikeShip computes the optimal transport cost to make all the relative spike-timing relationships (across neurons) identical between two spiking patterns. We show that this transport cost can be decomposed into a temporal rigid translation term and a vector of neuron-specific transport flows. SpikeShip can be effectively computed for high-dimensional neuronal ensembles, has a low (linear) computational cost, and is sensitive to higher-order correlations. We applied SpikeShip to large-scale Neuropixel recordings during spontaneous activity and visual encoding. We show that high-dimensional spiking sequences detected via SpikeShip reliably distinguish between different natural images and different behavioral states. These spiking sequences carried complementary information to conventional firing rate codes. SpikeShip opens new avenues for studying neural coding and memory consolidation by finding temporal patterns in high-dimensional neural ensembles.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.06.03.131573v3"/>
  <meta name="citation_id" content="2020.06.03.131573v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.06.03.131573v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.06.03.131573v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.06.03.131573v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.06.03.131573"/>
  <meta name="citation_num_pages" content="36"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.06.03.131573"/>
  <meta name="citation_author" content="Boris Sotomayor-Gómez"/>
  <meta name="citation_author_institution" content="Ernst Strüngmann Institute (ESI) for Neuroscience in Cooperation with Max Planck Society"/>
  <meta name="citation_author_institution" content="Department of Neuroinformatics, Donders Centre for Neuroscience, Radboud University Nijmegen"/>
  <meta name="citation_author_email" content="boris.sotomayor@esi-frankfurt.de"/>
  <meta name="citation_author_email" content="martin.vinck@esi-frankfurt.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6682-3761"/>
  <meta name="citation_author" content="Francesco P. Battaglia"/>
  <meta name="citation_author_institution" content="Donders Institute for Brain, Cognition and Behaviour, Radboud University Nijmegen"/>
  <meta name="citation_author" content="Martin Vinck"/>
  <meta name="citation_author_institution" content="Ernst Strüngmann Institute (ESI) for Neuroscience in Cooperation with Max Planck Society"/>
  <meta name="citation_author_institution" content="Department of Neuroinformatics, Donders Centre for Neuroscience, Radboud University Nijmegen"/>
  <meta name="citation_author_email" content="boris.sotomayor@esi-frankfurt.de"/>
  <meta name="citation_author_email" content="martin.vinck@esi-frankfurt.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4044-0970"/>
  <meta name="citation_reference" content="M.N. Havenith, S. Yu, J. Biederlack, N.H. Chen, W. Singer, D. Nikolic, Synchrony makes neurons fire in sequence, and stimulus properties determine who is ahead. J Neurosci 31(23), 8570–84 (2011)."/>
  <meta name="citation_reference" content="Havenith, Martha N Yu, Shan Biederlack, Julia Chen, Nan-Hui Singer, Wolf Nikolic, Danko Research Support, Non-U.S. Gov’t United States The Journal of neuroscience : the official journal of the Society for Neuroscience J Neurosci. 2011 Jun 8;31(23):8570–84. doi: 10.1523/JNEUROSCI.2817-10.2011."/>
  <meta name="citation_reference" content="M. Vinck, B. Lima, T. Womelsdorf, R. Oostenveld, W. Singer, S. Neuenschwander, P. Fries, Gamma-phase shifting in awake monkey visual cortex. Journal of Neuroscience 30(4), 1250–1257 (2010)"/>
  <meta name="citation_reference" content="A. Luczak, P. Bartho, K.D. Harris, Spontaneous events outline the realm of possible sensory responses in neocortical populations. Neuron 62(3), 413–25 (2009)."/>
  <meta name="citation_reference" content="Luczak, Artur Bartho, Peter Harris, Kenneth D Neuron. 2009 May 14;62(3):413–25. doi: 10.1016/j.neuron.2009.03.014."/>
  <meta name="citation_reference" content="T. Lu, L. Liang, X. Wang, Temporal and rate representations of timevarying signals in the auditory cortex of awake primates. Nat Neurosci 4(11), 1131–8 (2001)."/>
  <meta name="citation_reference" content="Lu, T Liang, L Wang, X Nat Neurosci. 2001 Nov;4(11):1131–8. doi: 10.1038/nn737."/>
  <meta name="citation_reference" content="E. Pastalkova, V. Itskov, A. Amarasingham, G. Buzsaki, Internally generated cell assembly sequences in the rat hippocampus. Science 321(5894), 1322–7 (2008)."/>
  <meta name="citation_reference" content="Pastalkova, Eva Itskov, Vladimir Amarasingham, Asohan Buzsaki, Gyorgy Research Support, N.I.H., Extramural Research Support, Non-U.S. Gov’t Research Support, U.S. Gov’t, Non-P.H.S. United States Science (New York, N.Y.) Science. 2008 Sep 5;321(5894):1322–7. doi: 10.1126/science.1159775."/>
  <meta name="citation_reference" content="Y. Ikegaya, G. Aaron, R. Cossart, D. Aronov, I. Lampl, D. Ferster, R. Yuste, Synfire chains and cortical songs: temporal modules of cortical activity. Science 304(5670), 559–564 (2004)"/>
  <meta name="citation_reference" content="M. Wehr, G. Laurent, Odour encoding by temporal sequences of firing in oscillating neural assemblies. Nature 384(6605), 162–6 (1996)."/>
  <meta name="citation_reference" content="Wehr, M Laurent, G England Nature. 1996 Nov 14;384(6605):162–6. doi: 10.1038/384162a0."/>
  <meta name="citation_reference" content="H.S. Seung, D.D. Lee, The manifold ways of perception. science 290(5500), 2268–2269 (2000)"/>
  <meta name="citation_reference" content="J.J. Jun, N.A. Steinmetz, J.H. Siegle, D.J. Denman, M. Bauza, B. Barbarits, A.K. Lee, C.A. Anastassiou, A. Andrei, Ç. Aydin, et al., Fully integrated silicon probes for high-density recording of neural activity. Nature 551(7679), 232–236 (2017)"/>
  <meta name="citation_reference" content="C. Stringer, M. Pachitariu, N. Steinmetz, C.B. Reddy, M. Carandini, K.D. Harris, Spontaneous behaviors drive multidimensional, brainwide activity. Science 364(6437) (2019)"/>
  <meta name="citation_reference" content="C. Stringer, M. Michaelos, D. Tsyboulski, S.E. Lindo, M. Pachitariu, High-precision coding in visual cortex. Cell 184(10), 2767–2778 (2021)"/>
  <meta name="citation_reference" content="J.H. Siegle, X. Jia, S. Durand, S. Gale, C. Bennett, N. Graddis, G. Heller, T.K. Ramirez, H. Choi, J.A. Luviano, et al., Survey of spiking in the mouse visual system reveals functional hierarchy. Nature 592(7852), 86–92 (2021)"/>
  <meta name="citation_reference" content="L. Chang, D.Y. Tsao, The code for facial identity in the primate brain. Cell 169(6), 1013–1028 (2017)"/>
  <meta name="citation_reference" content="M.M. Churchland, J.P. Cunningham, M.T. Kaufman, J.D. Foster, P. Nuyujukian, S.I. Ryu, K.V. Shenoy, Neural population dynamics during reaching. Nature 487(7405), 51–56 (2012)"/>
  <meta name="citation_reference" content="G. Dragoi, G. Buzsaki, Temporal encoding of place sequences by hippocampal cell assemblies. Neuron 50(1), 145–57 (2006)"/>
  <meta name="citation_reference" content="Z.F. Mainen, T.J. Sejnowski, Reliability of spike timing in neocortical neurons. Science 268(5216), 1503–1506 (1995)"/>
  <meta name="citation_reference" content="H. Markram, J. Lubke, M. Frotscher, B. Sakmann, Regulation of synaptic efficacy by coincidence of postsynaptic aps and epsps. Science 275(5297), 213–5 (1997)."/>
  <meta name="citation_reference" content="Markram, H Lubke, J Frotscher, M Sakmann, B New York, N.Y. Science. 1997 Jan 10;275(5297):213–5."/>
  <meta name="citation_reference" content="Y. Dan, M.M. Poo, Spike timing-dependent plasticity of neural circuits. Neuron 44(1), 23–30 (2004)."/>
  <meta name="citation_reference" content="Dan, Yang Poo, Mu-Ming Neuron. 2004 Sep 30;44(1):23–30. doi: 10.1016/j.neuron.2004.09.007."/>
  <meta name="citation_reference" content="L.F. Abbott, K.I. Blum, Functional significance of long-term potentiation for sequence learning and prediction. Cereb Cortex 6(3), 406–16 (1996)."/>
  <meta name="citation_reference" content="Abbott, L F Blum, K I New York, N.Y. : 1991 Cereb Cortex. 1996 May-Jun;6(3):406–16."/>
  <meta name="citation_reference" content="D.V. Buonomano, W. Maass, State-dependent computations: spatiotemporal processing in cortical networks. Nat Rev Neurosci 10(2), 113–25 (2009)."/>
  <meta name="citation_reference" content="Buonomano, Dean V Maass, Wolfgang England Nat Rev Neurosci. 2009 Feb;10(2):113–25. doi: 10.1038/nrn2558. Epub 2009 Jan 15."/>
  <meta name="citation_reference" content="W. Singer, A. Lazar, Does the cerebral cortex exploit high-dimensional, non-linear dynamics for information processing? Frontiers in computational neuroscience 10, 99 (2016)"/>
  <meta name="citation_reference" content="N. Steinmetz, M. Pachitariu, C. Stringer, M. Carandini, K. Harris. Eight-probe neuropixels recordings during spontaneous behaviors (2019). https://doi.org/10.25378/janelia.7739750.v4. URL https://janelia.figshare.com/articles/dataset/Eight-probe_Neuropixels_recordings_during_spontaneous_behaviors/7739750/4"/>
  <meta name="citation_reference" content="J.D. Victor, K.P. Purpura, Nature and precision of temporal coding in visual cortex: a metric-space analysis. Journal of neurophysiology 76(2), 1310–1326 (1996)"/>
  <meta name="citation_reference" content="J.D. Victor, K.P. Purpura, Metric-space analysis of spike trains: theory, algorithms and application. Network: computation in neural systems 8(2), 127–164 (1997)"/>
  <meta name="citation_reference" content="T. Kreuz, D. Chicharro, C. Houghton, R.G. Andrzejak, F. Mormann, Monitoring spike train synchrony. Journal of neurophysiology 109(5), 1457–1472 (2013)"/>
  <meta name="citation_reference" content="E. Satuvuori, M. Mulansky, N. Bozanic, I. Malvestio, F. Zeldenrust, K. Lenk, T. Kreuz, Measures of spike train synchrony for data with multiple time scales. Journal of neuroscience methods 287, 25–38 (2017)"/>
  <meta name="citation_reference" content="L. Grossberger, F.P. Battaglia, M. Vinck, Unsupervised clustering of temporal patterns in high-dimensional neuronal ensembles using a novel dissimilarity measure. PLoS computational biology 14(7), e1006, 283 (2018)"/>
  <meta name="citation_reference" content="L. Van der Maaten, G. Hinton, Visualizing data using t-sne. Journal of machine learning research 9(11) (2008)"/>
  <meta name="citation_reference" content="L. Van Der Maaten, Accelerating t-sne using tree-based algorithms. The Journal of Machine Learning Research 15(1), 3221–3245 (2014)"/>
  <meta name="citation_reference" content="G. Hinton, Stochastic neighbor embedding. Advances in neural information processing systems 15, 857–864 (2003)"/>
  <meta name="citation_reference" content="L. McInnes, J. Healy, S. Astels, hdbscan: Hierarchical density based clustering. The Journal of Open Source Software 2(11) (2017)"/>
  <meta name="citation_reference" content="E. Satuvuori, T. Kreuz, Which spike train distance is most suitable for distinguishing rate and temporal coding? Journal of neuroscience methods 299, 22–33 (2018)"/>
  <meta name="citation_reference" content="S. Thorpe, A. Delorme, R. Van Rullen, Spike-based strategies for rapid processing. Neural networks 14(6–7), 715–725 (2001)"/>
  <meta name="citation_reference" content="M.J. McGinley, M. Vinck, J. Reimer, R. Batista-Brito, E. Zagha, C.R. Cadwell, A.S. Tolias, J.A. Cardin, D.A. McCormick, Waking state: Rapid variations modulate neural and behavioral responses. Neuron 87(6), 1143–61 (2015)."/>
  <meta name="citation_reference" content="McGinley, Matthew J Vinck, Martin Reimer, Jacob Batista-Brito, Renata Zagha, Edward Cadwell, Cathryn R Tolias, Andreas S Cardin, Jessica A McCormick, David A Neuron. 2015 Sep 23;87(6):1143–61. doi: 10.1016/j.neuron.2015.09.012."/>
  <meta name="citation_reference" content="A.H. Williams, B. Poole, N. Maheswaranathan, A.K. Dhawale, T. Fisher, C.D. Wilson, D.H. Brann, E.M. Trautmann, S. Ryu, R. Shusterman, et al., Discovering precise temporal patterns in large-scale neural recordings through robust and interpretable time warping. Neuron 105(2), 246–259 (2020)"/>
  <meta name="citation_reference" content="T.H. Cormen, C.E. Leiserson, R.L. Rivest, C. Stein, Introduction to algorithms (MIT press, 2009)"/>
  <meta name="citation_reference" content="C. Bleich, M.L. Overton, A linear-time algorithm for the weighted median problem (Courant Institute of Mathematical Sciences, New York University, 1983)"/>
  <meta name="citation_reference" content="F. Bovo. Robustats. https://github.com/FilippoBovo/robustats (2020)"/>
  <meta name="citation_reference" content="M. Denker, A. Yegenoglu, S. Grün, in Neuroinformatics 2018 (2018), p. P19"/>
  <meta name="citation_reference" content="M. Mulansky, T. Kreuz, Pyspike—a python library for analyzing spike train synchrony. SoftwareX 5, 183–189 (2016)"/>
  <meta name="twitter:title" content="SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Neural coding and memory formation depend on temporal spiking sequences that span high-dimensional neural ensembles. The unsupervised discovery and characterization of these spiking sequences requires a suitable dissimilarity measure to spiking patterns. Here, we present a new dissimilarity measure based on optimal transport theory called SpikeShip, which compares multi-neuron spiking patterns based on all the relative spike-timing relationships among neurons. SpikeShip computes the optimal transport cost to make all the relative spike-timing relationships (across neurons) identical between two spiking patterns. We show that this transport cost can be decomposed into a temporal rigid translation term and a vector of neuron-specific transport flows. SpikeShip can be effectively computed for high-dimensional neuronal ensembles, has a low (linear) computational cost, and is sensitive to higher-order correlations. We applied SpikeShip to large-scale Neuropixel recordings during spontaneous activity and visual encoding. We show that high-dimensional spiking sequences detected via SpikeShip reliably distinguish between different natural images and different behavioral states. These spiking sequences carried complementary information to conventional firing rate codes. SpikeShip opens new avenues for studying neural coding and memory consolidation by finding temporal patterns in high-dimensional neural ensembles.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.06.03.131573v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Neural coding and memory formation depend on temporal spiking sequences that span high-dimensional neural ensembles. The unsupervised discovery and characterization of these spiking sequences requires a suitable dissimilarity measure to spiking patterns. Here, we present a new dissimilarity measure based on optimal transport theory called SpikeShip, which compares multi-neuron spiking patterns based on all the relative spike-timing relationships among neurons. SpikeShip computes the optimal transport cost to make all the relative spike-timing relationships (across neurons) identical between two spiking patterns. We show that this transport cost can be decomposed into a temporal rigid translation term and a vector of neuron-specific transport flows. SpikeShip can be effectively computed for high-dimensional neuronal ensembles, has a low (linear) computational cost, and is sensitive to higher-order correlations. We applied SpikeShip to large-scale Neuropixel recordings during spontaneous activity and visual encoding. We show that high-dimensional spiking sequences detected via SpikeShip reliably distinguish between different natural images and different behavioral states. These spiking sequences carried complementary information to conventional firing rate codes. SpikeShip opens new avenues for studying neural coding and memory consolidation by finding temporal patterns in high-dimensional neural ensembles.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-12-09"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.06.03.131573v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2889064"/>
  <title>
   SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bS2AtTUU__HTuXj1_0FEddK0hS2oNzx_STcvRFv_oK4"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2889064","apath":"\/biorxiv\/early\/2022\/12\/09\/2020.06.03.131573.atom","pisa":"biorxiv;2020.06.03.131573v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.06.03.131573v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.06.03.131573v3","title":"SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns","identifier":"node\/2889064"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zJOnygUqNhXXBkbQwndMbHOLvXtqass6vpIR4KJ0l6fogShq+S\\\\\\\/eJdmzGHqxYSw6vOF2Eky3DpOlCpd1gj4wKZ5Lp\\\\\\\/5XmQTpSA+qw1fmRYWh2UQ3fVvwSCNkuYdRf9f37s2A8d7jdxD8NWBR2A42MRgsjHyJiMr+5n63svC9XMgyXLoGYyZzsR8QI8spB5wUGGuV5\\\\\\\/oaSqsQUTj0xJF0\\\\\\\/Eck190Vs3vA5tIx\\\\\\\/dNDLZUpCOZxSq0\\\\\\\/FySEykGcVzqHOf93DENwl9xPn7sk3rKo5XgGAmrf4qrh0O7Q\\\\\\\/CzzSoBLxF20FNoxDU62y3vtT9\\\\\\\/tZAvfjkN6ik2hEs6ituK7XRrJdHQhzUek8mOQGXNMZM0CDo7HW6u6PBVo6QKQPIu5Vv12treS2FHpmzzzQx9FwY1MWZpz4VerGQbj65MCKWvfWH7jMnR+hQrl6iNwUL\\\\\\\/VW5farkFXkJlZjFIIpuRq1qloPuBv+yAres70TipBPJ\\\\\\\/DN3ddDVz\\\\\\\/u1ar+HJeU8I9lvn56mxvszbBY+VVrMqsBQ8qbV8JM5X\\\\\\\/mri7\\\\\\\/Lz8qmxHcFmjrCLUpkp+iO9WpEJdDWW+OmzRgBMa3F5AIbbWIQP5Mhr4GKda5+K7G668i+S37FLmqGdZd96Ck42o8r9\\\\\\\/tBqvz0jF2CkMmUxjW8wnPNZclvsHgmMfyioZ39G1wJEHTPCelrL5RucaZsFmb7RCV+9ihJTnzmxjer9heTsv+BrJKRi4ueCeNAytwqFypHnIkJ\\\\\\\/G9Y6i6ncWkRUVLUSxbkj5NCxdhRkFm0MdybRU4MrPlDxse9rDi3FaLFaiGiwfFsquhFDt+sIrh5exbH9KOACxWAt1\\\\\\\/bU1Yy7944K0FvGtt00UbajsRMilB67TecKXpXLb\\\\\\\/rmxNRLQT8kKA6\\\\\\\/z1oYL3lUN+UZo\\\\\\\/Q+A01zOpDeQdztN6pDLBmIZe8jV81vJiA4Wi5bbAv+ssY5fHwwmnnWKEQFkayVBZn1DpGeoHhJPSHo78kR7DQPnORlTvox7CPU9qpEqCSj1Fo6c06GWvYf0zXepdHTUMraHnqEt4RIte67RchYx\\\\\\\/BdGEV7QM1LIiFNFvo774K+mWICXZ55GT7tlpM5WnmZI8EGz0P9U\\\\\\\/t\\\\\\\/r5ml17DpPI7yY8P8lQBYE8GJEB9HXt58Duy5S9IGK2yjA83+HqkZPzCP0BMHue5QIW+T455MNsAIB1MhxFRqFWwenge3Tc+4+Zpxp28rt0CEH51x54Ejs0CHq9ePpbb9Hj9znR4BfKKBOqmdmqh1HgPr\\\\\\\/Wde1h4EcjlLVsL6e16Tc7v\\\\\\\/4o9ywAGrhGCL\\\\\\\/+UYTef0+VAaTMvlEgHFmb3\\\\\\\/yNtokD4pd5XBnAZmmhwPvTaalw1HGmuoANrU8HRDjMYkfT6P09rcKImo7R8hwsEw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022jMBuB0Pafmya51JIaebPNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002256bd2751dcce4fc5b95e37f6d521a31ae21a44cea1d5f5d1bbfcb356a3786850\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.06.03.131573v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2889064 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.06.03.131573v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1046009692">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1046009692" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_509149695" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-pts5leNiKZe60mVKgU3g3xuxZlw-2DrFVW7KID-bdzA"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2889064" data-node-nid="2889064" id="node-2889064--21559173415" data-pisa="biorxiv;2020.06.03.131573v3" data-pisa-master="biorxiv;2020.06.03.131573" data-apath="/biorxiv/early/2022/12/09/2020.06.03.131573.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-6682-3761" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Boris
                      </span>
                      <span class="nlm-surname">
                       Sotomayor-Gómez
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Francesco P.
                      </span>
                      <span class="nlm-surname">
                       Battaglia
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-4044-0970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Martin
                      </span>
                      <span class="nlm-surname">
                       Vinck
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.06.03.131573
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2889064--21559173415" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Boris Sotomayor-Gómez
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Ernst Strüngmann Institute (ESI) for Neuroscience in Cooperation with Max Planck Society
                       </span>
                       , Frankfurt,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Neuroinformatics, Donders Centre for Neuroscience, Radboud University Nijmegen
                       </span>
                       , Nijmegen,
                       <span class="nlm-country">
                        Netherlands
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Boris%2BSotomayor-G%C3%B3mez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sotomayor-G%C3%B3mez%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABoris%2BSotomayor-G%25C3%25B3mez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-6682-3761" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Boris Sotomayor-Gómez
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:boris.sotomayor@esi-frankfurt.de" class="" data-icon-position="" data-hide-link-title="0">
                        boris.sotomayor@esi-frankfurt.de
                       </a>
                       <a href="mailto:martin.vinck@esi-frankfurt.de" class="" data-icon-position="" data-hide-link-title="0">
                        martin.vinck@esi-frankfurt.de
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Francesco P. Battaglia
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Donders Institute for Brain, Cognition and Behaviour, Radboud University Nijmegen
                       </span>
                       , Nijmegen,
                       <span class="nlm-country">
                        Netherlands
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Francesco%2BP.%2BBattaglia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Battaglia%20FP&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AFrancesco%2BP.%2BBattaglia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Martin Vinck
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Ernst Strüngmann Institute (ESI) for Neuroscience in Cooperation with Max Planck Society
                       </span>
                       , Frankfurt,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Neuroinformatics, Donders Centre for Neuroscience, Radboud University Nijmegen
                       </span>
                       , Nijmegen,
                       <span class="nlm-country">
                        Netherlands
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Martin%2BVinck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Vinck%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMartin%2BVinck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-4044-0970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Martin Vinck
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:boris.sotomayor@esi-frankfurt.de" class="" data-icon-position="" data-hide-link-title="0">
                        boris.sotomayor@esi-frankfurt.de
                       </a>
                       <a href="mailto:martin.vinck@esi-frankfurt.de" class="" data-icon-position="" data-hide-link-title="0">
                        martin.vinck@esi-frankfurt.de
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.06.03.131573v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2889064" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2889064/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.03.131573v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2889064" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2889064/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.03.131573v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2889064" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2889064/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.03.131573v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2889064" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2889064/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.06.03.131573v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2889064" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2889064/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Neural coding and memory formation depend on temporal spiking sequences that span high-dimensional neural ensembles. The unsupervised discovery and characterization of these spiking sequences requires a suitable dissimilarity measure to spiking patterns. Here, we present a new dissimilarity measure based on optimal transport theory called SpikeShip, which compares multi-neuron spiking patterns based on all the relative spike-timing relationships among neurons. SpikeShip computes the optimal transport cost to make all the relative spike-timing relationships (across neurons) identical between two spiking patterns. We show that this transport cost can be decomposed into a temporal rigid translation term and a vector of neuron-specific transport flows. SpikeShip can be effectively computed for high-dimensional neuronal ensembles, has a low (linear) computational cost, and is sensitive to higher-order correlations. We applied SpikeShip to large-scale Neuropixel recordings during spontaneous activity and visual encoding. We show that high-dimensional spiking sequences detected via SpikeShip reliably distinguish between different natural images and different behavioral states. These spiking sequences carried complementary information to conventional firing rate codes. SpikeShip opens new avenues for studying neural coding and memory consolidation by finding temporal patterns in high-dimensional neural ensembles.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-5">
                             Information in the brain is encoded by very high-dimensional “ensembles” of neurons, which encode information with spikes. Populations of neurons can produce specific spike patterns depending on sensory inputs or internal variables [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              8
                             </a>
                             ]. With new recording techniques like Neuropixels [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              9
                             </a>
                             ], it has become possible to simultaneously record from thousands of single neurons [
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              10
                             </a>
                             –
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              12
                             </a>
                             ]. This offers new opportunities to uncover the relationship between multi-neuron spiking patterns and sensory inputs or motor outputs, yet also poses unique mathematical challenges for the unsupervised discovery of the “dictionary” of neuronal “code-words”.
                            </p>
                            <p id="p-6">
                             The notion of information encoding relies on the construction of a distance or dissimilarity measure in an N-dimensional space. For example, the distance between binary strings can be measured using the Hamming distance. In the brain, the distance between two multi-neuron spiking patterns is conventionally based on differences in the firing rates (spike / sec). Using this method, it has been shown for example that high-dimensional neural ensembles span a low-dimensional manifold that relates a to stimulus or behavioral variables in a meaningful way [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              13
                             </a>
                             ,
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              14
                             </a>
                             ]. However, firing rates do not capture the potentially rich information contained by the precise temporal order in which spikes are fired, e.g. neuron i firing at time t and neuron j firing at t + τ. For instance, we expect that any time-varying sensory stimulus or action sequence may be encoded by a unique multi-neuron temporal pattern of spiking. Indeed, multineuron temporal sequences can encode information about sensory stimuli and are required for the generation of complex motor patterns like bird songs [
                             <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              7
                             </a>
                             ,
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              15
                             </a>
                             ,
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              16
                             </a>
                             ]. Temporal sequences may also be critical for memory formation, because neural plasticity rules are highly sensitive to the temporal order in which spikes are fired [
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              17
                             </a>
                             –
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              21
                             </a>
                             ]. It is plausible that much of the information contained in spiking sequences has thus far not been discovered, as temporal correlations have typically been studied based on relatively small neural ensembles, whereas the number of pairwise spike-time relationships scales with N
                             <sup>
                              2
                             </sup>
                             .
                            </p>
                            <p id="p-7">
                             A major computational problem is thus to measure the dissimilarity of spiking patterns in terms of the relative spike timing between neurons. Developing such a measure has several challenges, including 1) Techniques that rely on binning spikes and require exact matches of patterns (e.g. information theoretical measures) have several disadvantages: They require a relatively large number of observations due to combinatorial explosion, lack robustness against spike time jitter and reduce temporal resolution due to binning. 2) Computational cost becomes a major constraint for high-dimensional ensembles of neurons, and a measure should ideally have a computational cost that is linear in the number of neurons and spikes. Here, we develop a novel dissimilarity measure for multi-neuron spiking patterns called SpikeShip, which has linear computational complexity of
                             <span class="inline-formula" id="inline-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-1.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-1.gif"/>
                               </noscript>
                              </span>
                             </span>
                             , and has the key advantage of being sensitive to higher-order structure. SpikeShip can be interpreted as the optimal transport cost to make all spike-timing relationships between two different multi-neuron spiking patterns identical. That is, it solves the optimal transport problem for the entire spiking pattern, and yields a unique decomposition of spike pattern dissimilarity in terms of a neuron-specific flows (which controls similarities in terms of relative spike times) as well as a global flow term (which controls the similarity in terms of absolute time). We demonstrated the power of the SpikeShip measure by applying it to large scale, high-dimensional neural ensembles in mice from [
                             <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                              12
                             </a>
                             ] and [
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              22
                             </a>
                             ], and demonstrating that temporal spiking sequences reliably distinguish between natural stimuli and different brain states. We discuss the properties of this measure compared to previous spike train measures like Victor-Purpura Distance (VP) [
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              23
                             </a>
                             ,
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              24
                             </a>
                             ], SPIKE [
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              25
                             </a>
                             ], and Rate-Independent SPIKE (RI-SPIKE) [
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              26
                             </a>
                             ]. Finally, we show that SpikeShip carries orthogonal information compared with the traditional firing rates code.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 Results
                            </h2>
                            <p>
                             Our overall goal is to develop a dissimilarity measure between multi-neuron spiking patterns that is exclusively based on the temporal order of spiking. Suppose that we have measured the spikes from N neurons (“spike trains”), with spike trains divided into M epochs of length T (measured in seconds or samples). Epochs could be defined by e.g. trials (e.g. stimulus presentations) or sliding windows. The problem is to find a dissimilarity measure with the following properties:
                            </p>
                            <ol class="list-ord" id="list-1">
                             <li id="list-item-1">
                              <p id="p-9">
                               The measure should depend on the temporal order of firing across neurons, but not on the spike count.
                              </p>
                             </li>
                             <li id="list-item-2">
                              <p id="p-10">
                               If two spike patterns are identical in terms of cross-neuron spike timing relationships (i.e. they are a temporally translated version of one another), then the dissimilarity measure should equal zero.
                              </p>
                             </li>
                             <li id="list-item-3">
                              <p id="p-11">
                               The measure does not require binning or smoothing, and is based on the exact timing of the spikes.
                              </p>
                             </li>
                             <li id="list-item-4">
                              <p id="p-12">
                               It should measure dissimilarity in a gradual way, and avoid the problem of “combinatorial explosion” that occurs with methods that search for exact matches in spiking patterns. Combinatorial explosion means that for a very large number of neurons, the probability of an exact match in spiking patterns becomes extremely small.
                              </p>
                             </li>
                            </ol>
                            <p id="p-13">
                             We introduce a measure that satisfies these constraints called the SpikeShip measure (see
                             <a id="xref-sec-8-1" class="xref-sec" href="#sec-8">
                              Methods
                             </a>
                             ). The idea of SpikeShip is to measure the dissimilarity between spike trains using the mathematical framework of optimal transport, as shown in
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Fig. 1
                             </a>
                             and
                             <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                              S1
                             </a>
                             . We will consider each spike train as a collection of “masses” (i.e. the spikes). All spikes from each active neuron, together, contribute a unit mass, i.e. the mass of each spike is normalized to the total mass. This ensures the rate invariance of the method. The question now is what the optimal way is of transporting the masses in time to make the two patterns identical in terms of the relative spike times among neurons. Importantly, similarity here is strictly defined based on relative timing among neurons, i.e. not on the absolute timing of the spikes. Intuitively, one would suspect that measuring the similarity of spike train patterns based on the relative spike timing among N neurons has a computational complexity of at least order N
                             <sup>
                              2
                             </sup>
                             , which would make the method impractical for larger data sets. However, we surprisingly show that there is a fundamental solution that can be computed in order N.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F1.large.jpg?width=800&height=600&carousel=1" title="Illustration of SpikeShip A) Example of two epochs with spike times tk = (10, 10, 10, 10, 10, 10) and tm = (25, 40, 45, 55, 60, 70) (note only one spike per neuron in this example). B) Distances between spike times tk and tm. C) The vector  contains the differences of spike times tk and tm. D) Computation of the median of . E) gmin is the optimal global shift such that fi = ci – gmin. The neuron-specific shifts  contain all the information about the structure of distances between tk and tm. SpikeShip equals ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Illustration of SpikeShip</span> A) Example of two epochs with spike times <em>t<sub>k</sub></em> = (10, 10, 10, 10, 10, 10) and <em>t<sub>m</sub></em> = (25, 40, 45, 55, 60, 70) (note only one spike per neuron in this example). B) Distances between spike times <em>t<sub>k</sub></em> and <em>t<sub>m</sub></em>. C) The vector  contains the differences of spike times <em>t<sub>k</sub></em> and <em>t<sub>m</sub></em>. D) Computation of the median of . E) <em>g</em><sup>min</sup> is the optimal global shift such that <em>f<sub>i</sub></em> = <em>c<sub>i</sub></em> – <em>g</em><sup>min</sup>. The neuron-specific shifts  contain all the information about the structure of distances between <em>t<sub>k</sub></em> and <em>t<sub>m</sub></em>. SpikeShip equals .</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F1.medium.gif" width="427" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F1.medium.gif" width="427" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Fig. 1
                              </span>
                              <span class="caption-title">
                               Illustration of SpikeShip
                              </span>
                              <p id="p-14" class="first-child">
                               A) Example of two epochs with spike times t
                               <sub>
                                k
                               </sub>
                               = (10, 10, 10, 10, 10, 10) and t
                               <sub>
                                m
                               </sub>
                               = (25, 40, 45, 55, 60, 70) (note only one spike per neuron in this example). B) Distances between spike times t
                               <sub>
                                k
                               </sub>
                               and t
                               <sub>
                                m
                               </sub>
                               . C) The vector
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               contains the differences of spike times t
                               <sub>
                                k
                               </sub>
                               and t
                               <sub>
                                m
                               </sub>
                               . D) Computation of the median of
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . E) g
                               <sup>
                                min
                               </sup>
                               is the optimal global shift such that f
                               <sub>
                                i
                               </sub>
                               = c
                               <sub>
                                i
                               </sub>
                               – g
                               <sup>
                                min
                               </sup>
                               . The neuron-specific shifts
                               <span class="inline-formula" id="inline-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               contain all the information about the structure of distances between t
                               <sub>
                                k
                               </sub>
                               and t
                               <sub>
                                m
                               </sub>
                               . SpikeShip equals
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F1/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p>
                             We show that the global optimal transport problem can be solved in two steps:
                            </p>
                            <ol class="list-ord" id="list-2">
                             <li id="list-item-5">
                              <p id="p-16">
                               We first compute the optimal transport flow to transform a spiking pattern in epoch k into the spiking pattern in epoch m, such that the patterns are identical in terms of absolute timing.
                              </p>
                             </li>
                             <li id="list-item-6">
                              <p id="p-17">
                               Transport cost is now minimized by computing a global temporal translation term and subtracting this term from all the individual spike shifts. This yields neuron-specific flows, and allows us to compute the total transport cost needed to make two patterns identical in terms of relative spike times.
                              </p>
                             </li>
                            </ol>
                            <p id="p-18">
                             The algorithm starts by computing the Earth Mover Distance (see
                             <a id="xref-sec-8-2" class="xref-sec" href="#sec-8">
                              Methods
                             </a>
                             ,
                             <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                              Fig. S1
                             </a>
                             ) for each neuron separately in step 1, shifting mass from each spike in pattern k to the spikes in pattern m. We denote the flows c
                             <sub>
                              i,u
                             </sub>
                             with moved mass w
                             <sub>
                              i,u
                             </sub>
                             , such that
                             <span class="inline-formula" id="inline-formula-6">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-6.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-6.gif"/>
                               </noscript>
                              </span>
                             </span>
                             .
                            </p>
                            <p id="p-19">
                             The EMDs, however, only indicate the similarity of absolute spike times between two epochs, i.e. are based on the alignment of the spikes relative to an event. Yet, we wish that our measure reflects the relative timing of spikes between neurons. We show that from the EMD flows computed in Step 1, we can subtract a global rigid translation term. This uniquely yields the minimum transport cost to transform the multi-neuron spiking pattern in epoch k such that its relative spike-timing relationships between neurons become identical to another pattern m. In the
                             <a id="xref-sec-8-3" class="xref-sec" href="#sec-8">
                              Methods
                             </a>
                             section we state our main theoretical result, namely that the optimal transport flows are given by
                             <span class="disp-formula" id="disp-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-2.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-2.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                            <p>
                             Here g
                             <sup>
                              min
                             </sup>
                             is the weighted median across all the original flows c
                             <sub>
                              i,u
                             </sub>
                             with associated mass w
                             <sub>
                              i,u
                             </sub>
                             . Thus, we can decompose the transport flow in two terms:
                            </p>
                            <ol class="list-ord" id="list-3">
                             <li id="list-item-7">
                              <p id="p-21">
                               an optimal global shift between two epochs, shared across all neurons; and
                              </p>
                             </li>
                             <li id="list-item-8">
                              <p id="p-22">
                               an optimal neuron-specific transport flow. We then define SpikeShip (see
                               <a id="xref-sec-8-4" class="xref-sec" href="#sec-8">
                                Methods
                               </a>
                               ) as
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-23">
                               Here
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the set of all neurons that fired a spike both in epoch k and m. The weight
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , effectively assigns an equal weight to each neuron that contains at least one spike.
                              </p>
                             </li>
                            </ol>
                            <p id="p-24">
                             The algorithm to compute SpikeShip has computational complexity
                             <span class="inline-formula" id="inline-formula-9">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-9.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-9.gif"/>
                               </noscript>
                              </span>
                             </span>
                             because the weighted median has complexity
                             <span class="inline-formula" id="inline-formula-10">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-10.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-10.gif"/>
                               </noscript>
                              </span>
                             </span>
                             (
                             <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                              Fig. S2
                             </a>
                             ). This means that SpikeShip performs much better in terms of computational complexity than previous measures like SPOTDis (which is
                             <span class="inline-formula" id="inline-formula-11">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-11.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-11.gif"/>
                               </noscript>
                              </span>
                             </span>
                             ) [
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              27
                             </a>
                             ] and it thus becomes feasible to compute for large ensembles of neurons, as we will show further below.
                            </p>
                            <p id="p-25">
                             Having computed a dissimilarity measure between multi-neuron spike trains, we can use embedding and clustering techniques to detect patterns in an unsupervised way. The rationale of our approach is that unsupervised clustering can be performed based on the dissimilarity matrices, rather than on the spike train data itself. To illustrate this, we generated 6 input patterns defined by the instantaneous rate of inhomogeneous Poisson process, as in [
                             <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                              27
                             </a>
                             ]. Noise was generated with random firing based on a homogenous Poisson process with a constant rate (i.e., homogenous noise) (See
                             <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                              Fig. 2A
                             </a>
                             ). We computed the pairwise distances between pairs of epochs using SpikeShip distance, yielding a dissimilarity matrix (2B). The patterns contained in the data can be visualized using manifold learning algorithms such as t-SNE, using the SpikeShip dissimilarity matrix as input [
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              28
                             </a>
                             –
                             <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                              30
                             </a>
                             ] (
                             <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                              Fig 2C
                             </a>
                             ). Furthermore, HDBSCAN [
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              31
                             </a>
                             ] automatically detected clusters on the basis of the SpikeShip dissimilarity matrix. These results illustrate how SpikeShip can unveil multi-neuron spiking patterns and shows its efficiency in simulated, high-dimensional data.
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F2.large.jpg?width=800&height=600&carousel=1" title="Dissimilarity matrices and clustering comparison. A) Six samples from simulated neuronal outputs according to an inhomogenous Poisson process plus one for Noise (using scripts provided by [27]). B) Sorted Dissimilarity matrix by pattern using SpikeShip dissimilarity measure (left), and a 2-dimensional t-SNE embedding using SpikeShip dissimilarity matrix (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Dissimilarity matrices and clustering comparison.</span> A) Six samples from simulated neuronal outputs according to an inhomogenous Poisson process plus one for Noise (using scripts provided by [27]). B) Sorted Dissimilarity matrix by pattern using SpikeShip dissimilarity measure (left), and a 2-dimensional t-SNE embedding using SpikeShip dissimilarity matrix (right).</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F2.medium.gif" width="439" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F2.medium.gif" width="439" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig. 2
                              </span>
                              <span class="caption-title">
                               Dissimilarity matrices and clustering comparison.
                              </span>
                              <p id="p-26" class="first-child">
                               A) Six samples from simulated neuronal outputs according to an inhomogenous Poisson process plus one for Noise (using scripts provided by [
                               <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">
                                27
                               </a>
                               ]). B) Sorted Dissimilarity matrix by pattern using SpikeShip dissimilarity measure (left), and a 2-dimensional t-SNE embedding using SpikeShip dissimilarity matrix (right).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-27">
                             An important property of SpikeShip is that it can distinguish spiking patterns even when they are multi-modal. To demonstrate this, we use multiple bimodal activation of Poisson patterns with patterned noise and homogeneous noise (See
                             <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                              Fig. S3A
                             </a>
                             ). In addition, we simulated a special case when neurons are “deactivated” in a certain segment of the epoch (See
                             <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                              Fig. S3B
                             </a>
                             ). We observed that, in both cases, SpikeShip can detect the patterns successfully and each cluster were well separated as shown in their dissimilarity matrices and 2D t-SNE embeddings (See
                             <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                              Fig. S3
                             </a>
                             ). Finally, compared to the SPOTDis measure, which is also based on optimal transport (SPOTDis), we found that SpikeShip can detect patterns that are defined by lower signal-to-noise ratios (See
                             <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                              Fig. S4
                             </a>
                             ).
                            </p>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Properties of SpikeShip
                             </h3>
                             <p>
                              Next, we show that, besides the properties discussed above, SpikeShip has several other desirable features that make it well suited as a dissimilarity measure for spike train patterns, and that distinguish it from previous spike train metrics:
                             </p>
                             <ol class="list-ord" id="list-4">
                              <li id="list-item-9">
                               <p id="p-29">
                                SpikeShip measures the dissimilarity of spiking patterns between a pair of epochs based on the relative spike timing between neurons. SpikeShip can therefore be used to detect patterns that are spontaneously occurring or that are not locked to the onset of an event, using e.g. sliding window approaches. By contrast, other measures like VP and SPIKE are not based on the relative spike timing. VP and SPIKE can, however, be used to compare spike trains in different epochs based on the absolute spike timing (
                                <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                                 Figure S5B
                                </a>
                                ). In this case, these measures are computed for each neuron separately by directly comparing the spike trains in different epochs, one neuron at a time.
                               </p>
                              </li>
                              <li id="list-item-10">
                               <p id="p-30">
                                SpikeShip has a linear dependence on differences in relative spike-timing between epochs (
                                <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                 Figure S5B
                                </a>
                                ). Conversely, because VP has a hyperparameter (q), it can have a highly non-linear dependence on spike timing. This is because as differences in timing become larger, VP is exclusively driven by insertions, such that the total cost does not increase (
                                <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                                 Figure S5B
                                </a>
                                ). We further observed that the two other measures, SPIKE and RI-SPIKE, were also influenced by the distance of spiking patterns to the edge of the window (
                                <a id="xref-fig-10-4" class="xref-fig" href="#F10">
                                 Figure S5B
                                </a>
                                ).
                               </p>
                              </li>
                              <li id="list-item-11">
                               <p id="p-31">
                                By design, SpikeShip is only sensitive to timing relationships, and not to firing rates. The reason is that all the spike trains due to normalization have the same mass. By contrast, VP should have a strong rate sensitivity because it also includes the insertion cost (See
                                <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                 Fig. S6
                                </a>
                                ). In fact, VP will be only rate sensitive when its hyperparameter q is very small or very large, and exhibits temporal sensitivity only for intermediate values of q. While SPIKE may have some rate sensitivity, RI-SPIKE is designed to have low rate sensitivity [
                                <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                                 26
                                </a>
                                ].
                               </p>
                              </li>
                             </ol>
                             <p id="p-32">
                              To illustrate these differences, we first examined an example previously shown in [
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               32
                              </a>
                              ], with three patterns that either differ by firing rate (1 and 2) or by timing (1 and 2 vs. 3) (
                              <a id="xref-fig-10-5" class="xref-fig" href="#F10">
                               Figure S5A
                              </a>
                              ). In this example, the VP distance is primarily dominated by rate differences, irrespective of the choice of q. In the example shown in
                              <a id="xref-fig-10-6" class="xref-fig" href="#F10">
                               Figure S5A
                              </a>
                              , three patterns are shown that either differ by firing rate (1 and 2) or by timing (1 and 2 vs. 3). For all q, VP does not assign the lowest between patterns 1 and 2 despite these two patterns having a very close temporal relationship. By contrast, SpikeShip, SPIKE, and RI-SPIKE do assign a much lower cost between patterns 1 and 2 (See
                              <a id="xref-fig-10-7" class="xref-fig" href="#F10">
                               Fig. S5A
                              </a>
                              ). Additionally, we noted that several measures require temporal alignment. To demonstrate this point, we simulated three Poisson patterns A, B, and C, as shown in
                              <a id="xref-fig-10-8" class="xref-fig" href="#F10">
                               Fig. S5B
                              </a>
                              . Here, pattern A contains Poisson spikes in a specific interval of the window length. Also, we generated the same amount of patterns but after applying a linear shift for half of the neural population and the entire neural population (pattern B and C, respectively) as illustrated in
                              <a id="xref-fig-10-9" class="xref-fig" href="#F10">
                               Fig. S5B
                              </a>
                              . We found that SPIKE and RI-SPIKE are affected by the position of the stimulus onset. Additionally, VP assigns a maximum distance when the cost of shifting spikes is greater than the cost to insert them. Thus, these measures require a clear definition of the stimulus onset to cover due to the definition of the window length affects their distances.
                             </p>
                             <p id="p-33">
                              Second, we studied different spiking sequences together with a local or global scaling of firing rates across epochs. As shown in
                              <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                               Fig. S6
                              </a>
                              , VP distance did not uniquely distinguish the patterns based on temporal or rate structure, and is dominated by differences in firing rates for a large range of q. By contrast, the performance of SpikeShip was not affected by the scaling of firing rates between epochs (See
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Fig. S7
                              </a>
                              and
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               S8
                              </a>
                              ). Finally, both SPIKE and RI-SPIKE were less effective than SpikeShip as they assign very high dissimilarities between the noise spike trains, such that the overall clustering is worse than for SpikeShip S9.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Application to Allen Brain Institute’s neural datasets
                             </h3>
                             <p id="p-34">
                              Next, we applied SpikeShip to several high-dimensional neural datasets. Previous work has shown that the visual system can very rapidly process natural images and extract categorical information [
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               33
                              </a>
                              ]. It has been proposed that this relies on a temporal coding strategy, whereby visual information is encoded based on the temporal sequence of spikes relative to stimulus onset [
                              <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                               33
                              </a>
                              ].
                             </p>
                             <p id="p-35">
                              Here, we used SpikeShip to determine whether temporal spiking sequences in six visual areas can reliably distinguish natural images from each other. To this end, we analysed Neuropixel data from 32 mice while they passively viewed natural images (dataset from the Allen Institute for Brain Science; see
                              <a id="xref-sec-8-5" class="xref-sec" href="#sec-8">
                               Methods
                              </a>
                              ). A total of 20 natural scenes were selected with 10 repetitions each (i.e. M = 200 epochs) as shown in
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3A
                              </a>
                              . To create a high-dimensional vector of neurons, we pooled together all recorded neurons (N = 8, 301) across the 32 mice (See
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3B
                              </a>
                              ).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F3.large.jpg?width=800&height=600&carousel=1" title="SpikeShip analyzis of spike sequences for natural scenes presentations (Allen Brain Institute). A) Sample of 10 Natural Scenes from Allen Brain Institute’s dataset. B) Raster plot of two epochs with N = 8, 301 neurons for each presentation. C) Left: Sorted dissimilarity matrix by image ID for 20 natural scenes presentations with 10 presentations each (M = 200). Right: 2-dimensional t-SNE embedding for each presentation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">SpikeShip analyzis of spike sequences for natural scenes presentations (Allen Brain Institute).</span> A) Sample of 10 Natural Scenes from Allen Brain Institute’s dataset. B) Raster plot of two epochs with <em>N</em> = 8, 301 neurons for each presentation. C) Left: Sorted dissimilarity matrix by image ID for 20 natural scenes presentations with 10 presentations each (<em>M</em> = 200). Right: 2-dimensional t-SNE embedding for each presentation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F3.medium.gif" width="308" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F3.medium.gif" width="308" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 3
                               </span>
                               <span class="caption-title">
                                SpikeShip analyzis of spike sequences for natural scenes presentations (Allen Brain Institute).
                               </span>
                               <p id="p-36" class="first-child">
                                A) Sample of 10 Natural Scenes from Allen Brain Institute’s dataset. B) Raster plot of two epochs with N = 8, 301 neurons for each presentation. C) Left: Sorted dissimilarity matrix by image ID for 20 natural scenes presentations with 10 presentations each (M = 200). Right: 2-dimensional t-SNE embedding for each presentation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-37">
                              The pairwise SpikeShip distance is presented in
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                              as a dissimilarity matrix sorted by the presentations of each natural scene. The t-SNE embedding revealed clear clustering of spiking patterns based on SpikeShip, such that different natural images could be reliably distinguished from each other (See
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3D
                              </a>
                              ). Hence, natural images yielded distinct temporal spiking sequences that were time-locked to stimulus onset (so that they could be extraced from combined data from multiple sessions), in support of the idea that the visual system may use a temporal coding strategy. In sum, these findings demonstrate that SpikeShip can unveil multi-neuron temporal spiking patterns from highdimensional recordings.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Comparison of SpikeShip vs firing rates in visual stimuli
                             </h3>
                             <p id="p-38">
                              We wondered how the information in temporal spiking sequences compared to the information carried by conventional firing rate codes. Our first question was, which one of these two codes conveyed more information. To determine this, we computed a distance matrix for the firing rates, by computing the Euclidean distance between the firing rate vectors. These distances were computed in the same time window as we used for SpikeShip (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ). We then computed a measure of discriminability between natural images based on the distances within and between images (see
                              <a id="xref-sec-8-6" class="xref-sec" href="#sec-8">
                               Methods
                              </a>
                              ,
                              <a id="xref-disp-formula-27-1" class="xref-disp-formula" href="#disp-formula-27">
                               Eq. 26
                              </a>
                              ).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F4.large.jpg?width=800&height=600&carousel=1" title="Comparison of pairwise distances between Firing rates and SpikeShip. A) Dissimilarity matrices for Allen Brain Institute data (sorted by natural scenes). Colormaps were changed to show more variability in distances. B) Discriminability index for dissimilarity matrices shown in A) by using firing rates (dFR) and SpikeShip (dSS)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparison of pairwise distances between Firing rates and SpikeShip.</span> A) Dissimilarity matrices for Allen Brain Institute data (sorted by natural scenes). Colormaps were changed to show more variability in distances. B) Discriminability index for dissimilarity matrices shown in A) by using firing rates (<em>d<sub>FR</sub></em>) and SpikeShip (<em>d<sub>SS</sub></em>).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F4.medium.gif" width="440" height="316"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F4.medium.gif" width="440" height="316"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 4
                               </span>
                               <span class="caption-title">
                                Comparison of pairwise distances between Firing rates and SpikeShip.
                               </span>
                               <p id="p-39" class="first-child">
                                A) Dissimilarity matrices for Allen Brain Institute data (sorted by natural scenes). Colormaps were changed to show more variability in distances. B) Discriminability index for dissimilarity matrices shown in A) by using firing rates (d
                                <sub>
                                 FR
                                </sub>
                                ) and SpikeShip (d
                                <sub>
                                 SS
                                </sub>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-40">
                              We found that there was a higher discriminability between natural images for SpikeShip as compared to firing rate vectors for all the natural scenes (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4B
                              </a>
                              ).
                             </p>
                             <p id="p-41">
                              Next, we wondered to what extent the information in SpikeShip was independent of the firing rate information. To this end, we computed the Spearman correlation between the SpikeShip distance and firing rate dissimilarity matrices, which contained information from all epochs across all natural images. We found that the dissimilarity matrices of SpikeShip and firing rates were only weakly correlated across epochs (Spearman correlation equals 0.1804). Accordingly, the t-SNE visualization shows that the relative locations of the clusters show major differences between both methods (
                              <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                               Fig. S10
                              </a>
                              ).
                             </p>
                             <p id="p-42">
                              Altogether, these results show that SpikeShip and firing rates contain different and to a large extent independent information about natural stimuli. Furthermore, SpikeShip allowed for a better separation of the different natural stimuli in comparison to firing rates. We further observed that both SPIKE and RI-SPIKE were less effective in separating the different patterns than SpikeShip, and showed a stronger correlation with firing rates (
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Fig. S11
                              </a>
                              ). These findings support the idea that the visual system may use spiking sequences to encode information about natural scenes, as proposed by e.g. [
                              <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">
                               33
                              </a>
                              ].
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Application to spontaneous activity
                             </h3>
                             <p id="p-43">
                              Next, we applied SpikeShip to high-dimensional neural recordings from multiple brain areas while mice spontaneously transitioned between different behavioral states. Previous works have shown that behavioral states have major effects on the firing rates of neurons across multiple brain areas [
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               10
                              </a>
                              ,
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               34
                              </a>
                              ]. Recent work has shown that different facial motion components outperform the prediction of firing rates as compared to running speed and pupil diameter (a measure of arousal). Hence, we wondered whether different behavioral states are accompanied and distinguished by specific spiking sequences.
                             </p>
                             <p id="p-44">
                              To investigate this, we analysed the data set of [
                              <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                               22
                              </a>
                              ], which contains multiareal recordings from > 1000 neurons in three mice. Similar to [
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               10
                              </a>
                              ], we distinguished between different states based on the facial motion components using the SVD (singular value decomposition), and identified low, medium, and high-motion epochs (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5A-C
                              </a>
                              ). We randomly selected one hundred epochs for both low-, medium and high-motion states and computed the SpikeShip dissimilarity matrix (M = 200, See 5D).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F5.large.jpg?width=800&height=600&carousel=1" title="Spontaneous activity analyzes. A) First 4 SVD dimensions of [22] dataset. B) subtraction of the median of the first four SVD motion dimensions (denoising). C) Normalized SVD motion used to detect low- and high-states. Circles represent samples of temporal windows for posterior analyzes. D) Raster plot for N = 2, 296 neurons for different epochs. E) Multi-spike sequence analyzes. Left: Dissimilarity Matrices. Middle: 2D t-SNE embedding. Right: 2D Spectral Embedding (Laplacian Eigenmaps). F) Mean SpikeShip distance from [10] experiments. G) Spearman correlation between firing rates and SpikeShip per experiment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Spontaneous activity analyzes.</span> A) First 4 SVD dimensions of [22] dataset. B) subtraction of the median of the first four SVD motion dimensions (denoising). C) Normalized SVD motion used to detect low- and high-states. Circles represent samples of temporal windows for posterior analyzes. D) Raster plot for <em>N</em> = 2, 296 neurons for different epochs. E) Multi-spike sequence analyzes. Left: Dissimilarity Matrices. Middle: 2D t-SNE embedding. Right: 2D Spectral Embedding (Laplacian Eigenmaps). F) Mean SpikeShip distance from [10] experiments. G) Spearman correlation between firing rates and SpikeShip per experiment.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F5.medium.gif" width="294" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F5.medium.gif" width="294" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 5
                               </span>
                               <span class="caption-title">
                                Spontaneous activity analyzes.
                               </span>
                               <p id="p-45" class="first-child">
                                A) First 4 SVD dimensions of [
                                <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">
                                 22
                                </a>
                                ] dataset. B) subtraction of the median of the first four SVD motion dimensions (denoising). C) Normalized SVD motion used to detect low- and high-states. Circles represent samples of temporal windows for posterior analyzes. D) Raster plot for N = 2, 296 neurons for different epochs. E) Multi-spike sequence analyzes. Left: Dissimilarity Matrices. Middle: 2D t-SNE embedding. Right: 2D Spectral Embedding (Laplacian Eigenmaps). F) Mean SpikeShip distance from [
                                <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">
                                 10
                                </a>
                                ] experiments. G) Spearman correlation between firing rates and SpikeShip per experiment.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-46">
                              In
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5E
                              </a>
                              , we show the dissimilarity matrix for one mouse (“Waksman”; N = 2, 688, for the other two mice see
                              <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                               Fig. S12
                              </a>
                              ). The first 100 epochs represent low-motion states, and the remaining 100 are the middle- and high-motion epochs. The dissimilarity matrix reveals that the spiking sequences during medium and high motion are relatively similar to each other, whereas there is a relatively high variability among sequences during low-motion epochs (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5E-F
                              </a>
                              ). Furthermore, both the t-SNE embedding and the spectral embedding show a separate state-space region for the spiking sequences during medium and high motion.
                             </p>
                             <p id="p-47">
                              We further wondered if SpikeShip contains orthogonal information compared to firing rates, similar to what we had observed for natural images. Again, we computed the Spearman correlations between the SpikeShip and the firing rate dissimilarity matrices. We found relatively weak correlations between the firing rate and SpikeShip dissimilarity matrices for the three mice: (0.189, 0.024, −0.064) for N = (1462, 2296, 2688) neurons, respectively (See
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5G
                              </a>
                              ).
                             </p>
                             <p id="p-48">
                              Altogether, these findings indicate that different brain states give rise to specific temporal correlation patterns across neurons, with relatively homogeneous spiking sequences during active behavior as compared to quiescence.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-7">
                            <h2 class="">
                             3 Discussion
                            </h2>
                            <p id="p-49">
                             We studied the problem of measuring the dissimilarity between two multineuron spiking patterns based on the relative spike-timing relationships across neurons (i.e. firing order). We developed a new measure called SpikeShip. SpikeShip solves the problem of optimally transporting the spikes of individual neurons, such that the global pattern of spike-timing relationships becomes identical between two epochs. Intuitively, one would think that such a measure has a computational complexity of at least
                             <span class="inline-formula" id="inline-formula-12">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-12.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-12.gif"/>
                               </noscript>
                              </span>
                             </span>
                             , but we show that it can be computed with
                             <span class="inline-formula" id="inline-formula-13">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-13.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-13.gif"/>
                               </noscript>
                              </span>
                             </span>
                             computational complexity; this is a major improvement over our previous work [
                             <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">
                              27
                             </a>
                             ]. We show that a dissimilarity between two spiking patterns can be decomposed into neuron-specific flows and a temporal rigid translation term. Importantly, SpikeShip is not restricted to the 2nd order correlations, but is based on the higher-order structure in the spike train.
                            </p>
                            <p id="p-50">
                             Our technique can also be used to align temporal sequences in an unsupervised way if there is a global jitter between spike patterns. This achieves a similar goal as a recent study [
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              35
                             </a>
                             ], described as “warping”; however, the method in our paper does not require any parameter estimation; furthermore, it can also be used to align trials corresponding to different stimuli or behavioral conditions. For example, suppose that we want to analyse the sequence of neural activation in relationship to some event, e.g. freely moving your arm. Suppose that we do not know the exact onset of the neuronal sequence in relation to moving your arm, and we record multiple realizations of this event. With SpikeShip, we can immediately decompose the transport cost between any two trials in terms of a global translation term (which is a non-linear computation) and a neuron-specific shift. This allows one to align trials that were recorded in different conditions, e.g. moving your arm left or right. And it yields, in essence, a state-space characterization of temporal trajectories (e.g. related to different movements), rather than a trajectory through a state-space of firing rate vectors.
                            </p>
                            <p id="p-51">
                             We applied SpikeShip to large, real neuronal datasets of experiments in mice. We found that SpikeShip can be used for the unsupervised decoding of different natural images from a high-dimensional temporal spiking pattern across N > 8000 neurons. Interestingly, we found that SpikeShip outperformed the classical firing rate vector, and that the spike timing information was only weakly redundant with the information in the firing rate vector. This suggests that spike timing information carries additional information relative to the firing rate, as has been hypothesized by [
                             <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">
                              33
                             </a>
                             ]. Interestingly, the SpikeShip technique does not require the explicit identifications of spike latencies, and is able to extract higher-order correlations from spiking patterns and also distinguish multimodal patterns from each other [
                             <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">
                              27
                             </a>
                             ]. Furthermore, as we showed here, the computation is extremely efficient. We further analysed large-scale recordings from the visual cortex, retrosplenial, sensorimotor, frontal, striatum, hippocampus, thalamus, and midbrain. We showed that the temporal structure of spike trains distinguished between low- and high-motion epochs, and again provided orthogonal information to the firing rate code. Furthermore, we found that spiking patterns become more homogeneous during high-motion epochs, which is consistent with the notion that arousal improves the reliability of signal transmission in the cortex [
                             <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                              34
                             </a>
                             ].
                            </p>
                            <p id="p-52">
                             SpikeShip has several advantages or distinct properties as compared to other measures: 1) It is useful to note that there are different measures that are designed for distinct computational problems. SpikeShip is explicitly designed to measure the similarity between patterns in terms of relative spike time relationships. This distinguishes it from e.g. the VP distance, which measures similarity in terms of absolute spike times. We note that while SpikeShip is designed to measure similarity based on relative spike timing, it can also be used to measure similarity in terms of the absolute timing of spikes. 2) SpikeShip is by design not sensitive to global or local scaling of firing rates, as opposed to VP. We note that also RI-SPIKE is designed to be insensitive to a scaling of firing rates. We note however that SpikeShip dissimilarity between any two epochs is based on the spike trains of those neurons that fire at least one spike. Thus, although the value of SpikeShip is not biased by firing rate, the firing rate can influence which neurons the measure is computed over, in particular when neurons have very low (baseline) firing rates. 3) SpikeShip has the major advantage of finding sequences based on higher-order temporal structure based on relative spike timing, with a computational cost of
                             <span class="inline-formula" id="inline-formula-14">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-14.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-14.gif"/>
                               </noscript>
                              </span>
                             </span>
                             . As shown here, SpikeShip is highly noise robust. 5) SpikeShip can be used for the detection of multi-modal patterns, which e.g. distinguishes it from methods that detect sequences based on the latency of firing. 6) SpikeShip is a bin-less measure, i.e. it is based on exact spike timings. 7) SpikeShip does not require exact matches between patterns, but is based on a metric distance (earth mover distance) that reflects the magnitude of differences in relative spike timings. This is an important difference relative to measures that e.g. require exact matches in patterns. Yet it also distinguishes it from e.g. VP, which shows a highly non-linear dependence on timing differences, and SPIKE and RI-SPIKE, which show a non-monotonic dependence on timing differences.
                            </p>
                            <p id="p-53">
                             Looking forward, SpikeShip opens new avenues to study temporal sequences in high-dimensional neural ensembles. Recent technological developments now allow for recordings of thousands of neurons simultaneously, either using electrical recordings or two-photon imaging [
                             <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                              9
                             </a>
                             ]. The technique developed here is applicable to both kinds of data, due to linear computation time. Application of SpikeShip to such kind of data might generate important insights into the role of temporal sequences in sensory coding and memory consolidation.
                            </p>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Materials and methods
                            </h2>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Derivation of SpikeShip
                             </h3>
                             <p id="p-54">
                              Here we derive a new dissimilarity measure, called SpikeShip, which has computational complexity
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for one pair of epochs. To derive this measure, we first consider the simplified case where each ith neuron fires one spike for all M epochs.
                             </p>
                             <div id="sec-10" class="subsection">
                              <h4>
                               SpikeShip for a single spike per epoch
                              </h4>
                              <p id="p-55">
                               Let
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               be a vector of flows for each neuron in epoch m. In other words, f
                               <sub>
                                1, km
                               </sub>
                               is the shift of the spike fired by the first neuron in the mth epoch. The total moving cost equals
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-56">
                               The problem statement is to find a flow vector
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               such that after moving the spikes, the resulting spike train patterns are identical between epoch k and epoch m, in terms of the full matrix of spike timing relationships. We wish to find the flow vector that satisfies this constraint with minimum cost F
                               <sub>
                                km
                               </sub>
                               .
                              </p>
                              <div id="sec-11" class="subsection">
                               <h5>
                                Example
                               </h5>
                               <p id="p-57">
                                Consider the following example: Suppose there are two epochs for N = 6 neurons with spike times
                                <span class="inline-formula" id="inline-formula-18">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-18.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-18.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="inline-formula" id="inline-formula-19">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-19.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-19.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . We will show that the flow vector with minimum cost, such that spike patterns have identical temporal structure, equals
                                <span class="inline-formula" id="inline-formula-20">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-20.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-20.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                (
                                <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                 Fig. 1
                                </a>
                                ).
                               </p>
                               <p id="p-58">
                                More formally, let t
                                <sub>
                                 m,i
                                </sub>
                                be the timing of the spike for the ith neuron in the mth epoch. We denote the spike times after (post) moving them in epoch m as
                                <span class="disp-formula" id="disp-formula-4">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-9.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-9.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                ∀i, where we omitted the subscripts k, m from the variable f
                                <sub>
                                 i,km
                                </sub>
                                for simplicity. The constraint that all the across-neuron spike timing relationships should be identical after moving implies that
                                <span class="disp-formula" id="disp-formula-5">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-10.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-10.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-59">
                                In other words, the delay between two spikes from two different neurons (i, j) should be identical between pattern k and m after moving the spikes. Substituting based on
                                <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">
                                 Eq. 4
                                </a>
                                , this can be expressed as
                                <span class="disp-formula" id="disp-formula-6">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-11.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-11.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-60">
                                Let c
                                <sub>
                                 i
                                </sub>
                                be the shift in spike timing for each neuron in epoch m, such that the spike train patterns become identical,
                                <span class="disp-formula" id="disp-formula-7">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-12.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-12.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-61">
                                Note that with this definition of c
                                <sub>
                                 i
                                </sub>
                                , the equation
                                <span class="disp-formula" id="disp-formula-8">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-13.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-13.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                holds for all (i, j). For all i, we can express f
                                <sub>
                                 i
                                </sub>
                                as a function of the shift c
                                <sub>
                                 i
                                </sub>
                                , such that
                                <span class="disp-formula" id="disp-formula-9">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-14.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-14.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-62">
                                We wish to solve
                                <span class="disp-formula" id="disp-formula-10">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-15.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-15.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                under the constraint of
                                <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                                 Eq 6
                                </a>
                                .
                               </p>
                               <p id="p-63">
                                From
                                <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">
                                 Eqs 6
                                </a>
                                ,
                                <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                                 8
                                </a>
                                and
                                <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                                 9
                                </a>
                                it follows that
                                <span class="disp-formula" id="disp-formula-11">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-16.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-16.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-64">
                                Hence the equation g
                                <sub>
                                 i
                                </sub>
                                = g
                                <sub>
                                 j
                                </sub>
                                holds for all (i, j). Thus, we can rewrite
                                <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                                 Eq. 10
                                </a>
                                to
                                <span class="disp-formula" id="disp-formula-12">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-17.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-17.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                to find a global shift that minimizes the L1-norm of the residuals. The solution to this equation is
                                <span class="disp-formula" id="disp-formula-13">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-18.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-18.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-65">
                                Thus, our main result is that the original shifts between the two spiking patterns can be written as the decomposition
                                <span class="disp-formula" id="disp-formula-14">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-19.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-19.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                for all i, i.e. the optimal transport in terms of neuron-specific shifts and a global temporal rigid translation term. Thus, the optimal transport between two spiking patterns (e.g. after stimulus onset) can be decomposed into the optimal transport in terms of neuron-specific shifts and a global temporal rigid translation term.
                               </p>
                              </div>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               Global shift definition for multiple spikes
                              </h4>
                              <p id="p-66">
                               We now consider the case where in each epoch, every neuron fires a variable number of spikes. We will show that a similar derivation for SpikeShip can be made based on the weighted median. Let n
                               <sub>
                                i,k
                               </sub>
                               and n
                               <sub>
                                i,m
                               </sub>
                               be the number of spikes for the i-th neuron in epoch k and m. We will also assign a weight to each spike, such that the total weight per neuron in the computation of SpikeShip is equal. In order to do so, we first find the smallest common multiple of the spike counts for each neuron, denoted
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . For instance, if n
                               <sub>
                                i,k
                               </sub>
                               = 2 and n
                               <sub>
                                i,m
                               </sub>
                               = 6 then the least common multiple equals 6. We now replicate each spike
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               times. Each spike will now have a weight of
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . (Note that in the actual computation, we do not replicate the spikes in practice, but use an algorithm similar to the one detailed in [
                               <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">
                                27
                               </a>
                               ]). Then, based on the optimal transport cost (EMD), we obtain shifts c
                               <sub>
                                i,u
                               </sub>
                               for the u-th spike of the i-th neuron,
                               <span class="inline-formula" id="inline-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Note that we have shown EMD for spike trains can be efficiently computed by first shifting the mass from most left-ward spike (i.e. first spike) out of
                               <span class="inline-formula" id="inline-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               spikes in epoch m to the most left-ward spike in epoch k [
                               <a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">
                                27
                               </a>
                               ], and then proceeding with the second spike etc.
                              </p>
                              <p id="p-67">
                               We use a similar derivation as the one above. Let t
                               <sub>
                                m,i,u
                               </sub>
                               be the timing of the u – th replicated spike for neuron i in the mth epoch, for all (i, m, u),
                               <span class="inline-formula" id="inline-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . We denote the spike times after (post) moving them in epoch m as
                               <span class="disp-formula" id="disp-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ∀(i, u), where we omitted the subscripts k, m from the variable f
                               <sub>
                                i,km,u
                               </sub>
                               for simplicity. The constraint that all the across-neuron spike timing relationships should be identical after moving implies that
                               <span class="disp-formula" id="disp-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-68">
                               In other words, the delay between two spikes from two different neurons (i, j) should be identical between pattern k and m after moving the spikes. Substituting based on
                               <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">
                                Eq. 15
                               </a>
                               , this can be expressed as
                               <span class="disp-formula" id="disp-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-69">
                               There is one additional constraint, namely
                               <span class="disp-formula" id="disp-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-70">
                               In other words, all the pairwise relationships after moving within the same neuron should be identical.
                              </p>
                              <p id="p-71">
                               Let c
                               <sub>
                                i, u
                               </sub>
                               be the shift in spike timing for each neuron in epoch m, such that the spiking patterns in the window become identical,
                               <span class="disp-formula" id="disp-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-72">
                               Note that with this definition of c
                               <sub>
                                i,u
                               </sub>
                               , the equation
                               <span class="disp-formula" id="disp-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               holds for all (i, j, u). For all i, u, we can express f
                               <sub>
                                i,u
                               </sub>
                               as a function of the shift c
                               <sub>
                                i, u
                               </sub>
                               , such that
                               <span class="disp-formula" id="disp-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-73">
                               Define
                               <span class="inline-formula" id="inline-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-74">
                               We wish to solve
                               <span class="disp-formula" id="disp-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               under the constraints of
                               <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">
                                Eq 17
                               </a>
                               ,
                               <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">
                                18
                               </a>
                               and
                               <a id="xref-disp-formula-20-1" class="xref-disp-formula" href="#disp-formula-20">
                                20
                               </a>
                               .
                              </p>
                              <p id="p-75">
                               Given these two constraints, the equation g
                               <sub>
                                i,u
                               </sub>
                               = g
                               <sub>
                                j,u
                               </sub>
                               holds for all (i, j, u). Thus, we can rewrite
                               <a id="xref-disp-formula-22-1" class="xref-disp-formula" href="#disp-formula-22">
                                Eq. 22
                               </a>
                               to
                               <span class="disp-formula" id="disp-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to find a global translation term that minimizes the L1-norm of the residuals. The solution to this this equation is the weighted median g,
                               <span class="disp-formula" id="disp-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-76">
                               Now average mover cost, SpikeShip, equals
                               <span class="disp-formula" id="disp-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the set of neurons that are active in both epochs k and m. Thus, we derive the result that the original shifts between the two spiking patterns can be written as the decomposition
                               <span class="disp-formula" id="disp-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for all (i, u), i.e. the optimal transport in terms of neuron-specific shifts and a global translation term. Thus, the optimal transport between two spiking patterns (e.g. after stimulus onset) can be decomposed into the optimal transport in terms of neuron-specific shifts and a global temporal rigid translation term. This computation has linear complexity
                               <span class="inline-formula" id="inline-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . We base the computation of the weighted median on [
                               <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                36
                               </a>
                               ,
                               <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                37
                               </a>
                               ] and an adaptation of the Robustats [
                               <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                38
                               </a>
                               ] Python library. In practice, the computation of c
                               <sub>
                                i,u
                               </sub>
                               (optimal transport per neuron) can be efficiently performed not by replicating the spikes to the common multiple integer (used in the derivation above), but by using a similar algorithm as in [
                               <a id="xref-ref-36-8" class="xref-bibr" href="#ref-36">
                                27
                               </a>
                               ].
                              </p>
                             </div>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Comparison with other spike train metrics
                             </h3>
                             <div id="sec-14" class="subsection">
                              <h4>
                               Victor-Púrpura distance
                              </h4>
                              <p id="p-77">
                               Victor-Púrpura metric (VP) combines both rate and temporal information by defining an hyper-parameter q related with the cost of shift between spikes. Thus, VP extracts rate information for small values of q, converging to the absolute difference of spike counts for two spike trains with n
                               <sub>
                                i
                               </sub>
                               and n
                               <sub>
                                j
                               </sub>
                               spikes when q = 0 (i.e., VP(q = 0) = |n
                               <sub>
                                i
                               </sub>
                               – n
                               <sub>
                                j
                               </sub>
                               |). On the other hand, for high values of q, VP distance maximizes the contribution of timing coding, converging to the sum of the total spike of both spike trains (i.e.,
                               <span class="inline-formula" id="inline-formula-31">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/inline-graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ). We used Elephant (Electrophysiology Analysis Toolkit) [
                               <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                39
                               </a>
                               ] to compute VP distance.
                              </p>
                             </div>
                             <div id="sec-15" class="subsection">
                              <h4>
                               SPIKE and RI-SPIKE distances
                              </h4>
                              <p id="p-78">
                               Both SPIKE and RI-SPIKE measure the similarity of two spike trains in terms of the absolute spike timing. RI-SPIKE was developed to avoid a rate bias in the computation of the dissimilarity. To perform the analyses for SPIKE and RI-SPIKE metrics, we used a python library for spike train similarity analysis called PySpike [
                               <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                40
                               </a>
                               ].
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               SPOTDis
                              </h4>
                              <p id="p-79">
                               Previously, we have developed a dissimilarity measure between multi-neuron temporal spiking patterns called SPOTDis (Spike Pattern Optimal Transport Dissimilarity) [
                               <a id="xref-ref-36-9" class="xref-bibr" href="#ref-36">
                                27
                               </a>
                               ]. SPOTDis is defined as the minimum energy (optimal transport) that is needed to transform all pairwise cross-correlations of one epoch k into the pairwise cross-correlations of another epoch m. This optimal transport is given by EMD (Earth Mover Distance). SPOTDis only measures pairwise correlations and has computational cost of order N
                               <sup>
                                2
                               </sup>
                               . We used SPOTDis python module [
                               <a id="xref-ref-36-10" class="xref-bibr" href="#ref-36">
                                27
                               </a>
                               ] to perform the analyses.
                              </p>
                             </div>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Application in high-dimensional neural data
                             </h3>
                             <div id="sec-18" class="subsection">
                              <h4>
                               Allen Brain Institute datasets
                              </h4>
                              <p id="p-80">
                               We used the free, public available datasets of Allen Brain Institute through AllenSDK (For more details, see
                               <a href="http://help.brain-map.org/display/">
                                http://help.brain-map.org/display/
                               </a>
                               observatory/Documentation). Neuropixels silicon probes [
                               <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">
                                9
                               </a>
                               ] were used to record neurons with precise spatial and temporal resolution [
                               <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                                12
                               </a>
                               ].
                              </p>
                              <p id="p-81">
                               We selected the cells of 32 mice during natural scenes presentations. The cells were selected considering a signal-noise ratio (SNR) such that SNR > 0. The neural activity from a total of N = 8, 301 cells was selected from the Primary visual area (VISp), Lateral visual area (VISl), Anterolateral visual area (VISal), Posteromedial visual area (VISpm), Rostrolateral visual area (VISrl), and Anteromedial visual area (VISam).
                              </p>
                              <p id="p-82">
                               The computation time for SpikeShip to compute one dissimilarity matrix of
                               <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                Fig. 3C
                               </a>
                               was on average 58.8 secs (see
                               <a id="xref-sec-8-7" class="xref-sec" href="#sec-8">
                                Methods
                               </a>
                               ), thus indicating a highly effective computation time for N = 8, 301 neurons. In contrast, we estimate the computation time for the previous SPOTDis measure [
                               <a id="xref-ref-36-11" class="xref-bibr" href="#ref-36">
                                27
                               </a>
                               ] to be approximately 175.71 hours (based on n ≈ 3.33 spikes per neuron).
                              </p>
                             </div>
                             <div id="sec-19" class="subsection">
                              <h4>
                               Spontaneous activity
                              </h4>
                              <p id="p-83">
                               For the spontaneous activity dataset, we used free, public available datasets [
                               <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">
                                22
                               </a>
                               ]. It contains data about three mice: “Waksman” (N = 2,688), “Robbins”(N = 2, 296), and “Krebs” (N = 1, 462). Probes were located in distinct cortical areas (visual, sensorimotor, frontal, and retrosplenial), hippocampus, thalamus, striatum, and midbrain [
                               <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">
                                10
                               </a>
                               ]. Furthermore, each experiment has information about the recorded spike times, cells, and the processed SVD components of whisker’s motion (i.e. pixel difference between consecutive frames of a mouse face movie).
                              </p>
                              <p id="p-84">
                               For our analyses, we added the first four SVD components (
                               <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                                Fig. 5A
                               </a>
                               ) and then we subtracted the median across dimensions (denoising, see
                               <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                                Fig. 5B
                               </a>
                               ). A sliding window was used to sum the activity SVD whisker’s motion across the interval [t, t + 0.5] seconds (
                               <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                                Fig. 5C,D
                               </a>
                               ). We selected one hundred epochs for both low- and high-motion states based on the SVD motion value, and ran SpikeShip across different amounts of cells, depending on the experiment. The manifold learning algorithm were t-SNE and Spectral Embedding from the python library Scikit Learn v0.22.1.
                              </p>
                              <p id="p-85">
                               For these experiments, SpikeShip took 25.5 secs in creating a dissimilarity matrix. Considering N ≈ 2, 500, n ≈ 5, based on
                               <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                                Fig. 5D
                               </a>
                               , the computation time of SPOTDis can be estimated as 34.06 hours (1 day and 10.06 hours; SU ≈ 4, 808).
                              </p>
                             </div>
                             <div id="sec-20" class="subsection">
                              <h4>
                               Discriminability index
                              </h4>
                              <p id="p-86">
                               Since information in temporal spiking sequences compared to the information from traditional firing rate codes could differ, we wanted to quantify to what extent the degree of discriminability within and between different visual stimuli differ in a dissimilarity matrix diss (i.e. within and between natural images) per stimulus id.
                              </p>
                              <p id="p-87">
                               To this end, we computed a “Discriminability index” (d), defined as the difference between the average distance within and between stimulus id, divided by the squared sum of their variances to the power of two.
                               <span class="disp-formula" id="disp-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-32.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/embed/graphic-32.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-88">
                               Therefore, it indicates how many standard deviations are two sets of distances away from each other.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             4 Supplementary section
                            </h2>
                            <div class="section" id="sec-22">
                             <h3>
                              Supplementary information
                             </h3>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F6.large.jpg?width=800&height=600&carousel=1" title="Illustration of SpikeShip for multiple spikes. (Top) Example of two epochs with spike times for two active neurons N0 and N1 (i.e., Akm = 2): tk = ((10, 15), (10)) and tm = ((35, 40, 45), (35, 40)). Spike counts per epoch m and k correspond to (tN0, m = 2, tN1, m = 1) and (tN0, k = 3, tN1, k = 2), respectively. (Middle) The difference of spike times is computed by normalizing the mass across neurons and between epochs. Such spike time difference is  with mass (i.e., weights) . Then, the global shift (i.e., gmin) equals 20. (Bottom) Neuronspecific shifts correspond to . Thus, ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Illustration of SpikeShip for multiple spikes.</span> (Top) Example of two epochs with spike times for two active neurons <em>N</em><sub>0</sub> and <em>N</em><sub>1</sub> (i.e., <em>A<sub>km</sub></em> = 2): <em>t<sub>k</sub></em> = ((10, 15), (10)) and <em>t<sub>m</sub></em> = ((35, 40, 45), (35, 40)). Spike counts per epoch <em>m</em> and <em>k</em> correspond to (<em>t</em><sub><em>N</em><sub>0</sub>, <em>m</em></sub> = 2, <em>t</em><sub><em>N</em><sub>1</sub>, <em>m</em></sub> = 1) and (<em>t</em><sub><em>N</em><sub>0</sub>, <em>k</em></sub> = 3, <em>t</em><sub><em>N</em><sub>1</sub>, <em>k</em></sub> = 2), respectively. (Middle) The difference of spike times is computed by normalizing the mass across neurons and between epochs. Such spike time difference is  with mass (i.e., weights) . Then, the global shift (i.e., <em>g<sup>min</sup></em>) equals 20. (Bottom) Neuronspecific shifts correspond to . Thus, .</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F6.medium.gif" width="406" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S1" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F6.medium.gif" width="406" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S1" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S1
                               </span>
                               <span class="caption-title">
                                Illustration of SpikeShip for multiple spikes.
                               </span>
                               <p id="p-89" class="first-child">
                                (Top) Example of two epochs with spike times for two active neurons N
                                <sub>
                                 0
                                </sub>
                                and N
                                <sub>
                                 1
                                </sub>
                                (i.e., A
                                <sub>
                                 km
                                </sub>
                                = 2): t
                                <sub>
                                 k
                                </sub>
                                = ((10, 15), (10)) and t
                                <sub>
                                 m
                                </sub>
                                = ((35, 40, 45), (35, 40)). Spike counts per epoch m and k correspond to (t
                                <sub>
                                 N
                                 <sub>
                                  0
                                 </sub>
                                 , m
                                </sub>
                                = 2, t
                                <sub>
                                 N
                                 <sub>
                                  1
                                 </sub>
                                 , m
                                </sub>
                                = 1) and (t
                                <sub>
                                 N
                                 <sub>
                                  0
                                 </sub>
                                 , k
                                </sub>
                                = 3, t
                                <sub>
                                 N
                                 <sub>
                                  1
                                 </sub>
                                 , k
                                </sub>
                                = 2), respectively. (Middle) The difference of spike times is computed by normalizing the mass across neurons and between epochs. Such spike time difference is
                                <span class="inline-formula" id="inline-formula-32">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-32.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-32.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                with mass (i.e., weights)
                                <span class="inline-formula" id="inline-formula-33">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-33.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-33.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . Then, the global shift (i.e., g
                                <sup>
                                 min
                                </sup>
                                ) equals 20. (Bottom) Neuronspecific shifts correspond to
                                <span class="inline-formula" id="inline-formula-34">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-34.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-34.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                . Thus,
                                <span class="inline-formula" id="inline-formula-35">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-35.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/12/09/2020.06.03.131573/F6/embed/inline-graphic-35.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F7.large.jpg?width=800&height=600&carousel=1" title="Accuracy and Speed-up comparison for single- and multi-spike patterns. A) Example of single spike trains for two epochs for 10 neurons. Patterns were generated as uniform sequences with n =1 spike per neuron per epoch. B) Computational speedup (log-scale) for SpikeShip vs. SPOTDis (serial execution) [27] for increasing amount of neurons N. Speed-up is approximately N when there is 1 spike per neuron, and it increases when n > 1 (i.e. the multi-spike pattern case). C) Example of three single-spike patterns: (−20, 0, 0, +20), (0, 0, 0, 0), and (−15, −15, +15, +15), from left to right. SpikeShip assigns a geometrically more appropriate transport cost between pattern 1 and 2 (F1,2 = 10) than SPOTDis (D1,2 = 12.5), considering their distance with pattern 0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Accuracy and Speed-up comparison for single- and multi-spike patterns.</span> A) Example of single spike trains for two epochs for 10 neurons. Patterns were generated as uniform sequences with <em>n</em> =1 spike per neuron per epoch. B) Computational speedup (log-scale) for SpikeShip vs. SPOTDis (serial execution) [27] for increasing amount of neurons <em>N</em>. Speed-up is approximately <em>N</em> when there is 1 spike per neuron, and it increases when <em>n</em> > 1 (i.e. the multi-spike pattern case). C) Example of three single-spike patterns: (−20, 0, 0, +20), (0, 0, 0, 0), and (−15, −15, +15, +15), from left to right. SpikeShip assigns a geometrically more appropriate transport cost between pattern 1 and 2 (<em>F</em><sub>1,2</sub> = 10) than SPOTDis (<em>D</em><sub>1,2</sub> = 12.5), considering their distance with pattern 0.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F7.medium.gif" width="425" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S2" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F7.medium.gif" width="425" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S2" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S2
                               </span>
                               <span class="caption-title">
                                Accuracy and Speed-up comparison for single- and multi-spike patterns.
                               </span>
                               <p id="p-90" class="first-child">
                                A) Example of single spike trains for two epochs for 10 neurons. Patterns were generated as uniform sequences with n =1 spike per neuron per epoch. B) Computational speedup (log-scale) for SpikeShip vs. SPOTDis (serial execution) [
                                <a id="xref-ref-36-12" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ] for increasing amount of neurons N. Speed-up is approximately N when there is 1 spike per neuron, and it increases when n > 1 (i.e. the multi-spike pattern case). C) Example of three single-spike patterns: (−20, 0, 0, +20), (0, 0, 0, 0), and (−15, −15, +15, +15), from left to right. SpikeShip assigns a geometrically more appropriate transport cost between pattern 1 and 2 (F
                                <sub>
                                 1,2
                                </sub>
                                = 10) than SPOTDis (D
                                <sub>
                                 1,2
                                </sub>
                                = 12.5), considering their distance with pattern 0.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F8.large.jpg?width=800&height=600&carousel=1" title="Multimodal activation and deactivation patterns can be detected using SpikeShip. Simulations from [27]. (A) Multiple bimodal activation patterns and examples of realizations for each pattern (N = 50 neurons). Simulation parameters were λin = 0.35 spks/sample, λout = 0.05 spks/sample, Tepoch = 300 and Tpulse = 20 samples. Bottom figures show sorted dissimilarity matrix and t-SNE for simulation with patterned noise (left) and homogeneous noise (right). (B) Multiple bimodal activation patterns and examples of realizations for each pattern (N = 50 neurons). Simulation parameters were λout = 0.02 spks/sample (i.e. the deactivation period), λin =0.3 spks/sample, Tepoch = 300 and Tdeactivation = 150 samples. Bottom figures show sorted dissimilarity matrix and t-SNE for simulation with patterned noise (left) and homogeneous noise (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Multimodal activation and deactivation patterns can be detected using SpikeShip.</span> Simulations from [27]. (A) Multiple bimodal activation patterns and examples of realizations for each pattern (<em>N</em> = 50 neurons). Simulation parameters were λ<em><sub>in</sub></em> = 0.35 spks/sample, λ<em><sub>out</sub></em> = 0.05 spks/sample, <em>T<sub>epoch</sub></em> = 300 and <em>T<sub>pulse</sub></em> = 20 samples. Bottom figures show sorted dissimilarity matrix and t-SNE for simulation with patterned noise (left) and homogeneous noise (right). (B) Multiple bimodal activation patterns and examples of realizations for each pattern (<em>N</em> = 50 neurons). Simulation parameters were λ<em><sub>out</sub></em> = 0.02 spks/sample (i.e. the deactivation period), λ<sub><em>in</em></sub> =0.3 spks/sample, <em>T<sub>epoch</sub></em> = 300 and <em>T<sub>deactivation</sub></em> = 150 samples. Bottom figures show sorted dissimilarity matrix and t-SNE for simulation with patterned noise (left) and homogeneous noise (right).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F8.medium.gif" width="278" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S3" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F8.medium.gif" width="278" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S3" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S3
                               </span>
                               <span class="caption-title">
                                Multimodal activation and deactivation patterns can be detected using SpikeShip.
                               </span>
                               <p id="p-91" class="first-child">
                                Simulations from [
                                <a id="xref-ref-36-13" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ]. (A) Multiple bimodal activation patterns and examples of realizations for each pattern (N = 50 neurons). Simulation parameters were λ
                                <sub>
                                 in
                                </sub>
                                = 0.35 spks/sample, λ
                                <sub>
                                 out
                                </sub>
                                = 0.05 spks/sample, T
                                <sub>
                                 epoch
                                </sub>
                                = 300 and T
                                <sub>
                                 pulse
                                </sub>
                                = 20 samples. Bottom figures show sorted dissimilarity matrix and t-SNE for simulation with patterned noise (left) and homogeneous noise (right). (B) Multiple bimodal activation patterns and examples of realizations for each pattern (N = 50 neurons). Simulation parameters were λ
                                <sub>
                                 out
                                </sub>
                                = 0.02 spks/sample (i.e. the deactivation period), λ
                                <sub>
                                 in
                                </sub>
                                =0.3 spks/sample, T
                                <sub>
                                 epoch
                                </sub>
                                = 300 and T
                                <sub>
                                 deactivation
                                </sub>
                                = 150 samples. Bottom figures show sorted dissimilarity matrix and t-SNE for simulation with patterned noise (left) and homogeneous noise (right).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F9.large.jpg?width=800&height=600&carousel=1" title="Performance of SpikeShip depends on the SNR but it outperforms SPOTDis. (A) Performance of SPOTDis (left) [27] and SpikeShip (right) measured with ARI score. We performed the same simulations as in [27]. Firing rate inside pulse period is varied, while firing rate outside pulse was varied. We simulated 5 patterns with 30 repetitions each, with λout = 0.05 spks/sample, and λin attaining values of 0.15, 0.2, 0.25, 0.35, 0.45 or 0.5 spks/sample, Tpulse = 30 and Tepoch = 1000 samples. The number of neurons was 25, 50 or 100, and 150 epochs of homogeneous noise. We show the mean and the standard deviation across 10 repetitions of the same simulation. Performance relative to ground truth increases with SNR. Lower SNRs are needed for achieving the same performance when the number of neurons is larger. (B) as (A), but now varying the pulse duration. Simulation parameters were λout = 0.05 spks/sample, and λin = 0.5, 0.4, 0.3, 0.2, 0.1 spks/sample, and Tpulse of 100, 200, 300, 400 or 500 samples, with Tepoch = 1000 samples; note that the product of λin. Tpulse remained constant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of SpikeShip depends on the SNR but it outperforms SPOTDis.</span> (A) Performance of SPOTDis (left) [27] and SpikeShip (right) measured with ARI score. We performed the same simulations as in [27]. Firing rate inside pulse period is varied, while firing rate outside pulse was varied. We simulated 5 patterns with 30 repetitions each, with λ<em><sub>out</sub></em> = 0.05 spks/sample, and λ<em><sub>in</sub></em> attaining values of 0.15, 0.2, 0.25, 0.35, 0.45 or 0.5 spks/sample, <em>T<sub>pulse</sub></em> = 30 and <em>T<sub>epoch</sub></em> = 1000 samples. The number of neurons was 25, 50 or 100, and 150 epochs of homogeneous noise. We show the mean and the standard deviation across 10 repetitions of the same simulation. Performance relative to ground truth increases with SNR. Lower SNRs are needed for achieving the same performance when the number of neurons is larger. (B) as (A), but now varying the pulse duration. Simulation parameters were λ<em><sub>out</sub></em> = 0.05 spks/sample, and λ<em><sub>in</sub></em> = 0.5, 0.4, 0.3, 0.2, 0.1 spks/sample, and <em>T<sub>pulse</sub></em> of 100, 200, 300, 400 or 500 samples, with <em>T<sub>epoch</sub></em> = 1000 samples; note that the product of λ<em><sub>in</sub></em>. <em>T<sub>pulse</sub></em> remained constant.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F9.medium.gif" width="440" height="365"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S4" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F9.medium.gif" width="440" height="365"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S4" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S4
                               </span>
                               <span class="caption-title">
                                Performance of SpikeShip depends on the SNR but it outperforms SPOTDis.
                               </span>
                               <p id="p-92" class="first-child">
                                (A) Performance of SPOTDis (left) [
                                <a id="xref-ref-36-14" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ] and SpikeShip (right) measured with ARI score. We performed the same simulations as in [
                                <a id="xref-ref-36-15" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ]. Firing rate inside pulse period is varied, while firing rate outside pulse was varied. We simulated 5 patterns with 30 repetitions each, with λ
                                <sub>
                                 out
                                </sub>
                                = 0.05 spks/sample, and λ
                                <sub>
                                 in
                                </sub>
                                attaining values of 0.15, 0.2, 0.25, 0.35, 0.45 or 0.5 spks/sample, T
                                <sub>
                                 pulse
                                </sub>
                                = 30 and T
                                <sub>
                                 epoch
                                </sub>
                                = 1000 samples. The number of neurons was 25, 50 or 100, and 150 epochs of homogeneous noise. We show the mean and the standard deviation across 10 repetitions of the same simulation. Performance relative to ground truth increases with SNR. Lower SNRs are needed for achieving the same performance when the number of neurons is larger. (B) as (A), but now varying the pulse duration. Simulation parameters were λ
                                <sub>
                                 out
                                </sub>
                                = 0.05 spks/sample, and λ
                                <sub>
                                 in
                                </sub>
                                = 0.5, 0.4, 0.3, 0.2, 0.1 spks/sample, and T
                                <sub>
                                 pulse
                                </sub>
                                of 100, 200, 300, 400 or 500 samples, with T
                                <sub>
                                 epoch
                                </sub>
                                = 1000 samples; note that the product of λ
                                <sub>
                                 in
                                </sub>
                                . T
                                <sub>
                                 pulse
                                </sub>
                                remained constant.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F10.large.jpg?width=800&height=600&carousel=1" title="Comparison of metrics varying stimulus onset. A) SpikeShip preserves timing relationship between spike trains. Simulations of three spike trains as in [32](Fig. 5D). SpikeShip extracts the temporal information based on the timing not rate of spike trains. B) SpikeShip preserves relative timing information for multi-neuron patterns. Top: Example of three multi-neuron spiking patterns. Onset of spike patterns B and C where shifted relative to pattern A. Colors correspond to the shift/delay applied to half (blue) and the full neural population (green). Bottom: Comparison between spike train distances measuring patterns (A, B) and pattern (A, C). SpikeShip can detect linear changes in the neural pattern when the delay is applied to the half of the neural population. When no changes are applied in the temporal information but a global delay. SpikeShip is invariant to delays applied to the entire neural population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparison of metrics varying stimulus onset.</span> A) SpikeShip preserves timing relationship between spike trains. Simulations of three spike trains as in [32](Fig. 5D). SpikeShip extracts the temporal information based on the timing not rate of spike trains. B) SpikeShip preserves relative timing information for multi-neuron patterns. Top: Example of three multi-neuron spiking patterns. Onset of spike patterns <em>B</em> and <em>C</em> where shifted relative to pattern <em>A</em>. Colors correspond to the shift/delay applied to half (blue) and the full neural population (green). Bottom: Comparison between spike train distances measuring patterns (<em>A, B</em>) and pattern (<em>A, C</em>). SpikeShip can detect linear changes in the neural pattern when the delay is applied to the half of the neural population. When no changes are applied in the temporal information but a global delay. SpikeShip is invariant to delays applied to the entire neural population.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F10.medium.gif" width="335" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S5" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F10.medium.gif" width="335" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S5" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S5
                               </span>
                               <span class="caption-title">
                                Comparison of metrics varying stimulus onset.
                               </span>
                               <p id="p-93" class="first-child">
                                A) SpikeShip preserves timing relationship between spike trains. Simulations of three spike trains as in [
                                <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                                 32
                                </a>
                                ](
                                <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                                 Fig. 5D
                                </a>
                                ). SpikeShip extracts the temporal information based on the timing not rate of spike trains. B) SpikeShip preserves relative timing information for multi-neuron patterns. Top: Example of three multi-neuron spiking patterns. Onset of spike patterns B and C where shifted relative to pattern A. Colors correspond to the shift/delay applied to half (blue) and the full neural population (green). Bottom: Comparison between spike train distances measuring patterns (A, B) and pattern (A, C). SpikeShip can detect linear changes in the neural pattern when the delay is applied to the half of the neural population. When no changes are applied in the temporal information but a global delay. SpikeShip is invariant to delays applied to the entire neural population.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F11" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F11.large.jpg?width=800&height=600&carousel=1" title="Performance of VP distance is affected by changes in both local and global scaling rates. Simulations from [27]. A) Global scaling. Same simulations as in Fig.S7. Victor-Purpura distance (VP) was used with different values of q. When q = 0, VP = |ni – nj |, with ni and nj the spike count of spike sequences i and j, respectively. Epochs are clustered based on rates. B) Local scaling. Same simulations as in Fig.S8. VP distance was used with different values of q. When q → ∞, VP = ni + nj. Besides high values of q aim to extract temporal information from spike trains, these 2D embeddings demonstrate that the contribution between rate and timing using VP is difficult to interpret and very sensitive to noise." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of VP distance is affected by changes in both local and global scaling rates.</span> Simulations from [27]. A) Global scaling. Same simulations as in Fig.S7. Victor-Purpura distance (VP) was used with different values of <em>q</em>. When <em>q</em> = 0, <em>VP</em> = |<em>n<sub>i</sub></em> – <em>n<sub>j</sub></em> |, with <em>n<sub>i</sub></em> and <em>n<sub>j</sub></em> the spike count of spike sequences <em>i</em> and <em>j</em>, respectively. Epochs are clustered based on rates. B) Local scaling. Same simulations as in Fig.S8. VP distance was used with different values of <em>q</em>. When <em>q</em> → ∞, <em>VP</em> = <em>n<sub>i</sub></em> + <em>n<sub>j</sub></em>. Besides high values of <em>q</em> aim to extract temporal information from spike trains, these 2D embeddings demonstrate that the contribution between rate and timing using VP is difficult to interpret and very sensitive to noise.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F11.medium.gif" width="262" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S6" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F11.medium.gif" width="262" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S6" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S6
                               </span>
                               <span class="caption-title">
                                Performance of VP distance is affected by changes in both local and global scaling rates.
                               </span>
                               <p id="p-94" class="first-child">
                                Simulations from [
                                <a id="xref-ref-36-16" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ]. A) Global scaling. Same simulations as in
                                <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                                 Fig.S7
                                </a>
                                . Victor-Purpura distance (VP) was used with different values of q. When q = 0, VP = |n
                                <sub>
                                 i
                                </sub>
                                – n
                                <sub>
                                 j
                                </sub>
                                |, with n
                                <sub>
                                 i
                                </sub>
                                and n
                                <sub>
                                 j
                                </sub>
                                the spike count of spike sequences i and j, respectively. Epochs are clustered based on rates. B) Local scaling. Same simulations as in
                                <a id="xref-fig-13-2" class="xref-fig" href="#F13">
                                 Fig.S8
                                </a>
                                . VP distance was used with different values of q. When q → ∞, VP = n
                                <sub>
                                 i
                                </sub>
                                + n
                                <sub>
                                 j
                                </sub>
                                . Besides high values of q aim to extract temporal information from spike trains, these 2D embeddings demonstrate that the contribution between rate and timing using VP is difficult to interpret and very sensitive to noise.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F12" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F12.large.jpg?width=800&height=600&carousel=1" title="Performance SpikeShip is not affected by a global scaling rate. Simulations from [27]. Shown are two different temporal patterns. Each temporal pattern can occur in a low (λin = 0.2 and λout = 0.02 spks/sample), medium (λin 0.4 and λout = 0.04 spks/sample) or high rate (λin = 0.7 and λout 0.07 spks/sample) state, with a constant ratio of λin/λout. In addition, the noise pattern can also occur in one of three rate states. The pulse duration was 30 samples. Shown at the bottom the sorted dissimilarity matrix with SpikeShip values, the t-SNE embedding with the ground-truth cluster labels and the t-SNE embedding with the HDBSCAN cluster labels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance SpikeShip is not affected by a global scaling rate.</span> Simulations from [27]. Shown are two different temporal patterns. Each temporal pattern can occur in a low (λ<em><sub>in</sub></em> = 0.2 and λ<em><sub>out</sub></em> = 0.02 spks/sample), medium (λ<em><sub>in</sub></em> 0.4 and λ<em><sub>out</sub></em> = 0.04 spks/sample) or high rate (λ<em><sub>in</sub></em> = 0.7 and λ<em><sub>out</sub></em> 0.07 spks/sample) state, with a constant ratio of λ<em><sub>in</sub></em>/λ<em><sub>out</sub></em>. In addition, the noise pattern can also occur in one of three rate states. The pulse duration was 30 samples. Shown at the bottom the sorted dissimilarity matrix with SpikeShip values, the t-SNE embedding with the ground-truth cluster labels and the t-SNE embedding with the HDBSCAN cluster labels.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F12.medium.gif" width="440" height="356"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S7" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F12.medium.gif" width="440" height="356"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S7" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S7
                               </span>
                               <span class="caption-title">
                                Performance SpikeShip is not affected by a global scaling rate.
                               </span>
                               <p id="p-95" class="first-child">
                                Simulations from [
                                <a id="xref-ref-36-17" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ]. Shown are two different temporal patterns. Each temporal pattern can occur in a low (λ
                                <sub>
                                 in
                                </sub>
                                = 0.2 and λ
                                <sub>
                                 out
                                </sub>
                                = 0.02 spks/sample), medium (λ
                                <sub>
                                 in
                                </sub>
                                0.4 and λ
                                <sub>
                                 out
                                </sub>
                                = 0.04 spks/sample) or high rate (λ
                                <sub>
                                 in
                                </sub>
                                = 0.7 and λ
                                <sub>
                                 out
                                </sub>
                                0.07 spks/sample) state, with a constant ratio of λ
                                <sub>
                                 in
                                </sub>
                                /λ
                                <sub>
                                 out
                                </sub>
                                . In addition, the noise pattern can also occur in one of three rate states. The pulse duration was 30 samples. Shown at the bottom the sorted dissimilarity matrix with SpikeShip values, the t-SNE embedding with the ground-truth cluster labels and the t-SNE embedding with the HDBSCAN cluster labels.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F13" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F13.large.jpg?width=800&height=600&carousel=1" title="Performance of SpikeShip is not affected by a local scaling rate Simulations from [27]. Shown are two temporal patterns. Each temporal pattern could occur in one of two rates states: In the first rate state, the first 25 neurons are firing at a low rate (λin = 0.3 and λout = 0.03 spks/sample), and the other 25 are firing at a high rate (λin = 0.7 and λout = 0.07 spks/sample). In the second rate state, the rate scaling is reversed. The pulse duration was 30 samples. Shown at the bottom the sorted dissimilarity matrix with SpikeShip values, the t-SNE embedding with the ground-truth cluster labels and the t-SNE embedding with the HDBSCAN cluster labels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of SpikeShip is not affected by a local scaling rate</span> Simulations from [27]. Shown are two temporal patterns. Each temporal pattern could occur in one of two rates states: In the first rate state, the first 25 neurons are firing at a low rate (λ<em><sub>in</sub></em> = 0.3 and λ<em><sub>out</sub></em> = 0.03 spks/sample), and the other 25 are firing at a high rate (λ<em><sub>in</sub></em> = 0.7 and λ<em><sub>out</sub></em> = 0.07 spks/sample). In the second rate state, the rate scaling is reversed. The pulse duration was 30 samples. Shown at the bottom the sorted dissimilarity matrix with SpikeShip values, the t-SNE embedding with the ground-truth cluster labels and the t-SNE embedding with the HDBSCAN cluster labels.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F13.medium.gif" width="440" height="356"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S8" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F13.medium.gif" width="440" height="356"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S8" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S8
                               </span>
                               <span class="caption-title">
                                Performance of SpikeShip is not affected by a local scaling rate
                               </span>
                               <p id="p-96" class="first-child">
                                Simulations from [
                                <a id="xref-ref-36-18" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ]. Shown are two temporal patterns. Each temporal pattern could occur in one of two rates states: In the first rate state, the first 25 neurons are firing at a low rate (λ
                                <sub>
                                 in
                                </sub>
                                = 0.3 and λ
                                <sub>
                                 out
                                </sub>
                                = 0.03 spks/sample), and the other 25 are firing at a high rate (λ
                                <sub>
                                 in
                                </sub>
                                = 0.7 and λ
                                <sub>
                                 out
                                </sub>
                                = 0.07 spks/sample). In the second rate state, the rate scaling is reversed. The pulse duration was 30 samples. Shown at the bottom the sorted dissimilarity matrix with SpikeShip values, the t-SNE embedding with the ground-truth cluster labels and the t-SNE embedding with the HDBSCAN cluster labels.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F14" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F14.large.jpg?width=800&height=600&carousel=1" title="Performance of SPIKE and RI-SPIKE are affected by both global and local scaling. Simulations from [27]. A) Global scaling. SPIKE and RI-SPIKE computations for globally scaled sequences. Top: dissimilarity matrices sorted by pattern id and scaling factor. Bottom: 2D t-SNE embeddings of epochs. B) Local scaling. SPIKE and RI-SPIKE computations for locally scaled sequences. Both A and B were computed using the same simulations as in Fig. S7. Top: dissimilarity matrices sorted by pattern id and scaling factor. Bottom: 2D t-SNE embeddings of epochs. First 180 epochs correspond to noise." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Performance of SPIKE and RI-SPIKE are affected by both global and local scaling.</span> Simulations from [27]. A) Global scaling. SPIKE and RI-SPIKE computations for globally scaled sequences. Top: dissimilarity matrices sorted by pattern id and scaling factor. Bottom: 2D t-SNE embeddings of epochs. B) Local scaling. SPIKE and RI-SPIKE computations for locally scaled sequences. Both A and B were computed using the same simulations as in Fig. S7. Top: dissimilarity matrices sorted by pattern id and scaling factor. Bottom: 2D t-SNE embeddings of epochs. First 180 epochs correspond to noise.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F14.medium.gif" width="440" height="162"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S9" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F14.medium.gif" width="440" height="162"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S9" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S9
                               </span>
                               <span class="caption-title">
                                Performance of SPIKE and RI-SPIKE are affected by both global and local scaling.
                               </span>
                               <p id="p-97" class="first-child">
                                Simulations from [
                                <a id="xref-ref-36-19" class="xref-bibr" href="#ref-36">
                                 27
                                </a>
                                ]. A) Global scaling. SPIKE and RI-SPIKE computations for globally scaled sequences. Top: dissimilarity matrices sorted by pattern id and scaling factor. Bottom: 2D t-SNE embeddings of epochs. B) Local scaling. SPIKE and RI-SPIKE computations for locally scaled sequences. Both A and B were computed using the same simulations as in
                                <a id="xref-fig-12-3" class="xref-fig" href="#F12">
                                 Fig. S7
                                </a>
                                . Top: dissimilarity matrices sorted by pattern id and scaling factor. Bottom: 2D t-SNE embeddings of epochs. First 180 epochs correspond to noise.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F15" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F15.large.jpg?width=800&height=600&carousel=1" title="Comparison between clusters from SpikeShip and Firing rates embeddings of Natural scenes. A) 2D t-SNE embeddings from SpikeShip and Firing rates’ dissimilarity matrices. Allocation of natural scenes’ clusters are different between the two embeddings. B) Scaled Euclidean pairwise distance between centroids of each cluster for both SpikeShip (Left) and Firing rates (Middle), and their difference (Right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Comparison between clusters from SpikeShip and Firing rates embeddings of Natural scenes.</span> A) 2D t-SNE embeddings from SpikeShip and Firing rates’ dissimilarity matrices. Allocation of natural scenes’ clusters are different between the two embeddings. B) Scaled Euclidean pairwise distance between centroids of each cluster for both SpikeShip (Left) and Firing rates (Middle), and their difference (Right).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F15.medium.gif" width="440" height="357"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S10" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F15.medium.gif" width="440" height="357"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S10" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S10
                               </span>
                               <span class="caption-title">
                                Comparison between clusters from SpikeShip and Firing rates embeddings of Natural scenes.
                               </span>
                               <p id="p-98" class="first-child">
                                A) 2D t-SNE embeddings from SpikeShip and Firing rates’ dissimilarity matrices. Allocation of natural scenes’ clusters are different between the two embeddings. B) Scaled Euclidean pairwise distance between centroids of each cluster for both SpikeShip (Left) and Firing rates (Middle), and their difference (Right).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F16" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F16.large.jpg?width=800&height=600&carousel=1" title="Analysis of large scale neural recordings during visual stimuli presentations with SPIKE and RI-SPIKE. Top: Dissimilarity matrices sorted by Natural Scene ID. Middle: 2D t-SNE embeddings from dissimilarity matrices colored by Natural Scene ID. Bottom: Clustering performance through ARI score and Spearman correlation between dissimilarity matrices computed via SPIKE, RI-SPIKE, Firing rates, and SpikeShip. Clustering performance of SPIKE and RI-SPIKE is lower than clustering performance by using the traditional firing rates and SpikeShip. SPIKE and RI-SPIKE are highly correlated, while Firing Rates and spikeship are highly uncorrelated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Analysis of large scale neural recordings during visual stimuli presentations with SPIKE and RI-SPIKE.</span> Top: Dissimilarity matrices sorted by Natural Scene ID. Middle: 2D t-SNE embeddings from dissimilarity matrices colored by Natural Scene ID. Bottom: Clustering performance through ARI score and Spearman correlation between dissimilarity matrices computed via SPIKE, RI-SPIKE, Firing rates, and SpikeShip. Clustering performance of SPIKE and RI-SPIKE is lower than clustering performance by using the traditional firing rates and SpikeShip. SPIKE and RI-SPIKE are highly correlated, while Firing Rates and spikeship are highly uncorrelated.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F16.medium.gif" width="324" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S11" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F16.medium.gif" width="324" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S11" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S11
                               </span>
                               <span class="caption-title">
                                Analysis of large scale neural recordings during visual stimuli presentations with SPIKE and RI-SPIKE.
                               </span>
                               <p id="p-99" class="first-child">
                                Top: Dissimilarity matrices sorted by Natural Scene ID. Middle: 2D t-SNE embeddings from dissimilarity matrices colored by Natural Scene ID. Bottom: Clustering performance through ARI score and Spearman correlation between dissimilarity matrices computed via SPIKE, RI-SPIKE, Firing rates, and SpikeShip. Clustering performance of SPIKE and RI-SPIKE is lower than clustering performance by using the traditional firing rates and SpikeShip. SPIKE and RI-SPIKE are highly correlated, while Firing Rates and spikeship are highly uncorrelated.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F17" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F17.large.jpg?width=800&height=600&carousel=1" title="Spontaneous activity analyses for 3 mice. Multi-spike sequence analyses for three mice (rows). Left: dissimilarity matrices. Middle: 2D t-SNE embedding. Right: 2D Spectral Embedding (Laplacian Eigenmaps)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1107559370" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Spontaneous activity analyses for 3 mice.</span> Multi-spike sequence analyses for three mice (rows). Left: dissimilarity matrices. Middle: 2D t-SNE embedding. Right: 2D Spectral Embedding (Laplacian Eigenmaps).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. S12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F17.medium.gif" width="440" height="434"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. S12" src="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F17.medium.gif" width="440" height="434"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S12" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/12/09/2020.06.03.131573/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. S12
                               </span>
                               <span class="caption-title">
                                Spontaneous activity analyses for 3 mice.
                               </span>
                               <p id="p-100" class="first-child">
                                Multi-spike sequence analyses for three mice (rows). Left: dissimilarity matrices. Middle: 2D t-SNE embedding. Right: 2D Spectral Embedding (Laplacian Eigenmaps).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Code availability
                            </h2>
                            <p id="p-101">
                             The code of SpikeShip from the github repository:
                             <a href="https://github.com/bsotomayorg/SpikeShip">
                              https://github.com/bsotomayorg/SpikeShip
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Declarations
                            </h2>
                            <p id="p-103">
                             Conception and problem statement: BSG, FPB, MV. Data analysis and simulations: BSG. Mathematical analysis: BSG and MV. Supervision: MV. Writing of main draft: BSG and MV.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-102">
                             This project was financed by the BMF (Bundesministerium fuer Bildung und Forschung), Computational Life Sciences, project BINDA (031L0167).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               Added many supplementary figures comparing other spike train measures
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.1" data-doi="10.1523/JNEUROSCI.2817-10.2011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Havenith
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Biederlack
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nikolic
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Synchrony makes neurons fire in sequence, and stimulus properties determine who is ahead
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Neurosci
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  23
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  8570
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  84
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DHavenith%26rft.auinit1%253DM.%2BN.%26rft.volume%253D31%26rft.issue%253D23%26rft.spage%253D8570%26rft.epage%253D8584%26rft.atitle%253DSynchrony%2BMakes%2BNeurons%2BFire%2Bin%2BSequence%252C%2Band%2BStimulus%2BProperties%2BDetermine%2BWho%2BIs%2BAhead%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.2817-10.2011%26rft_id%253Dinfo%253Apmid%252F21653861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8yMy84NTcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMTIvMDkvMjAyMC4wNi4wMy4xMzE1NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.2" data-doi="10.1523/JNEUROSCI.2817-10.2011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Havenith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Martha N Yu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Shan
                                  </span>
                                  <span class="cit-name-surname">
                                   Biederlack
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Julia
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nan-Hui
                                  </span>
                                  <span class="cit-name-surname">
                                   Singer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wolf
                                  </span>
                                  <span class="cit-name-surname">
                                   Nikolic
                                  </span>
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Danko Research Support, Non-U.S. Gov’t United States The Journal of neuroscience : the official journal of the Society for Neuroscience J Neurosci
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 <span class="cit-month">
                                  Jun
                                 </span>
                                 <span class="cit-day">
                                  8
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  23
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  8570
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  84
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1523/JNEUROSCI.2817-10.2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DHavenith%26rft.auinit1%253DM.%2BN.%26rft.volume%253D31%26rft.issue%253D23%26rft.spage%253D8570%26rft.epage%253D8584%26rft.atitle%253DSynchrony%2BMakes%2BNeurons%2BFire%2Bin%2BSequence%252C%2Band%2BStimulus%2BProperties%2BDetermine%2BWho%2BIs%2BAhead%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.2817-10.2011%26rft_id%253Dinfo%253Apmid%252F21653861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8yMy84NTcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMTIvMDkvMjAyMC4wNi4wMy4xMzE1NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.3" data-doi="10.1523/JNEUROSCI.1623-09.2010">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vinck
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lima
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Womelsdorf
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Oostenveld
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Neuenschwander
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fries
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Gamma-phase shifting in awake monkey visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1250
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1257
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DVinck%26rft.auinit1%253DM.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D1250%26rft.epage%253D1257%26rft.atitle%253DGamma-Phase%2BShifting%2Bin%2BAwake%2BMonkey%2BVisual%2BCortex%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.1623-09.2010%26rft_id%253Dinfo%253Apmid%252F20107053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjk6IjMwLzQvMTI1MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzEyLzA5LzIwMjAuMDYuMDMuMTMxNTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.4" data-doi="10.1016/j.neuron.2009.03.014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Luczak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bartho
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Spontaneous events outline the realm of possible sensory responses in neocortical populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  62
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  413
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D62%26rft.spage%253D413%26rft.atitle%253DSPONTANEOUS%2BEVENTS%2BOUTLINE%2BTHE%2BREALM%2BOF%2BPOSSIBLE%2BSENSORY%2BRESPONSES%2BIN%2BNEOCORTICAL%2BPOPULATIONS%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.03.014%26rft_id%253Dinfo%253Apmid%252F19447096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.03.014&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19447096&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000266146100012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.5" data-doi="10.1016/j.neuron.2009.03.014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Luczak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Artur Bartho
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kenneth D
                                  </span>
                                  <span class="cit-name-surname">
                                   Neuron
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 <span class="cit-month">
                                  May
                                 </span>
                                 <span class="cit-day">
                                  14
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  62
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  413
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.neuron.2009.03.014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNeuron%26rft.aulast%253DLuczak%26rft.auinit1%253DA.%26rft.volume%253D62%26rft.issue%253D3%26rft.spage%253D413%26rft.epage%253D425%26rft.atitle%253DSpontaneous%2Bevents%2Boutline%2Bthe%2Brealm%2Bof%2Bpossible%2Bsensory%2Bresponses%2Bin%2Bneocortical%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.03.014%26rft_id%253Dinfo%253Apmid%252F19447096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.03.014&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19447096&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000266146100012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.6" data-doi="10.1038/nn737">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Liang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Temporal and rate representations of timevarying signals in the auditory cortex of awake primates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Neurosci
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1131
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DLu%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D1131%26rft.epage%253D1138%26rft.atitle%253DTemporal%2Band%2Brate%2Brepresentations%2Bof%2Btime-varying%2Bsignals%2Bin%2Bthe%2Bauditory%2Bcortex%2Bof%2Bawake%2Bprimates.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn737%26rft_id%253Dinfo%253Apmid%252F11593234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn737&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11593234&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000171970900017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.7" data-doi="10.1038/nn737">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T Liang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  X Nat Neurosci
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 <span class="cit-month">
                                  Nov
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1131
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nn737
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DLu%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D1131%26rft.epage%253D1138%26rft.atitle%253DTemporal%2Band%2Brate%2Brepresentations%2Bof%2Btime-varying%2Bsignals%2Bin%2Bthe%2Bauditory%2Bcortex%2Bof%2Bawake%2Bprimates.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn737%26rft_id%253Dinfo%253Apmid%252F11593234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn737&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11593234&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000171970900017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.8" data-doi="10.1126/science.1159775">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pastalkova
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Itskov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Amarasingham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Buzsaki
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Internally generated cell assembly sequences in the rat hippocampus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  321
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5894
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1322
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPastalkova%26rft.auinit1%253DE.%26rft.volume%253D321%26rft.issue%253D5894%26rft.spage%253D1322%26rft.epage%253D1327%26rft.atitle%253DInternally%2BGenerated%2BCell%2BAssembly%2BSequences%2Bin%2Bthe%2BRat%2BHippocampus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1159775%26rft_id%253Dinfo%253Apmid%252F18772431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjEvNTg5NC8xMzIyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMTIvMDkvMjAyMC4wNi4wMy4xMzE1NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.9" data-doi="10.1126/science.1159775">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pastalkova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Eva Itskov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vladimir
                                  </span>
                                  <span class="cit-name-surname">
                                   Amarasingham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Asohan
                                  </span>
                                  <span class="cit-name-surname">
                                   Buzsaki
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Gyorgy Research Support, N.I.H., Extramural Research Support, Non-U.S. Gov’t Research Support, U.S. Gov’t, Non-P.H.S
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  United States Science (New York, N.Y.) Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 <span class="cit-month">
                                  Sep
                                 </span>
                                 <span class="cit-day">
                                  5
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  321
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5894
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1322
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1126/science.1159775
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPastalkova%26rft.auinit1%253DE.%26rft.volume%253D321%26rft.issue%253D5894%26rft.spage%253D1322%26rft.epage%253D1327%26rft.atitle%253DInternally%2BGenerated%2BCell%2BAssembly%2BSequences%2Bin%2Bthe%2BRat%2BHippocampus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1159775%26rft_id%253Dinfo%253Apmid%252F18772431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjEvNTg5NC8xMzIyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMTIvMDkvMjAyMC4wNi4wMy4xMzE1NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.10" data-doi="10.1126/science.1093173">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ikegaya
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Aaron
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cossart
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Aronov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lampl
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferster
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yuste
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Synfire chains and cortical songs: temporal modules of cortical activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  304
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5670
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  559
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  564
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DIkegaya%26rft.auinit1%253DY.%26rft.volume%253D304%26rft.issue%253D5670%26rft.spage%253D559%26rft.epage%253D564%26rft.atitle%253DSynfire%2BChains%2Band%2BCortical%2BSongs%253A%2BTemporal%2BModules%2Bof%2BCortical%2BActivity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1093173%26rft_id%253Dinfo%253Apmid%252F15105494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MC81NTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOS8yMDIwLjA2LjAzLjEzMTU3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [7] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.11" data-doi="10.1038/384162a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wehr
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Laurent
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Odour encoding by temporal sequences of firing in oscillating neural assemblies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  384
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6605
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  162
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWehr%26rft.auinit1%253DM.%26rft.volume%253D384%26rft.issue%253D6605%26rft.spage%253D162%26rft.epage%253D166%26rft.atitle%253DOdour%2Bencoding%2Bby%2Btemporal%2Bsequences%2Bof%2Bfiring%2Bin%2Boscillating%2Bneural%2Bassemblies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F384162a0%26rft_id%253Dinfo%253Apmid%252F8906790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/384162a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8906790&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.12" data-doi="10.1038/384162a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wehr
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M Laurent
                                  </span>
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  G England Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 <span class="cit-month">
                                  Nov
                                 </span>
                                 <span class="cit-day">
                                  14
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  384
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6605
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  162
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/384162a0
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWehr%26rft.auinit1%253DM.%26rft.volume%253D384%26rft.issue%253D6605%26rft.spage%253D162%26rft.epage%253D166%26rft.atitle%253DOdour%2Bencoding%2Bby%2Btemporal%2Bsequences%2Bof%2Bfiring%2Bin%2Boscillating%2Bneural%2Bassemblies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F384162a0%26rft_id%253Dinfo%253Apmid%252F8906790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/384162a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8906790&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [8] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.13" data-doi="10.1126/science.290.5500.2268">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Seung
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  The manifold ways of perception
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  science
                                 </abbr>
                                 <span class="cit-vol">
                                  290
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5500
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2268
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2269
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSeung%26rft.auinit1%253DH.%2BS.%26rft.volume%253D290%26rft.issue%253D5500%26rft.spage%253D2268%26rft.epage%253D2269%26rft.atitle%253DCOGNITION%253A%2BThe%2BManifold%2BWays%2Bof%2BPerception%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5500.2268%26rft_id%253Dinfo%253Apmid%252F11188725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTUwMC8yMjY4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMTIvMDkvMjAyMC4wNi4wMy4xMzE1NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [9] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.14" data-doi="10.1038/nature24636">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jun
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Siegle
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Denman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bauza
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Barbarits
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Anastassiou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Andrei
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ç.
                                  </span>
                                  <span class="cit-name-surname">
                                   Aydin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Fully integrated silicon probes for high-density recording of neural activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  551
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7679
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  232
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  236
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24636%26rft_id%253Dinfo%253Apmid%252F29120427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature24636&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29120427&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [10] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Stringer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pachitariu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Carandini
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Spontaneous behaviors drive multidimensional, brainwide activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  364
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6437
                                 </span>
                                 ) (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Stringer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Michaelos
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tsyboulski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lindo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pachitariu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  High-precision coding in visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  184
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2767
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2778
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D184%26rft.spage%253D2767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [12] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.17" data-doi="10.1038/s41586-020-03171-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Siegle
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Durand
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gale
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bennett
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Graddis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Heller
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ramirez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Choi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Luviano
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Survey of spiking in the mouse visual system reveals functional hierarchy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  592
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7852
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  86
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  92
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D592%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-03171-x%26rft_id%253Dinfo%253Apmid%252F33473216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-020-03171-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33473216&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [13] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.18" data-doi="10.1016/j.cell.2017.05.011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tsao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  The code for facial identity in the primate brain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  169
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1013
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1028
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D169%26rft.spage%253D1013%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.05.011%26rft_id%253Dinfo%253Apmid%252F28575666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.05.011&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28575666&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [14] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.19" data-doi="10.1038/nature11129">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Churchland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cunningham
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kaufman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Foster
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nuyujukian
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ryu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shenoy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Neural population dynamics during reaching
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  487
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7405
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  51
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  56
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChurchland%26rft.auinit1%253DM.%2BM.%26rft.volume%253D487%26rft.issue%253D7405%26rft.spage%253D51%26rft.epage%253D56%26rft.atitle%253DNeural%2Bpopulation%2Bdynamics%2Bduring%2Breaching.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11129%26rft_id%253Dinfo%253Apmid%252F22722855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature11129&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22722855&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000305982900048&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [15] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.20" data-doi="10.1016/j.neuron.2006.02.023">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dragoi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Buzsaki
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Temporal encoding of place sequences by hippocampal cell assemblies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  145
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  57
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D50%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2006.02.023%26rft_id%253Dinfo%253Apmid%252F16600862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2006.02.023&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16600862&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000236811300017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [16] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.21" data-doi="10.1126/science.7770778">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Z.F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mainen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sejnowski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Reliability of spike timing in neocortical neurons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  268
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5216
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1503
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1506
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMainen%26rft.auinit1%253DZ.%26rft.volume%253D268%26rft.issue%253D5216%26rft.spage%253D1503%26rft.epage%253D1506%26rft.atitle%253DReliability%2Bof%2Bspike%2Btiming%2Bin%2Bneocortical%2Bneurons%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7770778%26rft_id%253Dinfo%253Apmid%252F7770778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjgvNTIxNi8xNTAzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMTIvMDkvMjAyMC4wNi4wMy4xMzE1NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [17] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.22" data-doi="10.1126/science.275.5297.213">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Markram
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lubke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Frotscher
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sakmann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Regulation of synaptic efficacy by coincidence of postsynaptic aps and epsps
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  275
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5297
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  213
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMarkram%26rft.auinit1%253DH.%26rft.volume%253D275%26rft.issue%253D5297%26rft.spage%253D213%26rft.epage%253D215%26rft.atitle%253DRegulation%2Bof%2BSynaptic%2BEfficacy%2Bby%2BCoincidence%2Bof%2BPostsynaptic%2BAPs%2Band%2BEPSPs%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.275.5297.213%26rft_id%253Dinfo%253Apmid%252F8985014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvNTI5Ny8yMTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMi8wOS8yMDIwLjA2LjAzLjEzMTU3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Markram
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H Lubke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                  <span class="cit-name-surname">
                                   Frotscher
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                  <span class="cit-name-surname">
                                   Sakmann
                                  </span>
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  B New York, N.Y. Science.
                                 </abbr>
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 <span class="cit-month">
                                  Jan
                                 </span>
                                 <span class="cit-day">
                                  10
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  275
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5297
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  213
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DB%2BNew%2BYork%252C%2BN.Y.%2BScience.%26rft.volume%253D275%26rft.spage%253D213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.24" data-doi="10.1016/j.neuron.2004.09.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Poo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Spike timing-dependent plasticity of neural circuits
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  23
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D44%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2004.09.007%26rft_id%253Dinfo%253Apmid%252F15450157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2004.09.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15450157&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000224232600004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.25" data-doi="10.1016/j.neuron.2004.09.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Yang Poo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mu-Ming
                                  </span>
                                  <span class="cit-name-surname">
                                   Neuron
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 <span class="cit-month">
                                  Sep
                                 </span>
                                 <span class="cit-day">
                                  30
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  23
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.neuron.2004.09.007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DDan%26rft.auinit1%253DY.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D30%26rft.atitle%253DSpike%2Btiming-dependent%2Bplasticity%2Bof%2Bneural%2Bcircuits.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2004.09.007%26rft_id%253Dinfo%253Apmid%252F15450157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2004.09.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15450157&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000224232600004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.26" data-doi="10.1093/cercor/6.3.406">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Abbott
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Blum
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Functional significance of long-term potentiation for sequence learning and prediction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cereb Cortex
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  406
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  16
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCerebral%2BCortex%26rft.stitle%253DCereb%2BCortex%26rft.aulast%253DAbbott%26rft.auinit1%253DL.%2BF%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D406%26rft.epage%253D416%26rft.atitle%253DFunctional%2BSignificance%2Bof%2BLong-Term%2BPotentiation%2Bfor%2BSequence%2BLearning%2Band%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcercor%252F6.3.406%26rft_id%253Dinfo%253Apmid%252F8670667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/cercor/6.3.406&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8670667&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996UL40900008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abbott
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L F Blum
                                  </span>
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  K I New York, N.Y. : 1991 Cereb Cortex
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 <span class="cit-month">
                                  May-Jun
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  406
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  16
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DK%2BI%2BNew%2BYork%252C%2BN.Y.%2B%253A%2B1991%2BCereb%2BCortex%26rft.volume%253D6%26rft.spage%253D406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.28" data-doi="10.1038/nrn2558">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Buonomano
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Maass
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  State-dependent computations: spatiotemporal processing in cortical networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Rev Neurosci
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  113
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BNeuroscience%26rft.stitle%253DNat%2BRev%2BNeurosci%26rft.aulast%253DBuonomano%26rft.auinit1%253DD.%2BV.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D113%26rft.epage%253D125%26rft.atitle%253DState-dependent%2Bcomputations%253A%2Bspatiotemporal%2Bprocessing%2Bin%2Bcortical%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrn2558%26rft_id%253Dinfo%253Apmid%252F19145235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrn2558&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19145235&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000263091100014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.29" data-doi="10.1038/nrn2558">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Buonomano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Dean V Maass
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Wolfgang
                                  </span>
                                  <span class="cit-name-surname">
                                   England
                                  </span>
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Rev Neurosci
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 <span class="cit-month">
                                  Feb
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  113
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nrn2558
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  .
                                 </span>
                                 Epub 2009 Jan 15.
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BNeuroscience%26rft.stitle%253DNat%2BRev%2BNeurosci%26rft.aulast%253DBuonomano%26rft.auinit1%253DD.%2BV.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D113%26rft.epage%253D125%26rft.atitle%253DState-dependent%2Bcomputations%253A%2Bspatiotemporal%2Bprocessing%2Bin%2Bcortical%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrn2558%26rft_id%253Dinfo%253Apmid%252F19145235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrn2558&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19145235&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000263091100014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [21] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lazar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Does the cerebral cortex exploit high-dimensional, non-linear dynamics for information processing?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in computational neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  99
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bcomputational%2Bneuroscience%26rft.volume%253D10%26rft.spage%253D99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [22] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.06.03.131573v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pachitariu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Stringer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Carandini
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Eight-probe neuropixels recordings during spontaneous behaviors
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <a href="https://doi.org/10.25378/janelia.7739750.v4">
                                  https://doi.org/10.25378/janelia.7739750.v4
                                 </a>
                                 . URL
                                 <a href="https://janelia.figshare.com/articles/dataset/Eight-probe_Neuropixels_recordings_during_spontaneous_behaviors/7739750/4">
                                  https://janelia.figshare.com/articles/dataset/Eight-probe_Neuropixels_recordings_during_spontaneous_behaviors/7739750/4
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [23] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.32" data-doi="10.1152/jn.1996.76.2.1310">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Victor
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Purpura
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Nature and precision of temporal coding in visual cortex: a metric-space analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 <span class="cit-vol">
                                  76
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1310
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1326
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeurophysiology%26rft.stitle%253DJ.%2BNeurophysiol.%26rft.aulast%253DVictor%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D1310%26rft.epage%253D1326%26rft.atitle%253DNature%2Band%2Bprecision%2Bof%2Btemporal%2Bcoding%2Bin%2Bvisual%2Bcortex%253A%2Ba%2Bmetric-space%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.1996.76.2.1310%26rft_id%253Dinfo%253Apmid%252F8871238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.1996.76.2.1310&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8871238&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996VD97200050&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [24] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Victor
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Purpura
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Metric-space analysis of spike trains: theory, algorithms and application
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Network: computation in neural systems
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  127
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  164
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNetwork%253A%2Bcomputation%2Bin%2Bneural%2Bsystems%26rft.volume%253D8%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [25] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.34" data-doi="10.1152/jn.00873.2012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kreuz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chicharro
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Houghton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Andrzejak
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mormann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Monitoring spike train synchrony
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1457
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1472
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurophysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.00873.2012%26rft_id%253Dinfo%253Apmid%252F23221419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.00873.2012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23221419&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000315681100022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [26] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.35" data-doi="10.1016/j.jneumeth.2017.05.028">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Satuvuori
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mulansky
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bozanic
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Malvestio
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zeldenrust
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lenk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kreuz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Measures of spike train synchrony for data with multiple time scales
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neuroscience methods
                                 </abbr>
                                 <span class="cit-vol">
                                  287
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  25
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  38
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneuroscience%2Bmethods%26rft.volume%253D287%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jneumeth.2017.05.028%26rft_id%253Dinfo%253Apmid%252F28583477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jneumeth.2017.05.028&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28583477&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [27] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Grossberger
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Battaglia
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vinck
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Unsupervised clustering of temporal patterns in high-dimensional neuronal ensembles using a novel dissimilarity measure
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e1006
                                 </span>
                                 ,
                                 <span class="cit-page-range">
                                  283
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D14%26rft.spage%253D1006e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [28] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Van der Maaten
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Visualizing data using t-sne
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of machine learning research
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ) (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Der Maaten
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Accelerating t-sne using tree-based algorithms
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Machine Learning Research
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  3221
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3245
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BMachine%2BLearning%2BResearch%26rft.volume%253D15%26rft.spage%253D3221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [30] in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hinton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Stochastic neighbor embedding
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in neural information processing systems
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  857
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  864
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bneural%2Binformation%2Bprocessing%2Bsystems%26rft.volume%253D15%26rft.spage%253D857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [31] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                  <span class="cit-name-surname">
                                   McInnes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Healy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Astels
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  hdbscan: Hierarchical density based clustering
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Open Source Software
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ) (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [32] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.41" data-doi="10.1016/J.JNEUMETH.2018.02.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Satuvuori
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kreuz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Which spike train distance is most suitable for distinguishing rate and temporal coding?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neuroscience methods
                                 </abbr>
                                 <span class="cit-vol">
                                  299
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  22
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  33
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneuroscience%2Bmethods%26rft.volume%253D299%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1016%252FJ.JNEUMETH.2018.02.009%26rft_id%253Dinfo%253Apmid%252F29462713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/J.JNEUMETH.2018.02.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29462713&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [33] in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.42" data-doi="10.1016/S0893-6080(01)00083-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Thorpe
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Delorme
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Rullen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Spike-based strategies for rapid processing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural networks
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6–7
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  715
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  725
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2Bnetworks%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BNeural%2BNetwork%2BSociety%26rft.stitle%253DNeural%2BNetw%26rft.aulast%253DThorpe%26rft.auinit1%253DS.%26rft.volume%253D14%26rft.issue%253D6-7%26rft.spage%253D715%26rft.epage%253D725%26rft.atitle%253DSpike-based%2Bstrategies%2Bfor%2Brapid%2Bprocessing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0893-6080%252801%252900083-1%26rft_id%253Dinfo%253Apmid%252F11665765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0893-6080(01)00083-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11665765&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000171417300012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [34] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.43" data-doi="10.1016/j.neuron.2015.09.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                  <span class="cit-name-surname">
                                   McGinley
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vinck
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Reimer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Batista-Brito
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zagha
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cadwell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tolias
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cardin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                  <span class="cit-name-surname">
                                   McCormick
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Waking state: Rapid variations modulate neural and behavioral responses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  87
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1143
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  61
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D87%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2015.09.012%26rft_id%253Dinfo%253Apmid%252F26402600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2015.09.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26402600&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.06.03.131573v3.44" data-doi="10.1016/j.neuron.2015.09.012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McGinley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Matthew J Vinck
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Reimer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jacob
                                  </span>
                                  <span class="cit-name-surname">
                                   Batista-Brito
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Renata
                                  </span>
                                  <span class="cit-name-surname">
                                   Zagha
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edward
                                  </span>
                                  <span class="cit-name-surname">
                                   Cadwell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Cathryn R
                                  </span>
                                  <span class="cit-name-surname">
                                   Tolias
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andreas S
                                  </span>
                                  <span class="cit-name-surname">
                                   Cardin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jessica A
                                  </span>
                                  <span class="cit-name-surname">
                                   McCormick
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David A
                                  </span>
                                  <span class="cit-name-surname">
                                   Neuron
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 <span class="cit-month">
                                  Sep
                                 </span>
                                 <span class="cit-day">
                                  23
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  87
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1143
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  61
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.neuron.2015.09.012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.volume%253D87%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2015.09.012%26rft_id%253Dinfo%253Apmid%252F26402600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2015.09.012&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26402600&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F12%2F09%2F2020.06.03.131573.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [35] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Poole
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Maheswaranathan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dhawale
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fisher
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Trautmann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ryu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shusterman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Discovering precise temporal patterns in large-scale neural recordings through robust and interpretable time warping
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  246
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  259
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D105%26rft.spage%253D246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [36] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.06.03.131573v3.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cormen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Leiserson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rivest
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Stein
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Introduction to algorithms
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  MIT press
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [37] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.06.03.131573v3.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bleich
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Overton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  A linear-time algorithm for the weighted median problem
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Courant Institute of Mathematical Sciences, New York University
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1983
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [38] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.06.03.131573v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bovo
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Robustats
                                 </span>
                                 .
                                 <a href="https://github.com/FilippoBovo/robustats">
                                  https://github.com/FilippoBovo/robustats
                                 </a>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [39] in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Denker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yegenoglu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Grün
                                  </span>
                                 </span>
                                 , in
                                 <abbr class="cit-jnl-abbrev">
                                  Neuroinformatics 2018
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  P19
                                 </span>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [40] in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.03.131573v3.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mulansky
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kreuz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Pyspike—a python library for analyzing spike train synchrony
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  SoftwareX
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  183
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  189
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSoftwareX%26rft.volume%253D5%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.06.03.131573v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.10.21.465244v3" title="Climate of origin influences how a herbivorous mite responds to drought-stressed host plants" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.07.19.500653v3" title="Spherical harmonics representation of the steady-state membrane potential shift induced by tDCS in realistic neuron models" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted December 09, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.06.03.131573v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.06.03.131573v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.06.03.131573v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2889064"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-75Nyo0jB6HynJWnCyLGwIhGjzGhux1oecNWpO4ZDRGg"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484472054"/>
                                   <input type="hidden" name="captcha_token" value="4a4b6be4e9b873f6941754fbe49755d8"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2889064--3" data-node-nid="2889064" id="node-2889064--4838218599" data-pisa="biorxiv;2020.06.03.131573v3" data-pisa-master="biorxiv;2020.06.03.131573" data-seqnum="2889064" data-apath="/biorxiv/early/2022/12/09/2020.06.03.131573.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Boris
                                  </span>
                                  <span class="nlm-surname">
                                   Sotomayor-Gómez
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Francesco P.
                                  </span>
                                  <span class="nlm-surname">
                                   Battaglia
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Martin
                                  </span>
                                  <span class="nlm-surname">
                                   Vinck
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.06.03.131573;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.06.03.131573
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.06.03.131573v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&title=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&title=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&text=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&t=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&title=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&title=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&title=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2889064--5" data-node-nid="2889064" id="citation-node-2889064--6418248622" data-pisa="biorxiv;2020.06.03.131573v3" data-pisa-master="biorxiv;2020.06.03.131573" data-seqnum="2889064" data-apath="/biorxiv/early/2022/12/09/2020.06.03.131573.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     SpikeShip: A method for fast, unsupervised discovery of high-dimensional neural spiking patterns
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Boris
                                      </span>
                                      <span class="nlm-surname">
                                       Sotomayor-Gómez
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Francesco P.
                                      </span>
                                      <span class="nlm-surname">
                                       Battaglia
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Martin
                                      </span>
                                      <span class="nlm-surname">
                                       Vinck
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.06.03.131573;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.06.03.131573
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Sotomayor-G%C3%B3mez%2C+Boris&rft.au=Battaglia%2C+Francesco+P.&rft.au=Vinck%2C+Martin">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2889064/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2889064/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2889064/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2889064/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2889064/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2889064/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2889064/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2889064/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2889064/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2889064/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2889064/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2889064/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&count=horizontal&via=&text=SpikeShip%3A%20A%20method%20for%20fast%2C%20unsupervised%20discovery%20of%20high-dimensional%20neural%20spiking%20patterns&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.06.03.131573v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.06.03.131573v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
