<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.15.950840v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.15.950840v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.15.950840v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/05/03/2020.02.15.950840.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.15.950840v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.15.950840"/>
  <meta name="DC.Date" content="2022-05-03"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="We present a Python-based software package to automatically obtain phenomenological models of input-controlled synthetic biological circuits from descriptive models. From the parts and mechanism description of a synthetic biological circuit, it is easy to obtain a chemical reaction model of the circuit under the assumptions of mass-action kinetics using various existing tools. However, using these models to guide design decisions during an experiment is difficult due to a large number of reaction rate parameters and species in the model. Hence, phenomenological models are often developed that describe the effective relationships among the circuit inputs, outputs, and only the key states and parameters. In this paper, we present an algorithm to obtain these phenomenological models in an automated manner using a Python package for circuits with inputs that control the desired outputs. This model reduction approach combines the common assumptions of time-scale separation, conservation laws, and species’ abundance to obtain the reduced models that can be used for design of synthetic biological circuits. We consider an example of a simple gene expression circuit and another example of a layered genetic feedback control circuit to demonstrate the use of the model reduction procedure.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Ayush Pandey"/>
  <meta name="DC.Contributor" content="Richard M. Murray"/>
  <meta name="article:published_time" content="2022-05-03"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>We present a Python-based software package to automatically obtain phenomenological models of input-controlled synthetic biological circuits from descriptive models. From the parts and mechanism description of a synthetic biological circuit, it is easy to obtain a chemical reaction model of the circuit under the assumptions of mass-action kinetics using various existing tools. However, using these models to guide design decisions during an experiment is difficult due to a large number of reaction rate parameters and species in the model. Hence, phenomenological models are often developed that describe the effective relationships among the circuit inputs, outputs, and only the key states and parameters. In this paper, we present an algorithm to obtain these phenomenological models in an automated manner using a Python package for circuits with inputs that control the desired outputs. This model reduction approach combines the common assumptions of time-scale separation, conservation laws, and species’ abundance to obtain the reduced models that can be used for design of synthetic biological circuits. We consider an example of a simple gene expression circuit and another example of a layered genetic feedback control circuit to demonstrate the use of the model reduction procedure.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.15.950840v3"/>
  <meta name="citation_id" content="2020.02.15.950840v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.950840v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.950840v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.950840v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.15.950840"/>
  <meta name="citation_num_pages" content="12"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.15.950840"/>
  <meta name="citation_author" content="Ayush Pandey"/>
  <meta name="citation_author_institution" content="Department of Control and Dynamical Systems at California Institute of Technology"/>
  <meta name="citation_author_email" content="apandey@caltech.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3590-4459"/>
  <meta name="citation_author" content="Richard M. Murray"/>
  <meta name="citation_author_institution" content="Department of Control and Dynamical Systems at California Institute of Technology"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5785-7481"/>
  <meta name="citation_reference" content="A. Gorban, “Model reduction in chemical dynamics: slow invariant manifolds, singular perturbations, thermodynamic estimates, and analysis of reaction graph,” Current Opinion in Chemical Engineering, vol. 21, pp. 48–59, 2018."/>
  <meta name="citation_reference" content="C. W. Gear, T. J. Kaper, I. G. Kevrekidis, and A. Zagaris, “Projecting to a slow manifold: Singularly perturbed systems and legacy codes,” SIAM Journal on Applied Dynamical Systems, vol. 4, no. 3, pp. 711–732, 2005."/>
  <meta name="citation_reference" content="P. V. Kokotovic, R.E. O’Malley Jr, and P. Sannuti, “Singular perturbations and order reduction in control theory -an overview,” Automatica, vol. 12, no. 2, pp. 123–132, 1976."/>
  <meta name="citation_reference" content="N. Fenichel, “Geometric singular perturbation theory for ordinary differential equations,” Journal of Differential Equations, vol. 31, no. 1, pp. 53–98, 1979."/>
  <meta name="citation_reference" content="A. N. Tikhonov, “Systems of differential equations containing small parameters in the derivatives,” Matematicheskii Sbornik, vol. 73, no. 3, pp. 575–586, 1952."/>
  <meta name="citation_reference" content="A. Goeke, S. Walcher, and E. Zerz, “Determining “small parameters” for quasi-steady state,” Journal of Differential Equations, vol. 259, no. 3, pp. 1149–1180, 2015."/>
  <meta name="citation_reference" content="K. R. Schneider and T. Wilhelm, “Model reduction by extended quasi-steady-state approximation,” Journal of Mathematical Biology, vol. 40, no. 5, pp. 443–450, 2000."/>
  <meta name="citation_reference" content="N. Vora and P. Daoutidis, “Nonlinear model reduction of chemical reaction systems,” AIChE Journal, vol. 47, no. 10, pp. 2320–2332, 2001."/>
  <meta name="citation_reference" content="F. G. Helfferich, “Systematic approach to elucidation of multistep reaction networks,” The Journal of Physical Chemistry, vol. 93, no. 18, pp. 6676–6681, 1989."/>
  <meta name="citation_reference" content="J. A. Christiansen, “The elucidation of reaction mechanisms by the method of intermediates in quasi-stationary concentrations,” in Advances in Catalysis. Elsevier, 1953, vol. 5, pp. 311–353."/>
  <meta name="citation_reference" content="D. Del Vecchio and R. M. Murray, Biomolecular Feedback Systems. Princeton University Press Princeton, NJ, 2015."/>
  <meta name="citation_reference" content="A. Papachristodoulou, Y.-C. Chang, E. August, and J. Anderson, “Structured model reduction for dynamical networked systems,” in 49th IEEE Conference on Decision and Control (CDC). IEEE, 2010, pp. 2670–2675."/>
  <meta name="citation_reference" content="A. Pandey and R. M. Murray, “An automated model reduction tool to guide the design and analysis of synthetic biological circuits,” bioRxiv, p. 640276, 2019."/>
  <meta name="citation_reference" content="T. J. Snowden, P. H. Van Der Graaf, and M. J. Tindall, “A combined model reduction algorithm for controlled biochemical systems,” BMC Systems Biology, vol. 11, no. 1, p. 17, 2017."/>
  <meta name="citation_reference" content="B. Moore, “Principal component analysis in linear systems: Controllability, observability, and model reduction,” IEEE Transactions on Automatic Control, vol. 26, no. 1, pp. 17–32, 1981."/>
  <meta name="citation_reference" content="D. F. Enns, “Model reduction with balanced realizations: An error bound and a frequency weighted generalization,” in The 23rd IEEE Conference on Decision and Control, 1984, pp. 127–132."/>
  <meta name="citation_reference" content="J. Anderson, Y.-C. Chang, and A. Papachristodoulou, “Model decomposition and reduction tools for large-scale networks in systems biology,” Automatica, vol. 47, no. 6, pp. 1165–1174, 2011."/>
  <meta name="citation_reference" content="H. Sandberg and R. M. Murray, “Model reduction of interconnected linear systems,” Optimal Control Applications and Methods, vol. 30, no. 3, pp. 225–245, 2009."/>
  <meta name="citation_reference" content="V. Van Breusegem and G. Bastin, “Reduced order dynamical modelling of reaction systems: a singular perturbation approach,” in [1991] Proceedings of the 30th IEEE Conference on Decision and Control. IEEE, 1991, pp. 1049–1054."/>
  <meta name="citation_reference" content="F. Boulier, F. Lemaire, A. Sedoglavic, and A. Ürgüplü, “Towards an automated reduction method for polynomial ode models of biochemical reaction systems,” Mathematics in Computer Science, vol. 2, no. 3, pp. 443–464, 2009."/>
  <meta name="citation_reference" content="F. P. Davidescu and S.B. Jørgensen, “Structural parameter identifiability analysis for dynamic reaction networks,” Chemical Engineering Science, vol. 63, no. 19, pp. 4754–4762, 2008."/>
  <meta name="citation_reference" content="R. P. Dickinson and R. J. Gelinas, “Sensitivity analysis of ordinary differential equation systems - a direct method,” Journal of Computational Physics, vol. 21, no. 2, pp. 123–143, 1976."/>
  <meta name="citation_reference" content="H. K. Khalil and J. W. Grizzle, Nonlinear Systems. Prentice Hall, Upper Saddle River, NJ, 2002, vol. 3."/>
  <meta name="citation_reference" content="S. Prajna, A. Papachristodoulou, P. Seiler, and P. A. Parrilo, “Sostools and its control applications,” in Positive Polynomials in Control. Springer, 2005, pp. 273–292."/>
  <meta name="citation_reference" content="V. Chellaboina, W. M. Haddad, D. S. Bernstein, and D. A. Wilson, “Induced convolution operator norms of linear dynamical systems,” Mathematics of Control, Signals and Systems, vol. 13, no. 3, pp. 216–239, 2000."/>
  <meta name="citation_reference" content="V. Zahedzadeh, H. J. Marquez, and T. Chen, “Upper bounds for induced operator norms of nonlinear systems,” IEEE Transactions on Automatic Control, vol. 54, no. 5, pp. 1159–1165, 2009."/>
  <meta name="citation_reference" content="T. T. Georgiou, “On the computation of the gap metric,” Systems & Control Letters, vol. 11, no. 4, pp. 253–257, 1988."/>
  <meta name="citation_reference" content="B. D. Anderson, T. S. Brinsmead, and F. De Bruyne, “The vinnicombe metric for nonlinear operators,” IEEE Transactions on Automatic Control, vol. 47, no. 9, pp. 1450–1465, 2002."/>
  <meta name="citation_reference" content="V. Zahedzadeh, H. J. Marquez, and T. Chen, “On the computation of an upper bound on the gap metric for a class of nonlinear systems,” in 2008 American Control Conference. IEEE, 2008, pp. 1917–1922."/>
  <meta name="citation_reference" content="A. Pandey, “Auto-reduce : Python based automated model reduction package.” [Online]. Available: https://github.com/ayush9pandey/autoReduce"/>
  <meta name="citation_reference" content="R. D. McCardell, A. Pandey, and R. M. Murray, “Control of density and composition in an engineered two-member bacterial community,” bioRxiv, 2019."/>
  <meta name="citation_reference" content="A. Meurer, C. P. Smith, M. Paprocki, O. Č. ertík, S. B. Kirpichev, M. Rocklin, A. Kumar, S. Ivanov, J. K. Moore, S. Singh, T. Rathnayake, S. Vig, B. E. Granger, R. P. Muller, F. Bonazzi, H. Gupta, S. Vats, F. Johansson, F. Pedregosa, M. J. Curry, A. R. Terrel, v. Roučka, A. Saboo, I. Fernando, S. Kulal, R. Cimrman, and A. Scopatz, “Sympy: symbolic computing in python,” PeerJ Computer Science, vol. 3, p. e103, Jan. 2017. [Online]. Available: https://doi.org/10.7717/peerj-cs.103"/>
  <meta name="citation_reference" content="J. C. Doyle, B. A. Francis, and A. R. Tannenbaum, Feedback Control Theory. Courier Corporation, 2013."/>
  <meta name="citation_reference" content="C. Y. Hu and R. M. Murray, “Design of a genetic layered feedback controller in synthetic biological circuitry,” bioRxiv, p. 647057, 2019."/>
  <meta name="citation_reference" content="A. Pandey and R. M. Murray, “A two-state ribosome and protein model can robustly capture the chemical reaction dynamics of gene expression,” bioRxiv, 2020."/>
  <meta name="twitter:title" content="Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="We present a Python-based software package to automatically obtain phenomenological models of input-controlled synthetic biological circuits from descriptive models. From the parts and mechanism description of a synthetic biological circuit, it is easy to obtain a chemical reaction model of the circuit under the assumptions of mass-action kinetics using various existing tools. However, using these models to guide design decisions during an experiment is difficult due to a large number of reaction rate parameters and species in the model. Hence, phenomenological models are often developed that describe the effective relationships among the circuit inputs, outputs, and only the key states and parameters. In this paper, we present an algorithm to obtain these phenomenological models in an automated manner using a Python package for circuits with inputs that control the desired outputs. This model reduction approach combines the common assumptions of time-scale separation, conservation laws, and species’ abundance to obtain the reduced models that can be used for design of synthetic biological circuits. We consider an example of a simple gene expression circuit and another example of a layered genetic feedback control circuit to demonstrate the use of the model reduction procedure.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.950840v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="We present a Python-based software package to automatically obtain phenomenological models of input-controlled synthetic biological circuits from descriptive models. From the parts and mechanism description of a synthetic biological circuit, it is easy to obtain a chemical reaction model of the circuit under the assumptions of mass-action kinetics using various existing tools. However, using these models to guide design decisions during an experiment is difficult due to a large number of reaction rate parameters and species in the model. Hence, phenomenological models are often developed that describe the effective relationships among the circuit inputs, outputs, and only the key states and parameters. In this paper, we present an algorithm to obtain these phenomenological models in an automated manner using a Python package for circuits with inputs that control the desired outputs. This model reduction approach combines the common assumptions of time-scale separation, conservation laws, and species’ abundance to obtain the reduced models that can be used for design of synthetic biological circuits. We consider an example of a simple gene expression circuit and another example of a layered genetic feedback control circuit to demonstrate the use of the model reduction procedure.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-05-03"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.15.950840v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2525287"/>
  <title>
   Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0iLh7gQWCfY6k2IiSE-R5o1Dn3Halms6Vl8qkYuW8EU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.02.15.950840v3"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.02.15.950840v3"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.15.950840v3"}],"nid":"2525287","apath":"\/biorxiv\/early\/2022\/05\/03\/2020.02.15.950840.atom","pisa":"biorxiv;2020.02.15.950840v3","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.15.950840v3","title":"Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits","identifier":"node\/2525287"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022M0Sex5FMWAyC7OlKKETEjYURfjJ6sw1RxDZgC4RgMPyhc6YVUb8T3kN7In5esWlQ6+uST4SRbrAV1oROcyXmTieKokppQuOD2d8hna8QfMbw+pVqXyORdVGEzLbsoCtwO0uWcK9ykTmftT83KN+aOgNB63V98y383tu\\\\\\\/cMlv8wnXDSHavaaGmEJQkDiYQakEcPEpP9euwHUqaXraSDxgw+dkZS2V3SqsuRg1y3P3bH9R4aGCImwPy3ZQ1AUyhuTdpGNsCJk2NVGpBnwKxqmEs0I+qZBMA1HF0644wlmI22HtnLFxEFSkxoCDwBLTtTM7gKjERvA90GNLUew3RnLDE9uO5PaeO3LWbSGe7tVX\\\\\\\/h09tQh7RsO4csa4+vtDymBtcyYXJ\\\\\\\/n1QDrY7sQ9HeKWyyZ2zBfOHYwY0MPuk1rcHYeDarQ3zl9yUZqVUfkLzT7dGj2+tRBjRPoiAZj9Afg\\\\\\\/r2MrPFMgJKqbuDuF6xtNoTnPklEGTOf83TM6+5ZFeANxqsba7O59By1hlaa65SKfcfL1U+vwWJ8KfCGSvEecNqGIQ+UE5fd+rRYF+ek4TPi7Bj4MUCx7oXc181BbZJlbvh9yMOXFCfNG8Higfm\\\\\\\/mxHLSw\\\\\\\/Uw10hBH4V9+u3lhp5u636IVUsvi+YOS+PcvIxuRyKjU3kiYEgQiNvG0Hkh1GbeGxw3nzXaZL2GvLSKzUbxfMeCXZeQ6X8CPB3clDzfCnUbtnNuJnHWvb7R9Nb+A+4fwxjpeE+Qi+LNN68VgCEeMXujspf5jVOUakmaiOO7zgX7\\\\\\\/bW+CQMscTx3SB0HRntVk\\\\\\\/LW18+VJrN1wHFOdL\\\\\\\/9jD2w0A2MVoXS0xKNxYNBflfvNiqbDw\\\\\\\/jKAh+2MKFRZ5EcDzSkmMlGr4bnIqbCP+uXP76DtjxIwN9j7Fx9IuOv3Yrr5TOL3wlCzXL5orf5RsBe9\\\\\\\/kkjAKCFeC0unB6V7URF1JihEqCMJifAMh3owsdqlDaI7Ktr0FieYIQFw3pPKz0p94Q7MpUBLIq7NAPwKh9pBhLv+r9ELCRtmG0CgOlQlTFnVDl5cn\\\\\\\/9T4JyG4YWojSgarEkr9ATOPwh+7okqV13dY7V7Pl5LsB2haePP\\\\\\\/Lf2sEJunNfPaoBQj8JDDpavHW6cHikbQUSRY5P2xETuw6Oty0XgzbWonW0ubwiZtUV2RR22mbdR0iMVwFfB803oBhC4mxS0zgmymU3Evqzqt801aeAUYEFVC7VevBdWAKLT1J\\\\\\\/It4sxkbpSWw1f\\\\\\\/UbFAwds2w8Nj3wG5KXYH1VRUL3ap1eJp4ghl6Sd4RE701S+hnnM85cAxTmOUUAwlgvtHAu8zV\\\\\\\/tbaa+SfLyDICzzQvjAgWwoJAUsMr2gs\\\\\\\/2kA\\\\\\\/LcJ7qNaZV1L2lPjRQ9y9k=\\\u0022,\\\u0022iv\\\u0022:\\\u0022vU4VvJ1Hy+E5MTP14dSoSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ee54cd877e59f3038f9bfe5fdef210e19bc804df109514a4587e78ae8a4848e3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.15.950840v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2525287 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.15.950840v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1825099352">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1825099352" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1320895684" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-kDJRu6PejrCjGy7U2Y6DofbxNDPMgwYGiHrw-9mPTMs"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2525287" data-node-nid="2525287" id="node-2525287--2232076353" data-pisa="biorxiv;2020.02.15.950840v3" data-pisa-master="biorxiv;2020.02.15.950840" data-apath="/biorxiv/early/2022/05/03/2020.02.15.950840.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-3590-4459" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ayush
                      </span>
                      <span class="nlm-surname">
                       Pandey
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-5785-7481" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Richard M.
                      </span>
                      <span class="nlm-surname">
                       Murray
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.02.15.950840
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2525287--2232076353" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Ayush Pandey
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">
                        Department of Control and Dynamical Systems at California Institute of Technology
                       </span>
                       , Pasadena, CA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ayush%2BPandey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Pandey%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAyush%2BPandey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-3590-4459" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ayush Pandey
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:apandey@caltech.edu" class="" data-icon-position="" data-hide-link-title="0">
                        apandey@caltech.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Richard M. Murray
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-institution">
                        Department of Control and Dynamical Systems at California Institute of Technology
                       </span>
                       , Pasadena, CA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Richard%2BM.%2BMurray%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Murray%20RM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARichard%2BM.%2BMurray%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-5785-7481" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Richard M. Murray
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.02.15.950840v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2525287" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2525287/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.15.950840v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2525287" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2525287/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.15.950840v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2525287" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2525287/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.15.950840v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2525287" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2525287/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.15.950840v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2525287" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2525287/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.02.15.950840v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2525287" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2525287/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             We present a Python-based software package to automatically obtain phenomenological models of input-controlled synthetic biological circuits from descriptive models. From the parts and mechanism description of a synthetic biological circuit, it is easy to obtain a chemical reaction model of the circuit under the assumptions of mass-action kinetics using various existing tools. However, using these models to guide design decisions during an experiment is difficult due to a large number of reaction rate parameters and species in the model. Hence, phenomenological models are often developed that describe the effective relationships among the circuit inputs, outputs, and only the key states and parameters. In this paper, we present an algorithm to obtain these phenomenological models in an automated manner using a Python package for circuits with inputs that control the desired outputs. This model reduction approach combines the common assumptions of time-scale separation, conservation laws, and species’ abundance to obtain the reduced models that can be used for design of synthetic biological circuits. We consider an example of a simple gene expression circuit and another example of a layered genetic feedback control circuit to demonstrate the use of the model reduction procedure.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             I. I
                             <span class="sc">
                              ntroduction
                             </span>
                            </h2>
                            <p id="p-7">
                             Model reduction is a widely used tool in engineering design and analysis. Abstracting away the details of a system model to focus on modeling the properties of interest and its interactions is an important insight that is commonly used in control systems’ design. Reduced models are also useful to specify the desired objectives or the performance specifications of a system. To meet these objectives, the designer needs to map these reduced models to the level of system design and also mathematically characterize this mapping in order to understand and analyze the system performance. For biological systems, this is a challenge that obscures the use of mathematical models in experimental designs and analysis to some extent. One of the most common assumptions in biological model reduction is that of time-scale separation wherein processes work at different time scales. This assumption has been used to obtain reduced order models by assuming species at quasi-steady state.
                            </p>
                            <p id="p-8">
                             A recent review paper [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ] discusses basic ideas around quasi-steady state approximation (QSSA) based model reduction approaches. The paper also discusses other common approaches of model reduction such as finding slow invariant manifolds [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ], computational [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ] and geometric [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ] singular perturbation theory in reaction kinetics models. The latter approaches are based on utilizing time-scale separation in system dynamics. However, a common downside to singular perturbation theory [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ] based model reduction approaches is that finding the small parameter that transforms the system of ordinary differential equations into the required structure of singularly perturbed dynamics is non-trivial and hard to determine for general system dynamics. In [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ], a method to determine the small parameters is presented and applied to the enzymatic reaction system. However, it is not yet clear how this approach would scale up to the commonly encountered models in synthetic biology. Hence, the relaxed version of singular perturbation based model reduction using QSSA [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ]–[
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ] is more commonly used.
                            </p>
                            <p id="p-9">
                             With standard QSSA based model reduction we do not get bounds on the error in reduction or any other form of guarantee that ensures that the reduced models are correct. The assumptions that drive the QSSA methods are usually ad hoc and based on understanding of the system in consideration at particular operating conditions. Due to these disadvantages, the approach cannot be systematically used in a generalized fashion or be guaranteed to work for systems under various conditions often leading to incorrect models [11, Ch. 3]. A computational approach to improve QSSA based model reduction that provides bounds on error in reduction was shown in [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ]. In [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ], this approach was extended to ensure that the reduced models obtained by repeated application of combinatorial QSSA leads to models that are robust with respect to parameter variations along with minimizing the errors. Our work in this paper develops further on this work.
                            </p>
                            <p id="p-10">
                             A model reduction approach for controlled biochemical reaction systems is presented in [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ]. Algebraic approaches such as state elimination using conservation laws, parameter lumping and non-dimensionalization are combined with a balanced truncation method that retains the input-output mapping for the controlled biochemical system. Balanced truncation [
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ]–[
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ] is a model reduction method that is based on proposing a transformation of system coordinates so that the unobservable and uncontrollable modes can be eliminated. These methods have been shown to be successful for linear dynamical systems [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ]. However, extensions of these methods to nonlinear dynamics is still an unsolved research question. Other approaches also use transformation of coordinates to reduce a system model such as in [
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ] where it is shown that coordinate transformations can be proposed to bring the system into the desired singular perturbation form in order to identify the slow and the fast dynamics. Based on this approach, an automated algorithm has also been presented in [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ]. However, this algorithm assumes that fast and slow species have already been recognized in the model, which is not always an easy task. To develop better and more suitable model reduction methods for synthetic biological circuits, it is clear that we would need to combine algebraic approaches that take advantage of the structure of chemical reaction level models with the general approaches that exploit time-scale separation assumptions. The results in [
                             <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ] reinforce this point. Our approach in this paper is a step in this direction.
                            </p>
                            <div id="sec-2" class="subsection">
                             <h3>
                              A. Summary of results in this paper
                             </h3>
                             <p id="p-11">
                              In [
                              <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ], it was shown that for an autonomous nonlinear dynamical system reduced models can be obtained in an automated fashion by choosing the set of states to collapse in the QSSA procedure by using two metrics as discussed in the later sections. We extend these results in this paper by providing results that enable us to obtain robust reduced models for general nonlinear dynamical systems that are driven by a set of inputs as well as different initial conditions. We combine this extension with the assumptions of conservation laws and species abundance to obtain phenomenological Hill function models for synthetic biological circuits. We also present a Python package that implements this algorithm. We demonstrate our method using different examples of synthetic biological circuits by obtaining commonly encountered reduced order models in an automated manner.
                             </p>
                            </div>
                            <div id="sec-3" class="subsection">
                             <h3>
                              B. Significance of the results
                             </h3>
                             <p id="p-12">
                              In addition to guiding the design of synthetic biological circuits, the reduced models play an important role in the analysis of the circuits using the experimentally obtained data. Since the model parameters that lead to the desired outputs are not unique for CRN level models, it can be challenging to identify all of the parameters. This issue of parameter non-identifiability has been explored in detail in the literature [
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ]. Using the approach in this paper, we obtain reduced order models with lower number of parameters hence helping in the analysis of these circuits. The approach can also be used to find out a mathematical description of the non-identifiable manifold as it can give expressions for the lumped parameters which correspond to this manifold.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-4">
                            <h2 class="">
                             II. P
                             <span class="sc">
                              reliminaries
                             </span>
                            </h2>
                            <p id="p-13">
                             We denote an eigenvalue of a matrix P by λ(P). The maximum eigenvalue will be denoted by λ
                             <sub>
                              max
                             </sub>
                             (P). For a state-dependent matrix P (x) we denote the maximum eigenvalue of P over all values of x by
                             <span class="inline-formula" id="inline-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-1.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-1.gif"/>
                               </noscript>
                              </span>
                             </span>
                             . Throughout this paper, we consider the Euclidean 2-norm for vectors. For example, we use the notation ‖x ‖ for the 2-norm of x ∈ ℝ
                             <sup>
                              n
                             </sup>
                             and similarly for matrices ‖ · ‖ represents the induced 2-norm. The 2-norm for a signal x(t) defined for all t ≥ 0 is given by
                             <span class="disp-formula" id="disp-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-1.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-1.gif"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                            <div id="statement-1" class="statement">
                             <span class="statement-label">
                              Definition 1.
                             </span>
                             <p id="p-14">
                              A nonlinear dynamical system with affine inputs is defined as
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where x ∈ ℝ
                              <sup>
                               n
                              </sup>
                              are the states of the system, u ∈ ℝ
                              <sup>
                               q
                              </sup>
                              are the inputs and y ∈ ℝ
                              <sup>
                               p
                              </sup>
                              are the outputs. The vector of model parameters are given by Θ and the respective dynamics are given by the nonlinear functions f, g, and h. We assume that the initial conditions for the system are given by x(0) = x
                              <sub>
                               0
                              </sub>
                              ∈ ℝ
                              <sup>
                               n
                              </sup>
                              .
                             </p>
                            </div>
                            <div id="statement-2" class="statement">
                             <span class="statement-label">
                              Definition 2.
                             </span>
                             <p id="p-15">
                              A reduced order nonlinear dynamical system for the system in (1) is defined as
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and ŷ ∈ ℝ
                              <sup>
                               p
                              </sup>
                              , since we assume that the number of outputs are not reduced. The initial condition for the reduced model is given by
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                            </div>
                            <div id="statement-3" class="statement">
                             <span class="statement-label">
                              Definition 3.
                             </span>
                             <p id="p-16">
                              The sensitivity coefficient of a state variable x with respect to a parameter θ ∈ Θ is defined as
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <p id="p-17">
                             From [
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ], for dynamics of x given by the nonlinear function f, i.e.,
                             <span class="inline-formula" id="inline-formula-4">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-4.gif.backup.1651768043.7309"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-4.gif.backup.1651768043.7309"/>
                               </noscript>
                              </span>
                             </span>
                             we know that the sensitivity coefficients at x(t) = x
                             <sup>
                              ∗
                             </sup>
                             satisfy the sensitivity equation
                             <span class="inline-formula" id="inline-formula-5">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-5.gif.backup.1651768043.7884"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-5.gif.backup.1651768043.7884"/>
                               </noscript>
                              </span>
                             </span>
                             , where J
                             <sub>
                              f
                             </sub>
                             (x
                             <sup>
                              ∗
                             </sup>
                             ) ℝ
                             <sup>
                              n×n
                             </sup>
                             is defined as the Jacobian of the nonlinear function f at the point x
                             <sup>
                              ∗
                             </sup>
                             ,
                             <span class="disp-formula" id="disp-formula-5">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-5.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-5.gif"/>
                               </noscript>
                              </span>
                             </span>
                             and sensitivity to parameter matrix for the function f is defined as Z
                             <sub>
                              f
                             </sub>
                             ∈ ℝ
                             <sup>
                              n×1
                             </sup>
                             and given by,
                             <span class="disp-formula" id="disp-formula-6">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-6.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-6.gif"/>
                               </noscript>
                              </span>
                             </span>
                             For simplicity, we often drop the subscript f to write J
                             <sub>
                              f
                             </sub>
                             = J and Z
                             <sub>
                              f
                             </sub>
                             = Z.
                            </p>
                            <p id="p-18">
                             For simpler notations when working with the full and reduced model together, we define an augmented state vector
                             <span class="inline-formula" id="inline-formula-6">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-6.gif.backup.1651768043.9534"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-6.gif.backup.1651768043.9534"/>
                               </noscript>
                              </span>
                             </span>
                             as follows
                             <span class="disp-formula" id="disp-formula-7">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-7.gif.backup.1651768044.0051"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-7.gif.backup.1651768044.0051"/>
                               </noscript>
                              </span>
                             </span>
                             where x is the full state vector and
                             <span class="inline-formula" id="inline-formula-7">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-7.gif.backup.1651768044.0765"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-7.gif.backup.1651768044.0765"/>
                               </noscript>
                              </span>
                             </span>
                             is the reduced state vector. We extend this notation to all other corresponding elements such as
                             <span class="inline-formula" id="inline-formula-8">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-8.gif.backup.1651768044.1386"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-8.gif.backup.1651768044.1386"/>
                               </noscript>
                              </span>
                             </span>
                             , etc. as given below
                             <span class="disp-formula" id="disp-formula-8">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-8.gif.backup.1651768044.1978"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-8.gif.backup.1651768044.1978"/>
                               </noscript>
                              </span>
                             </span>
                            </p>
                           </div>
                           <div class="section" id="sec-5">
                            <h2 class="">
                             III. R
                             <span class="sc">
                              esults
                             </span>
                            </h2>
                            <p id="p-19">
                             Before presenting the main results, it is important to note the following assumptions that we make for the rest of this paper:
                            </p>
                            <ol class="list-ord" id="list-1">
                             <li id="list-item-1">
                              <p id="p-20">
                               We assume mass-action kinetics for chemical reaction network dynamics that describe the biological processes. Hence, our circuit models will be given by deterministic ordinary differential equations.
                              </p>
                             </li>
                             <li id="list-item-2">
                              <p id="p-21">
                               As discussed in the Introduction, we are only interested in the structured model reduction problem where the states of the full model retain their definitions in the reduced models. Hence, we do not consider coordinate transformations to reduce the models. As a result, the reduced state variables are related to the full state variables as follows:
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where T is a permutation matrix with only one nonzero element in each row or column, and x
                               <sub>
                                c
                               </sub>
                               is the set of collapsed state variables, i.e. the variables for which dynamics are collapsed in the process of reducing the model.
                              </p>
                             </li>
                             <li id="list-item-3">
                              <p id="p-22">
                               From the previous assumption, it is also implied that the inputs and outputs will always be retained in a reduced model.
                              </p>
                             </li>
                             <li id="list-item-4">
                              <p id="p-23">
                               As given in the nonlinear dynamical system in
                               <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                                equation (1)
                               </a>
                               , we assume affine input dynamics.
                              </p>
                             </li>
                            </ol>
                            <p id="p-24">
                             We are now ready to present our main results. Consider a forced nonlinear dynamical system with an input u given by
                             <span class="disp-formula" id="disp-formula-10">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-10.gif.backup.1651768044.2564"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-10.gif.backup.1651768044.2564"/>
                               </noscript>
                              </span>
                             </span>
                             a linear relationship for the output,
                             <span class="disp-formula" id="disp-formula-11">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-11.gif.backup.1651768044.3343"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-11.gif.backup.1651768044.3343"/>
                               </noscript>
                              </span>
                             </span>
                             and the reduced model
                             <span class="disp-formula" id="disp-formula-12">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-12.gif.backup.1651768044.3739"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-12.gif.backup.1651768044.3739"/>
                               </noscript>
                              </span>
                             </span>
                             <span class="disp-formula" id="disp-formula-13">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-13.gif.backup.1651768044.4304"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-13.gif.backup.1651768044.4304"/>
                               </noscript>
                              </span>
                             </span>
                             where x ∈ ℝ
                             <sup>
                              n
                             </sup>
                             and the reduced state variables
                             <span class="inline-formula" id="inline-formula-9">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-9.gif.backup.1651768044.4966"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-9.gif.backup.1651768044.4966"/>
                               </noscript>
                              </span>
                             </span>
                             such that
                             <span class="inline-formula" id="inline-formula-10">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-10.gif.backup.1651768044.5386"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-10.gif.backup.1651768044.5386"/>
                               </noscript>
                              </span>
                             </span>
                             . For simplicity of exposition, we assume that the input is scalar. The results that follow can be derived for systems with multiple inputs as well but with more complicated algebra. To choose the correct reduced model obtained after making the appropriate assumptions of time-scale separation, conservation laws, and species abundance assumptions, we consider the following metrics. We show how we can computationally associate these metrics with every possible reduced model to decide the best reduced model for a given system dynamics. Note that these results extend the results in [
                             <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ] for non-autonomous nonlinear dynamical systems.
                            </p>
                            <div id="sec-6" class="subsection">
                             <h3>
                              A. Error metric
                             </h3>
                             <p id="p-25">
                              Similar to [
                              <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ], we can derive a bound on norm of the error in the outputs e = ‖y −ŷ ‖
                              <sub>
                               2
                              </sub>
                              . Clearly, this is the most important metric of comparison between a full model and a reduced model as it compares the outputs of the two models for all time. Define an augmented system with states of the full and the reduced model stacked together as follows,
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Note that we omitted the variables in the notation above for f (x, Θ), g(x, Θ), and so on for brevity. The advantage of this augmented state dynamics is that the error in the outputs e naturally appears as the output for this augmented system since,
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-15.gif.backup.1651768044.5654"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-15.gif.backup.1651768044.5654"/>
                                </noscript>
                               </span>
                              </span>
                              As pointed out in [
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ], finding a bound on the error for general nonlinear systems might not always be possible and is usually associated with finding a Lyapunov function for the nonlinear system [
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ]. So, in the next theorem we present a bound on the error for the linearized dynamics of the full and reduced order model and discuss the methods available for nonlinear systems next. Define,
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and its linearization around a point x
                              <sup>
                               ∗
                              </sup>
                              ∈ ℝ
                              <sup>
                               n
                              </sup>
                              , given by the Jacobian evaluated at this point,
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <div id="statement-4" class="statement">
                              <span class="statement-label">
                               Theorem 1.
                              </span>
                              <p id="p-26">
                               If the linearized dynamics of the full and the reduced model are asymptotically stable, then the norm of the error in outputs is bounded above by
                               <span class="disp-formula" id="disp-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               if there exists a P = P
                               <sup>
                                T
                               </sup>
                               ≻ 0 that satisfies
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-27">
                               Before proving this theorem, note that the matrix P can be obtained for the linearized dynamics by solving the Lyapunov equation
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where Ā is the linearization of
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               around a point in state-space. To get a bound on the error without linearizing the dynamics, we could either compute the norm of the error in a brute force way computationally or find a storage function using techniques such as SOSTOOLS [
                               <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                24
                               </a>
                               ] to find an appropriate upper bound, as shown in [
                               <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                               ]. Also note that bound in the theorem can be further simplified as shown in [
                               <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                               ], [
                               <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                               ] to only depend on the reduced variable
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and the collapsed state variables.
                              </p>
                              <p id="p-28">
                               Proof. Consider a positive definite function
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               such that P satisfies the conditions in the theorem statement. Taking the derivative with respect to time, we get
                               <span class="disp-formula" id="disp-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <p id="p-29">
                              Now, for the norm of the error we can write,
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Writing
                              <span class="inline-formula" id="inline-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              proves the theorem. □
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              B. Robustness metric
                             </h3>
                             <p id="p-30">
                              In this section we discuss a metric that gives a measure of robustness of the reduced model with respect to the full model. We can use this metric to compare various possible reduced order models to find out the models for which the error in model reduction is the least sensitive to the model parameters. This metric is given by
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for θ ∈ Θ. The next result gives a bound on the robustness metric for the forced nonlinear dynamical system in
                              <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                               equation (8)
                              </a>
                              .
                             </p>
                             <div id="statement-5" class="statement">
                              <span class="statement-label">
                               Theorem 2.
                              </span>
                              <p id="p-31">
                               For the nonlinear controlled dynamics (1) and its reduced model (2), the norm of the sensitivity of the error in model reduction S
                               <sub>
                                e
                               </sub>
                               is bounded above by,
                               <span class="disp-formula" id="disp-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               if there exists
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               such that
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               at every point
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               in the system trajectory.
                              </p>
                             </div>
                             <p id="p-32">
                              Similar to the previous theorem, the condition can be simplified further as shown in [
                              <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ] to only depend on the reduced variable
                              <span class="inline-formula" id="inline-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and the collapsed variable x
                              <sub>
                               c
                              </sub>
                              . This simplification follows by defining a permutation matrix T that separates the state variables x into
                              <span class="inline-formula" id="inline-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and x
                              <sub>
                               c
                              </sub>
                              after time-scale separation assumptions have been applied. Also note that, unlike the previous theorem, it is simpler to find a matrix P that satisfies the given condition. This is due to the fact that the sensitivity system is a linear system where we can obtain the Jacobian matrix at each time point and use the continuous-time Lyapunov equation to solve for a P that satisfies the conditions required in the theorem statement.
                             </p>
                             <p id="p-33">
                              Proof. The proof follows similar to the proof of the previous theorem by defining a function
                              <span class="inline-formula" id="inline-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and calculating the bound for ‖S
                              <sub>
                               e
                              </sub>
                              ‖. Note that
                              <span class="inline-formula" id="inline-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is defined according to the notation given in Section II. The additional condition on P helps to give the bound in the final form as given in the theorem statement. See the Appendix for full proof. □
                             </p>
                             <p id="p-34">
                              The results above assumed that the outputs of the system are linearly related to the states y = Cx, however, we can derive similar results even without this assumption.
                             </p>
                             <div id="statement-6" class="statement">
                              <span class="statement-label">
                               Theorem 3.
                              </span>
                              <p id="p-35">
                               For the system dynamics in
                               <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">
                                equation (1)
                               </a>
                               with output dynamics given by y = h(x, Θ) and the reduced model dynamics given in
                               <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                                equation (2)
                               </a>
                               with output dynamics given by
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , the norm of the sensitivity of error is bounded above by,
                               <span class="disp-formula" id="disp-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               if there exists
                               <span class="inline-formula" id="inline-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               at every point
                               <span class="inline-formula" id="inline-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               such that
                               <span class="inline-formula" id="inline-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where
                               <span class="disp-formula" id="disp-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Proof. For S
                               <sub>
                                e
                               </sub>
                               , at a point
                               <span class="inline-formula" id="inline-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , we have the following from
                               <a id="xref-statement-3-1" class="xref-statement" href="#statement-3">
                                Definition 3
                               </a>
                               ,
                               <span class="disp-formula" id="disp-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <p id="p-36">
                              Using chain rule, we can write
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Defining the Jacobian
                              <span class="inline-formula" id="inline-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and parameter sensitivity matrices
                              <span class="inline-formula" id="inline-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with respect to h and substituting back, we can write,
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Now, consider the function
                              <span class="inline-formula" id="inline-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and proceed in a similar way as in the proof of previous results to write
                              <span class="inline-formula" id="inline-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Using the result from
                              <a id="xref-statement-5-1" class="xref-statement" href="#statement-5">
                               Theorem 2
                              </a>
                              and
                              <span class="inline-formula" id="inline-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,
                              <span class="disp-formula" id="disp-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              C. Input-Output mapping
                             </h3>
                             <p id="p-37">
                              For forced nonlinear systems, in addition to the error and the robustness metric, it is also important to consider the input-output mapping so that the response of a reduced model for an input is similar to that of the full model. If the mapping from u ↦ y is linear, the computation of induced system norms is a well studied topic, see for example [
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ]. However, for general nonlinear systems, this is still an active research area [
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ] with results only available under certain structural conditions on the system dynamics. Similar to the gap-metric for linear systems [
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ], [
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                              ], there have been a few results on computation of the gap metric for nonlinear systems [
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ]. We can use similar computational results in our model reduction procedure by assigning the gap metric to each nonlinear reduced order model.
                             </p>
                             <p id="p-38">
                              On the other hand, if we are only interested in the response to an input at steady state or at a fixed number of points in the response, then we can linearize the dynamics at these points and assess the induced system norm for each reduced model and use this as a metric while choosing a reduced order model. For the system operator H : u ↦ y, we define,
                              <span class="disp-formula" id="disp-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We will use this metric to demonstrate the system norm comparison between reduced models for input signals applied to the biological examples considered in the next section.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              D. Python software package
                             </h3>
                             <p id="p-39">
                              A Python based implementation of the model reduction algorithm discussed above called
                              <kbd>
                               AutoReduce
                              </kbd>
                              is available at [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ]. The algorithm is similar to Algorithm 1 given in [
                              <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ]. In [
                              <a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ], the algorithm was used to obtain a reduced order model for the circuit in [
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ]. We extend this algorithm using the results discussed in this paper and demonstrate its application to more general examples. The Python package works by loading an ODE model into the
                              <kbd>
                               System
                              </kbd>
                              object using Python Sympy [
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ]. Local sensitivity analysis using various approximate difference methods [
                              <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ] and a more accurate method that solves the sensitivity system ODEs directly are also implemented in this package. With the
                              <kbd>
                               Reduce
                              </kbd>
                              class, the following important tools are available:
                             </p>
                             <div id="sec-10" class="subsection">
                              <h4>
                               1) Time-scale separation
                              </h4>
                              <p id="p-40">
                               To solve the time-scale separation problem for a given model, the package methods can be used to set the dynamics of the given states to collapse (x
                               <sub>
                                c
                               </sub>
                               ) to zero and to automatically substitute back into the dynamics for the reduced state variables
                               <span class="inline-formula" id="inline-formula-34">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-34.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-34.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . This method automates the QSSA procedure and can often be useful to compute QSSA based reduced models without specifying parameter values.
                              </p>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               2) Conservation laws
                              </h4>
                              <p id="p-41">
                               Using conservations laws, we can eliminate states that are conserved from our ODE model. We integrate this appraoch in our method similar to the method in [
                               <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                               ]. The conservation laws can be explicitly defined and these are used in the software package accordingly to eliminate state variables and compute the reduced order models.
                              </p>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               3) Comparison metrics
                              </h4>
                              <p id="p-42">
                               The comparison metrics discussed above are implemented in this package as well. For any pair of full model and reduced model, metrics such as the norm of the error (‖e‖) or the robustness metric (‖S
                               <sub>
                                e
                               </sub>
                               ‖ can be computed.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             IV. E
                             <span class="sc">
                              xamples
                             </span>
                            </h2>
                            <p id="p-43">
                             In this section, we will present biologically relevant examples to demonstrate the model reduction procedure discussed in the previous section. We start with the canonical example of the enzymatic reaction system for which we get the Michaelis-Menten kinetics as the reduced order model. We show that we can use the software package discussed above to specify the conservation laws and get the reduced order model in an automated manner.
                            </p>
                            <div id="sec-14" class="subsection">
                             <h3>
                              A. Enzymatic reaction dynamics
                             </h3>
                             <p id="p-44">
                              We create the chemical reaction network model using mass-action kinetics for the enzymatic reaction system described by the following reactions:
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The full mass-action kinetics based model with four species is given by:
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              To obtain the reduced model, we use the Python package presented in this paper. After loading the
                              <kbd>
                               System
                              </kbd>
                              object in the package with the full model and the parameters given above, we specify the conservation laws for this system : E = E
                              <sub>
                               total
                              </sub>
                              C and S + C + P = S
                              <sub>
                               total
                              </sub>
                              . We also specify that the output of interest for this system is P, the product species. The output species (P) are always retained in our model reduction procedure as discussed in the previous section. On running the software to automatically obtain the reduced order model, it first eliminates the two state-variables E and S based on the conservation laws. In the next step, it solves for time-scale separation to obtain a possible reduced order model. For this reduced model, we computed the error metric (‖e‖ = 9.7 ×10
                              <sup>
                               −4
                              </sup>
                              ) as well as the robustness metric (‖S
                              <sub>
                               e
                              </sub>
                              ‖) over the model initial conditions. If the metrics are lower than the desired tolerance level, then we can conclude that the final reduced model is given by:
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where K
                              <sub>
                               L
                              </sub>
                              = kE
                              <sub>
                               total
                              </sub>
                              is the lumped parameter and
                              <span class="inline-formula" id="inline-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is a function of E
                              <sub>
                               total
                              </sub>
                              , S
                              <sub>
                               total
                              </sub>
                              and model parameters given by
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where K
                              <sub>
                               m
                              </sub>
                              = (d + k)/a. All of the simulations and code required for computations for this example is available at [
                              <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ].
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              B. Gene expression : TX-TL model
                             </h3>
                             <p id="p-45">
                              Consider a model of gene expression where a gene G transcribes to an mRNA species T which translates into the protein species X. In this system, the concentration of G controls the production of the output protein X. Hence, the system has an input u = [G] and one output y = [X]. The model dynamics can be derived as an ODE by writing down the mass-action kinetics for these chemical reactions:
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The transcription starts by the action of RNA polymerase P and the translation is modeled by the binding of ribosome R to the transcript T. We have also modeled the degradation of the mRNA species by the endonucleases E. The output species of interest in this case is the protein X. Since we can control the input gene concentration, we treat [G] as our input. The system dynamics with 8 states, x = [P C
                              <sub>
                               1
                              </sub>
                              T R C
                              <sub>
                               2
                              </sub>
                              E C
                              <sub>
                               3
                              </sub>
                              X]
                              <sup>
                               T
                              </sup>
                              ) and 11 parameters are given by:
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              In the
                              <kbd>
                               AutoReduce
                              </kbd>
                              software package, the system model as described above can be directly loaded using symbolic variables as follows:
                             </p>
                             <span class="preformat dialog">
                              sys = System(x, f, g, u, C, params)
                             </span>
                             <p id="p-46">
                              Since, we are only interested in the expression of the protein X driven by the input G, our goal is to obtain a simpler reduced order model for this system that preserves this mapping while also ensuring robust performance with respect to model parameters. We assume that the resources P, R, and E are conserved. So, we get the following conservation laws:
                              <span class="disp-formula" id="disp-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              These conservation laws can be set to the
                              <kbd>
                               Reduce
                              </kbd>
                              class object created for the
                              <kbd>
                               System
                              </kbd>
                              as follows:
                             </p>
                             <span class="preformat dialog">
                              sys_reduce = reduce(sys)
sys_reduce.set_conservation_laws([P,R,E])
                             </span>
                             <p id="p-47">
                              We can now obtain the various possible reduced order model for this transcription-translation model. From the dynamics of the system, we see that we would need to use the result of
                              <a id="xref-statement-5-2" class="xref-statement" href="#statement-5">
                               Theorem 2
                              </a>
                              . We would also need the error metric and the metric on the input-output mapping to compare the performance of various reduced models. We can use the following utility functions available in
                              <kbd>
                               AutoReduce
                              </kbd>
                              to compute the reduced models and the related metrics:
                             </p>
                             <span class="preformat dialog">
                              reduced_model =
sys_reduce.solve_timescale_separation(states)
reduced_model.get_error_metric()
reduced_model.get_robustness_metric()
reduced_model.get_gamma(x_point)
                             </span>
                             <p id="p-48">
                              In addition to these utility functions, the package also has automated functions that do all the computations required given the full model to get various possible reduced order models. For this example, we set the tolerance for maximum number of states in a reduced model to three. Among the various possible reduced models with three or fewer states, we use the computation of the metrics (as discussed above) to choose a “best” reduced order model.
                             </p>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Remark
                              </h4>
                              <p id="p-49">
                               We start by fixing the nominal parameter values and setting the initial conditions for the full CRN model. Note that the parameter values might not always be known exactly as is often the case with biological systems. However, for the numerical computations involved in the model reduction algorithm to work we need these nominal parameter values and initial conditions. The robustness metric that the algorithm computes can then be used to understand the sensitivity of each of the obtained reduced models with all the model parameters. Since we can eliminate the reduced models that have high robustness metric, the requirement of parameter values in the full model is not strict and rough guesses should suffice for the algorithm to work.
                              </p>
                              <p id="p-50">
                               On running the algorithm, we get a set of reduced order models as shown in
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                               . The detailed equations for some of the reduced models are given in
                               <a id="xref-app-2-1" class="xref-app" href="#app-2">
                                Appendix 2
                               </a>
                               . One of the reduced models that retains the states corresponding to the mRNA transcript and the output protein X is given below.
                              </p>
                              <div id="F1" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F1.large.jpg?width=800&height=600&carousel=1" title="The output response of the full CRN model compared alongside the output of various reduced order models for a step input. The legend indicates each of the reduced models with the corresponding norm of the error in the output. It is clear from the figure that we cannot choose a particular reduced model based only on the error metric." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846238129" data-figure-caption='<div class="highwire-markup">The output response of the full CRN model compared alongside the output of various reduced order models for a step input. The legend indicates each of the reduced models with the corresponding norm of the error in the output. It is clear from the figure that we cannot choose a particular reduced model based only on the error metric.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F1.medium.gif" width="440" height="307"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F1.medium.gif" width="440" height="307"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                                <span class="fig-label">
                                 Fig. 1.
                                </span>
                                <p id="p-51" class="first-child">
                                 The output response of the full CRN model compared alongside the output of various reduced order models for a step input. The legend indicates each of the reduced models with the corresponding norm of the error in the output. It is clear from the figure that we cannot choose a particular reduced model based only on the error metric.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-52">
                               Note that this is similar to the commonly used transcription-translation models [
                               <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                               ]. We can derive this model by lumping various parameters and making approximations based on the usual parameter values in the reduced model
                               <span class="inline-formula" id="inline-formula-36">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-36.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-36.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               given in the
                               <a id="xref-app-2-2" class="xref-app" href="#app-2">
                                Appendix 2
                               </a>
                               :
                               <span class="disp-formula" id="disp-formula-38">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-39.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-39.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-37">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-37.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-37.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are lumped parameters. This is the best reduced model in terms of the output error metric if we wish to restrict the number of states to two. But, it is also important to check whether this reduced model preserves the input-output mapping or not and if it is robust to various model parameters. The comparison of the robustness metric for all of the reduced models is shown in the heatmap in
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Figure 2
                               </a>
                               . Finally, in
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Figure 3
                               </a>
                               , we show the input-output mapping for various reduced models compared to the full CRN model. As discussed in the previous section, to compute system norms it is simpler to linearize the nonlinear models. In
                               <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                Figure 3
                               </a>
                               , the vertical lines show the points at which we linearize all of the models. Using the linearized model, we can compute any system norm of our choice. In this example, we compute the 2-norm [33, Ch.2] using the state-space matrices obtained after linearization [
                               <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                               ] at each of the chosen point in the trajectory. Hence, based on the three metrics, we can conclude that the reduced model that retains the states T, R, and X is the best possible reduced model for this gene expression example. The model equations are given in
                               <a id="xref-app-2-3" class="xref-app" href="#app-2">
                                Appendix 2
                               </a>
                               . All simulations, Python package setup, and parameters for this example are available at [
                               <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                               ].
                              </p>
                              <div id="F2" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F2.large.jpg?width=800&height=600&carousel=1" title="Robustness metric (‖Se‖) for different reduced models as computed according to the results in the previous section. From this heatmap, it is clear that to get lower sensitivity to the model parameters, we need to increase the size of  by 1 and consider models with three states retained. The reduced model with  is the least sensitive to the model parameters, hence it can be a good candidate for our choice of final reduced order model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846238129" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Robustness metric (‖<em>S</em><sub><em>e</em></sub>‖) for different reduced models as computed according to the results in the previous section. From this heatmap, it is clear that to get lower sensitivity to the model parameters, we need to increase the size of  by 1 and consider models with three states retained. The reduced model with  is the least sensitive to the model parameters, hence it can be a good candidate for our choice of final reduced order model.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F2.medium.gif" width="440" height="309"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F2.medium.gif" width="440" height="309"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Fig. 2.
                                </span>
                                <p id="p-53" class="first-child">
                                 Robustness metric (‖S
                                 <sub>
                                  e
                                 </sub>
                                 ‖) for different reduced models as computed according to the results in the previous section. From this heatmap, it is clear that to get lower sensitivity to the model parameters, we need to increase the size of
                                 <span class="inline-formula" id="inline-formula-38">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/F2/embed/inline-graphic-38.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/F2/embed/inline-graphic-38.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 by 1 and consider models with three states retained. The reduced model with
                                 <span class="inline-formula" id="inline-formula-39">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/F2/embed/inline-graphic-39.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/F2/embed/inline-graphic-39.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 is the least sensitive to the model parameters, hence it can be a good candidate for our choice of final reduced order model.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F3" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F3.large.jpg?width=800&height=600&carousel=1" title="Comparison of γ(H) for each of the reduced model linearized at the points denoted. From this metric, we can conclude that the reduced model with  preserves the input-output mapping when compared to the full CRN model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846238129" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Comparison of <em>γ</em>(<em>H</em>) for each of the reduced model linearized at the points denoted. From this metric, we can conclude that the reduced model with  preserves the input-output mapping when compared to the full CRN model.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F3.medium.gif" width="440" height="307"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F3.medium.gif" width="440" height="307"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Fig. 3.
                                </span>
                                <p id="p-54" class="first-child">
                                 Comparison of γ(H) for each of the reduced model linearized at the points denoted. From this metric, we can conclude that the reduced model with
                                 <span class="inline-formula" id="inline-formula-40">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/F3/embed/inline-graphic-40.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/F3/embed/inline-graphic-40.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 preserves the input-output mapping when compared to the full CRN model.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              C. Layered synthetic feedback controller
                             </h3>
                             <p id="p-55">
                              As the final example, we consider a genetic layered feedback controller recently shown in [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. The circuit implements feedback control mechanisms both at the molecular and the population level to control the system performance under various disturbances. A phenomenological model was developed for this circuit and presented in [
                              <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. This model was used for the analysis of the design and to study important properties of the system such as disturbance attenuation and stability under various perturbations. However, depending on the parts and components used to build this circuit experimentally, the detailed chemical reaction level model may or may not correspond to the phenomenological model used to make the design decisions and study performance specifications of this system. Hence, the model reduction approach presented in this paper can be used to computationally determine the phenomenological model from the chemical reaction level models in order to guide the experimental design. Since this circuit has inducer molecule concentrations that are used as inputs that control the output protein expression, the results in
                              <a id="xref-statement-4-1" class="xref-statement" href="#statement-4">
                               Theorem 1
                              </a>
                              ,
                              <a id="xref-statement-5-3" class="xref-statement" href="#statement-5">
                               2
                              </a>
                              , and the discussion in Section III-C are needed in order to study the reduced models.
                             </p>
                             <p id="p-56">
                              We begin by developing a chemical reaction level model from the proposed parts and component description as shown in
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              :
                              <span class="disp-formula" id="disp-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where P
                              <sub>
                               1
                              </sub>
                              is the transcription factor Rhl-R, I
                              <sub>
                               1
                              </sub>
                              is AHL-Rhl, T denotes the inactive transcript that gets activated by the dimerized complex C
                              <sub>
                               2
                              </sub>
                              to form the active transcript A
                              <sub>
                               T
                              </sub>
                              . This active transcript A
                              <sub>
                               T
                              </sub>
                              transcribes into the mRNA species given by M
                              <sub>
                               S
                              </sub>
                              . Finally, M
                              <sub>
                               S
                              </sub>
                              translates into the signalling protein Cin-R denoted by S. A small RNA R acts as a regulator that represses the expression of S as it binds to the active transcript A
                              <sub>
                               T
                              </sub>
                              to make it inactive. The protein S signals the production of the cis-regulatory mechanism which is governed by the following set of chemical reactions:
                              <span class="disp-formula" id="disp-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where I
                              <sub>
                               2
                              </sub>
                              is AHL-Cin and A
                              <sub>
                               C
                              </sub>
                              denotes the active transcript that leads to the downstream expression of the regulator sRNA R. M
                              <sub>
                               G
                              </sub>
                              and G denote the mRNA and the protein species of the fluorescent protein GFP. Using mass-action kinetics, we can obtain an ODE model for the circuit. Loading this model into the software package and setting the conservation laws for this circuit, our goal is to find out reduced order models for this circuit. We use the following conservation laws:
                              <span class="disp-formula" id="disp-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F4.large.jpg?width=800&height=600&carousel=1" title="Circuit design for the layered feedback controller from [34]. In this circuit, R is an sRNA regulator that interacts with the attenuator (Att) RNA. This interaction causes transcriptional termination, and therefore down regulates the gene expression of downstream genes. The promoter PRhl is activated by the (input 1) inducer molecule (Rhl-AHL), which expresses the transcription factor CinR that activates the Pcin promoter in presence of the Cin-AHL inducer (input 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846238129" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Circuit design for the layered feedback controller from [34]. In this circuit, R is an sRNA regulator that interacts with the attenuator (Att) RNA. This interaction causes transcriptional termination, and therefore down regulates the gene expression of downstream genes. The promoter P<sub>Rhl</sub> is activated by the (input 1) inducer molecule (Rhl-AHL), which expresses the transcription factor CinR that activates the P<sub>cin</sub> promoter in presence of the Cin-AHL inducer (input 2).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F4.medium.gif" width="440" height="151"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F4.medium.gif" width="440" height="151"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/03/2020.02.15.950840/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 4.
                               </span>
                               <p id="p-57" class="first-child">
                                Circuit design for the layered feedback controller from [
                                <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">
                                 34
                                </a>
                                ]. In this circuit, R is an sRNA regulator that interacts with the attenuator (Att) RNA. This interaction causes transcriptional termination, and therefore down regulates the gene expression of downstream genes. The promoter P
                                <sub>
                                 Rhl
                                </sub>
                                is activated by the (input 1) inducer molecule (Rhl-AHL), which expresses the transcription factor CinR that activates the P
                                <sub>
                                 cin
                                </sub>
                                promoter in presence of the Cin-AHL inducer (input 2).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-58">
                              The full model after substituting the conservation laws is given by:
                              <span class="disp-formula" id="disp-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We use the
                              <kbd>
                               AutoReduce
                              </kbd>
                              package for this system model to get various reduced models. Note that we can obtain these reduced model representations without the explicit information about all the parameters. So, if we assume that the fast states are known to us a priori, then we can use this software package to obtain the reduced models symbolically. Using the information from [
                              <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ], we can obtain a reduced model for the layered feedback control circuit. The reduced model obtained is given below. It is possible to show that this model is similar to the minimal model given in [
                              <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ]. The model is obtained by retaining the states R, M
                              <sub>
                               S
                              </sub>
                              , A
                              <sub>
                               T
                              </sub>
                              , S
                              <sup>
                               ∗
                              </sup>
                              , C
                              <sub>
                               C
                              </sub>
                              , M
                              <sub>
                               G
                              </sub>
                              , G, G
                              <sup>
                               ∗
                              </sup>
                              . The reduced model is given by
                              <span class="disp-formula" id="disp-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              On further simplification, introducing lumped parameters, and making approximations using parameter values from [
                              <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ], we can write the reduced model as follows:
                              <span class="disp-formula" id="disp-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              From this result, we conclude that the minimal model given in [
                              <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ] is a good approximation of the full CRN level system dynamics under some assumptions. With this systematic derivation of the reduced model, we can analyse the parameter regimes in which the minimal model is close to the full model.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             V. C
                             <span class="sc">
                              onclusion
                             </span>
                            </h2>
                            <p id="p-59">
                             Our main result in this paper gives a structured model reduction approach for forced nonlinear dynamical systems that model the chemical reactions occurring in a biological process using mass-action kinetics. We extended the results in [
                             <a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ] to give model reduction tools for controlled synthetic biological circuits (i.e. for systems with inputs). We derived three different metrics to compare various possible reduced order models viz. the normed error in outputs to compare the output responses between the full model and the reduced models, the sensitivity of error with respect to model parameters to compare the robustness of reduced models, and an induced norm metric to compare the input-output mapping for systems driven by inputs. Using these results, we presented an automated computational pipeline to derive phenomenological models of biological circuits directly from chemical reaction level descriptions. We implemented this automated pipeline as a Python package [
                             <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ] and demonstrated its utility using examples of model reduction of a simple gene expression transcription-translation circuit model and a layered genetic feedback controller.
                            </p>
                            <p id="p-60">
                             An important direction of future work could be to mathematically characterize the mapping from the chemical reaction level models to the phenomenological models describing the system behavior. The computational pipeline developed in this paper might be helpful towards that end.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             A
                             <span class="sc">
                              cknowledgment
                             </span>
                            </h2>
                            <p id="p-61">
                             We would like to thank Chelsea Hu for her help with the modeling of the layered feedback controller example. This research is sponsored in part by the National Science Foundation under grant number: CBET-1903477 and the Defense Advanced Research Projects Agency (Agreement HR0011-17-2-0008). The content of the information does not necessarily reflect the position or the policy of the Government, and no official endorsement should be inferred.
                            </p>
                           </div>
                           <div class="section app" id="app-1">
                            <h2 class="">
                             A
                             <span class="sc">
                              ppendix
                             </span>
                             1
                            </h2>
                            <p id="p-62">
                             Proof. For the nonlinear system in (8), we can write the sensitivity coefficients at any point x(t) = x
                             <sup>
                              ∗
                             </sup>
                             as
                             <span class="disp-formula" id="disp-formula-45">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-49.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-49.gif"/>
                               </noscript>
                              </span>
                             </span>
                             where θ ∈ Θ. Using chain rule, we can derive the sensitivity system equation given by
                             <span class="disp-formula" id="disp-formula-46">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-50.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-50.gif"/>
                               </noscript>
                              </span>
                             </span>
                             for a scalar u. Similarly, we can write a relation for the reduced model and hence for the augmented system given in
                             <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">
                              equation (12)
                             </a>
                             . For the augmented system, the sensitivity coefficients are denoted by
                             <span class="inline-formula" id="inline-formula-41">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-41.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-41.gif"/>
                               </noscript>
                              </span>
                             </span>
                             . For this system, at point
                             <span class="inline-formula" id="inline-formula-42">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-42.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-42.gif"/>
                               </noscript>
                              </span>
                             </span>
                             , consider a function
                             <span class="inline-formula" id="inline-formula-43">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-43.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-43.gif"/>
                               </noscript>
                              </span>
                             </span>
                             for
                             <span class="inline-formula" id="inline-formula-44">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-44.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-44.gif"/>
                               </noscript>
                              </span>
                             </span>
                             that satisfies the conditions given in the theorem statement. Taking the derivative of V with respect to time, we can write,
                             <span class="disp-formula" id="disp-formula-47">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-51.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-51.gif"/>
                               </noscript>
                              </span>
                             </span>
                             Substituting for
                             <span class="inline-formula" id="inline-formula-45">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-45.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-45.gif"/>
                               </noscript>
                              </span>
                             </span>
                             using the sensitivity system equation derived above as
                             <span class="inline-formula" id="inline-formula-46">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-46.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-46.gif"/>
                               </noscript>
                              </span>
                             </span>
                             we get
                             <span class="disp-formula" id="disp-formula-48">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-52.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-52.gif"/>
                               </noscript>
                              </span>
                             </span>
                             Now if there exists a
                             <span class="inline-formula" id="inline-formula-47">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-47.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-47.gif"/>
                               </noscript>
                              </span>
                             </span>
                             such that
                             <span class="disp-formula" id="disp-formula-49">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-53.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-53.gif"/>
                               </noscript>
                              </span>
                             </span>
                             then we get the following bound by manipulating the 2-norm of S
                             <sub>
                              e
                             </sub>
                             ,
                             <span class="disp-formula" id="disp-formula-50">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-54.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-54.gif"/>
                               </noscript>
                              </span>
                             </span>
                             Finally, applying a bound on the initial conditions for the sensitivity coefficients, we get
                             <span class="disp-formula" id="disp-formula-51">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-55.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-55.gif"/>
                               </noscript>
                              </span>
                             </span>
                             which gives us the desired result. This can be further simplified as shown in [
                             <a id="xref-ref-13-10" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ] to express this bound only in terms of
                             <span class="inline-formula" id="inline-formula-48">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-48.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-48.gif"/>
                               </noscript>
                              </span>
                             </span>
                             , the reduced state variables and x
                             <sub>
                              c
                             </sub>
                             , the collapsed state variables. □
                            </p>
                           </div>
                           <div class="section app" id="app-2">
                            <h2 class="">
                             A
                             <span class="sc">
                              ppendix
                             </span>
                             2
                            </h2>
                            <div class="section" id="sec-19">
                             <h3>
                              Reduced models for gene expression example
                             </h3>
                             <p id="p-63">
                              The model equations for various reduced order models discussed in the Results section are given below. For
                              <span class="inline-formula" id="inline-formula-49">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              we have
                              <span class="disp-formula" id="disp-formula-52">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-56.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-56.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For
                              <span class="inline-formula" id="inline-formula-50">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,we have
                              <span class="disp-formula" id="disp-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-57.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-57.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For
                              <span class="inline-formula" id="inline-formula-51">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              we have
                              <span class="disp-formula" id="disp-formula-54">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For
                              <span class="inline-formula" id="inline-formula-52">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/inline-graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                              we have
                              <span class="disp-formula" id="disp-formula-55">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/03/2020.02.15.950840/embed/graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              All other reduced model equations, simulations, and computations are available in [
                              <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ]. For a detailed study of gene expression models, the reader is referred to our latest work in [
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ].
                             </p>
                            </div>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               Email :
                               <span class="em-link">
                                <span class="em-addr">
                                 apandey{at}caltech.edu
                                </span>
                               </span>
                               ,
                               <span class="em-link">
                                <span class="em-addr">
                                 murray{at}cds.caltech.edu
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-5">
                               Fixed an error in the main theorem.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-3">
                              <p id="p-6">
                               <a href="https://github.com/ayush9pandey/autoReduce">
                                https://github.com/ayush9pandey/autoReduce
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             R
                             <span class="sc">
                              eferences
                             </span>
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gorban
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Model reduction in chemical dynamics: slow invariant manifolds, singular perturbations, thermodynamic estimates, and analysis of reaction graph
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Current Opinion in Chemical Engineering
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  48
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  59
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BChemical%2BEngineering%26rft.volume%253D21%26rft.spage%253D48%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gear
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kaper
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kevrekidis
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zagaris
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Projecting to a slow manifold: Singularly perturbed systems and legacy codes
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  SIAM Journal on Applied Dynamical Systems
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  711
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  732
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSIAM%2BJournal%2Bon%2BApplied%2BDynamical%2BSystems%26rft.volume%253D4%26rft.spage%253D711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kokotovic
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.E.
                                  </span>
                                  <span class="cit-name-surname">
                                   O’Malley
                                  </span>
                                  <span class="cit-name-suffix">
                                   Jr
                                  </span>
                                  .
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sannuti
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Singular perturbations and order reduction in control theory -an overview
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Automatica
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  123
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  132
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1976
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAutomatica%26rft.volume%253D12%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fenichel
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Geometric singular perturbation theory for ordinary differential equations
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Differential Equations
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  53
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  98
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1979
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDifferential%2BEquations%26rft.volume%253D31%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tikhonov
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Systems of differential equations containing small parameters in the derivatives
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Matematicheskii Sbornik
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  73
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  575
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  586
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1952
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMatematicheskii%2BSbornik%26rft.volume%253D73%26rft.spage%253D575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Goeke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Walcher
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zerz
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Determining “small parameters” for quasi-steady state
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Differential Equations
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  259
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1149
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1180
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDifferential%2BEquations%26rft.volume%253D259%26rft.spage%253D1149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schneider
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilhelm
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Model reduction by extended quasi-steady-state approximation
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Mathematical Biology
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  443
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  450
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DSchneider%26rft.auinit1%253DK.%2BR.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D443%26rft.epage%253D450%26rft.atitle%253DModel%2Breduction%2Bby%2Bextended%2Bquasi-steady-state%2Bapproximation.%26rft_id%253Dinfo%253Apmid%252F10885593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10885593&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F03%2F2020.02.15.950840.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.15.950840v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vora
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Daoutidis
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Nonlinear model reduction of chemical reaction systems
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  AIChE Journal
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  2320
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2332
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAIChE%2BJournal%26rft.volume%253D47%26rft.spage%253D2320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.15.950840v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F. G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Helfferich
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Systematic approach to elucidation of multistep reaction networks
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Physical Chemistry
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  18
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  6676
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6681
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1989
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPhysical%2BChemistry%26rft.volume%253D93%26rft.spage%253D6676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.15.950840v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Christiansen
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-chapter-title">
                                  The elucidation of reaction mechanisms by the method of intermediates in quasi-stationary concentrations
                                 </span>
                                 ,”
                                 <span class="cit-source">
                                  in Advances in Catalysis
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Elsevier
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1953
                                 </span>
                                 , vol.
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  311
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  353
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253Din%2BAdvances%2Bin%2BCatalysis%26rft.volume%253D5%26rft.spage%253D311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.15.950840v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Del Vecchio
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Biomolecular Feedback Systems
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Princeton University Press Princeton, NJ
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.15.950840v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Papachristodoulou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.-C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   August
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Anderson
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-chapter-title">
                                  Structured model reduction for dynamical networked systems
                                 </span>
                                 ,”
                                 <span class="cit-source">
                                  in 49th IEEE Conference on Decision and Control (CDC)
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  IEEE
                                 </span>
                                 ,
                                 <span class="cit-vol">
                                  2010
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  2670
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2675
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pandey
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  An automated model reduction tool to guide the design and analysis of synthetic biological circuits
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv, p. 640276
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Snowden
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Der Graaf
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tindall
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  A combined model reduction algorithm for controlled biochemical systems
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Systems Biology
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 , p.
                                 <span class="cit-fpage">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BSystems%2BBiology%26rft.volume%253D11%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.15" data-doi="10.1109/TAC.1981.1102568">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Principal component analysis in linear systems: Controllability, observability, and model reduction
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Transactions on Automatic Control
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  17
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1981
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTransactions%2Bon%2BAutomatic%2BControl%26rft.volume%253D26%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTAC.1981.1102568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1109/TAC.1981.1102568&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1981LG98000003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.15.950840v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Enns
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Model reduction with balanced realizations: An error bound and a frequency weighted generalization
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  in The 23rd IEEE Conference on Decision and Control
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  1984
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  127
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  132
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Anderson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.-C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Papachristodoulou
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Model decomposition and reduction tools for large-scale networks in systems biology
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Automatica
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1165
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1174
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAutomatica%26rft.volume%253D47%26rft.spage%253D1165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sandberg
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Model reduction of interconnected linear systems
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Optimal Control Applications and Methods
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  225
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  245
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DOptimal%2BControl%2BApplications%2Band%2BMethods%26rft.volume%253D30%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.15.950840v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Breusegem
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bastin
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-chapter-title">
                                  Reduced order dynamical modelling of reaction systems: a singular perturbation approach
                                 </span>
                                 ,”
                                 <span class="cit-source">
                                  in [1991] Proceedings of the 30th IEEE Conference on Decision and Control
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  IEEE
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1049
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1054
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Boulier
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lemaire
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sedoglavic
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ürgüplü
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Towards an automated reduction method for polynomial ode models of biochemical reaction systems
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Mathematics in Computer Science
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  443
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  464
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMathematics%2Bin%2BComputer%2BScience%26rft.volume%253D2%26rft.spage%253D443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Davidescu
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jørgensen
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Structural parameter identifiability analysis for dynamic reaction networks
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Chemical Engineering Science
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  63
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  19
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  4754
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4762
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DChemical%2BEngineering%2BScience%26rft.volume%253D63%26rft.spage%253D4754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Dickinson
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gelinas
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Sensitivity analysis of ordinary differential equation systems - a direct method
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Computational Physics
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  123
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  143
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1976
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BComputational%2BPhysics%26rft.volume%253D21%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.15.950840v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Khalil
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Grizzle
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Nonlinear Systems
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  Prentice Hall, Upper Saddle River, NJ
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 , vol.
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.15.950840v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Prajna
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Papachristodoulou
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Seiler
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Parrilo
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-chapter-title">
                                  Sostools and its control applications
                                 </span>
                                 ,”
                                 <span class="cit-source">
                                  in Positive Polynomials in Control
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  273
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  292
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chellaboina
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Haddad
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bernstein
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Induced convolution operator norms of linear dynamical systems
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Mathematics of Control, Signals and Systems
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  216
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  239
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMathematics%2Bof%2BControl%252C%2BSignals%2Band%2BSystems%26rft.volume%253D13%26rft.spage%253D216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zahedzadeh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Marquez
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Upper bounds for induced operator norms of nonlinear systems
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Transactions on Automatic Control
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  54
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1159
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1165
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTransactions%2Bon%2BAutomatic%2BControl%26rft.volume%253D54%26rft.spage%253D1159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Georgiou
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  On the computation of the gap metric
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  Systems & Control Letters
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  253
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  257
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1988
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystems%2B%2526%2BControl%2BLetters%26rft.volume%253D11%26rft.spage%253D253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Anderson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Brinsmead
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   De Bruyne
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  The vinnicombe metric for nonlinear operators
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  IEEE Transactions on Automatic Control
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 , no.
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1450
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1465
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIEEE%2BTransactions%2Bon%2BAutomatic%2BControl%26rft.volume%253D47%26rft.spage%253D1450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.02.15.950840v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zahedzadeh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Marquez
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-chapter-title">
                                  On the computation of an upper bound on the gap metric for a class of nonlinear systems
                                 </span>
                                 ,”
                                 <span class="cit-source">
                                  in 2008 American Control Conference
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  IEEE
                                 </span>
                                 ,
                                 <span class="cit-vol">
                                  2008
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1917
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1922
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.02.15.950840v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pandey
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  “Auto-reduce : Python based automated model reduction package.” [Online]
                                 </span>
                                 . Available:
                                 <a href="https://github.com/ayush9pandey/autoReduce">
                                  https://github.com/ayush9pandey/autoReduce
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   McCardell
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pandey
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Control of density and composition in an engineered two-member bacterial community
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.32" data-doi="10.7717/peerj-cs.103">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Meurer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Paprocki
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O. Č.
                                  </span>
                                  <span class="cit-name-surname">
                                   ertík
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kirpichev
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rocklin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kumar
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ivanov
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Rathnayake
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vig
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Granger
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Muller
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bonazzi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gupta
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vats
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Johansson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pedregosa
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Curry
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Terrel
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   v.
                                  </span>
                                  <span class="cit-name-surname">
                                   Roučka
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Saboo
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                  <span class="cit-name-surname">
                                   Fernando
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kulal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cimrman
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Scopatz
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Sympy: symbolic computing in python
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  PeerJ Computer Science
                                 </abbr>
                                 , vol.
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 , p.
                                 <span class="cit-fpage">
                                  e103
                                 </span>
                                 ,
                                 <span class="cit-month">
                                  Jan
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 . [Online]. Available:
                                 <a href="https://doi.org/10.7717/peerj-cs.103">
                                  https://doi.org/10.7717/peerj-cs.103
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPeerJ%2BComputer%2BScience%26rft.volume%253D3%26rft.spage%253D103e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj-cs.103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7717/peerj-cs.103&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.15.950840v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Doyle
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Francis
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Tannenbaum
                                  </span>
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Feedback Control Theory. Courier Corporation
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  Design of a genetic layered feedback controller in synthetic biological circuitry
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , p.
                                 <span class="cit-fpage">
                                  647057
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.15.950840v3.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pandey
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Murray
                                  </span>
                                 </span>
                                 , “
                                 <span class="cit-article-title">
                                  A two-state ribosome and protein model can robustly capture the chemical reaction dynamics of gene expression
                                 </span>
                                 ,”
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.15.950840v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.05.02.490336v1" title="Deep multi-omic network fusion for marker discovery of Alzheimer’s Disease" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.12.20.473413v2" title="Integrated view and comparative analysis of baseline protein expression in mouse and rat tissues" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted May 03, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.02.15.950840v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/05/03/2020.02.15.950840.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.02.15.950840v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.02.15.950840v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2525287"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-zL6kxBBOJhUthxl4zhdLbkUR_yYXoKsSjAoH1rHFIbc"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484467145"/>
                                   <input type="hidden" name="captcha_token" value="c89393b8462b058cfe4ca9df65a4de9f"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2525287--3" data-node-nid="2525287" id="node-2525287--4540233382" data-pisa="biorxiv;2020.02.15.950840v3" data-pisa-master="biorxiv;2020.02.15.950840" data-seqnum="2525287" data-apath="/biorxiv/early/2022/05/03/2020.02.15.950840.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Ayush
                                  </span>
                                  <span class="nlm-surname">
                                   Pandey
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Richard M.
                                  </span>
                                  <span class="nlm-surname">
                                   Murray
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.02.15.950840;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.02.15.950840
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.15.950840v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&title=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&title=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&text=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&t=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&title=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&title=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&title=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2525287--3" data-node-nid="2525287" id="citation-node-2525287--4909687305" data-pisa="biorxiv;2020.02.15.950840v3" data-pisa-master="biorxiv;2020.02.15.950840" data-seqnum="2525287" data-apath="/biorxiv/early/2022/05/03/2020.02.15.950840.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Model Reduction Tools For Phenomenological Modeling of Input-Controlled Biological Circuits
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Ayush
                                      </span>
                                      <span class="nlm-surname">
                                       Pandey
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Richard M.
                                      </span>
                                      <span class="nlm-surname">
                                       Murray
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.02.15.950840;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.02.15.950840
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Pandey%2C+Ayush&rft.au=Murray%2C+Richard+M.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2525287/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2525287/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2525287/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2525287/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2525287/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2525287/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2525287/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2525287/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2525287/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2525287/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2525287/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2525287/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&count=horizontal&via=&text=Model%20Reduction%20Tools%20For%20Phenomenological%20Modeling%20of%20Input-Controlled%20Biological%20Circuits&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.15.950840v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.02.15.950840v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/synthetic-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Synthetic Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
