<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.10.01.322255v4.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.10.01.322255v4.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.10.01.322255v4.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/08/2020.10.01.322255.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.10.01.322255v4"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Information-theoretic clustering for single-cell transcriptomics"/>
  <meta name="DC.Identifier" content="10.1101/2020.10.01.322255"/>
  <meta name="DC.Date" content="2022-03-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Single-cell sequencing (sc-Seq) experiments are producing increasingly large data sets. However, large data sets do not necessarily contain large amounts of information. Here, we introduce a formal framework for assessing the amount of information obtained from a sc-Seq experiment, which can be used throughout the sc-Seq analysis pipeline, including for feature selection, differential expression analysis, unsupervised clustering and semi-supervised cell annotation. Moreover, we show how this framework leads to a biologically intuitive definition of cell ‘type’ that can be used in common cell classification tasks. Based on this theory we describe an efficient method for the automatic unsupervised or semi-supervised identification of cell types from sc-Seq data, and provide an R package implementation. Our information-theoretic method performs comparably to the state-of-the-art on a series of benchmark sc-Seq data sets, and additionally provides a simple, biological interpretation of identified clusters.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Michael J. Casey"/>
  <meta name="DC.Contributor" content="Jörg Fliege"/>
  <meta name="DC.Contributor" content="Rubén J. Sánchez-García"/>
  <meta name="DC.Contributor" content="Ben D. MacArthur"/>
  <meta name="article:published_time" content="2022-03-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Information-theoretic clustering for single-cell transcriptomics"/>
  <meta name="citation_abstract" lang="en" content="<h3>ABSTRACT</h3>
<p>Single-cell sequencing (sc-Seq) experiments are producing increasingly large data sets. However, large data sets do not necessarily contain large amounts of information. Here, we introduce a formal framework for assessing the amount of information obtained from a sc-Seq experiment, which can be used throughout the sc-Seq analysis pipeline, including for feature selection, differential expression analysis, unsupervised clustering and semi-supervised cell annotation. Moreover, we show how this framework leads to a biologically intuitive definition of cell ‘type’ that can be used in common cell classification tasks. Based on this theory we describe an efficient method for the automatic unsupervised or semi-supervised identification of cell types from sc-Seq data, and provide an R package implementation. Our information-theoretic method performs comparably to the state-of-the-art on a series of benchmark sc-Seq data sets, and additionally provides a simple, biological interpretation of identified clusters.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.10.01.322255v4"/>
  <meta name="citation_id" content="2020.10.01.322255v4"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.10.01.322255v4"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.01.322255v4.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.01.322255v4.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.10.01.322255"/>
  <meta name="citation_num_pages" content="21"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.10.01.322255"/>
  <meta name="citation_author" content="Michael J. Casey"/>
  <meta name="citation_author_institution" content="Mathematical Sciences, University of Southampton"/>
  <meta name="citation_author_institution" content="Institute for Life Sciences, University of Southampton"/>
  <meta name="citation_author" content="Jörg Fliege"/>
  <meta name="citation_author_institution" content="Mathematical Sciences, University of Southampton"/>
  <meta name="citation_author" content="Rubén J. Sánchez-García"/>
  <meta name="citation_author_institution" content="Mathematical Sciences, University of Southampton"/>
  <meta name="citation_author_institution" content="Institute for Life Sciences, University of Southampton"/>
  <meta name="citation_author_institution" content="The Alan Turing Institute"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6479-3028"/>
  <meta name="citation_author" content="Ben D. MacArthur"/>
  <meta name="citation_author_institution" content="Mathematical Sciences, University of Southampton"/>
  <meta name="citation_author_institution" content="Institute for Life Sciences, University of Southampton"/>
  <meta name="citation_author_institution" content="The Alan Turing Institute"/>
  <meta name="citation_author_institution" content="Centre for Human Development, Stem Cells and Regeneration, Faculty of Medicine, University of Southampton"/>
  <meta name="citation_author_email" content="bdm@soton.ac.uk"/>
  <meta name="citation_reference" content="Ashburner, M., Ball, C. A., Blake, J. A., Botstein, D., Butler, H., Cherry, J. M., Davis, A. P., Dolinski, K., Dwight, S. S., Eppig, J. T. et al. (2000), ‘Gene ontology: tool for the unification of biology’, Nature genetics 25(1), 25–29."/>
  <meta name="citation_reference" content="Benjamini, Y. & Hochberg, Y. (1995), ‘Controlling the false discovery rate: a practical and powerful approach to multiple testing’, Journal of the Royal statistical society: series B (Methodological) 57(1), 289–300."/>
  <meta name="citation_reference" content="Blondel, V. D., Guillaume, J.-L., Lambiotte, R. & Lefebvre, E. (2008), ‘Fast unfolding of communities in large networks’, Journal of statistical mechanics: theory and experiment 2008(10), P10008."/>
  <meta name="citation_reference" content="Byrd, R. H., Lu, P., Nocedal, J. & Zhu, C. (1995), ‘A limited memory algorithm for bound constrained optimization’, SIAM Journal on scientific computing 16(5), 1190–1208."/>
  <meta name="citation_reference" content="Chan, T. E., Stumpf, M. P. & Babtie, A. C. (2017), ‘Gene regulatory network inference from single-cell data using multivariate information measures’, Cell systems 5(3), 251–267."/>
  <meta name="citation_reference" content="Cover, T. M. & Thomas, J. A. (2012), Elements of information theory, John Wiley & Sons."/>
  <meta name="citation_reference" content="Fisher, R. (2017), Statistical Methods For Research Workers, Gyan Books."/>
  <meta name="citation_reference" content="Freytag, S., Tian, L., Lönnstedt, I., Ng, M. & Bahlo, M. (2018), ‘Comparison of clustering tools in r for medium-sized 10x genomics single-cell rna-sequencing data’, F1000Research 7."/>
  <meta name="citation_reference" content="Gene Ontology Consortium (2021), ‘The gene ontology resource: enriching a gold mine’, Nucleic Acids Research 49(D1), D325– D334."/>
  <meta name="citation_reference" content="citation_title=‘Phenotypic Switching’;citation_pages=189-206;citation_year=2020"/>
  <meta name="citation_reference" content="Hausser, J. & Strimmer, K. (2009), ‘Entropy inference and the james-stein estimator, with application to nonlinear gene association networks.’, Journal of Machine Learning Research 10(7)."/>
  <meta name="citation_reference" content="Hicks, S. C., Townes, F. W., Teng, M. & Irizarry, R. A. (2018), ‘Missing data and technical variability in single-cell rna-sequencing experiments’, Biostatistics 19(4), 562–578."/>
  <meta name="citation_reference" content="Kharchenko, P. V., Silberstein, L. & Scadden, D. T. (2014), ‘Bayesian approach to single-cell differential expression analysis’, Nature methods 11(7), 740–742."/>
  <meta name="citation_reference" content="Kiselev, V. Y., Andrews, T. S. & Hemberg, M. (2019), ‘Challenges in unsupervised clustering of single-cell rna-seq data’, Nature Reviews Genetics 20(5), 273–282."/>
  <meta name="citation_reference" content="Kullback, S. & Leibler, R. A. (1951), ‘On information and sufficiency’, The annals of mathematical statistics 22(1), 79–86."/>
  <meta name="citation_reference" content="Luecken, M. D. & Theis, F. J. (2019), ‘Current best practices in single-cell rna-seq analysis: a tutorial’, Molecular systems biology 15(6), e8746."/>
  <meta name="citation_reference" content="R Core Team (2020), R: A Language and Environment for Statistical Computing, R Foundation for Statistical Computing, Vienna, Austria. URL: https://www.R-project.org/"/>
  <meta name="citation_reference" content="Rand, W. M. (1971), ‘Objective criteria for the evaluation of clustering methods’, Journal of the American Statistical association 66(336), 846–850."/>
  <meta name="citation_reference" content="Scrucca, L., Fop, M., Murphy, T. B. & Raftery, A. E. (2016), ‘mclust 5: clustering, classification and density estimation using gaussian finite mixture models’, The R journal 8(1), 289."/>
  <meta name="citation_reference" content="Shannon, C. E. (1948), ‘A mathematical theory of communication’, Bell system technical journal 27(3), 379–423."/>
  <meta name="citation_reference" content="Shorrocks, A. F. (1980), ‘The class of additively decomposable inequality measures’, Econometrica: Journal of the Econometric Society pp. 613–625."/>
  <meta name="citation_reference" content="Smith, R. C. & MacArthur, B. D. (2017), ‘Information-theoretic approaches to understanding stem cell variability’, Current Stem Cell Reports 3(3), 225–231."/>
  <meta name="citation_reference" content="Stuart, T., Butler, A., Hoffman, P., Hafemeister, C., Papalexi, E., Mauck III, W. M., Hao, Y., Stoeckius, M., Smibert, P. & Satija, R. (2019), ‘Comprehensive integration of single-cell data’, Cell 177(7), 1888–1902."/>
  <meta name="citation_reference" content="Stumpf, P. S., Du, X., Imanishi, H., Kunisaki, Y., Semba, Y., Noble, T., Smith, R. C., Rose-Zerili, M., West, J. J., Oreffo, R. O. et al. (2020), ‘Transfer learning efficiently maps bone marrow cell types from mouse to human using single-cell rna sequencing’, Communications biology 3(1), 1–11."/>
  <meta name="citation_reference" content="Svensson, V., Natarajan, K. N., Ly, L.-H., Miragaia, R. J., Labalette, C., Macaulay, I. C., Cvejic, A. & Teichmann, S. A. (2017), ‘Power analysis of single-cell rna-sequencing experiments’, Nature methods 14(4), 381."/>
  <meta name="citation_reference" content="Svensson, V., Vento-Tormo, R. & Teichmann, S. A. (2018), ‘Exponential scaling of single-cell rna-seq in the past decade’, Nature protocols 13(4), 599–604."/>
  <meta name="citation_reference" content="Tabula Muris Consortium (2018), ‘Single-cell transcriptomics of 20 mouse organs creates a tabula muris.’, Nature 562(7727), 367."/>
  <meta name="citation_reference" content="Theil, H. (1967), Economics and Information Theory, Studies in mathematical and managerial economics, North-Holland Publishing Company."/>
  <meta name="citation_reference" content="Tian, L., Dong, X., Freytag, S., LêCao, K.-A., Su, S., JalalAbadi, A., Amann-Zalcenstein, D., Weber, T. S., Seidi, A., Jabbari, J. S. et al. (2019), ‘Benchmarking single cell rna-sequencing analysis pipelines using mixture control experiments’, Nature methods 16(6), 479–487."/>
  <meta name="citation_reference" content="Trapnell, C. (2015), ‘Defining cell types and states with single-cell genomics’, Genome research 25(10), 1491–1498."/>
  <meta name="citation_reference" content="Van Rossum, G. & Drake, F. L. (2009), Python 3 Reference Manual, CreateSpace, Scotts Valley, CA."/>
  <meta name="citation_reference" content="Virtanen, P., Gommers, R., Oliphant, T. E., Haberland, M., Reddy, T., Cournapeau, D., Burovski, E., Peterson, P., Weckesser, W., Bright, J. et al. (2020), ‘Scipy 1.0: fundamental algorithms for scientific computing in python’, Nature methods 17(3), 261–272."/>
  <meta name="citation_reference" content="Zheng, G. X., Terry, J. M., Belgrader, P., Ryvkin, P., Bent, Z. W., Wilson, R., Ziraldo, S. B., Wheeler, T. D., McDermott, G. P., Zhu, J. et al. (2017), ‘Massively parallel digital transcriptional profiling of single cells’, Nature communications 8(1), 1–12."/>
  <meta name="citation_reference" content="Zhu, C., Byrd, R. H., Lu, P. & Nocedal, J. (1997), ‘Algorithm 778: L-BFGS-B: Fortran subroutines for large-scale bound-constrained optimization’, ACM Transactions on Mathematical Software (TOMS) 23(4), 550–560."/>
  <meta name="twitter:title" content="Information-theoretic clustering for single-cell transcriptomics"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Single-cell sequencing (sc-Seq) experiments are producing increasingly large data sets. However, large data sets do not necessarily contain large amounts of information. Here, we introduce a formal framework for assessing the amount of information obtained from a sc-Seq experiment, which can be used throughout the sc-Seq analysis pipeline, including for feature selection, differential expression analysis, unsupervised clustering and semi-supervised cell annotation. Moreover, we show how this framework leads to a biologically intuitive definition of cell ‘type’ that can be used in common cell classification tasks. Based on this theory we describe an efficient method for the automatic unsupervised or semi-supervised identification of cell types from sc-Seq data, and provide an R package implementation. Our information-theoretic method performs comparably to the state-of-the-art on a series of benchmark sc-Seq data sets, and additionally provides a simple, biological interpretation of identified clusters.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Information-theoretic clustering for single-cell transcriptomics"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.10.01.322255v4"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Single-cell sequencing (sc-Seq) experiments are producing increasingly large data sets. However, large data sets do not necessarily contain large amounts of information. Here, we introduce a formal framework for assessing the amount of information obtained from a sc-Seq experiment, which can be used throughout the sc-Seq analysis pipeline, including for feature selection, differential expression analysis, unsupervised clustering and semi-supervised cell annotation. Moreover, we show how this framework leads to a biologically intuitive definition of cell ‘type’ that can be used in common cell classification tasks. Based on this theory we describe an efficient method for the automatic unsupervised or semi-supervised identification of cell types from sc-Seq data, and provide an R package implementation. Our information-theoretic method performs comparably to the state-of-the-art on a series of benchmark sc-Seq data sets, and additionally provides a simple, biological interpretation of identified clusters.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-08"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.10.01.322255v4"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2436909"/>
  <title>
   Information-theoretic clustering for single-cell transcriptomics | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"oHgGrpcabvQZZ9LFvFt1eiUFI2qPvTV85nWNx6Ji0SI"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.10.01.322255v4"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.10.01.322255v4"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.10.01.322255v4"}],"nid":"2436909","apath":"\/biorxiv\/early\/2022\/03\/08\/2020.10.01.322255.atom","pisa":"biorxiv;2020.10.01.322255v4","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.10.01.322255v4","title":"Information-theoretic clustering for single-cell transcriptomics","identifier":"node\/2436909"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225mL8iRuPXTsh6qe44F6ima4D4qbEcP68UuiEMKyN3ib8sn2Zkclv8\\\\\\\/17MxfuxuCARnLFwvR6BdMxHbPX2dsgmP9C09KY7GIRH9LlS2hhimI9Kbuyll0n5guo6iB6L\\\\\\\/cnTNtbomQEBIr+KZC8D8YYhrhTzsEXyEa4xqvRdBJkoUXdHnNf6gFPkjgWvpazhfve9+xqOqWBbwl9i9+U8rA3RnzBACRjbKWIrYaujLwdQ7xuvvvnOS0TjBudmbUhEpZzEyJnJHw8\\\\\\\/r+hp2tGGsAtjMGBs0AB+Q1cM9MNYAj5ctQgbc9Ia9im6k\\\\\\\/zRkDQ5E80tnjaHVUg4t\\\\\\\/TMYwhqsxLbYqpsCmncwcH7YTNo4zzPucWuYbim8PhsuZwmp90b2iz0Qq0BwCiSzblmFHf9g0kvutEtAHBRiRfC18gCHEMUh0fFwXb0R7SRUM\\\\\\\/dmQS0FmTLxHfj\\\\\\\/pibYwjfgP405\\\\\\\/vOFYfPuFJ6lU2kOseY\\\\\\\/QVEv+zsotvKh8KvTPfMh0Fe2Qjn7wezHZFqS3BFJQ9uLErESMtJXYchie37JY9CEV7pvEujTDXMuOlSEtQrA80PG7n5\\\\\\\/sK7bDh6JQy86njNs4ey+lzKpGz9mGYlMizEZkOwJFcj4y6mvBVwKWMnlhXx1ZrPQmV\\\\\\\/PwKZ63ohu6x7b7TsFVZHhJKWa\\\\\\\/kyInwNVnPCt6UxoiEz3hkoE7s49tcfABmVDXZKKrr854PMX9l9PuLmJ9eAHCo8CdKZD3HnsLUM9H0Q6aCmXaEjuqRuyG+ZOtoPW15O+nRr4ldn\\\\\\\/LnObB5w5gNrC3QvmSBlH8e+yBAlds2A4mccagy4FSJcgp5ca+rSXYUafkUPV75iBi70EzG5jQDYcl7+VSyBlcKdy4kMC9PzGELfh3MXakr68DTTNPqldptYlpGFSLiuUGpVo0FQeG2uQuNu0FxkXRfIh4Tb3MKjHGOV9ETPPacDzAEdBhz5ftAnzkXYEeJvNpVge4BH3q7fhMiWcHhINTGYQTL3RmBFrS5n4DrzU7i+ZEemQKgbwpa3ssoZlRZg0p2oskpgG7jbcl8jwkoWJswen4sgpXnjVK51q9yOH1QRciqB76WDzpBB5Nq57uVw44+UixCIjXdm0EBiKCwbV7pUhbBGVBd+vZf+xWLqomihd0\\\\\\\/J9DOiSd1KW7w5jtdI0z324ND5GhYPZP\\\\\\\/lpM8Nslt4537rlj+MOKi3bIgU24uAxa9zyaf1Y\\\\\\\/NHuesVk8NOQmaGzYb8QM491vo3ueYlUOdvKMNOOKBv2rfIZ6YtREhDdMmfNdi\\\\\\\/6Px1sWme5+4B5kQq91KG72n4NNkUsx31iccddDUwLvS1dkVeyo0h3OtrkFWjOJUfcKy4gaR2NuU\\\\\\\/8NRwM7u2O9wFWrxoROVWVY\\\\\\\/5NmViDk=\\\u0022,\\\u0022iv\\\u0022:\\\u00224BXlDT6UXlAxAi0FtaBepQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e53a3d59005d9c2a8a96afca72afe272859553dc7702e8231baa24a29853ad4a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.10.01.322255v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2436909 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.10.01.322255v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_200560247">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_200560247" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1785499609" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-BeeewjkXi0oY7UjixnUCI1ig_7opNespSAuMHT3dGhc"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2436909" data-node-nid="2436909" id="node-2436909--2184628997" data-pisa="biorxiv;2020.10.01.322255v4" data-pisa-master="biorxiv;2020.10.01.322255" data-apath="/biorxiv/early/2022/03/08/2020.10.01.322255.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Information-theoretic clustering for single-cell transcriptomics
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Michael J.
                      </span>
                      <span class="nlm-surname">
                       Casey
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Jörg
                      </span>
                      <span class="nlm-surname">
                       Fliege
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0001-6479-3028" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Rubén J.
                      </span>
                      <span class="nlm-surname">
                       Sánchez-García
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Ben D.
                      </span>
                      <span class="nlm-surname">
                       MacArthur
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.10.01.322255
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2436909--2184628997" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Michael J. Casey
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Mathematical Sciences, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute for Life Sciences, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Michael%2BJ.%2BCasey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Casey%20MJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMichael%2BJ.%2BCasey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Jörg Fliege
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Mathematical Sciences, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=J%C3%B6rg%2BFliege%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Fliege%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJ%25C3%25B6rg%2BFliege%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Rubén J. Sánchez-García
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Mathematical Sciences, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute for Life Sciences, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        The Alan Turing Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Rub%C3%A9n%2BJ.%2BS%C3%A1nchez-Garc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=S%C3%A1nchez-Garc%C3%ADa%20RJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARub%25C3%25A9n%2BJ.%2BS%25C3%25A1nchez-Garc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-6479-3028" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Rubén J. Sánchez-García
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Ben D. MacArthur
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Mathematical Sciences, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Institute for Life Sciences, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        The Alan Turing Institute
                       </span>
                       , London,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Centre for Human Development, Stem Cells and Regeneration, Faculty of Medicine, University of Southampton
                       </span>
                       , Southampton,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ben%2BD.%2BMacArthur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=MacArthur%20BD&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABen%2BD.%2BMacArthur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:bdm@soton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        bdm@soton.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.10.01.322255v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2436909" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2436909/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.01.322255v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2436909" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2436909/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.01.322255v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2436909" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2436909/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.01.322255v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2436909" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2436909/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.10.01.322255v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2436909" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2436909/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             ABSTRACT
                            </h2>
                            <p id="p-2">
                             Single-cell sequencing (sc-Seq) experiments are producing increasingly large data sets. However, large data sets do not necessarily contain large amounts of information. Here, we introduce a formal framework for assessing the amount of information obtained from a sc-Seq experiment, which can be used throughout the sc-Seq analysis pipeline, including for feature selection, differential expression analysis, unsupervised clustering and semi-supervised cell annotation. Moreover, we show how this framework leads to a biologically intuitive definition of cell ‘type’ that can be used in common cell classification tasks. Based on this theory we describe an efficient method for the automatic unsupervised or semi-supervised identification of cell types from sc-Seq data, and provide an R package implementation. Our information-theoretic method performs comparably to the state-of-the-art on a series of benchmark sc-Seq data sets, and additionally provides a simple, biological interpretation of identified clusters.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-4">
                             Advances in single-cell sequencing (sc-Seq) technologies have enabled us to profile thousands of cells in a single experiment (
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              Svensson et al. 2018
                             </a>
                             ). In combination with advances in unsupervised analysis methods, particularly specialised clustering algorithms and dimensionality reduction techniques, these technologies have allowed us to dissect cellular identities in unprecedented detail and discover novel, functionally important, cell types (
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              Trapnell 2015
                             </a>
                             ). The goal of most sc-Seq studies (except those focused on methodology development) is to extract biological information, often concerning the mix of cell types present in the tissue sample, from the data obtained. Yet, data is not the same as information; and large, complex, data sets do not necessarily convey useful or usable information. Notably, current single-cell profiling technologies typically produce noisy data for numerous technical reasons, including low capture rate, sparsity due to shallow sequencing, and batch effects (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Kharchenko et al. 2014
                             </a>
                             ,
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              Hicks et al. 2018
                             </a>
                             ). Consequently, the relationship between biological information and sc-Seq data is complex and incompletely understood. There is, therefore, a need for formal, quantitative methods to evaluate this relationship.
                            </p>
                            <p id="p-5">
                             To address this challenge, we propose an information-theoretic framework based on a formal yet simple definition of gene expression heterogeneity, that can be used to precisely measure the amount of information contained in a sc-Seq data set. Our method measures the information obtained from a sc-Seq experiment and decomposes it into that which is explained by putative grouping of cells – for example, by time-point labels or by a proposed clustering into cell types – and that which remains unexplained. This allows us to assess the extent to which known or inferred mechanisms explain observed expression patterns, or fail to do so. The method is computationally simple and can be used to quickly assess the information content of a sc-Seq data set, identify features of importance, and determine the extent to which observed patterns of gene expression variability are explained by the presence of a mixture of cell types in a cellular population.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <p id="p-6">
                             High-throughput single-cell analysis methods, such as single cell sequencing, typically view cells as the objects of study and seek to compare cell identities with each other (
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              Kiselev et al. 2019
                             </a>
                             ). However, this cell-centric view is less well suited to quantifying gene expression heterogeneity, which is concerned with patterns of variation that arise from the mixing of cell types within a population, and may vary from gene to gene (
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Smith & MacArthur 2017
                             </a>
                             ). Our approach is motivated by an alternative gene-centric probabilistic view that seeks to more formally specify what is meant by gene expression homogeneity and heterogeneity.
                            </p>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Quantifying gene expression heterogeneity
                             </h3>
                             <p id="p-7">
                              Consider the expression pattern of a single gene g of interest in a population of N distinct cells. Assume that in total M transcripts of g are identified in the cell population (i.e. across all N cells profiled). Note that M represents the observed transcript count, which may differ from the true count due to technical artefacts. Now consider the stochastic process of assigning the M identified transcripts of gene g to the N cells profiled. Intuitively, the population is homogeneous with respect to expression of g if all the cells are statistically the same with respect to its expression. Mathematically, this means that the M transcripts of g will be assigned to the N cells independently and equiprobably – i.e. each transcript will be assigned to each cell with probability 1/N. Conversely, if the population is heterogeneous with respect to expression of g (that is, it consists of a mix of cell types, each expressing the gene differently), then transcripts will not be assigned uniformly, but rather will be assigned preferentially to distinct subsets of cells. Heterogeneity in experimentally observed patterns of expression can, therefore, be assessed in terms of deviation from this hypothetical homogeneous null model.
                             </p>
                             <p id="p-8">
                              The Kullback-Leibler divergence (KLD, also known as the relative entropy) is a measure of the information encoded in the difference between an observed and null distribution (
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               Kullback & Leibler 1951
                              </a>
                              ). In the sc-Seq context, the KLD of an experimentally observed gene expression distribution from the homogeneous null model, described above, measures the amount of information that is lost by assuming that the gene is homogeneously expressed in the sequenced cell population. We will denote this quantity I(g) and refer to it as the heterogeneity of g (see
                              <strong>
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Fig. 1
                               </a>
                              </strong>
                              for a schematic). Formally,
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where H(g) is the entropy of the expression of g in the population (see
                              <strong>
                               Methods
                              </strong>
                              for full details). For technical reasons, explained in the
                              <strong>
                               Methods
                              </strong>
                              , we will use natural logarithms in all calculations; I(g) therefore has units of nats.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F1.large.jpg?width=800&height=600&carousel=1" title="An information-theoretic view of sc-Seq data. Transcripts, or more generally counts, of a given gene are assigned to cells after sequencing. If the cell population is homogeneous with respect to the expression of g, then I(g) will be zero (top left population). In practice, the transcript assignment process is stochastic, and so there will always be some deviation from this ideal (bottom left population). The technical effects of this stochasticity on the information obtained may be reduced by using a shrinkage estimator to estimate the distribution of transcripts (see Methods). If the population is heterogeneous, then transcripts are expressed preferentially in a subset of cells and the information obtained from the experiment will be larger (top right population), reaching a maximum at log(N), where N is the number of cells sequenced, when only one cell expresses the gene (bottom right population, I(g) = ln(5) ≈ 1.61)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1811662779" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">An information-theoretic view of sc-Seq data.</span> Transcripts, or more generally counts, of a given gene are assigned to cells after sequencing. If the cell population is homogeneous with respect to the expression of <em>g</em>, then <em>I</em>(<em>g</em>) will be zero (top left population). In practice, the transcript assignment process is stochastic, and so there will always be some deviation from this ideal (bottom left population). The technical effects of this stochasticity on the information obtained may be reduced by using a shrinkage estimator to estimate the distribution of transcripts (see <strong>Methods</strong>). If the population is heterogeneous, then transcripts are expressed preferentially in a subset of cells and the information obtained from the experiment will be larger (top right population), reaching a maximum at log(<em>N</em>), where <em>N</em> is the number of cells sequenced, when only one cell expresses the gene (bottom right population, <em>I</em>(<em>g</em>) = ln(5) ≈ 1.61).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F1.medium.gif" width="440" height="201"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F1.medium.gif" width="440" height="201"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                An information-theoretic view of sc-Seq data.
                               </span>
                               <p id="p-9" class="first-child">
                                Transcripts, or more generally counts, of a given gene are assigned to cells after sequencing. If the cell population is homogeneous with respect to the expression of g, then I(g) will be zero (top left population). In practice, the transcript assignment process is stochastic, and so there will always be some deviation from this ideal (bottom left population). The technical effects of this stochasticity on the information obtained may be reduced by using a shrinkage estimator to estimate the distribution of transcripts (see
                                <strong>
                                 Methods
                                </strong>
                                ). If the population is heterogeneous, then transcripts are expressed preferentially in a subset of cells and the information obtained from the experiment will be larger (top right population), reaching a maximum at log(N), where N is the number of cells sequenced, when only one cell expresses the gene (bottom right population, I(g) = ln(5) ≈ 1.61).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-10">
                              Intuitively, if the cell population is unstructured with respect to the expression of g (i.e. if the cells are approximately interchangeable with respect to the expression of g) then the assumption of homogeneity is correct and I(g) ≈ 0. Conversely, the theoretical maximum for I(g) is log(N), which is achieved when H(g) = 0 and all transcripts of the gene are assigned to the same cell (see
                              <strong>
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Fig. 1
                               </a>
                              </strong>
                              ). Note that: (1) we do not need to know, or model, the particular expression distribution of g in the population, so no a priori assumptions about expression patterns are required to calculate I(g); (2) I(g) is agnostic concerning missing readings or counts so long as they are distributed uniformly at random; (3) since it quantifies the deviation from the homogeneous null model, I(g) measures the information obtained from the experiment concerning the expression of g.
                             </p>
                             <p id="p-11">
                              In practice, I(g) is associated with cellular diversity: the more distinct cell sub-populations present in a sample, and the more those sub-populations differ from one another with respect to their expression of g, the larger I(g) will be. Thus, I(g) is a parsimonious measure of expression heterogeneity that makes minimal assumptions concerning expression patterns and, therefore, imposes minimal technical requirements on data collection methodology or quality. As such, it can be used as the basis for numerous aspects of the sc-Seq pipeline, including feature selection, differential expression analysis, and unsupervised clustering.
                             </p>
                             <p id="p-12">
                              To validate these uses, we considered a series of single-cell RNA-sequencing bench-marking data sets: Svensson, a technical control (
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               Svensson et al. 2017
                              </a>
                              ); Tian, a mixture of three cancerous cell lines (
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               Tian et al. 2019
                              </a>
                              ); Zheng, a set of FACS separated peripheral blood mononuclear cells (PMBCs); Stumpf, a sampling of cells from mouse bone marrow (
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               Stumpf et al. 2020
                              </a>
                              ); and the Tabula Muris, a mouse cell atlas with cells from twelve organs (
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               Tabula Muris Consortium 2018
                              </a>
                              ).
                              <strong>
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Fig. 2a-e
                               </a>
                              </strong>
                              shows plots of I(g) for each gene profiled in these experiments. These plots confirm the intuition that the more distinct types present in a sample, and the more those cell types differ from one another with respect to their gene expression patterns, the greater the population heterogeneity will be, as measured by I(g). These results indicate that heterogeneity (as defined, here) is a simple measure of quality for technical controls, which can be used to select potentially informative features for further study. Moreover, it may be easily modified to account for the presence of multiple homogeneous cell sub-populations, and thereby act as a measure of cluster quality.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F2.large.jpg?width=800&height=600&carousel=1" title="Information-theoretic single-cell analysis. a-e) Plots of expression heterogeneity, I(g) (normalised by the theoretical maximum, log(N), where N is the number of cells) against log mean expression for the bench-marking sc-Seq data sets described in the main text. In each panel, each point represents a gene profiled. The number of genes associated with large values of I(g) increases with the number of cell types present in the population profiled. Panel a) shows data from a technical control (Svensson et al. 2017) (number of cell types, C = 1), b) a mixture of three cancerous cell lines (Tian et al. 2019) (C = 3), c) FACS sorted immune cells (Zheng et al. 2017) (C = 4), d) a sample of mouse bone marrow (Stumpf et al. 2020) (C = 14), and e) a multi-organ mouse cell atlas (Tabula Muris Consortium 2018) (C = 56). f-h) Biologically meaningful cell annotations are associated with high inter-cluster heterogeneity. Established cell annotations for the f) Tian, g) Zheng and h) Stumpf data are associated with higher inter-cluster heterogeneity than expected by chance (i.e., in randomly permuted clusters; significance is assessed using a one-sided exact test with 104 permutations; y axes show log10(p + 1)). In all panels the red line shows p " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1811662779" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Information-theoretic single-cell analysis.</span> <strong>a-e)</strong> Plots of expression heterogeneity, <em>I</em>(<em>g</em>) (normalised by the theoretical maximum, log(<em>N</em>), where <em>N</em> is the number of cells) against log mean expression for the bench-marking sc-Seq data sets described in the main text. In each panel, each point represents a gene profiled. The number of genes associated with large values of <em>I</em>(<em>g</em>) increases with the number of cell types present in the population profiled. Panel <strong>a)</strong> shows data from a technical control (Svensson et al. 2017) (number of cell types, <em>C</em> = 1), <strong>b)</strong> a mixture of three cancerous cell lines (Tian et al. 2019) (<em>C</em> = 3), <strong>c)</strong> FACS sorted immune cells (Zheng et al. 2017) (<em>C</em> = 4), <strong>d)</strong> a sample of mouse bone marrow (Stumpf et al. 2020) (<em>C</em> = 14), and <strong>e)</strong> a multi-organ mouse cell atlas (Tabula Muris Consortium 2018) (<em>C</em> = 56). <strong>f-h)</strong> Biologically meaningful cell annotations are associated with high inter-cluster heterogeneity. Established cell annotations for the <strong>f)</strong> <em>Tian</em>, <strong>g)</strong> <em>Zheng</em> and <strong>h)</strong> <em>Stumpf</em> data are associated with higher inter-cluster heterogeneity than expected by chance (i.e., in randomly permuted clusters; significance is assessed using a one-sided exact test with 10<sup>4</sup> permutations; y axes show log<sub>10</sub>(<em>p</em> + 1)). In all panels the red line shows <em>p <</em> 0.05, false discovery rate corrected for 500 trials (Benjamini & Hochberg 1995, Fisher 2017). Genes below this threshold are significantly differentially expressed between cell types as measured by <em>H</em><sub><em>S</em></sub>(<em>g</em>). <strong>i)</strong> Summary statistics for the total inter-cluster heterogeneity <em>H</em><sub><em>S</em></sub> = ∑<sub><em>g</em></sub><em>H</em><sub><em>S</em></sub>(<em>g</em>) based on established empirical and randomly permuted cell annotations (10<sup>4</sup> random permutations in each case). <strong>j)</strong> A Uniform Manifold Approximation and Projection (UMAP) (McInnes et al. 2018) plot of the top 500 genes by <em>I</em>(<em>g</em>) for the <em>Stumpf</em> data set. Each point is a cell, coloured by its scEC cluster.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F2.medium.gif" width="440" height="270"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F2.medium.gif" width="440" height="270"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Information-theoretic single-cell analysis.
                               </span>
                               <p id="p-13" class="first-child">
                                <strong>
                                 a-e)
                                </strong>
                                Plots of expression heterogeneity, I(g) (normalised by the theoretical maximum, log(N), where N is the number of cells) against log mean expression for the bench-marking sc-Seq data sets described in the main text. In each panel, each point represents a gene profiled. The number of genes associated with large values of I(g) increases with the number of cell types present in the population profiled. Panel
                                <strong>
                                 a)
                                </strong>
                                shows data from a technical control (
                                <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                                 Svensson et al. 2017
                                </a>
                                ) (number of cell types, C = 1),
                                <strong>
                                 b)
                                </strong>
                                a mixture of three cancerous cell lines (
                                <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                                 Tian et al. 2019
                                </a>
                                ) (C = 3),
                                <strong>
                                 c)
                                </strong>
                                FACS sorted immune cells (
                                <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                 Zheng et al. 2017
                                </a>
                                ) (C = 4),
                                <strong>
                                 d)
                                </strong>
                                a sample of mouse bone marrow (
                                <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                                 Stumpf et al. 2020
                                </a>
                                ) (C = 14), and
                                <strong>
                                 e)
                                </strong>
                                a multi-organ mouse cell atlas (
                                <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">
                                 Tabula Muris Consortium 2018
                                </a>
                                ) (C = 56).
                                <strong>
                                 f-h)
                                </strong>
                                Biologically meaningful cell annotations are associated with high inter-cluster heterogeneity. Established cell annotations for the
                                <strong>
                                 f)
                                </strong>
                                Tian,
                                <strong>
                                 g)
                                </strong>
                                Zheng and
                                <strong>
                                 h)
                                </strong>
                                Stumpf data are associated with higher inter-cluster heterogeneity than expected by chance (i.e., in randomly permuted clusters; significance is assessed using a one-sided exact test with 10
                                <sup>
                                 4
                                </sup>
                                permutations; y axes show log
                                <sub>
                                 10
                                </sub>
                                (p + 1)). In all panels the red line shows p < 0.05, false discovery rate corrected for 500 trials (
                                <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                                 Benjamini & Hochberg 1995
                                </a>
                                ,
                                <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                                 Fisher 2017
                                </a>
                                ). Genes below this threshold are significantly differentially expressed between cell types as measured by H
                                <sub>
                                 S
                                </sub>
                                (g).
                                <strong>
                                 i)
                                </strong>
                                Summary statistics for the total inter-cluster heterogeneity H
                                <sub>
                                 S
                                </sub>
                                = ∑
                                <sub>
                                 g
                                </sub>
                                H
                                <sub>
                                 S
                                </sub>
                                (g) based on established empirical and randomly permuted cell annotations (10
                                <sup>
                                 4
                                </sup>
                                random permutations in each case).
                                <strong>
                                 j)
                                </strong>
                                A Uniform Manifold Approximation and Projection (UMAP) (
                                <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                                 McInnes et al. 2018
                                </a>
                                ) plot of the top 500 genes by I(g) for the Stumpf data set. Each point is a cell, coloured by its scEC cluster.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Information-theoretic clustering
                             </h3>
                             <p id="p-14">
                              Let S be a discrete clustering of a population of cells – i.e. an assignment of the cells to a finite set of C non-intersecting sub-populations (also known as clusters). In the
                              <strong>
                               Methods
                              </strong>
                              , we show that I(g) can be decomposed into two parts: one part that quantifies the extent to which each of the sub-populations defined by S deviate from homogeneity, which we call the intra-cluster heterogeneity and write as h
                              <sub>
                               S
                              </sub>
                              (g); and one part that quantifies how differently the gene is expressed, in aggregate, between sub-populations, which we call the inter-cluster heterogeneity and write as H
                              <sub>
                               S
                              </sub>
                              (g). Namely, we show that
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for any proposed clustering S. Thus, the information obtained from an experiment concerning the expression of a gene g can be explicitly related to both local (within cluster) and global (between cluster) patterns of variation, for any proposed clustering. Full mathematical details of this decomposition, including formulae for H
                              <sub>
                               S
                              </sub>
                              (g) and h
                              <sub>
                               S
                              </sub>
                              (g), are provided in the
                              <strong>
                               Methods
                              </strong>
                              , and an illustration is given in
                              <strong>
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                              </strong>
                              . We also provide an
                              <kbd>
                               R
                              </kbd>
                              package for its calculation (see
                              <strong>
                               Methods
                              </strong>
                              ).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F3.large.jpg?width=800&height=600&carousel=1" title="Heterogeneity is additively decomposable. The heterogeneity of a population of cells with respect to the expression of a gene g can be decomposed into inter- and intra-cluster heterogeneities for any proposed clustering, S. The inter-cluster heterogeneity HS(g) is determined by independently aggregating all transcripts associated with each sub-population in S and then taking the KLD of the resulting distribution from the uniform distribution on C letters. It measures the extent to which transcripts are uniformly assigned to clusters. The intra-cluster heterogeneity hS(g) is determined by taking the weighted sum of the heterogeneities of each of the constituent sub-populations, considered independently. It represents the average heterogeneity of the proposed clusters, accounting for disparities in cluster sizes. Further details and formulae are provided in the Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1811662779" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Heterogeneity is additively decomposable.</span> The heterogeneity of a population of cells with respect to the expression of a gene <em>g</em> can be decomposed into inter- and intra-cluster heterogeneities for any proposed clustering, <em>S</em>. The inter-cluster heterogeneity <em>H</em><sub><em>S</em></sub>(<em>g</em>) is determined by independently aggregating all transcripts associated with each sub-population in <em>S</em> and then taking the KLD of the resulting distribution from the uniform distribution on <em>C</em> letters. It measures the extent to which transcripts are uniformly assigned to clusters. The intra-cluster heterogeneity <em>h</em><sub><em>S</em></sub>(<em>g</em>) is determined by taking the weighted sum of the heterogeneities of each of the constituent sub-populations, considered independently. It represents the average heterogeneity of the proposed clusters, accounting for disparities in cluster sizes. Further details and formulae are provided in the <strong>Methods</strong>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F3.medium.gif" width="440" height="145"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F3.medium.gif" width="440" height="145"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Heterogeneity is additively decomposable.
                               </span>
                               <p id="p-15" class="first-child">
                                The heterogeneity of a population of cells with respect to the expression of a gene g can be decomposed into inter- and intra-cluster heterogeneities for any proposed clustering, S. The inter-cluster heterogeneity H
                                <sub>
                                 S
                                </sub>
                                (g) is determined by independently aggregating all transcripts associated with each sub-population in S and then taking the KLD of the resulting distribution from the uniform distribution on C letters. It measures the extent to which transcripts are uniformly assigned to clusters. The intra-cluster heterogeneity h
                                <sub>
                                 S
                                </sub>
                                (g) is determined by taking the weighted sum of the heterogeneities of each of the constituent sub-populations, considered independently. It represents the average heterogeneity of the proposed clusters, accounting for disparities in cluster sizes. Further details and formulae are provided in the
                                <strong>
                                 Methods
                                </strong>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-16">
                              This decomposition can be used to assess cluster quality. For a proposed cluster to meaningfully represent a cell type, it must be associated with differential expression of some subset of marker genes. Because H
                              <sub>
                               S
                              </sub>
                              (g) measures the extent to which the expression of gene g deviates from the homogeneous null model it is a simple measure of the extent to which g is differentially expressed in the clusters defined by S. We therefore expect that biologically meaningful clustering based on a marker gene g will result in a high value for H
                              <sub>
                               S
                              </sub>
                              (g) and (by necessity) a low value for h
                              <sub>
                               S
                              </sub>
                              (g).
                             </p>
                             <p id="p-17">
                              This expectation is confirmed for the Tian, Zheng and Stumpf data sets for which we possess high-confidence a priori cell type annotations, derived either experimentally (for the Tian and Zheng data) or from expert annotation of a computational clustering (for the Stumpf data). In the set of genes most likely to be differentially expressed in these data sets (taken to be the top 500 genes by I(g) in each case) the majority are associated with substantially higher inter-cluster heterogeneity than expected by chance (i.e., in randomly permuted clusters). Indeed, in Tian 97.0%, in Zheng 94.4% and in Stumpf 99.6% of the tested genes had significantly higher inter-cluster heterogeneity than expected (one-sided exact test with 10
                              <sup>
                               4
                              </sup>
                              permutations, p < 0.05, false discovery rate corrected for 500 trials; see
                              <strong>
                               <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                                Fig. 2f-h
                               </a>
                              </strong>
                              for illustration) (
                              <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                               Benjamini & Hochberg 1995
                              </a>
                              ,
                              <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                               Fisher 2017
                              </a>
                              ).
                             </p>
                             <p id="p-18">
                              These results indicate that by simple information-theoretic reasoning, we are able to quantify the amount of information in the expression pattern of a single gene that is explained by a given clustering. However, a key strength of high-throughput single-cell profiling methods is that they allow the simultaneous profiling of thousands of genes in large cell populations.
                             </p>
                             <p id="p-19">
                              Our information-theoretic reasoning may be extended to a high-throughput single-cell profiling experiment by assuming that each gene is an independent source of information and making use of the fact that information from independent sources is additive (
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               Shannon 1948
                              </a>
                              ). Thus, we can determine the total information explained by a given clustering S by evaluating the sum H
                              <sub>
                               S
                              </sub>
                              = ∑
                              <sub>
                               g
                              </sub>
                              H
                              <sub>
                               S
                              </sub>
                              (g) over all genes profiled. The total information explained by S is a simple, easily computed measure for cluster evaluation that favours grouping of cells into homogeneous sub-populations and is maximised at ∑
                              <sub>
                               g
                              </sub>
                              I(g) if and only if the proposed clustering divides the population into indivisible homogeneous sub-populations and thereby accounts for all the heterogeneity contained in the sc-Seq data set. If so, then S is the maximum entropy partition of the cell population into distinct classes, and may therefore be considered as the most parsimonious way of assigning cell identities.
                             </p>
                             <p id="p-20">
                              To illustrate the association between total information explained, H
                              <sub>
                               S
                              </sub>
                              , and cluster quality, we again considered the established annotations of the Tian, Zheng and Stumpf data sets. As expected, in all cases the observed value of H
                              <sub>
                               S
                              </sub>
                              significantly exceeds that of all random label permutations (see
                              <strong>
                               <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                Fig. 2i
                               </a>
                              </strong>
                              ) indicating that H
                              <sub>
                               S
                              </sub>
                              is strongly associated with cluster quality and that these benchmark cell annotations are associated with homogeneous cell sub-populations. Notably, this association holds true independently of the method used to annotate cell identities (annotations were derived from genotypic information for the Tian data, surface protein expression for the Zheng data, and unsupervised clustering for the Stumpf data), indicating that H
                              <sub>
                               S
                              </sub>
                              provides a methodology agnostic, parameter-free, means to quantify cluster quality.
                             </p>
                             <p id="p-21">
                              Building on this empirical association, we reasoned that the most accurate cell type annotation processes should maximise H
                              <sub>
                               S
                              </sub>
                              for a given cell population. Based on this assumption, we established an automated process for cell type annotation from sc-Seq data that assigns each cell to one of C non-intersecting discrete identities such that H
                              <sub>
                               S
                              </sub>
                              is maximised. This optimisation constitutes a novel unsupervised clustering method developed explicitly for sc-Seq data, based on a biologically-motivated notion of expression heterogeneity. We refer to this method as scEC (single-cell Entropic Clustering).
                             </p>
                             <p id="p-22">
                              To determine the efficacy of scEC we assessed the similarity of the clustering returned by scEC and the established annotations for the Tian, Zheng and Stumpf data sets. The resulting scEC clusterings agreed strongly with the established cell annotations, achieving adjusted Rand Indices (ARI) of 0.99 for the Tian annotations, 0.87 for the Zheng annotations, and 0.69 for the Stumpf annotations (the adjusted Rand index is a measure of similarity of a proposed clustering to a known clustering,
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               Rand (1971)
                              </a>
                              ). To benchmark these results we also repeated the analysis using the Louvain method, a leading single-cell clustering algorithm, which achieved ARIs of 0.99, 0.99 and 0.35 for the Tian, Zheng, and Stumpf data sets respectively (
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               Blondel et al. 2008
                              </a>
                              ,
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               Luecken & Theis 2019
                              </a>
                              ,
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               Freytag et al. 2018
                              </a>
                              ,
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               Stuart et al. 2019
                              </a>
                              ).
                             </p>
                             <p id="p-23">
                              These results indicate that scEC is comparable in performance to the state-of-the-art in unsupervised single cell clustering. However, scEC has two additional advantages. First, unlike most alternatives, scEC is not a purely statistical process, but rather is anchored on a biologically-motivated definition of expression heterogeneity, and is specifically tailored to deciphering cell types from single cell expression data, rather than clusters in data more generally. Consequently, scEC has no free parameters, and can be implemented in a simple, robust way. Second, because it is based on a biologically-motivated definition of heterogeneity scEC is biologically intuitive and is, consequently, easily interpretable. For instance, because the scEC objective function, H
                              <sub>
                               S
                              </sub>
                              , is naturally built by summation of gene-level measurements, it can be used to directly identify the key drivers of a given clustering without relying on post-hoc differential gene expression testing.
                             </p>
                             <p id="p-24">
                              To demonstrate this feature, we compared the inter-cluster heterogeneity values (i.e., the values H
                              <sub>
                               S
                              </sub>
                              (g) for each of the genes profiled) for clusters identified by scEC and established annotations for the Tian & Stumpf data sets (see
                              <strong>
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Fig. 4
                               </a>
                              </strong>
                              for an illustration).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F4.large.jpg?width=800&height=600&carousel=1" title="Comparison of inter-cluster heterogeneity of scEC-generated clusters and established annotations. Plots of HS(g) based on an scEC-generated clustering (x-axis) and established annotations (y-axis) for the a) Zheng and b) Stumpf data sets (Zheng et al. 2017, Stumpf et al. 2020). In both panels each point represents a gene profiled and the red line indicates HscEC(g) = HKnown(g). Genes below the red line are better explained by the scEC clustering, while those above the line are better explained by the prior annotation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1811662779" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparison of inter-cluster heterogeneity of scEC-generated clusters and established annotations.</span> Plots of <em>H</em><sub><em>S</em></sub>(<em>g</em>) based on an scEC-generated clustering (<em>x</em>-axis) and established annotations (<em>y</em>-axis) for the <strong>a)</strong> <em>Zheng</em> and <strong>b)</strong> <em>Stumpf</em> data sets (Zheng et al. 2017, Stumpf et al. 2020). In both panels each point represents a gene profiled and the red line indicates <em>H</em><sub><em>scEC</em></sub>(<em>g</em>) = <em>H</em><sub><em>Known</em></sub>(<em>g</em>). Genes below the red line are better explained by the scEC clustering, while those above the line are better explained by the prior annotation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F4.medium.gif" width="440" height="239"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F4.medium.gif" width="440" height="239"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Comparison of inter-cluster heterogeneity of scEC-generated clusters and established annotations.
                               </span>
                               <p id="p-25" class="first-child">
                                Plots of H
                                <sub>
                                 S
                                </sub>
                                (g) based on an scEC-generated clustering (x-axis) and established annotations (y-axis) for the
                                <strong>
                                 a)
                                </strong>
                                Zheng and
                                <strong>
                                 b)
                                </strong>
                                Stumpf data sets (
                                <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                                 Zheng et al. 2017
                                </a>
                                ,
                                <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">
                                 Stumpf et al. 2020
                                </a>
                                ). In both panels each point represents a gene profiled and the red line indicates H
                                <sub>
                                 scEC
                                </sub>
                                (g) = H
                                <sub>
                                 Known
                                </sub>
                                (g). Genes below the red line are better explained by the scEC clustering, while those above the line are better explained by the prior annotation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-26">
                              We found that the values of H
                              <sub>
                               S
                              </sub>
                              (g) associated with scEC clusters was strongly positively correlated with those associated with established annotations for both the Zheng and Stumpf data sets (Pearson’s correlation coefficient of 0.94 for Zheng and 0.99 for Stumpf ; see
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               <strong>
                                Fig
                               </strong>
                               . 4
                              </a>
                              ). Moreover, the values associated with scEC clusters for the key marker genes used in each data set (Zheng : CD14, CD4, CD8 and NCAM1,
                              <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">
                               Zheng et al. (2017)
                              </a>
                              ; Stumpf : Cd34, Kitl, Spi1, Gata1 and Pax5,
                              <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">
                               Stumpf et al. (2020)
                              </a>
                              ) were significantly higher than expected by chance (one-sided exact test, 10
                              <sup>
                               4
                              </sup>
                              permutations, p < 0.05, false discovery rate corrected).
                             </p>
                             <p id="p-27">
                              This concordance indicates that scEC is generally able to recapitulate known cell clusters in an unbiased and parameter free way. However, despite this concordance, there were differences between scEC clusters and established annotations. To investigate these differences further we selected those genes for which the value of H
                              <sub>
                               S
                              </sub>
                              (g) differed substantially between scEC clusters and established annotations in each data set for further investigation. To illustrate we chose those genes for which ΔH
                              <sub>
                               S
                              </sub>
                              (g) = |H
                              <sub>
                               scEC
                              </sub>
                              (g) − H
                              <sub>
                               Known
                              </sub>
                              (g)| > 0.1 nats.
                             </p>
                             <p id="p-28">
                              In the Zheng data, we found that those genes with expression heterogeneity better explained by the scEC clustering were enriched for cell-cycle related genes (gene ontology enrichment, p = 1.33 × 10
                              <sup>
                               −8
                              </sup>
                              , false discovery rate corrected). Conversely, those genes with expression heterogeneity better explained by the established annotation were enriched for genes involved in the immune response (p = 1.49 × 10
                              <sup>
                               −3
                              </sup>
                              , false discovery rate corrected) (
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               Ashburner et al. 2000
                              </a>
                              ,
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               Gene Ontology Consortium 2021
                              </a>
                              ,
                              <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                               Benjamini & Hochberg 1995
                              </a>
                              ).
                             </p>
                             <p id="p-29">
                              In the Stumpf data set, we found that those genes with expression heterogeneity better explained by the scEC clustering were enriched for genes involved in erythrocyte differentiation and homeostasis (p = 1.80 × 10
                              <sup>
                               −2
                              </sup>
                              and p-value = 6.17 × 10
                              <sup>
                               −4
                              </sup>
                              respectively, false discovery rate corrected). Conversely, those genes with expression heterogeneity better explained by the established annotation were enriched for genes involved in the immune response (p = 2.57 × 10
                              <sup>
                               −18
                              </sup>
                              , false discovery rate corrected). In this example, scEC generally identifies additional erythrocyte sub-populations while merging the cell sub-types of the neutrophil lineage (see
                              <strong>
                               <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                                Fig. 2j
                               </a>
                              </strong>
                              for illustration and
                              <strong>
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Supplementary Table
                               </a>
                              </strong>
                              for contingency table of cluster assignments).
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2440485/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2440485/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2440485" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Supplementary Table
                               </span>
                               <p id="p-98" class="first-child">
                                Contingency table between cell annotations provided in (
                                <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">
                                 Stumpf et al. 2020
                                </a>
                                ) and scEC labels. Note that while the number of clusters for scEC was fixed at 14, only 13 clusters were realised; generally, the number of clusters returned will match the number specified, except in cases where any further splits in clusters provide only a minor increase in inter-cluster heterogeneity, so are harder to find by numerical optimisation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              Collectively, these differences reflect the preference of scEC for cellular annotations driven by biological processes involving larger tranches of genes. Depending on circumstance, this preference may be a benefit or a drawback. It is generally beneficial because it ensures that scEC annotations are robust to outliers (i.e. anomalous expression patterns of individual genes cannot easily distort scEC cluster assignments). Conversely, in circumstances in which key cellular functions are determined by a small number of genes, scEC may fail to disambiguate important cell populations from similar cell types. However, because of the simplicity of the scEC methodology this issue can this issue can be relatively easily addressed. In the standard formulation of scEC each gene contributes equally to the objective function H
                              <sub>
                               S
                              </sub>
                              = ∑
                              <sub>
                               g
                              </sub>
                              H
                              <sub>
                               S
                              </sub>
                              (g). This corresponds to assuming a uniform prior. However, alternatives may be easily considered that take into account different sources of prior knowledge. In general, each gene can contribute in a weighted way to the overall inter-cluster heterogeneity. In this case the objective becomes H
                              <sub>
                               S
                              </sub>
                              = ∑
                              <sub>
                               g
                              </sub>
                              w(g)H
                              <sub>
                               S
                              </sub>
                              (g), where w(g) is the weight of gene g. These weights can encode known biology – for example, by up-weighting known drivers of cellular identities, down-weighting genes associated with housekeeping processes and/or the cell cycle, or by weighting genes in inverse proportion to the size (i.e. the number of associated genes) of their leading ontology term.
                             </p>
                             <p id="p-31">
                              While incorporating such priors into scEC may be beneficial, defining them is a challenge and potentially introduces a source of bias into the resulting clustering and so should be approached with care. An alternative approach – which encodes prior information in a data-led, rather than algorithmic way – is to use a reference data set, such as a cell atlas, to benchmark cell annotations. Doing so leads to a semi-supervised version of scEC, which we explain below.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Semi-supervised cell classification
                             </h3>
                             <p id="p-32">
                              Taken together, these results indicate that our information-theoretic clustering method can be used to successfully identify clusters, corresponding to distinct homogeneous cell sub-populations, from sc-Seq data. However, scEC can be insensitive to important cellular processes driven by small numbers of genes. Moreover, annotating the recovered clusters (i.e. assigning biologically meaningful identities, for example based on marker gene expression patterns) still requires time-consuming manual inspection and differential gene expression analysis. We therefore sought to develop a semi-supervised version of scEC that combines clustering and annotation, by comparing experimentally derived cell profiles with those in a relevant reference database, such as a cell atlas.
                             </p>
                             <p id="p-33">
                              Assuming that individual cells in the reference data set have been clustered and correctly and unambiguously annotated, the semi-supervised implementation of scEC assigns cells of a novel, test data set, to the pre-identified clusters of the reference data set such that the total inter-cluster heterogeneity H
                              <sub>
                               S
                              </sub>
                              of the combined data set is maximised. The cells of the test data set then inherit the annotation of their assigned reference cluster, thereby assigning a biological identity without laborious manual inspection. A full mathematical description of this semi-supervised classification process is provided in the
                              <strong>
                               Methods
                              </strong>
                              .
                             </p>
                             <p id="p-34">
                              To illustrate this method, we again considered the Tian data set, which consists of sc-Seq data obtained from a mixture of cells drawn from three distinct cancerous cell lines (known as H1975, H2228 and HCC82). In addition to this data set, Tian and co-workers also generated a separate data set consisting of sc-Seq data obtained from a mixture of cells drawn from the original three cell lines and from a further two cancerous cell lines (known as A549 and H838) (
                              <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">
                               Tian et al. 2019
                              </a>
                              ). Since true cell annotations obtained by genotyping are available in both cases, we reasoned that these two data sets would collectively provide a good benchmark to assess the accuracy of our semi-supervised methodology. To test this possibility we used the Tian5 data set as a reference against which to compare the cell profiles in the Tian data.
                             </p>
                             <p id="p-35">
                              In this example, the semi-supervised cell annotation process proceeds in two steps. First the Tian data Tian5 data are concatenated to create a merged data set, and each cell of the reference Tian5 data set is assigned its known cellular identity, while each cell of the test Tian data set is assigned a fuzzy cluster identity, with uniform weighting across the set of five reference clusters (i.e. each cell i in the test set is assigned to cluster j with probability p
                              <sub>
                               ij
                              </sub>
                              , where, for each i elements of p
                              <sub>
                               ij
                              </sub>
                              are chosen randomly and uniformly). Second, the total inter-cluster heterogeneity H
                              <sub>
                               S
                              </sub>
                              = ∑
                              <sub>
                               g
                              </sub>
                              H
                              <sub>
                               S
                              </sub>
                              (g) of the merged data set is maximised, allowing the annotations of the cells in the test data set of vary, while keeping the identities of the reference cells fixed (details of the optimisation routine are provided in the
                              <strong>
                               Methods
                              </strong>
                              ). This semi-supervised assignment process thereby automatically annotates each cell of the test data set using the reference labels.
                             </p>
                             <p id="p-36">
                              The results of this semi-supervised assignment process are given in
                              <strong>
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Fig. 5
                               </a>
                              </strong>
                              . Cellular annotations obtained by comparing the Tian data with the reference Tian5 substantially agreed with their genotype annotations (ARI of 0.90), indicating that identities of cells in the Tian data were robustly recovered by our semi-supervised process. Notably, while it is possible for cells in the test data to be mis-assigned to the A549 and H838 identities – which are present in the Tian5 reference data but not in the Tian test data – this occurred only very rarely (only 6% of cells were mis-assigned in this way) despite considerable similarities in these cell lines. This example illustrates that our information-theoretic semi-supervised cell annotation process is able to reliably identify cell types in heterogeneous samples, even when differences between cell types are subtle.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F5.large.jpg?width=800&height=600&carousel=1" title="Semi-supervised classification of cancer cell lines. Confusion matrix for semi-supervised classification of cells in the Tian data, using the Tian5 data as a reference (Tian et al. 2019). Columns represent the established (i.e., true) experimentally-derived identities (H1975, H2228 and HCC82) while rows give predicted cell identities derived from the reference Tian5 data set (H1975, H2228, HCC82, A549 and H838), based on our semi-supervised process. In this case, the adjusted Rand index between the semi-supervised classification and true identities is 0.90." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1811662779" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Semi-supervised classification of cancer cell lines.</span> Confusion matrix for semi-supervised classification of cells in the <em>Tian</em> data, using the <em>Tian5</em> data as a reference (Tian et al. 2019). Columns represent the established (i.e., true) experimentally-derived identities (H1975, H2228 and HCC82) while rows give predicted cell identities derived from the reference <em>Tian5</em> data set (H1975, H2228, HCC82, A549 and H838), based on our semi-supervised process. In this case, the adjusted Rand index between the semi-supervised classification and true identities is 0.90.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F5.medium.gif" width="440" height="240"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F5.medium.gif" width="440" height="240"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/08/2020.10.01.322255/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Semi-supervised classification of cancer cell lines.
                               </span>
                               <p id="p-37" class="first-child">
                                Confusion matrix for semi-supervised classification of cells in the Tian data, using the Tian5 data as a reference (
                                <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">
                                 Tian et al. 2019
                                </a>
                                ). Columns represent the established (i.e., true) experimentally-derived identities (H1975, H2228 and HCC82) while rows give predicted cell identities derived from the reference Tian5 data set (H1975, H2228, HCC82, A549 and H838), based on our semi-supervised process. In this case, the adjusted Rand index between the semi-supervised classification and true identities is 0.90.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-6">
                            <h2 class="">
                             Conclusion
                            </h2>
                            <p id="p-38">
                             Traditional unsupervised clustering methods for single cell data are based on the, often implicit, assumption that cell types correspond to regions of high probability density in the joint gene expression distribution (
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              Greulich et al. 2020
                             </a>
                             ). Although useful, this is not a specifically biological assumption (similar assumptions form the basis of numerous clustering algorithms in other different disciples) and does not have a clear biological basis. Indeed, this assumption does not naturally accord with biological intuition that distinct cell populations should be ‘homogeneous’ and cells of the same type should be functionally interchangeable. Here, we have taken a different approach that seeks to encode biological intuition in a mathematical formulation, drawing on tools from information theory. We have proposed a formal measure of expression heterogeneity and have used this measure to establish a biologically-motivated clustering procedure, which we call single cell entropic clustering (scEC), that is specifically tailored for single-cell expression data.
                            </p>
                            <p id="p-39">
                             While successful and well-established clustering methods, such as the popular Louvain method, may be more general, scEC is more biologically intuitive and has an interpretation that will be familiar to many experimentalists. Importantly, the performance of scEC is comparable to a state-of-the-art on a series of benchmark sc-Seq data sets, suggesting that this underlying biological basis is justified. With sc-Seq data sets continually growing in size and complexity, there is a need for biologically grounded – rather than purely statistical – approaches to understanding them. scEC provides an efficient, parameter-free method for doing this. An
                             <kbd>
                              R
                             </kbd>
                             package for the implementation of scEC available at
                             <a href="https://github.com/mjcasy/scEC">
                              https://github.com/mjcasy/scEC
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-7">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Data collection
                             </h3>
                             <p id="p-40">
                              Count matrices for each data set were downloaded from their respective repositories (see
                              <strong>
                               Availability of Data and Materials
                              </strong>
                              ). For the Zheng and Tabula Muris data sets, the count matrices of the various cell types/tissues were concatenated using the
                              <kbd>
                               Matrix
                              </kbd>
                              (v1.3-2) package in
                              <kbd>
                               R
                              </kbd>
                              (v4.0.3) (
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               R Core Team 2020
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Data pre-processing
                             </h3>
                             <p id="p-41">
                              For each data set, those genes with less than 100 total transcripts in total (i.e. across all cells) were excluded from further analysis. For the Zheng data set, FACS identities were taken from the repository metadata, with the CD4+ Helper T-cells, CD4+/CD25+ Regulatory T-cells, CD4+/CD45RA+/CD25-Naïve T cells, CD4+/CD45RO+ Memory T-cells, CD8+ Cytotoxic T cells and CD8+/CD45RA+ Naïve Cytotoxic T-cells merged into the single identity of T-cells.
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Normalisation
                             </h3>
                             <p id="p-42">
                              Since sc-Seq data is noisy, rather than working with the experimentally observed proportions of transcripts assigned to each cell, we instead adopt a Bayesian approach to estimate the cellular frequencies using the James-Stein estimator, as explained below, and use this estimator in all subsequent calculations.
                             </p>
                             <p id="p-43">
                              We write
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for the number of transcripts of gene g associated with cell i in a population of N cells. Let
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              be the total number of transcripts of gene g, and
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              the fraction of transcripts of gene g expressed by cell i, for each 1 ≤ i ≤ N. Thus,
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and we can think of
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as the probability that a randomly selected transcript of gene g is associated with cell i. To take under-sampling into account, we adjust the observed probabilities
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as follows (let us drop the superscript g for simplicity). We define X = X
                              <sup>
                               g
                              </sup>
                              as the discrete random variable on the set {1, 2, … N} with probabilities
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where λ′ is the shrinkage intensity given by
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which is the James-Stein estimator, with the uniform distribution as the shrinkage target (
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               Hausser & Strimmer 2009
                              </a>
                              ). Shrinkage approaches deal with under-sampling and so are able to correct for the substantial sparsity observed in single-cell RNA-sequencing data. The shrinkage probabilities x
                              <sub>
                               i
                              </sub>
                              given in
                              <strong>
                               <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                                Eqn. 3
                               </a>
                              </strong>
                              are a compromise between the observed probabilities p
                              <sub>
                               i
                              </sub>
                              , which are unbiased but with a high variance, and the target uniform probabilities 1/N, which are biased but with low variance (
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               Hausser & Strimmer 2009
                              </a>
                              ,
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               Chan et al. 2017
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Feature selection
                             </h3>
                             <p id="p-44">
                              The Shannon entropy of X is, by definition,
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-45">
                              By convention, we assume that 0 · log 0 = 0. For the logarithm, we take base e for ease in differentiation (see later) rather than the usual base 2 (recall that log
                              <sub>
                               e
                              </sub>
                              (x) = log
                              <sub>
                               2
                              </sub>
                              (e) log
                              <sub>
                               2
                              </sub>
                              (x) for all x > 0), and thus H(X), and I(g) below, are measured in nats.
                             </p>
                             <p id="p-46">
                              The Shannon entropy is a measure of the uncertainty in the outcomes of the random variable X. It has a minimum value of zero, when x
                              <sub>
                               i
                              </sub>
                              = 1 for some i (e.g. if the gene g is expressed in only one cell of the population) and has a maximum value of log(N), when x
                              <sub>
                               i
                              </sub>
                              = 1/N for all i (e.g. if the gene is uniformly expressed in the cell population). The entropy may therefore be considered as a measure of the homogeneity of expression of the gene g in the cell population profiled (note that the association of high entropy with homogeneity is counter to the usual intuition: it occurs because we are concerned with the entropy of the generative process of assigning transcripts to cells, rather than transcript count distribution in the cell population, as is more usual. A high entropy transcript assignment process gives rise to a sharp transcript distribution, hence its association with homogeneity). By contrast, the quantity log(N) − H(X) also ranges between zero and log(N), yet is minimised when the gene is homogeneously expressed and so is a simple measure of expression heterogeneity, which we will denote I(g). We can rewrite this as
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-47">
                              The Kullback-Leibler divergence (KLD), or relative entropy, is a measure of the information encoded in the difference between two probability distributions (
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               Kullback & Leibler 1951
                              </a>
                              ). The relative entropy of a discrete probability distribution p
                              <sub>
                               1
                              </sub>
                              , …, p
                              <sub>
                               N
                              </sub>
                              from a discrete probability distribution q
                              <sub>
                               1
                              </sub>
                              , …, q
                              <sub>
                               N
                              </sub>
                              is, by definition,
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with the provision that q
                              <sub>
                               i
                              </sub>
                              = 0 implies p
                              <sub>
                               i
                              </sub>
                              = 0, and the convention that
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . From this definition and
                              <strong>
                               <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                                Eqn. 6
                               </a>
                              </strong>
                              , it is clear that our measure of heterogeneity is simply the relative entropy of the observed expression distribution from the uniform distribution U. Thus,
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where U denotes to the uniform distribution on the set {1, 2, … N} with probabilities u
                              <sub>
                               i
                              </sub>
                              = 1/N for all 1 ≤ i ≤ N.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Differential expression analysis
                             </h3>
                             <p id="p-48">
                              A crucial property of the relative entropy is that it is additively decomposable with respect to arbitrary groupings (
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               Shorrocks 1980
                              </a>
                              ). Informally, this means that if we have a clustering of the cells into disjoint groups, then I(g) can be reconstructed from inter-cluster and intra-cluster heterogeneities. Next, we formalise this additive decomposition and give a self-contained derivation (cf.
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               Theil (1967)
                              </a>
                              ).
                             </p>
                             <p id="p-49">
                              Consider a clustering S = {S
                              <sub>
                               1
                              </sub>
                              , …, S
                              <sub>
                               C
                              </sub>
                              } that unambiguously assigns each cell in the sample into one of C non-intersecting sub-populations S
                              <sub>
                               1
                              </sub>
                              , …, S
                              <sub>
                               C
                              </sub>
                              of sizes N
                              <sub>
                               1
                              </sub>
                              , …, N
                              <sub>
                               C
                              </sub>
                              . Note that
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the total number of cells. Let y
                              <sub>
                               k
                              </sub>
                              be the fraction of transcripts associated with cells in sub-population S
                              <sub>
                               k
                              </sub>
                              , adjusted by the shrinkage estimator. That is,
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with x
                              <sub>
                               i
                              </sub>
                              defined by
                              <strong>
                               <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">
                                Eqn. 3
                               </a>
                              </strong>
                              . This gives another discrete random variable Y with probability distribution y
                              <sub>
                               1
                              </sub>
                              , …, y
                              <sub>
                               C
                              </sub>
                              on the set {1, 2, …C}. For each k = 1, …, C, we can also assess the heterogeneity of the sub-population S
                              <sub>
                               k
                              </sub>
                              by considering the random variable Z
                              <sub>
                               k
                              </sub>
                              with probability distribution z
                              <sub>
                               i
                              </sub>
                              = x
                              <sub>
                               i
                              </sub>
                              /y
                              <sub>
                               k
                              </sub>
                              on the set i ∈ S
                              <sub>
                               k
                              </sub>
                              . Note that
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              so Z
                              <sub>
                               k
                              </sub>
                              is a random variable on the set (cluster) S
                              <sub>
                               k
                              </sub>
                              .
                             </p>
                             <p id="p-50">
                              We may rewrite I(g) in terms of Y and Z
                              <sub>
                               k
                              </sub>
                              , as follows:
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-51">
                              Expression A may be rewritten as:
                              <span class="disp-formula" id="disp-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-52">
                              This is the relative entropy of Y from the uniform distribution U
                              <sub>
                               group
                              </sub>
                              in which p
                              <sub>
                               k
                              </sub>
                              = N
                              <sub>
                               k
                              </sub>
                              /N for k = 1, …, C. Since y
                              <sub>
                               k
                              </sub>
                              is the proportion of transcripts assigned to the cluster S
                              <sub>
                               k
                              </sub>
                              , it measures the deviation from the assumption that the clusters are homogeneous in their expression of g (i.e. each cluster expresses g at the same level). Since it is a measure of the extent to which the population deviates from homogeneity between clusters, we will term this contribution the inter-cluster heterogeneity of g with respect to the clustering S, denoted H
                              <sub>
                               S
                              </sub>
                              . Informally, it is a measure of the extent to which the gene g is differentially expressed between clusters.
                             </p>
                             <p id="p-53">
                              Expression B may be rewritten as:
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-54">
                              This is the weighted sum of the relative entropies of the empirical distributions Z
                              <sub>
                               k
                              </sub>
                              (i.e. the observed gene expression distribution in group S
                              <sub>
                               k
                              </sub>
                              ) from the uniform distribution U
                              <sub>
                               k
                              </sub>
                              on S
                              <sub>
                               k
                              </sub>
                              (in which p
                              <sub>
                               i
                              </sub>
                              = 1/N
                              <sub>
                               k
                              </sub>
                              for each i ∈ S
                              <sub>
                               k
                              </sub>
                              ). It is the deviation from the assumption that the population consists of a mixture of homogeneous sub-populations according to the clustering S (where the expectation is taken with respect to the probability measure provided by Y). Since it is a measure of the expected extent to which the proposed sub-populations deviate from homogeneity within clusters, we will term this contribution the intra-cluster heterogeneity of g with respect to S, denoted h
                              <sub>
                               S
                              </sub>
                              (g).
                             </p>
                             <p id="p-55">
                              Taken together, these results show that I(g) can be decomposed into two well-defined parts that encode local and global properties of the expression distribution of g with respect to a given clustering S = {S
                              <sub>
                               1
                              </sub>
                              , …, S
                              <sub>
                               C
                              </sub>
                              }:
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-56">
                              The relative entropy is always non-negative and hence so are H
                              <sub>
                               S
                              </sub>
                              (g) and h
                              <sub>
                               S
                              </sub>
                              (g) for any S. Thus, both quantities range from zero to I(g). If S places one cell in each cluster (i.e. C = N) then Z
                              <sub>
                               k
                              </sub>
                              = U
                              <sub>
                               k
                              </sub>
                              for all k and thus h
                              <sub>
                               S
                              </sub>
                              (g) = 0. Conversely, if S places all cells in one cluster (i.e. C = 1) then Y = U
                              <sub>
                               S
                              </sub>
                              and thus H
                              <sub>
                               S
                              </sub>
                              (g) = 0. In this case, the population heterogeneity is equivalent to the intra-cluster heterogeneity of the trivial clustering.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Unsupervised clustering
                             </h3>
                             <p id="p-57">
                              The above measures of heterogeneity can be easily extended from one gene to many. In this case, the homogeneous null model is obtained by assuming that each gene is expressed homogeneously and independently. Because entropy from independent sources is additive (
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               Cover & Thomas 2012
                              </a>
                              ), the total heterogeneity of a single-cell RNA-sequencing data set under a given clustering is given by the sum:
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-58">
                              Here H
                              <sub>
                               S
                              </sub>
                              = ∑
                              <sub>
                               g
                              </sub>
                              H
                              <sub>
                               S
                              </sub>
                              (g) and h
                              <sub>
                               S
                              </sub>
                              = ∑
                              <sub>
                               g
                              </sub>
                              h
                              <sub>
                               S
                              </sub>
                              (g) represent the total inter-cluster and intra-cluster heterogeneity of the data with respect to a clustering S = {S
                              <sub>
                               1
                              </sub>
                              , …, S
                              <sub>
                               C
                              </sub>
                              }.
                             </p>
                             <p id="p-59">
                              Naturally, we want to identify a clustering (e.g. an assignment of cells to types) that produces maximally homogeneous sub-groups and thereby explains most of the expression heterogeneity for most genes in terms of inter-cluster heterogeneity. Mathematically, this means we want to find a clustering S that maximises H
                              <sub>
                               S
                              </sub>
                              or, equivalently, minimises h
                              <sub>
                               S
                              </sub>
                              . A brute force approach is not feasible, as the number of partitions of a set with n elements into two or more subsets grows exponentially with n.
                             </p>
                             <p id="p-60">
                              To approach the problem we therefore converted the problem from one of assigning discrete identities (i.e., a hard clustering problem) to one of assigning continuous identities (i.e., a fuzzy clustering problem). The advantage of this approach is that it defines the problem in terms of a continuous, differentiable objective function which can be approached with more efficient optimisation routines.
                             </p>
                             <div id="sec-14" class="subsection">
                              <h4>
                               Fuzzy clustering
                              </h4>
                              <p id="p-61">
                               We adopt a fuzzy conception of clustering in which cells are assigned to C (possibly) overlapping, fuzzy clusters, S
                               <sub>
                                1
                               </sub>
                               , …, S
                               <sub>
                                C
                               </sub>
                               . Each cell i has C corresponding membership functions μ
                               <sub>
                                ik
                               </sub>
                               ∈ [0, 1] for k = 1, 2, …, C, which assess the probability that cell i belongs to cluster S
                               <sub>
                                k
                               </sub>
                               . Thus,
                               <span class="disp-formula" id="disp-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-34.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-34.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-62">
                               Our information-theoretic framework can be adapted to this fuzzy setting. Population heterogeneity, I(g), is independent of the chosen clustering, so it remains unaffected by the adoption of fuzzy clusters, i.e. the total gene expression heterogeneity is unaffected by the choice of discrete or fuzzy cluster memberships. For the calculations of inter-cluster and intra-cluster heterogeneities, we extend the discrete random variables Y
                               <sup>
                                g
                               </sup>
                               and
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to the fuzzy setting, where Y
                               <sup>
                                g
                               </sup>
                               now measures the expression distribution of the gene g across the C fuzzy clusters, and
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               measures the expression distribution of the gene g within fuzzy cluster S
                               <sub>
                                k
                               </sub>
                               , as follows.
                              </p>
                              <p id="p-63">
                               We define the discrete random variable Y
                               <sup>
                                g
                               </sup>
                               on the set of fuzzy clusters S = {S
                               <sub>
                                1
                               </sub>
                               , …, S
                               <sub>
                                C
                               </sub>
                               } with probabilities
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               given by
                               <span class="disp-formula" id="disp-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-35.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-35.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-64">
                               We also define, for each S
                               <sub>
                                k
                               </sub>
                               ∈ S, a discrete random variable
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               on the set of cells, i = 1, …, N, with probabilities
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               given by,
                               <span class="disp-formula" id="disp-formula-31">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-36.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-36.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-65">
                               We can rewrite I(g) (which is independent of the clustering) in terms of Y
                               <sup>
                                g
                               </sup>
                               and
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (which depend on the clustering) as
                               <span class="disp-formula" id="disp-formula-32">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-37.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-37.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where,
                               <span class="disp-formula" id="disp-formula-33">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-38.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-38.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="disp-formula" id="disp-formula-34">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-39.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-39.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-66">
                               Note that
                               <strong>
                                <a id="xref-disp-formula-33-1" class="xref-disp-formula" href="#disp-formula-33">
                                 Eqn. 33
                                </a>
                               </strong>
                               is the entropy of the random variable Y
                               <sup>
                                g
                               </sup>
                               , however
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is not quite the entropy of
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (there is a missing 1/μ
                               <sub>
                                ik
                               </sub>
                               factor inside the logarithm).
                              </p>
                              <p id="p-67">
                               From
                               <strong>
                                <a id="xref-disp-formula-32-1" class="xref-disp-formula" href="#disp-formula-32">
                                 Eqn. 32
                                </a>
                               </strong>
                               , we can obtain a second decomposition,
                               <span class="disp-formula" id="disp-formula-35">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-40.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-40.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="disp-formula" id="disp-formula-36">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-41.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-41.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-37">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-42.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-42.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-38">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-43.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-43.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-68">
                               Here H(g) and h(g) represent the inter-cluster heterogeneity and intra-cluster heterogeneity of gene g with respect to the fuzzy clustering S.
                               <strong>
                                <a id="xref-disp-formula-36-1" class="xref-disp-formula" href="#disp-formula-36">
                                 Eqn. 36
                                </a>
                               </strong>
                               is also the KLD of the distribution of Y
                               <sup>
                                g
                               </sup>
                               from the uniform distribution U
                               <sub>
                                group
                               </sub>
                               given by u
                               <sub>
                                k
                               </sub>
                               = N
                               <sub>
                                k
                               </sub>
                               /N for k = 1, 2, …, C (note that
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ). Similarly,
                               <strong>
                                <a id="xref-disp-formula-37-1" class="xref-disp-formula" href="#disp-formula-37">
                                 Eqn. 37
                                </a>
                               </strong>
                               is the KLD of the distribution of Z
                               <sup>
                                g
                               </sup>
                               from the uniform distribution U
                               <sub>
                                k
                               </sub>
                               given by p
                               <sub>
                                i
                               </sub>
                               = μ
                               <sub>
                                ik
                               </sub>
                               /N
                               <sub>
                                k
                               </sub>
                               for i = 1, 2, …, N.
                              </p>
                              <p id="p-69">
                               Either H
                               <sub>
                                S
                               </sub>
                               = ∑
                               <sub>
                                g
                               </sub>
                               H
                               <sub>
                                S
                               </sub>
                               (g) or h
                               <sub>
                                S
                               </sub>
                               = ∑
                               <sub>
                                g
                               </sub>
                               h
                               <sub>
                                S
                               </sub>
                               (g) can serve as our objective function, f, for optimisation: optimising either quantity simultaneously identifies the clustering with the greatest heterogeneity between clusters (i.e. the greatest differential gene expression between clusters) and the least heterogeneity within clusters (i.e. in which the cells within each cluster are closest to being interchangeable with respect to their patterns of gene expression).
                              </p>
                              <p id="p-70">
                               Choosing inter-cluster heterogeneity H
                               <sub>
                                S
                               </sub>
                               , we solve the optimisation problem
                               <span class="disp-formula" id="disp-formula-39">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-44.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-44.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="disp-formula" id="disp-formula-40">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-45.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-45.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               using the limited-memory, box constrained BFGS (L-BFGS-B) optimisation algorithm from the
                               <kbd>
                                Python3
                               </kbd>
                               (v3.8.2) package
                               <kbd>
                                SciPy
                               </kbd>
                               (v1.5.3) (
                               <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                                Byrd et al. 1995
                               </a>
                               ,
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                Zhu et al. 1997
                               </a>
                               ,
                               <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                Van Rossum & Drake 2009
                               </a>
                               ,
                               <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                Virtanen et al. 2020
                               </a>
                               ).
                              </p>
                              <p id="p-71">
                               The L-BFGS-B algorithm is an efficient non-linear local optimisation method developed for solving large, dense problems such as this (
                               <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                                Zhu et al. 1997
                               </a>
                               )) but its speed and the quality of the solution produced depends on the availability of an explicit formulation of the gradient of the objective function. Therefore, we derive the required gradient by partially differentiating the total inter-cluster heterogeneity, H
                               <sub>
                                S
                               </sub>
                               , with respect to fuzzy cluster membership, i.e. the N · C membership functions μ
                               <sub>
                                rs
                               </sub>
                               (1 ≤ r ≤ N, 1 ≤ s ≤ C).
                              </p>
                              <p id="p-72">
                               Beginning by differentiating individual elements of f with respect to μ
                               <sub>
                                rs
                               </sub>
                               , from
                               <strong>
                                <a id="xref-disp-formula-30-1" class="xref-disp-formula" href="#disp-formula-30">
                                 Eqn. 30
                                </a>
                               </strong>
                               and
                               <strong>
                                <a id="xref-disp-formula-38-1" class="xref-disp-formula" href="#disp-formula-38">
                                 38
                                </a>
                               </strong>
                               , we have
                               <span class="disp-formula" id="disp-formula-41">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-46.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-46.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-42">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-47.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-47.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-73">
                               Using the product, chain and quotient rules, we can differentiate the g-summand, call it f
                               <sup>
                                g
                               </sup>
                               (= H
                               <sub>
                                S
                               </sub>
                               (g)), of the objective function f :
                               <span class="disp-formula" id="disp-formula-43">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-48.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-48.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-44">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-49.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-49.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-45">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-50.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-50.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-46">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-51.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-51.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-47">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-52.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-52.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-48">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-53.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-53.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-49">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-54.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-54.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-74">
                               The membership functions μ
                               <sub>
                                rs
                               </sub>
                               are not independent, since
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for all i, by
                               <strong>
                                <a id="xref-disp-formula-29-1" class="xref-disp-formula" href="#disp-formula-29">
                                 Eqn. 29
                                </a>
                               </strong>
                               . To incorporate this constraint, and the constraint μ
                               <sub>
                                ik
                               </sub>
                               ≥ 0 for all i, k, we introduce variables w
                               <sub>
                                ik
                               </sub>
                               given by
                               <span class="disp-formula" id="disp-formula-50">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-55.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-55.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and make the objective function f a function of the w
                               <sub>
                                ik
                               </sub>
                               . By the chain rule,
                               <span class="disp-formula" id="disp-formula-51">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-56.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-56.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-75">
                               Determining
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , let us write
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ; then,
                               <strong>
                                <a id="xref-disp-formula-50-1" class="xref-disp-formula" href="#disp-formula-50">
                                 Eqn. 50
                                </a>
                               </strong>
                               can be written as
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . By the quotient rule,
                               <span class="disp-formula" id="disp-formula-52">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-57.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-57.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                             <div id="sec-15" class="subsection">
                              <h4>
                               Numerical optimisation
                              </h4>
                              <p id="p-76">
                               The objective function H
                               <sub>
                                S
                               </sub>
                               , as formulated in
                               <strong>
                                <a id="xref-disp-formula-40-1" class="xref-disp-formula" href="#disp-formula-40">
                                 Eqn. 40
                                </a>
                               </strong>
                               , was optimised with respect to cluster memberships, μ
                               <sub>
                                ik
                               </sub>
                               using the L-BFGS-B optimisation algorithm from the
                               <kbd>
                                Python3
                               </kbd>
                               (v3.8.2) package
                               <kbd>
                                SciPy
                               </kbd>
                               (v1.5.3) (
                               <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                                Byrd et al. 1995
                               </a>
                               ,
                               <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                                Van Rossum & Drake 2009
                               </a>
                               ,
                               <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                                Virtanen et al. 2020
                               </a>
                               ), with random initial starting points (initial cluster memberships were defined by random sampling of a uniform distribution centred on zero). The gradient function, as formulated in
                               <strong>
                                <a id="xref-disp-formula-51-1" class="xref-disp-formula" href="#disp-formula-51">
                                 Eqn. 51
                                </a>
                               </strong>
                               , was supplied to the optimisation routine.
                              </p>
                              <p id="p-77">
                               Because initial conditions are randomly generated, and the L-BFGS-B algorithm only identifies local optima, the specific clustering found by the algorithm may depend on the choice of initial vector (
                               <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
                                Byrd et al. 1995
                               </a>
                               ,
                               <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">
                                Zhu et al. 1997
                               </a>
                               ). To make the implementation robust a multi-start process is adopted, in which the optimisation is repeated multiple times with different initial vectors, and the clustering S with the greatest inter-cluster heterogeneity H
                               <sub>
                                S
                               </sub>
                               is chosen.
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Permutation testing
                              </h4>
                              <p id="p-78">
                               Annotations for the Tian, Zheng and Stumpf data were taken from the respective repositories’ metadata. Random clusterings were generated by randomly permuting the established annotations 10,000 times each. Comparison of true and shuffled annotations was formulated as an exact one-side hypothesis test, generating exact p-values (
                               <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">
                                Fisher 2017
                               </a>
                               ). We control the false discovery rate arising from multiple testing for each data set using the
                               <kbd>
                                R
                               </kbd>
                               function p.adjust (
                               <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">
                                Benjamini & Hochberg 1995
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-17" class="subsection">
                              <h4>
                               Clustering comparison
                              </h4>
                              <p id="p-79">
                               We compare our clustering results to two standard clustering methods:
                               <kbd>
                                Seurat
                               </kbd>
                               (v3) clustering (Louvain community detection), and UMAP projection, both with default parameters, with the exception of the resolution, as described in
                               <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                                Stuart et al. (2019)
                               </a>
                               (
                               <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                                Blondel et al. 2008
                               </a>
                               ,
                               <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                                McInnes et al. 2018
                               </a>
                               ). The Adjusted Rand Index (ARI) was calculated for each data set using the associated function in the
                               <kbd>
                                R
                               </kbd>
                               package
                               <kbd>
                                mclust
                               </kbd>
                               (v5.4.7) (
                               <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                                Rand 1971
                               </a>
                               ,
                               <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                                Scrucca et al. 2016
                               </a>
                               ).
                              </p>
                             </div>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Semi-supervised classification
                             </h3>
                             <p id="p-80">
                              Consider a pair of count matrices, T = {T
                              <sub>
                               ref
                              </sub>
                              , T
                              <sub>
                               test
                              </sub>
                              }, where one count matrix, which we term the reference, T
                              <sub>
                               ref
                              </sub>
                              , has a known cluster structure (and therefore each cell has a unique, discrete cell type classification). The goal of reference mapping is to classify cells in the test matrix, T
                              <sub>
                               test
                              </sub>
                              , based on cellular classifications of the reference matrix.
                             </p>
                             <p id="p-81">
                              We normalise each count matrix separately as described in
                              <strong>
                               Normalisation
                              </strong>
                              , using the James-Stein-type estimator so that
                              <span class="inline-formula" id="inline-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                               Hausser & Strimmer 2009
                              </a>
                              ). We assume that the same set of genes are profiled in T
                              <sub>
                               ref
                              </sub>
                              and T
                              <sub>
                               test
                              </sub>
                              and derive the normalised combined count matrix, X
                              <sub>
                               mix
                              </sub>
                              , via the weighted concatenation:
                              <span class="disp-formula" id="disp-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where N
                              <sub>
                               ref
                              </sub>
                              is the number of cells in the reference data set, N
                              <sub>
                               test
                              </sub>
                              is the number of cells in the unclassified data set, and N is the total number of cells across both data sets.
                             </p>
                             <p id="p-82">
                              As in the unsupervised setting, we cluster the combined data set, X
                              <sub>
                               mix
                              </sub>
                              , by maximising H
                              <sub>
                               S
                              </sub>
                              of the combined data set with respect to a fuzzy clustering S, see
                              <strong>
                               <a id="xref-disp-formula-56-1" class="xref-disp-formula" href="#disp-formula-56">
                                Eqn. 56
                               </a>
                              </strong>
                              . However, unlike in the unsupervised setting, a subset of cellular cluster memberships are known a priori. Let R = R
                              <sub>
                               1
                              </sub>
                              , …, R
                              <sub>
                               C
                              </sub>
                              be the discrete clustering of the reference data set, with cluster sizes n
                              <sub>
                               1
                              </sub>
                              , …, n
                              <sub>
                               C
                              </sub>
                              , where
                              <span class="inline-formula" id="inline-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . As the cluster identities of cells originating from the reference data set are fixed, each cluster in the reference data set constitutes a subset of a cluster in the mixed data set, R
                              <sub>
                               k
                              </sub>
                              ∈ S
                              <sub>
                               k
                              </sub>
                              , where S = S
                              <sub>
                               1
                              </sub>
                              , …, S
                              <sub>
                               C
                              </sub>
                              is the clustering of the mixed data set. Note that number of clusters in the reference and in the combined data set is the same, i.e. C.
                             </p>
                             <p id="p-83">
                              Based on the co-mixing of known and unknown cellular identities,
                              <span class="inline-formula" id="inline-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and N
                              <sub>
                               k
                              </sub>
                              of the combined data set X
                              <sub>
                               mix
                              </sub>
                              are given by
                              <span class="disp-formula" id="disp-formula-54">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-59.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-59.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-55">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-60.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-60.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-84">
                              Based on these formulations of
                              <span class="inline-formula" id="inline-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/inline-graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and N
                              <sub>
                               k
                              </sub>
                              , the definition of H
                              <sub>
                               S
                              </sub>
                              follows as before,
                              <span class="disp-formula" id="disp-formula-56">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-61.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-61.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-85">
                              Similarly, the derivative of H
                              <sub>
                               S
                              </sub>
                              follows as before, except that the membership function, μ
                              <sub>
                               rs
                              </sub>
                              , is only defined with respect to cells of the test data set, 1 ≤ r ≤ N
                              <sub>
                               test
                              </sub>
                              & 1 ≤ i ≤ N
                              <sub>
                               test
                              </sub>
                              ,
                              <span class="disp-formula" id="disp-formula-57">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-62.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/08/2020.10.01.322255/embed/graphic-62.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-86">
                              Numerical optimisation follows as in the unsupervised setting, using the L-BFGS-B algorithm, except that the known cellular identities of the reference mean that only a single run of the algorithm is required, i.e. no multiple starts.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Declarations
                            </h2>
                            <div class="section" id="sec-20">
                             <h3>
                              Authors’ contributions
                             </h3>
                             <p id="p-87">
                              Conceptualization, MJC, RSG, BDM; Software, MJC; Investigation, MJC, JF, RJSG and BDM; Writing – Original Draft, MJC, RSG and BDM; Writing – Review & Editing, MJC, JF, RSG and BDM; Visualization, MJC; Supervision, RSG and BDM. All authors have seen and approved the manuscript.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Funding
                            </h2>
                            <p id="p-89">
                             RSG and BDM were partially supported by The Alan Turing Institute under the EPSRC grant EP/N510129/1.
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Availability of data and materials
                            </h2>
                            <p id="p-90">
                             The Svensson data was downloaded as file “svensson chromium control.h5a” from
                             <a href="https://data.caltech.edu/records/1264">
                              https://data.caltech.edu/records/1264
                             </a>
                            </p>
                            <p id="p-91">
                             The Tian data was downloaded as file “sincell with class.RDat” from
                             <a href="https://github.com/LuyiTian/sc_mixology">
                              https://github.com/LuyiTian/sc_mixology
                             </a>
                            </p>
                            <p id="p-92">
                             The Zheng data was downloaded as set of files “Gene / cell matrix (filtered)” in section “Single Cell 3’ Paper: Zheng et al. 2017” from
                             <a href="https://support.10xgenomics.com/single-cell-gene-expression/datasets">
                              https://support.10xgenomics.com/single-cell-gene-expression/datasets
                             </a>
                            </p>
                            <p id="p-93">
                             The Stumpf data was downloaded as file “RData.zip” from
                             <a href="https://data.mendeley.com/datasets/csvm3kpkxd/1?fbclid=IwAR3j_hvq5Zt0cdxBBM72Fqr8zXwVC6XRpkY2JtVwbcj1gzyNLMVsdoFYWgQ">
                              https://data.mendeley.com/datasets/csvm3kpkxd/1?fbclid=IwAR3j_hvq5Zt0cdxBBM72Fqr8zXwVC6XRpkY2JtVwbcj1gzyNLMVsdoFYWgQ
                             </a>
                            </p>
                            <p id="p-94">
                             The Tabula Muris data was downloaded as set of files “Single-cell RNA-seq data from microfluidic emulsion (v2)” from
                             <a href="https://tabula-muris.ds.czbiohub.org">
                              https://tabula-muris.ds.czbiohub.org
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Code availability
                            </h2>
                            <p id="p-95">
                             An
                             <kbd>
                              R
                             </kbd>
                             package for the implementation of the described methods is available on github
                             <a href="https://github.com/mjcasy/scEC">
                              https://github.com/mjcasy/scEC
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Ethics approval and consent to participate
                            </h2>
                            <p id="p-96">
                             Not applicable.
                            </p>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             Competing interests
                            </h2>
                            <p id="p-97">
                             The authors declare that they have no competing interests.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-88">
                             We thank P. Stumpf, J. Egan and B. Kitching-Morley for helpful discussions.
                            </p>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.1" data-doi="10.1038/75556">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashburner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ball
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blake
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Botstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Butler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cherry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dolinski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dwight
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eppig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Gene ontology: tool for the unification of biology’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  25
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  29
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/75556&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10802651&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F08%2F2020.10.01.322255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000086884000011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.2" data-doi="10.2307/2346101">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Benjamini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hochberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Controlling the false discovery rate: a practical and powerful approach to multiple testing’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the Royal statistical society: series B (Methodological)
                                 </abbr>
                                 <span class="cit-vol">
                                  57
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  289
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  300
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2Bstatistical%2Bsociety%253A%2Bseries%2BB%2B%2528Methodological%2529%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2346101%26rft_id%253Dinfo%253Apmid%252FWOS%253AA1995QE4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/2346101&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=WOS:A1995QE4&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F08%2F2020.10.01.322255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1995QE45300017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.3" data-doi="10.1088/1742-5468/2008/10/P10008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blondel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guillaume
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambiotte
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lefebvre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Fast unfolding of communities in large networks’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of statistical mechanics: theory and experiment
                                 </abbr>
                                 <span class="cit-vol">
                                  2008
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  P10008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstatistical%2Bmechanics%253A%2Btheory%2Band%2Bexperiment%26rft.volume%253D2008%26rft.spage%253D10008P%26rft_id%253Dinfo%253Adoi%252F10.1088%252F1742-5468%252F2008%252F10%252FP10008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1088/1742-5468/2008/10/P10008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Byrd
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nocedal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘A limited memory algorithm for bound constrained optimization’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  SIAM Journal on scientific computing
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1190
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1208
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSIAM%2BJournal%2Bon%2Bscientific%2Bcomputing%26rft.volume%253D16%26rft.spage%253D1190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stumpf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Babtie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Gene regulatory network inference from single-cell data using multivariate information measures’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Cell systems
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  251
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  267
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2Bsystems%26rft.volume%253D5%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.01.322255v4.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cover
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thomas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ),
                                 <span class="cit-source">
                                  Elements of information theory
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  John Wiley & Sons
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.01.322255v4.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fisher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ),
                                 <span class="cit-source">
                                  Statistical Methods For Research Workers
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  Gyan Books
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Freytag
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lönnstedt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bahlo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Comparison of clustering tools in r for medium-sized 10x genomics single-cell rna-sequencing data’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  F1000Research
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.9" data-doi="10.1093/nar/gkaa1113">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Gene Ontology Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘The gene ontology resource: enriching a gold mine’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  D1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  D325
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D334
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D49%26rft.spage%253DD325%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkaa1113%26rft_id%253Dinfo%253Apmid%252F33290552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkaa1113&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33290552&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F08%2F2020.10.01.322255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.01.322255v4.10">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    H.
                                   </span>
                                   <span class="cit-name-surname">
                                    Levine
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    M. K.
                                   </span>
                                   <span class="cit-name-surname">
                                    Jolly
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    P.
                                   </span>
                                   <span class="cit-name-surname">
                                    Kulkarni
                                   </span>
                                  </span>
                                  &
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    V.
                                   </span>
                                   <span class="cit-name-surname">
                                    Nanjundiah
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greulich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MacArthur
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ),
                                 <span class="cit-chapter-title">
                                  The physics of cell fate
                                 </span>
                                 , in
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Levine
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   M. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jolly
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kulkarni
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                  <span class="cit-name-surname">
                                   Nanjundiah
                                  </span>
                                 </span>
                                 , eds,
                                 <span class="cit-source">
                                  ‘Phenotypic Switching’
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  Academic Press
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  189
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  206
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hausser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Strimmer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Entropy inference and the james-stein estimator, with application to nonlinear gene association networks.’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Machine Learning Research
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.12" data-doi="10.1093/biostatistics/kxx053">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hicks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Townes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Irizarry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Missing data and technical variability in single-cell rna-sequencing experiments’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Biostatistics
                                 </abbr>
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  562
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  578
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.volume%253D19%26rft.spage%253D562%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxx053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxx053&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kharchenko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Silberstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Scadden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Bayesian approach to single-cell differential expression analysis’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  740
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  742
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D11%26rft.spage%253D740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kiselev
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V. Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrews
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hemberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Challenges in unsupervised clustering of single-cell rna-seq data’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  273
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  282
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D20%26rft.spage%253D273%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F08%2F2020.10.01.322255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.15" data-doi="10.1016/j.transproceed.2017.12.043">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kullback
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leibler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1951
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘On information and sufficiency’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  The annals of mathematical statistics
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  79
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  86
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2Bannals%2Bof%2Bmathematical%2Bstatistics%26rft.volume%253D22%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.transproceed.2017.12.043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.transproceed.2017.12.043&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.16" data-doi="10.15252/msb.20188746">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Luecken
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Theis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F. J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Current best practices in single-cell rna-seq analysis: a tutorial’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular systems biology
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e8746
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DLuecken%26rft.auinit1%253DM.%2BD.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253De8746%26rft.epage%253De8746%26rft.atitle%253DCurrent%2Bbest%2Bpractices%2Bin%2Bsingle-cell%2BRNA-seq%2Banalysis%253A%2Ba%2Btutorial%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20188746%26rft_id%253Dinfo%253Apmid%252F31217225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNS82L2U4NzQ2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDgvMjAyMC4xMC4wMS4zMjIyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McInnes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Healy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Melville
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Umap: Uniform manifold approximation and projection for dimension reduction’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  <span class="cit-pub-id-scheme-arxiv">
                                   arxiv:
                                  </span>
                                  1802.03426
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.01.322255v4.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  R Core Team
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ),
                                 <span class="cit-source">
                                  R: A Language and Environment for Statistical Computing
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  R Foundation for Statistical Computing
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  Vienna, Austria
                                 </span>
                                 . URL:
                                 <a href="https://www.R-project.org/">
                                  https://www.R-project.org/
                                 </a>
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.19" data-doi="10.2307/2284239">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1971
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Objective criteria for the evaluation of clustering methods’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the American Statistical association
                                 </abbr>
                                 <span class="cit-vol">
                                  66
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  336
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  846
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  850
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BStatistical%2Bassociation%26rft.volume%253D66%26rft.spage%253D846%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2284239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/2284239&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1971L498800028&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Scrucca
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fop
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Murphy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raftery
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘mclust 5: clustering, classification and density estimation using gaussian finite mixture models’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  The R journal
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  289
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BR%2Bjournal%26rft.volume%253D8%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shannon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1948
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘A mathematical theory of communication’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Bell system technical journal
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  379
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  423
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBell%2Bsystem%2Btechnical%2Bjournal%26rft.volume%253D27%26rft.spage%253D379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shorrocks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. F.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1980
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘The class of additively decomposable inequality measures’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Econometrica: Journal of the Econometric Society
                                 </abbr>
                                 pp.
                                 <span class="cit-fpage">
                                  613
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  625
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MacArthur
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Information-theoretic approaches to understanding stem cell variability’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Current Stem Cell Reports
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  225
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  231
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BStem%2BCell%2BReports%26rft.volume%253D3%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.24" data-doi="10.1016/j.cell.2019.05.031">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stuart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Butler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoffman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafemeister
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Papalexi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mauck
                                  </span>
                                  <span class="cit-name-suffix">
                                   III
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stoeckius
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smibert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Satija
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Comprehensive integration of single-cell data’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  177
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1888
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1902
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1888%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.05.031%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.05.031&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F08%2F2020.10.01.322255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stumpf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Du
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Imanishi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kunisaki
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Semba
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rose-Zerili
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   West
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oreffo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. O.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Transfer learning efficiently maps bone marrow cell types from mouse to human using single-cell rna sequencing’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Communications biology
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCommunications%2Bbiology%26rft.volume%253D3%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Svensson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Natarajan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ly
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.-H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miragaia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Labalette
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Macaulay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cvejic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teichmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Power analysis of single-cell rna-sequencing experiments’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  381
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D14%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Svensson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vento-Tormo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teichmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Exponential scaling of single-cell rna-seq in the past decade’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature protocols
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  599
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  604
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bprotocols%26rft.volume%253D13%26rft.spage%253D599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.28" data-doi="10.1038/s41586-018-0590-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Tabula Muris Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Single-cell transcriptomics of 20 mouse organs creates a tabula muris.’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  562
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7727
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  367
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D562%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0590-4%26rft_id%253Dinfo%253Apmid%252F30283141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0590-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30283141&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F08%2F2020.10.01.322255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.01.322255v4.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Theil
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1967
                                 </span>
                                 ),
                                 <span class="cit-chapter-title">
                                  Economics and Information Theory
                                 </span>
                                 ,
                                 <span class="cit-source">
                                  Studies in mathematical and managerial economics
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  North-Holland Publishing Company
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Freytag
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   LêCao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.-A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Su
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   JalalAbadi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Amann-Zalcenstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seidi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jabbari
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Benchmarking single cell rna-sequencing analysis pipelines using mixture control experiments’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  479
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  487
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D16%26rft.spage%253D479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.31" data-doi="10.1101/gr.190595.115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trapnell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Defining cell types and states with single-cell genomics’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1491
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1498
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.190595.115%26rft_id%253Dinfo%253Apmid%252F26430159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMC8xNDkxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDgvMjAyMC4xMC4wMS4zMjIyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.01.322255v4.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Rossum
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Drake
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F. L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ),
                                 <span class="cit-source">
                                  Python 3 Reference Manual
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  CreateSpace
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  Scotts Valley, CA
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Virtanen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gommers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oliphant
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haberland
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cournapeau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burovski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weckesser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bright
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Scipy 1.0: fundamental algorithms for scientific computing in python’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  261
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  272
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D17%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zheng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Terry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Belgrader
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ryvkin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bent
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ziraldo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wheeler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McDermott
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Massively parallel digital transcriptional profiling of single cells’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.01.322255v4.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Byrd
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nocedal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ),
                                 <span class="cit-article-title">
                                  ‘Algorithm 778: L-BFGS-B: Fortran subroutines for large-scale bound-constrained optimization’
                                 </span>
                                 ,
                                 <abbr class="cit-jnl-abbrev">
                                  ACM Transactions on Mathematical Software (TOMS)
                                 </abbr>
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  550
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  560
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DACM%2BTransactions%2Bon%2BMathematical%2BSoftware%2B%2528TOMS%2529%26rft.volume%253D23%26rft.spage%253D550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.10.01.322255v4" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.03.07.483225v1" title="Status of susceptibility of the visceral leishmaniasis vector, Phlebotomus argentipes (Diptera: Psychodidae: Phlebotominae), to insecticides used for vector control in Nepal" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2020.11.17.387340v3" title="Reproductive compensation and selection among viable embryos drive the evolution of polyembryony" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 08, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.10.01.322255v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.10.01.322255v4.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.10.01.322255v4" class="active" data-icon-position="" data-hide-link-title="0">
                                   Information-theoretic clustering for single-cell transcriptomics
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2436909"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-0gvPJt9RSfBkqkE8xwW0moLaq3jGzpr-MyKjaOms6PM"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484480884"/>
                                   <input type="hidden" name="captcha_token" value="4f553689106daa3de44a827050abf4df"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2436909--3" data-node-nid="2436909" id="node-2436909--4102677118" data-pisa="biorxiv;2020.10.01.322255v4" data-pisa-master="biorxiv;2020.10.01.322255" data-seqnum="2436909" data-apath="/biorxiv/early/2022/03/08/2020.10.01.322255.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Information-theoretic clustering for single-cell transcriptomics
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Michael J.
                                  </span>
                                  <span class="nlm-surname">
                                   Casey
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Jörg
                                  </span>
                                  <span class="nlm-surname">
                                   Fliege
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Rubén J.
                                  </span>
                                  <span class="nlm-surname">
                                   Sánchez-García
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Ben D.
                                  </span>
                                  <span class="nlm-surname">
                                   MacArthur
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.10.01.322255;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.10.01.322255
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.10.01.322255v4" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&title=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&title=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&text=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&t=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&title=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&title=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&title=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2436909--5" data-node-nid="2436909" id="citation-node-2436909--61596752234" data-pisa="biorxiv;2020.10.01.322255v4" data-pisa-master="biorxiv;2020.10.01.322255" data-seqnum="2436909" data-apath="/biorxiv/early/2022/03/08/2020.10.01.322255.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Information-theoretic clustering for single-cell transcriptomics
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Michael J.
                                      </span>
                                      <span class="nlm-surname">
                                       Casey
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Jörg
                                      </span>
                                      <span class="nlm-surname">
                                       Fliege
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Rubén J.
                                      </span>
                                      <span class="nlm-surname">
                                       Sánchez-García
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Ben D.
                                      </span>
                                      <span class="nlm-surname">
                                       MacArthur
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.10.01.322255;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.10.01.322255
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Casey%2C+Michael+J.&rft.au=Fliege%2C+J%C3%B6rg&rft.au=S%C3%A1nchez-Garc%C3%ADa%2C+Rub%C3%A9n+J.&rft.au=MacArthur%2C+Ben+D.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2436909/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2436909/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2436909/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2436909/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2436909/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2436909/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2436909/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2436909/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2436909/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2436909/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2436909/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2436909/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&count=horizontal&via=&text=Information-theoretic%20clustering%20for%20single-cell%20transcriptomics&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.10.01.322255v4&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.10.01.322255v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genomics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                    <li class="last even">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
